--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	f9810e06-75c5-465b-a4fc-c7b55fbce316	2015-02-20 18:26:29.378072
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	8e12a528-720a-4582-a545-f213052a1f24	2015-02-20 18:26:29.611623
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	87a3ec46-9d7f-4412-b277-09a0a21dfd99	2015-02-20 18:26:29.818263
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	65803cdb-a026-4556-a401-44ad262b7f86	2015-02-20 18:26:30.029359
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	5f872b73-c2ba-405e-8331-628c55cb53d3	2015-02-20 18:26:30.247191
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	bdb74445-491b-4cf4-a617-1a1a41637ecf	2015-02-20 18:26:30.451823
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	249996c2-e404-4aaa-9530-65dbb7c3b1fa	2015-02-20 18:26:30.654847
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	70e55259-4a81-428f-9480-83669ca11199	2015-02-20 18:26:30.854187
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	89e35681-8799-4af6-bfc0-adffd87168c5	2015-02-20 18:26:31.062215
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	28b4070f-516f-4836-8eab-622f80bd4d37	2015-02-20 18:26:31.266642
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	6e97adcd-8f65-4908-b11a-43cad6b49876	2015-02-20 18:26:31.468858
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	8966bb4f-e4b3-4b97-9af7-aec5d37b7bb0	2015-02-20 18:26:31.684023
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	9f92cd84-e844-4644-bf5b-dbbdfcbc2cd1	2015-02-20 18:26:31.933166
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	a3d107ac-3100-4540-8f22-79e913b07035	2015-02-20 18:26:32.129898
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: '"PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity."'\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	d42c6ce2-eddb-4f1b-a0bf-e3478397037b	2015-02-20 18:26:32.575282
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	d77a9453-e060-4919-ace6-0dc5fc0711bb	2015-02-20 18:26:32.785925
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	1a96de4a-7586-47a6-a5a7-e2c0dba515f2	2015-02-20 18:26:33.004828
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown senitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	af8f9bca-9273-4206-8d52-879b6586ede9	2015-02-20 18:26:33.216699
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	ba6f5ba0-4dc0-4203-b9b8-767fece64930	2015-02-20 18:26:33.422775
158	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	92548108-c7d3-48b0-9972-d2affd393afc	2015-02-20 18:26:42.252262
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions have been demonstrated\n  to have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	2be6c5b0-176d-4ff9-ae3a-49cbe6d5a8e0	2015-02-20 18:26:33.633756
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to the activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFRs can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions have been demonstrated\n  to have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	b4a32a48-b0ae-4aec-b4d7-f76cbf04cc18	2015-02-20 18:26:33.834537
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	0c92eae3-3d02-4a3b-bb19-76cbd7d1d254	2015-02-20 18:26:34.036003
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: N/A\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	f3281420-243b-4c38-860d-e93c2ea02d0b	2015-02-20 18:26:34.226552
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, and appear\n  to behave quite differently in different environments. In myelodysplastic syndromes\n  and acute myeloid leukemia (AML), IDH1 mutations have been shown to be associated\n  with worse outcome and shorter overall survival. They have also been linked to cytogenetically\n  normal AML, an intermediate risk subtype of the disease. However, in glioblastoma\n  and astrocytoma, patients with IDH1 mutations have shown better overall survival\n  than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1\n  mutations have been associated with specific cytogenetic abnormalities, 1p and 19q\n  deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	d9ce9334-698b-4d80-86f6-6beda24d9610	2015-02-20 18:26:34.413899
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: N/A\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	004b3a13-3d83-4786-af57-59db2b50c46c	2015-02-20 18:26:34.602192
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	80b9303a-0efd-4f6c-9b80-a5a61aca4dbd	2015-02-20 18:26:34.805546
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: N/A\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	91d3157c-e24e-4bac-abac-847ec944f4ce	2015-02-20 18:26:35.003619
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: N/A\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	8ac35238-e649-48da-8847-9063b6041d4c	2015-02-20 18:26:35.193117
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: N/A\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	2ca15bbd-b4e7-4544-ad75-ab5361ce11af	2015-02-20 18:26:35.383762
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: N/A\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	7c0e2491-7822-460d-8d5a-854f33e6fb5e	2015-02-20 18:26:35.570436
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: N/A\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	d1e944b0-5e1f-4061-a5ca-93e12f51ced3	2015-02-20 18:26:35.766172
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: N/A\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	3e78c853-ca45-494e-971d-2a407e037998	2015-02-20 18:26:35.971439
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1' is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	d5986a12-acfa-4174-a5f5-a0c3e90ab50d	2015-02-20 18:26:36.177068
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: N/A\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	01dd5bc1-f6c3-4884-bb4d-9306d377443e	2015-02-20 18:26:36.371377
50	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	b2e9b0be-9386-4a07-bf06-d12d79390fbf	2015-02-20 18:26:39.193215
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	8c0876aa-7bb8-4c89-8b8c-6a41f6b98b57	2015-02-20 18:26:36.561264
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: N/A\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	2b63ea52-775a-4e5e-a26f-4eac15f54317	2015-02-20 18:26:36.752156
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: N/A\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	622f13fd-4dd0-46d9-af2b-7b4a76b095fa	2015-02-20 18:26:36.943432
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARa\ndescription: N/A\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	65c49405-6091-4383-a5e1-6b560a5696b1	2015-02-20 18:26:37.129524
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: N/A\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	10a8020a-af99-433f-9a35-7a9f3abc3fae	2015-02-20 18:26:37.318172
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	3e214192-3713-4653-bc8e-167665db3847	2015-02-20 18:26:37.55954
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: N/A\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	af916f5c-bf5d-4465-9fc7-1d40c3173828	2015-02-20 18:26:37.745687
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	87f5b01a-0b23-4df9-8d53-09ca56a845b3	2015-02-20 18:26:38.027651
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: N/A\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	69463902-e456-4185-bc22-a62c62e067de	2015-02-20 18:26:38.236074
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: N/A\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	5a0e0782-097f-4103-aeb7-f091ac6ec999	2015-02-20 18:26:38.434975
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: N/A\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	e2a73d2e-ca2f-4e45-a815-d44084be6496	2015-02-20 18:26:38.623015
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	75b032e3-9894-4996-bcaf-43b7d82bfc43	2015-02-20 18:26:38.847071
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: N/A\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	fb3a5451-e627-4942-a2bb-fc1493368a1e	2015-02-20 18:26:39.047542
48	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	90761486-fc03-41e2-9c63-1f623a2f40a0	2015-02-20 18:26:39.10805
49	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	a27e9b9b-cbbd-4cbe-ad9d-17ff9e4478ca	2015-02-20 18:26:39.140162
51	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	fa144cbc-34e2-4e75-8391-44b34f10b48c	2015-02-20 18:26:39.219127
52	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	2622c9bc-7246-4889-bdb0-2533fe814d99	2015-02-20 18:26:39.250544
53	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	13c3e2cd-8f6a-4e7a-aa4a-3a19e65d6590	2015-02-20 18:26:39.276361
54	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	eb6acc94-b9d9-438e-b304-7c9b8be1f98f	2015-02-20 18:26:39.306751
55	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	17d942cf-7b2c-40f0-ab38-87026514a4f9	2015-02-20 18:26:39.334496
56	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	fe86ada1-6bd9-4857-8769-bdccd4ca5d0a	2015-02-20 18:26:39.357962
57	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	99d0a942-8450-4e3a-9fdf-c6da976862c3	2015-02-20 18:26:39.381327
58	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	233e678b-461e-4a9a-ad6f-59ef8b2a7849	2015-02-20 18:26:39.408283
59	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	60dbffd9-b0a7-4752-8840-cee46ef505c0	2015-02-20 18:26:39.43558
60	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	cbfa5dea-9eb1-484e-b8a0-2d16ec8d4ab4	2015-02-20 18:26:39.464925
61	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	ef5d986b-24d1-4e72-8c7c-78e031d0a32d	2015-02-20 18:26:39.494513
62	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	d94c9069-2228-4412-a4bb-b5b4bb220609	2015-02-20 18:26:39.522102
63	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	8315563a-4b0a-4ff1-9e91-d06b0249f61e	2015-02-20 18:26:39.551027
64	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	ceef028b-cff4-41a4-885c-4258640c446a	2015-02-20 18:26:39.578843
65	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	054687a8-2090-4602-9d9e-bc91d4e833a2	2015-02-20 18:26:39.606722
66	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	580a9a26-db42-4c58-965b-d7f97eccf637	2015-02-20 18:26:39.635114
67	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	cbfe955c-cba1-49dc-8b80-145f56d284ea	2015-02-20 18:26:39.659695
68	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	86af9a17-3422-43fb-9250-6aacff3abd69	2015-02-20 18:26:39.693572
69	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	7de34c38-1c33-43eb-9b8f-4fcee575b2eb	2015-02-20 18:26:39.720782
70	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	792cb8c9-581e-4118-bf1a-8bce129e337e	2015-02-20 18:26:39.747114
71	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	c5bbdfa8-90d7-4310-ab08-4ca6d49e4793	2015-02-20 18:26:39.771352
72	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	c3133652-f1ea-433f-ae76-953806cd002a	2015-02-20 18:26:39.796786
73	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	a0bde5b9-1952-4c13-86aa-bbb026f4b7a9	2015-02-20 18:26:39.821256
74	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	cddb5806-3773-481a-87de-55b27e1e9e49	2015-02-20 18:26:39.847409
75	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	3fe94ee3-eafe-47f7-84c3-619dc61cbef1	2015-02-20 18:26:39.896684
76	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	61ad24fb-7c6b-4230-b44e-178e6ac6d00c	2015-02-20 18:26:39.92082
77	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	1c5bb97e-f007-48c9-8364-bb07c9ec4856	2015-02-20 18:26:39.94325
78	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	7803a26c-ab4a-47ba-8911-e68f89ea42bf	2015-02-20 18:26:39.973744
89	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	9cffcc36-aca9-40b8-9e67-472312367860	2015-02-20 18:26:40.274217
79	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	2ada03bb-f531-4977-a145-68c70ee08b79	2015-02-20 18:26:39.998011
80	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	2d276ac0-c31a-40ba-9520-4353c63de079	2015-02-20 18:26:40.027657
81	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	27e65798-fd5f-4332-a293-07616e3e5e5f	2015-02-20 18:26:40.051443
82	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	c694179f-e086-43a6-8c8e-e662d550ad72	2015-02-20 18:26:40.084024
83	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	b3503063-8941-42ee-aa85-6aa1f71eaa6e	2015-02-20 18:26:40.11765
84	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	27d1d125-f458-4a41-b78f-2523e9a2585e	2015-02-20 18:26:40.144768
85	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	e9fa5835-5c4f-4279-8175-df0c13af17b2	2015-02-20 18:26:40.171966
86	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\n	1	\N	\N	2d586f0e-e12b-43d6-b24b-311bd832bc0f	2015-02-20 18:26:40.195591
87	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	ab903a0c-4928-4cf0-8d5b-41f54297eabe	2015-02-20 18:26:40.223313
88	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	54622249-99f7-4eeb-ba60-ad240bdb20f3	2015-02-20 18:26:40.248682
141	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 3\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	54032abe-fae5-4521-adbc-4aa9f8b6db77	2015-02-20 18:26:41.768329
142	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 4\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	c07e3e65-271d-4c16-9fe7-03f1a985f04b	2015-02-20 18:26:41.796389
90	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	588453b9-5f87-4ed0-a0b4-e92904a6dc7f	2015-02-20 18:26:40.302595
91	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	c66b2c88-bf1c-48e9-a406-ac610add156b	2015-02-20 18:26:40.333087
92	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	4912707e-e58d-48c5-a13d-709a635597c1	2015-02-20 18:26:40.356963
93	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	ad7e5f3e-d22d-4032-9da1-99d6670022f8	2015-02-20 18:26:40.380939
94	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	7f3681fd-5e5c-4cdd-bd37-b93abb3fd509	2015-02-20 18:26:40.405518
95	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	ab2b41c3-a9c6-4d49-8b4e-96db678ecb23	2015-02-20 18:26:40.435952
96	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	4065a871-dd4c-4169-904c-4883e94c9c93	2015-02-20 18:26:40.469072
97	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	7413a6d0-e1ec-4a3e-a835-d1682f025a7f	2015-02-20 18:26:40.494995
98	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-AFF3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	8095df68-433a-4ebf-8c52-ea57a60e12ca	2015-02-20 18:26:40.52467
99	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-BICC1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	e9c1c9ed-2de3-4f6e-8aa4-82c0688406b6	2015-02-20 18:26:40.568672
100	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CASP7\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	9753e402-e00d-4004-9c6c-f1fbb3e788e8	2015-02-20 18:26:40.594226
143	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	73709cd8-64f2-4595-8f73-0d893fd5386a	2015-02-20 18:26:41.822606
101	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CCDC6\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	b059f10f-6999-4cd3-89cc-feabab296b8d	2015-02-20 18:26:40.619315
102	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-KIAA1967\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	fb4c21bc-8d6c-45e1-8f08-d9f149a0ad79	2015-02-20 18:26:40.644224
103	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	222d529e-7d8d-4b72-8743-e5734bc5e271	2015-02-20 18:26:40.672459
104	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-OFD1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	4b19a032-c692-4a23-8f3d-f0e532c85d43	2015-02-20 18:26:40.70161
105	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-SLC45A3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	687dd6f0-fc2d-4adc-b8b8-67cf749437c5	2015-02-20 18:26:40.726202
106	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	b5d96279-adaf-40d3-a29f-29ed5dc52dc0	2015-02-20 18:26:40.756275
107	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: ITD\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	6fe6befb-f17b-4ff0-a8a7-313b76a81b1f	2015-02-20 18:26:40.786426
108	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: TKD\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	11d1d1cd-3193-4acf-ab48-58896bee68a7	2015-02-20 18:26:40.813004
109	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	ffa162cc-cbd4-4a1d-b2f6-631fa0f4f418	2015-02-20 18:26:40.843756
110	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	9e3f6a68-2115-40bb-8f74-1d059ef6d573	2015-02-20 18:26:40.868919
111	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	3ef11019-9207-43a3-bf6c-9467a4fd4c0c	2015-02-20 18:26:40.896058
144	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	db274d14-1c0b-4665-bc49-be84b525d7c8	2015-02-20 18:26:41.846034
112	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	2c780f1f-5239-43f3-88df-87888999c05c	2015-02-20 18:26:40.923845
113	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	f99571d7-15ab-4f7c-bb74-4b8704416d4c	2015-02-20 18:26:40.952373
114	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	38b716c3-51c0-4536-ab4a-be7cd3f6f6d4	2015-02-20 18:26:40.97604
115	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	9b8e3af8-922a-4551-9754-85aac90fda24	2015-02-20 18:26:41.004767
116	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	c86858ac-d4ca-4dd1-b49f-e95e06708b4e	2015-02-20 18:26:41.033167
117	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	86d93358-638a-4f47-991e-9048fb4085fd	2015-02-20 18:26:41.058185
118	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 11\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	81ebc5ef-e0b7-4038-b8c4-c4a45be18bac	2015-02-20 18:26:41.086115
119	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 13\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	9350c275-ccc0-4457-a95e-91aff1f4cf04	2015-02-20 18:26:41.116979
120	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 14\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	69bf44ea-c245-4a84-b5da-efbc2e48b4cd	2015-02-20 18:26:41.146412
121	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 17\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\n	1	\N	\N	69249f45-91a0-4105-920f-e4a803fed904	2015-02-20 18:26:41.173939
122	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 9\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	05b3c6ee-a613-4982-8f66-9941ec02deb6	2015-02-20 18:26:41.203531
123	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	2db1ec56-f744-47e2-a5ca-070ac90ad0df	2015-02-20 18:26:41.253586
124	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	0c6e0a37-b0ab-47b9-a2ba-708c977db9ed	2015-02-20 18:26:41.282575
125	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 1\ndescription: A study by Livre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	430e857b-62ff-4567-9c59-7b607dc030c9	2015-02-20 18:26:41.319599
126	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 2\ndescription: In a study by Pan et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	65446cc9-249a-4d69-bec4-4a72e0a3ca20	2015-02-20 18:26:41.344794
127	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	8af50b6e-0568-4f9e-8320-13f564c31efc	2015-02-20 18:26:41.369768
128	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	989a6a91-e21f-4d4e-8315-899fa9dab89f	2015-02-20 18:26:41.398195
129	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	7ff11981-6d06-4510-b910-8fc02e561da1	2015-02-20 18:26:41.43315
130	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	88e3965a-cebd-480a-afdc-4e1c31b3d56c	2015-02-20 18:26:41.459397
131	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	b2d7c539-197c-4068-9399-29c72c0128c5	2015-02-20 18:26:41.49147
132	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	f9c5b49a-f1eb-43cf-b34a-338c518cec9b	2015-02-20 18:26:41.518915
133	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	b51e772a-7b45-48f6-9027-de4a74025a0c	2015-02-20 18:26:41.554958
134	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	65b5caf4-ee2e-4a05-b2ab-f40e412d4be1	2015-02-20 18:26:41.581093
135	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	3c09f18a-313b-4396-9752-8a9f81911038	2015-02-20 18:26:41.609677
136	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	d622c720-912f-40f8-80c2-9ff50b13d620	2015-02-20 18:26:41.635412
137	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: EXON 12\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	e369ddf5-db9e-4133-b384-0b5b0258a6f8	2015-02-20 18:26:41.662743
138	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	f2eb80d1-e711-4e46-80c0-f73e8e01043d	2015-02-20 18:26:41.690912
139	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 1\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	f15d0baf-87e5-4a53-9318-bcd52c39b3b4	2015-02-20 18:26:41.715327
140	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 2\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	d3fcb89b-77a7-4f56-8a2b-245848de4f23	2015-02-20 18:26:41.743133
145	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	556be9af-52be-4c48-a10e-1c6843bf365d	2015-02-20 18:26:41.871006
146	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	e9bd27b0-5819-42e2-a648-2d2d5134a858	2015-02-20 18:26:41.896878
147	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	89c0e7ab-d126-4539-b9c2-744f7975e922	2015-02-20 18:26:41.948164
148	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	da9a88b1-a500-4993-94c9-41e149c4f5f9	2015-02-20 18:26:41.975085
149	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	2bed9728-9506-459b-8ad6-1b0a4f858ea5	2015-02-20 18:26:42.00056
150	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	a4d5bc6e-09bd-4d55-841d-aee8210016a4	2015-02-20 18:26:42.033827
151	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	632919d3-37d6-4747-9a45-840792ea93a4	2015-02-20 18:26:42.061713
152	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	a3e65829-da31-4d39-aef4-917955dd10e0	2015-02-20 18:26:42.086191
153	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	ab3a3de3-bdbe-4e7e-af8f-c8346cb9f99b	2015-02-20 18:26:42.114714
154	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	bfee66a0-ac45-4c80-86f2-3200cc491c3a	2015-02-20 18:26:42.138988
155	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 20\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	5aa839d4-2d5f-4e93-85f2-5bd143fc5130	2015-02-20 18:26:42.171486
156	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 9\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	a194438c-e856-40aa-b315-b50780860640	2015-02-20 18:26:42.199734
157	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	ebe851a0-929f-499e-987b-af69c051c3e3	2015-02-20 18:26:42.226747
159	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	f6e9973c-ed67-48a0-ba29-926fea004b0e	2015-02-20 18:26:42.287015
160	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	87802cbd-1b67-4638-b69e-0508651e7e6e	2015-02-20 18:26:42.312825
161	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	19ea94ce-f621-4e3c-b574-75f1af3ca468	2015-02-20 18:26:42.342018
162	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	38c560cd-8b80-4f0f-a55d-cd7e5db1279b	2015-02-20 18:26:42.37052
163	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	32886764-260b-484a-8eef-fc9cd15824cb	2015-02-20 18:26:42.397465
164	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	b5484006-dbc1-43b9-ac2a-ed57c088cd91	2015-02-20 18:26:42.423212
165	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	461463d4-6184-4618-9492-dabbafc6f00a	2015-02-20 18:26:42.448204
166	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	6db54ec7-e399-49f2-bf4c-d563bec911d5	2015-02-20 18:26:42.473034
167	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	5b27a864-e0c1-4325-984a-ab8697308dcb	2015-02-20 18:26:42.500722
168	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	71a1036c-0117-4728-9777-bfadb1934260	2015-02-20 18:26:42.527998
169	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	0924c907-52a2-4596-a232-f5c9e6559fa1	2015-02-20 18:26:42.557207
170	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	9864eca2-e0e7-4181-aeba-380148a4b1fa	2015-02-20 18:26:42.606071
171	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	d0b83b86-f586-48db-8640-6cb660afee34	2015-02-20 18:26:42.634199
246	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	af950d72-382a-4ee1-bf8e-85091118e7ac	2015-02-20 18:26:47.433207
172	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	58f3773a-2c84-4b8c-8be9-3608488b5404	2015-02-20 18:26:42.661106
173	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	5d2b8946-c054-42cd-bed6-6b815fabbbc7	2015-02-20 18:26:42.693559
174	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	df1fba30-fb8a-4067-a4a9-b4943dcb3002	2015-02-20 18:26:42.718261
175	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	40dd860e-8e68-4d0a-af89-8805f7df2adb	2015-02-20 18:26:42.742175
176	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	ffaca2b7-488e-46f8-8343-f417800e6a3a	2015-02-20 18:26:42.765978
177	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\n	1	\N	\N	294370ea-62e0-48af-be3b-c0392f3ed0e1	2015-02-20 18:26:42.791204
178	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\n	1	\N	\N	1beaae15-ea3f-4821-8e46-9a75820159df	2015-02-20 18:26:42.8186
179	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 7\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	d5cb95c9-5a84-4030-86f1-4193e7274eee	2015-02-20 18:26:42.848617
180	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 9\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	11b2a779-5de4-4ef1-b00c-7353774d8c15	2015-02-20 18:26:42.878414
181	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL fusions have been found to contribute to imatinib resistance in AML\n  cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 41\nvariant_id: 1\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	67030975-73f7-48be-9427-5d75a058ac61	2015-02-20 18:26:43.054396
182	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide\n  results in high rates of complete remission in patients.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 1\nsource_id: 41\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0e71e8f2-e02e-4a26-a3ba-841aa27c097d	2015-02-20 18:26:43.129947
183	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8a40c356-21f4-4b2e-8553-d6cdfea4d375	2015-02-20 18:26:43.209361
184	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 2\nsource_id: 43\nvariant_id: 1\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4c0f7779-3aec-43b3-bb23-f69ea6169ebe	2015-02-20 18:26:43.301813
185	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The clinical use of imatinib in patients with BCR-ABL fusion has resulted in\n  drastic sensitivity to the drug.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff1649b6-a784-4a8f-9c94-a7c3bcdfeaa9	2015-02-20 18:26:43.365526
186	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a84f574b-14c5-4559-bc36-a4fb43852061	2015-02-20 18:26:43.432181
187	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	95f9c126-7e0b-4344-8a9b-87c9031498d8	2015-02-20 18:26:43.489517
188	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis,\n  response rate, overall survival, and patient outcome in CML patients compared to\n  previous therapeutic regimens."\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	08a49a30-39a1-4e25-a0b1-bb4d37ef7898	2015-02-20 18:26:43.55095
189	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 2\nsource_id: 44\nvariant_id: 3\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3778c288-03b8-4cd9-9443-1f80553067f5	2015-02-20 18:26:43.612978
190	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have\n  shown to be resistant to imatinib treatment.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c5385c6f-0e6c-4928-9a64-f141a687dcd1	2015-02-20 18:26:43.722733
191	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 2\nsource_id: 44\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f6ead261-5b0d-4572-927b-547c17c346c6	2015-02-20 18:26:43.795279
192	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 3\nsource_id: 45\nvariant_id: 4\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c908c2b3-ff9f-49ab-97d9-f127d500d020	2015-02-20 18:26:43.889433
193	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 6\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4dce8287-da8a-4d07-ba1a-1729be691c77	2015-02-20 18:26:43.954559
194	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective in acheiving stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 5\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e6065eae-1c0d-476e-9a27-d5deece7ffc3	2015-02-20 18:26:44.036489
195	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 5\nsource_id: 47\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d6e95060-0d78-4de3-929d-83dd0ee9c904	2015-02-20 18:26:44.112276
196	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 6\nsource_id: 48\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	db30edfb-9dc5-4ce2-8014-c248b0539d75	2015-02-20 18:26:44.182559
197	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 6\nsource_id: 49\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	696df838-75f0-4afd-822e-a78ba1f61964	2015-02-20 18:26:44.245601
198	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 6\nsource_id: 50\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8d142c0c-e243-4232-85fa-081801ec0679	2015-02-20 18:26:44.313603
199	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 8\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	adb2bb15-6390-4e4b-8262-ccdcb960a40a	2015-02-20 18:26:44.375074
200	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 4\nsource_id: 47\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3c58f407-d727-4439-8f53-1d8762d2eaca	2015-02-20 18:26:44.45677
201	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 4\nsource_id: 48\nvariant_id: 7\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	857d4519-5857-4b18-be83-a41f17300276	2015-02-20 18:26:44.522864
202	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 7\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a11cbcbc-c3e5-4ec1-b285-996b99a373ed	2015-02-20 18:26:44.580524
203	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells\n  with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b16773c8-b8d2-439f-8bd0-0ff322475c34	2015-02-20 18:26:44.649231
204	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c95927c4-a985-4d92-87c1-84955ca0a553	2015-02-20 18:26:44.714339
205	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ec4c5a3c-da93-4096-b7f5-5cb6364736c8	2015-02-20 18:26:44.78771
206	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 11\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c13a79f5-d5be-43fc-8bf9-5504fa03501c	2015-02-20 18:26:44.855978
207	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6b5cb4ab-3c3f-416b-9bc0-15dd121b3f83	2015-02-20 18:26:44.913001
208	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 7\nsource_id: 53\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	764a396e-09cc-43e0-b912-52eb5fd0dcef	2015-02-20 18:26:44.997237
209	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in non-chronic sun damaged\n  skin and with superficial spreading melanoma\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 54\nvariant_id: 17\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ffc08036-38c1-4e39-b6fb-b33328e44a79	2015-02-20 18:26:45.066081
210	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 55\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5967fb12-c93a-457f-af5d-972175ef3c73	2015-02-20 18:26:45.127863
211	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 8\nsource_id: 55\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5fa66a7e-a531-4510-acf4-570a3ebc8c78	2015-02-20 18:26:45.172892
212	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 11\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2f75a4a5-f163-4361-99c4-bb3ceb9d7bc1	2015-02-20 18:26:45.236574
213	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3af2bcf9-014e-4b1e-8d45-91241ea251cc	2015-02-20 18:26:45.31029
214	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e87da9b4-9644-4f52-9a01-b7fe52082e85	2015-02-20 18:26:45.354815
215	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5e15f393-6ff6-4626-9b07-f7ce11516b02	2015-02-20 18:26:45.412457
216	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 58\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9971cf80-d260-43f3-9faa-702f945ea17d	2015-02-20 18:26:45.47875
217	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 9\nsource_id: 58\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a2ce01df-4403-4562-9331-9ff8961bf4c3	2015-02-20 18:26:45.555489
218	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 9\nsource_id: 59\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	69f0d013-0118-4485-a531-92892fa1eec9	2015-02-20 18:26:45.614655
219	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 17\ndisease_id: 9\nsource_id: 59\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6ced5f24-a259-4703-aa51-df2a459e017f	2015-02-20 18:26:45.668298
220	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 18\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7a76eab2-9e64-4981-91a8-53b75805349e	2015-02-20 18:26:45.727872
221	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 19\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c9aeb4f4-5146-4cc4-8a76-edc8c294bd9d	2015-02-20 18:26:45.784266
222	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 20\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ba4c747a-39be-4e02-b022-3502d522b406	2015-02-20 18:26:45.83545
223	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 19\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	90d87c1a-6f0a-4542-a298-efe161995bcb	2015-02-20 18:26:45.892251
224	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as a marker of poor prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 61\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	36658aef-2154-4d56-bbac-369c4ffd0bad	2015-02-20 18:26:45.94854
225	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates as well as shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  BRAF\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ccf75ad2-831c-4ecc-ae39-47dfa38833bd	2015-02-20 18:26:46.04015
226	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 8\nsource_id: 63\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b9526738-60f1-4c8e-8bf2-90739f17ff3f	2015-02-20 18:26:46.101998
227	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 8\nsource_id: 63\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	095a838c-dbfe-4a74-bba3-f9a3015e2400	2015-02-20 18:26:46.160322
228	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 21\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	577b1f49-e48a-4c14-81d3-36afbf4bc6f4	2015-02-20 18:26:46.227926
229	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 22\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5f6e7012-69a1-44f9-bb4c-2de6d3ac191a	2015-02-20 18:26:46.291267
230	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 65\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	42f83258-e1c1-4ba4-a63d-263909f8c17a	2015-02-20 18:26:46.35795
231	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 8\nsource_id: 65\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0fc7fa5d-8ef7-4b05-bf18-257e712b5d81	2015-02-20 18:26:46.410755
232	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter disease-free and overall Survival in a\n  Spanish cohort of melanoma patients.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 66\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3fe3b756-b67a-4e98-9376-78857559992b	2015-02-20 18:26:46.476054
233	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 4\nsource_id: 67\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4d52bf11-42d2-40f3-b52d-51348d16e542	2015-02-20 18:26:46.562871
234	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 68\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	266a66d5-fb47-437b-8233-83cb9712922e	2015-02-20 18:26:46.623646
235	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 69\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0aa04b4d-458e-45c1-b8ad-eb33d8ca15aa	2015-02-20 18:26:46.690217
236	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	143b7ec8-52e2-4307-a8d7-a6f06b20682e	2015-02-20 18:26:46.751775
237	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 71\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d5838d2f-c28e-49ab-b94f-d98c91493296	2015-02-20 18:26:46.818552
238	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8e4746bc-e998-459c-a630-4b1ba1bd7611	2015-02-20 18:26:46.881182
239	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 72\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	69ed8782-1b88-4247-9ff6-4122f276327c	2015-02-20 18:26:46.943271
240	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6271db80-bf16-4880-9f0b-1c585e49eb66	2015-02-20 18:26:47.022539
241	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK\n  inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification\n  of the BRAF V600E gene.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 23\ndisease_id: 7\nsource_id: 73\nvariant_id: 14\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	247e5f64-7c36-4040-93eb-33e8cf3b4c72	2015-02-20 18:26:47.113735
242	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 74\nvariant_id: 13\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d0e437eb-9073-47d9-b64c-c0df8adbafb4	2015-02-20 18:26:47.174217
243	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 15\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	50966973-c990-4938-9be5-fe6d8429974f	2015-02-20 18:26:47.237123
244	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 16\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	94e38273-7c8c-4c7d-9a61-f6c5490be97f	2015-02-20 18:26:47.302714
245	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 18\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0d07c188-fa15-497f-98a6-3fa49adcb5b9	2015-02-20 18:26:47.373888
247	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	73190162-08d5-4599-9ce6-a5b364993138	2015-02-20 18:26:47.493604
248	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 76\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3d194081-0311-4153-b6f1-0535eb3a3491	2015-02-20 18:26:47.557759
249	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 76\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	409f4aae-d3b5-4c03-8108-da80f71d994b	2015-02-20 18:26:47.631751
250	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 12\nsource_id: 77\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	463b7a5e-bc76-4392-b0ea-15a15ee3b9ec	2015-02-20 18:26:47.701979
251	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 13\nsource_id: 78\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2bdce689-31f0-4d93-b77d-8fcde7b8688a	2015-02-20 18:26:47.769492
252	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	83ae0605-75ad-480d-aabe-ae4665378d4d	2015-02-20 18:26:47.835559
253	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6af7d5ad-bd90-4cd4-8e59-93e796f88e04	2015-02-20 18:26:47.895992
254	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dd3f76ab-5ddf-4b88-857b-70115099c347	2015-02-20 18:26:47.954501
255	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	78fc1b78-a010-4cbe-90df-ee9d40d72cce	2015-02-20 18:26:48.018366
256	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 80\nvariant_id: 22\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bc4068b6-0d6b-4e14-b852-a32037bd30f1	2015-02-20 18:26:48.098883
257	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 24\ndisease_id: 15\nsource_id: 81\nvariant_id: 23\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3d3a203a-7e4c-4c51-ad4b-e97d23cea25e	2015-02-20 18:26:48.162945
258	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0d17d8c1-e432-4aa4-91f4-00230d7913c8	2015-02-20 18:26:48.225389
259	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 16\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7d70e16c-29c2-4608-bb20-a0d381cafaac	2015-02-20 18:26:48.286484
260	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 16\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c1c3d236-a834-4ee1-a8b4-7e7e462212a2	2015-02-20 18:26:48.353822
261	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	66fda192-9fb8-40b6-ad77-fcf84c5a1ffb	2015-02-20 18:26:48.416724
262	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 18\nsource_id: 84\nvariant_id: 25\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ee01e15b-0544-4250-aa5d-c05255507f26	2015-02-20 18:26:48.475938
263	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 18\nsource_id: 85\nvariant_id: 25\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	579a0530-8044-4bcf-bf11-d6789241949e	2015-02-20 18:26:48.535832
264	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 18\nsource_id: 84\nvariant_id: 26\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0d4bbaa7-8d20-4067-b257-02d6ef855d5c	2015-02-20 18:26:48.612461
265	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 18\nsource_id: 85\nvariant_id: 26\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0603e521-a806-4bb2-9324-2cbd9bc97596	2015-02-20 18:26:48.675577
266	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e5582fb0-8720-47c6-a2d0-0ea7fcfecc91	2015-02-20 18:26:48.737836
267	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a4c489ac-9000-4b52-9cb5-3874f398ec48	2015-02-20 18:26:48.796742
268	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 87\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1706ff4f-b4db-4863-8efe-ac204b19fcef	2015-02-20 18:26:48.852755
269	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 88\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b7b6ff0c-e5ff-445b-936b-1504f6218ed6	2015-02-20 18:26:48.910139
270	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 87\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ccef685e-61b5-45a7-a97b-9c1e11d1f2ba	2015-02-20 18:26:48.96834
271	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	677663b5-4d6d-4b9d-942c-b685ee84a950	2015-02-20 18:26:49.060449
272	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3028494a-e65c-4be9-8b79-066c2e276c4e	2015-02-20 18:26:49.114819
273	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7ddda34d-72b1-4b7d-9ccf-710f71e78bde	2015-02-20 18:26:49.179393
274	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 90\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4ce6b464-e578-4b2a-8a54-067a56b5777e	2015-02-20 18:26:49.245901
275	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 19\nsource_id: 90\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7bd46198-3d5b-4218-8a74-a75fab8ce4c3	2015-02-20 18:26:49.298722
276	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 20\nsource_id: 91\nvariant_id: 31\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3baa9efb-0c85-4a4a-9f77-0ae8a612dc63	2015-02-20 18:26:49.360972
277	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e7c196be-b848-4b2f-a60d-1aad2820b18b	2015-02-20 18:26:49.418595
278	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0005b890-8ae1-429f-bf41-5ee225a73c35	2015-02-20 18:26:49.505386
279	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 28\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	86322fd3-919f-41a5-970f-d4bda8727f33	2015-02-20 18:26:49.580304
280	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a7ed6fdb-14af-462a-a7e5-472b63217f92	2015-02-20 18:26:49.645753
281	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0c9fcdb2-5a99-4ab3-b176-27ce48910af8	2015-02-20 18:26:49.712673
282	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c2558f69-e45b-407d-9f1b-11c589d28311	2015-02-20 18:26:49.772486
283	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 95\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	96380ccc-eefc-49c3-a446-1a3641d766d7	2015-02-20 18:26:49.827854
284	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	22986657-366b-402f-b0dc-e72acc9f3924	2015-02-20 18:26:49.900297
285	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	86866d5f-ba05-4cdc-a240-40d4d9c6f1d5	2015-02-20 18:26:49.98512
286	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bc3b90d5-ca42-4659-bd59-8c0ea1b6a8a1	2015-02-20 18:26:50.040263
287	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1468fe3e-f15e-49cd-9921-3b603212da2f	2015-02-20 18:26:50.104411
288	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including\n  R882) were associated with worse overall survival compared to those without DNMT3A\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5a0a5965-d8f0-4f64-a53c-aa730f27a214	2015-02-20 18:26:50.163861
289	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 95\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	62e0a842-b707-4342-8779-2fda2d755de8	2015-02-20 18:26:50.224849
290	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	90a47f1d-dabd-43c1-b9f1-d5c7a7f74b1e	2015-02-20 18:26:50.287841
291	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	42704d44-056f-49ad-8561-ebdfda3a451e	2015-02-20 18:26:50.352012
307	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 98\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	963ffb25-d1e4-43bb-a62f-5db8940306a3	2015-02-20 18:26:51.432731
292	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	181f746f-e455-45f6-8230-ea9900611ca4	2015-02-20 18:26:50.435932
293	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable, cytogenetically normal AMLs\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e28f5c0d-bcb8-4680-8eb5-0f948257235c	2015-02-20 18:26:50.498115
294	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c1c472c8-492d-4f90-978d-3e6e935b7884	2015-02-20 18:26:50.563102
295	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e02a4db7-dae3-4a00-b8ea-d902fc9f5b25	2015-02-20 18:26:50.62353
296	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d52b97c7-a811-4cd0-9ac0-c79a23300bcc	2015-02-20 18:26:50.678157
297	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fe23ed52-53a1-440f-a140-96bc73cdcc1a	2015-02-20 18:26:50.738939
298	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e1551421-75f9-47af-8f86-0738836bb429	2015-02-20 18:26:50.799406
299	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3a2af05d-bf98-47f4-b489-fd8702c9246e	2015-02-20 18:26:50.936449
300	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a25b32e4-d1b1-4cf5-ad86-77005dd0399b	2015-02-20 18:26:50.99925
301	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c3b4ef16-5bc1-44ef-b23d-daef8f275ee4	2015-02-20 18:26:51.063496
302	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a0b58e6e-82c0-43a8-a8a7-7064e62079c7	2015-02-20 18:26:51.125166
303	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	96cf7dab-6dc8-417f-b4dd-74a2e8664ad8	2015-02-20 18:26:51.191753
304	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b07b35f3-41d1-4db0-9d11-ee8cdf487fc6	2015-02-20 18:26:51.247255
305	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ee6ee3f3-9b45-4dee-8c09-62cbbe02b53f	2015-02-20 18:26:51.307728
306	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 98\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8e58a5ff-9509-464f-8198-cb3340094616	2015-02-20 18:26:51.377724
308	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 99\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b53074e9-763b-48fd-ade1-de261db5489c	2015-02-20 18:26:51.517859
309	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Geftinib added to pemetrexed and carboplatin showed greater effectiveness than\n  premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 100\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cf2faad7-134e-41d6-9160-9794004cd291	2015-02-20 18:26:51.589949
310	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 101\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	77414d15-cb74-40f7-9878-0bbfaf8881f0	2015-02-20 18:26:51.648644
311	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The T790M mutation in EGFR has been shown to confer resistance to the tyrosine\n  kinase inhibitor erlotinib, and patients harboring this mutation that are placed\n  on the drug are likely to relapse.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 21\nsource_id: 102\nvariant_id: 34\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	13f5396e-a747-49ea-87dc-2bc8d41cec04	2015-02-20 18:26:51.716303
312	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 103\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1e836a98-0cca-409d-8beb-0f2333782957	2015-02-20 18:26:51.79285
313	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 104\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	13f49099-999c-4f03-b68e-06723413d9f7	2015-02-20 18:26:51.861282
314	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 32\ndisease_id: 4\nsource_id: 104\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8ae7ec79-d7d4-480f-9023-d764e70e1570	2015-02-20 18:26:51.917233
315	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when an L858R mutation is also present.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 33\ndisease_id: 4\nsource_id: 105\nvariant_id: 34\nrating: 1\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e6adb928-1c55-4f5a-8184-b9abf7fff5a4	2015-02-20 18:26:51.977998
316	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutations.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 106\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d3b6ac27-dabe-4f8a-aa79-e0e5da6198eb	2015-02-20 18:26:52.068308
317	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutations, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 107\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	28ea1ea2-b765-4e77-827e-0c24e252b9fb	2015-02-20 18:26:52.132187
318	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 35\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a1d6e286-ca88-4de7-9f03-b72f544e70b7	2015-02-20 18:26:52.20052
319	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 36\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	04f6525b-a995-4382-9705-441981626d1a	2015-02-20 18:26:52.27235
320	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2,\n  cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive\n  to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 37\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	434c6653-81d1-410d-966b-d803f5a4bb4f	2015-02-20 18:26:52.336894
321	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 38\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	95339df1-cb74-417e-ab2c-b4614b76b2f9	2015-02-20 18:26:52.397388
322	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally\n  transduced with wildtype or mutant ERBB2.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b3a2cf1e-9b8f-4c15-9b69-a5784fa947bc	2015-02-20 18:26:52.460086
399	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 46\ndisease_id: 28\nsource_id: 139\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	710aa6c7-1f16-4e51-9543-68dba5c86298	2015-02-20 18:26:57.637705
323	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell\n  lines retrovirally transduced with wildtype or mutant ERBB2.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 11\nsource_id: 108\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8d20f00b-18a6-4c76-bd5d-21526db666ac	2015-02-20 18:26:52.524227
324	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 40\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e2afca4c-1f49-4297-b23a-f3925409d71b	2015-02-20 18:26:52.602999
325	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the in-frame insertion of a single amino acid at position\n  780 was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 41\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8340eb34-8685-4c96-9bf9-91e2f6d914c9	2015-02-20 18:26:52.671173
326	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1a32d52d-1627-4862-bd61-799b2628b8c5	2015-02-20 18:26:52.741092
327	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 11\nsource_id: 108\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9087a77a-4877-4c57-ac0e-9a132bb437ff	2015-02-20 18:26:52.786167
328	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 43\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c53abc96-fb00-4f7b-bd91-d0c3835f7ba4	2015-02-20 18:26:52.848726
329	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 44\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d037e0eb-a051-4ff5-a8e9-680c14f4c72f	2015-02-20 18:26:52.914591
330	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 45\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	25033360-98f7-4d78-8808-6273b3ea4e04	2015-02-20 18:26:52.990688
331	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	861815fb-e776-4f7b-b14f-721de421d608	2015-02-20 18:26:53.060249
332	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4948c96c-1360-4963-8d9e-d536eb3eee2e	2015-02-20 18:26:53.127909
333	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 46\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ad766381-dd3f-498d-9058-1b6f8e0cba52	2015-02-20 18:26:53.191653
334	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	080536f5-0e65-4dfd-82bb-9aeda17f9399	2015-02-20 18:26:53.255478
335	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	70425b3f-9cb2-42f8-bdae-9e1e82f5fc72	2015-02-20 18:26:53.306384
336	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 47\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	34bd195b-8312-4c7d-addc-8b139975f487	2015-02-20 18:26:53.375636
337	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d8fcf417-789c-473c-bff7-d242323991d0	2015-02-20 18:26:53.438719
400	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 29\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	42b608d5-5ea7-47c3-a99c-c9b69bea447c	2015-02-20 18:26:57.702038
338	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8209c84c-c3be-4b69-8c40-ced2898aee57	2015-02-20 18:26:53.492962
339	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 48\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d185b9c1-a35c-4849-bfe6-be0b3b7a60c9	2015-02-20 18:26:53.577931
340	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	03ccbe55-9c99-49f4-a9d7-acbac3ac229d	2015-02-20 18:26:53.657089
341	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8d334357-a704-4b56-87e4-24c4bb4bbab9	2015-02-20 18:26:53.703734
342	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 49\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	298c2a30-cc25-4477-947d-857d97af7a7f	2015-02-20 18:26:53.773898
343	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 109\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8d284532-3da7-427c-84f6-66db4f347d3e	2015-02-20 18:26:53.838184
344	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bfe12611-bf99-41d2-a500-a3a22796f2f6	2015-02-20 18:26:53.887787
345	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 50\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5214d1fe-280f-4b8b-917d-6fb2b0e221ab	2015-02-20 18:26:53.956888
346	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in necrosis and shrinkage of the primary and metastatic sites\n  and overall stable disease. However, this response did not meet criteria for RECIST\n  partial response.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 22\nsource_id: 111\nvariant_id: 56\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	409a5b84-eee8-4101-923b-3fd67eb62f54	2015-02-20 18:26:54.031154
347	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 22\nsource_id: 111\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d28b9bd7-82f9-478c-bf37-ef09b8d79cd7	2015-02-20 18:26:54.125919
348	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 22\nsource_id: 111\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0602a032-ffec-4edd-947f-9e0d70ab1f6a	2015-02-20 18:26:54.175233
349	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	285d2748-ad58-46d8-b7c7-61996a383113	2015-02-20 18:26:54.240396
350	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 60\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	40ac7018-34f7-4713-a276-1c7a4e3947f3	2015-02-20 18:26:54.299131
351	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d882dc3a-30fe-4ffb-bb5d-9b5220fe6c1b	2015-02-20 18:26:54.360885
431	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 35\nsource_id: 157\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5edb428c-5bd5-4e95-9300-8d010079d7d4	2015-02-20 18:26:59.694211
352	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 40\ndisease_id: 1\nsource_id: 113\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8f663117-1cb8-4123-a00c-0f54c418596e	2015-02-20 18:26:54.425323
353	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 1\nsource_id: 114\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	193f2163-a242-4b6b-bcc4-38d1ae418f2b	2015-02-20 18:26:54.492445
354	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 1\nsource_id: 113\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1653342d-3aa7-486c-9c6d-5c299d674874	2015-02-20 18:26:54.555548
355	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9c20c207-bb0c-4cd6-bea3-0d791f6badbd	2015-02-20 18:26:54.63437
356	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1558e948-faea-4b4b-84e9-bdd2b1fcead4	2015-02-20 18:26:54.707549
357	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	73270f8d-2937-4aec-a653-465ce0115d6c	2015-02-20 18:26:54.772365
358	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	51e2498d-6a17-4187-aaaa-d049f69cfd3a	2015-02-20 18:26:54.843819
359	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ea0e04e4-a121-4448-b974-5cd520863bda	2015-02-20 18:26:54.938616
360	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	08b8ddae-04a0-4b9e-98de-936e120e4e12	2015-02-20 18:26:55.021638
361	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 116\nvariant_id: 60\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	02abad36-db51-4c47-b321-316613805b3c	2015-02-20 18:26:55.083837
362	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bd445956-830d-4f8d-91ee-3c61a0239a8a	2015-02-20 18:26:55.16541
363	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeats, but not\n  in wild type or D835 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 1\nsource_id: 118\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d169a861-a02a-4059-8f2e-d64707079cd1	2015-02-20 18:26:55.23395
364	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 1\nsource_id: 119\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1b3e1329-b6ea-4371-b3f1-7f5a86cd40ef	2015-02-20 18:26:55.296962
365	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 43\ndisease_id: 1\nsource_id: 120\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e86fee7b-632f-4071-8843-f7416a34a676	2015-02-20 18:26:55.362173
366	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 121\nvariant_id: 60\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c38a676b-0def-4b98-bd3d-25269c957838	2015-02-20 18:26:55.422736
367	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f7d18e59-64f2-4be5-b416-306558f691d8	2015-02-20 18:26:55.483173
368	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 40\ndisease_id: 1\nsource_id: 113\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f821181a-53a0-48eb-a338-cbf47f53fe8e	2015-02-20 18:26:55.539553
369	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 1\nsource_id: 113\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	006c98a7-aa5b-49fa-9a80-87b18de3751a	2015-02-20 18:26:55.62435
370	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 122\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3240e2a7-51cf-4789-96b7-caf17236aecc	2015-02-20 18:26:55.67986
371	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	55190ff1-3de2-4f8f-ac63-1aa6b87a26b9	2015-02-20 18:26:55.738633
372	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 1\nsource_id: 123\nvariant_id: 61\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	be6e0d08-6637-4e82-914f-dbdb16026e3d	2015-02-20 18:26:55.806866
373	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 1\nsource_id: 118\nvariant_id: 61\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ba08e284-6fc4-493b-8dbc-99420bd9622e	2015-02-20 18:26:55.866182
374	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 1\nsource_id: 119\nvariant_id: 61\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	144c26ce-5d9b-4a9e-8260-6be711042425	2015-02-20 18:26:55.924955
375	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8ea94ab2-a314-4716-ba78-9ffffc2864fa	2015-02-20 18:26:55.985988
376	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation was associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 124\nvariant_id: 61\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff0153e7-e367-4d7d-8f77-6c2b01a52b47	2015-02-20 18:26:56.074506
377	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 44\ndisease_id: 23\nsource_id: 125\nvariant_id: 62\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d0fb5195-50d8-4a1e-a1f0-80317c9a9661	2015-02-20 18:26:56.141198
378	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 45\ndisease_id: 23\nsource_id: 125\nvariant_id: 62\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2d3ae679-0cf1-4379-a1c0-1eab32a92f87	2015-02-20 18:26:56.19027
379	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 126\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	eaf7ead4-6d5d-4fc9-8ccf-451561055e9b	2015-02-20 18:26:56.25171
380	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 127\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2886d137-6e5c-4472-940b-5dfe498077d9	2015-02-20 18:26:56.320419
381	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 128\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bff1c7cf-533e-485b-b478-358859811ffc	2015-02-20 18:26:56.385004
382	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 24\nsource_id: 129\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff16ea90-a9d5-465f-bbca-e3d41fe93003	2015-02-20 18:26:56.446973
383	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 25\nsource_id: 130\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cacfdc2c-12c4-4ef0-b053-9963063b8036	2015-02-20 18:26:56.51712
384	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 131\nvariant_id: 63\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ec38e07b-846d-4a82-a797-15720cb27323	2015-02-20 18:26:56.601393
385	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 64\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	602f31a8-f1cd-4b1c-a507-dd138e9d6e58	2015-02-20 18:26:56.661539
386	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 64\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	37d6a8fc-9413-4c9f-a02a-1bba88b07b34	2015-02-20 18:26:56.72455
387	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 65\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1a6a866c-8f7a-4d59-a4da-c01a15eb4c6e	2015-02-20 18:26:56.784463
388	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	89a0c841-3a8e-4324-b63d-7e340bdc9d76	2015-02-20 18:26:56.844631
389	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 133\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8de195be-1a85-497b-a027-8724469757f6	2015-02-20 18:26:56.915093
390	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact\n  overall survival or disease free survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 134\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3a1ca67d-7cda-4b09-bfdb-eadcc2b48e25	2015-02-20 18:26:57.006531
391	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 135\nvariant_id: 67\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	76dee88f-fc15-4af1-ab9d-b1259aaf8136	2015-02-20 18:26:57.066645
392	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 27\nsource_id: 136\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	47cf3f9c-9b81-4e77-9da5-b69c56fd8a28	2015-02-20 18:26:57.134481
393	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 137\nvariant_id: 67\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9670782c-2859-437e-9d78-030401d14133	2015-02-20 18:26:57.192299
394	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 135\nvariant_id: 68\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4571cab2-5dc9-4847-b115-5f9fa186feb8	2015-02-20 18:26:57.24986
395	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with an IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 133\nvariant_id: 68\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f85b8d4b-67a5-48ee-aad5-e7ecf9d9c3a6	2015-02-20 18:26:57.302633
396	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutations such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 137\nvariant_id: 68\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8b7bea1f-267b-4f68-a0f7-b117c4fd3442	2015-02-20 18:26:57.363297
397	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F not associatted wth  lymphoid leukemia.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 2\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3d3cb6ea-0ba1-49fc-aa84-fbd0d51583b6	2015-02-20 18:26:57.510083
398	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 2\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7fc51e8d-fb8a-4d60-a08d-5de7e7dc9be7	2015-02-20 18:26:57.567984
401	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: rs10974944 SNP in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 140\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	56c0ea8f-ae72-4de6-9001-283dd2b87b3e	2015-02-20 18:26:57.761548
402	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 47\ndisease_id: 31\nsource_id: 141\nvariant_id: 69\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fc6f5986-86ca-4739-9125-922d05c05948	2015-02-20 18:26:57.829717
403	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 32\nsource_id: 142\nvariant_id: 70\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	740f5a51-e8cb-4a54-bb0b-d835b41592bc	2015-02-20 18:26:57.899043
404	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2c42e3cf-433c-4bcc-a61a-5cc5f740c660	2015-02-20 18:26:57.966648
405	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 71\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5c0051e3-0108-475c-bfa0-55a7221a7c76	2015-02-20 18:26:58.02964
406	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 145\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9930539d-7cc7-42b8-9d56-516d804936c4	2015-02-20 18:26:58.08783
407	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1daa7b09-bd55-4e4c-a951-095c5cfabaeb	2015-02-20 18:26:58.14452
408	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 146\nvariant_id: 71\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a610c1a6-36f3-427e-be41-5acb0f8c4141	2015-02-20 18:26:58.207027
409	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1ad91fd0-68a5-4656-89d2-ae28ef350221	2015-02-20 18:26:58.269399
410	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	beba5c1f-c52a-471f-a2e7-ae68f2229c20	2015-02-20 18:26:58.34625
411	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 73\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	688824d0-7271-4903-bfe7-22d651e4ba3c	2015-02-20 18:26:58.407857
412	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0bdfc0d1-e054-47b8-84bd-3ca8d5fe9e55	2015-02-20 18:26:58.465503
413	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 74\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2e93b676-069d-42d1-9388-30e417e3d9ed	2015-02-20 18:26:58.535366
414	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cf786be7-57a4-4a2c-8c17-707375f86737	2015-02-20 18:26:58.593813
415	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 146\nvariant_id: 75\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3404f822-2b42-4db1-a555-7142cbbf28b2	2015-02-20 18:26:58.652382
432	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 158\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2bd1ac10-a195-49f5-a6bb-5f52319cf0f3	2015-02-20 18:26:59.763753
416	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ed72fe51-f520-4d2f-8d07-73f87a5847d2	2015-02-20 18:26:58.718676
417	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	adc98f19-2c44-4a83-a45d-242b6c2fb65e	2015-02-20 18:26:58.775655
418	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	903fe75f-67c2-43e6-8c71-cf044712dc79	2015-02-20 18:26:58.828297
419	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fd53979b-2f5b-4c26-8e69-8632e9e89c4c	2015-02-20 18:26:58.89821
420	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d9148bc2-0a1f-457c-9f61-6490cdab710f	2015-02-20 18:26:58.967777
421	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	297244eb-63a4-4ade-9c4b-1ac009f009f2	2015-02-20 18:26:59.016502
422	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7a017030-373d-418d-a1b1-f4fb8f4172b9	2015-02-20 18:26:59.08935
423	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 33\nsource_id: 150\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ddb20406-f449-417e-8fae-67f2cf750482	2015-02-20 18:26:59.160177
424	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 48\ndisease_id: 33\nsource_id: 151\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cd6f4542-1696-4b44-8d74-7897103c33b1	2015-02-20 18:26:59.242451
425	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 152\nvariant_id: 78\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a39901ea-955a-4c10-8b2f-91030ce6c192	2015-02-20 18:26:59.316865
426	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 153\nvariant_id: 79\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b5f18a97-77d5-4f58-9ed7-a4518cc44960	2015-02-20 18:26:59.378827
427	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 154\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	87ea1463-3b62-44ee-97d4-bdf7dda0e387	2015-02-20 18:26:59.442464
428	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 154\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9abd854d-711f-4fbc-b0dd-b752bcaa5768	2015-02-20 18:26:59.493697
429	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 9\nsource_id: 155\nvariant_id: 80\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3784be27-7d10-43fe-9269-caa7979e82ae	2015-02-20 18:26:59.559208
430	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e4477d77-381a-4a6a-a46d-88b938ae2c2e	2015-02-20 18:26:59.629755
433	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 159\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1d4ade1a-33db-4683-808b-aefa00b53fbc	2015-02-20 18:26:59.826629
434	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 160\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7c997c0e-03a2-4df2-9454-555af0186ba5	2015-02-20 18:26:59.884039
435	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 7\nsource_id: 161\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	75d95514-d9d3-4124-92eb-77554d1c8fe6	2015-02-20 18:26:59.947481
436	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f7c68dc3-b8ec-4723-8143-cc7114ea7bce	2015-02-20 18:27:00.012225
437	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 161\nvariant_id: 81\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	86533562-268c-4109-8679-620864e6b1e2	2015-02-20 18:27:00.080754
438	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 162\nvariant_id: 82\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5d4f999d-02c4-47f4-9513-0d40d5a96389	2015-02-20 18:27:00.141994
439	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 162\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	31a2aeb4-24f5-461c-b5e7-e1de32bdbb1d	2015-02-20 18:27:00.216155
440	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 4\nsource_id: 163\nvariant_id: 83\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	35d2e3ef-d725-4456-b686-9d2736cb9f2b	2015-02-20 18:27:00.278813
441	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 50\ndisease_id: 4\nsource_id: 164\nvariant_id: 83\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	46ecb031-0f34-4d0a-a1b8-67a45375ebc7	2015-02-20 18:27:00.346361
442	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 51\ndisease_id: 4\nsource_id: 164\nvariant_id: 83\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9990879c-7410-4fd5-bdad-0503a6a5cb7f	2015-02-20 18:27:00.394207
443	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6ed6bc5c-b20a-4bd6-8fad-394ae6e66de3	2015-02-20 18:27:00.448752
444	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 36\nsource_id: 157\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	62c36a6c-4be6-4122-85b7-c24541289d72	2015-02-20 18:27:00.50296
445	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 158\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c6cf5df7-6226-4879-aaa0-a4baed8ab7ad	2015-02-20 18:27:00.554828
446	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d07ac244-8baa-4100-8ca0-6f90687ab8bd	2015-02-20 18:27:00.608876
447	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1b253631-4e44-482f-9266-9274407b711f	2015-02-20 18:27:00.711784
448	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b0f7ffdb-c28e-4a9f-89da-bc2366f33d15	2015-02-20 18:27:00.779359
449	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b80d824d-5c56-4c74-a319-725542bd8380	2015-02-20 18:27:00.839741
450	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 8\nsource_id: 166\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3cef4cc3-2059-4d7b-af5b-f22f2de1d702	2015-02-20 18:27:00.909292
451	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 8\nsource_id: 167\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a36aeb77-3132-4bea-91e5-348eb6d51675	2015-02-20 18:27:00.968422
452	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 90\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	44f98c54-8e82-4dfa-89e7-ba6f8fc3e1be	2015-02-20 18:27:01.028302
453	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 19\nsource_id: 90\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3f9ae38c-1d02-43f1-a749-7747e07c4790	2015-02-20 18:27:01.076026
454	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increased survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 22\ndisease_id: 37\nsource_id: 168\nvariant_id: 89\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d4394e49-9404-4b02-99c5-96b539b8bf3e	2015-02-20 18:27:01.137533
455	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0792c11e-0da4-4bdc-ba54-584fc8b40c82	2015-02-20 18:27:01.204329
456	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 52\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f86fcdcb-a008-46cc-89ce-ea39ca2708d5	2015-02-20 18:27:01.265727
457	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 53\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6ac9c99a-871b-4de7-a04d-a996c301a05b	2015-02-20 18:27:01.333686
458	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d334e229-6b69-4cd7-9fa5-53a6d0dd68d0	2015-02-20 18:27:01.396963
459	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a044557f-1142-4eb3-bdf9-a7f58cb5fd64	2015-02-20 18:27:01.460406
460	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype\n  (0/55).\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0e2bd1d8-fae7-40a5-b022-2827421f13ca	2015-02-20 18:27:01.522083
461	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid\n  leukemia.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	29ee090d-867e-4229-8d35-4ba83f5aec36	2015-02-20 18:27:01.578494
462	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid\n  leukemia\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	19e9607c-8e6f-417b-88b7-4cef45b4c6f5	2015-02-20 18:27:01.646474
463	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	418c41ec-3ea8-4695-941a-105a7a1b4c5c	2015-02-20 18:27:01.701978
464	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9ad37856-242c-4b39-85ce-e17f67596b09	2015-02-20 18:27:01.769339
465	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 1\nsource_id: 114\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1128da5f-a611-4a0a-93e8-c7b27e43fa32	2015-02-20 18:27:01.834235
466	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 173\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	671895c9-bfdc-4575-9527-d1d3035c07dc	2015-02-20 18:27:01.893803
467	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	35bca88b-9f2c-4094-bd1a-eed28466a3c5	2015-02-20 18:27:01.956951
468	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 54\ndisease_id: 1\nsource_id: 174\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2e5947c9-8f0d-487b-aef6-0f3c46374106	2015-02-20 18:27:02.028189
469	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 41\ndisease_id: 1\nsource_id: 174\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c4ae9b39-6b07-415c-9390-0368e62a4c57	2015-02-20 18:27:02.079158
470	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	819dc6d1-6e31-4535-b23d-ada892731da5	2015-02-20 18:27:02.137509
471	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	72fc3132-a766-4c92-a6d7-c90e3ec2e2de	2015-02-20 18:27:02.235031
472	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 28\ndisease_id: 1\nsource_id: 175\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3cccf498-7570-4397-9b26-adb4ebd506c0	2015-02-20 18:27:02.305834
473	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 55\ndisease_id: 1\nsource_id: 176\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f54112c5-819d-432d-9693-c7f58cdf3524	2015-02-20 18:27:02.372793
474	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 177\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	93647502-3799-48f0-98b5-0884286895b2	2015-02-20 18:27:02.429689
475	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 177\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e351d666-57a3-4ca9-9756-10a2da2a4f9d	2015-02-20 18:27:02.488289
476	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1 mutations.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d015d147-b483-4e19-b812-60a523ae922a	2015-02-20 18:27:02.546391
477	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 178\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b812ec73-6c5c-4c06-b3c8-39ceac997c6d	2015-02-20 18:27:02.615912
478	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 179\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0f97253e-668d-44d7-bbd4-d941b6157793	2015-02-20 18:27:02.679458
479	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 180\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6b6bfa54-88d6-445e-8d53-af17b27a5953	2015-02-20 18:27:02.743538
480	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  were longer for patients with Exon 12 NPM1 mutations.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	aed12677-d161-487c-8530-2f1da72a778c	2015-02-20 18:27:02.802296
481	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 122\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bc0d966c-e9ae-4cf4-9659-bb5bebc16a0d	2015-02-20 18:27:02.864036
482	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 181\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fb41b651-4dc7-4e67-8473-c8158f60ff90	2015-02-20 18:27:02.93107
483	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 182\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4166c47e-6e58-4753-8048-666443d53857	2015-02-20 18:27:02.99971
484	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1\n  and IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 175\nvariant_id: 90\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4a08fa5b-f050-424b-952b-5c68d16fccf9	2015-02-20 18:27:03.061445
485	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 88\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	265c4829-3119-48c7-85c0-7a9116d48f39	2015-02-20 18:27:03.115904
486	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 126\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9f857e21-f818-4b45-be51-8868138d22e9	2015-02-20 18:27:03.168172
487	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c0df9956-b75a-4472-b5b1-ace22e37dc82	2015-02-20 18:27:03.222572
488	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4b032d4b-5dd8-4808-b08a-d597990a0378	2015-02-20 18:27:03.279134
489	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 90\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	adb0bb5e-2371-46dd-aeb8-a5e7eaf3e885	2015-02-20 18:27:03.352109
490	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 183\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	eee1b060-eac6-4eee-975a-f9f007c96b68	2015-02-20 18:27:03.42555
491	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 180\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	10186bab-8971-4310-ad0c-7070a29b7672	2015-02-20 18:27:03.481029
492	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"AML with mutated NPM1" is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.'\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 184\nvariant_id: 90\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	690d5a2e-fc7a-4623-a3b1-f21fd581337c	2015-02-20 18:27:03.548916
493	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 91\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9074df54-011e-4387-980d-eb5d6957200e	2015-02-20 18:27:03.609934
494	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 91\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	14cd231c-8c66-4de7-afe0-032f7e0fec8c	2015-02-20 18:27:03.682748
495	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 54\ndisease_id: 1\nsource_id: 174\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f87a770f-668b-4c2d-bc36-0272a2d49757	2015-02-20 18:27:03.794503
496	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 185\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5a98e6c0-415f-48e8-8da4-daccd2a54844	2015-02-20 18:27:03.86874
497	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 186\nvariant_id: 92\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	48f03d41-4475-4c9f-939e-83f6af7967d1	2015-02-20 18:27:03.937586
498	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patients does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 187\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f0ec1257-4481-49ac-87b5-74b387452d12	2015-02-20 18:27:04.0129
499	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	225fa6e5-e5d4-42f9-a006-4affefc485f5	2015-02-20 18:27:04.078642
500	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 186\nvariant_id: 93\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	db5a118a-a5c9-4d5c-9737-d921cb4d89e2	2015-02-20 18:27:04.138709
501	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	516cfa9f-2198-4b3e-b460-2c783841f570	2015-02-20 18:27:04.197176
502	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 95\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1743293e-792c-4543-ac29-21b725ec2365	2015-02-20 18:27:04.262859
503	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 189\nvariant_id: 96\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bdb612be-98f5-4a38-9ae2-0c4812df0e8a	2015-02-20 18:27:04.32634
504	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 56\ndisease_id: 8\nsource_id: 190\nvariant_id: 97\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	154c36a4-a0cc-44af-9959-53cfb23993ae	2015-02-20 18:27:04.401102
505	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 98\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	95c5778e-f4bc-4d48-81f3-65ee8a50cdf5	2015-02-20 18:27:04.463133
506	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 98\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	66cbe158-d6c6-4266-8207-9eabd0541b44	2015-02-20 18:27:04.517817
507	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 191\nvariant_id: 98\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	59c44b27-bde6-436a-9ecf-d76c9cba7891	2015-02-20 18:27:04.578373
508	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 19\ndisease_id: 8\nsource_id: 192\nvariant_id: 98\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	50beca60-c752-4a8d-af17-d5167c577928	2015-02-20 18:27:04.639911
509	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 8\nsource_id: 193\nvariant_id: 99\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b0e11385-2579-4dea-8905-2116c46ee9d0	2015-02-20 18:27:04.703356
510	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 8\nsource_id: 193\nvariant_id: 100\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7c09a252-8958-4968-8447-15c3909ef58f	2015-02-20 18:27:04.764804
511	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e56fef04-70fc-403e-8283-e0984a102d64	2015-02-20 18:27:04.828242
512	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumors harboring PDGFRA D842V mutation are likely to be benign\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 38\nsource_id: 195\nvariant_id: 102\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2647d4fb-22cb-4e23-9899-b030e1ce6163	2015-02-20 18:27:04.897619
513	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 38\nsource_id: 150\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7657faf0-9c80-4f86-a1e0-74b259200d15	2015-02-20 18:27:04.956164
514	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 38\nsource_id: 196\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c527d366-87fe-40e2-a34e-d99a92faf358	2015-02-20 18:27:05.035776
515	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ffcc18fb-f404-45d9-99cd-a4ffd14a6034	2015-02-20 18:27:05.103651
516	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 103\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e17c2e52-4259-4a3a-b60a-208d30a1293c	2015-02-20 18:27:05.173975
517	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 104\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8d514289-c289-4c29-9fda-797c34b4e106	2015-02-20 18:27:05.238866
518	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 38\nsource_id: 194\nvariant_id: 105\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	993e2b2f-cdc9-4dc3-af8c-45794401e5fc	2015-02-20 18:27:05.351757
519	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 11\nsource_id: 197\nvariant_id: 106\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	adce6626-03f2-4aed-b323-22eceed245ae	2015-02-20 18:27:05.420936
520	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 198\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c7f19d40-d70c-40da-a99e-a69a2374ec8b	2015-02-20 18:27:05.48617
521	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 153\nvariant_id: 106\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	eb402428-81c8-4574-a6f8-9ef9567b557c	2015-02-20 18:27:05.55042
522	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d7b5054a-6363-41cd-b0b8-1934b5d5aaae	2015-02-20 18:27:05.612363
523	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f5592461-fa2e-4d3d-9e94-82888e596157	2015-02-20 18:27:05.668197
524	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff5c45ba-6e67-4b8a-a5e0-fd9f6f33d297	2015-02-20 18:27:05.727668
525	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 200\nvariant_id: 106\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	93c882ac-0efd-41ba-a0a8-38dea4b0000c	2015-02-20 18:27:05.789816
526	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 11\nsource_id: 197\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d8e5131e-6e11-49f1-b566-f6ad42a3db3f	2015-02-20 18:27:05.852445
527	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 198\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1f813c0c-1322-4d5e-a805-d9ad08594ad8	2015-02-20 18:27:05.909337
528	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 153\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7924f42a-db4f-42ee-8692-fc4552d8bb60	2015-02-20 18:27:05.968725
529	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2410b718-b276-49b2-b662-3962f5c5d2e6	2015-02-20 18:27:06.040048
530	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	69e335a0-9859-4dde-97fc-2bc85ca901b7	2015-02-20 18:27:06.099166
531	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5a1fe0dc-c2fd-49ed-b30f-9ee61cb6cc1a	2015-02-20 18:27:06.155328
532	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 200\nvariant_id: 107\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bd634426-ce0a-428a-a500-604b21d3b06e	2015-02-20 18:27:06.214088
533	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype PIK3CA\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 108\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6dac4d40-5baa-4944-942f-fb3fa2007a09	2015-02-20 18:27:06.276333
534	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 109\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	39b67852-1193-4ebc-84a8-5f02f5a0a4ef	2015-02-20 18:27:06.33584
535	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 11\nsource_id: 197\nvariant_id: 110\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	997e3f6f-ef6d-4f2a-90b8-49e044e4a6ac	2015-02-20 18:27:06.393199
536	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 198\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	085fcf7b-2121-4de3-99f8-34cae18b2c73	2015-02-20 18:27:06.466995
537	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 153\nvariant_id: 110\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e0a9b973-b725-4e51-b88a-8b954613ddd3	2015-02-20 18:27:06.537681
538	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1d6bcbb3-32ad-4288-85f8-1a84311c4d1c	2015-02-20 18:27:06.617562
539	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	abbd8ff6-6fe9-46d2-b8f0-780711e4b6b5	2015-02-20 18:27:06.668416
540	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8f4cb45f-3d4e-4d45-92d6-584a8569031b	2015-02-20 18:27:06.730752
541	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 39\nsource_id: 201\nvariant_id: 111\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	de2f2148-d9f9-4757-bf6d-23ce2556ed1c	2015-02-20 18:27:06.804151
542	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 20\nsource_id: 91\nvariant_id: 112\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fa7891b7-a247-403f-bf89-611a2e9f0344	2015-02-20 18:27:06.909194
543	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 3\nsource_id: 202\nvariant_id: 113\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1e61843f-17d0-4b4e-af89-669a3846997f	2015-02-20 18:27:06.972917
544	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 37\nsource_id: 203\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8d6cc975-0dff-49eb-941b-e25da9a5dfff	2015-02-20 18:27:07.042683
545	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 39\nsource_id: 201\nvariant_id: 114\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9ee6f91f-7db3-45dc-a060-5d8c3a2ee4f8	2015-02-20 18:27:07.107993
546	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 204\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	294a08c2-41e1-4fdc-bb55-543369af5f30	2015-02-20 18:27:07.180456
547	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 40\nsource_id: 204\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e0eef9fa-9aef-4cbb-8d6f-5c47826e1338	2015-02-20 18:27:07.249415
548	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 40\nsource_id: 205\nvariant_id: 116\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	239460b7-1a07-4bd2-a507-2a75152002ff	2015-02-20 18:27:07.323472
549	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong\n  repression of tyroid cancer cell growth.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 62\ndisease_id: 40\nsource_id: 206\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	26b4aa7d-4d79-4aae-8344-ffb75dcfcde3	2015-02-20 18:27:07.393731
550	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 40\nsource_id: 207\nvariant_id: 116\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4da4d711-0493-47ff-b48c-a71c5fd5e5a5	2015-02-20 18:27:07.4496
551	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 208\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9b8b982e-6c36-4de2-b620-89ca9f8e5cfb	2015-02-20 18:27:07.527502
552	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 208\nvariant_id: 118\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	33ca397b-94c4-40c1-a411-0656618adf9c	2015-02-20 18:27:07.587972
553	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 63\ndisease_id: 11\nsource_id: 209\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	035e1058-f84b-40e2-a626-c1ce21e0695e	2015-02-20 18:27:07.652885
554	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e49ad36e-2d79-4491-a104-897eda600cc0	2015-02-20 18:27:07.720663
555	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4f2255e3-3152-446d-bfa9-e787f4935b98	2015-02-20 18:27:07.781252
556	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	31927db4-5fb2-4a8f-b5d2-02d8a87fe515	2015-02-20 18:27:07.850724
557	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4257197b-5450-4b90-9dac-5afb770d4e05	2015-02-20 18:27:07.911996
558	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6431dcf9-11bd-4f43-bf92-a3a7d0873fe8	2015-02-20 18:27:07.974546
559	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 123\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	42d8cec9-87e8-474f-ba63-b8d27d5e6a32	2015-02-20 18:27:08.040569
560	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ed3dcf93-a66a-4f73-9746-72eedb053d07	2015-02-20 18:27:08.097753
561	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b916e888-678d-4616-82c5-7de86ef7f6d1	2015-02-20 18:27:08.156296
562	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b31ce5e4-b7b3-40bd-9459-305eb9ab9dbb	2015-02-20 18:27:08.210522
563	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3c8741db-71c1-4c3a-b61a-223186b456eb	2015-02-20 18:27:08.270241
564	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumors with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ab08a0dc-9b7a-457e-82fd-a5ffdd27e76e	2015-02-20 18:27:08.38031
565	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 64\ndisease_id: 41\nsource_id: 212\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	595ab2cf-4d51-4cde-bf55-fc8df5875939	2015-02-20 18:27:08.444817
566	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 4\nsource_id: 213\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ec89a2d8-8e68-454b-b600-76d64ba6c4c1	2015-02-20 18:27:08.518339
567	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 64\ndisease_id: 42\nsource_id: 214\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	24cf665b-e409-4765-9d52-6e7c5a680d97	2015-02-20 18:27:08.585508
568	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 64\ndisease_id: 42\nsource_id: 212\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	30941faa-1cf5-49bb-8ef0-e35c4884c2f5	2015-02-20 18:27:08.656888
569	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1fb6bb67-dd7b-4e47-96f1-4d2e68d369e1	2015-02-20 18:27:08.732655
570	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 130\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8baa78aa-74fd-4329-8208-5cab9ab47557	2015-02-20 18:27:08.787305
571	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 215\nvariant_id: 130\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	aa9a5e10-354d-42d3-976e-9515def30aa4	2015-02-20 18:27:08.858973
572	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 216\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	31d70a85-c898-47bc-8f38-9329cd5081a5	2015-02-20 18:27:08.92725
573	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 131\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ca5b53a7-1308-4300-a957-a733fb7af7a0	2015-02-20 18:27:08.983756
574	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 131\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cbcd9492-9b28-4217-a9da-6b9fa5f7c037	2015-02-20 18:27:09.048223
575	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 215\nvariant_id: 131\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bff447bc-96ae-475b-915f-1bf828475b33	2015-02-20 18:27:09.120155
576	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, the presence of U2AF mutations such as\n  S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 216\nvariant_id: 131\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f27f1224-f23b-4995-93a4-b1ef301f872d	2015-02-20 18:27:09.177793
577	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4e465bd0-54a2-4ed9-8290-4b3ec6d6bbb0	2015-02-20 18:27:09.242642
578	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e9a83bb6-5c77-490e-beda-558b3fc39257	2015-02-20 18:27:09.301279
579	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a594c552-8d86-4923-a258-14ce316fdbc5	2015-02-20 18:27:09.355931
580	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f72b2c98-261c-44d8-8c01-0936261ac681	2015-02-20 18:27:09.413666
581	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bbe62162-3958-4615-9f37-854a404bf864	2015-02-20 18:27:09.480164
582	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 219\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b7fdef89-3fb0-4563-9a43-8a3ed628df42	2015-02-20 18:27:09.550179
583	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6e7ba1d8-c85f-4878-a836-6f66dade1fca	2015-02-20 18:27:09.608916
584	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations in WT1 were associated with a worse overall prognosis than patients\n  wildtype for WT1 in young patients (15-50), primarily because of increased risk\n  of disease recurrence\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 220\nvariant_id: 132\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dbc25058-3426-4c7c-8c52-5dba98704991	2015-02-20 18:27:09.669692
585	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations  in WT1 were associated with increased risk of recurrence in young\n  patients (15-50) with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 220\nvariant_id: 132\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c39f82ea-3704-43f5-be16-9ebb7f55815d	2015-02-20 18:27:09.739306
586	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	95d61245-3c9e-4561-a6f3-98aebb147398	2015-02-20 18:27:09.798723
587	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	669b62bb-c608-4f7b-b3b6-a98c1133e90f	2015-02-20 18:27:09.907646
588	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	58f34126-10ed-4f3c-b535-2c9bb1481307	2015-02-20 18:27:09.964266
589	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e34a7510-c21a-40a2-8e18-b4ccbcfcf560	2015-02-20 18:27:10.024543
590	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a0bf84f2-acc0-4364-9333-eea178e4b306	2015-02-20 18:27:10.084934
591	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 219\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e47417ed-67d3-4969-9485-0c2af7794be9	2015-02-20 18:27:10.140664
592	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	488044c7-4ee2-4dec-8654-72a2fba4f81a	2015-02-20 18:27:10.196368
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 592, true);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
3	1612	Breast Cancer	2015-02-20 18:26:43.871468	2015-02-20 18:26:43.871468
6	769	Neuroblastoma	2015-02-20 18:26:44.161862	2015-02-20 18:26:44.161862
8	1909	Melanoma	2015-02-20 18:26:45.045656	2015-02-20 18:26:45.045656
22	4947	Cholangiocarcinoma	2015-02-20 18:26:54.013395	2015-02-20 18:26:54.013395
24	3069	Astrocytoma	2015-02-20 18:26:56.431404	2015-02-20 18:26:56.431404
34	1240	Leukemia	2015-02-20 18:26:59.612846	2015-02-20 18:26:59.612846
1	9119	Acute Myeloid Leukemia	2015-02-20 18:26:43.023027	2015-02-20 18:29:01.030999
2	8552	Chronic Myeloid Leukemia	2015-02-20 18:26:43.187054	2015-02-20 18:29:01.174002
4	3908	Non-small Cell Lung Carcinoma	2015-02-20 18:26:43.935064	2015-02-20 18:29:01.67839
5	1115	Sarcoma	2015-02-20 18:26:44.090567	2015-02-20 18:29:01.878189
7	9256	Colorectal Cancer	2015-02-20 18:26:44.957656	2015-02-20 18:29:02.286982
9	9256	Colorectal Cancer	2015-02-20 18:26:45.292584	2015-02-20 18:29:02.80384
10	1781	Thyroid Cancer	2015-02-20 18:26:46.607144	2015-02-20 18:29:03.022513
11	1612	Breast Cancer	2015-02-20 18:26:47.537659	2015-02-20 18:29:03.435417
12	8618	Oral Cavity Cancer	2015-02-20 18:26:47.678055	2015-02-20 18:29:04.061838
13	0050746	Mantle Cell Lymphoma	2015-02-20 18:26:47.746789	2015-02-20 18:29:04.199176
14	10534	Stomach Cancer	2015-02-20 18:26:47.818559	2015-02-20 18:29:04.435163
15	715	T-cell Leukemia	2015-02-20 18:26:48.146537	2015-02-20 18:29:04.584702
16	5635	Gastric Adenosquamous Carcinoma	2015-02-20 18:26:48.267143	2015-02-20 18:29:04.719877
17	1324	Lung Cancer	2015-02-20 18:26:48.397857	2015-02-20 18:29:04.937021
18	0060075	Estrogen-receptor Positive Breast Cancer	2015-02-20 18:26:48.460426	2015-02-20 18:29:05.075494
19	9952	Acute Lymphocytic Leukemia	2015-02-20 18:26:49.230738	2015-02-20 18:29:05.233198
20	5015	Hepatocellular Fibrolamellar Carcinoma	2015-02-20 18:26:49.341027	2015-02-20 18:29:05.366797
21	3908	Non-small Cell Lung Carcinoma	2015-02-20 18:26:51.69621	2015-02-20 18:29:05.540567
23	3909	Bronchogenic Lung Adenocarcinoma	2015-02-20 18:26:56.125748	2015-02-20 18:29:05.86137
25	3068	Glioblastoma Multiforme	2015-02-20 18:26:56.492961	2015-02-20 18:29:06.379896
26	3069	Astrocytoma	2015-02-20 18:26:56.581826	2015-02-20 18:29:06.779366
27	0050908	Myelodysplastic Syndrome	2015-02-20 18:26:57.118492	2015-02-20 18:29:06.934466
28	1240	Leukemia	2015-02-20 18:26:57.616765	2015-02-20 18:29:07.180351
29	10747	Lymphoid Leukemia	2015-02-20 18:26:57.680912	2015-02-20 18:29:07.334859
30	4960	Bone Marrow Cancer	2015-02-20 18:26:57.745697	2015-02-20 18:29:07.5081
31	8997	Polycythemia Vera	2015-02-20 18:26:57.811572	2015-02-20 18:29:07.640765
32	9119	Acute Myeloid Leukemia	2015-02-20 18:26:57.882406	2015-02-20 18:29:07.780212
33	9253	Gastrointestinal Stromal Tumor	2015-02-20 18:26:57.950244	2015-02-20 18:29:07.917197
35	9538	Multiple Myeloma	2015-02-20 18:26:59.673789	2015-02-20 18:29:08.34849
36	9119	Acute Myeloid Leukemia	2015-02-20 18:27:00.486965	2015-02-20 18:29:08.494221
37	3068	Glioblastoma Multiforme	2015-02-20 18:27:01.120287	2015-02-20 18:29:08.682926
38	9253	Gastrointestinal Stromal Tumor	2015-02-20 18:27:04.812957	2015-02-20 18:29:08.821791
39	9119	Acute Myeloid Leukemia	2015-02-20 18:27:06.783882	2015-02-20 18:29:08.959667
40	3973	Thyroid Medullary Carcinoma	2015-02-20 18:27:07.158878	2015-02-20 18:29:09.113395
41	6477	Invasive Bladder Transitional Cell Carcinoma	2015-02-20 18:27:08.422403	2015-02-20 18:29:09.330381
42	4007	Bladder Carcinoma	2015-02-20 18:27:08.568066	2015-02-20 18:29:09.538537
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 42, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Imatinib	\N	2015-02-20 18:26:43.01726	2015-02-20 18:26:43.01726
2	Arsenic Trioxide	\N	2015-02-20 18:26:43.107988	2015-02-20 18:26:43.107988
3	N/A	\N	2015-02-20 18:26:43.1829	2015-02-20 18:26:43.1829
4	Nilotinib	\N	2015-02-20 18:26:43.414682	2015-02-20 18:26:43.414682
5	Dasatinib	\N	2015-02-20 18:26:43.469803	2015-02-20 18:26:43.469803
6	MK-2206	\N	2015-02-20 18:26:43.846235	2015-02-20 18:26:43.846235
7	Crizotinib	\N	2015-02-20 18:26:43.931491	2015-02-20 18:26:43.931491
8	CH5424802	\N	2015-02-20 18:26:44.15756	2015-02-20 18:26:44.15756
9	TAE684	\N	2015-02-20 18:26:44.353897	2015-02-20 18:26:44.353897
10	Sorafenib	\N	2015-02-20 18:26:44.765736	2015-02-20 18:26:44.765736
11	Trametinib	\N	2015-02-20 18:26:44.838433	2015-02-20 18:26:44.838433
12	Panitumumab	\N	2015-02-20 18:26:44.954002	2015-02-20 18:26:44.954002
13	Dabrafenib	\N	2015-02-20 18:26:45.107714	2015-02-20 18:26:45.107714
14	Cetuximab	\N	2015-02-20 18:26:45.289195	2015-02-20 18:26:45.289195
15	PD0325901	\N	2015-02-20 18:26:45.459453	2015-02-20 18:26:45.459453
16	PLX4720	\N	2015-02-20 18:26:45.53754	2015-02-20 18:26:45.53754
17	Nutlin-3	\N	2015-02-20 18:26:45.648299	2015-02-20 18:26:45.648299
18	Capecitabine	\N	2015-02-20 18:26:45.710155	2015-02-20 18:26:45.710155
19	Vemurafenib	\N	2015-02-20 18:26:45.767171	2015-02-20 18:26:45.767171
20	Bevacizumab	\N	2015-02-20 18:26:45.818573	2015-02-20 18:26:45.818573
21	Dacarbazine	\N	2015-02-20 18:26:46.207331	2015-02-20 18:26:46.207331
22	Temozolomide	\N	2015-02-20 18:26:46.272852	2015-02-20 18:26:46.272852
23	AZD6244	\N	2015-02-20 18:26:47.067501	2015-02-20 18:26:47.067501
24	Palbociclib (PD-0332991)	\N	2015-02-20 18:26:48.142946	2015-02-20 18:26:48.142946
25	BYL719	\N	2015-02-20 18:26:48.456709	2015-02-20 18:26:48.456709
26	Palbociclib	\N	2015-02-20 18:26:48.516802	2015-02-20 18:26:48.516802
27	GW-2580	\N	2015-02-20 18:26:49.283088	2015-02-20 18:26:49.283088
28	Daunorubicin	\N	2015-02-20 18:26:49.556188	2015-02-20 18:26:49.556188
29	Idarubicin	\N	2015-02-20 18:26:49.627221	2015-02-20 18:26:49.627221
30	Gefitinib	\N	2015-02-20 18:26:51.356838	2015-02-20 18:26:51.356838
31	Erlotinib	\N	2015-02-20 18:26:51.414471	2015-02-20 18:26:51.414471
32	Premetrexed	\N	2015-02-20 18:26:51.901301	2015-02-20 18:26:51.901301
33	Stauroporine	\N	2015-02-20 18:26:51.956672	2015-02-20 18:26:51.956672
34	Neratinib	\N	2015-02-20 18:26:52.174734	2015-02-20 18:26:52.174734
35	Lapatinib	\N	2015-02-20 18:26:52.501725	2015-02-20 18:26:52.501725
36	Tamoxifen	\N	2015-02-20 18:26:53.036385	2015-02-20 18:26:53.036385
37	Fulvestrant	\N	2015-02-20 18:26:53.111949	2015-02-20 18:26:53.111949
38	Ponatinib	\N	2015-02-20 18:26:54.00969	2015-02-20 18:26:54.00969
39	Pazopanib	\N	2015-02-20 18:26:54.108487	2015-02-20 18:26:54.108487
40	SU5614	\N	2015-02-20 18:26:54.404898	2015-02-20 18:26:54.404898
41	ATRA	\N	2015-02-20 18:26:54.472036	2015-02-20 18:26:54.472036
42	CEP701	\N	2015-02-20 18:26:55.275677	2015-02-20 18:26:55.275677
43	AG1296	\N	2015-02-20 18:26:55.343707	2015-02-20 18:26:55.343707
44	Bortezomib	\N	2015-02-20 18:26:56.122186	2015-02-20 18:26:56.122186
45	Fasudil	\N	2015-02-20 18:26:56.171521	2015-02-20 18:26:56.171521
46	TG101348	\N	2015-02-20 18:26:57.613037	2015-02-20 18:26:57.613037
47	Pegylated IFN-2a	\N	2015-02-20 18:26:57.80812	2015-02-20 18:26:57.80812
48	SU11248	\N	2015-02-20 18:26:59.213772	2015-02-20 18:26:59.213772
49	Regorafenib	\N	2015-02-20 18:26:59.357923	2015-02-20 18:26:59.357923
50	NVP-BEZ235	\N	2015-02-20 18:27:00.328934	2015-02-20 18:27:00.328934
51	ARRY-142886	\N	2015-02-20 18:27:00.378071	2015-02-20 18:27:00.378071
52	O(6)-benzylguanine	\N	2015-02-20 18:27:01.245436	2015-02-20 18:27:01.245436
53	Carmustine	\N	2015-02-20 18:27:01.313357	2015-02-20 18:27:01.313357
54	NSC348884	\N	2015-02-20 18:27:02.007285	2015-02-20 18:27:02.007285
55	Valproic Acid	\N	2015-02-20 18:27:02.350852	2015-02-20 18:27:02.350852
56	17-AAG	\N	2015-02-20 18:27:04.376059	2015-02-20 18:27:04.376059
57	Temozolomide	\N	2015-02-20 18:27:04.685007	2015-02-20 18:27:04.685007
58	Crenolanib	\N	2015-02-20 18:27:04.80956	2015-02-20 18:27:04.80956
59	CH5132799	\N	2015-02-20 18:27:05.40249	2015-02-20 18:27:05.40249
60	Rapamycin	\N	2015-02-20 18:27:05.467218	2015-02-20 18:27:05.467218
61	Motesanib	\N	2015-02-20 18:27:07.155183	2015-02-20 18:27:07.155183
62	AZD1480	\N	2015-02-20 18:27:07.372965	2015-02-20 18:27:07.372965
63	Doxorubicin	\N	2015-02-20 18:27:07.629528	2015-02-20 18:27:07.629528
64	Everolimus	\N	2015-02-20 18:27:08.41886	2015-02-20 18:27:08.41886
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 64, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	B	\N	2015-02-20 18:26:43.028983	2015-02-20 18:26:43.028983
2	A	\N	2015-02-20 18:26:43.111483	2015-02-20 18:26:43.111483
3	C	\N	2015-02-20 18:26:43.285198	2015-02-20 18:26:43.285198
4	D	\N	2015-02-20 18:26:47.074338	2015-02-20 18:26:47.074338
\.


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Predictive	2015-02-20 18:26:43.025997	2015-02-20 18:26:43.025997
2	Diagnostic	2015-02-20 18:26:43.190347	2015-02-20 18:26:43.190347
3	Prognostic	2015-02-20 18:26:43.53193	2015-02-20 18:26:43.53193
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-02-20 18:26:29.334388	2015-02-20 18:26:29.334388	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-02-20 18:26:29.589827	2015-02-20 18:26:29.589827	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-02-20 18:26:29.797712	2015-02-20 18:26:29.797712	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-02-20 18:26:30.009272	2015-02-20 18:26:30.009272	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-02-20 18:26:30.222573	2015-02-20 18:26:30.222573	\N
6	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-02-20 18:26:30.431165	2015-02-20 18:26:30.431165	\N
7	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-02-20 18:26:30.637135	2015-02-20 18:26:30.637135	\N
8	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-02-20 18:26:30.836453	2015-02-20 18:26:30.836453	\N
9	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-02-20 18:26:31.045673	2015-02-20 18:26:31.045673	\N
10	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-02-20 18:26:31.246331	2015-02-20 18:26:31.246331	\N
11	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-02-20 18:26:31.450913	2015-02-20 18:26:31.450913	\N
12	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-02-20 18:26:31.662498	2015-02-20 18:26:31.662498	\N
13	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-02-20 18:26:31.907684	2015-02-20 18:26:31.907684	\N
14	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-02-20 18:26:32.112268	2015-02-20 18:26:32.112268	\N
15	5566	PRKACA	"PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity."	protein kinase, cAMP-dependent, catalytic, alpha	2015-02-20 18:26:32.554148	2015-02-20 18:26:32.554148	\N
16	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-02-20 18:26:32.764137	2015-02-20 18:26:32.764137	\N
17	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-02-20 18:26:32.981396	2015-02-20 18:26:32.981396	\N
18	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown senitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-02-20 18:26:33.194586	2015-02-20 18:26:33.194586	\N
19	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-02-20 18:26:33.400676	2015-02-20 18:26:33.400676	\N
20	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-02-20 18:26:33.603805	2015-02-20 18:26:33.603805	\N
21	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to the activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFRs can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-02-20 18:26:33.815652	2015-02-20 18:26:33.815652	\N
22	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-02-20 18:26:34.018198	2015-02-20 18:26:34.018198	\N
23	2624	GATA2	N/A	GATA binding protein 2	2015-02-20 18:26:34.215537	2015-02-20 18:26:34.215537	\N
24	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-02-20 18:26:34.4013	2015-02-20 18:26:34.4013	\N
25	3418	IDH2	N/A	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-02-20 18:26:34.590112	2015-02-20 18:26:34.590112	\N
26	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders.	Janus kinase 2	2015-02-20 18:26:34.793352	2015-02-20 18:26:34.793352	\N
27	3815	KIT	N/A	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-02-20 18:26:34.990948	2015-02-20 18:26:34.990948	\N
28	3845	KRAS	N/A	Kirsten rat sarcoma viral oncogene homolog	2015-02-20 18:26:35.181188	2015-02-20 18:26:35.181188	\N
29	5604	MAP2K1	N/A	mitogen-activated protein kinase kinase 1	2015-02-20 18:26:35.372085	2015-02-20 18:26:35.372085	\N
30	4209	MEF2D	N/A	myocyte enhancer factor 2D	2015-02-20 18:26:35.559309	2015-02-20 18:26:35.559309	\N
31	1436	CSF1R	N/A	colony stimulating factor 1 receptor	2015-02-20 18:26:35.754884	2015-02-20 18:26:35.754884	\N
32	4255	MGMT	N/A	O-6-methylguanine-DNA methyltransferase	2015-02-20 18:26:35.958821	2015-02-20 18:26:35.958821	\N
33	4869	NPM1	AML with mutated NPM1' is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-02-20 18:26:36.15571	2015-02-20 18:26:36.15571	\N
34	4893	NRAS	N/A	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-02-20 18:26:36.359201	2015-02-20 18:26:36.359201	\N
35	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-02-20 18:26:36.548567	2015-02-20 18:26:36.548567	\N
36	5156	PDGFRA	N/A	platelet-derived growth factor receptor, alpha polypeptide	2015-02-20 18:26:36.739964	2015-02-20 18:26:36.739964	\N
37	5371	PML	N/A	promyelocytic leukemia	2015-02-20 18:26:36.931669	2015-02-20 18:26:36.931669	\N
38	5914	RARa	N/A	retinoic acid receptor, alpha	2015-02-20 18:26:37.11777	2015-02-20 18:26:37.11777	\N
39	5728	PTEN	N/A	phosphatase and tensin homolog	2015-02-20 18:26:37.306664	2015-02-20 18:26:37.306664	\N
40	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-02-20 18:26:37.537073	2015-02-20 18:26:37.537073	\N
41	861	RUNX1	N/A	runt-related transcription factor 1	2015-02-20 18:26:37.734516	2015-02-20 18:26:37.734516	\N
42	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-02-20 18:26:38.00414	2015-02-20 18:26:38.00414	\N
43	7157	TP53	N/A	tumor protein p53	2015-02-20 18:26:38.22442	2015-02-20 18:26:38.22442	\N
44	7248	TSC1	N/A	tuberous sclerosis 1	2015-02-20 18:26:38.421752	2015-02-20 18:26:38.421752	\N
45	7249	TSC2	N/A	tuberous sclerosis 2	2015-02-20 18:26:38.612265	2015-02-20 18:26:38.612265	\N
46	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-02-20 18:26:38.823903	2015-02-20 18:26:38.823903	\N
47	7490	WT1	N/A	Wilms tumor 1	2015-02-20 18:26:39.033571	2015-02-20 18:26:39.033571	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-02-20 18:26:29.338518	2015-02-20 18:27:10.960021
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-02-20 18:26:29.344367	2015-02-20 18:27:11.574963
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-02-20 18:26:29.593211	2015-02-20 18:27:11.99111
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-02-20 18:26:29.599193	2015-02-20 18:27:12.374894
5	15676015	\N	Lee et al., 2005, APMIS	2015-02-20 18:26:29.80084	2015-02-20 18:27:12.744014
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-02-20 18:26:29.806527	2015-02-20 18:27:13.151595
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-02-20 18:26:30.012054	2015-02-20 18:27:13.514961
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-02-20 18:26:30.016966	2015-02-20 18:27:13.871786
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-02-20 18:26:30.22571	2015-02-20 18:27:14.288777
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-02-20 18:26:30.230924	2015-02-20 18:27:16.273022
11	12432268	\N	Diehl, Cancer Biol. Ther.	2015-02-20 18:26:30.434331	2015-02-20 18:27:16.689547
12	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-02-20 18:26:30.439799	2015-02-20 18:27:17.11024
13	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-02-20 18:26:31.04885	2015-02-20 18:27:17.567242
14	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-02-20 18:26:31.249463	2015-02-20 18:27:18.049948
15	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-02-20 18:26:31.254724	2015-02-20 18:27:18.467332
16	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-02-20 18:26:31.665553	2015-02-20 18:27:18.886837
17	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-20 18:26:31.670799	2015-02-20 18:27:19.303119
18	19357394	\N	Vardiman et al., 2009, Blood	2015-02-20 18:26:31.911305	2015-02-20 18:27:19.8493
19	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-02-20 18:26:31.916491	2015-02-20 18:27:20.244304
20	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 18:26:31.920623	2015-02-20 18:27:20.662847
21	24578576	\N	Honeyman et al., 2014, Science	2015-02-20 18:26:32.116027	2015-02-20 18:27:21.079207
22	24909179	\N	Moody et al., 2014, Oncogene	2015-02-20 18:26:32.560756	2015-02-20 18:27:21.496199
23	24167195	\N	Kim et al., 2013, Blood	2015-02-20 18:26:32.767507	2015-02-20 18:27:21.915854
24	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-02-20 18:26:32.773311	2015-02-20 18:27:22.331834
25	23953842	\N	Yewale et al., 2013, Biomaterials	2015-02-20 18:26:32.985081	2015-02-20 18:27:22.750711
26	18712184	\N	Charpidou et al., In Vivo	2015-02-20 18:26:32.990776	2015-02-20 18:27:23.167791
27	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-02-20 18:26:33.197753	2015-02-20 18:27:23.537548
28	10878580	\N	Yu et al., 2000, Bioessays	2015-02-20 18:26:33.203241	2015-02-20 18:27:23.897494
29	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-02-20 18:26:33.403828	2015-02-20 18:27:24.31977
30	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-02-20 18:26:33.409131	2015-02-20 18:27:25.157477
31	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-02-20 18:26:33.60733	2015-02-20 18:27:25.573889
32	23558953	\N	Wu et al., 2013, Cancer Discov	2015-02-20 18:26:33.616344	2015-02-20 18:27:26.006404
33	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-02-20 18:26:34.020433	2015-02-20 18:27:26.405533
34	23226219	\N	Chou et al., 2012, PLoS ONE	2015-02-20 18:26:36.159532	2015-02-20 18:27:26.826995
35	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-20 18:26:37.540494	2015-02-20 18:27:27.242429
36	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-02-20 18:26:37.546082	2015-02-20 18:27:27.668866
37	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-20 18:26:38.008023	2015-02-20 18:27:28.186292
38	24136165	\N	Cazzola et al., 2013, Blood	2015-02-20 18:26:38.01293	2015-02-20 18:27:28.67703
39	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-02-20 18:26:38.826557	2015-02-20 18:27:29.121002
40	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-02-20 18:26:38.831918	2015-02-20 18:27:29.542155
41	12476305		Nimmanapalli et al., 2002, Oncogene	2015-02-20 18:26:43.020464	2015-02-20 18:27:30.061258
42	20537386		An et al., 2010, Leuk. Res.	2015-02-20 18:26:43.184746	2015-02-20 18:27:30.478787
43	25212276		Schmidt et al., 2014, Leukemia	2015-02-20 18:26:43.2816	2015-02-20 18:27:31.002722
44	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-02-20 18:26:43.596854	2015-02-20 18:27:31.412122
45	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-02-20 18:26:43.848138	2015-02-20 18:27:31.811704
46	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-02-20 18:26:43.933231	2015-02-20 18:27:32.163506
47	21030459		Sasaki et al., 2010, Cancer Res.	2015-02-20 18:26:44.088559	2015-02-20 18:27:32.568949
48	21575866		Sakamoto et al., 2011, Cancer Cell	2015-02-20 18:26:44.159799	2015-02-20 18:27:32.991294
49	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-02-20 18:26:44.224529	2015-02-20 18:27:33.340721
50	22072639		Bresler et al., 2011, Sci Transl Med	2015-02-20 18:26:44.293499	2015-02-20 18:27:33.7369
51	18923525		George et al., 2008, Nature	2015-02-20 18:26:44.355662	2015-02-20 18:27:34.240555
52	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-02-20 18:26:44.767816	2015-02-20 18:27:34.657249
53	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-02-20 18:26:44.955833	2015-02-20 18:27:35.07509
54	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-02-20 18:26:45.043825	2015-02-20 18:27:35.492312
55	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-02-20 18:26:45.109555	2015-02-20 18:27:35.911726
56	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-02-20 18:26:45.218332	2015-02-20 18:27:36.80098
57	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-02-20 18:26:45.290937	2015-02-20 18:27:37.464942
58	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-02-20 18:26:45.46129	2015-02-20 18:27:37.958485
59	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-02-20 18:26:45.595602	2015-02-20 18:27:38.561131
60	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-02-20 18:26:45.712027	2015-02-20 18:27:39.079327
61	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-02-20 18:26:45.932236	2015-02-20 18:27:39.536582
62	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-02-20 18:26:45.996035	2015-02-20 18:27:39.970671
63	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-02-20 18:26:46.086254	2015-02-20 18:27:40.463436
64	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-02-20 18:26:46.209321	2015-02-20 18:27:41.282051
65	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-02-20 18:26:46.338655	2015-02-20 18:27:42.133696
66	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-02-20 18:26:46.45686	2015-02-20 18:27:42.626926
67	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-02-20 18:26:46.523346	2015-02-20 18:27:44.892663
68	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-02-20 18:26:46.60513	2015-02-20 18:27:45.327164
69	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-02-20 18:26:46.668253	2015-02-20 18:27:45.732771
70	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-02-20 18:26:46.73409	2015-02-20 18:27:46.254465
71	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-02-20 18:26:46.79334	2015-02-20 18:27:46.674177
72	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-02-20 18:26:46.924203	2015-02-20 18:27:47.088901
73	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-02-20 18:26:47.070426	2015-02-20 18:27:47.509016
74	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-02-20 18:26:47.157533	2015-02-20 18:27:47.924724
75	17070615		Gautschi et al., 2007, Lung Cancer	2015-02-20 18:26:47.352248	2015-02-20 18:27:48.444354
76	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-02-20 18:26:47.5357	2015-02-20 18:27:48.861231
77	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-02-20 18:26:47.676151	2015-02-20 18:27:49.288911
78	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-02-20 18:26:47.744901	2015-02-20 18:27:49.698601
79	10547574		Takano et al., 1999, J. Pathol.	2015-02-20 18:26:47.816708	2015-02-20 18:27:50.11667
80	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-02-20 18:26:48.059871	2015-02-20 18:27:50.533628
81	23079656		Sawai et al., 2012, Cancer Cell	2015-02-20 18:26:48.144805	2015-02-20 18:27:50.952744
82	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-02-20 18:26:48.208958	2015-02-20 18:27:51.376432
83	10224221		Donnellan et al., 1999, FASEB J.	2015-02-20 18:26:48.26522	2015-02-20 18:27:51.958329
84	25002028		Vora et al., 2014, Cancer Cell	2015-02-20 18:26:48.458568	2015-02-20 18:27:52.312178
85	23898052		Logan et al., 2013, Anticancer Res.	2015-02-20 18:26:48.518671	2015-02-20 18:27:52.734056
86	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-20 18:26:48.720455	2015-02-20 18:27:53.145523
87	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-02-20 18:26:48.836353	2015-02-20 18:27:53.561251
88	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-02-20 18:26:48.894326	2015-02-20 18:27:54.086204
89	14726504	Retrospective study	Frhling et al., 2004, J. Clin. Oncol.	2015-02-20 18:26:49.037087	2015-02-20 18:27:54.500829
90	24186003		Lilljebjrn et al., 2014, Leukemia	2015-02-20 18:26:49.228647	2015-02-20 18:27:55.023778
91	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-02-20 18:26:49.339197	2015-02-20 18:27:55.444225
92	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-02-20 18:26:49.402447	2015-02-20 18:27:55.859032
93	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-02-20 18:26:49.487902	2015-02-20 18:27:56.279019
94	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-02-20 18:26:49.756725	2015-02-20 18:27:56.696819
95	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-02-20 18:26:49.81152	2015-02-20 18:27:57.113455
96	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-02-20 18:26:49.967103	2015-02-20 18:27:57.532048
97	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-02-20 18:26:50.335572	2015-02-20 18:27:57.951874
98	24736073		Lim et al., 2014, J Thorac Oncol	2015-02-20 18:26:51.359024	2015-02-20 18:27:58.342356
99	24457318		Fukihara et al., 2014, Oncology	2015-02-20 18:26:51.498847	2015-02-20 18:27:58.78458
100	24585406		Li et al., 2014, Ann. Surg. Oncol.	2015-02-20 18:26:51.570119	2015-02-20 18:27:59.20229
101	24662454		Douillard et al., 2014, J Thorac Oncol	2015-02-20 18:26:51.631805	2015-02-20 18:27:59.622085
102	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-02-20 18:26:51.693251	2015-02-20 18:28:00.035938
103	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-02-20 18:26:51.768268	2015-02-20 18:28:00.560419
104	24636847		Li et al., 2014, Lung Cancer	2015-02-20 18:26:51.845335	2015-02-20 18:28:00.978005
105	24658966		Ai et al., 2014, Amino Acids	2015-02-20 18:26:51.958574	2015-02-20 18:28:01.39672
106	24729716		Li et al., 2014, Onco Targets Ther	2015-02-20 18:26:52.049991	2015-02-20 18:28:01.814739
107	24623981		Ding et al., 2014, Onco Targets Ther	2015-02-20 18:26:52.116496	2015-02-20 18:28:02.231761
108	23220880		Bose et al., 2013, Cancer Discov	2015-02-20 18:26:52.176499	2015-02-20 18:28:02.757114
109	24185510		Robinson et al., 2013, Nat. Genet.	2015-02-20 18:26:53.038206	2015-02-20 18:28:03.172987
110	24185512		Toy et al., 2013, Nat. Genet.	2015-02-20 18:26:53.1751	2015-02-20 18:28:03.804879
111	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-02-20 18:26:54.011637	2015-02-20 18:28:04.273846
112	17957027	Retrospective study	Gale et al., 2008, Blood	2015-02-20 18:26:54.344237	2015-02-20 18:28:04.737778
113	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-02-20 18:26:54.407627	2015-02-20 18:28:05.160104
114	19965647		Burnett et al., 2010, Blood	2015-02-20 18:26:54.474221	2015-02-20 18:28:05.575224
115	21537333		Schnittger et al., 2011, Leukemia	2015-02-20 18:26:54.689803	2015-02-20 18:28:05.991111
116	24801015		Port et al., 2014, Ann. Hematol.	2015-02-20 18:26:55.066225	2015-02-20 18:28:06.40896
117	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-02-20 18:26:55.149276	2015-02-20 18:28:06.827285
118	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-02-20 18:26:55.214495	2015-02-20 18:28:07.245367
119	14726387	Clinical trial	Smith et al., 2004, Blood	2015-02-20 18:26:55.277981	2015-02-20 18:28:07.668513
120	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-02-20 18:26:55.345815	2015-02-20 18:28:08.080142
121	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-02-20 18:26:55.403889	2015-02-20 18:28:08.462684
122	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-02-20 18:26:55.663676	2015-02-20 18:28:08.915603
123	22368270	Clinical trial	Man et al., 2012, Blood	2015-02-20 18:26:55.788078	2015-02-20 18:28:09.334288
124	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-02-20 18:26:56.032425	2015-02-20 18:28:09.753052
125	22624710		Barbacid, 2012, Cancer Cell	2015-02-20 18:26:56.123961	2015-02-20 18:28:10.168337
126	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-02-20 18:26:56.23604	2015-02-20 18:28:10.587184
127	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-02-20 18:26:56.304005	2015-02-20 18:28:11.01384
128	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-02-20 18:26:56.368093	2015-02-20 18:28:11.422418
129	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-02-20 18:26:56.429532	2015-02-20 18:28:11.842942
130	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-02-20 18:26:56.491053	2015-02-20 18:28:12.36493
131	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-02-20 18:26:56.579978	2015-02-20 18:28:12.781661
132	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-02-20 18:26:56.640804	2015-02-20 18:28:13.198744
133	21596855		Green et al., 2011, Blood	2015-02-20 18:26:56.891666	2015-02-20 18:28:13.527266
134	20421455	Retrospective study	Thol et al., 2010, Blood	2015-02-20 18:26:56.968351	2015-02-20 18:28:14.121285
135	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-02-20 18:26:57.050154	2015-02-20 18:28:14.556544
136	22033490		Patnaik et al., 2012, Leukemia	2015-02-20 18:26:57.115414	2015-02-20 18:28:15.078294
137	21997850		Lin et al., 2012, Ann. Hematol.	2015-02-20 18:26:57.176241	2015-02-20 18:28:15.600308
138	16081687	Retrospective study	Levine et al., 2005, Blood	2015-02-20 18:26:57.407594	2015-02-20 18:28:16.018301
139	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-02-20 18:26:57.614958	2015-02-20 18:28:16.438414
140	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-02-20 18:26:57.743656	2015-02-20 18:28:16.959672
141	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-02-20 18:26:57.809891	2015-02-20 18:28:17.481138
142	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-02-20 18:26:57.880498	2015-02-20 18:28:17.898829
143	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-02-20 18:26:57.948247	2015-02-20 18:28:18.340028
144	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-02-20 18:26:58.013692	2015-02-20 18:28:18.733886
145	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-02-20 18:26:58.072696	2015-02-20 18:28:19.152403
146	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-02-20 18:26:58.188581	2015-02-20 18:28:19.575513
147	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-02-20 18:26:58.253708	2015-02-20 18:28:19.988442
148	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-02-20 18:26:58.700464	2015-02-20 18:28:20.375326
149	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-02-20 18:26:58.951616	2015-02-20 18:28:20.778346
150	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-02-20 18:26:59.141574	2015-02-20 18:28:21.1413
151	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-02-20 18:26:59.216839	2015-02-20 18:28:22.494176
152	16618717	Retrospective study	Livre et al., 2006, Cancer Res.	2015-02-20 18:26:59.299836	2015-02-20 18:28:22.912349
153	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-02-20 18:26:59.360023	2015-02-20 18:28:23.244347
154	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-02-20 18:26:59.424556	2015-02-20 18:28:23.643382
155	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-02-20 18:26:59.539583	2015-02-20 18:28:24.063226
156	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-02-20 18:26:59.610459	2015-02-20 18:28:24.48218
157	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-02-20 18:26:59.671823	2015-02-20 18:28:25.314039
158	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-02-20 18:26:59.739867	2015-02-20 18:28:25.740148
159	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-02-20 18:26:59.80688	2015-02-20 18:28:26.150319
160	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-02-20 18:26:59.86735	2015-02-20 18:28:26.57406
161	18202412	In vitro study	Livre et al., 2008, J. Clin. Oncol.	2015-02-20 18:26:59.929144	2015-02-20 18:28:26.921958
162	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-02-20 18:27:00.121928	2015-02-20 18:28:27.408257
163	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-02-20 18:27:00.255332	2015-02-20 18:28:27.822583
164	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-02-20 18:27:00.330931	2015-02-20 18:28:28.243816
165	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-02-20 18:27:00.594083	2015-02-20 18:28:28.659626
166	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 18:27:00.88666	2015-02-20 18:28:29.180095
167	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 18:27:00.952327	2015-02-20 18:28:29.598073
168	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-02-20 18:27:01.117757	2015-02-20 18:28:30.019229
169	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-02-20 18:27:01.184466	2015-02-20 18:28:30.438847
170	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-02-20 18:27:01.441429	2015-02-20 18:28:30.956625
171	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-02-20 18:27:01.505701	2015-02-20 18:28:31.378762
172	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-02-20 18:27:01.684685	2015-02-20 18:28:31.79981
173	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-02-20 18:27:01.873776	2015-02-20 18:28:32.208533
174	21719597	In vitro study	Balusu et al., 2011, Blood	2015-02-20 18:27:02.009708	2015-02-20 18:28:32.730646
175	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-02-20 18:27:02.282896	2015-02-20 18:28:33.151753
176	24797300		Tassara et al., 2014, Blood	2015-02-20 18:27:02.352635	2015-02-20 18:28:33.567646
177	24927407		Ehninger et al., 2014, Blood Cancer J	2015-02-20 18:27:02.413719	2015-02-20 18:28:34.089988
178	16051734	Retrospective study	Dhner et al., 2005, Blood	2015-02-20 18:27:02.597366	2015-02-20 18:28:34.54849
179	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-02-20 18:27:02.659244	2015-02-20 18:28:35.02906
180	24855211		Linch et al., 2014, Blood	2015-02-20 18:27:02.725732	2015-02-20 18:28:35.444122
181	19047294	Retrospective study	Bchner et al., 2009, J. Clin. Oncol.	2015-02-20 18:27:02.913808	2015-02-20 18:28:35.831533
182	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-02-20 18:27:02.982868	2015-02-20 18:28:36.28152
183	24859829		Tian et al., 2014, Int. J. Hematol.	2015-02-20 18:27:03.406696	2015-02-20 18:28:36.697102
184	19357394		Vardiman et al., 2009, Blood	2015-02-20 18:27:03.528236	2015-02-20 18:28:37.115582
185	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-02-20 18:27:03.84719	2015-02-20 18:28:37.469293
186	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-02-20 18:27:03.917976	2015-02-20 18:28:37.839125
187	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-02-20 18:27:03.988606	2015-02-20 18:28:38.270907
188	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-02-20 18:27:04.055572	2015-02-20 18:28:38.996893
189	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-02-20 18:27:04.309336	2015-02-20 18:28:39.728007
190	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-02-20 18:27:04.378424	2015-02-20 18:28:40.149487
191	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-02-20 18:27:04.56121	2015-02-20 18:28:40.564572
192	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-02-20 18:27:04.620855	2015-02-20 18:28:41.088434
193	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-02-20 18:27:04.686874	2015-02-20 18:28:41.507346
194	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-02-20 18:27:04.811331	2015-02-20 18:28:42.024673
195	15146165		Lasota et al., 2004, Lab. Invest.	2015-02-20 18:27:04.880055	2015-02-20 18:28:42.442883
196	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-02-20 18:27:05.016722	2015-02-20 18:28:42.821503
197	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-02-20 18:27:05.404786	2015-02-20 18:28:44.401319
198	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 18:27:05.46918	2015-02-20 18:28:44.746135
199	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-02-20 18:27:05.594565	2015-02-20 18:28:45.162386
200	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-02-20 18:27:05.771088	2015-02-20 18:28:45.684244
201	8674046		Yoshida et al., 1996, Cancer Res.	2015-02-20 18:27:06.78078	2015-02-20 18:28:46.204318
202	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-02-20 18:27:06.955177	2015-02-20 18:28:46.832644
203	22479427		Carico et al., 2012, PLoS ONE	2015-02-20 18:27:07.025635	2015-02-20 18:28:47.253536
204	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-02-20 18:27:07.157079	2015-02-20 18:28:47.668818
205	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-20 18:27:07.301557	2015-02-20 18:28:48.190003
206	23056499		Couto et al., 2012, PLoS ONE	2015-02-20 18:27:07.375127	2015-02-20 18:28:48.925288
207	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-02-20 18:27:07.432965	2015-02-20 18:28:49.441822
208	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-20 18:27:07.506819	2015-02-20 18:28:49.870495
209	22698404		Jackson et al., 2012, Cancer Cell	2015-02-20 18:27:07.631462	2015-02-20 18:28:50.384585
210	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-02-20 18:27:07.704313	2015-02-20 18:28:50.800317
211	9569050		Berns et al., 1998, Br. J. Cancer	2015-02-20 18:27:07.76429	2015-02-20 18:28:51.218108
212	22923433		Iyer et al., 2012, Science	2015-02-20 18:27:08.420726	2015-02-20 18:28:51.636391
213	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-02-20 18:27:08.497192	2015-02-20 18:28:52.872384
214	22923433	Retrospective study	Iyer et al., 2012, Science	2015-02-20 18:27:08.566201	2015-02-20 18:28:53.30731
215	23029227		Qian et al., 2012, PLoS ONE	2015-02-20 18:27:08.7073	2015-02-20 18:28:53.831026
216	23861105		Wu et al., 2013, Am. J. Hematol.	2015-02-20 18:27:08.904439	2015-02-20 18:28:55.426895
217	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-02-20 18:27:09.21779	2015-02-20 18:28:55.813348
218	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-02-20 18:27:09.284894	2015-02-20 18:28:56.858262
219	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-02-20 18:27:09.528431	2015-02-20 18:28:58.392725
220	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-02-20 18:27:09.652915	2015-02-20 18:28:58.878944
\.


--
-- Data for Name: variant_origins; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_origins (id, origin) FROM stdin;
1	Somatic
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-02-20 18:26:39.091875	2015-02-20 18:26:39.091875
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-20 18:26:39.128301	2015-02-20 18:26:39.128301
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-20 18:26:39.183819	2015-02-20 18:26:39.183819
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-02-20 18:26:39.210954	2015-02-20 18:26:39.210954
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-02-20 18:26:39.242381	2015-02-20 18:26:39.242381
6	1	C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-20 18:26:39.267972	2015-02-20 18:26:39.267972
7	1	L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-20 18:26:39.299017	2015-02-20 18:26:39.299017
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-02-20 18:26:39.3253	2015-02-20 18:26:39.3253
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-02-20 18:26:39.350393	2015-02-20 18:26:39.350393
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-02-20 18:26:39.373109	2015-02-20 18:26:39.373109
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 18:26:39.401171	2015-02-20 18:26:39.401171
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-02-20 18:26:39.423543	2015-02-20 18:26:39.423543
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-02-20 18:26:39.458013	2015-02-20 18:26:39.458013
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-02-20 18:26:39.483145	2015-02-20 18:26:39.483145
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 18:26:39.514836	2015-02-20 18:26:39.514836
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 18:26:39.542165	2015-02-20 18:26:39.542165
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-02-20 18:26:39.571532	2015-02-20 18:26:39.571532
18	6	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-02-20 18:26:39.599507	2015-02-20 18:26:39.599507
19	6	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-02-20 18:26:39.625319	2015-02-20 18:26:39.625319
75	27	EXON 9	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-02-20 18:26:41.191097	2015-02-20 18:26:41.191097
97	34	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-20 18:26:41.838156	2015-02-20 18:26:41.838156
20	6	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 18:26:39.651584	2015-02-20 18:26:39.651584
21	7	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 18:26:39.684769	2015-02-20 18:26:39.684769
22	7	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 18:26:39.712563	2015-02-20 18:26:39.712563
23	8	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-02-20 18:26:39.738897	2015-02-20 18:26:39.738897
24	9	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-02-20 18:26:39.763942	2015-02-20 18:26:39.763942
25	11	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-20 18:26:39.788858	2015-02-20 18:26:39.788858
26	10	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-20 18:26:39.813402	2015-02-20 18:26:39.813402
27	12	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-02-20 18:26:39.839553	2015-02-20 18:26:39.839553
28	13	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-20 18:26:39.888855	2015-02-20 18:26:39.888855
29	13	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-20 18:26:39.913022	2015-02-20 18:26:39.913022
30	31	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-20 18:26:39.936149	2015-02-20 18:26:39.936149
31	14	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-20 18:26:39.964908	2015-02-20 18:26:39.964908
32	16	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-20 18:26:39.990449	2015-02-20 18:26:39.990449
76	27	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-02-20 18:26:41.226069	2015-02-20 18:26:41.226069
33	17	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-02-20 18:26:40.017638	2015-02-20 18:26:40.017638
34	17	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-02-20 18:26:40.043615	2015-02-20 18:26:40.043615
35	18	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.075966	2015-02-20 18:26:40.075966
36	18	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.109355	2015-02-20 18:26:40.109355
37	18	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.137203	2015-02-20 18:26:40.137203
38	18	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.163689	2015-02-20 18:26:40.163689
39	18	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-02-20 18:26:40.188141	2015-02-20 18:26:40.188141
40	18	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.211784	2015-02-20 18:26:40.211784
41	18	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.240795	2015-02-20 18:26:40.240795
42	18	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.265216	2015-02-20 18:26:40.265216
43	18	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.292136	2015-02-20 18:26:40.292136
44	18	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.32527	2015-02-20 18:26:40.32527
45	18	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 18:26:40.349485	2015-02-20 18:26:40.349485
46	19	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 18:26:40.37317	2015-02-20 18:26:40.37317
47	19	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 18:26:40.397825	2015-02-20 18:26:40.397825
48	19	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 18:26:40.42705	2015-02-20 18:26:40.42705
49	19	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 18:26:40.455986	2015-02-20 18:26:40.455986
50	19	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 18:26:40.486	2015-02-20 18:26:40.486
51	20	FGFR2-AFF3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.516971	2015-02-20 18:26:40.516971
52	20	FGFR2-BICC1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.541316	2015-02-20 18:26:40.541316
53	20	FGFR2-CASP7	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.586839	2015-02-20 18:26:40.586839
54	20	FGFR2-CCDC6	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.611803	2015-02-20 18:26:40.611803
55	20	FGFR2-KIAA1967	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.636839	2015-02-20 18:26:40.636839
56	20	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.664819	2015-02-20 18:26:40.664819
57	20	FGFR2-OFD1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.692259	2015-02-20 18:26:40.692259
58	20	FGFR2-SLC45A3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.718259	2015-02-20 18:26:40.718259
59	21	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 18:26:40.746123	2015-02-20 18:26:40.746123
60	22	ITD	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-02-20 18:26:40.774875	2015-02-20 18:26:40.774875
61	22	TKD	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-02-20 18:26:40.804884	2015-02-20 18:26:40.804884
62	23	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-02-20 18:26:40.835595	2015-02-20 18:26:40.835595
63	24	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 18:26:40.860975	2015-02-20 18:26:40.860975
64	24	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 18:26:40.887857	2015-02-20 18:26:40.887857
65	24	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 18:26:40.916036	2015-02-20 18:26:40.916036
66	24	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 18:26:40.943991	2015-02-20 18:26:40.943991
67	25	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-02-20 18:26:40.968871	2015-02-20 18:26:40.968871
68	25	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-02-20 18:26:40.994488	2015-02-20 18:26:40.994488
69	26	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-02-20 18:26:41.025443	2015-02-20 18:26:41.025443
70	27	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-02-20 18:26:41.051212	2015-02-20 18:26:41.051212
71	27	EXON 11	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-20 18:26:41.078031	2015-02-20 18:26:41.078031
72	27	EXON 13	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-02-20 18:26:41.107289	2015-02-20 18:26:41.107289
73	27	EXON 14	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-02-20 18:26:41.137436	2015-02-20 18:26:41.137436
74	27	EXON 17	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-02-20 18:26:41.163055	2015-02-20 18:26:41.163055
77	27	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-02-20 18:26:41.273248	2015-02-20 18:26:41.273248
78	28	EXON 1	A study by Livre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-02-20 18:26:41.303487	2015-02-20 18:26:41.303487
79	28	EXON 2	In a study by Pan et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-02-20 18:26:41.335812	2015-02-20 18:26:41.335812
80	28	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 18:26:41.36149	2015-02-20 18:26:41.36149
81	28	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 18:26:41.388667	2015-02-20 18:26:41.388667
82	28	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 18:26:41.423157	2015-02-20 18:26:41.423157
83	28	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 18:26:41.449886	2015-02-20 18:26:41.449886
84	28	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 18:26:41.482342	2015-02-20 18:26:41.482342
85	28	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 18:26:41.511159	2015-02-20 18:26:41.511159
86	29	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-02-20 18:26:41.543355	2015-02-20 18:26:41.543355
87	29	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-02-20 18:26:41.573394	2015-02-20 18:26:41.573394
88	30	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-20 18:26:41.602391	2015-02-20 18:26:41.602391
89	32	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-02-20 18:26:41.626309	2015-02-20 18:26:41.626309
90	33	EXON 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-02-20 18:26:41.653097	2015-02-20 18:26:41.653097
91	33	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-02-20 18:26:41.682869	2015-02-20 18:26:41.682869
92	34	EXON 1	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-02-20 18:26:41.708076	2015-02-20 18:26:41.708076
93	34	EXON 2	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-02-20 18:26:41.735245	2015-02-20 18:26:41.735245
94	34	EXON 3	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-20 18:26:41.760279	2015-02-20 18:26:41.760279
95	34	EXON 4	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-20 18:26:41.787737	2015-02-20 18:26:41.787737
96	34	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-20 18:26:41.815538	2015-02-20 18:26:41.815538
98	34	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 18:26:41.862356	2015-02-20 18:26:41.862356
99	34	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 18:26:41.888614	2015-02-20 18:26:41.888614
100	34	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 18:26:41.939764	2015-02-20 18:26:41.939764
101	36	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 18:26:41.967398	2015-02-20 18:26:41.967398
102	36	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 18:26:41.992956	2015-02-20 18:26:41.992956
103	36	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 18:26:42.025827	2015-02-20 18:26:42.025827
104	36	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 18:26:42.053416	2015-02-20 18:26:42.053416
105	36	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 18:26:42.077057	2015-02-20 18:26:42.077057
106	35	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 18:26:42.104044	2015-02-20 18:26:42.104044
107	35	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 18:26:42.131549	2015-02-20 18:26:42.131549
108	35	EXON 20	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-20 18:26:42.161912	2015-02-20 18:26:42.161912
109	35	EXON 9	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 18:26:42.189417	2015-02-20 18:26:42.189417
110	35	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-20 18:26:42.218419	2015-02-20 18:26:42.218419
111	37	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-02-20 18:26:42.243843	2015-02-20 18:26:42.243843
112	15	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-20 18:26:42.275335	2015-02-20 18:26:42.275335
113	39	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-02-20 18:26:42.30405	2015-02-20 18:26:42.30405
114	38	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-02-20 18:26:42.333585	2015-02-20 18:26:42.333585
115	40	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-02-20 18:26:42.363119	2015-02-20 18:26:42.363119
116	40	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-02-20 18:26:42.38902	2015-02-20 18:26:42.38902
117	42	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-02-20 18:26:42.415375	2015-02-20 18:26:42.415375
118	42	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-02-20 18:26:42.439601	2015-02-20 18:26:42.439601
119	43	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-02-20 18:26:42.465027	2015-02-20 18:26:42.465027
120	43	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-02-20 18:26:42.492419	2015-02-20 18:26:42.492419
121	43	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-02-20 18:26:42.519549	2015-02-20 18:26:42.519549
122	43	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 18:26:42.549803	2015-02-20 18:26:42.549803
123	43	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 18:26:42.597896	2015-02-20 18:26:42.597896
124	43	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 18:26:42.625811	2015-02-20 18:26:42.625811
125	43	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 18:26:42.65185	2015-02-20 18:26:42.65185
126	43	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 18:26:42.684907	2015-02-20 18:26:42.684907
127	44	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 18:26:42.71097	2015-02-20 18:26:42.71097
128	44	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 18:26:42.734455	2015-02-20 18:26:42.734455
129	45	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 18:26:42.758413	2015-02-20 18:26:42.758413
130	46	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-20 18:26:42.783023	2015-02-20 18:26:42.783023
131	46	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-20 18:26:42.80883	2015-02-20 18:26:42.80883
132	47	EXON 7	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-20 18:26:42.836669	2015-02-20 18:26:42.836669
133	47	EXON 9	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-20 18:26:42.870015	2015-02-20 18:26:42.870015
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_origin_id) FROM stdin;
1	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	Resistance or Non-Response	Supports	1	1	1	1	41	1	2015-02-20 18:26:43.034001	2015-02-20 18:26:43.034001	3	accepted	\N	\N	1
2	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	Sensitivity	Supports	1	2	2	1	41	1	2015-02-20 18:26:43.113833	2015-02-20 18:26:43.113833	5	accepted	\N	\N	1
3	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	Positive	Supports	2	2	3	2	42	1	2015-02-20 18:26:43.192903	2015-02-20 18:26:43.192903	4	accepted	\N	\N	1
4	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	Positive	Does Not Support	2	3	3	2	43	1	2015-02-20 18:26:43.287718	2015-02-20 18:26:43.287718	2	accepted	\N	\N	1
5	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.	Sensitivity	Supports	1	2	1	2	42	1	2015-02-20 18:26:43.35074	2015-02-20 18:26:43.35074	5	accepted	\N	\N	1
6	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	Sensitivity	Supports	1	1	4	2	42	1	2015-02-20 18:26:43.417547	2015-02-20 18:26:43.417547	4	accepted	\N	\N	1
7	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	Sensitivity	Supports	1	1	5	2	42	1	2015-02-20 18:26:43.474245	2015-02-20 18:26:43.474245	4	accepted	\N	\N	1
8	BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens."	Better Outcome	Supports	3	2	1	2	42	1	2015-02-20 18:26:43.534373	2015-02-20 18:26:43.534373	5	accepted	\N	\N	1
9	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	1	3	1	2	44	3	2015-02-20 18:26:43.599399	2015-02-20 18:26:43.599399	4	accepted	\N	\N	1
10	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	Resistance or Non-Response	Supports	1	1	1	2	42	2	2015-02-20 18:26:43.704469	2015-02-20 18:26:43.704469	4	accepted	\N	\N	1
11	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	1	3	1	2	44	2	2015-02-20 18:26:43.776728	2015-02-20 18:26:43.776728	4	accepted	\N	\N	1
12	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	Sensitivity	Does Not Support	1	1	6	3	45	4	2015-02-20 18:26:43.873925	2015-02-20 18:26:43.873925	3	accepted	\N	\N	1
13	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	1	1	7	4	46	6	2015-02-20 18:26:43.938676	2015-02-20 18:26:43.938676	4	accepted	\N	\N	1
14	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	Sensitivity	Supports	1	1	7	4	46	5	2015-02-20 18:26:44.015258	2015-02-20 18:26:44.015258	4	accepted	\N	\N	1
15	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	Resistance or Non-Response	Supports	1	3	7	5	47	8	2015-02-20 18:26:44.092914	2015-02-20 18:26:44.092914	3	accepted	\N	\N	1
16	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	Sensitivity	Supports	1	3	8	6	48	8	2015-02-20 18:26:44.164599	2015-02-20 18:26:44.164599	3	accepted	\N	\N	1
17	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	Sensitivity	Supports	1	3	7	6	49	8	2015-02-20 18:26:44.227492	2015-02-20 18:26:44.227492	3	accepted	\N	\N	1
18	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	Resistance or Non-Response	Supports	1	3	7	6	50	8	2015-02-20 18:26:44.296279	2015-02-20 18:26:44.296279	3	accepted	\N	\N	1
19	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	Sensitivity	Supports	1	3	9	6	51	8	2015-02-20 18:26:44.35838	2015-02-20 18:26:44.35838	4	accepted	\N	\N	1
20	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	Resistance or Non-Response	Supports	1	3	7	4	47	8	2015-02-20 18:26:44.419964	2015-02-20 18:26:44.419964	3	accepted	\N	\N	1
21	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	Sensitivity	Supports	1	3	8	4	48	7	2015-02-20 18:26:44.505149	2015-02-20 18:26:44.505149	3	accepted	\N	\N	1
22	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	1	1	7	4	46	7	2015-02-20 18:26:44.566735	2015-02-20 18:26:44.566735	4	accepted	\N	\N	1
23	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.	Sensitivity	Supports	1	3	9	6	51	9	2015-02-20 18:26:44.632845	2015-02-20 18:26:44.632845	3	accepted	\N	\N	1
24	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.	N/A	Does Not Support	1	3	9	6	51	9	2015-02-20 18:26:44.696318	2015-02-20 18:26:44.696318	3	accepted	\N	\N	1
25	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	Sensitivity	Supports	1	1	10	4	52	10	2015-02-20 18:26:44.77064	2015-02-20 18:26:44.77064	2	accepted	\N	\N	1
26	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	Sensitivity	Supports	1	3	11	4	52	10	2015-02-20 18:26:44.842503	2015-02-20 18:26:44.842503	3	accepted	\N	\N	1
27	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	Sensitivity	Supports	1	3	10	4	52	10	2015-02-20 18:26:44.899688	2015-02-20 18:26:44.899688	3	accepted	\N	\N	1
28	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	Resistance or Non-Response	Supports	1	1	12	7	53	17	2015-02-20 18:26:44.96031	2015-02-20 18:26:44.96031	3	accepted	\N	\N	1
29	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma	Positive	Supports	2	1	3	8	54	17	2015-02-20 18:26:45.049436	2015-02-20 18:26:45.049436	4	accepted	\N	\N	1
30	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	1	1	13	8	55	17	2015-02-20 18:26:45.111998	2015-02-20 18:26:45.111998	3	accepted	\N	\N	1
31	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	1	1	11	8	55	17	2015-02-20 18:26:45.159706	2015-02-20 18:26:45.159706	3	accepted	\N	\N	1
32	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	13	8	56	11	2015-02-20 18:26:45.221205	2015-02-20 18:26:45.221205	5	accepted	\N	\N	1
33	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	3	14	9	57	12	2015-02-20 18:26:45.296112	2015-02-20 18:26:45.296112	3	accepted	\N	\N	1
34	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	3	12	9	57	12	2015-02-20 18:26:45.34115	2015-02-20 18:26:45.34115	3	accepted	\N	\N	1
35	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	3	10	9	57	12	2015-02-20 18:26:45.39549	2015-02-20 18:26:45.39549	3	accepted	\N	\N	1
36	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	1	3	15	9	58	12	2015-02-20 18:26:45.463778	2015-02-20 18:26:45.463778	3	accepted	\N	\N	1
37	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	1	3	16	9	58	12	2015-02-20 18:26:45.540575	2015-02-20 18:26:45.540575	3	accepted	\N	\N	1
38	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	1	3	16	9	59	12	2015-02-20 18:26:45.598419	2015-02-20 18:26:45.598419	2	accepted	\N	\N	1
39	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	1	3	17	9	59	12	2015-02-20 18:26:45.651238	2015-02-20 18:26:45.651238	2	accepted	\N	\N	1
40	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	3	18	7	60	12	2015-02-20 18:26:45.714491	2015-02-20 18:26:45.714491	2	accepted	\N	\N	1
41	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	3	19	7	60	12	2015-02-20 18:26:45.770538	2015-02-20 18:26:45.770538	2	accepted	\N	\N	1
42	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	3	20	7	60	12	2015-02-20 18:26:45.821394	2015-02-20 18:26:45.821394	2	accepted	\N	\N	1
43	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	Sensitivity	Supports	1	3	19	7	60	12	2015-02-20 18:26:45.878017	2015-02-20 18:26:45.878017	2	accepted	\N	\N	1
44	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.	Poor Outcome	Supports	3	2	3	7	61	12	2015-02-20 18:26:45.935012	2015-02-20 18:26:45.935012	5	accepted	\N	\N	1
45	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	Resistance or Non-Response	Supports	1	1	14	7	62	12	2015-02-20 18:26:46.000431	2015-02-20 18:26:46.000431	3	accepted	\N	\N	1
46	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	Resistance or Non-Response	Does Not Support	1	3	3	8	63	12	2015-02-20 18:26:46.088985	2015-02-20 18:26:46.088985	3	accepted	\N	\N	1
47	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	Resistance or Non-Response	Supports	1	3	3	8	63	12	2015-02-20 18:26:46.146716	2015-02-20 18:26:46.146716	3	accepted	\N	\N	1
48	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N/A	Does Not Support	1	1	21	8	64	12	2015-02-20 18:26:46.211677	2015-02-20 18:26:46.211677	2	accepted	\N	\N	1
49	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N/A	Does Not Support	1	1	22	8	64	12	2015-02-20 18:26:46.276072	2015-02-20 18:26:46.276072	2	accepted	\N	\N	1
50	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	1	1	13	8	65	12	2015-02-20 18:26:46.34153	2015-02-20 18:26:46.34153	5	accepted	\N	\N	1
51	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	1	1	11	8	65	12	2015-02-20 18:26:46.395833	2015-02-20 18:26:46.395833	5	accepted	\N	\N	1
52	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.	Poor Outcome	Supports	3	1	3	8	66	12	2015-02-20 18:26:46.459363	2015-02-20 18:26:46.459363	3	accepted	\N	\N	1
53	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	1	1	13	4	67	12	2015-02-20 18:26:46.525931	2015-02-20 18:26:46.525931	2	accepted	\N	\N	1
54	thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	Positive	Supports	2	1	3	10	68	12	2015-02-20 18:26:46.609498	2015-02-20 18:26:46.609498	5	accepted	\N	\N	1
55	thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	Positive	Supports	2	1	3	10	69	12	2015-02-20 18:26:46.670939	2015-02-20 18:26:46.670939	5	accepted	\N	\N	1
56	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer	Positive	Supports	2	1	3	10	70	12	2015-02-20 18:26:46.736754	2015-02-20 18:26:46.736754	3	accepted	\N	\N	1
57	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	Poor Outcome	Does Not Support	3	1	3	10	71	12	2015-02-20 18:26:46.796758	2015-02-20 18:26:46.796758	5	accepted	\N	\N	1
58	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	Poor Outcome	Supports	3	1	3	10	70	12	2015-02-20 18:26:46.864443	2015-02-20 18:26:46.864443	3	accepted	\N	\N	1
59	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	Poor Outcome	Supports	3	1	3	10	72	12	2015-02-20 18:26:46.926851	2015-02-20 18:26:46.926851	3	accepted	\N	\N	1
60	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	Poor Outcome	Supports	3	1	3	10	70	12	2015-02-20 18:26:47.005248	2015-02-20 18:26:47.005248	3	accepted	\N	\N	1
61	COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	Resistance or Non-Response	Supports	1	4	23	7	73	14	2015-02-20 18:26:47.096818	2015-02-20 18:26:47.096818	4	accepted	\N	\N	1
62	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	13	8	74	13	2015-02-20 18:26:47.16032	2015-02-20 18:26:47.16032	3	accepted	\N	\N	1
63	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	13	8	56	15	2015-02-20 18:26:47.222808	2015-02-20 18:26:47.222808	5	accepted	\N	\N	1
64	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	13	8	56	16	2015-02-20 18:26:47.288835	2015-02-20 18:26:47.288835	5	accepted	\N	\N	1
65	Inceased copy number of CCND1 is associated with poorer overall survival.	Poor Outcome	Supports	3	1	3	4	75	18	2015-02-20 18:26:47.356234	2015-02-20 18:26:47.356234	3	accepted	\N	\N	1
66	Three studies have associated CCND1 expression with poorer survival.	Poor Outcome	Supports	3	1	3	4	75	19	2015-02-20 18:26:47.416044	2015-02-20 18:26:47.416044	3	accepted	\N	\N	1
67	Three studies have found no significant survival impact for CCND1 in lung cancer.	Poor Outcome	Does Not Support	3	1	3	4	75	19	2015-02-20 18:26:47.475724	2015-02-20 18:26:47.475724	3	accepted	\N	\N	1
68	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	Positive	Supports	2	1	3	11	76	20	2015-02-20 18:26:47.54011	2015-02-20 18:26:47.54011	4	accepted	\N	\N	1
69	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	Poor Outcome	Supports	3	1	3	11	76	20	2015-02-20 18:26:47.61802	2015-02-20 18:26:47.61802	4	accepted	\N	\N	1
70	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	Poor Outcome	Supports	3	1	3	12	77	20	2015-02-20 18:26:47.681921	2015-02-20 18:26:47.681921	4	accepted	\N	\N	1
71	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	Poor Outcome	Supports	3	1	3	13	78	20	2015-02-20 18:26:47.750826	2015-02-20 18:26:47.750826	4	accepted	\N	\N	1
72	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	Poor Outcome	Supports	3	3	3	14	79	21	2015-02-20 18:26:47.820856	2015-02-20 18:26:47.820856	4	accepted	\N	\N	1
73	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	Poor Outcome	Supports	3	3	3	14	79	21	2015-02-20 18:26:47.877304	2015-02-20 18:26:47.877304	4	accepted	\N	\N	1
74	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	Poor Outcome	Supports	3	3	3	14	79	21	2015-02-20 18:26:47.941	2015-02-20 18:26:47.941	4	accepted	\N	\N	1
75	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	Poor Outcome	Supports	3	3	3	14	79	21	2015-02-20 18:26:47.999261	2015-02-20 18:26:47.999261	4	accepted	\N	\N	1
76	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	Negative	Supports	2	3	3	14	80	22	2015-02-20 18:26:48.062534	2015-02-20 18:26:48.062534	3	accepted	\N	\N	1
77	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	Sensitivity	Supports	1	3	24	15	81	23	2015-02-20 18:26:48.148881	2015-02-20 18:26:48.148881	3	accepted	\N	\N	1
78	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	Poor Outcome	Supports	3	1	3	11	82	24	2015-02-20 18:26:48.211785	2015-02-20 18:26:48.211785	4	accepted	\N	\N	1
79	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	Poor Outcome	Supports	3	1	3	16	83	24	2015-02-20 18:26:48.270818	2015-02-20 18:26:48.270818	4	accepted	\N	\N	1
80	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	3	1	3	16	83	24	2015-02-20 18:26:48.333606	2015-02-20 18:26:48.333606	4	accepted	\N	\N	1
81	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	3	1	3	17	83	24	2015-02-20 18:26:48.40035	2015-02-20 18:26:48.40035	4	accepted	\N	\N	1
82	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	1	3	25	18	84	25	2015-02-20 18:26:48.462918	2015-02-20 18:26:48.462918	4	accepted	\N	\N	1
83	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	3	1	26	18	85	25	2015-02-20 18:26:48.521197	2015-02-20 18:26:48.521197	3	accepted	\N	\N	1
84	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	1	3	25	18	84	26	2015-02-20 18:26:48.599249	2015-02-20 18:26:48.599249	4	accepted	\N	\N	1
85	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	3	1	26	18	85	26	2015-02-20 18:26:48.656096	2015-02-20 18:26:48.656096	3	accepted	\N	\N	1
86	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	Poor Outcome	Supports	3	1	3	4	86	27	2015-02-20 18:26:48.724517	2015-02-20 18:26:48.724517	3	accepted	\N	\N	1
87	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	Poor Outcome	Supports	3	1	3	4	86	27	2015-02-20 18:26:48.781663	2015-02-20 18:26:48.781663	3	accepted	\N	\N	1
88	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	N/A	Does Not Support	1	1	3	1	87	29	2015-02-20 18:26:48.839184	2015-02-20 18:26:48.839184	4	accepted	\N	\N	1
89	CEBPA mutation was significantly associated with complete remission	Better Outcome	Supports	3	1	3	1	88	29	2015-02-20 18:26:48.89703	2015-02-20 18:26:48.89703	4	accepted	\N	\N	1
90	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	Better Outcome	Supports	3	1	3	1	87	29	2015-02-20 18:26:48.954871	2015-02-20 18:26:48.954871	4	accepted	\N	\N	1
91	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	Better Outcome	Supports	3	1	3	1	89	28	2015-02-20 18:26:49.041465	2015-02-20 18:26:49.041465	3	accepted	\N	\N	1
92	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	Better Outcome	Supports	3	1	3	1	89	28	2015-02-20 18:26:49.100513	2015-02-20 18:26:49.100513	3	accepted	\N	\N	1
93	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	N/A	Does Not Support	3	1	3	1	89	28	2015-02-20 18:26:49.162655	2015-02-20 18:26:49.162655	2	accepted	\N	\N	1
94	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	3	1	19	90	30	2015-02-20 18:26:49.233097	2015-02-20 18:26:49.233097	3	accepted	\N	\N	1
95	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	3	27	19	90	30	2015-02-20 18:26:49.28582	2015-02-20 18:26:49.28582	3	accepted	\N	\N	1
96	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	2	1	3	20	91	31	2015-02-20 18:26:49.343361	2015-02-20 18:26:49.343361	4	accepted	\N	\N	1
97	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A	Positive	Supports	2	1	3	1	92	32	2015-02-20 18:26:49.40514	2015-02-20 18:26:49.40514	3	accepted	\N	\N	1
98	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	Positive	Supports	2	1	3	1	93	32	2015-02-20 18:26:49.490509	2015-02-20 18:26:49.490509	3	accepted	\N	\N	1
99	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	N/A	Does Not Support	1	1	28	1	93	32	2015-02-20 18:26:49.560745	2015-02-20 18:26:49.560745	4	accepted	\N	\N	1
100	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Sensitivity	Supports	1	1	29	1	93	32	2015-02-20 18:26:49.631836	2015-02-20 18:26:49.631836	4	accepted	\N	\N	1
101	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	N/A	Does Not Support	3	1	3	1	93	32	2015-02-20 18:26:49.69933	2015-02-20 18:26:49.69933	4	accepted	\N	\N	1
102	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	3	1	3	1	94	32	2015-02-20 18:26:49.759261	2015-02-20 18:26:49.759261	4	accepted	\N	\N	1
103	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	Positive	Supports	2	1	3	1	95	32	2015-02-20 18:26:49.814052	2015-02-20 18:26:49.814052	3	accepted	\N	\N	1
104	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	3	1	3	1	92	32	2015-02-20 18:26:49.877686	2015-02-20 18:26:49.877686	4	accepted	\N	\N	1
105	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	N/A	Does Not Support	3	1	3	1	96	32	2015-02-20 18:26:49.970174	2015-02-20 18:26:49.970174	3	accepted	\N	\N	1
106	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	N/A	Does Not Support	3	1	3	1	96	32	2015-02-20 18:26:50.02627	2015-02-20 18:26:50.02627	3	accepted	\N	\N	1
107	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	3	1	3	1	94	32	2015-02-20 18:26:50.089471	2015-02-20 18:26:50.089471	5	accepted	\N	\N	1
108	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	3	1	3	1	94	32	2015-02-20 18:26:50.145294	2015-02-20 18:26:50.145294	5	accepted	\N	\N	1
109	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	Poor Outcome	Supports	3	1	3	1	95	32	2015-02-20 18:26:50.210915	2015-02-20 18:26:50.210915	5	accepted	\N	\N	1
110	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	Poor Outcome	Supports	3	1	3	1	92	32	2015-02-20 18:26:50.274041	2015-02-20 18:26:50.274041	4	accepted	\N	\N	1
111	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	3	1	3	1	97	32	2015-02-20 18:26:50.338159	2015-02-20 18:26:50.338159	4	accepted	\N	\N	1
112	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	3	1	3	1	97	32	2015-02-20 18:26:50.418424	2015-02-20 18:26:50.418424	4	accepted	\N	\N	1
113	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs	Poor Outcome	Supports	3	1	3	1	96	32	2015-02-20 18:26:50.477266	2015-02-20 18:26:50.477266	3	accepted	\N	\N	1
114	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	Poor Outcome	Supports	3	2	3	1	96	32	2015-02-20 18:26:50.545346	2015-02-20 18:26:50.545346	3	accepted	\N	\N	1
115	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	Negative	Supports	2	1	3	1	96	32	2015-02-20 18:26:50.606935	2015-02-20 18:26:50.606935	3	accepted	\N	\N	1
116	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	Positive	Supports	2	1	3	1	94	32	2015-02-20 18:26:50.662012	2015-02-20 18:26:50.662012	5	accepted	\N	\N	1
117	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	2	1	3	1	94	32	2015-02-20 18:26:50.717903	2015-02-20 18:26:50.717903	4	accepted	\N	\N	1
118	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	2	1	3	1	96	32	2015-02-20 18:26:50.785708	2015-02-20 18:26:50.785708	3	accepted	\N	\N	1
119	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	2	1	3	1	96	32	2015-02-20 18:26:50.922848	2015-02-20 18:26:50.922848	3	accepted	\N	\N	1
120	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	Better Outcome	Supports	3	1	3	1	96	32	2015-02-20 18:26:50.985452	2015-02-20 18:26:50.985452	3	accepted	\N	\N	1
121	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	N/A	Does Not Support	3	1	3	1	97	32	2015-02-20 18:26:51.049963	2015-02-20 18:26:51.049963	4	accepted	\N	\N	1
122	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	Poor Outcome	Supports	3	1	3	1	97	32	2015-02-20 18:26:51.111098	2015-02-20 18:26:51.111098	4	accepted	\N	\N	1
123	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	3	1	3	1	92	32	2015-02-20 18:26:51.175131	2015-02-20 18:26:51.175131	3	accepted	\N	\N	1
124	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	Poor Outcome	Supports	3	1	3	1	92	32	2015-02-20 18:26:51.230196	2015-02-20 18:26:51.230196	3	accepted	\N	\N	1
125	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	3	1	3	1	92	32	2015-02-20 18:26:51.293377	2015-02-20 18:26:51.293377	3	accepted	\N	\N	1
126	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.	N/A	Does Not Support	1	1	30	4	98	33	2015-02-20 18:26:51.361502	2015-02-20 18:26:51.361502	3	accepted	\N	\N	1
127	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.	N/A	Does Not Support	1	1	31	4	98	33	2015-02-20 18:26:51.418265	2015-02-20 18:26:51.418265	3	accepted	\N	\N	1
128	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	Sensitivity	Supports	1	1	3	4	99	33	2015-02-20 18:26:51.502829	2015-02-20 18:26:51.502829	4	accepted	\N	\N	1
129	Geftinib added to pemetrexed and carboplatin showed greater effectiveness than premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.	Sensitivity	Supports	1	1	30	4	100	33	2015-02-20 18:26:51.574304	2015-02-20 18:26:51.574304	3	accepted	\N	\N	1
130	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	Better Outcome	Supports	3	1	3	4	101	33	2015-02-20 18:26:51.634475	2015-02-20 18:26:51.634475	3	accepted	\N	\N	1
131	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	Resistance or Non-Response	Supports	1	2	31	21	102	34	2015-02-20 18:26:51.699145	2015-02-20 18:26:51.699145	5	accepted	\N	\N	1
132	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	Resistance or Non-Response	Supports	1	1	30	4	103	34	2015-02-20 18:26:51.770854	2015-02-20 18:26:51.770854	3	accepted	\N	\N	1
133	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	Sensitivity	Supports	1	1	31	4	104	34	2015-02-20 18:26:51.847999	2015-02-20 18:26:51.847999	3	accepted	\N	\N	1
134	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	Sensitivity	Supports	1	1	32	4	104	34	2015-02-20 18:26:51.904188	2015-02-20 18:26:51.904188	3	accepted	\N	\N	1
135	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.	Sensitivity	Supports	1	4	33	4	105	34	2015-02-20 18:26:51.962989	2015-02-20 18:26:51.962989	1	accepted	\N	\N	1
136	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.	Poor Outcome	Supports	3	1	3	4	106	34	2015-02-20 18:26:52.054426	2015-02-20 18:26:52.054426	4	accepted	\N	\N	1
137	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.	Poor Outcome	Supports	3	1	3	4	107	34	2015-02-20 18:26:52.119131	2015-02-20 18:26:52.119131	4	accepted	\N	\N	1
138	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	34	11	108	35	2015-02-20 18:26:52.180589	2015-02-20 18:26:52.180589	4	accepted	\N	\N	1
139	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	34	11	108	36	2015-02-20 18:26:52.254604	2015-02-20 18:26:52.254604	4	accepted	\N	\N	1
140	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	34	11	108	37	2015-02-20 18:26:52.31818	2015-02-20 18:26:52.31818	5	accepted	\N	\N	1
141	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	34	11	108	38	2015-02-20 18:26:52.382702	2015-02-20 18:26:52.382702	5	accepted	\N	\N	1
142	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	Sensitivity	Supports	1	3	34	11	108	39	2015-02-20 18:26:52.444838	2015-02-20 18:26:52.444838	5	accepted	\N	\N	1
143	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	Resistance or Non-Response	Supports	1	3	35	11	108	39	2015-02-20 18:26:52.506366	2015-02-20 18:26:52.506366	5	accepted	\N	\N	1
234	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	N/A	Does Not Support	3	1	3	33	143	75	2015-02-20 18:26:58.58004	2015-02-20 18:26:58.58004	3	accepted	\N	\N	1
144	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	1	3	34	11	108	40	2015-02-20 18:26:52.589457	2015-02-20 18:26:52.589457	5	accepted	\N	\N	1
145	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	34	11	108	41	2015-02-20 18:26:52.651131	2015-02-20 18:26:52.651131	5	accepted	\N	\N	1
146	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	1	3	34	11	108	42	2015-02-20 18:26:52.724445	2015-02-20 18:26:52.724445	5	accepted	\N	\N	1
147	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	Sensitivity	Supports	1	3	35	11	108	42	2015-02-20 18:26:52.772732	2015-02-20 18:26:52.772732	5	accepted	\N	\N	1
148	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	34	11	108	43	2015-02-20 18:26:52.829602	2015-02-20 18:26:52.829602	5	accepted	\N	\N	1
149	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	34	11	108	44	2015-02-20 18:26:52.899167	2015-02-20 18:26:52.899167	5	accepted	\N	\N	1
150	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.	Sensitivity	Supports	1	3	34	11	108	45	2015-02-20 18:26:52.970357	2015-02-20 18:26:52.970357	5	accepted	\N	\N	1
151	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	1	3	36	11	109	46	2015-02-20 18:26:53.042536	2015-02-20 18:26:53.042536	5	accepted	\N	\N	1
152	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	1	3	37	11	109	46	2015-02-20 18:26:53.114693	2015-02-20 18:26:53.114693	5	accepted	\N	\N	1
153	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	3	3	11	110	46	2015-02-20 18:26:53.177865	2015-02-20 18:26:53.177865	3	accepted	\N	\N	1
154	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	36	11	109	47	2015-02-20 18:26:53.241467	2015-02-20 18:26:53.241467	5	accepted	\N	\N	1
155	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	37	11	109	47	2015-02-20 18:26:53.29207	2015-02-20 18:26:53.29207	5	accepted	\N	\N	1
156	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	3	3	11	110	47	2015-02-20 18:26:53.358611	2015-02-20 18:26:53.358611	3	accepted	\N	\N	1
157	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	36	11	109	48	2015-02-20 18:26:53.424452	2015-02-20 18:26:53.424452	5	accepted	\N	\N	1
158	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	37	11	109	48	2015-02-20 18:26:53.477353	2015-02-20 18:26:53.477353	5	accepted	\N	\N	1
159	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	3	3	11	110	48	2015-02-20 18:26:53.54051	2015-02-20 18:26:53.54051	3	accepted	\N	\N	1
160	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	36	11	109	49	2015-02-20 18:26:53.635364	2015-02-20 18:26:53.635364	5	accepted	\N	\N	1
161	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	37	11	109	49	2015-02-20 18:26:53.689846	2015-02-20 18:26:53.689846	5	accepted	\N	\N	1
162	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	3	3	11	110	49	2015-02-20 18:26:53.753916	2015-02-20 18:26:53.753916	3	accepted	\N	\N	1
163	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	36	11	109	50	2015-02-20 18:26:53.824143	2015-02-20 18:26:53.824143	5	accepted	\N	\N	1
164	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	3	37	11	109	50	2015-02-20 18:26:53.874289	2015-02-20 18:26:53.874289	5	accepted	\N	\N	1
165	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	3	3	11	110	50	2015-02-20 18:26:53.93896	2015-02-20 18:26:53.93896	3	accepted	\N	\N	1
166	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.	Sensitivity	Supports	1	1	38	22	111	56	2015-02-20 18:26:54.015691	2015-02-20 18:26:54.015691	2	accepted	\N	\N	1
167	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be recurrent in multiple other cancer types.	Sensitivity	Supports	1	1	39	22	111	59	2015-02-20 18:26:54.111565	2015-02-20 18:26:54.111565	3	accepted	\N	\N	1
168	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be recurrent in multiple other cancer types.	Sensitivity	Supports	1	1	38	22	111	59	2015-02-20 18:26:54.161476	2015-02-20 18:26:54.161476	3	accepted	\N	\N	1
169	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	3	1	3	1	92	60	2015-02-20 18:26:54.226488	2015-02-20 18:26:54.226488	4	accepted	\N	\N	1
170	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	3	1	3	1	94	60	2015-02-20 18:26:54.281608	2015-02-20 18:26:54.281608	5	accepted	\N	\N	1
171	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	Resistance or Non-Response	Supports	1	1	3	1	112	60	2015-02-20 18:26:54.346869	2015-02-20 18:26:54.346869	4	accepted	\N	\N	1
172	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	1	3	40	1	113	60	2015-02-20 18:26:54.410117	2015-02-20 18:26:54.410117	3	accepted	\N	\N	1
173	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	Resistance or Non-Response	Supports	1	1	41	1	114	60	2015-02-20 18:26:54.478121	2015-02-20 18:26:54.478121	3	accepted	\N	\N	1
174	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	1	3	28	1	113	60	2015-02-20 18:26:54.537977	2015-02-20 18:26:54.537977	3	accepted	\N	\N	1
175	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	Resistance or Non-Response	Supports	1	1	3	1	112	60	2015-02-20 18:26:54.617212	2015-02-20 18:26:54.617212	3	accepted	\N	\N	1
176	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	N/A	Does Not Support	3	1	3	1	115	60	2015-02-20 18:26:54.692513	2015-02-20 18:26:54.692513	4	accepted	\N	\N	1
177	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	N/A	Does Not Support	3	1	3	1	112	60	2015-02-20 18:26:54.755393	2015-02-20 18:26:54.755393	2	accepted	\N	\N	1
178	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	3	1	3	1	115	60	2015-02-20 18:26:54.82253	2015-02-20 18:26:54.82253	4	accepted	\N	\N	1
179	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	3	1	3	1	115	60	2015-02-20 18:26:54.918344	2015-02-20 18:26:54.918344	4	accepted	\N	\N	1
180	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	Poor Outcome	Supports	3	1	3	1	115	60	2015-02-20 18:26:54.995193	2015-02-20 18:26:54.995193	4	accepted	\N	\N	1
181	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	Poor Outcome	Supports	3	2	3	1	116	60	2015-02-20 18:26:55.069493	2015-02-20 18:26:55.069493	2	accepted	\N	\N	1
182	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	Positive	Supports	2	1	3	1	117	60	2015-02-20 18:26:55.152043	2015-02-20 18:26:55.152043	3	accepted	\N	\N	1
183	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	Sensitivity	Supports	1	3	10	1	118	60	2015-02-20 18:26:55.217797	2015-02-20 18:26:55.217797	4	accepted	\N	\N	1
184	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	Sensitivity	Supports	1	1	42	1	119	60	2015-02-20 18:26:55.280873	2015-02-20 18:26:55.280873	4	accepted	\N	\N	1
185	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	Sensitivity	Supports	1	3	43	1	120	60	2015-02-20 18:26:55.348738	2015-02-20 18:26:55.348738	4	accepted	\N	\N	1
186	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	Poor Outcome	Supports	3	1	3	1	121	60	2015-02-20 18:26:55.406576	2015-02-20 18:26:55.406576	4	accepted	\N	\N	1
187	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	Poor Outcome	Supports	3	1	3	1	117	60	2015-02-20 18:26:55.463265	2015-02-20 18:26:55.463265	3	accepted	\N	\N	1
188	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	1	3	40	1	113	61	2015-02-20 18:26:55.523473	2015-02-20 18:26:55.523473	3	accepted	\N	\N	1
189	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	1	3	28	1	113	61	2015-02-20 18:26:55.582243	2015-02-20 18:26:55.582243	3	accepted	\N	\N	1
190	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	3	1	3	1	122	61	2015-02-20 18:26:55.666377	2015-02-20 18:26:55.666377	3	accepted	\N	\N	1
191	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	Positive	Does Not Support	2	1	3	1	117	61	2015-02-20 18:26:55.724705	2015-02-20 18:26:55.724705	3	accepted	\N	\N	1
192	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	Resistance or Non-Response	Supports	1	1	10	1	123	61	2015-02-20 18:26:55.791741	2015-02-20 18:26:55.791741	4	accepted	\N	\N	1
193	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.	Resistance or Non-Response	Supports	1	3	10	1	118	61	2015-02-20 18:26:55.851242	2015-02-20 18:26:55.851242	4	accepted	\N	\N	1
194	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	Sensitivity	Supports	1	1	42	1	119	61	2015-02-20 18:26:55.909866	2015-02-20 18:26:55.909866	4	accepted	\N	\N	1
195	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	N/A	Does Not Support	3	1	3	1	117	61	2015-02-20 18:26:55.969734	2015-02-20 18:26:55.969734	3	accepted	\N	\N	1
196	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3	Poor Outcome	Supports	3	1	3	1	124	61	2015-02-20 18:26:56.035702	2015-02-20 18:26:56.035702	4	accepted	\N	\N	1
197	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	1	1	44	23	125	62	2015-02-20 18:26:56.128024	2015-02-20 18:26:56.128024	4	accepted	\N	\N	1
198	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	1	1	45	23	125	62	2015-02-20 18:26:56.174371	2015-02-20 18:26:56.174371	4	accepted	\N	\N	1
199	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N/A	Does Not Support	3	1	3	1	126	63	2015-02-20 18:26:56.238911	2015-02-20 18:26:56.238911	3	accepted	\N	\N	1
200	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	Positive	Supports	2	1	3	1	127	63	2015-02-20 18:26:56.30686	2015-02-20 18:26:56.30686	2	accepted	\N	\N	1
201	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	N/A	Does Not Support	3	1	3	1	128	63	2015-02-20 18:26:56.370682	2015-02-20 18:26:56.370682	2	accepted	\N	\N	1
202	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	Better Outcome	Supports	3	1	3	24	129	63	2015-02-20 18:26:56.433695	2015-02-20 18:26:56.433695	3	accepted	\N	\N	1
203	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	Better Outcome	Supports	3	1	3	25	130	63	2015-02-20 18:26:56.495205	2015-02-20 18:26:56.495205	3	accepted	\N	\N	1
204	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	Poor Outcome	Supports	3	1	3	26	131	63	2015-02-20 18:26:56.584115	2015-02-20 18:26:56.584115	4	accepted	\N	\N	1
205	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	Positive	Supports	2	1	3	1	132	64	2015-02-20 18:26:56.645	2015-02-20 18:26:56.645	3	accepted	\N	\N	1
206	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	3	1	3	1	132	64	2015-02-20 18:26:56.705047	2015-02-20 18:26:56.705047	2	accepted	\N	\N	1
207	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	3	1	3	1	132	65	2015-02-20 18:26:56.768181	2015-02-20 18:26:56.768181	3	accepted	\N	\N	1
208	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	3	1	3	1	132	66	2015-02-20 18:26:56.828797	2015-02-20 18:26:56.828797	3	accepted	\N	\N	1
209	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	Better Outcome	Supports	3	1	3	1	133	67	2015-02-20 18:26:56.896058	2015-02-20 18:26:56.896058	3	accepted	\N	\N	1
210	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.	N/A	Does Not Support	3	1	3	1	134	67	2015-02-20 18:26:56.971103	2015-02-20 18:26:56.971103	3	accepted	\N	\N	1
211	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	N/A	Does Not Support	3	4	3	1	135	67	2015-02-20 18:26:57.052966	2015-02-20 18:26:57.052966	2	accepted	\N	\N	1
212	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	N/A	Does Not Support	3	1	3	27	136	67	2015-02-20 18:26:57.120907	2015-02-20 18:26:57.120907	3	accepted	\N	\N	1
213	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	N/A	Does Not Support	3	4	3	27	137	67	2015-02-20 18:26:57.178849	2015-02-20 18:26:57.178849	2	accepted	\N	\N	1
214	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	N/A	Does Not Support	3	4	3	1	135	68	2015-02-20 18:26:57.232915	2015-02-20 18:26:57.232915	2	accepted	\N	\N	1
215	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	Poor Outcome	Supports	3	1	3	1	133	68	2015-02-20 18:26:57.288894	2015-02-20 18:26:57.288894	3	accepted	\N	\N	1
216	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)	N/A	Does Not Support	3	4	3	27	137	68	2015-02-20 18:26:57.347781	2015-02-20 18:26:57.347781	2	accepted	\N	\N	1
217	JAK2 V617F not associatted wth  lymphoid leukemia.	Positive	Does Not Support	2	1	3	2	138	69	2015-02-20 18:26:57.411971	2015-02-20 18:26:57.411971	4	accepted	\N	\N	1
218	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	Positive	Supports	2	1	3	2	138	69	2015-02-20 18:26:57.554623	2015-02-20 18:26:57.554623	4	accepted	\N	\N	1
219	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells	Sensitivity	Supports	1	3	46	28	139	69	2015-02-20 18:26:57.619008	2015-02-20 18:26:57.619008	3	accepted	\N	\N	1
220	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	Negative	Supports	2	1	3	29	138	69	2015-02-20 18:26:57.68502	2015-02-20 18:26:57.68502	4	accepted	\N	\N	1
221	rs10974944 SNP in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer	Positive	Supports	2	1	3	30	140	69	2015-02-20 18:26:57.748011	2015-02-20 18:26:57.748011	4	accepted	\N	\N	1
222	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	Sensitivity	Supports	1	1	47	31	141	69	2015-02-20 18:26:57.813853	2015-02-20 18:26:57.813853	4	accepted	\N	\N	1
223	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT	Poor Outcome	Supports	3	1	3	32	142	70	2015-02-20 18:26:57.884893	2015-02-20 18:26:57.884893	4	accepted	\N	\N	1
224	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	N/A	Does Not Support	3	1	3	33	143	71	2015-02-20 18:26:57.952474	2015-02-20 18:26:57.952474	3	accepted	\N	\N	1
225	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	2	1	3	33	144	71	2015-02-20 18:26:58.016576	2015-02-20 18:26:58.016576	2	accepted	\N	\N	1
226	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	N/A	Does Not Support	3	1	3	33	145	71	2015-02-20 18:26:58.075268	2015-02-20 18:26:58.075268	3	accepted	\N	\N	1
227	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	Poor Outcome	Supports	3	1	3	33	144	71	2015-02-20 18:26:58.131987	2015-02-20 18:26:58.131987	3	accepted	\N	\N	1
228	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N/A	Does Not Support	3	1	3	17	146	71	2015-02-20 18:26:58.192095	2015-02-20 18:26:58.192095	2	accepted	\N	\N	1
229	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	1	1	8	147	71	2015-02-20 18:26:58.256382	2015-02-20 18:26:58.256382	3	accepted	\N	\N	1
230	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	1	1	8	147	72	2015-02-20 18:26:58.324813	2015-02-20 18:26:58.324813	3	accepted	\N	\N	1
231	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	Poor Outcome	Supports	3	1	3	33	143	73	2015-02-20 18:26:58.393729	2015-02-20 18:26:58.393729	3	accepted	\N	\N	1
232	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	2	1	3	33	144	74	2015-02-20 18:26:58.446966	2015-02-20 18:26:58.446966	3	accepted	\N	\N	1
233	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	1	1	8	147	74	2015-02-20 18:26:58.520304	2015-02-20 18:26:58.520304	3	accepted	\N	\N	1
235	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N/A	Does Not Support	3	1	3	17	146	75	2015-02-20 18:26:58.637677	2015-02-20 18:26:58.637677	2	accepted	\N	\N	1
236	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	3	1	8	148	76	2015-02-20 18:26:58.703074	2015-02-20 18:26:58.703074	3	accepted	\N	\N	1
237	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	3	4	8	148	76	2015-02-20 18:26:58.759434	2015-02-20 18:26:58.759434	3	accepted	\N	\N	1
238	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	3	10	8	148	76	2015-02-20 18:26:58.811732	2015-02-20 18:26:58.811732	3	accepted	\N	\N	1
239	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	Sensitivity	Supports	1	3	5	8	148	76	2015-02-20 18:26:58.880756	2015-02-20 18:26:58.880756	3	accepted	\N	\N	1
240	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	3	5	4	149	76	2015-02-20 18:26:58.955392	2015-02-20 18:26:58.955392	3	accepted	\N	\N	1
241	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	3	1	4	149	76	2015-02-20 18:26:58.997598	2015-02-20 18:26:58.997598	3	accepted	\N	\N	1
242	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	3	4	4	149	76	2015-02-20 18:26:59.071529	2015-02-20 18:26:59.071529	3	accepted	\N	\N	1
243	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	Resistance or Non-Response	Supports	1	3	1	33	150	77	2015-02-20 18:26:59.146492	2015-02-20 18:26:59.146492	3	accepted	\N	\N	1
244	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	Sensitivity	Supports	1	1	48	33	151	77	2015-02-20 18:26:59.220325	2015-02-20 18:26:59.220325	3	accepted	\N	\N	1
245	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	Resistance or Non-Response	Supports	1	1	14	7	152	78	2015-02-20 18:26:59.303059	2015-02-20 18:26:59.303059	4	accepted	\N	\N	1
246	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	Sensitivity	Does Not Support	1	1	49	7	153	79	2015-02-20 18:26:59.362629	2015-02-20 18:26:59.362629	4	accepted	\N	\N	1
247	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	1	1	30	4	154	79	2015-02-20 18:26:59.427067	2015-02-20 18:26:59.427067	3	accepted	\N	\N	1
248	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	1	1	31	4	154	79	2015-02-20 18:26:59.476836	2015-02-20 18:26:59.476836	3	accepted	\N	\N	1
249	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	N/A	Does Not Support	3	1	3	9	155	80	2015-02-20 18:26:59.542335	2015-02-20 18:26:59.542335	4	accepted	\N	\N	1
250	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	2	1	3	34	156	80	2015-02-20 18:26:59.615811	2015-02-20 18:26:59.615811	3	accepted	\N	\N	1
251	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	3	1	3	35	157	80	2015-02-20 18:26:59.6769	2015-02-20 18:26:59.6769	3	accepted	\N	\N	1
252	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	Positive	Does Not Support	2	1	3	4	158	80	2015-02-20 18:26:59.743313	2015-02-20 18:26:59.743313	3	accepted	\N	\N	1
253	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	N/A	Does Not Support	3	1	3	4	159	80	2015-02-20 18:26:59.809586	2015-02-20 18:26:59.809586	3	accepted	\N	\N	1
254	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	Poor Outcome	Supports	3	1	3	4	160	80	2015-02-20 18:26:59.870158	2015-02-20 18:26:59.870158	3	accepted	\N	\N	1
255	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	Resistance or Non-Response	Supports	1	3	14	7	161	81	2015-02-20 18:26:59.932004	2015-02-20 18:26:59.932004	3	accepted	\N	\N	1
256	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	Resistance or Non-Response	Supports	1	1	14	7	62	81	2015-02-20 18:26:59.993401	2015-02-20 18:26:59.993401	3	accepted	\N	\N	1
257	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	Poor Outcome	Supports	3	3	3	7	161	81	2015-02-20 18:27:00.062611	2015-02-20 18:27:00.062611	3	accepted	\N	\N	1
258	KRAS G12C occur more frequently in women than men	Positive	Supports	2	1	3	17	162	82	2015-02-20 18:27:00.124896	2015-02-20 18:27:00.124896	2	accepted	\N	\N	1
259	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	Positive	Supports	2	1	3	17	162	83	2015-02-20 18:27:00.196637	2015-02-20 18:27:00.196637	3	accepted	\N	\N	1
260	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	1	1	13	4	163	83	2015-02-20 18:27:00.257881	2015-02-20 18:27:00.257881	2	accepted	\N	\N	1
261	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	1	3	50	4	164	83	2015-02-20 18:27:00.333741	2015-02-20 18:27:00.333741	4	accepted	\N	\N	1
262	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	1	3	51	4	164	83	2015-02-20 18:27:00.380972	2015-02-20 18:27:00.380972	4	accepted	\N	\N	1
263	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	2	1	3	34	156	84	2015-02-20 18:27:00.43266	2015-02-20 18:27:00.43266	3	accepted	\N	\N	1
264	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	3	1	3	36	157	84	2015-02-20 18:27:00.489277	2015-02-20 18:27:00.489277	3	accepted	\N	\N	1
265	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	Negative	Does Not Support	2	1	3	4	158	84	2015-02-20 18:27:00.542558	2015-02-20 18:27:00.542558	3	accepted	\N	\N	1
266	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	Resistance or Non-Response	Supports	1	3	14	9	165	85	2015-02-20 18:27:00.596702	2015-02-20 18:27:00.596702	4	accepted	\N	\N	1
267	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	Sensitivity	Supports	1	3	14	9	165	85	2015-02-20 18:27:00.697195	2015-02-20 18:27:00.697195	4	accepted	\N	\N	1
268	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Sensitivity	Supports	1	1	14	9	165	85	2015-02-20 18:27:00.760786	2015-02-20 18:27:00.760786	4	accepted	\N	\N	1
269	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Better Outcome	Supports	3	1	14	9	165	85	2015-02-20 18:27:00.825987	2015-02-20 18:27:00.825987	4	accepted	\N	\N	1
270	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.	Resistance or Non-Response	Supports	1	3	23	8	166	86	2015-02-20 18:27:00.889666	2015-02-20 18:27:00.889666	3	accepted	\N	\N	1
271	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	Resistance or Non-Response	Supports	1	3	23	8	167	87	2015-02-20 18:27:00.954992	2015-02-20 18:27:00.954992	3	accepted	\N	\N	1
272	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	3	1	19	90	88	2015-02-20 18:27:01.014803	2015-02-20 18:27:01.014803	3	accepted	\N	\N	1
273	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	3	27	19	90	88	2015-02-20 18:27:01.061974	2015-02-20 18:27:01.061974	3	accepted	\N	\N	1
274	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.	Sensitivity	Supports	1	2	22	37	168	89	2015-02-20 18:27:01.122949	2015-02-20 18:27:01.122949	5	accepted	\N	\N	1
275	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.	Better Outcome	Supports	3	1	3	37	169	89	2015-02-20 18:27:01.187816	2015-02-20 18:27:01.187816	4	accepted	\N	\N	1
276	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	Sensitivity	Supports	1	4	52	37	169	89	2015-02-20 18:27:01.248046	2015-02-20 18:27:01.248046	4	accepted	\N	\N	1
277	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	Sensitivity	Supports	1	1	53	37	169	89	2015-02-20 18:27:01.316674	2015-02-20 18:27:01.316674	4	accepted	\N	\N	1
278	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	3	1	3	1	92	90	2015-02-20 18:27:01.376223	2015-02-20 18:27:01.376223	4	accepted	\N	\N	1
279	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.	Negative	Supports	2	1	3	1	170	90	2015-02-20 18:27:01.444535	2015-02-20 18:27:01.444535	4	accepted	\N	\N	1
280	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).	Negative	Does Not Support	2	1	3	1	171	90	2015-02-20 18:27:01.508288	2015-02-20 18:27:01.508288	3	accepted	\N	\N	1
281	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.	Positive	Supports	2	1	3	1	170	90	2015-02-20 18:27:01.564873	2015-02-20 18:27:01.564873	4	accepted	\N	\N	1
282	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia	Positive	Supports	2	1	3	1	171	90	2015-02-20 18:27:01.632341	2015-02-20 18:27:01.632341	3	accepted	\N	\N	1
283	NPM1 mutations were associated with normal karyotype in older (>60) patients	Positive	Supports	2	1	3	1	172	90	2015-02-20 18:27:01.687533	2015-02-20 18:27:01.687533	3	accepted	\N	\N	1
284	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	Positive	Supports	2	1	3	1	112	90	2015-02-20 18:27:01.752859	2015-02-20 18:27:01.752859	2	accepted	\N	\N	1
285	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	Resistance or Non-Response	Supports	1	1	41	1	114	90	2015-02-20 18:27:01.816667	2015-02-20 18:27:01.816667	3	accepted	\N	\N	1
286	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	Resistance or Non-Response	Supports	1	1	3	1	173	90	2015-02-20 18:27:01.877099	2015-02-20 18:27:01.877099	2	accepted	\N	\N	1
287	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)	Sensitivity	Supports	1	1	3	1	112	90	2015-02-20 18:27:01.939709	2015-02-20 18:27:01.939709	4	accepted	\N	\N	1
288	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	1	3	54	1	174	90	2015-02-20 18:27:02.013366	2015-02-20 18:27:02.013366	3	accepted	\N	\N	1
289	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	1	3	41	1	174	90	2015-02-20 18:27:02.062378	2015-02-20 18:27:02.062378	3	accepted	\N	\N	1
290	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	Sensitivity	Supports	1	1	3	1	172	90	2015-02-20 18:27:02.121838	2015-02-20 18:27:02.121838	3	accepted	\N	\N	1
291	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	Sensitivity	Supports	1	1	41	1	172	90	2015-02-20 18:27:02.176382	2015-02-20 18:27:02.176382	2	accepted	\N	\N	1
292	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	Sensitivity	Supports	1	1	28	1	175	90	2015-02-20 18:27:02.285675	2015-02-20 18:27:02.285675	2	accepted	\N	\N	1
293	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	Sensitivity	Supports	1	1	55	1	176	90	2015-02-20 18:27:02.355032	2015-02-20 18:27:02.355032	2	accepted	\N	\N	1
294	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	Sensitivity	Supports	1	4	3	1	177	90	2015-02-20 18:27:02.416335	2015-02-20 18:27:02.416335	2	accepted	\N	\N	1
295	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	Sensitivity	Supports	1	4	3	1	177	90	2015-02-20 18:27:02.47454	2015-02-20 18:27:02.47454	2	accepted	\N	\N	1
296	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.	Better Outcome	Supports	3	1	3	1	170	90	2015-02-20 18:27:02.532435	2015-02-20 18:27:02.532435	5	accepted	\N	\N	1
297	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	Better Outcome	Supports	3	1	3	1	178	90	2015-02-20 18:27:02.600219	2015-02-20 18:27:02.600219	4	accepted	\N	\N	1
298	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	Better Outcome	Supports	3	1	3	1	179	90	2015-02-20 18:27:02.662591	2015-02-20 18:27:02.662591	4	accepted	\N	\N	1
299	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	Better Outcome	Supports	3	1	3	1	180	90	2015-02-20 18:27:02.728307	2015-02-20 18:27:02.728307	4	accepted	\N	\N	1
300	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.	Better Outcome	Supports	3	1	3	1	171	90	2015-02-20 18:27:02.788624	2015-02-20 18:27:02.788624	3	accepted	\N	\N	1
301	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	3	1	3	1	122	90	2015-02-20 18:27:02.850336	2015-02-20 18:27:02.850336	3	accepted	\N	\N	1
302	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	Better Outcome	Supports	3	1	3	1	181	90	2015-02-20 18:27:02.916684	2015-02-20 18:27:02.916684	3	accepted	\N	\N	1
303	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	Better Outcome	Supports	3	1	3	1	182	90	2015-02-20 18:27:02.986274	2015-02-20 18:27:02.986274	3	accepted	\N	\N	1
304	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	Better Outcome	Supports	3	1	3	1	175	90	2015-02-20 18:27:03.046073	2015-02-20 18:27:03.046073	2	accepted	\N	\N	1
305	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	Better Outcome	Supports	3	1	3	1	88	90	2015-02-20 18:27:03.101542	2015-02-20 18:27:03.101542	3	accepted	\N	\N	1
306	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N/A	Does Not Support	3	1	3	1	126	90	2015-02-20 18:27:03.152576	2015-02-20 18:27:03.152576	3	accepted	\N	\N	1
307	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	Poor Outcome	Supports	3	1	3	1	171	90	2015-02-20 18:27:03.209163	2015-02-20 18:27:03.209163	4	accepted	\N	\N	1
308	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	3	1	3	1	115	90	2015-02-20 18:27:03.263377	2015-02-20 18:27:03.263377	4	accepted	\N	\N	1
309	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	3	1	3	1	115	90	2015-02-20 18:27:03.328312	2015-02-20 18:27:03.328312	4	accepted	\N	\N	1
310	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	Poor Outcome	Supports	3	1	3	1	183	90	2015-02-20 18:27:03.410281	2015-02-20 18:27:03.410281	3	accepted	\N	\N	1
311	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	Poor Outcome	Supports	3	1	3	1	180	90	2015-02-20 18:27:03.466592	2015-02-20 18:27:03.466592	3	accepted	\N	\N	1
312	"AML with mutated NPM1" is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	Positive	Supports	2	2	3	1	184	90	2015-02-20 18:27:03.531748	2015-02-20 18:27:03.531748	5	accepted	\N	\N	1
313	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	Negative	Does Not Support	2	1	3	1	94	91	2015-02-20 18:27:03.595595	2015-02-20 18:27:03.595595	4	accepted	\N	\N	1
314	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	2	1	3	1	94	91	2015-02-20 18:27:03.665433	2015-02-20 18:27:03.665433	4	accepted	\N	\N	1
315	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.	Sensitivity	Supports	1	3	54	1	174	91	2015-02-20 18:27:03.775133	2015-02-20 18:27:03.775133	3	accepted	\N	\N	1
316	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	Sensitivity	Supports	1	4	3	1	185	91	2015-02-20 18:27:03.850793	2015-02-20 18:27:03.850793	3	accepted	\N	\N	1
317	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	3	1	3	8	186	92	2015-02-20 18:27:03.921332	2015-02-20 18:27:03.921332	3	accepted	\N	\N	1
318	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	N/A	Does Not Support	3	1	3	1	187	93	2015-02-20 18:27:03.993382	2015-02-20 18:27:03.993382	3	accepted	\N	\N	1
319	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.	Poor Outcome	Supports	3	1	3	7	188	93	2015-02-20 18:27:04.058383	2015-02-20 18:27:04.058383	3	accepted	\N	\N	1
320	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	3	1	3	8	186	93	2015-02-20 18:27:04.123224	2015-02-20 18:27:04.123224	3	accepted	\N	\N	1
321	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.	Poor Outcome	Supports	3	2	3	7	188	94	2015-02-20 18:27:04.18237	2015-02-20 18:27:04.18237	3	accepted	\N	\N	1
322	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.	Poor Outcome	Supports	3	2	3	7	188	95	2015-02-20 18:27:04.240897	2015-02-20 18:27:04.240897	3	accepted	\N	\N	1
323	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	N/A	Does Not Support	3	1	3	1	189	96	2015-02-20 18:27:04.312185	2015-02-20 18:27:04.312185	3	accepted	\N	\N	1
324	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation	Sensitivity	Supports	1	1	56	8	190	97	2015-02-20 18:27:04.381793	2015-02-20 18:27:04.381793	2	accepted	\N	\N	1
325	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	Resistance or Non-Response	Supports	1	1	14	7	62	98	2015-02-20 18:27:04.44473	2015-02-20 18:27:04.44473	3	accepted	\N	\N	1
326	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	N/A	Does Not Support	1	1	14	7	62	98	2015-02-20 18:27:04.503435	2015-02-20 18:27:04.503435	3	accepted	\N	\N	1
327	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	Positive	Supports	2	1	3	8	191	98	2015-02-20 18:27:04.563806	2015-02-20 18:27:04.563806	3	accepted	\N	\N	1
328	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	Resistance or Non-Response	Supports	1	1	19	8	192	98	2015-02-20 18:27:04.62395	2015-02-20 18:27:04.62395	4	accepted	\N	\N	1
329	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	Sensitivity	Supports	1	1	57	8	193	99	2015-02-20 18:27:04.689306	2015-02-20 18:27:04.689306	2	accepted	\N	\N	1
330	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	Sensitivity	Supports	1	1	57	8	193	100	2015-02-20 18:27:04.748498	2015-02-20 18:27:04.748498	2	accepted	\N	\N	1
331	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	58	38	194	101	2015-02-20 18:27:04.815125	2015-02-20 18:27:04.815125	4	accepted	\N	\N	1
332	GIST tumors harboring PDGFRA D842V mutation are likely to be benign	Negative	Supports	2	1	3	38	195	102	2015-02-20 18:27:04.882918	2015-02-20 18:27:04.882918	3	accepted	\N	\N	1
333	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	Resistance or Non-Response	Supports	1	1	1	38	150	102	2015-02-20 18:27:04.942297	2015-02-20 18:27:04.942297	4	accepted	\N	\N	1
334	GIST cancer with D842V mutation is resistant to imatinib.	Resistance or Non-Response	Supports	1	1	1	38	196	102	2015-02-20 18:27:05.019477	2015-02-20 18:27:05.019477	4	accepted	\N	\N	1
335	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	58	38	194	102	2015-02-20 18:27:05.087428	2015-02-20 18:27:05.087428	4	accepted	\N	\N	1
336	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	58	38	194	103	2015-02-20 18:27:05.154672	2015-02-20 18:27:05.154672	4	accepted	\N	\N	1
337	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	58	38	194	104	2015-02-20 18:27:05.221339	2015-02-20 18:27:05.221339	4	accepted	\N	\N	1
338	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	Sensitivity	Supports	1	3	58	38	194	105	2015-02-20 18:27:05.334734	2015-02-20 18:27:05.334734	4	accepted	\N	\N	1
339	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	3	59	11	197	106	2015-02-20 18:27:05.407624	2015-02-20 18:27:05.407624	4	accepted	\N	\N	1
340	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	3	60	11	198	106	2015-02-20 18:27:05.471729	2015-02-20 18:27:05.471729	3	accepted	\N	\N	1
341	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	49	7	153	106	2015-02-20 18:27:05.533769	2015-02-20 18:27:05.533769	4	accepted	\N	\N	1
342	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	14	7	199	106	2015-02-20 18:27:05.597333	2015-02-20 18:27:05.597333	3	accepted	\N	\N	1
343	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	12	7	199	106	2015-02-20 18:27:05.65225	2015-02-20 18:27:05.65225	3	accepted	\N	\N	1
344	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	4	3	7	199	106	2015-02-20 18:27:05.710289	2015-02-20 18:27:05.710289	3	accepted	\N	\N	1
345	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	3	4	3	7	200	106	2015-02-20 18:27:05.774452	2015-02-20 18:27:05.774452	2	accepted	\N	\N	1
346	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	3	59	11	197	107	2015-02-20 18:27:05.837929	2015-02-20 18:27:05.837929	4	accepted	\N	\N	1
347	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	3	60	11	198	107	2015-02-20 18:27:05.891407	2015-02-20 18:27:05.891407	3	accepted	\N	\N	1
348	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	49	7	153	107	2015-02-20 18:27:05.953143	2015-02-20 18:27:05.953143	4	accepted	\N	\N	1
349	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	14	7	199	107	2015-02-20 18:27:06.018721	2015-02-20 18:27:06.018721	3	accepted	\N	\N	1
350	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	12	7	199	107	2015-02-20 18:27:06.083649	2015-02-20 18:27:06.083649	3	accepted	\N	\N	1
351	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	4	3	7	199	107	2015-02-20 18:27:06.141888	2015-02-20 18:27:06.141888	3	accepted	\N	\N	1
352	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	3	4	3	7	200	107	2015-02-20 18:27:06.19908	2015-02-20 18:27:06.19908	2	accepted	\N	\N	1
353	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	Resistance or Non-Response	Supports	1	1	14	7	62	108	2015-02-20 18:27:06.255228	2015-02-20 18:27:06.255228	3	accepted	\N	\N	1
354	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	N/A	Does Not Support	1	1	14	7	62	109	2015-02-20 18:27:06.321025	2015-02-20 18:27:06.321025	3	accepted	\N	\N	1
355	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	3	59	11	197	110	2015-02-20 18:27:06.376808	2015-02-20 18:27:06.376808	4	accepted	\N	\N	1
356	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	3	60	11	198	110	2015-02-20 18:27:06.445655	2015-02-20 18:27:06.445655	3	accepted	\N	\N	1
357	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	49	7	153	110	2015-02-20 18:27:06.516004	2015-02-20 18:27:06.516004	4	accepted	\N	\N	1
358	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	14	7	199	110	2015-02-20 18:27:06.600217	2015-02-20 18:27:06.600217	3	accepted	\N	\N	1
359	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	Resistance or Non-Response	Supports	1	4	12	7	199	110	2015-02-20 18:27:06.654192	2015-02-20 18:27:06.654192	3	accepted	\N	\N	1
360	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	4	3	7	199	110	2015-02-20 18:27:06.714705	2015-02-20 18:27:06.714705	3	accepted	\N	\N	1
361	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	1	1	41	39	201	111	2015-02-20 18:27:06.78724	2015-02-20 18:27:06.78724	5	accepted	\N	\N	1
362	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	2	1	3	20	91	112	2015-02-20 18:27:06.88976	2015-02-20 18:27:06.88976	4	accepted	\N	\N	1
363	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	Sensitivity	Supports	1	3	3	3	202	113	2015-02-20 18:27:06.957806	2015-02-20 18:27:06.957806	4	accepted	\N	\N	1
364	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	Poor Outcome	Does Not Support	3	1	3	37	203	113	2015-02-20 18:27:07.028568	2015-02-20 18:27:07.028568	3	accepted	\N	\N	1
365	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	1	1	41	39	201	114	2015-02-20 18:27:07.092924	2015-02-20 18:27:07.092924	5	accepted	\N	\N	1
366	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	1	3	61	40	204	115	2015-02-20 18:27:07.162529	2015-02-20 18:27:07.162529	3	accepted	\N	\N	1
367	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	1	3	61	40	204	116	2015-02-20 18:27:07.233398	2015-02-20 18:27:07.233398	3	accepted	\N	\N	1
368	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	Positive	Supports	2	1	3	40	205	116	2015-02-20 18:27:07.304108	2015-02-20 18:27:07.304108	5	accepted	\N	\N	1
369	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.	Sensitivity	Supports	1	3	62	40	206	116	2015-02-20 18:27:07.378151	2015-02-20 18:27:07.378151	3	accepted	\N	\N	1
370	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	Poor Outcome	Supports	3	1	3	40	207	116	2015-02-20 18:27:07.435728	2015-02-20 18:27:07.435728	4	accepted	\N	\N	1
371	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	3	4	3	27	208	117	2015-02-20 18:27:07.509487	2015-02-20 18:27:07.509487	3	accepted	\N	\N	1
372	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	3	4	3	27	208	118	2015-02-20 18:27:07.574455	2015-02-20 18:27:07.574455	3	accepted	\N	\N	1
373	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	Sensitivity	Supports	1	3	63	11	209	119	2015-02-20 18:27:07.63396	2015-02-20 18:27:07.63396	3	accepted	\N	\N	1
374	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	1	3	11	210	119	2015-02-20 18:27:07.706875	2015-02-20 18:27:07.706875	3	accepted	\N	\N	1
375	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	1	3	11	211	120	2015-02-20 18:27:07.76689	2015-02-20 18:27:07.76689	3	accepted	\N	\N	1
376	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	1	3	11	210	120	2015-02-20 18:27:07.83128	2015-02-20 18:27:07.83128	3	accepted	\N	\N	1
377	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	Poor Outcome	Supports	3	1	3	11	210	121	2015-02-20 18:27:07.895082	2015-02-20 18:27:07.895082	3	accepted	\N	\N	1
378	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	3	1	3	11	211	122	2015-02-20 18:27:07.956319	2015-02-20 18:27:07.956319	3	accepted	\N	\N	1
379	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	3	1	3	11	211	123	2015-02-20 18:27:08.023246	2015-02-20 18:27:08.023246	3	accepted	\N	\N	1
380	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	1	3	11	211	124	2015-02-20 18:27:08.083164	2015-02-20 18:27:08.083164	3	accepted	\N	\N	1
381	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	1	3	11	210	124	2015-02-20 18:27:08.137936	2015-02-20 18:27:08.137936	3	accepted	\N	\N	1
382	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	1	3	11	211	125	2015-02-20 18:27:08.196273	2015-02-20 18:27:08.196273	3	accepted	\N	\N	1
383	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	1	3	11	210	125	2015-02-20 18:27:08.25515	2015-02-20 18:27:08.25515	3	accepted	\N	\N	1
384	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	Poor Outcome	Supports	3	1	3	11	211	126	2015-02-20 18:27:08.3126	2015-02-20 18:27:08.3126	3	accepted	\N	\N	1
385	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	Sensitivity	Supports	1	1	64	41	212	127	2015-02-20 18:27:08.42627	2015-02-20 18:27:08.42627	3	accepted	\N	\N	1
386	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	Sensitivity	Supports	1	3	60	4	213	127	2015-02-20 18:27:08.501655	2015-02-20 18:27:08.501655	3	accepted	\N	\N	1
387	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	1	1	64	42	214	128	2015-02-20 18:27:08.570451	2015-02-20 18:27:08.570451	3	accepted	\N	\N	1
388	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	1	1	64	42	212	129	2015-02-20 18:27:08.642683	2015-02-20 18:27:08.642683	3	accepted	\N	\N	1
389	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	Negative	Does Not Support	2	4	3	1	215	130	2015-02-20 18:27:08.709926	2015-02-20 18:27:08.709926	3	accepted	\N	\N	1
390	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N/A	Does Not Support	3	4	3	1	215	130	2015-02-20 18:27:08.77271	2015-02-20 18:27:08.77271	2	accepted	\N	\N	1
391	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N/A	Does Not Support	3	4	3	27	215	130	2015-02-20 18:27:08.841251	2015-02-20 18:27:08.841251	2	accepted	\N	\N	1
392	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	3	4	3	27	216	130	2015-02-20 18:27:08.908832	2015-02-20 18:27:08.908832	3	accepted	\N	\N	1
393	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	Negative	Does Not Support	2	4	3	1	215	131	2015-02-20 18:27:08.970336	2015-02-20 18:27:08.970336	3	accepted	\N	\N	1
394	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N/A	Does Not Support	3	4	3	1	215	131	2015-02-20 18:27:09.029921	2015-02-20 18:27:09.029921	2	accepted	\N	\N	1
395	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N/A	Does Not Support	3	4	3	27	215	131	2015-02-20 18:27:09.100482	2015-02-20 18:27:09.100482	2	accepted	\N	\N	1
396	After adjust for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	3	4	3	27	216	131	2015-02-20 18:27:09.164151	2015-02-20 18:27:09.164151	3	accepted	\N	\N	1
397	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	1	1	3	1	217	132	2015-02-20 18:27:09.220382	2015-02-20 18:27:09.220382	3	accepted	\N	\N	1
398	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	N/A	Does Not Support	3	1	3	1	218	132	2015-02-20 18:27:09.287609	2015-02-20 18:27:09.287609	4	accepted	\N	\N	1
399	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N/A	Does Not Support	3	1	3	1	218	132	2015-02-20 18:27:09.342128	2015-02-20 18:27:09.342128	3	accepted	\N	\N	1
400	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	3	1	217	132	2015-02-20 18:27:09.400345	2015-02-20 18:27:09.400345	3	accepted	\N	\N	1
401	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	3	1	217	132	2015-02-20 18:27:09.462655	2015-02-20 18:27:09.462655	3	accepted	\N	\N	1
402	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	3	1	3	1	219	132	2015-02-20 18:27:09.53108	2015-02-20 18:27:09.53108	3	accepted	\N	\N	1
403	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	3	1	218	132	2015-02-20 18:27:09.595327	2015-02-20 18:27:09.595327	3	accepted	\N	\N	1
404	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	Poor Outcome	Supports	3	1	3	1	220	132	2015-02-20 18:27:09.655902	2015-02-20 18:27:09.655902	2	accepted	\N	\N	1
405	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	Poor Outcome	Supports	3	1	3	1	220	132	2015-02-20 18:27:09.717407	2015-02-20 18:27:09.717407	2	accepted	\N	\N	1
406	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	1	1	3	1	217	133	2015-02-20 18:27:09.784985	2015-02-20 18:27:09.784985	3	accepted	\N	\N	1
407	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	N/A	Does Not Support	3	1	3	1	218	133	2015-02-20 18:27:09.893049	2015-02-20 18:27:09.893049	4	accepted	\N	\N	1
408	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N/A	Does Not Support	3	1	3	1	218	133	2015-02-20 18:27:09.950549	2015-02-20 18:27:09.950549	3	accepted	\N	\N	1
409	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	3	1	217	133	2015-02-20 18:27:10.010736	2015-02-20 18:27:10.010736	3	accepted	\N	\N	1
410	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	3	1	217	133	2015-02-20 18:27:10.064746	2015-02-20 18:27:10.064746	3	accepted	\N	\N	1
411	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	3	1	3	1	219	133	2015-02-20 18:27:10.126247	2015-02-20 18:27:10.126247	3	accepted	\N	\N	1
412	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	3	1	218	133	2015-02-20 18:27:10.182076	2015-02-20 18:27:10.182076	3	accepted	\N	\N	1
\.


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 412, true);


--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 47, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-02-20 18:26:29.340855	2015-02-20 18:26:29.340855
1	2	2015-02-20 18:26:29.346139	2015-02-20 18:26:29.346139
2	3	2015-02-20 18:26:29.595526	2015-02-20 18:26:29.595526
2	4	2015-02-20 18:26:29.601177	2015-02-20 18:26:29.601177
3	5	2015-02-20 18:26:29.803197	2015-02-20 18:26:29.803197
3	6	2015-02-20 18:26:29.808182	2015-02-20 18:26:29.808182
4	7	2015-02-20 18:26:30.01396	2015-02-20 18:26:30.01396
4	8	2015-02-20 18:26:30.018793	2015-02-20 18:26:30.018793
5	9	2015-02-20 18:26:30.227857	2015-02-20 18:26:30.227857
5	10	2015-02-20 18:26:30.232488	2015-02-20 18:26:30.232488
6	11	2015-02-20 18:26:30.436663	2015-02-20 18:26:30.436663
6	12	2015-02-20 18:26:30.441383	2015-02-20 18:26:30.441383
7	11	2015-02-20 18:26:30.640146	2015-02-20 18:26:30.640146
7	12	2015-02-20 18:26:30.643526	2015-02-20 18:26:30.643526
8	11	2015-02-20 18:26:30.839556	2015-02-20 18:26:30.839556
8	12	2015-02-20 18:26:30.84283	2015-02-20 18:26:30.84283
9	13	2015-02-20 18:26:31.05096	2015-02-20 18:26:31.05096
10	14	2015-02-20 18:26:31.251632	2015-02-20 18:26:31.251632
10	15	2015-02-20 18:26:31.256485	2015-02-20 18:26:31.256485
11	14	2015-02-20 18:26:31.453989	2015-02-20 18:26:31.453989
11	15	2015-02-20 18:26:31.457137	2015-02-20 18:26:31.457137
12	16	2015-02-20 18:26:31.66772	2015-02-20 18:26:31.66772
12	17	2015-02-20 18:26:31.672357	2015-02-20 18:26:31.672357
13	18	2015-02-20 18:26:31.913463	2015-02-20 18:26:31.913463
13	19	2015-02-20 18:26:31.918106	2015-02-20 18:26:31.918106
13	20	2015-02-20 18:26:31.922394	2015-02-20 18:26:31.922394
14	21	2015-02-20 18:26:32.118393	2015-02-20 18:26:32.118393
15	21	2015-02-20 18:26:32.557485	2015-02-20 18:26:32.557485
15	22	2015-02-20 18:26:32.563092	2015-02-20 18:26:32.563092
16	23	2015-02-20 18:26:32.770205	2015-02-20 18:26:32.770205
16	24	2015-02-20 18:26:32.774962	2015-02-20 18:26:32.774962
17	25	2015-02-20 18:26:32.987426	2015-02-20 18:26:32.987426
17	26	2015-02-20 18:26:32.99256	2015-02-20 18:26:32.99256
18	27	2015-02-20 18:26:33.199976	2015-02-20 18:26:33.199976
18	28	2015-02-20 18:26:33.205306	2015-02-20 18:26:33.205306
19	29	2015-02-20 18:26:33.406034	2015-02-20 18:26:33.406034
19	30	2015-02-20 18:26:33.410717	2015-02-20 18:26:33.410717
20	31	2015-02-20 18:26:33.609803	2015-02-20 18:26:33.609803
20	32	2015-02-20 18:26:33.618461	2015-02-20 18:26:33.618461
21	31	2015-02-20 18:26:33.818793	2015-02-20 18:26:33.818793
21	32	2015-02-20 18:26:33.822243	2015-02-20 18:26:33.822243
22	33	2015-02-20 18:26:34.022004	2015-02-20 18:26:34.022004
22	18	2015-02-20 18:26:34.024733	2015-02-20 18:26:34.024733
33	34	2015-02-20 18:26:36.161892	2015-02-20 18:26:36.161892
33	18	2015-02-20 18:26:36.165141	2015-02-20 18:26:36.165141
40	35	2015-02-20 18:26:37.542746	2015-02-20 18:26:37.542746
40	36	2015-02-20 18:26:37.547656	2015-02-20 18:26:37.547656
42	37	2015-02-20 18:26:38.010345	2015-02-20 18:26:38.010345
42	38	2015-02-20 18:26:38.015578	2015-02-20 18:26:38.015578
46	39	2015-02-20 18:26:38.828787	2015-02-20 18:26:38.828787
46	40	2015-02-20 18:26:38.833879	2015-02-20 18:26:38.833879
\.


--
-- Data for Name: notifications; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY notifications (id, subscription_id, user_id, subscribable_id, subscribable_type, content, url, acknowledged, delivered, created_at, updated_at) FROM stdin;
\.


--
-- Name: notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('notifications_id_seq', 1, false);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20150220182109
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
20150220160805
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 220, true);


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
9	KIT Exon 17	\N	2015-02-20 18:26:57.930819	2015-02-20 18:26:57.930819
10	KIT Exon 11	\N	2015-02-20 18:26:57.995818	2015-02-20 18:26:57.995818
1	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-02-20 18:26:43.673714	2015-02-20 18:27:10.274542
2	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-02-20 18:26:43.989166	2015-02-20 18:27:10.279122
5	Erlotinib Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-02-20 18:26:51.748762	2015-02-20 18:27:10.282634
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-02-20 18:26:43.994696	2015-02-20 18:27:10.285449
4	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-02-20 18:26:45.268916	2015-02-20 18:27:10.2881
6	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-02-20 18:26:52.235329	2015-02-20 18:27:10.291248
7	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-02-20 18:26:53.158125	2015-02-20 18:27:10.294486
8	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-02-20 18:26:54.086752	2015-02-20 18:27:10.297992
11	NPM1 exon 12	To be filled in by Dr. KK, resident blood expert and knower of all things hematology. :) :) :) 	2015-02-20 18:27:03.639945	2015-02-20 18:27:10.301481
12	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-02-20 18:27:07.005918	2015-02-20 18:27:10.304894
13	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-02-20 18:27:07.213641	2015-02-20 18:27:10.30727
14	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-02-20 18:27:08.47467	2015-02-20 18:27:10.30979
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
3	1	2015-02-20 18:26:43.685172	2015-02-20 18:26:43.685172
2	1	2015-02-20 18:26:43.757823	2015-02-20 18:26:43.757823
2	1	2015-02-20 18:26:43.826937	2015-02-20 18:26:43.826937
6	2	2015-02-20 18:26:43.991953	2015-02-20 18:26:43.991953
6	3	2015-02-20 18:26:43.99711	2015-02-20 18:26:43.99711
5	2	2015-02-20 18:26:44.070317	2015-02-20 18:26:44.070317
8	3	2015-02-20 18:26:44.489228	2015-02-20 18:26:44.489228
7	2	2015-02-20 18:26:44.612176	2015-02-20 18:26:44.612176
7	3	2015-02-20 18:26:44.616679	2015-02-20 18:26:44.616679
11	4	2015-02-20 18:26:45.272402	2015-02-20 18:26:45.272402
13	4	2015-02-20 18:26:47.207102	2015-02-20 18:26:47.207102
15	4	2015-02-20 18:26:47.270021	2015-02-20 18:26:47.270021
16	4	2015-02-20 18:26:47.336432	2015-02-20 18:26:47.336432
34	5	2015-02-20 18:26:51.752143	2015-02-20 18:26:51.752143
35	6	2015-02-20 18:26:52.238179	2015-02-20 18:26:52.238179
36	6	2015-02-20 18:26:52.301758	2015-02-20 18:26:52.301758
38	6	2015-02-20 18:26:52.427423	2015-02-20 18:26:52.427423
41	6	2015-02-20 18:26:52.703711	2015-02-20 18:26:52.703711
43	6	2015-02-20 18:26:52.881671	2015-02-20 18:26:52.881671
44	6	2015-02-20 18:26:52.954854	2015-02-20 18:26:52.954854
45	6	2015-02-20 18:26:53.02025	2015-02-20 18:26:53.02025
46	7	2015-02-20 18:26:53.160615	2015-02-20 18:26:53.160615
46	7	2015-02-20 18:26:53.224788	2015-02-20 18:26:53.224788
47	7	2015-02-20 18:26:53.340778	2015-02-20 18:26:53.340778
47	7	2015-02-20 18:26:53.407001	2015-02-20 18:26:53.407001
48	7	2015-02-20 18:26:53.524664	2015-02-20 18:26:53.524664
48	7	2015-02-20 18:26:53.613905	2015-02-20 18:26:53.613905
49	7	2015-02-20 18:26:53.738694	2015-02-20 18:26:53.738694
49	7	2015-02-20 18:26:53.803861	2015-02-20 18:26:53.803861
50	7	2015-02-20 18:26:53.920958	2015-02-20 18:26:53.920958
50	7	2015-02-20 18:26:53.99068	2015-02-20 18:26:53.99068
56	8	2015-02-20 18:26:54.090272	2015-02-20 18:26:54.090272
59	8	2015-02-20 18:26:54.208132	2015-02-20 18:26:54.208132
70	9	2015-02-20 18:26:57.933637	2015-02-20 18:26:57.933637
71	10	2015-02-20 18:26:57.998301	2015-02-20 18:26:57.998301
71	10	2015-02-20 18:26:58.055721	2015-02-20 18:26:58.055721
71	10	2015-02-20 18:26:58.11701	2015-02-20 18:26:58.11701
71	10	2015-02-20 18:26:58.172782	2015-02-20 18:26:58.172782
71	10	2015-02-20 18:26:58.237399	2015-02-20 18:26:58.237399
71	10	2015-02-20 18:26:58.301898	2015-02-20 18:26:58.301898
74	9	2015-02-20 18:26:58.503624	2015-02-20 18:26:58.503624
74	9	2015-02-20 18:26:58.564369	2015-02-20 18:26:58.564369
76	10	2015-02-20 18:26:58.861691	2015-02-20 18:26:58.861691
76	10	2015-02-20 18:26:58.934423	2015-02-20 18:26:58.934423
76	10	2015-02-20 18:26:59.122056	2015-02-20 18:26:59.122056
77	10	2015-02-20 18:26:59.196442	2015-02-20 18:26:59.196442
77	10	2015-02-20 18:26:59.279096	2015-02-20 18:26:59.279096
91	11	2015-02-20 18:27:03.644305	2015-02-20 18:27:03.644305
91	11	2015-02-20 18:27:03.71323	2015-02-20 18:27:03.71323
91	11	2015-02-20 18:27:03.829357	2015-02-20 18:27:03.829357
91	11	2015-02-20 18:27:03.900927	2015-02-20 18:27:03.900927
101	1	2015-02-20 18:27:04.864504	2015-02-20 18:27:04.864504
102	1	2015-02-20 18:27:04.926904	2015-02-20 18:27:04.926904
102	1	2015-02-20 18:27:04.998283	2015-02-20 18:27:04.998283
102	1	2015-02-20 18:27:05.070214	2015-02-20 18:27:05.070214
102	1	2015-02-20 18:27:05.137753	2015-02-20 18:27:05.137753
103	1	2015-02-20 18:27:05.20429	2015-02-20 18:27:05.20429
104	1	2015-02-20 18:27:05.272019	2015-02-20 18:27:05.272019
105	1	2015-02-20 18:27:05.382846	2015-02-20 18:27:05.382846
113	12	2015-02-20 18:27:07.009181	2015-02-20 18:27:07.009181
113	12	2015-02-20 18:27:07.076323	2015-02-20 18:27:07.076323
115	13	2015-02-20 18:27:07.216409	2015-02-20 18:27:07.216409
116	13	2015-02-20 18:27:07.283263	2015-02-20 18:27:07.283263
127	14	2015-02-20 18:27:08.477646	2015-02-20 18:27:08.477646
127	14	2015-02-20 18:27:08.549779	2015-02-20 18:27:08.549779
128	14	2015-02-20 18:27:08.621439	2015-02-20 18:27:08.621439
129	14	2015-02-20 18:27:08.690246	2015-02-20 18:27:08.690246
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 14, true);


--
-- Name: variant_origins_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_origins_id_seq', 1, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 133, true);


--
-- PostgreSQL database dump complete
--

