--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	026c10f9-f937-4efd-9536-46578fe8aaf5	2015-02-18 20:00:00.931441
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	00ed8d4a-ff6a-4165-a716-353563d2ddaf	2015-02-18 20:00:01.402788
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	a691b638-6906-4b77-aa65-a2144d107c96	2015-02-18 20:00:01.797722
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	f750f24a-be24-4afb-add2-0ffd45214a7f	2015-02-18 20:00:02.441972
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	c4037fdb-98b4-4fca-9056-2219e1a19e81	2015-02-18 20:00:02.861713
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	f4f6e50f-9626-4ae0-ba96-cc9c09546ddd	2015-02-18 20:00:03.187375
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	c764fe7d-e492-4ce1-8373-7c20b1c854e6	2015-02-18 20:00:03.480819
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	1d823b04-6c29-4672-8033-d2f1a070b7ca	2015-02-18 20:00:03.707398
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	39ec2a2f-2eaa-4448-9e9f-e1ce9cd970ee	2015-02-18 20:00:03.956822
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	bc4756a8-30e4-448e-9fc0-c00f80e50e51	2015-02-18 20:00:04.210075
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	a369b39d-f6a3-4e01-b3da-af87227e6612	2015-02-18 20:00:04.426298
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	dff10cb8-1c9b-4047-8efa-21ce1b7458b6	2015-02-18 20:00:04.657664
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	e07cae9d-6c95-4e55-af2a-c1b573a77e9c	2015-02-18 20:00:04.880911
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	d0ef9eba-0fa2-4bd7-8f7f-9822e6962258	2015-02-18 20:00:05.07648
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: '"PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity."'\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	5d0f20dc-d9a4-47a1-b636-fbbf445bc531	2015-02-18 20:00:05.319033
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	224c4d19-7af0-4789-b1ad-ad1b45e9dd44	2015-02-18 20:00:05.542527
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	109e519a-46e9-495b-83ad-8fc185c33045	2015-02-18 20:00:05.800304
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown senitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	62448a53-e786-4b2e-8f42-7c4b2955957c	2015-02-18 20:00:06.024093
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	82bdd132-3b19-4191-91bd-9282194bfe01	2015-02-18 20:00:06.261131
158	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	516618e7-ab46-4a1a-8c5a-dd30210c4f69	2015-02-18 20:00:15.721401
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions have been demonstrated\n  to have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	9af159c7-534c-4350-8b23-33f32593a73c	2015-02-18 20:00:06.482742
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to the activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFRs can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions have been demonstrated\n  to have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	34ddc026-6438-4f82-930c-9df449f775b7	2015-02-18 20:00:06.690258
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	c4edf69e-76c1-4744-916f-e05215db5173	2015-02-18 20:00:06.921935
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: N/A\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	91c8b30b-4d4a-4818-9ea0-b90ca088231c	2015-02-18 20:00:07.143388
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, and appear\n  to behave quite differently in different environments. In myelodysplastic syndromes\n  and acute myeloid leukemia (AML), IDH1 mutations have been shown to be associated\n  with worse outcome and shorter overall survival. They have also been linked to cytogenetically\n  normal AML, an intermediate risk subtype of the disease. However, in glioblastoma\n  and astrocytoma, patients with IDH1 mutations have shown better overall survival\n  than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1\n  mutations have been associated with specific cytogenetic abnormalities, 1p and 19q\n  deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	708feab1-66a3-447e-befd-b1a80c375281	2015-02-18 20:00:07.339698
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: N/A\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	ece5bb56-a9dc-427d-8a9b-049de3c78856	2015-02-18 20:00:07.550395
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	3ca5e8a4-f771-4d4c-8298-c62bdffcbd4c	2015-02-18 20:00:07.757992
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: N/A\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	800ba0b1-b2c9-4a62-8666-891d635e9fb6	2015-02-18 20:00:07.955688
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: N/A\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	420cdfe1-b475-49f0-b64c-a2c0d3a06d8a	2015-02-18 20:00:08.149551
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: N/A\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	8d315e51-110c-4769-9b80-8b67adac06ce	2015-02-18 20:00:08.354144
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: N/A\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	f0e66a69-1ef9-4a95-8d2a-d75ddbc1f660	2015-02-18 20:00:08.568503
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: N/A\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	abf5e046-51b3-43d5-bbfd-e1cdfe6c87fe	2015-02-18 20:00:08.793102
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: N/A\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	c910408b-1df7-445f-aca8-def2fa4cb4fa	2015-02-18 20:00:09.011595
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1' is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	98219ffa-86ce-4e2c-8cbf-45c158680669	2015-02-18 20:00:09.237042
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: N/A\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	55ea3d2c-9ade-4b2e-a89c-afff138ecf0d	2015-02-18 20:00:09.480728
50	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	65d9e09c-0144-485d-8970-7733fbf4ec43	2015-02-18 20:00:12.548817
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	09f3e039-c3cc-4abe-933f-24793b62a91d	2015-02-18 20:00:09.753787
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: N/A\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	6df6a734-4906-447e-8db7-989612d09806	2015-02-18 20:00:09.952894
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: N/A\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	8523e65c-54a5-445b-a42e-1ea56af05105	2015-02-18 20:00:10.180264
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARa\ndescription: N/A\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	4c26bb2f-a2aa-4abc-b992-0f012910952c	2015-02-18 20:00:10.371228
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: N/A\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	7c56d2ae-5a3e-4356-8dc7-67aeb2ac5a43	2015-02-18 20:00:10.571301
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	993c83c8-6712-4746-8c20-efff5e16fdc3	2015-02-18 20:00:10.923325
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: N/A\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	44e716f1-3893-49a5-8edc-aca7f671391c	2015-02-18 20:00:11.115451
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	68a2fce3-a6b8-4787-9de7-0f00db6be966	2015-02-18 20:00:11.345225
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: N/A\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	57d89cb5-6c3c-4c04-9c2e-f6076c695ff7	2015-02-18 20:00:11.572226
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: N/A\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	d249ed22-82af-47c9-8630-f0550623732d	2015-02-18 20:00:11.779693
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: N/A\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	243a41b2-5f71-4b68-adea-5c3e0f4d3151	2015-02-18 20:00:11.975841
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	8e503175-14b9-454f-89f5-188e46808272	2015-02-18 20:00:12.217749
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: N/A\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	a9f541ee-21cb-4e89-9f2f-80ca16acc909	2015-02-18 20:00:12.432939
48	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	e5c9366a-2111-4993-af10-86f309ece133	2015-02-18 20:00:12.487293
49	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	54ff0aec-4690-4b04-bdd8-441cf4008b6c	2015-02-18 20:00:12.522601
51	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	5dad2b0a-b84b-452d-a3ab-8d2eb72dfa50	2015-02-18 20:00:12.578631
52	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	e9b0ab61-7207-428b-a24c-bbd38eabf66b	2015-02-18 20:00:12.624772
53	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	53306b45-4fb7-44c3-a5d6-f622acab6a9d	2015-02-18 20:00:12.652342
54	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	9bc1a19b-0575-498d-aeac-adbd32e25033	2015-02-18 20:00:12.679266
55	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	a49c5e8e-a384-4ea3-ae35-9448ae29b14b	2015-02-18 20:00:12.708836
56	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	a8f9ba8a-3c7c-4099-8208-246bc5a61b2f	2015-02-18 20:00:12.733111
57	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	c3294764-b57a-485d-8dd4-234190480801	2015-02-18 20:00:12.758097
58	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	6f200656-6955-4a1e-b7f8-4803ba1e8851	2015-02-18 20:00:12.784132
59	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	473f1623-9295-4777-a55a-7f58950de4be	2015-02-18 20:00:12.813776
60	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	663f0c71-11a6-4257-8c3c-902a57ad8bab	2015-02-18 20:00:12.839705
61	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	1dab6587-f0db-4489-9ad9-4979106ccbe5	2015-02-18 20:00:12.865871
62	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	3c335e23-46bc-4760-91a4-4552f405bc27	2015-02-18 20:00:12.890835
63	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	2237d9d5-934c-41a4-9078-8afc016772ae	2015-02-18 20:00:12.917749
64	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	3dc355f2-c180-49fe-9ed4-854fc412b982	2015-02-18 20:00:12.944869
65	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	349dcbae-2836-44c6-8057-4d9586ebf343	2015-02-18 20:00:12.969958
66	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	4c78b681-4642-4b21-b3a9-b7e87c291c24	2015-02-18 20:00:12.995058
67	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	430f2b53-f6b2-4c17-9809-87a4f74afcd9	2015-02-18 20:00:13.023014
68	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	289b00a8-9dd5-4ffd-8d24-913ba28ca006	2015-02-18 20:00:13.051677
69	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	aead7443-8645-441a-b51b-a5083f979954	2015-02-18 20:00:13.083377
70	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	89871251-a2ed-4a19-95fc-87db34dfd075	2015-02-18 20:00:13.11914
71	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	1bfe32eb-909b-40ac-b575-1c6b85db771c	2015-02-18 20:00:13.144813
72	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	53da88c7-fcdd-4439-82d6-f4f412721eeb	2015-02-18 20:00:13.178069
73	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	79af8b53-47b7-4244-98de-332d912fad21	2015-02-18 20:00:13.20227
74	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	1dfd97b6-517f-46a4-b30c-2c2454033395	2015-02-18 20:00:13.241372
75	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	56193040-aae7-4cbe-b505-9207574f47b6	2015-02-18 20:00:13.272243
76	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	b0fe2848-fb5a-41b3-b8f2-f540bd807cc8	2015-02-18 20:00:13.298187
77	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	d3bf1177-3a62-4729-bc4e-6706ee439d3a	2015-02-18 20:00:13.341822
78	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	77100a53-6ef9-4865-bdbc-729ec39851bb	2015-02-18 20:00:13.366042
89	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	11c73867-078c-4bbd-a79c-3d04d9d64d90	2015-02-18 20:00:13.684509
79	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	e70a45d2-fd91-48d9-ba56-3fccda43a00d	2015-02-18 20:00:13.394982
80	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	edc6de76-35a7-4711-9871-a78e8164d4fd	2015-02-18 20:00:13.424086
81	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	94df6f17-8bbe-4906-897e-188ad0bb3775	2015-02-18 20:00:13.452596
82	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	75bf737d-010d-403f-a44b-abc629bf95f3	2015-02-18 20:00:13.487059
83	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	39a1beff-70a6-4f80-a64a-c3bd83350305	2015-02-18 20:00:13.51606
84	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	2fa92315-f922-4602-8b54-872cc97bc20c	2015-02-18 20:00:13.542342
85	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	9d4a7d04-fd3b-4e44-8f01-7cb616f1074f	2015-02-18 20:00:13.569819
86	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\n	1	\N	\N	fa0a3554-fb00-45f7-b721-437e9ac7d4d7	2015-02-18 20:00:13.599384
87	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	efd2e9e3-cbb6-4615-a91c-61a9e5715da1	2015-02-18 20:00:13.625416
88	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	fc966a8a-e6b5-42b3-96ca-85de294f82ae	2015-02-18 20:00:13.654707
141	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 3\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	67e5109e-2c91-41de-8269-44cddb73ae59	2015-02-18 20:00:15.226728
142	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 4\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	bff11d89-c280-42a7-bd2e-db3dcc3db171	2015-02-18 20:00:15.255899
90	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	e54677ed-8883-4612-8e6f-295ee34cc0ee	2015-02-18 20:00:13.710683
91	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	7868ddb6-2182-4c5f-9376-3e656eceb09e	2015-02-18 20:00:13.737351
92	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	7a32e7ed-84b4-40a1-b770-6a00acc00b2b	2015-02-18 20:00:13.763688
93	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	ebfe67c3-248b-4f01-8d79-c712479bbc18	2015-02-18 20:00:13.793042
94	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	ed8ac61c-0e89-4bc8-a4ca-f98a6f41d66e	2015-02-18 20:00:13.82023
95	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	a1830b2a-6ad8-4954-852f-1856787aeb7d	2015-02-18 20:00:13.846778
96	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	d3ff9c90-8cdf-4438-9e4e-31f7ee1a4aca	2015-02-18 20:00:13.874048
97	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	3ead3412-a11a-4e9d-b311-fb83e00748bf	2015-02-18 20:00:13.903144
98	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-AFF3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	60a3e397-2e66-4876-a023-04d6663421a3	2015-02-18 20:00:13.931279
99	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-BICC1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	cdf7de8c-d3d8-4336-b397-2000e3abb107	2015-02-18 20:00:13.955919
100	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CASP7\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	6b6294f9-fe9d-460a-8aa6-889f9a29acf5	2015-02-18 20:00:13.989872
143	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	f055d75f-b3d3-4006-a25e-982deeed8c73	2015-02-18 20:00:15.287759
101	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CCDC6\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	798d3dff-551a-4cf6-9c78-459082b7ac85	2015-02-18 20:00:14.042254
102	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-KIAA1967\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	44860787-15b5-49bb-9e69-25b0c96bffe4	2015-02-18 20:00:14.067933
103	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	fdb99d90-5ef0-4316-ba4a-c9f0112c4982	2015-02-18 20:00:14.097375
104	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-OFD1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	ece67506-ab00-44f5-af32-34ab02449d30	2015-02-18 20:00:14.127393
105	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-SLC45A3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	2dcd7198-497d-4929-b604-5fe8b0c5a0e3	2015-02-18 20:00:14.161831
106	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	80899211-a367-418d-8a0f-37251f103732	2015-02-18 20:00:14.190375
107	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: ITD\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	da5fe6d8-a125-4a4e-a56d-e0ecc4130051	2015-02-18 20:00:14.222032
108	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: TKD\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	f15c8b53-9940-4e20-b474-a1b2efa224cc	2015-02-18 20:00:14.254701
109	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	f9782f64-b626-48ec-81c4-d521f920e591	2015-02-18 20:00:14.286563
110	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	a89f3f2a-b865-4f0a-a0bd-5d9494cb920e	2015-02-18 20:00:14.315689
111	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	25161b48-ded0-43bd-a329-7473102109eb	2015-02-18 20:00:14.345099
144	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	388e756b-d3df-4616-a8ee-9c44d5d99ba1	2015-02-18 20:00:15.313873
112	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	a13d912b-c95b-43d5-a0a3-9b74601b7b6d	2015-02-18 20:00:14.374088
113	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	782939fb-85dd-4b4c-ba13-6b92f5f48d15	2015-02-18 20:00:14.400924
114	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	88a0a81f-1e62-4fcc-9b59-5ec4c3d796e2	2015-02-18 20:00:14.433019
115	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	159ff1a6-4cea-4493-9fbe-6722756b51c1	2015-02-18 20:00:14.461625
116	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	443d5fcc-5a3f-4dda-9d09-d68e2ec9cdc3	2015-02-18 20:00:14.491498
117	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	a64a9dd7-a0d1-403e-a72f-cbd931b1091b	2015-02-18 20:00:14.519657
118	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 11\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	af7bba36-081f-42cf-b83b-c9193318f094	2015-02-18 20:00:14.547282
119	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 13\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	fa4fdc4c-6939-4a8c-8f2e-009531e8b0e4	2015-02-18 20:00:14.573114
120	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 14\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	552ccb88-dc6c-47f1-8f86-e53c7edf9280	2015-02-18 20:00:14.601517
121	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 17\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\n	1	\N	\N	1c28c9c6-af15-43d6-9d0d-09652b13c113	2015-02-18 20:00:14.629828
122	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 9\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	4a3dc601-bb4a-4c6e-988a-1ab442f6168c	2015-02-18 20:00:14.656384
123	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	82c61a4d-7012-4fe3-9e3a-16bef8814ebc	2015-02-18 20:00:14.685263
124	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	115e9be3-198c-4de0-b2b9-de3dc809873f	2015-02-18 20:00:14.71491
125	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 1\ndescription: A study by Livre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	066d3b4e-99a0-498a-84e4-bd484a640aeb	2015-02-18 20:00:14.76702
126	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 2\ndescription: In a study by Pan et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	e38e9549-6dbf-407b-b77e-6385a03466e9	2015-02-18 20:00:14.796438
127	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	8a621c77-05ae-40f8-a217-93fd79058abe	2015-02-18 20:00:14.821593
128	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	3d639ffd-d77b-4018-9f1a-219b0bbea3a5	2015-02-18 20:00:14.85077
129	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	1b8d85eb-3b6b-434f-ac6f-5392bfd90f2f	2015-02-18 20:00:14.877459
130	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	3fc814ba-a232-4f06-b686-c05dd3f7e932	2015-02-18 20:00:14.909416
131	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	657afa68-fc30-4a82-a8c3-c67dc14c2456	2015-02-18 20:00:14.935061
132	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	417d0f9c-a10b-43ac-acd0-d1e072c66842	2015-02-18 20:00:14.962838
133	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	dfcf829e-0a81-48b3-a2ac-da9a3e2b4d6a	2015-02-18 20:00:14.991735
134	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	5604e545-b685-4733-a188-a68d16ebc34a	2015-02-18 20:00:15.019572
135	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	93b3ef4e-bba6-4ef9-9374-c7a45078cc82	2015-02-18 20:00:15.047833
136	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	b941509e-bc83-4ccf-bf7a-32d7ea6e3738	2015-02-18 20:00:15.07489
137	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: EXON 12\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	c42823b4-a27d-4fa6-8fb8-e1ea93ea4c4c	2015-02-18 20:00:15.101542
138	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	02dfc864-6a49-4962-b95c-755a81983ef2	2015-02-18 20:00:15.130593
139	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 1\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	006a893b-f097-49be-b326-9b9f727229ec	2015-02-18 20:00:15.166887
140	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 2\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	dbe71f06-1f48-43d6-ab18-3645c3b45dc4	2015-02-18 20:00:15.19891
145	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	62cc0412-d64a-4a1c-9282-d0fd031e99c3	2015-02-18 20:00:15.339195
146	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	38b1e498-d06c-4172-bae7-dcbc9894192a	2015-02-18 20:00:15.363289
147	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	670312bd-5836-4e08-89f8-0ece93558bd8	2015-02-18 20:00:15.39026
148	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	8ec2b4eb-eabd-4025-83ab-a61b1841ba79	2015-02-18 20:00:15.415849
149	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	d196d0a5-1584-40a1-a629-b83c70e755d2	2015-02-18 20:00:15.46081
150	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	bfdf27c7-359e-45f0-b6af-7f4c72d34643	2015-02-18 20:00:15.490585
151	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	1810e302-57cd-455f-a8e4-6adc007a8313	2015-02-18 20:00:15.515087
152	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	8900c92f-413b-44b2-b9ea-4460b30786aa	2015-02-18 20:00:15.545935
153	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	9cff63ea-2db0-48e1-83a7-a910f24fe5e5	2015-02-18 20:00:15.57192
154	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	cc2a8b07-36ac-4df2-ba7f-72950204278d	2015-02-18 20:00:15.604867
155	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 20\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	4e960078-c5da-48f6-93ed-5eba9fda4e5b	2015-02-18 20:00:15.630277
156	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 9\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	5a968e44-6bf6-416b-aa83-97b310b035b8	2015-02-18 20:00:15.658008
157	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	ea9b9c84-4d10-44b4-82c6-b209f362d51c	2015-02-18 20:00:15.692174
159	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	1964e78a-1a3c-4eaa-9f64-f3530de6e685	2015-02-18 20:00:15.745421
160	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	037441e4-d7db-4f43-bcd7-05343265db3e	2015-02-18 20:00:15.777261
161	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	e84f3840-877e-4b2c-a14a-a6dbbd4079f5	2015-02-18 20:00:15.803527
162	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	9b207908-a72a-4afb-8b08-f1b441801f0f	2015-02-18 20:00:15.839285
163	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	8a31cc3d-2087-4a13-9978-020e73a67c9f	2015-02-18 20:00:15.862268
164	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	4383f3df-f658-4901-b444-86bb631ee625	2015-02-18 20:00:15.889614
165	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	6e923500-7ead-4f39-8069-0088f03ee93e	2015-02-18 20:00:15.912627
166	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	08cc8e62-5559-4134-82c0-ce161ea0172a	2015-02-18 20:00:15.936788
167	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	eab2e91a-21d7-487e-ac91-95c67ecba4c6	2015-02-18 20:00:15.960958
168	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	0b83e112-cb61-45f5-8971-90dfa1927960	2015-02-18 20:00:15.993261
169	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	f5ea1e7c-2ea1-46da-8c15-b489ad3abea1	2015-02-18 20:00:16.025195
170	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	14f0ff93-9507-42b6-8f96-5926d26c818c	2015-02-18 20:00:16.059353
171	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	d433d9dc-b9a2-4f13-8f90-ea31a9e738c8	2015-02-18 20:00:16.085551
240	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Frequency of NPM1 mutations was not different in normal karyotype AML patients\n  with CEPBA, NRAS or KIT mutations.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 59\nvariant_id: 90\nrating: 3\n	1	\N	\N	0e742e1a-7c97-4355-a335-a74b9757ebae	2015-02-18 20:00:20.10073
172	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	f2341e97-27e0-43a4-8b9a-c938c5445d0a	2015-02-18 20:00:16.133931
173	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	57af8a48-8466-4165-8d11-d55e42493d43	2015-02-18 20:00:16.160391
174	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	87de4931-5f69-4520-9dac-312cf4719a94	2015-02-18 20:00:16.187308
175	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	0db701f9-a591-43a4-b8f3-13e5c2de7bf3	2015-02-18 20:00:16.214181
176	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	97851788-7f63-4bf3-8e96-3d0f8b5731c5	2015-02-18 20:00:16.246682
177	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\n	1	\N	\N	f139b807-44ee-4dfe-bb21-64fda14ef9a4	2015-02-18 20:00:16.276351
178	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\n	1	\N	\N	bbb102f2-e2ff-4fb0-ab76-680417ad1299	2015-02-18 20:00:16.30551
179	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 7\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	025ce44d-080f-40ff-9b01-26dd6d9ba5bd	2015-02-18 20:00:16.335633
180	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 9\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	57840617-fac9-418a-b364-0948b6f0d90e	2015-02-18 20:00:16.364712
181	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective in acheiving stable disease.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 41\nvariant_id: 5\nrating: 4\n	1	\N	\N	6da2686e-dd1d-420f-b48a-69731f1f82a6	2015-02-18 20:00:16.514443
182	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 41\nvariant_id: 6\nrating: 4\n	1	\N	\N	5d3101e0-a997-42dd-bdbf-aa72239afdd9	2015-02-18 20:00:16.609224
183	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 41\nvariant_id: 7\nrating: 4\n	1	\N	\N	05f6426d-3e0c-48a1-a0e9-b6839d1d1f7c	2015-02-18 20:00:16.679207
184	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 1\nsource_id: 42\nvariant_id: 8\nrating: 3\n	1	\N	\N	fd8ce56b-e7b7-475f-9505-663bb94a8833	2015-02-18 20:00:16.749142
185	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 2\nsource_id: 43\nvariant_id: 31\nrating: 4\n	1	\N	\N	4182f45a-abfe-4864-9fc1-1c51b9120347	2015-02-18 20:00:16.836399
186	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 2\nsource_id: 43\nvariant_id: 112\nrating: 4\n	1	\N	\N	c0955d99-9fb2-4085-a3e2-25d9e5ee2edf	2015-02-18 20:00:16.899802
187	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 3\nsource_id: 44\nvariant_id: 46\nrating: 5\n	1	\N	\N	ec6b8117-507e-4bd9-9639-74b5322d8b9e	2015-02-18 20:00:16.963124
188	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 4\ndisease_id: 3\nsource_id: 44\nvariant_id: 46\nrating: 5\n	1	\N	\N	7738cd1e-7deb-49aa-b92b-067869c0ae53	2015-02-18 20:00:17.014864
189	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 3\nsource_id: 45\nvariant_id: 46\nrating: 3\n	1	\N	\N	267cdaa3-6c8d-49dd-932e-058ad93b4667	2015-02-18 20:00:17.075344
190	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 3\nsource_id: 44\nvariant_id: 47\nrating: 5\n	1	\N	\N	06eea128-853e-4aed-8709-4030f483dbd4	2015-02-18 20:00:17.138699
191	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 4\ndisease_id: 3\nsource_id: 44\nvariant_id: 47\nrating: 5\n	1	\N	\N	660f8ef6-fbcb-4778-8798-93937458b1e9	2015-02-18 20:00:17.187831
192	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 3\nsource_id: 45\nvariant_id: 47\nrating: 3\n	1	\N	\N	bc863daf-35aa-464d-bdb2-c863ca60963b	2015-02-18 20:00:17.253675
193	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 3\nsource_id: 44\nvariant_id: 48\nrating: 5\n	1	\N	\N	088d61e2-6a5d-4e23-8cc2-0bb644a14244	2015-02-18 20:00:17.313389
194	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 4\ndisease_id: 3\nsource_id: 44\nvariant_id: 48\nrating: 5\n	1	\N	\N	0fd7daca-2189-4597-86c2-0b63a5613b93	2015-02-18 20:00:17.356182
195	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 3\nsource_id: 45\nvariant_id: 48\nrating: 3\n	1	\N	\N	f98397eb-2000-450a-b6e6-73cfc2e1a861	2015-02-18 20:00:17.420884
196	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 3\nsource_id: 44\nvariant_id: 49\nrating: 5\n	1	\N	\N	99d73986-71ff-4f78-87ce-f560f74123c1	2015-02-18 20:00:17.500913
197	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 4\ndisease_id: 3\nsource_id: 44\nvariant_id: 49\nrating: 5\n	1	\N	\N	b455b8d5-14cd-47c0-af17-5b64b32780a1	2015-02-18 20:00:17.545296
198	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 3\nsource_id: 45\nvariant_id: 49\nrating: 3\n	1	\N	\N	6ea342d0-d3d0-4eae-ade9-b6d689d54bc0	2015-02-18 20:00:17.610755
199	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 3\nsource_id: 44\nvariant_id: 50\nrating: 5\n	1	\N	\N	58a93045-d59d-4c52-a6be-040d4f87d703	2015-02-18 20:00:17.674141
200	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 4\ndisease_id: 3\nsource_id: 44\nvariant_id: 50\nrating: 5\n	1	\N	\N	8981d65a-3152-4249-af6a-cd8614a55285	2015-02-18 20:00:17.717497
201	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 3\nsource_id: 45\nvariant_id: 50\nrating: 3\n	1	\N	\N	48650b76-3b81-4431-86bc-495d8e37bfb3	2015-02-18 20:00:17.777047
202	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 4\nsource_id: 46\nvariant_id: 76\nrating: 3\n	1	\N	\N	1f508613-4a2a-4256-9405-f083cf7caf08	2015-02-18 20:00:17.846878
203	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 6\ndisease_id: 4\nsource_id: 46\nvariant_id: 76\nrating: 3\n	1	\N	\N	be02590d-039e-45f6-9b8f-b0e60031fbd6	2015-02-18 20:00:17.896988
204	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 76\nrating: 3\n	1	\N	\N	ba0e0747-1be7-4bc7-b896-3ec7191de7b1	2015-02-18 20:00:17.942138
205	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 8\ndisease_id: 1\nsource_id: 47\nvariant_id: 76\nrating: 3\n	1	\N	\N	c567fcf6-dbb2-4a19-8bce-f97e0545c22e	2015-02-18 20:00:18.016023
206	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 1\nsource_id: 47\nvariant_id: 76\nrating: 3\n	1	\N	\N	2d722ea5-e3eb-4084-871d-2b78a3a014b3	2015-02-18 20:00:18.083098
207	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 6\ndisease_id: 1\nsource_id: 47\nvariant_id: 76\nrating: 3\n	1	\N	\N	24d85bd7-e36c-4bf4-950f-4159b4195712	2015-02-18 20:00:18.124018
208	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 8\ndisease_id: 4\nsource_id: 46\nvariant_id: 76\nrating: 3\n	1	\N	\N	254408ed-876d-4d5b-b8bc-ab609a869750	2015-02-18 20:00:18.180228
209	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a case study, a patient with anal melanoma harboring an exon 11 KIT mutation\n  showed marked response 4 months after imatinib treatment.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 4\nsource_id: 48\nvariant_id: 76\nrating: 3\n	1	\N	\N	241b8b7b-17a5-4a89-bd3c-f83a806e00a0	2015-02-18 20:00:18.238022
210	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 5\nsource_id: 49\nvariant_id: 77\nrating: 3\n	1	\N	\N	6fcf0c5e-5bd3-4a88-95b0-c0eb5bb23a11	2015-02-18 20:00:18.293985
211	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 9\ndisease_id: 5\nsource_id: 50\nvariant_id: 77\nrating: 3\n	1	\N	\N	67276f8d-1467-4fc6-963a-a638ba3d6d9c	2015-02-18 20:00:18.354917
212	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 6\nsource_id: 51\nvariant_id: 70\nrating: 4\n	1	\N	\N	5b1db848-77df-466e-a0e0-f6a973ea274a	2015-02-18 20:00:18.418844
213	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 3\nsource_id: 52\nvariant_id: 51\nrating: 3\n	1	\N	\N	ba17ff83-8001-4cc9-9d5c-00bc9d169cd3	2015-02-18 20:00:18.486368
214	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 7\nsource_id: 52\nvariant_id: 52\nrating: 3\n	1	\N	\N	fea34dbe-6633-4910-be15-838a6da4d24d	2015-02-18 20:00:18.543871
215	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 3\nsource_id: 52\nvariant_id: 53\nrating: 3\n	1	\N	\N	a756c84a-c4cb-4101-9be0-5f8c090261f1	2015-02-18 20:00:18.602502
216	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 3\nsource_id: 52\nvariant_id: 54\nrating: 3\n	1	\N	\N	f2e7d977-b00e-47a3-b04a-37cb3b89a464	2015-02-18 20:00:18.679422
217	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 8\nsource_id: 52\nvariant_id: 55\nrating: 3\n	1	\N	\N	2dc36069-5098-4215-bed0-9f3179865213	2015-02-18 20:00:18.735965
218	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in "anti-tumor activitiy".\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 7\nsource_id: 53\nvariant_id: 56\nrating: 3\n	1	\N	\N	21074694-f0d2-4d83-ac82-0e43250ab22a	2015-02-18 20:00:18.788692
219	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 9\nsource_id: 52\nvariant_id: 57\nrating: 3\n	1	\N	\N	3f2132c9-dee1-4d1c-926c-61ca2ab61c2e	2015-02-18 20:00:18.849132
220	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 10\nsource_id: 52\nvariant_id: 58\nrating: 3\n	1	\N	\N	717532ae-83cc-465e-abf8-9fb4eb5d8185	2015-02-18 20:00:18.913738
241	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 60\nvariant_id: 90\nrating: 3\n	1	\N	\N	8e0401be-a596-458a-9c94-798e9a245515	2015-02-18 20:00:20.159022
221	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 7\nsource_id: 53\nvariant_id: 59\nrating: 3\n	1	\N	\N	51c6f17f-ce80-42b1-b1b5-97d2cd7124c4	2015-02-18 20:00:18.984164
222	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 7\nsource_id: 53\nvariant_id: 59\nrating: 3\n	1	\N	\N	81365f82-e82f-4363-b784-4dd28fc1055e	2015-02-18 20:00:19.031244
223	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratiinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 35\nrating: 4\n	1	\N	\N	d49f381f-a285-4bef-aee9-e4319386cdb7	2015-02-18 20:00:19.092178
224	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratiinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 36\nrating: 4\n	1	\N	\N	b9401efe-9044-4699-bf7c-3d069d5d7328	2015-02-18 20:00:19.145979
225	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratiinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 38\nrating: 5\n	1	\N	\N	d27ce77a-5dc0-450f-9c49-d7c4844a0a6e	2015-02-18 20:00:19.227394
226	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the in-frame insertion of a single amino acid at position\n  780 was shown to be sensitive to neratiinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 41\nrating: 5\n	1	\N	\N	e7f113e7-9776-4787-b3e8-30e9f3ade0ed	2015-02-18 20:00:19.286689
227	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratiinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 44\nrating: 5\n	1	\N	\N	dadbec8b-723b-420f-b241-dcff212fbae1	2015-02-18 20:00:19.343267
228	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratiinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 45\nrating: 5\n	1	\N	\N	b7e5cef6-e6a7-425f-912c-08a2a7ac34cb	2015-02-18 20:00:19.401383
229	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratiinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 43\nrating: 5\n	1	\N	\N	649b1a79-ffdb-463f-9afe-abc8b52d4516	2015-02-18 20:00:19.455257
230	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with the ABL T315I mutation, tumors have shown to be resistant to\n  imatinib treatment.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 11\nsource_id: 55\nvariant_id: 2\nrating: 4\n	1	\N	\N	b6bcf921-e8c7-4661-9492-61d6ff0a63d8	2015-02-18 20:00:19.519969
231	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines harboring T315I mutations were shown to be resistant to imatinib\n  and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 11\nsource_id: 56\nvariant_id: 2\nrating: 4\n	1	\N	\N	74421b2b-bce4-45de-9a4b-6dbc21b7e3e6	2015-02-18 20:00:19.576098
232	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines harboring E255K mutations were shown to be resistant to imatinib\n  and exhibited increased kinase activity.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 11\nsource_id: 56\nvariant_id: 3\nrating: 4\n	1	\N	\N	7df7bbe3-6baa-411b-9613-29e576f451dc	2015-02-18 20:00:19.633004
233	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 12\nsource_id: 57\nvariant_id: 115\nrating: 3\n	1	\N	\N	33a32cf2-2338-42df-9e41-4db1bdbdf833	2015-02-18 20:00:19.695821
234	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 12\nsource_id: 57\nvariant_id: 116\nrating: 3\n	1	\N	\N	915bd689-484e-455e-96f2-c65746a3c41a	2015-02-18 20:00:19.749995
235	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 4\nsource_id: 58\nvariant_id: 12\nrating: 3\n	1	\N	\N	48900cdc-8255-4b5d-b656-8f5b0efb23e6	2015-02-18 20:00:19.833777
236	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 4\nsource_id: 58\nvariant_id: 12\nrating: 3\n	1	\N	\N	1c49f28d-eb5e-4c74-9d88-73101ca87980	2015-02-18 20:00:19.882339
237	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with M2 FAB subtypes\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 59\nvariant_id: 90\nrating: 4\n	1	\N	\N	aede9c69-f4b4-4302-b4c7-8d79d2c32c5f	2015-02-18 20:00:19.93893
238	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with M3 FAB subtype (0/55).\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 60\nvariant_id: 90\nrating: 3\n	1	\N	\N	afc46dad-f5eb-492f-8a75-111df5a7a22f	2015-02-18 20:00:19.991445
239	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 59\nvariant_id: 90\nrating: 4\n	1	\N	\N	103e9231-573c-49bd-b616-ef3197b0f216	2015-02-18 20:00:20.043849
242	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 61\nvariant_id: 90\nrating: 3\n	1	\N	\N	c2943a30-6512-477c-9abf-7e173c21f46c	2015-02-18 20:00:20.220995
243	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 62\nvariant_id: 90\nrating: 2\n	1	\N	\N	c7665395-e458-422e-861e-ee7bb7242306	2015-02-18 20:00:20.280161
244	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 13\nsource_id: 63\nvariant_id: 90\nrating: 3\n	1	\N	\N	72449dce-eb37-464c-aa37-40f3f97a2b47	2015-02-18 20:00:20.355908
245	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 64\nvariant_id: 90\nrating: 2\n	1	\N	\N	da664ab1-287a-4ec8-82d6-b2d27a13b016	2015-02-18 20:00:20.416468
246	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult AML patients (median age 43)\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 62\nvariant_id: 90\nrating: 4\n	1	\N	\N	b80f30d8-9024-43d4-9d95-da43df879014	2015-02-18 20:00:20.47566
247	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 13\nsource_id: 65\nvariant_id: 90\nrating: 3\n	1	\N	\N	372dc802-79ed-4353-b9d0-2cf89517eea6	2015-02-18 20:00:20.529101
248	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 13\nsource_id: 65\nvariant_id: 90\nrating: 3\n	1	\N	\N	ab9b3b77-63dd-4d37-938b-ad88a09567c7	2015-02-18 20:00:20.573142
249	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 61\nvariant_id: 90\nrating: 3\n	1	\N	\N	c6177a0d-5d0d-41bd-a684-80b53a735dc9	2015-02-18 20:00:20.625544
250	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 13\nsource_id: 61\nvariant_id: 90\nrating: 2\n	1	\N	\N	30d56682-604c-4005-89b0-3e01d45a663d	2015-02-18 20:00:20.675445
251	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 16\ndisease_id: 13\nsource_id: 66\nvariant_id: 90\nrating: 2\n	1	\N	\N	1bbe2145-abbd-41a1-8fbc-c5f7952beb5b	2015-02-18 20:00:20.727816
252	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 17\ndisease_id: 13\nsource_id: 67\nvariant_id: 90\nrating: 2\n	1	\N	\N	94f06879-3f8a-474d-a493-bb119b6b59be	2015-02-18 20:00:20.797617
253	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 68\nvariant_id: 90\nrating: 2\n	1	\N	\N	848a2ef0-cfdf-45ab-86aa-dc11d50321c9	2015-02-18 20:00:20.853677
254	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 68\nvariant_id: 90\nrating: 2\n	1	\N	\N	cc3f347f-4f17-4f4b-a11b-a593144e6a1b	2015-02-18 20:00:20.921449
255	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1  .\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 59\nvariant_id: 90\nrating: 5\n	1	\N	\N	e2258f14-ccb8-41ed-90da-bf1a55e0cc06	2015-02-18 20:00:20.973752
256	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 69\nvariant_id: 90\nrating: 4\n	1	\N	\N	75b0246c-ebb7-4e77-95e2-9564e1671745	2015-02-18 20:00:21.030803
257	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 70\nvariant_id: 90\nrating: 4\n	1	\N	\N	3a849c22-279b-46fc-aba7-f3363c3f5ccd	2015-02-18 20:00:21.08623
258	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 71\nvariant_id: 90\nrating: 4\n	1	\N	\N	4a9ccbbb-cd50-41d4-af95-6535dce9ff33	2015-02-18 20:00:21.141487
259	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  was longer for patients with Exon 12 NPM1  .\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 60\nvariant_id: 90\nrating: 3\n	1	\N	\N	2ddf6a42-b892-413f-8390-1a54727ba4d6	2015-02-18 20:00:21.198237
260	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 72\nvariant_id: 90\nrating: 3\n	1	\N	\N	ae2cd76f-6e35-4b1b-86fa-782cc3c4ed72	2015-02-18 20:00:21.25533
261	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 73\nvariant_id: 90\nrating: 3\n	1	\N	\N	854e38f6-9ea7-4e76-a4f6-5dee5f99051c	2015-02-18 20:00:21.308615
262	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 74\nvariant_id: 90\nrating: 3\n	1	\N	\N	d7938c9b-e2cb-417f-a13f-3b5665901462	2015-02-18 20:00:21.369607
263	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1\n  and IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 66\nvariant_id: 90\nrating: 2\n	1	\N	\N	59bf0959-500f-4e61-82ef-aed3364e57e8	2015-02-18 20:00:21.445482
264	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 75\nvariant_id: 90\nrating: 3\n	1	\N	\N	ac8eceaf-a765-45b9-8547-81139e3b018c	2015-02-18 20:00:21.49234
265	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 76\nvariant_id: 90\nrating: 4\n	1	\N	\N	c0b3af63-8eb1-4aa5-8172-067eb552a205	2015-02-18 20:00:21.540523
266	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 77\nvariant_id: 90\nrating: 3\n	1	\N	\N	3efb672d-a217-4095-a9eb-f574a5096ffc	2015-02-18 20:00:21.59692
267	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 60\nvariant_id: 90\nrating: 4\n	1	\N	\N	021c0923-54f0-464f-a013-bb2a607879c9	2015-02-18 20:00:21.644066
268	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 78\nvariant_id: 90\nrating: 4\n	1	\N	\N	ee7b5aac-9a1d-4397-b1fe-d15aca125a4e	2015-02-18 20:00:21.700383
269	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 78\nvariant_id: 90\nrating: 4\n	1	\N	\N	741e5544-0d14-4ef1-93fc-e38ad7ce995e	2015-02-18 20:00:21.746752
270	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 79\nvariant_id: 90\nrating: 3\n	1	\N	\N	bbb3abda-ad76-4986-a023-8e19e68ab66f	2015-02-18 20:00:21.813302
271	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 71\nvariant_id: 90\nrating: 3\n	1	\N	\N	c43f44b1-b0c0-4b67-a7e6-d7cea1e1c6cf	2015-02-18 20:00:21.875592
272	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 80\nvariant_id: 91\nrating: 4\n	1	\N	\N	5ae6ff85-0631-41b3-8fbf-94c61c2f8e94	2015-02-18 20:00:21.951061
273	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 80\nvariant_id: 91\nrating: 4\n	1	\N	\N	81f22eeb-b4c4-43f3-a958-e8bf9e76432c	2015-02-18 20:00:22.007529
274	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 81\nvariant_id: 91\nrating: 3\n	1	\N	\N	4b6e9693-c5af-4a4b-9474-27e60b7223b4	2015-02-18 20:00:22.061961
275	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 4\nsource_id: 82\nvariant_id: 11\nrating: 5\n	1	\N	\N	1f51c724-fb96-4618-bc3e-af8462737de8	2015-02-18 20:00:22.125347
276	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 4\nsource_id: 83\nvariant_id: 13\nrating: 3\n	1	\N	\N	90ac1a71-7e0d-4a31-ba60-415bf69764e7	2015-02-18 20:00:22.181756
277	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 4\nsource_id: 82\nvariant_id: 15\nrating: 5\n	1	\N	\N	2864c432-e2ec-4c64-a52b-081a7fa0c9de	2015-02-18 20:00:22.242237
278	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 4\nsource_id: 82\nvariant_id: 16\nrating: 5\n	1	\N	\N	c0d804cc-347d-4318-b68f-42af0e127bd7	2015-02-18 20:00:22.304617
279	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 14\nsource_id: 84\nvariant_id: 113\nrating: 4\n	1	\N	\N	7d31603d-51fc-4940-b390-f54bf51de96c	2015-02-18 20:00:22.365782
280	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense  , including R233*, has been shown to be inactivating and loss-of-function,\n  but do not have prognostic value in glioblastoma multiforme patients.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 15\nsource_id: 85\nvariant_id: 113\nrating: 3\n	1	\N	\N	93b657cc-11f8-4eb3-ae4f-f5fa2c645327	2015-02-18 20:00:22.444284
281	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 19\ndisease_id: 16\nsource_id: 86\nvariant_id: 127\nrating: 3\n	1	\N	\N	a2ad7526-7651-4ec2-bc48-7f3c498c03c6	2015-02-18 20:00:22.504379
282	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 1\nsource_id: 87\nvariant_id: 127\nrating: 3\n	1	\N	\N	45cbf291-a018-47c2-9991-b05a07bad7a1	2015-02-18 20:00:22.567546
283	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 19\ndisease_id: 17\nsource_id: 88\nvariant_id: 128\nrating: 3\n	1	\N	\N	966abfdd-dbee-48c9-b0f4-c9672bf1164d	2015-02-18 20:00:22.635362
284	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 19\ndisease_id: 17\nsource_id: 86\nvariant_id: 129\nrating: 3\n	1	\N	\N	e047305f-a37a-4a18-9e9b-4ce93361d173	2015-02-18 20:00:22.697473
285	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 11\nsource_id: 55\nvariant_id: 1\nrating: 4\n	1	\N	\N	6885cabd-4abf-4363-a8c8-ce8618d3423c	2015-02-18 20:00:22.756581
286	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 11\nsource_id: 89\nvariant_id: 1\nrating: 2\n	1	\N	\N	10b393cf-2570-43a5-8e0b-be70b4f1d573	2015-02-18 20:00:22.80847
287	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL fusions have been found to contribute to imatinib resistance in AML\n  cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 13\nsource_id: 90\nvariant_id: 1\nrating: 3\n	1	\N	\N	37e41def-fa6e-4fdf-a693-4fca00b44ec4	2015-02-18 20:00:22.858062
288	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide\n  results in high rates of complete remission in patients.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 21\ndisease_id: 13\nsource_id: 90\nvariant_id: 1\nrating: 5\n	1	\N	\N	2a1c0ee3-0a74-4490-a953-c0304555797e	2015-02-18 20:00:22.933976
289	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The clinical use of imatinib in patients with BCR-ABL fusion has drastic sensitivity\n  to the drug.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 5\ndisease_id: 11\nsource_id: 55\nvariant_id: 1\nrating: 5\n	1	\N	\N	7fdf9ab4-1acc-4060-8caa-c5a24065c908	2015-02-18 20:00:22.982542
290	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has shown signifcantly\n  greater response in patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 11\nsource_id: 55\nvariant_id: 1\nrating: 4\n	1	\N	\N	88b2e78e-048c-48b3-b09d-4aa06dfba2aa	2015-02-18 20:00:23.034578
291	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has shown signifcantly\n  greater response in patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 8\ndisease_id: 11\nsource_id: 55\nvariant_id: 1\nrating: 4\n	1	\N	\N	7cb266a6-c3dc-4f26-89af-0c43221f395a	2015-02-18 20:00:23.076931
292	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis,\n  response rate, overall survival, and patient outcome in CML patients compared to\n  previous therapeutic regimens."\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 5\ndisease_id: 11\nsource_id: 55\nvariant_id: 1\nrating: 5\n	1	\N	\N	7af3894b-11c8-457f-bccc-0f9ac4c5ee0a	2015-02-18 20:00:23.125819
293	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 22\ndisease_id: 18\nsource_id: 91\nvariant_id: 4\nrating: 3\n	1	\N	\N	5e29ff6e-4297-44e6-a64b-935f0428ec6d	2015-02-18 20:00:23.189401
294	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 23\ndisease_id: 1\nsource_id: 92\nvariant_id: 7\nrating: 3\n	1	\N	\N	92e66732-6292-4828-a0a3-18fda21ca67a	2015-02-18 20:00:23.244363
295	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 19\nsource_id: 42\nvariant_id: 8\nrating: 3\n	1	\N	\N	2859a10b-ca42-4ea6-a46b-f7a1e7717c6d	2015-02-18 20:00:23.296688
296	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 20\nsource_id: 93\nvariant_id: 8\nrating: 3\n	1	\N	\N	a129cdad-e61f-4f89-aba1-424288cd23fc	2015-02-18 20:00:23.357173
297	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 20\nsource_id: 94\nvariant_id: 8\nrating: 4\n	1	\N	\N	b03d3f0d-2f34-4d26-a4b4-1bea91b3aed5	2015-02-18 20:00:23.429675
298	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 23\ndisease_id: 20\nsource_id: 92\nvariant_id: 8\nrating: 3\n	1	\N	\N	af261a0d-e2f4-422a-9ca7-9547da9189aa	2015-02-18 20:00:23.483974
299	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 20\nsource_id: 95\nvariant_id: 8\nrating: 3\n	1	\N	\N	afb9b33a-24e4-45cf-acc8-c1bfe6547781	2015-02-18 20:00:23.54134
300	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK with the R1275Q mutation\n  at a higher IC50 than those expressing F1174L and the SMS-KCNR cell line harboring\n  the R1275Q mutation was resistant to TAE684\nclinical_significance: NA\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 20\nsource_id: 94\nvariant_id: 9\nrating: 3\n	1	\N	\N	3c8d290b-c022-4ef5-90e7-461a8f104cf4	2015-02-18 20:00:23.590303
301	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 25\ndisease_id: 1\nsource_id: 96\nvariant_id: 10\nrating: 3\n	1	\N	\N	d238fcee-b481-4afc-abd8-531fa723fcb5	2015-02-18 20:00:23.643971
302	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 1\nsource_id: 96\nvariant_id: 10\nrating: 3\n	1	\N	\N	0b40de69-7b26-4911-a922-65f41f372a3b	2015-02-18 20:00:23.699314
303	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 1\nsource_id: 96\nvariant_id: 10\nrating: 2\n	1	\N	\N	8fb8b446-e379-455d-9e40-1dc1b0a79b20	2015-02-18 20:00:23.759301
304	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in nonchronic sun damage\n  skin and with superficial spreading melanoma\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 4\nsource_id: 97\nvariant_id: 17\nrating: 4\n	1	\N	\N	b6695142-a0b4-4f2a-9d70-bacd348c0add	2015-02-18 20:00:23.810517
305	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 21\nsource_id: 98\nvariant_id: 17\nrating: 3\n	1	\N	\N	066261b9-9f3a-446b-a449-87460074ca0f	2015-02-18 20:00:23.944118
306	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 4\nsource_id: 99\nvariant_id: 17\nrating: 3\n	1	\N	\N	13b779b8-2e80-43ca-8d94-1917df6cfa34	2015-02-18 20:00:24.012128
307	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 4\nsource_id: 99\nvariant_id: 17\nrating: 3\n	1	\N	\N	daf88a3b-beb1-4589-ab7f-1d98ad98205c	2015-02-18 20:00:24.061006
308	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 100\nvariant_id: 12\nrating: 5\n	1	\N	\N	ce802504-1472-49c1-904f-77eb7c671023	2015-02-18 20:00:24.114493
309	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 101\nvariant_id: 12\nrating: 5\n	1	\N	\N	22788410-19c3-428b-9b1a-2ce8c13bb848	2015-02-18 20:00:24.167002
310	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of PTC\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 102\nvariant_id: 12\nrating: 3\n	1	\N	\N	e4a9b07c-b7a7-42fd-9124-3e09caf2ccdc	2015-02-18 20:00:24.220738
311	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 27\ndisease_id: 4\nsource_id: 103\nvariant_id: 12\nrating: 2\n	1	\N	\N	84c7dcab-109b-4000-ab14-9217ee26ffeb	2015-02-18 20:00:24.273175
312	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 28\ndisease_id: 4\nsource_id: 103\nvariant_id: 12\nrating: 2\n	1	\N	\N	c0474b58-f4c9-45c7-b185-dd3933a44eb7	2015-02-18 20:00:24.315753
313	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 22\nsource_id: 104\nvariant_id: 12\nrating: 3\n	1	\N	\N	0dcd7f72-0fee-436c-a4fe-f8f370d7c6eb	2015-02-18 20:00:24.379826
314	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 22\nsource_id: 104\nvariant_id: 12\nrating: 3\n	1	\N	\N	c35e443b-1b1f-41fc-a812-b2321a91153b	2015-02-18 20:00:24.423526
315	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 22\nsource_id: 104\nvariant_id: 12\nrating: 3\n	1	\N	\N	c467393c-63a6-440b-a6d5-038979d0111f	2015-02-18 20:00:24.466326
316	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates and shorter progression free and overall\n  survival following cetuximab plus chemotherapy than those with wildtype BRAF\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 21\nsource_id: 105\nvariant_id: 12\nrating: 3\n	1	\N	\N	bc78ca53-eec2-409f-8c2c-38236e351e1e	2015-02-18 20:00:24.540892
317	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E resistance is triggered by ERK dependent or independent pathways.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 4\nsource_id: 106\nvariant_id: 12\nrating: 3\n	1	\N	\N	baaca28a-9a64-4e9a-a395-b01fe1a64e94	2015-02-18 20:00:24.596422
318	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 1\nsource_id: 107\nvariant_id: 12\nrating: 2\n	1	\N	\N	5e9328e4-d186-476c-81d7-0bee105a832a	2015-02-18 20:00:24.65135
319	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 4\nsource_id: 108\nvariant_id: 12\nrating: 5\n	1	\N	\N	1c103ac0-8ece-4e3d-a6c2-dd62bc2ba3c0	2015-02-18 20:00:24.701684
320	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 4\nsource_id: 108\nvariant_id: 12\nrating: 5\n	1	\N	\N	33bf9c86-8c61-42ec-89b8-301e4ad5bf3c	2015-02-18 20:00:24.752718
321	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 30\ndisease_id: 22\nsource_id: 109\nvariant_id: 12\nrating: 3\n	1	\N	\N	1b12e6e1-8a6a-4460-bfd7-2906b4ea3856	2015-02-18 20:00:24.800348
322	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 22\nsource_id: 109\nvariant_id: 12\nrating: 3\n	1	\N	\N	16b3ee3e-7503-40d5-94c0-038fcf1f25dd	2015-02-18 20:00:24.851713
323	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g.\n  dabrafenib) appears to be beneficial, and better than vice versa.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 32\ndisease_id: 4\nsource_id: 110\nvariant_id: 12\nrating: 2\n	1	\N	\N	479e8b92-ad2f-4d46-8051-3bf390de2492	2015-02-18 20:00:24.915105
324	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g.\n  dabrafenib) appears to be beneficial, and better than vice versa.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 4\nsource_id: 110\nvariant_id: 12\nrating: 2\n	1	\N	\N	6fdc3c66-e25f-47f8-a82a-97983177cc5b	2015-02-18 20:00:24.959016
325	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 33\ndisease_id: 21\nsource_id: 111\nvariant_id: 12\nrating: 2\n	1	\N	\N	b3a3c625-de96-4cf8-afb4-b615ef455eb8	2015-02-18 20:00:25.021201
326	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 34\ndisease_id: 21\nsource_id: 111\nvariant_id: 12\nrating: 2\n	1	\N	\N	4558e2f4-efa9-4c8c-8919-e0e8c695bffa	2015-02-18 20:00:25.091649
327	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 21\nsource_id: 111\nvariant_id: 12\nrating: 2\n	1	\N	\N	a457c83f-9470-47f5-9703-7d610e9280ed	2015-02-18 20:00:25.138174
328	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 22\nsource_id: 112\nvariant_id: 12\nrating: 2\n	1	\N	\N	45e45d02-369f-49ec-869d-72fb86a87f9a	2015-02-18 20:00:25.193487
329	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 36\ndisease_id: 22\nsource_id: 112\nvariant_id: 12\nrating: 2\n	1	\N	\N	c9a226af-9ff8-4e45-9bf5-a286dba621ae	2015-02-18 20:00:25.246107
330	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 34\ndisease_id: 21\nsource_id: 111\nvariant_id: 12\nrating: 2\n	1	\N	\N	bdfdd989-1026-4dfe-805a-e50b5bea61f0	2015-02-18 20:00:25.298822
331	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as poor prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 21\nsource_id: 113\nvariant_id: 12\nrating: 5\n	1	\N	\N	58ba92df-976f-41b1-9203-d3eb9b55c510	2015-02-18 20:00:25.35558
332	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 114\nvariant_id: 12\nrating: 5\n	1	\N	\N	b9540b01-ca4e-474c-91e9-c9185e59b9ec	2015-02-18 20:00:25.417439
333	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter Disease-Free and Overall Survival in a\n  Spanish cohort of Melanoma patients.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 4\nsource_id: 115\nvariant_id: 12\nrating: 3\n	1	\N	\N	ae0356ea-c9c8-4fa5-b1b3-2fb3b549c8fe	2015-02-18 20:00:25.479474
334	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 102\nvariant_id: 12\nrating: 3\n	1	\N	\N	82634804-5f7f-47ac-9269-89a4957c30cd	2015-02-18 20:00:25.531992
335	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 116\nvariant_id: 12\nrating: 3\n	1	\N	\N	94b504b2-6811-49f7-ac35-f0ccad33f302	2015-02-18 20:00:25.591322
336	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 9\nsource_id: 102\nvariant_id: 12\nrating: 3\n	1	\N	\N	ccb9db9b-c143-4324-bba9-3f77456dd12c	2015-02-18 20:00:25.671153
337	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK\n  inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification\n  of the BRAF V600E gene.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 21\nsource_id: 117\nvariant_id: 14\nrating: 4\n	1	\N	\N	61601814-0c76-4ebf-8990-cc70cd9c7e78	2015-02-18 20:00:25.731595
338	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 118\nvariant_id: 18\nrating: 3\n	1	\N	\N	029de0ac-7698-4ac4-8141-9d9a70e72531	2015-02-18 20:00:25.781776
339	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 118\nvariant_id: 19\nrating: 3\n	1	\N	\N	f16917ba-719d-4017-b60a-a812c096139a	2015-02-18 20:00:25.837959
340	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 118\nvariant_id: 19\nrating: 3\n	1	\N	\N	3d7fdc3c-2c15-42ee-b453-69755d54e6f8	2015-02-18 20:00:25.89419
341	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 119\nvariant_id: 20\nrating: 4\n	1	\N	\N	645f878f-d1b9-41e2-a6a2-1f68b9f3a19b	2015-02-18 20:00:25.952885
342	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 119\nvariant_id: 20\nrating: 4\n	1	\N	\N	e2872b22-e4bf-44ec-8324-b5d454824b0e	2015-02-18 20:00:26.002837
343	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 23\nsource_id: 120\nvariant_id: 20\nrating: 4\n	1	\N	\N	907e2537-1971-477f-b0f0-5952f3282c24	2015-02-18 20:00:26.064337
344	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 24\nsource_id: 121\nvariant_id: 20\nrating: 4\n	1	\N	\N	a712ec68-25a6-4405-8056-a5cbcbcc0679	2015-02-18 20:00:26.120061
345	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 25\nsource_id: 122\nvariant_id: 21\nrating: 4\n	1	\N	\N	45e38a74-d8a2-4aef-bb33-b17231ff5d7d	2015-02-18 20:00:26.185251
346	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 25\nsource_id: 122\nvariant_id: 21\nrating: 4\n	1	\N	\N	47ee378c-6d41-4d6a-9477-aa1d47ebfacc	2015-02-18 20:00:26.266489
347	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 25\nsource_id: 122\nvariant_id: 21\nrating: 4\n	1	\N	\N	35a9e318-37f5-4afa-8bf8-0124823a08fb	2015-02-18 20:00:26.320153
348	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 25\nsource_id: 122\nvariant_id: 21\nrating: 4\n	1	\N	\N	7f1a0879-36c3-4e14-8967-d92dcebec7ad	2015-02-18 20:00:26.374143
349	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 25\nsource_id: 123\nvariant_id: 22\nrating: 3\n	1	\N	\N	00839eda-897d-4d6b-802c-02bbbe00127d	2015-02-18 20:00:26.426935
350	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of CCND2-negative cells with 5-Aza-2'-deoxycytidine restored cyclin\n  D2 expression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 25\nsource_id: 123\nvariant_id: 22\nrating: 2\n	1	\N	\N	18a4db25-22a7-419e-9b15-e9cb2081b499	2015-02-18 20:00:26.47485
351	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 38\ndisease_id: 26\nsource_id: 124\nvariant_id: 23\nrating: 3\n	1	\N	\N	ee4eca65-5fa4-4f2e-a1f2-1af6528f44c3	2015-02-18 20:00:26.532031
352	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 125\nvariant_id: 24\nrating: 4\n	1	\N	\N	f6f3a307-4a13-4af4-a27e-e453fbc6043b	2015-02-18 20:00:26.589617
353	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 27\nsource_id: 126\nvariant_id: 24\nrating: 4\n	1	\N	\N	96bb60d3-fcab-4590-9595-253e68ee9826	2015-02-18 20:00:26.647424
354	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 27\nsource_id: 126\nvariant_id: 24\nrating: 4\n	1	\N	\N	3ee5ef9a-ded1-485f-8ed6-6bda1b775d69	2015-02-18 20:00:26.705606
355	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 28\nsource_id: 126\nvariant_id: 24\nrating: 4\n	1	\N	\N	6367374d-4277-467c-bc46-206fd17a1d55	2015-02-18 20:00:26.759984
356	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and\n  CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor\n  regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 29\nsource_id: 127\nvariant_id: 25\nrating: 4\n	1	\N	\N	18ce8839-28b8-4750-96aa-199f29066963	2015-02-18 20:00:26.83335
357	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 40\ndisease_id: 29\nsource_id: 128\nvariant_id: 25\nrating: 3\n	1	\N	\N	fac9dbc9-b866-4eee-89e0-a7b76a770e95	2015-02-18 20:00:26.888225
358	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and\n  CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor\n  regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 29\nsource_id: 127\nvariant_id: 26\nrating: 4\n	1	\N	\N	ca0e0dd1-a425-4b39-a854-b120dc7538cf	2015-02-18 20:00:26.948353
359	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 40\ndisease_id: 29\nsource_id: 128\nvariant_id: 26\nrating: 3\n	1	\N	\N	fda24bd2-9454-4543-b5a2-76ea60c00101	2015-02-18 20:00:26.995306
360	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 129\nvariant_id: 27\nrating: 3\n	1	\N	\N	41520609-b68f-41e6-b26b-1db82ff92ab2	2015-02-18 20:00:27.044365
361	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 129\nvariant_id: 27\nrating: 3\n	1	\N	\N	37557ba5-b102-4179-ba8a-a531cfc80e3c	2015-02-18 20:00:27.107125
362	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 130\nvariant_id: 29\nrating: 4\n	1	\N	\N	9d80d673-e050-40cf-ba3c-0b96f0ba1bb2	2015-02-18 20:00:27.165508
363	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 75\nvariant_id: 29\nrating: 4\n	1	\N	\N	857779e4-adb3-4479-90ef-22ece2510530	2015-02-18 20:00:27.2158
364	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 130\nvariant_id: 29\nrating: 4\n	1	\N	\N	9e3a7f7e-014f-4398-ade3-065733e04568	2015-02-18 20:00:27.27166
365	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 131\nvariant_id: 28\nrating: 3\n	1	\N	\N	ae08a788-418c-4c4b-b174-00caca36ef23	2015-02-18 20:00:27.351229
366	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 131\nvariant_id: 28\nrating: 3\n	1	\N	\N	6c4af667-1c72-49bf-8e34-a3de7ca929d5	2015-02-18 20:00:27.399028
367	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 131\nvariant_id: 28\nrating: 2\n	1	\N	\N	7b3dbd00-e354-45dc-bca3-26f6dc8288fb	2015-02-18 20:00:27.453755
368	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 30\nsource_id: 132\nvariant_id: 30\nrating: 3\n	1	\N	\N	481886c2-8d69-4c61-ab6e-9ad129268c25	2015-02-18 20:00:27.508143
369	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 41\ndisease_id: 30\nsource_id: 132\nvariant_id: 30\nrating: 3\n	1	\N	\N	5dfc3ac1-c5ac-4eb4-ab51-21e8fe61c707	2015-02-18 20:00:27.55229
370	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 133\nvariant_id: 32\nrating: 3\n	1	\N	\N	0ec5a7e1-3309-40cf-b030-0a673f48580a	2015-02-18 20:00:27.603004
371	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 80\nvariant_id: 32\nrating: 5\n	1	\N	\N	d2b47b68-f930-4635-8043-6ee61f1a410f	2015-02-18 20:00:27.647355
372	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 3\n	1	\N	\N	b9ed705b-96bb-4f7e-a81c-7595fe5db008	2015-02-18 20:00:27.70601
373	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 133\nvariant_id: 32\nrating: 3\n	1	\N	\N	62e1583f-74f8-443e-afb3-24a80303e008	2015-02-18 20:00:27.757365
374	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 133\nvariant_id: 32\nrating: 3\n	1	\N	\N	d5621fec-d7ca-4ec1-bbb6-5861534d375e	2015-02-18 20:00:27.827673
375	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 134\nvariant_id: 32\nrating: 3\n	1	\N	\N	70fe119b-f2bb-4613-9e6e-8f4dfa8b5cff	2015-02-18 20:00:27.882177
376	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 135\nvariant_id: 32\nrating: 3\n	1	\N	\N	e5a98d84-f68f-4f83-8866-d0e801cb1265	2015-02-18 20:00:27.937894
377	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: NA\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 16\ndisease_id: 13\nsource_id: 134\nvariant_id: 32\nrating: 4\n	1	\N	\N	30a22993-9880-45ba-abc1-c779768c0f9b	2015-02-18 20:00:27.983917
378	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 13\nsource_id: 134\nvariant_id: 32\nrating: 4\n	1	\N	\N	21ff87d9-352e-45cc-9f01-3e67f5ff6ab1	2015-02-18 20:00:28.037009
379	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 133\nvariant_id: 32\nrating: 3\n	1	\N	\N	b1698bc9-013e-448a-8781-d1ccd80bde26	2015-02-18 20:00:28.090129
380	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 136\nvariant_id: 32\nrating: 4\n	1	\N	\N	23cc9a40-336e-493b-9638-1f1c4a077a5e	2015-02-18 20:00:28.141927
381	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 4\n	1	\N	\N	c188d4ec-7c87-4177-b012-650bd0c0e097	2015-02-18 20:00:28.205018
382	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 134\nvariant_id: 32\nrating: 4\n	1	\N	\N	854f8738-541a-4fe9-ac75-d81b157ec322	2015-02-18 20:00:28.255679
383	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 133\nvariant_id: 32\nrating: 3\n	1	\N	\N	bf3a776f-9da8-479f-b68c-97b4c284d387	2015-02-18 20:00:28.3337
384	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 133\nvariant_id: 32\nrating: 3\n	1	\N	\N	afff4611-817c-49bf-81ab-160e589c68e1	2015-02-18 20:00:28.388251
385	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 80\nvariant_id: 32\nrating: 5\n	1	\N	\N	f15c88bb-2663-45e6-92b7-fb4b8e7998cd	2015-02-18 20:00:28.448263
386	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD  , concurrent DNMT3A mutations (including R882)\n  were associated with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 80\nvariant_id: 32\nrating: 5\n	1	\N	\N	2cfcaa3f-e76d-45c0-9eab-a8134b7d0bb2	2015-02-18 20:00:28.505822
387	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 135\nvariant_id: 32\nrating: 5\n	1	\N	\N	2461ae8e-d948-408a-911e-7389e4a667dc	2015-02-18 20:00:28.552019
388	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 136\nvariant_id: 32\nrating: 4\n	1	\N	\N	5d674de4-944a-49e4-ae56-5ae954c35ba6	2015-02-18 20:00:28.609061
389	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 4\n	1	\N	\N	b9905733-36b1-4e42-9faf-736515d5ad8e	2015-02-18 20:00:28.670311
390	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 80\nvariant_id: 32\nrating: 4\n	1	\N	\N	29f7c799-53f9-4aa8-bc50-189b2c811530	2015-02-18 20:00:28.720637
391	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 136\nvariant_id: 32\nrating: 4\n	1	\N	\N	b66b6807-761e-4cb5-827e-44835798838d	2015-02-18 20:00:28.767479
392	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 136\nvariant_id: 32\nrating: 4\n	1	\N	\N	79edc1ea-9a7c-45b6-84e4-d20c4e82d8c0	2015-02-18 20:00:28.842086
393	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable cytogenetically normal AMLs\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 133\nvariant_id: 32\nrating: 3\n	1	\N	\N	51a45295-bf19-4b7c-b79f-0d3278643dba	2015-02-18 20:00:28.896056
394	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 3\n	1	\N	\N	1166599c-221d-4c70-bf18-171b1b399643	2015-02-18 20:00:28.949305
395	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 3\n	1	\N	\N	7bbca04e-043e-4c30-b839-0d0353ee8f98	2015-02-18 20:00:29.003999
396	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 3\n	1	\N	\N	a6be31a7-ffb7-4b9c-bdbc-73486e546bc3	2015-02-18 20:00:29.056616
397	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 13\nsource_id: 133\nvariant_id: 32\nrating: 3\n	1	\N	\N	4e9bad06-da44-4c66-9613-777ef9e9436d	2015-02-18 20:00:29.10563
398	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 1\nsource_id: 137\nvariant_id: 33\nrating: 3\n	1	\N	\N	8897ae44-69f0-40ec-b8a3-81902ac5afa8	2015-02-18 20:00:29.15726
399	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 44\ndisease_id: 1\nsource_id: 137\nvariant_id: 33\nrating: 3\n	1	\N	\N	3c5ba7e3-bba3-4fe4-ab07-0af067f92551	2015-02-18 20:00:29.199998
400	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with tyrosine kinase inhibitors, the presence of L858R\n  mutation is prognostic for better progression free survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 138\nvariant_id: 33\nrating: 4\n	1	\N	\N	06609fc4-a9b3-4820-b853-cc118870535a	2015-02-18 20:00:29.252972
401	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Geftinib added to pemetrexed and carboplatin showed greater effectiveness than\n  premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 1\nsource_id: 139\nvariant_id: 33\nrating: 3\n	1	\N	\N	617a69dc-0588-4f5b-a7ee-a716acd4e200	2015-02-18 20:00:29.324782
402	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 140\nvariant_id: 33\nrating: 3\n	1	\N	\N	16545cd3-97dd-4044-a467-1609f2f78fb2	2015-02-18 20:00:29.390729
403	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 1\nsource_id: 141\nvariant_id: 34\nrating: 3\n	1	\N	\N	0c7f082b-7bbc-4dd1-b6d2-a76d8d4e0166	2015-02-18 20:00:29.441402
404	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 44\ndisease_id: 1\nsource_id: 142\nvariant_id: 34\nrating: 3\n	1	\N	\N	00213143-faa1-49d9-b0be-b6d6c958d512	2015-02-18 20:00:29.496904
405	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 45\ndisease_id: 1\nsource_id: 142\nvariant_id: 34\nrating: 3\n	1	\N	\N	5ec822d2-87f8-4303-b6ed-751d37355001	2015-02-18 20:00:29.540867
406	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when L858R mutation is also present.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 46\ndisease_id: 1\nsource_id: 143\nvariant_id: 34\nrating: 1\n	1	\N	\N	edaf35e8-027b-40d6-bc84-be7a034357d1	2015-02-18 20:00:29.597221
407	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation has statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 144\nvariant_id: 34\nrating: 4\n	1	\N	\N	f9a165c7-c904-4b02-8708-f5667815b4b5	2015-02-18 20:00:29.660248
408	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutation, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 145\nvariant_id: 34\nrating: 4\n	1	\N	\N	13359056-169c-49df-8649-ee60b9c5a601	2015-02-18 20:00:29.709807
409	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the in-frame deletion of amino acids 755-759 was shown\n  to be sensitive to neratiinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 37\nrating: 5\n	1	\N	\N	b848f5af-0905-43de-80c3-658e5cacdd6b	2015-02-18 20:00:29.757428
410	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation showed mild response to neratinib in MCF10A cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 39\nrating: 5\n	1	\N	\N	5d5df3c5-3019-41bb-81af-554bfd402b9a	2015-02-18 20:00:29.827456
411	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell\n  lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 47\ndisease_id: 3\nsource_id: 54\nvariant_id: 39\nrating: 5\n	1	\N	\N	85c14bc4-4a77-4a8f-a4e3-2aa176740331	2015-02-18 20:00:29.881961
412	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 40\nrating: 5\n	1	\N	\N	ca4c4c9d-1129-4869-af3b-0ff4aadb4694	2015-02-18 20:00:29.936323
413	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 3\nsource_id: 54\nvariant_id: 42\nrating: 5\n	1	\N	\N	607e8cdb-f651-46f1-9303-14c547e6c0bd	2015-02-18 20:00:29.987827
414	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 146\nvariant_id: 60\nrating: 3\n	1	\N	\N	bb50b433-381c-44e3-8184-d61f1cd8cbe6	2015-02-18 20:00:30.040374
415	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 62\nvariant_id: 60\nrating: 4\n	1	\N	\N	cd67ca30-0cbd-4a46-a045-414583157808	2015-02-18 20:00:30.08922
434	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 13\nsource_id: 153\nvariant_id: 61\nrating: 4\n	1	\N	\N	a3742c76-4162-48a3-bd55-c9d39297f1bf	2015-02-18 20:00:31.154682
416	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 48\ndisease_id: 13\nsource_id: 147\nvariant_id: 60\nrating: 3\n	1	\N	\N	d6616abe-93e2-4bcc-ae87-728a20896a85	2015-02-18 20:00:30.144342
417	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 13\nsource_id: 63\nvariant_id: 60\nrating: 3\n	1	\N	\N	3ce48772-0158-411c-8ee5-e5e3e9b20f98	2015-02-18 20:00:30.196368
418	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 16\ndisease_id: 13\nsource_id: 147\nvariant_id: 60\nrating: 3\n	1	\N	\N	716ba47e-24d4-427a-9f93-bc2da01d99e8	2015-02-18 20:00:30.341386
419	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 62\nvariant_id: 60\nrating: 3\n	1	\N	\N	ae3dadbc-724f-4103-85e4-f5d9d9fe9946	2015-02-18 20:00:30.396928
420	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeats, but not\n  in wild type or D835 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 13\nsource_id: 148\nvariant_id: 60\nrating: 4\n	1	\N	\N	0ff08928-3dfc-4090-855a-487570c85253	2015-02-18 20:00:30.456538
421	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 is effective in AML patients with FLT3 mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 13\nsource_id: 149\nvariant_id: 60\nrating: 4\n	1	\N	\N	8fc21197-8300-4f8e-b405-853aa9dedb9a	2015-02-18 20:00:30.517171
422	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 50\ndisease_id: 13\nsource_id: 150\nvariant_id: 60\nrating: 4\n	1	\N	\N	aeb68555-af93-404d-9442-886056f6cdd5	2015-02-18 20:00:30.565877
423	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 76\nvariant_id: 60\nrating: 4\n	1	\N	\N	12b4bd1a-1650-40e0-8390-315970960c79	2015-02-18 20:00:30.614416
424	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 78\nvariant_id: 60\nrating: 4\n	1	\N	\N	a9f5d563-bd83-4d67-a63d-43934b280058	2015-02-18 20:00:30.665716
425	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 62\nvariant_id: 60\nrating: 2\n	1	\N	\N	1c8de018-e636-4273-a15b-f97f73c6aaa5	2015-02-18 20:00:30.721991
426	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD  , concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 80\nvariant_id: 60\nrating: 5\n	1	\N	\N	f4d54b5f-907c-48c7-a8cc-43e2e1d103ac	2015-02-18 20:00:30.773321
427	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 151\nvariant_id: 60\nrating: 4\n	1	\N	\N	3c4f1b3c-6e57-493c-99a5-ce652322f38e	2015-02-18 20:00:30.821744
428	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 78\nvariant_id: 60\nrating: 4\n	1	\N	\N	9364622c-f32c-42be-b58e-b6f7bb4d0f25	2015-02-18 20:00:30.867578
429	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 78\nvariant_id: 60\nrating: 4\n	1	\N	\N	45469ec4-f316-4252-b12f-bbeb693098c3	2015-02-18 20:00:30.921045
430	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 78\nvariant_id: 60\nrating: 4\n	1	\N	\N	49665b01-4847-45a6-8874-2d7e57a58b99	2015-02-18 20:00:30.969546
431	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 146\nvariant_id: 60\nrating: 3\n	1	\N	\N	db7ff7b0-67d6-4ea5-be54-7916f2815bc0	2015-02-18 20:00:31.016849
432	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 13\nsource_id: 152\nvariant_id: 60\nrating: 2\n	1	\N	\N	5fda95e6-1bf7-4398-97f9-046a9041e232	2015-02-18 20:00:31.060561
433	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 146\nvariant_id: 61\nrating: 3\n	1	\N	\N	4e3602cf-b17b-4053-9074-b92cc53359da	2015-02-18 20:00:31.106258
435	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat , but not\n  in wild type or D835 mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 13\nsource_id: 148\nvariant_id: 61\nrating: 4\n	1	\N	\N	fab61758-acc3-4387-8e1d-fb578de6aaac	2015-02-18 20:00:31.210751
436	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 48\ndisease_id: 13\nsource_id: 147\nvariant_id: 61\nrating: 3\n	1	\N	\N	fe03ff02-3234-434b-aec6-5d35768c64ed	2015-02-18 20:00:31.256301
437	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 16\ndisease_id: 13\nsource_id: 147\nvariant_id: 61\nrating: 3\n	1	\N	\N	d4671981-024e-4d76-9daf-267a760fc9a3	2015-02-18 20:00:31.300193
438	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 is effective in AML patients with FLT3 mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 13\nsource_id: 149\nvariant_id: 61\nrating: 4\n	1	\N	\N	adfb2855-c171-4e1b-b551-c783c8ae6e79	2015-02-18 20:00:31.349643
439	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 72\nvariant_id: 61\nrating: 3\n	1	\N	\N	4b4da7c1-e5ca-4f2d-86e4-b3a833c1e48e	2015-02-18 20:00:31.401532
440	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 146\nvariant_id: 61\nrating: 3\n	1	\N	\N	aa9c7fe0-1220-4e0b-9e72-f9473fed46d5	2015-02-18 20:00:31.451898
441	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation is associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 154\nvariant_id: 61\nrating: 4\n	1	\N	\N	61708fa6-fac7-447d-a740-94aaaba00484	2015-02-18 20:00:31.505901
442	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 51\ndisease_id: 31\nsource_id: 155\nvariant_id: 62\nrating: 4\n	1	\N	\N	45a78ba2-0899-426b-9f7c-6823eea227dd	2015-02-18 20:00:31.56711
443	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 52\ndisease_id: 31\nsource_id: 155\nvariant_id: 62\nrating: 4\n	1	\N	\N	00534369-e903-4fa1-acf4-e2d53598ea09	2015-02-18 20:00:31.614139
444	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 156\nvariant_id: 63\nrating: 2\n	1	\N	\N	4c054f5b-1c16-4c68-a520-2d807ca89746	2015-02-18 20:00:31.665273
445	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 32\nsource_id: 157\nvariant_id: 63\nrating: 3\n	1	\N	\N	6e17c2ee-d81b-4617-935c-4612989945eb	2015-02-18 20:00:31.719049
446	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 33\nsource_id: 158\nvariant_id: 63\nrating: 3\n	1	\N	\N	29ec270d-167c-4107-a9a5-1fea01fa53fd	2015-02-18 20:00:31.763232
447	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 159\nvariant_id: 63\nrating: 2\n	1	\N	\N	2a35d97a-f74f-4d2b-90f6-648b2a6d3cc1	2015-02-18 20:00:31.822761
448	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 34\nsource_id: 160\nvariant_id: 63\nrating: 4\n	1	\N	\N	835b6f00-baaf-465b-b1de-d03880a5ed1e	2015-02-18 20:00:31.870097
449	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 161\nvariant_id: 64\nrating: 3\n	1	\N	\N	9d73548b-5fa0-45d8-9da8-0d4bbb73d019	2015-02-18 20:00:31.947035
450	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 161\nvariant_id: 64\nrating: 2\n	1	\N	\N	1ac252ff-a28b-4cc8-bb84-2059207189e9	2015-02-18 20:00:31.998355
451	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 161\nvariant_id: 65\nrating: 3\n	1	\N	\N	626b1f37-9325-45a6-9342-faba58f72012	2015-02-18 20:00:32.04579
452	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 161\nvariant_id: 66\nrating: 3\n	1	\N	\N	07ee5597-3110-47a0-81c1-6537882ca396	2015-02-18 20:00:32.10389
453	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 162\nvariant_id: 67\nrating: 3\n	1	\N	\N	ece4b342-d0bf-44ac-9a0f-1285bb9cc5b1	2015-02-18 20:00:32.150024
454	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation do not impact\n  overall survival or disease free survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 163\nvariant_id: 67\nrating: 3\n	1	\N	\N	3c30ca45-1881-4a49-8683-7ab19d4a6b8d	2015-02-18 20:00:32.20432
455	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 35\nsource_id: 164\nvariant_id: 67\nrating: 3\n	1	\N	\N	6d188885-c88c-4372-85df-ae0f49c9ff1c	2015-02-18 20:00:32.253842
456	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 165\nvariant_id: 67\nrating: 2\n	1	\N	\N	edfd692c-b9c7-4338-abe1-c0f7e05b43ac	2015-02-18 20:00:32.309177
457	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 35\nsource_id: 166\nvariant_id: 67\nrating: 2\n	1	\N	\N	c3ee1b7c-d1f1-4abe-b857-bde4dc8f6a4f	2015-02-18 20:00:32.361497
458	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 165\nvariant_id: 68\nrating: 2\n	1	\N	\N	2106ed77-0507-4e47-a971-f39e39197c9b	2015-02-18 20:00:32.426236
459	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 35\nsource_id: 166\nvariant_id: 68\nrating: 2\n	1	\N	\N	ade95d96-301c-421b-a352-31e4a0ca9bff	2015-02-18 20:00:32.482078
460	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 162\nvariant_id: 68\nrating: 3\n	1	\N	\N	52c0d20f-ec78-4328-8444-78564a86b3f8	2015-02-18 20:00:32.536217
461	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 36\nsource_id: 167\nvariant_id: 69\nrating: 4\n	1	\N	\N	18733b86-16e8-463a-b26b-6769382cace8	2015-02-18 20:00:32.586076
462	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F not associatted wth  lymphoid leukemia.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 11\nsource_id: 167\nvariant_id: 69\nrating: 4\n	1	\N	\N	1cad257a-70f6-452a-8e5c-eef08df49024	2015-02-18 20:00:32.634199
463	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 11\nsource_id: 167\nvariant_id: 69\nrating: 4\n	1	\N	\N	450927ee-e41c-4b43-ac08-242e8b1bf675	2015-02-18 20:00:32.685896
464	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: rs10974944 SNP in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 37\nsource_id: 168\nvariant_id: 69\nrating: 4\n	1	\N	\N	d3f3090a-2d05-402c-ada4-885b8c5a21f8	2015-02-18 20:00:32.740866
465	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of peglated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 53\ndisease_id: 38\nsource_id: 169\nvariant_id: 69\nrating: 4\n	1	\N	\N	370561c6-2f73-4cad-ab1b-909ada1b0da6	2015-02-18 20:00:32.804749
466	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 54\ndisease_id: 39\nsource_id: 170\nvariant_id: 69\nrating: 3\n	1	\N	\N	2611c65c-ead5-473f-987a-d39dcc3e8a9c	2015-02-18 20:00:32.86025
467	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 5\nsource_id: 171\nvariant_id: 71\nrating: 2\n	1	\N	\N	6ec9b229-2bb4-4f52-9cbc-3bec5616946e	2015-02-18 20:00:32.910654
468	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 4\nsource_id: 172\nvariant_id: 71\nrating: 3\n	1	\N	\N	2bf11216-f2eb-4cc2-adad-22c76c978217	2015-02-18 20:00:32.967924
469	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 5\nsource_id: 173\nvariant_id: 71\nrating: 3\n	1	\N	\N	0b4312c2-a580-46e7-9bc6-081e8ff3ff75	2015-02-18 20:00:33.033382
470	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between KIT mutations in exon 11, exon 9\n  or wild type in survival among GIST patients.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 5\nsource_id: 174\nvariant_id: 71\nrating: 3\n	1	\N	\N	18998907-8c94-4bb8-bcfb-a234dd3c7acb	2015-02-18 20:00:33.086801
471	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 28\nsource_id: 175\nvariant_id: 71\nrating: 2\n	1	\N	\N	04eb2c54-be35-42c2-991a-b2a6b780c98e	2015-02-18 20:00:33.154462
472	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 5\nsource_id: 171\nvariant_id: 71\nrating: 3\n	1	\N	\N	b5b7639c-3d50-4fa6-b759-5855b51685c3	2015-02-18 20:00:33.208345
473	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 4\nsource_id: 172\nvariant_id: 72\nrating: 3\n	1	\N	\N	9f83069d-ba27-49b3-9b07-1b1f6198a9c9	2015-02-18 20:00:33.261545
474	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 5\nsource_id: 174\nvariant_id: 73\nrating: 3\n	1	\N	\N	4985d476-9ed0-4d38-be6e-82920e7eaab3	2015-02-18 20:00:33.309132
475	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 5\nsource_id: 171\nvariant_id: 74\nrating: 3\n	1	\N	\N	fb7b8a2e-0fb3-421d-9a5b-22ab70a0b48b	2015-02-18 20:00:33.356889
476	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 4\nsource_id: 172\nvariant_id: 74\nrating: 3\n	1	\N	\N	1ee0de1e-f0e0-4909-9b3b-f15a33f98686	2015-02-18 20:00:33.416144
477	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between KIT mutations in exon 11, exon 9\n  or wild type in survival among GIST patients.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 5\nsource_id: 174\nvariant_id: 75\nrating: 3\n	1	\N	\N	e13d1d05-7cde-4fd3-a1a3-c3b16af9b302	2015-02-18 20:00:33.469474
478	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 28\nsource_id: 175\nvariant_id: 75\nrating: 2\n	1	\N	\N	a571e722-66cc-4e76-af0e-71d604aa9260	2015-02-18 20:00:33.517269
479	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 21\nsource_id: 176\nvariant_id: 78\nrating: 4\n	1	\N	\N	a2dca6f9-ce20-4b5f-8f61-f42fbcbad2fa	2015-02-18 20:00:33.606735
480	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 1\nsource_id: 177\nvariant_id: 79\nrating: 3\n	1	\N	\N	a041ed7f-3f0c-4843-83df-8c61212428ed	2015-02-18 20:00:33.660675
481	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 44\ndisease_id: 1\nsource_id: 177\nvariant_id: 79\nrating: 3\n	1	\N	\N	0ab80972-9bee-41cf-90b9-adbf3201d3d6	2015-02-18 20:00:33.716753
482	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 55\ndisease_id: 21\nsource_id: 178\nvariant_id: 79\nrating: 4\n	1	\N	\N	5e866dcf-d67e-4c91-857b-1cb288d936a5	2015-02-18 20:00:33.772595
483	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 40\nsource_id: 179\nvariant_id: 80\nrating: 3\n	1	\N	\N	def9ff34-61b7-4959-8cd5-bcb56d0e9917	2015-02-18 20:00:33.835173
484	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung patients are not associated with smoking history, age\n  and gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 180\nvariant_id: 80\nrating: 3\n	1	\N	\N	91cecac0-965f-458e-9cfe-ce1782a3421b	2015-02-18 20:00:33.881984
485	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 22\nsource_id: 181\nvariant_id: 80\nrating: 4\n	1	\N	\N	7dac9da5-4dd4-4212-9f76-7dff8fb4a129	2015-02-18 20:00:33.935283
486	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 182\nvariant_id: 80\nrating: 3\n	1	\N	\N	fe4d7347-e10c-4cc0-a0c4-04c6eb7bd438	2015-02-18 20:00:33.984725
487	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 41\nsource_id: 183\nvariant_id: 80\nrating: 3\n	1	\N	\N	53ecf55e-78ee-4f91-ac59-e39305c69764	2015-02-18 20:00:34.040104
488	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 184\nvariant_id: 80\nrating: 3\n	1	\N	\N	53bf2d39-0d24-458d-8834-80cbfd99ceb1	2015-02-18 20:00:34.090275
489	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 21\nsource_id: 185\nvariant_id: 81\nrating: 3\n	1	\N	\N	da3ce1b0-04fc-43e0-9076-8c757e6bfb42	2015-02-18 20:00:34.153031
490	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 21\nsource_id: 105\nvariant_id: 81\nrating: 3\n	1	\N	\N	d3a595a2-28a8-417f-92da-721f2be7cddb	2015-02-18 20:00:34.209167
491	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 21\nsource_id: 185\nvariant_id: 81\nrating: 3\n	1	\N	\N	7bd7e917-1efc-40f4-b5e4-2bd1c372d967	2015-02-18 20:00:34.255869
492	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 28\nsource_id: 186\nvariant_id: 82\nrating: 2\n	1	\N	\N	66454e6b-18f1-4fb7-b541-0099e0378915	2015-02-18 20:00:34.310126
493	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 28\nsource_id: 186\nvariant_id: 83\nrating: 3\n	1	\N	\N	a47b4871-c5d7-405b-8bbd-d4af48b24643	2015-02-18 20:00:34.360063
494	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 18\ndisease_id: 1\nsource_id: 187\nvariant_id: 83\nrating: 2\n	1	\N	\N	9dbc85c6-d8f5-4b83-8db8-1f06272fceca	2015-02-18 20:00:34.421471
495	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 56\ndisease_id: 1\nsource_id: 188\nvariant_id: 83\nrating: 4\n	1	\N	\N	c44cb7fa-0b6c-42c8-9e48-73774479b364	2015-02-18 20:00:34.474442
496	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 57\ndisease_id: 1\nsource_id: 188\nvariant_id: 83\nrating: 4\n	1	\N	\N	01c4ca2c-efa9-460e-a77f-4874c3d4c4a3	2015-02-18 20:00:34.521741
497	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung patients are not associated with smoking history, age\n  and gender\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 1\nsource_id: 180\nvariant_id: 84\nrating: 3\n	1	\N	\N	4c4a878d-2f64-4fb4-ac78-6ad79785f349	2015-02-18 20:00:34.573775
498	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 40\nsource_id: 179\nvariant_id: 84\nrating: 3\n	1	\N	\N	e376eab7-4631-48e3-80f7-006234a7d882	2015-02-18 20:00:34.618896
499	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 42\nsource_id: 183\nvariant_id: 84\nrating: 3\n	1	\N	\N	773a9c98-4c4c-4fcd-aa4d-d2c53b334544	2015-02-18 20:00:34.678229
500	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft models\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 22\nsource_id: 189\nvariant_id: 85\nrating: 4\n	1	\N	\N	ff8db651-1aad-41a5-8d98-baa066bb0c14	2015-02-18 20:00:34.733209
501	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 22\nsource_id: 189\nvariant_id: 85\nrating: 4\n	1	\N	\N	15d93d58-6225-42bc-87e0-e6c8de5cbd00	2015-02-18 20:00:34.785372
502	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft models\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 22\nsource_id: 189\nvariant_id: 85\nrating: 4\n	1	\N	\N	8f3ef671-ffb4-436d-97eb-9c1fc7636838	2015-02-18 20:00:34.837219
503	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 22\nsource_id: 189\nvariant_id: 85\nrating: 4\n	1	\N	\N	e629e3df-0c49-46b1-ba27-35aec0a1bb2f	2015-02-18 20:00:34.894627
504	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 P124S confers increased resistance to inhibition by AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 37\ndisease_id: 4\nsource_id: 190\nvariant_id: 86\nrating: 3\n	1	\N	\N	0420c3a4-a01f-4d3a-87bb-a79936bf86a6	2015-02-18 20:00:34.94761
505	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 37\ndisease_id: 4\nsource_id: 191\nvariant_id: 87\nrating: 3\n	1	\N	\N	42b7745a-1c8a-47e5-a06a-75709679dc7f	2015-02-18 20:00:34.99631
506	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 30\nsource_id: 132\nvariant_id: 88\nrating: 3\n	1	\N	\N	2f9dc548-11fc-4c40-b0af-8bb9f796362b	2015-02-18 20:00:35.055785
507	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 41\ndisease_id: 30\nsource_id: 132\nvariant_id: 88\nrating: 3\n	1	\N	\N	65a68dd0-3ec5-4c9b-af1d-91a1927b6370	2015-02-18 20:00:35.106041
508	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increase survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 28\ndisease_id: 15\nsource_id: 192\nvariant_id: 89\nrating: 5\n	1	\N	\N	93f289f7-0168-4090-b944-c0820fd5035a	2015-02-18 20:00:35.16338
509	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 58\ndisease_id: 15\nsource_id: 193\nvariant_id: 89\nrating: 4\n	1	\N	\N	a1df28a7-0391-4087-8b1e-afaad371fdcf	2015-02-18 20:00:35.261403
510	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 59\ndisease_id: 15\nsource_id: 193\nvariant_id: 89\nrating: 4\n	1	\N	\N	c9f6f419-9de1-4542-887d-8a332a8ec5e6	2015-02-18 20:00:35.321211
511	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"AML with mutated NPM1" is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.'\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 2\ndisease_id: 13\nsource_id: 194\nvariant_id: 90\nrating: 5\n	1	\N	\N	bd23eaf2-a6be-4e15-9249-d11157ab86f6	2015-02-18 20:00:35.37298
512	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 4\nsource_id: 195\nvariant_id: 92\nrating: 3\n	1	\N	\N	d7a0711f-e1fb-4114-8d97-d76ddae3d272	2015-02-18 20:00:35.42179
513	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patient does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 196\nvariant_id: 93\nrating: 3\n	1	\N	\N	698fc2d0-f1af-437d-9107-8650b2fe6ed8	2015-02-18 20:00:35.475762
514	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 21\nsource_id: 197\nvariant_id: 93\nrating: 3\n	1	\N	\N	202a0bcc-c3ac-4a41-b16e-743eb880557f	2015-02-18 20:00:35.527607
515	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 4\nsource_id: 195\nvariant_id: 93\nrating: 3\n	1	\N	\N	a3942b27-19d4-41a3-8fd1-24a5dcaadecc	2015-02-18 20:00:35.580129
516	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 21\nsource_id: 197\nvariant_id: 94\nrating: 3\n	1	\N	\N	16ac410c-2ceb-423e-9c15-b70ac6cfa2bf	2015-02-18 20:00:35.631956
517	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 21\nsource_id: 197\nvariant_id: 95\nrating: 3\n	1	\N	\N	6e0b34e7-680c-4970-abff-decf3921b0f7	2015-02-18 20:00:35.684737
518	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 198\nvariant_id: 96\nrating: 3\n	1	\N	\N	28fdee17-2cbd-4e6e-8b27-9ce0d8afd344	2015-02-18 20:00:35.740198
519	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition\n  of HSP90 by 17-AAG is shown to be effective in a patient with metastatic malignant\n  melenoma with a NRAS G13D mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 60\ndisease_id: 4\nsource_id: 199\nvariant_id: 97\nrating: 2\n	1	\N	\N	10b0325f-612f-4aca-99fa-499d9f40ce04	2015-02-18 20:00:35.802435
520	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 4\nsource_id: 200\nvariant_id: 98\nrating: 3\n	1	\N	\N	6f983a76-cc0a-486f-9669-77b83bc121c8	2015-02-18 20:00:35.867989
521	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 21\nsource_id: 105\nvariant_id: 98\nrating: 3\n	1	\N	\N	79282db5-255a-4cb1-9417-772f40d1e6a7	2015-02-18 20:00:35.920479
522	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 34\ndisease_id: 4\nsource_id: 201\nvariant_id: 98\nrating: 4\n	1	\N	\N	189b7c93-ef2b-430b-88a1-f5e3fabcb148	2015-02-18 20:00:35.97053
523	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 21\nsource_id: 105\nvariant_id: 98\nrating: 3\n	1	\N	\N	42d46706-f67f-458c-b586-ad18545c500b	2015-02-18 20:00:36.026164
524	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 61\ndisease_id: 4\nsource_id: 202\nvariant_id: 99\nrating: 2\n	1	\N	\N	7e860fcb-115d-4753-a967-52680d079154	2015-02-18 20:00:36.075099
525	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 61\ndisease_id: 4\nsource_id: 202\nvariant_id: 100\nrating: 2\n	1	\N	\N	2542d562-4eed-40e9-b7a5-083cd296c391	2015-02-18 20:00:36.117835
526	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 43\nsource_id: 203\nvariant_id: 101\nrating: 4\n	1	\N	\N	c74937c7-1a01-4c44-aab6-da57e27f7eeb	2015-02-18 20:00:36.169641
527	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumor harboring PDGFRA D842V mutation are likely to be benign\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 43\nsource_id: 204\nvariant_id: 102\nrating: 3\n	1	\N	\N	c817f343-d968-4c28-b5ec-16e291c555eb	2015-02-18 20:00:36.232955
528	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 43\nsource_id: 49\nvariant_id: 102\nrating: 4\n	1	\N	\N	2bdedd87-855e-4013-a00a-52d2f6a1c02c	2015-02-18 20:00:36.295858
529	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 43\nsource_id: 205\nvariant_id: 102\nrating: 4\n	1	\N	\N	04f766a1-c7c7-4693-989d-b2c922f57d4f	2015-02-18 20:00:36.350158
530	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation resulting in imatinib resistance, crenolanib\n  showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 43\nsource_id: 203\nvariant_id: 102\nrating: 4\n	1	\N	\N	005edc90-af38-40a5-8f00-101b98bbab19	2015-02-18 20:00:36.406028
531	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation resulting in imatinib resistance, crenolanib\n  showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 43\nsource_id: 203\nvariant_id: 103\nrating: 4\n	1	\N	\N	86c53636-3075-4e55-ac02-d29f7a1584c8	2015-02-18 20:00:36.467718
532	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA deletion I843 mutation resulting in imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 43\nsource_id: 203\nvariant_id: 104\nrating: 4\n	1	\N	\N	cf562dba-cfff-48c5-bb54-b628c343276c	2015-02-18 20:00:36.531388
533	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA DI842-843IM mutation resulting in imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 43\nsource_id: 203\nvariant_id: 105\nrating: 4\n	1	\N	\N	ec07e055-38f8-4045-8530-66977ad6d550	2015-02-18 20:00:36.590104
534	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 55\ndisease_id: 21\nsource_id: 178\nvariant_id: 106\nrating: 4\n	1	\N	\N	76096edb-248c-4860-8303-cdc202c82dd2	2015-02-18 20:00:36.650751
535	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 29\ndisease_id: 21\nsource_id: 206\nvariant_id: 106\nrating: 3\n	1	\N	\N	fde7e528-e749-4a2e-9e15-bd726a92781c	2015-02-18 20:00:36.700453
536	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 21\nsource_id: 206\nvariant_id: 106\nrating: 3\n	1	\N	\N	09c01a5a-8fdb-4a55-bea6-8452803ba562	2015-02-18 20:00:36.740472
537	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 63\ndisease_id: 3\nsource_id: 207\nvariant_id: 106\nrating: 4\n	1	\N	\N	3419febd-599f-4036-bd29-a67309afefe2	2015-02-18 20:00:36.80275
538	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 3\nsource_id: 208\nvariant_id: 106\nrating: 3\n	1	\N	\N	e9419cbe-018e-4c7e-9e9c-40ea27fc0dea	2015-02-18 20:00:36.906686
539	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 21\nsource_id: 206\nvariant_id: 106\nrating: 3\n	1	\N	\N	83e6e929-7c7c-496a-93b2-971fbcce83ec	2015-02-18 20:00:36.959849
540	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 21\nsource_id: 209\nvariant_id: 106\nrating: 2\n	1	\N	\N	2c47a1e4-ee12-478e-919a-9aa43ed89346	2015-02-18 20:00:37.016808
541	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 55\ndisease_id: 21\nsource_id: 178\nvariant_id: 107\nrating: 4\n	1	\N	\N	431692cc-b044-44e0-8b0c-0186b083bdcd	2015-02-18 20:00:37.070799
542	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 29\ndisease_id: 21\nsource_id: 206\nvariant_id: 107\nrating: 3\n	1	\N	\N	8e6eb4cb-85df-4614-ab08-c55ea429414a	2015-02-18 20:00:37.125517
543	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 21\nsource_id: 206\nvariant_id: 107\nrating: 3\n	1	\N	\N	16302933-30dd-4626-87c8-b677b24c7eff	2015-02-18 20:00:37.173014
544	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 63\ndisease_id: 3\nsource_id: 207\nvariant_id: 107\nrating: 4\n	1	\N	\N	8fdce1d4-f596-4d7e-8e74-83885fad1835	2015-02-18 20:00:37.229827
545	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 3\nsource_id: 208\nvariant_id: 107\nrating: 3\n	1	\N	\N	ff094901-60d3-422e-a69c-f427e686624c	2015-02-18 20:00:37.300375
546	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 21\nsource_id: 206\nvariant_id: 107\nrating: 3\n	1	\N	\N	2efab12f-7b01-40fe-a719-8700b11a2007	2015-02-18 20:00:37.365687
547	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 21\nsource_id: 209\nvariant_id: 107\nrating: 2\n	1	\N	\N	6a4ed01b-11eb-49dd-9173-9731ad4987d6	2015-02-18 20:00:37.440364
548	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates and shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  PIK3CA\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 21\nsource_id: 105\nvariant_id: 108\nrating: 3\n	1	\N	\N	315ea0f4-f745-4c18-b921-657f5d266cbf	2015-02-18 20:00:37.495803
549	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 21\nsource_id: 105\nvariant_id: 109\nrating: 3\n	1	\N	\N	0ad6959c-1de8-4eb3-a2a8-cfb5346dccdd	2015-02-18 20:00:37.554465
550	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 55\ndisease_id: 21\nsource_id: 178\nvariant_id: 110\nrating: 4\n	1	\N	\N	acf81d8a-db28-4ec5-b22b-722d5794bea1	2015-02-18 20:00:37.619677
551	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase\n  resistance to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 29\ndisease_id: 21\nsource_id: 206\nvariant_id: 110\nrating: 3\n	1	\N	\N	12ebe5cb-fd22-4d1c-9814-5e2b6c3ec929	2015-02-18 20:00:37.667519
552	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase\n  resistance to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 21\nsource_id: 206\nvariant_id: 110\nrating: 3\n	1	\N	\N	c9e82294-628b-4756-9e8a-91507e0e1acc	2015-02-18 20:00:37.7122
553	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 63\ndisease_id: 3\nsource_id: 207\nvariant_id: 110\nrating: 4\n	1	\N	\N	470dec13-2a7c-4903-a62b-df5d54fc7f5e	2015-02-18 20:00:37.766985
554	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 3\nsource_id: 208\nvariant_id: 110\nrating: 3\n	1	\N	\N	707ea4fb-d417-4f9c-b4d0-117f79275f4f	2015-02-18 20:00:37.81841
555	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 21\nsource_id: 206\nvariant_id: 110\nrating: 3\n	1	\N	\N	01f03454-82fb-4906-ba72-1a43362f7d7f	2015-02-18 20:00:37.868708
556	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 44\nsource_id: 210\nvariant_id: 111\nrating: 5\n	1	\N	\N	c2eef3ca-4c70-45e4-af9e-7841b3ef4682	2015-02-18 20:00:37.920175
557	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 44\nsource_id: 210\nvariant_id: 114\nrating: 5\n	1	\N	\N	ae6cb9bb-b985-4422-aaae-b5238ef1fff4	2015-02-18 20:00:37.971929
558	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 12\nsource_id: 211\nvariant_id: 116\nrating: 5\n	1	\N	\N	f8551a4d-33a7-455e-be15-7e8ed114cb88	2015-02-18 20:00:38.021709
559	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 in vitro led to strong repression of tyroid cancer cell growth.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 64\ndisease_id: 12\nsource_id: 212\nvariant_id: 116\nrating: 3\n	1	\N	\N	83cbe936-c968-4169-b5e0-b5f68b9bbb12	2015-02-18 20:00:38.072745
560	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 12\nsource_id: 213\nvariant_id: 116\nrating: 4\n	1	\N	\N	dc3c2d6a-960b-4e05-8e0b-00a043e6acb7	2015-02-18 20:00:38.139792
561	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 35\nsource_id: 214\nvariant_id: 117\nrating: 3\n	1	\N	\N	9383a3df-0f7e-485b-84fb-8db21a0fc3e0	2015-02-18 20:00:38.191572
562	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 35\nsource_id: 214\nvariant_id: 118\nrating: 3\n	1	\N	\N	9cec4f35-5925-4a2b-be36-8ecfe84f06ee	2015-02-18 20:00:38.240916
563	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumor with wild type TP53.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 65\ndisease_id: 3\nsource_id: 215\nvariant_id: 119\nrating: 3\n	1	\N	\N	d3de7bbe-ea36-4dd1-bd3e-7cb9f8b1c44d	2015-02-18 20:00:38.301647
564	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 216\nvariant_id: 119\nrating: 3\n	1	\N	\N	541f7707-e37e-4924-bde1-591c59eaa63e	2015-02-18 20:00:38.357843
565	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R248 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 217\nvariant_id: 120\nrating: 3\n	1	\N	\N	30d44e10-821c-4ed0-b4c2-d7ac476995da	2015-02-18 20:00:38.408019
566	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 216\nvariant_id: 120\nrating: 3\n	1	\N	\N	5b097758-f351-468e-9aab-9c07b2f69870	2015-02-18 20:00:38.463097
567	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 216\nvariant_id: 121\nrating: 3\n	1	\N	\N	74e485bf-c1fd-4034-b0cc-6e17e6709609	2015-02-18 20:00:38.554726
568	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as R249 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 217\nvariant_id: 122\nrating: 3\n	1	\N	\N	7d6b9721-3c54-4d88-a05d-44131d443bba	2015-02-18 20:00:38.610698
569	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as R249 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 217\nvariant_id: 123\nrating: 3\n	1	\N	\N	2df33df6-4bab-48f4-987c-940e77ab172c	2015-02-18 20:00:38.665275
570	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R273 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 217\nvariant_id: 124\nrating: 3\n	1	\N	\N	0cb9deb1-812a-49c4-8c4c-41b05205c480	2015-02-18 20:00:38.717045
571	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 216\nvariant_id: 124\nrating: 3\n	1	\N	\N	a7ace52f-0865-47de-9669-b149879b0133	2015-02-18 20:00:38.76992
572	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R273 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 217\nvariant_id: 125\nrating: 3\n	1	\N	\N	83895a02-ce4f-4c32-81ce-ad8bf32fd831	2015-02-18 20:00:38.817332
573	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 216\nvariant_id: 125\nrating: 3\n	1	\N	\N	5abcdb4d-61ca-425f-a49e-42256fedd5ed	2015-02-18 20:00:38.873457
574	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as V173 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 3\nsource_id: 217\nvariant_id: 126\nrating: 3\n	1	\N	\N	e94b603b-1fde-4a9b-ab2a-7aeb62c384dc	2015-02-18 20:00:38.927481
575	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 218\nvariant_id: 130\nrating: 3\n	1	\N	\N	a7b6ea98-2590-42ad-8bd9-c1113c8d9c4d	2015-02-18 20:00:38.98505
576	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 218\nvariant_id: 130\nrating: 2\n	1	\N	\N	807e921e-aec0-4bea-aec1-db151140d293	2015-02-18 20:00:39.041055
577	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 35\nsource_id: 218\nvariant_id: 130\nrating: 2\n	1	\N	\N	86489016-9f26-4eb6-8ca6-ef1309a3a77c	2015-02-18 20:00:39.093268
578	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 35\nsource_id: 219\nvariant_id: 130\nrating: 3\n	1	\N	\N	19fea9f7-c7f8-4410-a7c2-f01673ed920d	2015-02-18 20:00:39.148229
579	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 218\nvariant_id: 131\nrating: 3\n	1	\N	\N	d3e3c8b8-5c22-431a-a6ba-1c08f49986fe	2015-02-18 20:00:39.209423
580	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 13\nsource_id: 218\nvariant_id: 131\nrating: 2\n	1	\N	\N	7d0e67b5-349c-4acc-8269-abe3b6a5c0fd	2015-02-18 20:00:39.264883
581	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 35\nsource_id: 218\nvariant_id: 131\nrating: 2\n	1	\N	\N	a73a8d3b-e1e4-46d4-b601-e3ecb8112188	2015-02-18 20:00:39.312529
582	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as S34Y/F\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 2\ndisease_id: 35\nsource_id: 219\nvariant_id: 131\nrating: 3\n	1	\N	\N	f33f2d18-4b28-40aa-8f2d-5db5b719ac1e	2015-02-18 20:00:39.367905
583	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 220\nvariant_id: 132\nrating: 3\n	1	\N	\N	945cf9a4-5af1-4758-8995-36f6a30bf916	2015-02-18 20:00:39.428732
584	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 221\nvariant_id: 132\nrating: 4\n	1	\N	\N	791abf3b-0d91-4269-8ed5-03f0298763a7	2015-02-18 20:00:39.480281
585	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No difference in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 221\nvariant_id: 132\nrating: 3\n	1	\N	\N	837e877a-9d9f-47f0-a89e-0ae27b9d545b	2015-02-18 20:00:39.527854
586	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 220\nvariant_id: 132\nrating: 3\n	1	\N	\N	7ad6c83f-8bca-4b1c-95e3-61f8d736ed87	2015-02-18 20:00:39.577895
587	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 220\nvariant_id: 132\nrating: 3\n	1	\N	\N	40447d89-cc3a-4e15-95c3-843b1589df12	2015-02-18 20:00:39.62771
588	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 222\nvariant_id: 132\nrating: 3\n	1	\N	\N	1bba26a6-3e2a-4299-9d64-04f52d0d28b7	2015-02-18 20:00:39.68375
589	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 221\nvariant_id: 132\nrating: 3\n	1	\N	\N	0bb74c2f-3c9d-4aec-85bc-e2497b41370e	2015-02-18 20:00:39.731478
590	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations in WT1 were associated with a worse overall prognosis than patients\n  wildtype for WT1 in young patients (15-50), primarily because of increased risk\n  of disease recurrence\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 223\nvariant_id: 132\nrating: 2\n	1	\N	\N	41e941ee-f390-4453-9fcb-44490b7c55c5	2015-02-18 20:00:39.790712
591	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations  in WT1 were associated with increased risk of recurrence in young\n  patients (15-50) with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 223\nvariant_id: 132\nrating: 2\n	1	\N	\N	e5a641ef-ac88-414e-9f28-d587c5590167	2015-02-18 20:00:39.843143
592	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 220\nvariant_id: 133\nrating: 3\n	1	\N	\N	e137d1c3-744a-4eff-b3c1-cd2e47ce2ef3	2015-02-18 20:00:39.891042
593	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 221\nvariant_id: 133\nrating: 4\n	1	\N	\N	5fbdbd57-56ce-4e0c-9b67-4cd05e806ddc	2015-02-18 20:00:39.948938
594	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No difference in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 221\nvariant_id: 133\nrating: 3\n	1	\N	\N	6c1b5314-da73-412c-a287-655081c38962	2015-02-18 20:00:40.011714
595	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 220\nvariant_id: 133\nrating: 3\n	1	\N	\N	905527a4-455f-4ff1-a697-ee2cb728a484	2015-02-18 20:00:40.065511
596	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 220\nvariant_id: 133\nrating: 3\n	1	\N	\N	077092e1-93f7-45eb-9c37-e7887eb8051a	2015-02-18 20:00:40.162099
597	417	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 222\nvariant_id: 133\nrating: 3\n	1	\N	\N	04f77076-8036-47c3-ae70-bfba43d2165e	2015-02-18 20:00:40.220345
598	418	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 2\ndisease_id: 13\nsource_id: 221\nvariant_id: 133\nrating: 3\n	1	\N	\N	68770309-1fd1-44a9-80e7-c67c19ff5d62	2015-02-18 20:00:40.276963
599	419	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The T790M mutation in EGFR has been shown to confer resistance to the tyrosine\n  kinase inhibitor erlotinib, and patients harboring this mutation that are placed\n  on the drug are likely to relapse.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 44\ndisease_id: 45\nsource_id: 224\nvariant_id: 34\nrating: 5\n	1	\N	\N	f59a05ef-a4ab-47da-99c6-e1b7addba27d	2015-02-18 20:00:40.333873
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 599, true);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
1	3908	non-small cell lung carcinoma	2015-02-18 20:00:16.486438	2015-02-18 20:02:45.404751
2	5015	hepatocellular fibrolamellar carcinoma	2015-02-18 20:00:16.799068	2015-02-18 20:02:45.665562
3	1612	breast cancer	2015-02-18 20:00:16.943284	2015-02-18 20:02:46.096018
4	1909	melanoma	2015-02-18 20:00:17.828824	2015-02-18 20:02:46.389312
5	9253	gastrointestinal stromal tumor	2015-02-18 20:00:18.278505	2015-02-18 20:02:46.531042
6	9119	acute myeloid leukemia	2015-02-18 20:00:18.403142	2015-02-18 20:02:46.700838
7	4947	cholangiocarcinoma	2015-02-18 20:00:18.529801	2015-02-18 20:02:46.923089
8	3907	lung squamous cell carcinoma	2015-02-18 20:00:18.721105	2015-02-18 20:02:47.126465
9	1781	thyroid cancer	2015-02-18 20:00:18.827904	2015-02-18 20:02:47.45637
10	10283	prostate cancer	2015-02-18 20:00:18.898284	2015-02-18 20:02:47.701046
11	8552	chronic myeloid leukemia	2015-02-18 20:00:19.506283	2015-02-18 20:02:47.851589
12	3973	thyroid medullary carcinoma	2015-02-18 20:00:19.682282	2015-02-18 20:02:48.039117
13	9119	acute myeloid leukemia	2015-02-18 20:00:19.921355	2015-02-18 20:02:48.186002
14	1612	breast cancer	2015-02-18 20:00:22.352463	2015-02-18 20:02:48.487878
15	3068	glioblastoma multiforme	2015-02-18 20:00:22.429922	2015-02-18 20:02:48.728359
16	6477	invasive bladder transitional cell carcinoma	2015-02-18 20:00:22.488731	2015-02-18 20:02:48.942951
17	4007	bladder carcinoma	2015-02-18 20:00:22.611556	2015-02-18 20:02:49.148633
18	1612	breast cancer	2015-02-18 20:00:23.169048	2015-02-18 20:02:49.462992
19	1115	sarcoma	2015-02-18 20:00:23.279016	2015-02-18 20:02:49.648089
20	769	neuroblastoma	2015-02-18 20:00:23.342687	2015-02-18 20:02:49.821961
21	9256	colorectal cancer	2015-02-18 20:00:23.852569	2015-02-18 20:02:49.990514
22	9256	colorectal cancer	2015-02-18 20:00:24.366312	2015-02-18 20:02:50.152851
23	8618	oral cavity cancer	2015-02-18 20:00:26.042488	2015-02-18 20:02:50.406559
24	0050746	mantle cell lymphoma	2015-02-18 20:00:26.104217	2015-02-18 20:02:50.545048
25	10534	stomach cancer	2015-02-18 20:00:26.16416	2015-02-18 20:02:50.804411
26	715	T-cell leukemia	2015-02-18 20:00:26.516878	2015-02-18 20:02:50.953317
27	5635	gastric adenosquamous carcinoma	2015-02-18 20:00:26.630523	2015-02-18 20:02:51.159954
28	1324	lung cancer	2015-02-18 20:00:26.743213	2015-02-18 20:02:51.390141
29	0060075	estrogen-receptor positive breast cancer	2015-02-18 20:00:26.81965	2015-02-18 20:02:51.522704
30	9952	acute lymphocytic leukemia	2015-02-18 20:00:27.492514	2015-02-18 20:02:51.684622
31	3909	bronchogenic lung adenocarcinoma	2015-02-18 20:00:31.550526	2015-02-18 20:02:51.872913
32	3069	astrocytoma	2015-02-18 20:00:31.704582	2015-02-18 20:02:52.144415
33	3068	glioblastoma multiforme	2015-02-18 20:00:31.750834	2015-02-18 20:02:52.383706
34	3069	astrocytoma	2015-02-18 20:00:31.85731	2015-02-18 20:02:52.629879
35	4972	myelodysplastic myeloproliferative cancer	2015-02-18 20:00:32.239682	2015-02-18 20:02:52.789942
36	10747	lymphoid leukemia	2015-02-18 20:00:32.572703	2015-02-18 20:02:52.941305
37	4960	bone marrow cancer	2015-02-18 20:00:32.726109	2015-02-18 20:02:53.12442
38	8997	polycythemia vera	2015-02-18 20:00:32.781399	2015-02-18 20:02:53.260956
39	1240	leukemia	2015-02-18 20:00:32.840526	2015-02-18 20:02:53.504529
40	1240	leukemia	2015-02-18 20:00:33.82125	2015-02-18 20:02:53.758194
41	9538	multiple myeloma	2015-02-18 20:00:34.027089	2015-02-18 20:02:53.99374
42	9119	acute myeloid leukemia	2015-02-18 20:00:34.66058	2015-02-18 20:02:54.166469
43	9253	gastrointestinal stromal tumor	2015-02-18 20:00:36.15673	2015-02-18 20:02:54.336977
44	9119	acute myeloid leukemia	2015-02-18 20:00:37.907121	2015-02-18 20:02:54.513353
45	3908	non-small cell lung carcinoma	2015-02-18 20:00:40.319366	2015-02-18 20:02:54.75974
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 45, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Crizotinib	\N	2015-02-18 20:00:16.480428	2015-02-18 20:00:16.480428
2	N/A	\N	2015-02-18 20:00:16.79527	2015-02-18 20:00:16.79527
3	Tamoxifen	\N	2015-02-18 20:00:16.939167	2015-02-18 20:00:16.939167
4	Fulvestrant	\N	2015-02-18 20:00:16.996312	2015-02-18 20:00:16.996312
5	Imatinib	\N	2015-02-18 20:00:17.824071	2015-02-18 20:00:17.824071
6	Nilotinib	\N	2015-02-18 20:00:17.878257	2015-02-18 20:00:17.878257
7	Sorafenib	\N	2015-02-18 20:00:17.925952	2015-02-18 20:00:17.925952
8	Dasatinib	\N	2015-02-18 20:00:17.993812	2015-02-18 20:00:17.993812
9	SU11248	\N	2015-02-18 20:00:18.340012	2015-02-18 20:00:18.340012
10	Pazopanib	\N	2015-02-18 20:00:18.463444	2015-02-18 20:00:18.463444
11	Ponatinib	\N	2015-02-18 20:00:18.773951	2015-02-18 20:00:18.773951
12	Neratinib	\N	2015-02-18 20:00:19.074883	2015-02-18 20:00:19.074883
13	Motesanib	\N	2015-02-18 20:00:19.678764	2015-02-18 20:00:19.678764
14	ATRA	\N	2015-02-18 20:00:20.3406	2015-02-18 20:00:20.3406
15	NSC348884	\N	2015-02-18 20:00:20.510749	2015-02-18 20:00:20.510749
16	Daunorubicin	\N	2015-02-18 20:00:20.709187	2015-02-18 20:00:20.709187
17	Valproic acid	\N	2015-02-18 20:00:20.763985	2015-02-18 20:00:20.763985
18	Dabrafenib	\N	2015-02-18 20:00:22.103924	2015-02-18 20:00:22.103924
19	Everolimus	\N	2015-02-18 20:00:22.485234	2015-02-18 20:00:22.485234
20	Rapamycin	\N	2015-02-18 20:00:22.550495	2015-02-18 20:00:22.550495
21	Arsenic trioxide	\N	2015-02-18 20:00:22.896551	2015-02-18 20:00:22.896551
22	MK-2206	\N	2015-02-18 20:00:23.165389	2015-02-18 20:00:23.165389
23	CH5424802	\N	2015-02-18 20:00:23.226777	2015-02-18 20:00:23.226777
24	TAE684	\N	2015-02-18 20:00:23.411732	2015-02-18 20:00:23.411732
25	Trametinib	\N	2015-02-18 20:00:23.627581	2015-02-18 20:00:23.627581
26	Panitumumab	\N	2015-02-18 20:00:23.848867	2015-02-18 20:00:23.848867
27	Dacarbazine	\N	2015-02-18 20:00:24.257484	2015-02-18 20:00:24.257484
28	Temozolomide	\N	2015-02-18 20:00:24.300366	2015-02-18 20:00:24.300366
29	Cetuximab	\N	2015-02-18 20:00:24.362762	2015-02-18 20:00:24.362762
30	PD0325901	\N	2015-02-18 20:00:24.784827	2015-02-18 20:00:24.784827
31	PLX4720	\N	2015-02-18 20:00:24.832714	2015-02-18 20:00:24.832714
32	Ipilimumab	\N	2015-02-18 20:00:24.894242	2015-02-18 20:00:24.894242
33	Capecitabine	\N	2015-02-18 20:00:25.001691	2015-02-18 20:00:25.001691
34	Vemurafenib	\N	2015-02-18 20:00:25.053572	2015-02-18 20:00:25.053572
35	Bevacizumab	\N	2015-02-18 20:00:25.123501	2015-02-18 20:00:25.123501
36	nutlin-3	\N	2015-02-18 20:00:25.228863	2015-02-18 20:00:25.228863
37	AZD6244	\N	2015-02-18 20:00:25.710392	2015-02-18 20:00:25.710392
38	Palbociclib (PD-0332991)	\N	2015-02-18 20:00:26.51297	2015-02-18 20:00:26.51297
39	BYL719	\N	2015-02-18 20:00:26.816167	2015-02-18 20:00:26.816167
40	Palbociclib	\N	2015-02-18 20:00:26.872705	2015-02-18 20:00:26.872705
41	GW-2580	\N	2015-02-18 20:00:27.538163	2015-02-18 20:00:27.538163
42	Idarubicin	\N	2015-02-18 20:00:28.02063	2015-02-18 20:00:28.02063
43	Gefitinib	\N	2015-02-18 20:00:29.140735	2015-02-18 20:00:29.140735
44	Erlotinib	\N	2015-02-18 20:00:29.185866	2015-02-18 20:00:29.185866
45	Premetrexed	\N	2015-02-18 20:00:29.527058	2015-02-18 20:00:29.527058
46	Stauroporine	\N	2015-02-18 20:00:29.581869	2015-02-18 20:00:29.581869
47	Lapatinib	\N	2015-02-18 20:00:29.863389	2015-02-18 20:00:29.863389
48	SU5614	\N	2015-02-18 20:00:30.128191	2015-02-18 20:00:30.128191
49	CEP701	\N	2015-02-18 20:00:30.496222	2015-02-18 20:00:30.496222
50	AG1296	\N	2015-02-18 20:00:30.550624	2015-02-18 20:00:30.550624
51	Bortezomib	\N	2015-02-18 20:00:31.545272	2015-02-18 20:00:31.545272
52	Fasudil	\N	2015-02-18 20:00:31.599268	2015-02-18 20:00:31.599268
53	Pegylated IFN-2a	\N	2015-02-18 20:00:32.7778	2015-02-18 20:00:32.7778
54	TG101348	\N	2015-02-18 20:00:32.836852	2015-02-18 20:00:32.836852
55	Regorafenib	\N	2015-02-18 20:00:33.756373	2015-02-18 20:00:33.756373
56	NVP-BEZ235	\N	2015-02-18 20:00:34.459584	2015-02-18 20:00:34.459584
57	ARRY-142886	\N	2015-02-18 20:00:34.505138	2015-02-18 20:00:34.505138
58	O(6)-benzylguanine	\N	2015-02-18 20:00:35.20404	2015-02-18 20:00:35.20404
59	Carmustine	\N	2015-02-18 20:00:35.304354	2015-02-18 20:00:35.304354
60	17-AAG	\N	2015-02-18 20:00:35.786855	2015-02-18 20:00:35.786855
61	Temozolomide	\N	2015-02-18 20:00:36.058787	2015-02-18 20:00:36.058787
62	Crenolanib	\N	2015-02-18 20:00:36.152856	2015-02-18 20:00:36.152856
63	CH5132799	\N	2015-02-18 20:00:36.784139	2015-02-18 20:00:36.784139
64	AZD1480	\N	2015-02-18 20:00:38.056377	2015-02-18 20:00:38.056377
65	Doxorubicin	\N	2015-02-18 20:00:38.279006	2015-02-18 20:00:38.279006
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 65, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	B	\N	2015-02-18 20:00:16.49358	2015-02-18 20:00:16.49358
2	C	\N	2015-02-18 20:00:16.73412	2015-02-18 20:00:16.73412
3	D	\N	2015-02-18 20:00:20.837689	2015-02-18 20:00:20.837689
4	A	\N	2015-02-18 20:00:22.74352	2015-02-18 20:00:22.74352
\.


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Predictive	2015-02-18 20:00:16.491169	2015-02-18 20:00:16.491169
2	Diagnostic	2015-02-18 20:00:16.801202	2015-02-18 20:00:16.801202
3	Prognostic	2015-02-18 20:00:18.404904	2015-02-18 20:00:18.404904
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-02-18 20:00:00.86171	2015-02-18 20:00:00.86171	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-02-18 20:00:01.382511	2015-02-18 20:00:01.382511	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-02-18 20:00:01.773309	2015-02-18 20:00:01.773309	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-02-18 20:00:02.421464	2015-02-18 20:00:02.421464	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-02-18 20:00:02.839805	2015-02-18 20:00:02.839805	\N
6	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-02-18 20:00:03.163646	2015-02-18 20:00:03.163646	\N
7	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-02-18 20:00:03.46512	2015-02-18 20:00:03.46512	\N
8	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-02-18 20:00:03.685923	2015-02-18 20:00:03.685923	\N
9	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-02-18 20:00:03.940078	2015-02-18 20:00:03.940078	\N
10	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-02-18 20:00:04.185504	2015-02-18 20:00:04.185504	\N
11	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-02-18 20:00:04.411547	2015-02-18 20:00:04.411547	\N
12	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-02-18 20:00:04.633855	2015-02-18 20:00:04.633855	\N
13	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-02-18 20:00:04.85339	2015-02-18 20:00:04.85339	\N
14	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-02-18 20:00:05.06145	2015-02-18 20:00:05.06145	\N
15	5566	PRKACA	"PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity."	protein kinase, cAMP-dependent, catalytic, alpha	2015-02-18 20:00:05.295214	2015-02-18 20:00:05.295214	\N
16	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-02-18 20:00:05.523864	2015-02-18 20:00:05.523864	\N
17	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-02-18 20:00:05.773982	2015-02-18 20:00:05.773982	\N
18	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown senitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-02-18 20:00:05.998682	2015-02-18 20:00:05.998682	\N
19	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-02-18 20:00:06.237604	2015-02-18 20:00:06.237604	\N
20	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-02-18 20:00:06.455189	2015-02-18 20:00:06.455189	\N
21	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to the activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFRs can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-02-18 20:00:06.671745	2015-02-18 20:00:06.671745	\N
22	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-02-18 20:00:06.900816	2015-02-18 20:00:06.900816	\N
23	2624	GATA2	N/A	GATA binding protein 2	2015-02-18 20:00:07.132499	2015-02-18 20:00:07.132499	\N
24	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-02-18 20:00:07.32415	2015-02-18 20:00:07.32415	\N
25	3418	IDH2	N/A	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-02-18 20:00:07.540034	2015-02-18 20:00:07.540034	\N
26	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders.	Janus kinase 2	2015-02-18 20:00:07.747845	2015-02-18 20:00:07.747845	\N
27	3815	KIT	N/A	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-02-18 20:00:07.944683	2015-02-18 20:00:07.944683	\N
28	3845	KRAS	N/A	Kirsten rat sarcoma viral oncogene homolog	2015-02-18 20:00:08.137552	2015-02-18 20:00:08.137552	\N
29	5604	MAP2K1	N/A	mitogen-activated protein kinase kinase 1	2015-02-18 20:00:08.342986	2015-02-18 20:00:08.342986	\N
30	4209	MEF2D	N/A	myocyte enhancer factor 2D	2015-02-18 20:00:08.55531	2015-02-18 20:00:08.55531	\N
31	1436	CSF1R	N/A	colony stimulating factor 1 receptor	2015-02-18 20:00:08.777854	2015-02-18 20:00:08.777854	\N
32	4255	MGMT	N/A	O-6-methylguanine-DNA methyltransferase	2015-02-18 20:00:08.999881	2015-02-18 20:00:08.999881	\N
33	4869	NPM1	AML with mutated NPM1' is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-02-18 20:00:09.216876	2015-02-18 20:00:09.216876	\N
34	4893	NRAS	N/A	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-02-18 20:00:09.46863	2015-02-18 20:00:09.46863	\N
35	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-02-18 20:00:09.737838	2015-02-18 20:00:09.737838	\N
36	5156	PDGFRA	N/A	platelet-derived growth factor receptor, alpha polypeptide	2015-02-18 20:00:09.938919	2015-02-18 20:00:09.938919	\N
37	5371	PML	N/A	promyelocytic leukemia	2015-02-18 20:00:10.168546	2015-02-18 20:00:10.168546	\N
38	5914	RARa	N/A	retinoic acid receptor, alpha	2015-02-18 20:00:10.360367	2015-02-18 20:00:10.360367	\N
39	5728	PTEN	N/A	phosphatase and tensin homolog	2015-02-18 20:00:10.561486	2015-02-18 20:00:10.561486	\N
40	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-02-18 20:00:10.901655	2015-02-18 20:00:10.901655	\N
41	861	RUNX1	N/A	runt-related transcription factor 1	2015-02-18 20:00:11.103148	2015-02-18 20:00:11.103148	\N
42	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-02-18 20:00:11.322717	2015-02-18 20:00:11.322717	\N
43	7157	TP53	N/A	tumor protein p53	2015-02-18 20:00:11.55713	2015-02-18 20:00:11.55713	\N
44	7248	TSC1	N/A	tuberous sclerosis 1	2015-02-18 20:00:11.769719	2015-02-18 20:00:11.769719	\N
45	7249	TSC2	N/A	tuberous sclerosis 2	2015-02-18 20:00:11.961507	2015-02-18 20:00:11.961507	\N
46	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-02-18 20:00:12.192556	2015-02-18 20:00:12.192556	\N
47	7490	WT1	N/A	Wilms tumor 1	2015-02-18 20:00:12.423413	2015-02-18 20:00:12.423413	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-02-18 20:00:00.890636	2015-02-18 20:00:40.980747
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-02-18 20:00:00.903117	2015-02-18 20:00:41.384228
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-02-18 20:00:01.38522	2015-02-18 20:00:41.800678
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-02-18 20:00:01.39036	2015-02-18 20:00:42.153512
5	15676015	\N	Lee et al., 2005, APMIS	2015-02-18 20:00:01.777142	2015-02-18 20:00:42.636227
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-02-18 20:00:01.785585	2015-02-18 20:00:43.056032
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-02-18 20:00:02.424469	2015-02-18 20:00:43.4714
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-02-18 20:00:02.42961	2015-02-18 20:00:44.30855
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-02-18 20:00:02.843681	2015-02-18 20:00:44.733202
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-02-18 20:00:02.849208	2015-02-18 20:00:45.143676
11	12432268	\N	Diehl, Cancer Biol. Ther.	2015-02-18 20:00:03.16681	2015-02-18 20:00:45.561605
12	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-02-18 20:00:03.171978	2015-02-18 20:00:45.979173
13	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-02-18 20:00:03.943305	2015-02-18 20:00:46.399065
14	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-02-18 20:00:04.188982	2015-02-18 20:00:46.816943
15	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-02-18 20:00:04.194516	2015-02-18 20:00:47.234852
16	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-02-18 20:00:04.636048	2015-02-18 20:00:47.658443
17	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-18 20:00:04.640784	2015-02-18 20:00:48.067968
18	19357394	\N	Vardiman et al., 2009, Blood	2015-02-18 20:00:04.858172	2015-02-18 20:00:48.487589
19	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-02-18 20:00:04.862808	2015-02-18 20:00:48.903516
20	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-02-18 20:00:04.867335	2015-02-18 20:00:49.323547
21	24578576	\N	Honeyman et al., 2014, Science	2015-02-18 20:00:05.063721	2015-02-18 20:00:49.743788
22	24909179	\N	Moody et al., 2014, Oncogene	2015-02-18 20:00:05.302051	2015-02-18 20:00:50.259713
23	24167195	\N	Kim et al., 2013, Blood	2015-02-18 20:00:05.526353	2015-02-18 20:00:50.67967
24	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-02-18 20:00:05.530473	2015-02-18 20:00:51.098006
25	23953842	\N	Yewale et al., 2013, Biomaterials	2015-02-18 20:00:05.777462	2015-02-18 20:00:51.51522
26	18712184	\N	Charpidou et al., In Vivo	2015-02-18 20:00:05.78306	2015-02-18 20:00:51.934071
27	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-02-18 20:00:06.002278	2015-02-18 20:00:52.350036
28	10878580	\N	Yu et al., 2000, Bioessays	2015-02-18 20:00:06.008624	2015-02-18 20:00:52.767884
29	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-02-18 20:00:06.240754	2015-02-18 20:00:53.188714
30	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-02-18 20:00:06.246408	2015-02-18 20:00:53.603364
31	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-02-18 20:00:06.458765	2015-02-18 20:00:54.020391
32	23558953	\N	Wu et al., 2013, Cancer Discov	2015-02-18 20:00:06.464436	2015-02-18 20:00:54.441548
33	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-02-18 20:00:06.904135	2015-02-18 20:00:54.855809
34	23226219	\N	Chou et al., 2012, PLoS ONE	2015-02-18 20:00:09.220213	2015-02-18 20:00:55.278154
35	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-18 20:00:10.90429	2015-02-18 20:00:55.796226
36	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-02-18 20:00:10.909093	2015-02-18 20:00:56.218186
37	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-18 20:00:11.325914	2015-02-18 20:00:56.63524
38	24136165	\N	Cazzola et al., 2013, Blood	2015-02-18 20:00:11.331044	2015-02-18 20:00:57.053405
39	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-02-18 20:00:12.195836	2015-02-18 20:00:57.559416
40	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-02-18 20:00:12.201769	2015-02-18 20:00:59.12845
41	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-02-18 20:00:16.484141	2015-02-18 20:00:59.547565
42	21030459		Sasaki et al., 2010, Cancer Res.	2015-02-18 20:00:16.73183	2015-02-18 20:00:59.977774
43	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-02-18 20:00:16.797153	2015-02-18 20:01:00.394047
44	24185510		Robinson et al., 2013, Nat. Genet.	2015-02-18 20:00:16.94143	2015-02-18 20:01:00.813469
45	24185512		Toy et al., 2013, Nat. Genet.	2015-02-18 20:00:17.060097	2015-02-18 20:01:01.221568
46	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-02-18 20:00:17.826424	2015-02-18 20:01:01.648187
47	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-02-18 20:00:17.997099	2015-02-18 20:01:02.063017
48	18421059	Observational study	Hodi et al., 2008, J. Clin. Oncol.	2015-02-18 20:00:18.222396	2015-02-18 20:01:02.480928
49	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-02-18 20:00:18.276628	2015-02-18 20:01:02.899452
50	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-02-18 20:00:18.341868	2015-02-18 20:01:03.533016
51	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-02-18 20:00:18.401249	2015-02-18 20:01:03.884672
52	23558953		Wu et al., 2013, Cancer Discov	2015-02-18 20:00:18.465226	2015-02-18 20:01:04.363859
53	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-02-18 20:00:18.775783	2015-02-18 20:01:04.783589
54	23220880		Bose et al., 2013, Cancer Discov	2015-02-18 20:00:19.076754	2015-02-18 20:01:05.199172
55	20537386		An et al., 2010, Leuk. Res.	2015-02-18 20:00:19.502645	2015-02-18 20:01:05.615067
56	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-02-18 20:00:19.561641	2015-02-18 20:01:06.244207
57	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-02-18 20:00:19.680571	2015-02-18 20:01:06.976909
58	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-02-18 20:00:19.814668	2015-02-18 20:01:07.394798
59	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-02-18 20:00:19.919503	2015-02-18 20:01:07.809036
60	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-02-18 20:00:19.978352	2015-02-18 20:01:08.238863
61	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-02-18 20:00:20.203863	2015-02-18 20:01:08.650037
62	17957027	Retrospective study	Gale et al., 2008, Blood	2015-02-18 20:00:20.265277	2015-02-18 20:01:09.207718
63	19965647		Burnett et al., 2010, Blood	2015-02-18 20:00:20.342419	2015-02-18 20:01:09.583899
64	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-02-18 20:00:20.393915	2015-02-18 20:01:09.958435
65	21719597	In vitro study	Balusu et al., 2011, Blood	2015-02-18 20:00:20.514825	2015-02-18 20:01:10.424929
66	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-02-18 20:00:20.710971	2015-02-18 20:01:10.843262
67	24797300		Tassara et al., 2014, Blood	2015-02-18 20:00:20.778798	2015-02-18 20:01:11.360281
68	24927407		Ehninger et al., 2014, Blood Cancer J	2015-02-18 20:00:20.835327	2015-02-18 20:01:11.885574
69	16051734	Retrospective study	Dhner et al., 2005, Blood	2015-02-18 20:00:21.013991	2015-02-18 20:01:12.299592
70	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-02-18 20:00:21.06868	2015-02-18 20:01:12.718616
71	24855211		Linch et al., 2014, Blood	2015-02-18 20:00:21.126604	2015-02-18 20:01:13.446284
72	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-02-18 20:00:21.238616	2015-02-18 20:01:13.86431
73	19047294	Retrospective study	Bchner et al., 2009, J. Clin. Oncol.	2015-02-18 20:00:21.292217	2015-02-18 20:01:14.282583
74	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-02-18 20:00:21.349781	2015-02-18 20:01:14.701024
75	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-02-18 20:00:21.47917	2015-02-18 20:01:15.120231
76	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-02-18 20:00:21.526522	2015-02-18 20:01:16.487486
77	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-02-18 20:00:21.578604	2015-02-18 20:01:18.402091
78	21537333		Schnittger et al., 2011, Leukemia	2015-02-18 20:00:21.681931	2015-02-18 20:01:18.774982
79	24859829		Tian et al., 2014, Int. J. Hematol.	2015-02-18 20:00:21.792433	2015-02-18 20:01:19.202667
80	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-02-18 20:00:21.93311	2015-02-18 20:01:19.745303
81	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-02-18 20:00:22.04782	2015-02-18 20:01:20.133954
82	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-02-18 20:00:22.106013	2015-02-18 20:01:20.551397
83	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-02-18 20:00:22.168007	2015-02-18 20:01:22.474522
84	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-02-18 20:00:22.350658	2015-02-18 20:01:22.958581
85	22479427		Carico et al., 2012, PLoS ONE	2015-02-18 20:00:22.427641	2015-02-18 20:01:23.373781
86	22923433		Iyer et al., 2012, Science	2015-02-18 20:00:22.487025	2015-02-18 20:01:23.732115
87	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-02-18 20:00:22.552269	2015-02-18 20:01:24.102734
88	22923433	Retrospective study	Iyer et al., 2012, Science	2015-02-18 20:00:22.609768	2015-02-18 20:01:24.523942
89	25212276		Schmidt et al., 2014, Leukemia	2015-02-18 20:00:22.79453	2015-02-18 20:01:24.941096
90	12476305		Nimmanapalli et al., 2002, Oncogene	2015-02-18 20:00:22.844332	2015-02-18 20:01:25.353671
91	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-02-18 20:00:23.167236	2015-02-18 20:01:25.764075
92	21575866		Sakamoto et al., 2011, Cancer Cell	2015-02-18 20:00:23.230179	2015-02-18 20:01:27.319243
93	22072639		Bresler et al., 2011, Sci Transl Med	2015-02-18 20:00:23.340592	2015-02-18 20:01:27.66766
94	18923525		George et al., 2008, Nature	2015-02-18 20:00:23.413925	2015-02-18 20:01:28.075222
95	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-02-18 20:00:23.524547	2015-02-18 20:01:28.491127
96	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-02-18 20:00:23.630574	2015-02-18 20:01:29.429901
97	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-02-18 20:00:23.795913	2015-02-18 20:01:29.846177
98	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-02-18 20:00:23.850825	2015-02-18 20:01:30.279962
99	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-02-18 20:00:23.994191	2015-02-18 20:01:30.998324
100	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-02-18 20:00:24.094771	2015-02-18 20:01:31.42045
101	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-02-18 20:00:24.149782	2015-02-18 20:01:31.851421
102	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-02-18 20:00:24.206883	2015-02-18 20:01:32.253419
103	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-02-18 20:00:24.259468	2015-02-18 20:01:32.668311
104	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-02-18 20:00:24.364621	2015-02-18 20:01:33.06634
105	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-02-18 20:00:24.523743	2015-02-18 20:01:33.502204
106	21505228	In vitro study	Corcoran et al., 2011, Oncotarget	2015-02-18 20:00:24.581239	2015-02-18 20:01:34.697038
107	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-02-18 20:00:24.632466	2015-02-18 20:01:35.068697
108	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-02-18 20:00:24.687958	2015-02-18 20:01:36.946026
109	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-02-18 20:00:24.786618	2015-02-18 20:01:37.311875
110	24577748	Retrospective study	Ackerman et al., 2014, Cancer	2015-02-18 20:00:24.896339	2015-02-18 20:01:37.693276
111	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-02-18 20:00:25.004215	2015-02-18 20:01:39.204348
112	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-02-18 20:00:25.179491	2015-02-18 20:01:39.668447
113	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-02-18 20:00:25.336903	2015-02-18 20:01:40.082046
114	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-02-18 20:00:25.403133	2015-02-18 20:01:41.796982
115	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-02-18 20:00:25.45829	2015-02-18 20:01:42.380311
116	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-02-18 20:00:25.577451	2015-02-18 20:01:42.799309
117	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-02-18 20:00:25.712244	2015-02-18 20:01:43.217057
118	17070615		Gautschi et al., 2007, Lung Cancer	2015-02-18 20:00:25.767938	2015-02-18 20:01:43.637225
119	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-02-18 20:00:25.932296	2015-02-18 20:01:44.006724
120	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-02-18 20:00:26.040064	2015-02-18 20:01:44.473904
121	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-02-18 20:00:26.102388	2015-02-18 20:01:44.934034
122	10547574		Takano et al., 1999, J. Pathol.	2015-02-18 20:00:26.162258	2015-02-18 20:01:45.408999
123	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-02-18 20:00:26.411205	2015-02-18 20:01:45.825688
124	23079656		Sawai et al., 2012, Cancer Cell	2015-02-18 20:00:26.515068	2015-02-18 20:01:46.185734
125	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-02-18 20:00:26.570682	2015-02-18 20:01:46.563874
126	10224221		Donnellan et al., 1999, FASEB J.	2015-02-18 20:00:26.6286	2015-02-18 20:01:46.975418
127	25002028		Vora et al., 2014, Cancer Cell	2015-02-18 20:00:26.817937	2015-02-18 20:01:47.394633
128	23898052		Logan et al., 2013, Anticancer Res.	2015-02-18 20:00:26.874702	2015-02-18 20:01:48.337133
129	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-18 20:00:27.030918	2015-02-18 20:01:49.850389
130	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-02-18 20:00:27.144554	2015-02-18 20:01:50.214445
131	14726504	Retrospective study	Frhling et al., 2004, J. Clin. Oncol.	2015-02-18 20:00:27.337692	2015-02-18 20:01:51.570413
132	24186003		Lilljebjrn et al., 2014, Leukemia	2015-02-18 20:00:27.490599	2015-02-18 20:01:52.931758
133	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-02-18 20:00:27.589107	2015-02-18 20:01:53.350492
134	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-02-18 20:00:27.869023	2015-02-18 20:01:53.871006
135	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-02-18 20:00:27.924411	2015-02-18 20:01:54.265827
136	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-02-18 20:00:28.128217	2015-02-18 20:01:56.422989
137	24736073		Lim et al., 2014, J Thorac Oncol	2015-02-18 20:00:29.142606	2015-02-18 20:01:56.809637
138	24457318		Fukihara et al., 2014, Oncology	2015-02-18 20:00:29.23697	2015-02-18 20:01:58.255898
139	24585406		Li et al., 2014, Ann. Surg. Oncol.	2015-02-18 20:00:29.288824	2015-02-18 20:01:58.68851
140	24662454		Douillard et al., 2014, J Thorac Oncol	2015-02-18 20:00:29.374975	2015-02-18 20:01:59.975352
141	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-02-18 20:00:29.426547	2015-02-18 20:02:02.536707
142	24636847		Li et al., 2014, Lung Cancer	2015-02-18 20:00:29.482918	2015-02-18 20:02:02.956207
143	24658966		Ai et al., 2014, Amino Acids	2015-02-18 20:00:29.583635	2015-02-18 20:02:03.372636
144	24729716		Li et al., 2014, Onco Targets Ther	2015-02-18 20:00:29.638613	2015-02-18 20:02:04.250941
145	24623981		Ding et al., 2014, Onco Targets Ther	2015-02-18 20:00:29.695767	2015-02-18 20:02:04.626536
146	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-02-18 20:00:30.022634	2015-02-18 20:02:05.149904
147	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-02-18 20:00:30.130716	2015-02-18 20:02:05.491554
148	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-02-18 20:00:30.44074	2015-02-18 20:02:06.209119
149	14726387	Clinical trial	Smith et al., 2004, Blood	2015-02-18 20:00:30.498024	2015-02-18 20:02:06.984835
150	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-02-18 20:00:30.552459	2015-02-18 20:02:07.656995
151	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-02-18 20:00:30.80954	2015-02-18 20:02:08.074692
152	24801015		Port et al., 2014, Ann. Hematol.	2015-02-18 20:00:31.047878	2015-02-18 20:02:08.493631
153	22368270	Clinical trial	Man et al., 2012, Blood	2015-02-18 20:00:31.14083	2015-02-18 20:02:08.949797
154	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-02-18 20:00:31.493148	2015-02-18 20:02:09.341577
155	22624710		Barbacid, 2012, Cancer Cell	2015-02-18 20:00:31.547746	2015-02-18 20:02:09.805821
156	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-02-18 20:00:31.652037	2015-02-18 20:02:10.903555
157	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-02-18 20:00:31.70284	2015-02-18 20:02:11.727882
158	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-02-18 20:00:31.749137	2015-02-18 20:02:12.157524
159	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-02-18 20:00:31.804447	2015-02-18 20:02:12.533149
160	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-02-18 20:00:31.855345	2015-02-18 20:02:13.295483
161	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-02-18 20:00:31.910265	2015-02-18 20:02:13.712984
162	21596855		Green et al., 2011, Blood	2015-02-18 20:00:32.135803	2015-02-18 20:02:14.13207
163	20421455	Retrospective study	Thol et al., 2010, Blood	2015-02-18 20:00:32.187369	2015-02-18 20:02:14.735451
164	22033490		Patnaik et al., 2012, Leukemia	2015-02-18 20:00:32.237765	2015-02-18 20:02:15.175649
165	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-02-18 20:00:32.295539	2015-02-18 20:02:15.592051
166	21997850		Lin et al., 2012, Ann. Hematol.	2015-02-18 20:00:32.348051	2015-02-18 20:02:16.010163
167	16081687	Retrospective study	Levine et al., 2005, Blood	2015-02-18 20:00:32.570531	2015-02-18 20:02:16.431258
168	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-02-18 20:00:32.723053	2015-02-18 20:02:17.058591
169	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-02-18 20:00:32.779697	2015-02-18 20:02:17.57841
170	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-02-18 20:00:32.838781	2015-02-18 20:02:18.005115
171	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-02-18 20:00:32.893873	2015-02-18 20:02:18.416071
172	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-02-18 20:00:32.953883	2015-02-18 20:02:18.828367
173	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-02-18 20:00:33.015157	2015-02-18 20:02:19.253999
174	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-02-18 20:00:33.073022	2015-02-18 20:02:19.66756
175	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-02-18 20:00:33.13515	2015-02-18 20:02:20.29325
176	16618717	Retrospective study	Livre et al., 2006, Cancer Res.	2015-02-18 20:00:33.589788	2015-02-18 20:02:20.730213
177	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-02-18 20:00:33.646693	2015-02-18 20:02:21.131696
178	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-02-18 20:00:33.758574	2015-02-18 20:02:22.590548
179	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-02-18 20:00:33.819224	2015-02-18 20:02:23.012798
180	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-02-18 20:00:33.86828	2015-02-18 20:02:23.433428
181	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-02-18 20:00:33.920472	2015-02-18 20:02:23.952205
182	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-02-18 20:00:33.971022	2015-02-18 20:02:24.370625
183	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-02-18 20:00:34.025236	2015-02-18 20:02:24.793638
184	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-02-18 20:00:34.076982	2015-02-18 20:02:25.209561
185	18202412	In vitro study	Livre et al., 2008, J. Clin. Oncol.	2015-02-18 20:00:34.136002	2015-02-18 20:02:25.622932
186	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-02-18 20:00:34.291886	2015-02-18 20:02:26.042729
187	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-02-18 20:00:34.401947	2015-02-18 20:02:26.460944
188	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-02-18 20:00:34.461442	2015-02-18 20:02:26.880376
189	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-02-18 20:00:34.717664	2015-02-18 20:02:27.291901
190	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-18 20:00:34.931972	2015-02-18 20:02:27.714222
191	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-18 20:00:34.982361	2015-02-18 20:02:28.12851
192	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-02-18 20:00:35.141024	2015-02-18 20:02:28.560915
193	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-02-18 20:00:35.206038	2015-02-18 20:02:29.067139
194	19357394		Vardiman et al., 2009, Blood	2015-02-18 20:00:35.359253	2015-02-18 20:02:29.49638
195	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-02-18 20:00:35.404857	2015-02-18 20:02:29.917262
196	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-02-18 20:00:35.455815	2015-02-18 20:02:30.329944
197	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-02-18 20:00:35.509562	2015-02-18 20:02:30.738373
198	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-02-18 20:00:35.725571	2015-02-18 20:02:31.158133
199	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-02-18 20:00:35.788862	2015-02-18 20:02:31.679744
200	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-02-18 20:00:35.843736	2015-02-18 20:02:32.099465
201	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-02-18 20:00:35.957388	2015-02-18 20:02:32.515952
202	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-02-18 20:00:36.060801	2015-02-18 20:02:32.869167
203	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-02-18 20:00:36.154782	2015-02-18 20:02:34.289497
204	15146165		Lasota et al., 2004, Lab. Invest.	2015-02-18 20:00:36.214622	2015-02-18 20:02:34.709547
205	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-02-18 20:00:36.336364	2015-02-18 20:02:35.131722
206	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-02-18 20:00:36.685294	2015-02-18 20:02:35.650169
207	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-02-18 20:00:36.786136	2015-02-18 20:02:36.170463
208	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-02-18 20:00:36.882919	2015-02-18 20:02:36.590634
209	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-02-18 20:00:37.000442	2015-02-18 20:02:37.995166
210	8674046		Yoshida et al., 1996, Cancer Res.	2015-02-18 20:00:37.905152	2015-02-18 20:02:38.468275
211	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-18 20:00:38.007693	2015-02-18 20:02:38.885957
212	23056499		Couto et al., 2012, PLoS ONE	2015-02-18 20:00:38.058216	2015-02-18 20:02:39.306938
213	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-02-18 20:00:38.118067	2015-02-18 20:02:39.831678
214	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-18 20:00:38.177508	2015-02-18 20:02:40.253718
215	22698404		Jackson et al., 2012, Cancer Cell	2015-02-18 20:00:38.281054	2015-02-18 20:02:40.663881
216	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-02-18 20:00:38.343825	2015-02-18 20:02:41.084873
217	9569050		Berns et al., 1998, Br. J. Cancer	2015-02-18 20:00:38.394343	2015-02-18 20:02:41.504465
218	23029227		Qian et al., 2012, PLoS ONE	2015-02-18 20:00:38.970132	2015-02-18 20:02:41.92281
219	23861105		Wu et al., 2013, Am. J. Hematol.	2015-02-18 20:00:39.133003	2015-02-18 20:02:42.440644
220	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-02-18 20:00:39.41044	2015-02-18 20:02:43.171729
221	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-02-18 20:00:39.463427	2015-02-18 20:02:43.691558
222	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-02-18 20:00:39.669552	2015-02-18 20:02:44.138505
223	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-02-18 20:00:39.773343	2015-02-18 20:02:44.558959
224	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-02-18 20:00:40.317136	2015-02-18 20:02:45.04861
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-02-18 20:00:12.474586	2015-02-18 20:00:12.474586
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-18 20:00:12.511597	2015-02-18 20:00:12.511597
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-18 20:00:12.54056	2015-02-18 20:00:12.54056
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-02-18 20:00:12.568381	2015-02-18 20:00:12.568381
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-02-18 20:00:12.616947	2015-02-18 20:00:12.616947
6	1	C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-18 20:00:12.644498	2015-02-18 20:00:12.644498
7	1	L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-18 20:00:12.669755	2015-02-18 20:00:12.669755
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-02-18 20:00:12.70084	2015-02-18 20:00:12.70084
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-02-18 20:00:12.725346	2015-02-18 20:00:12.725346
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-02-18 20:00:12.750098	2015-02-18 20:00:12.750098
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-18 20:00:12.776376	2015-02-18 20:00:12.776376
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-02-18 20:00:12.804353	2015-02-18 20:00:12.804353
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-02-18 20:00:12.832587	2015-02-18 20:00:12.832587
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-02-18 20:00:12.857231	2015-02-18 20:00:12.857231
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-18 20:00:12.883704	2015-02-18 20:00:12.883704
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-18 20:00:12.909681	2015-02-18 20:00:12.909681
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-02-18 20:00:12.935871	2015-02-18 20:00:12.935871
18	6	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-02-18 20:00:12.96217	2015-02-18 20:00:12.96217
19	6	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-02-18 20:00:12.987126	2015-02-18 20:00:12.987126
75	27	EXON 9	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-02-18 20:00:14.647989	2015-02-18 20:00:14.647989
97	34	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-18 20:00:15.303295	2015-02-18 20:00:15.303295
20	6	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-18 20:00:13.01427	2015-02-18 20:00:13.01427
21	7	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-18 20:00:13.042585	2015-02-18 20:00:13.042585
22	7	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-18 20:00:13.072453	2015-02-18 20:00:13.072453
23	8	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-02-18 20:00:13.107393	2015-02-18 20:00:13.107393
24	9	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-02-18 20:00:13.136734	2015-02-18 20:00:13.136734
25	11	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-18 20:00:13.167374	2015-02-18 20:00:13.167374
26	10	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-18 20:00:13.194647	2015-02-18 20:00:13.194647
27	12	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-02-18 20:00:13.227615	2015-02-18 20:00:13.227615
28	13	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-18 20:00:13.264392	2015-02-18 20:00:13.264392
29	13	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-18 20:00:13.290782	2015-02-18 20:00:13.290782
30	31	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-18 20:00:13.33136	2015-02-18 20:00:13.33136
31	14	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-18 20:00:13.358693	2015-02-18 20:00:13.358693
32	16	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-18 20:00:13.385849	2015-02-18 20:00:13.385849
76	27	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-02-18 20:00:14.6741	2015-02-18 20:00:14.6741
33	17	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-02-18 20:00:13.414593	2015-02-18 20:00:13.414593
34	17	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-02-18 20:00:13.443334	2015-02-18 20:00:13.443334
35	18	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.474672	2015-02-18 20:00:13.474672
36	18	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.507066	2015-02-18 20:00:13.507066
37	18	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.534344	2015-02-18 20:00:13.534344
38	18	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.560525	2015-02-18 20:00:13.560525
39	18	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-02-18 20:00:13.590666	2015-02-18 20:00:13.590666
40	18	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.616894	2015-02-18 20:00:13.616894
41	18	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.645189	2015-02-18 20:00:13.645189
42	18	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.673683	2015-02-18 20:00:13.673683
43	18	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.702508	2015-02-18 20:00:13.702508
44	18	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.728157	2015-02-18 20:00:13.728157
45	18	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-18 20:00:13.755488	2015-02-18 20:00:13.755488
46	19	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-18 20:00:13.784353	2015-02-18 20:00:13.784353
47	19	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-18 20:00:13.811218	2015-02-18 20:00:13.811218
48	19	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-18 20:00:13.838273	2015-02-18 20:00:13.838273
49	19	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-18 20:00:13.864917	2015-02-18 20:00:13.864917
50	19	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-18 20:00:13.89386	2015-02-18 20:00:13.89386
51	20	FGFR2-AFF3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:13.921836	2015-02-18 20:00:13.921836
52	20	FGFR2-BICC1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:13.947974	2015-02-18 20:00:13.947974
53	20	FGFR2-CASP7	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:13.981694	2015-02-18 20:00:13.981694
54	20	FGFR2-CCDC6	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:14.013787	2015-02-18 20:00:14.013787
55	20	FGFR2-KIAA1967	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:14.059615	2015-02-18 20:00:14.059615
56	20	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:14.089179	2015-02-18 20:00:14.089179
57	20	FGFR2-OFD1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:14.117846	2015-02-18 20:00:14.117846
58	20	FGFR2-SLC45A3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:14.149376	2015-02-18 20:00:14.149376
59	21	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-18 20:00:14.182866	2015-02-18 20:00:14.182866
60	22	ITD	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-02-18 20:00:14.213679	2015-02-18 20:00:14.213679
61	22	TKD	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-02-18 20:00:14.244492	2015-02-18 20:00:14.244492
62	23	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-02-18 20:00:14.275537	2015-02-18 20:00:14.275537
63	24	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-18 20:00:14.307899	2015-02-18 20:00:14.307899
64	24	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-18 20:00:14.334992	2015-02-18 20:00:14.334992
65	24	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-18 20:00:14.365888	2015-02-18 20:00:14.365888
66	24	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-18 20:00:14.393226	2015-02-18 20:00:14.393226
67	25	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-02-18 20:00:14.421151	2015-02-18 20:00:14.421151
68	25	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-02-18 20:00:14.451847	2015-02-18 20:00:14.451847
69	26	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-02-18 20:00:14.481915	2015-02-18 20:00:14.481915
70	27	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-02-18 20:00:14.510827	2015-02-18 20:00:14.510827
71	27	EXON 11	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-18 20:00:14.53769	2015-02-18 20:00:14.53769
72	27	EXON 13	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-02-18 20:00:14.565276	2015-02-18 20:00:14.565276
73	27	EXON 14	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-02-18 20:00:14.59349	2015-02-18 20:00:14.59349
74	27	EXON 17	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-02-18 20:00:14.622311	2015-02-18 20:00:14.622311
77	27	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-02-18 20:00:14.706492	2015-02-18 20:00:14.706492
78	28	EXON 1	A study by Livre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-02-18 20:00:14.732714	2015-02-18 20:00:14.732714
79	28	EXON 2	In a study by Pan et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-02-18 20:00:14.787867	2015-02-18 20:00:14.787867
80	28	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-18 20:00:14.813647	2015-02-18 20:00:14.813647
81	28	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-18 20:00:14.841487	2015-02-18 20:00:14.841487
82	28	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-18 20:00:14.868408	2015-02-18 20:00:14.868408
83	28	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-18 20:00:14.900031	2015-02-18 20:00:14.900031
84	28	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-18 20:00:14.927036	2015-02-18 20:00:14.927036
85	28	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-18 20:00:14.954406	2015-02-18 20:00:14.954406
86	29	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-02-18 20:00:14.982827	2015-02-18 20:00:14.982827
87	29	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-02-18 20:00:15.011034	2015-02-18 20:00:15.011034
88	30	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-18 20:00:15.040298	2015-02-18 20:00:15.040298
89	32	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-02-18 20:00:15.066703	2015-02-18 20:00:15.066703
90	33	EXON 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-02-18 20:00:15.093563	2015-02-18 20:00:15.093563
91	33	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-02-18 20:00:15.122783	2015-02-18 20:00:15.122783
92	34	EXON 1	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-02-18 20:00:15.158043	2015-02-18 20:00:15.158043
93	34	EXON 2	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-02-18 20:00:15.191709	2015-02-18 20:00:15.191709
94	34	EXON 3	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-18 20:00:15.217731	2015-02-18 20:00:15.217731
95	34	EXON 4	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-18 20:00:15.24721	2015-02-18 20:00:15.24721
96	34	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-18 20:00:15.277824	2015-02-18 20:00:15.277824
98	34	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-18 20:00:15.330582	2015-02-18 20:00:15.330582
99	34	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-18 20:00:15.355058	2015-02-18 20:00:15.355058
100	34	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-18 20:00:15.380706	2015-02-18 20:00:15.380706
101	36	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-18 20:00:15.408303	2015-02-18 20:00:15.408303
102	36	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-18 20:00:15.453185	2015-02-18 20:00:15.453185
103	36	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-18 20:00:15.478753	2015-02-18 20:00:15.478753
104	36	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-18 20:00:15.507577	2015-02-18 20:00:15.507577
105	36	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-18 20:00:15.53732	2015-02-18 20:00:15.53732
106	35	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-18 20:00:15.563384	2015-02-18 20:00:15.563384
107	35	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-18 20:00:15.59591	2015-02-18 20:00:15.59591
108	35	EXON 20	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-18 20:00:15.622483	2015-02-18 20:00:15.622483
109	35	EXON 9	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-18 20:00:15.649577	2015-02-18 20:00:15.649577
110	35	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-18 20:00:15.679541	2015-02-18 20:00:15.679541
111	37	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-02-18 20:00:15.712262	2015-02-18 20:00:15.712262
112	15	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-18 20:00:15.737617	2015-02-18 20:00:15.737617
113	39	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-02-18 20:00:15.769436	2015-02-18 20:00:15.769436
114	38	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-02-18 20:00:15.796049	2015-02-18 20:00:15.796049
115	40	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-02-18 20:00:15.828223	2015-02-18 20:00:15.828223
116	40	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-02-18 20:00:15.85479	2015-02-18 20:00:15.85479
117	42	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-02-18 20:00:15.878956	2015-02-18 20:00:15.878956
118	42	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-02-18 20:00:15.905004	2015-02-18 20:00:15.905004
119	43	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-02-18 20:00:15.929029	2015-02-18 20:00:15.929029
120	43	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-02-18 20:00:15.953312	2015-02-18 20:00:15.953312
121	43	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-02-18 20:00:15.982922	2015-02-18 20:00:15.982922
122	43	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-18 20:00:16.016892	2015-02-18 20:00:16.016892
123	43	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-18 20:00:16.049886	2015-02-18 20:00:16.049886
124	43	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-18 20:00:16.076534	2015-02-18 20:00:16.076534
125	43	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-18 20:00:16.106217	2015-02-18 20:00:16.106217
126	43	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-18 20:00:16.153044	2015-02-18 20:00:16.153044
127	44	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-18 20:00:16.17849	2015-02-18 20:00:16.17849
128	44	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-18 20:00:16.206612	2015-02-18 20:00:16.206612
129	45	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-18 20:00:16.237583	2015-02-18 20:00:16.237583
130	46	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-18 20:00:16.268236	2015-02-18 20:00:16.268236
131	46	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-18 20:00:16.297559	2015-02-18 20:00:16.297559
132	47	EXON 7	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-18 20:00:16.328072	2015-02-18 20:00:16.328072
133	47	EXON 9	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-18 20:00:16.354877	2015-02-18 20:00:16.354877
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating) FROM stdin;
1	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	Sensitivity	Supports	1	1	1	1	41	5	2015-02-18 20:00:16.495675	2015-02-18 20:00:16.495675	4
2	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	1	1	1	1	41	6	2015-02-18 20:00:16.59645	2015-02-18 20:00:16.59645	4
3	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	1	1	1	1	41	7	2015-02-18 20:00:16.659862	2015-02-18 20:00:16.659862	4
4	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	Resistance or Non-Response	Supports	1	2	1	1	42	8	2015-02-18 20:00:16.736435	2015-02-18 20:00:16.736435	3
5	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	2	1	2	2	43	31	2015-02-18 20:00:16.802854	2015-02-18 20:00:16.802854	4
6	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	2	1	2	2	43	112	2015-02-18 20:00:16.878122	2015-02-18 20:00:16.878122	4
7	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	1	2	3	3	44	46	2015-02-18 20:00:16.94669	2015-02-18 20:00:16.94669	5
8	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	Sensitivity	Supports	1	2	4	3	44	46	2015-02-18 20:00:17.000462	2015-02-18 20:00:17.000462	5
9	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	2	2	3	45	46	2015-02-18 20:00:17.062488	2015-02-18 20:00:17.062488	3
10	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	3	3	44	47	2015-02-18 20:00:17.118718	2015-02-18 20:00:17.118718	5
11	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	4	3	44	47	2015-02-18 20:00:17.175568	2015-02-18 20:00:17.175568	5
12	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	2	2	3	45	47	2015-02-18 20:00:17.236949	2015-02-18 20:00:17.236949	3
13	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	3	3	44	48	2015-02-18 20:00:17.301629	2015-02-18 20:00:17.301629	5
14	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	4	3	44	48	2015-02-18 20:00:17.344081	2015-02-18 20:00:17.344081	5
15	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	2	2	3	45	48	2015-02-18 20:00:17.408961	2015-02-18 20:00:17.408961	3
16	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	3	3	44	49	2015-02-18 20:00:17.485604	2015-02-18 20:00:17.485604	5
17	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	4	3	44	49	2015-02-18 20:00:17.532407	2015-02-18 20:00:17.532407	5
18	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	2	2	3	45	49	2015-02-18 20:00:17.596884	2015-02-18 20:00:17.596884	3
19	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	3	3	44	50	2015-02-18 20:00:17.660605	2015-02-18 20:00:17.660605	5
20	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	4	3	44	50	2015-02-18 20:00:17.704413	2015-02-18 20:00:17.704413	5
21	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	Sensitivity	Supports	1	2	2	3	45	50	2015-02-18 20:00:17.762015	2015-02-18 20:00:17.762015	3
22	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	2	5	4	46	76	2015-02-18 20:00:17.831353	2015-02-18 20:00:17.831353	3
23	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	2	6	4	46	76	2015-02-18 20:00:17.881365	2015-02-18 20:00:17.881365	3
24	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	2	7	4	46	76	2015-02-18 20:00:17.92948	2015-02-18 20:00:17.92948	3
25	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	2	8	1	47	76	2015-02-18 20:00:18.000123	2015-02-18 20:00:18.000123	3
26	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	2	5	1	47	76	2015-02-18 20:00:18.048324	2015-02-18 20:00:18.048324	3
27	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	2	6	1	47	76	2015-02-18 20:00:18.110959	2015-02-18 20:00:18.110959	3
28	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	Sensitivity	Supports	1	2	8	4	46	76	2015-02-18 20:00:18.166328	2015-02-18 20:00:18.166328	3
29	In a case study, a patient with anal melanoma harboring an exon 11 KIT mutation showed marked response 4 months after imatinib treatment.	Sensitivity	Supports	1	1	5	4	48	76	2015-02-18 20:00:18.224772	2015-02-18 20:00:18.224772	3
30	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	Resistance or Non-Response	Supports	1	2	5	5	49	77	2015-02-18 20:00:18.280492	2015-02-18 20:00:18.280492	3
31	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	Sensitivity	Supports	1	1	9	5	50	77	2015-02-18 20:00:18.343963	2015-02-18 20:00:18.343963	3
32	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT	Poor Outcome	Supports	3	1	2	6	51	70	2015-02-18 20:00:18.406517	2015-02-18 20:00:18.406517	4
33	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	10	3	52	51	2015-02-18 20:00:18.467304	2015-02-18 20:00:18.467304	3
34	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	10	7	52	52	2015-02-18 20:00:18.532031	2015-02-18 20:00:18.532031	3
35	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	10	3	52	53	2015-02-18 20:00:18.588516	2015-02-18 20:00:18.588516	3
36	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	10	3	52	54	2015-02-18 20:00:18.6602	2015-02-18 20:00:18.6602	3
37	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	10	8	52	55	2015-02-18 20:00:18.723854	2015-02-18 20:00:18.723854	3
38	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in "anti-tumor activitiy".	Sensitivity	Supports	1	1	11	7	53	56	2015-02-18 20:00:18.777828	2015-02-18 20:00:18.777828	3
39	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	10	9	52	57	2015-02-18 20:00:18.829935	2015-02-18 20:00:18.829935	3
40	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	10	10	52	58	2015-02-18 20:00:18.90168	2015-02-18 20:00:18.90168	3
41	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	10	7	53	59	2015-02-18 20:00:18.967233	2015-02-18 20:00:18.967233	3
42	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	11	7	53	59	2015-02-18 20:00:19.018975	2015-02-18 20:00:19.018975	3
43	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratiinib.	Sensitivity	Supports	1	2	12	3	54	35	2015-02-18 20:00:19.078943	2015-02-18 20:00:19.078943	4
44	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratiinib.	Sensitivity	Supports	1	2	12	3	54	36	2015-02-18 20:00:19.134461	2015-02-18 20:00:19.134461	4
45	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratiinib.	Sensitivity	Supports	1	2	12	3	54	38	2015-02-18 20:00:19.190885	2015-02-18 20:00:19.190885	5
46	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratiinib.	Sensitivity	Supports	1	2	12	3	54	41	2015-02-18 20:00:19.269966	2015-02-18 20:00:19.269966	5
47	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratiinib.	Sensitivity	Supports	1	2	12	3	54	44	2015-02-18 20:00:19.331347	2015-02-18 20:00:19.331347	5
48	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratiinib.	Sensitivity	Supports	1	2	12	3	54	45	2015-02-18 20:00:19.387109	2015-02-18 20:00:19.387109	5
49	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratiinib.	Sensitivity	Supports	1	2	12	3	54	43	2015-02-18 20:00:19.443781	2015-02-18 20:00:19.443781	5
50	In patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	Resistance or Non-Response	Supports	1	1	5	11	55	2	2015-02-18 20:00:19.50848	2015-02-18 20:00:19.50848	4
51	COS7 cell lines harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	1	2	5	11	56	2	2015-02-18 20:00:19.563969	2015-02-18 20:00:19.563969	4
52	COS7 cell lines harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	Resistance or Non-Response	Supports	1	2	5	11	56	3	2015-02-18 20:00:19.621837	2015-02-18 20:00:19.621837	4
53	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	1	2	13	12	57	115	2015-02-18 20:00:19.684198	2015-02-18 20:00:19.684198	3
54	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	1	2	13	12	57	116	2015-02-18 20:00:19.738492	2015-02-18 20:00:19.738492	3
55	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	Resistance or Non-Response	Does Not Support	1	2	2	4	58	12	2015-02-18 20:00:19.816857	2015-02-18 20:00:19.816857	3
56	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	Resistance or Non-Response	Supports	1	2	2	4	58	12	2015-02-18 20:00:19.868577	2015-02-18 20:00:19.868577	3
57	NPM1 mutations were not associated with M2 FAB subtypes	Negative	Supports	2	1	2	13	59	90	2015-02-18 20:00:19.923241	2015-02-18 20:00:19.923241	4
58	NPM1 mutations were not associated with M3 FAB subtype (0/55).	Negative	Does Not Support	2	1	2	13	60	90	2015-02-18 20:00:19.980586	2015-02-18 20:00:19.980586	3
59	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes	Positive	Supports	2	1	2	13	59	90	2015-02-18 20:00:20.032947	2015-02-18 20:00:20.032947	4
60	Frequency of NPM1 mutations was not different in normal karyotype AML patients with CEPBA, NRAS or KIT mutations.	Positive	Does Not Support	2	1	2	13	59	90	2015-02-18 20:00:20.082649	2015-02-18 20:00:20.082649	3
61	NPM1 mutations were associated with M5a and M5b FAB subtypes	Positive	Supports	2	1	2	13	60	90	2015-02-18 20:00:20.14034	2015-02-18 20:00:20.14034	3
62	NPM1 mutations were associated with normal karyotype in older (>60) patients	Positive	Supports	2	1	2	13	61	90	2015-02-18 20:00:20.206546	2015-02-18 20:00:20.206546	3
63	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	Positive	Supports	2	1	2	13	62	90	2015-02-18 20:00:20.268001	2015-02-18 20:00:20.268001	2
64	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	Resistance or Non-Response	Supports	1	1	14	13	63	90	2015-02-18 20:00:20.344612	2015-02-18 20:00:20.344612	3
65	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	Resistance or Non-Response	Supports	1	1	2	13	64	90	2015-02-18 20:00:20.396327	2015-02-18 20:00:20.396327	2
158	Inceased copy number of CCND1 is associated with poorer overall survival.	Poor Outcome	Supports	3	1	2	1	118	18	2015-02-18 20:00:25.770242	2015-02-18 20:00:25.770242	3
66	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult AML patients (median age 43)	Sensitivity	Supports	1	1	2	13	62	90	2015-02-18 20:00:20.461278	2015-02-18 20:00:20.461278	4
67	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	1	2	15	13	65	90	2015-02-18 20:00:20.517201	2015-02-18 20:00:20.517201	3
68	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	1	2	14	13	65	90	2015-02-18 20:00:20.5577	2015-02-18 20:00:20.5577	3
69	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	Sensitivity	Supports	1	1	2	13	61	90	2015-02-18 20:00:20.612374	2015-02-18 20:00:20.612374	3
70	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	Sensitivity	Supports	1	1	14	13	61	90	2015-02-18 20:00:20.659487	2015-02-18 20:00:20.659487	2
71	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	Sensitivity	Supports	1	1	16	13	66	90	2015-02-18 20:00:20.71594	2015-02-18 20:00:20.71594	2
72	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	Sensitivity	Supports	1	1	17	13	67	90	2015-02-18 20:00:20.781801	2015-02-18 20:00:20.781801	2
73	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	Sensitivity	Supports	1	3	2	13	68	90	2015-02-18 20:00:20.839334	2015-02-18 20:00:20.839334	2
74	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	Sensitivity	Supports	1	3	2	13	68	90	2015-02-18 20:00:20.90773	2015-02-18 20:00:20.90773	2
75	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1  .	Better Outcome	Supports	3	1	2	13	59	90	2015-02-18 20:00:20.961409	2015-02-18 20:00:20.961409	5
76	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	Better Outcome	Supports	3	1	2	13	69	90	2015-02-18 20:00:21.018502	2015-02-18 20:00:21.018502	4
77	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	Better Outcome	Supports	3	1	2	13	70	90	2015-02-18 20:00:21.071237	2015-02-18 20:00:21.071237	4
78	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	Better Outcome	Supports	3	1	2	13	71	90	2015-02-18 20:00:21.129446	2015-02-18 20:00:21.129446	4
79	Complete remission rates were higher and both disease-free and overall survival was longer for patients with Exon 12 NPM1  .	Better Outcome	Supports	3	1	2	13	60	90	2015-02-18 20:00:21.186696	2015-02-18 20:00:21.186696	3
80	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	3	1	2	13	72	90	2015-02-18 20:00:21.240985	2015-02-18 20:00:21.240985	3
81	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	Better Outcome	Supports	3	1	2	13	73	90	2015-02-18 20:00:21.295783	2015-02-18 20:00:21.295783	3
82	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	Better Outcome	Supports	3	1	2	13	74	90	2015-02-18 20:00:21.354305	2015-02-18 20:00:21.354305	3
83	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	Better Outcome	Supports	3	1	2	13	66	90	2015-02-18 20:00:21.428819	2015-02-18 20:00:21.428819	2
84	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	Better Outcome	Supports	3	1	2	13	75	90	2015-02-18 20:00:21.481496	2015-02-18 20:00:21.481496	3
85	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	3	1	2	13	76	90	2015-02-18 20:00:21.528871	2015-02-18 20:00:21.528871	4
86	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N/A	Does Not Support	3	1	2	13	77	90	2015-02-18 20:00:21.581117	2015-02-18 20:00:21.581117	3
87	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	Poor Outcome	Supports	3	1	2	13	60	90	2015-02-18 20:00:21.631704	2015-02-18 20:00:21.631704	4
88	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	3	1	2	13	78	90	2015-02-18 20:00:21.685768	2015-02-18 20:00:21.685768	4
89	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	3	1	2	13	78	90	2015-02-18 20:00:21.735155	2015-02-18 20:00:21.735155	4
90	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	Poor Outcome	Supports	3	1	2	13	79	90	2015-02-18 20:00:21.795941	2015-02-18 20:00:21.795941	3
91	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	Poor Outcome	Supports	3	1	2	13	71	90	2015-02-18 20:00:21.856189	2015-02-18 20:00:21.856189	3
92	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	Negative	Does Not Support	2	1	2	13	80	91	2015-02-18 20:00:21.935413	2015-02-18 20:00:21.935413	4
93	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	2	1	2	13	80	91	2015-02-18 20:00:21.990777	2015-02-18 20:00:21.990777	4
94	Mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	Sensitivity	Supports	1	3	2	13	81	91	2015-02-18 20:00:22.050231	2015-02-18 20:00:22.050231	3
95	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	18	4	82	11	2015-02-18 20:00:22.108396	2015-02-18 20:00:22.108396	5
96	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	18	4	83	13	2015-02-18 20:00:22.170356	2015-02-18 20:00:22.170356	3
97	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	18	4	82	15	2015-02-18 20:00:22.228863	2015-02-18 20:00:22.228863	5
98	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	18	4	82	16	2015-02-18 20:00:22.285867	2015-02-18 20:00:22.285867	5
99	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	Sensitivity	Supports	1	2	2	14	84	113	2015-02-18 20:00:22.354384	2015-02-18 20:00:22.354384	4
100	PTEN nonsense  , including R233*, has been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	Poor Outcome	Does Not Support	3	1	2	15	85	113	2015-02-18 20:00:22.432049	2015-02-18 20:00:22.432049	3
101	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	Sensitivity	Supports	1	1	19	16	86	127	2015-02-18 20:00:22.490652	2015-02-18 20:00:22.490652	3
102	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	Sensitivity	Supports	1	2	20	1	87	127	2015-02-18 20:00:22.554396	2015-02-18 20:00:22.554396	3
103	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	1	1	19	17	88	128	2015-02-18 20:00:22.614977	2015-02-18 20:00:22.614977	3
104	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	1	1	19	17	86	129	2015-02-18 20:00:22.681345	2015-02-18 20:00:22.681345	3
105	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	Positive	Supports	2	4	2	11	55	1	2015-02-18 20:00:22.745097	2015-02-18 20:00:22.745097	4
106	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	Positive	Does Not Support	2	2	2	11	89	1	2015-02-18 20:00:22.796848	2015-02-18 20:00:22.796848	2
107	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	Resistance or Non-Response	Supports	1	1	5	13	90	1	2015-02-18 20:00:22.846681	2015-02-18 20:00:22.846681	3
108	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	Sensitivity	Supports	1	4	21	13	90	1	2015-02-18 20:00:22.898907	2015-02-18 20:00:22.898907	5
109	The clinical use of imatinib in patients with BCR-ABL fusion has drastic sensitivity to the drug.	Sensitivity	Supports	1	4	5	11	55	1	2015-02-18 20:00:22.966442	2015-02-18 20:00:22.966442	5
110	The use of second-generation BCR-ABL targeted therapies has shown signifcantly greater response in patients resistant to imatinib.	Sensitivity	Supports	1	1	6	11	55	1	2015-02-18 20:00:23.019855	2015-02-18 20:00:23.019855	4
111	The use of second-generation BCR-ABL targeted therapies has shown signifcantly greater response in patients resistant to imatinib.	Sensitivity	Supports	1	1	8	11	55	1	2015-02-18 20:00:23.065051	2015-02-18 20:00:23.065051	4
112	BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens."	Better Outcome	Supports	3	4	5	11	55	1	2015-02-18 20:00:23.113951	2015-02-18 20:00:23.113951	5
113	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	Sensitivity	Does Not Support	1	1	22	18	91	4	2015-02-18 20:00:23.170989	2015-02-18 20:00:23.170989	3
114	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	Sensitivity	Supports	1	2	23	1	92	7	2015-02-18 20:00:23.232812	2015-02-18 20:00:23.232812	3
115	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	Resistance or Non-Response	Supports	1	2	1	19	42	8	2015-02-18 20:00:23.281217	2015-02-18 20:00:23.281217	3
116	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	Resistance or Non-Response	Supports	1	2	1	20	93	8	2015-02-18 20:00:23.345274	2015-02-18 20:00:23.345274	3
117	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	Sensitivity	Supports	1	2	24	20	94	8	2015-02-18 20:00:23.416691	2015-02-18 20:00:23.416691	4
118	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	Sensitivity	Supports	1	2	23	20	92	8	2015-02-18 20:00:23.468786	2015-02-18 20:00:23.468786	3
119	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	Sensitivity	Supports	1	2	1	20	95	8	2015-02-18 20:00:23.526995	2015-02-18 20:00:23.526995	3
120	TAE684 inhibits growth of Ba/F3 cells expressing ALK with the R1275Q mutation at a higher IC50 than those expressing F1174L and the SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684	NA	Does Not Support	1	2	24	20	94	9	2015-02-18 20:00:23.574735	2015-02-18 20:00:23.574735	3
121	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells in vitro	Sensitivity	Supports	1	2	25	1	96	10	2015-02-18 20:00:23.632997	2015-02-18 20:00:23.632997	3
122	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells in vitro	Sensitivity	Supports	1	2	7	1	96	10	2015-02-18 20:00:23.688102	2015-02-18 20:00:23.688102	3
123	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	Sensitivity	Supports	1	1	7	1	96	10	2015-02-18 20:00:23.742706	2015-02-18 20:00:23.742706	2
124	BRAF mutations are associated with melanoma arising in nonchronic sun damage skin and with superficial spreading melanoma	Positive	Supports	2	1	2	4	97	17	2015-02-18 20:00:23.798626	2015-02-18 20:00:23.798626	4
125	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	Resistance or Non-Response	Supports	1	1	26	21	98	17	2015-02-18 20:00:23.854688	2015-02-18 20:00:23.854688	3
126	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	1	1	18	4	99	17	2015-02-18 20:00:23.99728	2015-02-18 20:00:23.99728	3
127	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	1	1	25	4	99	17	2015-02-18 20:00:24.043757	2015-02-18 20:00:24.043757	3
128	thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.	Positive	Supports	2	1	2	9	100	12	2015-02-18 20:00:24.097128	2015-02-18 20:00:24.097128	5
129	thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.	Positive	Supports	2	1	2	9	101	12	2015-02-18 20:00:24.152025	2015-02-18 20:00:24.152025	5
130	BRAF V600E is shown to be associated with the tall-cell variant of PTC	Positive	Supports	2	1	2	9	102	12	2015-02-18 20:00:24.209515	2015-02-18 20:00:24.209515	3
131	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N/A	Does Not Support	1	1	27	4	103	12	2015-02-18 20:00:24.261652	2015-02-18 20:00:24.261652	2
132	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N/A	Does Not Support	1	1	28	4	103	12	2015-02-18 20:00:24.302825	2015-02-18 20:00:24.302825	2
133	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	2	29	22	104	12	2015-02-18 20:00:24.368332	2015-02-18 20:00:24.368332	3
134	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	2	26	22	104	12	2015-02-18 20:00:24.410912	2015-02-18 20:00:24.410912	3
135	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	2	7	22	104	12	2015-02-18 20:00:24.454094	2015-02-18 20:00:24.454094	3
136	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	Resistance or Non-Response	Supports	1	1	29	21	105	12	2015-02-18 20:00:24.526468	2015-02-18 20:00:24.526468	3
137	BRAF V600E resistance is triggered by ERK dependent or independent pathways.	Resistance or Non-Response	Supports	1	2	2	4	106	12	2015-02-18 20:00:24.583702	2015-02-18 20:00:24.583702	3
138	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	1	1	18	1	107	12	2015-02-18 20:00:24.634921	2015-02-18 20:00:24.634921	2
139	dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	1	1	18	4	108	12	2015-02-18 20:00:24.690381	2015-02-18 20:00:24.690381	5
140	dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	1	1	25	4	108	12	2015-02-18 20:00:24.740081	2015-02-18 20:00:24.740081	5
141	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	1	2	30	22	109	12	2015-02-18 20:00:24.789098	2015-02-18 20:00:24.789098	3
142	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	1	2	31	22	109	12	2015-02-18 20:00:24.837099	2015-02-18 20:00:24.837099	3
143	treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g. dabrafenib) appears to be beneficial, and better than vice versa.	Sensitivity	Supports	1	1	32	4	110	12	2015-02-18 20:00:24.898752	2015-02-18 20:00:24.898752	2
144	treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g. dabrafenib) appears to be beneficial, and better than vice versa.	Sensitivity	Supports	1	1	18	4	110	12	2015-02-18 20:00:24.946114	2015-02-18 20:00:24.946114	2
145	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	2	33	21	111	12	2015-02-18 20:00:25.006931	2015-02-18 20:00:25.006931	2
146	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	2	34	21	111	12	2015-02-18 20:00:25.056547	2015-02-18 20:00:25.056547	2
147	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	2	35	21	111	12	2015-02-18 20:00:25.126823	2015-02-18 20:00:25.126823	2
148	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	1	2	31	22	112	12	2015-02-18 20:00:25.181821	2015-02-18 20:00:25.181821	2
149	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	1	2	36	22	112	12	2015-02-18 20:00:25.231644	2015-02-18 20:00:25.231644	2
150	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	Sensitivity	Supports	1	2	34	21	111	12	2015-02-18 20:00:25.285176	2015-02-18 20:00:25.285176	2
151	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as poor prognosis.	Poor Outcome	Supports	3	4	2	21	113	12	2015-02-18 20:00:25.339277	2015-02-18 20:00:25.339277	5
152	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	Poor Outcome	Does Not Support	3	1	2	9	114	12	2015-02-18 20:00:25.405532	2015-02-18 20:00:25.405532	5
153	BRAF V600E is correlated with shorter Disease-Free and Overall Survival in a Spanish cohort of Melanoma patients.	Poor Outcome	Supports	3	1	2	4	115	12	2015-02-18 20:00:25.460704	2015-02-18 20:00:25.460704	3
154	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	Poor Outcome	Supports	3	1	2	9	102	12	2015-02-18 20:00:25.516642	2015-02-18 20:00:25.516642	3
155	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	Poor Outcome	Supports	3	1	2	9	116	12	2015-02-18 20:00:25.579954	2015-02-18 20:00:25.579954	3
156	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	Poor Outcome	Supports	3	1	2	9	102	12	2015-02-18 20:00:25.632427	2015-02-18 20:00:25.632427	3
157	COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	Resistance or Non-Response	Supports	1	3	37	21	117	14	2015-02-18 20:00:25.714373	2015-02-18 20:00:25.714373	4
159	Three studies have associated CCND1 expression with poorer survival.	Poor Outcome	Supports	3	1	2	1	118	19	2015-02-18 20:00:25.820804	2015-02-18 20:00:25.820804	3
160	Three studies have found no significant survival impact for CCND1 in lung cancer.	Poor Outcome	Supports	3	1	2	1	118	19	2015-02-18 20:00:25.876846	2015-02-18 20:00:25.876846	3
161	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	Positive	Supports	2	1	2	3	119	20	2015-02-18 20:00:25.93469	2015-02-18 20:00:25.93469	4
162	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	Poor Outcome	Supports	3	1	2	3	119	20	2015-02-18 20:00:25.988653	2015-02-18 20:00:25.988653	4
163	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	Poor Outcome	Supports	3	1	2	23	120	20	2015-02-18 20:00:26.045334	2015-02-18 20:00:26.045334	4
164	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	Poor Outcome	Supports	3	1	2	24	121	20	2015-02-18 20:00:26.106272	2015-02-18 20:00:26.106272	4
165	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	Poor Outcome	Supports	3	2	2	25	122	21	2015-02-18 20:00:26.167572	2015-02-18 20:00:26.167572	4
166	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	Poor Outcome	Supports	3	2	2	25	122	21	2015-02-18 20:00:26.253697	2015-02-18 20:00:26.253697	4
167	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	Poor Outcome	Supports	3	2	2	25	122	21	2015-02-18 20:00:26.305459	2015-02-18 20:00:26.305459	4
168	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	Poor Outcome	Supports	3	2	2	25	122	21	2015-02-18 20:00:26.362214	2015-02-18 20:00:26.362214	4
169	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	Negative	Supports	2	2	2	25	123	22	2015-02-18 20:00:26.413541	2015-02-18 20:00:26.413541	3
170	Treatment of CCND2-negative cells with 5-Aza-2'-deoxycytidine restored cyclin D2 expression.	Sensitivity	Supports	1	2	2	25	123	22	2015-02-18 20:00:26.462425	2015-02-18 20:00:26.462425	2
171	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	Sensitivity	Supports	1	2	38	26	124	23	2015-02-18 20:00:26.519166	2015-02-18 20:00:26.519166	3
172	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	Poor Outcome	Supports	3	1	2	3	125	24	2015-02-18 20:00:26.574735	2015-02-18 20:00:26.574735	4
173	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	Poor Outcome	Supports	3	1	2	27	126	24	2015-02-18 20:00:26.632555	2015-02-18 20:00:26.632555	4
174	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	3	1	2	27	126	24	2015-02-18 20:00:26.691257	2015-02-18 20:00:26.691257	4
175	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	3	1	2	28	126	24	2015-02-18 20:00:26.745367	2015-02-18 20:00:26.745367	4
176	In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	1	1	39	29	127	25	2015-02-18 20:00:26.821586	2015-02-18 20:00:26.821586	4
177	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	3	1	40	29	128	25	2015-02-18 20:00:26.877087	2015-02-18 20:00:26.877087	3
178	In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	1	1	39	29	127	26	2015-02-18 20:00:26.933495	2015-02-18 20:00:26.933495	4
179	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	3	1	40	29	128	26	2015-02-18 20:00:26.983721	2015-02-18 20:00:26.983721	3
180	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	Poor Outcome	Supports	3	1	2	1	129	27	2015-02-18 20:00:27.033335	2015-02-18 20:00:27.033335	3
181	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	Poor Outcome	Supports	3	1	2	1	129	27	2015-02-18 20:00:27.089375	2015-02-18 20:00:27.089375	3
182	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	N/A	Does Not Support	1	1	2	13	130	29	2015-02-18 20:00:27.147031	2015-02-18 20:00:27.147031	4
183	CEBPA mutation was significantly associated with complete remission	Better Outcome	Supports	3	1	2	13	75	29	2015-02-18 20:00:27.204211	2015-02-18 20:00:27.204211	4
184	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	Better Outcome	Supports	3	1	2	13	130	29	2015-02-18 20:00:27.259739	2015-02-18 20:00:27.259739	4
185	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	Better Outcome	Supports	3	1	2	13	131	28	2015-02-18 20:00:27.340068	2015-02-18 20:00:27.340068	3
186	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	Better Outcome	Supports	3	1	2	13	131	28	2015-02-18 20:00:27.387761	2015-02-18 20:00:27.387761	3
187	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	N/A	Does Not Support	3	1	2	13	131	28	2015-02-18 20:00:27.440135	2015-02-18 20:00:27.440135	2
188	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	2	5	30	132	30	2015-02-18 20:00:27.494705	2015-02-18 20:00:27.494705	3
189	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	2	41	30	132	30	2015-02-18 20:00:27.541106	2015-02-18 20:00:27.541106	3
190	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	Negative	Supports	2	1	2	13	133	32	2015-02-18 20:00:27.591712	2015-02-18 20:00:27.591712	3
191	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	Positive	Supports	2	1	2	13	80	32	2015-02-18 20:00:27.636231	2015-02-18 20:00:27.636231	5
192	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A	Positive	Supports	2	1	2	13	76	32	2015-02-18 20:00:27.689629	2015-02-18 20:00:27.689629	3
193	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	2	1	2	13	133	32	2015-02-18 20:00:27.745458	2015-02-18 20:00:27.745458	3
194	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	2	1	2	13	133	32	2015-02-18 20:00:27.797488	2015-02-18 20:00:27.797488	3
195	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	Positive	Supports	2	1	2	13	134	32	2015-02-18 20:00:27.871315	2015-02-18 20:00:27.871315	3
196	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	Positive	Supports	2	1	2	13	135	32	2015-02-18 20:00:27.926843	2015-02-18 20:00:27.926843	3
197	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	NA	Does Not Support	1	1	16	13	134	32	2015-02-18 20:00:27.972849	2015-02-18 20:00:27.972849	4
198	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Sensitivity	Supports	1	1	42	13	134	32	2015-02-18 20:00:28.023184	2015-02-18 20:00:28.023184	4
199	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	Better Outcome	Supports	3	1	2	13	133	32	2015-02-18 20:00:28.074817	2015-02-18 20:00:28.074817	3
200	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	N/A	Does Not Support	3	1	2	13	136	32	2015-02-18 20:00:28.130471	2015-02-18 20:00:28.130471	4
201	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	3	1	2	13	76	32	2015-02-18 20:00:28.185011	2015-02-18 20:00:28.185011	4
202	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	N/A	Does Not Support	3	1	2	13	134	32	2015-02-18 20:00:28.243433	2015-02-18 20:00:28.243433	4
203	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	N/A	Does Not Support	3	1	2	13	133	32	2015-02-18 20:00:28.297238	2015-02-18 20:00:28.297238	3
204	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	N/A	Does Not Support	3	1	2	13	133	32	2015-02-18 20:00:28.376605	2015-02-18 20:00:28.376605	3
205	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	3	1	2	13	80	32	2015-02-18 20:00:28.431677	2015-02-18 20:00:28.431677	5
206	In AML patients with FLT3-ITD  , concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	3	1	2	13	80	32	2015-02-18 20:00:28.486315	2015-02-18 20:00:28.486315	5
207	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	Poor Outcome	Supports	3	1	2	13	135	32	2015-02-18 20:00:28.540808	2015-02-18 20:00:28.540808	5
208	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	Poor Outcome	Supports	3	1	2	13	136	32	2015-02-18 20:00:28.593205	2015-02-18 20:00:28.593205	4
209	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	Poor Outcome	Supports	3	1	2	13	76	32	2015-02-18 20:00:28.649648	2015-02-18 20:00:28.649648	4
210	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	3	1	2	13	80	32	2015-02-18 20:00:28.709092	2015-02-18 20:00:28.709092	4
211	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	3	1	2	13	136	32	2015-02-18 20:00:28.755079	2015-02-18 20:00:28.755079	4
212	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	3	1	2	13	136	32	2015-02-18 20:00:28.805352	2015-02-18 20:00:28.805352	4
213	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable cytogenetically normal AMLs	Poor Outcome	Supports	3	1	2	13	133	32	2015-02-18 20:00:28.881969	2015-02-18 20:00:28.881969	3
214	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	3	1	2	13	76	32	2015-02-18 20:00:28.931867	2015-02-18 20:00:28.931867	3
215	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	Poor Outcome	Supports	3	1	2	13	76	32	2015-02-18 20:00:28.98359	2015-02-18 20:00:28.98359	3
216	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	3	1	2	13	76	32	2015-02-18 20:00:29.038323	2015-02-18 20:00:29.038323	3
217	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	Poor Outcome	Supports	3	4	2	13	133	32	2015-02-18 20:00:29.093002	2015-02-18 20:00:29.093002	3
218	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.	N/A	Does Not Support	1	1	43	1	137	33	2015-02-18 20:00:29.144845	2015-02-18 20:00:29.144845	3
219	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.	N/A	Does Not Support	1	1	44	1	137	33	2015-02-18 20:00:29.188243	2015-02-18 20:00:29.188243	3
220	In NSCLC patients treated with tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	Sensitivity	Supports	1	1	2	1	138	33	2015-02-18 20:00:29.240567	2015-02-18 20:00:29.240567	4
221	Geftinib added to pemetrexed and carboplatin showed greater effectiveness than premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.	Sensitivity	Supports	1	1	43	1	139	33	2015-02-18 20:00:29.292082	2015-02-18 20:00:29.292082	3
222	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	Better Outcome	Supports	3	1	2	1	140	33	2015-02-18 20:00:29.377499	2015-02-18 20:00:29.377499	3
223	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	Resistance or Non-Response	Supports	1	1	43	1	141	34	2015-02-18 20:00:29.429204	2015-02-18 20:00:29.429204	3
224	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective.	Sensitivity	Supports	1	1	44	1	142	34	2015-02-18 20:00:29.48524	2015-02-18 20:00:29.48524	3
225	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective.	Sensitivity	Supports	1	1	45	1	142	34	2015-02-18 20:00:29.52951	2015-02-18 20:00:29.52951	3
226	In NSCLC containing T790M mutation, staurosporine may be inhibitive of EGFR, especially when L858R mutation is also present.	Sensitivity	Supports	1	3	46	1	143	34	2015-02-18 20:00:29.585799	2015-02-18 20:00:29.585799	1
227	Patients with NSCLC harboring EGFR T790M mutation has statistically worse overall survival compared to patients with L858R or other exon 19 activating mutation.	Poor Outcome	Supports	3	1	2	1	144	34	2015-02-18 20:00:29.642056	2015-02-18 20:00:29.642056	4
228	In NSCLC patients with T790M and another activating mutation, their progression free survival is shorter compared to those who do not possess T790M mutation.	Poor Outcome	Supports	3	1	2	1	145	34	2015-02-18 20:00:29.698067	2015-02-18 20:00:29.698067	4
229	In MCF10A cell lines, the in-frame deletion of amino acids 755-759 was shown to be sensitive to neratiinib.	Sensitivity	Supports	1	2	12	3	54	37	2015-02-18 20:00:29.745087	2015-02-18 20:00:29.745087	5
230	The L755S mutation showed mild response to neratinib in MCF10A cell lines.	Sensitivity	Supports	1	2	12	3	54	39	2015-02-18 20:00:29.815535	2015-02-18 20:00:29.815535	5
231	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines.	Resistance or Non-Response	Supports	1	2	47	3	54	39	2015-02-18 20:00:29.867012	2015-02-18 20:00:29.867012	5
232	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	12	3	54	40	2015-02-18 20:00:29.923711	2015-02-18 20:00:29.923711	5
233	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	12	3	54	42	2015-02-18 20:00:29.973903	2015-02-18 20:00:29.973903	5
234	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	Positive	Supports	2	1	2	13	146	60	2015-02-18 20:00:30.025018	2015-02-18 20:00:30.025018	3
235	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	Resistance or Non-Response	Supports	1	1	2	13	62	60	2015-02-18 20:00:30.075805	2015-02-18 20:00:30.075805	4
236	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	1	2	48	13	147	60	2015-02-18 20:00:30.132924	2015-02-18 20:00:30.132924	3
237	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	Resistance or Non-Response	Supports	1	1	14	13	63	60	2015-02-18 20:00:30.184881	2015-02-18 20:00:30.184881	3
238	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	1	2	16	13	147	60	2015-02-18 20:00:30.237391	2015-02-18 20:00:30.237391	3
239	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	Resistance or Non-Response	Supports	1	1	2	13	62	60	2015-02-18 20:00:30.383531	2015-02-18 20:00:30.383531	3
240	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	Sensitivity	Supports	1	2	7	13	148	60	2015-02-18 20:00:30.443199	2015-02-18 20:00:30.443199	4
241	CEP701 is effective in AML patients with FLT3 mutation	Sensitivity	Supports	1	1	49	13	149	60	2015-02-18 20:00:30.500219	2015-02-18 20:00:30.500219	4
242	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	Sensitivity	Supports	1	2	50	13	150	60	2015-02-18 20:00:30.554638	2015-02-18 20:00:30.554638	4
243	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	3	1	2	13	76	60	2015-02-18 20:00:30.602907	2015-02-18 20:00:30.602907	4
244	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	N/A	Does Not Support	3	1	2	13	78	60	2015-02-18 20:00:30.650719	2015-02-18 20:00:30.650719	4
245	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	N/A	Does Not Support	3	1	2	13	62	60	2015-02-18 20:00:30.710659	2015-02-18 20:00:30.710659	2
246	In AML patients with FLT3-ITD  , concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	3	1	2	13	80	60	2015-02-18 20:00:30.761672	2015-02-18 20:00:30.761672	5
247	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	Poor Outcome	Supports	3	1	2	13	151	60	2015-02-18 20:00:30.811709	2015-02-18 20:00:30.811709	4
248	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	3	1	2	13	78	60	2015-02-18 20:00:30.855438	2015-02-18 20:00:30.855438	4
249	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	3	1	2	13	78	60	2015-02-18 20:00:30.910078	2015-02-18 20:00:30.910078	4
250	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	Poor Outcome	Supports	3	1	2	13	78	60	2015-02-18 20:00:30.959183	2015-02-18 20:00:30.959183	4
251	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	Poor Outcome	Supports	3	1	2	13	146	60	2015-02-18 20:00:31.005314	2015-02-18 20:00:31.005314	3
252	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	Poor Outcome	Supports	3	4	2	13	152	60	2015-02-18 20:00:31.050022	2015-02-18 20:00:31.050022	2
253	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	Positive	Does Not Support	2	1	2	13	146	61	2015-02-18 20:00:31.095375	2015-02-18 20:00:31.095375	3
254	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	Resistance or Non-Response	Supports	1	1	7	13	153	61	2015-02-18 20:00:31.143372	2015-02-18 20:00:31.143372	4
255	Sorafenib is effective in patients with FLT3 internal tandem repeat , but not in wild type or D835 mutation.	Resistance or Non-Response	Supports	1	2	7	13	148	61	2015-02-18 20:00:31.192806	2015-02-18 20:00:31.192806	4
256	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	1	2	48	13	147	61	2015-02-18 20:00:31.245022	2015-02-18 20:00:31.245022	3
257	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	1	2	16	13	147	61	2015-02-18 20:00:31.289377	2015-02-18 20:00:31.289377	3
258	CEP701 is effective in AML patients with FLT3 mutation	Sensitivity	Supports	1	1	49	13	149	61	2015-02-18 20:00:31.33881	2015-02-18 20:00:31.33881	4
259	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	3	1	2	13	72	61	2015-02-18 20:00:31.390633	2015-02-18 20:00:31.390633	3
260	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	N/A	Does Not Support	3	1	2	13	146	61	2015-02-18 20:00:31.435973	2015-02-18 20:00:31.435973	3
261	In AML patients, FLT3-TKD mutation is associated with poorer disease free survival compared to patients with wild type FLT3	Poor Outcome	Supports	3	1	2	13	154	61	2015-02-18 20:00:31.495356	2015-02-18 20:00:31.495356	4
262	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	1	1	51	31	155	62	2015-02-18 20:00:31.552763	2015-02-18 20:00:31.552763	4
263	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	1	1	52	31	155	62	2015-02-18 20:00:31.603079	2015-02-18 20:00:31.603079	4
264	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	Positive	Supports	2	1	2	13	156	63	2015-02-18 20:00:31.654228	2015-02-18 20:00:31.654228	2
265	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	Better Outcome	Supports	3	1	2	32	157	63	2015-02-18 20:00:31.706634	2015-02-18 20:00:31.706634	3
266	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	Better Outcome	Supports	3	1	2	33	158	63	2015-02-18 20:00:31.752644	2015-02-18 20:00:31.752644	3
267	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	N/A	Does Not Support	3	1	2	13	159	63	2015-02-18 20:00:31.806682	2015-02-18 20:00:31.806682	2
268	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	Poor Outcome	Supports	3	1	2	34	160	63	2015-02-18 20:00:31.859402	2015-02-18 20:00:31.859402	4
269	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	Positive	Supports	2	1	2	13	161	64	2015-02-18 20:00:31.913273	2015-02-18 20:00:31.913273	3
270	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	3	1	2	13	161	64	2015-02-18 20:00:31.985509	2015-02-18 20:00:31.985509	2
271	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	3	1	2	13	161	65	2015-02-18 20:00:32.034001	2015-02-18 20:00:32.034001	3
272	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	3	1	2	13	161	66	2015-02-18 20:00:32.092102	2015-02-18 20:00:32.092102	3
273	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	Better Outcome	Supports	3	1	2	13	162	67	2015-02-18 20:00:32.138061	2015-02-18 20:00:32.138061	3
274	In patients with IDH2 R140Q/L mutation, the presence of mutation do not impact overall survival or disease free survival.	N/A	Does Not Support	3	1	2	13	163	67	2015-02-18 20:00:32.191757	2015-02-18 20:00:32.191757	3
275	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	N/A	Does Not Support	3	1	2	35	164	67	2015-02-18 20:00:32.241663	2015-02-18 20:00:32.241663	3
276	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	N/A	Does Not Support	3	3	2	13	165	67	2015-02-18 20:00:32.297784	2015-02-18 20:00:32.297784	2
277	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	N/A	Does Not Support	3	3	2	35	166	67	2015-02-18 20:00:32.350379	2015-02-18 20:00:32.350379	2
278	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	N/A	Does Not Support	3	3	2	13	165	68	2015-02-18 20:00:32.407616	2015-02-18 20:00:32.407616	2
279	In patients suffering from MDS, the presence of IDH2 mutation such as R172K do not confer prognostic value (overall survival)	N/A	Does Not Support	3	3	2	35	166	68	2015-02-18 20:00:32.464774	2015-02-18 20:00:32.464774	2
280	In AML, patients with IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	Poor Outcome	Supports	3	1	2	13	162	68	2015-02-18 20:00:32.517991	2015-02-18 20:00:32.517991	3
281	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	Negative	Supports	2	1	2	36	167	69	2015-02-18 20:00:32.574986	2015-02-18 20:00:32.574986	4
282	JAK2 V617F not associatted wth  lymphoid leukemia.	Positive	Does Not Support	2	1	2	11	167	69	2015-02-18 20:00:32.621764	2015-02-18 20:00:32.621764	4
283	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	Positive	Supports	2	1	2	11	167	69	2015-02-18 20:00:32.667348	2015-02-18 20:00:32.667348	4
284	rs10974944 SNP in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer	Positive	Supports	2	1	2	37	168	69	2015-02-18 20:00:32.729259	2015-02-18 20:00:32.729259	4
285	In patients with JAK2 V617F, the use of peglated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	Sensitivity	Supports	1	1	53	38	169	69	2015-02-18 20:00:32.784783	2015-02-18 20:00:32.784783	4
286	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells	Sensitivity	Supports	1	2	54	39	170	69	2015-02-18 20:00:32.842568	2015-02-18 20:00:32.842568	3
287	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	2	1	2	5	171	71	2015-02-18 20:00:32.898306	2015-02-18 20:00:32.898306	2
288	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	1	5	4	172	71	2015-02-18 20:00:32.956298	2015-02-18 20:00:32.956298	3
289	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	N/A	Does Not Support	3	1	2	5	173	71	2015-02-18 20:00:33.018039	2015-02-18 20:00:33.018039	3
290	There is no significant association between KIT mutations in exon 11, exon 9 or wild type in survival among GIST patients.	N/A	Does Not Support	3	1	2	5	174	71	2015-02-18 20:00:33.075579	2015-02-18 20:00:33.075579	3
291	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N/A	Does Not Support	3	1	2	28	175	71	2015-02-18 20:00:33.13912	2015-02-18 20:00:33.13912	2
292	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	Poor Outcome	Supports	3	1	2	5	171	71	2015-02-18 20:00:33.195749	2015-02-18 20:00:33.195749	3
293	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	1	5	4	172	72	2015-02-18 20:00:33.248697	2015-02-18 20:00:33.248697	3
294	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	Poor Outcome	Supports	3	1	2	5	174	73	2015-02-18 20:00:33.294414	2015-02-18 20:00:33.294414	3
295	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	2	1	2	5	171	74	2015-02-18 20:00:33.346282	2015-02-18 20:00:33.346282	3
296	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	3	1	5	4	172	74	2015-02-18 20:00:33.405714	2015-02-18 20:00:33.405714	3
297	There is no significant association between KIT mutations in exon 11, exon 9 or wild type in survival among GIST patients.	N/A	Does Not Support	3	1	2	5	174	75	2015-02-18 20:00:33.458711	2015-02-18 20:00:33.458711	3
298	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N/A	Does Not Support	3	1	2	28	175	75	2015-02-18 20:00:33.506872	2015-02-18 20:00:33.506872	2
299	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	Resistance or Non-Response	Supports	1	1	29	21	176	78	2015-02-18 20:00:33.592096	2015-02-18 20:00:33.592096	4
300	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	1	1	43	1	177	79	2015-02-18 20:00:33.649423	2015-02-18 20:00:33.649423	3
301	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	1	1	44	1	177	79	2015-02-18 20:00:33.695077	2015-02-18 20:00:33.695077	3
302	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	Sensitivity	Does Not Support	1	1	55	21	178	79	2015-02-18 20:00:33.76083	2015-02-18 20:00:33.76083	4
303	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	2	1	2	40	179	80	2015-02-18 20:00:33.82333	2015-02-18 20:00:33.82333	3
304	KRAS mutations in lung patients are not associated with smoking history, age and gender	Positive	Does Not Support	2	1	2	1	180	80	2015-02-18 20:00:33.870577	2015-02-18 20:00:33.870577	3
305	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	N/A	Does Not Support	3	1	2	22	181	80	2015-02-18 20:00:33.922829	2015-02-18 20:00:33.922829	4
306	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	N/A	Does Not Support	3	1	2	1	182	80	2015-02-18 20:00:33.973404	2015-02-18 20:00:33.973404	3
307	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	3	1	2	41	183	80	2015-02-18 20:00:34.029034	2015-02-18 20:00:34.029034	3
308	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	Poor Outcome	Supports	3	1	2	1	184	80	2015-02-18 20:00:34.079272	2015-02-18 20:00:34.079272	3
309	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	Resistance or Non-Response	Supports	1	2	29	21	185	81	2015-02-18 20:00:34.139246	2015-02-18 20:00:34.139246	3
310	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	Resistance or Non-Response	Supports	1	1	29	21	105	81	2015-02-18 20:00:34.189308	2015-02-18 20:00:34.189308	3
311	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	Poor Outcome	Supports	3	2	2	21	185	81	2015-02-18 20:00:34.244374	2015-02-18 20:00:34.244374	3
312	KRAS G12C occur more frequently in women than men	Positive	Supports	2	1	2	28	186	82	2015-02-18 20:00:34.294789	2015-02-18 20:00:34.294789	2
313	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	Positive	Supports	2	1	2	28	186	83	2015-02-18 20:00:34.348454	2015-02-18 20:00:34.348454	3
314	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	1	1	18	1	187	83	2015-02-18 20:00:34.404707	2015-02-18 20:00:34.404707	2
315	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	1	2	56	1	188	83	2015-02-18 20:00:34.463572	2015-02-18 20:00:34.463572	4
316	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	1	2	57	1	188	83	2015-02-18 20:00:34.508994	2015-02-18 20:00:34.508994	4
317	KRAS mutations in lung patients are not associated with smoking history, age and gender	Negative	Does Not Support	2	1	2	1	180	84	2015-02-18 20:00:34.561647	2015-02-18 20:00:34.561647	3
318	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	2	1	2	40	179	84	2015-02-18 20:00:34.606024	2015-02-18 20:00:34.606024	3
319	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	3	1	2	42	183	84	2015-02-18 20:00:34.6642	2015-02-18 20:00:34.6642	3
320	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft models	Resistance or Non-Response	Supports	1	2	29	22	189	85	2015-02-18 20:00:34.719982	2015-02-18 20:00:34.719982	4
321	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Sensitivity	Supports	1	1	29	22	189	85	2015-02-18 20:00:34.773451	2015-02-18 20:00:34.773451	4
322	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft models	Sensitivity	Supports	1	2	29	22	189	85	2015-02-18 20:00:34.82582	2015-02-18 20:00:34.82582	4
323	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Better Outcome	Supports	3	1	29	22	189	85	2015-02-18 20:00:34.881094	2015-02-18 20:00:34.881094	4
324	MAP2K1 P124S confers increased resistance to inhibition by AZD6244 inhibition by about 5 fold.	Resistance or Non-Response	Supports	1	2	37	4	190	86	2015-02-18 20:00:34.934704	2015-02-18 20:00:34.934704	3
325	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	Resistance or Non-Response	Supports	1	2	37	4	191	87	2015-02-18 20:00:34.984773	2015-02-18 20:00:34.984773	3
326	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	2	5	30	132	88	2015-02-18 20:00:35.042724	2015-02-18 20:00:35.042724	3
327	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	2	41	30	132	88	2015-02-18 20:00:35.090712	2015-02-18 20:00:35.090712	3
328	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increase survival.	Sensitivity	Supports	1	4	28	15	192	89	2015-02-18 20:00:35.144471	2015-02-18 20:00:35.144471	5
329	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	Sensitivity	Supports	1	3	58	15	193	89	2015-02-18 20:00:35.208447	2015-02-18 20:00:35.208447	4
330	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	Sensitivity	Supports	1	1	59	15	193	89	2015-02-18 20:00:35.307001	2015-02-18 20:00:35.307001	4
331	"AML with mutated NPM1" is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	Positive	Supports	2	4	2	13	194	90	2015-02-18 20:00:35.361557	2015-02-18 20:00:35.361557	5
332	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	3	1	2	4	195	92	2015-02-18 20:00:35.407248	2015-02-18 20:00:35.407248	3
333	The presence of NRAS mutation in AML patient does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	N/A	Does Not Support	3	1	2	13	196	93	2015-02-18 20:00:35.458137	2015-02-18 20:00:35.458137	3
334	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.	Poor Outcome	Supports	3	1	2	21	197	93	2015-02-18 20:00:35.511864	2015-02-18 20:00:35.511864	3
335	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	3	1	2	4	195	93	2015-02-18 20:00:35.563737	2015-02-18 20:00:35.563737	3
336	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.	Poor Outcome	Supports	3	1	2	21	197	94	2015-02-18 20:00:35.6177	2015-02-18 20:00:35.6177	3
337	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.	Poor Outcome	Supports	3	1	2	21	197	95	2015-02-18 20:00:35.673378	2015-02-18 20:00:35.673378	3
338	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	N/A	Does Not Support	3	1	2	13	198	96	2015-02-18 20:00:35.727841	2015-02-18 20:00:35.727841	3
339	Due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG is shown to be effective in a patient with metastatic malignant melenoma with a NRAS G13D mutation	Sensitivity	Supports	1	1	60	4	199	97	2015-02-18 20:00:35.791231	2015-02-18 20:00:35.791231	2
340	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	Positive	Supports	2	1	2	4	200	98	2015-02-18 20:00:35.847633	2015-02-18 20:00:35.847633	3
341	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	N/A	Does Not Support	1	1	29	21	105	98	2015-02-18 20:00:35.907838	2015-02-18 20:00:35.907838	3
342	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	Resistance or Non-Response	Supports	1	1	34	4	201	98	2015-02-18 20:00:35.9597	2015-02-18 20:00:35.9597	4
343	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	Resistance or Non-Response	Supports	1	1	29	21	105	98	2015-02-18 20:00:36.012454	2015-02-18 20:00:36.012454	3
344	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	Sensitivity	Supports	1	1	61	4	202	99	2015-02-18 20:00:36.063066	2015-02-18 20:00:36.063066	2
345	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	Sensitivity	Supports	1	1	61	4	202	100	2015-02-18 20:00:36.106938	2015-02-18 20:00:36.106938	2
346	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	62	43	203	101	2015-02-18 20:00:36.158669	2015-02-18 20:00:36.158669	4
347	GIST tumor harboring PDGFRA D842V mutation are likely to be benign	Negative	Supports	2	1	2	43	204	102	2015-02-18 20:00:36.218345	2015-02-18 20:00:36.218345	3
348	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	Resistance or Non-Response	Supports	1	1	5	43	49	102	2015-02-18 20:00:36.278771	2015-02-18 20:00:36.278771	4
349	GIST cancer with D842V mutation is resistant to imatinib.	Resistance or Non-Response	Supports	1	1	5	43	205	102	2015-02-18 20:00:36.3388	2015-02-18 20:00:36.3388	4
350	In CHO cells with PDGFRA D842V mutation resulting in imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	62	43	203	102	2015-02-18 20:00:36.393981	2015-02-18 20:00:36.393981	4
351	In CHO cells with PDGFRA D842Y mutation resulting in imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	62	43	203	103	2015-02-18 20:00:36.455164	2015-02-18 20:00:36.455164	4
352	In CHO cells with PDGFRA deletion I843 mutation resulting in imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	62	43	203	104	2015-02-18 20:00:36.516249	2015-02-18 20:00:36.516249	4
353	In CHO cells with PDGFRA DI842-843IM mutation resulting in imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	62	43	203	105	2015-02-18 20:00:36.576662	2015-02-18 20:00:36.576662	4
354	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	55	21	178	106	2015-02-18 20:00:36.638918	2015-02-18 20:00:36.638918	4
355	The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	29	21	206	106	2015-02-18 20:00:36.687766	2015-02-18 20:00:36.687766	3
356	The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	26	21	206	106	2015-02-18 20:00:36.728572	2015-02-18 20:00:36.728572	3
357	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	2	63	3	207	106	2015-02-18 20:00:36.788437	2015-02-18 20:00:36.788437	4
358	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	2	20	3	208	106	2015-02-18 20:00:36.885587	2015-02-18 20:00:36.885587	3
359	The presence of exon 9 (such as E542K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	3	2	21	206	106	2015-02-18 20:00:36.948052	2015-02-18 20:00:36.948052	3
360	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	3	3	2	21	209	106	2015-02-18 20:00:37.003662	2015-02-18 20:00:37.003662	2
361	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	55	21	178	107	2015-02-18 20:00:37.058385	2015-02-18 20:00:37.058385	4
362	The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	29	21	206	107	2015-02-18 20:00:37.114192	2015-02-18 20:00:37.114192	3
363	The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	26	21	206	107	2015-02-18 20:00:37.154485	2015-02-18 20:00:37.154485	3
364	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	2	63	3	207	107	2015-02-18 20:00:37.217437	2015-02-18 20:00:37.217437	4
365	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	2	20	3	208	107	2015-02-18 20:00:37.270332	2015-02-18 20:00:37.270332	3
366	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	3	2	21	206	107	2015-02-18 20:00:37.350416	2015-02-18 20:00:37.350416	3
367	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	3	3	2	21	209	107	2015-02-18 20:00:37.421523	2015-02-18 20:00:37.421523	2
368	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	Resistance or Non-Response	Supports	1	1	29	21	105	108	2015-02-18 20:00:37.482442	2015-02-18 20:00:37.482442	3
369	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	N/A	Does Not Support	1	1	29	21	105	109	2015-02-18 20:00:37.542017	2015-02-18 20:00:37.542017	3
370	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	55	21	178	110	2015-02-18 20:00:37.600029	2015-02-18 20:00:37.600029	4
371	The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	29	21	206	110	2015-02-18 20:00:37.656415	2015-02-18 20:00:37.656415	3
372	The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	26	21	206	110	2015-02-18 20:00:37.695004	2015-02-18 20:00:37.695004	3
373	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	2	63	3	207	110	2015-02-18 20:00:37.75393	2015-02-18 20:00:37.75393	4
374	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	2	20	3	208	110	2015-02-18 20:00:37.806594	2015-02-18 20:00:37.806594	3
375	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	3	3	2	21	206	110	2015-02-18 20:00:37.85493	2015-02-18 20:00:37.85493	3
376	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	1	1	14	44	210	111	2015-02-18 20:00:37.909219	2015-02-18 20:00:37.909219	5
377	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	Sensitivity	Supports	1	1	14	44	210	114	2015-02-18 20:00:37.960461	2015-02-18 20:00:37.960461	5
378	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	Positive	Supports	2	1	2	12	211	116	2015-02-18 20:00:38.010243	2015-02-18 20:00:38.010243	5
379	The use of AZD1480 in vitro led to strong repression of tyroid cancer cell growth.	Sensitivity	Supports	1	2	64	12	212	116	2015-02-18 20:00:38.060364	2015-02-18 20:00:38.060364	3
380	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	Poor Outcome	Supports	3	1	2	12	213	116	2015-02-18 20:00:38.12269	2015-02-18 20:00:38.12269	4
381	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	3	3	2	35	214	117	2015-02-18 20:00:38.18003	2015-02-18 20:00:38.18003	3
382	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	3	3	2	35	214	118	2015-02-18 20:00:38.227227	2015-02-18 20:00:38.227227	3
383	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumor with wild type TP53.	Sensitivity	Supports	1	2	65	3	215	119	2015-02-18 20:00:38.283395	2015-02-18 20:00:38.283395	3
384	Breast cancer patients who harbor R175H mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	1	2	3	216	119	2015-02-18 20:00:38.346377	2015-02-18 20:00:38.346377	3
385	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R248 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	1	2	3	217	120	2015-02-18 20:00:38.396691	2015-02-18 20:00:38.396691	3
386	Breast cancer patients who harbor R248Q mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	1	2	3	216	120	2015-02-18 20:00:38.448373	2015-02-18 20:00:38.448373	3
387	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	Poor Outcome	Supports	3	1	2	3	216	121	2015-02-18 20:00:38.501522	2015-02-18 20:00:38.501522	3
388	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as R249 is prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	3	1	2	3	217	122	2015-02-18 20:00:38.594589	2015-02-18 20:00:38.594589	3
389	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as R249 is prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	3	1	2	3	217	123	2015-02-18 20:00:38.651618	2015-02-18 20:00:38.651618	3
390	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R273 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	1	2	3	217	124	2015-02-18 20:00:38.704781	2015-02-18 20:00:38.704781	3
391	Breast cancer patients who harbor R273C mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	1	2	3	216	124	2015-02-18 20:00:38.758115	2015-02-18 20:00:38.758115	3
392	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R273 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	3	1	2	3	217	125	2015-02-18 20:00:38.805773	2015-02-18 20:00:38.805773	3
393	Breast cancer patients who harbor R273H mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	3	1	2	3	216	125	2015-02-18 20:00:38.858657	2015-02-18 20:00:38.858657	3
394	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as V173 is prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	3	1	2	3	217	126	2015-02-18 20:00:38.908869	2015-02-18 20:00:38.908869	3
395	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	Negative	Does Not Support	2	3	2	13	218	130	2015-02-18 20:00:38.973075	2015-02-18 20:00:38.973075	3
396	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N/A	Does Not Support	3	3	2	13	218	130	2015-02-18 20:00:39.027859	2015-02-18 20:00:39.027859	2
397	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N/A	Does Not Support	3	3	2	35	218	130	2015-02-18 20:00:39.078616	2015-02-18 20:00:39.078616	2
398	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	3	3	2	35	219	130	2015-02-18 20:00:39.135468	2015-02-18 20:00:39.135468	3
399	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	Negative	Does Not Support	2	3	2	13	218	131	2015-02-18 20:00:39.192306	2015-02-18 20:00:39.192306	3
400	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N/A	Does Not Support	3	3	2	13	218	131	2015-02-18 20:00:39.253485	2015-02-18 20:00:39.253485	2
401	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N/A	Does Not Support	3	3	2	35	218	131	2015-02-18 20:00:39.300234	2015-02-18 20:00:39.300234	2
402	After adjust for age and cancer stage, presence of U2AF mutation such as S34Y/F is prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	3	3	2	35	219	131	2015-02-18 20:00:39.356619	2015-02-18 20:00:39.356619	3
403	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	1	1	2	13	220	132	2015-02-18 20:00:39.412943	2015-02-18 20:00:39.412943	3
404	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	N/A	Does Not Support	3	1	2	13	221	132	2015-02-18 20:00:39.46577	2015-02-18 20:00:39.46577	4
405	No difference in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N/A	Does Not Support	3	1	2	13	221	132	2015-02-18 20:00:39.516094	2015-02-18 20:00:39.516094	3
406	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	2	13	220	132	2015-02-18 20:00:39.565003	2015-02-18 20:00:39.565003	3
407	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	2	13	220	132	2015-02-18 20:00:39.615498	2015-02-18 20:00:39.615498	3
408	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	3	1	2	13	222	132	2015-02-18 20:00:39.671989	2015-02-18 20:00:39.671989	3
409	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	2	13	221	132	2015-02-18 20:00:39.719811	2015-02-18 20:00:39.719811	3
410	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	Poor Outcome	Supports	3	1	2	13	223	132	2015-02-18 20:00:39.776092	2015-02-18 20:00:39.776092	2
411	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	Poor Outcome	Supports	3	1	2	13	223	132	2015-02-18 20:00:39.830247	2015-02-18 20:00:39.830247	2
412	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	1	1	2	13	220	133	2015-02-18 20:00:39.878842	2015-02-18 20:00:39.878842	3
413	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	N/A	Does Not Support	3	1	2	13	221	133	2015-02-18 20:00:39.933801	2015-02-18 20:00:39.933801	4
414	No difference in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N/A	Does Not Support	3	1	2	13	221	133	2015-02-18 20:00:39.992152	2015-02-18 20:00:39.992152	3
415	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	2	13	220	133	2015-02-18 20:00:40.052472	2015-02-18 20:00:40.052472	3
416	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	2	13	220	133	2015-02-18 20:00:40.145133	2015-02-18 20:00:40.145133	3
417	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	3	1	2	13	222	133	2015-02-18 20:00:40.207481	2015-02-18 20:00:40.207481	3
418	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	3	1	2	13	221	133	2015-02-18 20:00:40.258196	2015-02-18 20:00:40.258196	3
419	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	Resistance or Non-Response	Supports	1	4	44	45	224	34	2015-02-18 20:00:40.321776	2015-02-18 20:00:40.321776	5
\.


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 419, true);


--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 47, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-02-18 20:00:00.898925	2015-02-18 20:00:00.898925
1	2	2015-02-18 20:00:00.905097	2015-02-18 20:00:00.905097
2	3	2015-02-18 20:00:01.387186	2015-02-18 20:00:01.387186
2	4	2015-02-18 20:00:01.392289	2015-02-18 20:00:01.392289
3	5	2015-02-18 20:00:01.779778	2015-02-18 20:00:01.779778
3	6	2015-02-18 20:00:01.787323	2015-02-18 20:00:01.787323
4	7	2015-02-18 20:00:02.426639	2015-02-18 20:00:02.426639
4	8	2015-02-18 20:00:02.431186	2015-02-18 20:00:02.431186
5	9	2015-02-18 20:00:02.846049	2015-02-18 20:00:02.846049
5	10	2015-02-18 20:00:02.851201	2015-02-18 20:00:02.851201
6	11	2015-02-18 20:00:03.168903	2015-02-18 20:00:03.168903
6	12	2015-02-18 20:00:03.173568	2015-02-18 20:00:03.173568
7	11	2015-02-18 20:00:03.467737	2015-02-18 20:00:03.467737
7	12	2015-02-18 20:00:03.470502	2015-02-18 20:00:03.470502
8	11	2015-02-18 20:00:03.68938	2015-02-18 20:00:03.68938
8	12	2015-02-18 20:00:03.693203	2015-02-18 20:00:03.693203
9	13	2015-02-18 20:00:03.945654	2015-02-18 20:00:03.945654
10	14	2015-02-18 20:00:04.191372	2015-02-18 20:00:04.191372
10	15	2015-02-18 20:00:04.196165	2015-02-18 20:00:04.196165
11	14	2015-02-18 20:00:04.413904	2015-02-18 20:00:04.413904
11	15	2015-02-18 20:00:04.416393	2015-02-18 20:00:04.416393
12	16	2015-02-18 20:00:04.638164	2015-02-18 20:00:04.638164
12	17	2015-02-18 20:00:04.642467	2015-02-18 20:00:04.642467
13	18	2015-02-18 20:00:04.860087	2015-02-18 20:00:04.860087
13	19	2015-02-18 20:00:04.864738	2015-02-18 20:00:04.864738
13	20	2015-02-18 20:00:04.869138	2015-02-18 20:00:04.869138
14	21	2015-02-18 20:00:05.065388	2015-02-18 20:00:05.065388
15	21	2015-02-18 20:00:05.298711	2015-02-18 20:00:05.298711
15	22	2015-02-18 20:00:05.304099	2015-02-18 20:00:05.304099
16	23	2015-02-18 20:00:05.528019	2015-02-18 20:00:05.528019
16	24	2015-02-18 20:00:05.532009	2015-02-18 20:00:05.532009
17	25	2015-02-18 20:00:05.779909	2015-02-18 20:00:05.779909
17	26	2015-02-18 20:00:05.785142	2015-02-18 20:00:05.785142
18	27	2015-02-18 20:00:06.004971	2015-02-18 20:00:06.004971
18	28	2015-02-18 20:00:06.01042	2015-02-18 20:00:06.01042
19	29	2015-02-18 20:00:06.243171	2015-02-18 20:00:06.243171
19	30	2015-02-18 20:00:06.248166	2015-02-18 20:00:06.248166
20	31	2015-02-18 20:00:06.461336	2015-02-18 20:00:06.461336
20	32	2015-02-18 20:00:06.46651	2015-02-18 20:00:06.46651
21	31	2015-02-18 20:00:06.674983	2015-02-18 20:00:06.674983
21	32	2015-02-18 20:00:06.678351	2015-02-18 20:00:06.678351
22	33	2015-02-18 20:00:06.90656	2015-02-18 20:00:06.90656
22	18	2015-02-18 20:00:06.910069	2015-02-18 20:00:06.910069
33	34	2015-02-18 20:00:09.222422	2015-02-18 20:00:09.222422
33	18	2015-02-18 20:00:09.225701	2015-02-18 20:00:09.225701
40	35	2015-02-18 20:00:10.906304	2015-02-18 20:00:10.906304
40	36	2015-02-18 20:00:10.910872	2015-02-18 20:00:10.910872
42	37	2015-02-18 20:00:11.328019	2015-02-18 20:00:11.328019
42	38	2015-02-18 20:00:11.332712	2015-02-18 20:00:11.332712
46	39	2015-02-18 20:00:12.198204	2015-02-18 20:00:12.198204
46	40	2015-02-18 20:00:12.203787	2015-02-18 20:00:12.203787
\.


--
-- Data for Name: notifications; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY notifications (id, subscription_id, user_id, subscribable_id, subscribable_type, content, url, acknowledged, delivered, created_at, updated_at) FROM stdin;
\.


--
-- Name: notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('notifications_id_seq', 1, false);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20150210204445
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 224, true);


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
4	KIT Exon 11	\N	2015-02-18 20:00:17.970936	2015-02-18 20:00:17.970936
5	KIT Exon 17	\N	2015-02-18 20:00:18.445821	2015-02-18 20:00:18.445821
1	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-02-18 20:00:16.5592	2015-02-18 20:00:40.421858
14	Erlotinib Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-02-18 20:00:40.3668	2015-02-18 20:00:40.443728
11	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-02-18 20:00:22.14916	2015-02-18 20:00:40.44866
7	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-02-18 20:00:19.117424	2015-02-18 20:00:40.451201
6	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-02-18 20:00:18.513096	2015-02-18 20:00:40.457854
10	NPM1 exon 12	To be filled in by Dr. KK, resident blood expert and knower of all things hematology. :) :) :) 	2015-02-18 20:00:21.974762	2015-02-18 20:00:40.46087
12	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-02-18 20:00:22.389982	2015-02-18 20:00:40.463821
13	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-02-18 20:00:22.534332	2015-02-18 20:00:40.467285
8	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-02-18 20:00:19.544629	2015-02-18 20:03:50.549683
2	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-02-18 20:00:16.641489	2015-02-18 20:03:50.558092
3	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-02-18 20:00:17.043142	2015-02-18 20:03:50.564173
9	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-02-18 20:00:19.720976	2015-02-18 20:03:50.570869
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
5	1	2015-02-18 20:00:16.577408	2015-02-18 20:00:16.577408
6	1	2015-02-18 20:00:16.638804	2015-02-18 20:00:16.638804
6	2	2015-02-18 20:00:16.643944	2015-02-18 20:00:16.643944
7	1	2015-02-18 20:00:16.713344	2015-02-18 20:00:16.713344
7	2	2015-02-18 20:00:16.717173	2015-02-18 20:00:16.717173
8	2	2015-02-18 20:00:16.778258	2015-02-18 20:00:16.778258
46	3	2015-02-18 20:00:17.045936	2015-02-18 20:00:17.045936
46	3	2015-02-18 20:00:17.102638	2015-02-18 20:00:17.102638
47	3	2015-02-18 20:00:17.219776	2015-02-18 20:00:17.219776
47	3	2015-02-18 20:00:17.284337	2015-02-18 20:00:17.284337
48	3	2015-02-18 20:00:17.392493	2015-02-18 20:00:17.392493
48	3	2015-02-18 20:00:17.47116	2015-02-18 20:00:17.47116
49	3	2015-02-18 20:00:17.57695	2015-02-18 20:00:17.57695
49	3	2015-02-18 20:00:17.642438	2015-02-18 20:00:17.642438
50	3	2015-02-18 20:00:17.746168	2015-02-18 20:00:17.746168
50	3	2015-02-18 20:00:17.804777	2015-02-18 20:00:17.804777
76	4	2015-02-18 20:00:17.973705	2015-02-18 20:00:17.973705
76	4	2015-02-18 20:00:18.147781	2015-02-18 20:00:18.147781
76	4	2015-02-18 20:00:18.208744	2015-02-18 20:00:18.208744
76	4	2015-02-18 20:00:18.261482	2015-02-18 20:00:18.261482
77	4	2015-02-18 20:00:18.326131	2015-02-18 20:00:18.326131
77	4	2015-02-18 20:00:18.384462	2015-02-18 20:00:18.384462
70	5	2015-02-18 20:00:18.448567	2015-02-18 20:00:18.448567
51	6	2015-02-18 20:00:18.515799	2015-02-18 20:00:18.515799
52	6	2015-02-18 20:00:18.571886	2015-02-18 20:00:18.571886
53	6	2015-02-18 20:00:18.626668	2015-02-18 20:00:18.626668
54	6	2015-02-18 20:00:18.706755	2015-02-18 20:00:18.706755
55	6	2015-02-18 20:00:18.75943	2015-02-18 20:00:18.75943
56	6	2015-02-18 20:00:18.814066	2015-02-18 20:00:18.814066
57	6	2015-02-18 20:00:18.879648	2015-02-18 20:00:18.879648
58	6	2015-02-18 20:00:18.948362	2015-02-18 20:00:18.948362
59	6	2015-02-18 20:00:19.060036	2015-02-18 20:00:19.060036
35	7	2015-02-18 20:00:19.120021	2015-02-18 20:00:19.120021
36	7	2015-02-18 20:00:19.175996	2015-02-18 20:00:19.175996
38	7	2015-02-18 20:00:19.254397	2015-02-18 20:00:19.254397
41	7	2015-02-18 20:00:19.315736	2015-02-18 20:00:19.315736
44	7	2015-02-18 20:00:19.373591	2015-02-18 20:00:19.373591
45	7	2015-02-18 20:00:19.428486	2015-02-18 20:00:19.428486
43	7	2015-02-18 20:00:19.487143	2015-02-18 20:00:19.487143
2	8	2015-02-18 20:00:19.548034	2015-02-18 20:00:19.548034
2	8	2015-02-18 20:00:19.607778	2015-02-18 20:00:19.607778
3	8	2015-02-18 20:00:19.664428	2015-02-18 20:00:19.664428
115	9	2015-02-18 20:00:19.723856	2015-02-18 20:00:19.723856
116	9	2015-02-18 20:00:19.798107	2015-02-18 20:00:19.798107
91	10	2015-02-18 20:00:21.97743	2015-02-18 20:00:21.97743
91	10	2015-02-18 20:00:22.034268	2015-02-18 20:00:22.034268
91	10	2015-02-18 20:00:22.090263	2015-02-18 20:00:22.090263
11	11	2015-02-18 20:00:22.151721	2015-02-18 20:00:22.151721
13	11	2015-02-18 20:00:22.212499	2015-02-18 20:00:22.212499
15	11	2015-02-18 20:00:22.270311	2015-02-18 20:00:22.270311
16	11	2015-02-18 20:00:22.335719	2015-02-18 20:00:22.335719
113	12	2015-02-18 20:00:22.392624	2015-02-18 20:00:22.392624
113	12	2015-02-18 20:00:22.469653	2015-02-18 20:00:22.469653
127	13	2015-02-18 20:00:22.537348	2015-02-18 20:00:22.537348
127	13	2015-02-18 20:00:22.594838	2015-02-18 20:00:22.594838
128	13	2015-02-18 20:00:22.664454	2015-02-18 20:00:22.664454
129	13	2015-02-18 20:00:22.729381	2015-02-18 20:00:22.729381
71	4	2015-02-18 20:00:32.93982	2015-02-18 20:00:32.93982
71	4	2015-02-18 20:00:32.998371	2015-02-18 20:00:32.998371
71	4	2015-02-18 20:00:33.057221	2015-02-18 20:00:33.057221
71	4	2015-02-18 20:00:33.119394	2015-02-18 20:00:33.119394
71	4	2015-02-18 20:00:33.178852	2015-02-18 20:00:33.178852
71	4	2015-02-18 20:00:33.236116	2015-02-18 20:00:33.236116
74	5	2015-02-18 20:00:33.389884	2015-02-18 20:00:33.389884
74	5	2015-02-18 20:00:33.444898	2015-02-18 20:00:33.444898
101	8	2015-02-18 20:00:36.195687	2015-02-18 20:00:36.195687
102	8	2015-02-18 20:00:36.264827	2015-02-18 20:00:36.264827
102	8	2015-02-18 20:00:36.321146	2015-02-18 20:00:36.321146
102	8	2015-02-18 20:00:36.380178	2015-02-18 20:00:36.380178
102	8	2015-02-18 20:00:36.439621	2015-02-18 20:00:36.439621
103	8	2015-02-18 20:00:36.498324	2015-02-18 20:00:36.498324
104	8	2015-02-18 20:00:36.56114	2015-02-18 20:00:36.56114
105	8	2015-02-18 20:00:36.619264	2015-02-18 20:00:36.619264
34	14	2015-02-18 20:00:40.369723	2015-02-18 20:00:40.369723
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 14, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 133, true);


--
-- PostgreSQL database dump complete
--

