--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: organizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY organizations (id, name, url, description) FROM stdin;
\.


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, username, created_at, updated_at, orcid, area_of_expertise, deleted, deleted_at, role, organization_id) FROM stdin;
\.


--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	664039dd-7dd3-4b60-b61c-98c821a590ef	2015-07-31 17:34:13.827018
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	9264bc5b-7378-4b77-936b-3c366575ea0f	2015-07-31 17:34:14.157491
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1b3d5c3d-e979-4469-b51d-54ca4c700775	2015-07-31 17:34:14.443718
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1a1d867d-daa6-42ab-bd1b-a9d3cb9b3a90	2015-07-31 17:34:14.710622
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e4f6d5b2-a765-492a-b138-702f0e097c10	2015-07-31 17:34:14.959767
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 672\nname: BRCA1\ndescription: BRCA1 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 1, early onset\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b1e1012b-a497-4051-a494-4a1967644165	2015-07-31 17:34:15.22838
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 675\nname: BRCA2\ndescription: BRCA2 mutations in the germline have become a hallmark for hereditary\n  breast and ovarian cancers. Variants that have been demonstrated to reduce the function\n  of the protein have been shown to increase risk for these cancers, as well as prostate\n  and pancreatic cancer. These findings have been the impetus for the increased popularity\n  of genetic testing of healthy indivudals to assess risk.\nofficial_name: breast cancer 2, early onset\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	ec718a7d-b43e-41c2-935e-9d7671216633	2015-07-31 17:34:15.500448
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e75718f0-a1ba-4c57-8220-63d1a153a247	2015-07-31 17:34:15.729424
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	5e74137b-9c5c-495b-8b1c-2b9a069de7fa	2015-07-31 17:34:15.963796
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	61604b36-be6a-47b1-83f1-d288d1bebfb6	2015-07-31 17:34:16.148448
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	9f3e3eb6-acd4-43ee-8f45-2eb6faae7442	2015-07-31 17:34:16.360821
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	cf7db244-a5bb-4130-85dc-37d73f138d56	2015-07-31 17:34:16.567956
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	80224b3d-f784-4f73-8c06-c550e2f2868a	2015-07-31 17:34:16.786772
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1ad411bd-173e-40ae-839c-3d3104976dbf	2015-07-31 17:34:17.08717
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	f9127a41-13a9-465b-b876-52696af7b7b4	2015-07-31 17:34:17.343161
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	d9074b39-c765-42dd-ad61-7996196b462f	2015-07-31 17:34:17.59073
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity.\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	509ee459-1f90-4c7b-9c42-bc3dcb7e8ed8	2015-07-31 17:34:17.803579
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b894fc11-23f5-4075-aa3b-9a071908118a	2015-07-31 17:34:18.015874
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	56fdc40d-282a-4296-bb62-adf6c2857801	2015-07-31 17:34:18.280632
146	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2c170dc7-f1dd-4656-a747-d1731908b605	2015-07-31 17:34:28.428609
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown sensitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	580787f1-db92-463d-a405-daee90e08503	2015-07-31 17:34:18.540306
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	4c78553d-b03a-4bc6-a5dd-d756c027d893	2015-07-31 17:34:18.785417
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	cef893bb-755a-456c-bd86-a507e912257d	2015-07-31 17:34:19.065299
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions that lead to pathway\n  actiation have been demonstrated to have oncogenic potential across multiple cancer\n  types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success\n  in treating over-active FGFR signalling, prompting use of diagnostic sequencing\n  targeting the FGFR genes, especially in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e5f43a92-b412-40fb-96ec-17dd92e45e4a	2015-07-31 17:34:19.327454
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	18dd5246-372a-47f3-a67e-d02d2aeecd33	2015-07-31 17:34:19.542127
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: GATA2 is a transcription factor involved in stem cell maintenance with\n  key roles in hematopoietic development. GATA2 mutations are associated with a variety\n  of inherited and acquired immune disorders including myelodysplastic syndrome and\n  acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance\n  GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell\n  lung cancer. Preclinical models have indicated therapeutic benefit from targeting\n  GATA2-mediated pathways in the context of KRAS-driven NSCLC.\nofficial_name: GATA binding protein 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	36a5b40d-2327-4b70-b5a7-dcadf513fa71	2015-07-31 17:34:19.780185
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. The implications of mutations in this gene vary greatly by cancer\n  type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations\n  have been associated with worse outcome, shorter overall survival, and normal karyotype.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Unlike the association\n  with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated\n  with specific cytogenetic abnormalities, 1p and 19q deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	087c3e9e-4062-4ab8-819f-46b2d6ae2dff	2015-07-31 17:34:20.029131
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: IDH2 mutations have been observed in a number of cancer types, including\n  sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in\n  the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial\n  (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described\n  as mutually exclusive in many of these cancer types. The most frequent mutations\n  involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic\n  enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall\n  prognosis in AML patients, its utility as a prognostic marker in MDS is still under\n  debate. Additionally, IDH2 (R140) has been associated with improved overall survival\n  in AML. IDH2 mutations have been associated with improved prognosis in gliomas.\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	044c2e9b-078f-4f27-99a1-a67aa8149a2b	2015-07-31 17:34:20.281502
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders. The variant is a known\n  activating mutation, and activated JAK2 is sufficient to drive myeloproliferative\n  disorders in mouse models. V617F, while most recurrent, is not the only mechanism\n  by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic\n  markers tested upon diagnosis with a myeloproliferative disorder.\nofficial_name: Janus kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	301c47aa-ab31-4bb5-9dcf-fcfb89577ee5	2015-07-31 17:34:20.505706
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: c-KIT activation has been shown to have oncogenic activity in gastrointestinal\n  stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted\n  therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive\n  patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can\n  be the result of many genomic events from genomic amplification to overexpression\n  to missense mutations. Missense mutations have been shown to be key players in mediating\n  clinical response and acquired resistance in patients being treated with these targeted\n  therapeutics.\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	82cefe0c-f21d-44ed-b7c0-0fa91959a77b	2015-07-31 17:34:20.716261
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	587aae41-a895-4f60-9999-dcf1a106469e	2015-07-31 17:34:20.979805
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: ''\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7e8b295e-0112-4e24-bbae-d998c834613e	2015-07-31 17:34:21.223324
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: ''\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2f72f236-73a0-43a2-97eb-2d9471735c2e	2015-07-31 17:34:21.437259
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: ''\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7ab2dfeb-d3a0-4d9f-8d31-fd5e248280cc	2015-07-31 17:34:21.663852
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: ''\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	74935ac0-eff9-497d-adec-bc2f86130e8b	2015-07-31 17:34:21.854779
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1 is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b23d4b35-d8f5-4160-8dc2-0ff41e6081c1	2015-07-31 17:34:22.041604
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: Mutations in the RAS family of proteins have frequently observed across\n  cancer types. The amino acid positions account for the overwhelming majority of\n  these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also\n  behave very differently when expressed in non-native tissue types, likely due to\n  differences in the C-terminal hyper-variable regions. Mis-regulation of isoform\n  expression has been shown to be a driving event in cancer, as well as missense mutations\n  at the three hotspots previously mentioned. While highly recurrent in cancer, targeted\n  these RAS mutants has also been very elusive, and has not yet become common practice\n  in the clinic.\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b1f83d22-89fc-4b86-8b2d-217a15eae227	2015-07-31 17:34:22.27112
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	10f861b5-a187-4306-82fa-bb8e7b099abd	2015-07-31 17:34:22.491979
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: ''\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	c656813f-a76b-468d-a037-078478cab34c	2015-07-31 17:34:22.700248
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: promyelocytic leukemia\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	adc2a0f5-cc6e-49a5-9dc1-df2b0a0d47d2	2015-07-31 17:34:22.903671
700	64	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '9'\nstart:\n- \n- '5073770'\nstop:\n- \n- '5073770'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	b4755470-d7d5-4378-851c-9e99c7c08287	2015-07-31 17:37:42.886923
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARA\ndescription: The PML-RARA fusion is the result of a recurrent, balanced translocation\n  between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event\n  in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown\n  sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA\n  fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for\n  treating these patients, and early results seem promising.\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	eb6a0350-4215-4e82-b6a5-b4d051c4d5e8	2015-07-31 17:34:23.096363
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: ''\nofficial_name: phosphatase and tensin homolog\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	054ac81b-9781-4e4c-9f74-6abd088400a6	2015-07-31 17:34:23.317721
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	240cb042-d597-42d5-b9da-75139a1bb0e1	2015-07-31 17:34:23.5646
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: ''\nofficial_name: runt-related transcription factor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	7fa456ac-dc19-414f-a48a-184bd3d668dd	2015-07-31 17:34:23.820478
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	220092b7-3f67-4926-9e39-89126ed7d453	2015-07-31 17:34:24.084794
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: TP53 mutations are universal across cancer types. Loss of tumor suppressors\n  is most recognized by large deleterious events, such as frameshift mutations, or\n  premature stop codons. In TP53 however, many of the observed mutations in cancer\n  are found to be single nucleotide variants, or missense mutations. These variants\n  are also very broadly distributed throughout the gene, not localizing in any particular\n  hotspot. While a large proportion of cancer genomics research is focused on somatic\n  variants, TP53 is also of note in the germline. Germline TP53 mutations are the\n  hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been\n  found to have prognostic impact on patient outcomes.\nofficial_name: tumor protein p53\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	1ebde46d-86f0-4e13-b75a-37ad3c846946	2015-07-31 17:34:24.329335
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: ''\nofficial_name: tuberous sclerosis 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	3d5f6534-059b-4920-bdf5-f3a6a2144369	2015-07-31 17:34:24.537098
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: ''\nofficial_name: tuberous sclerosis 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	cc5ad478-578a-4a30-9d31-349fe0c62aa9	2015-07-31 17:34:24.732575
48	48	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	457a19af-158f-41eb-9883-4f88dc2f696e	2015-07-31 17:34:25.01759
49	49	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: WT1 is a tumor suppressor gene associated with the development of Wilms'\n  Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently\n  identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy\n  resistance.\nofficial_name: Wilms tumor 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b5e6c4ab-a407-4cb5-91f3-9ced22fdb9ae	2015-07-31 17:34:25.268978
50	50	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4851\nname: NOTCH1\ndescription: ''\nofficial_name: notch 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	052ad2d5-72f3-41cd-83a2-6a9df65163f7	2015-07-31 17:34:25.482913
51	51	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4921\nname: DDR2\ndescription: ''\nofficial_name: discoidin domain receptor tyrosine kinase 2\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	818d5616-c12b-43f6-80b6-3e8b127c757d	2015-07-31 17:34:25.685845
52	52	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4233\nname: MET\ndescription: ''\nofficial_name: MET proto-oncogene, receptor tyrosine kinase\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	397c558c-ca46-4032-9962-ae1d4abc9483	2015-07-31 17:34:25.895023
53	53	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2313\nname: FLI1\ndescription: ''\nofficial_name: Fli-1 proto-oncogene, ETS transcription factor\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	48e27297-e921-4723-ac2f-485998452f26	2015-07-31 17:34:26.087671
54	54	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2130\nname: EWSR1\ndescription: ''\nofficial_name: EWS RNA-binding protein 1\nclinical_description: \ndeleted: false\ndeleted_at: \n	1	\N	\N	fff5197a-2948-4518-a1cb-7d5648117fd1	2015-07-31 17:34:26.29236
147	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ac900a5e-fa6d-4143-ad5a-da34f0283cd6	2015-07-31 17:34:28.448708
55	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4b829cee-71c3-44d6-9194-128ca3d54059	2015-07-31 17:34:26.360532
56	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c2596060-37cc-49f7-8627-d95fac4a65d7	2015-07-31 17:34:26.386504
57	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b111f851-2e60-4f56-981e-6c165b391208	2015-07-31 17:34:26.407779
58	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c6f1baaa-c162-4253-80fa-54d884fcea10	2015-07-31 17:34:26.427519
59	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6be583b6-5e1a-4828-b902-ce8ce5dddfbd	2015-07-31 17:34:26.447235
60	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	42324d1d-8523-4197-aac9-8284923be9e9	2015-07-31 17:34:26.465192
61	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	9ae13577-433a-4131-b33c-d24701f69f0b	2015-07-31 17:34:26.48497
62	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	24ef2984-6620-4385-8703-c9358cea70b3	2015-07-31 17:34:26.508014
63	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	98651672-1965-4d3b-9a25-c39d592b5409	2015-07-31 17:34:26.528483
64	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2f665fc5-2725-4903-b924-cc630990208a	2015-07-31 17:34:26.546817
65	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	6ba5445d-b63b-45d3-9ac1-c7d6341d870b	2015-07-31 17:34:26.571234
701	65	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55599321'\nstop:\n- \n- '55599321'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	6a487b53-8fdd-402e-8432-72a9b5289ef0	2015-07-31 17:37:42.904026
66	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	040284c5-f340-46fa-87ce-bd54b6834ec6	2015-07-31 17:34:26.591766
67	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	103016f3-af20-4581-a5e7-ce61d3e60ce4	2015-07-31 17:34:26.612173
68	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	be169aa1-2921-4f51-8584-08f91ea146d4	2015-07-31 17:34:26.636023
69	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d475a420-3de2-472f-8c94-8aca27a58183	2015-07-31 17:34:26.65718
70	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	33eea7ac-cccb-4859-9111-4485e3cfbf8a	2015-07-31 17:34:26.678206
71	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	fe688b01-97fe-4f80-b06c-3585d4a73068	2015-07-31 17:34:26.702705
72	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	09344f18-0e2a-4a9c-a9d5-eea58e92f279	2015-07-31 17:34:26.721813
73	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0e23010b-07bd-4245-8c8a-7a84a6939020	2015-07-31 17:34:26.743868
74	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a78063f7-77e9-4f01-8690-1661113cb508	2015-07-31 17:34:26.76374
75	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ad85da10-7790-4c6a-a8a1-6acc2b7a9de0	2015-07-31 17:34:26.791576
85	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	20ca27c8-d166-4c15-ac03-0022b098be42	2015-07-31 17:34:27.020025
76	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	048b8c23-47c5-4496-8288-1e19356e3bf0	2015-07-31 17:34:26.814294
77	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	485e6a7d-f0f8-450c-99b8-8463d6463178	2015-07-31 17:34:26.844261
78	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	35fc9b8c-d940-4dd3-a0ab-d26a61e17614	2015-07-31 17:34:26.863054
79	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	8cd01482-e48a-47f2-953c-b981be8f561a	2015-07-31 17:34:26.88571
80	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d21a9048-2075-4365-9b7c-e7539137be1b	2015-07-31 17:34:26.906794
81	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	47e5abec-4da2-441e-9054-0b1ca7a9bbdd	2015-07-31 17:34:26.934617
82	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a56926b8-1be1-4c63-b17f-963a0859ceb3	2015-07-31 17:34:26.958289
83	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	15baf5e1-0f00-42d6-a469-9c8f4afda3f2	2015-07-31 17:34:26.978989
84	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	3082cfab-c9fe-46a5-a31e-27e086b43c94	2015-07-31 17:34:26.998524
168	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	822c6e1b-0e13-4b5d-931f-57faa84838f8	2015-07-31 17:34:28.967923
86	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	67e9304c-83f1-4e96-b114-82b498a1e423	2015-07-31 17:34:27.042136
87	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5bf27d80-9c23-4a6a-955b-b47ae5dde561	2015-07-31 17:34:27.0707
88	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies in non-small cell lung cancer. While successful in amplified\n  EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib,\n  neratinib) in treating patients harboring this mutation before treatment is notably\n  lower. This lack of efficacy can likely be to blame for the poorer prognosis for\n  patients with this mutation as compared to patients with wildtype EGFR or other\n  types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired\n  resistance to TKI inhibition have been shown to harbor this mutation, implicating\n  it as a mechanism of acquired therapy resistence. The third generation TKI's are\n  being developed with this resistance problem in mind, and early stage experiments\n  have shown some efficacy of these drugs in previously resistant tumors with mutant\n  EGFR.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	dfd9222d-a67e-4ea6-9921-874b3a30adc3	2015-07-31 17:34:27.091096
89	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	edf018eb-5ffe-4b15-8188-cf871377c92c	2015-07-31 17:34:27.115828
90	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	cf4efff0-43d9-4095-9e0f-ba4350a34745	2015-07-31 17:34:27.134649
91	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	abb28b52-9cd3-4c92-bc4e-4dc5bbf48cbf	2015-07-31 17:34:27.155723
92	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b0cf5faf-f2c8-4d99-bf6c-61e58cf5cb36	2015-07-31 17:34:27.175933
93	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	76c94b07-6624-4d21-9708-6af368a9ca90	2015-07-31 17:34:27.1969
192	138	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: S2275FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at S2275,\n  have been shown to be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7703b582-4278-4e67-b373-76c40e648ec7	2015-07-31 17:34:29.489921
94	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	9a9309da-a622-4fe2-b65b-e4ce1e0826ab	2015-07-31 17:34:27.218119
95	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2c6695e7-acf6-4b5d-9c56-7c39f11fa9e8	2015-07-31 17:34:27.244969
96	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	cfe16b95-e312-44c6-86ac-31479dcd259d	2015-07-31 17:34:27.268956
97	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4982d132-99fe-46da-a03b-9ce043a61193	2015-07-31 17:34:27.289539
98	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ba06e728-4c00-4e94-831e-81b4c58b5a03	2015-07-31 17:34:27.314342
99	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	90034c3a-bb6a-4068-b8fa-ae31190e8f48	2015-07-31 17:34:27.335725
100	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	75346068-5a54-4c0d-8837-374312afaf71	2015-07-31 17:34:27.356339
101	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	cd08e2e2-8f43-4ad8-9a5f-d78739353303	2015-07-31 17:34:27.380678
102	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4270182f-993d-4562-8eda-eb0d4a54abdc	2015-07-31 17:34:27.401521
103	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5d75ba75-d82f-4746-b641-a875424c760c	2015-07-31 17:34:27.440702
104	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	dd776103-f8c0-4d49-b1b8-d9c821138342	2015-07-31 17:34:27.460814
105	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: P235R\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4b1b0fe1-459f-44ef-a03b-2db009f1bc17	2015-07-31 17:34:27.479825
106	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f8b96ca4-be29-4708-822c-f5471ce966ea	2015-07-31 17:34:27.504387
107	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-BAIAP2L1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	11e84124-c053-48f8-9044-1fea0ab318a7	2015-07-31 17:34:27.525762
108	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0def8815-0838-4d06-ac3e-fb733fcf1e05	2015-07-31 17:34:27.544611
109	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: ITD MUTATIONS\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7f3a2da0-cd35-4c3a-a343-9af55e85891e	2015-07-31 17:34:27.567595
110	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: TKD MUTATIONS\ndescription: FLT3 tyrosine kinase domain mutations (FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7d1a0f2a-b44d-4875-aeac-56ae4df36bf6	2015-07-31 17:34:27.596421
111	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	79b417ff-b068-41cd-b55b-42cada494f66	2015-07-31 17:34:27.62577
121	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: INTERNAL DUPLICATION\ndescription: c-KIT internal duplications have been observed in exon 11, within the\n  juxtamembrane domain. In a case study of an anal melanoma patient harboring this\n  event, imatinib confered marked response. Also, cells harboring exon 11 mutations\n  have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better\n  prognosis to patients treated with the drug in the first year.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	39ce473a-0e4d-4b2e-a514-55d93d2f5fa4	2015-07-31 17:34:27.876661
112	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0d96d9ce-8a3d-4a60-bab4-6a8fda56130d	2015-07-31 17:34:27.658072
113	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b97bf829-1542-46da-9fe3-e9aa27c6fa10	2015-07-31 17:34:27.68261
114	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	487be868-e968-4c19-bcd4-47c23cc21b7d	2015-07-31 17:34:27.705669
115	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5b4fc8d2-5606-4983-9ec9-4631e98ac5d6	2015-07-31 17:34:27.729375
116	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	27b84898-40da-4f92-af81-392225e6c553	2015-07-31 17:34:27.761274
117	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	400bc295-772b-4be6-abbd-6c4ec571f21a	2015-07-31 17:34:27.786415
118	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f13504fe-bd97-47dc-a146-39f6dabba598	2015-07-31 17:34:27.810778
119	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0e1d3e82-38a3-447b-aca0-e8bd59af64f7	2015-07-31 17:34:27.835747
120	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 11 MUTATIONS\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2cf48496-1dc6-4fad-93a9-1fef63c3600f	2015-07-31 17:34:27.856255
143	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 2 MUTATIONS\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	88f30222-b19d-4ef0-ac9b-d8cae850e2a9	2015-07-31 17:34:28.369847
122	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 13 MUTATIONS\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	8c84b13a-098b-48a8-a4a8-26074b765630	2015-07-31 17:34:27.899601
123	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 14 MUTATIONS\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	634739c2-20ca-4f88-bd97-996d73f314b6	2015-07-31 17:34:27.928192
124	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 17 MUTATIONS\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	83d8ae1d-e06f-4430-a9e0-64ab47f8a0b3	2015-07-31 17:34:27.951609
125	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: EXON 9 MUTATIONS\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	bb4ad8cb-537f-42c2-96b6-af144df39389	2015-07-31 17:34:27.97205
126	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	1bcfaea7-39e3-42be-b878-938237ca909e	2015-07-31 17:34:27.99573
127	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	45c3db20-e248-4405-ba5f-cf4f4bd1d9ff	2015-07-31 17:34:28.019602
128	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 1 MUTATIONS\ndescription: A study by Livre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4e78be5a-9c4d-4b88-bc31-ebb6a5fc0698	2015-07-31 17:34:28.046573
129	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: EXON 2 MUTATIONS\ndescription: In a study by Pao et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	3698967a-7d5d-49a5-8388-90af97a4db75	2015-07-31 17:34:28.084561
130	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5277f479-97b2-4aa1-953d-de64e1221191	2015-07-31 17:34:28.106734
131	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b01a74ac-1ebf-4571-8958-b26913d30965	2015-07-31 17:34:28.128738
132	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	840d90b3-f464-4003-beb0-3046d8a902e3	2015-07-31 17:34:28.14701
144	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 3 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	e1da8b37-0064-4195-ba51-df4ebceccb72	2015-07-31 17:34:28.388447
133	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ab1a0dba-f006-42d3-b793-3c6f68e169d2	2015-07-31 17:34:28.166248
134	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	93ff8cfe-bdfd-4108-bdab-e92a02c7f240	2015-07-31 17:34:28.185537
135	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	3a8c33fd-1c63-4b1c-8758-49196ce5867d	2015-07-31 17:34:28.209157
136	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4b46a9c6-1500-41a9-867d-0789f372f9ad	2015-07-31 17:34:28.22871
137	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	635e850f-a42b-4499-9bbf-c30dc53aff99	2015-07-31 17:34:28.250749
138	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c659dba0-a852-42a7-8767-4aa1722ae7e9	2015-07-31 17:34:28.271234
139	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ec770bf0-d008-4fa4-a30e-83262f7ce2e6	2015-07-31 17:34:28.290189
140	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 12 MUTATIONS\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f6469070-20bc-4eaa-a171-f23ac544d5c7	2015-07-31 17:34:28.311638
141	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d60fe522-c24a-4b6b-853b-189b5e8ff74f	2015-07-31 17:34:28.330641
142	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 1 MUTATIONS\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	75b8d7fc-e339-4646-a344-d7016da8abe8	2015-07-31 17:34:28.351594
145	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: EXON 4 MUTATIONS\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	384cd343-aba1-44c6-97d0-271ef47f825f	2015-07-31 17:34:28.40749
148	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	13ffe81d-e6f8-46fe-8ea5-8d48193a8c28	2015-07-31 17:34:28.467252
149	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f69b4023-0c39-438d-a3fe-1e8e7487e1b9	2015-07-31 17:34:28.486256
150	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	44f1b853-f441-43e1-bd69-1ebfd7137cd7	2015-07-31 17:34:28.50862
151	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: V536E\ndescription: ''\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4e7f999f-6bea-4f57-9b44-73bae20f5f66	2015-07-31 17:34:28.537008
152	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	030b749f-fca1-4285-a4a0-e850972fbb47	2015-07-31 17:34:28.556675
153	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c84ffcf3-53fb-451d-b703-c4b287229cbe	2015-07-31 17:34:28.584677
154	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f54212fc-fe47-4ee0-9c24-b693764f3b4e	2015-07-31 17:34:28.604528
155	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f322526d-a466-40b3-a73f-abd9afc14e98	2015-07-31 17:34:28.643832
156	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ed9c6afb-fb8b-4ebe-a65e-51b78e710bb1	2015-07-31 17:34:28.669544
157	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	1f5382d1-b3dc-453b-8ac3-88ec1c8f8245	2015-07-31 17:34:28.690947
169	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2325df8c-0179-4fd1-a809-960a8298db9e	2015-07-31 17:34:28.989402
158	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a6dc9873-2201-4f76-9844-603642395d81	2015-07-31 17:34:28.71437
159	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 20 MUTATIONS\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d5336f8e-d6ff-4df6-ae5a-77e24c870364	2015-07-31 17:34:28.73633
160	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: EXON 9 MUTATIONS\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	3f456820-b38a-46ca-9906-b16deb0caa0b	2015-07-31 17:34:28.756309
161	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c05a1b0b-2c86-4802-ae3f-92605b5b08a4	2015-07-31 17:34:28.77505
162	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d31c9aeb-ab9e-4eaa-951a-cd8a7855784e	2015-07-31 17:34:28.808848
163	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	af84737e-b59d-4a1e-a14c-c0f47610db0b	2015-07-31 17:34:28.843952
164	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 41\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	68cc4886-8ae3-4c37-990e-2bbe48f120d7	2015-07-31 17:34:28.872149
165	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	683e66ae-ec96-46ca-977f-18b65361c6e0	2015-07-31 17:34:28.897568
166	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d1b117cb-f9fc-4645-98fc-41e8d932418e	2015-07-31 17:34:28.922087
167	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ad122882-41ec-4d80-b666-c42a13215880	2015-07-31 17:34:28.94559
702	73	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55594258'\nstop:\n- \n- '55594258'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	894d674a-887f-4d1e-9337-f766b6737e6f	2015-07-31 17:37:42.924773
170	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5d8e684c-b0bf-4dd5-899c-28bd5510552b	2015-07-31 17:34:29.013191
171	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	19fd99d8-074e-433d-9d3c-a5c4d016cbf7	2015-07-31 17:34:29.036325
172	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c1d715a7-e052-486b-8e99-471d83ae76f5	2015-07-31 17:34:29.055386
173	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	9948e172-f8ca-4f48-9cd3-5985a2c5c1c0	2015-07-31 17:34:29.076766
174	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4f4004e6-6638-441f-806c-3903e702504a	2015-07-31 17:34:29.099235
175	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	c7b3b8f1-38dc-40cb-9b0b-0eb07929673e	2015-07-31 17:34:29.119177
176	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	769b6ed3-0d2a-44eb-a484-e15a5b145458	2015-07-31 17:34:29.139803
177	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0d0fe266-ce85-4d9b-85b5-2557efef2e13	2015-07-31 17:34:29.160989
178	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	8d091dd4-dc5f-4b9a-8a4f-9759843399e3	2015-07-31 17:34:29.184988
179	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7cd3370a-1de6-45e7-ab7d-707014cc2d11	2015-07-31 17:34:29.204346
180	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d4da7fc1-3bf8-46ff-9c77-6c1f9e14a04c	2015-07-31 17:34:29.223318
181	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	e6f25c9e-6c42-4e1c-b43d-502adb81c29e	2015-07-31 17:34:29.26268
182	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 48\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	ea378b47-343c-48cc-a234-03ba81d20122	2015-07-31 17:34:29.282784
183	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 7 MUTATIONS\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	893dfd3d-7c6d-49dc-95d9-b4595c52698a	2015-07-31 17:34:29.305079
184	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 49\nname: EXON 9 MUTATIONS\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0fcc58b3-9192-4342-8d8f-b491588eb1fa	2015-07-31 17:34:29.327073
185	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: LOSS-OF-FUNCTION\ndescription: BRCA1 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5b4b63de-801a-4b62-84fc-d95efa49484f	2015-07-31 17:34:29.347029
186	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: LOSS-OF-FUNCTION\ndescription: BRCA2 loss of function mutations have been shown to increase risk of\n  breast and ovarian cancer in those carrying the allele in their germline. Treating\n  BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant\n  response.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	81d29ec1-b953-4716-9088-6d8c33f7b022	2015-07-31 17:34:29.36665
187	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: EXON 19 DELETION\ndescription: Deletions within exon 19 of EGFR are most common in lung cancer. These\n  deletions, in non-small cell lung cancer, have been shown to be sensitive to the\n  EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest\n  that this event is a good prognostic marker in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	5e5dd359-b039-4d52-a133-cfb0981613d0	2015-07-31 17:34:29.387202
188	134	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: G719S\ndescription: While not as recurrent as the L858R mutation, EGFR G719S has also been\n  shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity\n  to the tyrosine kinase inhibitors gefitinib and erlotinib.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	466c1cbf-9f8e-4ab4-baab-0952aa0c6679	2015-07-31 17:34:29.408524
189	135	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: D1643H\ndescription: Activating mutations in NOTCH1, including D1643H, have been shown to\n  be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	1d64433e-bdd4-4ffa-a2a9-e38209283de0	2015-07-31 17:34:29.430663
190	136	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: R2328W\ndescription: Activating mutations in NOTCH1, including R2328W, have been shown to\n  be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	7d1f4ec1-675f-4e3c-a779-8483a2dbd5bd	2015-07-31 17:34:29.449737
191	137	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 50\nname: V2444FS\ndescription: Activating mutations in NOTCH1, including a frameshift insertion at V2444,\n  have been shown to be poor prognostic markers in lung cancer.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	9fd7e917-c471-4017-a9b4-429633c677e7	2015-07-31 17:34:29.470518
193	139	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: L63V\ndescription: Activating mutations in DDR2, including L63V, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	a4d9978f-ab5e-41ab-aae5-69a2339acc91	2015-07-31 17:34:29.508346
194	140	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: L239R\ndescription: Activating mutations in DDR2, including L239R, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	f3021412-11d1-4764-88ec-30eed1064a40	2015-07-31 17:34:29.529497
195	141	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G253C\ndescription: Activating mutations in DDR2, including G253C, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	43096f46-eebc-4580-a8c2-64ef32157118	2015-07-31 17:34:29.54748
196	142	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G505S\ndescription: Activating mutations in DDR2, including G505S, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	2c776fb4-2d7b-4eff-9f4c-14103043e696	2015-07-31 17:34:29.568026
197	143	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: I638F\ndescription: DDR2 I638F has been shown to be a loss of function mutation, but also\n  confers sensitivity to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	dd542f84-6cf5-4e1f-8382-db0b6ecb11e2	2015-07-31 17:34:29.585763
198	144	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: G774V\ndescription: Activating mutations in DDR2, including G774V, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	b1675ff9-84d9-409d-918a-f682b5dad302	2015-07-31 17:34:29.604085
199	145	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 51\nname: S768R\ndescription: Activating mutations in DDR2, including S768R, has been shown to be sensitive\n  to dasatinib in cell lines.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	67950f3f-d36b-45f2-8a4a-1a73efb9de90	2015-07-31 17:34:29.628677
200	146	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 52\nname: AMPLIFICATION\ndescription: MET amplification, like EGFR T790M, has been shown to be capable of driving\n  acquired resistance to dacomitinib in patients with lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	15075b5b-8185-4d42-868a-d786cff5591d	2015-07-31 17:34:29.647895
201	147	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12V\ndescription: KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	674ca328-514a-4c7f-8af4-87f72f79d0bc	2015-07-31 17:34:29.671122
202	148	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: G12A\ndescription: KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired\n  resistance to tyrosine kinase inhibitors in lung adenocarcinoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	4e5f9d5d-72da-4e49-9e30-c28fade39787	2015-07-31 17:34:29.690705
203	149	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 54\nname: EWS-FLI1\ndescription: EWS-FLI1 fusions have been found to be extremely common in Ewing's sarcomas.\n  The presence of the fusion is also thought to be a positive prognostic marker of\n  survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	d6c246e8-45a2-4113-883a-4cc51b6b6802	2015-07-31 17:34:29.709269
204	150	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: EWS-FLI1\ndescription: EWS-FLI1 fusions have been found to be extremely common in Ewing's sarcomas.\n  The presence of the fusion is also thought to be a positive prognostic marker of\n  survival.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	49a6ad70-b674-46e2-ab3e-d81d6ee78b2d	2015-07-31 17:34:29.73062
205	151	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 53\nname: TYPE 1 EWS-FLI1\ndescription: The most common EWS-FLI1 fusion (type 1) links EWS exon 7 to FLI1 exon\n  6. This fusion product is found to be associated with better outcome than other\n  EWS-FLI1 fusions in Ewing's Sarcoma.\ndeleted: false\ndeleted_at: \ngenome_build: \nchromosome: \nstart: \nstop: \nreference_bases: \nvariant_bases: \nrepresentative_transcript: \nchromosome2: \nstart2: \nstop2: \nreference_build: \n	1	\N	\N	0501ec5b-146f-4981-b9d2-21d7ee6d79a1	2015-07-31 17:34:29.752104
703	78	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398285'\nstop:\n- \n- '25398285'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	21563901-0ae3-4067-a6d3-f0c9bd589dce	2015-07-31 17:37:42.942136
704	79	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398284'\nstop:\n- \n- '25398284'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	45f19808-d341-4193-96c4-74dc9640cbe1	2015-07-31 17:37:42.957672
245	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In one patient with lung adenocarcinoma who had a S214C mutation, the\n  use of sorafenib has led to more than 5 years of survival and near remission.\ndisease_id: 12\nsource_id: 75\nvariant_id: 10\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	85ffaf20-dbb4-486d-bcbf-c3b4c2578e3a	2015-07-31 17:35:37.283313
246	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	281284d6-e3ad-44e6-8c44-b05431c1f49f	2015-07-31 17:35:37.343513
247	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A mutations (60% of which\n  were R882) were older, had higher white blood cell counts and had higher platelet\n  counts than patients wild-type for DNMT3A\ndisease_id: 13\nsource_id: 77\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0378ba6d-3d14-41fb-8357-669df0228f91	2015-07-31 17:35:37.384562
248	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Daunorubicin treatment resulted in similar overall survival and disease-free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0829b34e-bff3-4d76-b48f-deef6278f56d	2015-07-31 17:35:37.417484
249	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Idarubicin increases overall and disease free survival in de novo AML\n  patients with DNMT3A R882 mutations compared to those who do not harbor this mutation.\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	49c00913-2e4d-4028-bf76-547e8e810766	2015-07-31 17:35:37.457865
250	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no difference in the complete remission rate of de novo AML\n  patients with DNMT3A mutation compared to those who are wild type for DNMT3A.\ndisease_id: 13\nsource_id: 76\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ebf055d2-5463-4356-b456-48a69ae94a87	2015-07-31 17:35:37.490227
251	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: De novo AML patients with DNMT3A D882 mutation showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\ndisease_id: 13\nsource_id: 78\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	be6fe14b-8e76-4c8b-82cc-627caaa12ddf	2015-07-31 17:35:37.526716
252	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL\n  or CLL).\ndisease_id: 14\nsource_id: 79\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	15e6b3e4-fd66-4669-9798-74249c46028d	2015-07-31 17:35:37.579646
253	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\ndisease_id: 15\nsource_id: 79\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	28f288a9-776a-4de5-9a7d-2938292bfca4	2015-07-31 17:35:37.614863
254	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)\ndisease_id: 15\nsource_id: 79\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	07e5a5bd-00dc-4fd3-a092-3591e31d4df8	2015-07-31 17:35:37.644064
255	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads\n  to reduction in the percentage of cells harboring JAK2 V617F.\ndisease_id: 16\nsource_id: 80\nvariant_id: 64\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 9:5073770-5073770 (G->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	15a1163b-9089-475c-9e22-5260d86a54fa	2015-07-31 17:35:37.686138
256	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wild-type KIT\ndisease_id: 17\nsource_id: 81\nvariant_id: 65\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55599321-55599321 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d2c6f3c6-996c-4ee5-b73c-2bbf2a5c09ad	2015-07-31 17:35:37.733881
257	4	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: KIT Exon 17\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	481d27cd-de37-4b87-a8a6-48b3cc56cd98	2015-07-31 17:35:37.790958
258	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\ndisease_id: 18\nsource_id: 82\nvariant_id: 82\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:66729162-66729162 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f813fe73-523d-4567-8cca-74f1722a8fa3	2015-07-31 17:35:37.838193
259	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by\n  100 fold.\ndisease_id: 18\nsource_id: 83\nvariant_id: 83\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:66727451-66727451 (A->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	96b42bb8-c909-4638-91ab-f98e99a130b9	2015-07-31 17:35:37.88177
260	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 19\nsource_id: 84\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5ff0e59f-167c-4dc0-9d43-049fb23841df	2015-07-31 17:35:37.922622
261	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 19\nsource_id: 84\nvariant_id: 90\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6b9905e3-6b97-4a50-b605-9c6996acb3a0	2015-07-31 17:35:37.952012
262	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis than patients with wildtype NRAS.\ndisease_id: 19\nsource_id: 84\nvariant_id: 91\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	521a125a-0e79-4e09-89ad-c2cbee52ab09	2015-07-31 17:35:37.990715
263	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\ndisease_id: 18\nsource_id: 85\nvariant_id: 93\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115258744-115258744 (C->T)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1bc97a32-59f3-442b-b922-12f617007978	2015-07-31 17:35:38.032358
264	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma associated with NRAS Q61 mutation was more often associated\n  with those at the extremity than those at the trunk\ndisease_id: 18\nsource_id: 86\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	016846a8-75de-47fe-af0b-7465f0555196	2015-07-31 17:35:38.089114
265	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\ndisease_id: 18\nsource_id: 87\nvariant_id: 94\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e9ef2685-674d-4d90-b43a-1f97fc4b75a6	2015-07-31 17:35:38.125577
266	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide\n  resulted in disease free survival of 14 months\ndisease_id: 18\nsource_id: 88\nvariant_id: 95\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115256528-115256529 (TT->CA)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9edfeb3c-c594-4a27-8485-327b48849a91	2015-07-31 17:35:38.168315
267	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a melanoma patient with Q61R mutation, treatment with temozolomide\n  resulted in overall survival of 16 months\ndisease_id: 18\nsource_id: 88\nvariant_id: 96\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 1:115256528-115256529 (TT->CC)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	caf49fc7-143c-4326-9ff8-3d9a5c053900	2015-07-31 17:35:38.205529
705	81	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '12'\nstart:\n- \n- '25398281'\nstop:\n- \n- '25398281'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	51f31574-e956-4d1b-b8c4-a1ed11933f67	2015-07-31 17:37:42.975679
268	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST tumors harboring PDGFRA D842V mutation are more likely to be benign\n  than malignant.\ndisease_id: 20\nsource_id: 89\nvariant_id: 99\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a97102a8-2007-4440-8cfd-faea6f13da56	2015-07-31 17:35:38.254086
269	5	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Imatinib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	33c77e32-e82d-476c-b0dc-030f621175dd	2015-07-31 17:35:38.272767
270	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST cancer with D842V mutation is resistant to imatinib.\ndisease_id: 20\nsource_id: 90\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	17b79891-4764-46d9-8ed9-4a0a5dfa06f5	2015-07-31 17:35:38.313864
271	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\ndisease_id: 20\nsource_id: 91\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	204ef928-7681-4422-93fc-1f0ee35b2d15	2015-07-31 17:35:38.362554
272	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation\n  were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK\n  without this mutation.\ndisease_id: 21\nsource_id: 92\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8455b333-39c7-4eca-9d76-5b329a931e4f	2015-07-31 17:35:38.41412
273	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation\n  were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK\n  without this mutation.\ndisease_id: 12\nsource_id: 92\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fde1074d-f3e7-4b0f-b3f3-f224d63d516b	2015-07-31 17:35:38.447711
274	6	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Crizotinib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	ec4c6f53-f0bf-4d56-93d4-269022f44606	2015-07-31 17:35:38.46592
275	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant\n  in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\ndisease_id: 22\nsource_id: 93\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	24d28e11-2274-4195-8646-bbe1e1f9baef	2015-07-31 17:35:38.502172
276	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: High levels of crizotinib can overcome drug resistance of Ba/F3 cells\n  expressing the EML4-ALK fusion containing the F1174L mutation.\ndisease_id: 22\nsource_id: 94\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	37d98150-fdda-4e6c-8bbf-e83df1fa660c	2015-07-31 17:35:38.541931
277	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however,\n  cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing\n  F1174L.\ndisease_id: 22\nsource_id: 95\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b177a51f-f8df-40e2-8a71-2cfa7d419018	2015-07-31 17:35:38.588169
278	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to\n  TAE684.\ndisease_id: 22\nsource_id: 95\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f90d9e7b-9bff-4c8f-90d6-14a333108b19	2015-07-31 17:35:38.635659
279	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar\n  colony formation of AALE cells expressing ARAF S214C in vitro\ndisease_id: 12\nsource_id: 75\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c341ec59-c7a3-412b-8a94-d3c026c2d2cb	2015-07-31 17:35:38.670199
280	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar\n  colony formation of AALE cells expressing ARAF S214C in vitro\ndisease_id: 12\nsource_id: 75\nvariant_id: 10\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: X:47426121-47426121 (C->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b3d08427-a107-4b12-9f82-3c7d915c5fde	2015-07-31 17:35:38.698874
281	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF mutations are associated with melanoma arising in non-chronic sun\n  damaged skin and with superficial spreading melanoma\ndisease_id: 18\nsource_id: 96\nvariant_id: 17\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	556d99c1-cf63-45d3-acdb-219198b50772	2015-07-31 17:35:38.742987
282	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of young AML patients (18-60 years old), DNMT3A R882\n  mutations were associated with reduced relapse free survival in the entire cohort\n  as well as the subset of patients with cytogenetically normal AML\ndisease_id: 13\nsource_id: 97\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a89ad6ce-b72e-4ee6-905b-1e9ca2ea96c0	2015-07-31 17:35:38.788211
706	82	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '66729162'\nstop:\n- \n- '66729162'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	e741615d-6b5a-4e5f-8968-9d427e344baa	2015-07-31 17:37:43.005334
283	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations were associated with older age, higher white blood\n  cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of\n  cytogenetically normal AML patients\ndisease_id: 13\nsource_id: 98\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	04aeb705-656c-4c6b-a09e-8730f1428bec	2015-07-31 17:35:38.823652
284	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation\n  (64.5% of which were R882) had no prognostic value on overall, relapse free and\n  event free survival.\ndisease_id: 13\nsource_id: 97\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0de4302c-a208-42df-8e32-e778988cf6fb	2015-07-31 17:35:38.852488
285	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a large cohort of cytogenetically normal AML patients (18-60 years\n  old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value\n  for overall, relapse free and event free survival\ndisease_id: 13\nsource_id: 97\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	969a3c60-8eec-4bad-a6e1-ed6808cb2338	2015-07-31 17:35:38.88131
286	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status (60% of\n  which were R882) was not predictive of overall and relapse free survival in patients\n  with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 13\nsource_id: 77\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	13c879a1-4392-4da2-aab2-3884a5c08e6a	2015-07-31 17:35:38.917374
287	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with DNMT3A mutations (59% of which were R882) showed worse\n  survival (event-free and overall) outcome than those without DNMT3A mutation.\ndisease_id: 13\nsource_id: 78\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b058b4d4-2b17-4cb0-b271-8a7227f0051a	2015-07-31 17:35:38.947433
288	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutation was associated with reduced relapse free and overall\n  survival in ELN-unfavorable, cytogenetically normal AMLs\ndisease_id: 13\nsource_id: 97\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	52d8ef07-d5b6-4738-9817-b149e21cfbbd	2015-07-31 17:35:38.979326
289	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations\n  (including R882) were associated with worse overall survival compared to those without\n  DNMT3A mutation.\ndisease_id: 13\nsource_id: 78\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7ed82343-427f-4822-87bb-9d1f2b3ac470	2015-07-31 17:35:39.00969
290	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\ndisease_id: 13\nsource_id: 98\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	87d2cdfa-4b10-4c0d-8bb2-2226686ce840	2015-07-31 17:35:39.04009
291	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In older cytogenetically normal AML patients (>59 years), DNMT3A R882\n  mutation is prognostic for shorter disease free survival and overall survival compared\n  to patients without the mutation.\ndisease_id: 13\nsource_id: 99\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	75274942-39a0-481b-b48b-154bfe1e7b70	2015-07-31 17:35:39.083638
292	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutations were associated\n  with significantly reduced overall survival and relapse free survival in patients\n  wildtype for NPM1 and FLT3\ndisease_id: 13\nsource_id: 77\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	80c3ba13-871a-4dff-8216-9c1ca92370a1	2015-07-31 17:35:39.113234
293	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\ndisease_id: 13\nsource_id: 99\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	62038613-7f31-431e-8bd7-fc5aed0bfe2a	2015-07-31 17:35:39.145893
294	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status was not\n  predictive of overall and relapse free survival in patients with NPM1 mutations\n  and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 13\nsource_id: 77\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dba89654-fb37-4ce3-baa3-a5dc14641391	2015-07-31 17:35:39.190035
295	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations\n  were associated with worse overall survival compared to those without DNMT3A mutation.\ndisease_id: 13\nsource_id: 78\nvariant_id: 55\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1a120f74-de26-4850-ba5c-749586531193	2015-07-31 17:35:39.224487
707	83	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '66727451'\nstop:\n- \n- '66727451'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- C\n	2	\N	\N	b900a759-c619-4313-a74e-ab43c1922da6	2015-07-31 17:37:43.021509
296	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no significant association between wildtype KIT or KIT mutations\n  in exon 9 or 11 in survival among GIST patients.\ndisease_id: 23\nsource_id: 100\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f3e464ba-fbe8-487b-b075-59536826a057	2015-07-31 17:35:39.268889
297	7	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: KIT Exon 11\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	b2015507-aa3d-448a-8580-5e37a885f189	2015-07-31 17:35:39.287195
298	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 18\nsource_id: 101\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	af9090d6-7fc1-4f9c-b0e7-abe3e4c19f5c	2015-07-31 17:35:39.328234
299	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 18\nsource_id: 101\nvariant_id: 68\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6a7d23e4-d0c1-4bfd-bd57-38d8636e52e3	2015-07-31 17:35:39.366702
300	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: GIST patients with exon 14 KIT mutations had reduced overall survival\n  compared to patients wiltype for KIT\ndisease_id: 23\nsource_id: 100\nvariant_id: 69\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6c2a6e8e-34d7-49f5-82cc-4d0b4a5e8117	2015-07-31 17:35:39.396037
301	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with KIT mutation but not KIT amplification showed\n  response to imatinib treatment in a small cohort of patients.\ndisease_id: 18\nsource_id: 101\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	99ecee96-2d06-42d5-9fc9-872b96d168bf	2015-07-31 17:35:39.428876
302	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no significant association between wildtype KIT or KIT mutations\n  in exon 9 or 11 in survival among GIST patients.\ndisease_id: 23\nsource_id: 100\nvariant_id: 71\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c6fcf2ea-0842-482d-964e-c6688fe439e1	2015-07-31 17:35:39.466794
303	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The melanoma cell line WM3211, which harbors the L576P mutation, is not\n  sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the\n  L576P mutation alters the conformation of KIT and reduces the binding affinity of\n  imatinib.\ndisease_id: 18\nsource_id: 102\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	f948365f-49a2-4805-b39e-eb8f2c6f1660	2015-07-31 17:35:39.506096
304	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\ndisease_id: 18\nsource_id: 102\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b28a32a7-3135-441b-932d-96169ef5faf8	2015-07-31 17:35:39.554046
305	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC, Exon 2 KRAS mutations were associated with resistance to the\n  EGFR kinase inhibitors gefitinib and erlotinib\ndisease_id: 12\nsource_id: 103\nvariant_id: 75\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	347035ba-0b78-430b-a68d-8dc39d91c038	2015-07-31 17:35:39.599623
306	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In young AML patients (<60 years old), DNMT3A mutation status was not\n  predictive of overall and relapse free survival in patients with NPM1 mutations\n  and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\ndisease_id: 13\nsource_id: 77\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	68545c61-0e05-4208-82af-c36c895625eb	2015-07-31 17:35:39.63682
307	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage IV melanoma, NRAS mutation was associated with\n  reduced median survival compared to patients with wildtype NRAS\ndisease_id: 18\nsource_id: 104\nvariant_id: 88\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5ea96883-2442-4497-aa67-b27ff040e253	2015-07-31 17:35:39.674788
308	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of NRAS mutation in AML patients does not impact diease\n  prognosis (resistant disease, disease-free survival, complete remission rate, relapse\n  rate, induction death)\ndisease_id: 13\nsource_id: 105\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5e212f49-7964-45a3-9f61-f5b30b6008e0	2015-07-31 17:35:39.726653
309	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage IV melanoma, NRAS mutation was associated with\n  reduced median survival compared to patients with wildtype NRAS\ndisease_id: 18\nsource_id: 104\nvariant_id: 89\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1bd6d1d4-fddf-4156-bbd0-26f6a7d6d042	2015-07-31 17:35:39.757708
310	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NRAS mutation status does not impact overall, event-free or disease-free\n  survival in patients with AML\ndisease_id: 13\nsource_id: 106\nvariant_id: 92\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e88e99b4-f1d0-4e86-a473-4cdf592d1604	2015-07-31 17:35:39.79956
311	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring NRAS\n  mutation (primarily Q61) have a significantly lower response rate to cetuximab than\n  patients wildtype for NRAS.\ndisease_id: 19\nsource_id: 107\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fb63a62b-f7d9-4c66-a010-35c6efeea1cd	2015-07-31 17:35:39.836047
312	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 20\nsource_id: 108\nvariant_id: 98\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152092-55152093 (GA->AT)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8fd7d7cb-eccf-4576-a905-9bcd7a0d7c8c	2015-07-31 17:35:39.878764
313	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 20\nsource_id: 108\nvariant_id: 99\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152093-55152093 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d8ec6742-f7f7-48a7-8704-3d8c3fd65bbe	2015-07-31 17:35:39.922684
314	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\ndisease_id: 20\nsource_id: 108\nvariant_id: 100\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55152092-55152092 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d982acb6-cc97-4e32-ac1a-c5b0a6232ec3	2015-07-31 17:35:39.957572
315	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\ndisease_id: 20\nsource_id: 108\nvariant_id: 101\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a07f3973-2113-48a5-bd2f-31ad9439501a	2015-07-31 17:35:39.99637
316	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\ndisease_id: 20\nsource_id: 108\nvariant_id: 102\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e7b62bf4-5f33-42b9-9330-ae80adf04b26	2015-07-31 17:35:40.035786
317	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A single patient with BRAF V600E/V600M bi-allelic mutation responded\n  to the V600E drug dabrafenib.\ndisease_id: 18\nsource_id: 109\nvariant_id: 13\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a044cc65-ad80-4b86-baaf-38da4217f825	2015-07-31 17:35:40.081173
318	8	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Other V600's\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	ab413270-43a8-4a7b-b34e-cc13cbaba3da	2015-07-31 17:35:40.100994
319	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Medullary thyroid cancer cells with RET C634W mutation are insensitive\n  to motesanib, compared to wild-type RET.\ndisease_id: 24\nsource_id: 110\nvariant_id: 112\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43609950-43609950 (C->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7b9db641-1e60-4399-b40d-9d22eb7a6ef4	2015-07-31 17:35:40.138979
320	9	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: Motesanib Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	dd96991d-c7ea-4a46-9725-8677358cfb4f	2015-07-31 17:35:40.159967
321	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with medullary carcinoma, the presence of RET M918T mutation\n  is associated with increased probability of lymph node metastases.\ndisease_id: 24\nsource_id: 111\nvariant_id: 113\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5ba09856-8e46-4b3d-ba32-eaefc02193a9	2015-07-31 17:35:40.196691
322	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Compared to those who harbor a wild type RET, patients with RET M918T\n  mutation develop larger and more aggressive medullary thyroid cancer.\ndisease_id: 24\nsource_id: 112\nvariant_id: 113\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9c574c10-cb93-4e73-8929-03d1355f38ec	2015-07-31 17:35:40.253805
323	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Medullary thyroid cancer cells with RET M918T mutation are insensitive\n  to motesanib, compared to wild-type RET.\ndisease_id: 24\nsource_id: 110\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6f5f9394-5e87-429d-968f-5c92e9e9d006	2015-07-31 17:35:40.286744
324	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led\n  to strong repression of tyroid cancer cell growth.\ndisease_id: 24\nsource_id: 113\nvariant_id: 113\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:43617416-43617416 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	59da1fdb-526a-4530-9594-06d7d6079fcd	2015-07-31 17:35:40.336485
325	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\ndisease_id: 19\nsource_id: 114\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6ddc9d1b-29e1-47ff-ae9f-93a3b59a4dec	2015-07-31 17:35:40.378827
326	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Melanoma patients with BRAF V600E/K mutations had longer progression-free\n  survival, increased tumor regression,  and increased duration of response to combined\n  dabrafenib and trametinib treatment compared to dabrafenib alone\ndisease_id: 18\nsource_id: 115\nvariant_id: 17\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	94e74f62-2fd8-42b5-b6fd-4c871c70286e	2015-07-31 17:35:40.423761
327	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 18\nsource_id: 116\nvariant_id: 11\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453135-140453136 (CA->AT)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6d9e7ac6-3ffc-4eee-82ad-5b8bfec706ad	2015-07-31 17:35:40.464902
328	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: V600E is associated with adverse pathological features of colorectal\n  cancer. This can be concluded as a marker of poor prognosis.\ndisease_id: 19\nsource_id: 117\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1cc48d0a-ca54-41f5-9157-49ae80007329	2015-07-31 17:35:40.513025
329	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\ndisease_id: 25\nsource_id: 118\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1ddea5b8-1371-4af6-80a0-63b80660985f	2015-07-31 17:35:40.55975
330	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\ndisease_id: 25\nsource_id: 119\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	86939d79-cffb-41ba-923c-110b5b0def8c	2015-07-31 17:35:40.600124
331	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\ndisease_id: 25\nsource_id: 120\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f000b81d-e41f-4011-9e16-ea147dea1853	2015-07-31 17:35:40.647208
332	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\ndisease_id: 18\nsource_id: 121\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	d0828d67-9366-4b64-993a-408cbbe5bf75	2015-07-31 17:35:40.688137
333	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\ndisease_id: 18\nsource_id: 122\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	7881bad0-c2e5-4dfc-8360-77a3f1080fd6	2015-07-31 17:35:40.734472
334	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Unlike other studies that suggest a poorer outcome, BRAF mutation in\n  this study was not correlated with poorer prognosis in papillary thyroid cancer.\ndisease_id: 25\nsource_id: 123\nvariant_id: 12\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fa91b31f-b844-4bf7-8aaa-17b3ffcfb750	2015-07-31 17:35:40.787949
335	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is correlated with shorter disease-free and overall Survival\n  in a Spanish cohort of melanoma patients.\ndisease_id: 18\nsource_id: 124\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3ad1007e-a4b1-4d24-990f-99370ea0d97a	2015-07-31 17:35:40.829331
336	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF mutation correlated with poor prognosis in papillary thyroid cancer\n  in both older (>65 yo) and younger (<65 yo) cohorts.\ndisease_id: 25\nsource_id: 118\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eb834817-aa75-403b-8f69-72b497d7eb0c	2015-07-31 17:35:40.865382
337	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer\n  in a study of 187 patients with PTC and other thyroid diseases.\ndisease_id: 25\nsource_id: 125\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	51268b24-f059-4790-83ef-142546ca7817	2015-07-31 17:35:40.911002
338	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: V600E is correlated with disease recurrence in both age cohorts (>65\n  and <65 yo).\ndisease_id: 25\nsource_id: 118\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0d162e16-025e-44b9-b144-9d4d7973e026	2015-07-31 17:35:40.942019
339	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation\n  of RAS-GTP but does not show resistance to MEK inhibitors.\ndisease_id: 18\nsource_id: 126\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1a448850-b739-4cf1-b7a0-353ea6dd4d2f	2015-07-31 17:35:40.978953
708	87	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '5'\nstart:\n- \n- '170837546'\nstop:\n- \n- '170837547'\nreference_bases:\n- \n- '0'\nvariant_bases:\n- \n- TCAG\n	2	\N	\N	a9be182b-7c9e-44e0-8f7b-3854dbe82ac5	2015-07-31 17:37:43.038295
340	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation\n  unless BRAF inhibitor such as Sorafenib is introduced.\ndisease_id: 26\nsource_id: 127\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	45086610-b3ac-4169-a28b-57c351eabf12	2015-07-31 17:35:41.033795
341	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation\n  of RAS-GTP and resistance to RAF inhibitors.\ndisease_id: 18\nsource_id: 126\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	82ebaca0-e489-474f-83cb-8b400505893a	2015-07-31 17:35:41.068189
342	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely\n  injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited\n  tumor growth and reduced cellular proliferation\ndisease_id: 26\nsource_id: 128\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	f79e4dd0-0ea3-48cb-999e-e32b27c967e6	2015-07-31 17:35:41.107783
343	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\ndisease_id: 26\nsource_id: 129\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	9b613c9e-81c0-4943-a995-8d12702edc80	2015-07-31 17:35:41.156675
344	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\ndisease_id: 19\nsource_id: 130\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	d0f02eaa-8561-42f5-b3df-22984f2380e4	2015-07-31 17:35:41.203074
345	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\ndisease_id: 19\nsource_id: 130\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a51bd536-c712-4b03-b4fd-eba03944c742	2015-07-31 17:35:41.249663
346	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\ndisease_id: 12\nsource_id: 131\nvariant_id: 12\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b2b0f62e-ee11-4303-8c51-688066c56df2	2015-07-31 17:35:41.294078
347	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned\n  to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown\n  to be amplification of the BRAF V600E gene.\ndisease_id: 19\nsource_id: 132\nvariant_id: 14\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d53df070-5168-4791-a150-c52dc0d9c6d6	2015-07-31 17:35:41.352522
348	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 18\nsource_id: 116\nvariant_id: 15\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453137-140453137 (C->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	92b27860-e537-4a60-9378-497c9685ec20	2015-07-31 17:35:41.381979
349	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with other BRAF V600 mutations also respond well to the V600E\n  drug dabrafenib.\ndisease_id: 18\nsource_id: 116\nvariant_id: 16\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453137 (AC->CT)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	81e306d6-50c4-4c3c-abea-eee2b866084c	2015-07-31 17:35:41.425901
350	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\ndisease_id: 19\nsource_id: 133\nvariant_id: 75\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	48315ace-f410-4930-b29c-7414308dd62e	2015-07-31 17:35:41.475582
351	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 19\nsource_id: 133\nvariant_id: 103\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f6acfb85-c724-43d4-8079-867c47f691da	2015-07-31 17:35:41.512768
352	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 19\nsource_id: 133\nvariant_id: 104\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1f592fb9-6fe1-4254-bb57-a4105b05e5d9	2015-07-31 17:35:41.542405
709	93	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115258744'\nstop:\n- \n- '115258744'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	4665a6f1-921f-4725-ae95-8fda9453020a	2015-07-31 17:37:43.061605
353	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation status was not predictive of response to regorafenib\n  treatment in patients that had received standard therapy and progressed within 3\n  months of their last treatment\ndisease_id: 19\nsource_id: 133\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e15c1a14-99eb-416d-b715-32145b0bb7c7	2015-07-31 17:35:41.572256
354	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CH5424802 treatment resulted in significant tumor regression in xenograft\n  models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M\n  mutation\ndisease_id: 12\nsource_id: 93\nvariant_id: 7\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	235fc5f3-0603-4ad6-9644-ca061c705fdd	2015-07-31 17:35:41.601767
355	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\ndisease_id: 22\nsource_id: 134\nvariant_id: 8\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e2e735e4-510c-42b3-abd4-986ddac6b03d	2015-07-31 17:35:41.642174
356	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\ndisease_id: 22\nsource_id: 95\nvariant_id: 8\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29443695-29443695 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	93381bd7-94a4-44f3-8b75-005270f23d49	2015-07-31 17:35:41.674778
357	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation\ndisease_id: 22\nsource_id: 134\nvariant_id: 9\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:29432664-29432664 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	80143715-bc02-4ea8-9498-97a1b5049199	2015-07-31 17:35:41.705989
358	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring BRAF\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype BRAF\ndisease_id: 19\nsource_id: 107\nvariant_id: 12\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:140453136-140453136 (A->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	93cf546b-6189-4742-b81e-61bd02f110d3	2015-07-31 17:35:41.739971
359	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation status had no impact on ATRA treatment response in older\n  (>60) patients with AML\ndisease_id: 13\nsource_id: 135\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b11be7a7-4482-4957-9f4f-bd2fc0722ce5	2015-07-31 17:35:41.778233
360	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation was associated with improved overall survival in older\n  (>60) patients with AML\ndisease_id: 13\nsource_id: 135\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f5ca249c-e3e3-43a8-9ba4-5b3a782319dd	2015-07-31 17:35:41.80879
361	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEBPA mutation was significantly associated with complete remission\ndisease_id: 13\nsource_id: 136\nvariant_id: 29\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a679a633-1cdf-45e3-a998-8fb7dd7556fd	2015-07-31 17:35:41.847776
362	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\ndisease_id: 13\nsource_id: 137\nvariant_id: 28\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	161a9e9d-aed4-4dfb-9cd6-2675e793e018	2015-07-31 17:35:41.904698
363	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\ndisease_id: 13\nsource_id: 137\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	341e9523-087b-45d3-b143-dc37cf41d985	2015-07-31 17:35:41.938743
364	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\ndisease_id: 13\nsource_id: 137\nvariant_id: 28\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9eb9aa1b-56c8-45bc-8268-f9b84bcbd0c3	2015-07-31 17:35:41.967092
365	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Therapy-related AML was less common in patients with DNMT3A mutations\n  (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of\n  younger (18-60) AML patients\ndisease_id: 13\nsource_id: 97\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7d311c42-58eb-428e-a1e0-fc4d0a012094	2015-07-31 17:35:42.00085
366	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\ndisease_id: 13\nsource_id: 78\nvariant_id: 32\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	649dea31-8626-4301-87dc-909f8479a949	2015-07-31 17:35:42.031542
710	95	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115256528'\nstop:\n- \n- '115256529'\nreference_bases:\n- \n- TT\nvariant_bases:\n- \n- CA\n	2	\N	\N	00026da0-23bb-4111-b100-a6f0a8e028e0	2015-07-31 17:37:43.083581
367	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\ndisease_id: 13\nsource_id: 78\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c6f1b6cc-b03f-4a08-9f59-5a7eb266c2f2	2015-07-31 17:35:42.066844
368	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (64.5% R882) were associated with older age, higher\n  white blood cell count and cytogenetically normal AML in a large cohort of younger\n  (18-60) AML patients\ndisease_id: 13\nsource_id: 97\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cd9c0a3d-dabe-4a9a-95f7-5a3997445e5c	2015-07-31 17:35:42.096713
369	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A R882 mutations were associated with cytogenetically normal AML\n  in a large cohort of younger (18-60) AML patients\ndisease_id: 13\nsource_id: 97\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ebb39d54-e539-446e-94cc-6a0a7508707e	2015-07-31 17:35:42.128593
370	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates did not differ between patients with wildtype\n  or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML\ndisease_id: 13\nsource_id: 99\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3f7f3dab-c97d-4bb6-8469-dec19dd5176e	2015-07-31 17:35:42.159039
371	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations were associated with achievement of complete remission\n  in a large cohort of younger (18-60) AML patients\ndisease_id: 13\nsource_id: 97\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1b6c5741-b093-4343-a16f-94d1da851e8f	2015-07-31 17:35:42.188681
372	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rate was not different between young AML patients\n  (<60 years old) with or without DNMT3A mutations (60% of which were R882)\ndisease_id: 13\nsource_id: 77\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	36e72635-0b05-4f22-87a2-cd69f6f3c9f4	2015-07-31 17:35:42.221305
373	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: DNMT3A mutations (62% of which were R882) were associated with reduced\n  disease-free survival in patients with cytogenetically normal AML.\ndisease_id: 13\nsource_id: 99\nvariant_id: 32\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	546b0a90-78c7-41a4-af9c-aa3d75fabff9	2015-07-31 17:35:42.24909
374	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A mutation have shorter\n  overall survival and relapse-free survival than patients with wildtype DNMT3A\ndisease_id: 13\nsource_id: 77\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6630c8b2-897f-45d8-9e9e-61c492490798	2015-07-31 17:35:42.281299
375	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter\n  overall survival and relapse-free survival than patients with wildtype DNMT3A\ndisease_id: 13\nsource_id: 77\nvariant_id: 32\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bea41545-555b-473d-840a-52e960335175	2015-07-31 17:35:42.315999
376	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Following identification of this fusion in a patient, this fusion was\n  expressed in HEK 293T cells leading to oligomerization in the absence of ligand,\n  induced morphologic changes in the cell and increased cellular proliferation consistent\n  with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors\n  an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR\n  inhibitor PD173074.\ndisease_id: 27\nsource_id: 138\nvariant_id: 53\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fa9ba209-d375-4a73-99a6-7a1dbbc8a9db	2015-07-31 17:35:42.357928
377	10	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: FGFR fusions\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e3c969ff-cc64-461e-823a-b4ede82f081f	2015-07-31 17:35:42.381112
378	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Meta-analysis of studies involving cytogentically normal younger (<60)\n  patients showed reduced overall and relapse-free survival for patients with FLT3-ITD\ndisease_id: 13\nsource_id: 139\nvariant_id: 55\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	13588938-56d7-496e-8e32-8e0293743027	2015-07-31 17:35:42.443147
379	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment did not effect overall survival in patients <60 years\n  old with FLT3-ITD mutation\ndisease_id: 13\nsource_id: 140\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	77b3acf9-e9c0-45a0-838e-ad793cbd91e5	2015-07-31 17:35:42.483981
380	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3-ITD mutation without NPM1 was associated with increased relapse\n  risk and reduced overall survival in young adult AML patients (median age 43)\ndisease_id: 13\nsource_id: 141\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	02b0dd64-3bf7-480c-b573-efc57f44156b	2015-07-31 17:35:42.521902
410	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutation status of IDH1 did not change event-free survival of patients\n  with an NPM1 mutation\ndisease_id: 13\nsource_id: 145\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ac29df5d-2391-4397-80c8-ef7426fd071f	2015-07-31 17:35:43.611813
381	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Increasing level of FLT3-ITD mutant correlated with reduced disease-free\n  and overall survival as well as increased relapse rate\ndisease_id: 13\nsource_id: 141\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e71f4807-dee6-45fc-98a0-934dd8521b68	2015-07-31 17:35:42.556025
382	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: FLT3-ITD duplication length did not effect prognosis in patients with\n  an NPM1 mutation\ndisease_id: 13\nsource_id: 142\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	467ad442-8210-4075-a9c3-432ec920fbd4	2015-07-31 17:35:42.591572
383	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Size of the FLT3-ITD mutant duplication had no impact on overall survival\n  or relapse rate\ndisease_id: 13\nsource_id: 141\nvariant_id: 55\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	37352c89-c116-4d21-abde-bcfd984b121a	2015-07-31 17:35:42.630892
384	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced\n  event-free and overall survival in old (>59) and younger patients (<60) with an\n  NPM1 mutation\ndisease_id: 13\nsource_id: 142\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1143efc9-acf2-4543-a402-0e25c291e496	2015-07-31 17:35:42.660721
385	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival\n  compared to patients with wildtype FLT3\ndisease_id: 13\nsource_id: 142\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9fbac05b-a05b-4242-bac6-c4b2d7933191	2015-07-31 17:35:42.689006
386	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable,\n  a higher ratio was associated with reduced event-free survival in patients with\n  an NPM1 mutation\ndisease_id: 13\nsource_id: 142\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9cb9c2a5-5d04-41c4-aa29-07ea8bb8cb11	2015-07-31 17:35:42.722338
387	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to daunorubicin than cells with FLT3-ITD alone.\ndisease_id: 13\nsource_id: 143\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	86c264d2-3ac6-475f-b92b-786c88fb7a91	2015-07-31 17:35:42.760535
388	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells\n  with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\ndisease_id: 13\nsource_id: 143\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f9e24d22-28ce-4059-8953-1bbcccee7034	2015-07-31 17:35:42.79133
389	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Event-free survival was significantly improved in patients with both\n  NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone\n  2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis\n  3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal\n  karyotype patients were analyzed\ndisease_id: 13\nsource_id: 144\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	51f06d2f-a437-4093-bb0d-2d6093aed823	2015-07-31 17:35:42.831791
390	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to daunorubicin than cells with FLT3-ITD alone.\ndisease_id: 13\nsource_id: 143\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8f0eb2b5-95fd-43bd-8753-fce1288b50b5	2015-07-31 17:35:42.860918
391	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have\n  more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells\n  with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\ndisease_id: 13\nsource_id: 143\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	35b4bb69-d70a-403a-a37f-35518e011ad8	2015-07-31 17:35:42.897426
392	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutation status of IDH1 did not change event-free survival of patients\n  with an NPM1 mutation\ndisease_id: 13\nsource_id: 145\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	26f55a7f-00c9-4225-a4e9-53b94600e8b8	2015-07-31 17:35:42.948002
393	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring KRAS\n  mutations (primarily G12/G13) had lower response and disease control rates and shorter\n  progression free and overall survival following cetuximab plus chemotherapy than\n  those with wildtype KRAS\ndisease_id: 19\nsource_id: 107\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f1242126-5237-4cd2-8690-b269538a1ea2	2015-07-31 17:35:42.977398
394	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations were significantly associated with lack of response to\n  cetuximab in patients with advanced colorectal cancer\ndisease_id: 19\nsource_id: 146\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	644fc8be-6fa9-44b7-b202-117130dc7e00	2015-07-31 17:35:43.01597
395	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations were significantly associated with reduced progression-free\n  and overall survival in patients with advanced colorectal cancer\ndisease_id: 19\nsource_id: 146\nvariant_id: 77\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	89479d6f-02d2-4d9f-898c-94a983a3b5ab	2015-07-31 17:35:43.054508
396	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment\n  in isogenic SW48 cells and in a mouse xenograft model\ndisease_id: 26\nsource_id: 147\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5b2f4ec2-697d-4981-a789-c426f9c2625c	2015-07-31 17:35:43.097195
397	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment\n  in isogenic SW48 cells and in a mouse xenograft model\ndisease_id: 26\nsource_id: 147\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9ec2685d-f17b-417b-8740-d3491d03aed1	2015-07-31 17:35:43.126536
398	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\ndisease_id: 28\nsource_id: 148\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ca80e530-2085-4f2c-97b8-eb7f14487294	2015-07-31 17:35:43.173016
399	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML with mutated NPM1 is a provisional entity in WHO classification of\n  acute myeloid leukemia (AML). This mutation should be tested for in clinical trials\n  and is recommended for testing in patients with cytogentically normal AML.\ndisease_id: 13\nsource_id: 149\nvariant_id: 86\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b9d97dbb-3764-4c63-a659-850b75547cb4	2015-07-31 17:35:43.211657
400	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were not associated with the M3 acute myeloid leukemia\n  FAB subtype (0/55).\ndisease_id: 13\nsource_id: 150\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ed4135b3-cd25-41cd-8e5f-5d2ff74f447e	2015-07-31 17:35:43.254918
401	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\ndisease_id: 13\nsource_id: 151\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c104d573-49f7-49e3-9be1-18f5f7c12fc1	2015-07-31 17:35:43.290559
402	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with de novo AML as well as M4 and M5\n  FAB subtypes\ndisease_id: 13\nsource_id: 141\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b3beea69-c491-40b2-ae27-09dd362ac67f	2015-07-31 17:35:43.322135
403	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute\n  myeloid leukemia.\ndisease_id: 13\nsource_id: 151\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ab125dab-4782-4019-b7ba-001a7af7a1aa	2015-07-31 17:35:43.352087
404	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute\n  myeloid leukemia\ndisease_id: 13\nsource_id: 150\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a6dbc655-824b-432e-aad7-c8dababa91f7	2015-07-31 17:35:43.38733
405	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with normal karyotype in older (>60) patients\ndisease_id: 13\nsource_id: 152\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4934340a-c1da-4296-992a-510039e193ce	2015-07-31 17:35:43.42554
406	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment did not effect overall survival in patients <60 years\n  old with NPM1 mutation regardless of FLT3-ITD status\ndisease_id: 13\nsource_id: 140\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	37cdaa7f-5cd0-4e58-9ab6-93481e35b937	2015-07-31 17:35:43.4724
407	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: ATRA treatment improved overall and relapse-free survival in older (>60)\n  patients with mutant NPM1 and without FLT3-ITD\ndisease_id: 13\nsource_id: 152\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	58136adf-bf05-4e51-a6cf-fa8ac00133b6	2015-07-31 17:35:43.50015
408	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Young patients (18-60) with NPM1 mutations had improved overall survival\n  following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis.\n  However, the adjusted p-value for multiple testing is 0.11.\ndisease_id: 13\nsource_id: 153\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4d9d841f-53ff-4287-b35f-fa650b9feb1f	2015-07-31 17:35:43.537123
409	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: For patients with NPM1 mutation that achieved CR following induction\n  therapy, relapse-free survival was improved following treatment with valproic acid\ndisease_id: 13\nsource_id: 154\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	534c1ab6-20da-4c45-a4a6-e58de789eb95	2015-07-31 17:35:43.578636
411	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk\n  and increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\ndisease_id: 13\nsource_id: 141\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	36d388ff-3f13-4c96-b1e9-c4a66e5c021d	2015-07-31 17:35:43.641929
412	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\ndisease_id: 13\nsource_id: 152\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	392b0de8-f2f0-4230-8a6b-4a9d4e6dccd4	2015-07-31 17:35:43.67225
413	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates were higher and both disease-free and overall\n  survival were longer for patients with Exon 12 NPM1 mutations.\ndisease_id: 13\nsource_id: 150\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	31823b2c-efb7-4cc2-b4cf-b601a73d38b8	2015-07-31 17:35:43.706248
414	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Complete remission rates were higher and event-free survival was longer\n  for normal karyotype AML patients with Exon 12 NPM1 mutations.\ndisease_id: 13\nsource_id: 151\nvariant_id: 86\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c7e372a0-167b-414c-8cd4-cf5505716e31	2015-07-31 17:35:43.736543
415	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cytoplasmic localization of NPM in AML patients was an independent prognositic\n  factor associated with complete remission in response to induction therapy in a\n  multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).\ndisease_id: 13\nsource_id: 155\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	43e445de-02e6-4bf2-a726-ba68412857a5	2015-07-31 17:35:43.773834
416	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Event-free survival was significantly improved in patients with both\n  NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone\n  2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis\n  3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal\n  karyotype patients were analyzed\ndisease_id: 13\nsource_id: 144\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d1525919-2a85-4040-89c8-9284f6f64596	2015-07-31 17:35:43.808516
417	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Intermediate risk patients with mutant NPM1 had improved overall survival\n  with the presence of either IDH1 or IDH2 mutations than those wildtype for both\n  IDH1 and IDH2\ndisease_id: 13\nsource_id: 153\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	760548eb-494c-4aa8-92b5-fb63ba6111bb	2015-07-31 17:35:43.840092
418	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had\n  improved overall and relapse-free survival, complete remission rates and remission\n  duration compared to all other combinations of mutants\ndisease_id: 13\nsource_id: 156\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	474e2b48-c1c0-4440-9f32-228b156a5994	2015-07-31 17:35:43.887734
419	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation was associated with higher complete remission rates, lower\n  cumulative incidence of relapse and higher overall survival in intermediate risk\n  AML patients\ndisease_id: 13\nsource_id: 157\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e48bff45-1196-42a8-aa4c-478084eec48b	2015-07-31 17:35:43.925559
420	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\ndisease_id: 13\nsource_id: 158\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	facbeb6b-c468-4367-bef3-205f8f99ce18	2015-07-31 17:35:43.983167
421	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation without FLT3-ITD was significantly associated with complete\n  remission\ndisease_id: 13\nsource_id: 136\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7046c102-c304-42ff-a238-d418e2009035	2015-07-31 17:35:44.014212
422	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with increased complete remission rates\n  as well as longer overall disease free survival in normal karyotype AML patients\n  >59 years old, particularly in those >69 years old\ndisease_id: 13\nsource_id: 159\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4a518355-b7df-41ce-a519-cf64430cfa8f	2015-07-31 17:35:44.052129
423	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation\n  and without FLT3-ITD have increased overall survival, relapse-free survival and\n  response to induction chemotherapy\ndisease_id: 13\nsource_id: 160\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3388b044-2064-48ba-b5ea-4e263493cb4a	2015-07-31 17:35:44.090653
711	96	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '1'\nstart:\n- \n- '115256528'\nstop:\n- \n- '115256529'\nreference_bases:\n- \n- TT\nvariant_bases:\n- \n- CC\n	2	\N	\N	bab3bac7-0047-428f-b58a-7fbc14d2a3fe	2015-07-31 17:37:43.102341
424	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coocurrence of FLT3-ITD mutations significantly reduced overall survival\n  of normal karyotype AML patients\ndisease_id: 13\nsource_id: 150\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b02c50cc-30c6-4312-8e65-b46c4e5954a0	2015-07-31 17:35:44.121002
425	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\ndisease_id: 13\nsource_id: 161\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ba3b7386-1f7e-497c-814c-848639538633	2015-07-31 17:35:44.163968
426	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival\n  compared to patients with wildtype FLT3\ndisease_id: 13\nsource_id: 142\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5f925c2e-cfb4-41e1-b188-974a58c56cbb	2015-07-31 17:35:44.195417
427	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall\n  survival and cumulative incidence of relapse than those with NPM1 mutations alone\n  in intermediate risk AML patients\ndisease_id: 13\nsource_id: 157\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8fc3b018-a8ef-4bb0-8ec4-aad9dff088d1	2015-07-31 17:35:44.229009
428	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: TET2 mutation reduces overall survival of normal karyotype patients with\n  either an NPM1 mutation or NPM1 mutation without FLT3-ITD\ndisease_id: 13\nsource_id: 162\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	aae7a34b-74f6-4375-b1bc-dd4e11c7e51d	2015-07-31 17:35:44.271041
429	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable,\n  a higher ratio was associated with reduced event-free survival in patients with\n  an NPM1 mutation\ndisease_id: 13\nsource_id: 142\nvariant_id: 86\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a9af5dee-5290-48ab-a1db-7240c43fc928	2015-07-31 17:35:44.305452
430	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSC348884 combined with ATRA treatment induces apoptosis in primary AML\n  cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1\n  and FLT3-ITD\ndisease_id: 13\nsource_id: 163\nvariant_id: 86\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	bc5128ab-28a7-4d13-ba52-b620dd4fbdd5	2015-07-31 17:35:44.345299
431	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CD33 and CD123 expression was significantly increased in patients with\n  NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\ndisease_id: 13\nsource_id: 164\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bded9b02-bf5c-4010-8545-6c5813bab7d1	2015-07-31 17:35:44.394109
432	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CD33 expression was significantly increased in patients with NPM1 mutation\n  with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\ndisease_id: 13\nsource_id: 164\nvariant_id: 86\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5ebef54a-3ff0-4e6d-af70-cef5e0e75670	2015-07-31 17:35:44.425848
433	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\ndisease_id: 13\nsource_id: 78\nvariant_id: 87\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	114208c4-fb0f-40ec-b066-714806dbd197	2015-07-31 17:35:44.460443
434	11	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: NPM1 Exon 12\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	fec73d98-2076-43d9-8117-e281b2b9fee5	2015-07-31 17:35:44.480761
435	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\ndisease_id: 13\nsource_id: 78\nvariant_id: 87\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	195b5ddb-5367-42cf-a06e-88b8251d4d37	2015-07-31 17:35:44.524854
436	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\ndisease_id: 13\nsource_id: 163\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8091f29d-3871-4a35-b4cc-cd5d06c2a7e8	2015-07-31 17:35:44.558692
437	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM\n  when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization\n  of NPM in AML patients was associated with good response to induction therapy.\ndisease_id: 13\nsource_id: 155\nvariant_id: 87\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 5:170837546-170837547 (0->TCAG)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	96f16f5f-9718-4ecb-b4e5-5fd42a8eaf11	2015-07-31 17:35:44.594223
480	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Three studies have associated CCND1 expression with poorer survival.\ndisease_id: 12\nsource_id: 178\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cf5245ac-e469-4281-ae42-2b8b87519331	2015-07-31 17:35:46.208567
438	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation\n  status (primarily Q61) was not informative for disease control rate, progression\n  free survival or overall survival following cetuximab plus chemotherapy.\ndisease_id: 19\nsource_id: 107\nvariant_id: 94\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8ce00f81-ab00-43d5-8b64-ec2bbbc672c6	2015-07-31 17:35:44.632955
439	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Chemotherapy-refractory patients with colorectal cancer harboring Exon\n  20 PIK3CA mutations had lower response and disease control rates as well as shorter\n  progression free and overall survival following cetuximab plus chemotherapy than\n  those with wildtype PIK3CA\ndisease_id: 19\nsource_id: 107\nvariant_id: 105\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cbeaa132-674d-47f0-8c7b-279a560d0131	2015-07-31 17:35:44.665425
440	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA\n  mutations (Exon 9) had no significant impact on response rate, disease control rate,\n  progression free surivival or overall survival following cetuximab plus chemotherapy\n  than those with wildtype PIK3CA\ndisease_id: 19\nsource_id: 107\nvariant_id: 106\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	071ecaa6-5bdb-435c-8bf0-08c5ce49b6e9	2015-07-31 17:35:44.695162
441	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Following rapamycin treatment of mice with KrasG12D-induced lung tumors,\n  mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung\n  exhibit increased overall survival, increased apoptosis, reduced proliferation and\n  reduced tumor burden compared to Tsc1 wildtype mice.\ndisease_id: 12\nsource_id: 165\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e2e4bc93-6f9d-4fb3-af7e-db692f998333	2015-07-31 17:35:44.737847
442	12	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: TSC Loss\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	e06cc1d8-4ff2-4c44-92d8-01adaf1dccd1	2015-07-31 17:35:44.758443
443	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity\n  to everolimus than their wild-type counterparts.\ndisease_id: 27\nsource_id: 166\nvariant_id: 125\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8aa36445-dcb0-4ef7-a7b0-32f9cc8bc7be	2015-07-31 17:35:44.795422
444	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with an inferior response to induction\n  chemotherapy with a higher rate of resistant disease in young (15-60+, median 45)\n  patients with cytogenetically normal AML\ndisease_id: 13\nsource_id: 167\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	15abd96a-0031-44fd-bebb-dc70f091c3d2	2015-07-31 17:35:44.846484
445	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No differences in relapse-free or overall survival were identified between\n  young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in\n  cytogenetically normal AML\ndisease_id: 13\nsource_id: 168\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a2924b5f-2d4a-4e9b-aac4-d875b0f8616a	2015-07-31 17:35:44.894964
446	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rates of complete remission and refractory disease are not different\n  in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 13\nsource_id: 168\nvariant_id: 129\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	059c6010-3428-41a1-9cb9-15c7a4d42521	2015-07-31 17:35:44.925969
447	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutations  in WT1 were associated with increased risk of recurrence in\n  young patients (15-50) with cytogenetically normal AML\ndisease_id: 13\nsource_id: 169\nvariant_id: 129\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4f83245a-85b9-4381-87a5-1043cd1ba67f	2015-07-31 17:35:44.961482
448	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Mutations in WT1 were associated with a worse overall prognosis than\n  patients wildtype for WT1 in young patients (15-50), primarily because of increased\n  risk of disease recurrence\ndisease_id: 13\nsource_id: 169\nvariant_id: 129\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6ef27301-58d1-44fe-bc8d-bf8235ce3afe	2015-07-31 17:35:44.992436
449	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had\n  significantly lower complete remission and higher refractory disease rates than\n  those with wildtype WT1 and without FLT3-ITD following induction therapy in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 13\nsource_id: 168\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	29bb04bf-04f5-4bb8-a16a-457bfd0a0bf0	2015-07-31 17:35:45.040262
450	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\ndisease_id: 13\nsource_id: 170\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a055bc79-413c-4f8c-b09b-9586041025e1	2015-07-31 17:35:45.082175
692	45	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881332'\nstop:\n- \n- '37881332'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	da91c719-6a97-4437-84cf-ccdfc9d49c04	2015-07-31 17:37:42.753895
693	48	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419923'\nstop:\n- \n- '152419923'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- G\n	2	\N	\N	064e719c-3612-443b-88c3-676497273db7	2015-07-31 17:37:42.769516
451	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were a negative prognostic factor for overall survival\n  in young (15-60+, median 45) patients with cytogenetically normal AML\ndisease_id: 13\nsource_id: 167\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ca702fe1-1c81-41bf-83a1-5dd90746e40d	2015-07-31 17:35:45.119366
452	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\ndisease_id: 13\nsource_id: 167\nvariant_id: 129\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	014e8cfc-417f-4413-92f3-35d9e6f70f66	2015-07-31 17:35:45.160166
453	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with an inferior response to induction\n  chemotherapy with a higher rate of resistant disease in young (15-60+, median 45)\n  patients with cytogenetically normal AML\ndisease_id: 13\nsource_id: 167\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8ebff929-477e-41c1-9ad5-9725a494d721	2015-07-31 17:35:45.194862
454	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: No differences in relapse-free or overall survival were identified between\n  young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in\n  cytogenetically normal AML\ndisease_id: 13\nsource_id: 168\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9bbcf211-8b00-4acb-a4ea-d367b8ddcdf1	2015-07-31 17:35:45.229197
455	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rates of complete remission and refractory disease are not different\n  in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young\n  (16-60) patients with cytogenetically normal AML.\ndisease_id: 13\nsource_id: 168\nvariant_id: 130\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	656cfb0e-f6f9-4cda-9a7b-930459ea8825	2015-07-31 17:35:45.258647
456	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had\n  significantly lower complete remission and higher refractory disease rates than\n  those with wildtype WT1 and without FLT3-ITD following induction therapy in young\n  (16-60) patients with cytogenetically normal AML\ndisease_id: 13\nsource_id: 168\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4b476368-6afc-4377-acd5-677ecd33e71b	2015-07-31 17:35:45.293978
457	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were a negative prognostic factor for overall survival\n  in young (15-60+, median 45) patients with cytogenetically normal AML\ndisease_id: 13\nsource_id: 167\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	635c2ac0-f075-40bb-8ed8-f0e1d806c113	2015-07-31 17:35:45.324413
458	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\ndisease_id: 13\nsource_id: 167\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	762f3a0a-1352-4c57-8cb5-f8337b054205	2015-07-31 17:35:45.358827
459	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WT1 mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\ndisease_id: 13\nsource_id: 170\nvariant_id: 130\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d546294c-16d8-4255-9c17-c1307d29bf4a	2015-07-31 17:35:45.388145
460	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The combination of PARP inhibition (Olaparib) with BRCA mutation results\n  in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation\n  positive women with high grade serous or undifferentiated ovarian cancer.\ndisease_id: 29\nsource_id: 171\nvariant_id: 131\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 1\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7b0fdb92-d83e-49e5-8afc-b4636b33af82	2015-07-31 17:35:45.430488
461	13	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: BRCA Germline Variants\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	017aa68b-10a0-410f-b417-10936f64bb6b	2015-07-31 17:35:45.451399
462	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The combination of PARP inhibition (Olaparib) with BRCA mutation results\n  in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation\n  positive women with high grade serous or undifferentiated ovarian cancer.\ndisease_id: 29\nsource_id: 171\nvariant_id: 132\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 1\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7b5c509e-9717-4f57-bfe5-f9b3fa428471	2015-07-31 17:35:45.481429
463	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of BCR-ABL fusion is considered the characterizing feature\n  of chronic myeloid leukemia, and has been widely thought of as the initiating event\n  in the disease.\ndisease_id: 15\nsource_id: 172\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	91004e6e-54a4-423f-9a13-5a569f858beb	2015-07-31 17:35:45.534149
464	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Treatment of Philadelphia Chromosome positive leukemias with Arsenic\n  Trioxide results in high rates of complete remission in patients.\ndisease_id: 13\nsource_id: 173\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7d096692-7131-446d-ad8a-dcdeae5fd748	2015-07-31 17:35:45.594177
465	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The clinical use of imatinib in patients with BCR-ABL fusion has resulted\n  in drastic sensitivity to the drug.\ndisease_id: 15\nsource_id: 172\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8093e040-f23a-43df-a4f8-435ae1333ceb	2015-07-31 17:35:45.629509
466	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BCR-ABL inhibitors such as imatinib have lead to significantly improved\n  prognosis, response rate, overall survival, and patient outcome in CML patients\n  compared to previous therapeutic regimens.\ndisease_id: 15\nsource_id: 172\nvariant_id: 1\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	186df6c0-53c4-4663-a454-35ad3d663bd6	2015-07-31 17:35:45.665523
467	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of second-generation BCR-ABL targeted therapies has been effective\n  in patients resistant to imatinib.\ndisease_id: 15\nsource_id: 172\nvariant_id: 1\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	7937ba2d-fae6-4260-a33b-53fca6bbe456	2015-07-31 17:35:45.695567
468	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML\n  patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion,\n  and may imply the possibility that BCR-ABL is not universally the initiating event\n  in CML.\ndisease_id: 15\nsource_id: 174\nvariant_id: 1\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4a8b1cef-978a-4315-9f2d-a1962e158409	2015-07-31 17:35:45.739686
469	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: BCR-ABL fusions have been found to contribute to imatinib resistance\n  in AML cell lines.\ndisease_id: 13\nsource_id: 173\nvariant_id: 1\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	27fa8dc7-0d3b-42e5-af20-d0aeecf6ba91	2015-07-31 17:35:45.769632
470	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\ndisease_id: 15\nsource_id: 175\nvariant_id: 3\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	4be2301f-f03c-4932-96fa-a25744ac55c9	2015-07-31 17:35:45.81093
471	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors\n  have shown to be resistant to imatinib treatment.\ndisease_id: 15\nsource_id: 172\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	03a55245-188f-4574-bcb8-563fa3a10bc7	2015-07-31 17:35:45.856023
472	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\ndisease_id: 15\nsource_id: 175\nvariant_id: 2\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dce1ec5f-4de0-4234-930a-96b6370a794d	2015-07-31 17:35:45.896335
473	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity\n  to the AKT inhibitor MK-2206 in breast cancer cell lines.\ndisease_id: 30\nsource_id: 176\nvariant_id: 4\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 14:105246551-105246551 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	60f5f29f-8c02-4ac6-89a6-f354778526dd	2015-07-31 17:35:45.945422
474	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring the EML4-ALK fusion,\n  treatment with crizotinib has shown to be effective in acheiving stable disease.\ndisease_id: 12\nsource_id: 177\nvariant_id: 5\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	daf3b146-02e1-4de4-8426-b96ad36fb8a0	2015-07-31 17:35:45.986395
475	14	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: ALK Fusions\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	2e8397c3-0b82-462d-8597-ac250e071f9a	2015-07-31 17:35:46.012034
476	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring EML4-ALK fusion,\n  the C1156Y variant has been shown to confer resistance to crizotinib.\ndisease_id: 12\nsource_id: 177\nvariant_id: 6\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d99e7317-c0a2-4db4-9a74-5a4de12f2bc4	2015-07-31 17:35:46.040145
477	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with non-small cell lung cancer harboring EML4-ALK fusion,\n  the L1196M variant has been shown to confer resistance to crizotinib.\ndisease_id: 12\nsource_id: 177\nvariant_id: 7\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	afd5e491-608d-48a2-ae7c-fbdb192cb550	2015-07-31 17:35:46.082852
478	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Inceased copy number of CCND1 is associated with poorer overall survival.\ndisease_id: 12\nsource_id: 178\nvariant_id: 18\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	55c0dce4-6e2e-4d56-a1a7-e059de0ab6cf	2015-07-31 17:35:46.14776
479	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Three studies have found no significant survival impact for CCND1 in\n  lung cancer.\ndisease_id: 12\nsource_id: 178\nvariant_id: 19\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	68ae18b5-de31-4cc2-a43b-6873881408b4	2015-07-31 17:35:46.180167
481	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with the ER-positive subtype of\n  breast cancer.\ndisease_id: 31\nsource_id: 179\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ca07a0c1-aa0e-495f-b4a6-9a6eda60ed7e	2015-07-31 17:35:46.24776
482	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\ndisease_id: 31\nsource_id: 179\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2a020715-ac88-4050-86aa-9902902e2ee3	2015-07-31 17:35:46.287247
483	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with shorter overall survival\n  and increased metastasis in head and neck squamous cell carcinoma.\ndisease_id: 32\nsource_id: 180\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6390fdb6-3aa3-4707-8570-14912640ffe4	2015-07-31 17:35:46.334444
484	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D1 overexpression is associated with shorter overall survival\n  and increased metastasis in mantle cell lymphoma.\ndisease_id: 33\nsource_id: 181\nvariant_id: 20\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f25a1e6a-8563-48a1-b369-9a7d78709e1f	2015-07-31 17:35:46.374252
485	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with poor prognosis in gastric\n  cancers.\ndisease_id: 34\nsource_id: 182\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9b721a79-1f02-4269-8d37-1040ec8fffeb	2015-07-31 17:35:46.415419
486	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with the increased depth of cancer\n  invasion in gastric cancers.\ndisease_id: 34\nsource_id: 182\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	12e4fb45-3752-4235-b540-33c687570b57	2015-07-31 17:35:46.447669
487	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with the presence pf lymph node\n  metastasis in gastric cancers.\ndisease_id: 34\nsource_id: 182\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cb3669cb-4917-4200-b931-b45a818de3b6	2015-07-31 17:35:46.487956
488	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cyclin D2 overexpression is associated with vascular invasion by cancer\n  cells in gastric cancers.\ndisease_id: 34\nsource_id: 182\nvariant_id: 21\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a99dfe12-011f-43cc-bd04-3a8cf277f7a9	2015-07-31 17:35:46.522448
489	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\ndisease_id: 34\nsource_id: 183\nvariant_id: 22\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 5\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	90dd3dd3-ad6f-4402-8fe3-5c84e718168d	2015-07-31 17:35:46.559863
490	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with\n  Palbociclib (PD-0332991) significantly increased median survival.\ndisease_id: 35\nsource_id: 184\nvariant_id: 23\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b4af10c0-c750-468b-bf9a-abefeca57276	2015-07-31 17:35:46.611378
491	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with high expression of total cyclin E and low-molecular weight\n  cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard\n  ratio for these patients in 13.3 times higher than those with normal cyclin E levels.\ndisease_id: 31\nsource_id: 185\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9a320987-987f-491d-99d9-e9a31b5b83fb	2015-07-31 17:35:46.666719
492	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\ndisease_id: 36\nsource_id: 186\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c7e78fd4-62fc-48f0-ae18-530983e2fb0a	2015-07-31 17:35:46.705549
493	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\ndisease_id: 36\nsource_id: 186\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c338ec5e-b730-4a98-a657-f06ff542b583	2015-07-31 17:35:46.741708
494	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\ndisease_id: 37\nsource_id: 186\nvariant_id: 24\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c0fa60fc-354d-4dfc-a7d9-031d6ab11d08	2015-07-31 17:35:46.773048
495	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\ndisease_id: 38\nsource_id: 187\nvariant_id: 25\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1397ccc7-7492-4b52-ab55-ea3fdddd00cf	2015-07-31 17:35:46.811169
496	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\ndisease_id: 38\nsource_id: 188\nvariant_id: 25\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	c23b2a95-7104-4401-bbf6-082ad9c20d94	2015-07-31 17:35:46.849544
497	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\ndisease_id: 38\nsource_id: 187\nvariant_id: 26\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f40c989b-dde5-4984-b9a6-8dc5f3a0ceec	2015-07-31 17:35:46.882013
498	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\ndisease_id: 38\nsource_id: 188\nvariant_id: 26\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	a8e5ac80-5936-46a0-b7b0-9458b5abfddd	2015-07-31 17:35:46.912172
499	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A)\n  promoter hypermethylation is associated with shorter overall survival.\ndisease_id: 12\nsource_id: 189\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eb3d40a5-cfec-4ee8-8fc5-1afa5de0bddd	2015-07-31 17:35:46.948713
500	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A)\n  promoter hypermethylation is associated with shorter time to recurrence.\ndisease_id: 12\nsource_id: 189\nvariant_id: 27\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	949a5d83-f17d-4c27-87a8-2344d04174d4	2015-07-31 17:35:46.977797
501	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G253C mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 12\nsource_id: 190\nvariant_id: 141\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	11a2f94e-65b8-47d9-8c2a-8ff91e89856a	2015-07-31 17:35:47.021737
502	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G505S mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 12\nsource_id: 190\nvariant_id: 142\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3318b36f-4c40-4e93-ace9-eded8794dbbe	2015-07-31 17:35:47.058946
503	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the G774V mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 12\nsource_id: 190\nvariant_id: 144\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b501dec4-c24a-494d-8ed8-84785855111b	2015-07-31 17:35:47.093468
504	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the I638F mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 12\nsource_id: 190\nvariant_id: 143\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1212d442-9cde-4aad-8ec9-6534b91a7113	2015-07-31 17:35:47.122954
505	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the L239R mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 12\nsource_id: 190\nvariant_id: 140\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e7f3c989-8844-47f9-a8cd-7a0de199ee14	2015-07-31 17:35:47.165077
506	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In 3T3 fibroblasts expressing the L63V mutation have to be sensitive\n  to the tyrosine kinase inhibitor dasatinib.\ndisease_id: 12\nsource_id: 190\nvariant_id: 139\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0f2869d8-67a7-4a08-b6c1-dff1282dff5b	2015-07-31 17:35:47.195213
507	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient expressing a DDR2 S768R mutation, treatment with dasatinib\n  plus erlotinib acheived a partial response.\ndisease_id: 12\nsource_id: 190\nvariant_id: 145\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	5d3d3a6c-c38b-412e-b579-f87be7963e4f	2015-07-31 17:35:47.224949
508	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Exon 19 deletion has been shown to be correlated with gefitinib response.\ndisease_id: 12\nsource_id: 191\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fff13433-2639-4f69-8083-c74231d59e11	2015-07-31 17:35:47.273486
509	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Gefinitib has been shown to be effective in treating cell lines with\n  G719S missense mutations.\ndisease_id: 12\nsource_id: 192\nvariant_id: 134\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9620e2d1-080e-4405-b1f4-6ebab57b52c5	2015-07-31 17:35:47.312532
510	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no statistical difference in progression free survival between\n  lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\ndisease_id: 12\nsource_id: 193\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	3627362d-5708-48e4-afd2-b36bb539669d	2015-07-31 17:35:47.356094
511	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no statistical difference in progression free survival between\n  lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.\ndisease_id: 12\nsource_id: 193\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	fcfa236e-2906-470c-bd67-aea9efa86ac5	2015-07-31 17:35:47.39113
512	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\ndisease_id: 12\nsource_id: 194\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	953f07c1-498c-4ee2-ba0d-bbc109542d9a	2015-07-31 17:35:47.432476
513	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Median survival of patients with EGFR L858R mutation is better than those\n  with wild type EGFR.\ndisease_id: 12\nsource_id: 195\nvariant_id: 33\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55259514-55259515 (CT->AG)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	34fdf6d8-64ae-4aa9-b5ad-eacafbafdf5c	2015-07-31 17:35:47.480522
514	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Gefinitib has been shown to be effective in treating cell lines with\n  L858R missense mutations.\ndisease_id: 12\nsource_id: 192\nvariant_id: 33\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9af7bf77-33c4-44e4-b53f-2c2d62b8bb69	2015-07-31 17:35:47.511832
515	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The T790M mutation in EGFR has been shown to confer resistance to the\n  tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are\n  placed on the drug are likely to relapse.\ndisease_id: 39\nsource_id: 196\nvariant_id: 34\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	10ea7e34-abb1-4e7e-96a4-d73510e58c6d	2015-07-31 17:35:47.553966
516	15	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: EGFR TKI Resistance\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	89208870-335f-4a29-ac37-7f2a35c72f7a	2015-07-31 17:35:47.572395
517	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In non-small cell lung cancer, the appearance of T790M mutation leads\n  to resistance to gefitinib.\ndisease_id: 12\nsource_id: 197\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3e18e249-5531-4a29-b80a-4a4a83f8f9ce	2015-07-31 17:35:47.61098
518	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC patients with T790M and another activating mutations, their\n  progression free survival is shorter compared to those who do not possess T790M\n  mutation.\ndisease_id: 12\nsource_id: 198\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1614f333-c92f-4a7d-b6b8-09990ff77218	2015-07-31 17:35:47.647379
519	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with NSCLC harboring EGFR T790M mutation have statistically\n  worse overall survival compared to patients with L858R or other exon 19 activating\n  mutations.\ndisease_id: 12\nsource_id: 199\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	86bcdd8e-d263-45ea-ac53-e6a033919933	2015-07-31 17:35:47.702247
520	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a NSCLC patient with T790M and L858R mutation, combination treatment\n  with erlotinib and premetrexed has shown to be effective, reducing tumor size and\n  resulting in stable disease.\ndisease_id: 12\nsource_id: 200\nvariant_id: 34\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	b9518b74-6b61-4b3f-92a9-c0a546ea738c	2015-07-31 17:35:47.741958
521	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In NSCLC containing a T790M mutation, staurosporine may be inhibitive\n  of EGFR, especially when an L858R mutation is also present.\ndisease_id: 12\nsource_id: 201\nvariant_id: 34\nrating: 1\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 7:55249071-55249071 (C->T)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	079d2b0b-5514-4384-ba90-7ef21a75feb3	2015-07-31 17:35:47.789209
522	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\ndisease_id: 12\nsource_id: 202\nvariant_id: 34\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	dcd56308-fea1-4112-b66e-b29c573c5328	2015-07-31 17:35:47.828517
523	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the D769H mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 31\nsource_id: 203\nvariant_id: 35\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880261-37880261 (G->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0fceaca7-863d-414a-ba8e-dae86c859471	2015-07-31 17:35:47.878994
524	16	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: HER2 Activating\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	438dfd98-052d-441b-9c72-053ee2ec304d	2015-07-31 17:35:47.899669
525	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 31\nsource_id: 203\nvariant_id: 36\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880261-37880261 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eacd2ad7-1a7e-496c-aa2f-3520db44c713	2015-07-31 17:35:47.926779
694	49	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419922'\nstop:\n- \n- '152419922'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- A\n	2	\N	\N	7d7a0168-ce1f-4581-9669-db8bb42deddc	2015-07-31 17:37:42.787172
526	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines retrovirally transduced with wildtype or mutant\n  ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown\n  to be sensitive to neratinib.\ndisease_id: 31\nsource_id: 203\nvariant_id: 37\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ec1a2ef6-1551-4eef-aa40-0651eba8d583	2015-07-31 17:35:47.961553
527	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the G309A mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 31\nsource_id: 203\nvariant_id: 38\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37868205-37868205 (G->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a738d6a0-5032-4125-8dc0-f694dfbb939f	2015-07-31 17:35:47.994914
528	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L755S mutation was shown to confer resistance to lapatinib in MCF10A\n  cell lines retrovirally transduced with wildtype or mutant ERBB2.\ndisease_id: 31\nsource_id: 203\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880220-37880220 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	03cf9edc-61f5-4131-bb20-c15f23caf0c3	2015-07-31 17:35:48.033357
529	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L755S mutation showed mild response to neratinib in MCF10A cell lines\n  retrovirally transduced with wildtype or mutant ERBB2.\ndisease_id: 31\nsource_id: 203\nvariant_id: 39\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37880220-37880220 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	283bc914-1a8f-49bb-803d-1d6ff5a56ce5	2015-07-31 17:35:48.073008
530	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\ndisease_id: 31\nsource_id: 203\nvariant_id: 40\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c13dfd3f-66b5-4219-820c-15b086f79dda	2015-07-31 17:35:48.108772
531	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the in-frame insertion of a single amino acid at\n  position 780 was shown to be sensitive to neratinib.\ndisease_id: 31\nsource_id: 203\nvariant_id: 41\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	59d5663a-9540-4be5-8918-ace9cf819110	2015-07-31 17:35:48.141192
532	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\ndisease_id: 31\nsource_id: 203\nvariant_id: 42\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	9a188525-5df2-4dd3-9d8d-9350b812ba80	2015-07-31 17:35:48.178032
533	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the R896C mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 31\nsource_id: 203\nvariant_id: 43\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881616-37881616 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c7c94f06-b827-4b4e-9928-6b9970669066	2015-07-31 17:35:48.231132
534	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the V777L mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 31\nsource_id: 203\nvariant_id: 44\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881000-37881000 (G->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cfe9116e-60fc-423c-b3a5-2442fc99a27d	2015-07-31 17:35:48.267367
535	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In MCF10A cell lines, the V842I mutation was shown to be sensitive to\n  neratinib.\ndisease_id: 31\nsource_id: 203\nvariant_id: 45\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:37881332-37881332 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	38b96f95-dd34-4133-b957-ce83875cd0f2	2015-07-31 17:35:48.313408
536	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 31\nsource_id: 204\nvariant_id: 46\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	86e7e6dd-5759-4c86-8a15-054aeec1a558	2015-07-31 17:35:48.35988
537	17	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: ESR1 Ligand-Binding Domain\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	6e4f281c-4265-44bd-bd3a-9c51e49dbcf6	2015-07-31 17:35:48.381655
538	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The L536Q ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  MCF7 cell lines.\ndisease_id: 31\nsource_id: 205\nvariant_id: 46\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	783fabde-0063-4b5e-ae7b-2da44082521e	2015-07-31 17:35:48.421067
539	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 31\nsource_id: 204\nvariant_id: 47\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	201c05d5-7850-4625-b097-90e557e45f32	2015-07-31 17:35:48.461973
540	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The N538G ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 31\nsource_id: 205\nvariant_id: 47\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	7ad920e2-4f02-48af-ac66-48f52f20c01c	2015-07-31 17:35:48.504419
541	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 31\nsource_id: 204\nvariant_id: 48\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d404f43f-f019-4756-adc0-f75dd1d7a130	2015-07-31 17:35:48.546905
542	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537C ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 31\nsource_id: 205\nvariant_id: 48\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	d61ebbb1-cc3f-4606-8996-ede7b54bb9b9	2015-07-31 17:35:48.582624
543	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 31\nsource_id: 204\nvariant_id: 49\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419922-152419922 (T->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c6bc3cdd-b92b-486a-b920-4682000a2b6c	2015-07-31 17:35:48.622873
544	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537N ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 31\nsource_id: 205\nvariant_id: 49\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419922-152419922 (T->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	0305817e-5e51-4671-9704-0f26ac4dc696	2015-07-31 17:35:48.665474
545	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain\n  of ESR1 have shown resistance to hormone therapy.\ndisease_id: 31\nsource_id: 204\nvariant_id: 50\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	83887631-6afd-45bc-b157-ed3ceecb7d72	2015-07-31 17:35:48.708093
546	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The Y537S ligand binding domain mutations have been shown to result in\n  constitutive activity and continued responsiveness to anti-estrogen therapies in\n  vitro.\ndisease_id: 31\nsource_id: 205\nvariant_id: 50\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 6:152419923-152419923 (A->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	9ca6afb0-fd75-47ec-b5fc-b1a1f27853db	2015-07-31 17:35:48.760295
547	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a single patient with this fusion in intrahepatic cholangiocarcinoma.\n  Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic\n  sites and overall stable disease. However, this response did not meet criteria for\n  RECIST partial response.\ndisease_id: 40\nsource_id: 206\nvariant_id: 52\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	28784612-9ebb-4c55-bfc0-d8b1e2ec8553	2015-07-31 17:35:48.815306
548	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This\n  fusion is also shown to be recurrent in multiple other cancer types.\ndisease_id: 40\nsource_id: 206\nvariant_id: 54\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	820ff44c-ce93-4d94-a8b6-a5cf0601c406	2015-07-31 17:35:48.861203
549	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WBC counts were higher in AML patients with internal tandem repeat of\n  Flt3 compared to wild type\ndisease_id: 13\nsource_id: 207\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a792ede6-bd0f-44c8-9c2e-31c6d45d5481	2015-07-31 17:35:48.912965
550	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated\n  AML patients with FLT3 mutations\ndisease_id: 13\nsource_id: 208\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1f91f8bf-0bfb-4d76-943c-f92e98048fa3	2015-07-31 17:35:48.95462
551	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib is effective in patients with FLT3 internal tandem repeats,\n  but not in wild type or D835 mutation.\ndisease_id: 13\nsource_id: 209\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e6de5f47-5321-44fb-b276-f6a22ccdc483	2015-07-31 17:35:48.998575
552	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\ndisease_id: 13\nsource_id: 207\nvariant_id: 55\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	674f9565-59b7-44d8-9de7-c54ff611e191	2015-07-31 17:35:49.030317
553	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads\n  to poor survival and inferior disease free survival in patients with AML.\ndisease_id: 13\nsource_id: 210\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	97642ae8-34bd-43ee-af8e-ae4728f0b345	2015-07-31 17:35:49.073629
554	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML\n  cells with FLT3 internal tandem repeat\ndisease_id: 13\nsource_id: 211\nvariant_id: 55\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fe2a0555-c6fe-4537-9f61-2770550561e0	2015-07-31 17:35:49.111623
555	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: WBC counts were not significantly different in AML patients with FLT3\n  D835 mutations compared to wild type\ndisease_id: 13\nsource_id: 207\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	af1fa8a4-9229-4f18-9ab6-002b4ea6facf	2015-07-31 17:35:49.14173
556	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is\n  associated with sorafenib resistance\ndisease_id: 13\nsource_id: 212\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c912d388-2f8b-4799-bc9f-209f73c55d83	2015-07-31 17:35:49.180724
557	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\ndisease_id: 13\nsource_id: 209\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cdd2c615-0773-4553-8b03-1bebc09e6b40	2015-07-31 17:35:49.210837
558	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated\n  AML patients with FLT3 mutations\ndisease_id: 13\nsource_id: 208\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	51ce6712-5e38-4d9f-a4f1-28a79e476ef3	2015-07-31 17:35:49.246476
559	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\ndisease_id: 13\nsource_id: 207\nvariant_id: 56\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	439dccf5-8716-47d9-ba65-d988fd02ecd1	2015-07-31 17:35:49.291718
560	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML patients, FLT3-TKD mutation was associated with poorer disease\n  free survival compared to patients with wild type FLT3\ndisease_id: 13\nsource_id: 213\nvariant_id: 56\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7361b550-fe1e-4884-a4fe-4179147bd9df	2015-07-31 17:35:49.327552
561	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib\n  and fasudil led to dramatic tumor regression.\ndisease_id: 41\nsource_id: 214\nvariant_id: 57\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	0e010435-f2b1-4264-b7d2-09ddf5de2571	2015-07-31 17:35:49.374232
562	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In leukemia patients, IDH1 R132 mutation is associated with adults and\n  normal karyotype.\ndisease_id: 13\nsource_id: 215\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	31839230-989a-4486-945a-b4276a2f8d3d	2015-07-31 17:35:49.41916
563	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The complete remission and overall survival rates in patients with IDH1\n  R132 mutation is not significantly different from those who do not have this mutation\ndisease_id: 13\nsource_id: 216\nvariant_id: 58\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	547bbead-8890-4b45-b01f-b39ef7a1eacf	2015-07-31 17:35:49.460085
564	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\ndisease_id: 42\nsource_id: 217\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	cd21213f-2347-4bf6-975e-662d28e40a93	2015-07-31 17:35:49.499472
565	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with glioblastoma, those harboring IDH1 R132 mutation has\n  higher overall survival compared to those who do not have IDH1 mutation\ndisease_id: 43\nsource_id: 218\nvariant_id: 58\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	32599e72-c7ee-4aa0-957c-a5eb8e7ed530	2015-07-31 17:35:49.541686
566	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes\n  is associated with worse overall survival\ndisease_id: 44\nsource_id: 219\nvariant_id: 58\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0e4b8d01-aac4-4aa7-aadf-5182b970f995	2015-07-31 17:35:49.589449
567	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation is associated with patients of older age, high platelet\n  count during diagnosis, cytogenic normalcy and NPM1 mutation\ndisease_id: 13\nsource_id: 220\nvariant_id: 59\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113113-209113113 (G->A)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5142c7fd-acc8-47d6-ab92-41d30be37e7d	2015-07-31 17:35:49.631516
568	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 13\nsource_id: 220\nvariant_id: 59\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113113-209113113 (G->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e8e2cc12-2cd5-4001-b3fe-8a56f7a78bba	2015-07-31 17:35:49.665634
569	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 13\nsource_id: 220\nvariant_id: 60\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113112-209113112 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ae903776-643f-466a-8812-bf1ce9dac0ea	2015-07-31 17:35:49.698064
570	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\ndisease_id: 13\nsource_id: 220\nvariant_id: 61\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:209113112-209113112 (C->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b725a5c2-a5a3-4ca7-b25d-3ca9840949aa	2015-07-31 17:35:49.728716
571	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with IDH2 mutations such as R140Q/L have event free survival\n  and overall survival similar to those with wild-type IDH2\ndisease_id: 13\nsource_id: 221\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e5e7fd91-1c13-4c6f-8fa8-d6935dea23eb	2015-07-31 17:35:49.772689
572	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with IDH2 R140Q/L mutation, the presence of mutation does\n  not impact overall survival or disease free survival.\ndisease_id: 13\nsource_id: 222\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	447dac2a-3d0e-42f2-9b39-5cd3f9aff6fe	2015-07-31 17:35:49.824855
573	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients suffering from MDS, the presence of IDH2 mutation such as\n  R140Q/L do not confer prognostic value (overall survival)\ndisease_id: 45\nsource_id: 223\nvariant_id: 62\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	54d6b015-98b6-4c1a-8d1c-ac1ed47b2a7f	2015-07-31 17:35:49.86636
574	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: R140Q mutation in IDH2 does not have prognostic value in patients with\n  MDS.\ndisease_id: 45\nsource_id: 224\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	a737eec5-2aaa-41b6-86f4-980cfa19a6b9	2015-07-31 17:35:49.904333
575	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML, patients with IDH2 R140K mutation have improved overall survival\n  compared to those with wild-type IDH2\ndisease_id: 13\nsource_id: 225\nvariant_id: 62\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2b5c45fc-bf8e-4125-b481-e50aa47668a7	2015-07-31 17:35:49.941792
576	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: AML patients with IDH2 mutations such as R172K have event free survival\n  and overall survival similar to those with wild-type IDH2.\ndisease_id: 13\nsource_id: 221\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3587712a-9790-4643-8b03-974cc8650e05	2015-07-31 17:35:49.97642
577	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients suffering from MDS, the presence of IDH2 mutations such as\n  R172K do not confer prognostic value (overall survival)\ndisease_id: 45\nsource_id: 223\nvariant_id: 63\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b866cff4-bda1-4b7b-9140-5915c7a35b5a	2015-07-31 17:35:50.008747
578	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In AML, patients with an IDH2 R172K mutation have worse overall survival\n  compared to those with wild-type IDH2\ndisease_id: 13\nsource_id: 225\nvariant_id: 63\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 15:90631838-90631838 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fca177c8-f4ed-4121-a256-41598cdac30b	2015-07-31 17:35:50.038189
579	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with larger, more invasive tumors, greater\n  pathologic histology and older patients compared to tumors with wildtype KIT\ndisease_id: 23\nsource_id: 226\nvariant_id: 66\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	aeadabfb-6d95-452c-868a-2e78094da58e	2015-07-31 17:35:50.077434
580	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations were identified in morphologically benign, incidentally\n  discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients\n  indicating is not a prognostic marker for GIST\ndisease_id: 23\nsource_id: 227\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	30c939ec-5bf9-42f2-9d5f-7170ae09b8d7	2015-07-31 17:35:50.122015
581	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations detected in 5/60 patients showed no prognostic significance\n  in patients with small cell lung cancer\ndisease_id: 37\nsource_id: 228\nvariant_id: 66\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3a87f0bd-2f51-4e45-91c8-d7aeb9785c2d	2015-07-31 17:35:50.172152
582	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with worse overall and cause-specific prognosis\n  in patients with GIST compared to patients with wildtype KIT\ndisease_id: 23\nsource_id: 226\nvariant_id: 66\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	56c661af-6406-419a-9057-a711dc0f1224	2015-07-31 17:35:50.212221
583	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutation is associated with larger, more invasive tumors, greater\n  pathologic histology and older patients compared to tumors with wildtype KIT\ndisease_id: 23\nsource_id: 226\nvariant_id: 70\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e37da05f-cf0f-49e1-8289-ad184beab982	2015-07-31 17:35:50.250441
695	50	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '6'\nstart:\n- \n- '152419923'\nstop:\n- \n- '152419923'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- C\n	2	\N	\N	f28a804b-55ea-4012-bf1a-ba6735a2018c	2015-07-31 17:37:42.803629
584	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT mutations detected in 5/60 patients showed no prognostic significance\n  in patients with small cell lung cancer\ndisease_id: 37\nsource_id: 228\nvariant_id: 71\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	22d4e020-cdd2-4a5e-87dd-cff9e4b8d1a5	2015-07-31 17:35:50.302831
585	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a case study, a patient with anal melanoma harboring a 7 codon duplication\n  in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after\n  imatinib treatment.\ndisease_id: 46\nsource_id: 229\nvariant_id: 67\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e397f5c5-d2ca-4516-aaab-c45d51714632	2015-07-31 17:35:50.348245
586	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib,\n  nilotinib and imatinib.\ndisease_id: 12\nsource_id: 230\nvariant_id: 72\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	fc9d8658-6ca9-49fd-af8c-851991c36c5c	2015-07-31 17:35:50.396983
587	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248\n  is effective for those that are refractory to imatinib.\ndisease_id: 23\nsource_id: 231\nvariant_id: 73\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55594258-55594258 (T->C)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c3c940ad-324f-4873-901b-4e60bd22ed3a	2015-07-31 17:35:50.446607
588	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KIT V654A results in imatinib resistance in GIST patient-derived cell\n  lines.\ndisease_id: 23\nsource_id: 91\nvariant_id: 73\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 4:55594258-55594258 (T->C)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	1e414d77-4bd3-441d-adc2-f1322a2733d9	2015-07-31 17:35:50.482765
589	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with colorectal cancer who harbor KRAS mutation have low response\n  rate to cetuximab\ndisease_id: 19\nsource_id: 232\nvariant_id: 74\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8408049e-8c9d-4a91-89c0-c29ba04e3e0e	2015-07-31 17:35:50.52865
590	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations in lung cancer patients are not associated with smoking\n  history, age or gender\ndisease_id: 12\nsource_id: 233\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8ccf4787-0877-414b-9082-596092b65fbf	2015-07-31 17:35:50.576165
591	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with stage III colorectal cancer undergoing chemotherapy,\n  KRAS G12 mutation did not impact overall or disease free survival.\ndisease_id: 26\nsource_id: 234\nvariant_id: 76\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9c2cf1cf-f4b7-4d54-863c-0346d11dcf57	2015-07-31 17:35:50.623399
592	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: There is no strong association between KRAS mutation status and overall\n  or progression free survival in patients with NSCLC\ndisease_id: 12\nsource_id: 235\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	73bff76d-e75f-4e4a-a09d-b140639ede94	2015-07-31 17:35:50.660475
593	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of KRAS mutations in MM patients is prognostic for shorter\n  overall and progression free survival\ndisease_id: 47\nsource_id: 236\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	d260db0b-e143-4735-a4ac-d690a9b31dfa	2015-07-31 17:35:50.707476
594	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A meta-analysis showed KRAS mutation is associated with worse outcome\n  in patients with NSCLC\ndisease_id: 12\nsource_id: 237\nvariant_id: 76\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b8a57ed2-cf1e-4518-b06d-c32baf74372e	2015-07-31 17:35:50.749657
595	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 48\nsource_id: 238\nvariant_id: 148\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	c287ebc8-fb62-4e87-8214-ba5c87becf80	2015-07-31 17:35:50.791554
596	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12C occur more frequently in women than men\ndisease_id: 37\nsource_id: 239\nvariant_id: 78\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398285-25398285 (C->A)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3e2f2793-0b89-4e5c-a53f-eee4b2c3ecfd	2015-07-31 17:35:50.858889
597	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 48\nsource_id: 238\nvariant_id: 78\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	dba3434e-0ae1-4537-9cc7-78a39eef9b48	2015-07-31 17:35:50.888867
598	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12D mutation occurs in never smokers significantly more often than\n  in smokers\ndisease_id: 37\nsource_id: 239\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	c6d0c545-c5e0-487b-aff6-ad11fc350905	2015-07-31 17:35:50.93363
696	59	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113113'\nstop:\n- \n- '209113113'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	ae4afbd3-df06-4135-b8f8-e8f4167b5024	2015-07-31 17:37:42.821315
599	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\ndisease_id: 12\nsource_id: 240\nvariant_id: 79\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	b39f40d6-2922-4837-bbb2-f2d7ae7ad6b0	2015-07-31 17:35:50.980778
600	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 48\nsource_id: 238\nvariant_id: 79\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	cc0ed11b-6026-4f76-9699-c1d9ba24a159	2015-07-31 17:35:51.02157
601	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G12 mutations were the primary predictor of EGFR TKI resistance\n  in patients with lung adenocarcinoma.\ndisease_id: 48\nsource_id: 238\nvariant_id: 147\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	ffccd9da-ba5a-434a-aa53-ebc68fbe919f	2015-07-31 17:35:51.066035
602	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS mutations in lung cancer patients are not associated with smoking\n  history, age or gender\ndisease_id: 12\nsource_id: 233\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	fd9b68e5-952e-422a-b98d-5faf70699aa7	2015-07-31 17:35:51.111477
603	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of KRAS mutations in MM patients is prognostic for shorter\n  overall and progression free survival\ndisease_id: 49\nsource_id: 236\nvariant_id: 80\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5984065a-60d5-46d0-9204-624d56696d30	2015-07-31 17:35:51.143547
604	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G13D mutation is associated with better response to Cetuximab with\n  longer progression-free and overall survival in colorectal patients compared to\n  other KRAS\ndisease_id: 26\nsource_id: 147\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5886141d-a984-4094-85c4-30b68768cc9a	2015-07-31 17:35:51.180368
605	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: KRAS G13D mutation is associated with better response to Cetuximab with\n  longer progression-free and overall survival in colorectal patients compared to\n  other KRAS\ndisease_id: 26\nsource_id: 147\nvariant_id: 81\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398281-25398281 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2f8da9a1-13ec-488c-a1ef-fd43564e0d88	2015-07-31 17:35:51.215493
606	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804),\n  MET amplification and EGFR T790M were found to be the primary drivers of resistance.\ndisease_id: 12\nsource_id: 202\nvariant_id: 146\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	17682771-69f7-499a-b554-cf43d9757e74	2015-07-31 17:35:51.247941
607	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a randomized clinical trial, patients with MGMT promoter methyaltion\n  benefitted from temozolomide. This benefit was also methylation status dependent,\n  as those without methylation did not see increased survival.\ndisease_id: 28\nsource_id: 241\nvariant_id: 85\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 0\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	279e100f-d53c-4386-9731-c719ead0cb02	2015-07-31 17:35:51.293655
608	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\ndisease_id: 28\nsource_id: 148\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5f2698b0-a9dc-4e70-b40a-b8bfbea5bf9c	2015-07-31 17:35:51.325284
609	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\ndisease_id: 28\nsource_id: 148\nvariant_id: 85\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 0\n	1	\N	\N	d22f24ab-df91-4d82-96fd-b122f9cdecb8	2015-07-31 17:35:51.380072
610	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including D1643H, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 12\nsource_id: 242\nvariant_id: 135\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0aa99c7a-f1a1-47d5-b4ab-afda76d80c7a	2015-07-31 17:35:51.423899
611	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including R2328W, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 12\nsource_id: 242\nvariant_id: 136\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6dd6a5b2-4fc2-421f-9483-c1557d31bc3e	2015-07-31 17:35:51.454246
612	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including S2275fs, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 12\nsource_id: 242\nvariant_id: 138\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3de3e86b-c522-429e-8768-f9d35c9c7696	2015-07-31 17:35:51.491089
613	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NOTCH1 gain-of-function mutations, including V2444fs, have shown to be\n  correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.\ndisease_id: 12\nsource_id: 242\nvariant_id: 137\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9fcf1562-9374-453b-850e-f9d759e5088f	2015-07-31 17:35:51.523246
614	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with E542K/E545K mutations showed increased\n  sensitivity to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 31\nsource_id: 243\nvariant_id: 103\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7ff93bf6-6061-4bcb-af98-f24ee74b3074	2015-07-31 17:35:51.566167
615	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 31\nsource_id: 244\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e840451d-aa6c-47d0-8b20-be8783de3094	2015-07-31 17:35:51.60757
616	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 19\nsource_id: 245\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	bd7c6bd3-ad4c-4f46-92cb-762bcc184f69	2015-07-31 17:35:51.645774
617	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not\n  either alone, is prognostic for worse overall survival and cancer specific survival.\ndisease_id: 19\nsource_id: 246\nvariant_id: 103\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b50e26c8-16f7-4042-9d88-752213aa449c	2015-07-31 17:35:51.687411
618	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 19\nsource_id: 245\nvariant_id: 103\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936082-178936082 (G->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ad75f659-141a-4a47-ae2c-81219ca2fedc	2015-07-31 17:35:51.721385
619	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with E542K/E545K mutations showed increased\n  sensitivity to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 31\nsource_id: 243\nvariant_id: 104\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	06304d9e-1120-4815-a581-3dfed2cec17a	2015-07-31 17:35:51.754749
620	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 31\nsource_id: 244\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5d7f359c-f8c8-4689-b35d-848d3bca3266	2015-07-31 17:35:51.785133
621	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 19\nsource_id: 245\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	70deb499-3e4b-477e-a49b-1f8b5a0a2ccd	2015-07-31 17:35:51.822274
622	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not\n  either alone, is prognostic for worse overall survival and cancer specific survival.\ndisease_id: 19\nsource_id: 246\nvariant_id: 104\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	0bb8ebc4-a0d8-4fed-91ed-d4cb0ea1a9c7	2015-07-31 17:35:51.860205
623	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 9 (such as E545K) or exon 20 mutation may result\n  in poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 19\nsource_id: 245\nvariant_id: 104\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178936091-178936091 (G->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	7edb01f4-a4b0-4de1-a518-5e0fd0b7354a	2015-07-31 17:35:51.903398
624	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer cell lines with H1047R mutation showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\ndisease_id: 31\nsource_id: 243\nvariant_id: 107\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	903d4379-3982-4990-8a3b-6d304e72fe65	2015-07-31 17:35:51.935574
625	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Rapamycin inhibits transformation induced by mutation in PIK3CA\ndisease_id: 31\nsource_id: 244\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	abb072ec-655b-46b7-ad91-25a1f59220cb	2015-07-31 17:35:51.964645
626	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R)\n  were associated with lack of panitumumab or cetuximab response.\ndisease_id: 19\nsource_id: 245\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 4\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	fb228b1b-263f-47d6-9cb7-f87d7d428984	2015-07-31 17:35:51.994995
697	60	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113112'\nstop:\n- \n- '209113112'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	8b2b06f8-168a-4fce-b6a5-8da04d8a7819	2015-07-31 17:37:42.839622
627	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: The presence of exon 20 (such as H1047R) or exon 9 mutation may result\n  in poorer survival compared to those with wild-type PIK3CA mutation.\ndisease_id: 19\nsource_id: 245\nvariant_id: 107\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 3:178952085-178952085 (A->G)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8866d7e7-0c47-4f24-aec4-4a840654fb27	2015-07-31 17:35:52.029292
628	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Fusion protein degradation is accelerated upon ATRA treatment of acute\n  promyelocytic cells at pharmalogical concentrations.\ndisease_id: 50\nsource_id: 247\nvariant_id: 108\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	5555c8ed-47bd-4d94-97a4-b3f9b28d79fc	2015-07-31 17:35:52.069708
629	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\ndisease_id: 28\nsource_id: 248\nvariant_id: 110\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:89717672-89717672 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	39bd20c2-1823-4962-a56a-514e446da214	2015-07-31 17:35:52.113144
630	18	VariantGroup	\N	\N	\N	\N	\N	create	---\nname: PTEN Loss-of-Function\ndescription: \ndeleted: false\ndeleted_at: \n	1	\N	\N	018ff705-030a-4957-bd59-ea19fc7a6191	2015-07-31 17:35:52.13579
631	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cells with PTEN deficiency have been shown to exhibit slowed growth in\n  reponse to PI3K-mTOR inhibitors.\ndisease_id: 30\nsource_id: 249\nvariant_id: 110\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 10:89717672-89717672 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	199e246d-4afc-4cab-beac-6813cc99ed14	2015-07-31 17:35:52.17844
632	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Fusion protein degradation is accelerated upon ATRA treatment of acute\n  promyelocytic cells at pharmalogical concentrations.\ndisease_id: 50\nsource_id: 247\nvariant_id: 111\nrating: 5\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b7c72ccb-6ad6-448a-ab3e-7ddeb006a541	2015-07-31 17:35:52.218108
633	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with SF3B1 mutations had a statistically significant longer\n  overall survival as well as event free-survival. Both before and after adjustment\n  for age, karyotype and sex.\ndisease_id: 45\nsource_id: 250\nvariant_id: 114\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:198267359-198267359 (C->A)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	44c39bfb-bdcf-446f-a3b7-7111ba2c4d24	2015-07-31 17:35:52.259787
634	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Patients with SF3B1 mutations had a statistically significant longer\n  overall survival as well as event free-survival. Both before and after adjustment\n  for age, karyotype and sex.\ndisease_id: 45\nsource_id: 250\nvariant_id: 115\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 2:198266834-198266834 (T->C)\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 2\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	b39fb7a6-5d85-4966-8041-60e3009f787a	2015-07-31 17:35:52.293678
635	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 31\nsource_id: 251\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7578406-7578406 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2ceaa7bb-a242-4922-a790-27b74d869a7b	2015-07-31 17:35:52.336616
636	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast tumors in a mouse model with R175H mutation are more responsive\n  to doxorubicin than breast tumors with wild type TP53.\ndisease_id: 31\nsource_id: 252\nvariant_id: 116\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7578406-7578406 (C->T)\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3ee446db-42d2-4d18-a619-dc9ed91b0378	2015-07-31 17:35:52.380676
637	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 31\nsource_id: 251\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577538-7577538 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	3e508ca6-406b-4d2d-90be-4e1fac6ced44	2015-07-31 17:35:52.430279
638	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\ndisease_id: 31\nsource_id: 253\nvariant_id: 117\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577538-7577538 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	e2b61c9b-7a7a-4b0d-bc20-10706f61b533	2015-07-31 17:35:52.47025
639	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring R248W mutation, the prognosis is\n  worse than any other hotspot TP53 mutation, as well as worse than patients with\n  wild type TP53.\ndisease_id: 31\nsource_id: 251\nvariant_id: 118\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577539-7577539 (G->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f1df70d6-861b-46bf-a46c-1da72c61530e	2015-07-31 17:35:52.499368
640	417	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in conserved\n  regions such as R249 are prognostic for a worse overall survival compared to those\n  harboring wild-type TP53.\ndisease_id: 31\nsource_id: 253\nvariant_id: 119\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577535-7577535 (C->G)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	6720c921-718b-4fb6-8c15-9923a26b3540	2015-07-31 17:35:52.528744
698	61	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '209113112'\nstop:\n- \n- '209113112'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	59da2265-3edd-422c-a234-f8d26ce8f74f	2015-07-31 17:37:42.8554
641	418	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in conserved\n  regions such as R249 are prognostic for a worse overall survival compared to those\n  harboring wild-type TP53.\ndisease_id: 31\nsource_id: 253\nvariant_id: 120\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577536-7577536 (T->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	2a3b7d07-23dc-4335-bcb2-7717fd03b669	2015-07-31 17:35:52.557913
642	419	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 31\nsource_id: 251\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577121-7577121 (G->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	046e77e1-bdbd-43df-9b3e-561ad0df2397	2015-07-31 17:35:52.593426
643	420	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients  harboring TP53 mutation, mutations in DNA\n  contact regions such as R273 are prognostic for a worse relapse-free survival compared\n  to other non-silent mutations in TP53.\ndisease_id: 31\nsource_id: 253\nvariant_id: 121\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577121-7577121 (G->A)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	817f5452-7771-4982-a215-abe92a1c6ac4	2015-07-31 17:35:52.623262
644	421	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\ndisease_id: 31\nsource_id: 251\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577120-7577120 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	9cb4a339-ba9c-443f-a70d-031ac232003c	2015-07-31 17:35:52.659006
645	422	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\ndisease_id: 31\nsource_id: 253\nvariant_id: 122\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 17:7577120-7577120 (C->T)\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8ed0189e-d789-44d1-aa8f-5fbdc586a652	2015-07-31 17:35:52.689685
646	423	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Breast tumors with R175H mutations are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\ndisease_id: 31\nsource_id: 253\nvariant_id: 123\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	bc479432-314c-4f07-ba7b-6b3f3b94b668	2015-07-31 17:35:52.719296
647	424	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A patient with metastatic bladder cancer that responded well to the mTOR\n  inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\ndisease_id: 51\nsource_id: 254\nvariant_id: 124\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f4f06626-a14b-498c-8815-6cb4efe63dd9	2015-07-31 17:35:52.762349
648	425	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity\n  to everolimus than their wild-type counterparts.\ndisease_id: 27\nsource_id: 254\nvariant_id: 126\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	ef731a1c-1f2a-468b-afb3-613b5c676cca	2015-07-31 17:35:52.80053
649	426	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Age, sex, FAB subtype and karyotypes were not statistically significant\n  between AML patients with U2AF Q157P/R mutations and those who harbor wild type\n  U2AF.\ndisease_id: 13\nsource_id: 255\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	679d816d-e46c-40a0-8df3-f738a07b6c9f	2015-07-31 17:35:52.851867
650	427	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, those who harbor Q157P/R mutation of U2AF do not\n  show statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\ndisease_id: 13\nsource_id: 255\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	87e98797-e630-44e3-8dac-b5813d81c1ec	2015-07-31 17:35:52.883365
651	428	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not\n  show statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\ndisease_id: 45\nsource_id: 255\nvariant_id: 127\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8c3022a1-6d6a-44c5-a953-14525998a2a0	2015-07-31 17:35:52.914157
652	429	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: After adjusting for age and cancer stage, presence of U2AF mutation such\n  as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.\ndisease_id: 45\nsource_id: 256\nvariant_id: 127\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	42dbb76e-663a-4374-a8a2-9fdb5c372947	2015-07-31 17:35:52.973092
653	430	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Age, sex, FAB subtype and karyotypes were not statistically significant\n  between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\ndisease_id: 13\nsource_id: 255\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	afbb8567-4018-428f-9f14-0f7a7b5ce3f5	2015-07-31 17:35:53.004647
699	63	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '15'\nstart:\n- \n- '90631838'\nstop:\n- \n- '90631838'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	f295b4c4-0fdb-4983-a7e8-f6d2b74c1edb	2015-07-31 17:37:42.870884
654	431	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, complete remission rates are not different between\n  patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\ndisease_id: 13\nsource_id: 255\nvariant_id: 128\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	27ffbfd7-392f-4afa-8af5-a4a298ed35db	2015-07-31 17:35:53.036153
655	432	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with AML, complete remission rates are not different between\n  patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\ndisease_id: 45\nsource_id: 255\nvariant_id: 128\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 1\nclinical_significance: 6\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	af81a54c-a6db-4eb9-97a1-31747a5280dc	2015-07-31 17:35:53.066525
656	433	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: After adjusting for age and cancer stage, the presence of U2AF mutations\n  such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\ndisease_id: 45\nsource_id: 256\nvariant_id: 128\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 4\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 3\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	eb57b887-bf48-45ca-9921-4da8775b59af	2015-07-31 17:35:53.097996
657	434	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\ndisease_id: 12\nsource_id: 257\nvariant_id: 79\nrating: 2\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: 12:25398284-25398284 (C->T)\nevidence_level: 3\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 1\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8ab3d766-87f3-4cd4-b8e3-a36543161a44	2015-07-31 17:35:53.133434
658	435	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase\n  inhibitors Imatinib and GW-2580\ndisease_id: 52\nsource_id: 258\nvariant_id: 30\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	f918792a-037d-40c9-9fc2-64af50c18e50	2015-07-31 17:35:53.17929
659	436	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase\n  inhibitors Imatinib and GW-2580\ndisease_id: 52\nsource_id: 258\nvariant_id: 84\nrating: 3\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 2\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	46e1cc19-4b3b-4ff9-8bb3-45ae61e598e4	2015-07-31 17:35:53.221286
660	437	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA\n  resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\ndisease_id: 53\nsource_id: 259\nvariant_id: 31\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	df5febef-b9ca-4e17-b8fe-6289f73a0c23	2015-07-31 17:35:53.264334
661	438	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA\n  resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\ndisease_id: 53\nsource_id: 259\nvariant_id: 109\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 0\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	8027883e-607b-422c-a3b1-ba29e5955a5f	2015-07-31 17:35:53.293868
662	439	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: NSCLC patients with exon 19 deletions in EGFR had longer overall survival\n  than patients with an L858R mutation when treated with gefitinib and erlotinib (38\n  versus 17 months; P = 0.04).\ndisease_id: 12\nsource_id: 260\nvariant_id: 133\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 2\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 0\ndeleted: false\ndeleted_at: \ndrug_interaction_type: 2\n	1	\N	\N	6702705a-8a6a-4f3c-bb3e-6e036d5ed0c1	2015-07-31 17:35:53.35104
663	440	EvidenceItem	\N	\N	\N	\N	\N	create	---\ndescription: In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions\n  had longer overall survival than those with other types of EWS-FLI1 fusions.\ndisease_id: 54\nsource_id: 261\nvariant_id: 151\nrating: 4\nstatus: accepted\nremote_errors: \nremote_ids: \nvariant_hgvs: N/A\nevidence_level: 1\nevidence_type: 1\nvariant_origin: 0\nevidence_direction: 0\nclinical_significance: 4\ndeleted: false\ndeleted_at: \ndrug_interaction_type: \n	1	\N	\N	f6f16d41-a5bb-443b-9f9f-910eb43b2b2e	2015-07-31 17:35:53.402999
664	5	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'While imatinib has shown to be incredibly successful in treating philadelphia chromosome\n  positive CML, patients that have shown primary or secondary resistance to the drug\n  have been observed to harbor T315I and E255K ABL kinase domain mutations. These\n  mutations, among others, have been observed both in primary refractory disease and\n  acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations\n  have also been shown to confer resistance to imatinib. '\n	2	\N	\N	2c23b9d9-0783-4814-8605-e980ee0a2c20	2015-07-31 17:35:53.510963
665	14	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset\n  of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase\n  the activity of the ALK oncogene relative to normal cells. While EML4 is the most\n  common fusion partner, other 5'' partners have been observed. The EML4-ALK fusion\n  has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. '\n	2	\N	\N	8e944d3a-bfba-475d-b9ab-721941a81250	2015-07-31 17:35:53.524854
666	15	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine\n  kinase inhibitors have been developed to target EGFR pathway activity. One such\n  inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting.\n  However, the T790M missense mutation has shown to confer resistance to this inhibitor\n  in cell lines and case studies.  '\n	2	\N	\N	e03ad381-6423-47df-81a9-ef88db2a59ab	2015-07-31 17:35:53.536933
667	6	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'The ALK oncogene has long been considered a driving factor in non-small cell lung\n  cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be\n  effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance\n  to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive\n  this resistance. '\n	2	\N	\N	ce265927-e498-46ee-81fc-e743d0d2a444	2015-07-31 17:35:53.549142
668	8	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants\n  have also been observed and studied to a lesser degree. At first approximation,\n  many of these variants seem to behave similarly to V600E, and treatment with dabrafenib\n  has been shown to be effective. '\n	2	\N	\N	a297cd8e-a89e-4a0b-9c9f-731520ead8cf	2015-07-31 17:35:53.564262
669	16	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'HER2-positive breast cancer is widely recognized as a molecular subtype of breast\n  cancer. While amplification of ERBB2 is a common mechanism for activation of the\n  pathway, missense mutations have also demonstrated activation potential. These activating\n  missense mutations also confer senstivity to the targeted therapeutic neratinib. '\n	2	\N	\N	f0779cb5-a6e0-4c52-acc5-d4944ada7f58	2015-07-31 17:35:53.576862
670	17	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone\n  therapy has been widely effective in treating the disease, however sequencing of\n  resistant patients has uncovered a number of missense mutations in mediating this\n  resistance. Many of these mutations lie within the ligand-binding domain, and contribute\n  to constitutive activity of the receptor. This has lead to the development of estrogen\n  receptor degrading agents such as fulvestrant, which have shown early efficacy in\n  clinical trials.  '\n	2	\N	\N	f9a05c58-bff9-4f93-9897-e77ccbff7ac0	2015-07-31 17:35:53.590447
671	10	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'FGFR fusions have been demonstrated across many cancer types to have oncogenic\n  potential. FGFR2 and FGFR3 have been shown to be fused to many 3'' partners, which\n  may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2\n  fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib\n  and pazopanib have shown efficacy in case studies.  '\n	2	\N	\N	5218ff48-8f31-40d1-9149-24d3027fb9ed	2015-07-31 17:35:53.602972
672	18	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'PTEN loss is a common event in breast cancer, as well as others. The most common\n  single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell\n  lines harboring this mutation behave similarly to cell lines harboring larger insertion\n  or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded\n  growth. '\n	2	\N	\N	987ad2b5-f472-4fd3-979a-1e65917bf9c6	2015-07-31 17:35:53.617195
673	9	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment\n  of these patients with the targeted therapeutic motesanib has shown to be effective.\n  However, the missense mutations C634W and M918T have shown to confer motesanib resistance\n  in cell lines. '\n	2	\N	\N	e72e0938-e029-4acf-82b7-d45956034d56	2015-07-31 17:35:53.630472
674	12	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types.\n  Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor\n  everolimus has seen significant anti-tumorigenic activity. '\n	2	\N	\N	e8f806d0-38eb-41bf-ac74-8341fd529da3	2015-07-31 17:35:53.648734
675	13	VariantGroup	\N	\N	\N	\N	\N	update	---\ndescription:\n- \n- 'BRCA germline variants have long been recognized as important potential predictors\n  of breast and ovarian cancer risk. '\n	2	\N	\N	c7f4bf04-dc76-45bc-bcb7-7b11953d1eb6	2015-07-31 17:35:53.662129
676	4	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '14'\nstart:\n- \n- '105246551'\nstop:\n- \n- '105246551'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	b875e4a2-9b12-4ae1-8b91-1efe8fc4faae	2015-07-31 17:37:42.436488
677	8	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '29443695'\nstop:\n- \n- '29443695'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	5bb29760-82f2-4c7f-836a-23d150740258	2015-07-31 17:37:42.460806
678	9	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '29432664'\nstop:\n- \n- '29432664'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	bc75b1ef-276f-42a4-a0ba-562b08d501cd	2015-07-31 17:37:42.493064
679	10	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- X\nstart:\n- \n- '47426121'\nstop:\n- \n- '47426121'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	b28068dd-52b5-4074-baa8-2acbafdd8daa	2015-07-31 17:37:42.509483
680	11	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453135'\nstop:\n- \n- '140453136'\nreference_bases:\n- \n- CA\nvariant_bases:\n- \n- AT\n	2	\N	\N	a398eb83-09de-4c24-8503-2e4c193ed6f4	2015-07-31 17:37:42.531389
681	12	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453136'\nstop:\n- \n- '140453136'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	fa50b11d-0534-4996-8015-edf0efcb0ace	2015-07-31 17:37:42.548662
682	15	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453137'\nstop:\n- \n- '140453137'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	65f04696-b1b2-4d84-83ca-50aee3df4e3d	2015-07-31 17:37:42.571565
683	16	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '140453136'\nstop:\n- \n- '140453137'\nreference_bases:\n- \n- AC\nvariant_bases:\n- \n- CT\n	2	\N	\N	c4366a95-d851-482b-9b6e-5f25cc0f978a	2015-07-31 17:37:42.588699
684	33	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '55259514'\nstop:\n- \n- '55259515'\nreference_bases:\n- \n- CT\nvariant_bases:\n- \n- AG\n	2	\N	\N	3dbd2185-16d4-4139-a6cd-a7f5e430791b	2015-07-31 17:37:42.614363
685	34	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '7'\nstart:\n- \n- '55249071'\nstop:\n- \n- '55249071'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	e2fb11ef-7879-42d9-a999-54dad6467c12	2015-07-31 17:37:42.630777
686	35	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880261'\nstop:\n- \n- '37880261'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- C\n	2	\N	\N	738d7a05-f139-49d7-bfec-29b52990f4f1	2015-07-31 17:37:42.648684
687	36	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880261'\nstop:\n- \n- '37880261'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	27ea05a3-b34b-443c-acd3-57e935930918	2015-07-31 17:37:42.664957
688	38	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37868205'\nstop:\n- \n- '37868205'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- C\n	2	\N	\N	23778ce1-6945-49ca-aace-839c3ff9f7c9	2015-07-31 17:37:42.682932
689	39	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37880220'\nstop:\n- \n- '37880220'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	29e30ddd-80d8-40dc-b9a9-9c6282eb67f0	2015-07-31 17:37:42.701456
690	43	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881616'\nstop:\n- \n- '37881616'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	3c277235-0a47-454d-b69e-decd9f6e1d4f	2015-07-31 17:37:42.721541
691	44	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '37881000'\nstop:\n- \n- '37881000'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	49ab37d4-a76c-43b4-b965-84620a5a82bb	2015-07-31 17:37:42.738932
712	98	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152092'\nstop:\n- \n- '55152093'\nreference_bases:\n- \n- GA\nvariant_bases:\n- \n- AT\n	2	\N	\N	a75439b6-e94f-41c4-a22d-ea80843a25f7	2015-07-31 17:37:43.126831
713	99	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152093'\nstop:\n- \n- '55152093'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- T\n	2	\N	\N	36ed7a75-8f50-4aca-aadd-38bd57bceb65	2015-07-31 17:37:43.147713
714	100	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '4'\nstart:\n- \n- '55152092'\nstop:\n- \n- '55152092'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- T\n	2	\N	\N	8d0b927e-8770-4983-a6c5-93361f4dd193	2015-07-31 17:37:43.168801
715	103	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178936082'\nstop:\n- \n- '178936082'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	78ac1d72-a19a-4679-a7c5-302400405872	2015-07-31 17:37:43.186034
716	104	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178936091'\nstop:\n- \n- '178936091'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	d1f359c4-4394-438b-a034-e82504445471	2015-07-31 17:37:43.205715
717	107	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '3'\nstart:\n- \n- '178952085'\nstop:\n- \n- '178952085'\nreference_bases:\n- \n- A\nvariant_bases:\n- \n- G\n	2	\N	\N	68a44920-15d1-48b6-8f2b-e358c018e1f6	2015-07-31 17:37:43.221469
718	110	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '89717672'\nstop:\n- \n- '89717672'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	c0a4c69f-d1e7-4583-a4b5-46ae4a6cbaed	2015-07-31 17:37:43.239029
719	112	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '43609950'\nstop:\n- \n- '43609950'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	169926e0-56a5-454f-a837-e929b1a14594	2015-07-31 17:37:43.25974
720	113	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '10'\nstart:\n- \n- '43617416'\nstop:\n- \n- '43617416'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	da867907-3849-46ad-9e64-af1d903af461	2015-07-31 17:37:43.275295
721	114	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '198267359'\nstop:\n- \n- '198267359'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- A\n	2	\N	\N	b5a7b6ae-9f2d-4eb4-8b2d-93290e5c650c	2015-07-31 17:37:43.289838
722	115	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '2'\nstart:\n- \n- '198266834'\nstop:\n- \n- '198266834'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- C\n	2	\N	\N	e6fafc8f-c0ef-4d14-80cb-91b08dd61a5a	2015-07-31 17:37:43.305442
723	116	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7578406'\nstop:\n- \n- '7578406'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	fb5a659f-9e59-44dd-85fb-4e1e4d42cc27	2015-07-31 17:37:43.320627
724	117	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577538'\nstop:\n- \n- '7577538'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	5397b8ef-63f1-40c0-b60e-4b390ee2da6d	2015-07-31 17:37:43.334874
725	118	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577539'\nstop:\n- \n- '7577539'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	5010a141-03a6-4248-8fb0-09b335aa0d8b	2015-07-31 17:37:43.353121
726	119	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577535'\nstop:\n- \n- '7577535'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- G\n	2	\N	\N	7983c8cd-d86b-4d24-8cb1-45873f7d21c3	2015-07-31 17:37:43.368432
727	120	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577536'\nstop:\n- \n- '7577536'\nreference_bases:\n- \n- T\nvariant_bases:\n- \n- A\n	2	\N	\N	30bbe031-742a-4e39-b2eb-dbb1003a8073	2015-07-31 17:37:43.381857
728	121	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577121'\nstop:\n- \n- '7577121'\nreference_bases:\n- \n- G\nvariant_bases:\n- \n- A\n	2	\N	\N	b0324f26-d929-4600-9159-f4568dd28797	2015-07-31 17:37:43.396438
729	122	Variant	\N	\N	\N	\N	\N	update	---\nchromosome:\n- \n- '17'\nstart:\n- \n- '7577120'\nstop:\n- \n- '7577120'\nreference_bases:\n- \n- C\nvariant_bases:\n- \n- T\n	2	\N	\N	0b2b9111-f5ca-4f1f-8e08-9752a59490ef	2015-07-31 17:37:43.410984
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 729, true);


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
1	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5666a83b-82bb-49bd-a258-0bf608e06d14\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/1\n	\N	2015-07-31 17:34:13.898443	\N	\N	\N	default	2015-07-31 17:34:13.898595	2015-07-31 17:34:13.898595
2	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 454920e0-2a76-443f-bc72-600c26e4a3cd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/2\n	\N	2015-07-31 17:34:13.920577	\N	\N	\N	default	2015-07-31 17:34:13.920732	2015-07-31 17:34:13.920732
3	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4b5e4298-bde6-4aa6-9797-ce0ded465b66\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/3\n	\N	2015-07-31 17:34:14.169391	\N	\N	\N	default	2015-07-31 17:34:14.169508	2015-07-31 17:34:14.169508
4	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 273ac013-436d-4d84-ae29-ab4e645ab0ea\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/4\n	\N	2015-07-31 17:34:14.186389	\N	\N	\N	default	2015-07-31 17:34:14.186524	2015-07-31 17:34:14.186524
5	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c8e76647-f4c8-48a2-9a25-54fd125f13f2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/5\n	\N	2015-07-31 17:34:14.461212	\N	\N	\N	default	2015-07-31 17:34:14.461531	2015-07-31 17:34:14.461531
6	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a79f1e70-b422-4440-a2ca-005572b52b3b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/6\n	\N	2015-07-31 17:34:14.475773	\N	\N	\N	default	2015-07-31 17:34:14.475967	2015-07-31 17:34:14.475967
7	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0cd59cee-0fa9-4300-b214-88eaa4c6f4f0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/7\n	\N	2015-07-31 17:34:14.721462	\N	\N	\N	default	2015-07-31 17:34:14.721573	2015-07-31 17:34:14.721573
8	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1455bb27-b9eb-45e0-896a-93fc7b37181e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/8\n	\N	2015-07-31 17:34:14.732187	\N	\N	\N	default	2015-07-31 17:34:14.732302	2015-07-31 17:34:14.732302
9	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 83ceb1e6-5d99-4c30-8264-176a78804c7c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/9\n	\N	2015-07-31 17:34:14.971893	\N	\N	\N	default	2015-07-31 17:34:14.972035	2015-07-31 17:34:14.972035
10	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f1b0e851-e75c-4a98-acb0-52c5a2d8f9bf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/10\n	\N	2015-07-31 17:34:14.981739	\N	\N	\N	default	2015-07-31 17:34:14.982139	2015-07-31 17:34:14.982139
11	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a77a7566-aeac-4c04-b919-2a184831818c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/11\n	\N	2015-07-31 17:34:15.240642	\N	\N	\N	default	2015-07-31 17:34:15.24083	2015-07-31 17:34:15.24083
12	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 32154bc1-374f-4534-ac5b-4589b3f80f7e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/12\n	\N	2015-07-31 17:34:15.745455	\N	\N	\N	default	2015-07-31 17:34:15.745583	2015-07-31 17:34:15.745583
13	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 21efe954-94bc-44e5-ad5a-8cfbc5983661\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/13\n	\N	2015-07-31 17:34:15.757334	\N	\N	\N	default	2015-07-31 17:34:15.757448	2015-07-31 17:34:15.757448
14	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2716a3c6-632d-40b2-ad3e-cc7afc71b92b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/14\n	\N	2015-07-31 17:34:16.373291	\N	\N	\N	default	2015-07-31 17:34:16.373413	2015-07-31 17:34:16.373413
15	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 96be1b37-a98c-4ec2-b9bc-7e62bfdaab5b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/15\n	\N	2015-07-31 17:34:16.581124	\N	\N	\N	default	2015-07-31 17:34:16.581316	2015-07-31 17:34:16.581316
16	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e728658b-2b81-4b2b-ada6-9d2943df47c4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/16\n	\N	2015-07-31 17:34:16.592143	\N	\N	\N	default	2015-07-31 17:34:16.592279	2015-07-31 17:34:16.592279
17	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c93feb3a-1c56-49f2-a6b1-e553e4e33b73\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/17\n	\N	2015-07-31 17:34:17.098088	\N	\N	\N	default	2015-07-31 17:34:17.098198	2015-07-31 17:34:17.098198
18	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 94e63250-67bf-4818-a27e-1eaa898c50e7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/18\n	\N	2015-07-31 17:34:17.109655	\N	\N	\N	default	2015-07-31 17:34:17.109843	2015-07-31 17:34:17.109843
19	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bb61214a-331e-43cb-aad3-e4668e0edb26\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/19\n	\N	2015-07-31 17:34:17.359507	\N	\N	\N	default	2015-07-31 17:34:17.359641	2015-07-31 17:34:17.359641
20	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 31f370cd-86e9-4bec-ba3d-23420f6557f0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/20\n	\N	2015-07-31 17:34:17.369609	\N	\N	\N	default	2015-07-31 17:34:17.36972	2015-07-31 17:34:17.36972
21	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e969171c-80ea-4755-854d-be1f3d2f6b1f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/21\n	\N	2015-07-31 17:34:17.379363	\N	\N	\N	default	2015-07-31 17:34:17.379474	2015-07-31 17:34:17.379474
22	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 45fb5707-05b2-4731-8213-347d4e99afe9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/22\n	\N	2015-07-31 17:34:17.601972	\N	\N	\N	default	2015-07-31 17:34:17.602144	2015-07-31 17:34:17.602144
23	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5c6a6a30-e8a2-4acf-8f53-682e37f74dfd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/23\n	\N	2015-07-31 17:34:17.817707	\N	\N	\N	default	2015-07-31 17:34:17.81784	2015-07-31 17:34:17.81784
24	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9f15e655-cbb0-4397-aeff-d0f845542c2e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/24\n	\N	2015-07-31 17:34:18.039862	\N	\N	\N	default	2015-07-31 17:34:18.039979	2015-07-31 17:34:18.039979
25	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d74ef98c-8860-4816-9464-41d4c569055e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/25\n	\N	2015-07-31 17:34:18.049196	\N	\N	\N	default	2015-07-31 17:34:18.049339	2015-07-31 17:34:18.049339
26	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3fc3627a-50d6-4df2-a6a7-e96f6c9030cb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/26\n	\N	2015-07-31 17:34:18.292076	\N	\N	\N	default	2015-07-31 17:34:18.292191	2015-07-31 17:34:18.292191
27	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 907e1e9a-16d7-4ff3-a7db-dad27041ac50\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/27\n	\N	2015-07-31 17:34:18.301917	\N	\N	\N	default	2015-07-31 17:34:18.302032	2015-07-31 17:34:18.302032
28	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a428a8d9-e4c2-4a24-b1d6-c6f5b9a9d901\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/28\n	\N	2015-07-31 17:34:18.552561	\N	\N	\N	default	2015-07-31 17:34:18.552692	2015-07-31 17:34:18.552692
29	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3532d27f-2a1d-42b5-9e48-7843f94b0737\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/29\n	\N	2015-07-31 17:34:18.563478	\N	\N	\N	default	2015-07-31 17:34:18.563608	2015-07-31 17:34:18.563608
30	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5ffa5f28-89ba-4a5f-9644-056a65202f62\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/30\n	\N	2015-07-31 17:34:18.797708	\N	\N	\N	default	2015-07-31 17:34:18.797823	2015-07-31 17:34:18.797823
31	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2935a031-27e3-42c5-85a3-de36b9977021\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/31\n	\N	2015-07-31 17:34:18.807531	\N	\N	\N	default	2015-07-31 17:34:18.807653	2015-07-31 17:34:18.807653
32	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6a29f147-893a-4d23-8a83-9df39f7e652b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/32\n	\N	2015-07-31 17:34:19.080901	\N	\N	\N	default	2015-07-31 17:34:19.081032	2015-07-31 17:34:19.081032
33	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4386081d-e50e-4e62-bbe3-2ac441a6736f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/33\n	\N	2015-07-31 17:34:19.092651	\N	\N	\N	default	2015-07-31 17:34:19.092767	2015-07-31 17:34:19.092767
34	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 239e53be-2fb9-4661-9a41-baf355746db7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/34\n	\N	2015-07-31 17:34:19.558904	\N	\N	\N	default	2015-07-31 17:34:19.559046	2015-07-31 17:34:19.559046
35	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6e5bba4d-15ca-43eb-b6c9-f97e2df61ff4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/35\n	\N	2015-07-31 17:34:19.791341	\N	\N	\N	default	2015-07-31 17:34:19.791458	2015-07-31 17:34:19.791458
36	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 99750a47-5438-44c2-b4e2-b7b049b7b468\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/36\n	\N	2015-07-31 17:34:19.80073	\N	\N	\N	default	2015-07-31 17:34:19.800843	2015-07-31 17:34:19.800843
37	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e05a99ef-898b-4152-affa-d5efc41bffcf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/37\n	\N	2015-07-31 17:34:20.041023	\N	\N	\N	default	2015-07-31 17:34:20.041139	2015-07-31 17:34:20.041139
38	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 68fae287-b306-4c74-b40b-2907afe10f47\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/38\n	\N	2015-07-31 17:34:20.050651	\N	\N	\N	default	2015-07-31 17:34:20.050761	2015-07-31 17:34:20.050761
39	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 05ea17d5-ebf1-46fa-a9a3-d49e76085871\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/39\n	\N	2015-07-31 17:34:20.302326	\N	\N	\N	default	2015-07-31 17:34:20.302462	2015-07-31 17:34:20.302462
40	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3c607936-2622-43be-8ef6-03c12896dcde\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/40\n	\N	2015-07-31 17:34:20.517708	\N	\N	\N	default	2015-07-31 17:34:20.517823	2015-07-31 17:34:20.517823
41	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f725c4d9-a208-4774-98a3-b5da857c7c6c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/41\n	\N	2015-07-31 17:34:20.729618	\N	\N	\N	default	2015-07-31 17:34:20.729739	2015-07-31 17:34:20.729739
42	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cabb2034-8e35-4dd6-a5cf-193ab65da8b7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/42\n	\N	2015-07-31 17:34:20.990988	\N	\N	\N	default	2015-07-31 17:34:20.991119	2015-07-31 17:34:20.991119
43	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 81e4119b-f1df-4a3d-bb10-c077f96f4f85\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/43\n	\N	2015-07-31 17:34:22.0534	\N	\N	\N	default	2015-07-31 17:34:22.053641	2015-07-31 17:34:22.053641
44	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6edbe606-09d7-4848-a5dd-e15f988aa000\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/44\n	\N	2015-07-31 17:34:23.575788	\N	\N	\N	default	2015-07-31 17:34:23.575904	2015-07-31 17:34:23.575904
45	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fc724450-8a08-40a3-bb71-4d4d0c9d3cf2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/45\n	\N	2015-07-31 17:34:23.585343	\N	\N	\N	default	2015-07-31 17:34:23.585456	2015-07-31 17:34:23.585456
46	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7a87676b-d7f7-4a0f-be1c-bb7a72bea57e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/46\n	\N	2015-07-31 17:34:24.096107	\N	\N	\N	default	2015-07-31 17:34:24.096224	2015-07-31 17:34:24.096224
47	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9fd9652a-d0fb-488b-9528-0cda12f2073d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/47\n	\N	2015-07-31 17:34:24.105733	\N	\N	\N	default	2015-07-31 17:34:24.105932	2015-07-31 17:34:24.105932
48	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f67859ad-e72e-4908-851d-899b64359586\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/48\n	\N	2015-07-31 17:34:24.341699	\N	\N	\N	default	2015-07-31 17:34:24.341836	2015-07-31 17:34:24.341836
49	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 95920f61-69c7-46ee-9666-8b3212fdcc08\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/49\n	\N	2015-07-31 17:34:25.030868	\N	\N	\N	default	2015-07-31 17:34:25.030994	2015-07-31 17:34:25.030994
50	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d6d8dbd9-7f42-4144-9eac-b2032bbc8c15\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/50\n	\N	2015-07-31 17:34:25.042823	\N	\N	\N	default	2015-07-31 17:34:25.042955	2015-07-31 17:34:25.042955
75	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7c340779-46b5-4a87-81d9-552df76f01c6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/75\n	\N	2015-07-31 17:35:37.248541	\N	\N	\N	default	2015-07-31 17:35:37.248915	2015-07-31 17:35:37.248915
76	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cbbd1132-414d-4f8d-a95c-775f71763f67\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/76\n	\N	2015-07-31 17:35:37.330622	\N	\N	\N	default	2015-07-31 17:35:37.330736	2015-07-31 17:35:37.330736
77	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3a2bbb5c-d680-413b-992b-7f8ae7385630\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/77\n	\N	2015-07-31 17:35:37.373521	\N	\N	\N	default	2015-07-31 17:35:37.373645	2015-07-31 17:35:37.373645
78	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6e6b8715-bb04-4e8f-b315-a3a7be647c8b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/78\n	\N	2015-07-31 17:35:37.516172	\N	\N	\N	default	2015-07-31 17:35:37.516291	2015-07-31 17:35:37.516291
79	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 90e581a3-73b4-4fa3-b437-78b985cb9972\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/79\n	\N	2015-07-31 17:35:37.56376	\N	\N	\N	default	2015-07-31 17:35:37.563987	2015-07-31 17:35:37.563987
80	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fb47b042-9690-4daa-902b-38dd9fe0930c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/80\n	\N	2015-07-31 17:35:37.674406	\N	\N	\N	default	2015-07-31 17:35:37.674517	2015-07-31 17:35:37.674517
81	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5164090b-e9dd-4304-bbc3-960ad17226fb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/81\n	\N	2015-07-31 17:35:37.719431	\N	\N	\N	default	2015-07-31 17:35:37.719544	2015-07-31 17:35:37.719544
82	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 894543a2-c377-4929-8e33-b89f52157e11\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/82\n	\N	2015-07-31 17:35:37.825658	\N	\N	\N	default	2015-07-31 17:35:37.825925	2015-07-31 17:35:37.825925
83	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f666e455-a05b-460d-a3e2-e88ff0dfc56c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/83\n	\N	2015-07-31 17:35:37.870765	\N	\N	\N	default	2015-07-31 17:35:37.870895	2015-07-31 17:35:37.870895
84	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 525d4928-bd71-4537-a3ff-78461889d7d5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/84\n	\N	2015-07-31 17:35:37.9076	\N	\N	\N	default	2015-07-31 17:35:37.907712	2015-07-31 17:35:37.907712
85	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: deccba3e-abcd-4c09-8753-74a675768f29\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/85\n	\N	2015-07-31 17:35:38.02162	\N	\N	\N	default	2015-07-31 17:35:38.021773	2015-07-31 17:35:38.021773
86	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 84e90aed-eabc-498a-b280-5658f96a142b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/86\n	\N	2015-07-31 17:35:38.064224	\N	\N	\N	default	2015-07-31 17:35:38.064338	2015-07-31 17:35:38.064338
87	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ea768b09-23eb-4188-a32c-fcb3b08a92d4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/87\n	\N	2015-07-31 17:35:38.114318	\N	\N	\N	default	2015-07-31 17:35:38.114482	2015-07-31 17:35:38.114482
88	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cbc42342-3f15-45d1-b4f8-bf44ca323397\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/88\n	\N	2015-07-31 17:35:38.158318	\N	\N	\N	default	2015-07-31 17:35:38.158429	2015-07-31 17:35:38.158429
89	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f58cf47c-4d3f-49fd-a65a-11d57990c729\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/89\n	\N	2015-07-31 17:35:38.239294	\N	\N	\N	default	2015-07-31 17:35:38.239476	2015-07-31 17:35:38.239476
90	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 637d0357-0db7-4fc2-8977-715da9277eb8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/90\n	\N	2015-07-31 17:35:38.303117	\N	\N	\N	default	2015-07-31 17:35:38.303226	2015-07-31 17:35:38.303226
91	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 855243aa-5665-4f76-9e40-307078a7c87f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/91\n	\N	2015-07-31 17:35:38.347174	\N	\N	\N	default	2015-07-31 17:35:38.347285	2015-07-31 17:35:38.347285
92	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ddbb6ca8-527f-4aaa-9570-7827a2e49d6d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/92\n	\N	2015-07-31 17:35:38.401145	\N	\N	\N	default	2015-07-31 17:35:38.401278	2015-07-31 17:35:38.401278
93	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9ec9ad2a-80da-4025-8bb2-442ab7b2df3e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/93\n	\N	2015-07-31 17:35:38.489404	\N	\N	\N	default	2015-07-31 17:35:38.489519	2015-07-31 17:35:38.489519
94	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a7770e3a-8496-43a0-a386-57b85647eba3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/94\n	\N	2015-07-31 17:35:38.52926	\N	\N	\N	default	2015-07-31 17:35:38.529371	2015-07-31 17:35:38.529371
95	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ebe1436c-c489-4f5e-918a-0d23bf1b906c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/95\n	\N	2015-07-31 17:35:38.575116	\N	\N	\N	default	2015-07-31 17:35:38.575297	2015-07-31 17:35:38.575297
96	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 26ff3cd2-b6fe-4cf6-9697-cf54cd361f86\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/96\n	\N	2015-07-31 17:35:38.729837	\N	\N	\N	default	2015-07-31 17:35:38.730025	2015-07-31 17:35:38.730025
97	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0dcb1fab-4ec0-4c1d-b10b-9c9beba9fe20\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/97\n	\N	2015-07-31 17:35:38.774888	\N	\N	\N	default	2015-07-31 17:35:38.775062	2015-07-31 17:35:38.775062
98	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 17a6850e-e38e-4e73-af65-01acdaef7c3b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/98\n	\N	2015-07-31 17:35:38.813425	\N	\N	\N	default	2015-07-31 17:35:38.813549	2015-07-31 17:35:38.813549
99	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d4da70a5-f9e6-4eed-a02d-edd2c658e8d8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/99\n	\N	2015-07-31 17:35:39.07264	\N	\N	\N	default	2015-07-31 17:35:39.072767	2015-07-31 17:35:39.072767
100	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5c81266e-1674-48ce-b70b-dad5533fa94a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/100\n	\N	2015-07-31 17:35:39.251612	\N	\N	\N	default	2015-07-31 17:35:39.251721	2015-07-31 17:35:39.251721
101	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f9bee199-32f7-4b66-9239-d58dca40d065\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/101\n	\N	2015-07-31 17:35:39.316558	\N	\N	\N	default	2015-07-31 17:35:39.316783	2015-07-31 17:35:39.316783
102	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8d06145b-6363-499a-ba3d-e34577c23a8d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/102\n	\N	2015-07-31 17:35:39.495125	\N	\N	\N	default	2015-07-31 17:35:39.495236	2015-07-31 17:35:39.495236
103	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 24dc2ef9-d8c5-4b87-8fe1-c655944f25a6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/103\n	\N	2015-07-31 17:35:39.588942	\N	\N	\N	default	2015-07-31 17:35:39.58906	2015-07-31 17:35:39.58906
104	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d167b87d-6d8d-4c77-b877-5c6bf711c64b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/104\n	\N	2015-07-31 17:35:39.664125	\N	\N	\N	default	2015-07-31 17:35:39.66424	2015-07-31 17:35:39.66424
105	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 615f5452-5ed5-4f10-b18e-78fde4ced6cd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/105\n	\N	2015-07-31 17:35:39.71272	\N	\N	\N	default	2015-07-31 17:35:39.71283	2015-07-31 17:35:39.71283
106	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1229ea2d-8b46-4244-9ce5-a55781827496\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/106\n	\N	2015-07-31 17:35:39.783429	\N	\N	\N	default	2015-07-31 17:35:39.783539	2015-07-31 17:35:39.783539
107	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 03fd654b-6876-460d-8829-22934cc3bb6f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/107\n	\N	2015-07-31 17:35:39.825552	\N	\N	\N	default	2015-07-31 17:35:39.825662	2015-07-31 17:35:39.825662
108	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 33457469-ba23-4ecf-a045-b5c621ee06bb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/108\n	\N	2015-07-31 17:35:39.867046	\N	\N	\N	default	2015-07-31 17:35:39.867237	2015-07-31 17:35:39.867237
109	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3814d9db-2993-4619-9a81-c5d0217a5406\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/109\n	\N	2015-07-31 17:35:40.068104	\N	\N	\N	default	2015-07-31 17:35:40.068237	2015-07-31 17:35:40.068237
110	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 07280542-dd54-4583-9aa4-6cb4d24c117a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/110\n	\N	2015-07-31 17:35:40.126361	\N	\N	\N	default	2015-07-31 17:35:40.126475	2015-07-31 17:35:40.126475
111	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b10d4205-cafb-47d2-b7aa-e543d5433f23\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/111\n	\N	2015-07-31 17:35:40.186245	\N	\N	\N	default	2015-07-31 17:35:40.186372	2015-07-31 17:35:40.186372
112	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bafd60f4-c6a5-407b-8da3-e25117414261\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/112\n	\N	2015-07-31 17:35:40.239659	\N	\N	\N	default	2015-07-31 17:35:40.239793	2015-07-31 17:35:40.239793
113	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4f0db9bd-126c-4a78-ab97-88c88cce985b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/113\n	\N	2015-07-31 17:35:40.32526	\N	\N	\N	default	2015-07-31 17:35:40.325391	2015-07-31 17:35:40.325391
114	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 31b14168-b2b5-4e7f-8da2-a0586f6c10a1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/114\n	\N	2015-07-31 17:35:40.367264	\N	\N	\N	default	2015-07-31 17:35:40.367485	2015-07-31 17:35:40.367485
115	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fc6dcdd8-9b20-463e-98c4-bcff6775df2d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/115\n	\N	2015-07-31 17:35:40.412398	\N	\N	\N	default	2015-07-31 17:35:40.412532	2015-07-31 17:35:40.412532
116	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0c81f703-8d76-4bf1-9333-ac4d884e49af\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/116\n	\N	2015-07-31 17:35:40.452825	\N	\N	\N	default	2015-07-31 17:35:40.452936	2015-07-31 17:35:40.452936
117	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5154898f-5aa9-434c-aabe-9b2e3f4f900d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/117\n	\N	2015-07-31 17:35:40.4971	\N	\N	\N	default	2015-07-31 17:35:40.497214	2015-07-31 17:35:40.497214
118	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 08ae1b4a-fbeb-4cb2-96aa-64917f000234\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/118\n	\N	2015-07-31 17:35:40.54215	\N	\N	\N	default	2015-07-31 17:35:40.542333	2015-07-31 17:35:40.542333
119	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5755e48d-7b12-444a-b2a8-6130184cb0f8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/119\n	\N	2015-07-31 17:35:40.589827	\N	\N	\N	default	2015-07-31 17:35:40.589941	2015-07-31 17:35:40.589941
120	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 60853103-1116-413a-9210-6e489158002e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/120\n	\N	2015-07-31 17:35:40.632145	\N	\N	\N	default	2015-07-31 17:35:40.632319	2015-07-31 17:35:40.632319
121	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: da51773d-802c-4f55-a576-e246c844b7cc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/121\n	\N	2015-07-31 17:35:40.675621	\N	\N	\N	default	2015-07-31 17:35:40.675737	2015-07-31 17:35:40.675737
122	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9088984b-1741-4d8b-95de-424c32c55013\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/122\n	\N	2015-07-31 17:35:40.72239	\N	\N	\N	default	2015-07-31 17:35:40.722504	2015-07-31 17:35:40.722504
123	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3dcebdef-212a-42ac-9f20-c6c455c47342\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/123\n	\N	2015-07-31 17:35:40.776102	\N	\N	\N	default	2015-07-31 17:35:40.776216	2015-07-31 17:35:40.776216
124	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bd93cc38-8012-462d-bdda-aff604b16ed7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/124\n	\N	2015-07-31 17:35:40.81341	\N	\N	\N	default	2015-07-31 17:35:40.81352	2015-07-31 17:35:40.81352
125	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5cad4d98-f689-4f95-9fb1-e4d542ae7c95\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/125\n	\N	2015-07-31 17:35:40.899811	\N	\N	\N	default	2015-07-31 17:35:40.899926	2015-07-31 17:35:40.899926
126	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d0c7976e-e8a7-41e2-9e17-0d1e32bcdca8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/126\n	\N	2015-07-31 17:35:40.968928	\N	\N	\N	default	2015-07-31 17:35:40.969046	2015-07-31 17:35:40.969046
127	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 01954e13-15ba-48a9-b518-38459d30a6e7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/127\n	\N	2015-07-31 17:35:41.018097	\N	\N	\N	default	2015-07-31 17:35:41.018258	2015-07-31 17:35:41.018258
128	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f72f1aef-848d-41ad-9039-a39096922bbb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/128\n	\N	2015-07-31 17:35:41.095836	\N	\N	\N	default	2015-07-31 17:35:41.095947	2015-07-31 17:35:41.095947
129	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 66e8f671-ad8e-4d7b-aeb7-890ca64db9e1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/129\n	\N	2015-07-31 17:35:41.143041	\N	\N	\N	default	2015-07-31 17:35:41.143161	2015-07-31 17:35:41.143161
130	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3e587f1f-f587-4e37-a295-94f4268d768b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/130\n	\N	2015-07-31 17:35:41.191065	\N	\N	\N	default	2015-07-31 17:35:41.191179	2015-07-31 17:35:41.191179
131	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: da74ddd0-22c1-4db3-a065-ca90dda8e714\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/131\n	\N	2015-07-31 17:35:41.281597	\N	\N	\N	default	2015-07-31 17:35:41.281757	2015-07-31 17:35:41.281757
132	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1d0c5f6a-6a2d-473e-b17d-7e569668ff13\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/132\n	\N	2015-07-31 17:35:41.342203	\N	\N	\N	default	2015-07-31 17:35:41.342317	2015-07-31 17:35:41.342317
133	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2a2911b8-cacb-40d7-992e-052285eb9d4f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/133\n	\N	2015-07-31 17:35:41.458015	\N	\N	\N	default	2015-07-31 17:35:41.458128	2015-07-31 17:35:41.458128
134	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b8e6e31f-c08a-4e79-8b5e-9830801fccb8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/134\n	\N	2015-07-31 17:35:41.627282	\N	\N	\N	default	2015-07-31 17:35:41.627391	2015-07-31 17:35:41.627391
135	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b4e3d958-c960-4d51-8538-ad7533d4c1ba\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/135\n	\N	2015-07-31 17:35:41.768124	\N	\N	\N	default	2015-07-31 17:35:41.768236	2015-07-31 17:35:41.768236
136	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7fe2d6fa-2c77-4552-a048-0e9b8d3d9316\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/136\n	\N	2015-07-31 17:35:41.83714	\N	\N	\N	default	2015-07-31 17:35:41.837266	2015-07-31 17:35:41.837266
137	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 44027ae9-70f7-429c-ae48-a4e5fb987343\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/137\n	\N	2015-07-31 17:35:41.893296	\N	\N	\N	default	2015-07-31 17:35:41.893408	2015-07-31 17:35:41.893408
138	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 29a75a2a-edd1-44d3-bd80-518250546f34\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/138\n	\N	2015-07-31 17:35:42.345476	\N	\N	\N	default	2015-07-31 17:35:42.345589	2015-07-31 17:35:42.345589
139	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: caa91d0a-80fa-471a-b429-43f8fa7e088d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/139\n	\N	2015-07-31 17:35:42.431898	\N	\N	\N	default	2015-07-31 17:35:42.432008	2015-07-31 17:35:42.432008
140	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a43bc5dc-0374-4d6c-846c-867729037679\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/140\n	\N	2015-07-31 17:35:42.473166	\N	\N	\N	default	2015-07-31 17:35:42.473278	2015-07-31 17:35:42.473278
141	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 54eea36d-12c4-4a49-a02e-988dd4679419\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/141\n	\N	2015-07-31 17:35:42.511156	\N	\N	\N	default	2015-07-31 17:35:42.511267	2015-07-31 17:35:42.511267
142	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b257249d-107d-405b-a639-a84930f72de6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/142\n	\N	2015-07-31 17:35:42.581521	\N	\N	\N	default	2015-07-31 17:35:42.581639	2015-07-31 17:35:42.581639
143	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b0dfae1a-b8e1-4886-bde1-bb242d114c07\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/143\n	\N	2015-07-31 17:35:42.750234	\N	\N	\N	default	2015-07-31 17:35:42.750345	2015-07-31 17:35:42.750345
144	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d1b4561e-889d-4b6b-9be3-47f7267b20f0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/144\n	\N	2015-07-31 17:35:42.821707	\N	\N	\N	default	2015-07-31 17:35:42.821867	2015-07-31 17:35:42.821867
145	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ca248926-6255-4aa7-922f-c2ac5b708763\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/145\n	\N	2015-07-31 17:35:42.937022	\N	\N	\N	default	2015-07-31 17:35:42.937188	2015-07-31 17:35:42.937188
146	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d41f15c4-019c-4657-a1ac-1f3ef160222b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/146\n	\N	2015-07-31 17:35:43.005396	\N	\N	\N	default	2015-07-31 17:35:43.00553	2015-07-31 17:35:43.00553
147	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 3e256cdf-046f-4980-b7d8-880976ef19ad\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/147\n	\N	2015-07-31 17:35:43.086718	\N	\N	\N	default	2015-07-31 17:35:43.086865	2015-07-31 17:35:43.086865
148	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: cff36301-5b5f-4fcd-a69d-3f77005cec97\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/148\n	\N	2015-07-31 17:35:43.160383	\N	\N	\N	default	2015-07-31 17:35:43.160499	2015-07-31 17:35:43.160499
149	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9cd4db8e-1b3c-4d20-88fc-2d7056147613\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/149\n	\N	2015-07-31 17:35:43.200981	\N	\N	\N	default	2015-07-31 17:35:43.201092	2015-07-31 17:35:43.201092
150	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 572ffc09-82e9-4d74-b01e-c243ce0041b2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/150\n	\N	2015-07-31 17:35:43.243048	\N	\N	\N	default	2015-07-31 17:35:43.243165	2015-07-31 17:35:43.243165
151	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: daf545fa-2e1c-4c2a-83d1-adf93d5f5083\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/151\n	\N	2015-07-31 17:35:43.280281	\N	\N	\N	default	2015-07-31 17:35:43.280394	2015-07-31 17:35:43.280394
152	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: aa66c98b-f167-471d-861f-d051f93ce497\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/152\n	\N	2015-07-31 17:35:43.414694	\N	\N	\N	default	2015-07-31 17:35:43.414807	2015-07-31 17:35:43.414807
153	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b5161fed-2c58-4e91-ae32-369303e12b3d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/153\n	\N	2015-07-31 17:35:43.526423	\N	\N	\N	default	2015-07-31 17:35:43.526534	2015-07-31 17:35:43.526534
154	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b3cdc9db-5c41-45c1-b615-3dacad4ff074\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/154\n	\N	2015-07-31 17:35:43.568135	\N	\N	\N	default	2015-07-31 17:35:43.568313	2015-07-31 17:35:43.568313
155	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 680a3104-71d2-4514-949c-6cc41d4d17df\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/155\n	\N	2015-07-31 17:35:43.762451	\N	\N	\N	default	2015-07-31 17:35:43.762566	2015-07-31 17:35:43.762566
156	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8d96103d-bb75-47a0-9881-3451e4e2a5d5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/156\n	\N	2015-07-31 17:35:43.875122	\N	\N	\N	default	2015-07-31 17:35:43.87529	2015-07-31 17:35:43.87529
157	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 40e412e2-d05b-494e-8aae-009ad5f77c58\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/157\n	\N	2015-07-31 17:35:43.914913	\N	\N	\N	default	2015-07-31 17:35:43.915045	2015-07-31 17:35:43.915045
158	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6692878e-1f84-441c-9304-811a6d587573\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/158\n	\N	2015-07-31 17:35:43.969836	\N	\N	\N	default	2015-07-31 17:35:43.970007	2015-07-31 17:35:43.970007
159	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8085a8e1-1191-4534-9710-8101dfd53f44\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/159\n	\N	2015-07-31 17:35:44.041131	\N	\N	\N	default	2015-07-31 17:35:44.041244	2015-07-31 17:35:44.041244
160	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e3df5ba5-2c3e-41d7-ad0f-7dfb34e8c8ec\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/160\n	\N	2015-07-31 17:35:44.079489	\N	\N	\N	default	2015-07-31 17:35:44.0796	2015-07-31 17:35:44.0796
161	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d48bf6e8-f2fe-4545-ae07-149dbe90bf42\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/161\n	\N	2015-07-31 17:35:44.153067	\N	\N	\N	default	2015-07-31 17:35:44.153197	2015-07-31 17:35:44.153197
162	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: faf64f03-08a1-42b0-8d60-0d70900a1dd6\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/162\n	\N	2015-07-31 17:35:44.260814	\N	\N	\N	default	2015-07-31 17:35:44.260924	2015-07-31 17:35:44.260924
163	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 50682337-6013-485c-abc9-6f067f211844\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/163\n	\N	2015-07-31 17:35:44.33247	\N	\N	\N	default	2015-07-31 17:35:44.332592	2015-07-31 17:35:44.332592
164	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2a2b9dc1-d343-4283-95ab-40862b97590d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/164\n	\N	2015-07-31 17:35:44.382494	\N	\N	\N	default	2015-07-31 17:35:44.382615	2015-07-31 17:35:44.382615
165	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a4749a55-55ab-467f-b478-98552e6d81cb\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/165\n	\N	2015-07-31 17:35:44.726176	\N	\N	\N	default	2015-07-31 17:35:44.726311	2015-07-31 17:35:44.726311
166	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 9adad871-b90b-477c-acf1-d9ed7d851b7d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/166\n	\N	2015-07-31 17:35:44.783686	\N	\N	\N	default	2015-07-31 17:35:44.783801	2015-07-31 17:35:44.783801
167	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: daef918c-a095-42cd-b552-e67b8b3b66fd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/167\n	\N	2015-07-31 17:35:44.836353	\N	\N	\N	default	2015-07-31 17:35:44.836486	2015-07-31 17:35:44.836486
168	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c8662eda-c348-4bfd-bdb6-40c94921695d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/168\n	\N	2015-07-31 17:35:44.878841	\N	\N	\N	default	2015-07-31 17:35:44.878952	2015-07-31 17:35:44.878952
169	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0c53ef06-588e-4126-b3db-42cda14c9616\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/169\n	\N	2015-07-31 17:35:44.95096	\N	\N	\N	default	2015-07-31 17:35:44.951071	2015-07-31 17:35:44.951071
170	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bcd02e7a-7c5f-4144-acba-e0cab6bfd4f0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/170\n	\N	2015-07-31 17:35:45.065719	\N	\N	\N	default	2015-07-31 17:35:45.065831	2015-07-31 17:35:45.065831
171	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6fd179b1-0b77-4c60-b1f3-d2f07861b7e1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/171\n	\N	2015-07-31 17:35:45.417245	\N	\N	\N	default	2015-07-31 17:35:45.417359	2015-07-31 17:35:45.417359
172	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: eef05016-9c13-4887-a2a4-be2441f22ff7\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/172\n	\N	2015-07-31 17:35:45.524001	\N	\N	\N	default	2015-07-31 17:35:45.524135	2015-07-31 17:35:45.524135
173	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 27eba42f-edf6-460a-a06c-131b42e5ba5d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/173\n	\N	2015-07-31 17:35:45.583473	\N	\N	\N	default	2015-07-31 17:35:45.583595	2015-07-31 17:35:45.583595
174	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: fcb52727-7ff2-4b90-89fc-ee831479def1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/174\n	\N	2015-07-31 17:35:45.728763	\N	\N	\N	default	2015-07-31 17:35:45.728878	2015-07-31 17:35:45.728878
175	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f221fd01-1317-4ba3-9f3a-e034fb38807c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/175\n	\N	2015-07-31 17:35:45.800137	\N	\N	\N	default	2015-07-31 17:35:45.800252	2015-07-31 17:35:45.800252
176	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f3fd62f2-87af-473f-9293-8eac6d06e0fa\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/176\n	\N	2015-07-31 17:35:45.933387	\N	\N	\N	default	2015-07-31 17:35:45.933513	2015-07-31 17:35:45.933513
177	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a49ce74a-4b65-4cd8-9949-baa648006333\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/177\n	\N	2015-07-31 17:35:45.97377	\N	\N	\N	default	2015-07-31 17:35:45.973882	2015-07-31 17:35:45.973882
178	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 90533773-bd4d-49db-b3ce-774b29641016\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/178\n	\N	2015-07-31 17:35:46.137249	\N	\N	\N	default	2015-07-31 17:35:46.137366	2015-07-31 17:35:46.137366
179	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 53ed4de4-ef47-4597-805e-453ff5d8b8db\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/179\n	\N	2015-07-31 17:35:46.235851	\N	\N	\N	default	2015-07-31 17:35:46.235964	2015-07-31 17:35:46.235964
180	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5e41ce25-51e6-4fa4-99da-ca818b6add08\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/180\n	\N	2015-07-31 17:35:46.318676	\N	\N	\N	default	2015-07-31 17:35:46.318858	2015-07-31 17:35:46.318858
181	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a12543f0-24b6-4a9a-adcd-45b9ee568970\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/181\n	\N	2015-07-31 17:35:46.360783	\N	\N	\N	default	2015-07-31 17:35:46.360896	2015-07-31 17:35:46.360896
182	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c169a4b1-96c4-4afc-9638-69ed1be59b7e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/182\n	\N	2015-07-31 17:35:46.403786	\N	\N	\N	default	2015-07-31 17:35:46.403912	2015-07-31 17:35:46.403912
183	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d6ee8d9c-e6be-4063-a5c5-9dd29da54f31\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/183\n	\N	2015-07-31 17:35:46.549369	\N	\N	\N	default	2015-07-31 17:35:46.549483	2015-07-31 17:35:46.549483
184	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0f6c4e85-0b7f-47cd-bfdb-87f4b12e9f08\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/184\n	\N	2015-07-31 17:35:46.592317	\N	\N	\N	default	2015-07-31 17:35:46.592464	2015-07-31 17:35:46.592464
185	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b33171d1-2f94-42e0-9bb5-6621dec50d01\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/185\n	\N	2015-07-31 17:35:46.640694	\N	\N	\N	default	2015-07-31 17:35:46.654698	2015-07-31 17:35:46.654698
186	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 739a8aa6-239b-4714-8696-8a4e748e769c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/186\n	\N	2015-07-31 17:35:46.69349	\N	\N	\N	default	2015-07-31 17:35:46.693614	2015-07-31 17:35:46.693614
187	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c3af59f0-e50a-4c5e-853a-5a86242f9c5e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/187\n	\N	2015-07-31 17:35:46.799384	\N	\N	\N	default	2015-07-31 17:35:46.799495	2015-07-31 17:35:46.799495
188	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 0618a179-0ee2-4d55-a788-032cda5b1874\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/188\n	\N	2015-07-31 17:35:46.838742	\N	\N	\N	default	2015-07-31 17:35:46.838874	2015-07-31 17:35:46.838874
189	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 989a2b9b-c003-4e25-940f-e6939fcc1414\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/189\n	\N	2015-07-31 17:35:46.938528	\N	\N	\N	default	2015-07-31 17:35:46.9387	2015-07-31 17:35:46.9387
190	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 993dd412-d852-48c8-b84e-c7c99902c8a8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/190\n	\N	2015-07-31 17:35:47.008138	\N	\N	\N	default	2015-07-31 17:35:47.008254	2015-07-31 17:35:47.008254
191	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a03d82c1-80e6-4712-a170-96aab84eacbc\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/191\n	\N	2015-07-31 17:35:47.258792	\N	\N	\N	default	2015-07-31 17:35:47.258906	2015-07-31 17:35:47.258906
192	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ead9503d-8338-446e-b599-5a76dcce7339\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/192\n	\N	2015-07-31 17:35:47.301781	\N	\N	\N	default	2015-07-31 17:35:47.30189	2015-07-31 17:35:47.30189
193	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1fbff9d8-7db0-43ae-9159-5ef5da03f5cf\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/193\n	\N	2015-07-31 17:35:47.339899	\N	\N	\N	default	2015-07-31 17:35:47.340049	2015-07-31 17:35:47.340049
194	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 724e89c2-f347-4516-a61a-2ac929358e18\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/194\n	\N	2015-07-31 17:35:47.421521	\N	\N	\N	default	2015-07-31 17:35:47.421635	2015-07-31 17:35:47.421635
195	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d47c67cd-93fd-4f6d-8e35-329d4613d511\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/195\n	\N	2015-07-31 17:35:47.466548	\N	\N	\N	default	2015-07-31 17:35:47.466751	2015-07-31 17:35:47.466751
196	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 14a3f968-bc5d-4edd-9727-a6ce191896c0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/196\n	\N	2015-07-31 17:35:47.539755	\N	\N	\N	default	2015-07-31 17:35:47.539872	2015-07-31 17:35:47.539872
197	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1205f14c-827d-4943-a7e5-e75787ac1e30\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/197\n	\N	2015-07-31 17:35:47.597954	\N	\N	\N	default	2015-07-31 17:35:47.598065	2015-07-31 17:35:47.598065
198	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: dee685ea-7431-4896-a350-0d90d7f63eba\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/198\n	\N	2015-07-31 17:35:47.636478	\N	\N	\N	default	2015-07-31 17:35:47.636592	2015-07-31 17:35:47.636592
199	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f6028a6b-3dcf-4f23-8e26-d253d7f00f90\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/199\n	\N	2015-07-31 17:35:47.692376	\N	\N	\N	default	2015-07-31 17:35:47.692491	2015-07-31 17:35:47.692491
200	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f2ca481a-8f86-4590-9a48-a44b105914d1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/200\n	\N	2015-07-31 17:35:47.730013	\N	\N	\N	default	2015-07-31 17:35:47.730146	2015-07-31 17:35:47.730146
201	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f8afca5a-fbe8-480d-abd9-54d10d891afa\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/201\n	\N	2015-07-31 17:35:47.774913	\N	\N	\N	default	2015-07-31 17:35:47.775132	2015-07-31 17:35:47.775132
202	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 009c9a77-65ed-4dbc-952b-f49c922f4975\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/202\n	\N	2015-07-31 17:35:47.817682	\N	\N	\N	default	2015-07-31 17:35:47.817818	2015-07-31 17:35:47.817818
203	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 234a7348-3595-4b07-9553-38fa0fcd974a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/203\n	\N	2015-07-31 17:35:47.862793	\N	\N	\N	default	2015-07-31 17:35:47.862913	2015-07-31 17:35:47.862913
204	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8c76094a-eb6a-4fd0-ba00-3c9840ff57a4\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/204\n	\N	2015-07-31 17:35:48.34763	\N	\N	\N	default	2015-07-31 17:35:48.347741	2015-07-31 17:35:48.347741
205	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bbf5b190-e311-44dc-a14f-674eda27f0d9\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/205\n	\N	2015-07-31 17:35:48.410303	\N	\N	\N	default	2015-07-31 17:35:48.410418	2015-07-31 17:35:48.410418
206	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 27a6420b-38f1-488b-94c7-22cf2abb90c5\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/206\n	\N	2015-07-31 17:35:48.7973	\N	\N	\N	default	2015-07-31 17:35:48.79741	2015-07-31 17:35:48.79741
207	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a0c3cf1c-a74c-4268-9754-7a44cebc3cd2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/207\n	\N	2015-07-31 17:35:48.897235	\N	\N	\N	default	2015-07-31 17:35:48.897344	2015-07-31 17:35:48.897344
208	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a0d0a9c4-0858-4643-9e03-070e068ea81c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/208\n	\N	2015-07-31 17:35:48.938983	\N	\N	\N	default	2015-07-31 17:35:48.939091	2015-07-31 17:35:48.939091
209	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 27612117-fa86-4bc3-9021-bae7945d7d6e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/209\n	\N	2015-07-31 17:35:48.983707	\N	\N	\N	default	2015-07-31 17:35:48.983889	2015-07-31 17:35:48.983889
210	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a9b35cf0-53d5-4b52-9081-555a858e801a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/210\n	\N	2015-07-31 17:35:49.061937	\N	\N	\N	default	2015-07-31 17:35:49.062081	2015-07-31 17:35:49.062081
211	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: abb122b3-09c5-4a61-96fa-61f932d15fd1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/211\n	\N	2015-07-31 17:35:49.101567	\N	\N	\N	default	2015-07-31 17:35:49.101693	2015-07-31 17:35:49.101693
212	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6e095dcc-f632-4425-b777-f7b12945dd0d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/212\n	\N	2015-07-31 17:35:49.170134	\N	\N	\N	default	2015-07-31 17:35:49.17025	2015-07-31 17:35:49.17025
213	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 15862d99-1379-40d3-b3c2-3707bdb6f83f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/213\n	\N	2015-07-31 17:35:49.316998	\N	\N	\N	default	2015-07-31 17:35:49.317133	2015-07-31 17:35:49.317133
214	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 298f66cc-7929-4e94-805d-245d69be19c1\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/214\n	\N	2015-07-31 17:35:49.361198	\N	\N	\N	default	2015-07-31 17:35:49.361313	2015-07-31 17:35:49.361313
215	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: c3890d22-f1f0-4bd8-9b6f-a96951d9590c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/215\n	\N	2015-07-31 17:35:49.406957	\N	\N	\N	default	2015-07-31 17:35:49.40729	2015-07-31 17:35:49.40729
216	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 53697370-9b22-4896-95d8-222a4da9bada\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/216\n	\N	2015-07-31 17:35:49.448006	\N	\N	\N	default	2015-07-31 17:35:49.448499	2015-07-31 17:35:49.448499
217	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 06e1465a-f72b-4623-b614-e7d01be68f59\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/217\n	\N	2015-07-31 17:35:49.485669	\N	\N	\N	default	2015-07-31 17:35:49.485802	2015-07-31 17:35:49.485802
218	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2c752ee0-3e38-4d43-82bb-0c7624cf4846\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/218\n	\N	2015-07-31 17:35:49.524736	\N	\N	\N	default	2015-07-31 17:35:49.524858	2015-07-31 17:35:49.524858
219	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 42db0501-fac0-4808-afc3-a48232d83a61\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/219\n	\N	2015-07-31 17:35:49.57317	\N	\N	\N	default	2015-07-31 17:35:49.573282	2015-07-31 17:35:49.573282
220	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ffae2a74-7e2d-4f93-9bbd-cbcbf4eaf4b2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/220\n	\N	2015-07-31 17:35:49.61602	\N	\N	\N	default	2015-07-31 17:35:49.616134	2015-07-31 17:35:49.616134
221	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 61da6179-a90c-4271-b8f2-0f72fac46b19\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/221\n	\N	2015-07-31 17:35:49.759327	\N	\N	\N	default	2015-07-31 17:35:49.759438	2015-07-31 17:35:49.759438
222	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ad2e25c0-0fb1-48ad-972a-9889e41626d8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/222\n	\N	2015-07-31 17:35:49.814064	\N	\N	\N	default	2015-07-31 17:35:49.814174	2015-07-31 17:35:49.814174
223	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b95ef669-324e-4fab-bde6-fae339b7218e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/223\n	\N	2015-07-31 17:35:49.853517	\N	\N	\N	default	2015-07-31 17:35:49.853639	2015-07-31 17:35:49.853639
224	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d9e22f3f-256e-43c1-9b0c-6d2d20236483\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/224\n	\N	2015-07-31 17:35:49.894137	\N	\N	\N	default	2015-07-31 17:35:49.894248	2015-07-31 17:35:49.894248
225	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ed59d66f-68cd-4ee1-ab40-51000d096b7b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/225\n	\N	2015-07-31 17:35:49.929935	\N	\N	\N	default	2015-07-31 17:35:49.930046	2015-07-31 17:35:49.930046
226	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4cf54363-eb9f-4d27-b771-66dd59a20697\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/226\n	\N	2015-07-31 17:35:50.065106	\N	\N	\N	default	2015-07-31 17:35:50.06522	2015-07-31 17:35:50.06522
227	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5b18cd88-f391-4f58-9e46-f96cdbffe899\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/227\n	\N	2015-07-31 17:35:50.110741	\N	\N	\N	default	2015-07-31 17:35:50.110855	2015-07-31 17:35:50.110855
228	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 303353e3-7bdd-4459-95a5-330e9f676d73\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/228\n	\N	2015-07-31 17:35:50.16019	\N	\N	\N	default	2015-07-31 17:35:50.160316	2015-07-31 17:35:50.160316
229	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 8fee9893-88c0-4313-88f0-46696f977b6c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/229\n	\N	2015-07-31 17:35:50.332967	\N	\N	\N	default	2015-07-31 17:35:50.333135	2015-07-31 17:35:50.333135
230	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 18e1c047-ffbc-4e9b-9c76-bd5d4699027d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/230\n	\N	2015-07-31 17:35:50.385781	\N	\N	\N	default	2015-07-31 17:35:50.385898	2015-07-31 17:35:50.385898
231	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f337f732-c1f0-4dbe-b6a6-a80ccd16dc2b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/231\n	\N	2015-07-31 17:35:50.43577	\N	\N	\N	default	2015-07-31 17:35:50.435884	2015-07-31 17:35:50.435884
232	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 56bbbc5b-b1c0-479d-971a-d567440d9f02\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/232\n	\N	2015-07-31 17:35:50.515369	\N	\N	\N	default	2015-07-31 17:35:50.515486	2015-07-31 17:35:50.515486
233	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 7c144e2c-38be-49b9-8836-7e6e7aed2f5b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/233\n	\N	2015-07-31 17:35:50.56051	\N	\N	\N	default	2015-07-31 17:35:50.560619	2015-07-31 17:35:50.560619
234	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4bbc300d-eeb5-47a3-95cd-5a9af5cf10e8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/234\n	\N	2015-07-31 17:35:50.611915	\N	\N	\N	default	2015-07-31 17:35:50.612137	2015-07-31 17:35:50.612137
235	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b859baa0-60f4-4e6d-beed-2638e8378f0f\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/235\n	\N	2015-07-31 17:35:50.649981	\N	\N	\N	default	2015-07-31 17:35:50.650094	2015-07-31 17:35:50.650094
236	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e1538c54-26aa-4676-97af-d403a9c1b670\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/236\n	\N	2015-07-31 17:35:50.687528	\N	\N	\N	default	2015-07-31 17:35:50.68764	2015-07-31 17:35:50.68764
237	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 1262b191-d536-4585-a451-8a639f607e34\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/237\n	\N	2015-07-31 17:35:50.737904	\N	\N	\N	default	2015-07-31 17:35:50.738036	2015-07-31 17:35:50.738036
238	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: a7bee3ea-59d5-4e32-9c20-140dcf75a228\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/238\n	\N	2015-07-31 17:35:50.77936	\N	\N	\N	default	2015-07-31 17:35:50.779523	2015-07-31 17:35:50.779523
239	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ff219a91-da18-4223-a72f-6216f845605a\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/239\n	\N	2015-07-31 17:35:50.84243	\N	\N	\N	default	2015-07-31 17:35:50.84254	2015-07-31 17:35:50.84254
240	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6469d898-b56e-4bf6-9f26-0e0f4592833c\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/240\n	\N	2015-07-31 17:35:50.966342	\N	\N	\N	default	2015-07-31 17:35:50.96659	2015-07-31 17:35:50.96659
241	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: bfdb191e-edd1-4470-9ef4-1fe24406336b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/241\n	\N	2015-07-31 17:35:51.28099	\N	\N	\N	default	2015-07-31 17:35:51.281133	2015-07-31 17:35:51.281133
242	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 824df5bf-edac-4df1-aa24-0052e805fb52\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/242\n	\N	2015-07-31 17:35:51.413177	\N	\N	\N	default	2015-07-31 17:35:51.413293	2015-07-31 17:35:51.413293
243	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 01846e45-2b1f-4f99-80e1-2964e587c67d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/243\n	\N	2015-07-31 17:35:51.555503	\N	\N	\N	default	2015-07-31 17:35:51.555619	2015-07-31 17:35:51.555619
244	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 2bebc441-3726-4cd5-926b-debd9948c9a0\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/244\n	\N	2015-07-31 17:35:51.596361	\N	\N	\N	default	2015-07-31 17:35:51.596474	2015-07-31 17:35:51.596474
245	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: b8a66b5a-62da-4e1a-986d-cccf2e7e7430\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/245\n	\N	2015-07-31 17:35:51.63491	\N	\N	\N	default	2015-07-31 17:35:51.635021	2015-07-31 17:35:51.635021
246	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 099d4724-a75f-4269-b9fe-ca3119a2ad9b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/246\n	\N	2015-07-31 17:35:51.676831	\N	\N	\N	default	2015-07-31 17:35:51.676942	2015-07-31 17:35:51.676942
247	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 99857a1d-8d50-4eb8-b65e-ef3f84c3c2b8\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/247\n	\N	2015-07-31 17:35:52.056506	\N	\N	\N	default	2015-07-31 17:35:52.056648	2015-07-31 17:35:52.056648
248	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 81d1d271-d2a3-45a6-9ef6-3cb101346bb3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/248\n	\N	2015-07-31 17:35:52.101779	\N	\N	\N	default	2015-07-31 17:35:52.101891	2015-07-31 17:35:52.101891
249	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: e869cf05-1e90-4d81-9451-51880e5878da\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/249\n	\N	2015-07-31 17:35:52.16658	\N	\N	\N	default	2015-07-31 17:35:52.166699	2015-07-31 17:35:52.166699
250	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f8ddbd08-cf64-4836-bdc6-a648834f4842\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/250\n	\N	2015-07-31 17:35:52.246329	\N	\N	\N	default	2015-07-31 17:35:52.246511	2015-07-31 17:35:52.246511
251	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: da53d5e4-f3c2-48d4-b4cc-0c69ff0a8ef3\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/251\n	\N	2015-07-31 17:35:52.323976	\N	\N	\N	default	2015-07-31 17:35:52.324088	2015-07-31 17:35:52.324088
252	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 6805917f-5631-4889-8f48-c1ac97473d21\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/252\n	\N	2015-07-31 17:35:52.368818	\N	\N	\N	default	2015-07-31 17:35:52.368928	2015-07-31 17:35:52.368928
253	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 701f7ce7-aa96-426c-bd93-1b69db13c317\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/253\n	\N	2015-07-31 17:35:52.459093	\N	\N	\N	default	2015-07-31 17:35:52.459206	2015-07-31 17:35:52.459206
254	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 112e8795-fc69-4021-9116-6f2c2f465f1b\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/254\n	\N	2015-07-31 17:35:52.749058	\N	\N	\N	default	2015-07-31 17:35:52.749188	2015-07-31 17:35:52.749188
255	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: f3ebd5ec-79c5-4c05-b00f-97280096cc39\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/255\n	\N	2015-07-31 17:35:52.83639	\N	\N	\N	default	2015-07-31 17:35:52.836514	2015-07-31 17:35:52.836514
256	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 17b652d3-9ef6-40e7-8d6a-6e218b27cab2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/256\n	\N	2015-07-31 17:35:52.958041	\N	\N	\N	default	2015-07-31 17:35:52.958151	2015-07-31 17:35:52.958151
257	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: d88acf24-6ca4-4298-8a50-f68e3a5b00d2\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/257\n	\N	2015-07-31 17:35:53.123145	\N	\N	\N	default	2015-07-31 17:35:53.123258	2015-07-31 17:35:53.123258
258	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 5e5c039e-9e5e-4ed5-9030-84db482f411d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/258\n	\N	2015-07-31 17:35:53.163565	\N	\N	\N	default	2015-07-31 17:35:53.163729	2015-07-31 17:35:53.163729
259	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 529b912a-f088-4ffe-b4dc-b99026e0045d\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/259\n	\N	2015-07-31 17:35:53.251759	\N	\N	\N	default	2015-07-31 17:35:53.251918	2015-07-31 17:35:53.251918
260	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: ec924afe-55c0-4de2-b41d-5bdaad9cdb4e\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/260\n	\N	2015-07-31 17:35:53.338298	\N	\N	\N	default	2015-07-31 17:35:53.338411	2015-07-31 17:35:53.338411
261	0	0	--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: FetchSourceCitation\n  job_id: 4d7eea93-ecb3-468e-b4e1-36b5bfd03afd\n  queue_name: default\n  arguments:\n  - _aj_globalid: gid://civic-server/Source/261\n	\N	2015-07-31 17:35:53.388612	\N	\N	\N	default	2015-07-31 17:35:53.388824	2015-07-31 17:35:53.388824
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 261, true);


--
-- Data for Name: disease_ontology_mirrors; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY disease_ontology_mirrors (id, doid, name) FROM stdin;
\.


--
-- Name: disease_ontology_mirrors_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('disease_ontology_mirrors_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
18	1909	Melanoma	2015-07-31 17:35:37.828416	2015-07-31 17:35:37.828416
22	769	Neuroblastoma	2015-07-31 17:35:38.491732	2015-07-31 17:35:38.491732
30	1612	Breast Cancer	2015-07-31 17:35:45.935903	2015-07-31 17:35:45.935903
40	4947	Cholangiocarcinoma	2015-07-31 17:35:48.801432	2015-07-31 17:35:48.801432
42	3069	Astrocytoma	2015-07-31 17:35:49.489526	2015-07-31 17:35:49.489526
12	3908	Non-small Cell Lung Carcinoma	2015-07-31 17:35:37.254013	2015-07-31 17:37:34.102344
13	9119	Acute Myeloid Leukemia	2015-07-31 17:35:37.333205	2015-07-31 17:37:34.273029
14	10747	Lymphoid Leukemia	2015-07-31 17:35:37.567221	2015-07-31 17:37:34.454826
15	8552	Chronic Myeloid Leukemia	2015-07-31 17:35:37.604796	2015-07-31 17:37:34.629747
16	8997	Polycythemia Vera	2015-07-31 17:35:37.676691	2015-07-31 17:37:34.798918
17	9119	Acute Myeloid Leukemia	2015-07-31 17:35:37.721954	2015-07-31 17:37:34.967928
19	9256	Colorectal Cancer	2015-07-31 17:35:37.910033	2015-07-31 17:37:35.37437
20	9253	Gastrointestinal Stromal Tumor	2015-07-31 17:35:38.242703	2015-07-31 17:37:35.519754
21	0050905	Inflammatory Myofibroblastic Tumor	2015-07-31 17:35:38.403765	2015-07-31 17:37:35.650037
23	9253	Gastrointestinal Stromal Tumor	2015-07-31 17:35:39.255198	2015-07-31 17:37:35.985578
24	3973	Thyroid Medullary Carcinoma	2015-07-31 17:35:40.128735	2015-07-31 17:37:36.146163
25	1781	Thyroid Cancer	2015-07-31 17:35:40.546036	2015-07-31 17:37:36.348439
26	9256	Colorectal Cancer	2015-07-31 17:35:41.020998	2015-07-31 17:37:36.527384
27	4007	Bladder Carcinoma	2015-07-31 17:35:42.347675	2015-07-31 17:37:36.764855
28	3068	Glioblastoma Multiforme	2015-07-31 17:35:43.162685	2015-07-31 17:37:36.980418
29	2394	Ovarian Cancer	2015-07-31 17:35:45.419671	2015-07-31 17:37:37.312257
31	1612	Breast Cancer	2015-07-31 17:35:46.238115	2015-07-31 17:37:38.094296
32	5520	Head And Neck Squamous Cell Carcinoma	2015-07-31 17:35:46.322261	2015-07-31 17:37:38.239641
33	0050746	Mantle Cell Lymphoma	2015-07-31 17:35:46.363074	2015-07-31 17:37:38.377639
34	10534	Stomach Cancer	2015-07-31 17:35:46.4062	2015-07-31 17:37:38.647394
35	715	T-cell Leukemia	2015-07-31 17:35:46.595937	2015-07-31 17:37:38.801821
36	5635	Gastric Adenosquamous Carcinoma	2015-07-31 17:35:46.695709	2015-07-31 17:37:38.942315
37	1324	Lung Cancer	2015-07-31 17:35:46.763028	2015-07-31 17:37:39.217038
38	0060075	Estrogen-receptor Positive Breast Cancer	2015-07-31 17:35:46.801837	2015-07-31 17:37:39.351996
39	3908	Non-small Cell Lung Carcinoma	2015-07-31 17:35:47.542075	2015-07-31 17:37:39.543909
41	3909	Bronchogenic Lung Adenocarcinoma	2015-07-31 17:35:49.363514	2015-07-31 17:37:39.84946
43	3068	Glioblastoma Multiforme	2015-07-31 17:35:49.527065	2015-07-31 17:37:40.373228
44	0050908	Myelodysplastic Syndrome	2015-07-31 17:35:49.575755	2015-07-31 17:37:40.585029
45	0050908	Myelodysplastic Syndrome	2015-07-31 17:35:49.855882	2015-07-31 17:37:40.730002
46	14145	Malignant Anus Melanoma	2015-07-31 17:35:50.33618	2015-07-31 17:37:40.867326
47	9538	Multiple Myeloma	2015-07-31 17:35:50.691332	2015-07-31 17:37:41.071435
48	3910	Lung Adenocarcinoma	2015-07-31 17:35:50.781742	2015-07-31 17:37:41.288842
49	9119	Acute Myeloid Leukemia	2015-07-31 17:35:51.13391	2015-07-31 17:37:41.449907
50	9119	Acute Myeloid Leukemia	2015-07-31 17:35:52.059159	2015-07-31 17:37:41.603618
51	6477	Invasive Bladder Transitional Cell Carcinoma	2015-07-31 17:35:52.751469	2015-07-31 17:37:41.831875
52	9952	Acute Lymphocytic Leukemia	2015-07-31 17:35:53.166992	2015-07-31 17:37:42.005717
53	5015	Hepatocellular Fibrolamellar Carcinoma	2015-07-31 17:35:53.25396	2015-07-31 17:37:42.147219
54	3369	Peripheral Primitive Neuroectodermal Tumor	2015-07-31 17:35:53.391373	2015-07-31 17:37:42.323497
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 54, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
15	Sorafenib	\N	2015-07-31 17:35:37.303668	2015-07-31 17:35:37.303668
16	N/A	\N	2015-07-31 17:35:37.35212	2015-07-31 17:35:37.35212
17	Daunorubicin	\N	2015-07-31 17:35:37.428329	2015-07-31 17:35:37.428329
18	Idarubicin	\N	2015-07-31 17:35:37.465817	2015-07-31 17:35:37.465817
19	Pegylated IFN-2a	\N	2015-07-31 17:35:37.696846	2015-07-31 17:35:37.696846
20	AZD6244	\N	2015-07-31 17:35:37.846175	2015-07-31 17:35:37.846175
21	17-AAG	\N	2015-07-31 17:35:38.040094	2015-07-31 17:35:38.040094
22	Vemurafenib	\N	2015-07-31 17:35:38.136789	2015-07-31 17:35:38.136789
23	Temozolomide	\N	2015-07-31 17:35:38.180301	2015-07-31 17:35:38.180301
24	Imatinib	\N	2015-07-31 17:35:38.321668	2015-07-31 17:35:38.321668
25	Crizotinib	\N	2015-07-31 17:35:38.424499	2015-07-31 17:35:38.424499
26	CH5424802	\N	2015-07-31 17:35:38.509967	2015-07-31 17:35:38.509967
27	TAE684	\N	2015-07-31 17:35:38.596529	2015-07-31 17:35:38.596529
28	Trametinib	\N	2015-07-31 17:35:38.708495	2015-07-31 17:35:38.708495
29	Nilotinib	\N	2015-07-31 17:35:39.521149	2015-07-31 17:35:39.521149
30	Dasatinib	\N	2015-07-31 17:35:39.562031	2015-07-31 17:35:39.562031
31	Gefitinib	\N	2015-07-31 17:35:39.607989	2015-07-31 17:35:39.607989
32	Erlotinib	\N	2015-07-31 17:35:39.614422	2015-07-31 17:35:39.614422
33	Cetuximab	\N	2015-07-31 17:35:39.84402	2015-07-31 17:35:39.84402
34	Crenolanib	\N	2015-07-31 17:35:39.891694	2015-07-31 17:35:39.891694
35	Dabrafenib	\N	2015-07-31 17:35:40.089286	2015-07-31 17:35:40.089286
36	Motesanib	\N	2015-07-31 17:35:40.147317	2015-07-31 17:35:40.147317
37	AZD1480	\N	2015-07-31 17:35:40.346156	2015-07-31 17:35:40.346156
38	Panitumumab	\N	2015-07-31 17:35:40.386788	2015-07-31 17:35:40.386788
39	Dacarbazine	\N	2015-07-31 17:35:40.695791	2015-07-31 17:35:40.695791
40	Temozolomide	\N	2015-07-31 17:35:40.701328	2015-07-31 17:35:40.701328
41	PD0325901	\N	2015-07-31 17:35:41.116767	2015-07-31 17:35:41.116767
42	PLX4720	\N	2015-07-31 17:35:41.121439	2015-07-31 17:35:41.121439
43	Nutlin-3	\N	2015-07-31 17:35:41.168138	2015-07-31 17:35:41.168138
44	Capecitabine	\N	2015-07-31 17:35:41.214187	2015-07-31 17:35:41.214187
45	Bevacizumab	\N	2015-07-31 17:35:41.222141	2015-07-31 17:35:41.222141
46	Regorafenib	\N	2015-07-31 17:35:41.487091	2015-07-31 17:35:41.487091
47	PD173074	\N	2015-07-31 17:35:42.369106	2015-07-31 17:35:42.369106
48	ATRA	\N	2015-07-31 17:35:42.491872	2015-07-31 17:35:42.491872
49	SU5614	\N	2015-07-31 17:35:42.801537	2015-07-31 17:35:42.801537
50	Valproic Acid	\N	2015-07-31 17:35:43.588959	2015-07-31 17:35:43.588959
51	NSC348884	\N	2015-07-31 17:35:44.357864	2015-07-31 17:35:44.357864
52	Induction Therapy	\N	2015-07-31 17:35:44.603941	2015-07-31 17:35:44.603941
53	Rapamycin	\N	2015-07-31 17:35:44.746133	2015-07-31 17:35:44.746133
54	Everolimus	\N	2015-07-31 17:35:44.807013	2015-07-31 17:35:44.807013
55	Olaparib	\N	2015-07-31 17:35:45.438608	2015-07-31 17:35:45.438608
56	Arsenic Trioxide	\N	2015-07-31 17:35:45.605315	2015-07-31 17:35:45.605315
57	MK-2206	\N	2015-07-31 17:35:45.9545	2015-07-31 17:35:45.9545
58	Palbociclib (PD-0332991)	\N	2015-07-31 17:35:46.62051	2015-07-31 17:35:46.62051
59	Palbociclib	\N	2015-07-31 17:35:46.819399	2015-07-31 17:35:46.819399
60	BYL719	\N	2015-07-31 17:35:46.858325	2015-07-31 17:35:46.858325
61	Premetrexed	\N	2015-07-31 17:35:47.75334	2015-07-31 17:35:47.75334
62	Stauroporine	\N	2015-07-31 17:35:47.797769	2015-07-31 17:35:47.797769
63	Dacomitinib	\N	2015-07-31 17:35:47.835929	2015-07-31 17:35:47.835929
64	Neratinib	\N	2015-07-31 17:35:47.887216	2015-07-31 17:35:47.887216
65	Lapatinib	\N	2015-07-31 17:35:48.045351	2015-07-31 17:35:48.045351
66	Hormone Therapy	\N	2015-07-31 17:35:48.367838	2015-07-31 17:35:48.367838
67	Tamoxifen	\N	2015-07-31 17:35:48.429634	2015-07-31 17:35:48.429634
68	Fulvestrant	\N	2015-07-31 17:35:48.434222	2015-07-31 17:35:48.434222
69	Ponatinib	\N	2015-07-31 17:35:48.824049	2015-07-31 17:35:48.824049
70	Pazopanib	\N	2015-07-31 17:35:48.870774	2015-07-31 17:35:48.870774
71	CEP701	\N	2015-07-31 17:35:48.963641	2015-07-31 17:35:48.963641
72	AG1296	\N	2015-07-31 17:35:49.119461	2015-07-31 17:35:49.119461
73	Bortezomib	\N	2015-07-31 17:35:49.381704	2015-07-31 17:35:49.381704
74	Fasudil	\N	2015-07-31 17:35:49.386046	2015-07-31 17:35:49.386046
75	SU11248	\N	2015-07-31 17:35:50.45472	2015-07-31 17:35:50.45472
76	NVP-BEZ235	\N	2015-07-31 17:35:50.989693	2015-07-31 17:35:50.989693
77	ARRY-142886	\N	2015-07-31 17:35:50.994849	2015-07-31 17:35:50.994849
78	Carmustine	\N	2015-07-31 17:35:51.335339	2015-07-31 17:35:51.335339
79	O(6)-benzylguanine	\N	2015-07-31 17:35:51.389845	2015-07-31 17:35:51.389845
80	CH5132799	\N	2015-07-31 17:35:51.574303	2015-07-31 17:35:51.574303
81	MTOR Inhibitors	\N	2015-07-31 17:35:52.187116	2015-07-31 17:35:52.187116
82	Doxorubicin	\N	2015-07-31 17:35:52.393531	2015-07-31 17:35:52.393531
83	GW-2580	\N	2015-07-31 17:35:53.196104	2015-07-31 17:35:53.196104
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description, deleted, deleted_at) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-07-31 17:34:13.793019	2015-07-31 17:34:13.793019	\N	f	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-07-31 17:34:14.151121	2015-07-31 17:34:14.151121	\N	f	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-07-31 17:34:14.434466	2015-07-31 17:34:14.434466	\N	f	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-07-31 17:34:14.70456	2015-07-31 17:34:14.70456	\N	f	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-07-31 17:34:14.952847	2015-07-31 17:34:14.952847	\N	f	\N
6	672	BRCA1	BRCA1 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 1, early onset	2015-07-31 17:34:15.221179	2015-07-31 17:34:15.221179	\N	f	\N
7	675	BRCA2	BRCA2 mutations in the germline have become a hallmark for hereditary breast and ovarian cancers. Variants that have been demonstrated to reduce the function of the protein have been shown to increase risk for these cancers, as well as prostate and pancreatic cancer. These findings have been the impetus for the increased popularity of genetic testing of healthy indivudals to assess risk.	breast cancer 2, early onset	2015-07-31 17:34:15.494667	2015-07-31 17:34:15.494667	\N	f	\N
8	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-07-31 17:34:15.723169	2015-07-31 17:34:15.723169	\N	f	\N
9	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-07-31 17:34:15.95645	2015-07-31 17:34:15.95645	\N	f	\N
10	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-07-31 17:34:16.138411	2015-07-31 17:34:16.138411	\N	f	\N
11	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-07-31 17:34:16.354744	2015-07-31 17:34:16.354744	\N	f	\N
12	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-07-31 17:34:16.560511	2015-07-31 17:34:16.560511	\N	f	\N
13	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-07-31 17:34:16.779759	2015-07-31 17:34:16.779759	\N	f	\N
14	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-07-31 17:34:17.080685	2015-07-31 17:34:17.080685	\N	f	\N
15	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-07-31 17:34:17.336322	2015-07-31 17:34:17.336322	\N	f	\N
16	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-07-31 17:34:17.584234	2015-07-31 17:34:17.584234	\N	f	\N
17	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	protein kinase, cAMP-dependent, catalytic, alpha	2015-07-31 17:34:17.796577	2015-07-31 17:34:17.796577	\N	f	\N
18	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-07-31 17:34:18.00959	2015-07-31 17:34:18.00959	\N	f	\N
19	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-07-31 17:34:18.27427	2015-07-31 17:34:18.27427	\N	f	\N
20	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown sensitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-07-31 17:34:18.532265	2015-07-31 17:34:18.532265	\N	f	\N
21	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-07-31 17:34:18.778991	2015-07-31 17:34:18.778991	\N	f	\N
22	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-07-31 17:34:19.057189	2015-07-31 17:34:19.057189	\N	f	\N
23	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions that lead to pathway actiation have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-07-31 17:34:19.320084	2015-07-31 17:34:19.320084	\N	f	\N
24	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-07-31 17:34:19.535843	2015-07-31 17:34:19.535843	\N	f	\N
25	2624	GATA2	GATA2 is a transcription factor involved in stem cell maintenance with key roles in hematopoietic development. GATA2 mutations are associated with a variety of inherited and acquired immune disorders including myelodysplastic syndrome and acute myeloid leukemia. In addition to a role in hematopoiesis, the maintenance GATA2 expression has been implicated as a requirement in KRAS-driven non-small cell lung cancer. Preclinical models have indicated therapeutic benefit from targeting GATA2-mediated pathways in the context of KRAS-driven NSCLC.	GATA binding protein 2	2015-07-31 17:34:19.774004	2015-07-31 17:34:19.774004	\N	f	\N
26	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. The implications of mutations in this gene vary greatly by cancer type. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been associated with worse outcome, shorter overall survival, and normal karyotype. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Unlike the association with cytogenetically normal AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-07-31 17:34:20.020936	2015-07-31 17:34:20.020936	\N	f	\N
27	3418	IDH2	IDH2 mutations have been observed in a number of cancer types, including sarcomas, hematologic malignancies, colon cancer and brain cancer. Mutations in the two isocitrate dehydrogenase enzymes involved in cytoplasmic (IDH1) and mitochondrial (IDH2) conversion of alpha-ketoglutarate to D-2-hydroxyglutarate have been described as mutually exclusive in many of these cancer types. The most frequent mutations involve R132 (IDH1) and R172 (IDH2) involve the active site and result in neomorphic enzyme activity. Although IDH2 (R172) mutations are associated with poorer overall prognosis in AML patients, its utility as a prognostic marker in MDS is still under debate. Additionally, IDH2 (R140) has been associated with improved overall survival in AML. IDH2 mutations have been associated with improved prognosis in gliomas.	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-07-31 17:34:20.275254	2015-07-31 17:34:20.275254	\N	f	\N
28	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders. The variant is a known activating mutation, and activated JAK2 is sufficient to drive myeloproliferative disorders in mouse models. V617F, while most recurrent, is not the only mechanism by which JAK2 can be activated in patients. JAK2 is now one of the first diagnostic markers tested upon diagnosis with a myeloproliferative disorder.	Janus kinase 2	2015-07-31 17:34:20.498401	2015-07-31 17:34:20.498401	\N	f	\N
29	3815	KIT	c-KIT activation has been shown to have oncogenic activity in gastrointestinal stromal tumors (GISTs), melanomas, lung cancer, and other tumor types. The targeted therapeutics nilotinib and sunitinib have shown efficacy in treating KIT overactive patients, and are in late-stage trials in melanoma and GIST. KIT overactivity can be the result of many genomic events from genomic amplification to overexpression to missense mutations. Missense mutations have been shown to be key players in mediating clinical response and acquired resistance in patients being treated with these targeted therapeutics.	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-07-31 17:34:20.709912	2015-07-31 17:34:20.709912	\N	f	\N
30	3845	KRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	Kirsten rat sarcoma viral oncogene homolog	2015-07-31 17:34:20.97338	2015-07-31 17:34:20.97338	\N	f	\N
31	5604	MAP2K1		mitogen-activated protein kinase kinase 1	2015-07-31 17:34:21.217389	2015-07-31 17:34:21.217389	\N	f	\N
32	4209	MEF2D		myocyte enhancer factor 2D	2015-07-31 17:34:21.428279	2015-07-31 17:34:21.428279	\N	f	\N
33	1436	CSF1R		colony stimulating factor 1 receptor	2015-07-31 17:34:21.658103	2015-07-31 17:34:21.658103	\N	f	\N
34	4255	MGMT		O-6-methylguanine-DNA methyltransferase	2015-07-31 17:34:21.846722	2015-07-31 17:34:21.846722	\N	f	\N
35	4869	NPM1	AML with mutated NPM1 is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-07-31 17:34:22.035395	2015-07-31 17:34:22.035395	\N	f	\N
36	4893	NRAS	Mutations in the RAS family of proteins have frequently observed across cancer types. The amino acid positions account for the overwhelming majority of these mutations, G12, G13 and Q61. The isoforms, despite their raw similarity, also behave very differently when expressed in non-native tissue types, likely due to differences in the C-terminal hyper-variable regions. Mis-regulation of isoform expression has been shown to be a driving event in cancer, as well as missense mutations at the three hotspots previously mentioned. While highly recurrent in cancer, targeted these RAS mutants has also been very elusive, and has not yet become common practice in the clinic.	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-07-31 17:34:22.264763	2015-07-31 17:34:22.264763	\N	f	\N
37	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-07-31 17:34:22.485737	2015-07-31 17:34:22.485737	\N	f	\N
38	5156	PDGFRA		platelet-derived growth factor receptor, alpha polypeptide	2015-07-31 17:34:22.694472	2015-07-31 17:34:22.694472	\N	f	\N
39	5371	PML	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	promyelocytic leukemia	2015-07-31 17:34:22.897383	2015-07-31 17:34:22.897383	\N	f	\N
40	5914	RARA	The PML-RARA fusion is the result of a recurrent, balanced translocation between chromosomes 15 and 17, denoted as t(15;17)(q22;q12), and a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA (all-trans retinoic acid) in APL patients harboring the PML-RARA fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	retinoic acid receptor, alpha	2015-07-31 17:34:23.090406	2015-07-31 17:34:23.090406	\N	f	\N
41	5728	PTEN		phosphatase and tensin homolog	2015-07-31 17:34:23.312135	2015-07-31 17:34:23.312135	\N	f	\N
42	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-07-31 17:34:23.558501	2015-07-31 17:34:23.558501	\N	f	\N
43	861	RUNX1		runt-related transcription factor 1	2015-07-31 17:34:23.814691	2015-07-31 17:34:23.814691	\N	f	\N
44	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-07-31 17:34:24.078376	2015-07-31 17:34:24.078376	\N	f	\N
45	7157	TP53	TP53 mutations are universal across cancer types. Loss of tumor suppressors is most recognized by large deleterious events, such as frameshift mutations, or premature stop codons. In TP53 however, many of the observed mutations in cancer are found to be single nucleotide variants, or missense mutations. These variants are also very broadly distributed throughout the gene, not localizing in any particular hotspot. While a large proportion of cancer genomics research is focused on somatic variants, TP53 is also of note in the germline. Germline TP53 mutations are the hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been found to have prognostic impact on patient outcomes.	tumor protein p53	2015-07-31 17:34:24.323191	2015-07-31 17:34:24.323191	\N	f	\N
46	7248	TSC1		tuberous sclerosis 1	2015-07-31 17:34:24.530462	2015-07-31 17:34:24.530462	\N	f	\N
47	7249	TSC2		tuberous sclerosis 2	2015-07-31 17:34:24.726294	2015-07-31 17:34:24.726294	\N	f	\N
48	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-07-31 17:34:25.01135	2015-07-31 17:34:25.01135	\N	f	\N
49	7490	WT1	WT1 is a tumor suppressor gene associated with the development of Wilms' Tumor, from which it was named. Mutations in exon 7 and 9 of WT1 have been recurrently identified in acute myeloid leukemia and associated with poorer prognosis and chemotherapy resistance.	Wilms tumor 1	2015-07-31 17:34:25.2631	2015-07-31 17:34:25.2631	\N	f	\N
50	4851	NOTCH1		notch 1	2015-07-31 17:34:25.477287	2015-07-31 17:34:25.477287	\N	f	\N
51	4921	DDR2		discoidin domain receptor tyrosine kinase 2	2015-07-31 17:34:25.68015	2015-07-31 17:34:25.68015	\N	f	\N
52	4233	MET		MET proto-oncogene, receptor tyrosine kinase	2015-07-31 17:34:25.888834	2015-07-31 17:34:25.888834	\N	f	\N
53	2313	FLI1		Fli-1 proto-oncogene, ETS transcription factor	2015-07-31 17:34:26.080505	2015-07-31 17:34:26.080505	\N	f	\N
54	2130	EWSR1		EWS RNA-binding protein 1	2015-07-31 17:34:26.286767	2015-07-31 17:34:26.286767	\N	f	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-07-31 17:34:13.842987	2015-07-31 17:35:54.173282
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-07-31 17:34:13.91339	2015-07-31 17:35:54.590928
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-07-31 17:34:14.164021	2015-07-31 17:35:55.007561
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-07-31 17:34:14.178019	2015-07-31 17:35:55.52969
5	15676015	\N	Lee et al., 2005, APMIS	2015-07-31 17:34:14.454325	2015-07-31 17:35:55.947163
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-07-31 17:34:14.467902	2015-07-31 17:35:56.364996
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-07-31 17:34:14.71655	2015-07-31 17:35:56.732124
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-07-31 17:34:14.726429	2015-07-31 17:35:57.095708
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-07-31 17:34:14.966006	2015-07-31 17:35:57.514848
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-07-31 17:34:14.97673	2015-07-31 17:35:57.933026
11	24366442	\N	Nelson et al., 2014, Ann. Intern. Med.	2015-07-31 17:34:15.235294	2015-07-31 17:35:58.35043
12	12432268	\N	Diehl, Cancer Biol. Ther.	2015-07-31 17:34:15.737702	2015-07-31 17:35:58.774153
13	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-07-31 17:34:15.751609	2015-07-31 17:35:59.193814
14	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-07-31 17:34:16.367483	2015-07-31 17:35:59.70922
15	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-07-31 17:34:16.574844	2015-07-31 17:36:00.125478
16	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-07-31 17:34:16.587314	2015-07-31 17:36:00.54569
17	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-07-31 17:34:17.093192	2015-07-31 17:36:01.207102
18	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-07-31 17:34:17.103579	2015-07-31 17:36:01.690494
19	19357394	\N	Vardiman et al., 2009, Blood	2015-07-31 17:34:17.352412	2015-07-31 17:36:02.049678
20	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-07-31 17:34:17.364379	2015-07-31 17:36:02.421556
21	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-07-31 17:34:17.374316	2015-07-31 17:36:02.843735
22	24578576	\N	Honeyman et al., 2014, Science	2015-07-31 17:34:17.596297	2015-07-31 17:36:03.227773
23	24909179	\N	Moody et al., 2015, Oncogene	2015-07-31 17:34:17.812213	2015-07-31 17:36:03.653603
24	24167195	\N	Kim et al., 2013, Blood	2015-07-31 17:34:18.02424	2015-07-31 17:36:04.025244
25	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-07-31 17:34:18.044406	2015-07-31 17:36:04.419998
26	23953842	\N	Yewale et al., 2013, Biomaterials	2015-07-31 17:34:18.286599	2015-07-31 17:36:04.82562
27	18712184	\N	Charpidou et al., In Vivo	2015-07-31 17:34:18.296455	2015-07-31 17:36:05.244481
28	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-07-31 17:34:18.546404	2015-07-31 17:36:05.661092
29	10878580	\N	Yu et al., 2000, Bioessays	2015-07-31 17:34:18.557828	2015-07-31 17:36:05.9843
30	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-07-31 17:34:18.792129	2015-07-31 17:36:06.39478
31	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-07-31 17:34:18.802165	2015-07-31 17:36:06.809637
32	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-07-31 17:34:19.074715	2015-07-31 17:36:07.225918
33	23558953	\N	Wu et al., 2013, Cancer Discov	2015-07-31 17:34:19.087107	2015-07-31 17:36:07.652187
34	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-07-31 17:34:19.551576	2015-07-31 17:36:08.061975
35	22541434	\N	Kumar et al., 2012, Cell	2015-07-31 17:34:19.785789	2015-07-31 17:36:08.482308
36	25619630	\N	Mir et al., 2015, Cancer Med	2015-07-31 17:34:19.795875	2015-07-31 17:36:08.902078
37	24065766	\N	Lu et al., 2013, Genes Dev.	2015-07-31 17:34:20.035791	2015-07-31 17:36:09.32373
38	19228619	\N	Yan et al., 2009, N. Engl. J. Med.	2015-07-31 17:34:20.045653	2015-07-31 17:36:09.75261
39	22417203	\N	Patel et al., 2012, N. Engl. J. Med.	2015-07-31 17:34:20.296327	2015-07-31 17:36:10.155249
40	17133099	\N	Levine et al., 2007, Curr. Opin. Hematol.	2015-07-31 17:34:20.512103	2015-07-31 17:36:10.569681
41	23944364	\N	Stankov et al., 2014, Curr. Pharm. Des.	2015-07-31 17:34:20.723003	2015-07-31 17:36:10.958292
42	22589270	\N	Prior et al., 2012, Cancer Res.	2015-07-31 17:34:20.985752	2015-07-31 17:36:11.300316
43	23226219	\N	Chou et al., 2012, PLoS ONE	2015-07-31 17:34:22.047659	2015-07-31 17:36:11.681276
44	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-07-31 17:34:23.570517	2015-07-31 17:36:12.241682
45	25465739	\N	Perri et al., 2015, Crit. Rev. Oncol. Hematol.	2015-07-31 17:34:23.580382	2015-07-31 17:36:12.659154
46	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-07-31 17:34:24.090571	2015-07-31 17:36:13.078174
47	24136165	\N	Cazzola et al., 2013, Blood	2015-07-31 17:34:24.100533	2015-07-31 17:36:13.444847
48	20182602	\N	Olivier et al., 2010, Cold Spring Harb Perspect Biol	2015-07-31 17:34:24.336016	2015-07-31 17:36:13.808619
49	25311244	\N	Okeyo-Owuor et al., 2015, Leukemia	2015-07-31 17:34:25.024576	2015-07-31 17:36:14.229479
50	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-07-31 17:34:25.035485	2015-07-31 17:36:14.645712
75	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-07-31 17:35:37.199873	2015-07-31 17:36:15.018373
76	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-07-31 17:35:37.325762	2015-07-31 17:36:15.480358
77	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-07-31 17:35:37.368605	2015-07-31 17:36:15.895597
78	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-07-31 17:35:37.511124	2015-07-31 17:36:16.315403
79	16081687	Retrospective study	Levine et al., 2005, Blood	2015-07-31 17:35:37.558212	2015-07-31 17:36:16.71046
80	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-07-31 17:35:37.669503	2015-07-31 17:36:17.253802
81	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-07-31 17:35:37.714278	2015-07-31 17:36:17.671337
82	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-07-31 17:35:37.817533	2015-07-31 17:36:18.090247
83	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-07-31 17:35:37.865082	2015-07-31 17:36:18.547964
84	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-07-31 17:35:37.902815	2015-07-31 17:36:19.032832
85	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-07-31 17:35:38.01592	2015-07-31 17:36:19.403833
86	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-07-31 17:35:38.058362	2015-07-31 17:36:19.766314
87	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-07-31 17:35:38.109807	2015-07-31 17:36:20.106103
88	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-07-31 17:35:38.153497	2015-07-31 17:36:20.596466
89	15146165		Lasota et al., 2004, Lab. Invest.	2015-07-31 17:35:38.232248	2015-07-31 17:36:20.960939
90	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-07-31 17:35:38.298418	2015-07-31 17:36:21.341927
91	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-07-31 17:35:38.342353	2015-07-31 17:36:21.744882
92	21030459		Sasaki et al., 2010, Cancer Res.	2015-07-31 17:35:38.396554	2015-07-31 17:36:22.163322
93	21575866		Sakamoto et al., 2011, Cancer Cell	2015-07-31 17:35:38.484499	2015-07-31 17:36:22.582397
94	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-07-31 17:35:38.524451	2015-07-31 17:36:23.008352
95	18923525		George et al., 2008, Nature	2015-07-31 17:35:38.568135	2015-07-31 17:36:23.356302
96	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-07-31 17:35:38.723869	2015-07-31 17:36:23.729303
97	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-07-31 17:35:38.767632	2015-07-31 17:36:24.070317
98	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-07-31 17:35:38.808	2015-07-31 17:36:24.448376
99	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-07-31 17:35:39.067376	2015-07-31 17:36:24.985447
100	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-07-31 17:35:39.246967	2015-07-31 17:36:25.285759
101	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-07-31 17:35:39.311109	2015-07-31 17:36:25.715808
102	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-07-31 17:35:39.489941	2015-07-31 17:36:26.133305
103	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-07-31 17:35:39.583812	2015-07-31 17:36:26.550352
104	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-07-31 17:35:39.658613	2015-07-31 17:36:26.868799
105	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-07-31 17:35:39.708075	2015-07-31 17:36:27.281694
106	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-07-31 17:35:39.778474	2015-07-31 17:36:27.700232
107	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-07-31 17:35:39.820853	2015-07-31 17:36:28.13109
108	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-07-31 17:35:39.861575	2015-07-31 17:36:28.508156
109	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-07-31 17:35:40.06225	2015-07-31 17:36:28.95526
110	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-07-31 17:35:40.121282	2015-07-31 17:36:29.787515
111	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-07-31 17:35:40.179428	2015-07-31 17:36:30.413946
112	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-07-31 17:35:40.232378	2015-07-31 17:36:30.836699
113	23056499		Couto et al., 2012, PLoS ONE	2015-07-31 17:35:40.319196	2015-07-31 17:36:31.19681
114	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-07-31 17:35:40.361274	2015-07-31 17:36:31.564876
115	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-07-31 17:35:40.406612	2015-07-31 17:36:31.982014
116	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-07-31 17:35:40.447811	2015-07-31 17:36:32.402948
117	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-07-31 17:35:40.492025	2015-07-31 17:36:32.725431
118	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-07-31 17:35:40.535495	2015-07-31 17:36:33.548048
119	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-07-31 17:35:40.583951	2015-07-31 17:36:33.967139
120	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-07-31 17:35:40.623643	2015-07-31 17:36:34.386286
121	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-07-31 17:35:40.670424	2015-07-31 17:36:34.803504
122	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-07-31 17:35:40.717404	2015-07-31 17:36:35.329114
123	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-07-31 17:35:40.759614	2015-07-31 17:36:35.746416
124	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-07-31 17:35:40.808766	2015-07-31 17:36:36.110618
125	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-07-31 17:35:40.894224	2015-07-31 17:36:36.509831
126	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-07-31 17:35:40.96381	2015-07-31 17:36:37.101928
127	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-07-31 17:35:41.011224	2015-07-31 17:36:37.731504
128	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-07-31 17:35:41.090699	2015-07-31 17:36:38.145987
129	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-07-31 17:35:41.137701	2015-07-31 17:36:38.58117
130	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-07-31 17:35:41.185851	2015-07-31 17:36:38.980644
131	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-07-31 17:35:41.273842	2015-07-31 17:36:39.293854
132	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-07-31 17:35:41.323851	2015-07-31 17:36:39.718734
133	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-07-31 17:35:41.452846	2015-07-31 17:36:40.140073
134	22072639		Bresler et al., 2011, Sci Transl Med	2015-07-31 17:35:41.62277	2015-07-31 17:36:40.55503
135	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-07-31 17:35:41.763387	2015-07-31 17:36:40.958359
136	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-07-31 17:35:41.83135	2015-07-31 17:36:41.381778
137	14726504	Retrospective study	Frhling et al., 2004, J. Clin. Oncol.	2015-07-31 17:35:41.888264	2015-07-31 17:36:42.4256
138	23558953		Wu et al., 2013, Cancer Discov	2015-07-31 17:35:42.34021	2015-07-31 17:36:42.845627
139	24801015		Port et al., 2014, Ann. Hematol.	2015-07-31 17:35:42.427386	2015-07-31 17:36:43.261775
140	19965647		Burnett et al., 2010, Blood	2015-07-31 17:35:42.468385	2015-07-31 17:36:43.680393
141	17957027	Retrospective study	Gale et al., 2008, Blood	2015-07-31 17:35:42.506185	2015-07-31 17:36:44.018104
142	21537333		Schnittger et al., 2011, Leukemia	2015-07-31 17:35:42.576741	2015-07-31 17:36:44.620536
143	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-07-31 17:35:42.745602	2015-07-31 17:36:45.039742
144	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-07-31 17:35:42.816204	2015-07-31 17:36:45.453981
145	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-07-31 17:35:42.919532	2015-07-31 17:36:45.874028
146	18202412	In vitro study	Livre et al., 2008, J. Clin. Oncol.	2015-07-31 17:35:42.999658	2015-07-31 17:36:46.290457
147	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-07-31 17:35:43.080544	2015-07-31 17:36:46.9184
148	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-07-31 17:35:43.15455	2015-07-31 17:36:47.238661
149	19357394		Vardiman et al., 2009, Blood	2015-07-31 17:35:43.195692	2015-07-31 17:36:47.590853
150	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-07-31 17:35:43.237276	2015-07-31 17:36:48.169656
151	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-07-31 17:35:43.275558	2015-07-31 17:36:48.539917
152	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-07-31 17:35:43.40989	2015-07-31 17:36:48.906996
153	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-07-31 17:35:43.521677	2015-07-31 17:36:49.394227
154	24797300		Tassara et al., 2014, Blood	2015-07-31 17:35:43.562507	2015-07-31 17:36:49.861117
155	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-07-31 17:35:43.757264	2015-07-31 17:36:50.259954
156	19047294	Retrospective study	Bchner et al., 2009, J. Clin. Oncol.	2015-07-31 17:35:43.867017	2015-07-31 17:36:50.678788
157	24855211		Linch et al., 2014, Blood	2015-07-31 17:35:43.910087	2015-07-31 17:36:51.096306
158	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-07-31 17:35:43.948432	2015-07-31 17:36:51.515973
159	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-07-31 17:35:44.036391	2015-07-31 17:36:52.351055
160	16051734	Retrospective study	Dhner et al., 2005, Blood	2015-07-31 17:35:44.074759	2015-07-31 17:36:52.766563
161	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-07-31 17:35:44.147686	2015-07-31 17:36:53.079474
162	24859829		Tian et al., 2014, Int. J. Hematol.	2015-07-31 17:35:44.256243	2015-07-31 17:36:53.498184
163	21719597	In vitro study	Balusu et al., 2011, Blood	2015-07-31 17:35:44.327396	2015-07-31 17:36:53.820294
164	24927407		Ehninger et al., 2014, Blood Cancer J	2015-07-31 17:35:44.377471	2015-07-31 17:36:54.228782
165	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-07-31 17:35:44.720984	2015-07-31 17:36:54.611589
166	22923433	Retrospective study	Iyer et al., 2012, Science	2015-07-31 17:35:44.778025	2015-07-31 17:36:54.962983
167	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-07-31 17:35:44.831362	2015-07-31 17:36:55.378903
168	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-07-31 17:35:44.873966	2015-07-31 17:36:55.796221
169	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-07-31 17:35:44.945879	2015-07-31 17:36:56.212821
170	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-07-31 17:35:45.061105	2015-07-31 17:36:56.634791
171	23346317	Review	Goff, 2013, J Gynecol Oncol	2015-07-31 17:35:45.411706	2015-07-31 17:36:57.048414
172	20537386		An et al., 2010, Leuk. Res.	2015-07-31 17:35:45.518352	2015-07-31 17:36:57.468366
173	12476305		Nimmanapalli et al., 2002, Oncogene	2015-07-31 17:35:45.562322	2015-07-31 17:36:57.884807
174	25212276		Schmidt et al., 2014, Leukemia	2015-07-31 17:35:45.722388	2015-07-31 17:36:58.303516
175	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-07-31 17:35:45.794351	2015-07-31 17:36:58.82575
176	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-07-31 17:35:45.927777	2015-07-31 17:36:59.349329
177	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-07-31 17:35:45.968937	2015-07-31 17:36:59.767233
178	17070615		Gautschi et al., 2007, Lung Cancer	2015-07-31 17:35:46.132174	2015-07-31 17:37:00.236794
179	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-07-31 17:35:46.230802	2015-07-31 17:37:00.555515
180	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-07-31 17:35:46.311757	2015-07-31 17:37:01.020078
181	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-07-31 17:35:46.355956	2015-07-31 17:37:01.437646
182	10547574		Takano et al., 1999, J. Pathol.	2015-07-31 17:35:46.398031	2015-07-31 17:37:01.856695
183	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-07-31 17:35:46.544671	2015-07-31 17:37:02.272262
184	23079656		Sawai et al., 2012, Cancer Cell	2015-07-31 17:35:46.586918	2015-07-31 17:37:02.691444
185	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-07-31 17:35:46.635864	2015-07-31 17:37:03.108423
186	10224221		Donnellan et al., 1999, FASEB J.	2015-07-31 17:35:46.688343	2015-07-31 17:37:03.431676
187	23898052		Logan et al., 2013, Anticancer Res.	2015-07-31 17:35:46.794687	2015-07-31 17:37:03.783729
188	25002028		Vora et al., 2014, Cancer Cell	2015-07-31 17:35:46.833738	2015-07-31 17:37:04.152271
189	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-07-31 17:35:46.933527	2015-07-31 17:37:04.778188
190	22328973		Hammerman et al., 2011, Cancer Discov	2015-07-31 17:35:47.002663	2015-07-31 17:37:05.199226
191	16730237		Ji et al., 2006, Cancer Cell	2015-07-31 17:35:47.254	2015-07-31 17:37:05.585671
192	15118125		Paez et al., 2004, Science	2015-07-31 17:35:47.297008	2015-07-31 17:37:05.921344
193	24736073		Lim et al., 2014, J Thorac Oncol	2015-07-31 17:35:47.335246	2015-07-31 17:37:06.317533
194	24457318		Fukihara et al., 2014, Oncology	2015-07-31 17:35:47.416312	2015-07-31 17:37:06.659215
195	24662454		Douillard et al., 2014, J Thorac Oncol	2015-07-31 17:35:47.461045	2015-07-31 17:37:07.076261
196	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-07-31 17:35:47.534121	2015-07-31 17:37:07.597651
197	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-07-31 17:35:47.593189	2015-07-31 17:37:08.015662
198	24623981		Ding et al., 2014, Onco Targets Ther	2015-07-31 17:35:47.631509	2015-07-31 17:37:08.43221
199	24729716		Li et al., 2014, Onco Targets Ther	2015-07-31 17:35:47.687375	2015-07-31 17:37:08.85107
200	24636847		Li et al., 2014, Lung Cancer	2015-07-31 17:35:47.724554	2015-07-31 17:37:09.269966
201	24658966		Ai et al., 2014, Amino Acids	2015-07-31 17:35:47.766754	2015-07-31 17:37:09.686275
202	21430269		Sequist et al., 2011, Sci Transl Med	2015-07-31 17:35:47.812673	2015-07-31 17:37:10.109119
203	23220880		Bose et al., 2013, Cancer Discov	2015-07-31 17:35:47.857619	2015-07-31 17:37:10.500544
204	24185512		Toy et al., 2013, Nat. Genet.	2015-07-31 17:35:48.342992	2015-07-31 17:37:10.861152
205	24185510		Robinson et al., 2013, Nat. Genet.	2015-07-31 17:35:48.405483	2015-07-31 17:37:11.362759
206	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-07-31 17:35:48.792577	2015-07-31 17:37:11.783013
207	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-07-31 17:35:48.892624	2015-07-31 17:37:12.145044
208	14726387	Clinical trial	Smith et al., 2004, Blood	2015-07-31 17:35:48.934581	2015-07-31 17:37:12.489535
209	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-07-31 17:35:48.976805	2015-07-31 17:37:13.039556
210	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-07-31 17:35:49.056715	2015-07-31 17:37:13.421297
211	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-07-31 17:35:49.096821	2015-07-31 17:37:13.864875
212	22368270	Clinical trial	Man et al., 2012, Blood	2015-07-31 17:35:49.164213	2015-07-31 17:37:14.283319
213	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-07-31 17:35:49.311518	2015-07-31 17:37:14.606011
214	22624710		Barbacid, 2012, Cancer Cell	2015-07-31 17:35:49.356177	2015-07-31 17:37:15.01337
215	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-07-31 17:35:49.400791	2015-07-31 17:37:15.535892
216	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-07-31 17:35:49.442863	2015-07-31 17:37:15.95502
217	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-07-31 17:35:49.480373	2015-07-31 17:37:16.37284
218	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-07-31 17:35:49.519332	2015-07-31 17:37:16.794339
219	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-07-31 17:35:49.567678	2015-07-31 17:37:17.20798
220	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-07-31 17:35:49.611208	2015-07-31 17:37:17.577088
221	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-07-31 17:35:49.754055	2015-07-31 17:37:17.879036
222	20421455	Retrospective study	Thol et al., 2010, Blood	2015-07-31 17:35:49.809576	2015-07-31 17:37:18.211149
223	21997850		Lin et al., 2012, Ann. Hematol.	2015-07-31 17:35:49.84658	2015-07-31 17:37:18.530329
224	22033490		Patnaik et al., 2012, Leukemia	2015-07-31 17:35:49.88939	2015-07-31 17:37:18.983159
225	21596855		Green et al., 2011, Blood	2015-07-31 17:35:49.925435	2015-07-31 17:37:19.400811
226	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-07-31 17:35:50.05961	2015-07-31 17:37:19.745821
227	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-07-31 17:35:50.105786	2015-07-31 17:37:20.133683
228	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-07-31 17:35:50.155243	2015-07-31 17:37:20.496576
229	18421059	Observational study	Hodi et al., 2008, J. Clin. Oncol.	2015-07-31 17:35:50.327858	2015-07-31 17:37:20.966122
230	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-07-31 17:35:50.380592	2015-07-31 17:37:21.391446
231	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-07-31 17:35:50.430945	2015-07-31 17:37:21.803166
232	16618717	Retrospective study	Livre et al., 2006, Cancer Res.	2015-07-31 17:35:50.510237	2015-07-31 17:37:22.228253
233	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-07-31 17:35:50.555712	2015-07-31 17:37:22.528956
234	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-07-31 17:35:50.606061	2015-07-31 17:37:22.854051
235	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-07-31 17:35:50.644908	2015-07-31 17:37:23.264281
236	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-07-31 17:35:50.682506	2015-07-31 17:37:23.681363
237	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-07-31 17:35:50.732383	2015-07-31 17:37:24.100291
238	19794967		Marchetti et al., 2009, Neoplasia	2015-07-31 17:35:50.77439	2015-07-31 17:37:24.624853
239	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-07-31 17:35:50.837784	2015-07-31 17:37:25.145007
240	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-07-31 17:35:50.958464	2015-07-31 17:37:25.469568
241	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-07-31 17:35:51.276228	2015-07-31 17:37:25.873569
242	20007775		Westhoff et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-07-31 17:35:51.408107	2015-07-31 17:37:26.296431
243	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-07-31 17:35:51.549805	2015-07-31 17:37:26.818905
244	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-07-31 17:35:51.589653	2015-07-31 17:37:27.339491
245	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-07-31 17:35:51.630411	2015-07-31 17:37:27.755646
246	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-07-31 17:35:51.671868	2015-07-31 17:37:28.164953
247	8674046		Yoshida et al., 1996, Cancer Res.	2015-07-31 17:35:52.050698	2015-07-31 17:37:28.556127
248	22479427		Carico et al., 2012, PLoS ONE	2015-07-31 17:35:52.096638	2015-07-31 17:37:28.915808
249	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-07-31 17:35:52.161172	2015-07-31 17:37:29.327208
250	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-07-31 17:35:52.239684	2015-07-31 17:37:29.740854
251	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-07-31 17:35:52.319086	2015-07-31 17:37:30.158038
252	22698404		Jackson et al., 2012, Cancer Cell	2015-07-31 17:35:52.364031	2015-07-31 17:37:30.485531
253	9569050		Berns et al., 1998, Br. J. Cancer	2015-07-31 17:35:52.453808	2015-07-31 17:37:30.856755
254	22923433		Iyer et al., 2012, Science	2015-07-31 17:35:52.743338	2015-07-31 17:37:31.169857
255	23029227		Qian et al., 2012, PLoS ONE	2015-07-31 17:35:52.830112	2015-07-31 17:37:31.508988
256	23861105		Wu et al., 2013, Am. J. Hematol.	2015-07-31 17:35:52.953204	2015-07-31 17:37:31.936139
257	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-07-31 17:35:53.118583	2015-07-31 17:37:32.296997
258	24186003		Lilljebjrn et al., 2014, Leukemia	2015-07-31 17:35:53.155736	2015-07-31 17:37:32.67325
259	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-07-31 17:35:53.246982	2015-07-31 17:37:33.034401
260	16818686	Retrospective study	Jackman et al., 2006, Clin. Cancer Res.	2015-07-31 17:35:53.333209	2015-07-31 17:37:33.421888
261	9552022	Retrospective study	de Alava et al., 1998, J. Clin. Oncol.	2015-07-31 17:35:53.38194	2015-07-31 17:37:33.836372
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at, deleted, deleted_at, genome_build, chromosome, start, stop, reference_bases, variant_bases, representative_transcript, chromosome2, start2, stop2, reference_build) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-07-31 17:34:26.348937	2015-07-31 17:34:26.348937	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-07-31 17:34:26.377346	2015-07-31 17:34:26.377346	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-07-31 17:34:26.398081	2015-07-31 17:34:26.398081	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-07-31 17:34:26.439731	2015-07-31 17:34:26.439731	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
6	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-07-31 17:34:26.457967	2015-07-31 17:34:26.457967	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
7	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-07-31 17:34:26.477892	2015-07-31 17:34:26.477892	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-07-31 17:34:26.603845	2015-07-31 17:34:26.603845	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-07-31 17:34:26.628626	2015-07-31 17:34:26.628626	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-07-31 17:34:26.693882	2015-07-31 17:34:26.693882	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
18	8	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-07-31 17:34:26.714039	2015-07-31 17:34:26.714039	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
19	8	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-07-31 17:34:26.733404	2015-07-31 17:34:26.733404	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
20	8	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-07-31 17:34:26.755868	2015-07-31 17:34:26.755868	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
21	9	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-07-31 17:34:26.780682	2015-07-31 17:34:26.780682	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
22	9	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-07-31 17:34:26.806265	2015-07-31 17:34:26.806265	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
23	10	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-07-31 17:34:26.826749	2015-07-31 17:34:26.826749	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
24	11	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-07-31 17:34:26.856014	2015-07-31 17:34:26.856014	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
25	13	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-07-31 17:34:26.874544	2015-07-31 17:34:26.874544	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
26	12	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-07-31 17:34:26.898874	2015-07-31 17:34:26.898874	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
27	14	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-07-31 17:34:26.923668	2015-07-31 17:34:26.923668	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
28	15	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-07-31 17:34:26.948886	2015-07-31 17:34:26.948886	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
29	15	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-07-31 17:34:26.970419	2015-07-31 17:34:26.970419	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
30	33	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-07-31 17:34:26.990286	2015-07-31 17:34:26.990286	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
31	16	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-07-31 17:34:27.011519	2015-07-31 17:34:27.011519	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
32	18	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-07-31 17:34:27.032025	2015-07-31 17:34:27.032025	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
74	30	EXON 1 MUTATIONS	A study by Livre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-07-31 17:34:28.031764	2015-07-31 17:34:28.031764	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
37	20	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.147411	2015-07-31 17:34:27.147411	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
40	20	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.210752	2015-07-31 17:34:27.210752	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
41	20	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.23349	2015-07-31 17:34:27.23349	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
42	20	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.257674	2015-07-31 17:34:27.257674	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
75	30	EXON 2 MUTATIONS	In a study by Pao et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-07-31 17:34:28.075763	2015-07-31 17:34:28.075763	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
46	21	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-31 17:34:27.347167	2015-07-31 17:34:27.347167	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
47	21	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-31 17:34:27.373001	2015-07-31 17:34:27.373001	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
51	22	P235R		2015-07-31 17:34:27.472176	2015-07-31 17:34:27.472176	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
52	22	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-07-31 17:34:27.493881	2015-07-31 17:34:27.493881	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
53	23	FGFR3-BAIAP2L1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-07-31 17:34:27.517628	2015-07-31 17:34:27.517628	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
54	23	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-07-31 17:34:27.536771	2015-07-31 17:34:27.536771	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
55	24	ITD MUTATIONS	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-07-31 17:34:27.559614	2015-07-31 17:34:27.559614	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
56	24	TKD MUTATIONS	FLT3 tyrosine kinase domain mutations (FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-07-31 17:34:27.580298	2015-07-31 17:34:27.580298	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
57	25	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-07-31 17:34:27.614167	2015-07-31 17:34:27.614167	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
76	30	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-31 17:34:28.09605	2015-07-31 17:34:28.09605	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
58	26	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-07-31 17:34:27.642402	2015-07-31 17:34:27.642402	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
62	27	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-07-31 17:34:27.75132	2015-07-31 17:34:27.75132	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
66	29	EXON 11 MUTATIONS	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-07-31 17:34:27.848348	2015-07-31 17:34:27.848348	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
67	29	INTERNAL DUPLICATION	c-KIT internal duplications have been observed in exon 11, within the juxtamembrane domain. In a case study of an anal melanoma patient harboring this event, imatinib confered marked response. Also, cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-07-31 17:34:27.867384	2015-07-31 17:34:27.867384	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
68	29	EXON 13 MUTATIONS	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-07-31 17:34:27.891646	2015-07-31 17:34:27.891646	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
69	29	EXON 14 MUTATIONS	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-07-31 17:34:27.917361	2015-07-31 17:34:27.917361	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
70	29	EXON 17 MUTATIONS	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-07-31 17:34:27.942235	2015-07-31 17:34:27.942235	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
71	29	EXON 9 MUTATIONS	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-07-31 17:34:27.964612	2015-07-31 17:34:27.964612	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
72	29	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-07-31 17:34:27.985024	2015-07-31 17:34:27.985024	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
77	30	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-31 17:34:28.12035	2015-07-31 17:34:28.12035	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
80	30	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-31 17:34:28.177825	2015-07-31 17:34:28.177825	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
84	32	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-07-31 17:34:28.263801	2015-07-31 17:34:28.263801	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
85	34	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-07-31 17:34:28.282369	2015-07-31 17:34:28.282369	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
86	35	EXON 12 MUTATIONS	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurence with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-07-31 17:34:28.303365	2015-07-31 17:34:28.303365	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
88	36	EXON 1 MUTATIONS	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-07-31 17:34:28.343122	2015-07-31 17:34:28.343122	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
89	36	EXON 2 MUTATIONS	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-07-31 17:34:28.362606	2015-07-31 17:34:28.362606	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
90	36	EXON 3 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-07-31 17:34:28.380929	2015-07-31 17:34:28.380929	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
91	36	EXON 4 MUTATIONS	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-07-31 17:34:28.399185	2015-07-31 17:34:28.399185	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
92	36	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-07-31 17:34:28.421202	2015-07-31 17:34:28.421202	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
94	36	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-07-31 17:34:28.459697	2015-07-31 17:34:28.459697	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
111	40	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-07-31 17:34:28.887197	2015-07-31 17:34:28.887197	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
97	38	V536E		2015-07-31 17:34:28.527637	2015-07-31 17:34:28.527637	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
101	38	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-31 17:34:28.634789	2015-07-31 17:34:28.634789	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
102	38	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-31 17:34:28.656664	2015-07-31 17:34:28.656664	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
105	37	EXON 20 MUTATIONS	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-07-31 17:34:28.725851	2015-07-31 17:34:28.725851	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
106	37	EXON 9 MUTATIONS	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-07-31 17:34:28.748238	2015-07-31 17:34:28.748238	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
108	39	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-07-31 17:34:28.794534	2015-07-31 17:34:28.794534	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
109	17	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-07-31 17:34:28.832594	2015-07-31 17:34:28.832594	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
114	44	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-07-31 17:34:28.959611	2015-07-31 17:37:43.285119	f	\N	\N	2	198267359	198267359	C	A	\N	\N	\N	\N	\N
123	45	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-31 17:34:29.152723	2015-07-31 17:34:29.152723	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
124	46	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-07-31 17:34:29.176292	2015-07-31 17:34:29.176292	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
125	46	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-07-31 17:34:29.196399	2015-07-31 17:34:29.196399	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
126	47	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-07-31 17:34:29.215544	2015-07-31 17:34:29.215544	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-07-31 17:34:26.49698	2015-07-31 17:37:42.454977	f	\N	\N	2	29443695	29443695	G	T	\N	\N	\N	\N	\N
127	48	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-07-31 17:34:29.254658	2015-07-31 17:34:29.254658	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
128	48	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-07-31 17:34:29.274758	2015-07-31 17:34:29.274758	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
129	49	EXON 7 MUTATIONS	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-07-31 17:34:29.295269	2015-07-31 17:34:29.295269	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
130	49	EXON 9 MUTATIONS	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-07-31 17:34:29.318781	2015-07-31 17:34:29.318781	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
131	6	LOSS-OF-FUNCTION	BRCA1 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-07-31 17:34:29.33972	2015-07-31 17:34:29.33972	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
132	7	LOSS-OF-FUNCTION	BRCA2 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.	2015-07-31 17:34:29.359584	2015-07-31 17:34:29.359584	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
133	19	EXON 19 DELETION	Deletions within exon 19 of EGFR are most common in lung cancer. These deletions, in non-small cell lung cancer, have been shown to be sensitive to the EGFR tyrosine kinase inhibitors gefitinib and erlotinib. There is also data to suggest that this event is a good prognostic marker in lung adenocarcinoma.	2015-07-31 17:34:29.379651	2015-07-31 17:34:29.379651	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
134	19	G719S	While not as recurrent as the L858R mutation, EGFR G719S has also been shown to be an activating mutation. In lung cancer cell lines, it also confers sensitivity to the tyrosine kinase inhibitors gefitinib and erlotinib.	2015-07-31 17:34:29.399469	2015-07-31 17:34:29.399469	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
135	50	D1643H	Activating mutations in NOTCH1, including D1643H, have been shown to be poor prognostic markers in lung cancer.	2015-07-31 17:34:29.423368	2015-07-31 17:34:29.423368	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
136	50	R2328W	Activating mutations in NOTCH1, including R2328W, have been shown to be poor prognostic markers in lung cancer.	2015-07-31 17:34:29.441319	2015-07-31 17:34:29.441319	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
137	50	V2444FS	Activating mutations in NOTCH1, including a frameshift insertion at V2444, have been shown to be poor prognostic markers in lung cancer.	2015-07-31 17:34:29.462204	2015-07-31 17:34:29.462204	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
138	50	S2275FS	Activating mutations in NOTCH1, including a frameshift insertion at S2275, have been shown to be poor prognostic markers in lung cancer.	2015-07-31 17:34:29.482168	2015-07-31 17:34:29.482168	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
139	51	L63V	Activating mutations in DDR2, including L63V, has been shown to be sensitive to dasatinib in cell lines.	2015-07-31 17:34:29.501038	2015-07-31 17:34:29.501038	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
140	51	L239R	Activating mutations in DDR2, including L239R, has been shown to be sensitive to dasatinib in cell lines.	2015-07-31 17:34:29.522343	2015-07-31 17:34:29.522343	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
141	51	G253C	Activating mutations in DDR2, including G253C, has been shown to be sensitive to dasatinib in cell lines.	2015-07-31 17:34:29.540165	2015-07-31 17:34:29.540165	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
142	51	G505S	Activating mutations in DDR2, including G505S, has been shown to be sensitive to dasatinib in cell lines.	2015-07-31 17:34:29.560596	2015-07-31 17:34:29.560596	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
143	51	I638F	DDR2 I638F has been shown to be a loss of function mutation, but also confers sensitivity to dasatinib in cell lines.	2015-07-31 17:34:29.578484	2015-07-31 17:34:29.578484	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
144	51	G774V	Activating mutations in DDR2, including G774V, has been shown to be sensitive to dasatinib in cell lines.	2015-07-31 17:34:29.597084	2015-07-31 17:34:29.597084	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
145	51	S768R	Activating mutations in DDR2, including S768R, has been shown to be sensitive to dasatinib in cell lines.	2015-07-31 17:34:29.62014	2015-07-31 17:34:29.62014	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
146	52	AMPLIFICATION	MET amplification, like EGFR T790M, has been shown to be capable of driving acquired resistance to dacomitinib in patients with lung adenocarcinoma.	2015-07-31 17:34:29.640622	2015-07-31 17:34:29.640622	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
147	30	G12V	KRAS G12V, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-07-31 17:34:29.662479	2015-07-31 17:34:29.662479	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
148	30	G12A	KRAS G12A, like EGFR T790M, has been shown to be capable of driving acquired resistance to tyrosine kinase inhibitors in lung adenocarcinoma.	2015-07-31 17:34:29.683104	2015-07-31 17:34:29.683104	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
149	54	EWS-FLI1	EWS-FLI1 fusions have been found to be extremely common in Ewing's sarcomas. The presence of the fusion is also thought to be a positive prognostic marker of survival.	2015-07-31 17:34:29.701635	2015-07-31 17:34:29.701635	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
150	53	EWS-FLI1	EWS-FLI1 fusions have been found to be extremely common in Ewing's sarcomas. The presence of the fusion is also thought to be a positive prognostic marker of survival.	2015-07-31 17:34:29.720639	2015-07-31 17:34:29.720639	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
151	53	TYPE 1 EWS-FLI1	The most common EWS-FLI1 fusion (type 1) links EWS exon 7 to FLI1 exon 6. This fusion product is found to be associated with better outcome than other EWS-FLI1 fusions in Ewing's Sarcoma.	2015-07-31 17:34:29.744166	2015-07-31 17:34:29.744166	f	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-07-31 17:34:26.419827	2015-07-31 17:37:42.427616	f	\N	\N	14	105246551	105246551	C	T	\N	\N	\N	\N	\N
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-07-31 17:34:26.520906	2015-07-31 17:37:42.486156	f	\N	\N	2	29432664	29432664	C	T	\N	\N	\N	\N	\N
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-07-31 17:34:26.539618	2015-07-31 17:37:42.504436	f	\N	\N	X	47426121	47426121	C	G	\N	\N	\N	\N	\N
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-07-31 17:34:26.562299	2015-07-31 17:37:42.524629	f	\N	\N	7	140453135	140453136	CA	AT	\N	\N	\N	\N	\N
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-07-31 17:34:26.58269	2015-07-31 17:37:42.541847	f	\N	\N	7	140453136	140453136	A	T	\N	\N	\N	\N	\N
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-07-31 17:34:26.649852	2015-07-31 17:37:42.563772	f	\N	\N	7	140453137	140453137	C	T	\N	\N	\N	\N	\N
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-07-31 17:34:26.670935	2015-07-31 17:37:42.583438	f	\N	\N	7	140453136	140453137	AC	CT	\N	\N	\N	\N	\N
33	19	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-07-31 17:34:27.060513	2015-07-31 17:37:42.607911	f	\N	\N	7	55259514	55259515	CT	AG	\N	\N	\N	\N	\N
34	19	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies in non-small cell lung cancer. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-07-31 17:34:27.083137	2015-07-31 17:37:42.624471	f	\N	\N	7	55249071	55249071	C	T	\N	\N	\N	\N	\N
35	20	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.107553	2015-07-31 17:37:42.642437	f	\N	\N	17	37880261	37880261	G	C	\N	\N	\N	\N	\N
36	20	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.12721	2015-07-31 17:37:42.658394	f	\N	\N	17	37880261	37880261	G	T	\N	\N	\N	\N	\N
38	20	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.167919	2015-07-31 17:37:42.674963	f	\N	\N	17	37868205	37868205	G	C	\N	\N	\N	\N	\N
39	20	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-07-31 17:34:27.188412	2015-07-31 17:37:42.69319	f	\N	\N	17	37880220	37880220	T	C	\N	\N	\N	\N	\N
43	20	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.281526	2015-07-31 17:37:42.71601	f	\N	\N	17	37881616	37881616	C	T	\N	\N	\N	\N	\N
79	30	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-31 17:34:28.158726	2015-07-31 17:37:42.952722	f	\N	\N	12	25398284	25398284	C	T	\N	\N	\N	\N	\N
44	20	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.301723	2015-07-31 17:37:42.733417	f	\N	\N	17	37881000	37881000	G	T	\N	\N	\N	\N	\N
45	20	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-07-31 17:34:27.32742	2015-07-31 17:37:42.748847	f	\N	\N	17	37881332	37881332	G	A	\N	\N	\N	\N	\N
48	21	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-31 17:34:27.393155	2015-07-31 17:37:42.764649	f	\N	\N	6	152419923	152419923	A	G	\N	\N	\N	\N	\N
49	21	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-31 17:34:27.415075	2015-07-31 17:37:42.781957	f	\N	\N	6	152419922	152419922	T	A	\N	\N	\N	\N	\N
50	21	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-07-31 17:34:27.452756	2015-07-31 17:37:42.79782	f	\N	\N	6	152419923	152419923	A	C	\N	\N	\N	\N	\N
59	26	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-07-31 17:34:27.673566	2015-07-31 17:37:42.816628	f	\N	\N	2	209113113	209113113	G	A	\N	\N	\N	\N	\N
60	26	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-07-31 17:34:27.695415	2015-07-31 17:37:42.833556	f	\N	\N	2	209113112	209113112	C	T	\N	\N	\N	\N	\N
61	26	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-07-31 17:34:27.719751	2015-07-31 17:37:42.850448	f	\N	\N	2	209113112	209113112	C	A	\N	\N	\N	\N	\N
63	27	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-07-31 17:34:27.776499	2015-07-31 17:37:42.865438	f	\N	\N	15	90631838	90631838	C	T	\N	\N	\N	\N	\N
64	28	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-07-31 17:34:27.801784	2015-07-31 17:37:42.882202	f	\N	\N	9	5073770	5073770	G	T	\N	\N	\N	\N	\N
65	29	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-07-31 17:34:27.825641	2015-07-31 17:37:42.898481	f	\N	\N	4	55599321	55599321	A	T	\N	\N	\N	\N	\N
73	29	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-07-31 17:34:28.01091	2015-07-31 17:37:42.919258	f	\N	\N	4	55594258	55594258	T	C	\N	\N	\N	\N	\N
78	30	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-31 17:34:28.139531	2015-07-31 17:37:42.937638	f	\N	\N	12	25398285	25398285	C	A	\N	\N	\N	\N	\N
81	30	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-07-31 17:34:28.200785	2015-07-31 17:37:42.968642	f	\N	\N	12	25398281	25398281	C	T	\N	\N	\N	\N	\N
82	31	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-07-31 17:34:28.220523	2015-07-31 17:37:42.998946	f	\N	\N	15	66729162	66729162	C	T	\N	\N	\N	\N	\N
83	31	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-07-31 17:34:28.241544	2015-07-31 17:37:43.015817	f	\N	\N	15	66727451	66727451	A	C	\N	\N	\N	\N	\N
87	35	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-07-31 17:34:28.322941	2015-07-31 17:37:43.032942	f	\N	\N	5	170837546	170837547	0	TCAG	\N	\N	\N	\N	\N
93	36	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-07-31 17:34:28.441414	2015-07-31 17:37:43.055001	f	\N	\N	1	115258744	115258744	C	T	\N	\N	\N	\N	\N
95	36	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-07-31 17:34:28.478411	2015-07-31 17:37:43.074442	f	\N	\N	1	115256528	115256529	TT	CA	\N	\N	\N	\N	\N
96	36	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-07-31 17:34:28.498796	2015-07-31 17:37:43.094648	f	\N	\N	1	115256528	115256529	TT	CC	\N	\N	\N	\N	\N
98	38	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-31 17:34:28.548138	2015-07-31 17:37:43.117737	f	\N	\N	4	55152092	55152093	GA	AT	\N	\N	\N	\N	\N
99	38	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-31 17:34:28.574887	2015-07-31 17:37:43.140673	f	\N	\N	4	55152093	55152093	A	T	\N	\N	\N	\N	\N
100	38	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-07-31 17:34:28.596889	2015-07-31 17:37:43.161742	f	\N	\N	4	55152092	55152092	G	T	\N	\N	\N	\N	\N
103	37	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-07-31 17:34:28.683126	2015-07-31 17:37:43.179665	f	\N	\N	3	178936082	178936082	G	A	\N	\N	\N	\N	\N
104	37	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-07-31 17:34:28.7053	2015-07-31 17:37:43.198645	f	\N	\N	3	178936091	178936091	G	A	\N	\N	\N	\N	\N
107	37	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-07-31 17:34:28.767345	2015-07-31 17:37:43.216869	f	\N	\N	3	178952085	178952085	A	G	\N	\N	\N	\N	\N
110	41	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-07-31 17:34:28.860994	2015-07-31 17:37:43.232642	f	\N	\N	10	89717672	89717672	C	T	\N	\N	\N	\N	\N
112	42	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-07-31 17:34:28.913194	2015-07-31 17:37:43.254325	f	\N	\N	10	43609950	43609950	C	G	\N	\N	\N	\N	\N
113	42	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-07-31 17:34:28.937243	2015-07-31 17:37:43.270773	f	\N	\N	10	43617416	43617416	T	C	\N	\N	\N	\N	\N
115	44	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-07-31 17:34:28.98069	2015-07-31 17:37:43.300717	f	\N	\N	2	198266834	198266834	T	C	\N	\N	\N	\N	\N
116	45	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-07-31 17:34:29.002979	2015-07-31 17:37:43.315676	f	\N	\N	17	7578406	7578406	C	T	\N	\N	\N	\N	\N
117	45	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-07-31 17:34:29.027369	2015-07-31 17:37:43.329797	f	\N	\N	17	7577538	7577538	C	T	\N	\N	\N	\N	\N
118	45	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-07-31 17:34:29.047756	2015-07-31 17:37:43.344872	f	\N	\N	17	7577539	7577539	G	A	\N	\N	\N	\N	\N
119	45	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-31 17:34:29.067979	2015-07-31 17:37:43.363459	f	\N	\N	17	7577535	7577535	C	G	\N	\N	\N	\N	\N
120	45	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-31 17:34:29.091886	2015-07-31 17:37:43.377135	f	\N	\N	17	7577536	7577536	T	A	\N	\N	\N	\N	\N
121	45	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-31 17:34:29.111502	2015-07-31 17:37:43.391413	f	\N	\N	17	7577121	7577121	G	A	\N	\N	\N	\N	\N
122	45	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-07-31 17:34:29.132022	2015-07-31 17:37:43.406361	f	\N	\N	17	7577120	7577120	C	T	\N	\N	\N	\N	\N
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, description, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_hgvs, evidence_level, evidence_type, variant_origin, evidence_direction, clinical_significance, deleted, deleted_at, drug_interaction_type) FROM stdin;
37	In one patient with lung adenocarcinoma who had a S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	12	75	10	2015-07-31 17:35:37.257069	2015-07-31 17:35:37.257069	2	accepted	\N	\N	X:47426121-47426121 (C->G)	3	2	0	0	0	f	\N	\N
38	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	13	76	32	2015-07-31 17:35:37.335047	2015-07-31 17:35:37.335047	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
39	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older, had higher white blood cell counts and had higher platelet counts than patients wild-type for DNMT3A	13	77	32	2015-07-31 17:35:37.375939	2015-07-31 17:35:37.375939	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
40	Daunorubicin treatment resulted in similar overall survival and disease-free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	13	76	32	2015-07-31 17:35:37.409152	2015-07-31 17:35:37.409152	4	accepted	\N	\N	N/A	1	2	0	1	1	f	\N	\N
41	Idarubicin increases overall and disease free survival in de novo AML patients with DNMT3A R882 mutations compared to those who do not harbor this mutation.	13	76	32	2015-07-31 17:35:37.446582	2015-07-31 17:35:37.446582	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
42	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	13	76	32	2015-07-31 17:35:37.482397	2015-07-31 17:35:37.482397	4	accepted	\N	\N	N/A	1	1	0	1	3	f	\N	\N
43	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	13	78	32	2015-07-31 17:35:37.518499	2015-07-31 17:35:37.518499	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
44	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	14	79	64	2015-07-31 17:35:37.569053	2015-07-31 17:35:37.569053	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	0	0	0	5	f	\N	\N
45	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	15	79	64	2015-07-31 17:35:37.606647	2015-07-31 17:35:37.606647	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	0	0	0	4	f	\N	\N
46	JAK2 V617F is associated with myeloid neoplasms (AML, MDS, CMML/aCML)	15	79	64	2015-07-31 17:35:37.635398	2015-07-31 17:35:37.635398	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	0	0	0	4	f	\N	\N
47	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	16	80	64	2015-07-31 17:35:37.678299	2015-07-31 17:35:37.678299	4	accepted	\N	\N	9:5073770-5073770 (G->T)	1	2	0	0	0	f	\N	\N
48	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wild-type KIT	17	81	65	2015-07-31 17:35:37.724132	2015-07-31 17:35:37.724132	4	accepted	\N	\N	4:55599321-55599321 (A->T)	1	1	0	0	3	f	\N	\N
49	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.	18	82	82	2015-07-31 17:35:37.829977	2015-07-31 17:35:37.829977	3	accepted	\N	\N	15:66729162-66729162 (C->T)	2	2	0	0	1	f	\N	\N
50	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	18	83	83	2015-07-31 17:35:37.873705	2015-07-31 17:35:37.873705	3	accepted	\N	\N	15:66727451-66727451 (A->C)	2	2	0	0	1	f	\N	\N
51	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	19	84	89	2015-07-31 17:35:37.911703	2015-07-31 17:35:37.911703	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
52	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	19	84	90	2015-07-31 17:35:37.944072	2015-07-31 17:35:37.944072	3	accepted	\N	\N	N/A	0	1	0	0	3	f	\N	\N
53	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.	19	84	91	2015-07-31 17:35:37.976648	2015-07-31 17:35:37.976648	3	accepted	\N	\N	N/A	0	1	0	0	3	f	\N	\N
54	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation	18	85	93	2015-07-31 17:35:38.02418	2015-07-31 17:35:38.02418	2	accepted	\N	\N	1:115258744-115258744 (C->T)	3	2	0	0	0	f	\N	\N
55	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	18	86	94	2015-07-31 17:35:38.067161	2015-07-31 17:35:38.067161	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
56	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	18	87	94	2015-07-31 17:35:38.11727	2015-07-31 17:35:38.11727	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
57	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	18	88	95	2015-07-31 17:35:38.160629	2015-07-31 17:35:38.160629	2	accepted	\N	\N	1:115256528-115256529 (TT->CA)	3	2	0	0	0	f	\N	\N
58	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	18	88	96	2015-07-31 17:35:38.197414	2015-07-31 17:35:38.197414	2	accepted	\N	\N	1:115256528-115256529 (TT->CC)	3	2	0	0	0	f	\N	\N
59	GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than malignant.	20	89	99	2015-07-31 17:35:38.244903	2015-07-31 17:35:38.244903	3	accepted	\N	\N	4:55152093-55152093 (A->T)	1	0	0	0	5	f	\N	\N
60	GIST cancer with D842V mutation is resistant to imatinib.	20	90	99	2015-07-31 17:35:38.305482	2015-07-31 17:35:38.305482	4	accepted	\N	\N	4:55152093-55152093 (A->T)	1	2	0	0	1	f	\N	\N
61	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	20	91	99	2015-07-31 17:35:38.349527	2015-07-31 17:35:38.349527	4	accepted	\N	\N	4:55152093-55152093 (A->T)	1	2	0	0	1	f	\N	\N
62	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	21	92	8	2015-07-31 17:35:38.405714	2015-07-31 17:35:38.405714	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
63	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	12	92	8	2015-07-31 17:35:38.439625	2015-07-31 17:35:38.439625	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
64	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	22	93	8	2015-07-31 17:35:38.493431	2015-07-31 17:35:38.493431	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
65	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	22	94	8	2015-07-31 17:35:38.531494	2015-07-31 17:35:38.531494	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
66	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.	22	95	9	2015-07-31 17:35:38.578579	2015-07-31 17:35:38.578579	3	accepted	\N	\N	2:29432664-29432664 (C->T)	2	2	0	0	0	f	\N	\N
67	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.	22	95	9	2015-07-31 17:35:38.613695	2015-07-31 17:35:38.613695	3	accepted	\N	\N	2:29432664-29432664 (C->T)	2	2	0	0	1	f	\N	\N
68	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	12	75	10	2015-07-31 17:35:38.661613	2015-07-31 17:35:38.661613	3	accepted	\N	\N	X:47426121-47426121 (C->G)	2	2	0	0	0	f	\N	\N
69	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro	12	75	10	2015-07-31 17:35:38.691293	2015-07-31 17:35:38.691293	3	accepted	\N	\N	X:47426121-47426121 (C->G)	2	2	0	0	0	f	\N	\N
70	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma	18	96	17	2015-07-31 17:35:38.732191	2015-07-31 17:35:38.732191	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
71	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	13	97	32	2015-07-31 17:35:38.778465	2015-07-31 17:35:38.778465	3	accepted	\N	\N	N/A	0	1	0	0	3	f	\N	\N
72	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	13	98	32	2015-07-31 17:35:38.815848	2015-07-31 17:35:38.815848	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
73	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	13	97	32	2015-07-31 17:35:38.844609	2015-07-31 17:35:38.844609	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
74	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	13	97	32	2015-07-31 17:35:38.873086	2015-07-31 17:35:38.873086	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
75	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	13	77	32	2015-07-31 17:35:38.908238	2015-07-31 17:35:38.908238	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
76	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	13	78	32	2015-07-31 17:35:38.938904	2015-07-31 17:35:38.938904	5	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
77	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs	13	97	32	2015-07-31 17:35:38.971224	2015-07-31 17:35:38.971224	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
78	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	13	78	32	2015-07-31 17:35:39.000361	2015-07-31 17:35:39.000361	5	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
79	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	13	98	32	2015-07-31 17:35:39.031913	2015-07-31 17:35:39.031913	5	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
80	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	13	99	32	2015-07-31 17:35:39.075173	2015-07-31 17:35:39.075173	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
81	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	13	77	32	2015-07-31 17:35:39.105092	2015-07-31 17:35:39.105092	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
82	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	13	99	32	2015-07-31 17:35:39.137261	2015-07-31 17:35:39.137261	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
83	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	13	77	55	2015-07-31 17:35:39.181955	2015-07-31 17:35:39.181955	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
84	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	13	78	55	2015-07-31 17:35:39.21504	2015-07-31 17:35:39.21504	5	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
85	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	23	100	66	2015-07-31 17:35:39.257749	2015-07-31 17:35:39.257749	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
86	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	18	101	66	2015-07-31 17:35:39.319837	2015-07-31 17:35:39.319837	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
87	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	18	101	68	2015-07-31 17:35:39.358778	2015-07-31 17:35:39.358778	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
88	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	23	100	69	2015-07-31 17:35:39.387291	2015-07-31 17:35:39.387291	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
89	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	18	101	70	2015-07-31 17:35:39.420918	2015-07-31 17:35:39.420918	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
90	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.	23	100	71	2015-07-31 17:35:39.457601	2015-07-31 17:35:39.457601	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
177	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	13	145	58	2015-07-31 17:35:42.939672	2015-07-31 17:35:42.939672	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
91	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	18	102	72	2015-07-31 17:35:39.497522	2015-07-31 17:35:39.497522	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	2
92	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	18	102	72	2015-07-31 17:35:39.545708	2015-07-31 17:35:39.545708	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
93	In NSCLC, Exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	12	103	75	2015-07-31 17:35:39.591442	2015-07-31 17:35:39.591442	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
94	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	13	77	86	2015-07-31 17:35:39.628781	2015-07-31 17:35:39.628781	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
95	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	18	104	88	2015-07-31 17:35:39.666681	2015-07-31 17:35:39.666681	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
96	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	13	105	89	2015-07-31 17:35:39.715382	2015-07-31 17:35:39.715382	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
97	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	18	104	89	2015-07-31 17:35:39.748607	2015-07-31 17:35:39.748607	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
98	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	13	106	92	2015-07-31 17:35:39.787064	2015-07-31 17:35:39.787064	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
99	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	19	107	94	2015-07-31 17:35:39.827844	2015-07-31 17:35:39.827844	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
100	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	20	108	98	2015-07-31 17:35:39.870238	2015-07-31 17:35:39.870238	4	accepted	\N	\N	4:55152092-55152093 (GA->AT)	2	2	0	0	0	f	\N	\N
101	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	20	108	99	2015-07-31 17:35:39.91432	2015-07-31 17:35:39.91432	4	accepted	\N	\N	4:55152093-55152093 (A->T)	2	2	0	0	0	f	\N	\N
102	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	20	108	100	2015-07-31 17:35:39.949032	2015-07-31 17:35:39.949032	4	accepted	\N	\N	4:55152092-55152092 (G->T)	2	2	0	0	0	f	\N	\N
103	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	20	108	101	2015-07-31 17:35:39.98809	2015-07-31 17:35:39.98809	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
104	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.	20	108	102	2015-07-31 17:35:40.027405	2015-07-31 17:35:40.027405	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
105	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	18	109	13	2015-07-31 17:35:40.070826	2015-07-31 17:35:40.070826	3	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	\N
106	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	24	110	112	2015-07-31 17:35:40.130532	2015-07-31 17:35:40.130532	3	accepted	\N	\N	10:43609950-43609950 (C->G)	2	2	0	0	1	f	\N	\N
107	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	24	111	113	2015-07-31 17:35:40.188698	2015-07-31 17:35:40.188698	5	accepted	\N	\N	10:43617416-43617416 (T->C)	1	0	0	0	4	f	\N	\N
108	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	24	112	113	2015-07-31 17:35:40.242604	2015-07-31 17:35:40.242604	4	accepted	\N	\N	10:43617416-43617416 (T->C)	1	1	0	0	3	f	\N	\N
109	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	24	110	113	2015-07-31 17:35:40.278861	2015-07-31 17:35:40.278861	3	accepted	\N	\N	10:43617416-43617416 (T->C)	2	2	0	0	1	f	\N	\N
110	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.	24	113	113	2015-07-31 17:35:40.32795	2015-07-31 17:35:40.32795	3	accepted	\N	\N	10:43617416-43617416 (T->C)	2	2	0	0	0	f	\N	\N
111	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	19	114	17	2015-07-31 17:35:40.370469	2015-07-31 17:35:40.370469	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
112	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	18	115	17	2015-07-31 17:35:40.415191	2015-07-31 17:35:40.415191	3	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	0
113	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	18	116	11	2015-07-31 17:35:40.45526	2015-07-31 17:35:40.45526	5	accepted	\N	\N	7:140453135-140453136 (CA->AT)	1	2	0	0	0	f	\N	\N
114	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.	19	117	12	2015-07-31 17:35:40.500844	2015-07-31 17:35:40.500844	5	accepted	\N	\N	7:140453136-140453136 (A->T)	0	1	0	0	3	f	\N	\N
115	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer	25	118	12	2015-07-31 17:35:40.548363	2015-07-31 17:35:40.548363	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
116	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	25	119	12	2015-07-31 17:35:40.592131	2015-07-31 17:35:40.592131	5	accepted	\N	\N	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
117	Thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.	25	120	12	2015-07-31 17:35:40.635956	2015-07-31 17:35:40.635956	5	accepted	\N	\N	7:140453136-140453136 (A->T)	1	0	0	0	4	f	\N	\N
118	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	18	121	12	2015-07-31 17:35:40.678003	2015-07-31 17:35:40.678003	2	accepted	\N	\N	7:140453136-140453136 (A->T)	1	2	0	1	6	f	\N	2
119	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	18	122	12	2015-07-31 17:35:40.725472	2015-07-31 17:35:40.725472	5	accepted	\N	\N	7:140453136-140453136 (A->T)	1	2	0	0	0	f	\N	0
120	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	25	123	12	2015-07-31 17:35:40.779477	2015-07-31 17:35:40.779477	5	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	1	3	f	\N	\N
121	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.	18	124	12	2015-07-31 17:35:40.81715	2015-07-31 17:35:40.81715	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
122	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	25	118	12	2015-07-31 17:35:40.857058	2015-07-31 17:35:40.857058	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
123	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	25	125	12	2015-07-31 17:35:40.902301	2015-07-31 17:35:40.902301	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
124	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	25	118	12	2015-07-31 17:35:40.934026	2015-07-31 17:35:40.934026	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	1	0	0	3	f	\N	\N
125	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	18	126	12	2015-07-31 17:35:40.971288	2015-07-31 17:35:40.971288	3	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	1	1	f	\N	\N
126	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	26	127	12	2015-07-31 17:35:41.022889	2015-07-31 17:35:41.022889	3	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	1	f	\N	\N
127	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	18	126	12	2015-07-31 17:35:41.058118	2015-07-31 17:35:41.058118	3	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	1	f	\N	\N
128	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	26	128	12	2015-07-31 17:35:41.09805	2015-07-31 17:35:41.09805	3	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
129	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	26	129	12	2015-07-31 17:35:41.146787	2015-07-31 17:35:41.146787	2	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
130	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	19	130	12	2015-07-31 17:35:41.194725	2015-07-31 17:35:41.194725	2	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	0
131	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	19	130	12	2015-07-31 17:35:41.236661	2015-07-31 17:35:41.236661	2	accepted	\N	\N	7:140453136-140453136 (A->T)	2	2	0	0	0	f	\N	\N
132	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	12	131	12	2015-07-31 17:35:41.284855	2015-07-31 17:35:41.284855	2	accepted	\N	\N	7:140453136-140453136 (A->T)	3	2	0	0	1	f	\N	\N
133	COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	19	132	14	2015-07-31 17:35:41.344444	2015-07-31 17:35:41.344444	4	accepted	\N	\N	N/A	4	2	0	0	1	f	\N	\N
134	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	18	116	15	2015-07-31 17:35:41.373476	2015-07-31 17:35:41.373476	5	accepted	\N	\N	7:140453137-140453137 (C->T)	1	2	0	0	0	f	\N	\N
135	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	18	116	16	2015-07-31 17:35:41.415951	2015-07-31 17:35:41.415951	5	accepted	\N	\N	7:140453136-140453137 (AC->CT)	1	2	0	0	0	f	\N	\N
136	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	19	133	75	2015-07-31 17:35:41.461531	2015-07-31 17:35:41.461531	4	accepted	\N	\N	N/A	1	2	0	1	0	f	\N	\N
137	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	19	133	103	2015-07-31 17:35:41.504784	2015-07-31 17:35:41.504784	4	accepted	\N	\N	3:178936082-178936082 (G->A)	1	2	0	1	6	f	\N	\N
138	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	19	133	104	2015-07-31 17:35:41.53369	2015-07-31 17:35:41.53369	4	accepted	\N	\N	3:178936091-178936091 (G->A)	1	2	0	1	6	f	\N	\N
139	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	19	133	107	2015-07-31 17:35:41.563703	2015-07-31 17:35:41.563703	4	accepted	\N	\N	3:178952085-178952085 (A->G)	1	2	0	1	6	f	\N	\N
140	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	12	93	7	2015-07-31 17:35:41.593831	2015-07-31 17:35:41.593831	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
141	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	22	134	8	2015-07-31 17:35:41.630962	2015-07-31 17:35:41.630962	3	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	1	f	\N	\N
142	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	22	95	8	2015-07-31 17:35:41.665564	2015-07-31 17:35:41.665564	4	accepted	\N	\N	2:29443695-29443695 (G->T)	2	2	0	0	0	f	\N	\N
143	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation	22	134	9	2015-07-31 17:35:41.696544	2015-07-31 17:35:41.696544	3	accepted	\N	\N	2:29432664-29432664 (C->T)	2	2	0	0	0	f	\N	\N
144	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	19	107	12	2015-07-31 17:35:41.730095	2015-07-31 17:35:41.730095	3	accepted	\N	\N	7:140453136-140453136 (A->T)	1	2	0	0	1	f	\N	\N
145	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	13	135	29	2015-07-31 17:35:41.770633	2015-07-31 17:35:41.770633	4	accepted	\N	\N	N/A	1	2	0	1	6	f	\N	\N
146	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	13	135	29	2015-07-31 17:35:41.800983	2015-07-31 17:35:41.800983	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
147	CEBPA mutation was significantly associated with complete remission	13	136	29	2015-07-31 17:35:41.839636	2015-07-31 17:35:41.839636	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
148	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	13	137	28	2015-07-31 17:35:41.895888	2015-07-31 17:35:41.895888	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
149	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	13	137	28	2015-07-31 17:35:41.930726	2015-07-31 17:35:41.930726	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
150	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	13	137	28	2015-07-31 17:35:41.959073	2015-07-31 17:35:41.959073	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
151	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	13	97	32	2015-07-31 17:35:41.991252	2015-07-31 17:35:41.991252	3	accepted	\N	\N	N/A	1	0	0	0	5	f	\N	\N
152	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	13	78	32	2015-07-31 17:35:42.023662	2015-07-31 17:35:42.023662	5	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
153	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)	13	78	32	2015-07-31 17:35:42.05757	2015-07-31 17:35:42.05757	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
154	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	13	97	32	2015-07-31 17:35:42.088877	2015-07-31 17:35:42.088877	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
155	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	13	97	32	2015-07-31 17:35:42.120797	2015-07-31 17:35:42.120797	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
156	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	13	99	32	2015-07-31 17:35:42.150262	2015-07-31 17:35:42.150262	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
157	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	13	97	32	2015-07-31 17:35:42.179536	2015-07-31 17:35:42.179536	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
158	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	13	77	32	2015-07-31 17:35:42.21321	2015-07-31 17:35:42.21321	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
159	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	13	99	32	2015-07-31 17:35:42.240936	2015-07-31 17:35:42.240936	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
160	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	13	77	32	2015-07-31 17:35:42.273679	2015-07-31 17:35:42.273679	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
161	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	13	77	32	2015-07-31 17:35:42.307924	2015-07-31 17:35:42.307924	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
162	Following identification of this fusion in a patient, this fusion was expressed in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic changes in the cell and increased cellular proliferation consistent with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.	27	138	53	2015-07-31 17:35:42.349195	2015-07-31 17:35:42.349195	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
163	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	13	139	55	2015-07-31 17:35:42.434968	2015-07-31 17:35:42.434968	2	accepted	\N	\N	N/A	0	1	0	0	3	f	\N	\N
164	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	13	140	55	2015-07-31 17:35:42.475595	2015-07-31 17:35:42.475595	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
165	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	13	141	55	2015-07-31 17:35:42.513802	2015-07-31 17:35:42.513802	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
166	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	13	141	55	2015-07-31 17:35:42.548034	2015-07-31 17:35:42.548034	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
167	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	13	142	55	2015-07-31 17:35:42.583944	2015-07-31 17:35:42.583944	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
168	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	13	141	55	2015-07-31 17:35:42.622669	2015-07-31 17:35:42.622669	2	accepted	\N	\N	N/A	1	1	0	0	6	f	\N	\N
169	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	13	142	55	2015-07-31 17:35:42.652371	2015-07-31 17:35:42.652371	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
170	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	13	142	55	2015-07-31 17:35:42.681197	2015-07-31 17:35:42.681197	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
171	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	13	142	55	2015-07-31 17:35:42.712597	2015-07-31 17:35:42.712597	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
172	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	13	143	55	2015-07-31 17:35:42.752635	2015-07-31 17:35:42.752635	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
173	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	13	143	55	2015-07-31 17:35:42.78101	2015-07-31 17:35:42.78101	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
174	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	13	144	56	2015-07-31 17:35:42.824095	2015-07-31 17:35:42.824095	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
175	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	13	143	56	2015-07-31 17:35:42.853064	2015-07-31 17:35:42.853064	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
176	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	13	143	56	2015-07-31 17:35:42.888339	2015-07-31 17:35:42.888339	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
178	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	19	107	77	2015-07-31 17:35:42.969534	2015-07-31 17:35:42.969534	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
179	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	19	146	77	2015-07-31 17:35:43.008134	2015-07-31 17:35:43.008134	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
180	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	19	146	77	2015-07-31 17:35:43.040443	2015-07-31 17:35:43.040443	3	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
181	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	26	147	81	2015-07-31 17:35:43.088986	2015-07-31 17:35:43.088986	4	accepted	\N	\N	12:25398281-25398281 (C->T)	2	2	0	0	1	f	\N	\N
182	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model	26	147	81	2015-07-31 17:35:43.11845	2015-07-31 17:35:43.11845	4	accepted	\N	\N	12:25398281-25398281 (C->T)	2	2	0	0	0	f	\N	\N
183	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.	28	148	85	2015-07-31 17:35:43.164971	2015-07-31 17:35:43.164971	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
184	AML with mutated NPM1 is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	13	149	86	2015-07-31 17:35:43.203375	2015-07-31 17:35:43.203375	5	accepted	\N	\N	N/A	0	0	0	0	4	f	\N	\N
185	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).	13	150	86	2015-07-31 17:35:43.24536	2015-07-31 17:35:43.24536	3	accepted	\N	\N	N/A	1	0	0	1	4	f	\N	\N
186	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.	13	151	86	2015-07-31 17:35:43.282571	2015-07-31 17:35:43.282571	4	accepted	\N	\N	N/A	1	0	0	0	5	f	\N	\N
187	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	13	141	86	2015-07-31 17:35:43.314122	2015-07-31 17:35:43.314122	2	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
188	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.	13	151	86	2015-07-31 17:35:43.343951	2015-07-31 17:35:43.343951	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
189	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia	13	150	86	2015-07-31 17:35:43.378582	2015-07-31 17:35:43.378582	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
190	NPM1 mutations were associated with normal karyotype in older (>60) patients	13	152	86	2015-07-31 17:35:43.417049	2015-07-31 17:35:43.417049	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
191	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	13	140	86	2015-07-31 17:35:43.462749	2015-07-31 17:35:43.462749	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
192	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	13	152	86	2015-07-31 17:35:43.492129	2015-07-31 17:35:43.492129	2	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
193	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	13	153	86	2015-07-31 17:35:43.528688	2015-07-31 17:35:43.528688	2	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
194	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	13	154	86	2015-07-31 17:35:43.570495	2015-07-31 17:35:43.570495	2	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
195	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	13	145	86	2015-07-31 17:35:43.603821	2015-07-31 17:35:43.603821	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
196	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)	13	141	86	2015-07-31 17:35:43.633775	2015-07-31 17:35:43.633775	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
197	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	13	152	86	2015-07-31 17:35:43.663901	2015-07-31 17:35:43.663901	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
198	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.	13	150	86	2015-07-31 17:35:43.697713	2015-07-31 17:35:43.697713	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
199	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.	13	151	86	2015-07-31 17:35:43.72683	2015-07-31 17:35:43.72683	5	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
200	Cytoplasmic localization of NPM in AML patients was an independent prognositic factor associated with complete remission in response to induction therapy in a multivariate analysis (p=0.019) but not in a univariate analysis (p=0.07).	13	155	86	2015-07-31 17:35:43.765027	2015-07-31 17:35:43.765027	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
201	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	13	144	86	2015-07-31 17:35:43.798951	2015-07-31 17:35:43.798951	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
202	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	13	153	86	2015-07-31 17:35:43.831768	2015-07-31 17:35:43.831768	2	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
203	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	13	156	86	2015-07-31 17:35:43.878186	2015-07-31 17:35:43.878186	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
204	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	13	157	86	2015-07-31 17:35:43.917198	2015-07-31 17:35:43.917198	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
205	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	13	158	86	2015-07-31 17:35:43.973189	2015-07-31 17:35:43.973189	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
206	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	13	136	86	2015-07-31 17:35:44.005606	2015-07-31 17:35:44.005606	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
207	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	13	159	86	2015-07-31 17:35:44.043411	2015-07-31 17:35:44.043411	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
208	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	13	160	86	2015-07-31 17:35:44.081906	2015-07-31 17:35:44.081906	4	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
209	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	13	150	86	2015-07-31 17:35:44.113216	2015-07-31 17:35:44.113216	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
210	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	13	161	86	2015-07-31 17:35:44.155848	2015-07-31 17:35:44.155848	2	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
211	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	13	142	86	2015-07-31 17:35:44.184716	2015-07-31 17:35:44.184716	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
212	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	13	157	86	2015-07-31 17:35:44.220835	2015-07-31 17:35:44.220835	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
213	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	13	162	86	2015-07-31 17:35:44.263038	2015-07-31 17:35:44.263038	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
214	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	13	142	86	2015-07-31 17:35:44.297409	2015-07-31 17:35:44.297409	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
215	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	13	163	86	2015-07-31 17:35:44.335225	2015-07-31 17:35:44.335225	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	0
216	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	13	164	86	2015-07-31 17:35:44.384986	2015-07-31 17:35:44.384986	2	accepted	\N	\N	N/A	4	2	0	0	0	f	\N	\N
217	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	13	164	86	2015-07-31 17:35:44.417407	2015-07-31 17:35:44.417407	2	accepted	\N	\N	N/A	4	2	0	0	0	f	\N	\N
218	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	13	78	87	2015-07-31 17:35:44.452125	2015-07-31 17:35:44.452125	4	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	1	0	0	1	4	f	\N	\N
219	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	13	78	87	2015-07-31 17:35:44.516729	2015-07-31 17:35:44.516729	4	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	1	0	0	0	4	f	\N	\N
220	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.	13	163	87	2015-07-31 17:35:44.550625	2015-07-31 17:35:44.550625	3	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	2	2	0	0	0	f	\N	\N
221	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	13	155	87	2015-07-31 17:35:44.585641	2015-07-31 17:35:44.585641	3	accepted	\N	\N	5:170837546-170837547 (0->TCAG)	4	2	0	0	0	f	\N	\N
222	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	19	107	94	2015-07-31 17:35:44.624855	2015-07-31 17:35:44.624855	3	accepted	\N	\N	N/A	1	2	0	1	1	f	\N	\N
223	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	19	107	105	2015-07-31 17:35:44.655325	2015-07-31 17:35:44.655325	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
224	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	19	107	106	2015-07-31 17:35:44.686995	2015-07-31 17:35:44.686995	3	accepted	\N	\N	N/A	1	2	0	1	6	f	\N	\N
225	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	12	165	124	2015-07-31 17:35:44.728912	2015-07-31 17:35:44.728912	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
226	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	27	166	125	2015-07-31 17:35:44.786039	2015-07-31 17:35:44.786039	3	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
227	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	13	167	129	2015-07-31 17:35:44.838814	2015-07-31 17:35:44.838814	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
228	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	13	168	129	2015-07-31 17:35:44.882631	2015-07-31 17:35:44.882631	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
229	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	13	168	129	2015-07-31 17:35:44.91732	2015-07-31 17:35:44.91732	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
230	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	13	169	129	2015-07-31 17:35:44.95334	2015-07-31 17:35:44.95334	2	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
231	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	13	169	129	2015-07-31 17:35:44.983106	2015-07-31 17:35:44.983106	2	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
232	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	13	168	129	2015-07-31 17:35:45.032303	2015-07-31 17:35:45.032303	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
262	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	31	179	20	2015-07-31 17:35:46.239792	2015-07-31 17:35:46.239792	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
233	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	13	170	129	2015-07-31 17:35:45.068348	2015-07-31 17:35:45.068348	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
234	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	13	167	129	2015-07-31 17:35:45.105695	2015-07-31 17:35:45.105695	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
235	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	13	167	129	2015-07-31 17:35:45.148648	2015-07-31 17:35:45.148648	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
236	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	13	167	130	2015-07-31 17:35:45.186242	2015-07-31 17:35:45.186242	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
237	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	13	168	130	2015-07-31 17:35:45.219364	2015-07-31 17:35:45.219364	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
238	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	13	168	130	2015-07-31 17:35:45.250838	2015-07-31 17:35:45.250838	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
239	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	13	168	130	2015-07-31 17:35:45.284964	2015-07-31 17:35:45.284964	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
240	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	13	167	130	2015-07-31 17:35:45.315488	2015-07-31 17:35:45.315488	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
241	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	13	167	130	2015-07-31 17:35:45.34781	2015-07-31 17:35:45.34781	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
242	WT1 mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	13	170	130	2015-07-31 17:35:45.379795	2015-07-31 17:35:45.379795	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
243	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive women with high grade serous or undifferentiated ovarian cancer.	29	171	131	2015-07-31 17:35:45.421583	2015-07-31 17:35:45.421583	4	accepted	\N	\N	N/A	1	2	1	0	0	f	\N	\N
244	The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive women with high grade serous or undifferentiated ovarian cancer.	29	171	132	2015-07-31 17:35:45.473399	2015-07-31 17:35:45.473399	4	accepted	\N	\N	N/A	1	2	1	0	0	f	\N	\N
245	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	15	172	1	2015-07-31 17:35:45.52637	2015-07-31 17:35:45.52637	4	accepted	\N	\N	N/A	0	0	0	0	4	f	\N	\N
246	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	13	173	1	2015-07-31 17:35:45.585882	2015-07-31 17:35:45.585882	5	accepted	\N	\N	N/A	0	2	0	0	0	f	\N	\N
247	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.	15	172	1	2015-07-31 17:35:45.621242	2015-07-31 17:35:45.621242	5	accepted	\N	\N	N/A	0	2	0	0	0	f	\N	\N
248	BCR-ABL inhibitors such as imatinib have lead to significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens.	15	172	1	2015-07-31 17:35:45.656713	2015-07-31 17:35:45.656713	5	accepted	\N	\N	N/A	0	1	0	0	2	f	\N	\N
249	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.	15	172	1	2015-07-31 17:35:45.687478	2015-07-31 17:35:45.687478	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	2
250	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	15	174	1	2015-07-31 17:35:45.731117	2015-07-31 17:35:45.731117	2	accepted	\N	\N	N/A	2	0	0	1	4	f	\N	\N
251	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	13	173	1	2015-07-31 17:35:45.761374	2015-07-31 17:35:45.761374	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
252	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	15	175	3	2015-07-31 17:35:45.802748	2015-07-31 17:35:45.802748	4	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
253	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	15	172	2	2015-07-31 17:35:45.848076	2015-07-31 17:35:45.848076	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
254	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.	15	175	2	2015-07-31 17:35:45.888636	2015-07-31 17:35:45.888636	4	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
255	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	30	176	4	2015-07-31 17:35:45.937576	2015-07-31 17:35:45.937576	3	accepted	\N	\N	14:105246551-105246551 (C->T)	2	2	0	1	0	f	\N	\N
256	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.	12	177	5	2015-07-31 17:35:45.977428	2015-07-31 17:35:45.977428	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
257	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.	12	177	6	2015-07-31 17:35:46.031819	2015-07-31 17:35:46.031819	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
258	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.	12	177	7	2015-07-31 17:35:46.074189	2015-07-31 17:35:46.074189	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
259	Inceased copy number of CCND1 is associated with poorer overall survival.	12	178	18	2015-07-31 17:35:46.139612	2015-07-31 17:35:46.139612	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
260	Three studies have found no significant survival impact for CCND1 in lung cancer.	12	178	19	2015-07-31 17:35:46.172128	2015-07-31 17:35:46.172128	3	accepted	\N	\N	N/A	1	1	0	1	3	f	\N	\N
261	Three studies have associated CCND1 expression with poorer survival.	12	178	19	2015-07-31 17:35:46.200291	2015-07-31 17:35:46.200291	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
263	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	31	179	20	2015-07-31 17:35:46.274553	2015-07-31 17:35:46.274553	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
264	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	32	180	20	2015-07-31 17:35:46.324398	2015-07-31 17:35:46.324398	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
265	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	33	181	20	2015-07-31 17:35:46.365523	2015-07-31 17:35:46.365523	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
266	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	34	182	21	2015-07-31 17:35:46.407834	2015-07-31 17:35:46.407834	4	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
267	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	34	182	21	2015-07-31 17:35:46.439874	2015-07-31 17:35:46.439874	4	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
268	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	34	182	21	2015-07-31 17:35:46.47536	2015-07-31 17:35:46.47536	4	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
269	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	34	182	21	2015-07-31 17:35:46.509915	2015-07-31 17:35:46.509915	4	accepted	\N	\N	N/A	2	1	0	0	3	f	\N	\N
270	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	34	183	22	2015-07-31 17:35:46.552005	2015-07-31 17:35:46.552005	3	accepted	\N	\N	N/A	2	0	0	0	5	f	\N	\N
271	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	35	184	23	2015-07-31 17:35:46.598584	2015-07-31 17:35:46.598584	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
272	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	31	185	24	2015-07-31 17:35:46.657321	2015-07-31 17:35:46.657321	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
273	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	36	186	24	2015-07-31 17:35:46.697315	2015-07-31 17:35:46.697315	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
274	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	36	186	24	2015-07-31 17:35:46.732426	2015-07-31 17:35:46.732426	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
275	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	37	186	24	2015-07-31 17:35:46.764608	2015-07-31 17:35:46.764608	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
276	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	38	187	25	2015-07-31 17:35:46.803364	2015-07-31 17:35:46.803364	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
277	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	38	188	25	2015-07-31 17:35:46.841013	2015-07-31 17:35:46.841013	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	0
278	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	38	187	26	2015-07-31 17:35:46.873993	2015-07-31 17:35:46.873993	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
279	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	38	188	26	2015-07-31 17:35:46.904137	2015-07-31 17:35:46.904137	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	0
280	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	12	189	27	2015-07-31 17:35:46.940987	2015-07-31 17:35:46.940987	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
281	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	12	189	27	2015-07-31 17:35:46.969816	2015-07-31 17:35:46.969816	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
282	In 3T3 fibroblasts expressing the G253C mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	12	190	141	2015-07-31 17:35:47.010813	2015-07-31 17:35:47.010813	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
283	In 3T3 fibroblasts expressing the G505S mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	12	190	142	2015-07-31 17:35:47.046861	2015-07-31 17:35:47.046861	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
284	In 3T3 fibroblasts expressing the G774V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	12	190	144	2015-07-31 17:35:47.08477	2015-07-31 17:35:47.08477	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
285	In 3T3 fibroblasts expressing the I638F mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	12	190	143	2015-07-31 17:35:47.114781	2015-07-31 17:35:47.114781	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
286	In 3T3 fibroblasts expressing the L239R mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	12	190	140	2015-07-31 17:35:47.143982	2015-07-31 17:35:47.143982	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
287	In 3T3 fibroblasts expressing the L63V mutation have to be sensitive to the tyrosine kinase inhibitor dasatinib.	12	190	139	2015-07-31 17:35:47.186943	2015-07-31 17:35:47.186943	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
288	In a patient expressing a DDR2 S768R mutation, treatment with dasatinib plus erlotinib acheived a partial response.	12	190	145	2015-07-31 17:35:47.216895	2015-07-31 17:35:47.216895	3	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	0
289	Exon 19 deletion has been shown to be correlated with gefitinib response.	12	191	133	2015-07-31 17:35:47.261019	2015-07-31 17:35:47.261019	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
290	Gefinitib has been shown to be effective in treating cell lines with G719S missense mutations.	12	192	134	2015-07-31 17:35:47.304579	2015-07-31 17:35:47.304579	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
291	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	12	193	33	2015-07-31 17:35:47.342759	2015-07-31 17:35:47.342759	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	1	2	0	1	6	f	\N	2
292	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.	12	193	33	2015-07-31 17:35:47.381771	2015-07-31 17:35:47.381771	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	1	2	0	1	6	f	\N	2
293	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	12	194	33	2015-07-31 17:35:47.423997	2015-07-31 17:35:47.423997	4	accepted	\N	\N	7:55259514-55259515 (CT->AG)	1	2	0	0	0	f	\N	\N
294	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	12	195	33	2015-07-31 17:35:47.469177	2015-07-31 17:35:47.469177	3	accepted	\N	\N	7:55259514-55259515 (CT->AG)	1	1	0	0	2	f	\N	\N
295	Gefinitib has been shown to be effective in treating cell lines with L858R missense mutations.	12	192	33	2015-07-31 17:35:47.503753	2015-07-31 17:35:47.503753	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
296	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.	39	196	34	2015-07-31 17:35:47.544429	2015-07-31 17:35:47.544429	5	accepted	\N	\N	7:55249071-55249071 (C->T)	0	2	0	0	1	f	\N	\N
297	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	12	197	34	2015-07-31 17:35:47.601172	2015-07-31 17:35:47.601172	3	accepted	\N	\N	7:55249071-55249071 (C->T)	1	2	0	0	1	f	\N	\N
298	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.	12	198	34	2015-07-31 17:35:47.638887	2015-07-31 17:35:47.638887	4	accepted	\N	\N	7:55249071-55249071 (C->T)	1	1	0	0	3	f	\N	\N
299	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.	12	199	34	2015-07-31 17:35:47.694649	2015-07-31 17:35:47.694649	4	accepted	\N	\N	7:55249071-55249071 (C->T)	1	1	0	0	3	f	\N	\N
300	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.	12	200	34	2015-07-31 17:35:47.732819	2015-07-31 17:35:47.732819	3	accepted	\N	\N	7:55249071-55249071 (C->T)	3	2	0	0	0	f	\N	0
301	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.	12	201	34	2015-07-31 17:35:47.778962	2015-07-31 17:35:47.778962	1	accepted	\N	\N	7:55249071-55249071 (C->T)	4	2	0	0	0	f	\N	\N
302	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	12	202	34	2015-07-31 17:35:47.820197	2015-07-31 17:35:47.820197	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
303	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.	31	203	35	2015-07-31 17:35:47.866595	2015-07-31 17:35:47.866595	4	accepted	\N	\N	17:37880261-37880261 (G->C)	2	2	0	0	0	f	\N	\N
304	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.	31	203	36	2015-07-31 17:35:47.918684	2015-07-31 17:35:47.918684	4	accepted	\N	\N	17:37880261-37880261 (G->T)	2	2	0	0	0	f	\N	\N
305	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.	31	203	37	2015-07-31 17:35:47.953738	2015-07-31 17:35:47.953738	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
306	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.	31	203	38	2015-07-31 17:35:47.986023	2015-07-31 17:35:47.986023	5	accepted	\N	\N	17:37868205-37868205 (G->C)	2	2	0	0	0	f	\N	\N
307	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	31	203	39	2015-07-31 17:35:48.025107	2015-07-31 17:35:48.025107	5	accepted	\N	\N	17:37880220-37880220 (T->C)	2	2	0	0	1	f	\N	\N
308	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.	31	203	39	2015-07-31 17:35:48.0626	2015-07-31 17:35:48.0626	5	accepted	\N	\N	17:37880220-37880220 (T->C)	2	2	0	0	0	f	\N	\N
309	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	31	203	40	2015-07-31 17:35:48.100604	2015-07-31 17:35:48.100604	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
310	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.	31	203	41	2015-07-31 17:35:48.132927	2015-07-31 17:35:48.132927	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
311	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.	31	203	42	2015-07-31 17:35:48.169666	2015-07-31 17:35:48.169666	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
312	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.	31	203	43	2015-07-31 17:35:48.222754	2015-07-31 17:35:48.222754	5	accepted	\N	\N	17:37881616-37881616 (C->T)	2	2	0	0	0	f	\N	\N
313	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.	31	203	44	2015-07-31 17:35:48.258793	2015-07-31 17:35:48.258793	5	accepted	\N	\N	17:37881000-37881000 (G->T)	2	2	0	0	0	f	\N	\N
314	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.	31	203	45	2015-07-31 17:35:48.300133	2015-07-31 17:35:48.300133	5	accepted	\N	\N	17:37881332-37881332 (G->A)	2	2	0	0	0	f	\N	\N
315	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	31	204	46	2015-07-31 17:35:48.350604	2015-07-31 17:35:48.350604	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
316	The L536Q ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.	31	205	46	2015-07-31 17:35:48.412721	2015-07-31 17:35:48.412721	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
317	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	31	204	47	2015-07-31 17:35:48.453744	2015-07-31 17:35:48.453744	3	accepted	\N	\N	N/A	2	2	0	0	1	f	\N	\N
318	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	31	205	47	2015-07-31 17:35:48.496033	2015-07-31 17:35:48.496033	5	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
319	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	31	204	48	2015-07-31 17:35:48.53888	2015-07-31 17:35:48.53888	3	accepted	\N	\N	6:152419923-152419923 (A->G)	2	2	0	0	1	f	\N	\N
320	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	31	205	48	2015-07-31 17:35:48.574541	2015-07-31 17:35:48.574541	5	accepted	\N	\N	6:152419923-152419923 (A->G)	2	2	0	0	0	f	\N	2
321	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	31	204	49	2015-07-31 17:35:48.614234	2015-07-31 17:35:48.614234	3	accepted	\N	\N	6:152419922-152419922 (T->A)	2	2	0	0	1	f	\N	\N
322	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	31	205	49	2015-07-31 17:35:48.655706	2015-07-31 17:35:48.655706	5	accepted	\N	\N	6:152419922-152419922 (T->A)	2	2	0	0	0	f	\N	2
323	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.	31	204	50	2015-07-31 17:35:48.700179	2015-07-31 17:35:48.700179	3	accepted	\N	\N	6:152419923-152419923 (A->C)	2	2	0	0	1	f	\N	\N
324	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	31	205	50	2015-07-31 17:35:48.750967	2015-07-31 17:35:48.750967	5	accepted	\N	\N	6:152419923-152419923 (A->C)	2	2	0	0	0	f	\N	2
325	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.	40	206	52	2015-07-31 17:35:48.804005	2015-07-31 17:35:48.804005	2	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	\N
326	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.	40	206	54	2015-07-31 17:35:48.851516	2015-07-31 17:35:48.851516	3	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	2
327	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	13	207	55	2015-07-31 17:35:48.901032	2015-07-31 17:35:48.901032	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
328	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	13	208	55	2015-07-31 17:35:48.94283	2015-07-31 17:35:48.94283	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
329	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.	13	209	55	2015-07-31 17:35:48.987331	2015-07-31 17:35:48.987331	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
330	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	13	207	55	2015-07-31 17:35:49.021899	2015-07-31 17:35:49.021899	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
331	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	13	210	55	2015-07-31 17:35:49.064525	2015-07-31 17:35:49.064525	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
332	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	13	211	55	2015-07-31 17:35:49.103896	2015-07-31 17:35:49.103896	4	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	\N
333	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	13	207	56	2015-07-31 17:35:49.133856	2015-07-31 17:35:49.133856	3	accepted	\N	\N	N/A	1	0	0	1	4	f	\N	\N
334	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	13	212	56	2015-07-31 17:35:49.172672	2015-07-31 17:35:49.172672	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
335	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.	13	209	56	2015-07-31 17:35:49.202851	2015-07-31 17:35:49.202851	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
336	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations	13	208	56	2015-07-31 17:35:49.238214	2015-07-31 17:35:49.238214	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
337	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	13	207	56	2015-07-31 17:35:49.283427	2015-07-31 17:35:49.283427	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
338	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3	13	213	56	2015-07-31 17:35:49.319482	2015-07-31 17:35:49.319482	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
339	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	41	214	57	2015-07-31 17:35:49.365112	2015-07-31 17:35:49.365112	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	0
340	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	13	215	58	2015-07-31 17:35:49.410835	2015-07-31 17:35:49.410835	2	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
341	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	13	216	58	2015-07-31 17:35:49.450978	2015-07-31 17:35:49.450978	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
342	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	42	217	58	2015-07-31 17:35:49.491656	2015-07-31 17:35:49.491656	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
343	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	43	218	58	2015-07-31 17:35:49.528649	2015-07-31 17:35:49.528649	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
344	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	44	219	58	2015-07-31 17:35:49.578282	2015-07-31 17:35:49.578282	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
345	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	13	220	59	2015-07-31 17:35:49.618591	2015-07-31 17:35:49.618591	3	accepted	\N	\N	2:209113113-209113113 (G->A)	1	0	0	0	4	f	\N	\N
346	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	13	220	59	2015-07-31 17:35:49.655379	2015-07-31 17:35:49.655379	2	accepted	\N	\N	2:209113113-209113113 (G->A)	1	1	0	1	6	f	\N	\N
347	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	13	220	60	2015-07-31 17:35:49.690116	2015-07-31 17:35:49.690116	3	accepted	\N	\N	2:209113112-209113112 (C->T)	1	1	0	1	6	f	\N	\N
348	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	13	220	61	2015-07-31 17:35:49.719222	2015-07-31 17:35:49.719222	3	accepted	\N	\N	2:209113112-209113112 (C->A)	1	1	0	1	6	f	\N	\N
349	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	13	221	62	2015-07-31 17:35:49.761902	2015-07-31 17:35:49.761902	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
350	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.	13	222	62	2015-07-31 17:35:49.816618	2015-07-31 17:35:49.816618	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
351	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	45	223	62	2015-07-31 17:35:49.857402	2015-07-31 17:35:49.857402	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
352	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	45	224	62	2015-07-31 17:35:49.896587	2015-07-31 17:35:49.896587	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
353	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	13	225	62	2015-07-31 17:35:49.932211	2015-07-31 17:35:49.932211	3	accepted	\N	\N	N/A	1	1	0	0	2	f	\N	\N
354	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	13	221	63	2015-07-31 17:35:49.968397	2015-07-31 17:35:49.968397	2	accepted	\N	\N	15:90631838-90631838 (C->T)	1	1	0	1	6	f	\N	\N
355	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)	45	223	63	2015-07-31 17:35:49.998735	2015-07-31 17:35:49.998735	2	accepted	\N	\N	15:90631838-90631838 (C->T)	1	1	0	1	6	f	\N	\N
356	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	13	225	63	2015-07-31 17:35:50.030073	2015-07-31 17:35:50.030073	3	accepted	\N	\N	15:90631838-90631838 (C->T)	1	1	0	0	3	f	\N	\N
357	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	23	226	66	2015-07-31 17:35:50.067619	2015-07-31 17:35:50.067619	2	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
358	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	23	227	66	2015-07-31 17:35:50.113052	2015-07-31 17:35:50.113052	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
359	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	37	228	66	2015-07-31 17:35:50.162887	2015-07-31 17:35:50.162887	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
360	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	23	226	66	2015-07-31 17:35:50.203978	2015-07-31 17:35:50.203978	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
361	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	23	226	70	2015-07-31 17:35:50.242304	2015-07-31 17:35:50.242304	3	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
362	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	37	228	71	2015-07-31 17:35:50.278954	2015-07-31 17:35:50.278954	2	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
363	In a case study, a patient with anal melanoma harboring a 7 codon duplication in the juxtamambrane region (exon 11) of KIT showed marked response 4 months after imatinib treatment.	46	229	67	2015-07-31 17:35:50.338816	2015-07-31 17:35:50.338816	3	accepted	\N	\N	N/A	3	2	0	0	0	f	\N	\N
364	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	12	230	72	2015-07-31 17:35:50.388145	2015-07-31 17:35:50.388145	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
365	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	23	231	73	2015-07-31 17:35:50.438222	2015-07-31 17:35:50.438222	3	accepted	\N	\N	4:55594258-55594258 (T->C)	1	2	0	0	0	f	\N	\N
366	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	23	91	73	2015-07-31 17:35:50.475013	2015-07-31 17:35:50.475013	3	accepted	\N	\N	4:55594258-55594258 (T->C)	2	2	0	0	1	f	\N	\N
367	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	19	232	74	2015-07-31 17:35:50.5179	2015-07-31 17:35:50.5179	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
368	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	12	233	76	2015-07-31 17:35:50.564344	2015-07-31 17:35:50.564344	3	accepted	\N	\N	N/A	1	0	0	1	4	f	\N	\N
369	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	26	234	76	2015-07-31 17:35:50.615282	2015-07-31 17:35:50.615282	4	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
370	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	12	235	76	2015-07-31 17:35:50.652477	2015-07-31 17:35:50.652477	3	accepted	\N	\N	N/A	1	1	0	1	6	f	\N	\N
371	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	47	236	76	2015-07-31 17:35:50.693975	2015-07-31 17:35:50.693975	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
372	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	12	237	76	2015-07-31 17:35:50.740613	2015-07-31 17:35:50.740613	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
373	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	48	238	148	2015-07-31 17:35:50.783557	2015-07-31 17:35:50.783557	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
374	KRAS G12C occur more frequently in women than men	37	239	78	2015-07-31 17:35:50.845935	2015-07-31 17:35:50.845935	2	accepted	\N	\N	12:25398285-25398285 (C->A)	1	0	0	0	4	f	\N	\N
375	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	48	238	78	2015-07-31 17:35:50.880887	2015-07-31 17:35:50.880887	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
376	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	37	239	79	2015-07-31 17:35:50.920872	2015-07-31 17:35:50.920872	3	accepted	\N	\N	12:25398284-25398284 (C->T)	1	0	0	0	4	f	\N	\N
377	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.	12	240	79	2015-07-31 17:35:50.969821	2015-07-31 17:35:50.969821	4	accepted	\N	\N	12:25398284-25398284 (C->T)	2	2	0	0	0	f	\N	0
378	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	48	238	79	2015-07-31 17:35:51.011898	2015-07-31 17:35:51.011898	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
379	KRAS G12 mutations were the primary predictor of EGFR TKI resistance in patients with lung adenocarcinoma.	48	238	147	2015-07-31 17:35:51.054374	2015-07-31 17:35:51.054374	3	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	2
380	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender	12	233	80	2015-07-31 17:35:51.102057	2015-07-31 17:35:51.102057	3	accepted	\N	\N	N/A	1	0	0	1	4	f	\N	\N
381	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	49	236	80	2015-07-31 17:35:51.13566	2015-07-31 17:35:51.13566	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
382	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	26	147	81	2015-07-31 17:35:51.171711	2015-07-31 17:35:51.171711	4	accepted	\N	\N	12:25398281-25398281 (C->T)	1	2	0	0	0	f	\N	\N
383	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	26	147	81	2015-07-31 17:35:51.204732	2015-07-31 17:35:51.204732	4	accepted	\N	\N	12:25398281-25398281 (C->T)	1	1	0	0	2	f	\N	\N
384	In Lung Cancer patients with acquired resistance to Dacomitinib (PF00299804), MET amplification and EGFR T790M were found to be the primary drivers of resistance.	12	202	146	2015-07-31 17:35:51.237785	2015-07-31 17:35:51.237785	4	accepted	\N	\N	N/A	1	2	0	0	1	f	\N	\N
385	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.	28	241	85	2015-07-31 17:35:51.284687	2015-07-31 17:35:51.284687	5	accepted	\N	\N	N/A	0	2	0	0	0	f	\N	\N
386	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	28	148	85	2015-07-31 17:35:51.31631	2015-07-31 17:35:51.31631	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
387	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	28	148	85	2015-07-31 17:35:51.352987	2015-07-31 17:35:51.352987	4	accepted	\N	\N	N/A	4	2	0	0	0	f	\N	0
388	NOTCH1 gain-of-function mutations, including D1643H, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	12	242	135	2015-07-31 17:35:51.415612	2015-07-31 17:35:51.415612	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
389	NOTCH1 gain-of-function mutations, including R2328W, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	12	242	136	2015-07-31 17:35:51.445665	2015-07-31 17:35:51.445665	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
390	NOTCH1 gain-of-function mutations, including S2275fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	12	242	138	2015-07-31 17:35:51.479323	2015-07-31 17:35:51.479323	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
391	NOTCH1 gain-of-function mutations, including V2444fs, have shown to be correlated to poor prognosis in lung cancer. This is in a TP53 wild-type context.	12	242	137	2015-07-31 17:35:51.515005	2015-07-31 17:35:51.515005	4	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
392	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	31	243	103	2015-07-31 17:35:51.5578	2015-07-31 17:35:51.5578	4	accepted	\N	\N	3:178936082-178936082 (G->A)	2	2	0	0	0	f	\N	\N
393	Rapamycin inhibits transformation induced by mutation in PIK3CA	31	244	103	2015-07-31 17:35:51.598787	2015-07-31 17:35:51.598787	3	accepted	\N	\N	3:178936082-178936082 (G->A)	2	2	0	0	0	f	\N	\N
394	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	19	245	103	2015-07-31 17:35:51.637583	2015-07-31 17:35:51.637583	3	accepted	\N	\N	3:178936082-178936082 (G->A)	4	2	0	0	1	f	\N	2
395	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	19	246	103	2015-07-31 17:35:51.679148	2015-07-31 17:35:51.679148	2	accepted	\N	\N	3:178936082-178936082 (G->A)	4	1	0	0	3	f	\N	\N
396	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.	19	245	103	2015-07-31 17:35:51.713095	2015-07-31 17:35:51.713095	3	accepted	\N	\N	3:178936082-178936082 (G->A)	4	1	0	0	3	f	\N	\N
397	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	31	243	104	2015-07-31 17:35:51.745345	2015-07-31 17:35:51.745345	4	accepted	\N	\N	3:178936091-178936091 (G->A)	2	2	0	0	0	f	\N	\N
398	Rapamycin inhibits transformation induced by mutation in PIK3CA	31	244	104	2015-07-31 17:35:51.777035	2015-07-31 17:35:51.777035	3	accepted	\N	\N	3:178936091-178936091 (G->A)	2	2	0	0	0	f	\N	\N
399	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.	19	245	104	2015-07-31 17:35:51.810614	2015-07-31 17:35:51.810614	3	accepted	\N	\N	3:178936091-178936091 (G->A)	4	2	0	0	1	f	\N	2
400	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	19	246	104	2015-07-31 17:35:51.851098	2015-07-31 17:35:51.851098	2	accepted	\N	\N	3:178936091-178936091 (G->A)	4	1	0	0	3	f	\N	\N
401	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	19	245	104	2015-07-31 17:35:51.881021	2015-07-31 17:35:51.881021	3	accepted	\N	\N	3:178936091-178936091 (G->A)	4	1	0	0	3	f	\N	\N
402	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	31	243	107	2015-07-31 17:35:51.927099	2015-07-31 17:35:51.927099	4	accepted	\N	\N	3:178952085-178952085 (A->G)	2	2	0	0	0	f	\N	\N
403	Rapamycin inhibits transformation induced by mutation in PIK3CA	31	244	107	2015-07-31 17:35:51.956792	2015-07-31 17:35:51.956792	3	accepted	\N	\N	3:178952085-178952085 (A->G)	2	2	0	0	0	f	\N	\N
404	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.	19	245	107	2015-07-31 17:35:51.986019	2015-07-31 17:35:51.986019	3	accepted	\N	\N	3:178952085-178952085 (A->G)	4	2	0	0	1	f	\N	2
405	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	19	245	107	2015-07-31 17:35:52.020681	2015-07-31 17:35:52.020681	3	accepted	\N	\N	3:178952085-178952085 (A->G)	4	1	0	0	3	f	\N	\N
406	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	50	247	108	2015-07-31 17:35:52.061007	2015-07-31 17:35:52.061007	5	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
407	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	28	248	110	2015-07-31 17:35:52.104506	2015-07-31 17:35:52.104506	3	accepted	\N	\N	10:89717672-89717672 (C->T)	1	1	0	1	3	f	\N	\N
408	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	30	249	110	2015-07-31 17:35:52.168995	2015-07-31 17:35:52.168995	4	accepted	\N	\N	10:89717672-89717672 (C->T)	2	2	0	0	0	f	\N	\N
409	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.	50	247	111	2015-07-31 17:35:52.210012	2015-07-31 17:35:52.210012	5	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
410	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	45	250	114	2015-07-31 17:35:52.249993	2015-07-31 17:35:52.249993	3	accepted	\N	\N	2:198267359-198267359 (C->A)	4	1	0	0	2	f	\N	\N
411	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	45	250	115	2015-07-31 17:35:52.281447	2015-07-31 17:35:52.281447	3	accepted	\N	\N	2:198266834-198266834 (T->C)	4	1	0	0	2	f	\N	\N
412	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	31	251	116	2015-07-31 17:35:52.32818	2015-07-31 17:35:52.32818	3	accepted	\N	\N	17:7578406-7578406 (C->T)	1	1	0	0	3	f	\N	\N
413	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.	31	252	116	2015-07-31 17:35:52.371352	2015-07-31 17:35:52.371352	3	accepted	\N	\N	17:7578406-7578406 (C->T)	2	2	0	0	0	f	\N	\N
414	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	31	251	117	2015-07-31 17:35:52.421612	2015-07-31 17:35:52.421612	3	accepted	\N	\N	17:7577538-7577538 (C->T)	1	1	0	0	3	f	\N	\N
415	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	31	253	117	2015-07-31 17:35:52.461367	2015-07-31 17:35:52.461367	3	accepted	\N	\N	17:7577538-7577538 (C->T)	1	1	0	0	3	f	\N	\N
416	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	31	251	118	2015-07-31 17:35:52.491212	2015-07-31 17:35:52.491212	3	accepted	\N	\N	17:7577539-7577539 (G->A)	1	1	0	0	3	f	\N	\N
417	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	31	253	119	2015-07-31 17:35:52.520914	2015-07-31 17:35:52.520914	3	accepted	\N	\N	17:7577535-7577535 (C->G)	1	1	0	0	3	f	\N	\N
418	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.	31	253	120	2015-07-31 17:35:52.549527	2015-07-31 17:35:52.549527	3	accepted	\N	\N	17:7577536-7577536 (T->A)	1	1	0	0	3	f	\N	\N
419	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	31	251	121	2015-07-31 17:35:52.584771	2015-07-31 17:35:52.584771	3	accepted	\N	\N	17:7577121-7577121 (G->A)	1	1	0	0	3	f	\N	\N
420	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	31	253	121	2015-07-31 17:35:52.614585	2015-07-31 17:35:52.614585	3	accepted	\N	\N	17:7577121-7577121 (G->A)	1	1	0	0	3	f	\N	\N
421	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	31	251	122	2015-07-31 17:35:52.650124	2015-07-31 17:35:52.650124	3	accepted	\N	\N	17:7577120-7577120 (C->T)	1	1	0	0	3	f	\N	\N
422	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	31	253	122	2015-07-31 17:35:52.680907	2015-07-31 17:35:52.680907	3	accepted	\N	\N	17:7577120-7577120 (C->T)	1	1	0	0	3	f	\N	\N
423	Breast tumors with R175H mutations are more responsive to doxorubicin than breast tumors with wild type TP53.	31	253	123	2015-07-31 17:35:52.711803	2015-07-31 17:35:52.711803	3	accepted	\N	\N	N/A	1	1	0	0	3	f	\N	\N
424	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	51	254	124	2015-07-31 17:35:52.753305	2015-07-31 17:35:52.753305	3	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
425	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	27	254	126	2015-07-31 17:35:52.791512	2015-07-31 17:35:52.791512	3	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	\N
426	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	13	255	127	2015-07-31 17:35:52.840332	2015-07-31 17:35:52.840332	3	accepted	\N	\N	N/A	4	0	0	1	4	f	\N	\N
427	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	13	255	127	2015-07-31 17:35:52.875071	2015-07-31 17:35:52.875071	2	accepted	\N	\N	N/A	4	1	0	1	6	f	\N	\N
428	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	45	255	127	2015-07-31 17:35:52.905985	2015-07-31 17:35:52.905985	2	accepted	\N	\N	N/A	4	1	0	1	6	f	\N	\N
429	After adjusting for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	45	256	127	2015-07-31 17:35:52.960517	2015-07-31 17:35:52.960517	3	accepted	\N	\N	N/A	4	1	0	0	3	f	\N	\N
430	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	13	255	128	2015-07-31 17:35:52.995982	2015-07-31 17:35:52.995982	3	accepted	\N	\N	N/A	4	0	0	1	4	f	\N	\N
431	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	13	255	128	2015-07-31 17:35:53.028051	2015-07-31 17:35:53.028051	2	accepted	\N	\N	N/A	4	1	0	1	6	f	\N	\N
432	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	45	255	128	2015-07-31 17:35:53.057154	2015-07-31 17:35:53.057154	2	accepted	\N	\N	N/A	4	1	0	1	6	f	\N	\N
433	After adjusting for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.	45	256	128	2015-07-31 17:35:53.088766	2015-07-31 17:35:53.088766	3	accepted	\N	\N	N/A	4	1	0	0	3	f	\N	\N
434	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	12	257	79	2015-07-31 17:35:53.125426	2015-07-31 17:35:53.125426	2	accepted	\N	\N	12:25398284-25398284 (C->T)	3	2	0	0	1	f	\N	\N
435	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	52	258	30	2015-07-31 17:35:53.169742	2015-07-31 17:35:53.169742	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
436	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580	52	258	84	2015-07-31 17:35:53.212762	2015-07-31 17:35:53.212762	3	accepted	\N	\N	N/A	2	2	0	0	0	f	\N	2
437	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	53	259	31	2015-07-31 17:35:53.255527	2015-07-31 17:35:53.255527	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
438	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	53	259	109	2015-07-31 17:35:53.285938	2015-07-31 17:35:53.285938	4	accepted	\N	\N	N/A	1	0	0	0	4	f	\N	\N
439	NSCLC patients with exon 19 deletions in EGFR had longer overall survival than patients with an L858R mutation when treated with gefitinib and erlotinib (38 versus 17 months; P = 0.04).	12	260	133	2015-07-31 17:35:53.340825	2015-07-31 17:35:53.340825	4	accepted	\N	\N	N/A	1	2	0	0	0	f	\N	2
440	In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions had longer overall survival than those with other types of EWS-FLI1 fusions.	54	261	151	2015-07-31 17:35:53.393048	2015-07-31 17:35:53.393048	4	accepted	\N	\N	N/A	1	1	0	0	4	f	\N	\N
\.


--
-- Data for Name: drugs_evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs_evidence_items (drug_id, evidence_item_id) FROM stdin;
15	37
16	38
16	39
17	40
18	41
16	42
16	43
16	44
16	45
16	46
19	47
16	48
20	49
20	50
16	51
16	52
16	53
21	54
16	55
22	56
23	57
23	58
16	59
24	60
24	61
25	62
25	63
26	64
25	65
27	66
27	67
15	68
28	69
16	70
16	71
16	72
16	73
16	74
16	75
16	76
16	77
16	78
16	79
16	80
16	81
16	82
16	83
16	84
16	85
24	86
24	87
16	88
24	89
16	90
24	91
29	91
15	91
30	92
31	93
32	93
16	94
16	95
16	96
16	97
16	98
33	99
34	100
34	101
34	102
34	103
34	104
35	105
36	106
16	107
16	108
36	109
37	110
38	111
35	112
28	112
35	113
16	114
16	115
16	116
16	117
39	118
40	118
35	119
28	119
16	120
16	121
16	122
16	123
16	124
16	125
15	126
16	127
41	128
42	128
42	129
43	129
44	130
22	130
45	130
22	131
35	132
20	133
35	134
35	135
46	136
46	137
46	138
46	139
26	140
25	141
27	142
25	143
33	144
16	145
16	146
16	147
16	148
16	149
16	150
16	151
16	152
16	153
16	154
16	155
16	156
16	157
16	158
16	159
16	160
16	161
47	162
16	163
48	164
16	165
16	166
16	167
16	168
16	169
16	170
16	171
17	172
49	173
16	174
17	175
49	176
16	177
33	178
33	179
16	180
33	181
33	182
16	183
16	184
16	185
16	186
16	187
16	188
16	189
16	190
48	191
48	192
17	193
50	194
16	195
16	196
16	197
16	198
16	199
16	200
16	201
16	202
16	203
16	204
16	205
16	206
16	207
16	208
16	209
16	210
16	211
16	212
16	213
16	214
51	215
48	215
16	216
16	217
16	218
16	219
51	220
52	221
33	222
33	223
33	224
53	225
54	226
16	227
16	228
16	229
16	230
16	231
16	232
16	233
16	234
16	235
16	236
16	237
16	238
16	239
16	240
16	241
16	242
55	243
55	244
16	245
56	246
24	247
24	248
29	249
30	249
16	250
24	251
24	252
24	253
24	254
57	255
25	256
25	257
25	258
16	259
16	260
16	261
16	262
16	263
16	264
16	265
16	266
16	267
16	268
16	269
16	270
58	271
16	272
16	273
16	274
16	275
59	276
60	277
59	278
60	279
16	280
16	281
30	282
30	283
30	284
30	285
30	286
30	287
30	288
32	288
31	289
31	290
31	291
32	291
31	292
32	292
16	293
16	294
31	295
32	296
31	297
16	298
16	299
32	300
61	300
62	301
63	302
64	303
64	304
64	305
64	306
65	307
64	308
64	309
64	310
64	311
65	311
64	312
64	313
64	314
66	315
67	316
68	316
66	317
67	318
68	318
66	319
67	320
68	320
66	321
67	322
68	322
66	323
67	324
68	324
69	325
70	326
69	326
16	327
71	328
15	329
16	330
16	331
72	332
16	333
15	334
15	335
71	336
16	337
16	338
73	339
74	339
16	340
16	341
16	342
16	343
16	344
16	345
16	346
16	347
16	348
16	349
16	350
16	351
16	352
16	353
16	354
16	355
16	356
16	357
16	358
16	359
16	360
16	361
16	362
24	363
30	364
24	364
29	364
75	365
24	366
33	367
16	368
16	369
16	370
16	371
16	372
31	373
32	373
16	374
31	375
32	375
16	376
76	377
77	377
31	378
32	378
31	379
32	379
16	380
16	381
33	382
33	383
63	384
40	385
78	386
79	387
16	388
16	389
16	390
16	391
80	392
53	393
33	394
38	394
16	395
16	396
80	397
53	398
33	399
38	399
16	400
16	401
80	402
53	403
33	404
38	404
16	405
48	406
16	407
81	408
48	409
16	410
16	411
16	412
82	413
16	414
16	415
16	416
16	417
16	418
16	419
16	420
16	421
16	422
16	423
54	424
54	425
16	426
16	427
16	428
16	429
16	430
16	431
16	432
16	433
35	434
24	435
83	435
24	436
83	436
16	437
16	438
31	439
32	439
16	440
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 83, true);


--
-- Data for Name: events; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY events (id, created_at, updated_at, action, description, originating_user_id, subject_id, subject_type, state_params) FROM stdin;
\.


--
-- Name: events_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('events_id_seq', 1, false);


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 440, true);


--
-- Data for Name: feeds; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY feeds (id, created_at, updated_at, owner_id, event_id, acknowledged) FROM stdin;
\.


--
-- Name: feeds_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('feeds_id_seq', 1, false);


--
-- Data for Name: gene_aliases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases (id, name) FROM stdin;
1	CD246
2	NBLST3
3	AKT
4	CWS6
5	PKB
6	PKB-ALPHA
7	PRKBA
8	RAC
9	RAC-ALPHA
10	A-RAF
11	ARAF1
12	PKS2
13	RAFA1
14	ABL
15	JTK7
16	bcr/abl
17	c-ABL
18	c-ABL1
19	p150
20	v-abl
21	B-RAF1
22	BRAF1
23	NS7
24	RAFB1
25	BRCAI
26	BRCC1
27	BROVCA1
28	FANCS
29	IRIS
30	PNCA4
31	PPP1R53
32	PSCP
33	RNF53
34	BRCC2
35	BROVCA2
36	FACD
37	FAD
38	FAD1
39	FANCD
40	FANCD1
41	GLM3
42	PNCA2
43	XRCC11
44	BCL1
45	D11S287E
46	PRAD1
47	U21B31
48	KIAK0002
49	MPPH3
50	CCNE
51	pCCNE1
52	MCPH12
53	PLSTIRE
54	CMM3
55	PSK-J3
56	ARF
57	CDK4I
58	CDKN2
59	CMM2
60	INK4
61	INK4A
62	MLM
63	MTS-1
64	MTS1
65	P14
66	P14ARF
67	P16
68	P16-INK4A
69	P16INK4
70	P16INK4A
71	P19
72	P19ARF
73	TP16
74	C/EBP-alpha
75	CEBP
76	HSPF1
77	Hdj1
78	Hsp40
79	RSPH16B
80	Sis1
81	PKACA
82	PPNAD4
83	DNMT3A2
84	M.HsaIIIA
85	TBRS
86	ERBB
87	ERBB1
88	HER1
89	NISBD2
90	PIG61
91	mENA
92	CD340
93	HER-2
94	HER-2/neu
95	HER2
96	MLN 19
97	NEU
98	NGL
99	TKR1
100	ER
101	ESR
102	ESRA
103	ESTRR
104	Era
105	NR3A1
106	BBDS
107	BEK
108	BFR-1
109	CD332
110	CEK3
111	CFD1
112	ECT1
113	JWS
114	K-SAM
115	KGFR
116	TK14
117	TK25
118	ACH
119	CD333
120	CEK2
121	HSFGFR3EX
122	JTK4
123	CD135
124	FLK-2
125	FLK2
126	STK1
127	DCML
128	IMD21
129	MONOMAC
130	NFE1B
131	HEL-216
132	HEL-S-26
133	IDCD
134	IDH
135	IDP
136	IDPC
137	PICD
138	D2HGA2
139	ICD-M
140	IDHM
141	IDPM
142	mNADP-IDH
143	JTK10
144	THCYT3
145	C-Kit
146	CD117
147	PBT
148	SCFR
149	C-K-RAS
150	CFC2
151	K-RAS2A
152	K-RAS2B
153	K-RAS4A
154	K-RAS4B
155	KI-RAS
156	KRAS1
157	KRAS2
158	NS
159	NS3
160	RASK2
161	CFC3
162	MAPKK1
163	MEK1
164	MKK1
165	PRKMK1
166	C-FMS
167	CD115
168	CSF-1R
169	CSFR
170	FIM2
171	FMS
172	HDLS
173	M-CSF-R
174	B23
175	NPM
176	ALPS4
177	CMNS
178	N-ras
179	NCMS
180	NRAS1
181	NS6
182	CLOVE
183	CWS5
184	MCAP
185	MCM
186	MCMTC
187	PI3K
188	p110-alpha
189	CD140A
190	PDGFR-2
191	PDGFR2
192	RHEPDGFRA
193	MYL
194	PP8675
195	RNF71
196	TRIM19
197	NR1B1
198	RAR
199	10q23del
200	BZS
201	CWS1
202	DEC
203	GLM2
204	MHAM
205	MMAC1
206	PTEN1
207	TEP1
208	CDHF12
209	CDHR16
210	HSCR1
211	MEN2A
212	MEN2B
213	MTC1
214	PTC
215	RET-ELE1
216	RET51
217	AML1
218	AML1-EVI-1
219	AMLCR1
220	CBFA2
221	EVI-1
222	PEBP2aB
223	Hsh155
224	MDS
225	PRP10
226	PRPF10
227	SAP155
228	SF3b155
229	BCC7
230	LFS1
231	P53
232	TRP53
233	LAM
234	TSC
235	PPP1R160
236	TSC4
237	FP793
238	RN
239	RNU2AF1
240	U2AF35
241	U2AFBP
242	AWT1
243	EWS-WT1
244	GUD
245	NPHS4
246	WAGR
247	WIT-2
248	WT33
249	AOS5
250	AOVD1
251	TAN1
252	hN1
253	MIG20a
254	NTRKR3
255	TKT
256	TYRO10
257	AUTS9
258	DFNB97
259	HGFR
260	RCCP2
261	c-Met
262	EWSR2
263	SIC-1
264	EWS
265	EWS-FLI1
266	bK984G1.4
\.


--
-- Data for Name: gene_aliases_genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY gene_aliases_genes (gene_alias_id, gene_id) FROM stdin;
1	1
2	1
3	2
4	2
5	2
6	2
7	2
8	2
9	2
10	3
11	3
12	3
13	3
14	4
15	4
16	4
17	4
18	4
19	4
20	4
21	5
22	5
23	5
24	5
25	6
26	6
27	6
28	6
29	6
30	6
31	6
32	6
33	6
34	7
35	7
36	7
37	7
38	7
39	7
40	7
41	7
42	7
43	7
44	8
45	8
46	8
47	8
48	9
49	9
50	11
51	11
52	12
53	12
54	13
55	13
56	14
57	14
58	14
59	14
60	14
61	14
62	14
63	14
64	14
65	14
66	14
67	14
68	14
69	14
70	14
71	14
72	14
73	14
74	15
75	15
76	16
77	16
78	16
79	16
80	16
81	17
82	17
83	18
84	18
85	18
86	19
87	19
88	19
89	19
90	19
91	19
92	20
93	20
94	20
95	20
96	20
97	20
98	20
99	20
100	21
101	21
102	21
103	21
104	21
105	21
106	22
107	22
108	22
109	22
110	22
111	22
112	22
113	22
114	22
115	22
116	22
117	22
118	23
119	23
120	23
121	23
122	23
123	24
124	24
125	24
126	24
127	25
128	25
129	25
130	25
131	26
132	26
133	26
134	26
135	26
136	26
137	26
138	27
139	27
134	27
140	27
135	27
141	27
142	27
143	28
144	28
145	29
146	29
147	29
148	29
149	30
150	30
151	30
152	30
153	30
154	30
155	30
156	30
157	30
158	30
159	30
160	30
161	31
162	31
163	31
164	31
165	31
166	33
167	33
168	33
169	33
170	33
171	33
172	33
173	33
174	35
175	35
176	36
177	36
178	36
179	36
180	36
181	36
182	37
183	37
184	37
185	37
186	37
187	37
188	37
189	38
190	38
191	38
192	38
193	39
194	39
195	39
196	39
197	40
198	40
199	41
200	41
201	41
202	41
203	41
204	41
205	41
206	41
207	41
208	42
209	42
210	42
211	42
212	42
213	42
214	42
215	42
216	42
217	43
218	43
219	43
220	43
221	43
222	43
223	44
224	44
225	44
226	44
227	44
228	44
229	45
230	45
231	45
232	45
233	46
234	46
233	47
235	47
236	47
237	48
238	48
239	48
240	48
241	48
242	49
243	49
244	49
245	49
246	49
247	49
248	49
249	50
250	50
251	50
252	50
253	51
254	51
255	51
256	51
257	52
258	52
259	52
260	52
261	52
262	53
263	53
264	54
265	54
266	54
1	1
2	1
3	2
4	2
5	2
6	2
7	2
8	2
9	2
10	3
11	3
12	3
13	3
14	4
15	4
16	4
17	4
18	4
19	4
20	4
21	5
22	5
23	5
24	5
25	6
26	6
27	6
28	6
29	6
30	6
31	6
32	6
33	6
34	7
35	7
36	7
37	7
38	7
39	7
40	7
41	7
42	7
43	7
44	8
45	8
46	8
47	8
48	9
49	9
50	11
51	11
52	12
53	12
54	13
55	13
56	14
57	14
58	14
59	14
60	14
61	14
62	14
63	14
64	14
65	14
66	14
67	14
68	14
69	14
70	14
71	14
72	14
73	14
74	15
75	15
76	16
77	16
78	16
79	16
80	16
81	17
82	17
83	18
84	18
85	18
86	19
87	19
88	19
89	19
90	19
91	19
92	20
93	20
94	20
95	20
96	20
97	20
98	20
99	20
100	21
101	21
102	21
103	21
104	21
105	21
106	22
107	22
108	22
109	22
110	22
111	22
112	22
113	22
114	22
115	22
116	22
117	22
118	23
119	23
120	23
121	23
122	23
123	24
124	24
125	24
126	24
127	25
128	25
129	25
130	25
131	26
132	26
133	26
134	26
135	26
136	26
137	26
138	27
139	27
134	27
140	27
135	27
141	27
142	27
143	28
144	28
145	29
146	29
147	29
148	29
149	30
150	30
151	30
152	30
153	30
154	30
155	30
156	30
157	30
158	30
159	30
160	30
161	31
162	31
163	31
164	31
165	31
166	33
167	33
168	33
169	33
170	33
171	33
172	33
173	33
174	35
175	35
176	36
177	36
178	36
179	36
180	36
181	36
182	37
183	37
184	37
185	37
186	37
187	37
188	37
189	38
190	38
191	38
192	38
193	39
194	39
195	39
196	39
197	40
198	40
199	41
200	41
201	41
202	41
203	41
204	41
205	41
206	41
207	41
208	42
209	42
210	42
211	42
212	42
213	42
214	42
215	42
216	42
217	43
218	43
219	43
220	43
221	43
222	43
223	44
224	44
225	44
226	44
227	44
228	44
229	45
230	45
231	45
232	45
233	46
234	46
233	47
235	47
236	47
237	48
238	48
239	48
240	48
241	48
242	49
243	49
244	49
245	49
246	49
247	49
248	49
249	50
250	50
251	50
252	50
253	51
254	51
255	51
256	51
257	52
258	52
259	52
260	52
261	52
262	53
263	53
264	54
265	54
266	54
\.


--
-- Name: gene_aliases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('gene_aliases_id_seq', 266, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 54, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-07-31 17:34:13.908354	2015-07-31 17:34:13.908354
1	2	2015-07-31 17:34:13.923149	2015-07-31 17:34:13.923149
2	3	2015-07-31 17:34:14.173927	2015-07-31 17:34:14.173927
2	4	2015-07-31 17:34:14.189265	2015-07-31 17:34:14.189265
3	5	2015-07-31 17:34:14.464449	2015-07-31 17:34:14.464449
3	6	2015-07-31 17:34:14.478541	2015-07-31 17:34:14.478541
4	7	2015-07-31 17:34:14.723568	2015-07-31 17:34:14.723568
4	8	2015-07-31 17:34:14.734269	2015-07-31 17:34:14.734269
5	9	2015-07-31 17:34:14.974073	2015-07-31 17:34:14.974073
5	10	2015-07-31 17:34:14.985394	2015-07-31 17:34:14.985394
6	11	2015-07-31 17:34:15.24271	2015-07-31 17:34:15.24271
7	11	2015-07-31 17:34:15.50677	2015-07-31 17:34:15.50677
8	12	2015-07-31 17:34:15.747678	2015-07-31 17:34:15.747678
8	13	2015-07-31 17:34:15.761127	2015-07-31 17:34:15.761127
9	12	2015-07-31 17:34:15.969412	2015-07-31 17:34:15.969412
9	13	2015-07-31 17:34:15.972031	2015-07-31 17:34:15.972031
10	12	2015-07-31 17:34:16.155758	2015-07-31 17:34:16.155758
10	13	2015-07-31 17:34:16.158466	2015-07-31 17:34:16.158466
11	14	2015-07-31 17:34:16.375458	2015-07-31 17:34:16.375458
12	15	2015-07-31 17:34:16.584497	2015-07-31 17:34:16.584497
12	16	2015-07-31 17:34:16.594196	2015-07-31 17:34:16.594196
13	15	2015-07-31 17:34:16.792577	2015-07-31 17:34:16.792577
13	16	2015-07-31 17:34:16.794853	2015-07-31 17:34:16.794853
14	17	2015-07-31 17:34:17.10031	2015-07-31 17:34:17.10031
14	18	2015-07-31 17:34:17.111744	2015-07-31 17:34:17.111744
15	19	2015-07-31 17:34:17.361847	2015-07-31 17:34:17.361847
15	20	2015-07-31 17:34:17.371674	2015-07-31 17:34:17.371674
15	21	2015-07-31 17:34:17.381278	2015-07-31 17:34:17.381278
16	22	2015-07-31 17:34:17.60446	2015-07-31 17:34:17.60446
17	22	2015-07-31 17:34:17.80979	2015-07-31 17:34:17.80979
17	23	2015-07-31 17:34:17.819812	2015-07-31 17:34:17.819812
18	24	2015-07-31 17:34:18.042133	2015-07-31 17:34:18.042133
18	25	2015-07-31 17:34:18.051688	2015-07-31 17:34:18.051688
19	26	2015-07-31 17:34:18.294189	2015-07-31 17:34:18.294189
19	27	2015-07-31 17:34:18.304445	2015-07-31 17:34:18.304445
20	28	2015-07-31 17:34:18.555085	2015-07-31 17:34:18.555085
20	29	2015-07-31 17:34:18.565931	2015-07-31 17:34:18.565931
21	30	2015-07-31 17:34:18.799697	2015-07-31 17:34:18.799697
21	31	2015-07-31 17:34:18.809651	2015-07-31 17:34:18.809651
22	32	2015-07-31 17:34:19.083942	2015-07-31 17:34:19.083942
22	33	2015-07-31 17:34:19.094751	2015-07-31 17:34:19.094751
23	32	2015-07-31 17:34:19.333858	2015-07-31 17:34:19.333858
23	33	2015-07-31 17:34:19.336495	2015-07-31 17:34:19.336495
24	34	2015-07-31 17:34:19.561355	2015-07-31 17:34:19.561355
24	19	2015-07-31 17:34:19.563955	2015-07-31 17:34:19.563955
25	35	2015-07-31 17:34:19.793474	2015-07-31 17:34:19.793474
25	36	2015-07-31 17:34:19.802892	2015-07-31 17:34:19.802892
26	37	2015-07-31 17:34:20.043242	2015-07-31 17:34:20.043242
26	38	2015-07-31 17:34:20.052996	2015-07-31 17:34:20.052996
27	37	2015-07-31 17:34:20.289313	2015-07-31 17:34:20.289313
27	38	2015-07-31 17:34:20.293073	2015-07-31 17:34:20.293073
27	39	2015-07-31 17:34:20.304595	2015-07-31 17:34:20.304595
28	40	2015-07-31 17:34:20.519652	2015-07-31 17:34:20.519652
29	41	2015-07-31 17:34:20.731594	2015-07-31 17:34:20.731594
30	42	2015-07-31 17:34:20.993041	2015-07-31 17:34:20.993041
35	43	2015-07-31 17:34:22.057003	2015-07-31 17:34:22.057003
35	19	2015-07-31 17:34:22.060649	2015-07-31 17:34:22.060649
42	44	2015-07-31 17:34:23.578033	2015-07-31 17:34:23.578033
42	45	2015-07-31 17:34:23.587364	2015-07-31 17:34:23.587364
44	46	2015-07-31 17:34:24.098086	2015-07-31 17:34:24.098086
44	47	2015-07-31 17:34:24.108153	2015-07-31 17:34:24.108153
45	48	2015-07-31 17:34:24.343681	2015-07-31 17:34:24.343681
48	49	2015-07-31 17:34:25.033068	2015-07-31 17:34:25.033068
48	50	2015-07-31 17:34:25.045027	2015-07-31 17:34:25.045027
1	1	2015-07-31 17:35:26.017674	2015-07-31 17:35:26.017674
1	2	2015-07-31 17:35:26.023251	2015-07-31 17:35:26.023251
2	3	2015-07-31 17:35:26.242014	2015-07-31 17:35:26.242014
2	4	2015-07-31 17:35:26.245751	2015-07-31 17:35:26.245751
3	5	2015-07-31 17:35:26.443932	2015-07-31 17:35:26.443932
3	6	2015-07-31 17:35:26.447413	2015-07-31 17:35:26.447413
4	7	2015-07-31 17:35:26.633696	2015-07-31 17:35:26.633696
4	8	2015-07-31 17:35:26.636836	2015-07-31 17:35:26.636836
5	9	2015-07-31 17:35:26.833123	2015-07-31 17:35:26.833123
5	10	2015-07-31 17:35:26.836773	2015-07-31 17:35:26.836773
6	11	2015-07-31 17:35:27.089873	2015-07-31 17:35:27.089873
7	11	2015-07-31 17:35:27.297848	2015-07-31 17:35:27.297848
8	12	2015-07-31 17:35:27.509095	2015-07-31 17:35:27.509095
8	13	2015-07-31 17:35:27.512249	2015-07-31 17:35:27.512249
9	12	2015-07-31 17:35:27.695724	2015-07-31 17:35:27.695724
9	13	2015-07-31 17:35:27.698677	2015-07-31 17:35:27.698677
10	12	2015-07-31 17:35:27.874506	2015-07-31 17:35:27.874506
10	13	2015-07-31 17:35:27.877503	2015-07-31 17:35:27.877503
11	14	2015-07-31 17:35:28.042233	2015-07-31 17:35:28.042233
12	15	2015-07-31 17:35:28.219976	2015-07-31 17:35:28.219976
12	16	2015-07-31 17:35:28.223043	2015-07-31 17:35:28.223043
13	15	2015-07-31 17:35:28.400095	2015-07-31 17:35:28.400095
13	16	2015-07-31 17:35:28.403557	2015-07-31 17:35:28.403557
14	17	2015-07-31 17:35:28.582401	2015-07-31 17:35:28.582401
14	18	2015-07-31 17:35:28.586767	2015-07-31 17:35:28.586767
15	19	2015-07-31 17:35:28.820001	2015-07-31 17:35:28.820001
15	20	2015-07-31 17:35:28.822917	2015-07-31 17:35:28.822917
15	21	2015-07-31 17:35:28.825712	2015-07-31 17:35:28.825712
16	22	2015-07-31 17:35:28.998976	2015-07-31 17:35:28.998976
17	22	2015-07-31 17:35:29.185841	2015-07-31 17:35:29.185841
17	23	2015-07-31 17:35:29.188709	2015-07-31 17:35:29.188709
18	24	2015-07-31 17:35:29.365313	2015-07-31 17:35:29.365313
18	25	2015-07-31 17:35:29.368862	2015-07-31 17:35:29.368862
19	26	2015-07-31 17:35:29.632444	2015-07-31 17:35:29.632444
19	27	2015-07-31 17:35:29.635574	2015-07-31 17:35:29.635574
20	28	2015-07-31 17:35:29.836797	2015-07-31 17:35:29.836797
20	29	2015-07-31 17:35:29.840691	2015-07-31 17:35:29.840691
21	30	2015-07-31 17:35:30.050653	2015-07-31 17:35:30.050653
21	31	2015-07-31 17:35:30.054149	2015-07-31 17:35:30.054149
22	32	2015-07-31 17:35:30.255679	2015-07-31 17:35:30.255679
22	33	2015-07-31 17:35:30.259818	2015-07-31 17:35:30.259818
23	32	2015-07-31 17:35:30.477065	2015-07-31 17:35:30.477065
23	33	2015-07-31 17:35:30.480543	2015-07-31 17:35:30.480543
24	34	2015-07-31 17:35:30.668192	2015-07-31 17:35:30.668192
24	19	2015-07-31 17:35:30.671701	2015-07-31 17:35:30.671701
25	35	2015-07-31 17:35:30.857886	2015-07-31 17:35:30.857886
25	36	2015-07-31 17:35:30.860822	2015-07-31 17:35:30.860822
26	37	2015-07-31 17:35:31.041572	2015-07-31 17:35:31.041572
26	38	2015-07-31 17:35:31.044446	2015-07-31 17:35:31.044446
27	37	2015-07-31 17:35:31.240634	2015-07-31 17:35:31.240634
27	38	2015-07-31 17:35:31.243747	2015-07-31 17:35:31.243747
27	39	2015-07-31 17:35:31.247414	2015-07-31 17:35:31.247414
28	40	2015-07-31 17:35:31.448185	2015-07-31 17:35:31.448185
29	41	2015-07-31 17:35:31.623908	2015-07-31 17:35:31.623908
30	42	2015-07-31 17:35:31.807167	2015-07-31 17:35:31.807167
35	43	2015-07-31 17:35:32.71893	2015-07-31 17:35:32.71893
35	19	2015-07-31 17:35:32.723062	2015-07-31 17:35:32.723062
42	44	2015-07-31 17:35:34.021879	2015-07-31 17:35:34.021879
42	45	2015-07-31 17:35:34.02524	2015-07-31 17:35:34.02524
44	46	2015-07-31 17:35:34.4272	2015-07-31 17:35:34.4272
44	47	2015-07-31 17:35:34.430232	2015-07-31 17:35:34.430232
45	48	2015-07-31 17:35:34.617179	2015-07-31 17:35:34.617179
48	49	2015-07-31 17:35:35.163749	2015-07-31 17:35:35.163749
48	50	2015-07-31 17:35:35.167045	2015-07-31 17:35:35.167045
\.


--
-- Name: organizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('organizations_id_seq', 1, false);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
20150220160805
20150220182109
20150223170106
20150227204142
20150227212146
20150303202505
20150305200429
20150319181853
20150416182633
20150501151624
20150511191452
20150512184905
20150528170506
20150528211748
20150602202928
20150609210448
20150611153328
20150611154621
20150611200713
20150709222321
20150709230109
20150722183121
20150722185207
20150722185935
20150728191648
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 261, true);


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at, action_type, action_class) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Data for Name: tsv_releases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY tsv_releases (id, path, created_at, updated_at) FROM stdin;
\.


--
-- Name: tsv_releases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('tsv_releases_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at, deleted, deleted_at) FROM stdin;
4	KIT Exon 17	\N	2015-07-31 17:35:37.781567	2015-07-31 17:35:37.781567	f	\N
7	KIT Exon 11	\N	2015-07-31 17:35:39.282664	2015-07-31 17:35:39.282664	f	\N
11	NPM1 Exon 12	\N	2015-07-31 17:35:44.475762	2015-07-31 17:35:44.475762	f	\N
5	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-07-31 17:35:38.267584	2015-07-31 17:35:53.507189	f	\N
14	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-07-31 17:35:46.006103	2015-07-31 17:35:53.521411	f	\N
15	EGFR TKI Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-07-31 17:35:47.567504	2015-07-31 17:35:53.533707	f	\N
6	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-07-31 17:35:38.461276	2015-07-31 17:35:53.544946	f	\N
8	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-07-31 17:35:40.096225	2015-07-31 17:35:53.560823	f	\N
16	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-07-31 17:35:47.894729	2015-07-31 17:35:53.5728	f	\N
17	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-07-31 17:35:48.375451	2015-07-31 17:35:53.587014	f	\N
10	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-07-31 17:35:42.376362	2015-07-31 17:35:53.598952	f	\N
18	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-07-31 17:35:52.129485	2015-07-31 17:35:53.613343	f	\N
9	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-07-31 17:35:40.155176	2015-07-31 17:35:53.626745	f	\N
12	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-07-31 17:35:44.75343	2015-07-31 17:35:53.644916	f	\N
13	BRCA Germline Variants	BRCA germline variants have long been recognized as important potential predictors of breast and ovarian cancer risk. 	2015-07-31 17:35:45.446651	2015-07-31 17:35:53.658828	f	\N
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
65	4	2015-07-31 17:35:37.802888	2015-07-31 17:35:37.802888
99	5	2015-07-31 17:35:38.281556	2015-07-31 17:35:38.281556
99	5	2015-07-31 17:35:38.3301	2015-07-31 17:35:38.3301
99	5	2015-07-31 17:35:38.381498	2015-07-31 17:35:38.381498
8	6	2015-07-31 17:35:38.472081	2015-07-31 17:35:38.472081
66	7	2015-07-31 17:35:39.295905	2015-07-31 17:35:39.295905
66	7	2015-07-31 17:35:39.342656	2015-07-31 17:35:39.342656
70	4	2015-07-31 17:35:39.442449	2015-07-31 17:35:39.442449
72	7	2015-07-31 17:35:39.533169	2015-07-31 17:35:39.533169
72	7	2015-07-31 17:35:39.569537	2015-07-31 17:35:39.569537
98	5	2015-07-31 17:35:39.900577	2015-07-31 17:35:39.900577
99	5	2015-07-31 17:35:39.936515	2015-07-31 17:35:39.936515
100	5	2015-07-31 17:35:39.972382	2015-07-31 17:35:39.972382
101	5	2015-07-31 17:35:40.014328	2015-07-31 17:35:40.014328
102	5	2015-07-31 17:35:40.049116	2015-07-31 17:35:40.049116
13	8	2015-07-31 17:35:40.107694	2015-07-31 17:35:40.107694
112	9	2015-07-31 17:35:40.166468	2015-07-31 17:35:40.166468
113	9	2015-07-31 17:35:40.306882	2015-07-31 17:35:40.306882
11	8	2015-07-31 17:35:40.479479	2015-07-31 17:35:40.479479
15	8	2015-07-31 17:35:41.402322	2015-07-31 17:35:41.402322
16	8	2015-07-31 17:35:41.439937	2015-07-31 17:35:41.439937
53	10	2015-07-31 17:35:42.396263	2015-07-31 17:35:42.396263
87	11	2015-07-31 17:35:44.504602	2015-07-31 17:35:44.504602
87	11	2015-07-31 17:35:44.538297	2015-07-31 17:35:44.538297
87	11	2015-07-31 17:35:44.572945	2015-07-31 17:35:44.572945
87	11	2015-07-31 17:35:44.612309	2015-07-31 17:35:44.612309
124	12	2015-07-31 17:35:44.764845	2015-07-31 17:35:44.764845
125	12	2015-07-31 17:35:44.817694	2015-07-31 17:35:44.817694
131	13	2015-07-31 17:35:45.4592	2015-07-31 17:35:45.4592
132	13	2015-07-31 17:35:45.500002	2015-07-31 17:35:45.500002
3	5	2015-07-31 17:35:45.831098	2015-07-31 17:35:45.831098
2	5	2015-07-31 17:35:45.874938	2015-07-31 17:35:45.874938
2	5	2015-07-31 17:35:45.912688	2015-07-31 17:35:45.912688
5	14	2015-07-31 17:35:46.019473	2015-07-31 17:35:46.019473
6	14	2015-07-31 17:35:46.056151	2015-07-31 17:35:46.056151
6	6	2015-07-31 17:35:46.060164	2015-07-31 17:35:46.060164
7	14	2015-07-31 17:35:46.102137	2015-07-31 17:35:46.102137
7	6	2015-07-31 17:35:46.105791	2015-07-31 17:35:46.105791
34	15	2015-07-31 17:35:47.580424	2015-07-31 17:35:47.580424
34	15	2015-07-31 17:35:47.843453	2015-07-31 17:35:47.843453
35	16	2015-07-31 17:35:47.906048	2015-07-31 17:35:47.906048
36	16	2015-07-31 17:35:47.940895	2015-07-31 17:35:47.940895
38	16	2015-07-31 17:35:48.011605	2015-07-31 17:35:48.011605
41	16	2015-07-31 17:35:48.154609	2015-07-31 17:35:48.154609
43	16	2015-07-31 17:35:48.245089	2015-07-31 17:35:48.245089
44	16	2015-07-31 17:35:48.285751	2015-07-31 17:35:48.285751
45	16	2015-07-31 17:35:48.329499	2015-07-31 17:35:48.329499
46	17	2015-07-31 17:35:48.388047	2015-07-31 17:35:48.388047
46	17	2015-07-31 17:35:48.441324	2015-07-31 17:35:48.441324
47	17	2015-07-31 17:35:48.482041	2015-07-31 17:35:48.482041
47	17	2015-07-31 17:35:48.526319	2015-07-31 17:35:48.526319
48	17	2015-07-31 17:35:48.56108	2015-07-31 17:35:48.56108
48	17	2015-07-31 17:35:48.602248	2015-07-31 17:35:48.602248
49	17	2015-07-31 17:35:48.641493	2015-07-31 17:35:48.641493
49	17	2015-07-31 17:35:48.686991	2015-07-31 17:35:48.686991
50	17	2015-07-31 17:35:48.722213	2015-07-31 17:35:48.722213
50	17	2015-07-31 17:35:48.779161	2015-07-31 17:35:48.779161
52	10	2015-07-31 17:35:48.83388	2015-07-31 17:35:48.83388
54	10	2015-07-31 17:35:48.880637	2015-07-31 17:35:48.880637
66	7	2015-07-31 17:35:50.093203	2015-07-31 17:35:50.093203
66	7	2015-07-31 17:35:50.141907	2015-07-31 17:35:50.141907
66	7	2015-07-31 17:35:50.18824	2015-07-31 17:35:50.18824
66	7	2015-07-31 17:35:50.227995	2015-07-31 17:35:50.227995
70	4	2015-07-31 17:35:50.265483	2015-07-31 17:35:50.265483
67	7	2015-07-31 17:35:50.366031	2015-07-31 17:35:50.366031
72	7	2015-07-31 17:35:50.418753	2015-07-31 17:35:50.418753
73	7	2015-07-31 17:35:50.462936	2015-07-31 17:35:50.462936
73	7	2015-07-31 17:35:50.497376	2015-07-31 17:35:50.497376
148	15	2015-07-31 17:35:50.825292	2015-07-31 17:35:50.825292
78	15	2015-07-31 17:35:50.906372	2015-07-31 17:35:50.906372
79	15	2015-07-31 17:35:51.040452	2015-07-31 17:35:51.040452
147	15	2015-07-31 17:35:51.088198	2015-07-31 17:35:51.088198
146	15	2015-07-31 17:35:51.263658	2015-07-31 17:35:51.263658
110	18	2015-07-31 17:35:52.144595	2015-07-31 17:35:52.144595
110	18	2015-07-31 17:35:52.195835	2015-07-31 17:35:52.195835
124	12	2015-07-31 17:35:52.778518	2015-07-31 17:35:52.778518
126	12	2015-07-31 17:35:52.8161	2015-07-31 17:35:52.8161
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 18, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 151, true);


--
-- PostgreSQL database dump complete
--

