--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	920c9fd2-8997-4254-b614-a393c5d6e9a6	2015-02-20 16:15:28.892128
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, is a known oncogene. AKT\n  activation relies on the PI3K pathway, and is recognized as a critical node in the\n  pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been\n  shown to result in activation of the protein. Mutations in AKT1 have also been shown\n  to confer resistance to allosteric kinase inhibitors in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	62418641-9ccd-46cd-8c93-4bc03e426b5d	2015-02-20 16:15:29.190787
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	ba64c39d-8cca-4e9e-890d-2a186bc18df8	2015-02-20 16:15:29.450862
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia (CML). Cells harboring this\n  fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook\n  of the disease. However, additional mutations in ABL1 have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	629526d0-f57f-45de-9c85-b6e521a28179	2015-02-20 16:15:29.742494
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been correlated with poor prognosis in many cancer types, although there\n  is at least one study that questions this conclusion in papillary thyroid cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	492397ba-c873-42ec-a8f5-5a96da0607ca	2015-02-20 16:15:29.992122
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	4b967f6f-f6e9-4b79-b7b6-d24a4c806037	2015-02-20 16:15:30.237008
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	502b0280-a918-4873-9bc7-1b0f1adfc120	2015-02-20 16:15:30.445877
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, ranging from genomic amplification to changes in promoter methylation.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	011a81b5-0932-44a9-83a9-4bbff7be711c	2015-02-20 16:15:30.656187
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as its cyclin D counterparts,\n  has been implicated in various carcinomas, including breast, gastric, stomach and\n  colorectal. High levels of cyclin E, either by gene amplification or overexpression,\n  have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements\n  are correlated with later stage disease. In lung cancer, neoplastic cells with higher\n  levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly\n  expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	d9d0add0-ef8f-498a-a7ab-3be0180815c1	2015-02-20 16:15:30.855767
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	cdc5f64b-7ca0-4f06-86cf-f28bdb8cbfa8	2015-02-20 16:15:31.080291
634	42	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	688cd4ea-f359-4d4c-9245-ba77a0e7e455	2015-02-20 16:22:50.155531
635	43	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f66b53e5-ac14-4932-a929-3fcd97e1562e	2015-02-20 16:22:50.181637
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	212449ec-4e40-47de-8b45-fd9d6077741b	2015-02-20 16:15:31.286385
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. While no targeted therapeutic has been engaged in clinical trials, the prognostic\n  impact has been studied by a number of meta-analyses. One mechanism by which loss\n  of CDKN2A can occur is by hypermethylation of the promoter region for the gene.\n  However, the prognostic impact of promoter hypermethylation has been relatively\n  ambiguous. Many studies have suggesting poorer prognostic outcome for patients with\n  hypermethylation in colorectal, liver, and younger lung cancer patients. This being\n  said, there is still research to be done before this becomes a widely-accepted prognostic\n  indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	6f8659b3-f0cc-4d9f-a8ea-01fedb00ffe4	2015-02-20 16:15:31.494022
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications.\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	8b13e028-c1f2-4ab7-96ab-13786d4953d0	2015-02-20 16:15:31.723255
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In\n  one study, this fusion was observed in 15/15 FL-HCC cases examined and functional\n  studies found that the fusion retained kinase activity.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	0420ddba-0b6e-4430-aedb-9c4c68c3c706	2015-02-20 16:15:31.923846
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: '"PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker for this rare subtype of HCC. In one study, this fusion was\n  observed in 15/15 FL-HCC cases examined and functional studies found that the fusion\n  retained kinase activity."'\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	9a2c95c1-0b77-4353-86c9-736a86190282	2015-02-20 16:15:32.158865
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	12c3ef95-c938-494a-82eb-7e9eb1d899e8	2015-02-20 16:15:32.385664
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably\n  gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance\n  to these drugs, particularly the variant T790M, which has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has become a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	e95384bc-6048-4954-9d3e-e11309999c93	2015-02-20 16:15:32.693073
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast\n  cancer, and is treated in a separate manner than the other subtypes of breast cancer.\n  Apart from being amplified, ERBB2 activating mutations have been shown to have clinical\n  importance in HER2-negative breast cancer. These mutations have shown senitivity\n  to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical\n  sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	0a364fc6-57e1-4e69-857c-c97fa87073be	2015-02-20 16:15:32.940994
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, but has also\n  shown significance in endometrial, ovarian and other cancer types. ER-positive breast\n  cancer that is resistant to hormone therapy has instigated clinical sequencing efforts\n  to shed light on the mechanisms of this resistance. A number of mutations in the\n  ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	d3ff55d5-b256-4171-a18a-04f316eeb8ff	2015-02-20 16:15:33.182878
158	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	0adf3979-348c-4bf1-8c8e-c8058e3536fe	2015-02-20 16:15:42.934216
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFR can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions have been demonstrated\n  to have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	5139f95a-9b34-41d9-b6ff-5b26d1565bbc	2015-02-20 16:15:33.389877
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. Activation of these receptors can lead to the activation\n  of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by\n  which FGFRs can be misregulated vary between cancers. Amplification of the receptors\n  has been observed in lung and breast cancers, coding mutations and deletions have\n  been seen in many cancers, and more recently, FGFR fusions have been demonstrated\n  to have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	b5870e84-d535-4479-9536-e50c292ae319	2015-02-20 16:15:33.664345
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	e7cf9291-392b-4eea-929c-32b01c415fc7	2015-02-20 16:15:34.009028
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: N/A\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	80757d95-edcd-452f-bc99-9f0f9eb69026	2015-02-20 16:15:34.198987
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: IDH1 mutations have been observed in a number of cancer types, and appear\n  to behave quite differently in different environments. In myelodysplastic syndromes\n  and acute myeloid leukemia (AML), IDH1 mutations have been shown to be associated\n  with worse outcome and shorter overall survival. They have also been linked to cytogenetically\n  normal AML, an intermediate risk subtype of the disease. However, in glioblastoma\n  and astrocytoma, patients with IDH1 mutations have shown better overall survival\n  than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1\n  mutations have been associated with specific cytogenetic abnormalities, 1p and 19q\n  deletions.\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	4617fcdb-f11b-4980-9b0c-ca50d37d5a95	2015-02-20 16:15:34.435156
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: N/A\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	54433cfa-6742-4b88-82cd-d450a99542a6	2015-02-20 16:15:34.669094
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	49e674eb-1a28-4b43-b4a0-777f74bac96e	2015-02-20 16:15:34.862549
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: N/A\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	6c4d2054-d529-4b47-b00c-c3a38f5f8df9	2015-02-20 16:15:35.047082
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: N/A\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	188732cd-5a6b-4482-b4da-c040d173d489	2015-02-20 16:15:35.270039
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: N/A\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	33a8a1d3-756f-4097-af91-b1b101890676	2015-02-20 16:15:35.462454
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: N/A\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	61b08a00-a2c6-49a1-b576-15398c0b087c	2015-02-20 16:15:35.703005
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: N/A\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	f1867808-e4f5-4d56-ba60-4c3a1bf30515	2015-02-20 16:15:35.910059
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: N/A\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	46822daf-71b7-4260-a5b8-f08d9090ddbd	2015-02-20 16:15:36.107925
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: AML with mutated NPM1' is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development.\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	379b21e0-3c99-4955-b18b-a68730857588	2015-02-20 16:15:36.442005
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: N/A\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	d77de2d1-8331-4031-84ec-21722e4f8186	2015-02-20 16:15:36.652259
50	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL E255K\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation E255K has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	7e37c4a0-f5f5-4c16-b221-a4e9e8fc6aa0	2015-02-20 16:15:39.796759
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating mutations, the 542/545 region of the helical domain, and the 1047 region\n  of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways,\n  is the subject of an immense amount of research and development, and PI3K inhibition\n  has seen some limited success in recent clinical trials. While monotherapies seem\n  to be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	1a43d1a8-bcbf-4752-9ff0-5e389f89b66f	2015-02-20 16:15:36.909638
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: N/A\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	a492b475-baeb-4208-a84a-61769b79bfd6	2015-02-20 16:15:37.148965
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: N/A\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	2f1a6015-9b8b-4517-b427-354dfb9eaa3e	2015-02-20 16:15:37.339599
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARa\ndescription: N/A\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	d70b93e9-2ebb-40e4-9dd2-e82486f091ed	2015-02-20 16:15:37.528472
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: N/A\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	6d2a23bb-ce55-4e5a-9179-1dd6f95fc0a1	2015-02-20 16:15:37.768402
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	c30cf26f-805a-45e9-ba63-c99351816252	2015-02-20 16:15:38.036475
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: N/A\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	3f9c04b0-2e89-4fd0-81cf-6642cf5fac34	2015-02-20 16:15:38.248329
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	394226ac-f02f-4350-a938-187741bd5e53	2015-02-20 16:15:38.540489
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: N/A\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	44069199-f807-4af3-a4fc-738efa2de085	2015-02-20 16:15:38.765267
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: N/A\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	2c56fe0b-ac33-4c66-9246-b3858e8b1e0e	2015-02-20 16:15:38.952114
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: N/A\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	d78ebf65-6b3d-46d9-8fab-86eafcbbaed6	2015-02-20 16:15:39.139339
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	244ae274-4a61-458c-ae67-a1d14cae6cba	2015-02-20 16:15:39.49109
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: N/A\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	0d158f9b-e709-4eb0-9d06-1cf4b91547d1	2015-02-20 16:15:39.688118
48	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	2decef87-a474-499f-b0d7-31c3ca2cc882	2015-02-20 16:15:39.740134
49	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	fb3cf952-67f6-4c99-8f13-2549377a0484	2015-02-20 16:15:39.770689
636	44	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0d00fe45-1f38-4e1f-8c3c-70736320a355	2015-02-20 16:22:50.2097
637	45	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7de260d0-8a0a-4ae1-82bb-5e37569ca762	2015-02-20 16:22:50.240583
51	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	5e136317-6848-45d9-96a8-969b24a84cbd	2015-02-20 16:15:39.850443
52	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	aeb283ed-e0e3-408c-967a-ee654c05b700	2015-02-20 16:15:39.883404
53	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	d9d8935c-201f-4e37-9562-f655d70f5a8c	2015-02-20 16:15:39.910622
54	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	43d4f6ac-7b76-4236-94ce-fde2cb02446a	2015-02-20 16:15:39.93481
55	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	5e613d69-a968-4dcc-874f-02ee3379571c	2015-02-20 16:15:39.958491
56	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	35ee8ef8-8cbe-4a81-a657-4550c6082288	2015-02-20 16:15:39.982577
57	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	e0fbf01d-0173-4b66-8c93-fbc9128a5cfb	2015-02-20 16:15:40.013414
58	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	a4159471-0605-4d17-8c0c-1598466ace6a	2015-02-20 16:15:40.039259
59	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	204034df-884f-4c17-ac8a-a0eb554cccff	2015-02-20 16:15:40.065992
60	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	8ebfe482-d6e1-4c13-be42-14b3c543a0ff	2015-02-20 16:15:40.09558
61	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	1ca68e57-a50e-4b2c-9a85-090b12b719a1	2015-02-20 16:15:40.118713
62	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	bca2f140-0cf1-4b58-9340-1b3520b6561e	2015-02-20 16:15:40.140902
63	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	a1788f4a-949d-4c80-8b74-83a46c252fa6	2015-02-20 16:15:40.166935
64	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	66020789-da39-4661-8d4e-7539d9803b2e	2015-02-20 16:15:40.199646
65	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	c26a965c-9e9a-4c28-9ab2-1a6dca381050	2015-02-20 16:15:40.232239
66	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	881e2b6e-8fd4-4902-9204-06a51d279507	2015-02-20 16:15:40.25936
67	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	1746353f-1de4-4fa2-ad62-95b8b31969de	2015-02-20 16:15:40.289311
68	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	f1ba3d6d-5d8e-4d23-98a5-5bb5c25fa5d1	2015-02-20 16:15:40.3173
69	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	229ef23a-1a77-41c2-82d0-c14d85e70eaa	2015-02-20 16:15:40.346712
70	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	491510cd-e48b-487a-b76f-98756dcd2c03	2015-02-20 16:15:40.374991
71	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	53c51c0e-ceb3-404d-be87-66de438141ee	2015-02-20 16:15:40.413358
72	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	d74a688f-494f-4b18-beaf-3acdd2c57ecd	2015-02-20 16:15:40.439043
73	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	89d64fa5-8382-47cf-b1ca-fb98f3de5f5c	2015-02-20 16:15:40.469962
74	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	bea60755-d560-4b56-a4f7-3fe9ed0e1fdb	2015-02-20 16:15:40.494588
75	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	619ede65-e445-49d3-8aef-79031683b2ee	2015-02-20 16:15:40.529641
76	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: MUTATION\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	fbfde6e8-2bc8-4db0-9608-7f930f461658	2015-02-20 16:15:40.576104
77	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	ae8c9b09-957a-4358-a210-91634a009207	2015-02-20 16:15:40.601368
78	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	aff0d597-e6c0-453c-a055-fdf31351ddf6	2015-02-20 16:15:40.631309
89	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R678Q\ndescription: ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	60745e1c-285f-455d-9147-c0ac60680a8c	2015-02-20 16:15:40.945272
79	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	750b9ee7-f0d5-4037-8f90-925d66e9e269	2015-02-20 16:15:40.66357
80	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	c67fe2bf-4ced-46e2-8fd8-68fc6b2672a0	2015-02-20 16:15:40.688591
81	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	1740de3c-2280-403b-9de9-ad1020a1b211	2015-02-20 16:15:40.718538
82	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769H\ndescription: ERBB2 D769H was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	f6ebcfd8-352e-427f-a766-4003a06f1e13	2015-02-20 16:15:40.747784
83	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769Y\ndescription: ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	f228e1e5-e320-4f8b-b263-a7202ed51845	2015-02-20 16:15:40.772042
84	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: DEL 755-759\ndescription: ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants\n  to be functionally classified (Bose et al. 2012). This mutation was shown to be\n  an activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	bdc343f3-49b6-4f75-aa88-697c8a0e830a	2015-02-20 16:15:40.796219
85	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: G309A\ndescription: ERBB2 G309A was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	34f3f0c7-3e93-461d-9947-eaa4ea160b02	2015-02-20 16:15:40.824742
86	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755S\ndescription: ERBB2 L755S was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike\n  many of the other variants queried. This mutation was also shown to confer resistance\n  to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.\n	1	\N	\N	40c91336-d95f-499b-8970-37345ecd7802	2015-02-20 16:15:40.857382
87	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755W\ndescription: ERBB2 L755W was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	3065ddbf-bea8-41cf-8a87-1d0e8775d4af	2015-02-20 16:15:40.880809
88	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: P780INS\ndescription: ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was shown to be an activating mutation\n  in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	e2d35adb-f2bf-4d62-b845-f81e00d9bb17	2015-02-20 16:15:40.91569
141	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 3\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	bf417062-51fd-4a33-b09e-a4d3e65ed43e	2015-02-20 16:15:42.459905
142	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 4\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	88ecf9e9-b69b-40f6-97d9-eeb705cc3d05	2015-02-20 16:15:42.48198
90	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R896C\ndescription: ERBB2 R896C was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	ce85d23e-7179-44ce-849c-cd33858214ee	2015-02-20 16:15:40.972555
91	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V777L\ndescription: ERBB2 V777L was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	42b18200-f7a1-452a-b3ef-649252665fe1	2015-02-20 16:15:40.998074
92	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V842I\ndescription: ERBB2 V842I was one of the first ERBB2 variants to be functionally classified\n  (Bose et al. 2012). This mutation was shown to be an activating mutation in an in\n  vitro assay. In the same paper, this mutation (along with other ERBB2 activating\n  mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to\n  the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating\n  mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the\n  topic of current clinical trials and research.\n	1	\N	\N	6aaf0beb-6670-40b0-aa63-bfa25484b546	2015-02-20 16:15:41.027473
93	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	373c2f3d-f832-435c-8d17-5ef57d44716c	2015-02-20 16:15:41.057597
94	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	018522c0-6829-4c4a-8cc5-64d1aa216a56	2015-02-20 16:15:41.084585
95	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	10b85cb0-ff80-40e1-a08d-493499ae2ca0	2015-02-20 16:15:41.112178
96	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	5c45faf8-85ac-4dc8-8a00-50d9f6642656	2015-02-20 16:15:41.139738
97	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	5cc6bde2-4cdf-417c-a631-5398ab7de436	2015-02-20 16:15:41.164113
98	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-AFF3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	9660206c-aecc-434c-a9f1-6cce3521d2fa	2015-02-20 16:15:41.196842
99	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-BICC1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	f6fa7837-a2aa-4e11-95b0-78827882ea0f	2015-02-20 16:15:41.231333
100	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CASP7\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	ec38d9f0-d5c9-41bf-9490-aeec6881f056	2015-02-20 16:15:41.255588
143	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	22e936a0-c3cf-46ea-a994-943c10c67740	2015-02-20 16:15:42.503209
638	46	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3e3ca2a0-f006-4de6-a41c-939eb2f9240d	2015-02-20 16:22:50.263116
101	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CCDC6\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	c7fbef2c-b71c-4966-9a24-2d110c308456	2015-02-20 16:15:41.299579
102	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-KIAA1967\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	ec16e8b6-0937-4d8c-b73f-ca697c3fc830	2015-02-20 16:15:41.329069
103	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	f2a2ce09-4ef3-4bff-9458-35b712095342	2015-02-20 16:15:41.359478
104	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-OFD1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	82ac62ef-195c-45e8-b23f-2610613b20da	2015-02-20 16:15:41.384602
105	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-SLC45A3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	a343b324-1749-40b4-a50c-9ffa8a8d19a3	2015-02-20 16:15:41.420433
106	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	05f2a811-212a-4ec7-8f12-26a2b5dbc26e	2015-02-20 16:15:41.445308
107	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: ITD\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	6f80a4cd-0f4f-4b27-b546-35f0c21822b2	2015-02-20 16:15:41.471653
108	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: TKD\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	1864557a-7917-41af-9030-7ce0072375a5	2015-02-20 16:15:41.497375
109	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	3e335d91-73d5-4a83-8b08-a8e72fbc17e3	2015-02-20 16:15:41.527049
110	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	660c619a-109d-4947-8991-11ee7af21760	2015-02-20 16:15:41.551789
111	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	7dd67b3b-de31-49f3-a905-7cd7b4f72ea6	2015-02-20 16:15:41.581947
144	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	2cef6dfc-607e-47be-9de5-35b302b27dc4	2015-02-20 16:15:42.529409
639	47	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a2aa6d9f-271a-473b-8226-62a0276797ed	2015-02-20 16:22:50.285978
112	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	c3e4cee0-83b4-4436-bc94-be3176bdd6a5	2015-02-20 16:15:41.618803
113	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	ea0fcfe7-6736-4e90-a565-3782f048b3cb	2015-02-20 16:15:41.65328
114	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	02cd5bdf-1db1-499c-bd47-0d1ef7da0184	2015-02-20 16:15:41.681008
115	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	3ef74fc7-e27d-436c-a8db-d6f20930195f	2015-02-20 16:15:41.707534
116	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	744131b0-fa40-4fa7-973d-5bb892f4edbf	2015-02-20 16:15:41.742719
117	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	45efabae-cc04-48ae-b57e-e4dd33f0ef12	2015-02-20 16:15:41.771448
118	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 11\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	3e917696-7f0a-4874-b45b-dd3a42e5100f	2015-02-20 16:15:41.800677
119	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 13\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	8fa393f9-13ea-4a4b-8462-9c01a7c116b2	2015-02-20 16:15:41.826862
120	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 14\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	bb9119bf-ecc6-4e41-a574-a5f1c10b38a7	2015-02-20 16:15:41.859715
121	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 17\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop of the protein. In cell lines, mutations within this domain have been shown\n  to be sensitive to imatinib. However, in double KIT mutants in which the exon 17\n  mutation is a secondary mutation, cell lines have shown resistance to both imatinib\n  and sunitinib.\n	1	\N	\N	968ba960-d4bd-446a-ba4c-1c18592dae50	2015-02-20 16:15:41.887105
122	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 9\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	d7858443-52cc-4db5-860b-af004fe95926	2015-02-20 16:15:41.915402
123	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	35ad9634-7c9d-4a07-8999-900b788b2a85	2015-02-20 16:15:41.949901
124	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	6b7de469-02e6-47db-9b0f-bef8240bafe2	2015-02-20 16:15:41.974608
125	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 1\ndescription: A study by Livre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	3912d49b-6cb9-41f6-a3d8-97a7f21510f9	2015-02-20 16:15:42.025176
126	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 2\ndescription: In a study by Pan et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	4037f25e-cea7-4b94-88dc-b242d13f8e1d	2015-02-20 16:15:42.049444
640	48	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b3b8dd6e-9040-4a8d-ab58-dc6338d0d598	2015-02-20 16:22:50.306018
641	49	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	03ddfae9-f6ab-4613-90c4-af8e4979349d	2015-02-20 16:22:50.32647
127	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	f2312b94-32da-4b74-b0a5-7f57c9df126b	2015-02-20 16:15:42.074409
128	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	03829475-2d2f-4447-8e66-900bf0bec05c	2015-02-20 16:15:42.104172
129	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	1cce7aa2-7f81-491f-bd46-13e812a5f703	2015-02-20 16:15:42.135493
130	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	172bff3a-938f-415e-96db-fd773a6f1ad6	2015-02-20 16:15:42.161464
131	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	ea4f5669-08f3-465c-b00f-f8b94575a1f9	2015-02-20 16:15:42.185067
132	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	28236203-657d-49c7-b9ea-745724c7337e	2015-02-20 16:15:42.212176
133	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	4cb8ad96-a898-4b04-88e2-e65a7959ad11	2015-02-20 16:15:42.238077
134	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	59c7848b-d492-40f6-b9c3-e504cc38c696	2015-02-20 16:15:42.260943
135	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	f4f8dbfe-a439-4d9a-aec8-4e3bfc154023	2015-02-20 16:15:42.285687
136	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	37819582-7867-4330-a643-5dda5cc2587a	2015-02-20 16:15:42.309174
137	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: EXON 12\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	e5f1744c-018c-48d1-8ec5-827dd6b91fd8	2015-02-20 16:15:42.33497
138	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	f9e0ba9c-cbfe-4275-85d3-8da5c0ad570c	2015-02-20 16:15:42.370643
139	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 1\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	24ce45a7-05a2-420a-b18b-e187f7b2893b	2015-02-20 16:15:42.401151
140	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 2\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	129ca116-2c87-483b-a184-9bcfe2a58c50	2015-02-20 16:15:42.431402
145	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	78c09838-e4c5-4166-8434-169b70635ec5	2015-02-20 16:15:42.558955
146	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	7c44c0f3-b041-4238-9f5f-32fb1c7a8718	2015-02-20 16:15:42.586845
147	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	d412ecc8-97df-4924-b690-53e633901e48	2015-02-20 16:15:42.619741
148	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	8fba5cca-54ee-40c0-b9ff-37b213ef6f5a	2015-02-20 16:15:42.664612
149	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	9b617136-468d-4f79-af5d-7d8bf590e080	2015-02-20 16:15:42.692206
150	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	16997272-4794-4343-9e3d-04b94953a88b	2015-02-20 16:15:42.72291
151	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	10a93a38-9372-4e30-8e79-4b03a5e83631	2015-02-20 16:15:42.752031
152	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	ca8de458-e68c-4d34-b50a-9875d079a083	2015-02-20 16:15:42.779759
153	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	1a8fd987-6405-47bb-adca-0574df74b95f	2015-02-20 16:15:42.804092
154	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	95c0e9f2-52d8-4e15-b78b-a062a8069681	2015-02-20 16:15:42.827838
155	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 20\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	2fc67372-f2a1-49a7-9140-5e0a086e5ae7	2015-02-20 16:15:42.854017
156	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 9\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	dc4ce049-998b-4d17-87a3-513a80de759b	2015-02-20 16:15:42.882501
157	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	bd4ce4c4-83aa-4f94-a2a7-4575403e1a30	2015-02-20 16:15:42.908022
159	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	7143f162-4cf8-4e03-9f07-47b2475751fa	2015-02-20 16:15:42.961566
160	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	dfb0fbde-a771-4356-b291-f9b2dcab0b7c	2015-02-20 16:15:42.986519
161	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	df0a745b-cefe-4427-af47-c2625a741db6	2015-02-20 16:15:43.011494
162	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	e988f385-0924-4d47-9c42-3c260cf7d169	2015-02-20 16:15:43.038419
163	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	3c53daa4-4a8f-4064-8c1c-445b94ad0c8b	2015-02-20 16:15:43.068346
164	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	f168c0aa-7680-4294-9dc1-6d3dd00859bb	2015-02-20 16:15:43.096086
165	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	6bfd9e59-3a58-4b38-b073-8f4aab7c21a6	2015-02-20 16:15:43.123347
166	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	be24c79d-ae8b-4785-9a40-87c532ad8895	2015-02-20 16:15:43.15257
167	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	eb018d74-54c9-4e4a-8d92-d5f66f082f73	2015-02-20 16:15:43.178621
168	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	c1deb569-2c75-48f2-9469-bc4af7b58d40	2015-02-20 16:15:43.204534
169	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	3d38165c-fd1e-457f-929f-bc8905d2f4c8	2015-02-20 16:15:43.232246
170	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	456e2a2c-8faf-4133-8a7c-9d548590374e	2015-02-20 16:15:43.256872
171	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	6570522e-1375-4aa5-8440-b124b25dc6be	2015-02-20 16:15:43.300783
233	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 4\nsource_id: 67\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	686f4197-a092-4f48-89fc-17137251180d	2015-02-20 16:15:47.081037
172	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	aed50686-bc14-4237-98d8-ba17c0f0c5cc	2015-02-20 16:15:43.330055
173	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	c6e9bd10-81f3-4e83-af9d-060908a07454	2015-02-20 16:15:43.359081
174	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	83f5bde2-5f2a-4842-93e9-392572f63315	2015-02-20 16:15:43.382679
175	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	33efe85a-dd9b-4c3b-a89b-364821f6eab5	2015-02-20 16:15:43.410939
176	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	3c60cd52-c838-40dd-989e-1dabf1af67f0	2015-02-20 16:15:43.434672
177	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations\n  on overall survival in MDS has been debated, however, patients with U2AF1 mutations\n  were shown to be at an increased risk of transformation to secondary AML. The presence\n  of this mutation was not associated with a specific prognostic outcome in AML when\n  compared to U2AF1 wildtype patients.\n	1	\N	\N	d95658f4-eb38-4a46-9a3f-c74a88aff9e9	2015-02-20 16:15:43.461489
178	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1\n  mutations on overall survival in MDS has been debated, however, patients with U2AF1\n  mutations were shown to be at an increased risk of transformation to secondary AML.\n  The presence of this mutation was not associated with a specific prognostic outcome\n  in AML when compared to U2AF1 wildtype patients.\n	1	\N	\N	20d61ef8-d52c-47b5-95ae-962ddd6e1d0d	2015-02-20 16:15:43.486686
179	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 7\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	95c625a6-c466-4987-b1a8-f921c0b74b89	2015-02-20 16:15:43.511303
180	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 9\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	8caba694-2931-4c55-acc8-6bbe2f143de6	2015-02-20 16:15:43.540059
181	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL fusions have been found to contribute to imatinib resistance in AML\n  cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 41\nvariant_id: 1\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	756c00c7-5da0-4719-9dd9-f30e81fabd50	2015-02-20 16:15:43.693508
182	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide\n  results in high rates of complete remission in patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 1\nsource_id: 41\nvariant_id: 1\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	46b315c5-485d-41db-8718-6ae861a079be	2015-02-20 16:15:43.756087
183	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4ad7c75b-7b88-4b20-8cf0-cd8782202735	2015-02-20 16:15:43.820656
184	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 2\nsource_id: 43\nvariant_id: 1\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	86a4a718-8d99-4d98-a0fa-862e4868dfb2	2015-02-20 16:15:43.890548
185	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The clinical use of imatinib in patients with BCR-ABL fusion has resulted in\n  drastic sensitivity to the drug.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a37dda22-cdb2-42dc-b418-264bea117bf8	2015-02-20 16:15:43.969942
607	15	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	188f01ec-c899-4043-aaa2-b1cf575f9b84	2015-02-20 16:22:49.564572
186	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	922c8bfc-825f-4716-8bcc-5c5f23907c69	2015-02-20 16:15:44.037238
187	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has been effective in\n  patients resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 5\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b6e41bc6-36a1-4769-b6fb-483a287e5c08	2015-02-20 16:15:44.088598
188	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis,\n  response rate, overall survival, and patient outcome in CML patients compared to\n  previous therapeutic regimens."\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 1\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	677a771f-7e93-4ce6-8772-21d315c52ceb	2015-02-20 16:15:44.151991
189	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 2\nsource_id: 44\nvariant_id: 3\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b0a57f28-9a10-4780-b37b-85499d5f25d7	2015-02-20 16:15:44.215181
190	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have\n  shown to be resistant to imatinib treatment.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 2\nsource_id: 42\nvariant_id: 2\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	25822be3-622c-4a98-a4bd-eea68ce8b820	2015-02-20 16:15:44.307837
191	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations\n  were shown to be resistant to imatinib and exhibited increased kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 2\nsource_id: 44\nvariant_id: 2\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	260e2f6e-2187-4314-ab63-0d16495a2a23	2015-02-20 16:15:44.377548
192	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 3\nsource_id: 45\nvariant_id: 4\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	92dfbf9b-1b17-4ea4-9912-838ca72147d6	2015-02-20 16:15:44.442041
193	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 6\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4d68a60f-6158-41df-afcb-8f9c308a6e0d	2015-02-20 16:15:44.528692
194	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective in acheiving stable disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 5\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dc6f07fa-505f-4072-a0d6-a7ddd2ebd6a0	2015-02-20 16:15:44.60587
195	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 5\nsource_id: 47\nvariant_id: 8\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a7590860-52a8-463a-b83e-4a9ba09f08c1	2015-02-20 16:15:44.672308
196	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 6\nsource_id: 48\nvariant_id: 8\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	efc4976d-9074-432b-8233-ca33a203129f	2015-02-20 16:15:44.741828
197	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 6\nsource_id: 49\nvariant_id: 8\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c3b417db-fdaa-415f-8ca9-a2d713f4032c	2015-02-20 16:15:44.811338
198	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 6\nsource_id: 50\nvariant_id: 8\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6a162baa-b05c-4230-a6f1-cab0b3e40cbc	2015-02-20 16:15:44.872882
199	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 8\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	de63310e-0b40-4e99-abf7-cab2e126b0a8	2015-02-20 16:15:44.932131
200	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 7\ndisease_id: 4\nsource_id: 47\nvariant_id: 8\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d1451d08-682f-4e31-a75e-5dd422b45803	2015-02-20 16:15:44.990945
201	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 8\ndisease_id: 4\nsource_id: 48\nvariant_id: 7\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4168151c-8514-4172-b828-d536b6153cef	2015-02-20 16:15:45.079964
202	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 4\nsource_id: 46\nvariant_id: 7\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	00c9e215-926c-4759-8db4-b0e3ffcd83fa	2015-02-20 16:15:45.136844
203	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells\n  with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 9\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8d77485a-1672-4b52-a34a-e774e4bf0653	2015-02-20 16:15:45.206937
204	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 9\ndisease_id: 6\nsource_id: 51\nvariant_id: 9\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	884f8ec6-4803-4b51-b590-040c5b5a992f	2015-02-20 16:15:45.273661
205	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d70f0bbd-8fe8-4231-a0cf-323a0d2dc968	2015-02-20 16:15:45.335573
206	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 11\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d1e5c4d7-c8cd-463c-8480-5f54f7dd2cac	2015-02-20 16:15:45.390795
207	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells expressing ARAF S214C in vitro\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 4\nsource_id: 52\nvariant_id: 10\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b856828f-df10-4cfd-87c5-08266e7fcb9c	2015-02-20 16:15:45.449503
208	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 7\nsource_id: 53\nvariant_id: 17\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f1e57d0f-01bf-4381-9c85-448aba285e9d	2015-02-20 16:15:45.516928
209	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in non-chronic sun damaged\n  skin and with superficial spreading melanoma\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 54\nvariant_id: 17\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5fb7c4c9-20dd-419e-9f9a-d5e30a8358a6	2015-02-20 16:15:45.581597
210	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 55\nvariant_id: 17\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	28a54079-1eae-4d2d-9d8a-6682b82bd4c7	2015-02-20 16:15:45.682353
211	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 8\nsource_id: 55\nvariant_id: 17\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a68154a1-1b39-4032-ad54-0b81a3ffda9a	2015-02-20 16:15:45.734692
212	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 11\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1d485311-03d3-483f-8bba-2ee578d15a01	2015-02-20 16:15:45.79695
213	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8b934c66-0f93-45c4-ab33-73e181992700	2015-02-20 16:15:45.867183
214	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 12\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c1ad0ec0-1f7b-4b23-a12a-9e0e3f5405b0	2015-02-20 16:15:45.917404
215	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 9\nsource_id: 57\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	47b0fed0-6f41-4b14-91a3-58f86328e0de	2015-02-20 16:15:45.965881
216	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 15\ndisease_id: 9\nsource_id: 58\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	af23da7a-a344-49e9-a112-374101ac483a	2015-02-20 16:15:46.029927
217	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 9\nsource_id: 58\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0cb4816a-77fc-4d0d-a66a-97bcb52e3ba6	2015-02-20 16:15:46.084277
218	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 16\ndisease_id: 9\nsource_id: 59\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8ecd0693-062c-43f7-9f44-4e5e1677f5ae	2015-02-20 16:15:46.173224
219	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 17\ndisease_id: 9\nsource_id: 59\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c5527d43-e5cc-4445-ba32-802125651c06	2015-02-20 16:15:46.222889
220	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 18\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7113373c-fe11-4642-b27b-eb2921f29ece	2015-02-20 16:15:46.291958
221	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 19\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0305d373-0438-4e0e-9621-c474de5759bc	2015-02-20 16:15:46.341943
222	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 20\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9b246219-c067-479c-94d4-2aa41060df9e	2015-02-20 16:15:46.395651
223	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 19\ndisease_id: 7\nsource_id: 60\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d92e7f88-3e5c-4d55-89a0-f1836f0aa582	2015-02-20 16:15:46.454024
224	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as a marker of poor prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 61\nvariant_id: 12\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	01702021-9952-486e-aca3-9dfbcfe1b5a0	2015-02-20 16:15:46.511814
225	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates as well as shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  BRAF\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3b465cad-8e50-497a-8f0b-f5756e1c9e38	2015-02-20 16:15:46.578106
226	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 8\nsource_id: 63\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3766378d-6c4c-4e45-9550-244268b16115	2015-02-20 16:15:46.641679
227	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 8\nsource_id: 63\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	87f5ae19-e999-4667-aa52-011680d0a492	2015-02-20 16:15:46.721136
228	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 21\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f3bae09d-2b40-4cd1-93e4-c4812d51abee	2015-02-20 16:15:46.784921
229	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 22\ndisease_id: 8\nsource_id: 64\nvariant_id: 12\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f950af9f-84eb-49f3-a33f-c554b2474662	2015-02-20 16:15:46.837108
230	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 65\nvariant_id: 12\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e6175dc1-7bf8-4518-a299-d33badb9568d	2015-02-20 16:15:46.899868
231	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 8\nsource_id: 65\nvariant_id: 12\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	773cc4a3-4ac2-4f15-b022-a9606382ea65	2015-02-20 16:15:46.960842
232	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter disease-free and overall Survival in a\n  Spanish cohort of melanoma patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 66\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4ffa2c4a-2c66-441c-afbe-d9447d9e7622	2015-02-20 16:15:47.023048
608	16	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f62ac22d-03e3-4b83-bee6-2d19188a5241	2015-02-20 16:22:49.583617
234	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 68\nvariant_id: 12\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ce853dbe-6ef6-454e-a90e-93f3957d4be0	2015-02-20 16:15:47.143488
235	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with papillary\n  thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 69\nvariant_id: 12\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a6b30bd2-80b5-4ce4-bef3-00ae93a46521	2015-02-20 16:15:47.226971
236	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of papillary\n  thyroid cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	15d76015-0c8e-4caf-9ba5-c3300fa9766b	2015-02-20 16:15:47.285304
237	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 71\nvariant_id: 12\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	544a7680-4c79-4864-8c7e-41e92ce799a4	2015-02-20 16:15:47.34825
238	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1ef2ade8-8beb-40e8-bf82-8fb0ad98781f	2015-02-20 16:15:47.410007
239	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 72\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	16a2f683-ca55-4ebd-a2ef-a4e9f5dea6a6	2015-02-20 16:15:47.473233
240	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 10\nsource_id: 70\nvariant_id: 12\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d601e4b0-3573-427e-b9b8-fd78cb7c3c30	2015-02-20 16:15:47.529923
241	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK\n  inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification\n  of the BRAF V600E gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 23\ndisease_id: 7\nsource_id: 73\nvariant_id: 14\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	37225574-c512-4e5e-841f-29e8c6fbb71b	2015-02-20 16:15:47.593911
242	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 74\nvariant_id: 13\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	46e1d466-4c3a-47a0-bbbd-d31c00561024	2015-02-20 16:15:47.652384
243	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 15\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	99e48d2e-2dc3-4867-83f6-3b947507b13c	2015-02-20 16:15:47.742386
244	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 8\nsource_id: 56\nvariant_id: 16\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c9f2eef1-2d5d-4c57-b074-e250ca33c0e7	2015-02-20 16:15:47.799823
245	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 18\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	63491f54-d8e1-4632-9194-aa12d0ac0697	2015-02-20 16:15:47.869066
246	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 19\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1d447784-4309-4bb3-9ad7-2ecb7eab773f	2015-02-20 16:15:47.928708
247	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 75\nvariant_id: 19\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0dcc05e9-ca18-4d2b-b4e2-ee097d4811c7	2015-02-20 16:15:47.989516
248	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 76\nvariant_id: 20\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bf738d0d-a566-452d-a523-500a31eaf129	2015-02-20 16:15:48.047401
249	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 76\nvariant_id: 20\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	05ee70fc-2bdc-47b8-a441-33f5224109f9	2015-02-20 16:15:48.102054
250	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 12\nsource_id: 77\nvariant_id: 20\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bb549a70-71b0-4f9e-96da-0be75ce321fa	2015-02-20 16:15:48.159274
609	17	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6b84f1bc-ee47-46aa-bfb2-4d03c4e8b2d6	2015-02-20 16:22:49.604509
251	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 13\nsource_id: 78\nvariant_id: 20\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e2bd0e57-8678-44e8-b46a-201a36afcc6c	2015-02-20 16:15:48.242558
252	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	60d00cac-a72a-4bec-9929-4cfcb69917f3	2015-02-20 16:15:48.303594
253	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2ce800a6-6d32-4f22-9067-546da594b700	2015-02-20 16:15:48.371004
254	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1de24a33-6b95-4ecf-b093-cdbcc476e1a2	2015-02-20 16:15:48.434097
255	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 79\nvariant_id: 21\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e73e2c63-19e6-4b46-aa70-3d432c9bb0ac	2015-02-20 16:15:48.492551
256	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 14\nsource_id: 80\nvariant_id: 22\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f3d20424-9c8e-47df-a249-d07ad30a9ac0	2015-02-20 16:15:48.555034
257	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 24\ndisease_id: 15\nsource_id: 81\nvariant_id: 23\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	80cbc1d8-66f5-4402-9e7a-744bea4465e0	2015-02-20 16:15:48.616868
258	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 24\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3217fe42-50dd-4b5c-b374-5cab86612aa0	2015-02-20 16:15:48.700398
259	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 16\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fb641334-a772-423e-a0fd-11a45d28e28e	2015-02-20 16:15:48.771163
260	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 16\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b94cd23a-7be2-4608-a17c-08615d40db5b	2015-02-20 16:15:48.828902
261	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 83\nvariant_id: 24\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d2c234ea-3df5-45d3-ba64-58919ab6ae38	2015-02-20 16:15:48.891922
262	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 18\nsource_id: 84\nvariant_id: 25\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1d52f88c-92c8-436f-8c3f-31f30d3ebf83	2015-02-20 16:15:48.949486
263	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 18\nsource_id: 85\nvariant_id: 25\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7b86f7b1-b7db-4dd1-a897-5c050045f121	2015-02-20 16:15:49.016296
264	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In mouse xenograft studies of PIK3CA mutant breast cancers, the combination\n  of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading\n  to tumor regressions.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 25\ndisease_id: 18\nsource_id: 84\nvariant_id: 26\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a3835acd-7704-4bc5-b15c-1fdb12514e80	2015-02-20 16:15:49.089594
265	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 18\nsource_id: 85\nvariant_id: 26\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e6055760-0966-4d4f-b4da-7bdcf219e001	2015-02-20 16:15:49.15543
266	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 27\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c527d0d5-ddab-4aa7-8fd0-d2a26a7bc38e	2015-02-20 16:15:49.23746
610	18	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d62545c5-faea-4a4a-aea3-7aec341e55b8	2015-02-20 16:22:49.62448
611	19	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	4bc193b4-4b3e-4f1a-9776-140370e29405	2015-02-20 16:22:49.645122
267	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 86\nvariant_id: 27\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d508d77b-ded9-48a2-82cc-f3ea751db2e3	2015-02-20 16:15:49.291991
268	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 87\nvariant_id: 29\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	68192400-51c0-495f-84a5-2f5c348141e0	2015-02-20 16:15:49.357892
269	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 88\nvariant_id: 29\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	64591248-6541-4504-bd98-2f58055a602b	2015-02-20 16:15:49.415755
270	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 87\nvariant_id: 29\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d8b7eed3-a064-44d3-972a-bd6b257ef35e	2015-02-20 16:15:49.472508
271	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	23382df7-251e-4ce4-baaa-f60ec01e34a9	2015-02-20 16:15:49.533417
272	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7e192f78-24f3-4242-8c82-d8d04552d939	2015-02-20 16:15:49.594398
273	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 89\nvariant_id: 28\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	130e5b4f-eb9f-4965-b27d-3d4a69fd1c83	2015-02-20 16:15:49.675506
274	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 90\nvariant_id: 30\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	443babf4-b6b4-45a5-a47a-052278730812	2015-02-20 16:15:49.744111
275	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 19\nsource_id: 90\nvariant_id: 30\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	96a8eb53-fe05-42eb-b473-2e19ce3562f8	2015-02-20 16:15:49.79406
276	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 20\nsource_id: 91\nvariant_id: 31\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a801b310-5878-4557-a1a7-4a3acd373661	2015-02-20 16:15:49.861951
277	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	62b5ba91-1267-4c4b-ba1c-f94a8b122305	2015-02-20 16:15:49.9266
278	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	24ef10a3-03c5-4dba-944e-c660ec875d20	2015-02-20 16:15:49.989877
279	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 28\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	41d29630-3f56-4d00-94b4-3867b1b96935	2015-02-20 16:15:50.0492
280	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3e6468b1-0cac-4ab3-81f9-a626ffcc913c	2015-02-20 16:15:50.109115
281	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 93\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	28d1dd81-02ee-4fbe-a0c9-c8e6959f7047	2015-02-20 16:15:50.185617
282	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f1a57f79-a486-43d1-b56f-41e64e9f1e5b	2015-02-20 16:15:50.248901
283	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 95\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2195dc0e-e0e1-49d5-8a3f-f48dbee728a4	2015-02-20 16:15:50.318547
284	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	776fb2cd-4cd2-4ae0-baa9-10c19e68c6db	2015-02-20 16:15:50.381793
285	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	60b9e2eb-6b07-4b08-82f7-d3df3470aaf7	2015-02-20 16:15:50.439917
286	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a42e4a5a-4e34-4cbc-8ce6-1b72d051a7ce	2015-02-20 16:15:50.497563
287	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1a106f5f-d78e-4030-ba00-09e2c7a3d857	2015-02-20 16:15:50.561608
288	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including\n  R882) were associated with worse overall survival compared to those without DNMT3A\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2c5cc64f-bba0-49c5-a225-225bbe4b7951	2015-02-20 16:15:50.645531
289	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 95\nvariant_id: 32\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a11c2a8c-6b64-4eaf-b93d-f696aa07ea88	2015-02-20 16:15:50.709678
290	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1085224f-5916-4cc2-9e8e-a55bfa217ab4	2015-02-20 16:15:50.765531
291	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	84b11e69-958f-4c29-a3a5-71395e4a8047	2015-02-20 16:15:50.832762
292	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a51c26e4-a3e0-4841-b731-d348d93b58f2	2015-02-20 16:15:50.886713
293	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable, cytogenetically normal AMLs\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	19a7d56d-10ff-4014-aa23-1bbc2e0f457e	2015-02-20 16:15:50.950399
294	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	261785e2-4cb9-4d60-a8d5-05c0e394f965	2015-02-20 16:15:51.011483
295	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cef5c36d-c411-430f-a23a-bcb444b93cab	2015-02-20 16:15:51.093499
296	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7784e9e4-86a5-4111-a7be-e3a93d9e8836	2015-02-20 16:15:51.163704
297	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with intermediate\n  risk cytogenetics (including normal karyotype)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	19f4b53e-1b65-45bc-a428-8913b50eb59e	2015-02-20 16:15:51.222926
298	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d492d3ca-22bf-494c-ab9a-57faf949e027	2015-02-20 16:15:51.276498
299	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	63a1a4b4-232f-49b2-bf73-5597372d87b4	2015-02-20 16:15:51.342631
300	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 96\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4edaec56-7c72-4514-b37c-267576196d87	2015-02-20 16:15:51.401675
301	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	43fe5dd8-9450-4897-9152-81ca65a7d450	2015-02-20 16:15:51.464464
302	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 97\nvariant_id: 32\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4bcae638-0e69-4764-8935-1b53ffdd618e	2015-02-20 16:15:51.602762
303	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a2493eb1-cb1d-45cb-9949-b140d759847d	2015-02-20 16:15:51.668292
304	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1384eade-9743-456e-aa39-6131e546f81a	2015-02-20 16:15:51.725651
305	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 32\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2ef831a7-cc9b-46a2-a3a0-197ad35b5c05	2015-02-20 16:15:51.782652
306	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 98\nvariant_id: 33\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bde39b1d-eb9b-4d9f-bfbf-e6c78ceb1cdd	2015-02-20 16:15:51.85279
307	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 98\nvariant_id: 33\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	94cdd932-dc25-43c3-8104-9eaef5e8883e	2015-02-20 16:15:51.910609
308	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence\n  of L858R mutation is prognostic for better progression free survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 99\nvariant_id: 33\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	42395253-acf5-4c2b-94a1-9e6047533c8d	2015-02-20 16:15:51.980765
309	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Geftinib added to pemetrexed and carboplatin showed greater effectiveness than\n  premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 100\nvariant_id: 33\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6077f61b-38f4-4098-b2ae-572202c20f2b	2015-02-20 16:15:52.049432
310	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 101\nvariant_id: 33\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e03f7b38-5cd8-41ef-bd93-345ee0016ddc	2015-02-20 16:15:52.11637
311	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The T790M mutation in EGFR has been shown to confer resistance to the tyrosine\n  kinase inhibitor erlotinib, and patients harboring this mutation that are placed\n  on the drug are likely to relapse.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 31\ndisease_id: 21\nsource_id: 102\nvariant_id: 34\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	aa109013-f8cb-4abb-8492-ed6d99e300a9	2015-02-20 16:15:52.213297
312	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 103\nvariant_id: 34\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d216e212-c9af-48bc-b4cf-070463c898fa	2015-02-20 16:15:52.280017
313	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 104\nvariant_id: 34\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	791dfa6a-5b9f-49e1-926d-269d07746437	2015-02-20 16:15:52.347494
612	20	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3c5fe15c-2f9b-4948-8361-1952134a0b8c	2015-02-20 16:22:49.667035
314	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting\n  in stable disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 32\ndisease_id: 4\nsource_id: 104\nvariant_id: 34\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	71cd3485-8a6d-4062-acfb-a7da3ffc05c9	2015-02-20 16:15:52.399862
315	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when an L858R mutation is also present.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 33\ndisease_id: 4\nsource_id: 105\nvariant_id: 34\nrating: 1\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9a165b7d-9f00-4807-b097-4636410ee1b1	2015-02-20 16:15:52.467329
316	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 106\nvariant_id: 34\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	83c34f34-6af7-4b75-85f4-5314add18dee	2015-02-20 16:15:52.530948
317	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutations, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 107\nvariant_id: 34\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	65f1552b-b648-4c78-a3dc-2919ef6c9675	2015-02-20 16:15:52.592464
318	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 35\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cbffe8bb-4a34-44eb-8fef-a172dc08322b	2015-02-20 16:15:52.650814
319	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 36\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c3239786-b88a-49e4-b4ef-cf7510e20f67	2015-02-20 16:15:52.737573
320	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2,\n  cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive\n  to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 37\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ffbfadb0-9366-4fcc-ac9a-57ff8cd74a82	2015-02-20 16:15:52.808159
321	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 38\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	aeb81063-d422-4a9d-a385-8e48af21f998	2015-02-20 16:15:52.861767
322	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally\n  transduced with wildtype or mutant ERBB2.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 39\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	15b6c79d-195f-4f76-a76c-a8e9c632fa5b	2015-02-20 16:15:52.924892
323	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell\n  lines retrovirally transduced with wildtype or mutant ERBB2.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 35\ndisease_id: 11\nsource_id: 108\nvariant_id: 39\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	33140fb7-7584-40d4-bfa2-1673fa0dbd97	2015-02-20 16:15:52.984704
324	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive\n  to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 40\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	316955ef-c632-4a91-86b4-dc15b538b799	2015-02-20 16:15:53.043208
325	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the in-frame insertion of a single amino acid at position\n  780 was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 41\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	53d85aef-5dda-42a6-9024-b385f2087772	2015-02-20 16:15:53.098636
326	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 42\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	825293d3-e6ac-432c-bc2f-f97e016c9a3f	2015-02-20 16:15:53.161012
327	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive\n  to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony\n  forming assay.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 36\ndisease_id: 11\nsource_id: 108\nvariant_id: 42\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7534a76d-8aae-45dc-ab96-3f4d564d7db7	2015-02-20 16:15:53.244434
328	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 43\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ea55844a-37a8-4e20-88b9-3a5738d9acde	2015-02-20 16:15:53.315793
329	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 44\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5663d90e-9d80-4d76-9165-c0a5307b9d31	2015-02-20 16:15:53.37813
613	21	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	79a283be-7a58-4735-a0ec-53bc69333572	2015-02-20 16:22:49.688214
614	22	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e2c3f8ab-cb3c-49f7-ae66-15f4a49c7532	2015-02-20 16:22:49.709499
330	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 34\ndisease_id: 11\nsource_id: 108\nvariant_id: 45\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c99004dd-1b7c-47f5-9eb3-5d52ce9976a8	2015-02-20 16:15:53.447135
331	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 46\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d532e95e-913b-4d82-9380-65d33cc3d2a5	2015-02-20 16:15:53.515655
332	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The L536Q ligand binding domain mutation have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 11\nsource_id: 109\nvariant_id: 46\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c548d4ee-119d-4e1c-9dc5-a83cfd6c4a96	2015-02-20 16:15:53.564761
333	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 46\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	dfbb5387-f970-4123-8b8b-92490dad386c	2015-02-20 16:15:53.636348
334	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 47\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	57697015-3de6-4df9-be96-c77087b6f9d0	2015-02-20 16:15:53.700185
335	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The N538G ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 11\nsource_id: 109\nvariant_id: 47\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c2c59f97-5797-4f3d-8f2b-cd1cbe59cda4	2015-02-20 16:15:53.776183
336	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 47\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d2c116ef-c59e-472b-9cc0-a2695fa15ea8	2015-02-20 16:15:53.842826
337	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 48\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2cbfecdc-2d7b-412e-81e6-110c2deb1306	2015-02-20 16:15:53.911483
338	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537C ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 11\nsource_id: 109\nvariant_id: 48\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9547ab5e-f64e-45ae-a236-6769676de2a2	2015-02-20 16:15:53.959161
339	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 48\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4097f0cb-23bc-45f1-b71a-78efd0211896	2015-02-20 16:15:54.021971
340	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 49\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	28443b8f-03ed-46dc-81ff-ade0a5b5a081	2015-02-20 16:15:54.088965
341	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537N ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 11\nsource_id: 109\nvariant_id: 49\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8f07cb72-8354-410e-b70e-b519b5c696f8	2015-02-20 16:15:54.137939
342	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 49\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6a600ba6-000a-442d-b09a-a10df78f7d01	2015-02-20 16:15:54.19884
343	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 37\ndisease_id: 11\nsource_id: 109\nvariant_id: 50\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b1b70b51-98b8-4e6d-af47-a213627be2ef	2015-02-20 16:15:54.295752
344	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The Y537S ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 38\ndisease_id: 11\nsource_id: 109\nvariant_id: 50\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9d49938a-afd2-4564-a50b-c06d68137cfc	2015-02-20 16:15:54.343986
345	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of\n  ESR1 have shown resistance to hormone therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 11\nsource_id: 110\nvariant_id: 50\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bf0b3cda-230b-46d8-9b52-e64edce14503	2015-02-20 16:15:54.408419
346	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in necrosis and shrinkage of the primary and metastatic sites\n  and overall stable disease. However, this response did not meet criteria for RECIST\n  partial response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 22\nsource_id: 111\nvariant_id: 56\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9e90aeb0-260b-4f37-8b45-159ddc8c2b14	2015-02-20 16:15:54.477941
347	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 40\ndisease_id: 22\nsource_id: 111\nvariant_id: 59\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ae75f7d7-227f-4031-8c4f-10e19d42f55a	2015-02-20 16:15:54.555068
348	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be recurrent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 22\nsource_id: 111\nvariant_id: 59\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5f9091c1-5048-4d96-829a-77fc4a88ad95	2015-02-20 16:15:54.601388
349	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	60252d60-8ed1-4bd2-a7f7-3a0db5edef92	2015-02-20 16:15:54.674721
350	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 60\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5c21d1db-7592-4d39-8b06-0c3c0c95b1f1	2015-02-20 16:15:54.732748
351	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e10fc765-9ac9-4c25-8c9e-fe1a280f8626	2015-02-20 16:15:54.81823
352	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 41\ndisease_id: 1\nsource_id: 113\nvariant_id: 60\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bd7d201a-e5a9-4f46-8226-0570b8548e83	2015-02-20 16:15:54.880867
353	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 1\nsource_id: 114\nvariant_id: 60\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cbfafe46-924a-4f3a-83ea-955964d618d8	2015-02-20 16:15:54.943305
354	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 1\nsource_id: 113\nvariant_id: 60\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	da3eb0f0-3f0f-4722-b267-5e655ba87956	2015-02-20 16:15:55.003999
355	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	34803874-e7ad-4325-8e38-21dd6cab2871	2015-02-20 16:15:55.065265
356	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9386683f-e00e-4359-b472-634cc4c28020	2015-02-20 16:15:55.125971
357	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 60\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	de674762-204f-40f8-92da-c1760cdc5321	2015-02-20 16:15:55.188443
358	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	29673a64-d03d-4e21-88a6-b9caaa87833a	2015-02-20 16:15:55.251645
359	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff89fb60-62ed-42eb-a53d-d52f20da721f	2015-02-20 16:15:55.333229
360	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c67b32e9-3273-46d6-977f-766fcd55b55a	2015-02-20 16:15:55.401671
615	23	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2d15eb0e-8dff-4d96-bce9-5d0327056f10	2015-02-20 16:22:49.730796
616	24	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	edd1f04b-0090-4244-8304-d31cf4ca67db	2015-02-20 16:22:49.754117
361	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 116\nvariant_id: 60\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9ddf450e-0592-4dad-b761-a2b468fb8ddf	2015-02-20 16:15:55.462168
362	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 60\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cc8250ef-c84b-4a09-8bf1-0a8af9c528bd	2015-02-20 16:15:55.527968
363	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeats, but not\n  in wild type or D835 mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 1\nsource_id: 118\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1cf710f1-e39a-406a-8329-cb80cf9e43aa	2015-02-20 16:15:55.589005
364	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 1\nsource_id: 119\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	db48336b-cde6-47ca-a350-0ba69ab048f0	2015-02-20 16:15:55.645165
365	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 44\ndisease_id: 1\nsource_id: 120\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	83ef5df2-7c6b-4a9b-8a65-f25161186caa	2015-02-20 16:15:55.702536
366	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 121\nvariant_id: 60\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8446deaf-b361-4816-aa32-ac3b61bb860f	2015-02-20 16:15:55.786615
367	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 60\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	58436f45-4fab-4e59-9aaa-1f78a098e824	2015-02-20 16:15:55.844044
368	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 41\ndisease_id: 1\nsource_id: 113\nvariant_id: 61\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3f6d7c73-4389-410d-970d-0f2177918586	2015-02-20 16:15:55.90391
369	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 28\ndisease_id: 1\nsource_id: 113\nvariant_id: 61\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e8b58895-15fe-48ab-bcd7-c2a4570cccc8	2015-02-20 16:15:55.968258
370	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 122\nvariant_id: 61\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a4bdca45-9afd-4305-b16f-d4e86e662458	2015-02-20 16:15:56.032163
371	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 61\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a8df38c6-5a60-4636-a893-27aff96aa50d	2015-02-20 16:15:56.087908
372	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 1\nsource_id: 123\nvariant_id: 61\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e5000316-a305-45bd-8c98-52638e79924c	2015-02-20 16:15:56.152754
373	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat mutations,\n  but not in patients with wild type or D835 mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 1\nsource_id: 118\nvariant_id: 61\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	db0d0d33-cf7e-4501-9c2a-0b8881904e64	2015-02-20 16:15:56.21461
374	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML\n  patients with FLT3 mutations\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 1\nsource_id: 119\nvariant_id: 61\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4991513c-bc4c-4f79-a02b-e6d73afecd38	2015-02-20 16:15:56.29404
375	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 117\nvariant_id: 61\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	75dd9ec6-374b-4882-857f-9f9eb849d529	2015-02-20 16:15:56.349925
376	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation was associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 124\nvariant_id: 61\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4e49d2bd-c02c-4bc9-b8de-602e51459f9a	2015-02-20 16:15:56.40862
617	25	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	105dcbc7-4282-46f9-812e-434352165cbd	2015-02-20 16:22:49.777407
377	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 45\ndisease_id: 23\nsource_id: 125\nvariant_id: 62\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c2319258-1ef4-478d-aa77-86f79a89e10f	2015-02-20 16:15:56.480048
378	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 46\ndisease_id: 23\nsource_id: 125\nvariant_id: 62\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8330db7c-411e-4f35-a45d-525b3058b5b2	2015-02-20 16:15:56.533146
379	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 126\nvariant_id: 63\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f9951e97-4cb7-44af-886f-58dfa6175f9b	2015-02-20 16:15:56.592997
380	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 127\nvariant_id: 63\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1cfe3c19-a114-4561-a3e2-26f8b6ead015	2015-02-20 16:15:56.650413
381	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 128\nvariant_id: 63\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	46fd0c14-e241-49a3-947d-1881f6e94b01	2015-02-20 16:15:56.733485
382	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 24\nsource_id: 129\nvariant_id: 63\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0a15f2ff-6538-4811-8fbe-0a276c3df27a	2015-02-20 16:15:56.798657
383	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 25\nsource_id: 130\nvariant_id: 63\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8a9a90a1-9cbe-4d7f-9d08-4fb2fb7ffb05	2015-02-20 16:15:56.864769
384	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 131\nvariant_id: 63\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b97bfe39-1d88-4887-a6fe-0b3a71ff58a8	2015-02-20 16:15:56.939543
385	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 64\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c2f8a251-0e13-4ff5-9211-4759f6251195	2015-02-20 16:15:56.994898
386	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 64\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4822ec84-ebbe-41f8-8b1b-014686f9ef4c	2015-02-20 16:15:57.051293
387	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 65\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1deae829-8680-44da-8bf0-92549eb998c1	2015-02-20 16:15:57.110127
388	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 132\nvariant_id: 66\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fce4b517-ac3f-4073-8dd4-f9fb039922ce	2015-02-20 16:15:57.187681
389	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 133\nvariant_id: 67\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	af00846b-4eb2-4b17-bdce-98f9d706bcbf	2015-02-20 16:15:57.254616
390	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact\n  overall survival or disease free survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 134\nvariant_id: 67\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	59d1d817-b534-4813-aa51-3c23fe1fb47d	2015-02-20 16:15:57.314909
391	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 135\nvariant_id: 67\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b3795bd9-63a4-47c0-b85e-fc9af1f7ca37	2015-02-20 16:15:57.384992
392	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 27\nsource_id: 136\nvariant_id: 67\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ae02b643-2582-4cc5-9a08-12c2e24a75e5	2015-02-20 16:15:57.445179
618	26	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	98a9f596-65fb-465a-b163-dd1e60724645	2015-02-20 16:22:49.798181
619	27	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b8a8c1be-625d-4fdf-87b8-ff9a14dcaee7	2015-02-20 16:22:49.81985
620	28	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3d21ce7c-2b45-45d0-a1b2-92a2d1ae786b	2015-02-20 16:22:49.840169
393	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 137\nvariant_id: 67\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7bbb3f40-1aef-4046-80ab-b65c96bf945f	2015-02-20 16:15:57.503406
394	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 135\nvariant_id: 68\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0a89ac63-4c20-44f5-9a86-0c390cce6b09	2015-02-20 16:15:57.568916
395	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with an IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 133\nvariant_id: 68\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a3e7095b-f524-4123-a9ae-e733ccf6d17e	2015-02-20 16:15:57.656731
396	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutations such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 137\nvariant_id: 68\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1232ff97-b6fd-4aa2-9e02-f6fe292b13e3	2015-02-20 16:15:57.70956
397	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F not associatted wth  lymphoid leukemia.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 2\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	afa3c930-1f74-4286-a034-397b148ada38	2015-02-20 16:15:57.775376
398	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 2\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a082d145-3791-46c4-a96f-0591d6aeb9e6	2015-02-20 16:15:57.834381
399	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 47\ndisease_id: 28\nsource_id: 139\nvariant_id: 69\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f086b30e-3942-4d71-898e-2253dfcbe739	2015-02-20 16:15:57.906034
400	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 29\nsource_id: 138\nvariant_id: 69\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	519088dc-e855-4279-a3a8-9f4fd6c5c0da	2015-02-20 16:15:57.963133
401	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: rs10974944 SNP in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 140\nvariant_id: 69\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4489ee56-09a8-4495-acf7-1204aad04a98	2015-02-20 16:15:58.025773
402	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 48\ndisease_id: 31\nsource_id: 141\nvariant_id: 69\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7972821b-dd8c-4b5c-b6d6-8deeaf4a645c	2015-02-20 16:15:58.176756
403	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 32\nsource_id: 142\nvariant_id: 70\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	375fbdfc-4c19-4e5e-bde4-8cce9b75d05d	2015-02-20 16:15:58.25147
404	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 71\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c8feffa8-175f-4210-a2e4-15ca1ead8e35	2015-02-20 16:15:58.323938
405	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 71\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4eebe700-89b0-48d8-8e4c-9e91380da1df	2015-02-20 16:15:58.391489
406	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 145\nvariant_id: 71\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a9ae21ce-805d-4414-a4e2-f3e413a3c163	2015-02-20 16:15:58.457869
407	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 71\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9c2f8e37-98c6-4a9d-8de3-5ff6852d97d6	2015-02-20 16:15:58.523651
408	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 146\nvariant_id: 71\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7a3916c0-8f16-4e64-a006-9d934630b29c	2015-02-20 16:15:58.593225
409	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 71\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6848132e-78b2-4ede-ba01-5b80070712fd	2015-02-20 16:15:58.654254
410	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 72\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5d73b145-c42e-4372-9684-468c0279f1c3	2015-02-20 16:15:58.71451
411	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 73\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9b7c88ae-1604-4729-91df-dd4463efb4e1	2015-02-20 16:15:58.765766
412	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 144\nvariant_id: 74\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ee7de8fe-c951-406c-a00e-8d5da7ad1943	2015-02-20 16:15:58.819659
413	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 8\nsource_id: 147\nvariant_id: 74\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d75fa0fa-ba56-4d7d-a077-d296b1adf8b6	2015-02-20 16:15:58.879609
414	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between wildtype KIT or KIT mutations in\n  exon 9 or 11 in survival among GIST patients.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 33\nsource_id: 143\nvariant_id: 75\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8174b7a7-fcc5-499d-9a59-484cbc3ef8b0	2015-02-20 16:15:58.947821
415	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 146\nvariant_id: 75\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bce39b73-91ce-4bc4-aab4-07d9f427738e	2015-02-20 16:15:58.996377
416	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1c48ff61-6eb4-4166-b50c-04749edf8dc3	2015-02-20 16:15:59.056617
417	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff176c07-9f12-43d0-9a46-e6b91a6bf36a	2015-02-20 16:15:59.106847
418	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 10\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	da95b50a-71ac-4cc7-ae9f-477aa85bb542	2015-02-20 16:15:59.158867
419	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 8\nsource_id: 148\nvariant_id: 76\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	860069de-49f8-4444-8f50-ee303a1a1b82	2015-02-20 16:15:59.221492
420	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 5\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	32f42b06-2927-4dc7-90c6-7a942104ae29	2015-02-20 16:15:59.280505
421	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	aef866d9-cf54-4f82-8e98-54be312924d5	2015-02-20 16:15:59.327237
422	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 4\ndisease_id: 4\nsource_id: 149\nvariant_id: 76\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d2928c0d-efe9-4b89-a749-93f839c7d1bb	2015-02-20 16:15:59.370149
423	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 33\nsource_id: 150\nvariant_id: 77\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	543ce6d5-284c-4e8c-a722-a938810f1ded	2015-02-20 16:15:59.424799
424	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 33\nsource_id: 151\nvariant_id: 77\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2e97706a-2ab6-4f8c-8b28-84bee9a26020	2015-02-20 16:15:59.492428
425	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 152\nvariant_id: 78\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	53a7edfc-472e-46fe-ac41-f73f5beabc70	2015-02-20 16:15:59.55237
621	29	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3ab50055-b598-4f83-b441-f0e90a8a62f7	2015-02-20 16:22:49.861687
426	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 7\nsource_id: 153\nvariant_id: 79\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b784b496-5a10-485b-9b0e-622a42780eba	2015-02-20 16:15:59.603959
427	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 4\nsource_id: 154\nvariant_id: 79\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	df159a03-d82d-4595-82b1-22eebca75ea3	2015-02-20 16:15:59.682521
428	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 4\nsource_id: 154\nvariant_id: 79\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bcb7f21f-7004-40a3-982e-003bd2bcf5a0	2015-02-20 16:15:59.727971
429	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 9\nsource_id: 155\nvariant_id: 80\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2f9b574a-273e-4799-9332-8e2553b1a9fd	2015-02-20 16:15:59.780704
430	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 80\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	864acb58-7541-43e2-a82a-7b683a9107b1	2015-02-20 16:15:59.849146
431	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 35\nsource_id: 157\nvariant_id: 80\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	814799ac-f3ef-4ffe-ab1d-e213b15da103	2015-02-20 16:15:59.91875
432	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 158\nvariant_id: 80\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f6721ac7-0ef4-4d7d-9012-d1bec5a30c92	2015-02-20 16:15:59.976774
433	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 159\nvariant_id: 80\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f186e889-f563-418e-bfe4-82b79fc7aa02	2015-02-20 16:16:00.038743
434	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 160\nvariant_id: 80\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3cf10ad9-7836-4b87-a86b-874b450b0e04	2015-02-20 16:16:00.091589
435	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 7\nsource_id: 161\nvariant_id: 81\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	daa3bc87-52cd-4131-9f34-7292a9c00ca0	2015-02-20 16:16:00.158475
436	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 81\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e3d16ae8-a688-4fa8-adcb-580928ac557e	2015-02-20 16:16:00.212975
437	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 161\nvariant_id: 81\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a20a86b6-1905-4f2c-a17e-246e5c4bd1bf	2015-02-20 16:16:00.272107
438	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 162\nvariant_id: 82\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3477fbaf-ca17-4c29-8ddb-556453ed7401	2015-02-20 16:16:00.334508
439	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 17\nsource_id: 162\nvariant_id: 83\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7a20af83-eba2-46d0-a453-a60f25521017	2015-02-20 16:16:00.38713
440	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 4\nsource_id: 163\nvariant_id: 83\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c3f23773-b26d-4a6c-8cc5-be1b222eed23	2015-02-20 16:16:00.452964
441	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 51\ndisease_id: 4\nsource_id: 164\nvariant_id: 83\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8c835e41-6864-4800-9606-3effc78c2207	2015-02-20 16:16:00.512365
622	30	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	fa735f6c-c2a1-4dcd-b84e-f79da2af221c	2015-02-20 16:22:49.885059
623	31	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	799695f5-9066-40fc-a0e1-6aa77a9ed174	2015-02-20 16:22:49.910768
442	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in a lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 52\ndisease_id: 4\nsource_id: 164\nvariant_id: 83\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	92a5cb61-6138-4634-a7ff-d5cae0509276	2015-02-20 16:16:00.559537
443	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 34\nsource_id: 156\nvariant_id: 84\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	76509f1d-f2fd-4fa3-817e-e62105494a00	2015-02-20 16:16:00.628575
444	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 36\nsource_id: 157\nvariant_id: 84\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	72f7d561-9516-47c4-92d0-7a47e2a8f143	2015-02-20 16:16:00.684508
445	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung cancer patients are not associated with smoking history,\n  age or gender\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 4\nsource_id: 158\nvariant_id: 84\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bb734992-9fd2-4a54-94f8-9853576be2f4	2015-02-20 16:16:00.745178
446	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a8660934-5c1e-4506-a142-ac960c881ad3	2015-02-20 16:16:00.81086
447	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft model\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fc48e0cc-fa63-42a6-b216-ec73fbffaf36	2015-02-20 16:16:00.868196
448	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2699015c-eda0-4d89-8ce3-8161d5f55e4d	2015-02-20 16:16:00.913879
449	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 9\nsource_id: 165\nvariant_id: 85\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b792a856-c3bd-4733-b051-045efc0feda6	2015-02-20 16:16:00.966393
450	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 8\nsource_id: 166\nvariant_id: 86\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0502526c-cf42-4552-8f10-25fd1b6261b6	2015-02-20 16:16:01.028252
451	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 8\nsource_id: 167\nvariant_id: 87\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0b7cd49b-6d2b-4115-81ef-d0c0982fd58d	2015-02-20 16:16:01.083754
452	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 1\ndisease_id: 19\nsource_id: 90\nvariant_id: 88\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c012de93-b065-4080-9728-cecca52cbafc	2015-02-20 16:16:01.146121
453	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 19\nsource_id: 90\nvariant_id: 88\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7553bfcc-0ce4-4ade-9215-23046bba5514	2015-02-20 16:16:01.235841
454	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increased survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 22\ndisease_id: 37\nsource_id: 168\nvariant_id: 89\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	94c48fc5-6a15-4a15-9893-3ecdfe6dc125	2015-02-20 16:16:01.295
455	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation was associated with increased overall and disease-free\n  survival as well as tumor regression in patients with gliomas.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	16cb3bbe-67f0-4135-b375-42e392df436f	2015-02-20 16:16:01.364292
456	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 53\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c076a9c4-84b0-4af2-82e4-917cff3afa92	2015-02-20 16:16:01.43134
457	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 54\ndisease_id: 37\nsource_id: 169\nvariant_id: 89\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	17356c2b-bd4b-4bbb-b962-fd51ff11c6b5	2015-02-20 16:16:01.509529
624	32	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	405b9a7e-926b-4cdc-9e4d-7d0012e2aa82	2015-02-20 16:22:49.931124
625	33	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b3294d24-265c-4aad-ad16-c8626a6f234f	2015-02-20 16:22:49.94949
458	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 92\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f00f8b99-e1fe-495a-8027-572d54b5f6a4	2015-02-20 16:16:01.57228
459	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid\n  leukemia.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7bf90f6d-4e80-4c56-8f63-7867599221b6	2015-02-20 16:16:01.630175
460	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype\n  (0/55).\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	633baa76-8acd-4458-8c16-1a3d69484329	2015-02-20 16:16:01.689294
461	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid\n  leukemia.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	168ea9ef-1b37-4239-9d8b-1deabbe78680	2015-02-20 16:16:01.749579
462	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid\n  leukemia\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	616af353-8e63-4193-b14a-e67121797143	2015-02-20 16:16:01.80157
463	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7d266b2c-7a4a-415d-b051-48a9a28941c6	2015-02-20 16:16:01.859889
464	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 90\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	b06aaca4-c230-409c-8a49-6a4c501b27d5	2015-02-20 16:16:01.924646
465	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 1\nsource_id: 114\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f5f7621c-2c8b-45f4-a5f1-5597de39c408	2015-02-20 16:16:01.975312
466	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 173\nvariant_id: 90\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a13475f8-b73d-407b-9687-4735985d70bb	2015-02-20 16:16:02.036265
467	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult acute myeloid leukemia patients (median\n  age 43)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 112\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	303f41f4-fbe0-4216-b4ea-b98b7ed9294e	2015-02-20 16:16:02.097934
468	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 55\ndisease_id: 1\nsource_id: 174\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0843f141-5927-42d6-8044-8d6d5743cdf9	2015-02-20 16:16:02.159245
469	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and\n  FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 42\ndisease_id: 1\nsource_id: 174\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	df64c361-66e2-42dd-9799-a10d19d6b149	2015-02-20 16:16:02.209712
470	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	37bc3a91-ea05-4a47-8fbe-2a9a642504ed	2015-02-20 16:16:02.266331
471	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 1\nsource_id: 172\nvariant_id: 90\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3f7a579d-812a-4711-9ceb-72ddfb47497f	2015-02-20 16:16:02.327315
472	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 28\ndisease_id: 1\nsource_id: 175\nvariant_id: 90\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e87d3007-3146-4b2c-bdc6-7097e6ef813e	2015-02-20 16:16:02.394559
473	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 56\ndisease_id: 1\nsource_id: 176\nvariant_id: 90\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	408796a4-0a6b-4f84-be87-356e8fab9b21	2015-02-20 16:16:02.466637
474	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 177\nvariant_id: 90\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2067bea6-dacc-4c3a-89f9-c5b43df12eb3	2015-02-20 16:16:02.535671
475	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 177\nvariant_id: 90\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	72dc0c42-ce52-4d49-9a01-68cf594e0f9c	2015-02-20 16:16:02.602414
476	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1 mutations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 170\nvariant_id: 90\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	27190ad0-974e-42a5-99d5-fd32a120dffd	2015-02-20 16:16:02.664067
477	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 178\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	253c196c-2194-445a-b9e1-44164a7f734e	2015-02-20 16:16:02.721275
478	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 179\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	38126c0f-cbb4-41ec-b7a0-787109c83067	2015-02-20 16:16:02.828611
479	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 180\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d125aaf5-1109-4b66-9107-b23f992a9f42	2015-02-20 16:16:02.894871
480	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  were longer for patients with Exon 12 NPM1 mutations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c0dab104-1e06-4105-8bdb-642dc6f6390e	2015-02-20 16:16:02.957999
481	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 122\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9959853a-fa6f-489e-9336-c9769ce8e7a9	2015-02-20 16:16:03.018768
482	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 181\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	939b79b7-ee22-4728-9785-d25e439d0219	2015-02-20 16:16:03.077752
483	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 182\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1f71cb48-d19e-49eb-b5a8-5fef9e346a9b	2015-02-20 16:16:03.136896
484	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1\n  and IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 175\nvariant_id: 90\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9abf4511-934c-4918-8012-0f98dbe88a5a	2015-02-20 16:16:03.200321
485	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 88\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c0eb955f-6688-497b-9062-0fe8d85029b9	2015-02-20 16:16:03.264348
486	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 126\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4d02d4ab-3d22-4014-86fd-cd9be69f779b	2015-02-20 16:16:03.325369
487	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 171\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5a8e2fb9-bc0b-4ec3-a579-8dda52c663e9	2015-02-20 16:16:03.386196
488	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d047713d-388d-4114-bc82-c4d85f1cb96c	2015-02-20 16:16:03.451073
626	34	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e350ac76-46be-4890-84f9-52045e82ba49	2015-02-20 16:22:49.969662
627	35	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	eade953a-b491-465e-901a-4939a27d328e	2015-02-20 16:22:50.003965
489	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 115\nvariant_id: 90\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a68bba65-d0ba-4348-9adc-f4e2c7e5bd5a	2015-02-20 16:16:03.506037
490	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 183\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1a5d08b8-da93-486b-9517-9628d6f82864	2015-02-20 16:16:03.566102
491	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 180\nvariant_id: 90\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	536b0bfa-8819-4e5d-8e52-63fc11fd0233	2015-02-20 16:16:03.620031
492	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"AML with mutated NPM1" is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.'\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 1\nsource_id: 184\nvariant_id: 90\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6a62c12a-688e-4c5a-96ad-9b584b4d2463	2015-02-20 16:16:03.676212
493	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 91\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	241d20d4-cf1f-47be-90c5-98f2b4f7ee03	2015-02-20 16:16:03.736304
494	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 94\nvariant_id: 91\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6d3ea560-1eca-4f44-a559-f3888be50da7	2015-02-20 16:16:03.806676
495	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 55\ndisease_id: 1\nsource_id: 174\nvariant_id: 91\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0e4214ce-c4d5-4b9f-8a66-4540c4dde6ca	2015-02-20 16:16:03.880967
496	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 185\nvariant_id: 91\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7c63687f-59e0-4829-8f5e-f63c7bd0320f	2015-02-20 16:16:03.955368
497	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 186\nvariant_id: 92\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	278bc959-acbb-4c04-8f77-02e47bdef393	2015-02-20 16:16:04.021319
498	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patients does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 187\nvariant_id: 93\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bd3f761f-87ec-4a7c-bcf3-3404ff6aab40	2015-02-20 16:16:04.085959
499	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 93\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ad9ad418-c4c2-4ba8-bc0c-04ddfd8352f7	2015-02-20 16:16:04.144619
500	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 186\nvariant_id: 93\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c2e00bb4-bb91-41ab-b6f0-43a1f02a630a	2015-02-20 16:16:04.204602
501	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 94\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff1e791f-b9e4-4a4d-ba18-59b14e786c4c	2015-02-20 16:16:04.255735
502	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation have poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 7\nsource_id: 188\nvariant_id: 95\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	07bec0e8-bf7f-42b1-9a75-343b4c6b8edc	2015-02-20 16:16:04.361527
503	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 189\nvariant_id: 96\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	777af298-078a-422a-8ba8-61d5f3ef46bc	2015-02-20 16:16:04.424353
504	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Likely due to increased reliance of mutant NRAS on HSP90 for stabilization,\n  inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic\n  malignant melanoma with an NRAS G13D mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 8\nsource_id: 190\nvariant_id: 97\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f3194344-ec52-4174-b596-c9b52c221ad7	2015-02-20 16:16:04.482498
505	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 98\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a5c8bb5f-ac10-400f-8c23-dbfd0542db27	2015-02-20 16:16:04.547589
506	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 98\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	201640e8-986c-40e7-9549-36583d6ce08d	2015-02-20 16:16:04.609409
507	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 191\nvariant_id: 98\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	db4ff297-2f53-4639-8826-26a8bbea7bfa	2015-02-20 16:16:04.685869
508	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 19\ndisease_id: 8\nsource_id: 192\nvariant_id: 98\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d5525ebd-871b-4c9f-bb31-56b173a4e0a4	2015-02-20 16:16:04.745793
509	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 58\ndisease_id: 8\nsource_id: 193\nvariant_id: 99\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	665efb3f-7a77-4e1c-ab06-1fea3b2eabfd	2015-02-20 16:16:04.800139
510	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 58\ndisease_id: 8\nsource_id: 193\nvariant_id: 100\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c546acbd-5868-4798-931e-d1acc834bad0	2015-02-20 16:16:04.860401
511	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 38\nsource_id: 194\nvariant_id: 101\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	16265fdd-14f2-42b5-a265-ffc1b51d949c	2015-02-20 16:16:04.925861
512	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumors harboring PDGFRA D842V mutation are likely to be benign\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 38\nsource_id: 195\nvariant_id: 102\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7d03ac1e-d054-4411-a943-a6d42b3d0870	2015-02-20 16:16:04.992673
513	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 38\nsource_id: 150\nvariant_id: 102\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7239e68d-792d-4dea-9b20-54aa318ac4e3	2015-02-20 16:16:05.060126
514	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 38\nsource_id: 196\nvariant_id: 102\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	23958ad1-caa2-4871-aa12-b5d9ab52793b	2015-02-20 16:16:05.123556
515	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 38\nsource_id: 194\nvariant_id: 102\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	362bd186-9f82-461d-a177-29eb50d25248	2015-02-20 16:16:05.189584
516	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 38\nsource_id: 194\nvariant_id: 103\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	afdf8807-e6ac-4abf-9529-b10cfe7e3aca	2015-02-20 16:16:05.255139
517	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib\n  resistance, crenolanib was significantly more potent at inhibiting kinase activity\n  than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 38\nsource_id: 194\nvariant_id: 104\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	417e0864-53a7-48e8-8b8c-61c14dd2eea5	2015-02-20 16:16:05.321551
518	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance,\n  crenolanib was significantly more potent at inhibiting kinase activity than imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 59\ndisease_id: 38\nsource_id: 194\nvariant_id: 105\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cfd13dec-6c30-40bf-a1b8-ccc6b589d409	2015-02-20 16:16:05.395713
519	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 197\nvariant_id: 106\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1ef5ed54-56ee-4852-961a-aec8acc327b5	2015-02-20 16:16:05.483438
520	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 11\nsource_id: 198\nvariant_id: 106\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	58b87c7b-8cb8-4f36-ad03-baadb7ecdc4a	2015-02-20 16:16:05.543637
628	36	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	4f640d11-7ad6-4353-8623-9b6ba25f01e0	2015-02-20 16:22:50.029323
629	37	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ecb36430-f353-42ca-92ae-979fca25d78c	2015-02-20 16:22:50.05015
521	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 7\nsource_id: 153\nvariant_id: 106\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7e0f2b87-2c07-4ed6-b8ef-f59d3a4eaebd	2015-02-20 16:16:05.600618
522	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	93dbfbe1-66af-4ab3-afbf-8f0bd0ffd026	2015-02-20 16:16:05.674615
523	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	fb89f6a5-ce0c-4b45-bc72-909edb04807d	2015-02-20 16:16:05.732808
524	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutations were associated\n  with poorer survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 106\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	42bbb671-9347-4f0a-9f5a-54f6a6a23f29	2015-02-20 16:16:05.796975
525	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 200\nvariant_id: 106\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e6d40828-2daa-4819-8dc6-20c76be6b1f4	2015-02-20 16:16:05.86251
526	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 197\nvariant_id: 107\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c6d08615-6114-4da3-8b98-7749b02c8425	2015-02-20 16:16:05.96267
527	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 11\nsource_id: 198\nvariant_id: 107\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	8cbf329b-e955-4c76-8d0c-42c8978b5810	2015-02-20 16:16:06.021198
528	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 7\nsource_id: 153\nvariant_id: 107\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2bd37249-f819-495a-802c-135bb300a0d9	2015-02-20 16:16:06.082992
529	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6edf645e-e821-4099-8d36-4ee35862183d	2015-02-20 16:16:06.145973
530	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5252cc0d-73f9-40ec-aac2-964d2d2ba50f	2015-02-20 16:16:06.190874
531	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 107\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	050a9c8e-5828-4fd7-8640-1c6108f7b477	2015-02-20 16:16:06.256259
532	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 200\nvariant_id: 107\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	44aa53bf-3edc-418c-948e-9fbad9519e6a	2015-02-20 16:16:06.310247
533	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates as well as shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 108\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	210b7bf2-9005-4265-9735-ab7ed2f4469e	2015-02-20 16:16:06.37231
534	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 62\nvariant_id: 109\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ede50e30-559b-4304-8875-2a16e9fea0b4	2015-02-20 16:16:06.43258
535	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 60\ndisease_id: 11\nsource_id: 197\nvariant_id: 110\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0e3ff0d1-88b8-40d6-8be0-804074933e38	2015-02-20 16:16:06.490812
536	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 11\nsource_id: 198\nvariant_id: 110\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	9aae5db5-9cf8-4253-b3a8-1aeee333eb98	2015-02-20 16:16:06.556641
537	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 7\nsource_id: 153\nvariant_id: 110\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	575b8a00-854f-4d9e-bd0a-f569cf7b0d0f	2015-02-20 16:16:06.612837
538	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 14\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ff210782-77c1-4172-8ffe-029d91ff54d8	2015-02-20 16:16:06.675749
539	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were\n  associated with lack of panitumumab or cetuximab response.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 12\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f5f2ac3d-6fff-407d-b120-8e3d5b33c398	2015-02-20 16:16:06.720968
540	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 199\nvariant_id: 110\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ac148673-70fb-42d9-ac1f-312c869593c2	2015-02-20 16:16:06.782868
541	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 39\nsource_id: 201\nvariant_id: 111\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d600cf2b-7c05-4d01-942f-69dc5e1da53f	2015-02-20 16:16:06.851671
542	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 20\nsource_id: 91\nvariant_id: 112\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	528cc51d-158a-4fa1-a0c8-b34502b9497e	2015-02-20 16:16:06.910601
543	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 3\nsource_id: 202\nvariant_id: 113\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	bb50c947-8ca5-4a7f-a3f4-ab33d8c66d23	2015-02-20 16:16:06.98763
544	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense mutations, including R233*, have been shown to be inactivating\n  and loss-of-function, but do not have prognostic value in glioblastoma multiforme\n  patients.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 37\nsource_id: 203\nvariant_id: 113\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7244e917-1411-486b-8180-09694bf81489	2015-02-20 16:16:07.05967
545	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic\n  cells at pharmalogical concentrations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 39\nsource_id: 201\nvariant_id: 114\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	489ae063-6940-47fe-b323-0563d72d76da	2015-02-20 16:16:07.130717
546	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 62\ndisease_id: 40\nsource_id: 204\nvariant_id: 115\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	53844f7d-5776-443a-bb03-a1ee2ce1d558	2015-02-20 16:16:07.196786
547	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 62\ndisease_id: 40\nsource_id: 204\nvariant_id: 116\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	17e86783-cc9a-4935-8157-dc5b13b7e5ad	2015-02-20 16:16:07.271375
548	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 40\nsource_id: 205\nvariant_id: 116\nrating: 5\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d4ef7f49-7ebf-4d45-ba0e-aef921329c2e	2015-02-20 16:16:07.341696
549	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong\n  repression of tyroid cancer cell growth.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 63\ndisease_id: 40\nsource_id: 206\nvariant_id: 116\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0a22064b-83b8-410c-b991-d86eecb5be7d	2015-02-20 16:16:07.462078
550	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 40\nsource_id: 207\nvariant_id: 116\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	f1f06af1-bdd0-444d-ab70-24f9228d28ce	2015-02-20 16:16:07.522926
551	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 208\nvariant_id: 117\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ae0ac24f-f888-4530-9b7e-393402daedbb	2015-02-20 16:16:07.592665
630	38	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	44c813b9-fdac-47ac-ab96-7d6ab86eb492	2015-02-20 16:22:50.072352
631	39	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b5ede625-b63f-466a-8f25-13215adb49b9	2015-02-20 16:22:50.094596
552	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 208\nvariant_id: 118\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2628f976-e2f6-44d8-b01b-77653cf1a5a5	2015-02-20 16:16:07.657105
553	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin\n  than breast tumors with wild type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 64\ndisease_id: 11\nsource_id: 209\nvariant_id: 119\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	90c80142-89c5-482b-9fb1-333a8db3bacc	2015-02-20 16:16:07.718811
554	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 119\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	c19c9a58-50b6-405a-b085-adcf066b4292	2015-02-20 16:16:07.791513
555	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R248 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 120\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5048e087-a635-4e04-8f04-b1748e74f3be	2015-02-20 16:16:07.857782
556	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 120\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	adda0b78-0841-4ccd-bea7-6be2946bcda6	2015-02-20 16:16:07.921424
557	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 121\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	93f2c0db-95b2-4253-9af7-e9b4654b0166	2015-02-20 16:16:07.979173
558	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 122\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d05e450d-ba0e-4b30-980b-0830d2a7513e	2015-02-20 16:16:08.038801
559	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved regions\n  such as R249 are prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 123\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	edbb94b3-a1cd-4de9-a4c4-7c8d88ebf033	2015-02-20 16:16:08.096627
560	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 124\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ba2ee7d8-626c-4947-bac1-d0933023d157	2015-02-20 16:16:08.154257
561	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 124\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	55b090f6-88ba-432d-a861-5ff7c9ac3f99	2015-02-20 16:16:08.21777
562	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in DNA contact\n  regions such as R273 are prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 125\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	91046b71-65fe-48dc-b6f8-97f6c41713dd	2015-02-20 16:16:08.274742
563	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation have worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 210\nvariant_id: 125\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	73185fef-a993-431f-bae5-89f1f26eeefd	2015-02-20 16:16:08.336916
564	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumors with wild type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 211\nvariant_id: 126\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	7dbfe22f-7b6f-42cc-bf8f-8626eb9f094b	2015-02-20 16:16:08.395803
565	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 65\ndisease_id: 41\nsource_id: 212\nvariant_id: 127\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	0db54109-1210-47a7-a7e9-73ad70383f1f	2015-02-20 16:16:08.459157
566	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 61\ndisease_id: 4\nsource_id: 213\nvariant_id: 127\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ffd19b67-d706-4f8d-b404-105c8e2064df	2015-02-20 16:16:08.524767
567	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 65\ndisease_id: 42\nsource_id: 214\nvariant_id: 128\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	a6d4623a-5690-4e98-a437-22e226ebe2d7	2015-02-20 16:16:08.600396
568	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 65\ndisease_id: 42\nsource_id: 212\nvariant_id: 129\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	e81031ce-4e51-4e35-8187-7770dc0acf53	2015-02-20 16:16:08.677774
569	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 130\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	82b02a44-3bb7-4ebd-9adc-f1adaa5bdf04	2015-02-20 16:16:08.754718
570	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 130\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	411d883f-809a-4f95-8856-8e00091b8771	2015-02-20 16:16:08.811807
571	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 215\nvariant_id: 130\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	6ddec5f4-2973-4034-a321-072cbb588cf1	2015-02-20 16:16:08.878033
572	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 216\nvariant_id: 130\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	23e4f201-6fa1-4a4b-ad53-f2b206e2b560	2015-02-20 16:16:08.985828
573	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 131\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	ec29f888-1b63-4782-a8ea-9eabd1a9ddce	2015-02-20 16:16:09.048212
574	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 1\nsource_id: 215\nvariant_id: 131\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2e4427d2-be5c-48b7-b4db-3bed3275846c	2015-02-20 16:16:09.107514
575	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 215\nvariant_id: 131\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5cc5ccc7-0f1f-43a1-bfcd-e173e0d76927	2015-02-20 16:16:09.16792
576	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, the presence of U2AF mutations such as\n  S34Y/F are prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 27\nsource_id: 216\nvariant_id: 131\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	597d9be8-0d4c-41d5-8fc3-3fb5305a9d3d	2015-02-20 16:16:09.241488
577	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	3fff614b-06c8-4321-99d0-06c2c0852864	2015-02-20 16:16:09.306391
578	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	559f5b2d-ee96-42c9-a79e-9919b94fd90b	2015-02-20 16:16:09.37057
579	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	5b329a6e-8123-42af-a4a3-cb1c2bc25028	2015-02-20 16:16:09.43069
580	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	435bccf3-99d9-4230-b2c7-122a661cabd4	2015-02-20 16:16:09.491139
581	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 132\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	d2470f8c-07e6-49e8-a3de-ab3a375a5ec7	2015-02-20 16:16:09.5545
632	40	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b873f290-24c4-4a9f-af5f-cf39096ac7b0	2015-02-20 16:22:50.11599
633	41	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c71d844c-3e09-4d70-9a5f-1268863997be	2015-02-20 16:22:50.135421
582	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 219\nvariant_id: 132\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	2f92cfcb-07c6-46a8-9e3e-9e9df88ec39c	2015-02-20 16:16:09.61252
583	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 132\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	4ad5365c-7bb5-4c04-94ac-4fc38f0c6c9d	2015-02-20 16:16:09.684597
584	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations in WT1 were associated with a worse overall prognosis than patients\n  wildtype for WT1 in young patients (15-50), primarily because of increased risk\n  of disease recurrence\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 220\nvariant_id: 132\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	333cd782-518c-49ce-8489-03a09df064ea	2015-02-20 16:16:09.746907
585	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutations  in WT1 were associated with increased risk of recurrence in young\n  patients (15-50) with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 220\nvariant_id: 132\nrating: 2\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	11a4735a-58ff-4fc0-8962-50c04da44abc	2015-02-20 16:16:09.799841
586	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	817c3cb4-8e47-4c31-9884-c6f6294130b2	2015-02-20 16:16:09.862544
587	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 4\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cf0683bf-d3c9-40ad-b573-eaf5cf1d25e9	2015-02-20 16:16:09.927307
588	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No differences in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	cbf6b725-0eab-4d61-a790-894850c80751	2015-02-20 16:16:09.991765
589	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	027facfe-77f6-47b0-a000-2a613472383d	2015-02-20 16:16:10.051709
590	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 217\nvariant_id: 133\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	1391e508-ca9d-45a5-a6d2-bed18ea9c106	2015-02-20 16:16:10.11513
591	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 219\nvariant_id: 133\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	34edbece-2abf-45b1-abc3-afb82f9d7783	2015-02-20 16:16:10.174136
592	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 1\nsource_id: 218\nvariant_id: 133\nrating: 3\nstatus: \nremote_errors: \nremote_ids: \nvariant_origin_id: 1\n	1	\N	\N	04cc6444-ce49-4930-92b1-082e118c4cbf	2015-02-20 16:16:10.22898
593	1	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	eb0fd21d-20c8-49e0-b67d-83ef1fcfaf38	2015-02-20 16:22:49.239176
594	2	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1b5f3067-ae55-402a-ab37-06661592b5e9	2015-02-20 16:22:49.27995
595	3	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2ec5de4b-5262-410a-9726-8a4d2058a6ce	2015-02-20 16:22:49.302931
596	4	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d84c1f89-ccb2-47d0-8216-97fb40917629	2015-02-20 16:22:49.325316
597	5	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f52ec087-307d-4ed4-b98c-15a2d273fde0	2015-02-20 16:22:49.34746
598	6	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	81cda8a5-1cbc-4075-b012-37f10a30c6cc	2015-02-20 16:22:49.37138
599	7	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	72c3a832-7dbc-4281-b5af-a5b14bfc4b29	2015-02-20 16:22:49.391414
600	8	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	024ac5b8-e3a0-4d2e-9369-26ea0b0964be	2015-02-20 16:22:49.409098
601	9	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	58737ba8-28aa-47d1-82ad-f47d389b83e5	2015-02-20 16:22:49.436569
602	10	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	dfdbc35c-c97b-4960-b541-9039d9ac96f9	2015-02-20 16:22:49.458569
603	11	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	44b0221f-2d72-46ad-a1a1-41834eb5e470	2015-02-20 16:22:49.47806
604	12	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0706a6cf-dd03-4dd4-b752-26967e8a0d36	2015-02-20 16:22:49.500653
605	13	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	728d9fe9-2e31-4a07-bcd0-2e24c18b4b8b	2015-02-20 16:22:49.519338
606	14	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ec12ad12-f347-4f68-9ca9-29a1473d4158	2015-02-20 16:22:49.540075
642	50	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f164cfd5-3d3d-4084-8cef-c9e42a609372	2015-02-20 16:22:50.351667
643	51	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	836c205c-f337-41f2-82a0-0f05ec31451c	2015-02-20 16:22:50.371781
644	52	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	07ba0690-0ca1-4e28-8cc5-5d142876afcf	2015-02-20 16:22:50.39246
645	53	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8301e884-b729-4500-83a2-927011f8b6cb	2015-02-20 16:22:50.414767
646	54	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	adb359b7-d1e8-4961-97d2-9b3e573e9a89	2015-02-20 16:22:50.435784
647	55	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1e30f8c9-3d1f-4c31-87db-50466c4022c0	2015-02-20 16:22:50.455754
648	56	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bf8c9b0a-82c1-4b61-85e3-9c8f704b3ed2	2015-02-20 16:22:50.479913
649	57	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9f753bab-e84b-44ef-a0e8-68e2c3b553fd	2015-02-20 16:22:50.501363
650	58	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1556ae43-e79a-4aeb-bcef-828189a4fdb0	2015-02-20 16:22:50.520347
651	59	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	129c7bb4-d672-42ff-b48a-14b3312ec024	2015-02-20 16:22:50.548398
652	60	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8b9872f9-f0a9-4b0d-938e-89bc59249d54	2015-02-20 16:22:50.57159
653	61	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a1feaeab-c7de-4fcd-a5f0-72af72681bf5	2015-02-20 16:22:50.592858
654	62	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	768f594c-91d2-4f74-8afc-ac7b0e07e80d	2015-02-20 16:22:50.614342
655	63	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5e440f51-e73c-4fbb-b994-4a22caf0bff3	2015-02-20 16:22:50.635256
656	64	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2113d3a2-083e-4c48-ae74-3586c8155acf	2015-02-20 16:22:50.65659
657	65	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2deb0205-0146-4a52-9437-553bb0de183e	2015-02-20 16:22:50.678941
658	66	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	199a49bc-e028-40e9-b6b7-f3c75df25164	2015-02-20 16:22:50.705202
659	67	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	311a3cfd-d7a1-48a2-9ddc-7de8d1ea805d	2015-02-20 16:22:50.726275
660	68	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b7542b96-d569-43c2-998d-7f478dc57240	2015-02-20 16:22:50.76812
661	69	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5fc4006d-9339-4e18-90d0-bcb4f4d25801	2015-02-20 16:22:50.788004
662	70	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	41b38341-7bb5-47b9-bb32-4fac2105041a	2015-02-20 16:22:50.807694
663	71	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	21788746-affa-4456-aa5e-14133220f57d	2015-02-20 16:22:50.827318
664	72	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	75a33116-2c87-4eb7-a45d-b1563965cd37	2015-02-20 16:22:50.847182
665	73	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	87f60899-ffe8-40fa-9c8d-6f8a9a2e8d01	2015-02-20 16:22:50.867987
666	74	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6885dcb3-388e-4a61-aab9-2cea969a1eb1	2015-02-20 16:22:50.88909
667	75	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0893f71e-a281-4040-a8cd-a37356c80bb1	2015-02-20 16:22:50.909815
668	76	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	36564ed4-1feb-49cc-a6b6-6af546c83b56	2015-02-20 16:22:50.934559
669	77	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f000a922-3a83-4ea9-acde-c17b969a4992	2015-02-20 16:22:50.95544
670	78	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bcdc6db8-b2d4-4fe4-a4fc-bc6f9e1c5cb7	2015-02-20 16:22:50.975509
671	79	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	becec41b-1f5b-4814-9931-3013cd349115	2015-02-20 16:22:50.999959
672	80	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	75ac30fe-a114-44e4-9d5e-96496ca9c568	2015-02-20 16:22:51.022054
673	81	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2009e5b5-a179-45a6-8340-2d4ed8cf0cc3	2015-02-20 16:22:51.04296
674	82	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	57019d62-514d-43e8-95a5-49491be32dfc	2015-02-20 16:22:51.063
675	83	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	900af45b-6db8-43b8-99c7-2fe9260eb203	2015-02-20 16:22:51.084449
676	84	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	69e673cf-0cfe-4239-bae4-1672bc84c166	2015-02-20 16:22:51.103331
677	85	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0e1bf984-dc58-4cb7-ac72-833900ef2a34	2015-02-20 16:22:51.121994
678	86	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	87bcd991-4e91-4db8-b1b6-a2965fbab1e5	2015-02-20 16:22:51.143682
679	87	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	20ecac14-24df-4aed-a84e-3a57a15b5ebe	2015-02-20 16:22:51.163265
680	88	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d2e099b7-bb07-4c14-91d3-698d2f540c93	2015-02-20 16:22:51.184031
681	89	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	efacb098-3192-4c45-a2ed-2caa2ebb54fe	2015-02-20 16:22:51.204466
682	90	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	4a7a327c-7033-4c25-b607-7eae43e7ee13	2015-02-20 16:22:51.228354
683	91	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	80e23404-5cad-4e65-9eab-03696704b518	2015-02-20 16:22:51.247703
684	92	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	42acece8-f403-4bd0-b21a-176da2e56180	2015-02-20 16:22:51.268638
685	93	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	af0cb27b-0460-4369-a7de-18784538710b	2015-02-20 16:22:51.28871
686	94	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e3566229-04f3-4073-844c-10b2e4fbb5d5	2015-02-20 16:22:51.311372
687	95	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	40ef8a5d-5b17-47ed-b052-713609778675	2015-02-20 16:22:51.332781
688	96	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3293d5cb-67f0-41e8-b964-2761a13718dc	2015-02-20 16:22:51.357732
689	97	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3d05764a-c07a-4bee-a79f-3c3688b590e8	2015-02-20 16:22:51.379997
690	98	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	abb7af71-caef-4a5a-9184-982fca6009d4	2015-02-20 16:22:51.401793
691	99	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	299898c3-eb4f-4fc4-8871-21cb60767f31	2015-02-20 16:22:51.424606
692	100	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8df7e8e8-9563-49f2-9802-0921b686c085	2015-02-20 16:22:51.448017
693	101	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	318ce89a-a60a-48d5-a4c1-7c68f251dee5	2015-02-20 16:22:51.484146
694	102	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	fdf39dc3-a2c2-4f4c-a10e-f3f891d44606	2015-02-20 16:22:51.504995
695	103	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	93df0e66-2565-4584-b320-d646387bd72b	2015-02-20 16:22:51.526249
696	104	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1e2899e9-f462-41cd-9760-6d9c0a92c071	2015-02-20 16:22:51.547296
697	105	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6ec1b534-3e43-48d6-949a-73be2b93c450	2015-02-20 16:22:51.567916
698	106	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7d7407dc-8030-464f-9cdb-b30ba58fed87	2015-02-20 16:22:51.592272
699	107	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	997cb2ee-ebb9-454b-bd26-222e589cfec7	2015-02-20 16:22:51.613368
700	108	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c8486a4b-3512-4950-ab74-85b8baea82d9	2015-02-20 16:22:51.632254
701	109	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e1221205-a66d-4320-b1b3-126eca7977ae	2015-02-20 16:22:51.659272
702	110	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5d8d7cac-913e-4fa0-84dd-c7068cf5e02a	2015-02-20 16:22:51.680283
703	111	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7c864937-c208-4fa4-8237-394e512241c4	2015-02-20 16:22:51.700322
704	112	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	626c634c-a22c-40e4-a8e3-b2d1229b028d	2015-02-20 16:22:51.721526
705	113	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	357fb8b9-5751-45f8-832f-45c234a3f514	2015-02-20 16:22:51.743697
706	114	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f6a36cb5-6273-4910-8e6e-b0675b92e015	2015-02-20 16:22:51.766779
707	115	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	95c9b3ec-4f73-4262-9419-0a9cd06f0540	2015-02-20 16:22:51.789646
708	116	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	25cc7c23-e0b4-4051-8020-7c3505471915	2015-02-20 16:22:51.808971
709	117	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	4d005839-9451-4473-8265-4beaaa519c1d	2015-02-20 16:22:51.832536
710	118	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3b12aa1c-3e99-4bfa-993a-3ab9c72d234a	2015-02-20 16:22:51.852242
711	119	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6c0b5adb-6d59-4db1-84bc-1c43212145fa	2015-02-20 16:22:51.875437
712	120	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e878ca9f-f6e5-468c-a075-e50da3ee0c69	2015-02-20 16:22:51.897299
713	121	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5585a553-bbaa-480b-8d31-a91f5a164030	2015-02-20 16:22:51.918232
714	122	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ae3e1468-9ec9-422d-9165-d432c938ba39	2015-02-20 16:22:51.94466
715	123	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8cd4cf29-aa40-4b3d-8c4d-4281e036d7d6	2015-02-20 16:22:51.978555
716	124	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3307520e-304c-419e-aadd-41e4d9a30105	2015-02-20 16:22:52.011232
717	125	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bc1c4a3e-5ebf-4814-a896-dddfd826853f	2015-02-20 16:22:52.037234
718	126	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	91b12453-49af-400f-a1da-1a7de139cc9c	2015-02-20 16:22:52.057375
719	127	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	da39f9ac-ad6e-4ee2-9fe2-6a955e8babfc	2015-02-20 16:22:52.078713
720	128	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	53dd5733-913e-4b46-bc52-cb03e9eed622	2015-02-20 16:22:52.098203
721	129	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	77340531-c600-4ef7-b118-2926d39885d7	2015-02-20 16:22:52.118685
722	130	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2eabd379-d50c-4197-aaf7-b1b6efa0abf5	2015-02-20 16:22:52.139781
723	131	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d252674b-95a1-4f7d-b726-aec1333495be	2015-02-20 16:22:52.159444
724	132	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	30b87721-e189-4bc5-9647-147d7afa6146	2015-02-20 16:22:52.180753
725	133	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ed786ced-02cd-41eb-8826-2173f29d75eb	2015-02-20 16:22:52.216225
726	134	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6faf785d-4e7a-43da-89c0-d6667cbb0a1f	2015-02-20 16:22:52.235082
727	135	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	32da4aaa-ade8-4a8d-b396-11545dabb88a	2015-02-20 16:22:52.262756
728	136	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c5684030-fdab-416b-8252-881309f692c6	2015-02-20 16:22:52.2835
729	137	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b1cad451-e4bf-4b97-8fc4-9cabf9589264	2015-02-20 16:22:52.305843
730	138	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	cb37e5db-3cd0-4e1e-95d6-96ba3c25a5c9	2015-02-20 16:22:52.32715
731	139	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6f231f24-b992-4889-bd36-649eaf600020	2015-02-20 16:22:52.348219
732	140	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	80454d43-7cf4-4d3c-b3e4-571f9a2f3529	2015-02-20 16:22:52.367949
733	141	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	493ab13a-5af0-4275-aa8a-cdd3777caad8	2015-02-20 16:22:52.388534
734	142	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	65ee8f1e-c0b3-4d00-8d9f-f7f78334fd5c	2015-02-20 16:22:52.408697
735	143	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	cb37db8c-495b-478b-af1c-b8d16acfb507	2015-02-20 16:22:52.429142
736	144	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a2dd5947-0d09-4c4d-ad45-0c5d5d9e14bc	2015-02-20 16:22:52.449706
737	145	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ea519118-2f2d-4f35-9518-107fad2ad57c	2015-02-20 16:22:52.475798
738	146	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e0145153-5d6e-4beb-9b47-cd39fabf268e	2015-02-20 16:22:52.495434
739	147	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	234b382b-857f-4894-bf77-b1e61eea0442	2015-02-20 16:22:52.51681
740	148	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a64db202-4d84-49dc-97ca-53786cb72fc1	2015-02-20 16:22:52.539676
741	149	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	56ab637e-5c27-42bc-8a4a-6a90f6588e5e	2015-02-20 16:22:52.561221
742	150	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bf4e1d63-e8e9-4adc-b0ee-8afa5de44831	2015-02-20 16:22:52.582943
743	151	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	93cdfb5e-91a9-40f3-80ed-891c1e226898	2015-02-20 16:22:52.602846
744	152	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e055274b-cd2a-4ec0-8c9e-3be3970c1299	2015-02-20 16:22:52.62873
745	153	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	90a4fa16-2eac-4b76-8e8d-153440dff665	2015-02-20 16:22:52.648521
746	154	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d45b448d-0ea8-45c1-afce-e3bb27d28b09	2015-02-20 16:22:52.668536
747	155	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	13df4d29-1b1c-4e0e-bb6b-66a7e8dcb0ad	2015-02-20 16:22:52.689776
748	156	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	90fde553-78f1-47c4-b509-3bdde84bd2b3	2015-02-20 16:22:52.712199
749	157	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d988dd95-f1f1-467d-aa94-a864989d4f79	2015-02-20 16:22:52.73472
750	158	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	356b41be-4e61-42ed-b91a-dc12b88bd5fb	2015-02-20 16:22:52.755484
751	159	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6f9f47fd-505b-4652-a6f6-c489bf5743c2	2015-02-20 16:22:52.779932
752	160	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	56e3bf40-246f-4d45-9523-2ad009bad44c	2015-02-20 16:22:52.799816
753	161	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f31eb98b-3eb8-49b8-b1de-4f2fe7318760	2015-02-20 16:22:52.82096
754	162	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	07bfd573-0249-484f-b4fc-7ecf50ea4169	2015-02-20 16:22:52.842703
755	163	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b038c95d-44b9-4af7-b9b6-fdf80249613e	2015-02-20 16:22:52.865008
756	164	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f782a5a9-07f5-41d2-abb2-cb2a45cfbe5a	2015-02-20 16:22:52.902029
757	165	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	768d08e7-6105-4ba6-9040-78de9c04b397	2015-02-20 16:22:52.924829
758	166	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	208b72d5-5e0d-4a8a-a055-0a271dfe4c90	2015-02-20 16:22:52.944263
759	167	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	65a8569a-2802-4590-91b6-ce437a8ae31e	2015-02-20 16:22:52.964609
760	168	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	40b1ad1d-38bf-458a-99a3-062b3e1055bd	2015-02-20 16:22:52.988
761	169	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	df4f2624-73c5-406c-a4dc-0ce4efc07d94	2015-02-20 16:22:53.010408
762	170	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	68e63310-7783-422c-ac89-8c879bb56d01	2015-02-20 16:22:53.031986
763	171	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	38f529b3-87bc-41dd-a92f-7fcfe43a2c2e	2015-02-20 16:22:53.052823
764	172	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	346dbe15-8e89-4941-81f5-67b0aba38d84	2015-02-20 16:22:53.075419
765	173	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9a475a32-001b-4975-b3d6-22b7a7ca8f32	2015-02-20 16:22:53.097432
766	174	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	11535405-6ee4-4eba-97d2-2f26233b7925	2015-02-20 16:22:53.122841
767	175	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	56f0ef35-3984-470d-833e-3d21a16eb91e	2015-02-20 16:22:53.145327
768	176	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3365f4fc-3cff-4c11-9211-15cd43a09830	2015-02-20 16:22:53.165276
769	177	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3e84145d-ddef-4cfe-887f-5ff6dc336eff	2015-02-20 16:22:53.188542
770	178	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7724495d-051d-4d3e-bd6d-536d6fe7d190	2015-02-20 16:22:53.210518
771	179	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0241175a-3394-40c2-8e69-ab6558d7bbf8	2015-02-20 16:22:53.229905
772	180	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	16de696a-69a2-4021-ab6e-02ec9fb90e33	2015-02-20 16:22:53.25811
773	181	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1547c6e1-1408-47c6-bd69-596c7ede6fac	2015-02-20 16:22:53.279873
774	182	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bd75cf1f-362a-44cc-8a6f-44e105112fed	2015-02-20 16:22:53.300457
775	183	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6874325f-0788-4086-919b-ca33ea76ed80	2015-02-20 16:22:53.321278
776	184	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b7980e80-fd03-4ee1-bff9-de61c80135b7	2015-02-20 16:22:53.339799
777	185	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	efafa007-c003-414e-bb54-6b7e513e910a	2015-02-20 16:22:53.358263
778	186	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b3201cf7-e882-4f52-8617-48f33b107d77	2015-02-20 16:22:53.383768
779	187	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c9555b04-ea04-4515-87ca-f4846696687e	2015-02-20 16:22:53.404917
780	188	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	54c064e3-a8e1-46db-9ff0-3dfe0fe70a00	2015-02-20 16:22:53.428499
781	189	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	fec10a97-5ac3-4b6d-ac24-b40be8384f3d	2015-02-20 16:22:53.451678
782	190	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ec59b4fe-14b3-4cd6-8397-c94cfd865edb	2015-02-20 16:22:53.472049
783	191	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bd50dff1-c4b4-40ce-a681-843f7a5c8e6c	2015-02-20 16:22:53.492639
784	192	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	17fef3da-c128-441f-817e-817417f158de	2015-02-20 16:22:53.513128
785	193	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d0804182-b8dd-4e9d-8e6c-16f737ddfbdb	2015-02-20 16:22:53.535574
786	194	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2a48fd5c-1483-4df4-842b-d811313b51e4	2015-02-20 16:22:53.574154
787	195	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	94bba16f-4398-4758-9a6f-2f8eaf8c3ac8	2015-02-20 16:22:53.595301
788	196	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5d4460a3-7a4e-4f57-aa42-bf865985dc37	2015-02-20 16:22:53.617324
789	197	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7c55e186-c9fe-455c-bca8-28a5704d359f	2015-02-20 16:22:53.638496
790	198	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	61325ab9-0799-4ebf-90a0-263e251e8ae7	2015-02-20 16:22:53.662089
791	199	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	4e592d70-720d-4867-9f2a-89f63e5ba3aa	2015-02-20 16:22:53.683887
792	200	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	dbb08c82-eb61-4ead-b01f-6ff75ad11b08	2015-02-20 16:22:53.703862
793	201	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d2ea1100-7542-4a4c-a9cd-56e318781ea6	2015-02-20 16:22:53.725592
794	202	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	37bdef42-a588-496a-84b6-611e1677fec5	2015-02-20 16:22:53.746285
795	203	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	da9156f4-0fcb-40bc-af62-29e466c453ec	2015-02-20 16:22:53.774965
796	204	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	17f2379c-e315-4644-97dd-40732f0377e4	2015-02-20 16:22:53.796348
797	205	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5ec97057-1005-4c59-bda4-44c49fe71636	2015-02-20 16:22:53.817454
798	206	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	efc64ad6-51d2-4999-ac25-f29df169653e	2015-02-20 16:22:53.838904
799	207	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9416e0c2-8600-4714-9113-2e91941ee9af	2015-02-20 16:22:53.859532
800	208	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ab527956-634d-415f-a651-2b85b44f6ee2	2015-02-20 16:22:53.880729
801	209	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a0ed67c1-590d-4ba9-a3c7-e907bf1a57b9	2015-02-20 16:22:53.906258
802	210	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	182dc8f7-9a24-4c99-a8b7-3838285d496a	2015-02-20 16:22:53.926648
803	211	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	911a1080-115b-429e-b760-4a4bae9efc26	2015-02-20 16:22:53.946611
804	212	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	370e2066-3f65-47f4-a9d5-a58542476419	2015-02-20 16:22:53.966553
805	213	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b161e9f3-690e-471c-afae-02e7f0d9ab9f	2015-02-20 16:22:53.985987
806	214	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	83f566bf-6bef-4dff-9ee0-49634a5606f5	2015-02-20 16:22:54.008383
807	215	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e47d0afb-610f-4a96-91b7-13423d114bba	2015-02-20 16:22:54.030193
808	216	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	23e3d9e9-ac4f-4fcd-86ae-8268fdc827e4	2015-02-20 16:22:54.050481
809	217	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	282b5e24-3fd8-4451-aecf-ed7ece24036e	2015-02-20 16:22:54.071198
810	218	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	901fa8ad-d657-46ce-b428-6efd0ea6ebcb	2015-02-20 16:22:54.092757
811	219	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5d94c73f-a311-45be-8e72-b4c75f1b0eca	2015-02-20 16:22:54.113045
812	220	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	74ab8891-3bf4-4949-8c3b-ef14f79a2b63	2015-02-20 16:22:54.134602
813	221	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c83b0c49-ea44-4304-a3b5-bfa6d25043ac	2015-02-20 16:22:54.154947
814	222	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5ab62eff-7973-4cb2-bd2a-ccc2d1674cdc	2015-02-20 16:22:54.174776
815	223	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9ab69867-078f-45f8-b341-00fe7f2927ea	2015-02-20 16:22:54.196518
816	224	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2a7f5eda-2b3b-433e-be67-9376b8af7dcc	2015-02-20 16:22:54.237735
817	225	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	577850fb-5fa2-4420-8365-74756ef54982	2015-02-20 16:22:54.258465
818	226	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1ccd2866-5a2c-40be-97e9-1d146f1bdbc0	2015-02-20 16:22:54.28349
819	227	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	46507083-f831-4359-8ea0-6f4ac0acec8e	2015-02-20 16:22:54.304035
820	228	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c597a629-6be0-454b-9260-c7e93c43b54f	2015-02-20 16:22:54.32577
821	229	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9887b986-433a-49e2-af05-e359a08b83fd	2015-02-20 16:22:54.347136
822	230	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	aa1cbc6c-d1a3-4b6b-af9a-6c89c1691ae1	2015-02-20 16:22:54.367605
823	231	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3c4206b3-7630-429c-afe2-aa883e825bd9	2015-02-20 16:22:54.392804
824	232	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	4d1ed6e6-8590-4b05-b2ff-46d47fe4bdec	2015-02-20 16:22:54.414509
825	233	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	cdbf0fee-8f20-4158-927a-9c98f0460f5e	2015-02-20 16:22:54.440691
826	234	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2404fa2f-260a-437a-b8bc-16d3abae82dc	2015-02-20 16:22:54.46053
827	235	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c6d9b691-e8cd-4eb3-86bc-db321628f3d2	2015-02-20 16:22:54.481724
828	236	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	499791cd-6cdb-4bfd-838b-bc0cd1efc8fd	2015-02-20 16:22:54.502412
829	237	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	616202e8-347e-4411-92b5-cb36e158579f	2015-02-20 16:22:54.523858
830	238	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bf658351-0dd0-49c7-a74f-76a810366eba	2015-02-20 16:22:54.544168
831	239	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1e815b18-12e8-4597-b93f-745590cd3b0c	2015-02-20 16:22:54.569538
832	240	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	fd6eadd6-face-449e-bc41-27c1ea307e78	2015-02-20 16:22:54.589627
833	241	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	29409769-ee6a-4af3-8479-44d4bf4c568e	2015-02-20 16:22:54.612226
834	242	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	24d23600-c016-4f97-8af1-5ccabf9c3858	2015-02-20 16:22:54.634979
835	243	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7bc0554b-ad82-4bad-ab6f-e96df872f996	2015-02-20 16:22:54.658765
836	244	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	007bfec6-c2da-4ac4-8c25-98304f5a6260	2015-02-20 16:22:54.680203
837	245	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	26e3bbb0-dee5-46bb-bdf3-51bfe6b52b83	2015-02-20 16:22:54.699986
838	246	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c817e6ad-9616-4f6b-96e4-cd7821c0c933	2015-02-20 16:22:54.721875
839	247	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	849fdebf-0f09-484c-b535-ad91953fc0b6	2015-02-20 16:22:54.741376
840	248	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	906cbd08-4aa1-4ad2-ae01-a3ce69199b05	2015-02-20 16:22:54.764021
841	249	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7b311403-9272-40cd-8d9f-03cf3aa5edd5	2015-02-20 16:22:54.786336
842	250	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	39fee978-448f-49bd-b712-7a7862242464	2015-02-20 16:22:54.806026
843	251	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ed36220b-8471-40b2-8e51-40a91c046f20	2015-02-20 16:22:54.826522
844	252	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d1322465-cade-4042-8b45-b0e794268251	2015-02-20 16:22:54.847311
845	253	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	379f8a13-05a6-471e-b761-a87fb916adcc	2015-02-20 16:22:54.869257
846	254	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	713a0e1b-a3b5-4615-9161-86bc0f4ca020	2015-02-20 16:22:54.912335
847	255	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	97724949-1985-4019-80ba-20ac032ea8e7	2015-02-20 16:22:54.934359
848	256	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bcb0b154-9ba2-4141-877e-55bc9178a8ab	2015-02-20 16:22:54.955417
849	257	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3da8e1e6-7d73-48fb-a11b-a214f032bae0	2015-02-20 16:22:54.979019
850	258	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	604d697d-66b9-4673-9470-3e255ccfdf55	2015-02-20 16:22:55.00171
851	259	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2d096b2b-ef90-4ec0-9fe8-8a2a2ea87669	2015-02-20 16:22:55.02552
852	260	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b715891b-8222-42ec-9300-10667687336a	2015-02-20 16:22:55.044627
853	261	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d9f5035d-3f41-489c-80ee-9dae804b9c1c	2015-02-20 16:22:55.07349
854	262	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9333548f-5f03-4352-afe8-d352716f0c95	2015-02-20 16:22:55.094158
855	263	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d11cfaeb-b3ec-489a-91a0-99ead54cd386	2015-02-20 16:22:55.114432
856	264	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b358bced-59bd-466e-83df-3541439ab7cc	2015-02-20 16:22:55.136507
857	265	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7e75bd59-ef64-4689-9783-2a29a2bd73a2	2015-02-20 16:22:55.1579
858	266	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2f7103ed-adeb-4223-a0e9-6d245f4322a5	2015-02-20 16:22:55.184754
859	267	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d8c4792f-b07d-4f74-9cf6-c2a420a43884	2015-02-20 16:22:55.205844
860	268	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	747d557f-455b-4a7f-9814-d2312b37beea	2015-02-20 16:22:55.225885
861	269	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c51a0bfc-c9ba-44d5-aa9b-ebce0185fccc	2015-02-20 16:22:55.247203
862	270	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	25484d49-c667-4273-be9a-dea7dddedec0	2015-02-20 16:22:55.268412
863	271	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8d250d42-cc19-4fe0-a15c-87c870cfbc56	2015-02-20 16:22:55.291474
864	272	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5d77d96c-e90f-4949-9cb3-523e5ad1074d	2015-02-20 16:22:55.311396
865	273	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	497292ec-946f-4e66-a6c9-3ffa899bc6fa	2015-02-20 16:22:55.333068
866	274	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	be03a0e2-9028-4a32-b8c8-726072763778	2015-02-20 16:22:55.353314
867	275	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2c22aad5-d4f6-4f5c-a816-c3d1a564ac2b	2015-02-20 16:22:55.374084
868	276	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c5653aa1-b716-4d30-ad92-85ab740416a0	2015-02-20 16:22:55.395012
869	277	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f1261bc2-cb51-489e-8252-eed3ea512bf4	2015-02-20 16:22:55.415595
870	278	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5ca52641-cb0b-47e8-a0a5-be18ab034f0c	2015-02-20 16:22:55.43918
871	279	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1925a96b-d83a-4c0a-b1e7-d00a6af8dfdf	2015-02-20 16:22:55.460239
872	280	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	15d45e18-fbc8-4532-8bd7-d2f5e7b2cf3f	2015-02-20 16:22:55.480752
873	281	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	4b07c1b9-d5ab-4cc3-86fa-9aa78df4d121	2015-02-20 16:22:55.502499
874	282	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	958e8219-ab44-4de9-abea-52da48d841d4	2015-02-20 16:22:55.53794
875	283	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d90bc86b-6814-449d-9dc1-013c56dd70ea	2015-02-20 16:22:55.558256
876	284	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9c9db56d-0f73-469a-96a1-15e25bed8617	2015-02-20 16:22:55.577472
877	285	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	99afe799-f3cd-4610-a976-a936664c6899	2015-02-20 16:22:55.597839
878	286	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	714238cd-ea05-42e3-9871-9c0ac7be7709	2015-02-20 16:22:55.619138
879	287	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	72502b69-46ed-4b70-aec4-0b1247265477	2015-02-20 16:22:55.644603
880	288	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	970b61e4-a8d2-4bc7-9ca7-729745237202	2015-02-20 16:22:55.663919
881	289	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	35f9f2fe-8cc8-4453-be70-c0e0d1db32c4	2015-02-20 16:22:55.690948
882	290	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1a104eba-0b75-4146-91fe-fa217efbd71e	2015-02-20 16:22:55.710868
883	291	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b0e8448e-a992-4339-ab2a-ad4e2e7bf1c4	2015-02-20 16:22:55.731229
884	292	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0eeafac8-083f-43a9-bbf0-c25460cdcf4e	2015-02-20 16:22:55.750184
885	293	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a63e5e32-5354-4637-99b9-25776b395092	2015-02-20 16:22:55.773861
886	294	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	89c7feaf-3364-4edc-80ca-a8545a4b1279	2015-02-20 16:22:55.795305
887	295	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	64c0e3ac-6fb6-41d6-a6ad-df4f381efc90	2015-02-20 16:22:55.815394
888	296	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9812fc54-d0d9-4c21-a5c5-770b7475a729	2015-02-20 16:22:55.838919
889	297	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	32b2079e-4944-43e4-bc3e-30060a251e9c	2015-02-20 16:22:55.859581
890	298	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d4560a3e-63bb-4846-a944-6f99ba9ef2fc	2015-02-20 16:22:55.881269
891	299	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a1efb94f-7763-4733-b1d3-ecba63f0501d	2015-02-20 16:22:55.903609
892	300	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	cb8a0a85-78d7-42c0-b68d-52b1c821af60	2015-02-20 16:22:55.926814
893	301	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a1557a93-4fc8-47a1-8aa0-45c4a40b9b64	2015-02-20 16:22:55.945688
894	302	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	dac45183-a2d5-4d12-8f03-c1a39fee7b2b	2015-02-20 16:22:55.967763
895	303	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	93f90e80-6061-4652-878a-f9c2b703f521	2015-02-20 16:22:55.987612
896	304	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f9649233-2ba9-4430-b005-49691fd3baaa	2015-02-20 16:22:56.009018
897	305	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1f6f5225-6719-4097-9425-dfe49d9dead3	2015-02-20 16:22:56.029845
898	306	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a90fce62-bbd6-497b-92fb-5ff1764ff999	2015-02-20 16:22:56.050322
899	307	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1dae878c-4007-4c35-9f67-7a11be9426a7	2015-02-20 16:22:56.072115
900	308	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	db912559-c325-45b8-a681-0ddd499a2c1a	2015-02-20 16:22:56.090748
901	309	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	59759f4d-59ef-49f3-9252-03e5d55e23d4	2015-02-20 16:22:56.119657
902	310	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0711bfcf-4c32-463d-bd2c-bfe1609281c7	2015-02-20 16:22:56.141186
903	311	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	eea31222-a46d-4065-95d6-80a561f0922b	2015-02-20 16:22:56.179952
904	312	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ce18c123-5491-4ece-b24b-4fb586a656cc	2015-02-20 16:22:56.203276
905	313	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	cb90aad9-c8a0-4bab-a5c4-14ec94cbdea9	2015-02-20 16:22:56.229455
906	314	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ea6a967a-9689-4206-97a3-e028f788b712	2015-02-20 16:22:56.252779
907	315	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	acb0a1ca-4ee4-49ac-833d-78d4073126ed	2015-02-20 16:22:56.274464
908	316	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9a7cc189-85d7-4cc3-bc84-063047410cb6	2015-02-20 16:22:56.295588
909	317	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	def797a4-95d6-44d2-99af-80c8a2588a5b	2015-02-20 16:22:56.320223
910	318	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1d16bff0-94ed-4529-9de7-396c1d27ec25	2015-02-20 16:22:56.34095
911	319	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	09684992-5838-439c-94db-664fac84e58c	2015-02-20 16:22:56.362513
912	320	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6daf6093-2315-4917-8d72-b0b64e137ff6	2015-02-20 16:22:56.381737
913	321	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	84fff04d-5c6c-4743-a077-d84ebbe40fba	2015-02-20 16:22:56.405379
914	322	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2cb13276-70d3-4216-bbf5-9b485e4faa73	2015-02-20 16:22:56.425366
915	323	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	47421135-abd1-4a6b-881c-603ee3657add	2015-02-20 16:22:56.4464
916	324	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	60203c77-b47d-4c0c-bf45-275f9d78f7d8	2015-02-20 16:22:56.467958
917	325	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8bd947c3-4f8c-4f62-b43c-74071e9794b6	2015-02-20 16:22:56.488684
918	326	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a474a367-f099-4260-a671-c5e755ba2c38	2015-02-20 16:22:56.513667
919	327	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1aac2355-ecb5-4754-afc7-c4bcf3aa77db	2015-02-20 16:22:56.534946
920	328	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9d4c08db-2f4b-4e96-b2a0-604f8733ee73	2015-02-20 16:22:56.555503
921	329	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f5379104-1346-4f68-96b4-72ae6dc6f92e	2015-02-20 16:22:56.577672
922	330	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0c2da3ad-10a9-4b23-9b19-53d93b7749f6	2015-02-20 16:22:56.59961
923	331	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c21dcc58-87d2-41b1-9dc5-6ea44856fd96	2015-02-20 16:22:56.621578
924	332	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	201f404e-68b0-499a-8e86-376bc314f2d5	2015-02-20 16:22:56.64352
925	333	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a0c3e68b-906c-4746-8da9-dcd5ea3a1f7e	2015-02-20 16:22:56.667402
926	334	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b01743b8-ac6e-43f4-90fb-4e4bbb7c97d4	2015-02-20 16:22:56.687665
927	335	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	5ef2b727-e2e4-47b6-9af0-46ea6ef88428	2015-02-20 16:22:56.712067
928	336	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	07113635-1639-4b24-9267-07b4e4ef273e	2015-02-20 16:22:56.73363
929	337	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c58bffa1-f5c8-444f-8e5d-3ac5bba2f4a5	2015-02-20 16:22:56.757675
930	338	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	7e9f735c-caee-4689-844c-16eafc0de2a9	2015-02-20 16:22:56.798152
931	339	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	cb561afa-131b-452a-99e5-060d77917da6	2015-02-20 16:22:56.81743
932	340	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	e5b01a21-19c9-4385-8a94-30d777187b3a	2015-02-20 16:22:56.838614
933	341	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f69d6809-1b4f-4c5a-9866-1e03d50794ed	2015-02-20 16:22:56.862649
934	342	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	436cf0f9-2343-4963-9a98-010b75d3054b	2015-02-20 16:22:56.884357
935	343	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	2901422a-5152-4de7-97b1-a5cc3a050c05	2015-02-20 16:22:56.908634
936	344	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a9dd955a-90f7-45c2-a945-b9e549b58ace	2015-02-20 16:22:56.935827
937	345	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c2894a2e-9485-4667-841d-12e702810257	2015-02-20 16:22:56.956923
938	346	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	41814262-0ec0-42c1-b2e4-fc6276145534	2015-02-20 16:22:56.980084
939	347	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	99c32ceb-780d-4ca4-a798-f2f0c8637f2e	2015-02-20 16:22:57.003605
940	348	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b55f961f-7225-420e-a788-ade3ddb4bcd6	2015-02-20 16:22:57.02571
941	349	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f899c34c-2d61-4755-b31a-0a1722f2a4e9	2015-02-20 16:22:57.047034
942	350	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f5f8f67c-6343-4d25-88f3-a7b8f5cf0880	2015-02-20 16:22:57.068784
943	351	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6f1c6d97-27e9-4447-991d-063b07b9bb17	2015-02-20 16:22:57.089738
944	352	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ddae3c11-aef8-448d-bea4-148b8a9fe69a	2015-02-20 16:22:57.109576
945	353	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d7c6970a-c5ff-4179-8a8a-e386612bde8e	2015-02-20 16:22:57.130669
946	354	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8387d999-81d7-437a-a4ec-f382292b616c	2015-02-20 16:22:57.153066
947	355	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	06b34fbd-2f5a-4656-88af-059e34d80f69	2015-02-20 16:22:57.172866
948	356	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8f4532ac-5283-4725-a4fb-e19762666596	2015-02-20 16:22:57.194259
949	357	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	95cf4ceb-91b1-442b-954e-dab8648e5a06	2015-02-20 16:22:57.214794
950	358	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c3921f6e-e9b2-4f6a-8955-a1306c9c59a7	2015-02-20 16:22:57.23744
951	359	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6f623847-2d12-4eca-b9d7-e655dc4c3d4c	2015-02-20 16:22:57.256059
952	360	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	382cd21e-1cc1-46c8-a87c-d97cb3ac9d64	2015-02-20 16:22:57.275256
953	361	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	cd9a27fb-ba76-4bf1-a13b-39d6ed0c01a8	2015-02-20 16:22:57.29776
954	362	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	58ebe15b-9cb1-4bce-a03a-4a072f409c65	2015-02-20 16:22:57.32174
955	363	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	fbad32ef-90ea-4e1e-8481-21d3d3461830	2015-02-20 16:22:57.343615
956	364	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f0532c82-baec-4db1-b35e-6b22d217c55c	2015-02-20 16:22:57.370171
957	365	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1b3e8a1b-467e-44ac-9e43-74f993d1a0fa	2015-02-20 16:22:57.409126
958	366	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9c3c2ddb-2e13-494d-8a2d-39040034aeeb	2015-02-20 16:22:57.42949
959	367	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	8770b152-e326-4ac2-8103-cce079da1565	2015-02-20 16:22:57.448594
960	368	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	65d5f43e-e063-4046-9f9e-0864bf2b1a6f	2015-02-20 16:22:57.472259
961	369	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	580f0f6d-3815-460e-a4c1-64028350574c	2015-02-20 16:22:57.494537
962	370	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	01f6f6b1-2d75-444f-ab22-464102189f60	2015-02-20 16:22:57.515266
963	371	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	87ee0a22-cde1-4756-b3f6-ced9188a2477	2015-02-20 16:22:57.540674
964	372	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	093ef8f6-36af-4de3-aa0c-c91706aa80df	2015-02-20 16:22:57.56458
965	373	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	24cde9c3-2a94-4c7e-b6c3-c47f86c0fc04	2015-02-20 16:22:57.589605
966	374	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9597b414-4e50-4f9f-8036-d7f17d4d07b2	2015-02-20 16:22:57.611393
967	375	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	63f1a6c7-6b28-4c78-be67-3533ec20645b	2015-02-20 16:22:57.632012
968	376	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	005bcd5c-13a3-456c-b2ed-7c28330e9613	2015-02-20 16:22:57.652094
969	377	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	29929d0a-d6b1-4853-bfa5-5a0b0921f284	2015-02-20 16:22:57.672881
970	378	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1a96022b-8cde-429d-8738-fbbce9c64207	2015-02-20 16:22:57.695047
971	379	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bdafbf2a-b645-4e9b-8f3d-6e5606551050	2015-02-20 16:22:57.715164
972	380	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	1d22dd3d-f610-45aa-8bc5-cc7dfb51a4d3	2015-02-20 16:22:57.735515
973	381	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	f96a7458-0720-4913-b5b5-968ae210e3e5	2015-02-20 16:22:57.758285
974	382	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	34c41ce8-eab6-4813-8603-5d40600f0517	2015-02-20 16:22:57.778086
975	383	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	493cb5f2-13d8-4930-a511-b0662d7ba2cb	2015-02-20 16:22:57.79747
976	384	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6caf0780-b764-419a-8a33-430913eebcdf	2015-02-20 16:22:57.816595
977	385	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d687ce02-3b7d-4209-b305-cb3a18487fc3	2015-02-20 16:22:57.840209
978	386	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	14a41cdb-65de-4fb8-8f5e-ff2d58c7633d	2015-02-20 16:22:57.85987
979	387	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bc8752f0-fe92-4947-8d16-e4550212ffec	2015-02-20 16:22:57.881761
980	388	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	307496f7-22d4-43a5-94ec-87f00ccd40e6	2015-02-20 16:22:57.903345
981	389	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b0891039-ee83-4126-b0b2-ec692ce40ab3	2015-02-20 16:22:57.924237
982	390	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	6a09a84c-cdff-433e-a9ec-110652a3f568	2015-02-20 16:22:57.94726
983	391	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	357d6698-1fa4-451a-9c3a-b52cb1a07096	2015-02-20 16:22:57.982788
984	392	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	30f25752-23bd-478f-ac92-b5e085c4f981	2015-02-20 16:22:58.005288
985	393	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d35efa60-b0aa-4017-a100-27fd776421e0	2015-02-20 16:22:58.026299
986	394	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	634d1160-9936-43b1-9955-4484d0c4d8bb	2015-02-20 16:22:58.048229
987	395	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	b35f6eaf-505c-414e-9fd0-77bb67a4d42d	2015-02-20 16:22:58.07256
988	396	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3871d882-29bf-4db8-a2de-d6ff9bf809d1	2015-02-20 16:22:58.093675
989	397	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c0fa7f19-3210-44b8-83e5-e121b3aa408d	2015-02-20 16:22:58.114278
990	398	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	bfded27c-1b43-409b-8e84-b4d8092d6fdf	2015-02-20 16:22:58.134958
991	399	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3eb1f2ba-9555-4374-a46d-f9ea1e603cb5	2015-02-20 16:22:58.154764
992	400	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	a3a5ad29-d60a-4c68-bc12-e85df6f6686b	2015-02-20 16:22:58.175915
993	401	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	3f1f36db-7ba1-40e8-a412-9fc62d57557e	2015-02-20 16:22:58.200667
994	402	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d716f992-33b8-4c80-bab6-10074c0bebdd	2015-02-20 16:22:58.222372
995	403	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	d104f871-a6ae-4532-9ac4-8ab3052ec3eb	2015-02-20 16:22:58.243002
996	404	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	fb34cbd9-6b8e-447d-b912-802ecd3e545b	2015-02-20 16:22:58.262524
997	405	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	c9fdbd75-4106-463b-801d-77b333af2b21	2015-02-20 16:22:58.284087
998	406	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	918fe6ac-5fa3-4492-a430-84ee423a8d69	2015-02-20 16:22:58.30532
999	407	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	fda9978a-9397-4e3e-82d7-3d55cf452239	2015-02-20 16:22:58.326041
1000	408	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	449a99e1-1185-4f33-96cc-9b08befbf5c2	2015-02-20 16:22:58.346146
1001	409	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	fb8abdae-6709-4e71-86d9-708bc6675adc	2015-02-20 16:22:58.36459
1002	410	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	ff2a7041-714c-4126-bf1f-a3854013b905	2015-02-20 16:22:58.388953
1003	411	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	0d017a4c-a861-46e3-98db-2bf923bb7815	2015-02-20 16:22:58.40854
1004	412	EvidenceItem	\N	\N	\N	\N	\N	update	---\nstatus:\n- \n- accepted\n	2	\N	\N	9b06712d-9fcc-4de9-8a4b-f5f92715f4c6	2015-02-20 16:22:58.429375
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 1004, true);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: delayed_jobs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY delayed_jobs (id, priority, attempts, handler, last_error, run_at, locked_at, failed_at, locked_by, queue, created_at, updated_at) FROM stdin;
\.


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('delayed_jobs_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
1	9119	acute myeloid leukemia	2015-02-20 16:15:43.659956	2015-02-20 16:18:22.646389
2	8552	chronic myeloid leukemia	2015-02-20 16:15:43.802781	2015-02-20 16:18:22.872879
3	1612	breast cancer	2015-02-20 16:15:44.425661	2015-02-20 16:18:23.537607
4	3908	non-small cell lung carcinoma	2015-02-20 16:15:44.513503	2015-02-20 16:18:23.720864
5	1115	sarcoma	2015-02-20 16:15:44.65599	2015-02-20 16:18:24.064127
6	769	neuroblastoma	2015-02-20 16:15:44.724922	2015-02-20 16:18:24.263059
7	9256	colorectal cancer	2015-02-20 16:15:45.501384	2015-02-20 16:18:24.423434
8	1909	melanoma	2015-02-20 16:15:45.563458	2015-02-20 16:18:24.615221
9	9256	colorectal cancer	2015-02-20 16:15:45.851209	2015-02-20 16:18:24.780647
10	1781	thyroid cancer	2015-02-20 16:15:47.121626	2015-02-20 16:18:24.94569
11	1612	breast cancer	2015-02-20 16:15:48.031931	2015-02-20 16:18:25.288869
12	8618	oral cavity cancer	2015-02-20 16:15:48.142758	2015-02-20 16:18:25.803318
13	0050746	mantle cell lymphoma	2015-02-20 16:15:48.227109	2015-02-20 16:18:25.933743
14	10534	stomach cancer	2015-02-20 16:15:48.288091	2015-02-20 16:18:26.201712
15	715	T-cell leukemia	2015-02-20 16:15:48.600648	2015-02-20 16:18:26.393969
16	5635	gastric adenosquamous carcinoma	2015-02-20 16:15:48.747985	2015-02-20 16:18:26.546415
17	1324	lung cancer	2015-02-20 16:15:48.872895	2015-02-20 16:18:26.846854
18	0060075	estrogen-receptor positive breast cancer	2015-02-20 16:15:48.933893	2015-02-20 16:18:26.991846
19	9952	acute lymphocytic leukemia	2015-02-20 16:15:49.726664	2015-02-20 16:18:27.185984
20	5015	hepatocellular fibrolamellar carcinoma	2015-02-20 16:15:49.83809	2015-02-20 16:18:27.347873
21	3908	non-small cell lung carcinoma	2015-02-20 16:15:52.193545	2015-02-20 16:18:27.519313
22	4947	cholangiocarcinoma	2015-02-20 16:15:54.460565	2015-02-20 16:18:27.672186
23	3909	bronchogenic lung adenocarcinoma	2015-02-20 16:15:56.455699	2015-02-20 16:18:27.824352
24	3069	astrocytoma	2015-02-20 16:15:56.783251	2015-02-20 16:18:28.209392
25	3068	glioblastoma multiforme	2015-02-20 16:15:56.845976	2015-02-20 16:18:28.448467
26	3069	astrocytoma	2015-02-20 16:15:56.91602	2015-02-20 16:18:28.752581
27	0050908	myelodysplastic syndrome	2015-02-20 16:15:57.429046	2015-02-20 16:18:29.003198
28	1240	leukemia	2015-02-20 16:15:57.881395	2015-02-20 16:18:29.248037
29	10747	lymphoid leukemia	2015-02-20 16:15:57.947537	2015-02-20 16:18:29.413086
30	4960	bone marrow cancer	2015-02-20 16:15:58.007738	2015-02-20 16:18:29.59148
31	8997	polycythemia vera	2015-02-20 16:15:58.160945	2015-02-20 16:18:29.728059
32	9119	acute myeloid leukemia	2015-02-20 16:15:58.234776	2015-02-20 16:18:29.876459
33	9253	gastrointestinal stromal tumor	2015-02-20 16:15:58.307812	2015-02-20 16:18:30.014092
34	1240	leukemia	2015-02-20 16:15:59.828824	2015-02-20 16:18:30.362541
35	9538	multiple myeloma	2015-02-20 16:15:59.894633	2015-02-20 16:18:30.635214
36	9119	acute myeloid leukemia	2015-02-20 16:16:00.666942	2015-02-20 16:18:30.904091
37	3068	glioblastoma multiforme	2015-02-20 16:16:01.279962	2015-02-20 16:18:31.14501
38	9253	gastrointestinal stromal tumor	2015-02-20 16:16:04.90626	2015-02-20 16:18:31.304165
39	9119	acute myeloid leukemia	2015-02-20 16:16:06.829303	2015-02-20 16:18:31.472143
40	3973	thyroid medullary carcinoma	2015-02-20 16:16:07.181897	2015-02-20 16:18:31.648586
41	6477	invasive bladder transitional cell carcinoma	2015-02-20 16:16:08.442452	2015-02-20 16:18:31.885115
42	4007	bladder carcinoma	2015-02-20 16:16:08.57579	2015-02-20 16:18:32.173291
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 42, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Imatinib	\N	2015-02-20 16:15:43.653191	2015-02-20 16:15:43.653191
2	Arsenic trioxide	\N	2015-02-20 16:15:43.732564	2015-02-20 16:15:43.732564
3	N/A	\N	2015-02-20 16:15:43.797151	2015-02-20 16:15:43.797151
4	Nilotinib	\N	2015-02-20 16:15:44.017173	2015-02-20 16:15:44.017173
5	Dasatinib	\N	2015-02-20 16:15:44.07141	2015-02-20 16:15:44.07141
6	MK-2206	\N	2015-02-20 16:15:44.422012	2015-02-20 16:15:44.422012
7	Crizotinib	\N	2015-02-20 16:15:44.509956	2015-02-20 16:15:44.509956
8	CH5424802	\N	2015-02-20 16:15:44.720383	2015-02-20 16:15:44.720383
9	TAE684	\N	2015-02-20 16:15:44.912465	2015-02-20 16:15:44.912465
10	Sorafenib	\N	2015-02-20 16:15:45.313466	2015-02-20 16:15:45.313466
11	Trametinib	\N	2015-02-20 16:15:45.372656	2015-02-20 16:15:45.372656
12	Panitumumab	\N	2015-02-20 16:15:45.497278	2015-02-20 16:15:45.497278
13	Dabrafenib	\N	2015-02-20 16:15:45.658144	2015-02-20 16:15:45.658144
14	Cetuximab	\N	2015-02-20 16:15:45.847633	2015-02-20 16:15:45.847633
15	PD0325901	\N	2015-02-20 16:15:46.01111	2015-02-20 16:15:46.01111
16	PLX4720	\N	2015-02-20 16:15:46.063851	2015-02-20 16:15:46.063851
17	nutlin-3	\N	2015-02-20 16:15:46.202607	2015-02-20 16:15:46.202607
18	Capecitabine	\N	2015-02-20 16:15:46.274707	2015-02-20 16:15:46.274707
19	Vemurafenib	\N	2015-02-20 16:15:46.326006	2015-02-20 16:15:46.326006
20	Bevacizumab	\N	2015-02-20 16:15:46.376222	2015-02-20 16:15:46.376222
21	Dacarbazine	\N	2015-02-20 16:15:46.76343	2015-02-20 16:15:46.76343
22	Temozolomide	\N	2015-02-20 16:15:46.815683	2015-02-20 16:15:46.815683
23	AZD6244	\N	2015-02-20 16:15:47.572405	2015-02-20 16:15:47.572405
24	Palbociclib (PD-0332991)	\N	2015-02-20 16:15:48.596625	2015-02-20 16:15:48.596625
25	BYL719	\N	2015-02-20 16:15:48.930357	2015-02-20 16:15:48.930357
26	Palbociclib	\N	2015-02-20 16:15:48.995273	2015-02-20 16:15:48.995273
27	GW-2580	\N	2015-02-20 16:15:49.777842	2015-02-20 16:15:49.777842
28	Daunorubicin	\N	2015-02-20 16:15:50.03266	2015-02-20 16:15:50.03266
29	Idarubicin	\N	2015-02-20 16:15:50.09246	2015-02-20 16:15:50.09246
30	Gefitinib	\N	2015-02-20 16:15:51.834396	2015-02-20 16:15:51.834396
31	Erlotinib	\N	2015-02-20 16:15:51.891361	2015-02-20 16:15:51.891361
32	Premetrexed	\N	2015-02-20 16:15:52.383067	2015-02-20 16:15:52.383067
33	Stauroporine	\N	2015-02-20 16:15:52.442223	2015-02-20 16:15:52.442223
34	Neratinib	\N	2015-02-20 16:15:52.63248	2015-02-20 16:15:52.63248
35	Lapatinib	\N	2015-02-20 16:15:52.96732	2015-02-20 16:15:52.96732
36	lapatinib	\N	2015-02-20 16:15:53.202457	2015-02-20 16:15:53.202457
37	Tamoxifen	\N	2015-02-20 16:15:53.497861	2015-02-20 16:15:53.497861
38	Fulvestrant	\N	2015-02-20 16:15:53.546794	2015-02-20 16:15:53.546794
39	Ponatinib	\N	2015-02-20 16:15:54.456843	2015-02-20 16:15:54.456843
40	Pazopanib	\N	2015-02-20 16:15:54.534524	2015-02-20 16:15:54.534524
41	SU5614	\N	2015-02-20 16:15:54.861116	2015-02-20 16:15:54.861116
42	ATRA	\N	2015-02-20 16:15:54.920206	2015-02-20 16:15:54.920206
43	CEP701	\N	2015-02-20 16:15:55.627137	2015-02-20 16:15:55.627137
44	AG1296	\N	2015-02-20 16:15:55.684696	2015-02-20 16:15:55.684696
45	Bortezomib	\N	2015-02-20 16:15:56.452162	2015-02-20 16:15:56.452162
46	Fasudil	\N	2015-02-20 16:15:56.515142	2015-02-20 16:15:56.515142
47	TG101348	\N	2015-02-20 16:15:57.877818	2015-02-20 16:15:57.877818
48	Pegylated IFN-2a	\N	2015-02-20 16:15:58.157233	2015-02-20 16:15:58.157233
49	SU11248	\N	2015-02-20 16:15:59.472111	2015-02-20 16:15:59.472111
50	Regorafenib	\N	2015-02-20 16:15:59.587337	2015-02-20 16:15:59.587337
51	NVP-BEZ235	\N	2015-02-20 16:16:00.494212	2015-02-20 16:16:00.494212
52	ARRY-142886	\N	2015-02-20 16:16:00.542855	2015-02-20 16:16:00.542855
53	O(6)-benzylguanine	\N	2015-02-20 16:16:01.411714	2015-02-20 16:16:01.411714
54	Carmustine	\N	2015-02-20 16:16:01.485316	2015-02-20 16:16:01.485316
55	NSC348884	\N	2015-02-20 16:16:02.14177	2015-02-20 16:16:02.14177
56	Valproic acid	\N	2015-02-20 16:16:02.441876	2015-02-20 16:16:02.441876
57	17-AAG	\N	2015-02-20 16:16:04.462642	2015-02-20 16:16:04.462642
58	Temozolomide	\N	2015-02-20 16:16:04.782464	2015-02-20 16:16:04.782464
59	Crenolanib	\N	2015-02-20 16:16:04.90264	2015-02-20 16:16:04.90264
60	CH5132799	\N	2015-02-20 16:16:05.457244	2015-02-20 16:16:05.457244
61	Rapamycin	\N	2015-02-20 16:16:05.525105	2015-02-20 16:16:05.525105
62	Motesanib	\N	2015-02-20 16:16:07.178403	2015-02-20 16:16:07.178403
63	AZD1480	\N	2015-02-20 16:16:07.396459	2015-02-20 16:16:07.396459
64	Doxorubicin	\N	2015-02-20 16:16:07.698742	2015-02-20 16:16:07.698742
65	Everolimus	\N	2015-02-20 16:16:08.437951	2015-02-20 16:16:08.437951
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 65, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	B	\N	2015-02-20 16:15:43.665043	2015-02-20 16:15:43.665043
2	A	\N	2015-02-20 16:15:43.735138	2015-02-20 16:15:43.735138
3	C	\N	2015-02-20 16:15:43.873522	2015-02-20 16:15:43.873522
4	D	\N	2015-02-20 16:15:47.577291	2015-02-20 16:15:47.577291
\.


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Predictive	2015-02-20 16:15:43.662681	2015-02-20 16:15:43.662681
2	Diagnostic	2015-02-20 16:15:43.804644	2015-02-20 16:15:43.804644
3	Prognostic	2015-02-20 16:15:44.130868	2015-02-20 16:15:44.130868
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-02-20 16:15:28.848145	2015-02-20 16:15:28.848145	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-02-20 16:15:29.170769	2015-02-20 16:15:29.170769	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-02-20 16:15:29.427919	2015-02-20 16:15:29.427919	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia (CML). Cells harboring this fusion have shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, additional mutations in ABL1 have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-02-20 16:15:29.719994	2015-02-20 16:15:29.719994	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-02-20 16:15:29.970754	2015-02-20 16:15:29.970754	\N
6	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-02-20 16:15:30.21517	2015-02-20 16:15:30.21517	\N
7	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-02-20 16:15:30.426627	2015-02-20 16:15:30.426627	\N
8	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, ranging from genomic amplification to changes in promoter methylation. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-02-20 16:15:30.63421	2015-02-20 16:15:30.63421	\N
9	898	CCNE1	Cyclin E, while currenly not as widely implicated as its cyclin D counterparts, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. In lung cancer, neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their more lowly expressed counterparts.	cyclin E1	2015-02-20 16:15:30.8391	2015-02-20 16:15:30.8391	\N
10	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-02-20 16:15:31.057782	2015-02-20 16:15:31.057782	\N
11	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-02-20 16:15:31.267899	2015-02-20 16:15:31.267899	\N
12	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. One mechanism by which loss of CDKN2A can occur is by hypermethylation of the promoter region for the gene. However, the prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-02-20 16:15:31.473132	2015-02-20 16:15:31.473132	\N
13	1050	CEBPA	AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-02-20 16:15:31.696912	2015-02-20 16:15:31.696912	\N
14	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-02-20 16:15:31.905664	2015-02-20 16:15:31.905664	\N
15	5566	PRKACA	"PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity."	protein kinase, cAMP-dependent, catalytic, alpha	2015-02-20 16:15:32.133477	2015-02-20 16:15:32.133477	\N
16	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-02-20 16:15:32.363961	2015-02-20 16:15:32.363961	\N
17	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-02-20 16:15:32.670971	2015-02-20 16:15:32.670971	\N
18	2064	ERBB2	ERBB2, commonly referred to as HER2, is amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown senitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-02-20 16:15:32.920371	2015-02-20 16:15:32.920371	\N
19	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, but has also shown significance in endometrial, ovarian and other cancer types. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-02-20 16:15:33.159116	2015-02-20 16:15:33.159116	\N
20	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-02-20 16:15:33.367217	2015-02-20 16:15:33.367217	\N
21	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. Activation of these receptors can lead to the activation of the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFRs can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-02-20 16:15:33.644975	2015-02-20 16:15:33.644975	\N
22	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-02-20 16:15:33.987052	2015-02-20 16:15:33.987052	\N
23	2624	GATA2	N/A	GATA binding protein 2	2015-02-20 16:15:34.186229	2015-02-20 16:15:34.186229	\N
24	3417	IDH1	IDH1 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with specific cytogenetic abnormalities, 1p and 19q deletions.	isocitrate dehydrogenase 1 (NADP+), soluble	2015-02-20 16:15:34.42066	2015-02-20 16:15:34.42066	\N
25	3418	IDH2	N/A	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-02-20 16:15:34.657308	2015-02-20 16:15:34.657308	\N
26	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders.	Janus kinase 2	2015-02-20 16:15:34.850776	2015-02-20 16:15:34.850776	\N
27	3815	KIT	N/A	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-02-20 16:15:35.034309	2015-02-20 16:15:35.034309	\N
28	3845	KRAS	N/A	Kirsten rat sarcoma viral oncogene homolog	2015-02-20 16:15:35.257844	2015-02-20 16:15:35.257844	\N
29	5604	MAP2K1	N/A	mitogen-activated protein kinase kinase 1	2015-02-20 16:15:35.450731	2015-02-20 16:15:35.450731	\N
30	4209	MEF2D	N/A	myocyte enhancer factor 2D	2015-02-20 16:15:35.690812	2015-02-20 16:15:35.690812	\N
31	1436	CSF1R	N/A	colony stimulating factor 1 receptor	2015-02-20 16:15:35.897512	2015-02-20 16:15:35.897512	\N
32	4255	MGMT	N/A	O-6-methylguanine-DNA methyltransferase	2015-02-20 16:15:36.095591	2015-02-20 16:15:36.095591	\N
33	4869	NPM1	AML with mutated NPM1' is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-02-20 16:15:36.420921	2015-02-20 16:15:36.420921	\N
34	4893	NRAS	N/A	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-02-20 16:15:36.639644	2015-02-20 16:15:36.639644	\N
35	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating mutations, the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-02-20 16:15:36.895193	2015-02-20 16:15:36.895193	\N
36	5156	PDGFRA	N/A	platelet-derived growth factor receptor, alpha polypeptide	2015-02-20 16:15:37.137964	2015-02-20 16:15:37.137964	\N
37	5371	PML	N/A	promyelocytic leukemia	2015-02-20 16:15:37.327527	2015-02-20 16:15:37.327527	\N
38	5914	RARa	N/A	retinoic acid receptor, alpha	2015-02-20 16:15:37.515994	2015-02-20 16:15:37.515994	\N
39	5728	PTEN	N/A	phosphatase and tensin homolog	2015-02-20 16:15:37.756681	2015-02-20 16:15:37.756681	\N
40	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-02-20 16:15:38.015996	2015-02-20 16:15:38.015996	\N
41	861	RUNX1	N/A	runt-related transcription factor 1	2015-02-20 16:15:38.23672	2015-02-20 16:15:38.23672	\N
42	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-02-20 16:15:38.517796	2015-02-20 16:15:38.517796	\N
43	7157	TP53	N/A	tumor protein p53	2015-02-20 16:15:38.752748	2015-02-20 16:15:38.752748	\N
44	7248	TSC1	N/A	tuberous sclerosis 1	2015-02-20 16:15:38.940302	2015-02-20 16:15:38.940302	\N
45	7249	TSC2	N/A	tuberous sclerosis 2	2015-02-20 16:15:39.127096	2015-02-20 16:15:39.127096	\N
46	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-02-20 16:15:39.469303	2015-02-20 16:15:39.469303	\N
47	7490	WT1	N/A	Wilms tumor 1	2015-02-20 16:15:39.674964	2015-02-20 16:15:39.674964	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-02-20 16:15:28.854206	2015-02-20 16:16:10.789573
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-02-20 16:15:28.860023	2015-02-20 16:16:11.165192
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-02-20 16:15:29.173531	2015-02-20 16:16:13.496501
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-02-20 16:15:29.179426	2015-02-20 16:16:14.409617
5	15676015	\N	Lee et al., 2005, APMIS	2015-02-20 16:15:29.431636	2015-02-20 16:16:14.838808
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-02-20 16:15:29.437966	2015-02-20 16:16:15.261136
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-02-20 16:15:29.723508	2015-02-20 16:16:15.76809
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-02-20 16:15:29.729428	2015-02-20 16:16:16.185305
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-02-20 16:15:29.973882	2015-02-20 16:16:16.721111
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-02-20 16:15:29.979868	2015-02-20 16:16:17.229432
11	12432268	\N	Diehl, Cancer Biol. Ther.	2015-02-20 16:15:30.218297	2015-02-20 16:16:17.651012
12	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-02-20 16:15:30.224625	2015-02-20 16:16:18.068072
13	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-02-20 16:15:30.842163	2015-02-20 16:16:19.574077
14	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-02-20 16:15:31.061747	2015-02-20 16:16:20.049241
15	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-02-20 16:15:31.067266	2015-02-20 16:16:20.471725
16	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-02-20 16:15:31.476264	2015-02-20 16:16:21.021439
17	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-20 16:15:31.481707	2015-02-20 16:16:21.41039
18	19357394	\N	Vardiman et al., 2009, Blood	2015-02-20 16:15:31.70068	2015-02-20 16:16:21.838119
19	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-02-20 16:15:31.706132	2015-02-20 16:16:22.244097
20	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 16:15:31.710383	2015-02-20 16:16:22.675393
21	24578576	\N	Honeyman et al., 2014, Science	2015-02-20 16:15:31.909489	2015-02-20 16:16:23.080743
22	24909179	\N	Moody et al., 2014, Oncogene	2015-02-20 16:15:32.140659	2015-02-20 16:16:23.498436
23	24167195	\N	Kim et al., 2013, Blood	2015-02-20 16:15:32.367563	2015-02-20 16:16:24.024286
24	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-02-20 16:15:32.373062	2015-02-20 16:16:24.437091
25	23953842	\N	Yewale et al., 2013, Biomaterials	2015-02-20 16:15:32.674059	2015-02-20 16:16:24.896202
26	18712184	\N	Charpidou et al., In Vivo	2015-02-20 16:15:32.679484	2015-02-20 16:16:25.409094
27	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-02-20 16:15:32.923511	2015-02-20 16:16:26.862733
28	10878580	\N	Yu et al., 2000, Bioessays	2015-02-20 16:15:32.928797	2015-02-20 16:16:27.361266
29	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-02-20 16:15:33.162878	2015-02-20 16:16:27.799693
30	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-02-20 16:15:33.169424	2015-02-20 16:16:28.195853
31	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-02-20 16:15:33.370488	2015-02-20 16:16:28.612551
32	23558953	\N	Wu et al., 2013, Cancer Discov	2015-02-20 16:15:33.376799	2015-02-20 16:16:29.039024
33	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-02-20 16:15:33.990291	2015-02-20 16:16:30.85102
34	23226219	\N	Chou et al., 2012, PLoS ONE	2015-02-20 16:15:36.424474	2015-02-20 16:16:31.434696
35	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-20 16:15:38.018541	2015-02-20 16:16:33.209321
36	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-02-20 16:15:38.023878	2015-02-20 16:16:33.648362
37	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-20 16:15:38.520976	2015-02-20 16:16:34.045796
38	24136165	\N	Cazzola et al., 2013, Blood	2015-02-20 16:15:38.526633	2015-02-20 16:16:34.466257
39	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-02-20 16:15:39.472595	2015-02-20 16:16:34.88489
40	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-02-20 16:15:39.478422	2015-02-20 16:16:36.52278
41	12476305		Nimmanapalli et al., 2002, Oncogene	2015-02-20 16:15:43.657203	2015-02-20 16:16:36.969797
42	20537386		An et al., 2010, Leuk. Res.	2015-02-20 16:15:43.80058	2015-02-20 16:16:37.389811
43	25212276		Schmidt et al., 2014, Leukemia	2015-02-20 16:15:43.869733	2015-02-20 16:16:38.849727
44	15194504	Cell Line	Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.	2015-02-20 16:15:44.193457	2015-02-20 16:16:39.276472
45	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-02-20 16:15:44.423828	2015-02-20 16:16:39.706603
46	20979473	Clinical Trial	Choi et al., 2010, N. Engl. J. Med.	2015-02-20 16:15:44.511744	2015-02-20 16:16:40.215687
47	21030459		Sasaki et al., 2010, Cancer Res.	2015-02-20 16:15:44.654104	2015-02-20 16:16:41.611018
48	21575866		Sakamoto et al., 2011, Cancer Cell	2015-02-20 16:15:44.722156	2015-02-20 16:16:41.991187
49	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-02-20 16:15:44.789502	2015-02-20 16:16:42.40615
50	22072639		Bresler et al., 2011, Sci Transl Med	2015-02-20 16:15:44.853886	2015-02-20 16:16:42.836242
51	18923525		George et al., 2008, Nature	2015-02-20 16:15:44.914423	2015-02-20 16:16:43.263647
52	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-02-20 16:15:45.315434	2015-02-20 16:16:44.864054
53	23325582	Clinical trial	Peeters et al., 2013, Clin. Cancer Res.	2015-02-20 16:15:45.499145	2015-02-20 16:16:45.536422
54	21166657	Meta analysis	Lee et al., 2011, Br. J. Dermatol.	2015-02-20 16:15:45.561444	2015-02-20 16:16:46.058508
55	23020132	Clinical trial	Flaherty et al., 2012, N. Engl. J. Med.	2015-02-20 16:15:45.660031	2015-02-20 16:16:46.464642
56	23463675	Clinical trial	Ponti et al., 2013, J. Clin. Pathol.	2015-02-20 16:15:45.78042	2015-02-20 16:16:48.069664
57	19001320	In vitro study	Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-02-20 16:15:45.849444	2015-02-20 16:16:48.56433
58	23845441	In vitro study	Rad et al., 2013, Cancer Cell	2015-02-20 16:15:46.012957	2015-02-20 16:16:48.982411
59	23812671	In vitro study	Ji et al., 2013, Clin. Cancer Res.	2015-02-20 16:15:46.134308	2015-02-20 16:16:49.415456
60	22180495	In vivo study	Yang et al., 2012, Cancer Res.	2015-02-20 16:15:46.276524	2015-02-20 16:16:49.835185
61	24594804	Metaanalysis	Chen et al., 2014, PLoS ONE	2015-02-20 16:15:46.495292	2015-02-20 16:16:50.222143
62	20619739	Retrospective study	De Roock et al., 2010, Lancet Oncol.	2015-02-20 16:15:46.557995	2015-02-20 16:16:50.655903
63	24576830	In vitro study	Nissan et al., 2014, Cancer Res.	2015-02-20 16:15:46.620755	2015-02-20 16:16:51.071562
64	24586605	Retrospective study	Meckbach et al., 2014, PLoS ONE	2015-02-20 16:15:46.765449	2015-02-20 16:16:51.598716
65	24583796	Clinical trial	Menzies et al., 2014, Clin. Cancer Res.	2015-02-20 16:15:46.881957	2015-02-20 16:16:52.010088
66	24388723	Retrospective study	Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-02-20 16:15:47.003047	2015-02-20 16:16:52.428968
67	23524406	Case report	Rudin et al., 2013, J Thorac Oncol	2015-02-20 16:15:47.062777	2015-02-20 16:16:52.869416
68	24588959	Perspective study	Zhang et al., 2014, Diagn Pathol	2015-02-20 16:15:47.11973	2015-02-20 16:16:53.26883
69	24570209	Retrospective study	Crescenzi et al., 2014, Horm. Metab. Res.	2015-02-20 16:15:47.211238	2015-02-20 16:16:53.891662
70	21594703	Retrospective study	Howell et al., 2011, Ann. Surg. Oncol.	2015-02-20 16:15:47.268408	2015-02-20 16:16:54.30918
71	24354346	Retrospective study	Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-02-20 16:15:47.330691	2015-02-20 16:16:54.84173
72	24396464	Retrospective study	He et al., 2014, Oncol Lett	2015-02-20 16:15:47.454739	2015-02-20 16:16:55.249464
73	21098728	In vitro study	Corcoran et al., 2010, Sci Signal	2015-02-20 16:15:47.5747	2015-02-20 16:16:55.667365
74	23031422	Case report	Ponti et al., 2012, J Hematol Oncol	2015-02-20 16:15:47.632167	2015-02-20 16:16:56.084528
75	17070615		Gautschi et al., 2007, Lung Cancer	2015-02-20 16:15:47.848703	2015-02-20 16:16:56.502469
76	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-02-20 16:15:48.029925	2015-02-20 16:16:58.425333
77	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-02-20 16:15:48.139848	2015-02-20 16:16:58.910005
78	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-02-20 16:15:48.225276	2015-02-20 16:16:59.321784
79	10547574		Takano et al., 1999, J. Pathol.	2015-02-20 16:15:48.28617	2015-02-20 16:16:59.747863
80	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-02-20 16:15:48.536108	2015-02-20 16:17:00.15804
81	23079656		Sawai et al., 2012, Cancer Cell	2015-02-20 16:15:48.598651	2015-02-20 16:17:00.682568
82	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-02-20 16:15:48.660397	2015-02-20 16:17:01.120597
83	10224221		Donnellan et al., 1999, FASEB J.	2015-02-20 16:15:48.745	2015-02-20 16:17:01.51539
84	25002028		Vora et al., 2014, Cancer Cell	2015-02-20 16:15:48.932243	2015-02-20 16:17:01.936247
85	23898052		Logan et al., 2013, Anticancer Res.	2015-02-20 16:15:48.997108	2015-02-20 16:17:02.370548
86	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-20 16:15:49.220433	2015-02-20 16:17:02.717972
87	19965647	Retrospective study	Burnett et al., 2010, Blood	2015-02-20 16:15:49.341118	2015-02-20 16:17:03.084278
88	18450602	Retrospective study	Schlenk et al., 2008, N. Engl. J. Med.	2015-02-20 16:15:49.399491	2015-02-20 16:17:03.758342
89	14726504	Retrospective study	Frhling et al., 2004, J. Clin. Oncol.	2015-02-20 16:15:49.517591	2015-02-20 16:17:04.230951
90	24186003		Lilljebjrn et al., 2014, Leukemia	2015-02-20 16:15:49.723619	2015-02-20 16:17:04.648565
91	24578576	Retrospective study	Honeyman et al., 2014, Science	2015-02-20 16:15:49.836184	2015-02-20 16:17:05.072541
92	22490330	Retrospective study	Ribeiro et al., 2012, Blood	2015-02-20 16:15:49.908882	2015-02-20 16:17:05.496104
93	22081665	Retrospective study	LaRochelle et al., 2011, Oncotarget	2015-02-20 16:15:49.97158	2015-02-20 16:17:06.065531
94	21067377	Retrospective study	Ley et al., 2010, N. Engl. J. Med.	2015-02-20 16:15:50.23301	2015-02-20 16:17:06.435745
95	24512939	Retrospective study	El Ghannam et al., Blood Cells Mol. Dis.	2015-02-20 16:15:50.296407	2015-02-20 16:17:06.84173
96	23632886	Retrospective study	Gaidzik et al., 2013, Blood	2015-02-20 16:15:50.424227	2015-02-20 16:17:07.576992
97	22291079	Retrospective study	Marcucci et al., 2012, J. Clin. Oncol.	2015-02-20 16:15:50.809421	2015-02-20 16:17:09.161997
98	24736073		Lim et al., 2014, J Thorac Oncol	2015-02-20 16:15:51.836398	2015-02-20 16:17:09.558437
99	24457318		Fukihara et al., 2014, Oncology	2015-02-20 16:15:51.955089	2015-02-20 16:17:09.97828
100	24585406		Li et al., 2014, Ann. Surg. Oncol.	2015-02-20 16:15:52.028741	2015-02-20 16:17:10.497732
101	24662454		Douillard et al., 2014, J Thorac Oncol	2015-02-20 16:15:52.096544	2015-02-20 16:17:10.914756
102	25668228	Review	Denis et al., 2015, Clin. Chim. Acta	2015-02-20 16:15:52.190421	2015-02-20 16:17:11.437083
103	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-02-20 16:15:52.262115	2015-02-20 16:17:11.859497
104	24636847		Li et al., 2014, Lung Cancer	2015-02-20 16:15:52.330081	2015-02-20 16:17:13.431987
105	24658966		Ai et al., 2014, Amino Acids	2015-02-20 16:15:52.44411	2015-02-20 16:17:13.945369
106	24729716		Li et al., 2014, Onco Targets Ther	2015-02-20 16:15:52.507106	2015-02-20 16:17:14.369688
107	24623981		Ding et al., 2014, Onco Targets Ther	2015-02-20 16:15:52.576374	2015-02-20 16:17:14.781846
108	23220880		Bose et al., 2013, Cancer Discov	2015-02-20 16:15:52.634371	2015-02-20 16:17:15.314176
109	24185510		Robinson et al., 2013, Nat. Genet.	2015-02-20 16:15:53.499647	2015-02-20 16:17:15.827983
110	24185512		Toy et al., 2013, Nat. Genet.	2015-02-20 16:15:53.620695	2015-02-20 16:17:16.245164
111	24550739	Case study	Borad et al., 2014, PLoS Genet.	2015-02-20 16:15:54.458722	2015-02-20 16:17:16.667991
112	17957027	Retrospective study	Gale et al., 2008, Blood	2015-02-20 16:15:54.801938	2015-02-20 16:17:17.080313
113	15626738	In vitro study	Bagrintseva et al., 2005, Blood	2015-02-20 16:15:54.863958	2015-02-20 16:17:17.496724
114	19965647		Burnett et al., 2010, Blood	2015-02-20 16:15:54.922373	2015-02-20 16:17:19.107331
115	21537333		Schnittger et al., 2011, Leukemia	2015-02-20 16:15:55.109145	2015-02-20 16:17:19.793971
116	24801015		Port et al., 2014, Ann. Hematol.	2015-02-20 16:15:55.440089	2015-02-20 16:17:20.212311
117	11290608	Retrospective study	Yamamoto et al., 2001, Blood	2015-02-20 16:15:55.509113	2015-02-20 16:17:21.870338
118	18230792	In vitro study	Zhang et al., 2008, J. Natl. Cancer Inst.	2015-02-20 16:15:55.566615	2015-02-20 16:17:23.340543
119	14726387	Clinical trial	Smith et al., 2004, Blood	2015-02-20 16:15:55.628969	2015-02-20 16:17:23.738047
120	12357354	In vitro study	Tse et al., 2002, Leukemia	2015-02-20 16:15:55.686665	2015-02-20 16:17:24.116927
121	11585760	Retrospective study	Whitman et al., 2001, Cancer Res.	2015-02-20 16:15:55.764634	2015-02-20 16:17:24.497093
122	17965322	Retrospective study	Bacher et al., 2008, Blood	2015-02-20 16:15:56.015159	2015-02-20 16:17:24.935662
123	22368270	Clinical trial	Man et al., 2012, Blood	2015-02-20 16:15:56.135309	2015-02-20 16:17:25.333286
124	17940205	Retrospective study	Whitman et al., 2008, Blood	2015-02-20 16:15:56.387029	2015-02-20 16:17:25.950725
125	22624710		Barbacid, 2012, Cancer Cell	2015-02-20 16:15:56.453988	2015-02-20 16:17:26.374168
126	20805365	Retrospective study	Schnittger et al., 2010, Blood	2015-02-20 16:15:56.575766	2015-02-20 16:17:27.994643
127	20376086	Retrospective study	Ho et al., 2010, Leukemia	2015-02-20 16:15:56.63334	2015-02-20 16:17:28.48189
128	20368538	Retrospective study	Wagner et al., 2010, J. Clin. Oncol.	2015-02-20 16:15:56.717885	2015-02-20 16:17:29.196554
129	19933982	Retrospective study	Dubbink et al., 2009, Neurology	2015-02-20 16:15:56.781259	2015-02-20 16:17:29.615316
130	20127344	Retrospective study	Bleeker et al., 2010, Acta Neuropathol.	2015-02-20 16:15:56.843146	2015-02-20 16:17:30.031103
131	20494930	Retrospective study	Thol et al., 2010, Haematologica	2015-02-20 16:15:56.914076	2015-02-20 16:17:30.449275
132	20538800	Retrospective study	Abbas et al., 2010, Blood	2015-02-20 16:15:56.978628	2015-02-20 16:17:30.865659
133	21596855		Green et al., 2011, Blood	2015-02-20 16:15:57.237054	2015-02-20 16:17:31.300132
134	20421455	Retrospective study	Thol et al., 2010, Blood	2015-02-20 16:15:57.299195	2015-02-20 16:17:31.713785
135	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-02-20 16:15:57.360853	2015-02-20 16:17:32.121549
136	22033490		Patnaik et al., 2012, Leukemia	2015-02-20 16:15:57.426918	2015-02-20 16:17:32.537007
137	21997850		Lin et al., 2012, Ann. Hematol.	2015-02-20 16:15:57.487571	2015-02-20 16:17:32.955088
138	16081687	Retrospective study	Levine et al., 2005, Blood	2015-02-20 16:15:57.751458	2015-02-20 16:17:33.374046
139	18394554	In vitro study	Wernig et al., 2008, Cancer Cell	2015-02-20 16:15:57.879698	2015-02-20 16:17:33.791299
140	19287384	Retrospective study	Kilpivaara et al., 2009, Nat. Genet.	2015-02-20 16:15:58.005588	2015-02-20 16:17:34.209896
141	16709929	Clinical trial	Kiladjian et al., 2006, Blood	2015-02-20 16:15:58.159196	2015-02-20 16:17:34.642232
142	16384925	Retrospective study	Cairoli et al., 2006, Blood	2015-02-20 16:15:58.231927	2015-02-20 16:17:35.043701
143	16551858	Retrospective study	Wardelmann et al., 2006, Clin. Cancer Res.	2015-02-20 16:15:58.305971	2015-02-20 16:17:35.486467
144	10485475	Retrospective study	Taniguchi et al., 1999, Cancer Res.	2015-02-20 16:15:58.375493	2015-02-20 16:17:35.878185
145	12000708	Retrospective study	Corless et al., 2002, Am. J. Pathol.	2015-02-20 16:15:58.439562	2015-02-20 16:17:37.556609
146	15217946	Retrospective study	Boldrini et al., 2004, Clin. Cancer Res.	2015-02-20 16:15:58.574616	2015-02-20 16:17:37.968762
147	23775962	Clinical trial	Hodi et al., 2013, J. Clin. Oncol.	2015-02-20 16:15:58.639846	2015-02-20 16:17:38.492649
148	19671763	In vitro study	Woodman et al., 2009, Mol. Cancer Ther.	2015-02-20 16:15:59.041077	2015-02-20 16:17:39.997327
149	17372901	In vitro study	Antonescu et al., 2007, Int. J. Cancer	2015-02-20 16:15:59.266106	2015-02-20 16:17:40.450965
150	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-02-20 16:15:59.410005	2015-02-20 16:17:40.838966
151	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-02-20 16:15:59.474008	2015-02-20 16:17:42.662911
152	16618717	Retrospective study	Livre et al., 2006, Cancer Res.	2015-02-20 16:15:59.536882	2015-02-20 16:17:43.502307
153	24559322	Clinical trial	Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-02-20 16:15:59.589189	2015-02-20 16:17:43.920666
154	15696205	Retrospective study	Pao et al., 2005, PLoS Med.	2015-02-20 16:15:59.664771	2015-02-20 16:17:45.544023
155	19934290	Retrospective study	Ogino et al., 2009, Clin. Cancer Res.	2015-02-20 16:15:59.765423	2015-02-20 16:17:46.012265
156	24571676	Retrospective study	Andrade et al., 2014, BMC Cancer	2015-02-20 16:15:59.826229	2015-02-20 16:17:46.430162
157	18528420	Retrospective study	Chng et al., 2008, Leukemia	2015-02-20 16:15:59.892687	2015-02-20 16:17:46.847296
158	18794081	Retrospective study	Riely et al., 2008, Clin. Cancer Res.	2015-02-20 16:15:59.961494	2015-02-20 16:17:48.537439
159	11208838	Retrospective study	Schiller et al., 2001, J. Clin. Oncol.	2015-02-20 16:16:00.020438	2015-02-20 16:17:48.852246
160	15597105	Metaanalysis	Mascaux et al., 2005, Br. J. Cancer	2015-02-20 16:16:00.076126	2015-02-20 16:17:50.441136
161	18202412	In vitro study	Livre et al., 2008, J. Clin. Oncol.	2015-02-20 16:16:00.138748	2015-02-20 16:17:50.804801
162	23014527	Retrospective study	Dogan et al., 2012, Clin. Cancer Res.	2015-02-20 16:16:00.313417	2015-02-20 16:17:51.147137
163	23524406	case report	Rudin et al., 2013, J Thorac Oncol	2015-02-20 16:16:00.432174	2015-02-20 16:17:51.547142
164	19029981	In vivo study	Engelman et al., 2008, Nat. Med.	2015-02-20 16:16:00.496083	2015-02-20 16:17:51.905132
165	20978259	Clinical trial	De Roock et al., 2010, JAMA	2015-02-20 16:16:00.789345	2015-02-20 16:17:52.279036
166	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 16:16:01.00821	2015-02-20 16:17:52.697977
167	19915144	In vitro study	Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 16:16:01.066092	2015-02-20 16:17:53.104276
168	15758010	Clinical Trial	Hegi et al., 2005, N. Engl. J. Med.	2015-02-20 16:16:01.278179	2015-02-20 16:17:53.539495
169	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-02-20 16:16:01.347771	2015-02-20 16:17:54.073575
170	16076867	Retrospective study	Schnittger et al., 2005, Blood	2015-02-20 16:16:01.612616	2015-02-20 16:17:54.470871
171	16455956	Retrospective study	Thiede et al., 2006, Blood	2015-02-20 16:16:01.671801	2015-02-20 16:17:54.889216
172	19059939	Retrospective study	Schlenk et al., 2009, Haematologica	2015-02-20 16:16:01.842053	2015-02-20 16:17:55.308974
173	19587375	Retrospective study	Schnittger et al., 2009, Blood	2015-02-20 16:16:02.019267	2015-02-20 16:17:57.33688
174	21719597	In vitro study	Balusu et al., 2011, Blood	2015-02-20 16:16:02.143659	2015-02-20 16:17:57.814114
175	22417203	Retrospective study	Patel et al., 2012, N. Engl. J. Med.	2015-02-20 16:16:02.374357	2015-02-20 16:17:58.234537
176	24797300		Tassara et al., 2014, Blood	2015-02-20 16:16:02.44404	2015-02-20 16:17:58.649849
177	24927407		Ehninger et al., 2014, Blood Cancer J	2015-02-20 16:16:02.516094	2015-02-20 16:17:59.070839
178	16051734	Retrospective study	Dhner et al., 2005, Blood	2015-02-20 16:16:02.703645	2015-02-20 16:17:59.484764
179	20026798	Retrospective study	Becker et al., 2010, J. Clin. Oncol.	2015-02-20 16:16:02.805226	2015-02-20 16:17:59.904891
180	24855211		Linch et al., 2014, Blood	2015-02-20 16:16:02.873116	2015-02-20 16:18:00.393636
181	19047294	Retrospective study	Bchner et al., 2009, J. Clin. Oncol.	2015-02-20 16:16:03.061577	2015-02-20 16:18:02.250952
182	22430270	Retrospective study	Gaidzik et al., 2012, J. Clin. Oncol.	2015-02-20 16:16:03.120122	2015-02-20 16:18:02.931868
183	24859829		Tian et al., 2014, Int. J. Hematol.	2015-02-20 16:16:03.550374	2015-02-20 16:18:03.502886
184	19357394		Vardiman et al., 2009, Blood	2015-02-20 16:16:03.660382	2015-02-20 16:18:03.974389
185	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-02-20 16:16:03.935381	2015-02-20 16:18:04.392951
186	22180178	Clinical trial	Jakob et al., 2012, Cancer	2015-02-20 16:16:04.004207	2015-02-20 16:18:04.813108
187	15951308	Retrospective study	Bowen et al., 2005, Blood	2015-02-20 16:16:04.063323	2015-02-20 16:18:05.134488
188	24666267	Meta analysis	Therkildsen et al., 2014, Acta Oncol	2015-02-20 16:16:04.128287	2015-02-20 16:18:06.705242
189	16434492	Clinical trial	Bacher et al., 2006, Blood	2015-02-20 16:16:04.400664	2015-02-20 16:18:07.115032
190	18375819	Case report	Banerji et al., 2008, Mol. Cancer Ther.	2015-02-20 16:16:04.464528	2015-02-20 16:18:07.527704
191	23861977	Retrospective study	Tschandl et al., 2013, PLoS ONE	2015-02-20 16:16:04.661764	2015-02-20 16:18:08.051101
192	23569304	Clinical trial	Trunzer et al., 2013, J. Clin. Oncol.	2015-02-20 16:16:04.729352	2015-02-20 16:18:08.468377
193	21576590	Case report	Soon et al., 2011, Arch Dermatol	2015-02-20 16:16:04.784287	2015-02-20 16:18:08.883257
194	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-02-20 16:16:04.904591	2015-02-20 16:18:09.304108
195	15146165		Lasota et al., 2004, Lab. Invest.	2015-02-20 16:16:04.968599	2015-02-20 16:18:09.666438
196	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-02-20 16:16:05.108179	2015-02-20 16:18:10.066476
197	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-02-20 16:16:05.459909	2015-02-20 16:18:11.258941
198	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-02-20 16:16:05.527277	2015-02-20 16:18:11.705936
199	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-02-20 16:16:05.650692	2015-02-20 16:18:12.125841
200	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-02-20 16:16:05.841932	2015-02-20 16:18:12.541532
201	8674046		Yoshida et al., 1996, Cancer Res.	2015-02-20 16:16:06.826443	2015-02-20 16:18:12.948632
202	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-02-20 16:16:06.963282	2015-02-20 16:18:13.421419
203	22479427		Carico et al., 2012, PLoS ONE	2015-02-20 16:16:07.042143	2015-02-20 16:18:13.838987
204	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-02-20 16:16:07.180247	2015-02-20 16:18:14.332767
205	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-20 16:16:07.316774	2015-02-20 16:18:14.862339
206	23056499		Couto et al., 2012, PLoS ONE	2015-02-20 16:16:07.398794	2015-02-20 16:18:15.390855
207	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-02-20 16:16:07.504929	2015-02-20 16:18:15.792518
208	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-20 16:16:07.573174	2015-02-20 16:18:16.328018
209	22698404		Jackson et al., 2012, Cancer Cell	2015-02-20 16:16:07.701786	2015-02-20 16:18:17.050535
210	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-02-20 16:16:07.767141	2015-02-20 16:18:17.678043
211	9569050		Berns et al., 1998, Br. J. Cancer	2015-02-20 16:16:07.838405	2015-02-20 16:18:18.070415
212	22923433		Iyer et al., 2012, Science	2015-02-20 16:16:08.439828	2015-02-20 16:18:18.618691
213	19966866	Mouse model	Liang et al., 2010, Oncogene	2015-02-20 16:16:08.508943	2015-02-20 16:18:19.088563
214	22923433	Retrospective study	Iyer et al., 2012, Science	2015-02-20 16:16:08.57393	2015-02-20 16:18:19.602752
215	23029227		Qian et al., 2012, PLoS ONE	2015-02-20 16:16:08.730857	2015-02-20 16:18:20.109833
216	23861105		Wu et al., 2013, Am. J. Hematol.	2015-02-20 16:16:08.92612	2015-02-20 16:18:20.520654
217	18591546	Retrospective study	Virappane et al., 2008, J. Clin. Oncol.	2015-02-20 16:16:09.28552	2015-02-20 16:18:20.904348
218	19221039	Retrospective study	Gaidzik et al., 2009, Blood	2015-02-20 16:16:09.353188	2015-02-20 16:18:21.304708
219	18559874	Retrospective study	Paschka et al., 2008, J. Clin. Oncol.	2015-02-20 16:16:09.594066	2015-02-20 16:18:21.879601
220	19536888	Retrospective study	Renneville et al., 2009, Cancer	2015-02-20 16:16:09.729393	2015-02-20 16:18:22.392603
\.


--
-- Data for Name: variant_origins; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_origins (id, origin) FROM stdin;
1	somatic
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-02-20 16:15:39.729586	2015-02-20 16:15:39.729586
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-20 16:15:39.761763	2015-02-20 16:15:39.761763
3	4	BCR-ABL E255K	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-20 16:15:39.788742	2015-02-20 16:15:39.788742
4	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-02-20 16:15:39.821923	2015-02-20 16:15:39.821923
5	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-02-20 16:15:39.875823	2015-02-20 16:15:39.875823
6	1	C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-20 16:15:39.903408	2015-02-20 16:15:39.903408
7	1	L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-20 16:15:39.927632	2015-02-20 16:15:39.927632
8	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-02-20 16:15:39.95139	2015-02-20 16:15:39.95139
9	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-02-20 16:15:39.975276	2015-02-20 16:15:39.975276
10	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-02-20 16:15:40.003699	2015-02-20 16:15:40.003699
11	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 16:15:40.03137	2015-02-20 16:15:40.03137
12	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-02-20 16:15:40.055995	2015-02-20 16:15:40.055995
13	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-02-20 16:15:40.088518	2015-02-20 16:15:40.088518
14	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-02-20 16:15:40.111326	2015-02-20 16:15:40.111326
15	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 16:15:40.1336	2015-02-20 16:15:40.1336
16	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-20 16:15:40.156455	2015-02-20 16:15:40.156455
17	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-02-20 16:15:40.19004	2015-02-20 16:15:40.19004
18	6	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-02-20 16:15:40.22401	2015-02-20 16:15:40.22401
19	6	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-02-20 16:15:40.249788	2015-02-20 16:15:40.249788
75	27	EXON 9	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-02-20 16:15:41.903854	2015-02-20 16:15:41.903854
97	34	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-20 16:15:42.517488	2015-02-20 16:15:42.517488
20	6	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 16:15:40.280573	2015-02-20 16:15:40.280573
21	7	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 16:15:40.30882	2015-02-20 16:15:40.30882
22	7	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-20 16:15:40.338737	2015-02-20 16:15:40.338737
23	8	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-02-20 16:15:40.367228	2015-02-20 16:15:40.367228
24	9	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-02-20 16:15:40.403914	2015-02-20 16:15:40.403914
25	11	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-20 16:15:40.431526	2015-02-20 16:15:40.431526
26	10	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-20 16:15:40.461476	2015-02-20 16:15:40.461476
27	12	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-02-20 16:15:40.48638	2015-02-20 16:15:40.48638
28	13	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-20 16:15:40.52022	2015-02-20 16:15:40.52022
29	13	MUTATION	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-20 16:15:40.546498	2015-02-20 16:15:40.546498
30	31	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-20 16:15:40.59419	2015-02-20 16:15:40.59419
31	14	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-20 16:15:40.622646	2015-02-20 16:15:40.622646
32	16	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-20 16:15:40.655502	2015-02-20 16:15:40.655502
76	27	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-02-20 16:15:41.941882	2015-02-20 16:15:41.941882
33	17	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-02-20 16:15:40.680736	2015-02-20 16:15:40.680736
34	17	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-02-20 16:15:40.704551	2015-02-20 16:15:40.704551
35	18	D769H	ERBB2 D769H was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.740294	2015-02-20 16:15:40.740294
36	18	D769Y	ERBB2 D769Y was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.764186	2015-02-20 16:15:40.764186
37	18	DEL 755-759	ERBB2 in-frame deletion of 755-759 was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.788773	2015-02-20 16:15:40.788773
38	18	G309A	ERBB2 G309A was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.814687	2015-02-20 16:15:40.814687
39	18	L755S	ERBB2 L755S was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in MCF10A cell lines.	2015-02-20 16:15:40.849578	2015-02-20 16:15:40.849578
40	18	L755W	ERBB2 L755W was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.873317	2015-02-20 16:15:40.873317
41	18	P780INS	ERBB2 P780 insertion was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.906803	2015-02-20 16:15:40.906803
42	18	R678Q	ERBB2 R678Q was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.936968	2015-02-20 16:15:40.936968
43	18	R896C	ERBB2 R896C was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.961611	2015-02-20 16:15:40.961611
44	18	V777L	ERBB2 V777L was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:40.990096	2015-02-20 16:15:40.990096
45	18	V842I	ERBB2 V842I was one of the first ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-20 16:15:41.018214	2015-02-20 16:15:41.018214
46	19	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 16:15:41.046861	2015-02-20 16:15:41.046861
47	19	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 16:15:41.076811	2015-02-20 16:15:41.076811
48	19	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 16:15:41.102348	2015-02-20 16:15:41.102348
49	19	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 16:15:41.130648	2015-02-20 16:15:41.130648
50	19	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-20 16:15:41.15617	2015-02-20 16:15:41.15617
51	20	FGFR2-AFF3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.188122	2015-02-20 16:15:41.188122
52	20	FGFR2-BICC1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.22164	2015-02-20 16:15:41.22164
53	20	FGFR2-CASP7	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.248234	2015-02-20 16:15:41.248234
54	20	FGFR2-CCDC6	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.287898	2015-02-20 16:15:41.287898
55	20	FGFR2-KIAA1967	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.316764	2015-02-20 16:15:41.316764
56	20	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.351101	2015-02-20 16:15:41.351101
57	20	FGFR2-OFD1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.37693	2015-02-20 16:15:41.37693
58	20	FGFR2-SLC45A3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.408777	2015-02-20 16:15:41.408777
59	21	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-20 16:15:41.437723	2015-02-20 16:15:41.437723
60	22	ITD	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-02-20 16:15:41.463313	2015-02-20 16:15:41.463313
61	22	TKD	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-02-20 16:15:41.490056	2015-02-20 16:15:41.490056
62	23	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-02-20 16:15:41.518558	2015-02-20 16:15:41.518558
63	24	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 16:15:41.544147	2015-02-20 16:15:41.544147
64	24	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 16:15:41.574269	2015-02-20 16:15:41.574269
65	24	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 16:15:41.606925	2015-02-20 16:15:41.606925
66	24	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-20 16:15:41.640691	2015-02-20 16:15:41.640691
67	25	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-02-20 16:15:41.673748	2015-02-20 16:15:41.673748
68	25	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-02-20 16:15:41.697001	2015-02-20 16:15:41.697001
69	26	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-02-20 16:15:41.733398	2015-02-20 16:15:41.733398
70	27	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-02-20 16:15:41.764602	2015-02-20 16:15:41.764602
71	27	EXON 11	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-20 16:15:41.793031	2015-02-20 16:15:41.793031
72	27	EXON 13	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-02-20 16:15:41.8188	2015-02-20 16:15:41.8188
73	27	EXON 14	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-02-20 16:15:41.851745	2015-02-20 16:15:41.851745
74	27	EXON 17	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop of the protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-02-20 16:15:41.878845	2015-02-20 16:15:41.878845
77	27	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-02-20 16:15:41.966512	2015-02-20 16:15:41.966512
78	28	EXON 1	A study by Livre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-02-20 16:15:42.013161	2015-02-20 16:15:42.013161
79	28	EXON 2	In a study by Pan et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-02-20 16:15:42.041991	2015-02-20 16:15:42.041991
80	28	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 16:15:42.066768	2015-02-20 16:15:42.066768
81	28	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 16:15:42.097001	2015-02-20 16:15:42.097001
82	28	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 16:15:42.126067	2015-02-20 16:15:42.126067
83	28	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 16:15:42.153816	2015-02-20 16:15:42.153816
84	28	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 16:15:42.177466	2015-02-20 16:15:42.177466
85	28	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-20 16:15:42.20125	2015-02-20 16:15:42.20125
86	29	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-02-20 16:15:42.230425	2015-02-20 16:15:42.230425
87	29	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-02-20 16:15:42.253679	2015-02-20 16:15:42.253679
88	30	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-20 16:15:42.278708	2015-02-20 16:15:42.278708
89	32	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-02-20 16:15:42.301466	2015-02-20 16:15:42.301466
90	33	EXON 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-02-20 16:15:42.326655	2015-02-20 16:15:42.326655
91	33	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12 mutation, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-02-20 16:15:42.357373	2015-02-20 16:15:42.357373
92	34	EXON 1	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-02-20 16:15:42.389132	2015-02-20 16:15:42.389132
93	34	EXON 2	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-02-20 16:15:42.422803	2015-02-20 16:15:42.422803
94	34	EXON 3	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-20 16:15:42.451997	2015-02-20 16:15:42.451997
95	34	EXON 4	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-20 16:15:42.475037	2015-02-20 16:15:42.475037
96	34	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-20 16:15:42.496322	2015-02-20 16:15:42.496322
98	34	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 16:15:42.551101	2015-02-20 16:15:42.551101
99	34	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 16:15:42.578816	2015-02-20 16:15:42.578816
100	34	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-20 16:15:42.610065	2015-02-20 16:15:42.610065
101	36	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 16:15:42.637008	2015-02-20 16:15:42.637008
102	36	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 16:15:42.68126	2015-02-20 16:15:42.68126
103	36	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 16:15:42.714368	2015-02-20 16:15:42.714368
104	36	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 16:15:42.74405	2015-02-20 16:15:42.74405
105	36	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-20 16:15:42.772356	2015-02-20 16:15:42.772356
106	35	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 16:15:42.796267	2015-02-20 16:15:42.796267
107	35	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 16:15:42.820243	2015-02-20 16:15:42.820243
108	35	EXON 20	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-20 16:15:42.84602	2015-02-20 16:15:42.84602
109	35	EXON 9	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-20 16:15:42.874986	2015-02-20 16:15:42.874986
110	35	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-20 16:15:42.900007	2015-02-20 16:15:42.900007
111	37	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-02-20 16:15:42.926184	2015-02-20 16:15:42.926184
112	15	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-20 16:15:42.95048	2015-02-20 16:15:42.95048
113	39	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-02-20 16:15:42.979286	2015-02-20 16:15:42.979286
114	38	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-02-20 16:15:43.003728	2015-02-20 16:15:43.003728
115	40	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-02-20 16:15:43.029729	2015-02-20 16:15:43.029729
116	40	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-02-20 16:15:43.055816	2015-02-20 16:15:43.055816
117	42	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-02-20 16:15:43.086967	2015-02-20 16:15:43.086967
118	42	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-02-20 16:15:43.115561	2015-02-20 16:15:43.115561
119	43	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-02-20 16:15:43.140466	2015-02-20 16:15:43.140466
120	43	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-02-20 16:15:43.171198	2015-02-20 16:15:43.171198
121	43	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-02-20 16:15:43.19682	2015-02-20 16:15:43.19682
122	43	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 16:15:43.222644	2015-02-20 16:15:43.222644
123	43	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 16:15:43.249615	2015-02-20 16:15:43.249615
124	43	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 16:15:43.277407	2015-02-20 16:15:43.277407
125	43	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 16:15:43.320673	2015-02-20 16:15:43.320673
126	43	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-20 16:15:43.350714	2015-02-20 16:15:43.350714
127	44	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 16:15:43.375259	2015-02-20 16:15:43.375259
128	44	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 16:15:43.401311	2015-02-20 16:15:43.401311
129	45	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-20 16:15:43.427362	2015-02-20 16:15:43.427362
130	46	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-20 16:15:43.451124	2015-02-20 16:15:43.451124
131	46	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. The impact of U2AF1 mutations on overall survival in MDS has been debated, however, patients with U2AF1 mutations were shown to be at an increased risk of transformation to secondary AML. The presence of this mutation was not associated with a specific prognostic outcome in AML when compared to U2AF1 wildtype patients.	2015-02-20 16:15:43.478542	2015-02-20 16:15:43.478542
132	47	EXON 7	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-20 16:15:43.503819	2015-02-20 16:15:43.503819
133	47	EXON 9	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-20 16:15:43.532829	2015-02-20 16:15:43.532829
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating, status, remote_errors, remote_ids, variant_origin_id) FROM stdin;
3	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.		Supports	2	2	3	2	42	1	2015-02-20 16:15:43.806678	2015-02-20 16:22:49.297853	4	accepted	\N	\N	1
4	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.		Does Not Support	2	3	3	2	43	1	2015-02-20 16:15:43.875865	2015-02-20 16:22:49.318432	2	accepted	\N	\N	1
5	The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic sensitivity to the drug.		Supports	1	2	1	2	42	1	2015-02-20 16:15:43.952828	2015-02-20 16:22:49.339468	5	accepted	\N	\N	1
6	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.		Supports	1	1	4	2	42	1	2015-02-20 16:15:44.020348	2015-02-20 16:22:49.366403	4	accepted	\N	\N	1
7	The use of second-generation BCR-ABL targeted therapies has been effective in patients resistant to imatinib.		Supports	1	1	5	2	42	1	2015-02-20 16:15:44.075587	2015-02-20 16:22:49.386391	4	accepted	\N	\N	1
8	BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens."		Supports	3	2	1	2	42	1	2015-02-20 16:15:44.134172	2015-02-20 16:22:49.404381	5	accepted	\N	\N	1
9	COS7 cell lines transfected with BCR/ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.		Supports	1	3	1	2	44	3	2015-02-20 16:15:44.197833	2015-02-20 16:22:49.430428	4	accepted	\N	\N	1
10	In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.		Supports	1	1	1	2	42	2	2015-02-20 16:15:44.289915	2015-02-20 16:22:49.452743	4	accepted	\N	\N	1
11	COS7 cell lines transfected with BCR/ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.		Supports	1	3	1	2	44	2	2015-02-20 16:15:44.358499	2015-02-20 16:22:49.473135	4	accepted	\N	\N	1
12	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.		Does Not Support	1	1	6	3	45	4	2015-02-20 16:15:44.428464	2015-02-20 16:22:49.495754	3	accepted	\N	\N	1
14	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.		Supports	1	1	7	4	46	5	2015-02-20 16:15:44.586681	2015-02-20 16:22:49.535389	4	accepted	\N	\N	1
15	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.		Supports	1	3	7	5	47	8	2015-02-20 16:15:44.65834	2015-02-20 16:22:49.558839	3	accepted	\N	\N	1
16	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.		Supports	1	3	8	6	48	8	2015-02-20 16:15:44.727922	2015-02-20 16:22:49.578715	3	accepted	\N	\N	1
17	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.		Supports	1	3	7	6	49	8	2015-02-20 16:15:44.792267	2015-02-20 16:22:49.599512	3	accepted	\N	\N	1
18	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.		Supports	1	3	7	6	50	8	2015-02-20 16:15:44.85824	2015-02-20 16:22:49.618759	3	accepted	\N	\N	1
19	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition		Supports	1	3	9	6	51	8	2015-02-20 16:15:44.91685	2015-02-20 16:22:49.640386	4	accepted	\N	\N	1
20	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.		Supports	1	3	7	4	47	8	2015-02-20 16:15:44.977658	2015-02-20 16:22:49.662254	3	accepted	\N	\N	1
21	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation		Supports	1	3	8	4	48	7	2015-02-20 16:15:45.045443	2015-02-20 16:22:49.68299	3	accepted	\N	\N	1
22	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.		Supports	1	1	7	4	46	7	2015-02-20 16:15:45.122912	2015-02-20 16:22:49.703873	4	accepted	\N	\N	1
23	TAE684 inhibits growth of Ba/F3 cells expressing ALK mutations; however, cells with the R1275Q mutation were inhibited at a higher IC50 than those expressing F1174L.		Supports	1	3	9	6	51	9	2015-02-20 16:15:45.194035	2015-02-20 16:22:49.724449	3	accepted	\N	\N	1
24	The SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684.		Does Not Support	1	3	9	6	51	9	2015-02-20 16:15:45.256484	2015-02-20 16:22:49.748449	3	accepted	\N	\N	1
25	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.		Supports	1	1	10	4	52	10	2015-02-20 16:15:45.317988	2015-02-20 16:22:49.771832	2	accepted	\N	\N	1
27	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro		Supports	1	3	10	4	52	10	2015-02-20 16:15:45.434016	2015-02-20 16:22:49.81443	3	accepted	\N	\N	1
28	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)		Supports	1	1	12	7	53	17	2015-02-20 16:15:45.503846	2015-02-20 16:22:49.83439	3	accepted	\N	\N	1
29	BRAF mutations are associated with melanoma arising in non-chronic sun damaged skin and with superficial spreading melanoma		Supports	2	1	3	8	54	17	2015-02-20 16:15:45.565976	2015-02-20 16:22:49.856532	4	accepted	\N	\N	1
30	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone		Supports	1	1	13	8	55	17	2015-02-20 16:15:45.664213	2015-02-20 16:22:49.87787	3	accepted	\N	\N	1
31	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone		Supports	1	1	11	8	55	17	2015-02-20 16:15:45.720635	2015-02-20 16:22:49.904995	3	accepted	\N	\N	1
32	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	13	8	56	11	2015-02-20 16:15:45.783199	2015-02-20 16:22:49.925955	5	accepted	\N	\N	1
34	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	3	12	9	57	12	2015-02-20 16:15:45.900998	2015-02-20 16:22:49.964863	3	accepted	\N	\N	1
35	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	3	10	9	57	12	2015-02-20 16:15:45.948959	2015-02-20 16:22:49.996768	3	accepted	\N	\N	1
37	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation		Supports	1	3	16	9	58	12	2015-02-20 16:15:46.067386	2015-02-20 16:22:50.044773	3	accepted	\N	\N	1
38	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy		Supports	1	3	16	9	59	12	2015-02-20 16:15:46.137036	2015-02-20 16:22:50.067198	2	accepted	\N	\N	1
39	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy		Supports	1	3	17	9	59	12	2015-02-20 16:15:46.207051	2015-02-20 16:22:50.088416	2	accepted	\N	\N	1
40	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	3	18	7	60	12	2015-02-20 16:15:46.278946	2015-02-20 16:22:50.110708	2	accepted	\N	\N	1
41	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	3	19	7	60	12	2015-02-20 16:15:46.328797	2015-02-20 16:22:50.130365	2	accepted	\N	\N	1
42	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	3	20	7	60	12	2015-02-20 16:15:46.38052	2015-02-20 16:22:50.150382	2	accepted	\N	\N	1
43	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation		Supports	1	3	19	7	60	12	2015-02-20 16:15:46.435461	2015-02-20 16:22:50.176145	2	accepted	\N	\N	1
44	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as a marker of poor prognosis.		Supports	3	2	3	7	61	12	2015-02-20 16:15:46.49818	2015-02-20 16:22:50.200843	5	accepted	\N	\N	1
45	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF		Supports	1	1	14	7	62	12	2015-02-20 16:15:46.56187	2015-02-20 16:22:50.232332	3	accepted	\N	\N	1
46	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.		Does Not Support	1	3	3	8	63	12	2015-02-20 16:15:46.623401	2015-02-20 16:22:50.258099	3	accepted	\N	\N	1
47	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.		Supports	1	3	3	8	63	12	2015-02-20 16:15:46.707663	2015-02-20 16:22:50.281144	3	accepted	\N	\N	1
48	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.		Does Not Support	1	1	21	8	64	12	2015-02-20 16:15:46.768045	2015-02-20 16:22:50.300079	2	accepted	\N	\N	1
49	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.		Does Not Support	1	1	22	8	64	12	2015-02-20 16:15:46.820312	2015-02-20 16:22:50.321649	2	accepted	\N	\N	1
50	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.		Supports	1	1	13	8	65	12	2015-02-20 16:15:46.884824	2015-02-20 16:22:50.345453	5	accepted	\N	\N	1
51	Dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.		Supports	1	1	11	8	65	12	2015-02-20 16:15:46.939688	2015-02-20 16:22:50.365939	5	accepted	\N	\N	1
52	BRAF V600E is correlated with shorter disease-free and overall Survival in a Spanish cohort of melanoma patients.		Supports	3	1	3	8	66	12	2015-02-20 16:15:47.006117	2015-02-20 16:22:50.387515	3	accepted	\N	\N	1
53	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.		Supports	1	1	13	4	67	12	2015-02-20 16:15:47.066038	2015-02-20 16:22:50.408333	2	accepted	\N	\N	1
54	thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.		Supports	2	1	3	10	68	12	2015-02-20 16:15:47.125547	2015-02-20 16:22:50.430765	5	accepted	\N	\N	1
55	thyroid nodule with BRAF V600E mutation is highly correlated with papillary thyroid cancer.		Supports	2	1	3	10	69	12	2015-02-20 16:15:47.214455	2015-02-20 16:22:50.450777	5	accepted	\N	\N	1
57	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.		Does Not Support	3	1	3	10	71	12	2015-02-20 16:15:47.333374	2015-02-20 16:22:50.494755	5	accepted	\N	\N	1
58	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).		Supports	3	1	3	10	70	12	2015-02-20 16:15:47.3969	2015-02-20 16:22:50.515285	3	accepted	\N	\N	1
59	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.		Supports	3	1	3	10	72	12	2015-02-20 16:15:47.459221	2015-02-20 16:22:50.542014	3	accepted	\N	\N	1
60	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.		Supports	3	1	3	10	70	12	2015-02-20 16:15:47.51636	2015-02-20 16:22:50.565709	3	accepted	\N	\N	1
62	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.		Supports	1	1	13	8	74	13	2015-02-20 16:15:47.634746	2015-02-20 16:22:50.607532	3	accepted	\N	\N	1
63	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	13	8	56	15	2015-02-20 16:15:47.720433	2015-02-20 16:22:50.629848	5	accepted	\N	\N	1
64	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	13	8	56	16	2015-02-20 16:15:47.786444	2015-02-20 16:22:50.651451	5	accepted	\N	\N	1
65	Inceased copy number of CCND1 is associated with poorer overall survival.		Supports	3	1	3	4	75	18	2015-02-20 16:15:47.851325	2015-02-20 16:22:50.672942	3	accepted	\N	\N	1
66	Three studies have associated CCND1 expression with poorer survival.		Supports	3	1	3	4	75	19	2015-02-20 16:15:47.909175	2015-02-20 16:22:50.69846	3	accepted	\N	\N	1
68	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.		Supports	2	1	3	11	76	20	2015-02-20 16:15:48.034132	2015-02-20 16:22:50.762918	4	accepted	\N	\N	1
69	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.		Supports	3	1	3	11	76	20	2015-02-20 16:15:48.088419	2015-02-20 16:22:50.782832	4	accepted	\N	\N	1
70	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.		Supports	3	1	3	12	77	20	2015-02-20 16:15:48.145745	2015-02-20 16:22:50.802864	4	accepted	\N	\N	1
71	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.		Supports	3	1	3	13	78	20	2015-02-20 16:15:48.229378	2015-02-20 16:22:50.82216	4	accepted	\N	\N	1
72	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.		Supports	3	3	3	14	79	21	2015-02-20 16:15:48.290347	2015-02-20 16:22:50.842211	4	accepted	\N	\N	1
73	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.		Supports	3	3	3	14	79	21	2015-02-20 16:15:48.349266	2015-02-20 16:22:50.86167	4	accepted	\N	\N	1
74	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.		Supports	3	3	3	14	79	21	2015-02-20 16:15:48.41583	2015-02-20 16:22:50.883275	4	accepted	\N	\N	1
75	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.		Supports	3	3	3	14	79	21	2015-02-20 16:15:48.475739	2015-02-20 16:22:50.904883	4	accepted	\N	\N	1
77	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.		Supports	1	3	24	15	81	23	2015-02-20 16:15:48.603154	2015-02-20 16:22:50.949517	3	accepted	\N	\N	1
78	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.		Supports	3	1	3	11	82	24	2015-02-20 16:15:48.665065	2015-02-20 16:22:50.970232	4	accepted	\N	\N	1
79	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.		Supports	3	1	3	16	83	24	2015-02-20 16:15:48.750748	2015-02-20 16:22:50.99455	4	accepted	\N	\N	1
80	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.		Supports	3	1	3	16	83	24	2015-02-20 16:15:48.814008	2015-02-20 16:22:51.016257	4	accepted	\N	\N	1
81	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.		Supports	3	1	3	17	83	24	2015-02-20 16:15:48.876672	2015-02-20 16:22:51.037185	4	accepted	\N	\N	1
82	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.		Supports	1	3	25	18	84	25	2015-02-20 16:15:48.936156	2015-02-20 16:22:51.056843	4	accepted	\N	\N	1
83	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.		Supports	3	1	26	18	85	25	2015-02-20 16:15:48.999611	2015-02-20 16:22:51.079444	3	accepted	\N	\N	1
85	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.		Supports	3	1	26	18	85	26	2015-02-20 16:15:49.139237	2015-02-20 16:22:51.11697	3	accepted	\N	\N	1
86	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.		Supports	3	1	3	4	86	27	2015-02-20 16:15:49.223223	2015-02-20 16:22:51.136898	3	accepted	\N	\N	1
87	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.		Supports	3	1	3	4	86	27	2015-02-20 16:15:49.278628	2015-02-20 16:22:51.157451	3	accepted	\N	\N	1
88	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML		Does Not Support	1	1	3	1	87	29	2015-02-20 16:15:49.344463	2015-02-20 16:22:51.179108	4	accepted	\N	\N	1
89	CEBPA mutation was significantly associated with complete remission		Supports	3	1	3	1	88	29	2015-02-20 16:15:49.402443	2015-02-20 16:22:51.198049	4	accepted	\N	\N	1
90	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML		Supports	3	1	3	1	87	29	2015-02-20 16:15:49.457504	2015-02-20 16:22:51.222385	4	accepted	\N	\N	1
91	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations		Supports	3	1	3	1	89	28	2015-02-20 16:15:49.520337	2015-02-20 16:22:51.242735	3	accepted	\N	\N	1
92	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA		Supports	3	1	3	1	89	28	2015-02-20 16:15:49.572044	2015-02-20 16:22:51.262656	3	accepted	\N	\N	1
93	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA		Does Not Support	3	1	3	1	89	28	2015-02-20 16:15:49.633778	2015-02-20 16:22:51.282863	2	accepted	\N	\N	1
94	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580		Supports	1	3	1	19	90	30	2015-02-20 16:15:49.729664	2015-02-20 16:22:51.305164	3	accepted	\N	\N	1
95	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580		Supports	1	3	27	19	90	30	2015-02-20 16:15:49.780585	2015-02-20 16:22:51.326517	3	accepted	\N	\N	1
97	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A		Supports	2	1	3	1	92	32	2015-02-20 16:15:49.913191	2015-02-20 16:22:51.374281	3	accepted	\N	\N	1
98	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.		Supports	2	1	3	1	93	32	2015-02-20 16:15:49.974343	2015-02-20 16:22:51.396467	3	accepted	\N	\N	1
100	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.		Supports	1	1	29	1	93	32	2015-02-20 16:15:50.095206	2015-02-20 16:22:51.441788	4	accepted	\N	\N	1
101	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.		Does Not Support	3	1	3	1	93	32	2015-02-20 16:15:50.172104	2015-02-20 16:22:51.478609	4	accepted	\N	\N	1
102	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.		Supports	3	1	3	1	94	32	2015-02-20 16:15:50.235816	2015-02-20 16:22:51.500154	4	accepted	\N	\N	1
103	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients		Supports	2	1	3	1	95	32	2015-02-20 16:15:50.300941	2015-02-20 16:22:51.519497	3	accepted	\N	\N	1
105	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.		Does Not Support	3	1	3	1	96	32	2015-02-20 16:15:50.426885	2015-02-20 16:22:51.563017	3	accepted	\N	\N	1
106	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival		Does Not Support	3	1	3	1	96	32	2015-02-20 16:15:50.484349	2015-02-20 16:22:51.585505	3	accepted	\N	\N	1
107	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.		Supports	3	1	3	1	94	32	2015-02-20 16:15:50.544593	2015-02-20 16:22:51.607912	5	accepted	\N	\N	1
108	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.		Supports	3	1	3	1	94	32	2015-02-20 16:15:50.632559	2015-02-20 16:22:51.627288	5	accepted	\N	\N	1
109	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.		Supports	3	1	3	1	95	32	2015-02-20 16:15:50.696561	2015-02-20 16:22:51.653468	5	accepted	\N	\N	1
110	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3		Supports	3	1	3	1	92	32	2015-02-20 16:15:50.752764	2015-02-20 16:22:51.674091	4	accepted	\N	\N	1
111	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.		Supports	3	1	3	1	97	32	2015-02-20 16:15:50.812238	2015-02-20 16:22:51.694559	4	accepted	\N	\N	1
112	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.		Supports	3	1	3	1	97	32	2015-02-20 16:15:50.873598	2015-02-20 16:22:51.716347	4	accepted	\N	\N	1
113	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable, cytogenetically normal AMLs		Supports	3	1	3	1	96	32	2015-02-20 16:15:50.931547	2015-02-20 16:22:51.738015	3	accepted	\N	\N	1
114	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML		Supports	3	2	3	1	96	32	2015-02-20 16:15:50.991546	2015-02-20 16:22:51.761392	3	accepted	\N	\N	1
115	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients		Supports	2	1	3	1	96	32	2015-02-20 16:15:51.0782	2015-02-20 16:22:51.784279	3	accepted	\N	\N	1
116	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes		Supports	2	1	3	1	94	32	2015-02-20 16:15:51.141367	2015-02-20 16:22:51.803225	5	accepted	\N	\N	1
117	DNMT3A mutations (59% of which were R882) were associated with intermediate risk cytogenetics (including normal karyotype)		Supports	2	1	3	1	94	32	2015-02-20 16:15:51.205651	2015-02-20 16:22:51.825813	4	accepted	\N	\N	1
118	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients		Supports	2	1	3	1	96	32	2015-02-20 16:15:51.261939	2015-02-20 16:22:51.847426	3	accepted	\N	\N	1
119	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients		Supports	2	1	3	1	96	32	2015-02-20 16:15:51.324923	2015-02-20 16:22:51.870459	3	accepted	\N	\N	1
120	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients		Supports	3	1	3	1	96	32	2015-02-20 16:15:51.382094	2015-02-20 16:22:51.89012	3	accepted	\N	\N	1
121	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML		Does Not Support	3	1	3	1	97	32	2015-02-20 16:15:51.448659	2015-02-20 16:22:51.912542	4	accepted	\N	\N	1
122	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.		Supports	3	1	3	1	97	32	2015-02-20 16:15:51.588571	2015-02-20 16:22:51.937618	4	accepted	\N	\N	1
123	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A		Supports	3	1	3	1	92	32	2015-02-20 16:15:51.649318	2015-02-20 16:22:51.969298	3	accepted	\N	\N	1
124	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)		Supports	3	1	3	1	92	32	2015-02-20 16:15:51.711583	2015-02-20 16:22:52.004625	3	accepted	\N	\N	1
126	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.		Does Not Support	1	1	30	4	98	33	2015-02-20 16:15:51.83897	2015-02-20 16:22:52.052196	3	accepted	\N	\N	1
127	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.		Does Not Support	1	1	31	4	98	33	2015-02-20 16:15:51.894537	2015-02-20 16:22:52.073518	3	accepted	\N	\N	1
128	In NSCLC patients treated with EGFR tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.		Supports	1	1	3	4	99	33	2015-02-20 16:15:51.958481	2015-02-20 16:22:52.092498	4	accepted	\N	\N	1
130	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.		Supports	3	1	3	4	101	33	2015-02-20 16:15:52.099864	2015-02-20 16:22:52.133849	3	accepted	\N	\N	1
131	The T790M mutation in EGFR has been shown to confer resistance to the tyrosine kinase inhibitor erlotinib, and patients harboring this mutation that are placed on the drug are likely to relapse.		Supports	1	2	31	21	102	34	2015-02-20 16:15:52.195973	2015-02-20 16:22:52.154188	5	accepted	\N	\N	1
132	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.		Supports	1	1	30	4	103	34	2015-02-20 16:15:52.264727	2015-02-20 16:22:52.174271	3	accepted	\N	\N	1
133	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.		Supports	1	1	31	4	104	34	2015-02-20 16:15:52.33424	2015-02-20 16:22:52.211216	3	accepted	\N	\N	1
134	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective, reducing tumor size and resulting in stable disease.		Supports	1	1	32	4	104	34	2015-02-20 16:15:52.386651	2015-02-20 16:22:52.230269	3	accepted	\N	\N	1
135	In NSCLC containing a T790M mutation, staurosporine may be inhibitive of EGFR, especially when an L858R mutation is also present.		Supports	1	4	33	4	105	34	2015-02-20 16:15:52.448378	2015-02-20 16:22:52.256523	1	accepted	\N	\N	1
136	Patients with NSCLC harboring EGFR T790M mutation have statistically worse overall survival compared to patients with L858R or other exon 19 activating mutations.		Supports	3	1	3	4	106	34	2015-02-20 16:15:52.511587	2015-02-20 16:22:52.277706	4	accepted	\N	\N	1
137	In NSCLC patients with T790M and another activating mutations, their progression free survival is shorter compared to those who do not possess T790M mutation.		Supports	3	1	3	4	107	34	2015-02-20 16:15:52.579077	2015-02-20 16:22:52.300753	4	accepted	\N	\N	1
138	In MCF10A cell lines, the D769H mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	35	2015-02-20 16:15:52.637231	2015-02-20 16:22:52.321344	4	accepted	\N	\N	1
139	In MCF10A cell lines, the D769Y mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	36	2015-02-20 16:15:52.724182	2015-02-20 16:22:52.34273	4	accepted	\N	\N	1
141	In MCF10A cell lines, the G309A mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	38	2015-02-20 16:15:52.849202	2015-02-20 16:22:52.383151	5	accepted	\N	\N	1
142	The L755S mutation showed mild response to neratinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.		Supports	1	3	34	11	108	39	2015-02-20 16:15:52.910313	2015-02-20 16:22:52.402994	5	accepted	\N	\N	1
143	The L755S mutation was shown to confer resistance to lapatinib in MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2.		Supports	1	3	35	11	108	39	2015-02-20 16:15:52.970009	2015-02-20 16:22:52.423366	5	accepted	\N	\N	1
144	MCF10A cells transduced with ERBB2 harboring an L755W mutation were sensitive to the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.		Supports	1	3	34	11	108	40	2015-02-20 16:15:53.024258	2015-02-20 16:22:52.444409	5	accepted	\N	\N	1
145	In MCF10A cell lines, the in-frame insertion of a single amino acid at position 780 was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	41	2015-02-20 16:15:53.082775	2015-02-20 16:22:52.470037	5	accepted	\N	\N	1
146	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.		Supports	1	3	34	11	108	42	2015-02-20 16:15:53.147623	2015-02-20 16:22:52.490313	5	accepted	\N	\N	1
147	MCF10A cells transduced with ERBB2 harboring an R678Q mutation were sensitive to lapatinib and the irreversible kinase inhibitor neratinib in an matrigel colony forming assay.		Supports	1	3	36	11	108	42	2015-02-20 16:15:53.205348	2015-02-20 16:22:52.511459	5	accepted	\N	\N	1
148	In MCF10A cell lines, the R896C mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	43	2015-02-20 16:15:53.300597	2015-02-20 16:22:52.530502	5	accepted	\N	\N	1
149	In MCF10A cell lines, the V777L mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	44	2015-02-20 16:15:53.365193	2015-02-20 16:22:52.555738	5	accepted	\N	\N	1
150	In MCF10A cell lines, the V842I mutation was shown to be sensitive to neratinib.		Supports	1	3	34	11	108	45	2015-02-20 16:15:53.430349	2015-02-20 16:22:52.576878	5	accepted	\N	\N	1
151	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.		Supports	1	3	37	11	109	46	2015-02-20 16:15:53.502083	2015-02-20 16:22:52.597513	5	accepted	\N	\N	1
152	The L536Q ligand binding domain mutation have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in MCF7 cell lines.		Supports	1	3	38	11	109	46	2015-02-20 16:15:53.549537	2015-02-20 16:22:52.62277	5	accepted	\N	\N	1
153	MCF7 cell lines harboring the L536Q mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	46	2015-02-20 16:15:53.623346	2015-02-20 16:22:52.642927	3	accepted	\N	\N	1
155	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	38	11	109	47	2015-02-20 16:15:53.736103	2015-02-20 16:22:52.684708	5	accepted	\N	\N	1
156	MCF7 cell lines harboring the N538G mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	47	2015-02-20 16:15:53.828327	2015-02-20 16:22:52.70608	3	accepted	\N	\N	1
157	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	37	11	109	48	2015-02-20 16:15:53.893252	2015-02-20 16:22:52.726851	5	accepted	\N	\N	1
158	The Y537C ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	38	11	109	48	2015-02-20 16:15:53.944036	2015-02-20 16:22:52.750113	5	accepted	\N	\N	1
159	MCF7 cell lines harboring the Y537C mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	48	2015-02-20 16:15:54.007461	2015-02-20 16:22:52.774748	3	accepted	\N	\N	1
160	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	37	11	109	49	2015-02-20 16:15:54.074992	2015-02-20 16:22:52.794116	5	accepted	\N	\N	1
161	The Y537N ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	38	11	109	49	2015-02-20 16:15:54.123833	2015-02-20 16:22:52.816018	5	accepted	\N	\N	1
163	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	37	11	109	50	2015-02-20 16:15:54.275148	2015-02-20 16:22:52.859961	5	accepted	\N	\N	1
164	The Y537S ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	38	11	109	50	2015-02-20 16:15:54.33107	2015-02-20 16:22:52.897129	5	accepted	\N	\N	1
165	MCF7 cell lines harboring the L537S mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	50	2015-02-20 16:15:54.391924	2015-02-20 16:22:52.919914	3	accepted	\N	\N	1
167	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be recurrent in multiple other cancer types.		Supports	1	1	40	22	111	59	2015-02-20 16:15:54.537527	2015-02-20 16:22:52.959565	3	accepted	\N	\N	1
168	In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be recurrent in multiple other cancer types.		Supports	1	1	39	22	111	59	2015-02-20 16:15:54.586862	2015-02-20 16:22:52.979324	3	accepted	\N	\N	1
169	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	3	1	3	1	92	60	2015-02-20 16:15:54.656401	2015-02-20 16:22:53.005225	4	accepted	\N	\N	1
170	In AML patients with FLT3-ITD mutations, concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.		Supports	3	1	3	1	94	60	2015-02-20 16:15:54.718767	2015-02-20 16:22:53.026542	5	accepted	\N	\N	1
171	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)		Supports	1	1	3	1	112	60	2015-02-20 16:15:54.80473	2015-02-20 16:22:53.047154	4	accepted	\N	\N	1
172	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.		Supports	1	3	41	1	113	60	2015-02-20 16:15:54.867219	2015-02-20 16:22:53.068298	3	accepted	\N	\N	1
173	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation		Supports	1	1	42	1	114	60	2015-02-20 16:15:54.925183	2015-02-20 16:22:53.092176	3	accepted	\N	\N	1
174	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.		Supports	1	3	28	1	113	60	2015-02-20 16:15:54.990057	2015-02-20 16:22:53.117124	3	accepted	\N	\N	1
175	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate		Supports	1	1	3	1	112	60	2015-02-20 16:15:55.046465	2015-02-20 16:22:53.139018	3	accepted	\N	\N	1
176	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation		Does Not Support	3	1	3	1	115	60	2015-02-20 16:15:55.111804	2015-02-20 16:22:53.159966	4	accepted	\N	\N	1
177	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate		Does Not Support	3	1	3	1	112	60	2015-02-20 16:15:55.174552	2015-02-20 16:22:53.181323	2	accepted	\N	\N	1
178	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3		Supports	3	1	3	1	115	60	2015-02-20 16:15:55.234353	2015-02-20 16:22:53.204792	4	accepted	\N	\N	1
179	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation		Supports	3	1	3	1	115	60	2015-02-20 16:15:55.312985	2015-02-20 16:22:53.224609	4	accepted	\N	\N	1
180	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation		Supports	3	1	3	1	115	60	2015-02-20 16:15:55.379594	2015-02-20 16:22:53.251837	4	accepted	\N	\N	1
181	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD		Supports	3	2	3	1	116	60	2015-02-20 16:15:55.44269	2015-02-20 16:22:53.274481	2	accepted	\N	\N	1
182	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type		Supports	2	1	3	1	117	60	2015-02-20 16:15:55.513273	2015-02-20 16:22:53.295079	3	accepted	\N	\N	1
183	Sorafenib is effective in patients with FLT3 internal tandem repeats, but not in wild type or D835 mutation.		Supports	1	3	10	1	118	60	2015-02-20 16:15:55.571087	2015-02-20 16:22:53.316211	4	accepted	\N	\N	1
184	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations		Supports	1	1	43	1	119	60	2015-02-20 16:15:55.631987	2015-02-20 16:22:53.334904	4	accepted	\N	\N	1
185	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat		Supports	1	3	44	1	120	60	2015-02-20 16:15:55.689316	2015-02-20 16:22:53.353311	4	accepted	\N	\N	1
186	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.		Supports	3	1	3	1	121	60	2015-02-20 16:15:55.76945	2015-02-20 16:22:53.377765	4	accepted	\N	\N	1
187	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3		Supports	3	1	3	1	117	60	2015-02-20 16:15:55.83036	2015-02-20 16:22:53.400032	3	accepted	\N	\N	1
189	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.		Supports	1	3	28	1	113	61	2015-02-20 16:15:55.954849	2015-02-20 16:22:53.444443	3	accepted	\N	\N	1
190	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed		Supports	3	1	3	1	122	61	2015-02-20 16:15:56.018696	2015-02-20 16:22:53.466969	3	accepted	\N	\N	1
191	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type		Does Not Support	2	1	3	1	117	61	2015-02-20 16:15:56.07325	2015-02-20 16:22:53.486873	3	accepted	\N	\N	1
192	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance		Supports	1	1	10	1	123	61	2015-02-20 16:15:56.13927	2015-02-20 16:22:53.507822	4	accepted	\N	\N	1
194	CEP701 was effective in reducing blast counts of 5/14 heavily pre-treated AML patients with FLT3 mutations		Supports	1	1	43	1	119	61	2015-02-20 16:15:56.273465	2015-02-20 16:22:53.549616	4	accepted	\N	\N	1
195	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3		Does Not Support	3	1	3	1	117	61	2015-02-20 16:15:56.336851	2015-02-20 16:22:53.589844	3	accepted	\N	\N	1
197	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.		Supports	1	1	45	23	125	62	2015-02-20 16:15:56.45796	2015-02-20 16:22:53.633129	4	accepted	\N	\N	1
198	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.		Supports	1	1	46	23	125	62	2015-02-20 16:15:56.518463	2015-02-20 16:22:53.655957	4	accepted	\N	\N	1
199	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation		Does Not Support	3	1	3	1	126	63	2015-02-20 16:15:56.57868	2015-02-20 16:22:53.678696	3	accepted	\N	\N	1
200	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.		Supports	2	1	3	1	127	63	2015-02-20 16:15:56.636	2015-02-20 16:22:53.698593	2	accepted	\N	\N	1
201	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation		Does Not Support	3	1	3	1	128	63	2015-02-20 16:15:56.720622	2015-02-20 16:22:53.720296	2	accepted	\N	\N	1
202	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.		Supports	3	1	3	24	129	63	2015-02-20 16:15:56.785701	2015-02-20 16:22:53.740672	3	accepted	\N	\N	1
203	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation		Supports	3	1	3	25	130	63	2015-02-20 16:15:56.848998	2015-02-20 16:22:53.768076	3	accepted	\N	\N	1
204	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival		Supports	3	1	3	26	131	63	2015-02-20 16:15:56.918287	2015-02-20 16:22:53.79145	4	accepted	\N	\N	1
205	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation		Supports	2	1	3	1	132	64	2015-02-20 16:15:56.981847	2015-02-20 16:22:53.812242	3	accepted	\N	\N	1
206	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	3	1	3	1	132	64	2015-02-20 16:15:57.036045	2015-02-20 16:22:53.833073	2	accepted	\N	\N	1
207	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	3	1	3	1	132	65	2015-02-20 16:15:57.095632	2015-02-20 16:22:53.854213	3	accepted	\N	\N	1
209	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2		Supports	3	1	3	1	133	67	2015-02-20 16:15:57.24162	2015-02-20 16:22:53.900939	3	accepted	\N	\N	1
210	In patients with IDH2 R140Q/L mutation, the presence of mutation does not impact overall survival or disease free survival.		Does Not Support	3	1	3	1	134	67	2015-02-20 16:15:57.301833	2015-02-20 16:22:53.921297	3	accepted	\N	\N	1
211	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2		Does Not Support	3	4	3	1	135	67	2015-02-20 16:15:57.365663	2015-02-20 16:22:53.941819	2	accepted	\N	\N	1
212	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.		Does Not Support	3	1	3	27	136	67	2015-02-20 16:15:57.431636	2015-02-20 16:22:53.961119	3	accepted	\N	\N	1
213	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)		Does Not Support	3	4	3	27	137	67	2015-02-20 16:15:57.490229	2015-02-20 16:22:53.981191	2	accepted	\N	\N	1
214	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.		Does Not Support	3	4	3	1	135	68	2015-02-20 16:15:57.551526	2015-02-20 16:22:54.001913	2	accepted	\N	\N	1
215	In AML, patients with an IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2		Supports	3	1	3	1	133	68	2015-02-20 16:15:57.635	2015-02-20 16:22:54.023379	3	accepted	\N	\N	1
216	In patients suffering from MDS, the presence of IDH2 mutations such as R172K do not confer prognostic value (overall survival)		Does Not Support	3	4	3	27	137	68	2015-02-20 16:15:57.694503	2015-02-20 16:22:54.04523	2	accepted	\N	\N	1
217	JAK2 V617F not associatted wth  lymphoid leukemia.		Does Not Support	2	1	3	2	138	69	2015-02-20 16:15:57.754069	2015-02-20 16:22:54.06516	4	accepted	\N	\N	1
218	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).		Supports	2	1	3	2	138	69	2015-02-20 16:15:57.821066	2015-02-20 16:22:54.087663	4	accepted	\N	\N	1
219	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells		Supports	1	3	47	28	139	69	2015-02-20 16:15:57.884916	2015-02-20 16:22:54.10705	3	accepted	\N	\N	1
220	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).		Supports	2	1	3	29	138	69	2015-02-20 16:15:57.950012	2015-02-20 16:22:54.127124	4	accepted	\N	\N	1
222	In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.		Supports	1	1	48	31	141	69	2015-02-20 16:15:58.163243	2015-02-20 16:22:54.169479	4	accepted	\N	\N	1
223	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT		Supports	3	1	3	32	142	70	2015-02-20 16:15:58.237798	2015-02-20 16:22:54.190912	4	accepted	\N	\N	1
224	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.		Does Not Support	3	1	3	33	143	71	2015-02-20 16:15:58.310118	2015-02-20 16:22:54.213337	3	accepted	\N	\N	1
225	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT		Supports	2	1	3	33	144	71	2015-02-20 16:15:58.378247	2015-02-20 16:22:54.253347	2	accepted	\N	\N	1
226	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST		Does Not Support	3	1	3	33	145	71	2015-02-20 16:15:58.444033	2015-02-20 16:22:54.278205	3	accepted	\N	\N	1
227	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT		Supports	3	1	3	33	144	71	2015-02-20 16:15:58.507499	2015-02-20 16:22:54.29906	3	accepted	\N	\N	1
228	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer		Does Not Support	3	1	3	17	146	71	2015-02-20 16:15:58.577235	2015-02-20 16:22:54.319203	2	accepted	\N	\N	1
230	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	3	1	1	8	147	72	2015-02-20 16:15:58.701749	2015-02-20 16:22:54.362575	3	accepted	\N	\N	1
231	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT		Supports	3	1	3	33	143	73	2015-02-20 16:15:58.749252	2015-02-20 16:22:54.386272	3	accepted	\N	\N	1
233	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	3	1	1	8	147	74	2015-02-20 16:15:58.865258	2015-02-20 16:22:54.435621	3	accepted	\N	\N	1
234	There is no significant association between wildtype KIT or KIT mutations in exon 9 or 11 in survival among GIST patients.		Does Not Support	3	1	3	33	143	75	2015-02-20 16:15:58.931895	2015-02-20 16:22:54.455308	3	accepted	\N	\N	1
235	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer		Does Not Support	3	1	3	17	146	75	2015-02-20 16:15:58.983901	2015-02-20 16:22:54.47602	2	accepted	\N	\N	1
236	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	3	1	8	148	76	2015-02-20 16:15:59.044045	2015-02-20 16:22:54.497119	3	accepted	\N	\N	1
237	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	3	4	8	148	76	2015-02-20 16:15:59.08972	2015-02-20 16:22:54.518162	3	accepted	\N	\N	1
238	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	3	10	8	148	76	2015-02-20 16:15:59.141448	2015-02-20 16:22:54.539179	3	accepted	\N	\N	1
239	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.		Supports	1	3	5	8	148	76	2015-02-20 16:15:59.200374	2015-02-20 16:22:54.564469	3	accepted	\N	\N	1
240	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	3	5	4	149	76	2015-02-20 16:15:59.268572	2015-02-20 16:22:54.583633	3	accepted	\N	\N	1
241	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	3	1	4	149	76	2015-02-20 16:15:59.314992	2015-02-20 16:22:54.606746	3	accepted	\N	\N	1
242	Ba/F3 cells harboring the KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	3	4	4	149	76	2015-02-20 16:15:59.357674	2015-02-20 16:22:54.628583	3	accepted	\N	\N	1
243	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.		Supports	1	3	1	33	150	77	2015-02-20 16:15:59.412406	2015-02-20 16:22:54.652152	3	accepted	\N	\N	1
244	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.		Supports	1	1	49	33	151	77	2015-02-20 16:15:59.476415	2015-02-20 16:22:54.674803	3	accepted	\N	\N	1
245	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab		Supports	1	1	14	7	152	78	2015-02-20 16:15:59.539443	2015-02-20 16:22:54.694218	4	accepted	\N	\N	1
247	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib		Supports	1	1	30	4	154	79	2015-02-20 16:15:59.667512	2015-02-20 16:22:54.736373	3	accepted	\N	\N	1
248	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib		Supports	1	1	31	4	154	79	2015-02-20 16:15:59.714696	2015-02-20 16:22:54.757912	3	accepted	\N	\N	1
249	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.		Does Not Support	3	1	3	9	155	80	2015-02-20 16:15:59.768063	2015-02-20 16:22:54.781025	4	accepted	\N	\N	1
250	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.		Supports	2	1	3	34	156	80	2015-02-20 16:15:59.832769	2015-02-20 16:22:54.800963	3	accepted	\N	\N	1
251	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival		Supports	3	1	3	35	157	80	2015-02-20 16:15:59.896863	2015-02-20 16:22:54.8214	3	accepted	\N	\N	1
252	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender		Does Not Support	2	1	3	4	158	80	2015-02-20 16:15:59.963974	2015-02-20 16:22:54.840872	3	accepted	\N	\N	1
253	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC		Does Not Support	3	1	3	4	159	80	2015-02-20 16:16:00.023573	2015-02-20 16:22:54.862964	3	accepted	\N	\N	1
254	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC		Supports	3	1	3	4	160	80	2015-02-20 16:16:00.078755	2015-02-20 16:22:54.906994	3	accepted	\N	\N	1
255	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer		Supports	1	3	14	7	161	81	2015-02-20 16:16:00.141327	2015-02-20 16:22:54.929471	3	accepted	\N	\N	1
257	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer		Supports	3	3	3	7	161	81	2015-02-20 16:16:00.258489	2015-02-20 16:22:54.973109	3	accepted	\N	\N	1
258	KRAS G12C occur more frequently in women than men		Supports	2	1	3	17	162	82	2015-02-20 16:16:00.315902	2015-02-20 16:22:54.996359	2	accepted	\N	\N	1
259	KRAS G12D mutation occurs in never smokers significantly more often than in smokers		Supports	2	1	3	17	162	83	2015-02-20 16:16:00.373368	2015-02-20 16:22:55.019422	3	accepted	\N	\N	1
260	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.		Supports	1	1	13	4	163	83	2015-02-20 16:16:00.434937	2015-02-20 16:22:55.039715	2	accepted	\N	\N	1
261	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.		Supports	1	3	51	4	164	83	2015-02-20 16:16:00.498848	2015-02-20 16:22:55.065092	4	accepted	\N	\N	1
262	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in a lung cancer model with KRAS G12D mutation led to marked tumor regression.		Supports	1	3	52	4	164	83	2015-02-20 16:16:00.545986	2015-02-20 16:22:55.088119	4	accepted	\N	\N	1
264	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival		Supports	3	1	3	36	157	84	2015-02-20 16:16:00.669417	2015-02-20 16:22:55.129916	3	accepted	\N	\N	1
265	KRAS mutations in lung cancer patients are not associated with smoking history, age or gender		Does Not Support	2	1	3	4	158	84	2015-02-20 16:16:00.730588	2015-02-20 16:22:55.15278	3	accepted	\N	\N	1
267	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model		Supports	1	3	14	9	165	85	2015-02-20 16:16:00.855861	2015-02-20 16:22:55.200724	4	accepted	\N	\N	1
268	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS		Supports	1	1	14	9	165	85	2015-02-20 16:16:00.902046	2015-02-20 16:22:55.220612	4	accepted	\N	\N	1
269	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS		Supports	3	1	14	9	165	85	2015-02-20 16:16:00.954134	2015-02-20 16:22:55.241726	4	accepted	\N	\N	1
270	The MAP2K1 P124S mutation confers increased resistance to AZD6244 inhibition by about 5 fold.		Supports	1	3	23	8	166	86	2015-02-20 16:16:01.011105	2015-02-20 16:22:55.262896	3	accepted	\N	\N	1
271	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.		Supports	1	3	23	8	167	87	2015-02-20 16:16:01.070923	2015-02-20 16:22:55.284986	3	accepted	\N	\N	1
272	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580		Supports	1	3	1	19	90	88	2015-02-20 16:16:01.13069	2015-02-20 16:22:55.306167	3	accepted	\N	\N	1
273	Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580		Supports	1	3	27	19	90	88	2015-02-20 16:16:01.221079	2015-02-20 16:22:55.327873	3	accepted	\N	\N	1
274	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.		Supports	1	2	22	37	168	89	2015-02-20 16:16:01.282116	2015-02-20 16:22:55.34742	5	accepted	\N	\N	1
275	MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.		Supports	3	1	3	37	169	89	2015-02-20 16:16:01.350565	2015-02-20 16:22:55.367947	4	accepted	\N	\N	1
277	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.		Supports	1	1	54	37	169	89	2015-02-20 16:16:01.489735	2015-02-20 16:22:55.410317	4	accepted	\N	\N	1
278	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	3	1	3	1	92	90	2015-02-20 16:16:01.555273	2015-02-20 16:22:55.434122	4	accepted	\N	\N	1
279	NPM1 mutations were not associated with the M2 FAB subtype of acute myeloid leukemia.		Supports	2	1	3	1	170	90	2015-02-20 16:16:01.615167	2015-02-20 16:22:55.455181	4	accepted	\N	\N	1
280	NPM1 mutations were not associated with the M3 acute myeloid leukemia FAB subtype (0/55).		Does Not Support	2	1	3	1	171	90	2015-02-20 16:16:01.675366	2015-02-20 16:22:55.474648	3	accepted	\N	\N	1
281	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes of acute myeloid leukemia.		Supports	2	1	3	1	170	90	2015-02-20 16:16:01.731985	2015-02-20 16:22:55.496982	4	accepted	\N	\N	1
282	NPM1 mutations were associated with M5a and M5b FAB subtypes of acute myeloid leukemia		Supports	2	1	3	1	171	90	2015-02-20 16:16:01.787754	2015-02-20 16:22:55.517458	3	accepted	\N	\N	1
283	NPM1 mutations were associated with normal karyotype in older (>60) patients		Supports	2	1	3	1	172	90	2015-02-20 16:16:01.845968	2015-02-20 16:22:55.553145	3	accepted	\N	\N	1
284	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes		Supports	2	1	3	1	112	90	2015-02-20 16:16:01.9067	2015-02-20 16:22:55.572525	2	accepted	\N	\N	1
285	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status		Supports	1	1	42	1	114	90	2015-02-20 16:16:01.961715	2015-02-20 16:22:55.591951	3	accepted	\N	\N	1
286	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)		Supports	1	1	3	1	173	90	2015-02-20 16:16:02.022286	2015-02-20 16:22:55.613813	2	accepted	\N	\N	1
287	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult acute myeloid leukemia patients (median age 43)		Supports	1	1	3	1	112	90	2015-02-20 16:16:02.084801	2015-02-20 16:22:55.637989	4	accepted	\N	\N	1
288	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD		Supports	1	3	55	1	174	90	2015-02-20 16:16:02.146144	2015-02-20 16:22:55.658573	3	accepted	\N	\N	1
289	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not in cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD		Supports	1	3	42	1	174	90	2015-02-20 16:16:02.191749	2015-02-20 16:22:55.684964	3	accepted	\N	\N	1
291	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD		Supports	1	1	42	1	172	90	2015-02-20 16:16:02.312327	2015-02-20 16:22:55.726151	2	accepted	\N	\N	1
292	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.		Supports	1	1	28	1	175	90	2015-02-20 16:16:02.377503	2015-02-20 16:22:55.745303	2	accepted	\N	\N	1
293	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid		Supports	1	1	56	1	176	90	2015-02-20 16:16:02.448414	2015-02-20 16:22:55.768324	2	accepted	\N	\N	1
294	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy		Supports	1	4	3	1	177	90	2015-02-20 16:16:02.520237	2015-02-20 16:22:55.789483	2	accepted	\N	\N	1
295	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy		Supports	1	4	3	1	177	90	2015-02-20 16:16:02.586489	2015-02-20 16:22:55.810067	2	accepted	\N	\N	1
297	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy		Supports	3	1	3	1	178	90	2015-02-20 16:16:02.706643	2015-02-20 16:22:55.853298	4	accepted	\N	\N	1
299	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients		Supports	3	1	3	1	180	90	2015-02-20 16:16:02.877325	2015-02-20 16:22:55.897271	4	accepted	\N	\N	1
300	Complete remission rates were higher and both disease-free and overall survival were longer for patients with Exon 12 NPM1 mutations.		Supports	3	1	3	1	171	90	2015-02-20 16:16:02.941393	2015-02-20 16:22:55.920179	3	accepted	\N	\N	1
301	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed		Supports	3	1	3	1	122	90	2015-02-20 16:16:03.002621	2015-02-20 16:22:55.940562	3	accepted	\N	\N	1
302	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants		Supports	3	1	3	1	181	90	2015-02-20 16:16:03.064291	2015-02-20 16:22:55.962852	3	accepted	\N	\N	1
303	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients		Supports	3	1	3	1	182	90	2015-02-20 16:16:03.123063	2015-02-20 16:22:55.981901	3	accepted	\N	\N	1
304	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence of either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2		Supports	3	1	3	1	175	90	2015-02-20 16:16:03.187041	2015-02-20 16:22:56.003191	2	accepted	\N	\N	1
305	NPM1 mutation without FLT3-ITD was significantly associated with complete remission		Supports	3	1	3	1	88	90	2015-02-20 16:16:03.247239	2015-02-20 16:22:56.024374	3	accepted	\N	\N	1
306	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation		Does Not Support	3	1	3	1	126	90	2015-02-20 16:16:03.309233	2015-02-20 16:22:56.045176	3	accepted	\N	\N	1
307	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients		Supports	3	1	3	1	171	90	2015-02-20 16:16:03.371229	2015-02-20 16:22:56.065945	4	accepted	\N	\N	1
308	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3		Supports	3	1	3	1	115	90	2015-02-20 16:16:03.429552	2015-02-20 16:22:56.085832	4	accepted	\N	\N	1
309	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation		Supports	3	1	3	1	115	90	2015-02-20 16:16:03.489886	2015-02-20 16:22:56.113278	4	accepted	\N	\N	1
310	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD		Supports	3	1	3	1	183	90	2015-02-20 16:16:03.553044	2015-02-20 16:22:56.135058	3	accepted	\N	\N	1
311	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients		Supports	3	1	3	1	180	90	2015-02-20 16:16:03.605161	2015-02-20 16:22:56.17479	3	accepted	\N	\N	1
313	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)		Does Not Support	2	1	3	1	94	91	2015-02-20 16:16:03.723061	2015-02-20 16:22:56.224312	4	accepted	\N	\N	1
314	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)		Supports	2	1	3	1	94	91	2015-02-20 16:16:03.793651	2015-02-20 16:22:56.245688	4	accepted	\N	\N	1
315	NSC348884 induced apoptosis in OPI-AML3 cells harboring an NPM1 mutation.		Supports	1	3	55	1	174	91	2015-02-20 16:16:03.867004	2015-02-20 16:22:56.269554	3	accepted	\N	\N	1
316	NPM1 mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.		Supports	1	4	3	1	185	91	2015-02-20 16:16:03.938245	2015-02-20 16:22:56.290574	3	accepted	\N	\N	1
317	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS		Supports	3	1	3	8	186	92	2015-02-20 16:16:04.007039	2015-02-20 16:22:56.314473	3	accepted	\N	\N	1
318	The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)		Does Not Support	3	1	3	1	187	93	2015-02-20 16:16:04.067775	2015-02-20 16:22:56.335967	3	accepted	\N	\N	1
319	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.		Supports	3	1	3	7	188	93	2015-02-20 16:16:04.130955	2015-02-20 16:22:56.355504	3	accepted	\N	\N	1
320	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS		Supports	3	1	3	8	186	93	2015-02-20 16:16:04.186517	2015-02-20 16:22:56.376493	3	accepted	\N	\N	1
321	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.		Supports	3	2	3	7	188	94	2015-02-20 16:16:04.243822	2015-02-20 16:22:56.400409	3	accepted	\N	\N	1
322	Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis.		Supports	3	2	3	7	188	95	2015-02-20 16:16:04.343383	2015-02-20 16:22:56.419537	3	accepted	\N	\N	1
323	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML		Does Not Support	3	1	3	1	189	96	2015-02-20 16:16:04.403281	2015-02-20 16:22:56.440371	3	accepted	\N	\N	1
324	Likely due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG was shown to be effective in a patient with metastatic malignant melanoma with an NRAS G13D mutation		Supports	1	1	57	8	190	97	2015-02-20 16:16:04.466944	2015-02-20 16:22:56.462666	2	accepted	\N	\N	1
325	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.		Supports	1	1	14	7	62	98	2015-02-20 16:16:04.530719	2015-02-20 16:22:56.483356	3	accepted	\N	\N	1
326	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.		Does Not Support	1	1	14	7	62	98	2015-02-20 16:16:04.591539	2015-02-20 16:22:56.507639	3	accepted	\N	\N	1
328	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.		Supports	1	1	19	8	192	98	2015-02-20 16:16:04.732464	2015-02-20 16:22:56.550411	4	accepted	\N	\N	1
329	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months		Supports	1	1	58	8	193	99	2015-02-20 16:16:04.786799	2015-02-20 16:22:56.571082	2	accepted	\N	\N	1
330	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months		Supports	1	1	58	8	193	100	2015-02-20 16:16:04.847212	2015-02-20 16:22:56.593254	2	accepted	\N	\N	1
331	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	59	38	194	101	2015-02-20 16:16:04.908472	2015-02-20 16:22:56.616619	4	accepted	\N	\N	1
332	GIST tumors harboring PDGFRA D842V mutation are likely to be benign		Supports	2	1	3	38	195	102	2015-02-20 16:16:04.972927	2015-02-20 16:22:56.636751	3	accepted	\N	\N	1
334	GIST cancer with D842V mutation is resistant to imatinib.		Supports	1	1	1	38	196	102	2015-02-20 16:16:05.110972	2015-02-20 16:22:56.682496	4	accepted	\N	\N	1
335	In CHO cells with PDGFRA D842V mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	59	38	194	102	2015-02-20 16:16:05.174381	2015-02-20 16:22:56.705337	4	accepted	\N	\N	1
336	In CHO cells with PDGFRA D842Y mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	59	38	194	103	2015-02-20 16:16:05.237234	2015-02-20 16:22:56.728407	4	accepted	\N	\N	1
337	In CHO cells with the PDGFRA deletion I843 mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	59	38	194	104	2015-02-20 16:16:05.305037	2015-02-20 16:22:56.752127	4	accepted	\N	\N	1
338	In CHO cells with PDGFRA D842-843IM mutation that have shown imatinib resistance, crenolanib was significantly more potent at inhibiting kinase activity than imatinib.		Supports	1	3	59	38	194	105	2015-02-20 16:16:05.382789	2015-02-20 16:22:56.791491	4	accepted	\N	\N	1
339	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	3	60	11	197	106	2015-02-20 16:16:05.464426	2015-02-20 16:22:56.812029	4	accepted	\N	\N	1
340	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	3	61	11	198	106	2015-02-20 16:16:05.529952	2015-02-20 16:22:56.83354	3	accepted	\N	\N	1
341	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	50	7	153	106	2015-02-20 16:16:05.587354	2015-02-20 16:22:56.856787	4	accepted	\N	\N	1
342	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.		Supports	1	4	14	7	199	106	2015-02-20 16:16:05.654999	2015-02-20 16:22:56.879054	3	accepted	\N	\N	1
343	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.		Supports	1	4	12	7	199	106	2015-02-20 16:16:05.716696	2015-02-20 16:22:56.903366	3	accepted	\N	\N	1
344	The presence of exon 9 (such as E542K) or exon 20 mutations were associated with poorer survival compared to those with wild-type PIK3CA mutation.		Supports	3	4	3	7	199	106	2015-02-20 16:16:05.776492	2015-02-20 16:22:56.930013	3	accepted	\N	\N	1
345	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.		Supports	3	4	3	7	200	106	2015-02-20 16:16:05.844831	2015-02-20 16:22:56.951965	2	accepted	\N	\N	1
346	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	3	60	11	197	107	2015-02-20 16:16:05.948907	2015-02-20 16:22:56.973878	4	accepted	\N	\N	1
347	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	3	61	11	198	107	2015-02-20 16:16:06.006543	2015-02-20 16:22:56.997543	3	accepted	\N	\N	1
349	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.		Supports	1	4	14	7	199	107	2015-02-20 16:16:06.126478	2015-02-20 16:22:57.041049	3	accepted	\N	\N	1
350	The presence of a PIK3CA mutation in exon 9 (such as E542K) or exon 20 were associated with lack of panitumumab or cetuximab response.		Supports	1	4	12	7	199	107	2015-02-20 16:16:06.177338	2015-02-20 16:22:57.062891	3	accepted	\N	\N	1
351	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.		Supports	3	4	3	7	199	107	2015-02-20 16:16:06.235446	2015-02-20 16:22:57.084717	3	accepted	\N	\N	1
352	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.		Supports	3	4	3	7	200	107	2015-02-20 16:16:06.296397	2015-02-20 16:22:57.10423	2	accepted	\N	\N	1
353	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates as well as shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA		Supports	1	1	14	7	62	108	2015-02-20 16:16:06.351631	2015-02-20 16:22:57.125388	3	accepted	\N	\N	1
354	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA		Does Not Support	1	1	14	7	62	109	2015-02-20 16:16:06.418413	2015-02-20 16:22:57.145202	3	accepted	\N	\N	1
355	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	3	60	11	197	110	2015-02-20 16:16:06.47756	2015-02-20 16:22:57.167344	4	accepted	\N	\N	1
356	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	3	61	11	198	110	2015-02-20 16:16:06.541554	2015-02-20 16:22:57.189249	3	accepted	\N	\N	1
357	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	50	7	153	110	2015-02-20 16:16:06.594699	2015-02-20 16:22:57.208638	4	accepted	\N	\N	1
358	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.		Supports	1	4	14	7	199	110	2015-02-20 16:16:06.660113	2015-02-20 16:22:57.232394	3	accepted	\N	\N	1
359	The presence of a PIK3CA mutation in exon 9 or exon 20 (such as H1047R) were associated with lack of panitumumab or cetuximab response.		Supports	1	4	12	7	199	110	2015-02-20 16:16:06.708414	2015-02-20 16:22:57.250949	3	accepted	\N	\N	1
361	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.		Supports	1	1	42	39	201	111	2015-02-20 16:16:06.832483	2015-02-20 16:22:57.290959	5	accepted	\N	\N	1
363	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.		Supports	1	3	3	3	202	113	2015-02-20 16:16:06.965955	2015-02-20 16:22:57.338464	4	accepted	\N	\N	1
364	PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.		Does Not Support	3	1	3	37	203	113	2015-02-20 16:16:07.045382	2015-02-20 16:22:57.362536	3	accepted	\N	\N	1
365	Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.		Supports	1	1	42	39	201	114	2015-02-20 16:16:07.116127	2015-02-20 16:22:57.403862	5	accepted	\N	\N	1
366	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.		Supports	1	3	62	40	204	115	2015-02-20 16:16:07.184121	2015-02-20 16:22:57.423834	3	accepted	\N	\N	1
367	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.		Supports	1	3	62	40	204	116	2015-02-20 16:16:07.248649	2015-02-20 16:22:57.44345	3	accepted	\N	\N	1
368	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.		Supports	2	1	3	40	205	116	2015-02-20 16:16:07.319636	2015-02-20 16:22:57.466168	5	accepted	\N	\N	1
369	The use of AZD1480 on RET-mutated/rearranged cell lines in vitro led to strong repression of tyroid cancer cell growth.		Supports	1	3	63	40	206	116	2015-02-20 16:16:07.448341	2015-02-20 16:22:57.489186	3	accepted	\N	\N	1
370	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.		Supports	3	1	3	40	207	116	2015-02-20 16:16:07.507565	2015-02-20 16:22:57.509299	4	accepted	\N	\N	1
371	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.		Supports	3	4	3	27	208	117	2015-02-20 16:16:07.575877	2015-02-20 16:22:57.535272	3	accepted	\N	\N	1
372	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.		Supports	3	4	3	27	208	118	2015-02-20 16:16:07.63798	2015-02-20 16:22:57.55758	3	accepted	\N	\N	1
373	Breast tumors in a mouse model with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.		Supports	1	3	64	11	209	119	2015-02-20 16:16:07.705123	2015-02-20 16:22:57.583529	3	accepted	\N	\N	1
374	Breast cancer patients who harbor R175H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	3	1	3	11	210	119	2015-02-20 16:16:07.771519	2015-02-20 16:22:57.605732	3	accepted	\N	\N	1
375	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R248 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	3	1	3	11	211	120	2015-02-20 16:16:07.840984	2015-02-20 16:22:57.626847	3	accepted	\N	\N	1
376	Breast cancer patients who harbor R248Q mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	3	1	3	11	210	120	2015-02-20 16:16:07.908086	2015-02-20 16:22:57.646691	3	accepted	\N	\N	1
377	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.		Supports	3	1	3	11	210	121	2015-02-20 16:16:07.964041	2015-02-20 16:22:57.667885	3	accepted	\N	\N	1
378	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.		Supports	3	1	3	11	211	122	2015-02-20 16:16:08.025353	2015-02-20 16:22:57.689753	3	accepted	\N	\N	1
379	In breast cancer patients harboring TP53 mutation, mutations in conserved regions such as R249 are prognostic for a worse overall survival compared to those harboring wild-type TP53.		Supports	3	1	3	11	211	123	2015-02-20 16:16:08.081823	2015-02-20 16:22:57.709319	3	accepted	\N	\N	1
380	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	3	1	3	11	211	124	2015-02-20 16:16:08.139942	2015-02-20 16:22:57.730151	3	accepted	\N	\N	1
381	Breast cancer patients who harbor R273C mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	3	1	3	11	210	124	2015-02-20 16:16:08.199461	2015-02-20 16:22:57.751026	3	accepted	\N	\N	1
382	In breast cancer patients harboring TP53 mutation, mutations in DNA contact regions such as R273 are prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	3	1	3	11	211	125	2015-02-20 16:16:08.26164	2015-02-20 16:22:57.772493	3	accepted	\N	\N	1
383	Breast cancer patients who harbor R273H mutation have worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	3	1	3	11	210	125	2015-02-20 16:16:08.315837	2015-02-20 16:22:57.792457	3	accepted	\N	\N	1
384	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumors with wild type TP53.		Supports	3	1	3	11	211	126	2015-02-20 16:16:08.381601	2015-02-20 16:22:57.811631	3	accepted	\N	\N	1
385	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.		Supports	1	1	65	41	212	127	2015-02-20 16:16:08.445227	2015-02-20 16:22:57.833192	3	accepted	\N	\N	1
386	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.		Supports	1	3	61	4	213	127	2015-02-20 16:16:08.511558	2015-02-20 16:22:57.854662	3	accepted	\N	\N	1
387	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.		Supports	1	1	65	42	214	128	2015-02-20 16:16:08.578028	2015-02-20 16:22:57.874929	3	accepted	\N	\N	1
388	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.		Supports	1	1	65	42	212	129	2015-02-20 16:16:08.660426	2015-02-20 16:22:57.897278	3	accepted	\N	\N	1
1	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.		Supports	1	1	1	1	41	1	2015-02-20 16:15:43.668898	2015-02-20 16:22:49.187419	3	accepted	\N	\N	1
2	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.		Supports	1	2	2	1	41	1	2015-02-20 16:15:43.736923	2015-02-20 16:22:49.273977	5	accepted	\N	\N	1
13	In patients with non-small cell lung cancer harboring EML4-ALK fusion, the C1156Y variant has been shown to confer resistance to crizotinib.		Supports	1	1	7	4	46	6	2015-02-20 16:15:44.51589	2015-02-20 16:22:49.514041	4	accepted	\N	\N	1
26	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells expressing ARAF S214C in vitro		Supports	1	3	11	4	52	10	2015-02-20 16:15:45.375392	2015-02-20 16:22:49.793023	3	accepted	\N	\N	1
33	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	3	14	9	57	12	2015-02-20 16:15:45.853458	2015-02-20 16:22:49.94455	3	accepted	\N	\N	1
390	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.		Does Not Support	3	4	3	1	215	130	2015-02-20 16:16:08.798657	2015-02-20 16:22:57.940124	2	accepted	\N	\N	1
391	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.		Does Not Support	3	4	3	27	215	130	2015-02-20 16:16:08.856555	2015-02-20 16:22:57.961357	2	accepted	\N	\N	1
392	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.		Supports	3	4	3	27	216	130	2015-02-20 16:16:08.971156	2015-02-20 16:22:57.99875	3	accepted	\N	\N	1
393	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.		Does Not Support	2	4	3	1	215	131	2015-02-20 16:16:09.034663	2015-02-20 16:22:58.020926	3	accepted	\N	\N	1
394	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.		Does Not Support	3	4	3	1	215	131	2015-02-20 16:16:09.091504	2015-02-20 16:22:58.042644	2	accepted	\N	\N	1
395	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.		Does Not Support	3	4	3	27	215	131	2015-02-20 16:16:09.148033	2015-02-20 16:22:58.067536	2	accepted	\N	\N	1
396	After adjust for age and cancer stage, the presence of U2AF mutations such as S34Y/F are prognostic for poorer survival outcomes in patients with MDS.		Supports	3	4	3	27	216	131	2015-02-20 16:16:09.219852	2015-02-20 16:22:58.087305	3	accepted	\N	\N	1
398	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML		Does Not Support	3	1	3	1	218	132	2015-02-20 16:16:09.356294	2015-02-20 16:22:58.130019	4	accepted	\N	\N	1
399	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML		Does Not Support	3	1	3	1	218	132	2015-02-20 16:16:09.417554	2015-02-20 16:22:58.149255	3	accepted	\N	\N	1
400	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	3	1	3	1	217	132	2015-02-20 16:16:09.475493	2015-02-20 16:22:58.170818	3	accepted	\N	\N	1
401	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	3	1	3	1	217	132	2015-02-20 16:16:09.53734	2015-02-20 16:22:58.194682	3	accepted	\N	\N	1
402	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients		Supports	3	1	3	1	219	132	2015-02-20 16:16:09.596669	2015-02-20 16:22:58.216427	3	accepted	\N	\N	1
403	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML		Supports	3	1	3	1	218	132	2015-02-20 16:16:09.66169	2015-02-20 16:22:58.237869	3	accepted	\N	\N	1
404	Mutations in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence		Supports	3	1	3	1	220	132	2015-02-20 16:16:09.733217	2015-02-20 16:22:58.257468	2	accepted	\N	\N	1
405	Mutations  in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML		Supports	3	1	3	1	220	132	2015-02-20 16:16:09.786289	2015-02-20 16:22:58.278781	2	accepted	\N	\N	1
406	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	1	1	3	1	217	133	2015-02-20 16:16:09.848835	2015-02-20 16:22:58.298674	3	accepted	\N	\N	1
407	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.		Does Not Support	3	1	3	1	218	133	2015-02-20 16:16:09.913987	2015-02-20 16:22:58.319798	4	accepted	\N	\N	1
408	No differences in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML		Does Not Support	3	1	3	1	218	133	2015-02-20 16:16:09.969485	2015-02-20 16:22:58.341149	3	accepted	\N	\N	1
409	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	3	1	3	1	217	133	2015-02-20 16:16:10.032462	2015-02-20 16:22:58.359762	3	accepted	\N	\N	1
410	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	3	1	3	1	217	133	2015-02-20 16:16:10.09567	2015-02-20 16:22:58.383789	3	accepted	\N	\N	1
411	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients		Supports	3	1	3	1	219	133	2015-02-20 16:16:10.154428	2015-02-20 16:22:58.402611	3	accepted	\N	\N	1
36	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation		Supports	1	3	15	9	58	12	2015-02-20 16:15:46.015678	2015-02-20 16:22:50.023321	3	accepted	\N	\N	1
56	BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid cancer		Supports	2	1	3	10	70	12	2015-02-20 16:15:47.271225	2015-02-20 16:22:50.474181	3	accepted	\N	\N	1
61	COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.		Supports	1	4	23	7	73	14	2015-02-20 16:15:47.579418	2015-02-20 16:22:50.587967	4	accepted	\N	\N	1
67	Three studies have found no significant survival impact for CCND1 in lung cancer.		Does Not Support	3	1	3	4	75	19	2015-02-20 16:15:47.969175	2015-02-20 16:22:50.721746	3	accepted	\N	\N	1
76	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.		Supports	2	3	3	14	80	22	2015-02-20 16:15:48.539011	2015-02-20 16:22:50.92869	3	accepted	\N	\N	1
84	In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.		Supports	1	3	25	18	84	26	2015-02-20 16:15:49.066857	2015-02-20 16:22:51.098079	4	accepted	\N	\N	1
96	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.		Supports	2	1	3	20	91	31	2015-02-20 16:15:49.842026	2015-02-20 16:22:51.351209	4	accepted	\N	\N	1
99	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.		Does Not Support	1	1	28	1	93	32	2015-02-20 16:15:50.03551	2015-02-20 16:22:51.41846	4	accepted	\N	\N	1
104	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	3	1	3	1	92	32	2015-02-20 16:15:50.3681	2015-02-20 16:22:51.541949	4	accepted	\N	\N	1
125	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A		Supports	3	1	3	1	92	32	2015-02-20 16:15:51.768045	2015-02-20 16:22:52.028825	3	accepted	\N	\N	1
129	Geftinib added to pemetrexed and carboplatin showed greater effectiveness than premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.		Supports	1	1	30	4	100	33	2015-02-20 16:15:52.033135	2015-02-20 16:22:52.113773	3	accepted	\N	\N	1
140	In MCF10A cell lines retrovirally transduced with wildtype or mutant ERBB2, cells expressing the in-frame deletion of amino acids 755-759 were shown to be sensitive to neratinib.		Supports	1	3	34	11	108	37	2015-02-20 16:15:52.789744	2015-02-20 16:22:52.362897	5	accepted	\N	\N	1
154	The N538G ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	3	37	11	109	47	2015-02-20 16:15:53.686265	2015-02-20 16:22:52.663123	5	accepted	\N	\N	1
162	MCF7 cell lines harboring the Y537N mutation in the ligand-binding domain of ESR1 have shown resistance to hormone therapy.		Supports	1	3	3	11	110	49	2015-02-20 16:15:54.185343	2015-02-20 16:22:52.837103	3	accepted	\N	\N	1
166	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.		Supports	1	1	39	22	111	56	2015-02-20 16:15:54.46277	2015-02-20 16:22:52.938527	2	accepted	\N	\N	1
188	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.		Supports	1	3	41	1	113	61	2015-02-20 16:15:55.886154	2015-02-20 16:22:53.422701	3	accepted	\N	\N	1
193	Sorafenib is effective in patients with FLT3 internal tandem repeat mutations, but not in patients with wild type or D835 mutation.		Supports	1	3	10	1	118	61	2015-02-20 16:15:56.19583	2015-02-20 16:22:53.530421	4	accepted	\N	\N	1
196	In AML patients, FLT3-TKD mutation was associated with poorer disease free survival compared to patients with wild type FLT3		Supports	3	1	3	1	124	61	2015-02-20 16:15:56.389652	2015-02-20 16:22:53.611452	4	accepted	\N	\N	1
208	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	3	1	3	1	132	66	2015-02-20 16:15:57.171346	2015-02-20 16:22:53.875579	3	accepted	\N	\N	1
221	rs10974944 SNP in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer		Supports	2	1	3	30	140	69	2015-02-20 16:15:58.010753	2015-02-20 16:22:54.149979	4	accepted	\N	\N	1
229	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	3	1	1	8	147	71	2015-02-20 16:15:58.642302	2015-02-20 16:22:54.341574	3	accepted	\N	\N	1
232	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT		Supports	2	1	3	33	144	74	2015-02-20 16:15:58.806518	2015-02-20 16:22:54.408321	3	accepted	\N	\N	1
246	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	50	7	153	79	2015-02-20 16:15:59.591563	2015-02-20 16:22:54.716784	4	accepted	\N	\N	1
256	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS		Supports	1	1	14	7	62	81	2015-02-20 16:16:00.199149	2015-02-20 16:22:54.950054	3	accepted	\N	\N	1
263	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.		Supports	2	1	3	34	156	84	2015-02-20 16:16:00.606618	2015-02-20 16:22:55.109462	3	accepted	\N	\N	1
266	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft model		Supports	1	3	14	9	165	85	2015-02-20 16:16:00.793686	2015-02-20 16:22:55.178257	4	accepted	\N	\N	1
276	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.		Supports	1	4	53	37	169	89	2015-02-20 16:16:01.415072	2015-02-20 16:22:55.390021	4	accepted	\N	\N	1
290	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved		Supports	1	1	3	1	172	90	2015-02-20 16:16:02.252348	2015-02-20 16:22:55.705299	3	accepted	\N	\N	1
296	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1 mutations.		Supports	3	1	3	1	170	90	2015-02-20 16:16:02.643582	2015-02-20 16:22:55.833627	5	accepted	\N	\N	1
298	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old		Supports	3	1	3	1	179	90	2015-02-20 16:16:02.807925	2015-02-20 16:22:55.875054	4	accepted	\N	\N	1
312	"AML with mutated NPM1" is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.		Supports	2	2	3	1	184	90	2015-02-20 16:16:03.662957	2015-02-20 16:22:56.198189	5	accepted	\N	\N	1
327	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk		Supports	2	1	3	8	191	98	2015-02-20 16:16:04.664359	2015-02-20 16:22:56.52857	3	accepted	\N	\N	1
333	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.		Supports	1	1	1	38	150	102	2015-02-20 16:16:05.046349	2015-02-20 16:22:56.660965	4	accepted	\N	\N	1
348	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	50	7	153	107	2015-02-20 16:16:06.069569	2015-02-20 16:22:57.01987	4	accepted	\N	\N	1
360	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.		Supports	3	4	3	7	199	110	2015-02-20 16:16:06.764907	2015-02-20 16:22:57.2702	3	accepted	\N	\N	1
362	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.		Supports	2	1	3	20	91	112	2015-02-20 16:16:06.897321	2015-02-20 16:22:57.316098	4	accepted	\N	\N	1
389	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.		Does Not Support	2	4	3	1	215	130	2015-02-20 16:16:08.733491	2015-02-20 16:22:57.91822	3	accepted	\N	\N	1
397	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	1	1	3	1	217	132	2015-02-20 16:16:09.288211	2015-02-20 16:22:58.108205	3	accepted	\N	\N	1
412	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML		Supports	3	1	3	1	218	133	2015-02-20 16:16:10.215582	2015-02-20 16:22:58.424246	3	accepted	\N	\N	1
\.


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 412, true);


--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 47, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-02-20 16:15:28.856587	2015-02-20 16:15:28.856587
1	2	2015-02-20 16:15:28.86175	2015-02-20 16:15:28.86175
2	3	2015-02-20 16:15:29.17608	2015-02-20 16:15:29.17608
2	4	2015-02-20 16:15:29.181029	2015-02-20 16:15:29.181029
3	5	2015-02-20 16:15:29.434234	2015-02-20 16:15:29.434234
3	6	2015-02-20 16:15:29.440029	2015-02-20 16:15:29.440029
4	7	2015-02-20 16:15:29.725987	2015-02-20 16:15:29.725987
4	8	2015-02-20 16:15:29.731504	2015-02-20 16:15:29.731504
5	9	2015-02-20 16:15:29.976428	2015-02-20 16:15:29.976428
5	10	2015-02-20 16:15:29.981507	2015-02-20 16:15:29.981507
6	11	2015-02-20 16:15:30.220918	2015-02-20 16:15:30.220918
6	12	2015-02-20 16:15:30.226674	2015-02-20 16:15:30.226674
7	11	2015-02-20 16:15:30.429944	2015-02-20 16:15:30.429944
7	12	2015-02-20 16:15:30.433692	2015-02-20 16:15:30.433692
8	11	2015-02-20 16:15:30.637303	2015-02-20 16:15:30.637303
8	12	2015-02-20 16:15:30.640554	2015-02-20 16:15:30.640554
9	13	2015-02-20 16:15:30.844326	2015-02-20 16:15:30.844326
10	14	2015-02-20 16:15:31.064001	2015-02-20 16:15:31.064001
10	15	2015-02-20 16:15:31.069822	2015-02-20 16:15:31.069822
11	14	2015-02-20 16:15:31.271016	2015-02-20 16:15:31.271016
11	15	2015-02-20 16:15:31.274224	2015-02-20 16:15:31.274224
12	16	2015-02-20 16:15:31.478529	2015-02-20 16:15:31.478529
12	17	2015-02-20 16:15:31.483416	2015-02-20 16:15:31.483416
13	18	2015-02-20 16:15:31.702934	2015-02-20 16:15:31.702934
13	19	2015-02-20 16:15:31.707846	2015-02-20 16:15:31.707846
13	20	2015-02-20 16:15:31.712273	2015-02-20 16:15:31.712273
14	21	2015-02-20 16:15:31.911808	2015-02-20 16:15:31.911808
15	21	2015-02-20 16:15:32.13695	2015-02-20 16:15:32.13695
15	22	2015-02-20 16:15:32.143036	2015-02-20 16:15:32.143036
16	23	2015-02-20 16:15:32.369798	2015-02-20 16:15:32.369798
16	24	2015-02-20 16:15:32.374713	2015-02-20 16:15:32.374713
17	25	2015-02-20 16:15:32.676239	2015-02-20 16:15:32.676239
17	26	2015-02-20 16:15:32.68153	2015-02-20 16:15:32.68153
18	27	2015-02-20 16:15:32.925802	2015-02-20 16:15:32.925802
18	28	2015-02-20 16:15:32.930417	2015-02-20 16:15:32.930417
19	29	2015-02-20 16:15:33.165662	2015-02-20 16:15:33.165662
19	30	2015-02-20 16:15:33.171631	2015-02-20 16:15:33.171631
20	31	2015-02-20 16:15:33.373094	2015-02-20 16:15:33.373094
20	32	2015-02-20 16:15:33.378552	2015-02-20 16:15:33.378552
21	31	2015-02-20 16:15:33.648442	2015-02-20 16:15:33.648442
21	32	2015-02-20 16:15:33.652086	2015-02-20 16:15:33.652086
22	33	2015-02-20 16:15:33.993142	2015-02-20 16:15:33.993142
22	18	2015-02-20 16:15:33.996741	2015-02-20 16:15:33.996741
33	34	2015-02-20 16:15:36.427028	2015-02-20 16:15:36.427028
33	18	2015-02-20 16:15:36.430624	2015-02-20 16:15:36.430624
40	35	2015-02-20 16:15:38.02058	2015-02-20 16:15:38.02058
40	36	2015-02-20 16:15:38.025469	2015-02-20 16:15:38.025469
42	37	2015-02-20 16:15:38.52317	2015-02-20 16:15:38.52317
42	38	2015-02-20 16:15:38.52905	2015-02-20 16:15:38.52905
46	39	2015-02-20 16:15:39.474931	2015-02-20 16:15:39.474931
46	40	2015-02-20 16:15:39.480297	2015-02-20 16:15:39.480297
\.


--
-- Data for Name: notifications; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY notifications (id, subscription_id, user_id, subscribable_id, subscribable_type, content, url, acknowledged, delivered, created_at, updated_at) FROM stdin;
\.


--
-- Name: notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('notifications_id_seq', 1, false);


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20150220160805
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
20150205210725
20150210191015
20150210204445
20150219202227
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 220, true);


--
-- Data for Name: subscriptions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY subscriptions (id, user_id, subscribable_id, subscribable_type, type, created_at, updated_at) FROM stdin;
\.


--
-- Name: subscriptions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('subscriptions_id_seq', 1, false);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
9	KIT Exon 17	\N	2015-02-20 16:15:58.284729	2015-02-20 16:15:58.284729
10	KIT Exon 11	\N	2015-02-20 16:15:58.355485	2015-02-20 16:15:58.355485
1	Imatinib Resistance	While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. 	2015-02-20 16:15:44.265182	2015-02-20 16:16:10.307431
2	ALK Fusions	ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. 	2015-02-20 16:15:44.560721	2015-02-20 16:16:10.312313
5	Erlotinib Resistance	EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  	2015-02-20 16:15:52.244032	2015-02-20 16:16:10.315652
3	Crizotinib Resistance	The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. 	2015-02-20 16:15:44.568261	2015-02-20 16:16:10.318277
4	Other V600's	While BRAF V600E is nearly ubiquitous in many cancer types, other V600 variants have also been observed and studied to a lesser degree. At first approximation, many of these variants seem to behave similarly to V600E, and treatment with dabrafenib has been shown to be effective. 	2015-02-20 16:15:45.827984	2015-02-20 16:16:10.320641
6	HER2 Activating	HER2-positive breast cancer is widely recognized as a molecular subtype of breast cancer. While amplification of ERBB2 is a common mechanism for activation of the pathway, missense mutations have also demonstrated activation potential. These activating missense mutations also confer senstivity to the targeted therapeutic neratinib. 	2015-02-20 16:15:52.68261	2015-02-20 16:16:10.322831
7	ESR1 Ligand-Binding Domain	ER-positive breast cancer is the most common of the breast cancer subtypes. Hormone therapy has been widely effective in treating the disease, however sequencing of resistant patients has uncovered a number of missense mutations in mediating this resistance. Many of these mutations lie within the ligand-binding domain, and contribute to constitutive activity of the receptor. This has lead to the development of estrogen receptor degrading agents such as fulvestrant, which have shown early efficacy in clinical trials.  	2015-02-20 16:15:53.603101	2015-02-20 16:16:10.325043
8	FGFR fusions	FGFR fusions have been demonstrated across many cancer types to have oncogenic potential. FGFR2 and FGFR3 have been shown to be fused to many 3' partners, which may act as a vehicle for pathway activation. Treatment of cell lines harboring FGFR2 fusions with pazopanib has shown mild "anti-tumor" activities, and both ponatinib and pazopanib have shown efficacy in case studies.  	2015-02-20 16:15:54.50838	2015-02-20 16:16:10.327273
11	NPM1 exon 12	To be filled in by Dr. KK, resident blood expert and knower of all things hematology. :) :) :) 	2015-02-20 16:16:03.775134	2015-02-20 16:16:10.329786
12	PTEN Loss-of-Function	PTEN loss is a common event in breast cancer, as well as others. The most common single nucleotide variant resulting in PTEN loss is a R233 nonsense mutation. Cell lines harboring this mutation behave similarly to cell lines harboring larger insertion or deletion events, in that they both respond to PI3K-mTOR inhibitors with impeded growth. 	2015-02-20 16:16:07.020367	2015-02-20 16:16:10.332107
13	Motesanib Resistance	RET activation is a common oncogenic marker of medullary thyroid carcinoma. Treatment of these patients with the targeted therapeutic motesanib has shown to be effective. However, the missense mutations C634W and M918T have shown to confer motesanib resistance in cell lines. 	2015-02-20 16:16:07.230415	2015-02-20 16:16:10.3343
14	TSC Loss	Inactivating events in TSC1 and TSC2 have been observed in a number of cancer types. Cell lines and case studies have shown that in the loss of TSC, the mTOR inhibitor everolimus has seen significant anti-tumorigenic activity. 	2015-02-20 16:16:08.490603	2015-02-20 16:16:10.33664
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
3	1	2015-02-20 16:15:44.273873	2015-02-20 16:15:44.273873
2	1	2015-02-20 16:15:44.34325	2015-02-20 16:15:44.34325
2	1	2015-02-20 16:15:44.405644	2015-02-20 16:15:44.405644
6	2	2015-02-20 16:15:44.56469	2015-02-20 16:15:44.56469
6	3	2015-02-20 16:15:44.570981	2015-02-20 16:15:44.570981
5	2	2015-02-20 16:15:44.638957	2015-02-20 16:15:44.638957
8	3	2015-02-20 16:15:45.02952	2015-02-20 16:15:45.02952
7	2	2015-02-20 16:15:45.174736	2015-02-20 16:15:45.174736
7	3	2015-02-20 16:15:45.178537	2015-02-20 16:15:45.178537
11	4	2015-02-20 16:15:45.830979	2015-02-20 16:15:45.830979
13	4	2015-02-20 16:15:47.681238	2015-02-20 16:15:47.681238
15	4	2015-02-20 16:15:47.770811	2015-02-20 16:15:47.770811
16	4	2015-02-20 16:15:47.833661	2015-02-20 16:15:47.833661
34	5	2015-02-20 16:15:52.246803	2015-02-20 16:15:52.246803
35	6	2015-02-20 16:15:52.686101	2015-02-20 16:15:52.686101
36	6	2015-02-20 16:15:52.773942	2015-02-20 16:15:52.773942
38	6	2015-02-20 16:15:52.894334	2015-02-20 16:15:52.894334
41	6	2015-02-20 16:15:53.126933	2015-02-20 16:15:53.126933
43	6	2015-02-20 16:15:53.345979	2015-02-20 16:15:53.345979
44	6	2015-02-20 16:15:53.414101	2015-02-20 16:15:53.414101
45	6	2015-02-20 16:15:53.481759	2015-02-20 16:15:53.481759
46	7	2015-02-20 16:15:53.606044	2015-02-20 16:15:53.606044
46	7	2015-02-20 16:15:53.66924	2015-02-20 16:15:53.66924
47	7	2015-02-20 16:15:53.811573	2015-02-20 16:15:53.811573
47	7	2015-02-20 16:15:53.875447	2015-02-20 16:15:53.875447
48	7	2015-02-20 16:15:53.990807	2015-02-20 16:15:53.990807
48	7	2015-02-20 16:15:54.058358	2015-02-20 16:15:54.058358
49	7	2015-02-20 16:15:54.16558	2015-02-20 16:15:54.16558
49	7	2015-02-20 16:15:54.234332	2015-02-20 16:15:54.234332
50	7	2015-02-20 16:15:54.377495	2015-02-20 16:15:54.377495
50	7	2015-02-20 16:15:54.43825	2015-02-20 16:15:54.43825
56	8	2015-02-20 16:15:54.51263	2015-02-20 16:15:54.51263
59	8	2015-02-20 16:15:54.638698	2015-02-20 16:15:54.638698
70	9	2015-02-20 16:15:58.288906	2015-02-20 16:15:58.288906
71	10	2015-02-20 16:15:58.359148	2015-02-20 16:15:58.359148
71	10	2015-02-20 16:15:58.424577	2015-02-20 16:15:58.424577
71	10	2015-02-20 16:15:58.490931	2015-02-20 16:15:58.490931
71	10	2015-02-20 16:15:58.558807	2015-02-20 16:15:58.558807
71	10	2015-02-20 16:15:58.626237	2015-02-20 16:15:58.626237
71	10	2015-02-20 16:15:58.687705	2015-02-20 16:15:58.687705
74	9	2015-02-20 16:15:58.851008	2015-02-20 16:15:58.851008
74	9	2015-02-20 16:15:58.910167	2015-02-20 16:15:58.910167
76	10	2015-02-20 16:15:59.184795	2015-02-20 16:15:59.184795
76	10	2015-02-20 16:15:59.250205	2015-02-20 16:15:59.250205
76	10	2015-02-20 16:15:59.395424	2015-02-20 16:15:59.395424
77	10	2015-02-20 16:15:59.455965	2015-02-20 16:15:59.455965
77	10	2015-02-20 16:15:59.51977	2015-02-20 16:15:59.51977
91	11	2015-02-20 16:16:03.778177	2015-02-20 16:16:03.778177
91	11	2015-02-20 16:16:03.850386	2015-02-20 16:16:03.850386
91	11	2015-02-20 16:16:03.917553	2015-02-20 16:16:03.917553
91	11	2015-02-20 16:16:03.987321	2015-02-20 16:16:03.987321
101	1	2015-02-20 16:16:04.953692	2015-02-20 16:16:04.953692
102	1	2015-02-20 16:16:05.028511	2015-02-20 16:16:05.028511
102	1	2015-02-20 16:16:05.093046	2015-02-20 16:16:05.093046
102	1	2015-02-20 16:16:05.15916	2015-02-20 16:16:05.15916
102	1	2015-02-20 16:16:05.218528	2015-02-20 16:16:05.218528
103	1	2015-02-20 16:16:05.286206	2015-02-20 16:16:05.286206
104	1	2015-02-20 16:16:05.366916	2015-02-20 16:16:05.366916
105	1	2015-02-20 16:16:05.43846	2015-02-20 16:16:05.43846
113	12	2015-02-20 16:16:07.02339	2015-02-20 16:16:07.02339
113	12	2015-02-20 16:16:07.096653	2015-02-20 16:16:07.096653
115	13	2015-02-20 16:16:07.233075	2015-02-20 16:16:07.233075
116	13	2015-02-20 16:16:07.302295	2015-02-20 16:16:07.302295
127	14	2015-02-20 16:16:08.493802	2015-02-20 16:16:08.493802
127	14	2015-02-20 16:16:08.558977	2015-02-20 16:16:08.558977
128	14	2015-02-20 16:16:08.640667	2015-02-20 16:16:08.640667
129	14	2015-02-20 16:16:08.710992	2015-02-20 16:16:08.710992
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 14, true);


--
-- Name: variant_origins_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_origins_id_seq', 1, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 133, true);


--
-- PostgreSQL database dump complete
--

