--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	6b6bab8e-b2f1-476f-8b4a-d33d62114a32	2015-02-05 21:09:34.326053
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, has been known as an oncogene\n  for quite some time. AKT activation relies on the PI3K pathway, and is recognized\n  as a critical node in the pathway. The E17 hotspot is the most characterized of\n  AKT1 mutations, and has been shown to result in activation of the protein. Mutations\n  in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors\n  in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	073374a6-7722-42d1-8a55-04ed32dca6d1	2015-02-05 21:09:34.537239
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	37604d6a-2b88-46e2-9843-fc7006d29bd3	2015-02-05 21:09:34.746168
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia. Treatment of cells harboring\n  this fusion has shown sensitivity to imatinib, greatly improving the prognostic\n  outlook of the disease. However, mutations in this gene have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	0a14915e-491c-4d95-8431-c6af9c9b6cdd	2015-02-05 21:09:35.047094
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been shown to be correlated with poor prognosis in many cancer types,\n  there is at least one study that questions this conclusion in papillary thyroid\n  cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	b0ddca48-6499-467b-acb1-a5f2b61f4178	2015-02-05 21:09:35.255005
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	8d1a89d4-3393-4c58-ae07-c3785354ba5c	2015-02-05 21:09:35.479521
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	a34e91ca-87e8-44f2-b440-9a0aeaef6459	2015-02-05 21:09:35.683004
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	1fc864a1-0c17-4bad-8a38-24bad8373774	2015-02-05 21:09:35.890613
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as it's counterparts,\n  cyclin D, has been shown to implicated in various carcinomas, including breast,\n  gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification\n  or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma,\n  and these measurements are correlated with later stage disease. It has also been\n  shown, in lung cancer specifically, that neoplastic cells with higher levels of\n  the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	62e6db9c-482f-41c0-b424-673d61d46dad	2015-02-05 21:09:36.070554
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	5f86d908-378d-428b-a673-e2c91ccf4d5d	2015-02-05 21:09:36.251636
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: N/A\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	726b49db-9e8b-41c4-9ba7-f1680914cae3	2015-02-05 21:09:41.917863
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	0d38f8a1-a96e-4a58-95bc-54b8e6ebf941	2015-02-05 21:09:36.441018
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the promoter\n  region for the gene. While no targeted therapeutic has been engaged in clinical\n  trials, the prognostic impact has been studied by a number of meta-analyses. The\n  prognostic impact of promoter hypermethylation has been relatively ambiguous. Many\n  studies have shown significant p-values suggesting poorer prognostic outcome for\n  patients with hypermethylation in colorectal, liver, and younger lung cancer patients.\n  This being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	24666169-4064-4044-95b9-2f7b404d01be	2015-02-05 21:09:36.648273
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: "'AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications."\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	fd8ec0a9-a5b2-411c-8525-5ca12d13b2d6	2015-02-05 21:09:36.86771
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. A\n  study has found this fusion in every FL-HCC sample sequenced, and in vitro assays\n  have shown that the fusion retains kinase function.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	17be0f57-d2b6-43e4-a5b0-4fb4adba3c0d	2015-02-05 21:09:37.049991
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker. A study has found this fusion in every sample sequenced, and\n  in vitro assays have shown that the fusion retains kinase function.\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	8e853277-c6dc-4133-ae88-67df783ac82f	2015-02-05 21:09:37.312955
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	0f10a352-9051-4a5a-b340-bf2806570195	2015-02-05 21:09:37.504328
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have show efficacy in EGFR amplfied tumors, most notably\n  gefitinibb and erlotinib. mutations in EGFR have been shown to confer resistance\n  to these drugs however, and the variant T790M has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has been a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	3e36bcac-3c9f-42f4-9749-ceb5c58b4ea7	2015-02-05 21:09:37.727377
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is seen to be amplified in HER2-positive\n  breast cancer, and is treated in a separate manner than the other subtypes of breast\n  cancer. Apart from being amplified, ERBB2 activating mutations have been shown to\n  have clinical importance in HER2-negative breast cancer. These mutations have shown\n  senitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance\n  of clinical sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	97e7ca2b-74a7-49df-9bb2-7ac512653d54	2015-02-05 21:09:37.929906
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, and has shown\n  significance in endometrial, ovarian and other cancer types as well. ER-positive\n  breast cancer that is resistant to hormone therapy has instigated clinical sequencing\n  efforts to shed light on the mechanisms of this resistance. A number of mutations\n  in the ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	46e52686-2e85-401c-b1fb-eba65af0d03a	2015-02-05 21:09:38.153161
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARa\ndescription: N/A\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	c3aac798-ce33-4d89-a642-8e8d24e148e1	2015-02-05 21:09:42.160749
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: N/A\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	dfbbbd54-0952-4f10-9071-ac754e914502	2015-02-05 21:09:42.345846
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. The active proteins have the potential to activate\n  the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which\n  FGFR can be misregulated vary between cancers. Amplification of the receptors has\n  been observed in lung and breast cancers, coding mutations and deletions have been\n  seen in many cancers, and more recently, FGFR fusions have been demonstrated to\n  have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	394ea043-257b-4fed-9cc1-1a9c3d9af4a8	2015-02-05 21:09:38.346255
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. The active proteins have the potential to activate\n  the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which\n  FGFR can be misregulated vary between cancers. Amplification of the receptors has\n  been observed in lung and breast cancers, coding mutations and deletions have been\n  seen in many cancers, and more recently, FGFR fusions have been demonstrated to\n  have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	ed8165ed-8fbf-44f4-b966-268d26399fcc	2015-02-05 21:09:38.596325
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	e49e14a1-5c00-4357-b36e-830f5bd7481d	2015-02-05 21:09:38.829669
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: N/A\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	4b28b109-b0c3-46cc-9e5c-d755c9c221a1	2015-02-05 21:09:39.01788
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: N/A\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	22c053bd-3a0b-4ba7-a4b7-c0edd9dd2dda	2015-02-05 21:09:39.206784
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: N/A\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	5e24d625-bcba-449e-a701-4b7b8bcf5a76	2015-02-05 21:09:39.405402
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	e603c666-f718-4257-9d63-5805e4dc3186	2015-02-05 21:09:39.621861
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: N/A\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	edcf151c-60c5-4a1f-a51f-f8039c5186d0	2015-02-05 21:09:39.794645
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: N/A\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	79fee7f2-64ca-43a8-8dcb-84e0c9fb8062	2015-02-05 21:09:40.09266
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: N/A\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	94d7e1dd-e385-431d-bc94-5380cefcf9a8	2015-02-05 21:09:40.303291
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: N/A\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	be17c28e-ed3e-41af-8e00-cc807d437b95	2015-02-05 21:09:40.535572
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: N/A\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	b8af09c0-b0a3-42f2-bc1b-9480541b9567	2015-02-05 21:09:40.722646
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: N/A\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	1568a3ee-b524-41e3-85fc-59f1ae80c07d	2015-02-05 21:09:40.921006
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: "'AML with mutated NPM1' is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development."\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	285a2b3f-3242-48fb-9a77-08eda5f4688c	2015-02-05 21:09:41.119376
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: N/A\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	5eccdf1d-8e66-4cc7-8ecb-c527277ea317	2015-02-05 21:09:41.307814
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating  , the 542/545 region of the helical domain, and the 1047 region of the\n  kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the\n  subject of an immense amount of research and development, and PI3K inhibition has\n  seen some limited success in recent clinical trials. While monotherapies seem to\n  be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	6a3fe527-64bc-4f5f-bde2-24a8227774bf	2015-02-05 21:09:41.536601
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: N/A\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	61c6040d-ad7c-4b16-903f-67a265785e7f	2015-02-05 21:09:41.710487
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	c4602a0b-bc8a-4d24-94da-723b60cefcf3	2015-02-05 21:09:42.551132
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: N/A\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	a459ba0a-273e-4a87-90ee-7be7369000b5	2015-02-05 21:09:42.740947
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	64e9a8aa-1385-45cd-a2f7-88d2c4637ccd	2015-02-05 21:09:42.938683
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: N/A\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	0dca0a7b-b791-44d2-82a6-19f3574475d1	2015-02-05 21:09:43.168704
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: N/A\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	103e7663-d4d3-4fb0-80ab-8ca97788b3b0	2015-02-05 21:09:43.411776
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: N/A\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	43b6ad20-bfc0-494a-aa27-c4cb6d137d48	2015-02-05 21:09:44.275957
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	8969ee9c-d151-4b03-836a-9f5e2097ec19	2015-02-05 21:09:45.170019
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: N/A\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	2512f6c6-9923-44d1-87a8-9833538eec13	2015-02-05 21:09:46.035058
48	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	cbba9b09-f78d-4899-9fec-c11a29bf47a2	2015-02-05 21:09:46.096782
49	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	e1918803-67a1-4f4d-abc1-049182850fc0	2015-02-05 21:09:46.110189
50	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	7d5ba8f1-834f-4ce1-9cdb-bffc0f8bf0af	2015-02-05 21:09:46.122321
51	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	3b4e973b-549f-4bc0-b327-968fb59e5ea4	2015-02-05 21:09:46.133085
52	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	c5bee42a-ed96-446b-ac97-bdfc83121aee	2015-02-05 21:09:46.146194
53	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	216f39a4-9486-4320-9223-952f1e0b19bc	2015-02-05 21:09:46.162932
54	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	27d3e6e0-7014-4f4c-88c3-32103ae938f8	2015-02-05 21:09:46.177916
55	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	3ac4fae6-6800-4c2e-bcd0-bb18787249ee	2015-02-05 21:09:46.190923
56	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	a42ea299-eb62-4e55-8cf8-92af7a8ae5cb	2015-02-05 21:09:46.200868
57	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	d63ddc8e-0e99-43c6-82b0-bc7458521976	2015-02-05 21:09:46.212062
58	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	4e7f3609-44fe-4395-a53a-829140a46578	2015-02-05 21:09:46.226335
59	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	6caef931-f10f-4ade-af19-a576930839e4	2015-02-05 21:09:46.238691
60	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	8d8cfcf0-8376-4771-a755-3f96a8e6385b	2015-02-05 21:09:46.250307
61	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	bcc60a51-57a9-42b9-82a0-03ede8a8864b	2015-02-05 21:09:46.263001
62	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	f5db2b37-0826-448f-97e1-9e9889987590	2015-02-05 21:09:46.274765
63	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	ea81335f-3e48-45f2-bddc-5c7fd5ad0cfb	2015-02-05 21:09:46.286248
64	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	99873bd6-ad76-4fff-bea7-d650833c022a	2015-02-05 21:09:46.301085
65	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	b5b978e3-44bf-4c90-a86e-04efa0ec3220	2015-02-05 21:09:46.313444
66	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	e493eeb3-ea8f-4504-bc0c-e719bde8c973	2015-02-05 21:09:46.326587
67	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: EXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	7f2b1c68-dbf0-4ee5-875a-2e62fbfff1aa	2015-02-05 21:09:46.337786
68	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	9d07ec5a-15db-4d17-ba55-6600fa4fc67c	2015-02-05 21:09:46.348471
127	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 1\ndescription: A study by Li√®vre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	a8e84558-e01c-4191-95b3-2bb4aca2c796	2015-02-05 21:09:47.204296
69	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	aee2e3e0-01ac-49ec-b227-0ae5a42feefc	2015-02-05 21:09:46.363715
70	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	ff59b673-95d7-48d9-a93d-1f608590fa45	2015-02-05 21:09:46.376831
71	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	0dce4aca-a709-4566-9ffa-4df744765219	2015-02-05 21:09:46.387173
72	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	d0fc67c2-94a1-4daf-85c3-4407c4a8b4d6	2015-02-05 21:09:46.397893
73	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	01308a72-70ac-4014-8c02-af86b38fbd8e	2015-02-05 21:09:46.410955
74	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	e9050c13-aae9-49fd-bf16-8289ad439980	2015-02-05 21:09:46.422772
75	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	4893b6b0-72eb-4165-b18e-be3a08856afb	2015-02-05 21:09:46.434303
76	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	02543dfd-2bd3-48c9-a283-48fbf6b4964b	2015-02-05 21:09:46.447069
77	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: WHOLE GENE\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	9b6f7d3b-2b84-486b-983f-0016fc2affa5	2015-02-05 21:09:46.462121
78	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	0203cc9b-162e-4645-bc40-2d2129a51b8d	2015-02-05 21:09:46.477202
79	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	a38ada4a-b3bc-4fec-a154-bc6eb1c0cbb7	2015-02-05 21:09:46.489415
80	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	426743a8-68d3-4870-977a-30c4acf453e1	2015-02-05 21:09:46.503028
81	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: R882H\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	d199f92b-7312-4b0e-bbdc-56f9cc10b3cf	2015-02-05 21:09:46.517549
82	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	4e130e25-203b-40ce-b250-08f6f032da0d	2015-02-05 21:09:46.531882
83	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	1f1c20a4-e973-4d0e-83c3-fad60b9ce8d7	2015-02-05 21:09:46.547154
84	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769H\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	36c4f798-e3e7-41f3-b2f6-e0bc039331bd	2015-02-05 21:09:46.563501
85	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: DEL 755-759\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	685adbca-7a19-47c2-bdce-a490b09ffa3d	2015-02-05 21:09:46.576491
86	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: G309A\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	ea7858fa-823d-457a-b292-29b629c9e079	2015-02-05 21:09:46.588957
87	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755S\ndescription: This variant was one of the first of the ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was not shown to be an activating mutation,\n  unlike many of the other variants queried. This mutation was also shown to confer\n  resistance to the tyrosine kinase inhibitor lapatinib in cell lines.\n	1	\N	\N	4a38314e-3ae9-433a-b7e2-f4ca12c885e7	2015-02-05 21:09:46.6009
88	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755W\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	b19fde31-d656-4b8d-9167-0085b1004497	2015-02-05 21:09:46.617635
89	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: P780INS\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	c0725082-c9ff-4af2-b4ba-7158bba0207b	2015-02-05 21:09:46.629383
90	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R678Q\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	80f51b6e-f603-4beb-8bd6-4c538a197bb2	2015-02-05 21:09:46.643146
128	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 2\ndescription: In a study by Pan et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	f6b2084d-5de4-4cb0-ba24-5af27b55c997	2015-02-05 21:09:47.22425
91	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R896C\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	4f477085-e574-4aec-8617-6909bb94c8cf	2015-02-05 21:09:46.654471
92	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V777L\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	3e919ec7-cb33-4c4a-b165-81f397ebcb0d	2015-02-05 21:09:46.666184
93	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V842I\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	30b88695-d759-467d-8bac-c0be115ff0ab	2015-02-05 21:09:46.681211
94	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	7a3b31dd-fab7-4b75-b404-281d6fb54970	2015-02-05 21:09:46.696401
95	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	cd20d9ab-c039-422b-b194-69d540ef9294	2015-02-05 21:09:46.7122
96	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	ffe6ac9d-08e9-43b2-9ed2-8fcc42fc11fe	2015-02-05 21:09:46.728395
97	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	5b0bc2fb-e709-490f-bc28-fe49bf5a6fcb	2015-02-05 21:09:46.741429
98	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	9b98e9cb-a87d-4862-a7fd-9ed6f36e7ee3	2015-02-05 21:09:46.755831
99	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-AFF3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	6381b733-ceaf-487f-9621-6ed8cbbc31ad	2015-02-05 21:09:46.772841
100	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-BICC1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	5b48d4e9-0bfd-4058-a75e-e89c4bbc3603	2015-02-05 21:09:46.787347
101	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CASP7\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	4c9b52bc-b9cb-4084-885b-7bd9ce2bcbc8	2015-02-05 21:09:46.804033
143	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 3\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	9fcf0398-fbd0-48e3-bc8e-0dc4f1c4b916	2015-02-05 21:09:47.421674
102	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CCDC6\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	f315fbba-0611-4ac1-99fa-e7b8b01af35b	2015-02-05 21:09:46.819546
103	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-KIAA1967\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	6f1389eb-aecb-4820-bd80-c299b8629324	2015-02-05 21:09:46.836337
104	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	37bee6e6-c850-445b-a22d-cf05beacd36e	2015-02-05 21:09:46.85419
105	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-OFD1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	14ce06ef-6a8a-4163-8eef-1eb4a2291d73	2015-02-05 21:09:46.865797
106	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-SLC45A3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	43cbddc9-69ee-4334-8083-2abc6d3071d2	2015-02-05 21:09:46.884356
107	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	33f53d24-8899-477f-93b2-0568d85e96f1	2015-02-05 21:09:46.89847
108	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: ITD\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	b6e27f9c-b1a0-4a14-86f7-58a5dddc8e50	2015-02-05 21:09:46.911556
109	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: TKD\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	f622ab69-8ec5-413f-a031-9e913c5ac1a6	2015-02-05 21:09:46.924514
110	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	b059ba1c-b323-4c81-aff7-fa4f5a85ec59	2015-02-05 21:09:46.937905
111	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	d5a8094e-75bf-4479-b29d-4c3e0b0a4c45	2015-02-05 21:09:46.951745
112	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	d327b1b6-29af-4c5a-a1d9-3ec21c2ab996	2015-02-05 21:09:46.971004
144	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 4\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	b6a48cd2-b76c-4f83-a4b8-efd62044acf7	2015-02-05 21:09:47.43202
113	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132G\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	d6608108-cdad-43cb-a3d6-d747c17eb65e	2015-02-05 21:09:46.986085
114	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	dc1a2cd5-8ec6-4b9d-97e7-c46cf4375f38	2015-02-05 21:09:46.998944
115	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	67a24506-6d98-4d60-a1f2-0d4aeed24296	2015-02-05 21:09:47.013342
116	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	e713a819-5058-46df-8c42-76ccd5ccc6da	2015-02-05 21:09:47.02592
117	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	b954f7c2-ff05-4f8c-92fb-f5cc67bf3274	2015-02-05 21:09:47.042048
118	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	3a86316f-b0fc-4680-a7b1-5f6c9fe4db9f	2015-02-05 21:09:47.057415
119	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	a5fb0fd1-3e40-467f-90c1-653dd1e0d860	2015-02-05 21:09:47.076319
120	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 11\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	e943ca88-7f7b-4a9d-8ac1-2542f245e405	2015-02-05 21:09:47.090025
121	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 13\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	83dd186e-b183-43ac-b7e3-7ff9372e45b9	2015-02-05 21:09:47.105377
122	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 14\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	9611827d-64a4-4e8a-9ac1-da3d65310b30	2015-02-05 21:09:47.122031
123	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 17\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop protein. In cell lines, mutations within this domain have been shown to be\n  sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation\n  is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.\n	1	\N	\N	6c7f99e6-4688-4cf1-bcee-755a78ea8d06	2015-02-05 21:09:47.136431
124	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 9\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	c9421b95-e139-4f3b-b49f-669a6c076309	2015-02-05 21:09:47.151866
125	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	eafb9104-ea8b-45af-9364-faed90a9311e	2015-02-05 21:09:47.164245
126	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	0fe7990d-906c-4c4c-b39c-3f115ebf3aa9	2015-02-05 21:09:47.177288
129	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	fd46c900-171a-47a9-918a-fcbed99415c1	2015-02-05 21:09:47.237272
130	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	eac6f14f-ec1f-4982-98ce-2f41ec169f3f	2015-02-05 21:09:47.250032
131	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	76f509f7-dc36-4f71-a5f8-d679cd6f3d9c	2015-02-05 21:09:47.262351
132	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	73c6816c-bd97-4a24-a01c-c768396052a1	2015-02-05 21:09:47.273763
133	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	043abefb-a7c4-45a0-8c78-2415dc05e7ab	2015-02-05 21:09:47.290044
134	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	af342cc8-e61b-4d2a-91a9-b33f65f8e307	2015-02-05 21:09:47.306822
135	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	52e593c8-ccf3-47c1-98fe-6b975afd7cfe	2015-02-05 21:09:47.319184
136	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	efac4086-c9ee-4136-b75f-318ab4db5f5c	2015-02-05 21:09:47.332229
137	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	1263e754-db7a-4fae-bdc0-9317233e36c3	2015-02-05 21:09:47.345542
138	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	2d884d48-26ff-406c-8af7-19c03b51481a	2015-02-05 21:09:47.359803
139	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: EXON 12\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	f928807b-b2d0-4797-966d-e756b7dbb01f	2015-02-05 21:09:47.373665
140	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	b0595b04-888f-4ff8-8493-e9ff69142c5c	2015-02-05 21:09:47.389403
141	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 1\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	97c1b403-a9d0-4d8f-bdde-01badb5c75af	2015-02-05 21:09:47.402583
142	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 2\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	b15df071-f599-490b-8352-48b730cb595e	2015-02-05 21:09:47.412011
145	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	80c3531d-72c4-47b8-ad93-7efeaf4bddd9	2015-02-05 21:09:47.445216
146	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	4b9dafdf-29a5-4df8-9dae-0c2b64303bb9	2015-02-05 21:09:47.455682
147	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	35389929-ee92-458a-b09a-5ee14cc4e1bf	2015-02-05 21:09:47.465526
148	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	05da9be9-6030-42ab-9be5-a5fe10a5fb12	2015-02-05 21:09:47.476546
149	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	cbfc2151-6af1-4dfa-8d47-369204e2a2f1	2015-02-05 21:09:47.48883
150	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	ba332f9b-9a4d-458a-9a09-e310a379e28f	2015-02-05 21:09:47.500194
151	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	7875a998-ccec-4dd6-bdfe-333dcee0986b	2015-02-05 21:09:47.509991
152	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	e838914c-933e-4f12-bb8e-a591e2ac3b61	2015-02-05 21:09:47.523111
153	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	82ce81b7-4c9d-4ce8-b663-1b5461d93064	2015-02-05 21:09:47.534581
154	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	026ada53-a868-4783-ae07-b21374e8aefb	2015-02-05 21:09:47.547197
155	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	072337a2-b1cf-43d3-b421-e7ad8071a272	2015-02-05 21:09:47.559632
156	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	389fcbcc-2b5a-4ee0-9cc9-72ed6e47f468	2015-02-05 21:09:47.574436
157	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 20\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	07acdcb8-c23c-414c-96f1-280489629f2a	2015-02-05 21:09:47.588075
158	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 9\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	68968714-b1c5-4342-8826-dc27aeee619c	2015-02-05 21:09:47.603026
159	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	23d3e9aa-2e47-464e-820b-9d1788a9c0dc	2015-02-05 21:09:47.617128
160	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	344fc2fb-7f98-443e-9bc5-2c0f5857a088	2015-02-05 21:09:47.629617
161	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	c71be4aa-6086-47f2-bb20-5470d766e87e	2015-02-05 21:09:47.64129
162	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	9b4e0325-b713-43cf-b353-219a4fd58b6b	2015-02-05 21:09:47.655339
163	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	238cf814-a8cc-44da-b4d7-96285dfefa82	2015-02-05 21:09:47.667914
164	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	b258632a-ba92-4124-aac2-0cbee1f08e5b	2015-02-05 21:09:47.67882
165	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	546f8ca6-9e42-4295-96da-924a6589ae13	2015-02-05 21:09:47.691925
166	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	b2437a2f-d65d-4614-8f21-88d7d6bfa9c7	2015-02-05 21:09:47.703424
167	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	d124eec6-2cda-4893-82c3-2ea48d3ed006	2015-02-05 21:09:47.712531
168	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	3f4fa837-9ab5-4d12-948e-247d79626d02	2015-02-05 21:09:47.722431
169	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	d69215d6-50d0-4a9f-88d2-ba1f570ca355	2015-02-05 21:09:47.731687
170	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	0698bb6c-b694-4e93-a880-f072112fe33a	2015-02-05 21:09:47.744346
171	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	09c3d764-b402-4da4-8967-7fc317f3c2c4	2015-02-05 21:09:47.755196
172	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	cfc11f50-133c-4b31-aed6-27744c17eee8	2015-02-05 21:09:47.763895
173	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	e41c9fad-30b7-446c-916c-c4a199eabbbb	2015-02-05 21:09:47.774461
174	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	4f97e16c-2589-45bf-8ba0-cf0944f48b02	2015-02-05 21:09:47.785552
175	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	d93d3f58-29d9-43d2-b1fa-5e30d29ff733	2015-02-05 21:09:47.797824
176	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	82fce1af-7dea-42fc-8dce-ea6a6db4cc01	2015-02-05 21:09:47.808181
177	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	b56e2ece-ff5f-496d-823a-b3105f1e2cb0	2015-02-05 21:09:47.81789
178	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	5610713c-bc09-4a95-b915-3251dfc10f0c	2015-02-05 21:09:47.830171
179	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. Although this mutation has not\n  been determined to provide prognostic information in AML, patients with MDS were\n  shown to be at an increased risk of transformation to secondary AML. However, the\n  prognostic importance of U2AF1 mutations in MDS has been debated.\n	1	\N	\N	3542cad2-440e-4617-bdc3-de225513dcce	2015-02-05 21:09:47.841682
180	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. Although this mutation\n  has not been determined to provide prognostic information in AML, patients with\n  MDS were shown to be at an increased risk of transformation to secondary AML. However,\n  the prognostic importance of U2AF1 mutations in MDS has been debated.\n	1	\N	\N	813038c8-5d35-48d6-a55c-9f6497ec9a04	2015-02-05 21:09:47.854137
181	134	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 7\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	ed1e7dfa-8900-4461-99b9-da73d481f751	2015-02-05 21:09:47.864451
182	135	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 9\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	30667149-2968-40f3-bb19-7b798aa51902	2015-02-05 21:09:47.876297
183	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 40\nvariant_id: 131\nrating: 3\n	1	\N	\N	10e6473e-06de-4685-9bd1-f98a15e1c31b	2015-02-05 21:09:48.013004
184	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 40\nvariant_id: 130\nrating: 3\n	1	\N	\N	f6e82c1c-d66f-44a3-a9de-ec895b2f2c26	2015-02-05 21:09:48.069161
185	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 2\nsource_id: 41\nvariant_id: 129\nrating: 3\n	1	\N	\N	0601cd20-b91d-41b2-b74b-f4fa6b514012	2015-02-05 21:09:48.099354
186	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 3\nsource_id: 42\nvariant_id: 129\nrating: 3\n	1	\N	\N	4267ea4a-6ef9-4ba7-81c2-3a96f9538d3a	2015-02-05 21:09:48.132505
187	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 43\nvariant_id: 115\nrating: 4\n	1	\N	\N	caa69312-80fa-41e1-a293-acbf410a811c	2015-02-05 21:09:48.168692
188	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense  , including R233*, has been shown to be inactivating and loss-of-function,\n  but do not have prognostic value in glioblastoma multiforme patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 5\nsource_id: 44\nvariant_id: 115\nrating: 3\n	1	\N	\N	c469ef5c-74c8-4b83-a52f-60627f1d27ab	2015-02-05 21:09:48.211272
189	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 45\nvariant_id: 10\nrating: 5\n	1	\N	\N	eb56b362-f905-4b21-944e-a827068a07d6	2015-02-05 21:09:48.250876
190	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 45\nvariant_id: 14\nrating: 5\n	1	\N	\N	54e78fd5-6b53-4d62-bb9e-14eaab3bb33f	2015-02-05 21:09:48.286357
191	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 45\nvariant_id: 15\nrating: 5\n	1	\N	\N	fcfd983f-546c-4243-93ac-19dc90aea1b4	2015-02-05 21:09:48.317946
192	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 46\nvariant_id: 12\nrating: 3\n	1	\N	\N	bbc43a79-5508-4990-9ab2-676a6db19cdc	2015-02-05 21:09:48.354334
193	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1  .\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 47\nvariant_id: 92\nrating: 5\n	1	\N	\N	63582a60-8fef-4e97-a340-b2c154048731	2015-02-05 21:09:48.386667
194	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with M2 FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 47\nvariant_id: 92\nrating: 4\n	1	\N	\N	3f284385-a30d-4e6e-8ace-70ae211c2867	2015-02-05 21:09:48.428976
195	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 47\nvariant_id: 92\nrating: 4\n	1	\N	\N	27c66034-0b5e-4e6c-b9a8-c3af2f276d96	2015-02-05 21:09:48.477917
196	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult AML patients (median age 43)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 92\nrating: 4\n	1	\N	\N	166be0a5-328c-47af-967b-b52b920c41a4	2015-02-05 21:09:48.506795
197	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 92\nrating: 4\n	1	\N	\N	dd83c232-1019-4841-aaf3-d9e3f2a983b5	2015-02-05 21:09:48.538753
198	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 50\nvariant_id: 92\nrating: 4\n	1	\N	\N	ea394d18-c62e-434c-8fe3-f11f86f48ada	2015-02-05 21:09:48.565594
199	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 51\nvariant_id: 92\nrating: 4\n	1	\N	\N	8d7de1a2-660d-4a74-943d-2b3badd78e28	2015-02-05 21:09:48.598999
200	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 52\nvariant_id: 92\nrating: 4\n	1	\N	\N	110173c5-d6ec-49e7-b81a-aa4bdd7f71c7	2015-02-05 21:09:48.622697
201	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 53\nvariant_id: 92\nrating: 4\n	1	\N	\N	9fa44c17-0aa8-40f4-bce8-635e4433fad0	2015-02-05 21:09:48.650911
202	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 92\nrating: 4\n	1	\N	\N	a813f9b7-5409-4c34-953a-f05aee5715f4	2015-02-05 21:09:48.684106
203	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 92\nrating: 4\n	1	\N	\N	321ff35c-8718-4d2a-bf6d-dca652fd9a2f	2015-02-05 21:09:48.712345
204	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 93\nrating: 4\n	1	\N	\N	7dfefe0f-f976-41e7-90a2-f5ba7a2ae67b	2015-02-05 21:09:48.7359
205	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 93\nrating: 4\n	1	\N	\N	19d5b5dc-f2d8-4572-a22c-fc4a854cb6d1	2015-02-05 21:09:48.761402
206	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 7\nsource_id: 56\nvariant_id: 92\nrating: 3\n	1	\N	\N	a53fa2c4-e448-4ab8-a5a2-29fb557a9ab0	2015-02-05 21:09:48.789734
207	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 6\ndisease_id: 7\nsource_id: 56\nvariant_id: 92\nrating: 3\n	1	\N	\N	f54acd6c-cadb-4d7a-869b-aacff2c6e9ee	2015-02-05 21:09:48.803151
208	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 7\nsource_id: 57\nvariant_id: 92\nrating: 3\n	1	\N	\N	51675d47-952a-4c02-b99e-752897c0efce	2015-02-05 21:09:48.837142
209	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Frequency of NPM1 mutations was not different in normal karyotype AML patients\n  with CEPBA, NRAS or KIT mutations.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 47\nvariant_id: 92\nrating: 3\n	1	\N	\N	5e346c3a-e388-4322-8c6d-81e9f84daac5	2015-02-05 21:09:48.870763
210	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with M3 FAB subtype (0/55).\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 53\nvariant_id: 92\nrating: 3\n	1	\N	\N	4001f689-9c03-4a73-9f55-27f748e206ce	2015-02-05 21:09:48.905556
211	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 53\nvariant_id: 92\nrating: 3\n	1	\N	\N	c01a96c7-6068-48a0-bc16-251a9f12fd18	2015-02-05 21:09:48.944702
212	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 58\nvariant_id: 92\nrating: 3\n	1	\N	\N	ab651ca8-56cc-4e95-ab0f-86c83e7458ac	2015-02-05 21:09:48.974045
213	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 58\nvariant_id: 92\nrating: 3\n	1	\N	\N	ae1a68a8-f8af-4c72-95c2-f792e97e1616	2015-02-05 21:09:49.000736
214	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 59\nvariant_id: 92\nrating: 3\n	1	\N	\N	1540c6bd-736b-420a-88dd-4c4ee662805a	2015-02-05 21:09:49.030291
215	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  was longer for patients with Exon 12 NPM1  .\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 53\nvariant_id: 92\nrating: 3\n	1	\N	\N	bd3067c2-a1d4-4a2a-966a-f866e6e09cdd	2015-02-05 21:09:49.05653
216	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 60\nvariant_id: 92\nrating: 3\n	1	\N	\N	20cc0bc1-9bc2-4818-9e9e-88bcf25481d8	2015-02-05 21:09:49.087986
217	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 61\nvariant_id: 92\nrating: 3\n	1	\N	\N	0b46a1ae-7711-4873-9b19-3f302a38eb37	2015-02-05 21:09:49.118474
218	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 62\nvariant_id: 92\nrating: 3\n	1	\N	\N	cc3aec9f-cc8c-425b-b4ef-b831a913558d	2015-02-05 21:09:49.14889
219	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 63\nvariant_id: 92\nrating: 3\n	1	\N	\N	0e88a528-ab19-48a8-ab3d-1f0e5a86b41f	2015-02-05 21:09:49.175162
220	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 52\nvariant_id: 92\nrating: 3\n	1	\N	\N	eed9cd0a-f6b6-491f-a428-6612eab9d954	2015-02-05 21:09:49.202616
221	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 64\nvariant_id: 93\nrating: 3\n	1	\N	\N	aed22102-794b-4592-ba49-4cfae8f11b65	2015-02-05 21:09:49.229319
222	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 7\nsource_id: 58\nvariant_id: 92\nrating: 2\n	1	\N	\N	b7f72da4-b73c-4257-8dc2-9cb12c6409e5	2015-02-05 21:09:49.252762
242	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 13\nsource_id: 74\nvariant_id: 56\nrating: 3\n	1	\N	\N	5bf8fb8b-a8f6-4926-a79b-f12bb5fce098	2015-02-05 21:09:49.865024
223	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 7\nsource_id: 65\nvariant_id: 92\nrating: 2\n	1	\N	\N	1c3712e1-9331-4a43-ab7f-28fee1422e3c	2015-02-05 21:09:49.280677
224	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 8\ndisease_id: 7\nsource_id: 66\nvariant_id: 92\nrating: 2\n	1	\N	\N	b8441c12-f359-43b2-97d8-5a25849e2c43	2015-02-05 21:09:49.309502
225	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 92\nrating: 2\n	1	\N	\N	beeb30c6-b97b-4db9-bd2c-04edad23da0a	2015-02-05 21:09:49.337409
226	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 67\nvariant_id: 92\nrating: 2\n	1	\N	\N	cb6c0196-ccdf-41d9-9b20-3bb2838157cb	2015-02-05 21:09:49.362997
227	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence either IDH1 or IDH2 mutations than those wildtype for both IDH1 and\n  IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 65\nvariant_id: 92\nrating: 2\n	1	\N	\N	20530440-c522-4a28-93b5-2bb8c1a03592	2015-02-05 21:09:49.389322
228	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 68\nvariant_id: 92\nrating: 2\n	1	\N	\N	5e73c89f-791c-4b8d-ac85-c3dd0304e9f7	2015-02-05 21:09:49.420113
229	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 68\nvariant_id: 92\nrating: 2\n	1	\N	\N	0ff98b72-9a89-428e-b4ee-55d5c483eb93	2015-02-05 21:09:49.448008
230	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 69\nvariant_id: 92\nrating: 3\n	1	\N	\N	eef7d673-ec4a-409f-a2ca-acbb305fae88	2015-02-05 21:09:49.479327
231	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 6\nsource_id: 70\nvariant_id: 11\nrating: 3\n	1	\N	\N	4036eec6-bacc-4f0d-ad49-1a636e64c6ed	2015-02-05 21:09:49.504363
232	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 6\nsource_id: 70\nvariant_id: 11\nrating: 3\n	1	\N	\N	742d17ed-9613-4af2-83ed-adee3e364347	2015-02-05 21:09:49.529926
233	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 9\ndisease_id: 8\nsource_id: 71\nvariant_id: 117\nrating: 3\n	1	\N	\N	a11d959f-7b59-4c1b-b315-65b20bd13727	2015-02-05 21:09:49.561104
234	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 9\ndisease_id: 8\nsource_id: 71\nvariant_id: 118\nrating: 3\n	1	\N	\N	c29a55fd-19cf-4ecb-ada9-10f5111ebafb	2015-02-05 21:09:49.597654
235	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with the ABL T315I mutation, tumors have shown to be resistant to\n  imatinib treatment.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 9\nsource_id: 72\nvariant_id: 2\nrating: 4\n	1	\N	\N	244076e0-fa73-41e4-8daf-be65977b9707	2015-02-05 21:09:49.641948
236	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in "anti-tumor activitiy".\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 10\nsource_id: 73\nvariant_id: 57\nrating: 3\n	1	\N	\N	b7732787-f9e7-4b6c-8eb0-7055e4d55316	2015-02-05 21:09:49.692402
237	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 11\nsource_id: 74\nvariant_id: 52\nrating: 3\n	1	\N	\N	8152d4bc-1346-4cf2-ae42-3631ea040660	2015-02-05 21:09:49.727581
238	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 11\nsource_id: 74\nvariant_id: 54\nrating: 3\n	1	\N	\N	61034d11-f5a2-4268-b6f8-64e4e437e78e	2015-02-05 21:09:49.758285
239	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 11\nsource_id: 74\nvariant_id: 55\nrating: 3\n	1	\N	\N	0d457dcc-e12e-4f5a-82c7-56dbf8199f62	2015-02-05 21:09:49.784411
240	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 10\nsource_id: 74\nvariant_id: 53\nrating: 3\n	1	\N	\N	305cc062-b8e8-46bf-a89c-1c4cdfd3a1f5	2015-02-05 21:09:49.807125
241	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 12\nsource_id: 74\nvariant_id: 59\nrating: 3\n	1	\N	\N	451cc2fa-c2f0-485b-be67-3040fe75550e	2015-02-05 21:09:49.833501
243	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 14\nsource_id: 74\nvariant_id: 58\nrating: 3\n	1	\N	\N	0d567dfb-6738-4e50-8880-d5f70475f25a	2015-02-05 21:09:49.894961
244	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 15\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	58a0e3f9-0d07-4739-a9c4-d5dca1ab7ccb	2015-02-05 21:09:49.923677
245	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 15\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	67964200-3ab7-4960-81df-d72c57a9f71e	2015-02-05 21:09:49.93463
246	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 10\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	10ad25e7-824f-47ed-9515-2f67aa195c9b	2015-02-05 21:09:49.956622
247	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 10\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	98e7c55a-4fcf-4d14-bdc8-51281a314306	2015-02-05 21:09:49.967343
248	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 16\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	54af5ee2-93f4-4172-b239-8a499a77249b	2015-02-05 21:09:49.997135
249	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 16\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	93bfa6b0-6931-4d44-844f-e36a66bae03b	2015-02-05 21:09:50.009992
250	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 17\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	a98e34f0-2c13-47d3-992e-bdfceab4439b	2015-02-05 21:09:50.038316
251	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 17\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	e8c584cb-f5cc-436d-8136-a9f873a56ed2	2015-02-05 21:09:50.049149
252	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 18\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	14609b4d-c2c6-4df8-a533-e8998e622027	2015-02-05 21:09:50.080413
253	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 18\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	70f4e2d2-b203-4ffb-8565-46e114f0a50c	2015-02-05 21:09:50.092298
254	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 19\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	53e91798-4dcb-49f4-a331-b11de09c3501	2015-02-05 21:09:50.120729
255	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 19\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	a3f69ab0-768b-4a9b-882e-185435569e1a	2015-02-05 21:09:50.131394
256	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 20\nsource_id: 75\nvariant_id: 72\nrating: 4\n	1	\N	\N	ee2efdae-9a68-42dd-9918-43b088be7417	2015-02-05 21:09:50.157183
257	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 6\nsource_id: 76\nvariant_id: 78\nrating: 3\n	1	\N	\N	f099124c-17e3-49da-b68c-abf68ef5621a	2015-02-05 21:09:50.190403
292	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 24\nsource_id: 87\nvariant_id: 90\nrating: 3\n	1	\N	\N	98591fbf-a89c-4344-b6fe-948347932de0	2015-02-05 21:09:51.004812
258	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 6\nsource_id: 76\nvariant_id: 78\nrating: 3\n	1	\N	\N	3563b504-edb3-4ce3-bcb4-bfb2b61b8b33	2015-02-05 21:09:50.202421
259	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 6\nsource_id: 76\nvariant_id: 78\nrating: 3\n	1	\N	\N	4ad0bcdb-8ce6-4e00-857c-4f8ba508b829	2015-02-05 21:09:50.214098
260	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 3\nsource_id: 77\nvariant_id: 78\nrating: 3\n	1	\N	\N	a234f92b-e899-4e77-91ad-b39652708dce	2015-02-05 21:09:50.248565
261	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 3\nsource_id: 77\nvariant_id: 78\nrating: 3\n	1	\N	\N	f18c8d87-c856-44c1-8f77-595ce5b68c8c	2015-02-05 21:09:50.259191
262	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 3\nsource_id: 77\nvariant_id: 78\nrating: 3\n	1	\N	\N	cd1a2bc1-6aaa-4175-8812-ddbed333aea1	2015-02-05 21:09:50.272659
263	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 16\ndisease_id: 21\nsource_id: 78\nvariant_id: 79\nrating: 3\n	1	\N	\N	253c3755-8fbc-474e-a8b2-23b71d0c7d2d	2015-02-05 21:09:50.303385
264	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 6\nsource_id: 76\nvariant_id: 78\nrating: 3\n	1	\N	\N	1c01a551-edf1-43cd-9f90-83c522cbb284	2015-02-05 21:09:50.330602
265	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 21\nsource_id: 79\nvariant_id: 79\nrating: 3\n	1	\N	\N	279e7254-7b3a-4979-8878-8d84f6c73df7	2015-02-05 21:09:50.359533
266	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a case study, a patient with anal melanoma harboring an exon 11 KIT mutation\n  showed marked response 4 months after imatinib treatment.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 6\nsource_id: 80\nvariant_id: 78\nrating: 3\n	1	\N	\N	8a71aec0-8de9-47dc-8ba9-19902e702df6	2015-02-05 21:09:50.390914
267	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 47\nrating: 5\n	1	\N	\N	24419cba-102a-4d1d-bbb8-e850e1bfd02e	2015-02-05 21:09:50.418301
268	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 47\nrating: 5\n	1	\N	\N	87cd5ccd-b685-4549-bf6b-0623ddc4ba5c	2015-02-05 21:09:50.431857
269	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 48\nrating: 5\n	1	\N	\N	f7d10b73-6435-4398-b092-982b84b48664	2015-02-05 21:09:50.460597
270	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 48\nrating: 5\n	1	\N	\N	4b1c1fbc-b210-4e21-8f60-962d3f94888c	2015-02-05 21:09:50.471134
271	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 49\nrating: 5\n	1	\N	\N	4391bc44-6551-4f12-a794-b22a451ce6dc	2015-02-05 21:09:50.496605
272	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 49\nrating: 5\n	1	\N	\N	92b47a7f-26d3-4123-871c-1f2af266beed	2015-02-05 21:09:50.506352
273	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 50\nrating: 5\n	1	\N	\N	f24b57c6-99ee-4f38-a960-15e098738d95	2015-02-05 21:09:50.533397
274	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 50\nrating: 5\n	1	\N	\N	3917564f-1315-4329-9a33-d8a710fcfd79	2015-02-05 21:09:50.543932
275	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 51\nrating: 5\n	1	\N	\N	b4ba40e2-39a1-4f46-9f05-1fd7be3cf3d2	2015-02-05 21:09:50.566591
276	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 51\nrating: 5\n	1	\N	\N	d1a9df02-7201-42fc-be05-4338fd875a98	2015-02-05 21:09:50.580072
277	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 47\nrating: 3\n	1	\N	\N	d463724f-d615-42c5-83ec-893d59702171	2015-02-05 21:09:50.60655
278	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 48\nrating: 3\n	1	\N	\N	e7689b89-8d8e-4a6b-9033-1f01eb5c180e	2015-02-05 21:09:50.633554
279	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 49\nrating: 3\n	1	\N	\N	5e30f3ec-51c6-46ae-a016-498a0788c71f	2015-02-05 21:09:50.657581
280	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 50\nrating: 3\n	1	\N	\N	29f09f31-c173-4b0b-a550-91bcb5176c44	2015-02-05 21:09:50.686803
281	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 51\nrating: 3\n	1	\N	\N	10ee8d1a-93f4-457f-b146-1230873e1fc6	2015-02-05 21:09:50.714846
282	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from an ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 22\nsource_id: 83\nvariant_id: 114\nrating: 4\n	1	\N	\N	e058cadc-73bb-442d-b9e5-352bcd7d5fc3	2015-02-05 21:09:50.755604
283	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from an ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 22\nsource_id: 83\nvariant_id: 32\nrating: 4\n	1	\N	\N	8755d5b1-be8c-4504-8518-d7f04ee68913	2015-02-05 21:09:50.785865
284	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 3\nsource_id: 84\nvariant_id: 7\nrating: 3\n	1	\N	\N	9f216d8c-43ba-48b0-b3eb-6e96671224ba	2015-02-05 21:09:50.810957
285	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 19\ndisease_id: 3\nsource_id: 85\nvariant_id: 6\nrating: 4\n	1	\N	\N	fe0e79d9-5c66-4441-b58a-5238bbeaa000	2015-02-05 21:09:50.841585
286	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, this\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 19\ndisease_id: 3\nsource_id: 85\nvariant_id: 5\nrating: 4\n	1	\N	\N	4a19132e-4d4b-422d-88be-7dc634df93a3	2015-02-05 21:09:50.870836
287	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 19\ndisease_id: 3\nsource_id: 85\nvariant_id: 4\nrating: 4\n	1	\N	\N	ef624f85-1d66-4ea0-ad9f-cf459226f17b	2015-02-05 21:09:50.899982
288	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein undergoes proteolysis in APL cells upon ATRA treatment in cell\n  lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 23\nsource_id: 86\nvariant_id: 113\nrating: 5\n	1	\N	\N	b938e90b-8cdb-4077-ba83-a5bf33897cff	2015-02-05 21:09:50.929894
289	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment at pharmalogical\n  concentrations.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 23\nsource_id: 86\nvariant_id: 116\nrating: 5\n	1	\N	\N	398c100a-47f7-46ee-8da8-005e0f5991e3	2015-02-05 21:09:50.947514
290	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 24\nsource_id: 87\nvariant_id: 31\nrating: 3\n	1	\N	\N	e1126f4c-aced-4c0a-832f-f3c3c60a723c	2015-02-05 21:09:50.97002
291	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 24\nsource_id: 87\nvariant_id: 31\nrating: 3\n	1	\N	\N	4cd05f7e-a02d-40cf-a215-822c71018eaf	2015-02-05 21:09:50.984201
293	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 24\nsource_id: 87\nvariant_id: 90\nrating: 3\n	1	\N	\N	f8ee175c-fb11-4ec9-9f06-e1e0ded4f7dd	2015-02-05 21:09:51.016032
294	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 88\nvariant_id: 119\nrating: 3\n	1	\N	\N	aae324f4-5db4-479f-aae7-707a497841d7	2015-02-05 21:09:51.040443
295	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 88\nvariant_id: 120\nrating: 3\n	1	\N	\N	6b5fae9a-a414-4ea8-b397-8684d8fbeca4	2015-02-05 21:09:51.05749
296	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 134\nrating: 4\n	1	\N	\N	18443122-48be-4ed8-8704-bccf774664fe	2015-02-05 21:09:51.081282
297	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 135\nrating: 4\n	1	\N	\N	c589e65f-1a1b-46d3-883b-45dd2c359409	2015-02-05 21:09:51.100786
298	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 134\nrating: 3\n	1	\N	\N	646ae70c-c00b-464e-b1e0-3e4d0d9ed92d	2015-02-05 21:09:51.121615
299	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No difference in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 134\nrating: 3\n	1	\N	\N	b4baf342-41d3-4d5f-87d4-0a7382f4f2f6	2015-02-05 21:09:51.142848
300	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 134\nrating: 3\n	1	\N	\N	be2c9408-f4d2-4d46-8712-11637b474773	2015-02-05 21:09:51.162899
301	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 134\nrating: 3\n	1	\N	\N	c9ecf618-e3a0-47eb-86ed-74cd55766893	2015-02-05 21:09:51.186191
302	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 91\nvariant_id: 134\nrating: 3\n	1	\N	\N	a08de5bc-beec-492b-bd0b-5cac5a58c32c	2015-02-05 21:09:51.20727
303	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 134\nrating: 3\n	1	\N	\N	1eb5280f-e1bc-40c2-b658-eda041f2c987	2015-02-05 21:09:51.228841
304	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 135\nrating: 3\n	1	\N	\N	3da09011-d677-461e-b7b2-25360b62cf96	2015-02-05 21:09:51.247902
305	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No difference in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 135\nrating: 3\n	1	\N	\N	1357e616-bf28-4382-bd50-739f5db8e7d9	2015-02-05 21:09:51.264901
306	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 135\nrating: 3\n	1	\N	\N	d67e8e42-5f92-4273-be76-96c35a80d6a7	2015-02-05 21:09:51.288776
307	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 135\nrating: 3\n	1	\N	\N	f0e487f9-bb89-449c-a35b-d0d7df657b55	2015-02-05 21:09:51.306396
308	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 91\nvariant_id: 135\nrating: 3\n	1	\N	\N	a6389e97-b280-452b-b24c-b2d16b420d12	2015-02-05 21:09:51.328191
326	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as R249 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 125\nrating: 3\n	1	\N	\N	6f2fe8ac-dd0e-4172-a56f-869325f31961	2015-02-05 21:09:51.699861
309	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 135\nrating: 3\n	1	\N	\N	89bf63b9-d847-4e59-9bab-7de689b000e5	2015-02-05 21:09:51.346461
310	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: in WT1 were associated with a worse overall prognosis than patients wildtype\n  for WT1 in young patients (15-50), primarily because of increased risk of disease\n  recurrence\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 92\nvariant_id: 134\nrating: 2\n	1	\N	\N	012aa946-066f-41c1-90e5-6380ca0e21f9	2015-02-05 21:09:51.364194
311	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: in WT1 were associated with increased risk of recurrence in young patients (15-50)\n  with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 92\nvariant_id: 134\nrating: 2\n	1	\N	\N	bc4f55c4-eb2a-437b-859a-c63010c9a7a8	2015-02-05 21:09:51.388301
312	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 93\nvariant_id: 132\nrating: 3\n	1	\N	\N	9e4d174e-a2ce-499a-90b4-51775543652d	2015-02-05 21:09:51.406818
313	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 93\nvariant_id: 133\nrating: 3\n	1	\N	\N	0bd9c8e1-047b-44d2-9504-b8554093c8f5	2015-02-05 21:09:51.423862
314	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 94\nvariant_id: 132\nrating: 3\n	1	\N	\N	6eb332a9-c997-4194-abd1-b9ef538c8902	2015-02-05 21:09:51.446948
315	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as S34Y/F\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 94\nvariant_id: 133\nrating: 3\n	1	\N	\N	9c9c1779-14f8-4935-8a4a-58d5096ca7e7	2015-02-05 21:09:51.463819
316	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 93\nvariant_id: 132\nrating: 2\n	1	\N	\N	92d095f1-8dc6-48e3-b382-698f1f5a5a93	2015-02-05 21:09:51.480087
317	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 93\nvariant_id: 133\nrating: 2\n	1	\N	\N	159145df-a789-4471-a75c-cd8c2bd123dc	2015-02-05 21:09:51.50229
318	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 93\nvariant_id: 132\nrating: 2\n	1	\N	\N	36ac59a2-7b99-4beb-8847-d564a882a238	2015-02-05 21:09:51.51916
319	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 93\nvariant_id: 133\nrating: 2\n	1	\N	\N	562e6bb4-f449-4c80-9595-d77198ad9ade	2015-02-05 21:09:51.538401
320	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumor with wild type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 21\ndisease_id: 11\nsource_id: 95\nvariant_id: 121\nrating: 3\n	1	\N	\N	3d6c921b-22f1-421b-a471-22f020f5dd05	2015-02-05 21:09:51.559296
321	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 121\nrating: 3\n	1	\N	\N	89194b0f-37b9-4e15-8c90-0c0286c28232	2015-02-05 21:09:51.576832
322	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R248 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 122\nrating: 3\n	1	\N	\N	af45e034-ac8f-47fd-8542-643c31c8db7a	2015-02-05 21:09:51.601549
323	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 122\nrating: 3\n	1	\N	\N	ce93fc6d-a5b4-45e6-99fe-a1dc2286d0f2	2015-02-05 21:09:51.618732
324	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 123\nrating: 3\n	1	\N	\N	3efffa49-36c6-4793-bcb3-aa9bde4e2890	2015-02-05 21:09:51.637728
325	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as R249 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 124\nrating: 3\n	1	\N	\N	0779af8a-65cb-456b-be0c-ab483adfed24	2015-02-05 21:09:51.675705
327	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R273 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 126\nrating: 3\n	1	\N	\N	f6131ba3-40ba-4244-b2b0-0e7d18e35811	2015-02-05 21:09:51.718468
328	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 126\nrating: 3\n	1	\N	\N	82774afa-3025-4e98-a59e-f35b46bf495f	2015-02-05 21:09:51.738979
329	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R273 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 127\nrating: 3\n	1	\N	\N	daaed03e-b9b7-47e5-b244-a9319b2ff437	2015-02-05 21:09:51.757934
330	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 127\nrating: 3\n	1	\N	\N	63c72a85-d37f-4e90-ad61-a276e7d648d1	2015-02-05 21:09:51.776314
331	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as V173 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 128\nrating: 3\n	1	\N	\N	9021e870-6cd5-4b56-9f7f-39bf97d77134	2015-02-05 21:09:51.795688
332	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 98\nvariant_id: 118\nrating: 5\n	1	\N	\N	e7a10466-091d-458a-9afb-a40b98a8a7dc	2015-02-05 21:09:51.818448
333	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 99\nvariant_id: 118\nrating: 4\n	1	\N	\N	6d09223d-c0f1-41b3-ba78-71ea648f6a2b	2015-02-05 21:09:51.839214
334	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 in vitro led to strong repression of tyroid cancer cell growth.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 22\ndisease_id: 8\nsource_id: 100\nvariant_id: 118\nrating: 3\n	1	\N	\N	4fcc0242-2392-4675-82b5-78b4027406d5	2015-02-05 21:09:51.860815
335	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 P124S confers increased resistance to inhibition by AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 23\ndisease_id: 6\nsource_id: 101\nvariant_id: 88\nrating: 3\n	1	\N	\N	9de61ca0-de89-4061-b15f-30638df972b2	2015-02-05 21:09:51.886234
336	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 23\ndisease_id: 6\nsource_id: 101\nvariant_id: 89\nrating: 3\n	1	\N	\N	8cbc8cfb-6703-494d-8e77-c288804e4671	2015-02-05 21:09:51.907917
337	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 11\nsource_id: 102\nvariant_id: 108\nrating: 4\n	1	\N	\N	a2de09df-c4e4-40b9-b23e-c22c715edeab	2015-02-05 21:09:51.930796
338	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 11\nsource_id: 102\nvariant_id: 109\nrating: 4\n	1	\N	\N	9c19195d-bb3c-49de-bb5a-3747d441a53b	2015-02-05 21:09:51.949432
339	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 11\nsource_id: 102\nvariant_id: 112\nrating: 4\n	1	\N	\N	3fbe5ea9-b730-4ed5-bbad-d99289d3b510	2015-02-05 21:09:51.970192
340	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 26\nsource_id: 103\nvariant_id: 108\nrating: 4\n	1	\N	\N	3106dd82-720d-454b-83b8-96085d30e694	2015-02-05 21:09:51.994366
341	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 26\nsource_id: 103\nvariant_id: 109\nrating: 4\n	1	\N	\N	da36785c-0dc1-4ad7-bce9-09b0024ceb71	2015-02-05 21:09:52.017287
342	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 26\nsource_id: 103\nvariant_id: 112\nrating: 4\n	1	\N	\N	a1400e55-8a0f-4326-b95c-4e06d0c5f447	2015-02-05 21:09:52.037745
343	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 26\nsource_id: 104\nvariant_id: 108\nrating: 3\n	1	\N	\N	cdb65ac4-d4f0-45be-900d-0a461293cc60	2015-02-05 21:09:52.061971
344	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 26\nsource_id: 104\nvariant_id: 108\nrating: 3\n	1	\N	\N	bf936794-fd80-4c63-a9a0-6749baee8104	2015-02-05 21:09:52.075674
345	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 26\nsource_id: 104\nvariant_id: 109\nrating: 3\n	1	\N	\N	895690d1-4e46-4370-b1af-4551d6cf17f5	2015-02-05 21:09:52.098669
346	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 26\nsource_id: 104\nvariant_id: 109\nrating: 3\n	1	\N	\N	5df87bd3-1e8f-4c3f-af8d-60b1a93c3e8b	2015-02-05 21:09:52.108743
347	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase\n  resistance to drugs such as  panitumumab or cetuximab.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 26\nsource_id: 104\nvariant_id: 112\nrating: 3\n	1	\N	\N	4ada1c73-bba7-44c2-bfbf-406b6088a6da	2015-02-05 21:09:52.130361
348	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase\n  resistance to drugs such as  panitumumab or cetuximab.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 26\nsource_id: 104\nvariant_id: 112\nrating: 3\n	1	\N	\N	41ae5499-5ced-40de-ad10-a3dcf8e2f747	2015-02-05 21:09:52.141967
349	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates and shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 110\nrating: 3\n	1	\N	\N	c10bb289-6dcc-48a6-a755-b8a83595e573	2015-02-05 21:09:52.163003
350	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 111\nrating: 3\n	1	\N	\N	a59fbde4-c292-4c4a-8cd0-03c5fc33e33d	2015-02-05 21:09:52.182291
351	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 11\nsource_id: 106\nvariant_id: 112\nrating: 3\n	1	\N	\N	4eac872b-39c8-4573-b577-477b9bbb2d7e	2015-02-05 21:09:52.206468
352	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 11\nsource_id: 106\nvariant_id: 108\nrating: 3\n	1	\N	\N	0fd336a2-f7c4-4cf8-aee9-10d8eef6d045	2015-02-05 21:09:52.232458
353	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 11\nsource_id: 106\nvariant_id: 109\nrating: 3\n	1	\N	\N	4565683d-ece0-451e-b842-4bd24605b9a7	2015-02-05 21:09:52.255184
354	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 104\nvariant_id: 108\nrating: 3\n	1	\N	\N	0b285b0d-e678-4f41-9e88-65652de7410c	2015-02-05 21:09:52.275581
355	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 104\nvariant_id: 109\nrating: 3\n	1	\N	\N	141d71ce-2ef9-4ce7-9780-189d414c3a00	2015-02-05 21:09:52.296591
356	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 104\nvariant_id: 112\nrating: 3\n	1	\N	\N	292840d4-57cd-4ce6-a9ec-067dd6eab714	2015-02-05 21:09:52.316405
357	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 107\nvariant_id: 108\nrating: 2\n	1	\N	\N	22ccc016-f8d8-47c4-af7a-e29d3e1fcf64	2015-02-05 21:09:52.341205
358	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 107\nvariant_id: 109\nrating: 2\n	1	\N	\N	7fa45918-9e9a-42f7-8efe-273c2d02d36c	2015-02-05 21:09:52.361014
359	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 103\nrating: 4\n	1	\N	\N	64d46828-c85b-4513-b32e-a0aaa4993352	2015-02-05 21:09:52.392485
360	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation resulting in imatinib resistance, crenolanib\n  showed relatively high potency.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 104\nrating: 4\n	1	\N	\N	0299d3b7-8e7b-4b5d-95c1-21e63b776a73	2015-02-05 21:09:52.416189
361	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation resulting in imatinib resistance, crenolanib\n  showed relatively high potency.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 105\nrating: 4\n	1	\N	\N	1382c9c5-3861-4df2-aef5-aaa53891a85e	2015-02-05 21:09:52.43832
362	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA deletion I843 mutation resulting in imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 106\nrating: 4\n	1	\N	\N	7a9544b9-341e-4efa-914e-cb0af72f7f71	2015-02-05 21:09:52.459852
363	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA DI842-843IM mutation resulting in imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 107\nrating: 4\n	1	\N	\N	2f2ca9c1-efd2-4580-9fba-e7c7a1423f6a	2015-02-05 21:09:52.476516
364	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 27\nsource_id: 79\nvariant_id: 104\nrating: 4\n	1	\N	\N	9973b540-3fa3-43f8-b39b-5c43db7ba640	2015-02-05 21:09:52.498223
365	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 27\nsource_id: 109\nvariant_id: 104\nrating: 4\n	1	\N	\N	8ec6acd8-89ff-4b9b-8d11-9db4696f64e9	2015-02-05 21:09:52.521997
366	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumor harboring PDGFRA D842V mutation are likely to be benign\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 27\nsource_id: 110\nvariant_id: 104\nrating: 3\n	1	\N	\N	30adc71f-3eb4-4945-bb13-91cf71024f17	2015-02-05 21:09:52.547136
367	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 6\nsource_id: 111\nvariant_id: 100\nrating: 4\n	1	\N	\N	7b726977-db40-4bba-a05c-56e32785b842	2015-02-05 21:09:52.577151
368	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 100\nrating: 3\n	1	\N	\N	47cb5fe9-3d1c-4306-b39c-9ff48ce040ea	2015-02-05 21:09:52.600764
369	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 100\nrating: 3\n	1	\N	\N	dc53c545-399d-45ea-bcb3-fd53a7e1de4c	2015-02-05 21:09:52.644069
370	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patient does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 112\nvariant_id: 95\nrating: 3\n	1	\N	\N	fc9f98e2-8ed7-4d12-beca-464053d3806e	2015-02-05 21:09:52.667026
371	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 113\nvariant_id: 98\nrating: 3\n	1	\N	\N	e7343349-6ce9-47cd-8d9b-72e56cd35b83	2015-02-05 21:09:52.691411
372	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 114\nvariant_id: 95\nrating: 3\n	1	\N	\N	5b7a2cfc-0023-42c7-b34d-bc0166652fbd	2015-02-05 21:09:52.711008
373	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 114\nvariant_id: 96\nrating: 3\n	1	\N	\N	f444d73e-ad49-490d-93db-e122cc2fc415	2015-02-05 21:09:52.732995
374	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 114\nvariant_id: 97\nrating: 3\n	1	\N	\N	dc773c10-3514-47e8-971e-fe598409d406	2015-02-05 21:09:52.750139
375	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 115\nvariant_id: 94\nrating: 3\n	1	\N	\N	91df3924-9108-45d6-b529-bb4d38f2d9db	2015-02-05 21:09:52.772761
376	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 115\nvariant_id: 95\nrating: 3\n	1	\N	\N	911cc645-f722-4eef-b44e-c56f63f16a0e	2015-02-05 21:09:52.792895
377	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 116\nvariant_id: 100\nrating: 3\n	1	\N	\N	c9d24320-0986-4e55-bb52-cde51954bce1	2015-02-05 21:09:52.818434
378	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition\n  of HSP90 by 17-AAG is shown to be effective in a patient with metastatic malignant\n  melenoma with a NRAS G13D mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 6\nsource_id: 117\nvariant_id: 99\nrating: 2\n	1	\N	\N	f2426db8-a5af-4bc3-ac5e-24758c0c0340	2015-02-05 21:09:52.847987
379	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 6\nsource_id: 118\nvariant_id: 101\nrating: 2\n	1	\N	\N	357f0e53-4144-4d05-a16c-b7960dbc10e1	2015-02-05 21:09:52.878465
380	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 6\nsource_id: 118\nvariant_id: 102\nrating: 2\n	1	\N	\N	cf55a996-a0c9-42d9-b18a-b69a4e62838d	2015-02-05 21:09:52.900303
398	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 26\nsource_id: 127\nvariant_id: 83\nrating: 3\n	1	\N	\N	592b0971-950c-44c5-84da-2e905f6c8d2c	2015-02-05 21:09:53.341758
381	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"AML with mutated NPM1" is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.'\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 119\nvariant_id: 92\nrating: 5\n	1	\N	\N	18dd929c-6953-48b4-80b9-d41475ce7b83	2015-02-05 21:09:52.921678
382	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increase survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 32\ndisease_id: 5\nsource_id: 120\nvariant_id: 91\nrating: 5\n	1	\N	\N	dacf12b9-b970-434f-8bea-aa18beede864	2015-02-05 21:09:52.945479
383	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 33\ndisease_id: 5\nsource_id: 121\nvariant_id: 91\nrating: 4\n	1	\N	\N	18298c7f-6002-4b1e-bbb8-09611fb144fb	2015-02-05 21:09:52.971077
384	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 34\ndisease_id: 5\nsource_id: 121\nvariant_id: 91\nrating: 4\n	1	\N	\N	361cfa89-d38f-4278-8509-af014cdcb10d	2015-02-05 21:09:52.99528
385	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 122\nvariant_id: 80\nrating: 4\n	1	\N	\N	cdd376c8-d475-4a8c-8335-c51740ca4a08	2015-02-05 21:09:53.024892
386	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 3\nsource_id: 123\nvariant_id: 85\nrating: 4\n	1	\N	\N	2c37911b-8651-4a70-a1c1-a4e9a5fb3c3d	2015-02-05 21:09:53.053963
387	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 36\ndisease_id: 3\nsource_id: 123\nvariant_id: 85\nrating: 4\n	1	\N	\N	39e413ec-e60e-4340-b881-69fdb337cef4	2015-02-05 21:09:53.069931
388	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft models\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 28\nsource_id: 124\nvariant_id: 87\nrating: 4\n	1	\N	\N	f3d538aa-1469-4ace-b675-d22cd0aa937c	2015-02-05 21:09:53.100866
389	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 28\nsource_id: 124\nvariant_id: 87\nrating: 4\n	1	\N	\N	fe6994a9-a35f-4a1c-ae68-e718beb0e7cc	2015-02-05 21:09:53.127147
390	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft models\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 28\nsource_id: 124\nvariant_id: 87\nrating: 4\n	1	\N	\N	22bf4406-6667-4d7e-ae74-54505167297f	2015-02-05 21:09:53.148645
391	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 28\nsource_id: 124\nvariant_id: 87\nrating: 4\n	1	\N	\N	5d685656-e161-42db-b481-76441f836845	2015-02-05 21:09:53.1708
392	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 26\nsource_id: 103\nvariant_id: 81\nrating: 4\n	1	\N	\N	4ea19d7f-8804-479c-8a76-4633cd12e1d3	2015-02-05 21:09:53.188675
393	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 28\nsource_id: 125\nvariant_id: 82\nrating: 4\n	1	\N	\N	78319eb6-38a8-4201-b2a1-683b4835e33e	2015-02-05 21:09:53.215621
394	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 3\nsource_id: 126\nvariant_id: 81\nrating: 3\n	1	\N	\N	47a0a0c4-ed8b-4ce5-adee-8b845bd25d14	2015-02-05 21:09:53.245697
395	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 3\nsource_id: 126\nvariant_id: 81\nrating: 3\n	1	\N	\N	7d988382-ae52-4308-b3dc-588f4c0ab13b	2015-02-05 21:09:53.259399
396	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 26\nsource_id: 127\nvariant_id: 83\nrating: 3\n	1	\N	\N	b7d9aadf-9e76-4776-a1de-b21d5ec9b281	2015-02-05 21:09:53.28439
397	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 83\nrating: 3\n	1	\N	\N	d9c9ec26-cc27-49b0-99c5-c309395417b5	2015-02-05 21:09:53.313942
399	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 29\nsource_id: 128\nvariant_id: 82\nrating: 3\n	1	\N	\N	3bb70f6e-fefa-48fd-84bd-5cbdb818df09	2015-02-05 21:09:53.368102
400	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 29\nsource_id: 128\nvariant_id: 86\nrating: 3\n	1	\N	\N	5dc1ceef-5c07-4468-8d70-2a5c2d3aeb50	2015-02-05 21:09:53.388188
401	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 129\nvariant_id: 85\nrating: 3\n	1	\N	\N	431cfe6a-6ac4-4dd6-a6fc-a92f1cf229b4	2015-02-05 21:09:53.417086
402	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 31\nsource_id: 130\nvariant_id: 82\nrating: 3\n	1	\N	\N	757dd630-e088-4468-8f3d-4da77fa1f3d3	2015-02-05 21:09:53.450668
403	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 32\nsource_id: 130\nvariant_id: 86\nrating: 3\n	1	\N	\N	9f2e1320-0a92-4139-8592-2968adda85f1	2015-02-05 21:09:53.479335
404	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung patients are not associated with smoking history, age\n  and gender\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 131\nvariant_id: 82\nrating: 3\n	1	\N	\N	2c3b0b07-6c82-45a6-9976-8657290cdb6c	2015-02-05 21:09:53.506258
405	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 132\nvariant_id: 82\nrating: 3\n	1	\N	\N	beb3a31b-1e8a-4a21-b133-42c9d5e3ca3e	2015-02-05 21:09:53.533121
406	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 133\nvariant_id: 82\nrating: 3\n	1	\N	\N	e0f5d20a-d314-4d50-935d-6b57c1aefc45	2015-02-05 21:09:53.557097
407	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung patients are not associated with smoking history, age\n  and gender\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 131\nvariant_id: 86\nrating: 3\n	1	\N	\N	29a60060-71e6-4f20-b67c-73d03c25ad4f	2015-02-05 21:09:53.577416
408	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 3\nsource_id: 134\nvariant_id: 85\nrating: 2\n	1	\N	\N	9f686d59-a63c-48e5-8107-c154e20ef8cc	2015-02-05 21:09:53.597782
409	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 129\nvariant_id: 84\nrating: 2\n	1	\N	\N	08f2db91-5d63-48bc-8a0b-e49e35d7ca04	2015-02-05 21:09:53.634891
410	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 6\nsource_id: 135\nvariant_id: 73\nrating: 3\n	1	\N	\N	ddae197d-5670-4581-baaa-2e8c348f8284	2015-02-05 21:09:53.661504
411	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 6\nsource_id: 135\nvariant_id: 74\nrating: 3\n	1	\N	\N	68cd96f7-4ca7-4dd5-a7fd-8e1ce665293d	2015-02-05 21:09:53.680245
412	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 6\nsource_id: 135\nvariant_id: 76\nrating: 3\n	1	\N	\N	72a9d499-e462-46eb-ab3c-2aa08dd551c5	2015-02-05 21:09:53.708535
413	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 136\nvariant_id: 73\nrating: 3\n	1	\N	\N	19f80718-cc36-4133-aa8d-86b2fc3eac8e	2015-02-05 21:09:53.739386
414	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between KIT mutations in exon 11, exon 9\n  or wild type in survival among GIST patients.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 137\nvariant_id: 73\nrating: 3\n	1	\N	\N	ee297e36-6fda-4275-8f5c-8b55c9afa156	2015-02-05 21:09:53.768108
415	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 138\nvariant_id: 73\nrating: 3\n	1	\N	\N	89b73742-f7ca-4277-b976-e411292dcd8d	2015-02-05 21:09:53.797889
416	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 137\nvariant_id: 75\nrating: 3\n	1	\N	\N	c4e3b556-4ca5-4e7d-93fb-4cd48d826b45	2015-02-05 21:09:53.825661
417	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 138\nvariant_id: 76\nrating: 3\n	1	\N	\N	4249795e-fb55-4b14-8744-df536cf36965	2015-02-05 21:09:53.851772
418	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between KIT mutations in exon 11, exon 9\n  or wild type in survival among GIST patients.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 137\nvariant_id: 77\nrating: 3\n	1	\N	\N	072bd78c-29f3-4c61-8854-1b0c10a1b766	2015-02-05 21:09:53.877111
419	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 138\nvariant_id: 73\nrating: 2\n	1	\N	\N	fca68b5d-dee4-48f0-b29c-0e2e3a142175	2015-02-05 21:09:53.904923
420	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 139\nvariant_id: 73\nrating: 2\n	1	\N	\N	2a8b6ce1-931d-4f26-807c-989201193760	2015-02-05 21:09:53.935688
421	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 139\nvariant_id: 77\nrating: 2\n	1	\N	\N	af575faa-af87-446d-b11a-3585eaa30a20	2015-02-05 21:09:53.962082
422	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of peglated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 33\nsource_id: 140\nvariant_id: 71\nrating: 4\n	1	\N	\N	e2e4ee41-68a5-49be-90fe-305aa2322b28	2015-02-05 21:09:53.992394
423	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F not associatted wth  lymphoid leukemia.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 9\nsource_id: 141\nvariant_id: 71\nrating: 4\n	1	\N	\N	0c43ee15-dedb-4eff-ad59-7e8a20efe1c9	2015-02-05 21:09:54.025612
424	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 9\nsource_id: 141\nvariant_id: 71\nrating: 4\n	1	\N	\N	6da53ec0-04b7-41d8-89ba-f81482409398	2015-02-05 21:09:54.049473
425	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 34\nsource_id: 141\nvariant_id: 71\nrating: 4\n	1	\N	\N	9f6abc2c-bce6-4ad7-b17d-7820eaaa5fac	2015-02-05 21:09:54.07828
426	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: rs10974944 SNP in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 35\nsource_id: 142\nvariant_id: 71\nrating: 4\n	1	\N	\N	5a8e2a63-d8da-46ba-ac6e-4e1422db64e7	2015-02-05 21:09:54.111683
427	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 40\ndisease_id: 36\nsource_id: 143\nvariant_id: 71\nrating: 3\n	1	\N	\N	0489d67d-4251-4129-9fd0-18c519f69ea2	2015-02-05 21:09:54.150528
428	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation do not impact\n  overall survival or disease free survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 144\nvariant_id: 69\nrating: 3\n	1	\N	\N	4c7a8a3d-6f3c-4734-8fe6-06a91e40d48e	2015-02-05 21:09:54.178812
429	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 145\nvariant_id: 69\nrating: 3\n	1	\N	\N	04eefc7b-0d93-41d8-86bb-b94ff68d0ffa	2015-02-05 21:09:54.211053
430	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 145\nvariant_id: 70\nrating: 3\n	1	\N	\N	fdab50a7-c369-4b8c-b269-b90b8e4fb3b7	2015-02-05 21:09:54.236779
431	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 25\nsource_id: 146\nvariant_id: 69\nrating: 3\n	1	\N	\N	c0f12215-ed62-4958-9245-6b5260e22888	2015-02-05 21:09:54.26537
432	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 147\nvariant_id: 69\nrating: 2\n	1	\N	\N	df119762-5b2d-435e-a0e4-8111a26c9846	2015-02-05 21:09:54.295402
433	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 147\nvariant_id: 70\nrating: 2\n	1	\N	\N	e0e89ac9-d091-4eb8-9e09-519642ec574d	2015-02-05 21:09:54.320004
434	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 148\nvariant_id: 69\nrating: 2\n	1	\N	\N	a9db38c0-f6d9-431a-8aa7-d4d17ac7a287	2015-02-05 21:09:54.352211
435	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 148\nvariant_id: 70\nrating: 2\n	1	\N	\N	0108471a-17a6-4ffb-be2a-25cf5fffb9ef	2015-02-05 21:09:54.376078
436	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 37\nsource_id: 149\nvariant_id: 64\nrating: 4\n	1	\N	\N	59fab376-045a-48af-bd99-e66eddb255b9	2015-02-05 21:09:54.406231
437	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 150\nvariant_id: 65\nrating: 3\n	1	\N	\N	fabf685e-ddeb-4457-9173-25f473f899f1	2015-02-05 21:09:54.432551
438	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 150\nvariant_id: 67\nrating: 3\n	1	\N	\N	a6adaf64-e48d-4f8f-8c2b-466648c961d7	2015-02-05 21:09:54.458366
439	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 150\nvariant_id: 68\nrating: 3\n	1	\N	\N	521c7eb8-9867-4422-aac9-ebe10a781114	2015-02-05 21:09:54.478746
440	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 38\nsource_id: 151\nvariant_id: 64\nrating: 3\n	1	\N	\N	6003d582-0e62-4c3a-af9e-f154745f9c34	2015-02-05 21:09:54.509249
441	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 39\nsource_id: 152\nvariant_id: 64\nrating: 3\n	1	\N	\N	82116d71-86a7-49fc-a4aa-b860b0295e2b	2015-02-05 21:09:54.532474
442	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 153\nvariant_id: 64\nrating: 2\n	1	\N	\N	a4008f52-aafa-4763-9364-8f9de165e297	2015-02-05 21:09:54.556678
443	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 154\nvariant_id: 64\nrating: 2\n	1	\N	\N	812bdaf1-2078-40ef-90e9-760aea066c9c	2015-02-05 21:09:54.581976
444	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 150\nvariant_id: 65\nrating: 2\n	1	\N	\N	270bb8ee-143f-433c-9965-819a491e46c7	2015-02-05 21:09:54.601566
445	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 40\nsource_id: 155\nvariant_id: 63\nrating: 4\n	1	\N	\N	d53b3eea-c1f7-4e18-8a84-815ac0cf846b	2015-02-05 21:09:54.628753
446	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 40\nsource_id: 155\nvariant_id: 63\nrating: 4\n	1	\N	\N	e2e188c7-9cdc-46c4-9324-878c5c4e1702	2015-02-05 21:09:54.655515
447	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD  , concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 61\nrating: 5\n	1	\N	\N	ff077a55-079a-48fd-94ea-1f2ae55e4399	2015-02-05 21:09:54.676028
448	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat , but not\n  in wild type or D835 mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 156\nvariant_id: 61\nrating: 4\n	1	\N	\N	62d8f703-73f0-4aeb-a943-1f2e929b0fda	2015-02-05 21:09:54.700466
449	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 157\nvariant_id: 62\nrating: 4\n	1	\N	\N	a51d431f-7bc0-43ef-bfd9-9af03e6cfd6c	2015-02-05 21:09:54.723296
450	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat , but not\n  in wild type or D835 mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 156\nvariant_id: 62\nrating: 4\n	1	\N	\N	369077f6-01df-4744-b2ea-13ae256aa234	2015-02-05 21:09:54.74378
451	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 is effective in AML patients with FLT3 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 7\nsource_id: 158\nvariant_id: 61\nrating: 4\n	1	\N	\N	b4ea2f6e-98ca-42c7-a59b-8f530d419a61	2015-02-05 21:09:54.770693
452	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 is effective in AML patients with FLT3 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 7\nsource_id: 158\nvariant_id: 62\nrating: 4\n	1	\N	\N	c1f09286-a5e5-4406-bc64-d70fc0d1ed37	2015-02-05 21:09:54.793135
453	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 44\ndisease_id: 7\nsource_id: 159\nvariant_id: 61\nrating: 4\n	1	\N	\N	f5548d36-637a-48eb-8e9e-3efc0004829b	2015-02-05 21:09:54.822384
454	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 61\nrating: 4\n	1	\N	\N	efabee1d-38ea-48db-b0ed-92fcb37e7ef9	2015-02-05 21:09:54.847443
455	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 61\nrating: 4\n	1	\N	\N	9cdb4a9e-8da9-4ef1-b93f-7b1623f55d37	2015-02-05 21:09:54.874232
456	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 61\nrating: 4\n	1	\N	\N	bb1d3cbf-4ab5-40b3-a771-7ccd8399f096	2015-02-05 21:09:54.900235
457	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 160\nvariant_id: 61\nrating: 4\n	1	\N	\N	bcc54cda-2054-405b-9f21-bc0a6696b8c4	2015-02-05 21:09:54.927916
458	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 61\nrating: 4\n	1	\N	\N	1a8bf9e5-35f4-4bb5-88d5-0b96beb8ad42	2015-02-05 21:09:54.953334
459	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 61\nrating: 4\n	1	\N	\N	b009a55e-aee3-4b12-a047-6d5a07529e7a	2015-02-05 21:09:54.978004
460	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 61\nrating: 4\n	1	\N	\N	ab6a91e6-e090-4427-89a5-566393dc39a2	2015-02-05 21:09:55.002502
461	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation is associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 161\nvariant_id: 62\nrating: 4\n	1	\N	\N	da8543a8-128c-4ca3-a342-5d33670bf174	2015-02-05 21:09:55.034826
462	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 45\ndisease_id: 7\nsource_id: 162\nvariant_id: 61\nrating: 3\n	1	\N	\N	4f274557-9981-4cc7-a30a-fc898c12d523	2015-02-05 21:09:55.060504
463	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 45\ndisease_id: 7\nsource_id: 162\nvariant_id: 62\nrating: 3\n	1	\N	\N	c54e20a6-553c-4483-9655-23869009bdb6	2015-02-05 21:09:55.087027
464	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 7\nsource_id: 57\nvariant_id: 61\nrating: 3\n	1	\N	\N	49bd9537-6525-4526-85fa-17746a405fd6	2015-02-05 21:09:55.110676
465	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 7\nsource_id: 162\nvariant_id: 61\nrating: 3\n	1	\N	\N	ba27778f-5627-433c-969b-97edb1fcadf9	2015-02-05 21:09:55.135948
466	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 7\nsource_id: 162\nvariant_id: 62\nrating: 3\n	1	\N	\N	15ae47d3-1464-44b5-b163-441d21b995e0	2015-02-05 21:09:55.15654
467	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 61\nrating: 3\n	1	\N	\N	b7ea3969-8ef4-4630-8361-83b183b86d05	2015-02-05 21:09:55.179278
468	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 61\nrating: 3\n	1	\N	\N	b9651eae-ee29-45eb-83f3-4e218cf3a27e	2015-02-05 21:09:55.201078
469	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 61\nrating: 3\n	1	\N	\N	45f8d546-73aa-4db2-9c8e-4b6803142653	2015-02-05 21:09:55.223571
470	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 62\nrating: 3\n	1	\N	\N	1fcbfecb-68ec-4562-810d-43b8be4ae82d	2015-02-05 21:09:55.245711
471	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 62\nrating: 3\n	1	\N	\N	fa4f6247-e2d8-431e-a89b-6d92f0164671	2015-02-05 21:09:55.265407
472	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 60\nvariant_id: 62\nrating: 3\n	1	\N	\N	72ec6ccd-1cb7-461d-af4a-c2555f130a6d	2015-02-05 21:09:55.284767
473	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 61\nrating: 2\n	1	\N	\N	1fcbe27a-603f-4f17-bccc-a3ec5ddc4239	2015-02-05 21:09:55.303932
474	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 164\nvariant_id: 61\nrating: 2\n	1	\N	\N	e226350f-829e-40dd-9b41-eb104b0e323d	2015-02-05 21:09:55.327067
475	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 38\nrating: 5\n	1	\N	\N	b319f299-1b1e-4010-9a22-d77a384cbd6b	2015-02-05 21:09:55.351517
476	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 39\nrating: 5\n	1	\N	\N	6a1d8a1d-28ad-43d9-bc0a-a5dfde6d059e	2015-02-05 21:09:55.374153
477	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 40\nrating: 5\n	1	\N	\N	fc076989-1493-4ec1-a66e-a5eb3a486d93	2015-02-05 21:09:55.395146
478	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 41\nrating: 5\n	1	\N	\N	def2b539-a70e-4126-8d1b-19e4da956e53	2015-02-05 21:09:55.417333
479	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 43\nrating: 5\n	1	\N	\N	1e3e584b-c37b-47bb-aaad-4d36d35e3e69	2015-02-05 21:09:55.442106
480	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: This mutation is sensitive to the irreversible kinase inhibitor, neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 45\nrating: 5\n	1	\N	\N	1dca2d26-73bd-4625-8046-f911778ae3ab	2015-02-05 21:09:55.463342
481	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 46\nrating: 5\n	1	\N	\N	59b0ea4d-e7e8-436a-96a8-5d163f0f550e	2015-02-05 21:09:55.484955
482	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 37\nrating: 4\n	1	\N	\N	63cf368a-96dd-4db8-a3f4-54438d208dea	2015-02-05 21:09:55.505583
483	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with tyrosine kinase inhibitors, the presence of L858R\n  mutation is prognostic for better progression free survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 166\nvariant_id: 35\nrating: 4\n	1	\N	\N	0a9741f8-0b35-4a35-a26c-af2d8d4e778b	2015-02-05 21:09:55.545112
484	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation has statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 167\nvariant_id: 36\nrating: 4\n	1	\N	\N	00df18bf-376c-4844-b87d-16a87f140e70	2015-02-05 21:09:55.572332
485	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutation, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 168\nvariant_id: 36\nrating: 4\n	1	\N	\N	ef4408d9-6f10-4e05-860b-21f2d1689663	2015-02-05 21:09:55.597094
486	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 3\nsource_id: 169\nvariant_id: 36\nrating: 3\n	1	\N	\N	d82a1adf-a262-413c-8854-ec0c081f36bf	2015-02-05 21:09:55.626396
487	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 47\ndisease_id: 3\nsource_id: 169\nvariant_id: 36\nrating: 3\n	1	\N	\N	a15fbde9-1ddc-4739-b69f-bb04bdefcfe5	2015-02-05 21:09:55.643892
488	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 3\nsource_id: 170\nvariant_id: 35\nrating: 3\n	1	\N	\N	a5252fd1-386a-4c26-addf-8d013f27bbf8	2015-02-05 21:09:55.672219
489	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 3\nsource_id: 170\nvariant_id: 35\nrating: 3\n	1	\N	\N	29700f21-af6d-487b-9fff-5198b9a9d8b4	2015-02-05 21:09:55.685913
490	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Geftinib added to pemetrexed and carboplatin showed greater effectiveness than\n  premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 3\nsource_id: 171\nvariant_id: 35\nrating: 3\n	1	\N	\N	357851a3-809e-444a-b051-58eb1fdf7412	2015-02-05 21:09:55.714263
491	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 3\nsource_id: 172\nvariant_id: 36\nrating: 3\n	1	\N	\N	a98396e3-95c0-4c56-99be-bba4296f832d	2015-02-05 21:09:55.742394
492	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 173\nvariant_id: 35\nrating: 3\n	1	\N	\N	4f6e8615-d7bb-4b87-b0ae-c43cc143ea4a	2015-02-05 21:09:55.771151
493	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when L858R mutation is also present.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 48\ndisease_id: 3\nsource_id: 174\nvariant_id: 36\nrating: 1\n	1	\N	\N	e4cfe816-d0ed-4111-9c3f-cfad56d2c755	2015-02-05 21:09:55.803068
494	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 33\nrating: 5\n	1	\N	\N	992900c0-43aa-42d5-a1a9-6de047c06c57	2015-02-05 21:09:55.830213
495	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 33\nrating: 5\n	1	\N	\N	ce729dd1-6fc6-495d-bddf-4636e28dff88	2015-02-05 21:09:55.853707
496	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD  , concurrent DNMT3A mutations (including R882)\n  were associated with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 33\nrating: 5\n	1	\N	\N	207bb39d-c2fa-4163-bd70-801bf828922d	2015-02-05 21:09:55.873583
497	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 175\nvariant_id: 33\nrating: 5\n	1	\N	\N	b5c49ece-6ac8-4b2c-8b3d-9fae4213a1b0	2015-02-05 21:09:55.90169
498	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 176\nvariant_id: 33\nrating: 4\n	1	\N	\N	09773d70-078c-4e3f-ab06-716804768de0	2015-02-05 21:09:55.928829
499	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 7\nsource_id: 176\nvariant_id: 33\nrating: 4\n	1	\N	\N	43ce87b7-a930-452e-983b-a5a9b39e444f	2015-02-05 21:09:55.946699
500	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 177\nvariant_id: 33\nrating: 4\n	1	\N	\N	fa18e453-a915-427c-b77e-c955e08908dd	2015-02-05 21:09:55.97176
501	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 4\n	1	\N	\N	f495feb7-438b-4bf0-b45a-22a8b403e1dd	2015-02-05 21:09:55.993847
502	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 176\nvariant_id: 33\nrating: 4\n	1	\N	\N	bb33772c-52af-4849-9708-d9072d64039a	2015-02-05 21:09:56.025528
503	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 177\nvariant_id: 33\nrating: 4\n	1	\N	\N	a933b105-855b-4c55-9c01-c06604fa8776	2015-02-05 21:09:56.053194
504	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 4\n	1	\N	\N	e9ada7f3-706d-4d65-b7d0-793694a43a76	2015-02-05 21:09:56.078614
505	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 33\nrating: 4\n	1	\N	\N	5a1661c5-9ea9-48aa-b01a-86e7901276fe	2015-02-05 21:09:56.106505
506	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 177\nvariant_id: 33\nrating: 4\n	1	\N	\N	95d5c722-86f6-417e-9e0f-5a3d82f19581	2015-02-05 21:09:56.135168
507	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 177\nvariant_id: 33\nrating: 4\n	1	\N	\N	12f24a3e-3e65-4526-9653-de45d0db2f8b	2015-02-05 21:09:56.160331
508	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 3\n	1	\N	\N	4ae5beeb-a855-4e44-819b-e0bb7e74383d	2015-02-05 21:09:56.183449
509	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	90c4c78e-d7a3-4813-ac72-4f89ce7e89b3	2015-02-05 21:09:56.206429
510	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	7933e4aa-8dbb-40d0-9d8b-eb4d0c0326a8	2015-02-05 21:09:56.231635
511	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	da5eee30-7d0c-4dcd-a3e2-40a988bac689	2015-02-05 21:09:56.255448
512	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 176\nvariant_id: 33\nrating: 3\n	1	\N	\N	e0ff652b-bab7-4b85-9b28-717d6f8b2ba6	2015-02-05 21:09:56.283215
513	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 175\nvariant_id: 33\nrating: 3\n	1	\N	\N	06e7eb01-fcf3-4210-8ced-75dc3fe82c26	2015-02-05 21:09:56.310323
514	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	eed953a6-0abe-4e9f-8cb0-b1c93f79b56d	2015-02-05 21:09:56.337032
515	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	197d1fdc-2e28-4e63-99c8-0e11ef5a78ea	2015-02-05 21:09:56.363074
516	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	e9830b62-3a73-4a8d-a1d3-90789f78fd33	2015-02-05 21:09:56.386421
517	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable cytogenetically normal AMLs\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	bf752fe0-5a09-4317-abc0-7f39c902533c	2015-02-05 21:09:56.41062
518	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 3\n	1	\N	\N	aea7f1ff-5070-4779-9591-8257185efb12	2015-02-05 21:09:56.447496
519	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 3\n	1	\N	\N	79f97450-fc75-4869-ac99-59bf12fe9477	2015-02-05 21:09:56.467029
520	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 3\n	1	\N	\N	970824dc-7468-4762-8f31-6951f48ed1fe	2015-02-05 21:09:56.4879
521	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	4aca2f96-98f5-49ce-ace4-10f39eea9701	2015-02-05 21:09:56.506037
522	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 57\nvariant_id: 30\nrating: 4\n	1	\N	\N	06c98778-8ee9-401b-ac0a-97ca28d5d78c	2015-02-05 21:09:56.527936
523	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 69\nvariant_id: 30\nrating: 4\n	1	\N	\N	85a9bb4f-c3f9-4034-a288-9f55cc9d1e40	2015-02-05 21:09:56.550507
524	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 57\nvariant_id: 30\nrating: 4\n	1	\N	\N	786493f1-e508-4c98-b928-f976a0f11e87	2015-02-05 21:09:56.57162
525	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 179\nvariant_id: 29\nrating: 3\n	1	\N	\N	87e91a47-72d2-4d3e-bd1e-42d51253c266	2015-02-05 21:09:56.598756
526	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 179\nvariant_id: 29\nrating: 3\n	1	\N	\N	af494721-1d3b-410a-82e1-80591c65e52e	2015-02-05 21:09:56.620232
527	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 179\nvariant_id: 29\nrating: 2\n	1	\N	\N	891a805e-d71c-427d-8c0b-ecfd41a5c767	2015-02-05 21:09:56.644734
528	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 180\nvariant_id: 28\nrating: 3\n	1	\N	\N	c42c1463-f706-4eeb-868a-37c88a3c6d60	2015-02-05 21:09:56.671589
529	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 180\nvariant_id: 28\nrating: 3\n	1	\N	\N	22c9284c-8d75-4eb7-9724-ec26da6ec77b	2015-02-05 21:09:56.69508
530	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and\n  CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor\n  regressions.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 41\nsource_id: 181\nvariant_id: 27\nrating: 4\n	1	\N	\N	77f832a6-1ffb-4a33-b633-8718129251fe	2015-02-05 21:09:56.723475
531	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 51\ndisease_id: 41\nsource_id: 182\nvariant_id: 27\nrating: 3\n	1	\N	\N	b4654d77-9521-46ff-990d-c981bb5ec726	2015-02-05 21:09:56.751306
532	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and\n  CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor\n  regressions.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 41\nsource_id: 181\nvariant_id: 26\nrating: 4\n	1	\N	\N	c4e96084-da4d-41ad-81eb-4ed5e9298199	2015-02-05 21:09:56.773159
533	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 51\ndisease_id: 41\nsource_id: 182\nvariant_id: 26\nrating: 3\n	1	\N	\N	2553ccbd-172c-45a1-820f-4e1263f34a82	2015-02-05 21:09:56.794981
534	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 183\nvariant_id: 25\nrating: 4\n	1	\N	\N	d95fab27-7c65-48b3-a653-e36cecd83274	2015-02-05 21:09:56.823475
535	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 42\nsource_id: 184\nvariant_id: 25\nrating: 4\n	1	\N	\N	4c3bf362-f793-4760-bf02-4cff5d113a18	2015-02-05 21:09:56.8496
536	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 42\nsource_id: 184\nvariant_id: 25\nrating: 4\n	1	\N	\N	630ac169-786d-47ff-a024-5f359ee097f9	2015-02-05 21:09:56.868357
537	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 184\nvariant_id: 25\nrating: 4\n	1	\N	\N	a12414dd-0f47-4e97-9307-d2a353356e6e	2015-02-05 21:09:56.893528
538	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 52\ndisease_id: 43\nsource_id: 185\nvariant_id: 24\nrating: 3\n	1	\N	\N	2a3f234f-018f-4115-92da-fd6e302e7d9c	2015-02-05 21:09:56.925466
539	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 186\nvariant_id: 21\nrating: 4\n	1	\N	\N	995ef9c8-80bb-431c-aa6e-2a2fe8a7de44	2015-02-05 21:09:56.955863
540	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 186\nvariant_id: 21\nrating: 4\n	1	\N	\N	b8bfe47a-adec-4f31-bbfa-0566ebedecf0	2015-02-05 21:09:56.979773
541	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 186\nvariant_id: 21\nrating: 4\n	1	\N	\N	8ea33297-4487-405f-934a-5a53b0e83806	2015-02-05 21:09:57.006534
542	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 186\nvariant_id: 21\nrating: 4\n	1	\N	\N	14571aa9-b40d-49f9-a232-7cb1d957bea8	2015-02-05 21:09:57.034756
543	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 187\nvariant_id: 22\nrating: 3\n	1	\N	\N	56dc7d7f-d51d-4324-b958-5a1e48d1f905	2015-02-05 21:09:57.060785
544	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of CCND2-negative cells with 5-Aza-2'-deoxycytidine restored cyclin\n  D2 expression.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 187\nvariant_id: 22\nrating: 2\n	1	\N	\N	06666974-2358-4052-8d3c-dc6a15ad7f20	2015-02-05 21:09:57.085341
545	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 188\nvariant_id: 11\nrating: 5\n	1	\N	\N	6a82ed64-4a39-47a8-8d38-492f8ae77141	2015-02-05 21:09:57.111373
546	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 53\ndisease_id: 6\nsource_id: 188\nvariant_id: 11\nrating: 5\n	1	\N	\N	48b747d6-4b81-4d0c-b1f5-1e6c396106b9	2015-02-05 21:09:57.128475
547	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as poor prognosis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 26\nsource_id: 189\nvariant_id: 11\nrating: 5\n	1	\N	\N	003fff42-d842-44b0-807c-8eee5161e67d	2015-02-05 21:09:57.155369
548	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 190\nvariant_id: 11\nrating: 5\n	1	\N	\N	aece9015-db18-4d0b-b9cc-276c8366e1ee	2015-02-05 21:09:57.178875
549	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 191\nvariant_id: 11\nrating: 5\n	1	\N	\N	9106958f-5c99-4031-9a64-a35b55ad5c54	2015-02-05 21:09:57.207292
550	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 192\nvariant_id: 11\nrating: 5\n	1	\N	\N	860c3056-6b87-4632-ba52-57212c831230	2015-02-05 21:09:57.233715
551	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK\n  inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification\n  of the BRAF V600E gene.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 26\nsource_id: 193\nvariant_id: 13\nrating: 4\n	1	\N	\N	6bc1112c-a3a7-4d42-9db5-3e39f0947241	2015-02-05 21:09:57.341041
552	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in nonchronic sun damage\n  skin and with superficial spreading melanoma\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 194\nvariant_id: 16\nrating: 4\n	1	\N	\N	6a2d6e4c-dc18-455f-9eb4-3b20c1195e6a	2015-02-05 21:09:57.367001
553	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 27\ndisease_id: 26\nsource_id: 195\nvariant_id: 16\nrating: 3\n	1	\N	\N	0eb4ff32-15d9-453c-adfe-e4693191b26f	2015-02-05 21:09:57.389585
554	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 54\ndisease_id: 28\nsource_id: 196\nvariant_id: 11\nrating: 3\n	1	\N	\N	8e5a17d3-3af6-4cc9-b23d-a6fb604a4b81	2015-02-05 21:09:57.414958
555	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 55\ndisease_id: 28\nsource_id: 196\nvariant_id: 11\nrating: 3\n	1	\N	\N	4e473026-c9d4-4faf-9a5c-f1375be1665e	2015-02-05 21:09:57.43073
556	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 28\nsource_id: 197\nvariant_id: 11\nrating: 3\n	1	\N	\N	f5d160ed-8963-437a-9349-45be97841e21	2015-02-05 21:09:57.454619
557	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 27\ndisease_id: 28\nsource_id: 197\nvariant_id: 11\nrating: 3\n	1	\N	\N	533cda2a-c4f3-4ccb-a768-49f88b7fdbb8	2015-02-05 21:09:57.465636
558	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 28\nsource_id: 197\nvariant_id: 11\nrating: 3\n	1	\N	\N	4dd63e6b-e3b9-410b-bf56-7a977ea3f398	2015-02-05 21:09:57.478532
559	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 198\nvariant_id: 16\nrating: 3\n	1	\N	\N	2d87202c-1490-4536-b07b-fe3ece2cfdef	2015-02-05 21:09:57.505036
560	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 53\ndisease_id: 6\nsource_id: 198\nvariant_id: 16\nrating: 3\n	1	\N	\N	3698f153-004d-4201-9b4b-9a6a8059bada	2015-02-05 21:09:57.516691
561	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates and shorter progression free and overall\n  survival following cetuximab plus chemotherapy than those with wildtype BRAF\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 11\nrating: 3\n	1	\N	\N	0153f655-c182-49f9-8d5a-87ba095acdbd	2015-02-05 21:09:57.54131
562	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter Disease-Free and Overall Survival in a\n  Spanish cohort of Melanoma patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 199\nvariant_id: 11\nrating: 3\n	1	\N	\N	219d6079-d573-40ab-af07-77b1f4d63cd4	2015-02-05 21:09:57.564993
563	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of PTC\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 200\nvariant_id: 11\nrating: 3\n	1	\N	\N	f6caca87-4fd0-488f-b564-22b9252de659	2015-02-05 21:09:57.589663
564	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 200\nvariant_id: 11\nrating: 3\n	1	\N	\N	747db58c-abdc-41ea-94a0-769e2280ed9c	2015-02-05 21:09:57.610995
565	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 201\nvariant_id: 11\nrating: 3\n	1	\N	\N	88eefeb0-ffa7-47ce-bafc-8040d6df3828	2015-02-05 21:09:57.634714
566	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 200\nvariant_id: 11\nrating: 3\n	1	\N	\N	1aa99040-81b5-47c8-9b43-cabc0d6883d3	2015-02-05 21:09:57.653032
567	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E resistance is triggered by ERK dependent or independent pathways.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 6\nsource_id: 202\nvariant_id: 11\nrating: 3\n	1	\N	\N	b88a8dfb-f598-4b02-87f0-9727840e572f	2015-02-05 21:09:57.673781
568	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g.\n  dabrafenib) appears to be beneficial, and better than vice versa.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 56\ndisease_id: 6\nsource_id: 203\nvariant_id: 11\nrating: 2\n	1	\N	\N	9d6cdb79-7776-43d6-a264-dfe361d1fc6a	2015-02-05 21:09:57.699077
569	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g.\n  dabrafenib) appears to be beneficial, and better than vice versa.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 203\nvariant_id: 11\nrating: 2\n	1	\N	\N	a8e189ce-2288-44d7-8f4f-00eceb9352d6	2015-02-05 21:09:57.709703
570	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 6\nsource_id: 204\nvariant_id: 11\nrating: 2\n	1	\N	\N	df512a9c-61f7-4de2-a191-41c5cf72c089	2015-02-05 21:09:57.737475
571	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 32\ndisease_id: 6\nsource_id: 204\nvariant_id: 11\nrating: 2\n	1	\N	\N	e85aa215-039e-4deb-9d49-67009bc3731a	2015-02-05 21:09:57.750775
572	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 58\ndisease_id: 26\nsource_id: 205\nvariant_id: 11\nrating: 2\n	1	\N	\N	18d98f2c-ae53-4faf-8546-4cfdad68a964	2015-02-05 21:09:57.775656
573	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 26\nsource_id: 205\nvariant_id: 11\nrating: 2\n	1	\N	\N	d993ece3-7248-459e-a518-af983477c93d	2015-02-05 21:09:57.785613
574	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 59\ndisease_id: 26\nsource_id: 205\nvariant_id: 11\nrating: 2\n	1	\N	\N	6ac0f9ce-6c7f-427e-8c3d-14535a2577ce	2015-02-05 21:09:57.797347
575	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 55\ndisease_id: 28\nsource_id: 206\nvariant_id: 11\nrating: 2\n	1	\N	\N	6ec2641f-1c21-450a-bd08-800a09767c3d	2015-02-05 21:09:57.821484
576	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 28\nsource_id: 206\nvariant_id: 11\nrating: 2\n	1	\N	\N	f7a26e28-0677-4c8b-a5b5-6654029bfa6e	2015-02-05 21:09:57.836158
577	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 3\nsource_id: 134\nvariant_id: 11\nrating: 2\n	1	\N	\N	210309ac-9470-40bf-b3ca-1e48c323fa30	2015-02-05 21:09:57.859741
578	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 26\nsource_id: 205\nvariant_id: 11\nrating: 2\n	1	\N	\N	891ab8f6-fd11-44f4-b612-8989089c973c	2015-02-05 21:09:57.878488
579	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 207\nvariant_id: 19\nrating: 4\n	1	\N	\N	62c5b782-552d-4576-8720-a9a04758be86	2015-02-05 21:09:57.902557
580	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 207\nvariant_id: 19\nrating: 4\n	1	\N	\N	3b5a3461-daa6-4646-a244-d04b966a98f2	2015-02-05 21:09:57.928464
581	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 45\nsource_id: 208\nvariant_id: 19\nrating: 4\n	1	\N	\N	4e094c65-05f5-4be9-a33e-91209377dd3a	2015-02-05 21:09:57.950024
582	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 46\nsource_id: 209\nvariant_id: 19\nrating: 4\n	1	\N	\N	f0a7574c-617b-4b92-8e2a-b4ff34d4a660	2015-02-05 21:09:57.973892
583	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 210\nvariant_id: 17\nrating: 3\n	1	\N	\N	a7cb6e4e-9a88-4cf3-9f82-7f6359fb98a3	2015-02-05 21:09:57.992263
584	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 210\nvariant_id: 18\nrating: 3\n	1	\N	\N	312a8cb3-29a9-4e25-92b9-97e197c8229b	2015-02-05 21:09:58.019434
585	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 210\nvariant_id: 18\nrating: 3\n	1	\N	\N	430ed112-fcd6-4a88-8953-725564116813	2015-02-05 21:09:58.042107
586	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells in vitro\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 53\ndisease_id: 3\nsource_id: 211\nvariant_id: 9\nrating: 3\n	1	\N	\N	eb76a37f-593c-4285-9ba3-6e740489f252	2015-02-05 21:09:58.066986
587	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells in vitro\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 3\nsource_id: 211\nvariant_id: 9\nrating: 3\n	1	\N	\N	c2acd5ac-0eea-4fc7-8d81-5ff71c70a879	2015-02-05 21:09:58.084446
588	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 3\nsource_id: 211\nvariant_id: 9\nrating: 2\n	1	\N	\N	fac86033-0db3-4936-86f5-85f2eae92e97	2015-02-05 21:09:58.107628
589	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 61\ndisease_id: 47\nsource_id: 212\nvariant_id: 7\nrating: 4\n	1	\N	\N	2c75a594-d65e-4a40-8d7c-45c1c0de68d6	2015-02-05 21:09:58.147017
590	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 47\nsource_id: 213\nvariant_id: 7\nrating: 3\n	1	\N	\N	14d41ace-adf7-4b53-a0a7-2bba7e2f9eea	2015-02-05 21:09:58.174709
591	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 3\nsource_id: 213\nvariant_id: 6\nrating: 3\n	1	\N	\N	14d2f765-b612-4f9b-9bab-3feb5e07e16c	2015-02-05 21:09:58.196668
592	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK with the R1275Q mutation\n  at a higher IC50 than those expressing F1174L and the SMS-KCNR cell line harboring\n  the R1275Q mutation was resistant to TAE684\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 61\ndisease_id: 47\nsource_id: 212\nvariant_id: 8\nrating: 3\n	1	\N	\N	d0444c93-b53d-4ce1-b08a-e6811f46256c	2015-02-05 21:09:58.21964
593	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 48\nsource_id: 84\nvariant_id: 7\nrating: 3\n	1	\N	\N	90a5328f-3901-42ea-b529-e39fcc4680b5	2015-02-05 21:09:58.248842
594	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 47\nsource_id: 214\nvariant_id: 7\nrating: 3\n	1	\N	\N	1f6ae848-90d1-437f-b263-6962c7378890	2015-02-05 21:09:58.277499
595	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 47\nsource_id: 215\nvariant_id: 7\nrating: 3\n	1	\N	\N	fa6e6684-8fbd-4f20-91ff-95afc64e71f7	2015-02-05 21:09:58.30961
596	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 63\ndisease_id: 49\nsource_id: 216\nvariant_id: 3\nrating: 3\n	1	\N	\N	2e6449df-5487-42fa-8fdc-ef6f4635a621	2015-02-05 21:09:58.343422
597	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide\n  results in high rates of complete remission in patients.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 64\ndisease_id: 7\nsource_id: 217\nvariant_id: 1\nrating: 5\n	1	\N	\N	ee9c1f49-a61b-466e-b34f-a8a3e3176188	2015-02-05 21:09:58.372
598	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The clinical use of imatinib in patients with BCR-ABL fusion has drastic sensitivity\n  to the drug.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 10\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 5\n	1	\N	\N	a31ecd61-b650-4111-8e64-c20cecc0a984	2015-02-05 21:09:58.395972
599	417	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis,\n  response rate, overall survival, and patient outcome in CML patients compared to\n  previous therapeutic regimens."\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 10\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 5\n	1	\N	\N	87f7542f-7ac1-483d-8373-f48d0d757fd3	2015-02-05 21:09:58.422136
600	418	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has shown signifcantly\n  greater response in patients resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 4\n	1	\N	\N	68fc2680-7c38-47b9-a5bf-8673232d92a7	2015-02-05 21:09:58.451249
601	419	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has shown signifcantly\n  greater response in patients resistant to imatinib.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 15\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 4\n	1	\N	\N	b82a288b-7ea3-4dab-b842-29d219ea9f93	2015-02-05 21:09:58.464709
602	420	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 4\n	1	\N	\N	600b4adf-7d89-4bea-a4c7-f6b1fe0a4ca3	2015-02-05 21:09:58.487932
603	421	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL fusions have been found to contribute to imatinib resistance in AML\n  cell lines.\nclinical_significance: ''\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 7\nsource_id: 217\nvariant_id: 1\nrating: 3\n	1	\N	\N	6b6ae117-f076-4702-aa40-40a813413f13	2015-02-05 21:09:58.508217
604	422	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: ''\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 9\nsource_id: 218\nvariant_id: 1\nrating: 2\n	1	\N	\N	d036ca33-b355-484c-8f60-ec8b948b7789	2015-02-05 21:09:58.534321
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 604, true);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
1	4007	bladder carcinoma	2015-02-05 21:09:47.999901	2015-02-05 21:12:06.848492
2	6477	invasive bladder transitional cell carcinoma	2015-02-05 21:09:48.094141	2015-02-05 21:12:07.082243
3	3908	non-small cell lung carcinoma	2015-02-05 21:09:48.125086	2015-02-05 21:12:07.302447
4	1612	breast cancer	2015-02-05 21:09:48.161121	2015-02-05 21:12:07.732214
5	3068	glioblastoma multiforme	2015-02-05 21:09:48.201054	2015-02-05 21:12:07.936277
6	1909	melanoma	2015-02-05 21:09:48.244347	2015-02-05 21:12:08.137631
7	9119	acute myeloid leukemia	2015-02-05 21:09:48.380793	2015-02-05 21:12:08.290506
8	3973	thyroid medullary carcinoma	2015-02-05 21:09:49.55616	2015-02-05 21:12:08.549448
9	8552	chronic myeloid leukemia	2015-02-05 21:09:49.633828	2015-02-05 21:12:08.734551
10	4947	cholangiocarcinoma	2015-02-05 21:09:49.686755	2015-02-05 21:12:08.917016
11	1612	breast cancer	2015-02-05 21:09:49.720594	2015-02-05 21:12:09.269844
12	10283	prostate cancer	2015-02-05 21:09:49.826279	2015-02-05 21:12:09.4727
13	3907	lung squamous cell carcinoma	2015-02-05 21:09:49.858831	2015-02-05 21:12:09.65388
14	1781	thyroid cancer	2015-02-05 21:09:49.890266	2015-02-05 21:12:09.837153
15	4007	bladder carcinoma	2015-02-05 21:09:49.918397	2015-02-05 21:12:10.050248
16	3068	glioblastoma multiforme	2015-02-05 21:09:49.991733	2015-02-05 21:12:10.256092
17	8618	oral cavity cancer	2015-02-05 21:09:50.032735	2015-02-05 21:12:10.505001
18	4947	cholangiocarcinoma	2015-02-05 21:09:50.075536	2015-02-05 21:12:10.685325
19	3907	lung squamous cell carcinoma	2015-02-05 21:09:50.114398	2015-02-05 21:12:10.885973
20	9119	acute myeloid leukemia	2015-02-05 21:09:50.152105	2015-02-05 21:12:11.033639
21	9253	gastrointestinal stromal tumor	2015-02-05 21:09:50.298395	2015-02-05 21:12:11.175001
22	5015	hepatocellular fibrolamellar carcinoma	2015-02-05 21:09:50.751289	2015-02-05 21:12:11.34878
23	9119	acute myeloid leukemia	2015-02-05 21:09:50.923302	2015-02-05 21:12:11.499675
24	9952	acute lymphocytic leukemia	2015-02-05 21:09:50.963827	2015-02-05 21:12:11.671449
25	4972	myelodysplastic myeloproliferative cancer	2015-02-05 21:09:51.036039	2015-02-05 21:12:11.837203
26	9256	colorectal cancer	2015-02-05 21:09:51.988768	2015-02-05 21:12:12.030294
27	9253	gastrointestinal stromal tumor	2015-02-05 21:09:52.383007	2015-02-05 21:12:12.217847
28	9256	colorectal cancer	2015-02-05 21:09:53.095157	2015-02-05 21:12:12.405546
29	1240	leukemia	2015-02-05 21:09:53.363255	2015-02-05 21:12:12.670022
30	1324	lung cancer	2015-02-05 21:09:53.411739	2015-02-05 21:12:12.916482
31	9538	multiple myeloma	2015-02-05 21:09:53.44475	2015-02-05 21:12:13.105845
32	9119	acute myeloid leukemia	2015-02-05 21:09:53.473937	2015-02-05 21:12:13.274245
33	8997	polycythemia vera	2015-02-05 21:09:53.986283	2015-02-05 21:12:13.420403
34	10747	lymphoid leukemia	2015-02-05 21:09:54.070513	2015-02-05 21:12:13.793755
35	4960	bone marrow cancer	2015-02-05 21:09:54.100922	2015-02-05 21:12:13.986285
36	1240	leukemia	2015-02-05 21:09:54.142735	2015-02-05 21:12:14.294672
37	3069	astrocytoma	2015-02-05 21:09:54.397607	2015-02-05 21:12:14.557975
38	3069	astrocytoma	2015-02-05 21:09:54.500322	2015-02-05 21:12:14.825191
39	3068	glioblastoma multiforme	2015-02-05 21:09:54.527962	2015-02-05 21:12:15.108763
40	3909	bronchogenic lung adenocarcinoma	2015-02-05 21:09:54.624058	2015-02-05 21:12:15.275833
41	0060075	estrogen-receptor positive breast cancer	2015-02-05 21:09:56.715713	2015-02-05 21:12:15.451105
42	5635	gastric adenosquamous carcinoma	2015-02-05 21:09:56.844716	2015-02-05 21:12:15.632964
43	715	T-cell leukemia	2015-02-05 21:09:56.920435	2015-02-05 21:12:15.787191
44	10534	stomach cancer	2015-02-05 21:09:56.947628	2015-02-05 21:12:16.071238
45	8618	oral cavity cancer	2015-02-05 21:09:57.945273	2015-02-05 21:12:16.38764
46	0050746	mantle cell lymphoma	2015-02-05 21:09:57.969494	2015-02-05 21:12:16.539103
47	769	neuroblastoma	2015-02-05 21:09:58.14258	2015-02-05 21:12:16.738903
48	1115	sarcoma	2015-02-05 21:09:58.243194	2015-02-05 21:12:16.992011
49	1612	breast cancer	2015-02-05 21:09:58.336143	2015-02-05 21:12:17.538895
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 49, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Everolimus	\N	2015-02-05 21:09:47.993785	2015-02-05 21:09:47.993785
2	Rapamycin	\N	2015-02-05 21:09:48.121432	2015-02-05 21:09:48.121432
3	N/A	\N	2015-02-05 21:09:48.157737	2015-02-05 21:09:48.157737
4	Dabrafenib	\N	2015-02-05 21:09:48.24035	2015-02-05 21:09:48.24035
5	NSC348884	\N	2015-02-05 21:09:48.782977	2015-02-05 21:09:48.782977
6	ATRA	\N	2015-02-05 21:09:48.797662	2015-02-05 21:09:48.797662
7	Daunorubicin	\N	2015-02-05 21:09:49.272462	2015-02-05 21:09:49.272462
8	Valproic acid	\N	2015-02-05 21:09:49.30051	2015-02-05 21:09:49.30051
9	Motesanib	\N	2015-02-05 21:09:49.552331	2015-02-05 21:09:49.552331
10	Imatinib	\N	2015-02-05 21:09:49.629261	2015-02-05 21:09:49.629261
11	Ponatinib	\N	2015-02-05 21:09:49.670113	2015-02-05 21:09:49.670113
12	Pazopanib	\N	2015-02-05 21:09:49.716973	2015-02-05 21:09:49.716973
13	Nilotinib	\N	2015-02-05 21:09:50.197948	2015-02-05 21:09:50.197948
14	Sorafenib	\N	2015-02-05 21:09:50.209532	2015-02-05 21:09:50.209532
15	Dasatinib	\N	2015-02-05 21:09:50.241607	2015-02-05 21:09:50.241607
16	SU11248	\N	2015-02-05 21:09:50.295324	2015-02-05 21:09:50.295324
17	Tamoxifen	\N	2015-02-05 21:09:50.412085	2015-02-05 21:09:50.412085
18	Fulvestrant	\N	2015-02-05 21:09:50.427183	2015-02-05 21:09:50.427183
19	Crizotinib	\N	2015-02-05 21:09:50.804755	2015-02-05 21:09:50.804755
20	GW-2580	\N	2015-02-05 21:09:50.979015	2015-02-05 21:09:50.979015
21	Doxorubicin	\N	2015-02-05 21:09:51.553861	2015-02-05 21:09:51.553861
22	AZD1480	\N	2015-02-05 21:09:51.854945	2015-02-05 21:09:51.854945
23	AZD6244	\N	2015-02-05 21:09:51.877965	2015-02-05 21:09:51.877965
24	CH5132799	\N	2015-02-05 21:09:51.924025	2015-02-05 21:09:51.924025
25	Regorafenib	\N	2015-02-05 21:09:51.984885	2015-02-05 21:09:51.984885
26	Cetuximab	\N	2015-02-05 21:09:52.055384	2015-02-05 21:09:52.055384
27	Panitumumab	\N	2015-02-05 21:09:52.07025	2015-02-05 21:09:52.07025
28	Crenolanib	\N	2015-02-05 21:09:52.379373	2015-02-05 21:09:52.379373
29	Vemurafenib	\N	2015-02-05 21:09:52.567948	2015-02-05 21:09:52.567948
30	17-AAG	\N	2015-02-05 21:09:52.83877	2015-02-05 21:09:52.83877
31	Temozolomide¬†	\N	2015-02-05 21:09:52.867553	2015-02-05 21:09:52.867553
32	Temozolomide	\N	2015-02-05 21:09:52.938981	2015-02-05 21:09:52.938981
33	O(6)-benzylguanine	\N	2015-02-05 21:09:52.963923	2015-02-05 21:09:52.963923
34	Carmustine	\N	2015-02-05 21:09:52.986313	2015-02-05 21:09:52.986313
35	NVP-BEZ235	\N	2015-02-05 21:09:53.04613	2015-02-05 21:09:53.04613
36	ARRY-142886	\N	2015-02-05 21:09:53.064264	2015-02-05 21:09:53.064264
37	Gefitinib	\N	2015-02-05 21:09:53.237954	2015-02-05 21:09:53.237954
38	Erlotinib	\N	2015-02-05 21:09:53.254105	2015-02-05 21:09:53.254105
39	Pegylated IFN-Œ±‚Äì2a	\N	2015-02-05 21:09:53.982492	2015-02-05 21:09:53.982492
40	TG101348	\N	2015-02-05 21:09:54.137921	2015-02-05 21:09:54.137921
41	Bortezomib	\N	2015-02-05 21:09:54.620657	2015-02-05 21:09:54.620657
42	Fasudil	\N	2015-02-05 21:09:54.636511	2015-02-05 21:09:54.636511
43	CEP701	\N	2015-02-05 21:09:54.76324	2015-02-05 21:09:54.76324
44	AG1296	\N	2015-02-05 21:09:54.814849	2015-02-05 21:09:54.814849
45	SU5614	\N	2015-02-05 21:09:55.050756	2015-02-05 21:09:55.050756
46	Neratinib	\N	2015-02-05 21:09:55.344141	2015-02-05 21:09:55.344141
47	Premetrexed	\N	2015-02-05 21:09:55.634586	2015-02-05 21:09:55.634586
48	Stauroporine	\N	2015-02-05 21:09:55.793316	2015-02-05 21:09:55.793316
49	Idarubicin	\N	2015-02-05 21:09:55.919706	2015-02-05 21:09:55.919706
50	BYL719	\N	2015-02-05 21:09:56.711605	2015-02-05 21:09:56.711605
51	Palbociclib	\N	2015-02-05 21:09:56.742407	2015-02-05 21:09:56.742407
52	Palbociclib (PD-0332991)	\N	2015-02-05 21:09:56.915356	2015-02-05 21:09:56.915356
53	Trametinib	\N	2015-02-05 21:09:57.123606	2015-02-05 21:09:57.123606
54	PD0325901	\N	2015-02-05 21:09:57.408338	2015-02-05 21:09:57.408338
55	PLX4720	\N	2015-02-05 21:09:57.426205	2015-02-05 21:09:57.426205
56	Ipilimumab	\N	2015-02-05 21:09:57.692894	2015-02-05 21:09:57.692894
57	Dacarbazine	\N	2015-02-05 21:09:57.729987	2015-02-05 21:09:57.729987
58	Capecitabine	\N	2015-02-05 21:09:57.769569	2015-02-05 21:09:57.769569
59	Bevacizumab	\N	2015-02-05 21:09:57.793019	2015-02-05 21:09:57.793019
60	nutlin-3	\N	2015-02-05 21:09:57.830255	2015-02-05 21:09:57.830255
61	TAE684	\N	2015-02-05 21:09:58.139376	2015-02-05 21:09:58.139376
62	CH5424802	\N	2015-02-05 21:09:58.167233	2015-02-05 21:09:58.167233
63	MK-2206	\N	2015-02-05 21:09:58.330948	2015-02-05 21:09:58.330948
64	Arsenic trioxide	\N	2015-02-05 21:09:58.363181	2015-02-05 21:09:58.363181
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 64, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	B	\N	2015-02-05 21:09:48.005296	2015-02-05 21:09:48.005296
2	C	\N	2015-02-05 21:09:48.127001	2015-02-05 21:09:48.127001
3	D	\N	2015-02-05 21:09:49.224589	2015-02-05 21:09:49.224589
4	A	\N	2015-02-05 21:09:50.492237	2015-02-05 21:09:50.492237
\.


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Predictive	2015-02-05 21:09:48.002414	2015-02-05 21:09:48.002414
2	Prognostic	2015-02-05 21:09:48.20474	2015-02-05 21:09:48.20474
3	Diagnostic	2015-02-05 21:09:48.422039	2015-02-05 21:09:48.422039
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-02-05 21:09:34.295665	2015-02-05 21:09:34.295665	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, has been known as an oncogene for quite some time. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-02-05 21:09:34.525528	2015-02-05 21:09:34.525528	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-02-05 21:09:34.732506	2015-02-05 21:09:34.732506	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia. Treatment of cells harboring this fusion has shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, mutations in this gene have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-02-05 21:09:35.032621	2015-02-05 21:09:35.032621	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been shown to be correlated with poor prognosis in many cancer types, there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-02-05 21:09:35.242871	2015-02-05 21:09:35.242871	\N
6	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-02-05 21:09:35.465957	2015-02-05 21:09:35.465957	\N
7	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-02-05 21:09:35.670578	2015-02-05 21:09:35.670578	\N
8	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-02-05 21:09:35.882359	2015-02-05 21:09:35.882359	\N
9	898	CCNE1	Cyclin E, while currenly not as widely implicated as it's counterparts, cyclin D, has been shown to implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	cyclin E1	2015-02-05 21:09:36.059496	2015-02-05 21:09:36.059496	\N
10	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-02-05 21:09:36.240865	2015-02-05 21:09:36.240865	\N
11	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-02-05 21:09:36.43256	2015-02-05 21:09:36.43256	\N
12	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the promoter region for the gene. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-02-05 21:09:36.633876	2015-02-05 21:09:36.633876	\N
28	3845	KRAS	N/A	Kirsten rat sarcoma viral oncogene homolog	2015-02-05 21:09:40.088022	2015-02-05 21:09:40.088022	\N
29	5604	MAP2K1	N/A	mitogen-activated protein kinase kinase 1	2015-02-05 21:09:40.299821	2015-02-05 21:09:40.299821	\N
30	4209	MEF2D	N/A	myocyte enhancer factor 2D	2015-02-05 21:09:40.532424	2015-02-05 21:09:40.532424	\N
13	1050	CEBPA	'AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-02-05 21:09:36.849152	2015-02-05 21:09:36.849152	\N
14	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. A study has found this fusion in every FL-HCC sample sequenced, and in vitro assays have shown that the fusion retains kinase function.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-02-05 21:09:37.042902	2015-02-05 21:09:37.042902	\N
15	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker. A study has found this fusion in every sample sequenced, and in vitro assays have shown that the fusion retains kinase function.	protein kinase, cAMP-dependent, catalytic, alpha	2015-02-05 21:09:37.305776	2015-02-05 21:09:37.305776	\N
16	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-02-05 21:09:37.490527	2015-02-05 21:09:37.490527	\N
17	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have show efficacy in EGFR amplfied tumors, most notably gefitinibb and erlotinib. mutations in EGFR have been shown to confer resistance to these drugs however, and the variant T790M has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has been a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-02-05 21:09:37.71585	2015-02-05 21:09:37.71585	\N
18	2064	ERBB2	ERBB2, commonly referred to as HER2, is seen to be amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown senitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-02-05 21:09:37.918669	2015-02-05 21:09:37.918669	\N
19	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, and has shown significance in endometrial, ovarian and other cancer types as well. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-02-05 21:09:38.141041	2015-02-05 21:09:38.141041	\N
20	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. The active proteins have the potential to activate the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-02-05 21:09:38.335576	2015-02-05 21:09:38.335576	\N
21	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. The active proteins have the potential to activate the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-02-05 21:09:38.585791	2015-02-05 21:09:38.585791	\N
22	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-02-05 21:09:38.815865	2015-02-05 21:09:38.815865	\N
23	2624	GATA2	N/A	GATA binding protein 2	2015-02-05 21:09:39.014747	2015-02-05 21:09:39.014747	\N
24	3417	IDH1	N/A	isocitrate dehydrogenase 1 (NADP+), soluble	2015-02-05 21:09:39.20129	2015-02-05 21:09:39.20129	\N
25	3418	IDH2	N/A	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-02-05 21:09:39.40144	2015-02-05 21:09:39.40144	\N
26	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders.	Janus kinase 2	2015-02-05 21:09:39.615396	2015-02-05 21:09:39.615396	\N
27	3815	KIT	N/A	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-02-05 21:09:39.791958	2015-02-05 21:09:39.791958	\N
31	1436	CSF1R	N/A	colony stimulating factor 1 receptor	2015-02-05 21:09:40.719783	2015-02-05 21:09:40.719783	\N
32	4255	MGMT	N/A	O-6-methylguanine-DNA methyltransferase	2015-02-05 21:09:40.917779	2015-02-05 21:09:40.917779	\N
33	4869	NPM1	'AML with mutated NPM1' is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-02-05 21:09:41.109276	2015-02-05 21:09:41.109276	\N
34	4893	NRAS	N/A	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-02-05 21:09:41.303674	2015-02-05 21:09:41.303674	\N
35	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating  , the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-02-05 21:09:41.532254	2015-02-05 21:09:41.532254	\N
36	5156	PDGFRA	N/A	platelet-derived growth factor receptor, alpha polypeptide	2015-02-05 21:09:41.707792	2015-02-05 21:09:41.707792	\N
37	5371	PML	N/A	promyelocytic leukemia	2015-02-05 21:09:41.914544	2015-02-05 21:09:41.914544	\N
38	5914	RARa	N/A	retinoic acid receptor, alpha	2015-02-05 21:09:42.156581	2015-02-05 21:09:42.156581	\N
39	5728	PTEN	N/A	phosphatase and tensin homolog	2015-02-05 21:09:42.341975	2015-02-05 21:09:42.341975	\N
40	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-02-05 21:09:42.533064	2015-02-05 21:09:42.533064	\N
41	861	RUNX1	N/A	runt-related transcription factor 1	2015-02-05 21:09:42.737432	2015-02-05 21:09:42.737432	\N
42	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-02-05 21:09:42.92424	2015-02-05 21:09:42.92424	\N
43	7157	TP53	N/A	tumor protein p53	2015-02-05 21:09:43.165199	2015-02-05 21:09:43.165199	\N
44	7248	TSC1	N/A	tuberous sclerosis 1	2015-02-05 21:09:43.407936	2015-02-05 21:09:43.407936	\N
45	7249	TSC2	N/A	tuberous sclerosis 2	2015-02-05 21:09:44.272317	2015-02-05 21:09:44.272317	\N
46	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-02-05 21:09:45.15373	2015-02-05 21:09:45.15373	\N
47	7490	WT1	N/A	Wilms tumor 1	2015-02-05 21:09:46.030902	2015-02-05 21:09:46.030902	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-02-05 21:09:34.302753	2015-02-05 21:10:00.359864
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-02-05 21:09:34.307659	2015-02-05 21:10:00.920606
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-02-05 21:09:34.527594	2015-02-05 21:10:01.474614
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-02-05 21:09:34.531953	2015-02-05 21:10:01.866958
5	15676015	\N	Lee et al., 2005, APMIS	2015-02-05 21:09:34.735145	2015-02-05 21:10:02.402068
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-02-05 21:09:34.740232	2015-02-05 21:10:03.027888
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-02-05 21:09:35.035438	2015-02-05 21:10:03.637436
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-02-05 21:09:35.040781	2015-02-05 21:10:04.057133
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-02-05 21:09:35.244736	2015-02-05 21:10:04.472831
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-02-05 21:09:35.24919	2015-02-05 21:10:04.888701
11	12432268	\N	Diehl, Cancer Biol. Ther.	2015-02-05 21:09:35.468394	2015-02-05 21:10:05.420211
12	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-02-05 21:09:35.472828	2015-02-05 21:10:05.776164
13	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-02-05 21:09:36.062676	2015-02-05 21:10:06.247932
14	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-02-05 21:09:36.242515	2015-02-05 21:10:06.668877
15	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-02-05 21:09:36.245712	2015-02-05 21:10:07.190457
16	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-02-05 21:09:36.637725	2015-02-05 21:10:07.625141
17	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-05 21:09:36.641751	2015-02-05 21:10:08.024981
18	19357394	\N	Vardiman et al., 2009, Blood	2015-02-05 21:09:36.852092	2015-02-05 21:10:08.439578
19	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-02-05 21:09:36.856352	2015-02-05 21:10:08.858308
20	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-02-05 21:09:36.862094	2015-02-05 21:10:09.378713
21	24578576	\N	Honeyman et al., 2014, Science	2015-02-05 21:09:37.044738	2015-02-05 21:10:09.797175
22	24167195	\N	Kim et al., 2013, Blood	2015-02-05 21:09:37.492765	2015-02-05 21:10:10.218856
23	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-02-05 21:09:37.49716	2015-02-05 21:10:10.639248
24	23953842	\N	Yewale et al., 2013, Biomaterials	2015-02-05 21:09:37.718391	2015-02-05 21:10:11.01808
25	18712184	\N	Charpidou et al., In Vivo	2015-02-05 21:09:37.721705	2015-02-05 21:10:11.384694
26	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-02-05 21:09:37.920509	2015-02-05 21:10:12.306403
27	10878580	\N	Yu et al., 2000, Bioessays	2015-02-05 21:09:37.924161	2015-02-05 21:10:12.724284
28	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-02-05 21:09:38.143053	2015-02-05 21:10:13.172846
29	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-02-05 21:09:38.147546	2015-02-05 21:10:13.669978
30	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-02-05 21:09:38.338124	2015-02-05 21:10:14.078414
31	23558953	\N	Wu et al., 2013, Cancer Discov	2015-02-05 21:09:38.341521	2015-02-05 21:10:14.524488
32	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-02-05 21:09:38.820675	2015-02-05 21:10:15.027512
33	23226219	\N	Chou et al., 2012, PLoS ONE	2015-02-05 21:09:41.111223	2015-02-05 21:10:15.474454
118	21576590		Soon et al., 2011, Arch Dermatol	2015-02-05 21:09:52.869808	2015-02-05 21:10:58.478627
34	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-05 21:09:42.536049	2015-02-05 21:10:16.281407
35	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-02-05 21:09:42.543673	2015-02-05 21:10:16.645069
36	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-05 21:09:42.927588	2015-02-05 21:10:17.26811
37	24136165	\N	Cazzola et al., 2013, Blood	2015-02-05 21:09:42.931535	2015-02-05 21:10:17.690069
38	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-02-05 21:09:45.157269	2015-02-05 21:10:18.186688
39	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-02-05 21:09:45.162201	2015-02-05 21:10:18.848494
40	23071027			2015-02-05 21:09:47.996711	2015-02-05 21:10:19.307093
41	22923433		Iyer et al., 2012, Science	2015-02-05 21:09:48.092556	2015-02-05 21:10:19.822277
42	19966866		Liang et al., 2010, Oncogene	2015-02-05 21:09:48.123349	2015-02-05 21:10:20.204727
43	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-02-05 21:09:48.159556	2015-02-05 21:10:20.587513
44	22479427		Carico et al., 2012, PLoS ONE	2015-02-05 21:09:48.19771	2015-02-05 21:10:21.049348
45	23463675		Ponti et al., 2013, J. Clin. Pathol.	2015-02-05 21:09:48.242809	2015-02-05 21:10:21.621546
46	23031422		Ponti et al., 2012, J Hematol Oncol	2015-02-05 21:09:48.346964	2015-02-05 21:10:22.055532
47	16076867		Schnittger et al., 2005, Blood	2015-02-05 21:09:48.379339	2015-02-05 21:10:22.442211
48	17957027		Gale et al., 2008, Blood	2015-02-05 21:09:48.499879	2015-02-05 21:10:22.961878
49	22490330		Ribeiro et al., 2012, Blood	2015-02-05 21:09:48.531439	2015-02-05 21:10:23.377498
50	16051734		D√∂hner et al., 2005, Blood	2015-02-05 21:09:48.560953	2015-02-05 21:10:23.805746
51	20026798		Becker et al., 2010, J. Clin. Oncol.	2015-02-05 21:09:48.593732	2015-02-05 21:10:24.330974
52	24855211		Linch et al., 2014, Blood	2015-02-05 21:09:48.617969	2015-02-05 21:10:24.69566
53	16455956		Thiede et al., 2006, Blood	2015-02-05 21:09:48.645786	2015-02-05 21:10:25.178977
54	21537333		Schnittger et al., 2011, Leukemia	2015-02-05 21:09:48.678553	2015-02-05 21:10:25.680954
55	21067377		Ley et al., 2010, N. Engl. J. Med.	2015-02-05 21:09:48.731505	2015-02-05 21:10:26.218601
56	21719597		Balusu et al., 2011, Blood	2015-02-05 21:09:48.784463	2015-02-05 21:10:26.945415
57	19965647		Burnett et al., 2010, Blood	2015-02-05 21:09:48.830015	2015-02-05 21:10:27.461474
58	19059939		Schlenk et al., 2009, Haematologica	2015-02-05 21:09:48.968151	2015-02-05 21:10:27.999935
59	20805365		Schnittger et al., 2010, Blood	2015-02-05 21:09:49.025154	2015-02-05 21:10:28.622867
60	17965322		Bacher et al., 2008, Blood	2015-02-05 21:09:49.082426	2015-02-05 21:10:29.04104
61	19047294		B√ºchner et al., 2009, J. Clin. Oncol.	2015-02-05 21:09:49.112255	2015-02-05 21:10:29.566869
62	22430270		Gaidzik et al., 2012, J. Clin. Oncol.	2015-02-05 21:09:49.143605	2015-02-05 21:10:29.954713
63	24859829		Tian et al., 2014, Int. J. Hematol.	2015-02-05 21:09:49.169509	2015-02-05 21:10:30.480655
64	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-02-05 21:09:49.222867	2015-02-05 21:10:30.914419
65	22417203		Patel et al., 2012, N. Engl. J. Med.	2015-02-05 21:09:49.27409	2015-02-05 21:10:31.431615
66	24797300		Tassara et al., 2014, Blood	2015-02-05 21:09:49.302589	2015-02-05 21:10:32.024604
67	19587375		Schnittger et al., 2009, Blood	2015-02-05 21:09:49.357849	2015-02-05 21:10:32.499002
68	24927407		Ehninger et al., 2014, Blood Cancer J	2015-02-05 21:09:49.414992	2015-02-05 21:10:32.875845
69	18450602		Schlenk et al., 2008, N. Engl. J. Med.	2015-02-05 21:09:49.472408	2015-02-05 21:10:33.321686
70	24576830		Nissan et al., 2014, Cancer Res.	2015-02-05 21:09:49.499518	2015-02-05 21:10:33.843447
71	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-02-05 21:09:49.5544	2015-02-05 21:10:34.6636
72	20537386		An et al., 2010, Leuk. Res.	2015-02-05 21:09:49.63209	2015-02-05 21:10:35.094688
73	24550739		Borad et al., 2014, PLoS Genet.	2015-02-05 21:09:49.672389	2015-02-05 21:10:35.537549
74	23558953		Wu et al., 2013, Cancer Discov	2015-02-05 21:09:49.71883	2015-02-05 21:10:36.146402
75	16384925		Cairoli et al., 2006, Blood	2015-02-05 21:09:50.150673	2015-02-05 21:10:36.646072
76	19671763		Woodman et al., 2009, Mol. Cancer Ther.	2015-02-05 21:09:50.184264	2015-02-05 21:10:37.179227
77	17372901		Antonescu et al., 2007, Int. J. Cancer	2015-02-05 21:09:50.243356	2015-02-05 21:10:37.747666
78	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-02-05 21:09:50.296904	2015-02-05 21:10:38.191044
79	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-02-05 21:09:50.354806	2015-02-05 21:10:38.879349
80	18421059		Hodi et al., 2008, J. Clin. Oncol.	2015-02-05 21:09:50.385158	2015-02-05 21:10:39.264479
81	24185510		Robinson et al., 2013, Nat. Genet.	2015-02-05 21:09:50.413747	2015-02-05 21:10:39.702989
82	24185512		Toy et al., 2013, Nat. Genet.	2015-02-05 21:09:50.600884	2015-02-05 21:10:40.315595
83	24578576		Honeyman et al., 2014, Science	2015-02-05 21:09:50.749594	2015-02-05 21:10:40.84126
84	21030459		Sasaki et al., 2010, Cancer Res.	2015-02-05 21:09:50.806331	2015-02-05 21:10:41.266332
85	20979473		Choi et al., 2010, N. Engl. J. Med.	2015-02-05 21:09:50.836016	2015-02-05 21:10:41.796432
86	8674046		Yoshida et al., 1996, Cancer Res.	2015-02-05 21:09:50.920633	2015-02-05 21:10:42.5993
87	24186003		Lilljebj√∂rn et al., 2014, Leukemia	2015-02-05 21:09:50.961986	2015-02-05 21:10:43.145231
88	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-05 21:09:51.034494	2015-02-05 21:10:44.19379
89	19221039		Gaidzik et al., 2009, Blood	2015-02-05 21:09:51.074552	2015-02-05 21:10:44.595837
90	18591546		Virappane et al., 2008, J. Clin. Oncol.	2015-02-05 21:09:51.114411	2015-02-05 21:10:45.12724
91	18559874		Paschka et al., 2008, J. Clin. Oncol.	2015-02-05 21:09:51.201622	2015-02-05 21:10:45.667917
92	19536888		Renneville et al., 2009, Cancer	2015-02-05 21:09:51.359944	2015-02-05 21:10:46.147976
93	23029227		Qian et al., 2012, PLoS ONE	2015-02-05 21:09:51.402275	2015-02-05 21:10:46.514812
94	23861105		Wu et al., 2013, Am. J. Hematol.	2015-02-05 21:09:51.44099	2015-02-05 21:10:46.991243
95	22698404		Jackson et al., 2012, Cancer Cell	2015-02-05 21:09:51.555266	2015-02-05 21:10:47.38172
96	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-02-05 21:09:51.572442	2015-02-05 21:10:47.825694
97	9569050		Berns et al., 1998, Br. J. Cancer	2015-02-05 21:09:51.596209	2015-02-05 21:10:48.197567
98	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-05 21:09:51.813862	2015-02-05 21:10:48.665228
99	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-02-05 21:09:51.83233	2015-02-05 21:10:49.038455
100	23056499		Couto et al., 2012, PLoS ONE	2015-02-05 21:09:51.856573	2015-02-05 21:10:49.509857
101	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-05 21:09:51.880348	2015-02-05 21:10:50.140379
102	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-02-05 21:09:51.925591	2015-02-05 21:10:51.176715
103	24559322		Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-02-05 21:09:51.986216	2015-02-05 21:10:51.590834
104	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-02-05 21:09:52.057044	2015-02-05 21:10:51.99079
105	20619739		De Roock et al., 2010, Lancet Oncol.	2015-02-05 21:09:52.158786	2015-02-05 21:10:52.644235
106	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-02-05 21:09:52.200025	2015-02-05 21:10:53.048486
107	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-02-05 21:09:52.334344	2015-02-05 21:10:53.47669
108	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-02-05 21:09:52.3813	2015-02-05 21:10:53.882037
109	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-02-05 21:09:52.516339	2015-02-05 21:10:54.514594
110	15146165		Lasota et al., 2004, Lab. Invest.	2015-02-05 21:09:52.541727	2015-02-05 21:10:54.845262
111	23569304		Trunzer et al., 2013, J. Clin. Oncol.	2015-02-05 21:09:52.570203	2015-02-05 21:10:55.346655
112	15951308		Bowen et al., 2005, Blood	2015-02-05 21:09:52.662393	2015-02-05 21:10:55.709449
113	16434492		Bacher et al., 2006, Blood	2015-02-05 21:09:52.685273	2015-02-05 21:10:56.077236
114	24666267		Therkildsen et al., 2014, Acta Oncol	2015-02-05 21:09:52.706074	2015-02-05 21:10:56.590499
115	22180178		Jakob et al., 2012, Cancer	2015-02-05 21:09:52.768068	2015-02-05 21:10:57.013739
116	23861977		Tschandl et al., 2013, PLoS ONE	2015-02-05 21:09:52.812204	2015-02-05 21:10:57.531106
117	18375819		Banerji et al., 2008, Mol. Cancer Ther.	2015-02-05 21:09:52.840879	2015-02-05 21:10:57.946862
119	19357394		Vardiman et al., 2009, Blood	2015-02-05 21:09:52.914789	2015-02-05 21:10:59.414184
120	15758010		Hegi et al., 2005, N. Engl. J. Med.	2015-02-05 21:09:52.940726	2015-02-05 21:10:59.8326
121	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-02-05 21:09:52.965841	2015-02-05 21:11:00.158114
122	16618717		Li√®vre et al., 2006, Cancer Res.	2015-02-05 21:09:53.019638	2015-02-05 21:11:00.661836
123	19029981		Engelman et al., 2008, Nat. Med.	2015-02-05 21:09:53.04752	2015-02-05 21:11:01.09655
124	20978259		De Roock et al., 2010, JAMA	2015-02-05 21:09:53.092333	2015-02-05 21:11:01.500935
125	19934290		Ogino et al., 2009, Clin. Cancer Res.	2015-02-05 21:09:53.210082	2015-02-05 21:11:02.031848
126	15696205		Pao et al., 2005, PLoS Med.	2015-02-05 21:09:53.240167	2015-02-05 21:11:02.648669
127	18202412		Li√®vre et al., 2008, J. Clin. Oncol.	2015-02-05 21:09:53.278367	2015-02-05 21:11:03.001206
128	24571676		Andrade et al., 2014, BMC Cancer	2015-02-05 21:09:53.361555	2015-02-05 21:11:03.429287
129	23014527		Dogan et al., 2012, Clin. Cancer Res.	2015-02-05 21:09:53.408412	2015-02-05 21:11:04.012237
130	18528420		Chng et al., 2008, Leukemia	2015-02-05 21:09:53.442767	2015-02-05 21:11:04.615527
131	18794081		Riely et al., 2008, Clin. Cancer Res.	2015-02-05 21:09:53.500988	2015-02-05 21:11:05.282205
132	11208838		Schiller et al., 2001, J. Clin. Oncol.	2015-02-05 21:09:53.526129	2015-02-05 21:11:06.000206
133	15597105		Mascaux et al., 2005, Br. J. Cancer	2015-02-05 21:09:53.549084	2015-02-05 21:11:06.628135
134	23524406		Rudin et al., 2013, J Thorac Oncol	2015-02-05 21:09:53.592082	2015-02-05 21:11:07.055746
135	23775962		Hodi et al., 2013, J. Clin. Oncol.	2015-02-05 21:09:53.656316	2015-02-05 21:11:07.461839
136	12000708		Corless et al., 2002, Am. J. Pathol.	2015-02-05 21:09:53.731881	2015-02-05 21:11:07.889414
137	16551858		Wardelmann et al., 2006, Clin. Cancer Res.	2015-02-05 21:09:53.761061	2015-02-05 21:11:08.617246
138	10485475		Taniguchi et al., 1999, Cancer Res.	2015-02-05 21:09:53.791	2015-02-05 21:11:09.026921
139	15217946		Boldrini et al., 2004, Clin. Cancer Res.	2015-02-05 21:09:53.929514	2015-02-05 21:11:09.456362
140	16709929		Kiladjian et al., 2006, Blood	2015-02-05 21:09:53.98458	2015-02-05 21:11:09.980201
141	16081687		Levine et al., 2005, Blood	2015-02-05 21:09:54.01915	2015-02-05 21:11:10.40005
142	19287384		Kilpivaara et al., 2009, Nat. Genet.	2015-02-05 21:09:54.098555	2015-02-05 21:11:10.815335
143	18394554		Wernig et al., 2008, Cancer Cell	2015-02-05 21:09:54.140618	2015-02-05 21:11:11.217501
144	20421455		Thol et al., 2010, Blood	2015-02-05 21:09:54.17222	2015-02-05 21:11:11.691614
145	21596855		Green et al., 2011, Blood	2015-02-05 21:09:54.203806	2015-02-05 21:11:12.063284
146	22033490		Patnaik et al., 2012, Leukemia	2015-02-05 21:09:54.260039	2015-02-05 21:11:12.47639
147	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-02-05 21:09:54.287457	2015-02-05 21:11:12.912807
148	21997850		Lin et al., 2012, Ann. Hematol.	2015-02-05 21:09:54.342811	2015-02-05 21:11:13.311205
149	20494930		Thol et al., 2010, Haematologica	2015-02-05 21:09:54.395683	2015-02-05 21:11:13.726682
150	20538800		Abbas et al., 2010, Blood	2015-02-05 21:09:54.426579	2015-02-05 21:11:14.145082
151	19933982		Dubbink et al., 2009, Neurology	2015-02-05 21:09:54.498324	2015-02-05 21:11:14.565612
152	20127344		Bleeker et al., 2010, Acta Neuropathol.	2015-02-05 21:09:54.526437	2015-02-05 21:11:14.963135
153	20376086		Ho et al., 2010, Leukemia	2015-02-05 21:09:54.548605	2015-02-05 21:11:15.417241
154	20368538		Wagner et al., 2010, J. Clin. Oncol.	2015-02-05 21:09:54.574853	2015-02-05 21:11:15.83226
155	22624710		Barbacid, 2012, Cancer Cell	2015-02-05 21:09:54.622332	2015-02-05 21:11:16.229126
156	18230792		Zhang et al., 2008, J. Natl. Cancer Inst.	2015-02-05 21:09:54.694764	2015-02-05 21:11:16.607744
157	22368270		Man et al., 2012, Blood	2015-02-05 21:09:54.716312	2015-02-05 21:11:17.069194
158	14726387		Smith et al., 2004, Blood	2015-02-05 21:09:54.764723	2015-02-05 21:11:17.977185
159	12357354		Tse et al., 2002, Leukemia	2015-02-05 21:09:54.816788	2015-02-05 21:11:18.884448
160	11585760		Whitman et al., 2001, Cancer Res.	2015-02-05 21:09:54.922649	2015-02-05 21:11:19.509051
161	17940205		Whitman et al., 2008, Blood	2015-02-05 21:09:55.027744	2015-02-05 21:11:20.267091
162	15626738		Bagrintseva et al., 2005, Blood	2015-02-05 21:09:55.053149	2015-02-05 21:11:20.729824
163	11290608		Yamamoto et al., 2001, Blood	2015-02-05 21:09:55.174664	2015-02-05 21:11:21.097521
164	24801015		Port et al., 2014, Ann. Hematol.	2015-02-05 21:09:55.32249	2015-02-05 21:11:21.56468
165	23220880		Bose et al., 2013, Cancer Discov	2015-02-05 21:09:55.345672	2015-02-05 21:11:22.005581
166	24457318		Fukihara et al., 2014, Oncology	2015-02-05 21:09:55.539944	2015-02-05 21:11:22.399867
167	24729716		Li et al., 2014, Onco Targets Ther	2015-02-05 21:09:55.564843	2015-02-05 21:11:22.817272
168	24623981		Ding et al., 2014, Onco Targets Ther	2015-02-05 21:09:55.592772	2015-02-05 21:11:23.345287
169	24636847		Li et al., 2014, Lung Cancer	2015-02-05 21:09:55.620056	2015-02-05 21:11:23.91269
170	24736073		Lim et al., 2014, J Thorac Oncol	2015-02-05 21:09:55.665943	2015-02-05 21:11:24.528537
171	24585406		Li et al., 2014, Ann. Surg. Oncol.	2015-02-05 21:09:55.70627	2015-02-05 21:11:25.548767
172	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-02-05 21:09:55.734327	2015-02-05 21:11:26.234648
173	24662454		Douillard et al., 2014, J Thorac Oncol	2015-02-05 21:09:55.765137	2015-02-05 21:11:27.659399
174	24658966		Ai et al., 2014, Amino Acids	2015-02-05 21:09:55.796016	2015-02-05 21:11:28.853972
175	24512939		El Ghannam et al., Blood Cells Mol. Dis.	2015-02-05 21:09:55.89487	2015-02-05 21:11:29.698823
176	22081665		LaRochelle et al., 2011, Oncotarget	2015-02-05 21:09:55.922224	2015-02-05 21:11:30.668725
177	22291079		Marcucci et al., 2012, J. Clin. Oncol.	2015-02-05 21:09:55.966181	2015-02-05 21:11:31.73034
178	23632886		Gaidzik et al., 2013, Blood	2015-02-05 21:09:56.201138	2015-02-05 21:11:33.017485
179	14726504		Fr√∂hling et al., 2004, J. Clin. Oncol.	2015-02-05 21:09:56.593421	2015-02-05 21:11:33.693088
180	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-05 21:09:56.664333	2015-02-05 21:11:34.094567
181	25002028		Vora et al., 2014, Cancer Cell	2015-02-05 21:09:56.713885	2015-02-05 21:11:34.682664
182	23898052		Logan et al., 2013, Anticancer Res.	2015-02-05 21:09:56.745555	2015-02-05 21:11:36.478558
183	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-02-05 21:09:56.815527	2015-02-05 21:11:37.34506
184	10224221		Donnellan et al., 1999, FASEB J.	2015-02-05 21:09:56.841836	2015-02-05 21:11:37.893995
185	23079656		Sawai et al., 2012, Cancer Cell	2015-02-05 21:09:56.917458	2015-02-05 21:11:38.556213
186	10547574		Takano et al., 1999, J. Pathol.	2015-02-05 21:09:56.944867	2015-02-05 21:11:39.421314
187	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-02-05 21:09:57.056015	2015-02-05 21:11:40.185384
188	24583796		Menzies et al., 2014, Clin. Cancer Res.	2015-02-05 21:09:57.106615	2015-02-05 21:11:41.053754
189	24594804		Chen et al., 2014, PLoS ONE	2015-02-05 21:09:57.148679	2015-02-05 21:11:41.939346
190	24588959		Zhang et al., 2014, Diagn Pathol	2015-02-05 21:09:57.174298	2015-02-05 21:11:43.207633
191	24570209		Crescenzi et al., 2014, Horm. Metab. Res.	2015-02-05 21:09:57.197569	2015-02-05 21:11:44.18143
192	24354346		Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-02-05 21:09:57.228617	2015-02-05 21:11:44.771354
193	21098728		Corcoran et al., 2010, Sci Signal	2015-02-05 21:09:57.333909	2015-02-05 21:11:46.690837
194	21166657		Lee et al., 2011, Br. J. Dermatol.	2015-02-05 21:09:57.360842	2015-02-05 21:11:51.212369
195	23325582		Peeters et al., 2013, Clin. Cancer Res.	2015-02-05 21:09:57.383102	2015-02-05 21:11:51.72538
196	23845441		Rad et al., 2013, Cancer Cell	2015-02-05 21:09:57.410149	2015-02-05 21:11:52.408692
197	19001320		Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-02-05 21:09:57.448584	2015-02-05 21:11:52.791546
198	23020132		Flaherty et al., 2012, N. Engl. J. Med.	2015-02-05 21:09:57.497328	2015-02-05 21:11:53.332091
199	24388723		Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-02-05 21:09:57.560087	2015-02-05 21:11:53.737964
200	21594703		Howell et al., 2011, Ann. Surg. Oncol.	2015-02-05 21:09:57.5827	2015-02-05 21:11:54.157748
201	24396464		He et al., 2014, Oncol Lett	2015-02-05 21:09:57.629309	2015-02-05 21:11:54.68376
202	21505228		Corcoran et al., 2011, Oncotarget	2015-02-05 21:09:57.668924	2015-02-05 21:11:57.276696
203	24577748		Ackerman et al., 2014, Cancer	2015-02-05 21:09:57.69441	2015-02-05 21:11:57.724355
204	24586605		Meckbach et al., 2014, PLoS ONE	2015-02-05 21:09:57.731576	2015-02-05 21:11:58.455622
205	22180495		Yang et al., 2012, Cancer Res.	2015-02-05 21:09:57.771019	2015-02-05 21:12:01.167825
206	23812671		Ji et al., 2013, Clin. Cancer Res.	2015-02-05 21:09:57.817105	2015-02-05 21:12:01.567192
207	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-02-05 21:09:57.897006	2015-02-05 21:12:01.985786
208	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-02-05 21:09:57.943145	2015-02-05 21:12:02.423273
209	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-02-05 21:09:57.96769	2015-02-05 21:12:02.819094
210	17070615		Gautschi et al., 2007, Lung Cancer	2015-02-05 21:09:57.987786	2015-02-05 21:12:03.174095
211	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-02-05 21:09:58.060821	2015-02-05 21:12:03.550685
212	18923525		George et al., 2008, Nature	2015-02-05 21:09:58.141127	2015-02-05 21:12:04.186197
213	21575866		Sakamoto et al., 2011, Cancer Cell	2015-02-05 21:09:58.16952	2015-02-05 21:12:04.586105
214	22072639		Bresler et al., 2011, Sci Transl Med	2015-02-05 21:09:58.269998	2015-02-05 21:12:05.014652
215	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-02-05 21:09:58.298172	2015-02-05 21:12:05.409572
216	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-02-05 21:09:58.334196	2015-02-05 21:12:05.766813
217	12476305		Nimmanapalli et al., 2002, Oncogene	2015-02-05 21:09:58.364987	2015-02-05 21:12:06.113718
218	25212276		Schmidt et al., 2014, Leukemia	2015-02-05 21:09:58.526627	2015-02-05 21:12:06.574611
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-02-05 21:09:46.091574	2015-02-05 21:09:46.091574
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-05 21:09:46.107094	2015-02-05 21:09:46.107094
3	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-02-05 21:09:46.118125	2015-02-05 21:09:46.118125
4	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-02-05 21:09:46.128943	2015-02-05 21:09:46.128943
5	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-05 21:09:46.143651	2015-02-05 21:09:46.143651
6	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-05 21:09:46.158198	2015-02-05 21:09:46.158198
7	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-02-05 21:09:46.174848	2015-02-05 21:09:46.174848
8	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-02-05 21:09:46.186428	2015-02-05 21:09:46.186428
9	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-02-05 21:09:46.197364	2015-02-05 21:09:46.197364
10	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-05 21:09:46.209274	2015-02-05 21:09:46.209274
11	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-02-05 21:09:46.222816	2015-02-05 21:09:46.222816
12	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-02-05 21:09:46.234986	2015-02-05 21:09:46.234986
13	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-02-05 21:09:46.24682	2015-02-05 21:09:46.24682
14	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-05 21:09:46.259858	2015-02-05 21:09:46.259858
15	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-05 21:09:46.271461	2015-02-05 21:09:46.271461
16	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-02-05 21:09:46.282406	2015-02-05 21:09:46.282406
17	6	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-02-05 21:09:46.296481	2015-02-05 21:09:46.296481
18	6	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-02-05 21:09:46.31053	2015-02-05 21:09:46.31053
19	6	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-05 21:09:46.322727	2015-02-05 21:09:46.322727
20	7	EXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-05 21:09:46.334732	2015-02-05 21:09:46.334732
21	7	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-05 21:09:46.345024	2015-02-05 21:09:46.345024
22	7	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-05 21:09:46.358159	2015-02-05 21:09:46.358159
23	8	EXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-05 21:09:46.372768	2015-02-05 21:09:46.372768
24	8	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-02-05 21:09:46.384173	2015-02-05 21:09:46.384173
25	9	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-02-05 21:09:46.394981	2015-02-05 21:09:46.394981
26	11	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-05 21:09:46.406799	2015-02-05 21:09:46.406799
27	10	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-05 21:09:46.418306	2015-02-05 21:09:46.418306
28	12	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-02-05 21:09:46.43065	2015-02-05 21:09:46.43065
29	13	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-05 21:09:46.444337	2015-02-05 21:09:46.444337
30	13	WHOLE GENE	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-05 21:09:46.45724	2015-02-05 21:09:46.45724
31	31	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-05 21:09:46.473339	2015-02-05 21:09:46.473339
32	14	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-05 21:09:46.484586	2015-02-05 21:09:46.484586
125	43	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-05 21:09:47.761249	2015-02-05 21:09:47.761249
33	16	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-05 21:09:46.498192	2015-02-05 21:09:46.498192
34	16	R882H	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-05 21:09:46.51429	2015-02-05 21:09:46.51429
35	17	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-02-05 21:09:46.528944	2015-02-05 21:09:46.528944
36	17	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-02-05 21:09:46.542684	2015-02-05 21:09:46.542684
37	18	D769H	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.558332	2015-02-05 21:09:46.558332
38	18	DEL 755-759	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.57315	2015-02-05 21:09:46.57315
39	18	G309A	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.584763	2015-02-05 21:09:46.584763
40	18	L755S	This variant was one of the first of the ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in cell lines.	2015-02-05 21:09:46.596939	2015-02-05 21:09:46.596939
41	18	L755W	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.614067	2015-02-05 21:09:46.614067
42	18	P780INS	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.626162	2015-02-05 21:09:46.626162
43	18	R678Q	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.638751	2015-02-05 21:09:46.638751
44	18	R896C	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.650837	2015-02-05 21:09:46.650837
45	18	V777L	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.661297	2015-02-05 21:09:46.661297
46	18	V842I	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-05 21:09:46.676642	2015-02-05 21:09:46.676642
47	19	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-05 21:09:46.6913	2015-02-05 21:09:46.6913
48	19	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-05 21:09:46.708315	2015-02-05 21:09:46.708315
49	19	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-05 21:09:46.724235	2015-02-05 21:09:46.724235
50	19	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-05 21:09:46.737108	2015-02-05 21:09:46.737108
51	19	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-05 21:09:46.750788	2015-02-05 21:09:46.750788
52	20	FGFR2-AFF3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.768176	2015-02-05 21:09:46.768176
53	20	FGFR2-BICC1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.784192	2015-02-05 21:09:46.784192
54	20	FGFR2-CASP7	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.800412	2015-02-05 21:09:46.800412
55	20	FGFR2-CCDC6	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.815636	2015-02-05 21:09:46.815636
56	20	FGFR2-KIAA1967	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.831643	2015-02-05 21:09:46.831643
57	20	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.851334	2015-02-05 21:09:46.851334
58	20	FGFR2-OFD1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.861523	2015-02-05 21:09:46.861523
59	20	FGFR2-SLC45A3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.880618	2015-02-05 21:09:46.880618
60	21	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-05 21:09:46.894789	2015-02-05 21:09:46.894789
61	22	ITD	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-02-05 21:09:46.908939	2015-02-05 21:09:46.908939
62	22	TKD	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-02-05 21:09:46.919638	2015-02-05 21:09:46.919638
63	23	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-02-05 21:09:46.933979	2015-02-05 21:09:46.933979
64	24	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-05 21:09:46.947218	2015-02-05 21:09:46.947218
65	24	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-05 21:09:46.966378	2015-02-05 21:09:46.966378
66	24	R132G	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-05 21:09:46.981595	2015-02-05 21:09:46.981595
67	24	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-05 21:09:46.9951	2015-02-05 21:09:46.9951
68	24	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-05 21:09:47.0093	2015-02-05 21:09:47.0093
69	25	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-02-05 21:09:47.022494	2015-02-05 21:09:47.022494
70	25	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-02-05 21:09:47.034241	2015-02-05 21:09:47.034241
71	26	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-02-05 21:09:47.053145	2015-02-05 21:09:47.053145
72	27	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-02-05 21:09:47.071019	2015-02-05 21:09:47.071019
73	27	EXON 11	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-05 21:09:47.085589	2015-02-05 21:09:47.085589
74	27	EXON 13	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-02-05 21:09:47.10167	2015-02-05 21:09:47.10167
75	27	EXON 14	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-02-05 21:09:47.11781	2015-02-05 21:09:47.11781
76	27	EXON 17	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-02-05 21:09:47.132725	2015-02-05 21:09:47.132725
77	27	EXON 9	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-02-05 21:09:47.147045	2015-02-05 21:09:47.147045
78	27	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-02-05 21:09:47.160462	2015-02-05 21:09:47.160462
79	27	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-02-05 21:09:47.173767	2015-02-05 21:09:47.173767
80	28	EXON 1	A study by Li√®vre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-02-05 21:09:47.199983	2015-02-05 21:09:47.199983
81	28	EXON 2	In a study by Pan et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-02-05 21:09:47.220066	2015-02-05 21:09:47.220066
82	28	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-05 21:09:47.233945	2015-02-05 21:09:47.233945
83	28	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-05 21:09:47.24624	2015-02-05 21:09:47.24624
84	28	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-05 21:09:47.258562	2015-02-05 21:09:47.258562
85	28	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-05 21:09:47.270834	2015-02-05 21:09:47.270834
86	28	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-05 21:09:47.284472	2015-02-05 21:09:47.284472
87	28	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-05 21:09:47.30095	2015-02-05 21:09:47.30095
88	29	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-02-05 21:09:47.315894	2015-02-05 21:09:47.315894
89	29	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-02-05 21:09:47.328502	2015-02-05 21:09:47.328502
90	30	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-05 21:09:47.341813	2015-02-05 21:09:47.341813
91	32	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-02-05 21:09:47.356002	2015-02-05 21:09:47.356002
92	33	EXON 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-02-05 21:09:47.370872	2015-02-05 21:09:47.370872
93	33	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-02-05 21:09:47.384955	2015-02-05 21:09:47.384955
94	34	EXON 1	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-02-05 21:09:47.399833	2015-02-05 21:09:47.399833
95	34	EXON 2	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-02-05 21:09:47.409336	2015-02-05 21:09:47.409336
96	34	EXON 3	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-05 21:09:47.418923	2015-02-05 21:09:47.418923
97	34	EXON 4	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-05 21:09:47.428592	2015-02-05 21:09:47.428592
98	34	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-05 21:09:47.442169	2015-02-05 21:09:47.442169
99	34	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-05 21:09:47.452938	2015-02-05 21:09:47.452938
100	34	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-05 21:09:47.462101	2015-02-05 21:09:47.462101
101	34	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-05 21:09:47.473546	2015-02-05 21:09:47.473546
102	34	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-05 21:09:47.483695	2015-02-05 21:09:47.483695
103	36	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-05 21:09:47.496869	2015-02-05 21:09:47.496869
104	36	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-05 21:09:47.507293	2015-02-05 21:09:47.507293
105	36	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-05 21:09:47.517808	2015-02-05 21:09:47.517808
106	36	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-05 21:09:47.530995	2015-02-05 21:09:47.530995
107	36	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-05 21:09:47.542821	2015-02-05 21:09:47.542821
108	35	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-05 21:09:47.555596	2015-02-05 21:09:47.555596
109	35	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-05 21:09:47.571203	2015-02-05 21:09:47.571203
110	35	EXON 20	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-05 21:09:47.584719	2015-02-05 21:09:47.584719
111	35	EXON 9	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-05 21:09:47.598613	2015-02-05 21:09:47.598613
112	35	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-05 21:09:47.613045	2015-02-05 21:09:47.613045
113	37	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-02-05 21:09:47.625129	2015-02-05 21:09:47.625129
114	15	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-05 21:09:47.638148	2015-02-05 21:09:47.638148
115	39	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-02-05 21:09:47.651833	2015-02-05 21:09:47.651833
116	38	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-02-05 21:09:47.664462	2015-02-05 21:09:47.664462
117	40	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-02-05 21:09:47.676032	2015-02-05 21:09:47.676032
118	40	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-02-05 21:09:47.687947	2015-02-05 21:09:47.687947
119	42	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-02-05 21:09:47.700227	2015-02-05 21:09:47.700227
120	42	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-02-05 21:09:47.709842	2015-02-05 21:09:47.709842
121	43	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-02-05 21:09:47.71965	2015-02-05 21:09:47.71965
122	43	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-02-05 21:09:47.728775	2015-02-05 21:09:47.728775
123	43	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-02-05 21:09:47.741108	2015-02-05 21:09:47.741108
124	43	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-05 21:09:47.7524	2015-02-05 21:09:47.7524
126	43	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-05 21:09:47.771919	2015-02-05 21:09:47.771919
127	43	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-05 21:09:47.782035	2015-02-05 21:09:47.782035
128	43	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-05 21:09:47.793869	2015-02-05 21:09:47.793869
129	44	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-05 21:09:47.805304	2015-02-05 21:09:47.805304
130	44	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-05 21:09:47.814909	2015-02-05 21:09:47.814909
131	45	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-05 21:09:47.826371	2015-02-05 21:09:47.826371
132	46	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. Although this mutation has not been determined to provide prognostic information in AML, patients with MDS were shown to be at an increased risk of transformation to secondary AML. However, the prognostic importance of U2AF1 mutations in MDS has been debated.	2015-02-05 21:09:47.838633	2015-02-05 21:09:47.838633
133	46	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. Although this mutation has not been determined to provide prognostic information in AML, patients with MDS were shown to be at an increased risk of transformation to secondary AML. However, the prognostic importance of U2AF1 mutations in MDS has been debated.	2015-02-05 21:09:47.85115	2015-02-05 21:09:47.85115
134	47	EXON 7	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-05 21:09:47.861489	2015-02-05 21:09:47.861489
135	47	EXON 9	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-05 21:09:47.873229	2015-02-05 21:09:47.873229
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating) FROM stdin;
1	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.		Supports	1	1	1	1	40	131	2015-02-05 21:09:48.00731	2015-02-05 21:09:48.00731	3
2	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.		Supports	1	1	1	1	40	130	2015-02-05 21:09:48.065219	2015-02-05 21:09:48.065219	3
3	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.		Supports	1	1	1	2	41	129	2015-02-05 21:09:48.095476	2015-02-05 21:09:48.095476	3
4	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.		Supports	1	2	2	3	42	129	2015-02-05 21:09:48.128457	2015-02-05 21:09:48.128457	3
5	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.		Supports	1	2	3	4	43	115	2015-02-05 21:09:48.16249	2015-02-05 21:09:48.16249	4
6	PTEN nonsense  , including R233*, has been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.		Supports	2	1	3	5	44	115	2015-02-05 21:09:48.206436	2015-02-05 21:09:48.206436	3
7	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	4	6	45	10	2015-02-05 21:09:48.245698	2015-02-05 21:09:48.245698	5
8	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	4	6	45	14	2015-02-05 21:09:48.280116	2015-02-05 21:09:48.280116	5
9	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.		Supports	1	1	4	6	45	15	2015-02-05 21:09:48.313071	2015-02-05 21:09:48.313071	5
10	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.		Supports	1	1	4	6	46	12	2015-02-05 21:09:48.349678	2015-02-05 21:09:48.349678	3
11	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1  .		Supports	2	1	3	7	47	92	2015-02-05 21:09:48.382298	2015-02-05 21:09:48.382298	5
12	NPM1 mutations were not associated with M2 FAB subtypes		Supports	3	1	3	7	47	92	2015-02-05 21:09:48.424686	2015-02-05 21:09:48.424686	4
13	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes		Supports	3	1	3	7	47	92	2015-02-05 21:09:48.47414	2015-02-05 21:09:48.47414	4
14	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult AML patients (median age 43)		Supports	1	1	3	7	48	92	2015-02-05 21:09:48.502417	2015-02-05 21:09:48.502417	4
15	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	2	1	3	7	49	92	2015-02-05 21:09:48.533271	2015-02-05 21:09:48.533271	4
16	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy		Supports	2	1	3	7	50	92	2015-02-05 21:09:48.562366	2015-02-05 21:09:48.562366	4
17	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old		Supports	2	1	3	7	51	92	2015-02-05 21:09:48.595623	2015-02-05 21:09:48.595623	4
18	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients		Supports	2	1	3	7	52	92	2015-02-05 21:09:48.619438	2015-02-05 21:09:48.619438	4
19	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients		Supports	2	1	3	7	53	92	2015-02-05 21:09:48.647338	2015-02-05 21:09:48.647338	4
20	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3		Supports	2	1	3	7	54	92	2015-02-05 21:09:48.680529	2015-02-05 21:09:48.680529	4
21	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation		Supports	2	1	3	7	54	92	2015-02-05 21:09:48.708401	2015-02-05 21:09:48.708401	4
22	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)		Does Not Support	3	1	3	7	55	93	2015-02-05 21:09:48.733017	2015-02-05 21:09:48.733017	4
23	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)		Supports	3	1	3	7	55	93	2015-02-05 21:09:48.756753	2015-02-05 21:09:48.756753	4
24	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD		Supports	1	2	5	7	56	92	2015-02-05 21:09:48.785777	2015-02-05 21:09:48.785777	3
25	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD		Supports	1	2	6	7	56	92	2015-02-05 21:09:48.799345	2015-02-05 21:09:48.799345	3
26	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status		Supports	1	1	6	7	57	92	2015-02-05 21:09:48.831657	2015-02-05 21:09:48.831657	3
27	Frequency of NPM1 mutations was not different in normal karyotype AML patients with CEPBA, NRAS or KIT mutations.		Does Not Support	3	1	3	7	47	92	2015-02-05 21:09:48.866285	2015-02-05 21:09:48.866285	3
28	NPM1 mutations were not associated with M3 FAB subtype (0/55).		Does Not Support	3	1	3	7	53	92	2015-02-05 21:09:48.900906	2015-02-05 21:09:48.900906	3
29	NPM1 mutations were associated with M5a and M5b FAB subtypes		Supports	3	1	3	7	53	92	2015-02-05 21:09:48.938447	2015-02-05 21:09:48.938447	3
30	NPM1 mutations were associated with normal karyotype in older (>60) patients		Supports	3	1	3	7	58	92	2015-02-05 21:09:48.970285	2015-02-05 21:09:48.970285	3
31	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved		Supports	1	1	3	7	58	92	2015-02-05 21:09:48.997373	2015-02-05 21:09:48.997373	3
32	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation		Does Not Support	2	1	3	7	59	92	2015-02-05 21:09:49.026868	2015-02-05 21:09:49.026868	3
33	Complete remission rates were higher and both disease-free and overall survival was longer for patients with Exon 12 NPM1  .		Supports	2	1	3	7	53	92	2015-02-05 21:09:49.053027	2015-02-05 21:09:49.053027	3
34	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed		Supports	2	1	3	7	60	92	2015-02-05 21:09:49.084199	2015-02-05 21:09:49.084199	3
35	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants		Supports	2	1	3	7	61	92	2015-02-05 21:09:49.114201	2015-02-05 21:09:49.114201	3
36	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients		Supports	2	1	3	7	62	92	2015-02-05 21:09:49.145348	2015-02-05 21:09:49.145348	3
37	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD		Supports	2	1	3	7	63	92	2015-02-05 21:09:49.171421	2015-02-05 21:09:49.171421	3
38	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients		Supports	2	1	3	7	52	92	2015-02-05 21:09:49.199177	2015-02-05 21:09:49.199177	3
39	Mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.		Supports	1	3	3	7	64	93	2015-02-05 21:09:49.226	2015-02-05 21:09:49.226	3
40	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD		Supports	1	1	6	7	58	92	2015-02-05 21:09:49.249222	2015-02-05 21:09:49.249222	2
41	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.		Supports	1	1	7	7	65	92	2015-02-05 21:09:49.275564	2015-02-05 21:09:49.275564	2
42	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid		Supports	1	1	8	7	66	92	2015-02-05 21:09:49.305665	2015-02-05 21:09:49.305665	2
43	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes		Supports	3	1	3	7	48	92	2015-02-05 21:09:49.334219	2015-02-05 21:09:49.334219	2
44	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)		Supports	1	1	3	7	67	92	2015-02-05 21:09:49.359526	2015-02-05 21:09:49.359526	2
45	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2		Supports	1	1	3	7	65	92	2015-02-05 21:09:49.38565	2015-02-05 21:09:49.38565	2
46	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy		Supports	1	3	3	7	68	92	2015-02-05 21:09:49.416608	2015-02-05 21:09:49.416608	2
47	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy		Supports	1	3	3	7	68	92	2015-02-05 21:09:49.443208	2015-02-05 21:09:49.443208	2
48	NPM1 mutation without FLT3-ITD was significantly associated with complete remission		Supports	2	1	3	7	69	92	2015-02-05 21:09:49.474363	2015-02-05 21:09:49.474363	3
49	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.		Does Not Support	1	2	3	6	70	11	2015-02-05 21:09:49.501132	2015-02-05 21:09:49.501132	3
50	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.		Supports	1	2	3	6	70	11	2015-02-05 21:09:49.526457	2015-02-05 21:09:49.526457	3
51	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.		Supports	1	2	9	8	71	117	2015-02-05 21:09:49.557647	2015-02-05 21:09:49.557647	3
52	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.		Supports	1	2	9	8	71	118	2015-02-05 21:09:49.592241	2015-02-05 21:09:49.592241	3
53	In patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.		Supports	1	1	10	9	72	2	2015-02-05 21:09:49.635748	2015-02-05 21:09:49.635748	4
54	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in "anti-tumor activitiy".		Supports	1	1	11	10	73	57	2015-02-05 21:09:49.688591	2015-02-05 21:09:49.688591	3
55	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.		Supports	1	2	12	11	74	52	2015-02-05 21:09:49.72226	2015-02-05 21:09:49.72226	3
56	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.		Supports	1	2	12	11	74	54	2015-02-05 21:09:49.754875	2015-02-05 21:09:49.754875	3
57	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.		Supports	1	2	12	11	74	55	2015-02-05 21:09:49.781068	2015-02-05 21:09:49.781068	3
58	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.		Supports	1	2	12	10	74	53	2015-02-05 21:09:49.803993	2015-02-05 21:09:49.803993	3
59	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.		Supports	1	2	12	12	74	59	2015-02-05 21:09:49.827863	2015-02-05 21:09:49.827863	3
60	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.		Supports	1	2	12	13	74	56	2015-02-05 21:09:49.86032	2015-02-05 21:09:49.86032	3
61	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.		Supports	1	2	12	14	74	58	2015-02-05 21:09:49.89187	2015-02-05 21:09:49.89187	3
62	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	12	15	73	60	2015-02-05 21:09:49.92011	2015-02-05 21:09:49.92011	3
63	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	11	15	73	60	2015-02-05 21:09:49.931328	2015-02-05 21:09:49.931328	3
64	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	12	10	73	60	2015-02-05 21:09:49.952793	2015-02-05 21:09:49.952793	3
65	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	11	10	73	60	2015-02-05 21:09:49.964179	2015-02-05 21:09:49.964179	3
66	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	12	16	73	60	2015-02-05 21:09:49.993367	2015-02-05 21:09:49.993367	3
97	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.		Supports	1	1	3	11	82	49	2015-02-05 21:09:50.653973	2015-02-05 21:09:50.653973	3
67	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	11	16	73	60	2015-02-05 21:09:50.006706	2015-02-05 21:09:50.006706	3
68	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	12	17	73	60	2015-02-05 21:09:50.034654	2015-02-05 21:09:50.034654	3
69	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	11	17	73	60	2015-02-05 21:09:50.045761	2015-02-05 21:09:50.045761	3
70	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	12	18	73	60	2015-02-05 21:09:50.077519	2015-02-05 21:09:50.077519	3
71	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	11	18	73	60	2015-02-05 21:09:50.08897	2015-02-05 21:09:50.08897	3
72	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	12	19	73	60	2015-02-05 21:09:50.116243	2015-02-05 21:09:50.116243	3
73	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.		Supports	1	1	11	19	73	60	2015-02-05 21:09:50.12811	2015-02-05 21:09:50.12811	3
74	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT		Supports	2	1	3	20	75	72	2015-02-05 21:09:50.153377	2015-02-05 21:09:50.153377	4
75	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	2	10	6	76	78	2015-02-05 21:09:50.187162	2015-02-05 21:09:50.187162	3
76	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	2	13	6	76	78	2015-02-05 21:09:50.19961	2015-02-05 21:09:50.19961	3
77	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.		Supports	1	2	14	6	76	78	2015-02-05 21:09:50.211185	2015-02-05 21:09:50.211185	3
78	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	2	15	3	77	78	2015-02-05 21:09:50.245104	2015-02-05 21:09:50.245104	3
79	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	2	10	3	77	78	2015-02-05 21:09:50.256176	2015-02-05 21:09:50.256176	3
80	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.		Supports	1	2	13	3	77	78	2015-02-05 21:09:50.269458	2015-02-05 21:09:50.269458	3
81	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.		Supports	1	1	16	21	78	79	2015-02-05 21:09:50.299762	2015-02-05 21:09:50.299762	3
82	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.		Supports	1	2	15	6	76	78	2015-02-05 21:09:50.326492	2015-02-05 21:09:50.326492	3
83	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.		Supports	1	2	10	21	79	79	2015-02-05 21:09:50.356539	2015-02-05 21:09:50.356539	3
84	In a case study, a patient with anal melanoma harboring an exon 11 KIT mutation showed marked response 4 months after imatinib treatment.		Supports	1	1	10	6	80	78	2015-02-05 21:09:50.386786	2015-02-05 21:09:50.386786	3
85	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	2	17	11	81	47	2015-02-05 21:09:50.415219	2015-02-05 21:09:50.415219	5
86	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	2	18	11	81	47	2015-02-05 21:09:50.42887	2015-02-05 21:09:50.42887	5
87	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	2	17	11	81	48	2015-02-05 21:09:50.455946	2015-02-05 21:09:50.455946	5
88	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	2	18	11	81	48	2015-02-05 21:09:50.468002	2015-02-05 21:09:50.468002	5
89	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	4	17	11	81	49	2015-02-05 21:09:50.493476	2015-02-05 21:09:50.493476	5
90	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	4	18	11	81	49	2015-02-05 21:09:50.503288	2015-02-05 21:09:50.503288	5
91	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	4	17	11	81	50	2015-02-05 21:09:50.530101	2015-02-05 21:09:50.530101	5
92	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	4	18	11	81	50	2015-02-05 21:09:50.540798	2015-02-05 21:09:50.540798	5
93	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	4	17	11	81	51	2015-02-05 21:09:50.563004	2015-02-05 21:09:50.563004	5
94	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.		Supports	1	4	18	11	81	51	2015-02-05 21:09:50.575782	2015-02-05 21:09:50.575782	5
95	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.		Supports	1	2	3	11	82	47	2015-02-05 21:09:50.602367	2015-02-05 21:09:50.602367	3
96	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.		Supports	1	1	3	11	82	48	2015-02-05 21:09:50.630313	2015-02-05 21:09:50.630313	3
98	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.		Supports	1	1	3	11	82	50	2015-02-05 21:09:50.681818	2015-02-05 21:09:50.681818	3
99	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.		Supports	1	1	3	11	82	51	2015-02-05 21:09:50.710266	2015-02-05 21:09:50.710266	3
100	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from an ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.		Supports	3	1	3	22	83	114	2015-02-05 21:09:50.752587	2015-02-05 21:09:50.752587	4
101	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from an ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.		Supports	3	1	3	22	83	32	2015-02-05 21:09:50.78096	2015-02-05 21:09:50.78096	4
102	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.		Supports	1	2	19	3	84	7	2015-02-05 21:09:50.807603	2015-02-05 21:09:50.807603	3
103	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.		Supports	1	4	19	3	85	6	2015-02-05 21:09:50.837824	2015-02-05 21:09:50.837824	4
104	In patients with non-small cell lung cancer harboring EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.		Supports	1	4	19	3	85	5	2015-02-05 21:09:50.865788	2015-02-05 21:09:50.865788	4
105	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective.		Supports	1	4	19	3	85	4	2015-02-05 21:09:50.896534	2015-02-05 21:09:50.896534	4
106	Fusion protein undergoes proteolysis in APL cells upon ATRA treatment in cell lines.		Supports	1	1	6	23	86	113	2015-02-05 21:09:50.926178	2015-02-05 21:09:50.926178	5
107	Fusion protein degradation is accelerated upon ATRA treatment at pharmalogical concentrations.		Supports	1	1	6	23	86	116	2015-02-05 21:09:50.943741	2015-02-05 21:09:50.943741	5
108	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580		Supports	1	2	10	24	87	31	2015-02-05 21:09:50.965417	2015-02-05 21:09:50.965417	3
109	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580		Supports	1	2	20	24	87	31	2015-02-05 21:09:50.980674	2015-02-05 21:09:50.980674	3
110	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580		Supports	1	2	10	24	87	90	2015-02-05 21:09:51.001039	2015-02-05 21:09:51.001039	3
111	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580		Supports	1	2	20	24	87	90	2015-02-05 21:09:51.012383	2015-02-05 21:09:51.012383	3
112	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.		Supports	2	3	3	25	88	119	2015-02-05 21:09:51.03729	2015-02-05 21:09:51.03729	3
113	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.		Supports	2	3	3	25	88	120	2015-02-05 21:09:51.054669	2015-02-05 21:09:51.054669	3
114	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML		Does Not Support	2	1	3	7	89	134	2015-02-05 21:09:51.077647	2015-02-05 21:09:51.077647	4
115	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.		Does Not Support	2	1	3	7	89	135	2015-02-05 21:09:51.097302	2015-02-05 21:09:51.097302	4
116	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	1	1	3	7	90	134	2015-02-05 21:09:51.116146	2015-02-05 21:09:51.116146	3
117	No difference in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML		Does Not Support	2	1	3	7	89	134	2015-02-05 21:09:51.138009	2015-02-05 21:09:51.138009	3
118	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	2	1	3	7	90	134	2015-02-05 21:09:51.159877	2015-02-05 21:09:51.159877	3
119	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	2	1	3	7	90	134	2015-02-05 21:09:51.181509	2015-02-05 21:09:51.181509	3
120	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients		Supports	2	1	3	7	91	134	2015-02-05 21:09:51.20318	2015-02-05 21:09:51.20318	3
121	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML		Supports	2	1	3	7	89	134	2015-02-05 21:09:51.224422	2015-02-05 21:09:51.224422	3
122	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	1	1	3	7	90	135	2015-02-05 21:09:51.244683	2015-02-05 21:09:51.244683	3
123	No difference in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML		Does Not Support	2	1	3	7	89	135	2015-02-05 21:09:51.262025	2015-02-05 21:09:51.262025	3
124	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	2	1	3	7	90	135	2015-02-05 21:09:51.283218	2015-02-05 21:09:51.283218	3
125	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML		Supports	2	1	3	7	90	135	2015-02-05 21:09:51.303185	2015-02-05 21:09:51.303185	3
126	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients		Supports	2	1	3	7	91	135	2015-02-05 21:09:51.323969	2015-02-05 21:09:51.323969	3
127	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML		Supports	2	1	3	7	89	135	2015-02-05 21:09:51.343357	2015-02-05 21:09:51.343357	3
128	in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence		Supports	2	1	3	7	92	134	2015-02-05 21:09:51.361456	2015-02-05 21:09:51.361456	2
129	in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML		Supports	2	1	3	7	92	134	2015-02-05 21:09:51.383465	2015-02-05 21:09:51.383465	2
130	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.		Does Not Support	3	3	3	7	93	132	2015-02-05 21:09:51.403786	2015-02-05 21:09:51.403786	3
131	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.		Does Not Support	3	3	3	7	93	133	2015-02-05 21:09:51.420569	2015-02-05 21:09:51.420569	3
132	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.		Supports	2	3	3	25	94	132	2015-02-05 21:09:51.442625	2015-02-05 21:09:51.442625	3
133	After adjust for age and cancer stage, presence of U2AF mutation such as S34Y/F is prognostic for poorer survival outcomes in patients with MDS.		Supports	2	3	3	25	94	133	2015-02-05 21:09:51.460841	2015-02-05 21:09:51.460841	3
134	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.		Does Not Support	2	3	3	7	93	132	2015-02-05 21:09:51.477178	2015-02-05 21:09:51.477178	2
135	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.		Does Not Support	2	3	3	7	93	133	2015-02-05 21:09:51.498899	2015-02-05 21:09:51.498899	2
136	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.		Does Not Support	2	3	3	25	93	132	2015-02-05 21:09:51.515806	2015-02-05 21:09:51.515806	2
137	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.		Does Not Support	2	3	3	25	93	133	2015-02-05 21:09:51.533481	2015-02-05 21:09:51.533481	2
138	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumor with wild type TP53.		Supports	1	2	21	11	95	121	2015-02-05 21:09:51.556513	2015-02-05 21:09:51.556513	3
139	Breast cancer patients who harbor R175H mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	2	1	3	11	96	121	2015-02-05 21:09:51.573941	2015-02-05 21:09:51.573941	3
140	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R248 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	2	1	3	11	97	122	2015-02-05 21:09:51.598052	2015-02-05 21:09:51.598052	3
141	Breast cancer patients who harbor R248Q mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	2	1	3	11	96	122	2015-02-05 21:09:51.615046	2015-02-05 21:09:51.615046	3
142	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.		Supports	2	1	3	11	96	123	2015-02-05 21:09:51.634518	2015-02-05 21:09:51.634518	3
143	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as R249 is prognostic for a worse overall survival compared to those harboring wild-type TP53.		Supports	2	1	3	11	97	124	2015-02-05 21:09:51.656779	2015-02-05 21:09:51.656779	3
144	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as R249 is prognostic for a worse overall survival compared to those harboring wild-type TP53.		Supports	2	1	3	11	97	125	2015-02-05 21:09:51.696663	2015-02-05 21:09:51.696663	3
145	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R273 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	2	1	3	11	97	126	2015-02-05 21:09:51.71465	2015-02-05 21:09:51.71465	3
146	Breast cancer patients who harbor R273C mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	2	1	3	11	96	126	2015-02-05 21:09:51.735326	2015-02-05 21:09:51.735326	3
147	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R273 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.		Supports	2	1	3	11	97	127	2015-02-05 21:09:51.754755	2015-02-05 21:09:51.754755	3
148	Breast cancer patients who harbor R273H mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.		Supports	2	1	3	11	96	127	2015-02-05 21:09:51.773306	2015-02-05 21:09:51.773306	3
149	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as V173 is prognostic for a worse overall survival compared to those harboring wild-type TP53.		Supports	2	1	3	11	97	128	2015-02-05 21:09:51.79271	2015-02-05 21:09:51.79271	3
150	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.		Supports	3	1	3	8	98	118	2015-02-05 21:09:51.815561	2015-02-05 21:09:51.815561	5
151	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.		Supports	2	1	3	8	99	118	2015-02-05 21:09:51.834439	2015-02-05 21:09:51.834439	4
152	The use of AZD1480 in vitro led to strong repression of tyroid cancer cell growth.		Supports	1	2	22	8	100	118	2015-02-05 21:09:51.858008	2015-02-05 21:09:51.858008	3
153	MAP2K1 P124S confers increased resistance to inhibition by AZD6244 inhibition by about 5 fold.		Supports	1	2	23	6	101	88	2015-02-05 21:09:51.882375	2015-02-05 21:09:51.882375	3
154	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.		Supports	1	2	23	6	101	89	2015-02-05 21:09:51.904289	2015-02-05 21:09:51.904289	3
155	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	2	24	11	102	108	2015-02-05 21:09:51.926826	2015-02-05 21:09:51.926826	4
156	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	2	24	11	102	109	2015-02-05 21:09:51.946003	2015-02-05 21:09:51.946003	4
157	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.		Supports	1	2	24	11	102	112	2015-02-05 21:09:51.967079	2015-02-05 21:09:51.967079	4
158	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	25	26	103	108	2015-02-05 21:09:51.990976	2015-02-05 21:09:51.990976	4
159	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	25	26	103	109	2015-02-05 21:09:52.012381	2015-02-05 21:09:52.012381	4
160	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	25	26	103	112	2015-02-05 21:09:52.032836	2015-02-05 21:09:52.032836	4
161	The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.		Supports	1	3	26	26	104	108	2015-02-05 21:09:52.058303	2015-02-05 21:09:52.058303	3
162	The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.		Supports	1	3	27	26	104	108	2015-02-05 21:09:52.072628	2015-02-05 21:09:52.072628	3
163	The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.		Supports	1	3	26	26	104	109	2015-02-05 21:09:52.094997	2015-02-05 21:09:52.094997	3
164	The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.		Supports	1	3	27	26	104	109	2015-02-05 21:09:52.10569	2015-02-05 21:09:52.10569	3
165	The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase resistance to drugs such as  panitumumab or cetuximab.		Supports	1	3	26	26	104	112	2015-02-05 21:09:52.126902	2015-02-05 21:09:52.126902	3
166	The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase resistance to drugs such as  panitumumab or cetuximab.		Supports	1	3	27	26	104	112	2015-02-05 21:09:52.138851	2015-02-05 21:09:52.138851	3
167	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA		Supports	1	1	26	26	105	110	2015-02-05 21:09:52.160165	2015-02-05 21:09:52.160165	3
168	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA		Does Not Support	1	1	26	26	105	111	2015-02-05 21:09:52.178945	2015-02-05 21:09:52.178945	3
169	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	2	2	11	106	112	2015-02-05 21:09:52.201826	2015-02-05 21:09:52.201826	3
170	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	2	2	11	106	108	2015-02-05 21:09:52.228375	2015-02-05 21:09:52.228375	3
171	Rapamycin inhibits transformation induced by mutation in PIK3CA		Supports	1	2	2	11	106	109	2015-02-05 21:09:52.251709	2015-02-05 21:09:52.251709	3
172	The presence of exon 9 (such as E542K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.		Supports	2	3	3	26	104	108	2015-02-05 21:09:52.272642	2015-02-05 21:09:52.272642	3
173	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.		Supports	2	3	3	26	104	109	2015-02-05 21:09:52.2928	2015-02-05 21:09:52.2928	3
174	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.		Supports	2	3	3	26	104	112	2015-02-05 21:09:52.312489	2015-02-05 21:09:52.312489	3
175	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.		Supports	2	3	3	26	107	108	2015-02-05 21:09:52.337463	2015-02-05 21:09:52.337463	2
176	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.		Supports	2	3	3	26	107	109	2015-02-05 21:09:52.357551	2015-02-05 21:09:52.357551	2
177	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib showed relatively high potency.		Supports	1	2	28	27	108	103	2015-02-05 21:09:52.384936	2015-02-05 21:09:52.384936	4
178	In CHO cells with PDGFRA D842V mutation resulting in imatinib resistance, crenolanib showed relatively high potency.		Supports	1	2	28	27	108	104	2015-02-05 21:09:52.41183	2015-02-05 21:09:52.41183	4
179	In CHO cells with PDGFRA D842Y mutation resulting in imatinib resistance, crenolanib showed relatively high potency.		Supports	1	2	28	27	108	105	2015-02-05 21:09:52.43224	2015-02-05 21:09:52.43224	4
180	In CHO cells with PDGFRA deletion I843 mutation resulting in imatinib resistance, crenolanib showed relatively high potency.		Supports	1	2	28	27	108	106	2015-02-05 21:09:52.456383	2015-02-05 21:09:52.456383	4
181	In CHO cells with PDGFRA DI842-843IM mutation resulting in imatinib resistance, crenolanib showed relatively high potency.		Supports	1	2	28	27	108	107	2015-02-05 21:09:52.473492	2015-02-05 21:09:52.473492	4
182	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.		Supports	1	1	10	27	79	104	2015-02-05 21:09:52.495158	2015-02-05 21:09:52.495158	4
183	GIST cancer with D842V mutation is resistant to imatinib.		Supports	1	1	10	27	109	104	2015-02-05 21:09:52.51811	2015-02-05 21:09:52.51811	4
184	GIST tumor harboring PDGFRA D842V mutation are likely to be benign		Supports	3	1	3	27	110	104	2015-02-05 21:09:52.543476	2015-02-05 21:09:52.543476	3
185	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.		Supports	1	1	29	6	111	100	2015-02-05 21:09:52.572047	2015-02-05 21:09:52.572047	4
186	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.		Does Not Support	1	1	26	26	105	100	2015-02-05 21:09:52.596203	2015-02-05 21:09:52.596203	3
187	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.		Supports	1	1	26	26	105	100	2015-02-05 21:09:52.63702	2015-02-05 21:09:52.63702	3
188	The presence of NRAS mutation in AML patient does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)		Does Not Support	2	1	3	7	112	95	2015-02-05 21:09:52.663869	2015-02-05 21:09:52.663869	3
189	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML		Does Not Support	2	1	3	7	113	98	2015-02-05 21:09:52.686631	2015-02-05 21:09:52.686631	3
190	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.		Supports	2	1	3	26	114	95	2015-02-05 21:09:52.707749	2015-02-05 21:09:52.707749	3
191	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.		Supports	2	1	3	26	114	96	2015-02-05 21:09:52.72933	2015-02-05 21:09:52.72933	3
192	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.		Supports	2	1	3	26	114	97	2015-02-05 21:09:52.746444	2015-02-05 21:09:52.746444	3
193	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS		Supports	2	1	3	6	115	94	2015-02-05 21:09:52.769551	2015-02-05 21:09:52.769551	3
194	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS		Supports	2	1	3	6	115	95	2015-02-05 21:09:52.787179	2015-02-05 21:09:52.787179	3
195	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk		Supports	3	1	3	6	116	100	2015-02-05 21:09:52.814075	2015-02-05 21:09:52.814075	3
196	Due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG is shown to be effective in a patient with metastatic malignant melenoma with a NRAS G13D mutation		Supports	1	1	30	6	117	99	2015-02-05 21:09:52.842467	2015-02-05 21:09:52.842467	2
197	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months		Supports	1	1	31	6	118	101	2015-02-05 21:09:52.873692	2015-02-05 21:09:52.873692	2
198	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months		Supports	1	1	31	6	118	102	2015-02-05 21:09:52.89675	2015-02-05 21:09:52.89675	2
199	"AML with mutated NPM1" is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.		Supports	3	4	3	7	119	92	2015-02-05 21:09:52.916625	2015-02-05 21:09:52.916625	5
200	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increase survival.		Supports	1	4	32	5	120	91	2015-02-05 21:09:52.942255	2015-02-05 21:09:52.942255	5
201	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.		Supports	1	3	33	5	121	91	2015-02-05 21:09:52.967185	2015-02-05 21:09:52.967185	4
202	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.		Supports	1	1	34	5	121	91	2015-02-05 21:09:52.989688	2015-02-05 21:09:52.989688	4
203	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab		Supports	1	1	26	26	122	80	2015-02-05 21:09:53.021336	2015-02-05 21:09:53.021336	4
204	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in lung cancer model with KRAS G12D mutation led to marked tumor regression.		Supports	1	2	35	3	123	85	2015-02-05 21:09:53.049152	2015-02-05 21:09:53.049152	4
205	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in lung cancer model with KRAS G12D mutation led to marked tumor regression.		Supports	1	2	36	3	123	85	2015-02-05 21:09:53.066091	2015-02-05 21:09:53.066091	4
206	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft models		Supports	1	2	26	28	124	87	2015-02-05 21:09:53.096741	2015-02-05 21:09:53.096741	4
207	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS		Supports	1	1	26	28	124	87	2015-02-05 21:09:53.120869	2015-02-05 21:09:53.120869	4
208	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft models		Supports	1	2	26	28	124	87	2015-02-05 21:09:53.14558	2015-02-05 21:09:53.14558	4
209	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS		Supports	2	1	26	28	124	87	2015-02-05 21:09:53.167712	2015-02-05 21:09:53.167712	4
210	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment		Does Not Support	1	1	25	26	103	81	2015-02-05 21:09:53.18546	2015-02-05 21:09:53.18546	4
211	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.		Does Not Support	2	1	3	28	125	82	2015-02-05 21:09:53.212084	2015-02-05 21:09:53.212084	4
212	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib		Supports	1	1	37	3	126	81	2015-02-05 21:09:53.241721	2015-02-05 21:09:53.241721	3
213	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib		Supports	1	1	38	3	126	81	2015-02-05 21:09:53.255972	2015-02-05 21:09:53.255972	3
214	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer		Supports	1	2	26	26	127	83	2015-02-05 21:09:53.280173	2015-02-05 21:09:53.280173	3
215	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS		Supports	1	1	26	26	105	83	2015-02-05 21:09:53.310238	2015-02-05 21:09:53.310238	3
216	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer		Supports	2	2	3	26	127	83	2015-02-05 21:09:53.334977	2015-02-05 21:09:53.334977	3
217	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.		Supports	3	1	3	29	128	82	2015-02-05 21:09:53.364862	2015-02-05 21:09:53.364862	3
218	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.		Supports	3	1	3	29	128	86	2015-02-05 21:09:53.384665	2015-02-05 21:09:53.384665	3
219	KRAS G12D mutation occurs in never smokers significantly more often than in smokers		Supports	3	1	3	30	129	85	2015-02-05 21:09:53.413118	2015-02-05 21:09:53.413118	3
220	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival		Supports	2	1	3	31	130	82	2015-02-05 21:09:53.446455	2015-02-05 21:09:53.446455	3
221	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival		Supports	2	1	3	32	130	86	2015-02-05 21:09:53.476252	2015-02-05 21:09:53.476252	3
222	KRAS mutations in lung patients are not associated with smoking history, age and gender		Does Not Support	3	1	3	3	131	82	2015-02-05 21:09:53.503129	2015-02-05 21:09:53.503129	3
223	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC		Does Not Support	2	1	3	3	132	82	2015-02-05 21:09:53.529553	2015-02-05 21:09:53.529553	3
224	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC		Supports	2	1	3	3	133	82	2015-02-05 21:09:53.551444	2015-02-05 21:09:53.551444	3
225	KRAS mutations in lung patients are not associated with smoking history, age and gender		Does Not Support	3	1	3	3	131	86	2015-02-05 21:09:53.573999	2015-02-05 21:09:53.573999	3
226	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.		Supports	1	1	4	3	134	85	2015-02-05 21:09:53.593543	2015-02-05 21:09:53.593543	2
227	KRAS G12C occur more frequently in women than men		Supports	3	1	3	30	129	84	2015-02-05 21:09:53.630608	2015-02-05 21:09:53.630608	2
228	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	2	1	10	6	135	73	2015-02-05 21:09:53.657989	2015-02-05 21:09:53.657989	3
229	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	2	1	10	6	135	74	2015-02-05 21:09:53.677014	2015-02-05 21:09:53.677014	3
230	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.		Supports	2	1	10	6	135	76	2015-02-05 21:09:53.704154	2015-02-05 21:09:53.704154	3
231	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST		Does Not Support	2	1	3	21	136	73	2015-02-05 21:09:53.734058	2015-02-05 21:09:53.734058	3
232	There is no significant association between KIT mutations in exon 11, exon 9 or wild type in survival among GIST patients.		Does Not Support	2	1	3	21	137	73	2015-02-05 21:09:53.764058	2015-02-05 21:09:53.764058	3
233	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT		Supports	2	1	3	21	138	73	2015-02-05 21:09:53.793407	2015-02-05 21:09:53.793407	3
234	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT		Supports	2	1	3	21	137	75	2015-02-05 21:09:53.820033	2015-02-05 21:09:53.820033	3
235	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT		Supports	3	1	3	21	138	76	2015-02-05 21:09:53.848161	2015-02-05 21:09:53.848161	3
236	There is no significant association between KIT mutations in exon 11, exon 9 or wild type in survival among GIST patients.		Does Not Support	2	1	3	21	137	77	2015-02-05 21:09:53.871918	2015-02-05 21:09:53.871918	3
237	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT		Supports	3	1	3	21	138	73	2015-02-05 21:09:53.89963	2015-02-05 21:09:53.89963	2
238	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer		Does Not Support	2	1	3	30	139	73	2015-02-05 21:09:53.931937	2015-02-05 21:09:53.931937	2
239	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer		Does Not Support	2	1	3	30	139	77	2015-02-05 21:09:53.957686	2015-02-05 21:09:53.957686	2
240	In patients with JAK2 V617F, the use of peglated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.		Supports	1	1	39	33	140	71	2015-02-05 21:09:53.988245	2015-02-05 21:09:53.988245	4
241	JAK2 V617F not associatted wth  lymphoid leukemia.		Does Not Support	3	1	3	9	141	71	2015-02-05 21:09:54.021587	2015-02-05 21:09:54.021587	4
242	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).		Supports	3	1	3	9	141	71	2015-02-05 21:09:54.043691	2015-02-05 21:09:54.043691	4
243	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).		Supports	3	1	3	34	141	71	2015-02-05 21:09:54.072176	2015-02-05 21:09:54.072176	4
244	rs10974944 SNP in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer		Supports	3	1	3	35	142	71	2015-02-05 21:09:54.106852	2015-02-05 21:09:54.106852	4
245	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells		Supports	1	2	40	36	143	71	2015-02-05 21:09:54.144679	2015-02-05 21:09:54.144679	3
246	In patients with IDH2 R140Q/L mutation, the presence of mutation do not impact overall survival or disease free survival.		Does Not Support	2	1	3	7	144	69	2015-02-05 21:09:54.174308	2015-02-05 21:09:54.174308	3
247	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2		Supports	2	1	3	7	145	69	2015-02-05 21:09:54.205918	2015-02-05 21:09:54.205918	3
248	In AML, patients with IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2		Supports	2	1	3	7	145	70	2015-02-05 21:09:54.231425	2015-02-05 21:09:54.231425	3
249	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.		Does Not Support	2	1	3	25	146	69	2015-02-05 21:09:54.261697	2015-02-05 21:09:54.261697	3
250	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2		Does Not Support	2	3	3	7	147	69	2015-02-05 21:09:54.290188	2015-02-05 21:09:54.290188	2
251	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.		Does Not Support	2	3	3	7	147	70	2015-02-05 21:09:54.315422	2015-02-05 21:09:54.315422	2
252	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)		Does Not Support	2	3	3	25	148	69	2015-02-05 21:09:54.345818	2015-02-05 21:09:54.345818	2
253	In patients suffering from MDS, the presence of IDH2 mutation such as R172K do not confer prognostic value (overall survival)		Does Not Support	2	3	3	25	148	70	2015-02-05 21:09:54.371397	2015-02-05 21:09:54.371397	2
254	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival		Supports	2	1	3	37	149	64	2015-02-05 21:09:54.400256	2015-02-05 21:09:54.400256	4
255	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation		Supports	3	1	3	7	150	65	2015-02-05 21:09:54.428696	2015-02-05 21:09:54.428696	3
256	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	2	1	3	7	150	67	2015-02-05 21:09:54.454807	2015-02-05 21:09:54.454807	3
257	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	2	1	3	7	150	68	2015-02-05 21:09:54.47534	2015-02-05 21:09:54.47534	3
258	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.		Supports	2	1	3	38	151	64	2015-02-05 21:09:54.502292	2015-02-05 21:09:54.502292	3
259	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation		Supports	2	1	3	39	152	64	2015-02-05 21:09:54.529205	2015-02-05 21:09:54.529205	3
260	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.		Supports	3	1	3	7	153	64	2015-02-05 21:09:54.550744	2015-02-05 21:09:54.550744	2
261	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation		Does Not Support	2	1	3	7	154	64	2015-02-05 21:09:54.576591	2015-02-05 21:09:54.576591	2
262	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.		Does Not Support	2	1	3	7	150	65	2015-02-05 21:09:54.597357	2015-02-05 21:09:54.597357	2
263	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.		Supports	1	1	41	40	155	63	2015-02-05 21:09:54.625296	2015-02-05 21:09:54.625296	4
264	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.		Supports	1	1	42	40	155	63	2015-02-05 21:09:54.639281	2015-02-05 21:09:54.639281	4
265	In AML patients with FLT3-ITD  , concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.		Supports	2	1	3	7	55	61	2015-02-05 21:09:54.672336	2015-02-05 21:09:54.672336	5
266	Sorafenib is effective in patients with FLT3 internal tandem repeat , but not in wild type or D835 mutation.		Supports	1	2	14	7	156	61	2015-02-05 21:09:54.696708	2015-02-05 21:09:54.696708	4
267	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance		Supports	1	1	14	7	157	62	2015-02-05 21:09:54.718307	2015-02-05 21:09:54.718307	4
268	Sorafenib is effective in patients with FLT3 internal tandem repeat , but not in wild type or D835 mutation.		Supports	1	2	14	7	156	62	2015-02-05 21:09:54.740101	2015-02-05 21:09:54.740101	4
269	CEP701 is effective in AML patients with FLT3 mutation		Supports	1	1	43	7	158	61	2015-02-05 21:09:54.766313	2015-02-05 21:09:54.766313	4
270	CEP701 is effective in AML patients with FLT3 mutation		Supports	1	1	43	7	158	62	2015-02-05 21:09:54.789468	2015-02-05 21:09:54.789468	4
271	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat		Supports	1	2	44	7	159	61	2015-02-05 21:09:54.818802	2015-02-05 21:09:54.818802	4
272	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)		Supports	1	1	3	7	48	61	2015-02-05 21:09:54.843151	2015-02-05 21:09:54.843151	4
273	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	2	1	3	7	49	61	2015-02-05 21:09:54.867631	2015-02-05 21:09:54.867631	4
274	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation		Does Not Support	2	1	3	7	54	61	2015-02-05 21:09:54.8942	2015-02-05 21:09:54.8942	4
275	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.		Supports	2	1	3	7	160	61	2015-02-05 21:09:54.924429	2015-02-05 21:09:54.924429	4
276	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3		Supports	2	1	3	7	54	61	2015-02-05 21:09:54.947341	2015-02-05 21:09:54.947341	4
277	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation		Supports	2	1	3	7	54	61	2015-02-05 21:09:54.97412	2015-02-05 21:09:54.97412	4
278	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation		Supports	2	1	3	7	54	61	2015-02-05 21:09:54.998135	2015-02-05 21:09:54.998135	4
279	In AML patients, FLT3-TKD mutation is associated with poorer disease free survival compared to patients with wild type FLT3		Supports	2	1	3	7	161	62	2015-02-05 21:09:55.02946	2015-02-05 21:09:55.02946	4
280	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.		Supports	1	2	45	7	162	61	2015-02-05 21:09:55.056153	2015-02-05 21:09:55.056153	3
281	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.		Supports	1	2	45	7	162	62	2015-02-05 21:09:55.0812	2015-02-05 21:09:55.0812	3
282	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation		Supports	1	1	6	7	57	61	2015-02-05 21:09:55.105686	2015-02-05 21:09:55.105686	3
283	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.		Supports	1	2	7	7	162	61	2015-02-05 21:09:55.129978	2015-02-05 21:09:55.129978	3
284	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.		Supports	1	2	7	7	162	62	2015-02-05 21:09:55.151844	2015-02-05 21:09:55.151844	3
285	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type		Supports	3	1	3	7	163	61	2015-02-05 21:09:55.176151	2015-02-05 21:09:55.176151	3
286	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate		Supports	1	1	3	7	48	61	2015-02-05 21:09:55.196858	2015-02-05 21:09:55.196858	3
287	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3		Supports	2	1	3	7	163	61	2015-02-05 21:09:55.219566	2015-02-05 21:09:55.219566	3
288	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type		Does Not Support	3	1	3	7	163	62	2015-02-05 21:09:55.242542	2015-02-05 21:09:55.242542	3
289	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3		Does Not Support	2	1	3	7	163	62	2015-02-05 21:09:55.262133	2015-02-05 21:09:55.262133	3
290	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed		Supports	2	1	3	7	60	62	2015-02-05 21:09:55.281169	2015-02-05 21:09:55.281169	3
291	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate		Does Not Support	2	1	3	7	48	61	2015-02-05 21:09:55.300669	2015-02-05 21:09:55.300669	2
292	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD		Supports	2	4	3	7	164	61	2015-02-05 21:09:55.323997	2015-02-05 21:09:55.323997	2
293	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.		Supports	1	2	46	11	165	38	2015-02-05 21:09:55.347009	2015-02-05 21:09:55.347009	5
294	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.		Supports	1	2	46	11	165	39	2015-02-05 21:09:55.368818	2015-02-05 21:09:55.368818	5
295	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.		Supports	1	2	46	11	165	40	2015-02-05 21:09:55.391682	2015-02-05 21:09:55.391682	5
296	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.		Supports	1	2	46	11	165	41	2015-02-05 21:09:55.413801	2015-02-05 21:09:55.413801	5
297	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.		Supports	1	2	46	11	165	43	2015-02-05 21:09:55.437312	2015-02-05 21:09:55.437312	5
298	This mutation is sensitive to the irreversible kinase inhibitor, neratinib.		Supports	1	2	46	11	165	45	2015-02-05 21:09:55.460199	2015-02-05 21:09:55.460199	5
299	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.		Supports	1	2	46	11	165	46	2015-02-05 21:09:55.481816	2015-02-05 21:09:55.481816	5
300	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.		Supports	1	2	46	11	165	37	2015-02-05 21:09:55.50228	2015-02-05 21:09:55.50228	4
301	In NSCLC patients treated with tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.		Supports	1	1	3	3	166	35	2015-02-05 21:09:55.541507	2015-02-05 21:09:55.541507	4
402	Three studies have associated CCND1 expression with poorer survival.		Supports	2	1	3	3	210	18	2015-02-05 21:09:58.015155	2015-02-05 21:09:58.015155	3
302	Patients with NSCLC harboring EGFR T790M mutation has statistically worse overall survival compared to patients with L858R or other exon 19 activating mutation.		Supports	2	1	3	3	167	36	2015-02-05 21:09:55.566509	2015-02-05 21:09:55.566509	4
303	In NSCLC patients with T790M and another activating mutation, their progression free survival is shorter compared to those who do not possess T790M mutation.		Supports	2	1	3	3	168	36	2015-02-05 21:09:55.594342	2015-02-05 21:09:55.594342	4
304	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective.		Supports	1	1	38	3	169	36	2015-02-05 21:09:55.621705	2015-02-05 21:09:55.621705	3
305	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective.		Supports	1	1	47	3	169	36	2015-02-05 21:09:55.638558	2015-02-05 21:09:55.638558	3
306	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.		Does Not Support	1	1	37	3	170	35	2015-02-05 21:09:55.667919	2015-02-05 21:09:55.667919	3
307	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.		Does Not Support	1	1	38	3	170	35	2015-02-05 21:09:55.682047	2015-02-05 21:09:55.682047	3
308	Geftinib added to pemetrexed and carboplatin showed greater effectiveness than premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.		Supports	1	1	37	3	171	35	2015-02-05 21:09:55.708356	2015-02-05 21:09:55.708356	3
309	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.		Supports	1	1	37	3	172	36	2015-02-05 21:09:55.736487	2015-02-05 21:09:55.736487	3
310	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.		Supports	2	1	3	3	173	35	2015-02-05 21:09:55.767186	2015-02-05 21:09:55.767186	3
311	In NSCLC containing T790M mutation, staurosporine may be inhibitive of EGFR, especially when L858R mutation is also present.		Supports	1	3	48	3	174	36	2015-02-05 21:09:55.798666	2015-02-05 21:09:55.798666	1
312	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes		Supports	3	1	3	7	55	33	2015-02-05 21:09:55.825721	2015-02-05 21:09:55.825721	5
313	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.		Supports	2	1	3	7	55	33	2015-02-05 21:09:55.850025	2015-02-05 21:09:55.850025	5
314	In AML patients with FLT3-ITD  , concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.		Supports	2	1	3	7	55	33	2015-02-05 21:09:55.868583	2015-02-05 21:09:55.868583	5
315	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.		Supports	2	1	3	7	175	33	2015-02-05 21:09:55.897587	2015-02-05 21:09:55.897587	5
316	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.		Supports	1	1	49	7	176	33	2015-02-05 21:09:55.923905	2015-02-05 21:09:55.923905	4
317	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.		Does Not Support	1	1	7	7	176	33	2015-02-05 21:09:55.943563	2015-02-05 21:09:55.943563	4
318	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML		Does Not Support	2	1	3	7	177	33	2015-02-05 21:09:55.967829	2015-02-05 21:09:55.967829	4
319	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD		Does Not Support	2	1	3	7	49	33	2015-02-05 21:09:55.990194	2015-02-05 21:09:55.990194	4
320	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.		Does Not Support	2	1	3	7	176	33	2015-02-05 21:09:56.019337	2015-02-05 21:09:56.019337	4
321	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.		Supports	2	1	3	7	177	33	2015-02-05 21:09:56.0489	2015-02-05 21:09:56.0489	4
322	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3		Supports	2	1	3	7	49	33	2015-02-05 21:09:56.07279	2015-02-05 21:09:56.07279	4
323	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.		Supports	2	1	3	7	55	33	2015-02-05 21:09:56.101312	2015-02-05 21:09:56.101312	4
324	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.		Supports	2	1	3	7	177	33	2015-02-05 21:09:56.129882	2015-02-05 21:09:56.129882	4
325	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.		Supports	2	1	3	7	177	33	2015-02-05 21:09:56.155924	2015-02-05 21:09:56.155924	4
326	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A		Supports	3	1	3	7	49	33	2015-02-05 21:09:56.178684	2015-02-05 21:09:56.178684	3
327	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients		Supports	3	1	3	7	178	33	2015-02-05 21:09:56.202724	2015-02-05 21:09:56.202724	3
328	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients		Supports	3	1	3	7	178	33	2015-02-05 21:09:56.225839	2015-02-05 21:09:56.225839	3
329	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients		Supports	3	1	3	7	178	33	2015-02-05 21:09:56.251908	2015-02-05 21:09:56.251908	3
330	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.		Supports	3	1	3	7	176	33	2015-02-05 21:09:56.279029	2015-02-05 21:09:56.279029	3
331	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients		Supports	3	1	3	7	175	33	2015-02-05 21:09:56.304802	2015-02-05 21:09:56.304802	3
332	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.		Does Not Support	2	1	3	7	178	33	2015-02-05 21:09:56.332101	2015-02-05 21:09:56.332101	3
333	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival		Does Not Support	2	1	3	7	178	33	2015-02-05 21:09:56.357096	2015-02-05 21:09:56.357096	3
334	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients		Supports	2	1	3	7	178	33	2015-02-05 21:09:56.381706	2015-02-05 21:09:56.381706	3
335	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable cytogenetically normal AMLs		Supports	2	1	3	7	178	33	2015-02-05 21:09:56.406699	2015-02-05 21:09:56.406699	3
336	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A		Supports	2	1	3	7	49	33	2015-02-05 21:09:56.443903	2015-02-05 21:09:56.443903	3
337	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)		Supports	2	1	3	7	49	33	2015-02-05 21:09:56.462743	2015-02-05 21:09:56.462743	3
338	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A		Supports	2	1	3	7	49	33	2015-02-05 21:09:56.484642	2015-02-05 21:09:56.484642	3
339	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML		Supports	2	4	3	7	178	33	2015-02-05 21:09:56.502355	2015-02-05 21:09:56.502355	3
340	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML		Does Not Support	1	1	3	7	57	30	2015-02-05 21:09:56.524738	2015-02-05 21:09:56.524738	4
341	CEBPA mutation was significantly associated with complete remission		Supports	2	1	3	7	69	30	2015-02-05 21:09:56.546097	2015-02-05 21:09:56.546097	4
342	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML		Supports	2	1	3	7	57	30	2015-02-05 21:09:56.567634	2015-02-05 21:09:56.567634	4
343	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations		Supports	2	1	3	7	179	29	2015-02-05 21:09:56.595167	2015-02-05 21:09:56.595167	3
344	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA		Supports	2	1	3	7	179	29	2015-02-05 21:09:56.616829	2015-02-05 21:09:56.616829	3
345	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA		Does Not Support	2	1	3	7	179	29	2015-02-05 21:09:56.640058	2015-02-05 21:09:56.640058	2
346	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.		Supports	2	1	3	3	180	28	2015-02-05 21:09:56.666858	2015-02-05 21:09:56.666858	3
347	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.		Supports	2	1	3	3	180	28	2015-02-05 21:09:56.691078	2015-02-05 21:09:56.691078	3
348	In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.		Supports	1	1	50	41	181	27	2015-02-05 21:09:56.71879	2015-02-05 21:09:56.71879	4
349	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.		Supports	2	1	51	41	182	27	2015-02-05 21:09:56.747442	2015-02-05 21:09:56.747442	3
350	In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.		Supports	1	1	50	41	181	26	2015-02-05 21:09:56.769927	2015-02-05 21:09:56.769927	4
351	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.		Supports	2	1	51	41	182	26	2015-02-05 21:09:56.790138	2015-02-05 21:09:56.790138	3
352	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.		Supports	2	1	3	11	183	25	2015-02-05 21:09:56.819469	2015-02-05 21:09:56.819469	4
353	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.		Supports	2	1	3	42	184	25	2015-02-05 21:09:56.846346	2015-02-05 21:09:56.846346	4
354	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.		Supports	2	1	3	42	184	25	2015-02-05 21:09:56.86436	2015-02-05 21:09:56.86436	4
355	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.		Supports	2	1	3	30	184	25	2015-02-05 21:09:56.889989	2015-02-05 21:09:56.889989	4
356	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.		Supports	1	2	52	43	185	24	2015-02-05 21:09:56.922112	2015-02-05 21:09:56.922112	3
357	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.		Supports	2	2	3	44	186	21	2015-02-05 21:09:56.949829	2015-02-05 21:09:56.949829	4
358	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.		Supports	2	2	3	44	186	21	2015-02-05 21:09:56.975412	2015-02-05 21:09:56.975412	4
359	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.		Supports	2	2	3	44	186	21	2015-02-05 21:09:57.000741	2015-02-05 21:09:57.000741	4
360	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.		Supports	2	2	3	44	186	21	2015-02-05 21:09:57.029892	2015-02-05 21:09:57.029892	4
361	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.		Supports	3	2	3	44	187	22	2015-02-05 21:09:57.057614	2015-02-05 21:09:57.057614	3
362	Treatment of CCND2-negative cells with 5-Aza-2'-deoxycytidine restored cyclin D2 expression.		Supports	1	2	3	44	187	22	2015-02-05 21:09:57.08094	2015-02-05 21:09:57.08094	2
363	dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.		Supports	1	1	4	6	188	11	2015-02-05 21:09:57.108284	2015-02-05 21:09:57.108284	5
364	dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.		Supports	1	1	53	6	188	11	2015-02-05 21:09:57.125242	2015-02-05 21:09:57.125242	5
365	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as poor prognosis.		Supports	2	4	3	26	189	11	2015-02-05 21:09:57.151503	2015-02-05 21:09:57.151503	5
366	thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.		Supports	3	1	3	14	190	11	2015-02-05 21:09:57.175897	2015-02-05 21:09:57.175897	5
367	thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.		Supports	3	1	3	14	191	11	2015-02-05 21:09:57.201752	2015-02-05 21:09:57.201752	5
368	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.		Does Not Support	2	1	3	14	192	11	2015-02-05 21:09:57.230208	2015-02-05 21:09:57.230208	5
369	COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.		Supports	1	3	23	26	193	13	2015-02-05 21:09:57.337205	2015-02-05 21:09:57.337205	4
370	BRAF mutations are associated with melanoma arising in nonchronic sun damage skin and with superficial spreading melanoma		Supports	3	1	3	6	194	16	2015-02-05 21:09:57.362259	2015-02-05 21:09:57.362259	4
371	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)		Supports	1	1	27	26	195	16	2015-02-05 21:09:57.384986	2015-02-05 21:09:57.384986	3
372	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation		Supports	1	2	54	28	196	11	2015-02-05 21:09:57.411498	2015-02-05 21:09:57.411498	3
373	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation		Supports	1	2	55	28	196	11	2015-02-05 21:09:57.427688	2015-02-05 21:09:57.427688	3
374	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	2	26	28	197	11	2015-02-05 21:09:57.450319	2015-02-05 21:09:57.450319	3
375	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	2	27	28	197	11	2015-02-05 21:09:57.462199	2015-02-05 21:09:57.462199	3
376	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.		Supports	1	2	14	28	197	11	2015-02-05 21:09:57.475538	2015-02-05 21:09:57.475538	3
377	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone		Supports	1	1	4	6	198	16	2015-02-05 21:09:57.500017	2015-02-05 21:09:57.500017	3
378	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone		Supports	1	1	53	6	198	16	2015-02-05 21:09:57.513263	2015-02-05 21:09:57.513263	3
379	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF		Supports	1	1	26	26	105	11	2015-02-05 21:09:57.535675	2015-02-05 21:09:57.535675	3
380	BRAF V600E is correlated with shorter Disease-Free and Overall Survival in a Spanish cohort of Melanoma patients.		Supports	2	1	3	6	199	11	2015-02-05 21:09:57.561923	2015-02-05 21:09:57.561923	3
381	BRAF V600E is shown to be associated with the tall-cell variant of PTC		Supports	3	1	3	14	200	11	2015-02-05 21:09:57.584758	2015-02-05 21:09:57.584758	3
382	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).		Supports	2	1	3	14	200	11	2015-02-05 21:09:57.60775	2015-02-05 21:09:57.60775	3
383	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.		Supports	2	1	3	14	201	11	2015-02-05 21:09:57.630963	2015-02-05 21:09:57.630963	3
384	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.		Supports	2	1	3	14	200	11	2015-02-05 21:09:57.649963	2015-02-05 21:09:57.649963	3
385	BRAF V600E resistance is triggered by ERK dependent or independent pathways.		Supports	1	2	3	6	202	11	2015-02-05 21:09:57.67058	2015-02-05 21:09:57.67058	3
386	treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g. dabrafenib) appears to be beneficial, and better than vice versa.		Supports	1	1	56	6	203	11	2015-02-05 21:09:57.695747	2015-02-05 21:09:57.695747	2
387	treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g. dabrafenib) appears to be beneficial, and better than vice versa.		Supports	1	1	4	6	203	11	2015-02-05 21:09:57.706668	2015-02-05 21:09:57.706668	2
388	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.		Does Not Support	1	1	57	6	204	11	2015-02-05 21:09:57.733041	2015-02-05 21:09:57.733041	2
389	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.		Does Not Support	1	1	32	6	204	11	2015-02-05 21:09:57.746932	2015-02-05 21:09:57.746932	2
390	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	2	58	26	205	11	2015-02-05 21:09:57.772321	2015-02-05 21:09:57.772321	2
391	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	2	29	26	205	11	2015-02-05 21:09:57.782557	2015-02-05 21:09:57.782557	2
392	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.		Supports	1	2	59	26	205	11	2015-02-05 21:09:57.794447	2015-02-05 21:09:57.794447	2
393	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy		Supports	1	2	55	28	206	11	2015-02-05 21:09:57.818552	2015-02-05 21:09:57.818552	2
394	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy		Supports	1	2	60	28	206	11	2015-02-05 21:09:57.831966	2015-02-05 21:09:57.831966	2
395	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.		Supports	1	1	4	3	134	11	2015-02-05 21:09:57.855216	2015-02-05 21:09:57.855216	2
396	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation		Supports	1	2	29	26	205	11	2015-02-05 21:09:57.874864	2015-02-05 21:09:57.874864	2
397	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.		Supports	3	1	3	11	207	19	2015-02-05 21:09:57.898598	2015-02-05 21:09:57.898598	4
398	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.		Supports	2	1	3	11	207	19	2015-02-05 21:09:57.922508	2015-02-05 21:09:57.922508	4
399	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.		Supports	2	1	3	45	208	19	2015-02-05 21:09:57.946498	2015-02-05 21:09:57.946498	4
400	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.		Supports	2	1	3	46	209	19	2015-02-05 21:09:57.970945	2015-02-05 21:09:57.970945	4
401	Inceased copy number of CCND1 is associated with poorer overall survival.		Supports	2	1	3	3	210	17	2015-02-05 21:09:57.98925	2015-02-05 21:09:57.98925	3
403	Three studies have found no significant survival impact for CCND1 in lung cancer.		Supports	2	1	3	3	210	18	2015-02-05 21:09:58.038234	2015-02-05 21:09:58.038234	3
404	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells in vitro		Supports	1	2	53	3	211	9	2015-02-05 21:09:58.063396	2015-02-05 21:09:58.063396	3
405	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells in vitro		Supports	1	2	14	3	211	9	2015-02-05 21:09:58.08093	2015-02-05 21:09:58.08093	3
406	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.		Supports	1	1	14	3	211	9	2015-02-05 21:09:58.104114	2015-02-05 21:09:58.104114	2
407	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition		Supports	1	2	61	47	212	7	2015-02-05 21:09:58.143905	2015-02-05 21:09:58.143905	4
408	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.		Supports	1	2	62	47	213	7	2015-02-05 21:09:58.171436	2015-02-05 21:09:58.171436	3
409	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation		Supports	1	2	62	3	213	6	2015-02-05 21:09:58.192396	2015-02-05 21:09:58.192396	3
410	TAE684 inhibits growth of Ba/F3 cells expressing ALK with the R1275Q mutation at a higher IC50 than those expressing F1174L and the SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684		Does Not Support	1	2	61	47	212	8	2015-02-05 21:09:58.215376	2015-02-05 21:09:58.215376	3
411	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.		Supports	1	2	19	48	84	7	2015-02-05 21:09:58.244692	2015-02-05 21:09:58.244692	3
412	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.		Supports	1	2	19	47	214	7	2015-02-05 21:09:58.273987	2015-02-05 21:09:58.273987	3
413	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.		Supports	1	2	19	47	215	7	2015-02-05 21:09:58.301451	2015-02-05 21:09:58.301451	3
414	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.		Does Not Support	1	1	63	49	216	3	2015-02-05 21:09:58.338215	2015-02-05 21:09:58.338215	3
415	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.		Supports	1	4	64	7	217	1	2015-02-05 21:09:58.367997	2015-02-05 21:09:58.367997	5
416	The clinical use of imatinib in patients with BCR-ABL fusion has drastic sensitivity to the drug.		Supports	1	4	10	9	72	1	2015-02-05 21:09:58.392738	2015-02-05 21:09:58.392738	5
417	BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens."		Supports	2	4	10	9	72	1	2015-02-05 21:09:58.416753	2015-02-05 21:09:58.416753	5
418	The use of second-generation BCR-ABL targeted therapies has shown signifcantly greater response in patients resistant to imatinib.		Supports	1	1	13	9	72	1	2015-02-05 21:09:58.446492	2015-02-05 21:09:58.446492	4
419	The use of second-generation BCR-ABL targeted therapies has shown signifcantly greater response in patients resistant to imatinib.		Supports	1	1	15	9	72	1	2015-02-05 21:09:58.461908	2015-02-05 21:09:58.461908	4
420	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.		Supports	3	4	3	9	72	1	2015-02-05 21:09:58.483581	2015-02-05 21:09:58.483581	4
421	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.		Supports	1	1	10	7	217	1	2015-02-05 21:09:58.504635	2015-02-05 21:09:58.504635	3
422	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.		Does Not Support	3	2	3	9	218	1	2015-02-05 21:09:58.528493	2015-02-05 21:09:58.528493	2
\.


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 422, true);


--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 47, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-02-05 21:09:34.30469	2015-02-05 21:09:34.30469
1	2	2015-02-05 21:09:34.308831	2015-02-05 21:09:34.308831
2	3	2015-02-05 21:09:34.529055	2015-02-05 21:09:34.529055
2	4	2015-02-05 21:09:34.533168	2015-02-05 21:09:34.533168
3	5	2015-02-05 21:09:34.73714	2015-02-05 21:09:34.73714
3	6	2015-02-05 21:09:34.74185	2015-02-05 21:09:34.74185
4	7	2015-02-05 21:09:35.037412	2015-02-05 21:09:35.037412
4	8	2015-02-05 21:09:35.042411	2015-02-05 21:09:35.042411
5	9	2015-02-05 21:09:35.246225	2015-02-05 21:09:35.246225
5	10	2015-02-05 21:09:35.250609	2015-02-05 21:09:35.250609
6	11	2015-02-05 21:09:35.470035	2015-02-05 21:09:35.470035
6	12	2015-02-05 21:09:35.474389	2015-02-05 21:09:35.474389
7	11	2015-02-05 21:09:35.674136	2015-02-05 21:09:35.674136
7	12	2015-02-05 21:09:35.677278	2015-02-05 21:09:35.677278
8	11	2015-02-05 21:09:35.884379	2015-02-05 21:09:35.884379
8	12	2015-02-05 21:09:35.886533	2015-02-05 21:09:35.886533
9	13	2015-02-05 21:09:36.06459	2015-02-05 21:09:36.06459
10	14	2015-02-05 21:09:36.243625	2015-02-05 21:09:36.243625
10	15	2015-02-05 21:09:36.246909	2015-02-05 21:09:36.246909
11	14	2015-02-05 21:09:36.434416	2015-02-05 21:09:36.434416
11	15	2015-02-05 21:09:36.436714	2015-02-05 21:09:36.436714
12	16	2015-02-05 21:09:36.639309	2015-02-05 21:09:36.639309
12	17	2015-02-05 21:09:36.642968	2015-02-05 21:09:36.642968
13	18	2015-02-05 21:09:36.853688	2015-02-05 21:09:36.853688
13	19	2015-02-05 21:09:36.858031	2015-02-05 21:09:36.858031
13	20	2015-02-05 21:09:36.863477	2015-02-05 21:09:36.863477
14	21	2015-02-05 21:09:37.046269	2015-02-05 21:09:37.046269
15	21	2015-02-05 21:09:37.307738	2015-02-05 21:09:37.307738
16	22	2015-02-05 21:09:37.494121	2015-02-05 21:09:37.494121
16	23	2015-02-05 21:09:37.498674	2015-02-05 21:09:37.498674
17	24	2015-02-05 21:09:37.719569	2015-02-05 21:09:37.719569
17	25	2015-02-05 21:09:37.722881	2015-02-05 21:09:37.722881
18	26	2015-02-05 21:09:37.921781	2015-02-05 21:09:37.921781
18	27	2015-02-05 21:09:37.925633	2015-02-05 21:09:37.925633
19	28	2015-02-05 21:09:38.144666	2015-02-05 21:09:38.144666
19	29	2015-02-05 21:09:38.148987	2015-02-05 21:09:38.148987
20	30	2015-02-05 21:09:38.339398	2015-02-05 21:09:38.339398
20	31	2015-02-05 21:09:38.342617	2015-02-05 21:09:38.342617
21	30	2015-02-05 21:09:38.587644	2015-02-05 21:09:38.587644
21	31	2015-02-05 21:09:38.590354	2015-02-05 21:09:38.590354
22	32	2015-02-05 21:09:38.823377	2015-02-05 21:09:38.823377
22	18	2015-02-05 21:09:38.82554	2015-02-05 21:09:38.82554
33	33	2015-02-05 21:09:41.112372	2015-02-05 21:09:41.112372
33	18	2015-02-05 21:09:41.114638	2015-02-05 21:09:41.114638
40	34	2015-02-05 21:09:42.538448	2015-02-05 21:09:42.538448
40	35	2015-02-05 21:09:42.546274	2015-02-05 21:09:42.546274
42	36	2015-02-05 21:09:42.929199	2015-02-05 21:09:42.929199
42	37	2015-02-05 21:09:42.933164	2015-02-05 21:09:42.933164
46	38	2015-02-05 21:09:45.159607	2015-02-05 21:09:45.159607
46	39	2015-02-05 21:09:45.163516	2015-02-05 21:09:45.163516
\.


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20150205210725
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 218, true);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
1	TSC Loss	\N	2015-02-05 21:09:48.040624	2015-02-05 21:09:48.040624
2	PTEN Loss-of-Function	\N	2015-02-05 21:09:48.178126	2015-02-05 21:09:48.178126
3	Other V600's	\N	2015-02-05 21:09:48.260017	2015-02-05 21:09:48.260017
4	NPM1 exon 12	\N	2015-02-05 21:09:48.39563	2015-02-05 21:09:48.39563
5	NF1 loss	\N	2015-02-05 21:09:49.511159	2015-02-05 21:09:49.511159
6	Motesanib Resistance	\N	2015-02-05 21:09:49.569146	2015-02-05 21:09:49.569146
7	Imatinib Resistance	\N	2015-02-05 21:09:49.650043	2015-02-05 21:09:49.650043
8	FGFR fusions	\N	2015-02-05 21:09:49.699729	2015-02-05 21:09:49.699729
9	Exon 17	\N	2015-02-05 21:09:50.163802	2015-02-05 21:09:50.163802
10	Exon 11	\N	2015-02-05 21:09:50.222257	2015-02-05 21:09:50.222257
11	ESR1 Ligand-Binding Domain	\N	2015-02-05 21:09:50.438488	2015-02-05 21:09:50.438488
12	DNAJB1 Fusions	\N	2015-02-05 21:09:50.763064	2015-02-05 21:09:50.763064
13	Crizotinib Resistance	\N	2015-02-05 21:09:50.817643	2015-02-05 21:09:50.817643
14	ALK Fusions	\N	2015-02-05 21:09:50.849005	2015-02-05 21:09:50.849005
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
131	1	2015-02-05 21:09:48.051481	2015-02-05 21:09:48.051481
130	1	2015-02-05 21:09:48.078194	2015-02-05 21:09:48.078194
129	1	2015-02-05 21:09:48.107982	2015-02-05 21:09:48.107982
129	1	2015-02-05 21:09:48.141002	2015-02-05 21:09:48.141002
115	2	2015-02-05 21:09:48.18026	2015-02-05 21:09:48.18026
115	2	2015-02-05 21:09:48.222993	2015-02-05 21:09:48.222993
10	3	2015-02-05 21:09:48.262077	2015-02-05 21:09:48.262077
14	3	2015-02-05 21:09:48.295519	2015-02-05 21:09:48.295519
15	3	2015-02-05 21:09:48.32898	2015-02-05 21:09:48.32898
12	3	2015-02-05 21:09:48.362809	2015-02-05 21:09:48.362809
92	4	2015-02-05 21:09:48.398082	2015-02-05 21:09:48.398082
92	4	2015-02-05 21:09:48.439795	2015-02-05 21:09:48.439795
92	4	2015-02-05 21:09:48.486282	2015-02-05 21:09:48.486282
92	4	2015-02-05 21:09:48.514772	2015-02-05 21:09:48.514772
92	4	2015-02-05 21:09:48.547292	2015-02-05 21:09:48.547292
92	4	2015-02-05 21:09:48.574924	2015-02-05 21:09:48.574924
92	4	2015-02-05 21:09:48.606141	2015-02-05 21:09:48.606141
92	4	2015-02-05 21:09:48.629667	2015-02-05 21:09:48.629667
92	4	2015-02-05 21:09:48.661119	2015-02-05 21:09:48.661119
92	4	2015-02-05 21:09:48.692159	2015-02-05 21:09:48.692159
92	4	2015-02-05 21:09:48.719712	2015-02-05 21:09:48.719712
93	4	2015-02-05 21:09:48.743022	2015-02-05 21:09:48.743022
93	4	2015-02-05 21:09:48.768853	2015-02-05 21:09:48.768853
92	4	2015-02-05 21:09:48.814201	2015-02-05 21:09:48.814201
92	4	2015-02-05 21:09:48.849284	2015-02-05 21:09:48.849284
92	4	2015-02-05 21:09:48.880322	2015-02-05 21:09:48.880322
92	4	2015-02-05 21:09:48.919369	2015-02-05 21:09:48.919369
92	4	2015-02-05 21:09:48.953454	2015-02-05 21:09:48.953454
92	4	2015-02-05 21:09:48.983747	2015-02-05 21:09:48.983747
92	4	2015-02-05 21:09:49.011014	2015-02-05 21:09:49.011014
92	4	2015-02-05 21:09:49.038048	2015-02-05 21:09:49.038048
92	4	2015-02-05 21:09:49.064541	2015-02-05 21:09:49.064541
92	4	2015-02-05 21:09:49.096535	2015-02-05 21:09:49.096535
92	4	2015-02-05 21:09:49.127383	2015-02-05 21:09:49.127383
92	4	2015-02-05 21:09:49.156466	2015-02-05 21:09:49.156466
92	4	2015-02-05 21:09:49.186221	2015-02-05 21:09:49.186221
92	4	2015-02-05 21:09:49.21046	2015-02-05 21:09:49.21046
93	4	2015-02-05 21:09:49.237022	2015-02-05 21:09:49.237022
92	4	2015-02-05 21:09:49.260321	2015-02-05 21:09:49.260321
92	4	2015-02-05 21:09:49.288327	2015-02-05 21:09:49.288327
92	4	2015-02-05 21:09:49.32181	2015-02-05 21:09:49.32181
92	4	2015-02-05 21:09:49.344564	2015-02-05 21:09:49.344564
92	4	2015-02-05 21:09:49.370651	2015-02-05 21:09:49.370651
92	4	2015-02-05 21:09:49.39833	2015-02-05 21:09:49.39833
92	4	2015-02-05 21:09:49.428652	2015-02-05 21:09:49.428652
92	4	2015-02-05 21:09:49.459241	2015-02-05 21:09:49.459241
92	4	2015-02-05 21:09:49.486719	2015-02-05 21:09:49.486719
11	5	2015-02-05 21:09:49.513449	2015-02-05 21:09:49.513449
11	5	2015-02-05 21:09:49.5373	2015-02-05 21:09:49.5373
117	6	2015-02-05 21:09:49.573615	2015-02-05 21:09:49.573615
118	6	2015-02-05 21:09:49.607758	2015-02-05 21:09:49.607758
2	7	2015-02-05 21:09:49.652583	2015-02-05 21:09:49.652583
57	8	2015-02-05 21:09:49.702225	2015-02-05 21:09:49.702225
52	8	2015-02-05 21:09:49.739092	2015-02-05 21:09:49.739092
54	8	2015-02-05 21:09:49.767735	2015-02-05 21:09:49.767735
55	8	2015-02-05 21:09:49.791856	2015-02-05 21:09:49.791856
53	8	2015-02-05 21:09:49.814472	2015-02-05 21:09:49.814472
59	8	2015-02-05 21:09:49.842935	2015-02-05 21:09:49.842935
56	8	2015-02-05 21:09:49.873645	2015-02-05 21:09:49.873645
58	8	2015-02-05 21:09:49.90348	2015-02-05 21:09:49.90348
60	8	2015-02-05 21:09:49.941134	2015-02-05 21:09:49.941134
60	8	2015-02-05 21:09:49.975773	2015-02-05 21:09:49.975773
60	8	2015-02-05 21:09:50.017062	2015-02-05 21:09:50.017062
60	8	2015-02-05 21:09:50.058992	2015-02-05 21:09:50.058992
60	8	2015-02-05 21:09:50.099105	2015-02-05 21:09:50.099105
60	8	2015-02-05 21:09:50.138942	2015-02-05 21:09:50.138942
72	9	2015-02-05 21:09:50.166113	2015-02-05 21:09:50.166113
78	10	2015-02-05 21:09:50.22428	2015-02-05 21:09:50.22428
78	10	2015-02-05 21:09:50.280698	2015-02-05 21:09:50.280698
79	10	2015-02-05 21:09:50.31053	2015-02-05 21:09:50.31053
78	10	2015-02-05 21:09:50.340337	2015-02-05 21:09:50.340337
79	10	2015-02-05 21:09:50.368759	2015-02-05 21:09:50.368759
78	10	2015-02-05 21:09:50.39776	2015-02-05 21:09:50.39776
47	11	2015-02-05 21:09:50.440505	2015-02-05 21:09:50.440505
48	11	2015-02-05 21:09:50.478031	2015-02-05 21:09:50.478031
49	11	2015-02-05 21:09:50.513727	2015-02-05 21:09:50.513727
50	11	2015-02-05 21:09:50.550705	2015-02-05 21:09:50.550705
51	11	2015-02-05 21:09:50.589588	2015-02-05 21:09:50.589588
47	11	2015-02-05 21:09:50.614587	2015-02-05 21:09:50.614587
48	11	2015-02-05 21:09:50.64189	2015-02-05 21:09:50.64189
49	11	2015-02-05 21:09:50.6645	2015-02-05 21:09:50.6645
50	11	2015-02-05 21:09:50.695006	2015-02-05 21:09:50.695006
51	11	2015-02-05 21:09:50.736908	2015-02-05 21:09:50.736908
114	12	2015-02-05 21:09:50.765017	2015-02-05 21:09:50.765017
32	12	2015-02-05 21:09:50.793008	2015-02-05 21:09:50.793008
7	13	2015-02-05 21:09:50.820604	2015-02-05 21:09:50.820604
6	14	2015-02-05 21:09:50.851183	2015-02-05 21:09:50.851183
6	13	2015-02-05 21:09:50.853894	2015-02-05 21:09:50.853894
5	14	2015-02-05 21:09:50.880583	2015-02-05 21:09:50.880583
5	13	2015-02-05 21:09:50.884457	2015-02-05 21:09:50.884457
4	14	2015-02-05 21:09:50.906917	2015-02-05 21:09:50.906917
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 14, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 135, true);


--
-- PostgreSQL database dump complete
--

