--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

SET search_path = public, pg_catalog;

--
-- Data for Name: audits; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY audits (id, auditable_id, auditable_type, associated_id, associated_type, user_id, user_type, username, action, audited_changes, version, comment, remote_address, request_uuid, created_at) FROM stdin;
1	1	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 238\nname: ALK\ndescription: ALK amplifications, fusions and mutations have been shown to be driving\n  events in non-small cell lung cancer. While crizontinib has demonstrated efficacy\n  in treating the amplification, mutations in ALK have been shown to confer resistance\n  to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied\n  success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been\n  shown to be cytostatic in ALK-altered cell lines.\nofficial_name: anaplastic lymphoma receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	b1e32a53-713c-469f-82f9-a4dc0eb431a4	2015-02-06 19:00:57.522118
2	2	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 207\nname: AKT1\ndescription: AKT1, also referred to as protein kinase B, has been known as an oncogene\n  for quite some time. AKT activation relies on the PI3K pathway, and is recognized\n  as a critical node in the pathway. The E17 hotspot is the most characterized of\n  AKT1 mutations, and has been shown to result in activation of the protein. Mutations\n  in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors\n  in vitro.\nofficial_name: v-akt murine thymoma viral oncogene homolog 1\nclinical_description: \n	1	\N	\N	4180f4d4-7a21-4449-b705-62a3c32fc8b9	2015-02-06 19:00:57.729794
3	3	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 369\nname: ARAF\ndescription: ARAF has recently become increasingly considered for its oncogenic potential.\n  Its potential as a target for informing clinical action was demonstrated by a single\n  case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated\n  with sorafenib, acheived near-complete clinical remission. This finding has brought\n  new focus on ARAF as a marker that should be assayed for in cancer treatment.\nofficial_name: A-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	44bf0b8d-f484-45d7-827b-efbcc962d1e1	2015-02-06 19:00:57.911454
4	4	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 25\nname: ABL1\ndescription: ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein\n  that has become a signature of chronic myeloid leukemia. Treatment of cells harboring\n  this fusion has shown sensitivity to imatinib, greatly improving the prognostic\n  outlook of the disease. However, mutations in this gene have been shown to confer\n  resistance to imatinib. In these resistance cases, second-generation tyrosine kinase\n  inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are\n  currently undergoing clinical trials for treating acquired resistance in CML.\nofficial_name: ABL proto-oncogene 1, non-receptor tyrosine kinase\nclinical_description: \n	1	\N	\N	6f5aad90-47d6-42ab-bac1-6931d06a3583	2015-02-06 19:00:58.090882
5	5	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 673\nname: BRAF\ndescription: BRAF mutations are found to be recurrent in many cancer types. Of these,\n  the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E\n  has been determined to be an activating mutation, and cells that harbor it, along\n  with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is\n  also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK\n  inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations\n  have also been shown to be correlated with poor prognosis in many cancer types,\n  there is at least one study that questions this conclusion in papillary thyroid\n  cancer.\nofficial_name: B-Raf proto-oncogene, serine/threonine kinase\nclinical_description: \n	1	\N	\N	a2ffa47d-a45f-4980-ad29-550ac0a218c6	2015-02-06 19:00:58.307337
6	6	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 595\nname: CCND1\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D1\nclinical_description: \n	1	\N	\N	2961a110-1856-457e-8981-3bb12a59d79b	2015-02-06 19:00:58.491018
7	7	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 894\nname: CCND2\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin\n  D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been\n  shown to lead to poorer outcomes in a number of studies, and currently there are\n  no FDA-approved targeted therapies.\nofficial_name: cyclin D2\nclinical_description: \n	1	\N	\N	131d526f-1902-4b35-bab0-4f270ed00924	2015-02-06 19:00:58.667459
8	8	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 896\nname: CCND3\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared\n  to the amplifcations and overexpressions of the other cyclin D's. In a mouse study,\n  the targeted therapeutic palbociclib significantly increased the median survival\n  of the cyclin D3 knockouts.\nofficial_name: cyclin D3\nclinical_description: \n	1	\N	\N	a7709d38-c40d-4dd6-8081-9e130822123a	2015-02-06 19:00:58.843406
9	9	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 898\nname: CCNE1\ndescription: Cyclin E, while currenly not as widely implicated as it's counterparts,\n  cyclin D, has been shown to implicated in various carcinomas, including breast,\n  gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification\n  or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma,\n  and these measurements are correlated with later stage disease. It has also been\n  shown, in lung cancer specifically, that neoplastic cells with higher levels of\n  the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.\nofficial_name: cyclin E1\nclinical_description: \n	1	\N	\N	5bf36b95-db2d-49da-bcf5-bd0320ae298c	2015-02-06 19:00:59.092224
10	10	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1021\nname: CDK6\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 6\nclinical_description: \n	1	\N	\N	360a37b3-e467-43cc-9ae2-db0e5d20e64e	2015-02-06 19:00:59.288625
37	37	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5371\nname: PML\ndescription: N/A\nofficial_name: promyelocytic leukemia\nclinical_description: \n	1	\N	\N	5e8c9190-4e7d-42c6-b372-16da009ecb3d	2015-02-06 19:01:04.49161
11	11	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1019\nname: CDK4\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\nofficial_name: cyclin-dependent kinase 4\nclinical_description: \n	1	\N	\N	a14f811d-9bdd-4499-8c6b-ed4667153d57	2015-02-06 19:00:59.469362
12	12	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1029\nname: CDKN2A\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the promoter\n  region for the gene. While no targeted therapeutic has been engaged in clinical\n  trials, the prognostic impact has been studied by a number of meta-analyses. The\n  prognostic impact of promoter hypermethylation has been relatively ambiguous. Many\n  studies have shown significant p-values suggesting poorer prognostic outcome for\n  patients with hypermethylation in colorectal, liver, and younger lung cancer patients.\n  This being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\nofficial_name: cyclin-dependent kinase inhibitor 2A\nclinical_description: \n	1	\N	\N	db671b24-9f10-4944-bfa1-d85cdb03d432	2015-02-06 19:00:59.669471
13	13	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1050\nname: CEBPA\ndescription: "'AML with mutated CEBPA' is a provisional entity in the WHO classification\n  of acute myeloid leukemia (AML) and is recommended to be tested for in patients\n  with AML. CEBPA mutations are particularly associated with cytogenetically normal\n  AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation\n  in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha\n  protein while C-terminal mutations reduce the DNA-binding potential of this transcription\n  factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1\n  and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations\n  may have prognostic implications."\nofficial_name: CCAAT/enhancer binding protein (C/EBP), alpha\nclinical_description: \n	1	\N	\N	64855346-bd3f-4272-acd7-9053a0b19c4a	2015-02-06 19:00:59.857131
14	14	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3337\nname: DNAJB1\ndescription: DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating\n  p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1\n  to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. A\n  study has found this fusion in every FL-HCC sample sequenced, and in vitro assays\n  have shown that the fusion retains kinase function.\nofficial_name: DnaJ (Hsp40) homolog, subfamily B, member 1\nclinical_description: \n	1	\N	\N	616d8f55-59fa-4e70-916b-1192dba2336d	2015-02-06 19:01:00.042705
15	15	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5566\nname: PRKACA\ndescription: PRKACA has been studied in breast cancer and has been found to mediate\n  resistance to HER2 targeted therapies. It has also been found to contain a mutation\n  hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular\n  fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be\n  a diagnostic marker. A study has found this fusion in every sample sequenced, and\n  in vitro assays have shown that the fusion retains kinase function.\nofficial_name: protein kinase, cAMP-dependent, catalytic, alpha\nclinical_description: \n	1	\N	\N	4366cccb-5199-4c2f-8940-7bd52d78af3a	2015-02-06 19:01:00.26086
16	16	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1788\nname: DNMT3A\ndescription: DNMT3A is one of several epigenetic modifiers identified as recurrently\n  mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically\n  normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant\n  negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers.\n  AML patient bone marrow harboring R882 mutations were similarly demonstrated to\n  be hypomethylated compared to patients with wildtype DNMT3A. These studies also\n  indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner\n  from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes\n  and treatment protocols may need to be considered for these two classes of DNMT3A\n  mutations.\nofficial_name: DNA (cytosine-5-)-methyltransferase 3 alpha\nclinical_description: \n	1	\N	\N	e71f3079-1a35-4f2d-8b87-15c1c7a8e293	2015-02-06 19:01:00.461867
17	17	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1956\nname: EGFR\ndescription: EGFR is widely recognized for its importance in cancer. Amplification\n  and mutations have been shown to be driving events in many cancer types. Its role\n  in non-small cell lung cancer has spurred many research and drug development efforts.\n  Tyrosine kinase inhibitors have show efficacy in EGFR amplfied tumors, most notably\n  gefitinibb and erlotinib. mutations in EGFR have been shown to confer resistance\n  to these drugs however, and the variant T790M has been functionally characterized\n  as a resistance marker for both of these drugs. The later generation TKI's have\n  seen some success in treating these resistant cases, and targeted sequencing of\n  the EGFR locus has been a common practice in non-small cell lung cancer.\nofficial_name: epidermal growth factor receptor\nclinical_description: \n	1	\N	\N	ac00de6b-f08c-4f0e-bc7e-10012cc6edd7	2015-02-06 19:01:00.65669
18	18	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2064\nname: ERBB2\ndescription: ERBB2, commonly referred to as HER2, is seen to be amplified in HER2-positive\n  breast cancer, and is treated in a separate manner than the other subtypes of breast\n  cancer. Apart from being amplified, ERBB2 activating mutations have been shown to\n  have clinical importance in HER2-negative breast cancer. These mutations have shown\n  senitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance\n  of clinical sequencing efforts in treating breast cancer.\nofficial_name: erb-b2 receptor tyrosine kinase 2\nclinical_description: \n	1	\N	\N	f78d3e86-6018-4d57-b533-97258c38667f	2015-02-06 19:01:00.841693
19	19	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2099\nname: ESR1\ndescription: ESR1 has been a focus in breast cancer for quite some time, and has shown\n  significance in endometrial, ovarian and other cancer types as well. ER-positive\n  breast cancer that is resistant to hormone therapy has instigated clinical sequencing\n  efforts to shed light on the mechanisms of this resistance. A number of mutations\n  in the ligand binding domain of the protein have been implicated in hormone resistance\n  and anti-estrogen therapies. This has spurred efforts to develop therapeutics that\n  act to degrade the protein, rather than act as an antagonist. These agents are currently\n  in clinical trials and have seen some success, highlighting the importance of sequencing\n  efforts in treating breast cancer.\nofficial_name: estrogen receptor 1\nclinical_description: \n	1	\N	\N	6c1d0f57-0e5e-4da8-ba28-4cba0422cff5	2015-02-06 19:01:01.076055
38	38	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5914\nname: RARa\ndescription: N/A\nofficial_name: retinoic acid receptor, alpha\nclinical_description: \n	1	\N	\N	bf4f27d0-7951-455a-b69f-7b6b085a8bde	2015-02-06 19:01:04.6794
39	39	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5728\nname: PTEN\ndescription: N/A\nofficial_name: phosphatase and tensin homolog\nclinical_description: \n	1	\N	\N	6b6ffc17-ae52-4041-b453-5539100d3a0b	2015-02-06 19:01:04.878441
20	20	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2263\nname: FGFR2\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. The active proteins have the potential to activate\n  the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which\n  FGFR can be misregulated vary between cancers. Amplification of the receptors has\n  been observed in lung and breast cancers, coding mutations and deletions have been\n  seen in many cancers, and more recently, FGFR fusions have been demonstrated to\n  have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 2\nclinical_description: \n	1	\N	\N	cd6b1cf1-e28b-4fe0-b729-621b77c671c0	2015-02-06 19:01:01.296295
21	21	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2261\nname: FGFR3\ndescription: The FGFR proteins are involved in a wide array of pathways known to play\n  a signficant role in cancer. The active proteins have the potential to activate\n  the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which\n  FGFR can be misregulated vary between cancers. Amplification of the receptors has\n  been observed in lung and breast cancers, coding mutations and deletions have been\n  seen in many cancers, and more recently, FGFR fusions have been demonstrated to\n  have oncogenic potential across multiple cancer types. The targeted therapeutics\n  ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR\n  signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially\n  in lung cancer patients.\nofficial_name: fibroblast growth factor receptor 3\nclinical_description: \n	1	\N	\N	a652ed7f-e64b-4c1e-b81e-9e68421b9919	2015-02-06 19:01:01.475671
22	22	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2322\nname: FLT3\ndescription: FLT3 is an important cytokine receptor involved in normal hematopoiesis.\n  Mutations in this gene are common in acute myeloid leukemia (AML) and screening\n  for mutations in this gene has been recommended by the World Health Organization\n  in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML).\n  FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated\n  with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have\n  been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD\n  mutations are still up for debate.\nofficial_name: fms-related tyrosine kinase 3\nclinical_description: \n	1	\N	\N	f5144ef2-4353-4c35-b622-872e5996f9bb	2015-02-06 19:01:01.701684
23	23	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 2624\nname: GATA2\ndescription: N/A\nofficial_name: GATA binding protein 2\nclinical_description: \n	1	\N	\N	2050d5f3-46ba-42b4-97bd-54f0d34e29cc	2015-02-06 19:01:01.867708
24	24	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3417\nname: IDH1\ndescription: N/A\nofficial_name: isocitrate dehydrogenase 1 (NADP+), soluble\nclinical_description: \n	1	\N	\N	03183847-9088-4f7e-b959-5964d7e06863	2015-02-06 19:01:02.0365
25	25	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3418\nname: IDH2\ndescription: N/A\nofficial_name: isocitrate dehydrogenase 2 (NADP+), mitochondrial\nclinical_description: \n	1	\N	\N	92a162a9-5367-401d-8cfe-38f55495d792	2015-02-06 19:01:02.205987
26	26	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3717\nname: JAK2\ndescription: JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative\n  diseases and cancers. The mutation V617F is the most clinically relevant variant,\n  and is seen in around half of myeloproliferative disorders.\nofficial_name: Janus kinase 2\nclinical_description: \n	1	\N	\N	a73fc5f0-bf5e-4156-a21c-6440da3de1b8	2015-02-06 19:01:02.402121
27	27	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3815\nname: KIT\ndescription: N/A\nofficial_name: v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	c49617b9-5796-4f5b-9e67-8e01d976796f	2015-02-06 19:01:02.567617
28	28	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 3845\nname: KRAS\ndescription: N/A\nofficial_name: Kirsten rat sarcoma viral oncogene homolog\nclinical_description: \n	1	\N	\N	1a496214-5382-49e9-bcd1-2f29c2200e4c	2015-02-06 19:01:02.760926
29	29	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5604\nname: MAP2K1\ndescription: N/A\nofficial_name: mitogen-activated protein kinase kinase 1\nclinical_description: \n	1	\N	\N	f119fb18-64d7-4813-a403-9a6d3bd9bc1c	2015-02-06 19:01:02.945216
30	30	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4209\nname: MEF2D\ndescription: N/A\nofficial_name: myocyte enhancer factor 2D\nclinical_description: \n	1	\N	\N	79fe0089-2b53-4016-89bc-dc3f0ce1c1d8	2015-02-06 19:01:03.115488
31	31	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 1436\nname: CSF1R\ndescription: N/A\nofficial_name: colony stimulating factor 1 receptor\nclinical_description: \n	1	\N	\N	4157ed4f-c26d-43a9-906b-7c3572e6adcb	2015-02-06 19:01:03.283841
32	32	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4255\nname: MGMT\ndescription: N/A\nofficial_name: O-6-methylguanine-DNA methyltransferase\nclinical_description: \n	1	\N	\N	e612bc0a-ae02-44c5-9e0e-cd27c903955f	2015-02-06 19:01:03.521496
33	33	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4869\nname: NPM1\ndescription: "'AML with mutated NPM1' is provisional entity in the WHO classification\n  of AML and is recommended to be tested for in patients with cytogenetically normal\n  AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they\n  have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently\n  W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations\n  in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing\n  the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia,\n  indicating it may require additional mutations to promote leukemic development."\nofficial_name: nucleophosmin (nucleolar phosphoprotein B23, numatrin)\nclinical_description: \n	1	\N	\N	56c4dcbf-09f8-4b5c-babf-5f593064af59	2015-02-06 19:01:03.780414
34	34	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 4893\nname: NRAS\ndescription: N/A\nofficial_name: neuroblastoma RAS viral (v-ras) oncogene homolog\nclinical_description: \n	1	\N	\N	c164aaa6-317a-4ebb-8b45-e77513ceb4ed	2015-02-06 19:01:03.959143
35	35	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5290\nname: PIK3CA\ndescription: PIK3CA is the most recurrently mutated gene in breast cancer, and has\n  been found to important in a number of cancer types. An integral part of the PI3K\n  pathway, PIK3CA has long been described as an oncogene, with two main hotspots for\n  activating  , the 542/545 region of the helical domain, and the 1047 region of the\n  kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the\n  subject of an immense amount of research and development, and PI3K inhibition has\n  seen some limited success in recent clinical trials. While monotherapies seem to\n  be limited in their potential, there is a recent interest in pursuing PI3K inhibition\n  as part of a combination therapy regiment with inhibition partners including TKI's,\n  MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.\nofficial_name: phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha\nclinical_description: \n	1	\N	\N	76c70eb4-c413-46ec-b2be-4ec07bda9020	2015-02-06 19:01:04.13931
36	36	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5156\nname: PDGFRA\ndescription: N/A\nofficial_name: platelet-derived growth factor receptor, alpha polypeptide\nclinical_description: \n	1	\N	\N	0543e791-a7d1-449e-b72f-4ba5b53cdebb	2015-02-06 19:01:04.310146
40	40	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 5979\nname: RET\ndescription: RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine\n  kinase receptor and are associated with thyroid cancers. RET point mutations are\n  the most common mutations identified in medullary thyroid cancer (MTC) with germline\n  and somatic mutations in RET associated with hereditary and sporadic forms, respectively.\n  The most common somatic form mutation is M918T (exon 16) and a variety of other\n  mutations effecting exons 10, 11 and 15 have been described. The prognostic significance\n  of these mutations have been hotly debated in the field, however, data suggests\n  that some RET mutation may confer drug resistence. No RET-specific agents are currently\n  clinically available but several promiscuous kinase inhibitors that target RET,\n  among others, have been approved for MTC treatment.\nofficial_name: ret proto-oncogene\nclinical_description: \n	1	\N	\N	2aa30965-edc0-44ac-a26e-4eaf277a0cfc	2015-02-06 19:01:05.124442
41	41	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 861\nname: RUNX1\ndescription: N/A\nofficial_name: runt-related transcription factor 1\nclinical_description: \n	1	\N	\N	e8c5c1af-cdca-4124-9f96-4f2bfe39a411	2015-02-06 19:01:05.289351
42	42	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 23451\nname: SF3B1\ndescription: SF3B1 mutations have been described in several myeloid malignancies,\n  predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies\n  and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has\n  been identified as recurrently mutated in MDS and other malignanices. The mutations\n  affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally\n  deleterious with K700E described as a major hotspot mutation. MDS patients with\n  SF3B1 mutations have been reported to have better overall and event-free survival\n  than their wildtype counterparts. Additionally, these mutations are highly associated\n  with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with\n  ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring\n  sideroblasts).\nofficial_name: splicing factor 3b, subunit 1, 155kDa\nclinical_description: \n	1	\N	\N	f5ba568c-009c-4b87-babe-5ffc21c44dcc	2015-02-06 19:01:05.51437
43	43	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7157\nname: TP53\ndescription: N/A\nofficial_name: tumor protein p53\nclinical_description: \n	1	\N	\N	22f8cddb-4a06-4266-a081-f614a1f4be49	2015-02-06 19:01:05.724447
44	44	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7248\nname: TSC1\ndescription: N/A\nofficial_name: tuberous sclerosis 1\nclinical_description: \n	1	\N	\N	f5b5114c-8708-4d92-99a1-bd28b23e6a30	2015-02-06 19:01:05.897778
45	45	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7249\nname: TSC2\ndescription: N/A\nofficial_name: tuberous sclerosis 2\nclinical_description: \n	1	\N	\N	d7c8996c-320e-4798-967b-cd47cde4761d	2015-02-06 19:01:06.066139
46	46	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7307\nname: U2AF1\ndescription: U2AF1 is one of several spliceosome complex genes frequently mutated\n  in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes\n  (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot\n  mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein.\n  These mutations have been associated with altered splicing patterns in vitro and\n  in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation\n  to secondary acute myeloid leukemia, however, the impact of these mutations on overall\n  survival has been an area of debate.\nofficial_name: U2 small nuclear RNA auxiliary factor 1\nclinical_description: \n	1	\N	\N	30e1edbe-a66f-4e06-bbb4-9557ce46a3da	2015-02-06 19:01:06.247183
47	47	Gene	\N	\N	\N	\N	\N	create	---\nentrez_id: 7490\nname: WT1\ndescription: N/A\nofficial_name: Wilms tumor 1\nclinical_description: \n	1	\N	\N	1c6464f1-2f84-436b-943c-9affd431a268	2015-02-06 19:01:06.427009
48	1	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL\ndescription: The BCR-ABL fusion protein, commonly referred to as the philadelphia\n  chromosome, is the most well-studied fusion gene in cancer. It has widely been considered\n  the initiating event in chronic myelogenous leukemia (CML), but despite its ability\n  initiate disease in mice, its status an initiating mutation is in dispute. In what\n  is commonly used as the poster-child for targeted therapeutics, the development\n  and use of imatinib in the clinic has led to profound improvements in the prognosis\n  of the disease. However, imatinib resistance is still seen in patients with mutations\n  in the ABL kinase domain of the fusion, most notably the T315I variant. In patients\n  resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's\n  (dasatinib and nilotinib) have seen some success in delivering a tumor response.\n	1	\N	\N	1fa2150a-1955-40e3-909d-c0901af58353	2015-02-06 19:01:06.47253
49	2	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 4\nname: BCR-ABL T315I\ndescription: While the efficacy of imatinib has revolutionized chronic myelogenous\n  leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and\n  acquired resistance as a result of seleciton have been seen in a small subset of\n  CML patients. The ABL kinase domain mutation T315I has been shown to be one such\n  mutation that confers resistance to imatinib. Second generation TKI's (dasatinib\n  and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\n	1	\N	\N	0411e02b-36cf-48b8-aea5-d54abbcf1060	2015-02-06 19:01:06.482776
50	3	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 2\nname: E17K\ndescription: AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal,\n  lung, and ovarian cancer. It has been convincingly shown to be an activating mutation\n  resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation\n  decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and\n  other AKT1 mutations, are the subject of much research and development for therapeutics.\n	1	\N	\N	a3d130d5-99b4-46ee-8e9c-e2c781d7440e	2015-02-06 19:01:06.492158
51	4	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK\ndescription: The EML4-ALK fusion has been seen in non-small cell lung cancer, and\n  appears to be an alternative mechanism for ALK activation. Cells with this fusion\n  have been shown to be sensitive to the ALK inhibitor crizotinib.\n	1	\N	\N	74f1d5e9-aee7-43a4-a6fe-f968ce6dc310	2015-02-06 19:01:06.501099
52	5	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK C1156Y\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	006738b9-fe19-4c5f-aab3-11a8a65c3f15	2015-02-06 19:01:06.511159
53	6	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: EML4-ALK L1196M\ndescription: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion,\n  this variant has been shown to confer resistance to crizotinib.\n	1	\N	\N	a17d3c33-991a-48bf-b30a-2fee2b85227a	2015-02-06 19:01:06.521474
54	7	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: F1174L\ndescription: ALK F1174L has been observed as recurrent in neuroblastoma, non-small\n  cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing\n  this mutation have shown resistance to low doses of criztonib. However, increased\n  dosage can overcome this resistance in cell lines studies. TAE684 has also proven\n  effective in both NSCLC and neuroblastoma F1174L containing cells.\n	1	\N	\N	d6fcde19-3435-4b43-9c73-664277779e63	2015-02-06 19:01:06.53104
55	8	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 1\nname: R1275Q\ndescription: ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma,\n  non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma\n  cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This,\n  and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC\n  cell lines.\n	1	\N	\N	6ba350a2-ec93-4fbd-ae94-495efaa65552	2015-02-06 19:01:06.541593
56	9	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 3\nname: S214C\ndescription: ARAF S214C has been found to be a recurrent oncogenic mutation in non-small\n  cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib\n  in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this\n  mutation, sorafenib also acheived near-complete clinical remission. This case has\n  brought more interest to the variant from a research and clinical perspective.\n	1	\N	\N	3752232a-cc18-42d3-bdd3-19f0b08ab6a2	2015-02-06 19:01:06.550888
57	10	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600D\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	aa2e3d61-41e6-4014-b8f1-40b7c17ad988	2015-02-06 19:01:06.559449
58	11	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E\ndescription: BRAF V600E has been shown to be recurrent in many cancer types. It is\n  one of the most widely studied variants in cancer. This variant is correlated with\n  poor prognosis in certain cancer types, including colorectal cancer and papillary\n  thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective\n  in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib\n  has also shown to be effective when combined with the MEK inhibitor trametinib in\n  colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA\n  mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib,\n  and a number of combination therapies have been successful in treating V600E mutations.\n  While the drugs cetuximab and panitumumab have been largely shown to be ineffective\n  without supplementary treatment.\n	1	\N	\N	5214aff4-4601-4dd0-9fa3-2eba9b1f21ab	2015-02-06 19:01:06.567976
59	12	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E/V600M\ndescription: A case study of a single patient harboring both a V600E and a V600M mutation,\n  dabrafenib was shown to acheive clinical response.\n	1	\N	\N	e02238a7-f4cc-4994-892c-3bb19d8c73a5	2015-02-06 19:01:06.576461
60	13	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600E AMPLIFICATION\ndescription: Amplification of BRAF V600E has been shown to confer resistance to MEK\n  inhibitors. For more information on the V600 locus, see the V600E entry.\n	1	\N	\N	7ac03940-cc4e-401a-b832-85056c2d81fb	2015-02-06 19:01:06.585141
61	14	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600M\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	82aa0fe0-9cda-4982-9600-d1eb8cca58f0	2015-02-06 19:01:06.593429
62	15	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600R\ndescription: Patients harboring mutations in valine 600 residue of BRAF have been\n  shown to be sensitive to dabrafenib. For more information on the V600 locus, see\n  the V600E entry.\n	1	\N	\N	c0dc1c31-10c8-4935-832d-a8afc90b587d	2015-02-06 19:01:06.602027
63	16	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 5\nname: V600\ndescription: BRAF mutations of the valine 600 residue have been shown to be recurrent\n  in many cancer types. Of the V600 mutations, V600E is the most widely researched.\n  V600 mutations as a whole have been correlated to poorer prognosis in colorectal\n  and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity\n  to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual\n  mutation pages on the left sidebar.\n	1	\N	\N	c6fcfeef-a2fd-4ce6-b8fb-64d33dddb63a	2015-02-06 19:01:06.61104
64	17	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: AMPLIFICATION\ndescription: CCND1 amplification has been implicated in poorer prognosis in non-small\n  cell lung cancer.\n	1	\N	\N	2f419771-a9a8-48df-83a2-f43a983273d9	2015-02-06 19:01:06.619748
65	18	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: EXPRESSION\ndescription: CCND1 expression, and its prognositc impact, is still in dispute. Three\n  experiments in non-small cell lung cancer have shown it to have no impact on survival,\n  but three additional studies have shown it results in poorer prognosis. There is\n  also some ambiguity in how the boundaries between expression and overexpression\n  are defined.\n	1	\N	\N	e7385fe0-8914-45fd-a390-150c95c574e2	2015-02-06 19:01:06.628234
66	19	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 6\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	6b389e19-3927-4df8-878f-163000988890	2015-02-06 19:01:06.638575
67	20	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: EXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	5e58ce84-9228-46cc-b5a4-0cf831040ff2	2015-02-06 19:01:06.650135
68	21	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: OVEREXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	17c8ec2f-ddae-40d9-b23d-8cd1412252c8	2015-02-06 19:01:06.660425
127	80	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 1\ndescription: A study by Li√®vre et al in 2006 showed that colorectal cancer patients\n  with KRAS exon 1 mutations had low cetuximab response rates.\n	1	\N	\N	f52222d1-8f10-4848-99e3-5ba0d24eedf1	2015-02-06 19:01:07.229686
69	22	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 7\nname: PROMOTER DEMETHYLATION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	09ea9901-8879-4ca3-aa25-ac2defa242a1	2015-02-06 19:01:06.669819
70	23	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: EXPRESSION\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are misregulated\n  are widely variable, and range from genomic amplification to promoter methylation\n  changes. While Cyclin D2 has only been found to be significantly misregulated in\n  glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation\n  has been shown to lead to poorer outcomes in a number of studies, and currently\n  there are no FDA-approved targeted therapies.\n	1	\N	\N	f6ee9aa0-907b-4293-990e-eae9bcdf941e	2015-02-06 19:01:06.680557
71	24	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 8\nname: LOSS\ndescription: Cyclin D has been shown in many cancer types to be misregulated. Well\n  established for their oncogenic properties, the cyclins and the cyclin-dependent\n  kinases (CDK's) they activate have been the focus of major research and development\n  efforts over the past decade. The methods by which the cyclins are deregulated are\n  widely variable, and range from genomic amplification to promoter methylation changes.\n  Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL),\n  a seemingly unique trend when compared to the amplifcations and overexpressions\n  of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic\n  palbociclib significantly increased the median survival of a Notch-driven model\n  of T-ALL.\n	1	\N	\N	01d2e5e7-f6f9-4a10-a22d-3514e306e102	2015-02-06 19:01:06.689327
72	25	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 9\nname: OVEREXPRESSION\ndescription: Cyclin E, while currenly not as widely implicated as its counterpart,\n  cyclin D, has been implicated in various carcinomas, including breast, gastric,\n  stomach and colorectal. High levels of cyclin E, either by gene amplification or\n  overexpression, are correlated with later stage disease and have been shown to lead\n  to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer\n  specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex\n  are more radiosensitive than their lowly expressed counterparts.\n	1	\N	\N	60b1515c-1d05-4a18-8f17-e8f30abcb735	2015-02-06 19:01:06.698394
73	26	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 11\nname: EXPRESSION\ndescription: CDK4, along with its partner CDK6, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	36a25733-af92-4ca4-b576-398008b7bdae	2015-02-06 19:01:06.70707
74	27	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 10\nname: EXPRESSION\ndescription: CDK6, along with its partner CDK4, are key players in cell cycle progression.\n  The complex has been implicated in a number of cancer types, and is the focus of\n  therapeutic research and development. One targeted therapy for CDK inhibition is\n  palbociclib, which may slow the growth of advanced stage breast cancers. It has\n  also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors\n  to PI3K inhibitors.\n	1	\N	\N	c1f7d9ae-9eb8-492f-a38b-fa5a2a905b05	2015-02-06 19:01:06.715836
75	28	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 12\nname: PROMOTER HYPERMETHYLATION\ndescription: CDKN2A loss has been shown to be a significant event in a number of cancer\n  types. One mechanism by which this can occur is by hypermethylation of the CDKN2A\n  promoter region. While no targeted therapeutic has been engaged in clinical trials,\n  the prognostic impact has been studied by a number of meta-analyses. The prognostic\n  impact of promoter hypermethylation has been relatively ambiguous. Many studies\n  have shown significant p-values suggesting poorer prognostic outcomes for patients\n  with hypermethylation in colorectal, liver, and younger lung cancer patients. This\n  being said, there is still research to be done before this becomes a widely-accepted\n  prognostic indicator.\n	1	\N	\N	228c6af4-7e08-43aa-b87a-0c3756223d6a	2015-02-06 19:01:06.726098
76	29	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: N-TERMINAL FRAME SHIFT\ndescription: CEBPA N-terminal frame shift mutations that result in a premature stop\n  codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML).\n  CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3\n  mutations should also be assessed in CN-AML patients as concurrent mutations may\n  have prognostic implications.\n	1	\N	\N	8803ec7a-584a-4cbe-8d3e-2cb5cc72ba50	2015-02-06 19:01:06.736044
77	30	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 13\nname: WHOLE GENE\ndescription: CEBPA mutations are associated with cytogenetically normal acute myeloid\n  leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should\n  also be assessed in CN-AML patients as concurrent mutations may have prognostic\n  implications.\n	1	\N	\N	425a7fd8-b81a-4f47-9212-2dddceb5738a	2015-02-06 19:01:06.744599
78	31	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 31\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	67953396-b822-4502-8f8d-fdf76d149a80	2015-02-06 19:01:06.754934
79	32	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 14\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in the rare form of adolescent\n  liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion\n  was observed in 15/15 FL-HCC cases examined and functional studies found that the\n  fusion retained kinase activity. The presence of this fusion may be used as a diagnostic\n  marker for this rare tumor type.\n	1	\N	\N	9f8ec055-e962-4544-b6c3-84b4d0919641	2015-02-06 19:01:06.763768
80	33	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: R882\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	1029004f-d06c-4141-943d-350b6cedd2ca	2015-02-06 19:01:06.773937
81	34	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 16\nname: R882H\ndescription: DNMT3A R882 mutations are associated with cytogenetically normal acute\n  myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A\n  have largely been associated with poorer prognosis, however this is not consistent\n  across all studies. This may be a result of patient age or combining R882 and non-R882\n  mutations during analysis as studies have indicated independent mechanisms of action\n  and differential prognostic implications for these mutation types. One study that\n  independently analyzed R882 and non-R882 mutations showed R882 mutations were associated\n  with poorer prognosis than patients with wildtype and non-R882 mutations, but only\n  in older patients with AML.\n	1	\N	\N	6a6db37b-1c92-408c-b9b1-848d01ca04ed	2015-02-06 19:01:06.782895
82	35	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: L858R\ndescription: EGFR L858R has long been recognized as a functionally significant mutation\n  in cancer, and is one of the most prevalent single mutations in lung cancer. Best\n  described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity\n  to first and second generation TKI's like gefitinib and neratinib. NSCLC patients\n  with this mutation treated with TKI's show increased overall and progression-free\n  survival, as compared to chemotherapy alone. Third generation TKI's are currently\n  in clinical trials that specifically focus on mutant forms of EGFR, a few of which\n  have shown efficacy in treating patients that failed to respond to earlier generation\n  TKI therapies.\n	1	\N	\N	5ad1d329-4001-46d9-8a74-eeed91362f94	2015-02-06 19:01:06.793572
83	36	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 17\nname: T790M\ndescription: EGFR T790M was one of the very first mutations recognized to confer resistance\n  to targeted therapies. While successful in amplified EGFR, the efficacy of the first\n  and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients\n  harboring this mutation before treatment is notably lower. This lack of efficacy\n  can likely be to blame for the poorer prognosis for patients with this mutation\n  as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately\n  half of EGFR mutant tumors with acquired resistance to TKI inhibition have been\n  shown to harbor this mutation, implicating it as a mechanism of acquired therapy\n  resistence. The third generation TKI's are being developed with this resistance\n  problem in mind, and early stage experiments have shown some efficacy of these drugs\n  in previously resistant tumors with mutant EGFR.\n	1	\N	\N	b2e227b8-f8e7-4ae5-b44c-0aefe0c1ec22	2015-02-06 19:01:06.803715
84	37	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: D769H\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	61544b3f-f4b0-4772-9551-fc06101d9971	2015-02-06 19:01:06.812407
85	38	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: DEL 755-759\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	34b9ff4b-0681-400c-9654-bae0941a0a78	2015-02-06 19:01:06.821614
86	39	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: G309A\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	840e71b0-1f08-4e57-a15e-1738e1ec2ae5	2015-02-06 19:01:06.831104
87	40	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755S\ndescription: This variant was one of the first of the ERBB2 variants to be functionally\n  classified (Bose et al. 2012). This mutation was not shown to be an activating mutation,\n  unlike many of the other variants queried. This mutation was also shown to confer\n  resistance to the tyrosine kinase inhibitor lapatinib in cell lines.\n	1	\N	\N	ad48b9ae-6ec3-414c-8054-4eccc2093ec2	2015-02-06 19:01:06.839559
88	41	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: L755W\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	f480a48b-a1dd-4e60-8811-f49b038fc08d	2015-02-06 19:01:06.848374
89	42	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: P780INS\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	2c7b7605-83c6-41de-8c1b-b65c14004fc4	2015-02-06 19:01:06.857093
90	43	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R678Q\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	8f125c83-ae94-4b94-994a-d739129c3162	2015-02-06 19:01:06.866247
128	81	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: EXON 2\ndescription: In a study by Pan et al in 2005, non-small cell lung cancer patients\n  harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors\n  gefinitib and erlotinib.\n	1	\N	\N	49e57659-216d-4b85-b62e-0d07f5ac2ec3	2015-02-06 19:01:07.244599
91	44	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: R896C\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	c072ade5-c668-49b7-a74e-47291d217198	2015-02-06 19:01:06.875265
92	45	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V777L\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	39ee0d72-e7f2-47f2-b120-065710634cb4	2015-02-06 19:01:06.88403
93	46	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 18\nname: V842I\ndescription: This variant was one of the first of the ERBB2 variants (aka HER2) to\n  be functionally classified (Bose et al. 2012). This mutation was shown to be an\n  activating mutation in an in vitro assay. In the same paper, this mutation (along\n  with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been\n  shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may\n  show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase\n  inhibitors, which is the topic of current clinical trials and research.\n	1	\N	\N	58999ee7-e0ac-4b1f-b961-53758c8024e0	2015-02-06 19:01:06.892743
94	47	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: L536Q\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of\n  these ligand binding domain mutations, and is commonly implicated in this hormone\n  resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant,\n  may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	bfa025d8-b474-432a-bf78-243162f2ff57	2015-02-06 19:01:06.901612
95	48	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: N538G\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	ed2f313b-9e83-4f82-937a-dcb65711e4e4	2015-02-06 19:01:06.911508
96	49	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537C\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	bbf9693e-f53f-4dce-8d25-802d9fad3b1c	2015-02-06 19:01:06.920832
97	50	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537N\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	2b6959c6-8be7-4d84-8c71-731d85d9cf57	2015-02-06 19:01:06.929721
98	51	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 19\nname: Y537S\ndescription: ESR1 biology has become a central focus in breast cancer therapy. In\n  ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer\n  resistance to hormone therapy. This evidence has led to an increased use of targeted\n  sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of\n  these ligand binding domain  , and is commonly implicated in this hormone resistance.\n  Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial\n  in treating ESR1 mutant, hormone resistant breast cancers.\n	1	\N	\N	d7e760e5-1ee7-4850-aedf-d3c2a28def8a	2015-02-06 19:01:06.939173
99	52	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-AFF3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	4acf1236-0825-4002-9285-8b019370d1b8	2015-02-06 19:01:06.948383
100	53	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-BICC1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	b49f8b14-10e0-42fb-ab22-6716748a8553	2015-02-06 19:01:06.95721
101	54	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CASP7\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	f72f3090-34a4-42b5-9a96-cafc3ad064b5	2015-02-06 19:01:06.966337
143	96	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 3\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	8acd4b7f-6882-439e-b23b-40113ea4c407	2015-02-06 19:01:07.387587
102	55	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-CCDC6\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	bbd2b542-0a46-49f2-b224-e35a4e348d30	2015-02-06 19:01:06.975602
103	56	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-KIAA1967\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	1a7e4050-fa8d-41f3-90e5-665ba31e4191	2015-02-06 19:01:06.984614
104	57	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-MGEA5\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. Additionally, tumor size reduction was\n  achieved by both ponatinib and pazopanib treatments administered separately in a\n  single patient with intrahepatic cholangiocarcinoma and this fusion. The authors\n  use these cases to highlight the need for enhanced clinical sequencing efforts.\n	1	\N	\N	2a355c14-41a6-488d-9313-6d027ad00570	2015-02-06 19:01:06.993509
105	58	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-OFD1\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	ec59a739-ed39-4f10-bfda-bfce912d8809	2015-02-06 19:01:07.006215
106	59	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 20\nname: FGFR2-SLC45A3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. Cell lines harboring these\n  fusions were shown to respond to pazopanib. The authors use these cases to highlight\n  the need for enhanced clinical sequencing efforts.\n	1	\N	\N	28ed7520-f001-4604-ad4f-9cd7ade99c8c	2015-02-06 19:01:07.015548
107	60	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 21\nname: FGFR3-TACC3\ndescription: In a clinical sequencing program for advanced stage cancers, Wu et al\n  (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with\n  many different cancer types. These fusions were also found to retain oligomerization\n  capability, and result in enhanced cell proliferation. These fusions were shown\n  to respond to pazopanib. The authors use these cases to highlight the need for enhanced\n  clinical sequencing efforts.\n	1	\N	\N	e86e5a66-3767-483e-a702-39bfca9f3fa5	2015-02-06 19:01:07.026546
108	61	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: ITD\ndescription: FLT3-ITD (internal tandem duplications) frequently occur in patients\n  with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid\n  leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically\n  normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of\n  FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated\n  sequence resulting in an in-frame duplication event. The length of these duplications\n  can vary widely which may have prognostic consequences, but this has not been conclusively\n  determined. FLT3-ITD mutations overall have generally been associated with poor\n  prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the\n  prognosis associated with this variant.\n	1	\N	\N	6f513ed2-8739-4e66-aa1c-b2c8efa11209	2015-02-06 19:01:07.035192
109	62	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 22\nname: TKD\ndescription: FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common\n  than FLT3-ITD (internal tandem duplication) mutations and may not confer the same\n  prognostic impact. Although the majority of mutations are point mutations effecting\n  D835 (most frequently D835Y), a small proportion involve an in-frame deletion of\n  I836. These mutations are within the activation loop of the second tyrosine kinase\n  domain of FLT3 and thought to result in constitutive activation of the receptor.\n	1	\N	\N	174aec9f-c7ef-4289-a042-ee9f3867871d	2015-02-06 19:01:07.045144
110	63	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 23\nname: EXPRESSION\ndescription: GATA2 misregulation has been observed in a number of hematologic malignancies,\n  as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using\n  the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung\n  cancer.\n	1	\N	\N	0e0a993f-ce11-4f63-bbb6-b18a4ed03c59	2015-02-06 19:01:07.054707
111	64	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	07a3698f-f0e0-4a1a-ad1d-3449fe6b30ba	2015-02-06 19:01:07.06442
112	65	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132C\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	6330f83a-6802-429e-8bf5-2f5ce32079f3	2015-02-06 19:01:07.073919
144	97	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 4\ndescription: In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS\n  exon 3 and 4 mutations were correlated with poorer overall survival.\n	1	\N	\N	b0f91ed9-9602-44b8-95d8-39cc04581566	2015-02-06 19:01:07.396361
113	66	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132G\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	cad634a6-d690-43d6-b180-ec1e0823c77a	2015-02-06 19:01:07.08345
114	67	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132H\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	e73e4d6e-49f1-4993-9667-5dc32c76a511	2015-02-06 19:01:07.094067
115	68	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 24\nname: R132L\ndescription: IDH1 R132 mutations have been observed in a number of cancer types, and\n  appear to behave quite differently in different environments. In myelodysplastic\n  syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown\n  to be associated with worse outcome and shorter overall survival. They have also\n  been linked to cytogenetically normal AML, an intermediate risk subtype of the disease.\n  However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown\n  better overall survival than patients with wild-type IDH1. Also in contrast to AML,\n  in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic\n  abnormalities, 1p and 19q deletions.\n	1	\N	\N	a62cd662-a9e5-4def-b8e9-5e0e0dff0f96	2015-02-06 19:01:07.103011
116	69	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R140Q/L\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival\n  as compared to their wild-type counterparts. In myelodysplastic syndromes, however,\n  no prognostic link was discovered between IDH2 mutation status and overall survival.\n	1	\N	\N	dd2dd648-3677-4c0e-a776-0f12136c709e	2015-02-06 19:01:07.113516
117	70	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 25\nname: R172K\ndescription: IDH2 mutations have been observed in a number of hematologic malignancies.\n  In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis\n  and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic\n  syndromes, studies did not find a prognostic association between this variant and\n  patient outcomes.\n	1	\N	\N	d9568823-ddb5-4341-9998-44dead0865ad	2015-02-06 19:01:07.123683
118	71	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 26\nname: V617F\ndescription: JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases,\n  occuring in around half of all MPD's. While less associated with cancer, when it\n  is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The\n  V617F mutation is an activating mutation, resulting in increased kinase activity.\n  The mutation seems to be restricted to hematologic malignancies. Treatment of JAK\n  mutant diseases with ruxolitinib has seen some clinical success.\n	1	\N	\N	8ccf5cae-8908-454c-9675-4906f2e118af	2015-02-06 19:01:07.132988
119	72	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: D816V\ndescription: KIT D816V is a mutation observed in acute myeloid leukemia (AML). This\n  variant has been linked to poorer prognosis and worse outcome in AML patients.\n	1	\N	\N	e2ca8ec9-b0e8-4361-a11e-5f17bee9da96	2015-02-06 19:01:07.142625
120	73	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 11\ndescription: c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are\n  very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis\n  than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity\n  to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients\n  treated with the drug in the first year.\n	1	\N	\N	62e1e417-a89a-455d-82fc-addd20fb9fdc	2015-02-06 19:01:07.151513
121	74	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 13\ndescription: c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations.\n  These mutations lie within the tyrosine kinase 1 domain, and are found primarily\n  in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations\n  show sensitivity to imatinib and sunitinib treatment. However, only imatinib has\n  seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant\n  cases.\n	1	\N	\N	c7a54917-fb05-4f96-ad15-bf4a977bb692	2015-02-06 19:01:07.15976
122	75	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 14\ndescription: c-KIT exon 14 mutations lie within the tyrosine kinase domain of the\n  protein. While relatively rare in primary gastrointestinal tumors, they are notably\n  more prevalent in refractory disease, suggesting a role in imatinib resistance.\n  Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.\n	1	\N	\N	d8ce701a-d967-4b9a-bed4-ac5d0f439d1b	2015-02-06 19:01:07.168698
123	76	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 17\ndescription: c-KIT exon 17 mutations lie within the TK2 domain, containing the activation\n  loop protein. In cell lines, mutations within this domain have been shown to be\n  sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation\n  is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.\n	1	\N	\N	da4d87fe-9b43-49e1-97eb-719872a9730b	2015-02-06 19:01:07.178457
124	77	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: EXON 9\ndescription: c-KIT exon 9 mutations lie within the dimerization motif of the protein.\n  Relative to other KIT mutations, exon 9 mutations have been associated with better\n  overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and\n  in vivo.\n	1	\N	\N	8c57198b-cfb1-4ad7-845c-09de3081e456	2015-02-06 19:01:07.187043
125	78	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: L576P\ndescription: KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain.\n  It is one of the most recurrent KIT mutations in melanoma, and both in vitro and\n  in vivo studies have shown sensitivity to imatinib.\n	1	\N	\N	dda701bb-f719-4f7a-b3b7-6063fba060be	2015-02-06 19:01:07.209576
126	79	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 27\nname: V654A\ndescription: KIT V654A is an exon 13 mutation that lies within the tyrosine kinase\n  1 domain of the protein. It has been shown to be an activating mutation by in vitro\n  studies. This mutation is associated with imatinib resistance in melanoma patients.\n  However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have\n  seen success in acheiving tumor response.\n	1	\N	\N	ae7d3fcd-995b-4041-9e55-83896139c30a	2015-02-06 19:01:07.220469
129	82	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	e67da26f-6003-4456-b6b3-4aef6d4a1835	2015-02-06 19:01:07.254702
130	83	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12/G13\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation\n  may result in a less responsive tumor when treated with first-generation TKI's like\n  gefitinib.\n	1	\N	\N	fbe51f08-8733-443e-a2e2-b3f7665e9660	2015-02-06 19:01:07.264382
131	84	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12C\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	c8165c5a-9ebc-4ac8-a18b-ab3b26099091	2015-02-06 19:01:07.273168
132	85	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G12D\ndescription: While the KRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G13 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	e7d2c67c-a4fe-4c22-834d-bfdbfc638fee	2015-02-06 19:01:07.283828
133	86	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	39d58c56-5b7e-42a8-88b9-c65b941a0b7d	2015-02-06 19:01:07.293535
134	87	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 28\nname: G13D\ndescription: While the KRAS G13 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated. Often associated with tumors that\n  are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients\n  with this mutation seems to be worse than the KRAS wild-type cohort. This mutation,\n  along with the mutations affecting the neighboring G12 position, may result in a\n  less responsive tumor when treated with first-generation TKI's like gefitinib.\n	1	\N	\N	a5c08ad7-a89e-4247-a2bf-41835fb9cb3a	2015-02-06 19:01:07.302187
135	88	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: P124S\ndescription: MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder\n  and colon cancer to a lesser degree. The P124S mutation has been shown to contribute\n  to AZD6244 resistance in melanoma cell lines, but considerably less so than its\n  Q56P counterpart.\n	1	\N	\N	b742d0bd-9a09-4ee5-9863-73e49255c8da	2015-02-06 19:01:07.310806
136	89	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 29\nname: Q56P\ndescription: MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This\n  mutation has been shown to confer considerable resistance to AZD6244 treatment of\n  melanoma cell lines.\n	1	\N	\N	78bf1761-9303-4cfc-93ae-913a8719ac12	2015-02-06 19:01:07.320015
137	90	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 30\nname: MEF2D-CSF1R\ndescription: MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia.\n  In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\n	1	\N	\N	f62e03ed-62df-483f-9273-af9f7405ebd4	2015-02-06 19:01:07.329685
138	91	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 32\nname: PROMOTER METHYLATION\ndescription: MGMT promoter methylation has been observed to impact tumor progression\n  in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating\n  agent carmustine has shown efficacy. In patients lacking methylation, combining\n  carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more\n  experiments are required. Clinical trials have also shown selective sensitivity\n  of promoter methylation-positive patients to temozolomide, making a case for wider\n  methylation screening in GBM patients.\n	1	\N	\N	3c936d01-9c1a-4c30-bf9d-43a1f6eb4404	2015-02-06 19:01:07.340048
139	92	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: EXON 12\ndescription: NPM1 exon 12 mutations are frequently identified in patients with cytogenetically\n  normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status\n  should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon\n  12 mutations have been identified as a predictor of good prognostic outcomes in\n  the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively\n  analyzed in the context of a number of therapies including variable results following\n  ATRA treatment as well as improved response to high-dose daunorubicin or valproic\n  acid. Additionally, multiple groups have shown increased surface expression of CD33\n  associated with NPM1 mutation, suggesting these patients may respond to anti-CD33\n  therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response\n  to induction therapy.\n	1	\N	\N	ae3ae613-0641-4947-a0a3-a73410323b30	2015-02-06 19:01:07.349673
140	93	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 33\nname: W288FS\ndescription: NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most\n  common NPM1 mutation identified in acute myeloid leukemia. This mutation results\n  in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response\n  to induction therapy. Although it is the most extensively studied NPM1 exon 12,\n  it is generally grouped with other exon 12 mutations for patient analysis (see NPM1\n  Exon 12 variants for more information).\n	1	\N	\N	dd8e9a24-fb0b-4f53-9551-7b3f015766a3	2015-02-06 19:01:07.358356
141	94	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 1\ndescription: NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown\n  to be correlated with poorer overall survival relative to wild-type NRAS in melanoma\n  patients.\n	1	\N	\N	9ca0df93-a37c-4408-a73b-971f56fab469	2015-02-06 19:01:07.367286
142	95	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: EXON 2\ndescription: NRAS exon 2 mutations have been shown to be correlated with poorer overall\n  survival in melanoma patients and colorectal cancer patients, however no prognostic\n  impact was seen in acute myeloid leukemia patients.\n	1	\N	\N	c04108b9-72d2-4e95-a1ba-4c7510a662ca	2015-02-06 19:01:07.377125
145	98	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G12\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	e95f1d6c-f8f0-48e6-9f33-9edc6f69925d	2015-02-06 19:01:07.405019
146	99	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: G13D\ndescription: While the NRAS G12 region is a widely studied recurrent region in cancer,\n  its impact on clinical action is still debated.\n	1	\N	\N	522453d1-145a-4de2-a4c4-a1106da0ce53	2015-02-06 19:01:07.413553
147	100	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations\n  at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide.\n  However, in colorectal cancer patients, mutations at this locus have been shown\n  to confer resistance to cetuximab. The prognostic impact of mutations at this locus\n  is currently under study.\n	1	\N	\N	fcaa68a6-5b02-4815-92e3-50b1be92b2eb	2015-02-06 19:01:07.423555
148	101	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61L\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	5a9b8388-b463-4c3f-ab7d-6ee8bbf3a8b2	2015-02-06 19:01:07.433373
149	102	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 34\nname: Q61R\ndescription: NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal\n  stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at\n  this locus responded to treatment with the akylating agent temozolomide. However,\n  in colorectal cancer patients, mutations at this locus have been shown to confer\n  resistance to cetuximab. The prognostic impact of mutations at this locus is currently\n  under study.\n	1	\N	\N	0dadea17-861c-412e-b82c-a8672f63d8c0	2015-02-06 19:01:07.442177
150	103	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842I\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	4073692d-aa79-4cdd-af5e-41aa67474866	2015-02-06 19:01:07.451055
151	104	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842V\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	54958bed-ec36-47c2-a2fd-7368297e14c4	2015-02-06 19:01:07.45967
152	105	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: D842Y\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	02ee7acc-569a-4af1-bab9-4bbfe01007b2	2015-02-06 19:01:07.469778
153	106	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DEL I843\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	e86c3e21-9345-48b3-8258-2c8518c5a9b1	2015-02-06 19:01:07.480301
154	107	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 36\nname: DI842-843IM\ndescription: PDGFRA D842 mutations are characterized broadly as imatinib resistance\n  mutations. This is most well characterized in gastrointestinal stromal tumors, but\n  other cell lines containing these mutations have been shown to be resistant as well.\n  In imatinib resistant cell lines, a number of other therapeutics have demonstrated\n  efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).\n	1	\N	\N	ab130b79-8021-40e1-a1ab-10b9cb78a470	2015-02-06 19:01:07.489372
155	108	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E542K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	76647773-47c4-4b29-8153-75001a51d8b2	2015-02-06 19:01:07.497622
156	109	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: E545K\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	da8e0d24-af70-4802-b709-5479c2f9f08b	2015-02-06 19:01:07.506723
157	110	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 20\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	d2f78d60-d154-45ee-a13f-cf35df501915	2015-02-06 19:01:07.515501
158	111	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: EXON 9\ndescription: PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in\n  breast cancer, and are highly recurrent mutations in many other cancer types. E545K,\n  and possibly the other mutations in the E545 region, may present patients with a\n  poorer prognosis than patients with either patients with other PIK3CA variant or\n  wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer\n  resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies\n  for variants in PIK3CA are still in early clinical trial phases.\n	1	\N	\N	0787e740-6253-4258-9a53-3b7977b15194	2015-02-06 19:01:07.524849
159	112	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 35\nname: H1047R\ndescription: PIK3CA H1047R is one of the most recurrent mutations in cancer, especially\n  breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation.\n  Meta-analyses have shown that patients harboring this mutation may have worse overall\n  survival, but other studies have shown no difference between H1047R and other PIK3CA\n  mutants from a prognostic standpoint. While very prevalent, targeted therapies for\n  this particular mutation are still in early clinical trial phases.\n	1	\N	\N	a7ef1084-30e1-43a5-89df-5ab62021a20b	2015-02-06 19:01:07.533515
160	113	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 37\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and arsenic trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	2c31e8e7-f359-47ed-ad07-2f4f8b43a530	2015-02-06 19:01:07.542274
161	114	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 15\nname: DNAJB1-PRKACA\ndescription: This fusion has been found to be very recurrent in a rare form of adolescent\n  liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found\n  Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined,\n  and functional studies found that the fusion retained kinase activity. The presence\n  of this fusion may be used as a diagnostic marker for this rare tumor type.\n	1	\N	\N	f8c58001-1f95-49a1-a6cf-f6be64cd3a34	2015-02-06 19:01:07.551103
162	115	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 39\nname: R233*\ndescription: PTEN R233* has been shown to be a loss of function mutation, and PTEN\n  loss has been the subject of considerable research in breast cancer. PTEN loss may\n  sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data\n  to support that PTEN loss is both associated with poorer prognosis, and no change\n  in prognosis.\n	1	\N	\N	ee1af668-cd89-4e93-b297-c16ad6374273	2015-02-06 19:01:07.559775
163	116	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 38\nname: PML-RARA\ndescription: The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic\n  leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA\n  in APL patients harboring the PML-RARa fusion. Recent interest has been shown in\n  combining ATRA and Arsenic Trioxide for treating these patients, and early results\n  seem promising.\n	1	\N	\N	16b7896a-7d26-4288-a696-33dc22c78d43	2015-02-06 19:01:07.568524
164	117	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: C634W\ndescription: RET C639W has been implicated as an alternate mechanism of activating\n  RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting\n  agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the C639W mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib.\n	1	\N	\N	d7466bb7-cb2e-4f79-b480-1169e9f1a12e	2015-02-06 19:01:07.577504
165	118	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 40\nname: M918T\ndescription: RET M819T is the most common somatically acquired mutation in medullary\n  thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents,\n  promiscuous kinase inhibitors have seen some success in treating RET overactivity.\n  Data suggests however, that the M918T mutation may lead to drug resistance, especially\n  against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T\n  leads to more aggressive MTC with a poorer prognosis.\n	1	\N	\N	d4bb693e-db62-44d9-8400-d43c3766d6b1	2015-02-06 19:01:07.588217
166	119	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K666N\ndescription: SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients.\n	1	\N	\N	91b6bfb3-37c2-472e-ae3b-27b36086acd2	2015-02-06 19:01:07.596991
167	120	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 42\nname: K700E\ndescription: SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic\n  leukemias, and more recently, breast cancer. This somatic mutation has been linked\n  to better overall outcome and event-free survival in MDS patients. Additionally,\n  these mutations are the most common SF3B1 mutation observed in MDS and highly associated\n  with subtypes of MDS that are defined by ringed sideroblasts.\n	1	\N	\N	726a8e91-9c1a-46be-8d9e-45e0cbacf9d1	2015-02-06 19:01:07.605708
168	121	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R175H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R175H variant seems not only to result in loss of tumor-suppression, but also acts\n  as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell\n  lines harboring this mutant is also more responsive to treatment with doxorubicin\n  than its wild-type counterparts. While the prognostic impact of individual TP53\n  mutations is influenced by the cohort being studied, it has been shown that the\n  R175H mutation is correlated with worse overall survival than wild-type TP53, but\n  is not as detrimental as the R248W variant.\n	1	\N	\N	39ee03cf-75a8-4253-a88a-7d726644abfd	2015-02-06 19:01:07.614406
169	122	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248Q\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival. The R248Q mutation has also shown an increased invasive behavior in cell\n  lines. This is specific to the 248Q variant.\n	1	\N	\N	3321d276-7318-483c-b6c1-52ea43002111	2015-02-06 19:01:07.62302
170	123	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R248W\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R248 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, R248 mutations have been shown to confer worse overall\n  survival.\n	1	\N	\N	68f14bcd-3759-4a30-b2ea-85b3901f98ee	2015-02-06 19:01:07.633635
171	124	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249T\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	cf903c8e-cab6-4853-8759-e95386f116a5	2015-02-06 19:01:07.644052
172	125	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R249W\ndescription: This mutant is also more responsive to treatment with doxorubicin than\n  its wild-type counterparts. While the prognostic impact of individual TP53 mutations\n  is influenced by the cohort being studied, it has been suggested that the R249 mutants\n  have been correlated with worse overall survival in breast cancer patients when\n  compared to wild-type.\n	1	\N	\N	d42f1e9d-6140-457d-9b1f-3fe86eba36bc	2015-02-06 19:01:07.65428
173	126	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273C\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	40d38c16-3953-42ec-bdca-37240339f019	2015-02-06 19:01:07.664551
174	127	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: R273H\ndescription: While loss-of-function events in TP53 are very common in cancer, the\n  R273 variants seem not only to result in loss of tumor-suppression, but also act\n  as a gain-of-function mutation that can promote tumorigenesis in mouse models. This\n  mutant is also more responsive to treatment with doxorubicin than its wild-type\n  counterparts. While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R273 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	6a921cfa-f24a-4c6b-b418-ad1528fe9484	2015-02-06 19:01:07.675375
175	128	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 43\nname: V173G/A\ndescription: While the prognostic impact of individual TP53 mutations is influenced\n  by the cohort being studied, it has been suggested that the R249 mutants have been\n  correlated with worse overall survival in breast cancer patients when compared to\n  wild-type.\n	1	\N	\N	c2e5bfc0-0ec8-4e82-9b08-5255cf59653a	2015-02-06 19:01:07.685867
176	129	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: FRAMESHIFT TRUNCATION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	502ee868-0bff-4e2f-91b6-27727d490b8c	2015-02-06 19:01:07.698136
177	130	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 44\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	e8511d69-8eae-4afa-ab1e-9cb3de2f77ea	2015-02-06 19:01:07.708481
178	131	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 45\nname: LOSS-OF-FUNCTION\ndescription: In a small cohort study of bladder cancer, patients with TSC1 mutations\n  showed better responses to and increased treatment duration tolerance with the mTOR\n  inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with\n  TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2\n  wildtype cells.\n	1	\N	\N	e024e7fe-e96b-4a71-a479-cb598fe54987	2015-02-06 19:01:07.719201
179	132	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: Q157P/R\ndescription: U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is less common than the S34F mutation, occurs in the second zinc finger domain of\n  U2AF1 and has been demonstrated to alter splicing. Although this mutation has not\n  been determined to provide prognostic information in AML, patients with MDS were\n  shown to be at an increased risk of transformation to secondary AML. However, the\n  prognostic importance of U2AF1 mutations in MDS has been debated.\n	1	\N	\N	9dc4a3ca-7204-483c-b846-d008b7e8bf83	2015-02-06 19:01:07.727911
180	133	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 46\nname: S34Y/F\ndescription: U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid\n  leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation\n  is is the most commonly identified variant in MDS, occurs in the first zinc finger\n  domain of U2AF1 and has been demonstrated to alter splicing. Although this mutation\n  has not been determined to provide prognostic information in AML, patients with\n  MDS were shown to be at an increased risk of transformation to secondary AML. However,\n  the prognostic importance of U2AF1 mutations in MDS has been debated.\n	1	\N	\N	d75a3b6e-ce9f-416b-8a9b-073f071d004f	2015-02-06 19:01:07.737093
181	134	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 7\ndescription: WT1 exon 7 mutations have been shown to be recurrent in acute myeloid\n  leukemia. Many sources have examined the prognostic impact of these, agreeing that\n  the mutant exon 7 cohort is correlated with worse overall survival and a number\n  of poor prognistic outcomes. This may be the result of an overall poor response\n  to chemotherapy from WT1 mutant tumors.\n	1	\N	\N	9dbfb082-6163-4e90-ae68-8e30d36b2d03	2015-02-06 19:01:07.748206
182	135	Variant	\N	\N	\N	\N	\N	create	---\ngene_id: 47\nname: EXON 9\ndescription: WT1 exon 9 mutations have been shown to be recurrent in acute myeloid\n  leukemia, although at a less frequent rate than their exon 7 counterparts. Many\n  sources have examined the prognostic impact of these, agreeing that the mutant exon\n  9 cohort is correlated with worse overall survival and a number of poor prognistic\n  outcomes. This may be the result of an overall poor response to chemotherapy from\n  WT1 mutant tumors.\n	1	\N	\N	9c9b1481-4243-49de-b836-d11f6618ebd0	2015-02-06 19:01:07.756993
183	1	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 40\nvariant_id: 131\nrating: 3\n	1	\N	\N	8c85c072-4050-4b17-8e53-d36ee9c35c70	2015-02-06 19:01:07.879621
184	2	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus\n  than their wild-type counterparts.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 1\nsource_id: 40\nvariant_id: 130\nrating: 3\n	1	\N	\N	679af1ab-d446-464c-9a4d-6481a782e64a	2015-02-06 19:01:07.931225
185	3	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A patient with metastatic bladder cancer that responded well to the mTOR inhibitor\n  everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del,\n  p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations\n  also had mild responses to everolimus.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 1\ndisease_id: 2\nsource_id: 41\nvariant_id: 129\nrating: 3\n	1	\N	\N	929f7f4d-f440-4a4d-bf07-760147d98dad	2015-02-06 19:01:07.956024
186	4	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice\n  with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit\n  increased overall survival, increased apoptosis, reduced proliferation and reduced\n  tumor burden compared to Tsc1 wildtype mice.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 3\nsource_id: 42\nvariant_id: 129\nrating: 3\n	1	\N	\N	02d8ab2f-9d1d-456f-aa6e-41a19b0aecd0	2015-02-06 19:01:07.985687
187	5	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse\n  to PI3K-mTOR inhibitors.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 4\nsource_id: 43\nvariant_id: 115\nrating: 4\n	1	\N	\N	6a49837f-8f97-4a15-8395-f432374f4bef	2015-02-06 19:01:08.013212
188	6	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PTEN nonsense  , including R233*, has been shown to be inactivating and loss-of-function,\n  but do not have prognostic value in glioblastoma multiforme patients.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 5\nsource_id: 44\nvariant_id: 115\nrating: 3\n	1	\N	\N	4b0e9018-16b5-4f0a-af4d-6cdc5fd6ce74	2015-02-06 19:01:08.043601
189	7	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 45\nvariant_id: 10\nrating: 5\n	1	\N	\N	e80f4de8-1e0b-414b-88ee-975a2da54653	2015-02-06 19:01:08.068265
190	8	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 45\nvariant_id: 14\nrating: 5\n	1	\N	\N	1793b05e-a044-455d-b82e-138bbc29c5eb	2015-02-06 19:01:08.090266
191	9	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with other BRAF V600 mutations also respond well to the V600E drug\n  dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 45\nvariant_id: 15\nrating: 5\n	1	\N	\N	c87c2ad2-a0cd-4b12-bd3b-067f8c889aa5	2015-02-06 19:01:08.112442
192	10	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A single patient with BRAF V600E/V600M bi-allelic mutation responded to the\n  V600E drug dabrafenib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 46\nvariant_id: 12\nrating: 3\n	1	\N	\N	e6659f61-a269-4e76-a706-ef249480eb0d	2015-02-06 19:01:08.135364
193	11	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and event-free survival was longer for\n  normal karyotype AML patients with Exon 12 NPM1  .\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 47\nvariant_id: 92\nrating: 5\n	1	\N	\N	71ecf83f-e25e-4986-9b52-98091fd1bd83	2015-02-06 19:01:08.161626
194	12	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with M2 FAB subtypes\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 47\nvariant_id: 92\nrating: 4\n	1	\N	\N	2145fa9c-5f66-4ddf-a826-e30929d33369	2015-02-06 19:01:08.202358
195	13	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M4, M5a and M5b FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 47\nvariant_id: 92\nrating: 4\n	1	\N	\N	23a40fd6-313f-49d9-b00d-3c3c6fded54e	2015-02-06 19:01:08.222485
196	14	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and\n  increased overall survival in young adult AML patients (median age 43)\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 92\nrating: 4\n	1	\N	\N	bacb1b46-bdc1-4fb4-a8e3-f1385fc56d0a	2015-02-06 19:01:08.245326
197	15	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 92\nrating: 4\n	1	\N	\N	06df1edd-2c58-4084-b65f-1edd5efaf195	2015-02-06 19:01:08.269721
198	16	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and\n  without FLT3-ITD have increased overall survival, relapse-free survival and response\n  to induction chemotherapy\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 50\nvariant_id: 92\nrating: 4\n	1	\N	\N	7e4f3735-33ca-4b6e-af7d-4f61a5aedb91	2015-02-06 19:01:08.29422
199	17	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with increased complete remission rates as well\n  as longer overall disease free survival in normal karyotype AML patients >59 years\n  old, particularly in those >69 years old\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 51\nvariant_id: 92\nrating: 4\n	1	\N	\N	c79a174b-5e2c-47a6-acf2-17501fcdb8c7	2015-02-06 19:01:08.321621
200	18	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with higher complete remission rates, lower cumulative\n  incidence of relapse and higher overall survival in intermediate risk AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 52\nvariant_id: 92\nrating: 4\n	1	\N	\N	73f23817-7e2b-456f-9734-c84b1717e491	2015-02-06 19:01:08.346956
201	19	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coocurrence of FLT3-ITD mutations significantly reduced overall survival of\n  normal karyotype AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 53\nvariant_id: 92\nrating: 4\n	1	\N	\N	9149d4f4-600e-4a4f-8a70-597b145ce923	2015-02-06 19:01:08.371459
202	20	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 92\nrating: 4\n	1	\N	\N	85d3d289-1719-46ef-a746-783d4c21b85c	2015-02-06 19:01:08.39526
203	21	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 92\nrating: 4\n	1	\N	\N	8004b48a-b6be-4a9b-8b9c-f4989dfdbeae	2015-02-06 19:01:08.414894
204	22	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No NPM1 mutations were identified in patients with favorable risk cytogenetics\n  (79/215 patients)\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 93\nrating: 4\n	1	\N	\N	cb3ed7e5-c79b-46ff-b5fd-2cba10121b79	2015-02-06 19:01:08.439738
205	23	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with intermediate risk cytogenetics (including\n  normal karyotype)\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 93\nrating: 4\n	1	\N	\N	d9486dda-afad-4ba9-ac82-db6250c30a73	2015-02-06 19:01:08.461004
206	24	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 5\ndisease_id: 7\nsource_id: 56\nvariant_id: 92\nrating: 3\n	1	\N	\N	145163da-9ffa-44b7-bbf2-942abf25fb4c	2015-02-06 19:01:08.48501
207	25	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells\n  harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 6\ndisease_id: 7\nsource_id: 56\nvariant_id: 92\nrating: 3\n	1	\N	\N	eea02590-b4e8-4319-b225-2cc481b18b39	2015-02-06 19:01:08.495522
208	26	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  NPM1 mutation regardless of FLT3-ITD status\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 7\nsource_id: 57\nvariant_id: 92\nrating: 3\n	1	\N	\N	c8211ad6-666b-4149-914c-8880d0ddc217	2015-02-06 19:01:08.51766
209	27	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Frequency of NPM1 mutations was not different in normal karyotype AML patients\n  with CEPBA, NRAS or KIT mutations.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 47\nvariant_id: 92\nrating: 3\n	1	\N	\N	442f5262-7fab-480e-9aff-ef20c0adfdcf	2015-02-06 19:01:08.539329
210	28	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were not associated with M3 FAB subtype (0/55).\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 53\nvariant_id: 92\nrating: 3\n	1	\N	\N	820bd069-cff3-4b16-bbb9-956fa2482f94	2015-02-06 19:01:08.56167
211	29	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with M5a and M5b FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 53\nvariant_id: 92\nrating: 3\n	1	\N	\N	82e793de-c318-4b63-ba53-c7820fed68f7	2015-02-06 19:01:08.584581
212	30	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with normal karyotype in older (>60) patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 58\nvariant_id: 92\nrating: 3\n	1	\N	\N	e6cbba12-2545-4861-8a23-c76d15983c2a	2015-02-06 19:01:08.607325
213	31	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations were associated with improved complete remission\n  rates as well as overall and disease free survival when all ages and karyotype were\n  analyzed; however, when only normal karyotype patients were analyzed, only complete\n  remission rates were improved\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 58\nvariant_id: 92\nrating: 3\n	1	\N	\N	d70e0f84-0036-4baa-a83e-11c8f56cef1c	2015-02-06 19:01:08.627575
214	32	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation status of IDH1 did not change event-free survival of patients with\n  an NPM1 mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 59\nvariant_id: 92\nrating: 3\n	1	\N	\N	ae4aa669-9c8b-49fe-9c29-d356eefa8615	2015-02-06 19:01:08.648733
215	33	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates were higher and both disease-free and overall survival\n  was longer for patients with Exon 12 NPM1  .\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 53\nvariant_id: 92\nrating: 3\n	1	\N	\N	aefda216-25d9-48df-b17d-fadf1da8d8b2	2015-02-06 19:01:08.669939
216	34	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 60\nvariant_id: 92\nrating: 3\n	1	\N	\N	055bbf5c-c451-4c41-b904-ca141df1b28d	2015-02-06 19:01:08.693249
217	35	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved\n  overall and relapse-free survival, complete remission rates and remission duration\n  compared to all other combinations of mutants\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 61\nvariant_id: 92\nrating: 3\n	1	\N	\N	358451d8-36a1-4112-9f2f-8492e2cb9ed0	2015-02-06 19:01:08.716681
218	36	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation was associated with increased overall, event-free and relapse-free\n  survival and remission after induction therapy in the entire cohort and in the cytogenetically\n  normal subset of patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 62\nvariant_id: 92\nrating: 3\n	1	\N	\N	00c64012-93ac-4a6a-8bca-a5f48002aa84	2015-02-06 19:01:08.738533
219	37	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TET2 mutation reduces overall survival of normal karyotype patients with either\n  an NPM1 mutation or NPM1 mutation without FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 63\nvariant_id: 92\nrating: 3\n	1	\N	\N	b5d87bf8-5e84-4430-ad21-f383276dc674	2015-02-06 19:01:08.760417
220	38	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival\n  and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate\n  risk AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 52\nvariant_id: 92\nrating: 3\n	1	\N	\N	444a6b37-5917-49d1-a0a6-61df49cd24f4	2015-02-06 19:01:08.787381
221	39	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected\n  into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM\n  in AML patients was associated with good response to induction therapy.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 64\nvariant_id: 93\nrating: 3\n	1	\N	\N	370b4009-7559-4441-8d42-95a860bca044	2015-02-06 19:01:08.812417
222	40	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment improved overall and relapse-free survival in older (>60) patients\n  with mutant NPM1 and without FLT3-ITD\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 7\nsource_id: 58\nvariant_id: 92\nrating: 2\n	1	\N	\N	a9c594d4-efe2-4f43-adc8-b169c228d557	2015-02-06 19:01:08.835501
223	41	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young patients (18-60) with NPM1 mutations had improved overall survival following\n  high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However,\n  the adjusted p-value for multiple testing is 0.11.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 7\nsource_id: 65\nvariant_id: 92\nrating: 2\n	1	\N	\N	159567e3-d7cd-4bab-846b-4951ddce45a0	2015-02-06 19:01:08.863107
224	42	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: For patients with NPM1 mutation that achieved CR following induction therapy,\n  relapse-free survival was improved following treatment with valproic acid\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 8\ndisease_id: 7\nsource_id: 66\nvariant_id: 92\nrating: 2\n	1	\N	\N	94cb26b8-c02f-4fbe-a472-82f99f0a5ba8	2015-02-06 19:01:08.887019
225	43	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 92\nrating: 2\n	1	\N	\N	42b760b8-0821-46db-ba30-b7dedf1e28c5	2015-02-06 19:01:08.907597
226	44	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse\n  in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction\n  in NPM1 levels following first line therapy (N=15)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 67\nvariant_id: 92\nrating: 2\n	1	\N	\N	7c09a099-2651-41b0-8da5-84f354dd2931	2015-02-06 19:01:08.932555
227	45	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Intermediate risk patients with mutant NPM1 had improved overall survival with\n  the presence either IDH1 or IDH2 mutations than those wildtype for both IDH1 and\n  IDH2\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 65\nvariant_id: 92\nrating: 2\n	1	\N	\N	a3efb0d8-664d-4e65-b54f-afb04476af27	2015-02-06 19:01:08.955505
228	46	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 expression was significantly increased in patients with NPM1 mutation with\n  or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 68\nvariant_id: 92\nrating: 2\n	1	\N	\N	3233c95e-edf7-4bcc-ae06-85a3eca78cfd	2015-02-06 19:01:08.977432
229	47	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CD33 and CD123 expression was significantly increased in patients with NPM1\n  mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33\n  and anti-CD123 therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 68\nvariant_id: 92\nrating: 2\n	1	\N	\N	da484b33-2dec-4421-8502-0e30e64eaf5d	2015-02-06 19:01:08.998596
230	48	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NPM1 mutation without FLT3-ITD was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 69\nvariant_id: 92\nrating: 3\n	1	\N	\N	75688f36-a2b3-4a45-9c56-7fed7f43ac13	2015-02-06 19:01:09.026521
231	49	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  but does not show resistance to MEK inhibitors.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 6\nsource_id: 70\nvariant_id: 11\nrating: 3\n	1	\N	\N	92602464-6109-4edb-853b-425599aa6b62	2015-02-06 19:01:09.049569
232	50	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP\n  and resistance to RAF inhibitors.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 6\nsource_id: 70\nvariant_id: 11\nrating: 3\n	1	\N	\N	03c955a3-3b85-46f6-9c0b-123fe39ad460	2015-02-06 19:01:09.073981
233	51	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 9\ndisease_id: 8\nsource_id: 71\nvariant_id: 117\nrating: 3\n	1	\N	\N	346ce20b-3e7c-48ad-a381-97f49961d2b5	2015-02-06 19:01:09.101367
234	52	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib,\n  compared to wild-type RET.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 9\ndisease_id: 8\nsource_id: 71\nvariant_id: 118\nrating: 3\n	1	\N	\N	30e30091-5fdd-4ae0-8c31-5571aa27f608	2015-02-06 19:01:09.12359
235	53	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with the ABL T315I mutation, tumors have shown to be resistant to\n  imatinib treatment.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 9\nsource_id: 72\nvariant_id: 2\nrating: 4\n	1	\N	\N	7e626cc6-5627-4ad8-92e4-ff54fe3d16b4	2015-02-06 19:01:09.152117
236	54	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib\n  treatment resulted in "anti-tumor activitiy".\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 10\nsource_id: 73\nvariant_id: 57\nrating: 3\n	1	\N	\N	bfd8180b-82c6-4a09-8e2f-e3bd8acd636a	2015-02-06 19:01:09.193157
237	55	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 11\nsource_id: 74\nvariant_id: 52\nrating: 3\n	1	\N	\N	bc79612d-67ef-42a4-8017-6b2fb1eb9298	2015-02-06 19:01:09.218823
238	56	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 11\nsource_id: 74\nvariant_id: 54\nrating: 3\n	1	\N	\N	5e269539-ba11-4552-a443-e079dc70ba12	2015-02-06 19:01:09.241356
239	57	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 11\nsource_id: 74\nvariant_id: 55\nrating: 3\n	1	\N	\N	57965333-94ee-485f-b065-c246fa1abe4f	2015-02-06 19:01:09.266482
240	58	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 10\nsource_id: 74\nvariant_id: 53\nrating: 3\n	1	\N	\N	3e401f1a-a081-43fa-8931-e4e925ed182f	2015-02-06 19:01:09.288897
241	59	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 12\nsource_id: 74\nvariant_id: 59\nrating: 3\n	1	\N	\N	3b4309b2-d912-41dc-8b0c-5bed29cbdbc0	2015-02-06 19:01:09.311261
242	60	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 13\nsource_id: 74\nvariant_id: 56\nrating: 3\n	1	\N	\N	232888e4-c3fc-44ad-9fa1-c3295c4e3b4c	2015-02-06 19:01:09.335445
243	61	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 12\ndisease_id: 14\nsource_id: 74\nvariant_id: 58\nrating: 3\n	1	\N	\N	27d9591b-0e37-45a5-b357-a744f855ff21	2015-02-06 19:01:09.356652
244	62	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 15\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	4b19d577-465f-42e1-9ff5-da601a7d5112	2015-02-06 19:01:09.378726
245	63	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 15\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	de58676a-f728-4ea5-aadc-19b7f417dbcf	2015-02-06 19:01:09.38828
246	64	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 10\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	5a5ffd0f-9d64-4c2c-98ec-ab16e48116aa	2015-02-06 19:01:09.40921
247	65	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 10\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	5a7500df-3b10-4216-8b1f-f57d4e7c6b5a	2015-02-06 19:01:09.420084
248	66	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 16\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	07452355-447d-42e6-a84e-5b05cd051e84	2015-02-06 19:01:09.446289
249	67	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 16\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	c0dd7a5d-aeef-4858-8d1f-387fb8215a7b	2015-02-06 19:01:09.456103
250	68	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 17\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	c3c7c0f2-7b1c-468c-b760-6acf7909d59e	2015-02-06 19:01:09.479552
251	69	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 17\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	5a01ab68-b126-4169-aa76-4a58bdf27318	2015-02-06 19:01:09.489725
252	70	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 18\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	7389de08-b361-4725-9c04-2123fdbf4731	2015-02-06 19:01:09.511734
253	71	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 18\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	e9e3548e-cf55-49c8-8ddd-501fa7462edf	2015-02-06 19:01:09.521011
254	72	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 12\ndisease_id: 19\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	cbe3cbfc-7244-4d35-9a3c-5958c8933d1d	2015-02-06 19:01:09.546104
255	73	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma.\n  Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This\n  fusion is also shown to be reccurent in multiple other cancer types.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 11\ndisease_id: 19\nsource_id: 73\nvariant_id: 60\nrating: 3\n	1	\N	\N	e0b29d80-2e16-49b9-a8d6-8f977e46e79e	2015-02-06 19:01:09.555512
256	74	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In acute myloid leukemia patients, D816 mutation is associated with earlier\n  relapse and poorer prognosis than wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 20\nsource_id: 75\nvariant_id: 72\nrating: 4\n	1	\N	\N	a9069fba-d924-427f-95e3-c39270f5fe58	2015-02-06 19:01:09.581624
257	75	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 6\nsource_id: 76\nvariant_id: 78\nrating: 3\n	1	\N	\N	cbd41830-d7ca-4a0e-a8c2-aed7a049ec0b	2015-02-06 19:01:09.609049
258	76	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 6\nsource_id: 76\nvariant_id: 78\nrating: 3\n	1	\N	\N	d7913148-d3d2-4a16-8add-e89d613ff71c	2015-02-06 19:01:09.619968
259	77	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive\n  to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation\n  alters the conformation of KIT and reduces the binding affinity of imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 6\nsource_id: 76\nvariant_id: 78\nrating: 3\n	1	\N	\N	8f2c2c11-d1f7-4f75-ad4b-8074a499e1ee	2015-02-06 19:01:09.630571
260	78	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 3\nsource_id: 77\nvariant_id: 78\nrating: 3\n	1	\N	\N	1ed3a545-f72a-4271-bc32-f1efac59fe8f	2015-02-06 19:01:09.659867
261	79	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 3\nsource_id: 77\nvariant_id: 78\nrating: 3\n	1	\N	\N	5458aca4-a73e-4626-a841-b80a6e8b3e00	2015-02-06 19:01:09.671559
262	80	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib\n  and imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 13\ndisease_id: 3\nsource_id: 77\nvariant_id: 78\nrating: 3\n	1	\N	\N	27eb8d47-a509-4f33-90be-c535d8da007d	2015-02-06 19:01:09.681653
263	81	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective\n  for those that are refractory to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 16\ndisease_id: 21\nsource_id: 78\nvariant_id: 79\nrating: 3\n	1	\N	\N	51e82c3a-f3b1-418c-8176-41fd889572c2	2015-02-06 19:01:09.706791
264	82	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive\n  to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation\n  of KIT but did not alter the binding affinity of dasatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 15\ndisease_id: 6\nsource_id: 76\nvariant_id: 78\nrating: 3\n	1	\N	\N	70b40dcd-b381-4301-a806-8e9b286dd1b5	2015-02-06 19:01:09.727516
265	83	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT V654A results in imatinib resistance in GIST patient-derived cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 21\nsource_id: 79\nvariant_id: 79\nrating: 3\n	1	\N	\N	03d65a61-d80f-40bb-ab92-f0de70ce1571	2015-02-06 19:01:09.752079
266	84	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a case study, a patient with anal melanoma harboring an exon 11 KIT mutation\n  showed marked response 4 months after imatinib treatment.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 6\nsource_id: 80\nvariant_id: 78\nrating: 3\n	1	\N	\N	32ceaf96-d292-4c96-a99e-f53c56ea0f18	2015-02-06 19:01:09.774474
267	85	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 47\nrating: 5\n	1	\N	\N	1bbd3280-e4c1-486b-9d11-e56efcd9242a	2015-02-06 19:01:09.799939
268	86	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 47\nrating: 5\n	1	\N	\N	0a2700d7-97e4-42d0-ab00-f4ecf6899544	2015-02-06 19:01:09.811559
269	87	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 48\nrating: 5\n	1	\N	\N	02c27d5f-984b-4a32-8dbd-7082ed9f3929	2015-02-06 19:01:09.834493
270	88	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 48\nrating: 5\n	1	\N	\N	570c9bab-31d5-4c95-8109-239619498e37	2015-02-06 19:01:09.844814
271	89	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 49\nrating: 5\n	1	\N	\N	8adcc20f-a954-497e-b0e3-60c31c35f0d0	2015-02-06 19:01:09.869347
272	90	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 49\nrating: 5\n	1	\N	\N	59553554-accc-4eee-9709-eed67f2dca2f	2015-02-06 19:01:09.878957
273	91	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 50\nrating: 5\n	1	\N	\N	afbb1c7c-1f1e-4906-b7f8-7f993fb13234	2015-02-06 19:01:09.901554
274	92	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 50\nrating: 5\n	1	\N	\N	9651f467-7471-45b0-baa2-80175a0840c0	2015-02-06 19:01:09.911561
275	93	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 17\ndisease_id: 11\nsource_id: 81\nvariant_id: 51\nrating: 5\n	1	\N	\N	ec796be6-2f0f-4ebc-af29-07507d97ef85	2015-02-06 19:01:09.933754
276	94	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: 5 ligand binding domain mutations have been shown to result in constitutive\n  activity and continued responsiveness to anti-estrogen therapies in vitro.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 18\ndisease_id: 11\nsource_id: 81\nvariant_id: 51\nrating: 5\n	1	\N	\N	1a88e586-ae07-4f84-8a0f-3b4eb0f40e6a	2015-02-06 19:01:09.943509
277	95	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 47\nrating: 3\n	1	\N	\N	595634ae-7a1d-4b99-b041-d8ac97705ba7	2015-02-06 19:01:09.966265
278	96	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 48\nrating: 3\n	1	\N	\N	3e6daa1f-1172-4b96-aec3-27e4a2da4a72	2015-02-06 19:01:09.988223
279	97	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 49\nrating: 3\n	1	\N	\N	30acf81b-a051-47b5-a2aa-b0280a61f4f6	2015-02-06 19:01:10.011437
280	98	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 50\nrating: 3\n	1	\N	\N	894b5da5-520b-4654-a8fa-f43197644b35	2015-02-06 19:01:10.033762
281	99	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Surveys implicate ligand binding domain mutants in mediating clinical resistance\n  to hormonal therapy and suggest that more potent ER antagonists may be of substantial\n  therapeutic benefit.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 82\nvariant_id: 51\nrating: 3\n	1	\N	\N	4f1ecade-e9ea-4a37-8483-40e580d73d34	2015-02-06 19:01:10.06669
282	100	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from an ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 22\nsource_id: 83\nvariant_id: 114\nrating: 4\n	1	\N	\N	630c2bbc-ccdc-4bb3-a137-e3303afd5e69	2015-02-06 19:01:10.091405
283	101	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting\n  from an ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular\n  carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary\n  and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation\n  and Western blot analyses confirmed that the chimeric protein is expressed in tumor\n  tissue, and a cell culture assay indicated that it retains kinase activity.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 22\nsource_id: 83\nvariant_id: 32\nrating: 4\n	1	\N	\N	b603a865-473d-48c9-9095-39ffadb180e7	2015-02-06 19:01:10.114084
284	102	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 3\nsource_id: 84\nvariant_id: 7\nrating: 3\n	1	\N	\N	d4fac264-9288-4f4f-8911-8b7d38a6bbdc	2015-02-06 19:01:10.137666
285	103	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 19\ndisease_id: 3\nsource_id: 85\nvariant_id: 6\nrating: 4\n	1	\N	\N	d2988859-0146-48f5-b793-494601607c75	2015-02-06 19:01:10.163056
286	104	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring EML4-ALK fusion, this\n  variant has been shown to confer resistance to crizotinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 19\ndisease_id: 3\nsource_id: 85\nvariant_id: 5\nrating: 4\n	1	\N	\N	c72d3158-4dfa-4de5-8a1b-255ead89bde6	2015-02-06 19:01:10.190442
287	105	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment\n  with crizotinib has shown to be effective.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 19\ndisease_id: 3\nsource_id: 85\nvariant_id: 4\nrating: 4\n	1	\N	\N	516b46a6-568d-43fc-a6a2-cd6481518700	2015-02-06 19:01:10.219132
288	106	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein undergoes proteolysis in APL cells upon ATRA treatment in cell\n  lines.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 23\nsource_id: 86\nvariant_id: 113\nrating: 5\n	1	\N	\N	819f582f-ccbd-4c3b-860f-f0ed7e8adbf5	2015-02-06 19:01:10.242467
289	107	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Fusion protein degradation is accelerated upon ATRA treatment at pharmalogical\n  concentrations.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 23\nsource_id: 86\nvariant_id: 116\nrating: 5\n	1	\N	\N	ec4b2aa2-fa9d-42b6-b0fc-57bcb885264e	2015-02-06 19:01:10.258996
290	108	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 24\nsource_id: 87\nvariant_id: 31\nrating: 3\n	1	\N	\N	04a51a84-2bcc-48c4-ac2c-7b61e69f33dc	2015-02-06 19:01:10.280033
291	109	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 24\nsource_id: 87\nvariant_id: 31\nrating: 3\n	1	\N	\N	da1ad56a-fb84-47a1-b06c-b3de4e55ee9a	2015-02-06 19:01:10.29231
292	110	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 10\ndisease_id: 24\nsource_id: 87\nvariant_id: 90\nrating: 3\n	1	\N	\N	a6cf9721-a080-448d-bf0f-9311d45b63ac	2015-02-06 19:01:10.31256
293	111	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors\n  Imatinib and GW-2580\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 20\ndisease_id: 24\nsource_id: 87\nvariant_id: 90\nrating: 3\n	1	\N	\N	622ae904-0553-462f-90bf-5a701ee1a0c3	2015-02-06 19:01:10.322473
294	112	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 88\nvariant_id: 119\nrating: 3\n	1	\N	\N	bbd06361-8f22-47ba-9721-590adc4fe637	2015-02-06 19:01:10.34279
295	113	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with SF3B1 mutations had a statistically significant longer overall\n  survival as well as event free-survival. Both before and after adjustment for age,\n  karyotype and sex.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 88\nvariant_id: 120\nrating: 3\n	1	\N	\N	94d1e432-5ce1-4e88-ae48-f7c15f4467f2	2015-02-06 19:01:10.359642
296	114	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 134\nrating: 4\n	1	\N	\N	bb040c92-fca7-4944-9307-2dcb36534071	2015-02-06 19:01:10.377037
297	115	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rates of complete remission and refractory disease are not different in patients\n  with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60)\n  patients with cytogenetically normal AML.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 135\nrating: 4\n	1	\N	\N	f82dd9b1-b5d3-43b0-9f90-5c835fe4fbda	2015-02-06 19:01:10.393731
298	116	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 134\nrating: 3\n	1	\N	\N	6a659b6e-a28e-4379-af18-aa44767ca470	2015-02-06 19:01:10.41103
299	117	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No difference in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 134\nrating: 3\n	1	\N	\N	d1d0871d-6636-49b5-8fb5-93c2686f69a8	2015-02-06 19:01:10.427017
300	118	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 134\nrating: 3\n	1	\N	\N	7bffc2aa-68cb-4739-ad6f-5c297ef0489e	2015-02-06 19:01:10.444639
301	119	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 134\nrating: 3\n	1	\N	\N	3f09d606-6bca-4bbb-ba9e-2874823069a3	2015-02-06 19:01:10.460993
302	120	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 91\nvariant_id: 134\nrating: 3\n	1	\N	\N	6eaf03ba-c10e-4ef1-9b0f-87142c201f2c	2015-02-06 19:01:10.478422
303	121	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 134\nrating: 3\n	1	\N	\N	8ee89ccc-daae-4745-88bb-ba63628cb966	2015-02-06 19:01:10.496867
304	122	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with an inferior response to induction chemotherapy\n  with a higher rate of resistant disease in young (15-60+, median 45) patients with\n  cytogenetically normal AML\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 135\nrating: 3\n	1	\N	\N	97af6644-9003-4bd3-9357-f0c3a2f1b7d5	2015-02-06 19:01:10.51296
305	123	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: No difference in relapse-free or overall survival were identified between young\n  (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically\n  normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 135\nrating: 3\n	1	\N	\N	314fc902-2be5-4b5b-96ff-6437c14bbe92	2015-02-06 19:01:10.528448
306	124	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were associated with a higher cumulative incidence of relapse\n  and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically\n  normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 135\nrating: 3\n	1	\N	\N	9940acae-ccf4-4b41-87f2-29f4b4a7677f	2015-02-06 19:01:10.544652
307	125	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT1 mutations were a negative prognostic factor for overall survival in young\n  (15-60+, median 45) patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 90\nvariant_id: 135\nrating: 3\n	1	\N	\N	f0b6bacb-9e9d-41de-9c7d-9c241c66f704	2015-02-06 19:01:10.562207
308	126	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WT mutations were associated with shorter overall and disease free survival\n  in a cohort of cytogenetically normal, young (<60) AML patients\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 91\nvariant_id: 135\nrating: 3\n	1	\N	\N	c5e18c3d-4cfa-4014-9ef8-ffd237ab0e49	2015-02-06 19:01:10.578118
309	127	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly\n  lower complete remission and higher refractory disease rates than those with wildtype\n  WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with\n  cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 89\nvariant_id: 135\nrating: 3\n	1	\N	\N	aeac2c65-c6c0-4bc9-9cf1-60b8153fcff2	2015-02-06 19:01:10.595482
310	128	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: in WT1 were associated with a worse overall prognosis than patients wildtype\n  for WT1 in young patients (15-50), primarily because of increased risk of disease\n  recurrence\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 92\nvariant_id: 134\nrating: 2\n	1	\N	\N	b0b7c41c-464b-4be3-a120-67230dffa2e2	2015-02-06 19:01:10.615111
311	129	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: in WT1 were associated with increased risk of recurrence in young patients (15-50)\n  with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 92\nvariant_id: 134\nrating: 2\n	1	\N	\N	0b5ac494-910f-441b-a3c0-961789af8854	2015-02-06 19:01:10.631231
312	130	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 93\nvariant_id: 132\nrating: 3\n	1	\N	\N	66c3f3fb-d1ff-4806-a24e-a944ff49749d	2015-02-06 19:01:10.648608
313	131	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Age, sex, FAB subtype and karyotypes were not statistically significant between\n  AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 93\nvariant_id: 133\nrating: 3\n	1	\N	\N	afe17bf6-d541-438b-95c7-ecac5da06a07	2015-02-06 19:01:10.665015
314	132	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 94\nvariant_id: 132\nrating: 3\n	1	\N	\N	57a924cb-cd02-4e41-9e91-037523122840	2015-02-06 19:01:10.682399
315	133	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: After adjust for age and cancer stage, presence of U2AF mutation such as S34Y/F\n  is prognostic for poorer survival outcomes in patients with MDS.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 94\nvariant_id: 133\nrating: 3\n	1	\N	\N	8eae7e61-1449-411d-8c6d-a34018dbf837	2015-02-06 19:01:10.698527
316	134	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 93\nvariant_id: 132\nrating: 2\n	1	\N	\N	66f2a1fa-02b6-480c-bf57-fc90f118922d	2015-02-06 19:01:10.716223
317	135	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 93\nvariant_id: 133\nrating: 2\n	1	\N	\N	1db3d0a2-a830-4de8-a69f-7428e92d65a4	2015-02-06 19:01:10.734137
318	136	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show\n  statistical significance in complete remission rate compared to those who harbor\n  wild type U2AF.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 93\nvariant_id: 132\nrating: 2\n	1	\N	\N	4eff7281-9cc9-4d6f-9825-e2727afa5dff	2015-02-06 19:01:10.754562
319	137	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with AML, complete remission rates are not different between patients\n  who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 93\nvariant_id: 133\nrating: 2\n	1	\N	\N	753dbb0e-1201-48d6-8c4d-af8636ce36e9	2015-02-06 19:01:10.773723
320	138	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast tumor with R175H mutation are more responsive to doxorubicin than breast\n  tumor with wild type TP53.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 21\ndisease_id: 11\nsource_id: 95\nvariant_id: 121\nrating: 3\n	1	\N	\N	49ff3bd2-6eb2-44a8-a249-470aa782bcb4	2015-02-06 19:01:10.793359
321	139	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R175H mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 121\nrating: 3\n	1	\N	\N	c468c750-86fb-4077-975f-23095b2f8ec6	2015-02-06 19:01:10.812553
322	140	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R248 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 122\nrating: 3\n	1	\N	\N	ba076adb-b01f-4c89-bad4-bb56cfd38f0c	2015-02-06 19:01:10.832076
323	141	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R248Q mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 122\nrating: 3\n	1	\N	\N	614b9178-83ea-40ef-a435-242b2294d33c	2015-02-06 19:01:10.847989
324	142	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring R248W mutation, the prognosis is worse than\n  any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 123\nrating: 3\n	1	\N	\N	56081efa-06af-475b-8bdf-0ad5b4898ead	2015-02-06 19:01:10.876381
325	143	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as R249 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 124\nrating: 3\n	1	\N	\N	69e7e1e1-dfe3-401e-8216-6332f7c61f68	2015-02-06 19:01:10.891963
326	144	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as R249 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 125\nrating: 3\n	1	\N	\N	3e7308f7-bfde-4c04-b679-e820bf2b6a34	2015-02-06 19:01:10.90835
327	145	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R273 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 126\nrating: 3\n	1	\N	\N	9ed7c304-9901-47ca-9cfb-cb0a5a25163e	2015-02-06 19:01:10.925358
328	146	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273C mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 126\nrating: 3\n	1	\N	\N	113a62e7-2300-4a22-aaf9-fdfc0a105166	2015-02-06 19:01:10.940845
329	147	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients  harboring TP53 mutation, mutations in DNA contact\n  region such as R273 is prognostic for a worse relapse-free survival compared to\n  other non-silent mutations in TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 127\nrating: 3\n	1	\N	\N	481c89bc-c3e1-48a8-9c6c-e9b3459745ca	2015-02-06 19:01:10.957721
330	148	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer patients who harbor R273H mutation has worse overall survival\n  than those with wild type TP53, but have better prognosis than those with R248W\n  mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 96\nvariant_id: 127\nrating: 3\n	1	\N	\N	83e19a98-e108-4195-a816-882df3ee514c	2015-02-06 19:01:10.974749
331	149	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In breast cancer patients harboring TP53 mutation, mutations in conserved region\n  such as V173 is prognostic for a worse overall survival compared to those harboring\n  wild-type TP53.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 97\nvariant_id: 128\nrating: 3\n	1	\N	\N	7ecbe1f8-b9b1-470b-bf99-2cbfe635d00e	2015-02-06 19:01:10.991071
332	150	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with medullary carcinoma, the presence of RET M918T mutation is\n  associated with increased probability of lymph node metastases.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 98\nvariant_id: 118\nrating: 5\n	1	\N	\N	250d82a0-ebb0-47d1-8453-89f45b6e53bb	2015-02-06 19:01:11.010374
333	151	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Compared to those who harbor a wild type RET, patients with RET M918T mutation\n  develop larger and more aggressive medullary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 8\nsource_id: 99\nvariant_id: 118\nrating: 4\n	1	\N	\N	5c15b61d-7074-41db-a346-d81e73fc205a	2015-02-06 19:01:11.02846
334	152	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of AZD1480 in vitro led to strong repression of tyroid cancer cell growth.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 22\ndisease_id: 8\nsource_id: 100\nvariant_id: 118\nrating: 3\n	1	\N	\N	393a867e-7a0b-4751-90f7-7371307fb78d	2015-02-06 19:01:11.048125
335	153	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 P124S confers increased resistance to inhibition by AZD6244 inhibition\n  by about 5 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 23\ndisease_id: 6\nsource_id: 101\nvariant_id: 88\nrating: 3\n	1	\N	\N	1d1ec072-1a15-4e55-a669-d66d204a0094	2015-02-06 19:01:11.068363
336	154	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 23\ndisease_id: 6\nsource_id: 101\nvariant_id: 89\nrating: 3\n	1	\N	\N	6568a636-d560-4618-9c66-75f87af865bf	2015-02-06 19:01:11.085041
337	155	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 11\nsource_id: 102\nvariant_id: 108\nrating: 4\n	1	\N	\N	5e9c69ad-c606-40ae-aa13-b6eb1fd2a7b4	2015-02-06 19:01:11.103419
338	156	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity\n  to CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 11\nsource_id: 102\nvariant_id: 109\nrating: 4\n	1	\N	\N	d51a97cb-4bc7-4818-92bc-cb5a7993c309	2015-02-06 19:01:11.118868
339	157	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Breast cancer cell lines with H1047R mutation showed increased sensitivity to\n  CH5132799 than cells with wild-type PIK3CA gene.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 24\ndisease_id: 11\nsource_id: 102\nvariant_id: 112\nrating: 4\n	1	\N	\N	47908a69-2ddd-46cb-8545-a6563720c498	2015-02-06 19:01:11.134555
340	158	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 26\nsource_id: 103\nvariant_id: 108\nrating: 4\n	1	\N	\N	1cb33d17-27ca-4aeb-ad19-51a74271c8d1	2015-02-06 19:01:11.15488
341	159	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 26\nsource_id: 103\nvariant_id: 109\nrating: 4\n	1	\N	\N	a40b784b-ecc7-44d2-8928-aef48908b888	2015-02-06 19:01:11.174601
342	160	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 26\nsource_id: 103\nvariant_id: 112\nrating: 4\n	1	\N	\N	07a097df-3222-4f7c-ba5d-996bdc96931c	2015-02-06 19:01:11.191439
343	161	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 26\nsource_id: 104\nvariant_id: 108\nrating: 3\n	1	\N	\N	03e5e35b-ce54-4d7d-877e-779558d17f83	2015-02-06 19:01:11.210956
344	162	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 26\nsource_id: 104\nvariant_id: 108\nrating: 3\n	1	\N	\N	7b5221e0-add1-4414-b1f5-a8de9437a416	2015-02-06 19:01:11.221396
345	163	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 26\nsource_id: 104\nvariant_id: 109\nrating: 3\n	1	\N	\N	366134ea-6606-4ca7-82a4-85a3095938a7	2015-02-06 19:01:11.237826
346	164	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance\n  to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 26\nsource_id: 104\nvariant_id: 109\nrating: 3\n	1	\N	\N	a2ac78bd-e201-4aa6-a18b-a3187c2109c5	2015-02-06 19:01:11.247128
347	165	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase\n  resistance to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 26\ndisease_id: 26\nsource_id: 104\nvariant_id: 112\nrating: 3\n	1	\N	\N	de31e61e-9abd-442f-a469-1f21f21286a2	2015-02-06 19:01:11.266484
348	166	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase\n  resistance to drugs such as  panitumumab or cetuximab.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 27\ndisease_id: 26\nsource_id: 104\nvariant_id: 112\nrating: 3\n	1	\N	\N	f83e3042-6116-4aac-837b-fcbf98d19f5b	2015-02-06 19:01:11.278468
349	167	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA\n  mutations had lower response and disease control rates and shorter progression free\n  and overall survival following cetuximab plus chemotherapy than those with wildtype\n  PIK3CA\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 110\nrating: 3\n	1	\N	\N	5ffb970d-db78-4ac3-a25b-0574ae16621d	2015-02-06 19:01:11.297914
350	168	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations\n  (Exon 9) had no significant impact on response rate, disease control rate, progression\n  free surivival or overall survival following cetuximab plus chemotherapy than those\n  with wildtype PIK3CA\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 111\nrating: 3\n	1	\N	\N	04137501-0f66-4657-b1fe-75ade4fe6e36	2015-02-06 19:01:11.314365
351	169	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 11\nsource_id: 106\nvariant_id: 112\nrating: 3\n	1	\N	\N	8ae5744e-bac3-4294-bd3f-cfdeb8cc4ca3	2015-02-06 19:01:11.33118
352	170	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 11\nsource_id: 106\nvariant_id: 108\nrating: 3\n	1	\N	\N	82124deb-e6a4-49ba-bb3b-40c3919f9b11	2015-02-06 19:01:11.346997
353	171	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Rapamycin inhibits transformation induced by mutation in PIK3CA\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 2\ndisease_id: 11\nsource_id: 106\nvariant_id: 109\nrating: 3\n	1	\N	\N	de111788-5f0a-420e-8952-1339c0114233	2015-02-06 19:01:11.362961
354	172	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E542K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 104\nvariant_id: 108\nrating: 3\n	1	\N	\N	01c15c50-b9ab-4223-bcdb-1e68e741fc61	2015-02-06 19:01:11.37885
355	173	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 104\nvariant_id: 109\nrating: 3\n	1	\N	\N	04389571-7f4e-469f-a811-8e90b2dae714	2015-02-06 19:01:11.401037
356	174	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer\n  survival compared to those with wild-type PIK3CA mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 104\nvariant_id: 112\nrating: 3\n	1	\N	\N	100591ad-88be-44e9-b501-0d1a9d564e3f	2015-02-06 19:01:11.417942
357	175	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 107\nvariant_id: 108\nrating: 2\n	1	\N	\N	0bca8014-4db3-4008-864f-3dfac706c0b8	2015-02-06 19:01:11.435589
358	176	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either\n  alone, is prognostic for worse overall survival and cancer specific survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 26\nsource_id: 107\nvariant_id: 109\nrating: 2\n	1	\N	\N	7eed1c88-b3cd-4680-bd5e-f1da1695a207	2015-02-06 19:01:11.451263
359	177	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 103\nrating: 4\n	1	\N	\N	ced9cd28-ad12-4399-96af-1a94113dd493	2015-02-06 19:01:11.476366
360	178	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842V mutation resulting in imatinib resistance, crenolanib\n  showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 104\nrating: 4\n	1	\N	\N	f056e7b5-194a-477d-ba7c-0272b43090a8	2015-02-06 19:01:11.492387
361	179	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA D842Y mutation resulting in imatinib resistance, crenolanib\n  showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 105\nrating: 4\n	1	\N	\N	2a4c8ad5-4dc5-42b6-a775-1c6d805e8eb4	2015-02-06 19:01:11.509184
362	180	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA deletion I843 mutation resulting in imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 106\nrating: 4\n	1	\N	\N	96fb4a65-8923-4a17-bee7-f158e53b6965	2015-02-06 19:01:11.527538
363	181	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In CHO cells with PDGFRA DI842-843IM mutation resulting in imatinib resistance,\n  crenolanib showed relatively high potency.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 28\ndisease_id: 27\nsource_id: 108\nvariant_id: 107\nrating: 4\n	1	\N	\N	1cb5c0f4-efec-42a5-a60a-fdaea2cc991d	2015-02-06 19:01:11.543993
364	182	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib,\n  double mutation of V561D and D842V mutants are resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 27\nsource_id: 79\nvariant_id: 104\nrating: 4\n	1	\N	\N	4728549c-6197-4a00-a458-26878b1b1add	2015-02-06 19:01:11.560272
365	183	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST cancer with D842V mutation is resistant to imatinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 27\nsource_id: 109\nvariant_id: 104\nrating: 4\n	1	\N	\N	b352f4f0-5615-4f18-a035-173d32f07032	2015-02-06 19:01:11.57955
366	184	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST tumor harboring PDGFRA D842V mutation are likely to be benign\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 27\nsource_id: 110\nvariant_id: 104\nrating: 3\n	1	\N	\N	9f2270e3-85c0-46c1-863f-62ca88c4d856	2015-02-06 19:01:11.603212
367	185	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 29\ndisease_id: 6\nsource_id: 111\nvariant_id: 100\nrating: 4\n	1	\N	\N	1df48cdc-c4bb-4a7b-88c5-4c60f6a76076	2015-02-06 19:01:11.634615
368	186	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status\n  (primarily Q61) was not informative for disease control rate, progression free survival\n  or overall survival following cetuximab plus chemotherapy.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 100\nrating: 3\n	1	\N	\N	1c5c2e84-0d95-4650-a53a-909b3f9c6720	2015-02-06 19:01:11.652653
369	187	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation\n  (primarily Q61) have a significantly lower response rate to cetuximab than patients\n  wildtype for NRAS.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 100\nrating: 3\n	1	\N	\N	2d049503-66cd-4fdc-8056-e524afc6bd6c	2015-02-06 19:01:11.668979
370	188	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of NRAS mutation in AML patient does not impact diease prognosis\n  (resistant disease, disease-free survival, complete remission rate, relapse rate,\n  induction death)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 112\nvariant_id: 95\nrating: 3\n	1	\N	\N	95fc7fe8-9dae-4add-8d04-7b23297332f1	2015-02-06 19:01:11.689571
371	189	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: NRAS mutation status does not impact overall, event-free or disease-free survival\n  in patients with AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 113\nvariant_id: 98\nrating: 3\n	1	\N	\N	9188046e-51fc-4fa1-a36b-6ef41c8f320c	2015-02-06 19:01:11.706827
372	190	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 114\nvariant_id: 95\nrating: 3\n	1	\N	\N	d2f066ad-6746-4a2a-b3ff-b123af16c709	2015-02-06 19:01:11.726253
373	191	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 114\nvariant_id: 96\nrating: 3\n	1	\N	\N	4072e48a-9d77-49f6-b89c-35290ecfeb2c	2015-02-06 19:01:11.74253
374	192	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer harboring NRAS mutation has poorer survival\n  outcome and worse prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 26\nsource_id: 114\nvariant_id: 97\nrating: 3\n	1	\N	\N	5f025918-3b1b-4f94-b219-01da4d34c2a7	2015-02-06 19:01:11.762583
375	193	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 115\nvariant_id: 94\nrating: 3\n	1	\N	\N	36a6c879-ed77-4bed-b82f-e8a1183b06c6	2015-02-06 19:01:11.78347
376	194	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage IV melanoma, NRAS mutation was associated with reduced\n  median survival compared to patients with wildtype NRAS\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 115\nvariant_id: 95\nrating: 3\n	1	\N	\N	b1dc8b52-5b75-440d-a767-4d9f933887e8	2015-02-06 19:01:11.802324
377	195	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma associated with NRAS Q61 mutation was more often associated with those\n  at the extremity than those at the trunk\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 116\nvariant_id: 100\nrating: 3\n	1	\N	\N	fda53398-1d51-4d94-95af-4e37654c21cc	2015-02-06 19:01:11.820113
378	196	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition\n  of HSP90 by 17-AAG is shown to be effective in a patient with metastatic malignant\n  melenoma with a NRAS G13D mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 30\ndisease_id: 6\nsource_id: 117\nvariant_id: 99\nrating: 2\n	1	\N	\N	e9dbc305-cf37-4553-baa2-cdda2418f65f	2015-02-06 19:01:11.838945
379	197	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted\n  in disease free survival of 14 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 6\nsource_id: 118\nvariant_id: 101\nrating: 2\n	1	\N	\N	c82c9b96-4c32-424c-b854-ad48275db27d	2015-02-06 19:01:11.860619
380	198	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a melanoma patient with Q61R mutation, treatment with temozolomide resulted\n  in overall survival of 16 months\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 31\ndisease_id: 6\nsource_id: 118\nvariant_id: 102\nrating: 2\n	1	\N	\N	d8a37701-8543-4301-9fef-c2427e76c505	2015-02-06 19:01:11.877145
381	199	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: '"AML with mutated NPM1" is a provisional entity in WHO classification of acute\n  myeloid leukemia (AML). This mutation should be tested for in clinical trials and\n  is recommended for testing in patients with cytogentically normal AML.'\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 119\nvariant_id: 92\nrating: 5\n	1	\N	\N	6f24e746-96dc-494e-a49b-b93f694834b0	2015-02-06 19:01:11.895848
382	200	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted\n  from temozolomide. This benefit was also methylation status dependent, as those\n  without methylation did not see increase survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 32\ndisease_id: 5\nsource_id: 120\nvariant_id: 91\nrating: 5\n	1	\N	\N	41d17ef9-40ce-43f7-9b21-a4072a88d571	2015-02-06 19:01:11.91578
383	201	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor,\n  such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may\n  be useful in treating patients with initial resistance to alkylating agents.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 33\ndisease_id: 5\nsource_id: 121\nvariant_id: 91\nrating: 4\n	1	\N	\N	9430c095-74da-4adf-b3f7-dbbdc47f4584	2015-02-06 19:01:11.938579
384	202	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: MGMT promoter methylation has been shown to confer sensitivity to alkylating\n  agents, such as carmustine, in glioblastoma cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 34\ndisease_id: 5\nsource_id: 121\nvariant_id: 91\nrating: 4\n	1	\N	\N	02271beb-d7e8-45ac-b0d0-72328b231922	2015-02-06 19:01:11.957779
385	203	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with colorectal cancer who harbor KRAS mutation have low response rate\n  to cetuximab\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 122\nvariant_id: 80\nrating: 4\n	1	\N	\N	cb35b323-a603-42bb-98c7-99443ad8c9df	2015-02-06 19:01:11.975467
386	204	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 35\ndisease_id: 3\nsource_id: 123\nvariant_id: 85\nrating: 4\n	1	\N	\N	65f2c309-7bc2-473e-8053-04f41f4e696a	2015-02-06 19:01:11.996955
387	205	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy,\n  in lung cancer model with KRAS G12D mutation led to marked tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 36\ndisease_id: 3\nsource_id: 123\nvariant_id: 85\nrating: 4\n	1	\N	\N	f7d823dd-aa7d-43f0-8007-452f44a8f3b1	2015-02-06 19:01:12.009599
388	206	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft models\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 28\nsource_id: 124\nvariant_id: 87\nrating: 4\n	1	\N	\N	c1a0dea0-57bc-44f7-8d65-5ef4fb5adb58	2015-02-06 19:01:12.031392
389	207	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 28\nsource_id: 124\nvariant_id: 87\nrating: 4\n	1	\N	\N	2cf9343c-2055-40d4-b8a1-77098e65e647	2015-02-06 19:01:12.047325
390	208	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in\n  isogenic SW48 cells and in a mouse xenograft models\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 28\nsource_id: 124\nvariant_id: 87\nrating: 4\n	1	\N	\N	14831975-473b-482a-a197-84349d4a446f	2015-02-06 19:01:12.066573
391	209	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G13D mutation is associated with better response to Cetuximab with longer\n  progression-free and overall survival in colorectal patients compared to other KRAS\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 28\nsource_id: 124\nvariant_id: 87\nrating: 4\n	1	\N	\N	1960affc-8426-49d6-bbc6-550211b12457	2015-02-06 19:01:12.086601
392	210	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutation status was not predictive of response to regorafenib treatment\n  in patients that had received standard therapy and progressed within 3 months of\n  their last treatment\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 25\ndisease_id: 26\nsource_id: 103\nvariant_id: 81\nrating: 4\n	1	\N	\N	fb8e2171-8ae8-4f73-a1bc-426c131ade82	2015-02-06 19:01:12.102796
393	211	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12\n  mutation did not impact overall or disease free survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 28\nsource_id: 125\nvariant_id: 82\nrating: 4\n	1	\N	\N	e6045ba7-3430-4ad7-89ee-085b5de16424	2015-02-06 19:01:12.119797
394	212	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 3\nsource_id: 126\nvariant_id: 81\nrating: 3\n	1	\N	\N	336f3572-a7b9-4c86-bd05-63230880c856	2015-02-06 19:01:12.141449
395	213	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR\n  kinase inhibitors gefitinib and erlotinib\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 3\nsource_id: 126\nvariant_id: 81\nrating: 3\n	1	\N	\N	5ee13692-bc07-43c2-b147-1cbf49755546	2015-02-06 19:01:12.152557
396	214	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with lack of response to cetuximab\n  in patients with advanced colorectal cancer\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 26\nsource_id: 127\nvariant_id: 83\nrating: 3\n	1	\N	\N	385a098f-e820-4837-9e40-ce1f1a8c53eb	2015-02-06 19:01:12.17137
397	215	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations\n  (primarily G12/G13) had lower response and disease control rates and shorter progression\n  free and overall survival following cetuximab plus chemotherapy than those with\n  wildtype KRAS\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 83\nrating: 3\n	1	\N	\N	bbda3abf-dd8c-4a03-8116-8f5581820e84	2015-02-06 19:01:12.189094
398	216	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations were significantly associated with reduced progression-free and\n  overall survival in patients with advanced colorectal cancer\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 26\nsource_id: 127\nvariant_id: 83\nrating: 3\n	1	\N	\N	0de8ef7a-eb25-45ea-93cb-17a174dd0008	2015-02-06 19:01:12.205513
399	217	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 29\nsource_id: 128\nvariant_id: 82\nrating: 3\n	1	\N	\N	4c0da831-6eee-4b69-9c87-c4ce218068e6	2015-02-06 19:01:12.223881
400	218	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Children with early age leukaemia who had second hand smoke exposure are more\n  likely to harbor KRAS mutation.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 29\nsource_id: 128\nvariant_id: 86\nrating: 3\n	1	\N	\N	809ceace-3883-48f6-9af0-25f2329c8434	2015-02-06 19:01:12.23992
401	219	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12D mutation occurs in never smokers significantly more often than in\n  smokers\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 129\nvariant_id: 85\nrating: 3\n	1	\N	\N	0b4461d7-faca-414a-bc86-cb90e531c535	2015-02-06 19:01:12.259559
402	220	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 31\nsource_id: 130\nvariant_id: 82\nrating: 3\n	1	\N	\N	efc00555-dd51-4666-993d-1c9ea0b49760	2015-02-06 19:01:12.278789
403	221	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of KRAS mutations in MM patients is prognostic for shorter overall\n  and progression free survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 32\nsource_id: 130\nvariant_id: 86\nrating: 3\n	1	\N	\N	9e870090-1982-48e5-82ed-766f678b88d1	2015-02-06 19:01:12.297313
404	222	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung patients are not associated with smoking history, age\n  and gender\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 131\nvariant_id: 82\nrating: 3\n	1	\N	\N	d59833a9-5549-4f25-8e92-cb302467a218	2015-02-06 19:01:12.314603
405	223	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no strong association between KRAS mutation status and overall or progression\n  free survival in patients with NSCLC\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 132\nvariant_id: 82\nrating: 3\n	1	\N	\N	b7d2c18f-0b80-4093-8190-df6c518fd975	2015-02-06 19:01:12.334521
406	224	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A meta-analysis showed KRAS mutation is associated with worse outcome in patients\n  with NSCLC\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 133\nvariant_id: 82\nrating: 3\n	1	\N	\N	b7fbf478-2c61-4123-89cf-c03346d746be	2015-02-06 19:01:12.355392
407	225	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS mutations in lung patients are not associated with smoking history, age\n  and gender\nclinical_significance: Negative\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 131\nvariant_id: 86\nrating: 3\n	1	\N	\N	9b7a031c-1f30-4d70-b7de-4484069c92ed	2015-02-06 19:01:12.382431
408	226	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 3\nsource_id: 134\nvariant_id: 85\nrating: 2\n	1	\N	\N	a8b523d8-f597-40ca-aaef-86eac05e4e33	2015-02-06 19:01:12.402224
409	227	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KRAS G12C occur more frequently in women than men\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 129\nvariant_id: 84\nrating: 2\n	1	\N	\N	a144a1fa-1832-4e7d-bc80-70e808e28274	2015-02-06 19:01:12.419596
410	228	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 6\nsource_id: 135\nvariant_id: 73\nrating: 3\n	1	\N	\N	fe48dcee-accf-4d39-841a-dc020182b5de	2015-02-06 19:01:12.437085
411	229	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 6\nsource_id: 135\nvariant_id: 74\nrating: 3\n	1	\N	\N	ec947ef3-edbb-4e49-ad9b-6fcfb41b4df9	2015-02-06 19:01:12.452863
412	230	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with KIT mutation but not KIT amplification showed response\n  to imatinib treatment in a small cohort of patients.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 6\nsource_id: 135\nvariant_id: 76\nrating: 3\n	1	\N	\N	1b0aa0cc-ff6f-497c-89ef-71a34c2eca13	2015-02-06 19:01:12.472397
413	231	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations were identified in morphologically benign, incidentally discovered\n  GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating\n  is not a prognostic marker for GIST\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 136\nvariant_id: 73\nrating: 3\n	1	\N	\N	0bdb8603-ca15-4794-8e33-55805579f455	2015-02-06 19:01:12.490413
414	232	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between KIT mutations in exon 11, exon 9\n  or wild type in survival among GIST patients.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 137\nvariant_id: 73\nrating: 3\n	1	\N	\N	d142a0dd-7b8c-42a5-9848-eb433e07ce7a	2015-02-06 19:01:12.508108
415	233	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with worse overall and cause-specific prognosis in\n  patients with GIST compared to patients with wildtype KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 138\nvariant_id: 73\nrating: 3\n	1	\N	\N	05a6c995-c610-4d78-a06c-cf04451ddb3a	2015-02-06 19:01:12.52823
416	234	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: GIST patients with exon 14 KIT mutations had reduced overall survival compared\n  to patients wiltype for KIT\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 137\nvariant_id: 75\nrating: 3\n	1	\N	\N	b5b461cc-5e74-4663-b74d-3c9f8c12c319	2015-02-06 19:01:12.544602
417	235	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 138\nvariant_id: 76\nrating: 3\n	1	\N	\N	a72f1868-934f-4af0-9822-0d6cacc5fc2e	2015-02-06 19:01:12.56034
418	236	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no significant association between KIT mutations in exon 11, exon 9\n  or wild type in survival among GIST patients.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 137\nvariant_id: 77\nrating: 3\n	1	\N	\N	94044c39-1a35-4f40-a2b3-fc0c881d33ad	2015-02-06 19:01:12.580004
419	237	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutation is associated with larger, more invasive tumors, greater pathologic\n  histology and older patients compared to tumors with wildtype KIT\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 21\nsource_id: 138\nvariant_id: 73\nrating: 2\n	1	\N	\N	4fe093d0-ccff-4f66-9876-06e0892daf3f	2015-02-06 19:01:12.598654
420	238	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 139\nvariant_id: 73\nrating: 2\n	1	\N	\N	488c00b5-9676-4596-8b47-08d6e3510f90	2015-02-06 19:01:12.617083
421	239	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: KIT mutations detected in 5/60 patients showed no prognostic significance in\n  patients with small cell lung cancer\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 139\nvariant_id: 77\nrating: 2\n	1	\N	\N	4f9b3afd-5f43-4adb-80a4-37145b1b8ed2	2015-02-06 19:01:12.632997
422	240	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with JAK2 V617F, the use of peglated IFN-alpha-2a leads to reduction\n  in the percentage of cells harboring JAK2 V617F.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 39\ndisease_id: 33\nsource_id: 140\nvariant_id: 71\nrating: 4\n	1	\N	\N	9940a368-1878-4c0e-bcd5-ef420bb74697	2015-02-06 19:01:12.655884
423	241	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F not associatted wth  lymphoid leukemia.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 9\nsource_id: 141\nvariant_id: 71\nrating: 4\n	1	\N	\N	e5a886d1-82f5-45c4-bdc4-ac770ac1b106	2015-02-06 19:01:12.673679
424	242	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical\n  CML).\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 9\nsource_id: 141\nvariant_id: 71\nrating: 4\n	1	\N	\N	b6002aec-2692-4f34-8575-64c2eeac921e	2015-02-06 19:01:12.692854
425	243	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or\n  CLL).\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 34\nsource_id: 141\nvariant_id: 71\nrating: 4\n	1	\N	\N	077a18f6-4955-468e-b9a6-d0a78c7855b2	2015-02-06 19:01:12.710776
426	244	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: rs10974944 SNP in the Jak2 locus is associated with increased predisposition\n  for JAK2 V617F mutation and its associated cancer\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 35\nsource_id: 142\nvariant_id: 71\nrating: 4\n	1	\N	\N	a3ec7a25-3fad-4885-ae5c-69863f2d36c7	2015-02-06 19:01:12.72955
427	245	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 40\ndisease_id: 36\nsource_id: 143\nvariant_id: 71\nrating: 3\n	1	\N	\N	dd81c252-bf5d-4e05-89f1-b61f39d0f0eb	2015-02-06 19:01:12.750972
428	246	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with IDH2 R140Q/L mutation, the presence of mutation do not impact\n  overall survival or disease free survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 144\nvariant_id: 69\nrating: 3\n	1	\N	\N	fd42ce14-a57d-42d9-aa8e-e2673e7cdbfe	2015-02-06 19:01:12.770663
429	247	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R140K mutation have improved overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 145\nvariant_id: 69\nrating: 3\n	1	\N	\N	e3d40072-8c31-4275-9c8b-56ecec15fa56	2015-02-06 19:01:12.789756
430	248	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML, patients with IDH2 R172K mutation have worse overall survival compared\n  to those with wild-type IDH2\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 145\nvariant_id: 70\nrating: 3\n	1	\N	\N	2245d05b-322c-4937-9abe-044fc45343e4	2015-02-06 19:01:12.808889
431	249	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: R140Q mutation in IDH2 does not have prognostic value in patients with MDS.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 25\nsource_id: 146\nvariant_id: 69\nrating: 3\n	1	\N	\N	4bc749bd-27cc-4fd7-b9a0-17a34cb380de	2015-02-06 19:01:12.826051
432	250	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R140Q/L have event free survival and\n  overall survival similar to those with wild-type IDH2\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 147\nvariant_id: 69\nrating: 2\n	1	\N	\N	73e00cb5-2fd1-4ba8-952a-c503bcee5bdf	2015-02-06 19:01:12.843418
433	251	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with IDH2 mutations such as R172K have event free survival and\n  overall survival similar to those with wild-type IDH2.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 7\nsource_id: 147\nvariant_id: 70\nrating: 2\n	1	\N	\N	3dcca3e9-cd8f-4822-9906-8059daa9dd2f	2015-02-06 19:01:12.859792
434	252	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L\n  do not confer prognostic value (overall survival)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 148\nvariant_id: 69\nrating: 2\n	1	\N	\N	82122f5c-8ebe-4229-870f-cf2f62a67896	2015-02-06 19:01:12.877022
435	253	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients suffering from MDS, the presence of IDH2 mutation such as R172K\n  do not confer prognostic value (overall survival)\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 3\ndrug_id: 3\ndisease_id: 25\nsource_id: 148\nvariant_id: 70\nrating: 2\n	1	\N	\N	9248ae76-2652-4dc7-b0cd-f3797fea6785	2015-02-06 19:01:12.893132
436	254	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated\n  with worse overall survival\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 37\nsource_id: 149\nvariant_id: 64\nrating: 4\n	1	\N	\N	35fbf2b6-f3d1-4353-88f4-69f291bda0c4	2015-02-06 19:01:12.913227
437	255	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation is associated with patients of older age, high platelet count\n  during diagnosis, cytogenic normalcy and NPM1 mutation\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 150\nvariant_id: 65\nrating: 3\n	1	\N	\N	32cd62b2-3783-49d0-8d8d-1c1c6415c313	2015-02-06 19:01:12.933391
438	256	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 150\nvariant_id: 67\nrating: 3\n	1	\N	\N	5efa421d-6fe4-4a58-8523-2953eb49ace7	2015-02-06 19:01:12.950524
439	257	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 150\nvariant_id: 68\nrating: 3\n	1	\N	\N	56ed5d13-6f4f-4c82-a33f-51f3a6fb2984	2015-02-06 19:01:12.968328
440	258	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic\n  for better survival compared to patients who harbor wild type mutation.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 38\nsource_id: 151\nvariant_id: 64\nrating: 3\n	1	\N	\N	95226a94-7100-4ec1-8ca3-812e109612eb	2015-02-06 19:01:12.991191
441	259	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In patients with glioblastoma, those harboring IDH1 R132 mutation has higher\n  overall survival compared to those who do not have IDH1 mutation\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 39\nsource_id: 152\nvariant_id: 64\nrating: 3\n	1	\N	\N	d3b629ee-a83b-4ae2-afa7-b50528fc439e	2015-02-06 19:01:13.013334
442	260	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In leukemia patients, IDH1 R132 mutation is associated with adults and normal\n  karyotype.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 153\nvariant_id: 64\nrating: 2\n	1	\N	\N	e9015a46-0afb-49c6-92ec-8a70a2aa69e8	2015-02-06 19:01:13.0335
443	261	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The complete remission and overall survival rates in patients with IDH1 R132\n  mutation is not significantly different from those who do not have this mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 154\nvariant_id: 64\nrating: 2\n	1	\N	\N	5e34f2cf-42f6-46e1-ac66-5c5da60840af	2015-02-06 19:01:13.052111
444	262	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: IDH1 R132 mutation in patients with AML is not associated with any prognostic\n  value compared to patients with wild-type IDH1.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 150\nvariant_id: 65\nrating: 2\n	1	\N	\N	19a88a74-b4cd-47e4-8ece-f58d8d38ac88	2015-02-06 19:01:13.08022
445	263	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 41\ndisease_id: 40\nsource_id: 155\nvariant_id: 63\nrating: 4\n	1	\N	\N	c0ee0f86-c028-4704-9973-fe8199431ef8	2015-02-06 19:01:13.102678
446	264	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and\n  fasudil led to dramatic tumor regression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 42\ndisease_id: 40\nsource_id: 155\nvariant_id: 63\nrating: 4\n	1	\N	\N	f6d8373d-7784-47b4-8767-0a511fe9ee96	2015-02-06 19:01:13.113416
447	265	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD  , concurrent DNMT3A mutations were associated\n  with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 61\nrating: 5\n	1	\N	\N	6c9b84c3-18de-4377-b107-853d601757a1	2015-02-06 19:01:13.129416
448	266	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat , but not\n  in wild type or D835 mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 156\nvariant_id: 61\nrating: 4\n	1	\N	\N	46d65ecf-0be7-4127-9fa5-214dbbd5e31d	2015-02-06 19:01:13.149909
449	267	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated\n  with sorafenib resistance\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 7\nsource_id: 157\nvariant_id: 62\nrating: 4\n	1	\N	\N	bf01cd99-a6bc-4699-84ba-9eb548cdc9cb	2015-02-06 19:01:13.170725
450	268	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib is effective in patients with FLT3 internal tandem repeat , but not\n  in wild type or D835 mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 7\nsource_id: 156\nvariant_id: 62\nrating: 4\n	1	\N	\N	9102dab0-4ed0-4701-a027-e3f58da9333b	2015-02-06 19:01:13.186614
451	269	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 is effective in AML patients with FLT3 mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 7\nsource_id: 158\nvariant_id: 61\nrating: 4\n	1	\N	\N	e47f0986-71f7-4232-877f-14b2b960953f	2015-02-06 19:01:13.205119
452	270	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEP701 is effective in AML patients with FLT3 mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 43\ndisease_id: 7\nsource_id: 158\nvariant_id: 62\nrating: 4\n	1	\N	\N	4e47df1d-2723-4d18-a750-dceedc421ef6	2015-02-06 19:01:13.223208
453	271	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with\n  FLT3 internal tandem repeat\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 44\ndisease_id: 7\nsource_id: 159\nvariant_id: 61\nrating: 4\n	1	\N	\N	26cb531f-db88-4b9d-aec3-a6e2111e230a	2015-02-06 19:01:13.241687
454	272	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD mutation without NPM1 was associated with increased relapse risk and\n  reduced overall survival in young adult AML patients (median age 43)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 61\nrating: 4\n	1	\N	\N	df49062c-a245-4024-ae33-85e7cd2bbc72	2015-02-06 19:01:13.257786
455	273	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status was not predictive\n  of overall and relapse free survival in patients with NPM1 mutations and wildtype\n  FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 61\nrating: 4\n	1	\N	\N	a924623f-7331-4780-9aa5-aaccc28bcca5	2015-02-06 19:01:13.274617
456	274	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3-ITD duplication length did not effect prognosis in patients with an NPM1\n  mutation\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 61\nrating: 4\n	1	\N	\N	9330c41f-e247-4046-bac3-473aed7004c9	2015-02-06 19:01:13.291298
457	275	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor\n  survival and inferior disease free survival in patients with AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 160\nvariant_id: 61\nrating: 4\n	1	\N	\N	21ccc77e-a8e6-453c-b137-946c4ba01d55	2015-02-06 19:01:13.308336
458	276	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared\n  to patients with wildtype FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 61\nrating: 4\n	1	\N	\N	9cd3a032-3163-4179-afec-2e483456bb1f	2015-02-06 19:01:13.32499
459	277	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher\n  ratio was associated with reduced event-free survival in patients with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 61\nrating: 4\n	1	\N	\N	997adfe4-2b6d-437a-b197-a3bc70b8755a	2015-02-06 19:01:13.341301
460	278	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free\n  and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 54\nvariant_id: 61\nrating: 4\n	1	\N	\N	7edbf4a4-89a5-415b-a50b-2c0200595da8	2015-02-06 19:01:13.357463
461	279	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients, FLT3-TKD mutation is associated with poorer disease free survival\n  compared to patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 161\nvariant_id: 62\nrating: 4\n	1	\N	\N	29555ab5-8123-4ff8-8b4f-66ed5d97fda4	2015-02-06 19:01:13.377801
462	280	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 45\ndisease_id: 7\nsource_id: 162\nvariant_id: 61\nrating: 3\n	1	\N	\N	1bfcbe37-756f-41c9-a18d-8ff414a331f8	2015-02-06 19:01:13.396735
463	281	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with\n  FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 45\ndisease_id: 7\nsource_id: 162\nvariant_id: 62\nrating: 3\n	1	\N	\N	12d14782-a3cd-44e6-88dc-69af1c4a2486	2015-02-06 19:01:13.412378
464	282	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: ATRA treatment did not effect overall survival in patients <60 years old with\n  FLT3-ITD mutation\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 6\ndisease_id: 7\nsource_id: 57\nvariant_id: 61\nrating: 3\n	1	\N	\N	f597e445-1e27-4932-b701-cce508f150e9	2015-02-06 19:01:13.428276
465	283	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 7\nsource_id: 162\nvariant_id: 61\nrating: 3\n	1	\N	\N	25146588-c634-4481-a22e-2bd2e25d83df	2015-02-06 19:01:13.446153
466	284	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more\n  resistance to daunorubicin than cells with FLT3-ITD alone.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 7\ndisease_id: 7\nsource_id: 162\nvariant_id: 62\nrating: 3\n	1	\N	\N	3b2f4419-d5d5-4150-a257-944a7dc9a0e8	2015-02-06 19:01:13.465498
467	285	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared\n  to wild type\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 61\nrating: 3\n	1	\N	\N	2b825c86-26ed-44a5-992d-2593900a8f03	2015-02-06 19:01:13.483773
468	286	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Increasing level of FLT3-ITD mutant correlated with reduced disease-free and\n  overall survival as well as increased relapse rate\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 61\nrating: 3\n	1	\N	\N	70679d5d-e129-4c24-9d34-c286f15d90ef	2015-02-06 19:01:13.502332
469	287	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with internal tandem repeat of FLT3 has worse overall survival\n  than patients with wild type FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 61\nrating: 3\n	1	\N	\N	381f0d6c-f2d7-4d46-8a46-84424703b22b	2015-02-06 19:01:13.520979
470	288	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: WBC counts were not significantly different in AML patients with FLT3 D835 mutations\n  compared to wild type\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 62\nrating: 3\n	1	\N	\N	78f3b6df-1225-40c0-a7d6-5b8b9e22d80c	2015-02-06 19:01:13.537585
471	289	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with D835 mutation of FLT3 did not have worse overall survival\n  than patients with wild type FLT3\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 163\nvariant_id: 62\nrating: 3\n	1	\N	\N	be7fe10a-d3c0-4c75-9d07-1cf0afe397b3	2015-02-06 19:01:13.55529
472	290	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Event-free survival was significantly improved in patients with both NPM1 and\n  FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either\n  NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither\n  FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype\n  patients were analyzed\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 60\nvariant_id: 62\nrating: 3\n	1	\N	\N	33e870b2-d024-4049-b708-8224e57a5fe7	2015-02-06 19:01:13.571887
473	291	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Size of the FLT3-ITD mutant duplication had no impact on overall survival or\n  relapse rate\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 48\nvariant_id: 61\nrating: 2\n	1	\N	\N	03f0feea-0179-4797-9191-fb09fddefdb2	2015-02-06 19:01:13.588399
474	292	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Meta-analysis of studies involving cytogentically normal younger (<60) patients\n  showed reduced overall and relapse-free survival for patients with FLT3-ITD\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 164\nvariant_id: 61\nrating: 2\n	1	\N	\N	645028b3-32bd-4775-9107-1530f37077fc	2015-02-06 19:01:13.606625
475	293	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 38\nrating: 5\n	1	\N	\N	2aa49862-1103-4c7e-a12d-e5c3f44a0455	2015-02-06 19:01:13.628002
476	294	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 39\nrating: 5\n	1	\N	\N	0fbc1d7b-a824-4475-a360-3342b33257ca	2015-02-06 19:01:13.646012
477	295	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 40\nrating: 5\n	1	\N	\N	7cb61a8c-43d2-4136-9f6c-9185bfbc61fb	2015-02-06 19:01:13.661879
478	296	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 41\nrating: 5\n	1	\N	\N	46a14f26-55da-4527-8cc2-701c061048c7	2015-02-06 19:01:13.678185
479	297	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 43\nrating: 5\n	1	\N	\N	71c81a38-5123-47e0-a2f1-ebea3c0cd0aa	2015-02-06 19:01:13.694535
480	298	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: This mutation is sensitive to the irreversible kinase inhibitor, neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 45\nrating: 5\n	1	\N	\N	56db5a63-dca1-4458-a350-0cad9ea0c400	2015-02-06 19:01:13.711952
481	299	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 46\nrating: 5\n	1	\N	\N	37affcd5-5d8e-4f2c-9596-78a7680b825a	2015-02-06 19:01:13.740788
482	300	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: All of these mutations were sensitive to the irreversible kinase inhibitor,\n  neratinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 46\ndisease_id: 11\nsource_id: 165\nvariant_id: 37\nrating: 4\n	1	\N	\N	40c385f6-4889-4bc2-a1e7-ad5fc1301bce	2015-02-06 19:01:13.759441
483	301	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients treated with tyrosine kinase inhibitors, the presence of L858R\n  mutation is prognostic for better progression free survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 166\nvariant_id: 35\nrating: 4\n	1	\N	\N	3fdff4be-a041-41f1-b852-e29a74834eb2	2015-02-06 19:01:13.778268
484	302	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with NSCLC harboring EGFR T790M mutation has statistically worse overall\n  survival compared to patients with L858R or other exon 19 activating mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 167\nvariant_id: 36\nrating: 4\n	1	\N	\N	5c538fb4-8409-4ec7-87d0-6d61c8e0e56a	2015-02-06 19:01:13.79815
485	303	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients with T790M and another activating mutation, their progression\n  free survival is shorter compared to those who do not possess T790M mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 168\nvariant_id: 36\nrating: 4\n	1	\N	\N	45291493-a9aa-46fb-a94e-2185b7e2a3cb	2015-02-06 19:01:13.815831
486	304	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 3\nsource_id: 169\nvariant_id: 36\nrating: 3\n	1	\N	\N	1c176a45-a355-4074-9718-a3b4205c3019	2015-02-06 19:01:13.834962
487	305	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a NSCLC patient with T790M and L858R mutation, combination treatment with\n  erlotinib and premetrexed has shown to be effective.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 47\ndisease_id: 3\nsource_id: 169\nvariant_id: 36\nrating: 3\n	1	\N	\N	44daf19e-1aa1-4c19-ae34-d462be7f6626	2015-02-06 19:01:13.845973
488	306	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 3\nsource_id: 170\nvariant_id: 35\nrating: 3\n	1	\N	\N	2eb56705-ed46-4d82-b24a-138259f10a8b	2015-02-06 19:01:13.865069
489	307	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no statistical difference in progression free survival between lung\n  cancer patients treated with gefitinib or erlotinib.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 38\ndisease_id: 3\nsource_id: 170\nvariant_id: 35\nrating: 3\n	1	\N	\N	f6d3cc6e-f453-42e8-8bf0-45203182c0a3	2015-02-06 19:01:13.874472
490	308	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Geftinib added to pemetrexed and carboplatin showed greater effectiveness than\n  premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 3\nsource_id: 171\nvariant_id: 35\nrating: 3\n	1	\N	\N	256e9d02-3919-430e-9ffb-1387ce5aaa17	2015-02-06 19:01:13.891822
491	309	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In non-small cell lung cancer, the appearance of T790M mutation leads to resistance\n  to gefitinib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 37\ndisease_id: 3\nsource_id: 172\nvariant_id: 36\nrating: 3\n	1	\N	\N	ebb200fb-f5d7-4fcc-ad86-43b5d24459d4	2015-02-06 19:01:13.909599
492	310	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Median survival of patients with EGFR L858R mutation is better than those with\n  wild type EGFR.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 173\nvariant_id: 35\nrating: 3\n	1	\N	\N	d5cee447-77d5-4dd5-be3f-8cddeeffcdec	2015-02-06 19:01:13.926979
493	311	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC containing T790M mutation, staurosporine may be inhibitive of EGFR,\n  especially when L858R mutation is also present.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 48\ndisease_id: 3\nsource_id: 174\nvariant_id: 36\nrating: 1\n	1	\N	\N	40e2103a-5b0a-4c86-acb4-2006172a0c7e	2015-02-06 19:01:13.947656
494	312	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (59% of which were R882) were associated with an intermediate\n  risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 33\nrating: 5\n	1	\N	\N	1ce7ac40-9429-401e-831f-775b1e7c530e	2015-02-06 19:01:13.966011
495	313	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: AML patients with DNMT3A mutations (59% of which were R882) showed worse survival\n  (event-free and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 33\nrating: 5\n	1	\N	\N	25ccc196-cc65-4f03-8108-e82da3d59858	2015-02-06 19:01:13.983927
496	314	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In AML patients with FLT3-ITD  , concurrent DNMT3A mutations (including R882)\n  were associated with worse overall survival compared to those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 33\nrating: 5\n	1	\N	\N	7aad6711-a3b8-4117-8257-9ac4f29dca84	2015-02-06 19:01:14.000274
497	315	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In cytogenetically normal AML patients, DNMT3A R882 mutations are associated\n  with lower overall and disease free survival as compared to patients with wild type\n  DNMT3A.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 175\nvariant_id: 33\nrating: 5\n	1	\N	\N	386a2c36-8024-44ed-bd1e-d0b47bec5558	2015-02-06 19:01:14.020133
498	316	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Idarubicin increases the overall survival and disease free survival in de novo\n  AML patients with DNMT3A R882 mutation compared to those who do not harbor this\n  mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 49\ndisease_id: 7\nsource_id: 176\nvariant_id: 33\nrating: 4\n	1	\N	\N	8c0df2c6-8c8c-43c5-a4f6-b4766a4a6cdd	2015-02-06 19:01:14.039051
499	317	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Daunorubicin treatment resulted in similar overall survival and disease free\n  survival in de novo AML patients with DNMT3A R882 mutation compared to those who\n  do not harbor this mutation.\nclinical_significance: NA\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 7\ndisease_id: 7\nsource_id: 176\nvariant_id: 33\nrating: 4\n	1	\N	\N	66a40f59-af94-4949-be30-3b142a76d37e	2015-02-06 19:01:14.055669
500	318	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rates did not differ between patients with wildtype or mutant\n  DNMT3A (62% of which affected R882) and cytogenetically normal AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 177\nvariant_id: 33\nrating: 4\n	1	\N	\N	16419400-ca63-4286-a501-308513a30059	2015-02-06 19:01:14.073681
501	319	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutation status (60% of which\n  were R882) was not predictive of overall and relapse free survival in patients with\n  NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 4\n	1	\N	\N	07d3ee35-2fc1-4411-8111-41363d450901	2015-02-06 19:01:14.089916
502	320	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: There is no difference in the complete remission rate of de novo AML patients\n  with DNMT3A mutation compared to those who are wild type for DNMT3A.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 176\nvariant_id: 33\nrating: 4\n	1	\N	\N	f005c8c6-4d52-4781-98c1-7069fa0040cc	2015-02-06 19:01:14.106628
503	321	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (62% of which were R882) were associated with reduced disease-free\n  survival in patients with cytogenetically normal AML.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 177\nvariant_id: 33\nrating: 4\n	1	\N	\N	41b802c6-a5bf-4815-95c1-14d1d5bb6cb6	2015-02-06 19:01:14.123061
504	322	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In young AML patients (<60 years old), DNMT3A mutations were associated with\n  significantly reduced overall survival and relapse free survival in patients wildtype\n  for NPM1 and FLT3\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 4\n	1	\N	\N	4007d9b4-0b02-4e57-b9c0-82cf92ab167a	2015-02-06 19:01:14.14284
505	323	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free\n  and overall) outcome than those without DNMT3A mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 55\nvariant_id: 33\nrating: 4\n	1	\N	\N	ca962df0-be97-4f75-961c-53bcfc0cd98b	2015-02-06 19:01:14.159821
506	324	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation\n  is prognostic for shorter disease free survival and overall survival compared to\n  patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 177\nvariant_id: 33\nrating: 4\n	1	\N	\N	6d898f28-ddd5-4207-a6e3-9f44be06769e	2015-02-06 19:01:14.17662
507	325	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations\n  other than R882 are prognostic for shorter disease free survival and overall survival\n  compared to patients without the mutation.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 177\nvariant_id: 33\nrating: 4\n	1	\N	\N	6fee87e2-4dc4-4a2f-942b-53b29e8511d8	2015-02-06 19:01:14.192595
508	326	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutations (60% of which were\n  R882) were older in age, had higher white blood cell counts and had higher platelet\n  counts than patients wildtype for DNMT3A\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 3\n	1	\N	\N	929eb575-140a-4dba-b111-f95cf6dd33bf	2015-02-06 19:01:14.209267
509	327	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations (64.5% R882) were associated with older age, higher white blood\n  cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML\n  patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	c3a2244f-00ab-48fe-8207-7ffa20447b5e	2015-02-06 19:01:14.230102
510	328	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Therapy-related AML was less common in patients with DNMT3A mutations (64.5%\n  of which were R882) than patients wildtype for DNMT3A in a large cohort of younger\n  (18-60) AML patients\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	7e43aef9-6e89-4666-8dd8-93c62f4fcd3a	2015-02-06 19:01:14.246941
511	329	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with cytogenetically normal AML in a large\n  cohort of younger (18-60) AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	911f8258-e2b9-409f-8663-4d5f4a05c762	2015-02-06 19:01:14.262808
512	330	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations occur most often in de novo AML patients with intermediate\n  cytogenic risk.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 176\nvariant_id: 33\nrating: 3\n	1	\N	\N	e0f7e967-13e0-4c0f-afb5-4d072d79d0eb	2015-02-06 19:01:14.280622
513	331	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutations were associated with older age, higher white blood cell\n  count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically\n  normal AML patients\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 175\nvariant_id: 33\nrating: 3\n	1	\N	\N	64eb4ca6-2440-43cb-8595-55de3ee00625	2015-02-06 19:01:14.299142
514	332	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5%\n  of which were R882) had no prognostic value on overall, relapse free and event free\n  survival.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	040403d4-8c04-4a5d-8b74-08cbc5396f20	2015-02-06 19:01:14.317129
515	333	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of cytogenetically normal AML patients (18-60 years old),\n  DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall,\n  relapse free and event free survival\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	653c5d5a-e594-4e8a-b25a-a05fd09915f3	2015-02-06 19:01:14.335007
516	334	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A mutations were associated with achievement of complete remission in a\n  large cohort of younger (18-60) AML patients\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	3ebaf3d9-a7e1-440f-8822-633a303f6a3a	2015-02-06 19:01:14.366322
517	335	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: DNMT3A R882 mutation was associated with reduced relapse free and overall survival\n  in ELN-unfavorable cytogenetically normal AMLs\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	99f4f4b9-fc8f-449f-9bc3-7b2c28d1704b	2015-02-06 19:01:14.383708
518	336	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A mutation have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 3\n	1	\N	\N	e552dbae-b52d-4d74-98a7-e2fe6c34e68e	2015-02-06 19:01:14.400858
519	337	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Complete remission rate was not different between young AML patients (<60 years\n  old) with or without DNMT3A mutations (60% of which were R882)\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 3\n	1	\N	\N	75cc575b-8153-44b7-ba37-938dbfa803fa	2015-02-06 19:01:14.417736
520	338	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall\n  survival and relapse-free survival than patients with wildtype DNMT3A\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 49\nvariant_id: 33\nrating: 3\n	1	\N	\N	869a97f1-20a5-4e14-b609-4d955107086e	2015-02-06 19:01:14.434808
521	339	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations\n  were associated with reduced relapse free survival in the entire cohort as well\n  as the subset of patients with cytogenetically normal AML\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 7\nsource_id: 178\nvariant_id: 33\nrating: 3\n	1	\N	\N	3c1f1dd2-7e75-406b-9b11-a92dca06c725	2015-02-06 19:01:14.451332
522	340	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation status had no impact on ATRA treatment response in older (>60)\n  patients with AML\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 57\nvariant_id: 30\nrating: 4\n	1	\N	\N	42fb69f6-b364-4026-a157-9d6fa107c71d	2015-02-06 19:01:14.468781
523	341	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was significantly associated with complete remission\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 69\nvariant_id: 30\nrating: 4\n	1	\N	\N	3a8b1397-0534-4b2f-99ab-419374449cbc	2015-02-06 19:01:14.485856
524	342	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CEBPA mutation was associated with improved overall survival in older (>60)\n  patients with AML\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 57\nvariant_id: 30\nrating: 4\n	1	\N	\N	33484328-79a6-4608-b600-7530e4f3bfa5	2015-02-06 19:01:14.502285
525	343	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Overall survival was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA mutations\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 179\nvariant_id: 29\nrating: 3\n	1	\N	\N	03f87327-c2bb-4dba-9acd-6e5f0a2c5b81	2015-02-06 19:01:14.526334
526	344	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Remission duration was significantly longer in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 179\nvariant_id: 29\nrating: 3\n	1	\N	\N	aa024c26-4735-43b1-8eb5-6d6454ba69ff	2015-02-06 19:01:14.543587
527	345	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically\n  normal AML patients with CEBPA\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 7\nsource_id: 179\nvariant_id: 29\nrating: 2\n	1	\N	\N	d0944f24-f0eb-4def-a612-65f1b5123b22	2015-02-06 19:01:14.56015
528	346	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter time to recurrence.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 180\nvariant_id: 28\nrating: 3\n	1	\N	\N	4cefdeac-d761-4d32-8f06-37e000142ac0	2015-02-06 19:01:14.580969
529	347	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter\n  hypermethylation is associated with shorter overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 180\nvariant_id: 28\nrating: 3\n	1	\N	\N	8e6f82a2-eb08-428c-88e8-8a3d4cb9a070	2015-02-06 19:01:14.597461
530	348	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and\n  CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor\n  regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 41\nsource_id: 181\nvariant_id: 27\nrating: 4\n	1	\N	\N	c5643560-43e5-4320-8430-1717aa37bbc6	2015-02-06 19:01:14.619379
531	349	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 51\ndisease_id: 41\nsource_id: 182\nvariant_id: 27\nrating: 3\n	1	\N	\N	1bd1ff8d-1b48-46d5-9888-b56f6ee1045a	2015-02-06 19:01:14.641711
532	350	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and\n  CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor\n  regressions.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 50\ndisease_id: 41\nsource_id: 181\nvariant_id: 26\nrating: 4\n	1	\N	\N	5302c85f-3846-4f1f-bfe1-6037b31d0c0a	2015-02-06 19:01:14.658988
533	351	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Palbociclib has been shown to dramatically improve progression-free survival\n  by over 18 months in patients with ER+ Breast Cancer.\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 51\ndisease_id: 41\nsource_id: 182\nvariant_id: 26\nrating: 3\n	1	\N	\N	144b8ca1-f23c-47de-b01e-db78e7c12882	2015-02-06 19:01:14.677997
534	352	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Patients with high expression of total cyclin E and low-molecular weight cyclin\n  E is strongly associated with poor prognosis in breast cancer, and the hazard ratio\n  for these patients in 13.3 times higher than those with normal cyclin E levels.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 183\nvariant_id: 25\nrating: 4\n	1	\N	\N	8ce11eb2-248e-431b-9205-852f3d8d05d2	2015-02-06 19:01:14.696772
535	353	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced\n  stage gastric carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 42\nsource_id: 184\nvariant_id: 25\nrating: 4\n	1	\N	\N	0f7f0d3c-5a31-4edf-a1c5-dcf9a536712f	2015-02-06 19:01:14.716099
536	354	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 42\nsource_id: 184\nvariant_id: 25\nrating: 4\n	1	\N	\N	f51b620f-3b84-4a93-81aa-096d30bd9815	2015-02-06 19:01:14.734167
537	355	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Elevated levels of CCNE1 mRNA has been shown to be correlated with increased\n  depth of tumor invasion in gastic carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 30\nsource_id: 184\nvariant_id: 25\nrating: 4\n	1	\N	\N	ee76a90c-5db6-4841-b54f-f84ffeef95a2	2015-02-06 19:01:14.754496
538	356	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib\n  (PD-0332991) significantly increased median survival.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 52\ndisease_id: 43\nsource_id: 185\nvariant_id: 24\nrating: 3\n	1	\N	\N	f097d130-05e5-431e-aa0b-390cf95c7bc1	2015-02-06 19:01:14.776274
539	357	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the presence pf lymph node metastasis\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 186\nvariant_id: 21\nrating: 4\n	1	\N	\N	cc6aec87-83b8-481c-8b2a-4ef6c23cf14f	2015-02-06 19:01:14.797362
540	358	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with the increased depth of cancer invasion\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 186\nvariant_id: 21\nrating: 4\n	1	\N	\N	5c209e5e-c128-4ecb-b48e-5cbca96e6ae6	2015-02-06 19:01:14.814145
541	359	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 186\nvariant_id: 21\nrating: 4\n	1	\N	\N	e4aaa155-844b-42e8-a3b3-53881784c243	2015-02-06 19:01:14.830668
542	360	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D2 overexpression is associated with vascular invasion by cancer cells\n  in gastric cancers.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 186\nvariant_id: 21\nrating: 4\n	1	\N	\N	7e553268-afdb-4464-8235-ea671f473342	2015-02-06 19:01:14.847333
543	361	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CCND2 hypomethylation is seen to be more common in stage III and IV gastric\n  tumors than stage I and II.\nclinical_significance: Negative\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 187\nvariant_id: 22\nrating: 3\n	1	\N	\N	90fb82ef-f24c-4c30-aa34-d610128f295f	2015-02-06 19:01:14.867422
544	362	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of CCND2-negative cells with 5-Aza-2'-deoxycytidine restored cyclin\n  D2 expression.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 44\nsource_id: 187\nvariant_id: 22\nrating: 2\n	1	\N	\N	3858a27b-5924-4d9c-8431-6cbb9cb28aa8	2015-02-06 19:01:14.883612
545	363	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 188\nvariant_id: 11\nrating: 5\n	1	\N	\N	cf06ac1c-0288-4a9d-bb89-b1b36e86f8d8	2015-02-06 19:01:14.903503
546	364	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: dabrafenib with trametinib provides higher response rate and lower toxicity-as\n  compared to chemotherapy-in patients with melanoma.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 53\ndisease_id: 6\nsource_id: 188\nvariant_id: 11\nrating: 5\n	1	\N	\N	690e820f-b30b-496b-aae8-790c4bba90b1	2015-02-06 19:01:14.916527
547	365	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is associated with adverse pathological features of colorectal cancer.\n  This can be concluded as poor prognosis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 26\nsource_id: 189\nvariant_id: 11\nrating: 5\n	1	\N	\N	3340dcbe-c9d8-4c99-b2b1-8ca4fe4dd112	2015-02-06 19:01:14.936447
548	366	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 190\nvariant_id: 11\nrating: 5\n	1	\N	\N	cef4cb81-1ad6-4df2-9072-3123212d6a04	2015-02-06 19:01:14.955189
549	367	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 191\nvariant_id: 11\nrating: 5\n	1	\N	\N	0822d1a5-fe34-46b0-9d5d-7f751582eecf	2015-02-06 19:01:15.044985
550	368	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Unlike other studies that suggest a poorer outcome, BRAF mutation in this study\n  was not correlated with poorer prognosis in papillary thyroid cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Does Not Support\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 192\nvariant_id: 11\nrating: 5\n	1	\N	\N	7406bca4-e5dd-41ff-b567-0e06a5299a2c	2015-02-06 19:01:15.065327
551	369	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK\n  inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification\n  of the BRAF V600E gene.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 3\ndrug_id: 23\ndisease_id: 26\nsource_id: 193\nvariant_id: 13\nrating: 4\n	1	\N	\N	d065e8a4-54fe-4a0c-a081-d4277e8689d9	2015-02-06 19:01:15.083517
552	370	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutations are associated with melanoma arising in nonchronic sun damage\n  skin and with superficial spreading melanoma\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 194\nvariant_id: 16\nrating: 4\n	1	\N	\N	d5cb916a-f45d-4113-a767-f01f932a2d21	2015-02-06 19:01:15.103974
553	371	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations\n  were associated with poor progression free survival regardless of treatment (panitumumab\n  with best supportive care or best supportive care alone)\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 27\ndisease_id: 26\nsource_id: 195\nvariant_id: 16\nrating: 3\n	1	\N	\N	4dcc3cb0-e14c-45f8-ad56-7d661443ffef	2015-02-06 19:01:15.122244
554	372	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 54\ndisease_id: 28\nsource_id: 196\nvariant_id: 11\nrating: 3\n	1	\N	\N	ed7cb487-eb04-4408-8f1d-60efa71faaca	2015-02-06 19:01:15.141799
555	373	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected\n  with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor\n  growth and reduced cellular proliferation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 55\ndisease_id: 28\nsource_id: 196\nvariant_id: 11\nrating: 3\n	1	\N	\N	70db81ab-32b4-44e1-8a8e-6efbc93b3992	2015-02-06 19:01:15.153026
556	374	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 26\ndisease_id: 28\nsource_id: 197\nvariant_id: 11\nrating: 3\n	1	\N	\N	c2a53705-1779-485d-92b2-0d867728fa94	2015-02-06 19:01:15.175562
557	375	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 27\ndisease_id: 28\nsource_id: 197\nvariant_id: 11\nrating: 3\n	1	\N	\N	b7ae74a0-d0a9-4567-a025-0bb44e3ca231	2015-02-06 19:01:15.187329
558	376	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless\n  BRAF inhibitor such as Sorafenib is introduced.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 28\nsource_id: 197\nvariant_id: 11\nrating: 3\n	1	\N	\N	d1bd7137-0367-42fa-8da3-3e5020a1a20d	2015-02-06 19:01:15.197551
559	377	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 198\nvariant_id: 16\nrating: 3\n	1	\N	\N	a9401aab-9e88-4eb1-95e9-a49aaf9aea25	2015-02-06 19:01:15.21812
560	378	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Melanoma patients with BRAF V600E/K mutations had longer progression-free survival,\n  increased tumor regression,  and increased duration of response to combined dabrafenib\n  and trametinib treatment compared to dabrafenib alone\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 53\ndisease_id: 6\nsource_id: 198\nvariant_id: 16\nrating: 3\n	1	\N	\N	87fdc8a2-b0fe-492b-ba7e-9db9e61fd26c	2015-02-06 19:01:15.229846
561	379	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations\n  had lower response and disease control rates and shorter progression free and overall\n  survival following cetuximab plus chemotherapy than those with wildtype BRAF\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 26\ndisease_id: 26\nsource_id: 105\nvariant_id: 11\nrating: 3\n	1	\N	\N	7b7a5fb7-3744-4cfe-ba16-aebadebf3cf0	2015-02-06 19:01:15.248793
562	380	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with shorter Disease-Free and Overall Survival in a\n  Spanish cohort of Melanoma patients.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 6\nsource_id: 199\nvariant_id: 11\nrating: 3\n	1	\N	\N	13bac61e-4eeb-4b88-a15d-a8ef55d318d5	2015-02-06 19:01:15.269877
563	381	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is shown to be associated with the tall-cell variant of PTC\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 200\nvariant_id: 11\nrating: 3\n	1	\N	\N	04ed0834-354c-48a4-ad18-8a637dfe4c12	2015-02-06 19:01:15.287941
564	382	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: V600E is correlated with disease recurrence in both age cohorts (>65 and <65\n  yo).\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 200\nvariant_id: 11\nrating: 3\n	1	\N	\N	06be2fdb-2a6c-4d42-8503-8251ec7632d7	2015-02-06 19:01:15.307064
565	383	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in\n  a study of 187 patients with PTC and other thyroid diseases.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 201\nvariant_id: 11\nrating: 3\n	1	\N	\N	266feaca-ef6a-400c-bd5c-18df89acd6cd	2015-02-06 19:01:15.325094
566	384	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF mutation correlated with poor prognosis in papillary thyroid cancer in\n  both older (>65 yo) and younger (<65 yo) cohorts.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 14\nsource_id: 200\nvariant_id: 11\nrating: 3\n	1	\N	\N	a31e30cc-c9c0-4bec-91f0-2a1d18203ec0	2015-02-06 19:01:15.343436
567	385	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF V600E resistance is triggered by ERK dependent or independent pathways.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 6\nsource_id: 202\nvariant_id: 11\nrating: 3\n	1	\N	\N	9a82d85c-c698-4827-9fd3-e5a107b16def	2015-02-06 19:01:15.361676
568	386	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g.\n  dabrafenib) appears to be beneficial, and better than vice versa.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 56\ndisease_id: 6\nsource_id: 203\nvariant_id: 11\nrating: 2\n	1	\N	\N	350376aa-406d-406b-8ec3-a0c2e4e3d2a1	2015-02-06 19:01:15.38415
569	387	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g.\n  dabrafenib) appears to be beneficial, and better than vice versa.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 6\nsource_id: 203\nvariant_id: 11\nrating: 2\n	1	\N	\N	82a40aae-f52f-4cc4-8032-4f5398186b7b	2015-02-06 19:01:15.395524
570	388	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 57\ndisease_id: 6\nsource_id: 204\nvariant_id: 11\nrating: 2\n	1	\N	\N	0ba54a3b-97b5-4847-8a55-fc06b2b32cfa	2015-02-06 19:01:15.417055
571	389	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BRAF status does not predict prognosis in patients treated with dacarbazine\n  or temozolomide.\nclinical_significance: N/A\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 32\ndisease_id: 6\nsource_id: 204\nvariant_id: 11\nrating: 2\n	1	\N	\N	8e020bd6-f403-46b9-a0eb-68a0dcb02a07	2015-02-06 19:01:15.428565
572	390	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 58\ndisease_id: 26\nsource_id: 205\nvariant_id: 11\nrating: 2\n	1	\N	\N	8876f72b-d5b8-47d5-8d8b-fc9ec30fbe2e	2015-02-06 19:01:15.45217
591	409	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 treatment resulted in significant tumor regression in xenograft models\n  produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 3\nsource_id: 213\nvariant_id: 6\nrating: 3\n	1	\N	\N	f5364942-f2f9-4b09-99fb-2758b1120b29	2015-02-06 19:01:15.802083
573	391	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 26\nsource_id: 205\nvariant_id: 11\nrating: 2\n	1	\N	\N	0f3c47d8-3570-4a5f-b54a-d9dfdb76404a	2015-02-06 19:01:15.463696
574	392	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated\n  with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased\n  survival and reduced tumor burden compared to single and double agent therapies.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 59\ndisease_id: 26\nsource_id: 205\nvariant_id: 11\nrating: 2\n	1	\N	\N	da9a66ed-5c08-430d-9aa3-b967d7014c4a	2015-02-06 19:01:15.476068
575	393	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 55\ndisease_id: 28\nsource_id: 206\nvariant_id: 11\nrating: 2\n	1	\N	\N	f9922a7d-3ee8-44e2-8e20-01e364f25859	2015-02-06 19:01:15.494736
576	394	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely\n  injected with the A357 colorectal cell line with a BRAF V600E mutation effectively\n  inhibited tumor growth significantly more than single agent therapy\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 60\ndisease_id: 28\nsource_id: 206\nvariant_id: 11\nrating: 2\n	1	\N	\N	ffd70ca1-6fa4-4886-99a4-289d41c9e495	2015-02-06 19:01:15.506693
577	395	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance\n  to dabrafenib.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 4\ndisease_id: 3\nsource_id: 134\nvariant_id: 11\nrating: 2\n	1	\N	\N	8da28255-c77e-4a3a-b2c6-f89dc9a0f2d2	2015-02-06 19:01:15.524706
578	396	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cell lines expressing the BRAF V600E mutation responded better to vemurafenib\n  treatment than cells wildtype for BRAF as measured by reduced cellular proliferation\n  and inhibition of MET and ERK phosphorylation\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 29\ndisease_id: 26\nsource_id: 205\nvariant_id: 11\nrating: 2\n	1	\N	\N	b078d32d-f5b1-43e9-aa50-91147ac9a548	2015-02-06 19:01:15.541934
579	397	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with the ER-positive subtype of breast\n  cancer.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 207\nvariant_id: 19\nrating: 4\n	1	\N	\N	c8060172-1a59-4ab5-88f3-f6646414ccf1	2015-02-06 19:01:15.564533
580	398	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In ER-positive breast cancer patients, Cyclin D1 overexpression is associated\n  with shorter overall survival and increased metastasis.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 11\nsource_id: 207\nvariant_id: 19\nrating: 4\n	1	\N	\N	31d3eb49-1715-4f3e-b659-d4c2358a0965	2015-02-06 19:01:15.582651
581	399	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in head and neck squamous cell carcinoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 45\nsource_id: 208\nvariant_id: 19\nrating: 4\n	1	\N	\N	9e4deb03-1a0e-488c-a94c-8073bea16b0c	2015-02-06 19:01:15.602358
582	400	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cyclin D1 overexpression is associated with shorter overall survival and increased\n  metastasis in mantle cell lymphoma.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 46\nsource_id: 209\nvariant_id: 19\nrating: 4\n	1	\N	\N	7f778371-1728-4cd0-b867-c3e8e39626be	2015-02-06 19:01:15.621781
583	401	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Inceased copy number of CCND1 is associated with poorer overall survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 210\nvariant_id: 17\nrating: 3\n	1	\N	\N	74ebbb00-b2c6-4881-b361-09a7b6661d4b	2015-02-06 19:01:15.64024
584	402	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have associated CCND1 expression with poorer survival.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 210\nvariant_id: 18\nrating: 3\n	1	\N	\N	791dd9ac-6735-493e-abf0-9fca226b5517	2015-02-06 19:01:15.659024
585	403	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Three studies have found no significant survival impact for CCND1 in lung cancer.\nclinical_significance: Poor Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 1\ndrug_id: 3\ndisease_id: 3\nsource_id: 210\nvariant_id: 18\nrating: 3\n	1	\N	\N	8b5c40a6-7bbb-4dce-bc8e-1e9f7f6a0e47	2015-02-06 19:01:15.675602
586	404	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony\n  formation of AALE cells in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 53\ndisease_id: 3\nsource_id: 211\nvariant_id: 9\nrating: 3\n	1	\N	\N	e9879cd8-28e6-469e-ad23-2cbb87e9af67	2015-02-06 19:01:15.696482
587	405	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony\n  formation of AALE cells in vitro\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 14\ndisease_id: 3\nsource_id: 211\nvariant_id: 9\nrating: 3\n	1	\N	\N	48c96b27-30b9-4d06-af7b-44931da4d460	2015-02-06 19:01:15.729054
588	406	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: In one patient with S214C mutation, the use of sorafenib has led to more than\n  5 years of survival and near remission.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 14\ndisease_id: 3\nsource_id: 211\nvariant_id: 9\nrating: 2\n	1	\N	\N	1d0012f7-7b5d-4606-aec3-fa379bf22cfd	2015-02-06 19:01:15.745866
589	407	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing\n  ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 61\ndisease_id: 47\nsource_id: 212\nvariant_id: 7\nrating: 4\n	1	\N	\N	23890870-c939-4ea6-b81e-0236bad1a676	2015-02-06 19:01:15.766612
590	408	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in\n  a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 62\ndisease_id: 47\nsource_id: 213\nvariant_id: 7\nrating: 3\n	1	\N	\N	5d780cf7-bc26-4353-9048-1df42a534cc0	2015-02-06 19:01:15.786218
592	410	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: TAE684 inhibits growth of Ba/F3 cells expressing ALK with the R1275Q mutation\n  at a higher IC50 than those expressing F1174L and the SMS-KCNR cell line harboring\n  the R1275Q mutation was resistant to TAE684\nclinical_significance: NA\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 61\ndisease_id: 47\nsource_id: 212\nvariant_id: 8\nrating: 3\n	1	\N	\N	47afd6ad-95ea-4618-a28f-2e4a12d07f5f	2015-02-06 19:01:15.818242
593	411	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were\n  more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without\n  this mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 48\nsource_id: 84\nvariant_id: 7\nrating: 3\n	1	\N	\N	35747520-ac46-4b58-a770-dcbd2968377e	2015-02-06 19:01:15.838509
594	412	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Cells lines expressing ALK harboring the F1174L mutation are less sensitive\n  to growth inhibition by crizotinib than cells expressing the R1275Q mutation.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 47\nsource_id: 214\nvariant_id: 7\nrating: 3\n	1	\N	\N	0d2b7ba7-c292-41db-bc29-e22534dbc9d5	2015-02-06 19:01:15.857342
595	413	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing\n  the EML4-ALK fusion containing the F1174L mutation.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 2\ndrug_id: 19\ndisease_id: 47\nsource_id: 215\nvariant_id: 7\nrating: 3\n	1	\N	\N	cf8239ee-2018-4f06-90ce-e17fb50ee8f0	2015-02-06 19:01:15.875503
596	414	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the\n  AKT inhibitor MK-2206 in breast cancer cell lines.\nclinical_significance: Sensitivity\nevidence_direction: Does Not Support\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 63\ndisease_id: 49\nsource_id: 216\nvariant_id: 3\nrating: 3\n	1	\N	\N	cdf1bd4c-39c7-4d34-ba7d-29df02e18017	2015-02-06 19:01:15.897445
597	415	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide\n  results in high rates of complete remission in patients.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 64\ndisease_id: 7\nsource_id: 217\nvariant_id: 1\nrating: 5\n	1	\N	\N	eaf33065-ecac-4c6d-8456-20447a4db201	2015-02-06 19:01:15.917357
598	416	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The clinical use of imatinib in patients with BCR-ABL fusion has drastic sensitivity\n  to the drug.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 4\ndrug_id: 10\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 5\n	1	\N	\N	a376e326-c53c-478b-99c5-6bbfc3c99a8b	2015-02-06 19:01:15.934025
599	417	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis,\n  response rate, overall survival, and patient outcome in CML patients compared to\n  previous therapeutic regimens."\nclinical_significance: Better Outcome\nevidence_direction: Supports\nevidence_type_id: 2\nevidence_level_id: 4\ndrug_id: 10\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 5\n	1	\N	\N	01a0e0ac-d528-4f2b-be7c-438a28e753c2	2015-02-06 19:01:15.950437
600	418	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has shown signifcantly\n  greater response in patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 13\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 4\n	1	\N	\N	ddf1f502-f569-4079-b9aa-11f9e06abc29	2015-02-06 19:01:15.967667
601	419	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The use of second-generation BCR-ABL targeted therapies has shown signifcantly\n  greater response in patients resistant to imatinib.\nclinical_significance: Sensitivity\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 15\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 4\n	1	\N	\N	12df3f37-68e1-4c06-80cd-8039261500a3	2015-02-06 19:01:15.976931
602	420	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: The presence of BCR-ABL fusion is considered the characterizing feature of chronic\n  myeloid leukemia, and has been widely thought of as the initiating event in the\n  disease.\nclinical_significance: Positive\nevidence_direction: Supports\nevidence_type_id: 3\nevidence_level_id: 4\ndrug_id: 3\ndisease_id: 9\nsource_id: 72\nvariant_id: 1\nrating: 4\n	1	\N	\N	eaf94b92-7d80-4ac2-9174-5a2a2e421aa8	2015-02-06 19:01:15.994403
603	421	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: BCR-ABL fusions have been found to contribute to imatinib resistance in AML\n  cell lines.\nclinical_significance: Resistance or Non-Response\nevidence_direction: Supports\nevidence_type_id: 1\nevidence_level_id: 1\ndrug_id: 10\ndisease_id: 7\nsource_id: 217\nvariant_id: 1\nrating: 3\n	1	\N	\N	8216376c-a35f-40bc-a73f-efd17a1ec205	2015-02-06 19:01:16.012879
604	422	EvidenceItem	\N	\N	\N	\N	\N	create	---\ntext: Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient\n  has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may\n  imply the possibility that BCR-ABL is not universally the initiating event in CML.\nclinical_significance: Positive\nevidence_direction: Does Not Support\nevidence_type_id: 3\nevidence_level_id: 2\ndrug_id: 3\ndisease_id: 9\nsource_id: 218\nvariant_id: 1\nrating: 2\n	1	\N	\N	1b86c2eb-723f-4d3d-bae5-8f55d828c6ac	2015-02-06 19:01:16.031572
\.


--
-- Name: audits_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('audits_id_seq', 604, true);


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY users (id, email, name, url, nickname, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: authorizations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY authorizations (id, user_id, provider, uid, created_at, updated_at) FROM stdin;
\.


--
-- Name: authorizations_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('authorizations_id_seq', 1, false);


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY comments (id, title, comment, commentable_id, commentable_type, user_id, role, created_at, updated_at) FROM stdin;
\.


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('comments_id_seq', 1, false);


--
-- Data for Name: data_versions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY data_versions (id, version) FROM stdin;
\.


--
-- Name: data_versions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('data_versions_id_seq', 1, false);


--
-- Data for Name: definitions; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY definitions (id, term, text, created_at, updated_at) FROM stdin;
\.


--
-- Name: definitions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('definitions_id_seq', 1, false);


--
-- Data for Name: diseases; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY diseases (id, doid, name, created_at, updated_at) FROM stdin;
1	4007	bladder carcinoma	2015-02-06 19:01:07.866987	2015-02-06 19:03:52.763698
2	6477	invasive bladder transitional cell carcinoma	2015-02-06 19:01:07.951414	2015-02-06 19:03:52.99362
3	3908	non-small cell lung carcinoma	2015-02-06 19:01:07.978048	2015-02-06 19:03:53.403081
4	1612	breast cancer	2015-02-06 19:01:08.007986	2015-02-06 19:03:53.672916
5	3068	glioblastoma multiforme	2015-02-06 19:01:08.036943	2015-02-06 19:03:53.861574
6	1909	melanoma	2015-02-06 19:01:08.064298	2015-02-06 19:03:54.11052
7	9119	acute myeloid leukemia	2015-02-06 19:01:08.156056	2015-02-06 19:03:54.252378
8	3973	thyroid medullary carcinoma	2015-02-06 19:01:09.096424	2015-02-06 19:03:54.398355
9	8552	chronic myeloid leukemia	2015-02-06 19:01:09.146241	2015-02-06 19:03:54.567484
10	4947	cholangiocarcinoma	2015-02-06 19:01:09.188715	2015-02-06 19:03:54.725251
11	1612	breast cancer	2015-02-06 19:01:09.214803	2015-02-06 19:03:55.045106
12	10283	prostate cancer	2015-02-06 19:01:09.306499	2015-02-06 19:03:55.283018
13	3907	lung squamous cell carcinoma	2015-02-06 19:01:09.330484	2015-02-06 19:03:55.486784
14	1781	thyroid cancer	2015-02-06 19:01:09.352552	2015-02-06 19:03:55.689577
15	4007	bladder carcinoma	2015-02-06 19:01:09.374355	2015-02-06 19:03:55.951725
16	3068	glioblastoma multiforme	2015-02-06 19:01:09.44152	2015-02-06 19:03:56.305362
17	8618	oral cavity cancer	2015-02-06 19:01:09.474168	2015-02-06 19:03:56.586985
18	4947	cholangiocarcinoma	2015-02-06 19:01:09.507522	2015-02-06 19:03:56.993532
19	3907	lung squamous cell carcinoma	2015-02-06 19:01:09.541895	2015-02-06 19:03:57.144146
20	9119	acute myeloid leukemia	2015-02-06 19:01:09.577606	2015-02-06 19:03:57.285111
21	9253	gastrointestinal stromal tumor	2015-02-06 19:01:09.702656	2015-02-06 19:03:57.416837
22	5015	hepatocellular fibrolamellar carcinoma	2015-02-06 19:01:10.087007	2015-02-06 19:03:57.545585
23	9119	acute myeloid leukemia	2015-02-06 19:01:10.238373	2015-02-06 19:03:57.683303
24	9952	acute lymphocytic leukemia	2015-02-06 19:01:10.275572	2015-02-06 19:03:58.099926
25	4972	myelodysplastic myeloproliferative cancer	2015-02-06 19:01:10.337533	2015-02-06 19:03:58.251747
26	9256	colorectal cancer	2015-02-06 19:01:11.150226	2015-02-06 19:03:58.415796
27	9253	gastrointestinal stromal tumor	2015-02-06 19:01:11.471867	2015-02-06 19:03:58.553793
28	9256	colorectal cancer	2015-02-06 19:01:12.027261	2015-02-06 19:03:58.717072
29	1240	leukemia	2015-02-06 19:01:12.219925	2015-02-06 19:03:58.963151
30	1324	lung cancer	2015-02-06 19:01:12.255623	2015-02-06 19:03:59.263311
31	9538	multiple myeloma	2015-02-06 19:01:12.274454	2015-02-06 19:03:59.570941
32	9119	acute myeloid leukemia	2015-02-06 19:01:12.292487	2015-02-06 19:03:59.755185
33	8997	polycythemia vera	2015-02-06 19:01:12.651753	2015-02-06 19:04:00.152421
34	10747	lymphoid leukemia	2015-02-06 19:01:12.706514	2015-02-06 19:04:00.294678
35	4960	bone marrow cancer	2015-02-06 19:01:12.725343	2015-02-06 19:04:00.462266
36	1240	leukemia	2015-02-06 19:01:12.746863	2015-02-06 19:04:00.73905
37	3069	astrocytoma	2015-02-06 19:01:12.908	2015-02-06 19:04:00.974297
38	3069	astrocytoma	2015-02-06 19:01:12.98696	2015-02-06 19:04:01.221955
39	3068	glioblastoma multiforme	2015-02-06 19:01:13.008537	2015-02-06 19:04:01.416654
40	3909	bronchogenic lung adenocarcinoma	2015-02-06 19:01:13.098709	2015-02-06 19:04:01.558288
41	0060075	estrogen-receptor positive breast cancer	2015-02-06 19:01:14.61405	2015-02-06 19:04:01.699501
42	5635	gastric adenosquamous carcinoma	2015-02-06 19:01:14.712142	2015-02-06 19:04:01.873838
43	715	T-cell leukemia	2015-02-06 19:01:14.771435	2015-02-06 19:04:02.023412
44	10534	stomach cancer	2015-02-06 19:01:14.793424	2015-02-06 19:04:02.255038
45	8618	oral cavity cancer	2015-02-06 19:01:15.597748	2015-02-06 19:04:02.5782
46	0050746	mantle cell lymphoma	2015-02-06 19:01:15.617634	2015-02-06 19:04:02.708687
47	769	neuroblastoma	2015-02-06 19:01:15.762407	2015-02-06 19:04:02.906708
48	1115	sarcoma	2015-02-06 19:01:15.833937	2015-02-06 19:04:03.090151
49	1612	breast cancer	2015-02-06 19:01:15.892046	2015-02-06 19:04:03.435959
\.


--
-- Name: diseases_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('diseases_id_seq', 49, true);


--
-- Data for Name: drugs; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY drugs (id, name, pubchem_id, created_at, updated_at) FROM stdin;
1	Everolimus	\N	2015-02-06 19:01:07.86217	2015-02-06 19:01:07.86217
2	Rapamycin	\N	2015-02-06 19:01:07.974794	2015-02-06 19:01:07.974794
3	N/A	\N	2015-02-06 19:01:08.004379	2015-02-06 19:01:08.004379
4	Dabrafenib	\N	2015-02-06 19:01:08.061588	2015-02-06 19:01:08.061588
5	NSC348884	\N	2015-02-06 19:01:08.479619	2015-02-06 19:01:08.479619
6	ATRA	\N	2015-02-06 19:01:08.491371	2015-02-06 19:01:08.491371
7	Daunorubicin	\N	2015-02-06 19:01:08.856512	2015-02-06 19:01:08.856512
8	Valproic acid	\N	2015-02-06 19:01:08.881458	2015-02-06 19:01:08.881458
9	Motesanib	\N	2015-02-06 19:01:09.093503	2015-02-06 19:01:09.093503
10	Imatinib	\N	2015-02-06 19:01:09.143474	2015-02-06 19:01:09.143474
11	Ponatinib	\N	2015-02-06 19:01:09.184808	2015-02-06 19:01:09.184808
12	Pazopanib	\N	2015-02-06 19:01:09.212076	2015-02-06 19:01:09.212076
13	Nilotinib	\N	2015-02-06 19:01:09.61581	2015-02-06 19:01:09.61581
14	Sorafenib	\N	2015-02-06 19:01:09.626371	2015-02-06 19:01:09.626371
15	Dasatinib	\N	2015-02-06 19:01:09.653555	2015-02-06 19:01:09.653555
16	SU11248	\N	2015-02-06 19:01:09.699879	2015-02-06 19:01:09.699879
17	Tamoxifen	\N	2015-02-06 19:01:09.792925	2015-02-06 19:01:09.792925
18	Fulvestrant	\N	2015-02-06 19:01:09.807153	2015-02-06 19:01:09.807153
19	Crizotinib	\N	2015-02-06 19:01:10.131486	2015-02-06 19:01:10.131486
20	GW-2580	\N	2015-02-06 19:01:10.286579	2015-02-06 19:01:10.286579
21	Doxorubicin	\N	2015-02-06 19:01:10.787968	2015-02-06 19:01:10.787968
22	AZD1480	\N	2015-02-06 19:01:11.041413	2015-02-06 19:01:11.041413
23	AZD6244	\N	2015-02-06 19:01:11.062217	2015-02-06 19:01:11.062217
24	CH5132799	\N	2015-02-06 19:01:11.097926	2015-02-06 19:01:11.097926
25	Regorafenib	\N	2015-02-06 19:01:11.147674	2015-02-06 19:01:11.147674
26	Cetuximab	\N	2015-02-06 19:01:11.205344	2015-02-06 19:01:11.205344
27	Panitumumab	\N	2015-02-06 19:01:11.217348	2015-02-06 19:01:11.217348
28	Crenolanib	\N	2015-02-06 19:01:11.467965	2015-02-06 19:01:11.467965
29	Vemurafenib	\N	2015-02-06 19:01:11.617911	2015-02-06 19:01:11.617911
30	17-AAG	\N	2015-02-06 19:01:11.833328	2015-02-06 19:01:11.833328
31	Temozolomide¬†	\N	2015-02-06 19:01:11.854503	2015-02-06 19:01:11.854503
32	Temozolomide	\N	2015-02-06 19:01:11.910116	2015-02-06 19:01:11.910116
33	O(6)-benzylguanine	\N	2015-02-06 19:01:11.931041	2015-02-06 19:01:11.931041
34	Carmustine	\N	2015-02-06 19:01:11.953349	2015-02-06 19:01:11.953349
35	NVP-BEZ235	\N	2015-02-06 19:01:11.990986	2015-02-06 19:01:11.990986
36	ARRY-142886	\N	2015-02-06 19:01:12.005306	2015-02-06 19:01:12.005306
37	Gefitinib	\N	2015-02-06 19:01:12.134974	2015-02-06 19:01:12.134974
38	Erlotinib	\N	2015-02-06 19:01:12.148109	2015-02-06 19:01:12.148109
39	Pegylated IFN-Œ±‚Äì2a	\N	2015-02-06 19:01:12.646907	2015-02-06 19:01:12.646907
40	TG101348	\N	2015-02-06 19:01:12.744189	2015-02-06 19:01:12.744189
41	Bortezomib	\N	2015-02-06 19:01:13.096071	2015-02-06 19:01:13.096071
42	Fasudil	\N	2015-02-06 19:01:13.109279	2015-02-06 19:01:13.109279
43	CEP701	\N	2015-02-06 19:01:13.199582	2015-02-06 19:01:13.199582
44	AG1296	\N	2015-02-06 19:01:13.236079	2015-02-06 19:01:13.236079
45	SU5614	\N	2015-02-06 19:01:13.3913	2015-02-06 19:01:13.3913
46	Neratinib	\N	2015-02-06 19:01:13.622017	2015-02-06 19:01:13.622017
47	Premetrexed	\N	2015-02-06 19:01:13.841661	2015-02-06 19:01:13.841661
48	Stauroporine	\N	2015-02-06 19:01:13.941962	2015-02-06 19:01:13.941962
49	Idarubicin	\N	2015-02-06 19:01:14.033349	2015-02-06 19:01:14.033349
50	BYL719	\N	2015-02-06 19:01:14.610587	2015-02-06 19:01:14.610587
51	Palbociclib	\N	2015-02-06 19:01:14.634492	2015-02-06 19:01:14.634492
52	Palbociclib (PD-0332991)	\N	2015-02-06 19:01:14.768892	2015-02-06 19:01:14.768892
53	Trametinib	\N	2015-02-06 19:01:14.910595	2015-02-06 19:01:14.910595
54	PD0325901	\N	2015-02-06 19:01:15.136098	2015-02-06 19:01:15.136098
55	PLX4720	\N	2015-02-06 19:01:15.148545	2015-02-06 19:01:15.148545
56	Ipilimumab	\N	2015-02-06 19:01:15.377768	2015-02-06 19:01:15.377768
57	Dacarbazine	\N	2015-02-06 19:01:15.411499	2015-02-06 19:01:15.411499
58	Capecitabine	\N	2015-02-06 19:01:15.446461	2015-02-06 19:01:15.446461
59	Bevacizumab	\N	2015-02-06 19:01:15.471848	2015-02-06 19:01:15.471848
60	nutlin-3	\N	2015-02-06 19:01:15.501947	2015-02-06 19:01:15.501947
61	TAE684	\N	2015-02-06 19:01:15.759781	2015-02-06 19:01:15.759781
62	CH5424802	\N	2015-02-06 19:01:15.780019	2015-02-06 19:01:15.780019
63	MK-2206	\N	2015-02-06 19:01:15.889154	2015-02-06 19:01:15.889154
64	Arsenic trioxide	\N	2015-02-06 19:01:15.91133	2015-02-06 19:01:15.91133
\.


--
-- Name: drugs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('drugs_id_seq', 64, true);


--
-- Data for Name: evidence_levels; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_levels (id, level, description, created_at, updated_at) FROM stdin;
1	B	\N	2015-02-06 19:01:07.872405	2015-02-06 19:01:07.872405
2	C	\N	2015-02-06 19:01:07.979385	2015-02-06 19:01:07.979385
3	D	\N	2015-02-06 19:01:08.808069	2015-02-06 19:01:08.808069
4	A	\N	2015-02-06 19:01:09.864882	2015-02-06 19:01:09.864882
\.


--
-- Data for Name: evidence_types; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_types (id, evidence_type, created_at, updated_at) FROM stdin;
1	Predictive	2015-02-06 19:01:07.86997	2015-02-06 19:01:07.86997
2	Prognostic	2015-02-06 19:01:08.038481	2015-02-06 19:01:08.038481
3	Diagnostic	2015-02-06 19:01:08.18307	2015-02-06 19:01:08.18307
\.


--
-- Data for Name: genes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes (id, entrez_id, name, description, official_name, created_at, updated_at, clinical_description) FROM stdin;
1	238	ALK	ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines.	anaplastic lymphoma receptor tyrosine kinase	2015-02-06 19:00:57.462688	2015-02-06 19:00:57.462688	\N
2	207	AKT1	AKT1, also referred to as protein kinase B, has been known as an oncogene for quite some time. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro.	v-akt murine thymoma viral oncogene homolog 1	2015-02-06 19:00:57.716182	2015-02-06 19:00:57.716182	\N
3	369	ARAF	ARAF has recently become increasingly considered for its oncogenic potential. Its potential as a target for informing clinical action was demonstrated by a single case of advanced lung adenocarcinoma harboring an S214C mutation that, when treated with sorafenib, acheived near-complete clinical remission. This finding has brought new focus on ARAF as a marker that should be assayed for in cancer treatment.	A-Raf proto-oncogene, serine/threonine kinase	2015-02-06 19:00:57.896685	2015-02-06 19:00:57.896685	\N
4	25	ABL1	ABL1 is most relevant to cancer in its role in the BCR-ABL fusion protein that has become a signature of chronic myeloid leukemia. Treatment of cells harboring this fusion has shown sensitivity to imatinib, greatly improving the prognostic outlook of the disease. However, mutations in this gene have been shown to confer resistance to imatinib. In these resistance cases, second-generation tyrosine kinase inhibitors such as dasatinib and nilotinib have exhibited some efficacy and are currently undergoing clinical trials for treating acquired resistance in CML.	ABL proto-oncogene 1, non-receptor tyrosine kinase	2015-02-06 19:00:58.077196	2015-02-06 19:00:58.077196	\N
5	673	BRAF	BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been shown to be correlated with poor prognosis in many cancer types, there is at least one study that questions this conclusion in papillary thyroid cancer.	B-Raf proto-oncogene, serine/threonine kinase	2015-02-06 19:00:58.294031	2015-02-06 19:00:58.294031	\N
6	595	CCND1	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D1	2015-02-06 19:00:58.477855	2015-02-06 19:00:58.477855	\N
7	894	CCND2	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly deregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	cyclin D2	2015-02-06 19:00:58.656558	2015-02-06 19:00:58.656558	\N
8	896	CCND3	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-ALL, a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. In a mouse study, the targeted therapeutic palbociclib significantly increased the median survival of the cyclin D3 knockouts.	cyclin D3	2015-02-06 19:00:58.832774	2015-02-06 19:00:58.832774	\N
9	898	CCNE1	Cyclin E, while currenly not as widely implicated as it's counterparts, cyclin D, has been shown to implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, have been shown to lead to poorer prognosis in gastic carcinoma, and these measurements are correlated with later stage disease. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	cyclin E1	2015-02-06 19:00:59.08234	2015-02-06 19:00:59.08234	\N
10	1021	CDK6	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 6	2015-02-06 19:00:59.274857	2015-02-06 19:00:59.274857	\N
11	1019	CDK4	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	cyclin-dependent kinase 4	2015-02-06 19:00:59.458863	2015-02-06 19:00:59.458863	\N
12	1029	CDKN2A	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the promoter region for the gene. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcome for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	cyclin-dependent kinase inhibitor 2A	2015-02-06 19:00:59.655788	2015-02-06 19:00:59.655788	\N
28	3845	KRAS	N/A	Kirsten rat sarcoma viral oncogene homolog	2015-02-06 19:01:02.756533	2015-02-06 19:01:02.756533	\N
29	5604	MAP2K1	N/A	mitogen-activated protein kinase kinase 1	2015-02-06 19:01:02.940809	2015-02-06 19:01:02.940809	\N
30	4209	MEF2D	N/A	myocyte enhancer factor 2D	2015-02-06 19:01:03.109859	2015-02-06 19:01:03.109859	\N
13	1050	CEBPA	'AML with mutated CEBPA' is a provisional entity in the WHO classification of acute myeloid leukemia (AML) and is recommended to be tested for in patients with AML. CEBPA mutations are particularly associated with cytogenetically normal AML (CN-AML). CEBPA is an intronless gene that is required for granulocyte formation in mice. N-terminal nonsense mutations result in a dominant negative C/EBP-alpha protein while C-terminal mutations reduce the DNA-binding potential of this transcription factor. CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	CCAAT/enhancer binding protein (C/EBP), alpha	2015-02-06 19:00:59.838593	2015-02-06 19:00:59.838593	\N
14	3337	DNAJB1	DNAJB1 works upstream of MDM2, stabilizing the complex and facilitating p53 turnover. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker for this rare subtype of HCC. A study has found this fusion in every FL-HCC sample sequenced, and in vitro assays have shown that the fusion retains kinase function.	DnaJ (Hsp40) homolog, subfamily B, member 1	2015-02-06 19:01:00.032965	2015-02-06 19:01:00.032965	\N
15	5566	PRKACA	PRKACA has been studied in breast cancer and has been found to mediate resistance to HER2 targeted therapies. It has also been found to contain a mutation hotspot that contributes to neoplastic behavior in neuroendocrine cancers. In hepatocellular fibrolamellar carcinoma (FL-HCC), the fusion DNAJB1 to PRKACA is suggested to be a diagnostic marker. A study has found this fusion in every sample sequenced, and in vitro assays have shown that the fusion retains kinase function.	protein kinase, cAMP-dependent, catalytic, alpha	2015-02-06 19:01:00.252583	2015-02-06 19:01:00.252583	\N
16	1788	DNMT3A	DNMT3A is one of several epigenetic modifiers identified as recurrently mutated in acute myeloid leukemia (AML). DNMT3A mutations are associated with cytogenetically normal AML. In vitro experiments indicate that the R882H mutation acts in a dominant negative manner to disrupt the de novo methyltransferase activity of wildtype homotetramers. AML patient bone marrow harboring R882 mutations were similarly demonstrated to be hypomethylated compared to patients with wildtype DNMT3A. These studies also indicated that non-R882 DNMT3A mutations may act in a functionally distinct manner from R882 mutations. Alternative mechanisms indicate independent prognostic outcomes and treatment protocols may need to be considered for these two classes of DNMT3A mutations.	DNA (cytosine-5-)-methyltransferase 3 alpha	2015-02-06 19:01:00.450452	2015-02-06 19:01:00.450452	\N
17	1956	EGFR	EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer has spurred many research and drug development efforts. Tyrosine kinase inhibitors have show efficacy in EGFR amplfied tumors, most notably gefitinibb and erlotinib. mutations in EGFR have been shown to confer resistance to these drugs however, and the variant T790M has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has been a common practice in non-small cell lung cancer.	epidermal growth factor receptor	2015-02-06 19:01:00.642228	2015-02-06 19:01:00.642228	\N
18	2064	ERBB2	ERBB2, commonly referred to as HER2, is seen to be amplified in HER2-positive breast cancer, and is treated in a separate manner than the other subtypes of breast cancer. Apart from being amplified, ERBB2 activating mutations have been shown to have clinical importance in HER2-negative breast cancer. These mutations have shown senitivity to the tyrosine kinase inhibitor neratinib, and highlight the importance of clinical sequencing efforts in treating breast cancer.	erb-b2 receptor tyrosine kinase 2	2015-02-06 19:01:00.828012	2015-02-06 19:01:00.828012	\N
19	2099	ESR1	ESR1 has been a focus in breast cancer for quite some time, and has shown significance in endometrial, ovarian and other cancer types as well. ER-positive breast cancer that is resistant to hormone therapy has instigated clinical sequencing efforts to shed light on the mechanisms of this resistance. A number of mutations in the ligand binding domain of the protein have been implicated in hormone resistance and anti-estrogen therapies. This has spurred efforts to develop therapeutics that act to degrade the protein, rather than act as an antagonist. These agents are currently in clinical trials and have seen some success, highlighting the importance of sequencing efforts in treating breast cancer.	estrogen receptor 1	2015-02-06 19:01:01.061937	2015-02-06 19:01:01.061937	\N
20	2263	FGFR2	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. The active proteins have the potential to activate the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 2	2015-02-06 19:01:01.283375	2015-02-06 19:01:01.283375	\N
21	2261	FGFR3	The FGFR proteins are involved in a wide array of pathways known to play a signficant role in cancer. The active proteins have the potential to activate the RAS-MAPK pathway and the PI3K-AKT pathway, among others. The mechanisms by which FGFR can be misregulated vary between cancers. Amplification of the receptors has been observed in lung and breast cancers, coding mutations and deletions have been seen in many cancers, and more recently, FGFR fusions have been demonstrated to have oncogenic potential across multiple cancer types. The targeted therapeutics ponatinib, dovitinib and pazopanib have seen success in treating over-active FGFR signalling, prompting use of diagnostic sequencing targeting the FGFR genes, especially in lung cancer patients.	fibroblast growth factor receptor 3	2015-02-06 19:01:01.464346	2015-02-06 19:01:01.464346	\N
22	2322	FLT3	FLT3 is an important cytokine receptor involved in normal hematopoiesis. Mutations in this gene are common in acute myeloid leukemia (AML) and screening for mutations in this gene has been recommended by the World Health Organization in patients with AML, particularly in cases of cytogenetically normal AML (CN-AML). FLT3 mutations commonly co-occur with mutations such as NPM1 that are associated with CN-AML and likely modulate prognostic impact. While FLT3-ITD mutations have been associated with poorer prognosis in AML, the prognostic impact of FLT3-TKD mutations are still up for debate.	fms-related tyrosine kinase 3	2015-02-06 19:01:01.689531	2015-02-06 19:01:01.689531	\N
23	2624	GATA2	N/A	GATA binding protein 2	2015-02-06 19:01:01.862971	2015-02-06 19:01:01.862971	\N
24	3417	IDH1	N/A	isocitrate dehydrogenase 1 (NADP+), soluble	2015-02-06 19:01:02.032151	2015-02-06 19:01:02.032151	\N
25	3418	IDH2	N/A	isocitrate dehydrogenase 2 (NADP+), mitochondrial	2015-02-06 19:01:02.201413	2015-02-06 19:01:02.201413	\N
26	3717	JAK2	JAK2 is a kinase that is misregulated or mutated in a number of myeloproliferative diseases and cancers. The mutation V617F is the most clinically relevant variant, and is seen in around half of myeloproliferative disorders.	Janus kinase 2	2015-02-06 19:01:02.397796	2015-02-06 19:01:02.397796	\N
27	3815	KIT	N/A	v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog	2015-02-06 19:01:02.563122	2015-02-06 19:01:02.563122	\N
31	1436	CSF1R	N/A	colony stimulating factor 1 receptor	2015-02-06 19:01:03.279026	2015-02-06 19:01:03.279026	\N
32	4255	MGMT	N/A	O-6-methylguanine-DNA methyltransferase	2015-02-06 19:01:03.516756	2015-02-06 19:01:03.516756	\N
33	4869	NPM1	'AML with mutated NPM1' is provisional entity in the WHO classification of AML and is recommended to be tested for in patients with cytogenetically normal AML (CN-AML). Evaluation of FLT3 mutations should be evaluated concurrently as they have prognostic consequences. NPM1 mutations are concentrated in exon 12, most frequently W288fs which results in cytoplasmic sequestration of the protein. Exon 12 NPM1 mutations in the absence of FLT3-ITD are associated with good prognostic outcomes. Mice expressing the Npm1-W288fs mutation develop myeloproliferative neoplasms but not overt leukemia, indicating it may require additional mutations to promote leukemic development.	nucleophosmin (nucleolar phosphoprotein B23, numatrin)	2015-02-06 19:01:03.766691	2015-02-06 19:01:03.766691	\N
34	4893	NRAS	N/A	neuroblastoma RAS viral (v-ras) oncogene homolog	2015-02-06 19:01:03.955974	2015-02-06 19:01:03.955974	\N
35	5290	PIK3CA	PIK3CA is the most recurrently mutated gene in breast cancer, and has been found to important in a number of cancer types. An integral part of the PI3K pathway, PIK3CA has long been described as an oncogene, with two main hotspots for activating  , the 542/545 region of the helical domain, and the 1047 region of the kinase domain. PIK3CA, and its interaction with the AKT and mTOR pathways, is the subject of an immense amount of research and development, and PI3K inhibition has seen some limited success in recent clinical trials. While monotherapies seem to be limited in their potential, there is a recent interest in pursuing PI3K inhibition as part of a combination therapy regiment with inhibition partners including TKI's, MEK inhibitors, PARP inhibitors, and in breast cancer, aromatase inhibitors.	phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha	2015-02-06 19:01:04.135858	2015-02-06 19:01:04.135858	\N
36	5156	PDGFRA	N/A	platelet-derived growth factor receptor, alpha polypeptide	2015-02-06 19:01:04.304868	2015-02-06 19:01:04.304868	\N
37	5371	PML	N/A	promyelocytic leukemia	2015-02-06 19:01:04.487229	2015-02-06 19:01:04.487229	\N
38	5914	RARa	N/A	retinoic acid receptor, alpha	2015-02-06 19:01:04.676386	2015-02-06 19:01:04.676386	\N
39	5728	PTEN	N/A	phosphatase and tensin homolog	2015-02-06 19:01:04.87367	2015-02-06 19:01:04.87367	\N
40	5979	RET	RET mutations and the RET fusion RET-PTC lead to activation of this tyrosine kinase receptor and are associated with thyroid cancers. RET point mutations are the most common mutations identified in medullary thyroid cancer (MTC) with germline and somatic mutations in RET associated with hereditary and sporadic forms, respectively. The most common somatic form mutation is M918T (exon 16) and a variety of other mutations effecting exons 10, 11 and 15 have been described. The prognostic significance of these mutations have been hotly debated in the field, however, data suggests that some RET mutation may confer drug resistence. No RET-specific agents are currently clinically available but several promiscuous kinase inhibitors that target RET, among others, have been approved for MTC treatment.	ret proto-oncogene	2015-02-06 19:01:05.109972	2015-02-06 19:01:05.109972	\N
41	861	RUNX1	N/A	runt-related transcription factor 1	2015-02-06 19:01:05.285121	2015-02-06 19:01:05.285121	\N
42	23451	SF3B1	SF3B1 mutations have been described in several myeloid malignancies, predominantly myelodysplastic syndromes (MDS), as well as other hematologic malignancies and breast cancer. SF3B1 is one of several genes involved in RNA splicing that has been identified as recurrently mutated in MDS and other malignanices. The mutations affecting SF3B1 are typically heterozygous, point mutations suspected to be functionally deleterious with K700E described as a major hotspot mutation. MDS patients with SF3B1 mutations have been reported to have better overall and event-free survival than their wildtype counterparts. Additionally, these mutations are highly associated with subtypes of MDS characterized by ringed sideroblasts (refractory anemia with ringed sideroblasts and refractory cytopenia with multilineage dysplasia and ring sideroblasts).	splicing factor 3b, subunit 1, 155kDa	2015-02-06 19:01:05.499779	2015-02-06 19:01:05.499779	\N
43	7157	TP53	N/A	tumor protein p53	2015-02-06 19:01:05.719968	2015-02-06 19:01:05.719968	\N
44	7248	TSC1	N/A	tuberous sclerosis 1	2015-02-06 19:01:05.893204	2015-02-06 19:01:05.893204	\N
45	7249	TSC2	N/A	tuberous sclerosis 2	2015-02-06 19:01:06.061628	2015-02-06 19:01:06.061628	\N
46	7307	U2AF1	U2AF1 is one of several spliceosome complex genes frequently mutated in a variety of hematologic malignancies, particularly de novo myelodysplastic syndromes (MDS), as well as solid tumors such as lung and pancreatic cancers. Two hotspot mutations (S34 and Q157) occur within the two zinc-finger domains of the U2AF1 protein. These mutations have been associated with altered splicing patterns in vitro and in vivo. U2AF1 mutations in MDS have been associated with an increased risk of transformation to secondary acute myeloid leukemia, however, the impact of these mutations on overall survival has been an area of debate.	U2 small nuclear RNA auxiliary factor 1	2015-02-06 19:01:06.233516	2015-02-06 19:01:06.233516	\N
47	7490	WT1	N/A	Wilms tumor 1	2015-02-06 19:01:06.421974	2015-02-06 19:01:06.421974	\N
\.


--
-- Data for Name: sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY sources (id, pubmed_id, study_type, description, created_at, updated_at) FROM stdin;
1	24889366	\N	Rossi et al., 2014, Int. J. Oncol.	2015-02-06 19:00:57.488673	2015-02-06 19:01:17.405943
2	23401436	\N	Shaw et al., 2013, J. Clin. Oncol.	2015-02-06 19:00:57.501919	2015-02-06 19:01:17.787011
3	16095999	\N	Bellacosa et al., 2005, Adv. Cancer Res.	2015-02-06 19:00:57.718793	2015-02-06 19:01:18.195109
4	15023437	\N	Fresno Vara et al., 2004, Cancer Treat. Rev.	2015-02-06 19:00:57.723821	2015-02-06 19:01:19.654389
5	15676015	\N	Lee et al., 2005, APMIS	2015-02-06 19:00:57.899915	2015-02-06 19:01:21.328152
6	24569458	\N	Imielinski et al., 2014, J. Clin. Invest.	2015-02-06 19:00:57.904998	2015-02-06 19:01:23.174651
7	15719031	\N	Ren, 2005, Nat. Rev. Cancer	2015-02-06 19:00:58.079829	2015-02-06 19:01:23.521981
8	17457302	\N	Weisberg et al., 2007, Nat. Rev. Cancer	2015-02-06 19:00:58.084887	2015-02-06 19:01:23.936958
9	19724843	\N	Li et al., 2009, Oncol. Rep.	2015-02-06 19:00:58.296781	2015-02-06 19:01:25.08784
10	23594689	\N	Pakneshan et al., 2013, Pathology	2015-02-06 19:00:58.301424	2015-02-06 19:01:25.711201
11	12432268	\N	Diehl, Cancer Biol. Ther.	2015-02-06 19:00:58.480598	2015-02-06 19:01:26.128708
12	24387133	\N	Casimiro et al., 2014, Expert Opin Investig Drugs	2015-02-06 19:00:58.485178	2015-02-06 19:01:26.547091
13	14965268	\N	Mazumder et al., 2004, Curr Cancer Drug Targets	2015-02-06 19:00:59.085503	2015-02-06 19:01:29.270763
14	20370706	\N	Graf et al., 2010, Mini Rev Med Chem	2015-02-06 19:00:59.277606	2015-02-06 19:01:29.687815
15	24089445	\N	Sheppard et al., 2013, Clin. Cancer Res.	2015-02-06 19:00:59.282431	2015-02-06 19:01:30.098584
16	20473920	\N	Shima et al., 2011, Int. J. Cancer	2015-02-06 19:00:59.65835	2015-02-06 19:01:30.621587
17	23111194	\N	Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-06 19:00:59.663386	2015-02-06 19:01:31.352971
18	19357394	\N	Vardiman et al., 2009, Blood	2015-02-06 19:00:59.842301	2015-02-06 19:01:32.084077
19	11242107	\N	Pabst et al., 2001, Nat. Genet.	2015-02-06 19:00:59.847108	2015-02-06 19:01:32.710426
20	9012825	\N	Zhang et al., 1997, Proc. Natl. Acad. Sci. U.S.A.	2015-02-06 19:00:59.851284	2015-02-06 19:01:33.130458
21	24578576	\N	Honeyman et al., 2014, Science	2015-02-06 19:01:00.035833	2015-02-06 19:01:33.550323
22	24167195	\N	Kim et al., 2013, Blood	2015-02-06 19:01:00.452956	2015-02-06 19:01:33.964666
23	24656771	\N	Russler-Germain et al., 2014, Cancer Cell	2015-02-06 19:01:00.456681	2015-02-06 19:01:34.382089
24	23953842	\N	Yewale et al., 2013, Biomaterials	2015-02-06 19:01:00.644918	2015-02-06 19:01:35.950126
25	18712184	\N	Charpidou et al., In Vivo	2015-02-06 19:01:00.649493	2015-02-06 19:01:36.470404
26	16947083	\N	Badache et al., 2006, J Mammary Gland Biol Neoplasia	2015-02-06 19:01:00.830703	2015-02-06 19:01:36.894467
27	10878580	\N	Yu et al., 2000, Bioessays	2015-02-06 19:01:00.835344	2015-02-06 19:01:37.410547
28	24185510	\N	Robinson et al., 2013, Nat. Genet.	2015-02-06 19:01:01.064542	2015-02-06 19:01:37.847327
29	22279420	\N	Mohibi et al., 2011, J Carcinog	2015-02-06 19:01:01.069629	2015-02-06 19:01:38.250782
30	18636142	\N	Katoh, 2008, Int. J. Oncol.	2015-02-06 19:01:01.285924	2015-02-06 19:01:38.66439
31	23558953	\N	Wu et al., 2013, Cancer Discov	2015-02-06 19:01:01.290505	2015-02-06 19:01:40.482093
32	12951584	\N	Stirewalt et al., 2003, Nat. Rev. Cancer	2015-02-06 19:01:01.692563	2015-02-06 19:01:40.828674
33	23226219	\N	Chou et al., 2012, PLoS ONE	2015-02-06 19:01:03.769674	2015-02-06 19:01:41.208708
118	21576590		Soon et al., 2011, Arch Dermatol	2015-02-06 19:01:11.856067	2015-02-06 19:02:41.135133
34	18073307	\N	Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-06 19:01:05.112702	2015-02-06 19:01:42.459614
35	25465739	\N	Perri et al., 2014, Crit. Rev. Oncol. Hematol.	2015-02-06 19:01:05.117624	2015-02-06 19:01:42.841363
36	21995386	\N	Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-06 19:01:05.502659	2015-02-06 19:01:44.703303
37	24136165	\N	Cazzola et al., 2013, Blood	2015-02-06 19:01:05.507706	2015-02-06 19:01:45.095138
38	25311244	\N	Okeyo-Owuor et al., 2014, Leukemia	2015-02-06 19:01:06.236552	2015-02-06 19:01:45.45967
39	22158538	\N	Graubert et al., 2012, Nat. Genet.	2015-02-06 19:01:06.241452	2015-02-06 19:01:45.982119
40	23071027			2015-02-06 19:01:07.864421	2015-02-06 19:01:46.497851
41	22923433		Iyer et al., 2012, Science	2015-02-06 19:01:07.949945	2015-02-06 19:01:47.95973
42	19966866		Liang et al., 2010, Oncogene	2015-02-06 19:01:07.976648	2015-02-06 19:01:48.37787
43	20085938		Courtney et al., 2010, J. Clin. Oncol.	2015-02-06 19:01:08.00635	2015-02-06 19:01:49.07222
44	22479427		Carico et al., 2012, PLoS ONE	2015-02-06 19:01:08.035309	2015-02-06 19:01:49.527893
45	23463675		Ponti et al., 2013, J. Clin. Pathol.	2015-02-06 19:01:08.063	2015-02-06 19:01:50.03253
46	23031422		Ponti et al., 2012, J Hematol Oncol	2015-02-06 19:01:08.130987	2015-02-06 19:01:50.407072
47	16076867		Schnittger et al., 2005, Blood	2015-02-06 19:01:08.154432	2015-02-06 19:01:50.780527
48	17957027		Gale et al., 2008, Blood	2015-02-06 19:01:08.239869	2015-02-06 19:01:52.137614
49	22490330		Ribeiro et al., 2012, Blood	2015-02-06 19:01:08.265312	2015-02-06 19:01:52.764311
50	16051734		D√∂hner et al., 2005, Blood	2015-02-06 19:01:08.2883	2015-02-06 19:01:53.219776
51	20026798		Becker et al., 2010, J. Clin. Oncol.	2015-02-06 19:01:08.31618	2015-02-06 19:01:53.709106
52	24855211		Linch et al., 2014, Blood	2015-02-06 19:01:08.340303	2015-02-06 19:01:54.226011
53	16455956		Thiede et al., 2006, Blood	2015-02-06 19:01:08.366263	2015-02-06 19:01:54.749371
54	21537333		Schnittger et al., 2011, Leukemia	2015-02-06 19:01:08.389968	2015-02-06 19:01:55.149575
55	21067377		Ley et al., 2010, N. Engl. J. Med.	2015-02-06 19:01:08.434813	2015-02-06 19:01:55.559446
56	21719597		Balusu et al., 2011, Blood	2015-02-06 19:01:08.481019	2015-02-06 19:01:56.773558
57	19965647		Burnett et al., 2010, Blood	2015-02-06 19:01:08.512733	2015-02-06 19:01:57.256301
58	19059939		Schlenk et al., 2009, Haematologica	2015-02-06 19:01:08.60305	2015-02-06 19:01:57.675712
59	20805365		Schnittger et al., 2010, Blood	2015-02-06 19:01:08.644723	2015-02-06 19:01:58.196383
60	17965322		Bacher et al., 2008, Blood	2015-02-06 19:01:08.687561	2015-02-06 19:01:58.611735
61	19047294		B√ºchner et al., 2009, J. Clin. Oncol.	2015-02-06 19:01:08.712335	2015-02-06 19:01:59.020566
62	22430270		Gaidzik et al., 2012, J. Clin. Oncol.	2015-02-06 19:01:08.734338	2015-02-06 19:02:00.866498
63	24859829		Tian et al., 2014, Int. J. Hematol.	2015-02-06 19:01:08.756227	2015-02-06 19:02:02.096347
64	15659725		Falini et al., 2005, N. Engl. J. Med.	2015-02-06 19:01:08.806572	2015-02-06 19:02:02.451758
65	22417203		Patel et al., 2012, N. Engl. J. Med.	2015-02-06 19:01:08.858284	2015-02-06 19:02:02.897811
66	24797300		Tassara et al., 2014, Blood	2015-02-06 19:01:08.882878	2015-02-06 19:02:03.478906
67	19587375		Schnittger et al., 2009, Blood	2015-02-06 19:01:08.927426	2015-02-06 19:02:03.944767
68	24927407		Ehninger et al., 2014, Blood Cancer J	2015-02-06 19:01:08.972999	2015-02-06 19:02:04.671281
69	18450602		Schlenk et al., 2008, N. Engl. J. Med.	2015-02-06 19:01:09.02198	2015-02-06 19:02:06.484593
70	24576830		Nissan et al., 2014, Cancer Res.	2015-02-06 19:01:09.045164	2015-02-06 19:02:06.842051
71	21422803		Coxon et al., 2012, J. Endocrinol. Invest.	2015-02-06 19:01:09.094971	2015-02-06 19:02:07.228159
72	20537386		An et al., 2010, Leuk. Res.	2015-02-06 19:01:09.144877	2015-02-06 19:02:07.920362
73	24550739		Borad et al., 2014, PLoS Genet.	2015-02-06 19:01:09.186883	2015-02-06 19:02:08.478487
74	23558953		Wu et al., 2013, Cancer Discov	2015-02-06 19:01:09.213491	2015-02-06 19:02:09.059701
75	16384925		Cairoli et al., 2006, Blood	2015-02-06 19:01:09.576189	2015-02-06 19:02:11.92521
76	19671763		Woodman et al., 2009, Mol. Cancer Ther.	2015-02-06 19:01:09.604846	2015-02-06 19:02:12.403906
77	17372901		Antonescu et al., 2007, Int. J. Cancer	2015-02-06 19:01:09.655221	2015-02-06 19:02:12.924764
78	16638875		Prenen et al., 2006, Clin. Cancer Res.	2015-02-06 19:01:09.701389	2015-02-06 19:02:13.340641
79	16954519		Heinrich et al., 2006, J. Clin. Oncol.	2015-02-06 19:01:09.746499	2015-02-06 19:02:14.801171
80	18421059		Hodi et al., 2008, J. Clin. Oncol.	2015-02-06 19:01:09.770444	2015-02-06 19:02:15.329031
81	24185510		Robinson et al., 2013, Nat. Genet.	2015-02-06 19:01:09.794674	2015-02-06 19:02:16.056166
82	24185512		Toy et al., 2013, Nat. Genet.	2015-02-06 19:01:09.962088	2015-02-06 19:02:16.475511
83	24578576		Honeyman et al., 2014, Science	2015-02-06 19:01:10.085566	2015-02-06 19:02:16.92637
84	21030459		Sasaki et al., 2010, Cancer Res.	2015-02-06 19:01:10.133351	2015-02-06 19:02:17.726247
85	20979473		Choi et al., 2010, N. Engl. J. Med.	2015-02-06 19:01:10.158019	2015-02-06 19:02:18.144268
86	8674046		Yoshida et al., 1996, Cancer Res.	2015-02-06 19:01:10.237023	2015-02-06 19:02:18.564124
87	24186003		Lilljebj√∂rn et al., 2014, Leukemia	2015-02-06 19:01:10.273887	2015-02-06 19:02:18.981821
88	21995386		Papaemmanuil et al., 2011, N. Engl. J. Med.	2015-02-06 19:01:10.336085	2015-02-06 19:02:20.445653
89	19221039		Gaidzik et al., 2009, Blood	2015-02-06 19:01:10.3727	2015-02-06 19:02:20.871652
90	18591546		Virappane et al., 2008, J. Clin. Oncol.	2015-02-06 19:01:10.406808	2015-02-06 19:02:23.892057
91	18559874		Paschka et al., 2008, J. Clin. Oncol.	2015-02-06 19:01:10.474193	2015-02-06 19:02:26.516025
92	19536888		Renneville et al., 2009, Cancer	2015-02-06 19:01:10.610392	2015-02-06 19:02:26.915826
93	23029227		Qian et al., 2012, PLoS ONE	2015-02-06 19:01:10.644175	2015-02-06 19:02:27.47372
94	23861105		Wu et al., 2013, Am. J. Hematol.	2015-02-06 19:01:10.677988	2015-02-06 19:02:27.859496
95	22698404		Jackson et al., 2012, Cancer Cell	2015-02-06 19:01:10.789366	2015-02-06 19:02:29.058446
96	16489069		Olivier et al., 2006, Clin. Cancer Res.	2015-02-06 19:01:10.808183	2015-02-06 19:02:29.372496
97	9569050		Berns et al., 1998, Br. J. Cancer	2015-02-06 19:01:10.82599	2015-02-06 19:02:29.983084
98	18073307		Elisei et al., 2008, J. Clin. Endocrinol. Metab.	2015-02-06 19:01:11.005565	2015-02-06 19:02:30.303179
99	9839497		Egawa et al., 1998, Jpn. J. Clin. Oncol.	2015-02-06 19:01:11.024359	2015-02-06 19:02:31.824
100	23056499		Couto et al., 2012, PLoS ONE	2015-02-06 19:01:11.042944	2015-02-06 19:02:32.246358
101	19915144		Emery et al., 2009, Proc. Natl. Acad. Sci. U.S.A.	2015-02-06 19:01:11.063726	2015-02-06 19:02:32.664947
102	21558396		Tanaka et al., 2011, Clin. Cancer Res.	2015-02-06 19:01:11.099315	2015-02-06 19:02:33.084208
103	24559322		Sartore-Bianchi et al., 2014, Expert Rev Anticancer Ther	2015-02-06 19:01:11.149005	2015-02-06 19:02:33.604789
104	19223544		Sartore-Bianchi et al., 2009, Cancer Res.	2015-02-06 19:01:11.20669	2015-02-06 19:02:34.750924
105	20619739		De Roock et al., 2010, Lancet Oncol.	2015-02-06 19:01:11.293572	2015-02-06 19:02:35.278537
106	15647370		Kang et al., 2005, Proc. Natl. Acad. Sci. U.S.A.	2015-02-06 19:01:11.326997	2015-02-06 19:02:35.693111
107	22357840		Liao et al., 2012, Clin. Cancer Res.	2015-02-06 19:01:11.431324	2015-02-06 19:02:36.112396
108	22745105		Heinrich et al., 2012, Clin. Cancer Res.	2015-02-06 19:01:11.46966	2015-02-06 19:02:36.527982
109	15928335		Corless et al., 2005, J. Clin. Oncol.	2015-02-06 19:01:11.574397	2015-02-06 19:02:36.946492
110	15146165		Lasota et al., 2004, Lab. Invest.	2015-02-06 19:01:11.596761	2015-02-06 19:02:37.467495
111	23569304		Trunzer et al., 2013, J. Clin. Oncol.	2015-02-06 19:01:11.619364	2015-02-06 19:02:37.884772
112	15951308		Bowen et al., 2005, Blood	2015-02-06 19:01:11.685202	2015-02-06 19:02:38.303886
113	16434492		Bacher et al., 2006, Blood	2015-02-06 19:01:11.702583	2015-02-06 19:02:38.622274
114	24666267		Therkildsen et al., 2014, Acta Oncol	2015-02-06 19:01:11.720448	2015-02-06 19:02:39.138373
115	22180178		Jakob et al., 2012, Cancer	2015-02-06 19:01:11.777589	2015-02-06 19:02:39.556831
116	23861977		Tschandl et al., 2013, PLoS ONE	2015-02-06 19:01:11.815935	2015-02-06 19:02:40.081199
117	18375819		Banerji et al., 2008, Mol. Cancer Ther.	2015-02-06 19:01:11.834802	2015-02-06 19:02:40.717815
119	19357394		Vardiman et al., 2009, Blood	2015-02-06 19:01:11.890287	2015-02-06 19:02:42.105986
120	15758010		Hegi et al., 2005, N. Engl. J. Med.	2015-02-06 19:01:11.911642	2015-02-06 19:02:42.480097
121	11070098		Esteller et al., 2000, N. Engl. J. Med.	2015-02-06 19:01:11.932468	2015-02-06 19:02:43.943635
122	16618717		Li√®vre et al., 2006, Cancer Res.	2015-02-06 19:01:11.971077	2015-02-06 19:02:44.571499
123	19029981		Engelman et al., 2008, Nat. Med.	2015-02-06 19:01:11.992798	2015-02-06 19:02:45.201677
124	20978259		De Roock et al., 2010, JAMA	2015-02-06 19:01:12.025831	2015-02-06 19:02:46.486656
125	19934290		Ogino et al., 2009, Clin. Cancer Res.	2015-02-06 19:01:12.115636	2015-02-06 19:02:47.184106
126	15696205		Pao et al., 2005, PLoS Med.	2015-02-06 19:01:12.136688	2015-02-06 19:02:47.510841
127	18202412		Li√®vre et al., 2008, J. Clin. Oncol.	2015-02-06 19:01:12.166196	2015-02-06 19:02:48.226317
128	24571676		Andrade et al., 2014, BMC Cancer	2015-02-06 19:01:12.218483	2015-02-06 19:02:49.419653
129	23014527		Dogan et al., 2012, Clin. Cancer Res.	2015-02-06 19:01:12.254187	2015-02-06 19:02:50.216926
130	18528420		Chng et al., 2008, Leukemia	2015-02-06 19:01:12.272945	2015-02-06 19:02:50.632224
131	18794081		Riely et al., 2008, Clin. Cancer Res.	2015-02-06 19:01:12.310233	2015-02-06 19:02:51.256892
132	11208838		Schiller et al., 2001, J. Clin. Oncol.	2015-02-06 19:01:12.330097	2015-02-06 19:02:51.779947
133	15597105		Mascaux et al., 2005, Br. J. Cancer	2015-02-06 19:01:12.350221	2015-02-06 19:02:52.302491
134	23524406		Rudin et al., 2013, J Thorac Oncol	2015-02-06 19:01:12.397911	2015-02-06 19:02:52.667281
135	23775962		Hodi et al., 2013, J. Clin. Oncol.	2015-02-06 19:01:12.4326	2015-02-06 19:02:54.740341
136	12000708		Corless et al., 2002, Am. J. Pathol.	2015-02-06 19:01:12.486046	2015-02-06 19:02:55.954876
137	16551858		Wardelmann et al., 2006, Clin. Cancer Res.	2015-02-06 19:01:12.503367	2015-02-06 19:02:57.417588
138	10485475		Taniguchi et al., 1999, Cancer Res.	2015-02-06 19:01:12.522274	2015-02-06 19:02:58.879789
139	15217946		Boldrini et al., 2004, Clin. Cancer Res.	2015-02-06 19:01:12.61292	2015-02-06 19:02:59.852801
140	16709929		Kiladjian et al., 2006, Blood	2015-02-06 19:01:12.650303	2015-02-06 19:03:00.450246
141	16081687		Levine et al., 2005, Blood	2015-02-06 19:01:12.669353	2015-02-06 19:03:00.866375
142	19287384		Kilpivaara et al., 2009, Nat. Genet.	2015-02-06 19:01:12.723791	2015-02-06 19:03:01.286676
143	18394554		Wernig et al., 2008, Cancer Cell	2015-02-06 19:01:12.74558	2015-02-06 19:03:02.333404
144	20421455		Thol et al., 2010, Blood	2015-02-06 19:01:12.766423	2015-02-06 19:03:03.993416
145	21596855		Green et al., 2011, Blood	2015-02-06 19:01:12.785448	2015-02-06 19:03:04.416239
146	22033490		Patnaik et al., 2012, Leukemia	2015-02-06 19:01:12.822088	2015-02-06 19:03:04.792591
147	22616558		Zhou et al., 2012, Leuk. Lymphoma	2015-02-06 19:01:12.839194	2015-02-06 19:03:05.147049
148	21997850		Lin et al., 2012, Ann. Hematol.	2015-02-06 19:01:12.872556	2015-02-06 19:03:05.520922
149	20494930		Thol et al., 2010, Haematologica	2015-02-06 19:01:12.906646	2015-02-06 19:03:07.380552
150	20538800		Abbas et al., 2010, Blood	2015-02-06 19:01:12.927814	2015-02-06 19:03:08.698999
151	19933982		Dubbink et al., 2009, Neurology	2015-02-06 19:01:12.985132	2015-02-06 19:03:10.405039
152	20127344		Bleeker et al., 2010, Acta Neuropathol.	2015-02-06 19:01:13.006848	2015-02-06 19:03:10.788269
153	20376086		Ho et al., 2010, Leukemia	2015-02-06 19:01:13.029304	2015-02-06 19:03:11.205633
154	20368538		Wagner et al., 2010, J. Clin. Oncol.	2015-02-06 19:01:13.047048	2015-02-06 19:03:11.727654
155	22624710		Barbacid, 2012, Cancer Cell	2015-02-06 19:01:13.097458	2015-02-06 19:03:12.248413
156	18230792		Zhang et al., 2008, J. Natl. Cancer Inst.	2015-02-06 19:01:13.145473	2015-02-06 19:03:12.574343
157	22368270		Man et al., 2012, Blood	2015-02-06 19:01:13.166064	2015-02-06 19:03:12.980157
158	14726387		Smith et al., 2004, Blood	2015-02-06 19:01:13.201077	2015-02-06 19:03:13.438937
159	12357354		Tse et al., 2002, Leukemia	2015-02-06 19:01:13.237563	2015-02-06 19:03:13.780105
160	11585760		Whitman et al., 2001, Cancer Res.	2015-02-06 19:01:13.304159	2015-02-06 19:03:14.801316
161	17940205		Whitman et al., 2008, Blood	2015-02-06 19:01:13.372446	2015-02-06 19:03:16.291726
162	15626738		Bagrintseva et al., 2005, Blood	2015-02-06 19:01:13.392753	2015-02-06 19:03:16.741543
163	11290608		Yamamoto et al., 2001, Blood	2015-02-06 19:01:13.47915	2015-02-06 19:03:17.089164
164	24801015		Port et al., 2014, Ann. Hematol.	2015-02-06 19:01:13.601996	2015-02-06 19:03:17.680928
165	23220880		Bose et al., 2013, Cancer Discov	2015-02-06 19:01:13.62342	2015-02-06 19:03:21.201552
166	24457318		Fukihara et al., 2014, Oncology	2015-02-06 19:01:13.774072	2015-02-06 19:03:22.172471
167	24729716		Li et al., 2014, Onco Targets Ther	2015-02-06 19:01:13.79322	2015-02-06 19:03:22.551199
168	24623981		Ding et al., 2014, Onco Targets Ther	2015-02-06 19:01:13.811628	2015-02-06 19:03:22.919466
169	24636847		Li et al., 2014, Lung Cancer	2015-02-06 19:01:13.830859	2015-02-06 19:03:23.26372
170	24736073		Lim et al., 2014, J Thorac Oncol	2015-02-06 19:01:13.860427	2015-02-06 19:03:23.633689
171	24585406		Li et al., 2014, Ann. Surg. Oncol.	2015-02-06 19:01:13.887677	2015-02-06 19:03:24.050102
172	15728811		Kobayashi et al., 2005, N. Engl. J. Med.	2015-02-06 19:01:13.905362	2015-02-06 19:03:24.46886
173	24662454		Douillard et al., 2014, J Thorac Oncol	2015-02-06 19:01:13.922721	2015-02-06 19:03:24.992679
174	24658966		Ai et al., 2014, Amino Acids	2015-02-06 19:01:13.943553	2015-02-06 19:03:25.513097
175	24512939		El Ghannam et al., Blood Cells Mol. Dis.	2015-02-06 19:01:14.014993	2015-02-06 19:03:26.035196
176	22081665		LaRochelle et al., 2011, Oncotarget	2015-02-06 19:01:14.03473	2015-02-06 19:03:26.453771
177	22291079		Marcucci et al., 2012, J. Clin. Oncol.	2015-02-06 19:01:14.069247	2015-02-06 19:03:26.980085
178	23632886		Gaidzik et al., 2013, Blood	2015-02-06 19:01:14.225512	2015-02-06 19:03:27.497744
179	14726504		Fr√∂hling et al., 2004, J. Clin. Oncol.	2015-02-06 19:01:14.521184	2015-02-06 19:03:28.019495
180	23111194		Bradly et al., 2012, Diagn. Mol. Pathol.	2015-02-06 19:01:14.576463	2015-02-06 19:03:28.438753
181	25002028		Vora et al., 2014, Cancer Cell	2015-02-06 19:01:14.612559	2015-02-06 19:03:29.31275
182	23898052		Logan et al., 2013, Anticancer Res.	2015-02-06 19:01:14.636303	2015-02-06 19:03:29.695063
183	12432043		Keyomarsi et al., 2002, N. Engl. J. Med.	2015-02-06 19:01:14.692456	2015-02-06 19:03:30.053727
184	10224221		Donnellan et al., 1999, FASEB J.	2015-02-06 19:01:14.710691	2015-02-06 19:03:30.410297
185	23079656		Sawai et al., 2012, Cancer Cell	2015-02-06 19:01:14.770254	2015-02-06 19:03:30.787381
186	10547574		Takano et al., 1999, J. Pathol.	2015-02-06 19:01:14.791921	2015-02-06 19:03:31.156313
187	14612939		Oshimo et al., 2003, Int. J. Oncol.	2015-02-06 19:01:14.862771	2015-02-06 19:03:31.719705
188	24583796		Menzies et al., 2014, Clin. Cancer Res.	2015-02-06 19:01:14.898001	2015-02-06 19:03:33.127861
189	24594804		Chen et al., 2014, PLoS ONE	2015-02-06 19:01:14.931164	2015-02-06 19:03:33.462988
190	24588959		Zhang et al., 2014, Diagn Pathol	2015-02-06 19:01:14.950325	2015-02-06 19:03:35.151263
191	24570209		Crescenzi et al., 2014, Horm. Metab. Res.	2015-02-06 19:01:15.040349	2015-02-06 19:03:35.542071
192	24354346		Walczyk et al., 2014, Clin. Endocrinol. (Oxf)	2015-02-06 19:01:15.061009	2015-02-06 19:03:35.959717
193	21098728		Corcoran et al., 2010, Sci Signal	2015-02-06 19:01:15.07884	2015-02-06 19:03:36.37919
194	21166657		Lee et al., 2011, Br. J. Dermatol.	2015-02-06 19:01:15.099641	2015-02-06 19:03:38.291576
195	23325582		Peeters et al., 2013, Clin. Cancer Res.	2015-02-06 19:01:15.117841	2015-02-06 19:03:38.901729
196	23845441		Rad et al., 2013, Cancer Cell	2015-02-06 19:01:15.137545	2015-02-06 19:03:39.301635
197	19001320		Di Nicolantonio et al., 2008, J. Clin. Oncol.	2015-02-06 19:01:15.170788	2015-02-06 19:03:39.72156
198	23020132		Flaherty et al., 2012, N. Engl. J. Med.	2015-02-06 19:01:15.213418	2015-02-06 19:03:40.145252
199	24388723		Nagore et al., 2014, J. Am. Acad. Dermatol.	2015-02-06 19:01:15.264775	2015-02-06 19:03:40.556043
200	21594703		Howell et al., 2011, Ann. Surg. Oncol.	2015-02-06 19:01:15.283608	2015-02-06 19:03:40.975193
201	24396464		He et al., 2014, Oncol Lett	2015-02-06 19:01:15.320592	2015-02-06 19:03:42.43518
202	21505228		Corcoran et al., 2011, Oncotarget	2015-02-06 19:01:15.357176	2015-02-06 19:03:42.854166
203	24577748		Ackerman et al., 2014, Cancer	2015-02-06 19:01:15.379389	2015-02-06 19:03:43.273098
204	24586605		Meckbach et al., 2014, PLoS ONE	2015-02-06 19:01:15.412924	2015-02-06 19:03:43.688939
205	22180495		Yang et al., 2012, Cancer Res.	2015-02-06 19:01:15.447977	2015-02-06 19:03:44.110247
206	23812671		Ji et al., 2013, Clin. Cancer Res.	2015-02-06 19:01:15.490234	2015-02-06 19:03:44.530087
207	15961768		Arnold et al., 2005, J. Clin. Oncol.	2015-02-06 19:01:15.557433	2015-02-06 19:03:44.939799
208	16309541		Thomas et al., 2005, Int J Exp Pathol	2015-02-06 19:01:15.596307	2015-02-06 19:03:45.48793
209	17891190		Jares et al., 2007, Nat. Rev. Cancer	2015-02-06 19:01:15.616144	2015-02-06 19:03:45.990122
210	17070615		Gautschi et al., 2007, Lung Cancer	2015-02-06 19:01:15.635259	2015-02-06 19:03:46.427606
211	24569458		Imielinski et al., 2014, J. Clin. Invest.	2015-02-06 19:01:15.690807	2015-02-06 19:03:46.928842
212	18923525		George et al., 2008, Nature	2015-02-06 19:01:15.761164	2015-02-06 19:03:48.560971
213	21575866		Sakamoto et al., 2011, Cancer Cell	2015-02-06 19:01:15.781582	2015-02-06 19:03:48.912277
214	22072639		Bresler et al., 2011, Sci Transl Med	2015-02-06 19:01:15.85305	2015-02-06 19:03:49.432473
215	21948233		Heuckmann et al., 2011, Clin. Cancer Res.	2015-02-06 19:01:15.871053	2015-02-06 19:03:51.364684
216	23888070		Beaver et al., 2013, Clin. Cancer Res.	2015-02-06 19:01:15.89061	2015-02-06 19:03:51.738062
217	12476305		Nimmanapalli et al., 2002, Oncogene	2015-02-06 19:01:15.91293	2015-02-06 19:03:52.147616
218	25212276		Schmidt et al., 2014, Leukemia	2015-02-06 19:01:16.027323	2015-02-06 19:03:52.474446
\.


--
-- Data for Name: variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variants (id, gene_id, name, description, created_at, updated_at) FROM stdin;
1	4	BCR-ABL	The BCR-ABL fusion protein, commonly referred to as the philadelphia chromosome, is the most well-studied fusion gene in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML), but despite its ability initiate disease in mice, its status an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response.	2015-02-06 19:01:06.465919	2015-02-06 19:01:06.465919
2	4	BCR-ABL T315I	While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of seleciton have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.	2015-02-06 19:01:06.480118	2015-02-06 19:01:06.480118
3	2	E17K	AKT1 E17K is a recurrent mutation that has been observed in breast, colorectal, lung, and ovarian cancer. It has been convincingly shown to be an activating mutation resulting in PI3K/AKT/mTOR pathway activity. It has been suggested that this mutation decreases the cell's sensitivity to AKT1 allosteric kinase ihibitors. This, and other AKT1 mutations, are the subject of much research and development for therapeutics.	2015-02-06 19:01:06.489144	2015-02-06 19:01:06.489144
4	1	EML4-ALK	The EML4-ALK fusion has been seen in non-small cell lung cancer, and appears to be an alternative mechanism for ALK activation. Cells with this fusion have been shown to be sensitive to the ALK inhibitor crizotinib.	2015-02-06 19:01:06.498418	2015-02-06 19:01:06.498418
5	1	EML4-ALK C1156Y	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-06 19:01:06.508584	2015-02-06 19:01:06.508584
6	1	EML4-ALK L1196M	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	2015-02-06 19:01:06.518664	2015-02-06 19:01:06.518664
7	1	F1174L	ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells.	2015-02-06 19:01:06.528137	2015-02-06 19:01:06.528137
8	1	R1275Q	ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This, and the geldanamycin deriviative 17-DMAG, has been shown to be effective in NSCLC cell lines.	2015-02-06 19:01:06.538961	2015-02-06 19:01:06.538961
9	3	S214C	ARAF S214C has been found to be a recurrent oncogenic mutation in non-small cell lung cancer. It has been shown to confer sensitivity to sorafenib and trametenib in cell lines. In a case study of advanced stage lung adenocarcinoma harboring this mutation, sorafenib also acheived near-complete clinical remission. This case has brought more interest to the variant from a research and clinical perspective.	2015-02-06 19:01:06.548218	2015-02-06 19:01:06.548218
10	5	V600D	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-06 19:01:06.556827	2015-02-06 19:01:06.556827
11	5	V600E	BRAF V600E has been shown to be recurrent in many cancer types. It is one of the most widely studied variants in cancer. This variant is correlated with poor prognosis in certain cancer types, including colorectal cancer and papillary thyroid cancer. The targeted therapeutic dabrafenib has been shown to be effective in clinical trials with an array of BRAF mutations and cancer types. Dabrafenib has also shown to be effective when combined with the MEK inhibitor trametinib in colorectal cancer and melanoma. However, in patients with TP53, KRAS, and CDK2NA mutations, dabrafenib resistance has been reported. Ipilimumab, regorafenib, vemurafenib, and a number of combination therapies have been successful in treating V600E mutations. While the drugs cetuximab and panitumumab have been largely shown to be ineffective without supplementary treatment.	2015-02-06 19:01:06.56543	2015-02-06 19:01:06.56543
12	5	V600E/V600M	A case study of a single patient harboring both a V600E and a V600M mutation, dabrafenib was shown to acheive clinical response.	2015-02-06 19:01:06.574106	2015-02-06 19:01:06.574106
13	5	V600E AMPLIFICATION	Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.	2015-02-06 19:01:06.582375	2015-02-06 19:01:06.582375
14	5	V600M	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-06 19:01:06.590965	2015-02-06 19:01:06.590965
15	5	V600R	Patients harboring mutations in valine 600 residue of BRAF have been shown to be sensitive to dabrafenib. For more information on the V600 locus, see the V600E entry.	2015-02-06 19:01:06.599435	2015-02-06 19:01:06.599435
16	5	V600	BRAF mutations of the valine 600 residue have been shown to be recurrent in many cancer types. Of the V600 mutations, V600E is the most widely researched. V600 mutations as a whole have been correlated to poorer prognosis in colorectal and papilarry thyroid cancers. V600 mutations have also been shown to confer sensitivity to the BRAF inhibitor dabrafenib. For a more detailed summary, click the individual mutation pages on the left sidebar.	2015-02-06 19:01:06.608393	2015-02-06 19:01:06.608393
17	6	AMPLIFICATION	CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.	2015-02-06 19:01:06.617117	2015-02-06 19:01:06.617117
18	6	EXPRESSION	CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.	2015-02-06 19:01:06.625531	2015-02-06 19:01:06.625531
19	6	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-06 19:01:06.636025	2015-02-06 19:01:06.636025
20	7	EXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-06 19:01:06.646758	2015-02-06 19:01:06.646758
21	7	OVEREXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-06 19:01:06.657587	2015-02-06 19:01:06.657587
22	7	PROMOTER DEMETHYLATION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-06 19:01:06.667213	2015-02-06 19:01:06.667213
23	8	EXPRESSION	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.	2015-02-06 19:01:06.677651	2015-02-06 19:01:06.677651
24	8	LOSS	Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.	2015-02-06 19:01:06.686648	2015-02-06 19:01:06.686648
25	9	OVEREXPRESSION	Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.	2015-02-06 19:01:06.695689	2015-02-06 19:01:06.695689
26	11	EXPRESSION	CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-06 19:01:06.704581	2015-02-06 19:01:06.704581
27	10	EXPRESSION	CDK6, along with its partner CDK4, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.	2015-02-06 19:01:06.713202	2015-02-06 19:01:06.713202
28	12	PROMOTER HYPERMETHYLATION	CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.	2015-02-06 19:01:06.723135	2015-02-06 19:01:06.723135
29	13	N-TERMINAL FRAME SHIFT	CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis, however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-06 19:01:06.733289	2015-02-06 19:01:06.733289
30	13	WHOLE GENE	CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.	2015-02-06 19:01:06.742151	2015-02-06 19:01:06.742151
31	31	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-06 19:01:06.750657	2015-02-06 19:01:06.750657
32	14	DNAJB1-PRKACA	This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-06 19:01:06.761349	2015-02-06 19:01:06.761349
125	43	R249W	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-06 19:01:07.650952	2015-02-06 19:01:07.650952
33	16	R882	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-06 19:01:06.770503	2015-02-06 19:01:06.770503
34	16	R882H	DNMT3A R882 mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) with R882H as the most common form. Mutations in DNMT3A have largely been associated with poorer prognosis, however this is not consistent across all studies. This may be a result of patient age or combining R882 and non-R882 mutations during analysis as studies have indicated independent mechanisms of action and differential prognostic implications for these mutation types. One study that independently analyzed R882 and non-R882 mutations showed R882 mutations were associated with poorer prognosis than patients with wildtype and non-R882 mutations, but only in older patients with AML.	2015-02-06 19:01:06.780347	2015-02-06 19:01:06.780347
35	17	L858R	EGFR L858R has long been recognized as a functionally significant mutation in cancer, and is one of the most prevalent single mutations in lung cancer. Best described in non-small cell lung cancer (NSCLC), the mutation seems to confer sensitivity to first and second generation TKI's like gefitinib and neratinib. NSCLC patients with this mutation treated with TKI's show increased overall and progression-free survival, as compared to chemotherapy alone. Third generation TKI's are currently in clinical trials that specifically focus on mutant forms of EGFR, a few of which have shown efficacy in treating patients that failed to respond to earlier generation TKI therapies.	2015-02-06 19:01:06.790567	2015-02-06 19:01:06.790567
36	17	T790M	EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. The third generation TKI's are being developed with this resistance problem in mind, and early stage experiments have shown some efficacy of these drugs in previously resistant tumors with mutant EGFR.	2015-02-06 19:01:06.801002	2015-02-06 19:01:06.801002
37	18	D769H	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.809756	2015-02-06 19:01:06.809756
38	18	DEL 755-759	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.819005	2015-02-06 19:01:06.819005
39	18	G309A	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.828607	2015-02-06 19:01:06.828607
40	18	L755S	This variant was one of the first of the ERBB2 variants to be functionally classified (Bose et al. 2012). This mutation was not shown to be an activating mutation, unlike many of the other variants queried. This mutation was also shown to confer resistance to the tyrosine kinase inhibitor lapatinib in cell lines.	2015-02-06 19:01:06.836955	2015-02-06 19:01:06.836955
41	18	L755W	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.845706	2015-02-06 19:01:06.845706
42	18	P780INS	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.854529	2015-02-06 19:01:06.854529
43	18	R678Q	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.863215	2015-02-06 19:01:06.863215
44	18	R896C	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.872591	2015-02-06 19:01:06.872591
45	18	V777L	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.881491	2015-02-06 19:01:06.881491
46	18	V842I	This variant was one of the first of the ERBB2 variants (aka HER2) to be functionally classified (Bose et al. 2012). This mutation was shown to be an activating mutation in an in vitro assay. In the same paper, this mutation (along with other ERBB2 activating mutations) in MCF10A breast cancer cell lines have been shown to be sensitive to the kinase inhibitor neratinib. More recent evidence may show that HER2 acitivating mutations confer sensitivity to a host of tyrosine kinase inhibitors, which is the topic of current clinical trials and research.	2015-02-06 19:01:06.890247	2015-02-06 19:01:06.890247
47	19	L536Q	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y536Q is one of these ligand binding domain mutations, and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-06 19:01:06.899109	2015-02-06 19:01:06.899109
48	19	N538G	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. N538G is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-06 19:01:06.908335	2015-02-06 19:01:06.908335
49	19	Y537C	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537C is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-06 19:01:06.918203	2015-02-06 19:01:06.918203
50	19	Y537N	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537N is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-06 19:01:06.927079	2015-02-06 19:01:06.927079
51	19	Y537S	ESR1 biology has become a central focus in breast cancer therapy. In ER+ tumors, mutations in the ESR1 ligand binding domain have been shown to confer resistance to hormone therapy. This evidence has led to an increased use of targeted sequencing of the estrogen receptor in breast and ovarian cancer. Y537S is one of these ligand binding domain  , and is commonly implicated in this hormone resistance. Preliminary data suggests ER-degrading agents, such as fulvestrant, may be beneficial in treating ESR1 mutant, hormone resistant breast cancers.	2015-02-06 19:01:06.936658	2015-02-06 19:01:06.936658
52	20	FGFR2-AFF3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:06.945759	2015-02-06 19:01:06.945759
53	20	FGFR2-BICC1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:06.954481	2015-02-06 19:01:06.954481
54	20	FGFR2-CASP7	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:06.963539	2015-02-06 19:01:06.963539
55	20	FGFR2-CCDC6	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:06.972871	2015-02-06 19:01:06.972871
56	20	FGFR2-KIAA1967	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:06.982163	2015-02-06 19:01:06.982163
57	20	FGFR2-MGEA5	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:06.990858	2015-02-06 19:01:06.990858
58	20	FGFR2-OFD1	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:07.000952	2015-02-06 19:01:07.000952
59	20	FGFR2-SLC45A3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:07.012856	2015-02-06 19:01:07.012856
60	21	FGFR3-TACC3	In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.	2015-02-06 19:01:07.023371	2015-02-06 19:01:07.023371
61	22	ITD	FLT3-ITD (internal tandem duplications) frequently occur in patients with hematologic malignancies such as chronic myelogenous leukemia, acute myeloid leukemia (AML) and myelodysplastic syndrome, but particularly in cytogenetically normal AML (CN-AML). These duplication events disrupt the juxtamembrane domain of FLT3 and can be the result of a duplication of internal FLT3 sequence or other unrelated sequence resulting in an in-frame duplication event. The length of these duplications can vary widely which may have prognostic consequences, but this has not been conclusively determined. FLT3-ITD mutations overall have generally been associated with poor prognosis. Additional genes associated with CN-AML such as NPM1 may modulate the prognosis associated with this variant.	2015-02-06 19:01:07.0327	2015-02-06 19:01:07.0327
62	22	TKD	FLT3 tyrosine kinase domain mutations (aka FLT3-TKD) are much less common than FLT3-ITD (internal tandem duplication) mutations and may not confer the same prognostic impact. Although the majority of mutations are point mutations effecting D835 (most frequently D835Y), a small proportion involve an in-frame deletion of I836. These mutations are within the activation loop of the second tyrosine kinase domain of FLT3 and thought to result in constitutive activation of the receptor.	2015-02-06 19:01:07.042198	2015-02-06 19:01:07.042198
63	23	EXPRESSION	GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.	2015-02-06 19:01:07.052204	2015-02-06 19:01:07.052204
64	24	R132	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-06 19:01:07.061733	2015-02-06 19:01:07.061733
65	24	R132C	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-06 19:01:07.070784	2015-02-06 19:01:07.070784
66	24	R132G	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-06 19:01:07.080377	2015-02-06 19:01:07.080377
67	24	R132H	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-06 19:01:07.090988	2015-02-06 19:01:07.090988
68	24	R132L	IDH1 R132 mutations have been observed in a number of cancer types, and appear to behave quite differently in different environments. In myelodysplastic syndromes and acute myeloid leukemia (AML), IDH1 R132 mutations have been shown to be associated with worse outcome and shorter overall survival. They have also been linked to cytogenetically normal AML, an intermediate risk subtype of the disease. However, in glioblastoma and astrocytoma, patients with IDH1 mutations have shown better overall survival than patients with wild-type IDH1. Also in contrast to AML, in glioblastoma, IDH1 mutations have been associated with the specific cytogenetic abnormalities, 1p and 19q deletions.	2015-02-06 19:01:07.100497	2015-02-06 19:01:07.100497
69	25	R140Q/L	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R140Q/L mutants have shown improved overall survival as compared to their wild-type counterparts. In myelodysplastic syndromes, however, no prognostic link was discovered between IDH2 mutation status and overall survival.	2015-02-06 19:01:07.109598	2015-02-06 19:01:07.109598
70	25	R172K	IDH2 mutations have been observed in a number of hematologic malignancies. In acute myeloid leukemia, the R172K mutation has been linked with poorer prognosis and worse overall survival than IDH2 wild-type patients. However, in myelodisplastic syndromes, studies did not find a prognostic association between this variant and patient outcomes.	2015-02-06 19:01:07.121086	2015-02-06 19:01:07.121086
71	26	V617F	JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases, occuring in around half of all MPD's. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.	2015-02-06 19:01:07.13004	2015-02-06 19:01:07.13004
72	27	D816V	KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients.	2015-02-06 19:01:07.13962	2015-02-06 19:01:07.13962
73	27	EXON 11	c-KIT mutations in exon 11 lie within the juxtamembrane domain, and are very recurrent in gastrointestinal stromal tumors, often bearing a poorer prognosis than other KIT mutations. Cells harboring exon 11 mutations have shown sensitivity to the tyrosine kinase inhibitor imatinib, offering a better prognosis to patients treated with the drug in the first year.	2015-02-06 19:01:07.148823	2015-02-06 19:01:07.148823
74	27	EXON 13	c-KIT exon 13 mutations are relatively rare compared to other c-KIT mutations. These mutations lie within the tyrosine kinase 1 domain, and are found primarily in melanoma and gastrointestinal stromal tumors. Cell lines harboring these mutations show sensitivity to imatinib and sunitinib treatment. However, only imatinib has seen use in clinical settings to date, with sunitinib entering trials in imatinib-resistant cases.	2015-02-06 19:01:07.157383	2015-02-06 19:01:07.157383
75	27	EXON 14	c-KIT exon 14 mutations lie within the tyrosine kinase domain of the protein. While relatively rare in primary gastrointestinal tumors, they are notably more prevalent in refractory disease, suggesting a role in imatinib resistance. Unlike mutations in other KIT exons, exon 14 mutations seem relatively rare in melanoma.	2015-02-06 19:01:07.166274	2015-02-06 19:01:07.166274
76	27	EXON 17	c-KIT exon 17 mutations lie within the TK2 domain, containing the activation loop protein. In cell lines, mutations within this domain have been shown to be sensitive to imatinib. However, in double KIT mutants in which the exon 17 mutation is a secondary mutation, cell lines have shown resistance to both imatinib and sunitinib.	2015-02-06 19:01:07.175889	2015-02-06 19:01:07.175889
77	27	EXON 9	c-KIT exon 9 mutations lie within the dimerization motif of the protein. Relative to other KIT mutations, exon 9 mutations have been associated with better overall survival. In exon 9 mutants, imatinib has shown efficacy both in vitro and in vivo.	2015-02-06 19:01:07.184399	2015-02-06 19:01:07.184399
78	27	L576P	KIT L576P is an exon 11 mutation that lies within the juxtamembrane domain. It is one of the most recurrent KIT mutations in melanoma, and both in vitro and in vivo studies have shown sensitivity to imatinib.	2015-02-06 19:01:07.206157	2015-02-06 19:01:07.206157
79	27	V654A	KIT V654A is an exon 13 mutation that lies within the tyrosine kinase 1 domain of the protein. It has been shown to be an activating mutation by in vitro studies. This mutation is associated with imatinib resistance in melanoma patients. However, second generation TKI's such as sunitinib and midostaurin (PKC 412) have seen success in acheiving tumor response.	2015-02-06 19:01:07.217197	2015-02-06 19:01:07.217197
80	28	EXON 1	A study by Li√®vre et al in 2006 showed that colorectal cancer patients with KRAS exon 1 mutations had low cetuximab response rates.	2015-02-06 19:01:07.227215	2015-02-06 19:01:07.227215
81	28	EXON 2	In a study by Pan et al in 2005, non-small cell lung cancer patients harboring KRAS exon 2 mutations were associated with resistance to the EGFR inhibitors gefinitib and erlotinib.	2015-02-06 19:01:07.241957	2015-02-06 19:01:07.241957
82	28	G12	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-06 19:01:07.252058	2015-02-06 19:01:07.252058
83	28	G12/G13	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-06 19:01:07.261641	2015-02-06 19:01:07.261641
84	28	G12C	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-06 19:01:07.270618	2015-02-06 19:01:07.270618
85	28	G12D	While the KRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G13 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-06 19:01:07.280785	2015-02-06 19:01:07.280785
86	28	G13	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-06 19:01:07.290963	2015-02-06 19:01:07.290963
87	28	G13D	While the KRAS G13 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated. Often associated with tumors that are wild-type for other drivers (EGFR and ALK specifically), the prognosis for patients with this mutation seems to be worse than the KRAS wild-type cohort. This mutation, along with the mutations affecting the neighboring G12 position, may result in a less responsive tumor when treated with first-generation TKI's like gefitinib.	2015-02-06 19:01:07.299533	2015-02-06 19:01:07.299533
88	29	P124S	MAP2K1 P124S is a recurrent mutation in melanoma, and is seen in bladder and colon cancer to a lesser degree. The P124S mutation has been shown to contribute to AZD6244 resistance in melanoma cell lines, but considerably less so than its Q56P counterpart.	2015-02-06 19:01:07.308376	2015-02-06 19:01:07.308376
89	29	Q56P	MAP2K1 Q56P is a recurrent mutation in melanoma and gastric cancer. This mutation has been shown to confer considerable resistance to AZD6244 treatment of melanoma cell lines.	2015-02-06 19:01:07.317437	2015-02-06 19:01:07.317437
90	30	MEF2D-CSF1R	MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.	2015-02-06 19:01:07.326869	2015-02-06 19:01:07.326869
91	32	PROMOTER METHYLATION	MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.	2015-02-06 19:01:07.336953	2015-02-06 19:01:07.336953
92	33	EXON 12	NPM1 exon 12 mutations are frequently identified in patients with cytogenetically normal acute myeloid leukemia (AML) and often co-occur with FLT3-ITD. FLT3 status should also be evaluated as co-occurance with FLT3-ITD may impact prognosis. Exon 12 mutations have been identified as a predictor of good prognostic outcomes in the absence of FLT3-ITD. Due to their high frequency, NPM1 mutations have been retrospectively analyzed in the context of a number of therapies including variable results following ATRA treatment as well as improved response to high-dose daunorubicin or valproic acid. Additionally, multiple groups have shown increased surface expression of CD33 associated with NPM1 mutation, suggesting these patients may respond to anti-CD33 therapy. Cytoplasmic sequestration of NPM1 (NPM1c) is associated with a good response to induction therapy.	2015-02-06 19:01:07.347079	2015-02-06 19:01:07.347079
93	33	W288FS	NPM1 W288fs (aka NPM1-A) is located in exon 12 of NPM1 and is the most common NPM1 mutation identified in acute myeloid leukemia. This mutation results in cytoplasmic localization of NPM1 (NPM1c) which is associated with a good response to induction therapy. Although it is the most extensively studied NPM1 exon 12, it is generally grouped with other exon 12 mutations for patient analysis (see NPM1 Exon 12 variants for more information).	2015-02-06 19:01:07.355887	2015-02-06 19:01:07.355887
94	34	EXON 1	NRAS exon 1 mutations were studied by Jakob et al in 2012 and were shown to be correlated with poorer overall survival relative to wild-type NRAS in melanoma patients.	2015-02-06 19:01:07.364569	2015-02-06 19:01:07.364569
95	34	EXON 2	NRAS exon 2 mutations have been shown to be correlated with poorer overall survival in melanoma patients and colorectal cancer patients, however no prognostic impact was seen in acute myeloid leukemia patients.	2015-02-06 19:01:07.373839	2015-02-06 19:01:07.373839
96	34	EXON 3	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-06 19:01:07.38475	2015-02-06 19:01:07.38475
97	34	EXON 4	In a multi-gene prognostic survey, Therkildsen et al 2014 found NRAS exon 3 and 4 mutations were correlated with poorer overall survival.	2015-02-06 19:01:07.3937	2015-02-06 19:01:07.3937
98	34	G12	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-06 19:01:07.402485	2015-02-06 19:01:07.402485
99	34	G13D	While the NRAS G12 region is a widely studied recurrent region in cancer, its impact on clinical action is still debated.	2015-02-06 19:01:07.411063	2015-02-06 19:01:07.411063
100	34	Q61	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. Two melanoma patients, each harboring mutations at this locus (Q61L and Q61R), responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-06 19:01:07.420996	2015-02-06 19:01:07.420996
101	34	Q61L	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-06 19:01:07.430764	2015-02-06 19:01:07.430764
102	34	Q61R	NRAS Q61 mutations have been found in multiple myeloma, gastrointestinal stromal tumors, melanoma, and others. A melanoma patient harboring a mutation at this locus responded to treatment with the akylating agent temozolomide. However, in colorectal cancer patients, mutations at this locus have been shown to confer resistance to cetuximab. The prognostic impact of mutations at this locus is currently under study.	2015-02-06 19:01:07.43948	2015-02-06 19:01:07.43948
103	36	D842I	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-06 19:01:07.448279	2015-02-06 19:01:07.448279
104	36	D842V	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-06 19:01:07.457197	2015-02-06 19:01:07.457197
105	36	D842Y	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-06 19:01:07.467323	2015-02-06 19:01:07.467323
106	36	DEL I843	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-06 19:01:07.47769	2015-02-06 19:01:07.47769
107	36	DI842-843IM	PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412).	2015-02-06 19:01:07.486805	2015-02-06 19:01:07.486805
108	35	E542K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-06 19:01:07.49516	2015-02-06 19:01:07.49516
109	35	E545K	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-06 19:01:07.503978	2015-02-06 19:01:07.503978
110	35	EXON 20	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-06 19:01:07.513016	2015-02-06 19:01:07.513016
111	35	EXON 9	PIK3CA E545K/E542K are the second most recurrent PIK3CA mutations in breast cancer, and are highly recurrent mutations in many other cancer types. E545K, and possibly the other mutations in the E545 region, may present patients with a poorer prognosis than patients with either patients with other PIK3CA variant or wild-type PIK3CA. There is also data to suggest that E545/542 mutations may confer resistance to EGFR inhibitors like cetuximab. While very prevalent, targeted therapies for variants in PIK3CA are still in early clinical trial phases.	2015-02-06 19:01:07.522169	2015-02-06 19:01:07.522169
112	35	H1047R	PIK3CA H1047R is one of the most recurrent mutations in cancer, especially breast cancer. Of PIK3CA mutant breast cancers, over half harbor this mutation. Meta-analyses have shown that patients harboring this mutation may have worse overall survival, but other studies have shown no difference between H1047R and other PIK3CA mutants from a prognostic standpoint. While very prevalent, targeted therapies for this particular mutation are still in early clinical trial phases.	2015-02-06 19:01:07.53089	2015-02-06 19:01:07.53089
113	37	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.	2015-02-06 19:01:07.539855	2015-02-06 19:01:07.539855
114	15	DNAJB1-PRKACA	This fusion has been found to be very recurrent in a rare form of adolescent liver cancer, hepatocellular fibrolamellar carcinoma. In a 2014 study, authors found Honeyman et al observed this fusion in all 15 of the FL-HCC cases they examined, and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.	2015-02-06 19:01:07.548531	2015-02-06 19:01:07.548531
115	39	R233*	PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.	2015-02-06 19:01:07.556987	2015-02-06 19:01:07.556987
116	38	PML-RARA	The PML-RARa fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and Arsenic Trioxide for treating these patients, and early results seem promising.	2015-02-06 19:01:07.566141	2015-02-06 19:01:07.566141
117	40	C634W	RET C639W has been implicated as an alternate mechanism of activating RET in medullary thyroid cancer. While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the C639W mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib.	2015-02-06 19:01:07.574994	2015-02-06 19:01:07.574994
118	40	M918T	RET M819T is the most common somatically acquired mutation in medullary thyroid cancer (MTC). While there currently are no RET-specific inhibiting agents, promiscuous kinase inhibitors have seen some success in treating RET overactivity. Data suggests however, that the M918T mutation may lead to drug resistance, especially against the VEGFR-inhibitor motesanib. It has also been suggested that RET M819T leads to more aggressive MTC with a poorer prognosis.	2015-02-06 19:01:07.585355	2015-02-06 19:01:07.585355
119	42	K666N	SF3B1 K666N is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients.	2015-02-06 19:01:07.594522	2015-02-06 19:01:07.594522
120	42	K700E	SF3B1 K700E is a variant found in myelodysplastic syndromes, chronic leukemias, and more recently, breast cancer. This somatic mutation has been linked to better overall outcome and event-free survival in MDS patients. Additionally, these mutations are the most common SF3B1 mutation observed in MDS and highly associated with subtypes of MDS that are defined by ringed sideroblasts.	2015-02-06 19:01:07.603049	2015-02-06 19:01:07.603049
121	43	R175H	While loss-of-function events in TP53 are very common in cancer, the R175H variant seems not only to result in loss of tumor-suppression, but also acts as a gain-of-function mutation that promotes tumorigenesis in mouse models. Cell lines harboring this mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been shown that the R175H mutation is correlated with worse overall survival than wild-type TP53, but is not as detrimental as the R248W variant.	2015-02-06 19:01:07.611925	2015-02-06 19:01:07.611925
122	43	R248Q	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival. The R248Q mutation has also shown an increased invasive behavior in cell lines. This is specific to the 248Q variant.	2015-02-06 19:01:07.620557	2015-02-06 19:01:07.620557
123	43	R248W	While loss-of-function events in TP53 are very common in cancer, the R248 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, R248 mutations have been shown to confer worse overall survival.	2015-02-06 19:01:07.630966	2015-02-06 19:01:07.630966
124	43	R249T	This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-06 19:01:07.640933	2015-02-06 19:01:07.640933
126	43	R273C	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-06 19:01:07.66117	2015-02-06 19:01:07.66117
127	43	R273H	While loss-of-function events in TP53 are very common in cancer, the R273 variants seem not only to result in loss of tumor-suppression, but also act as a gain-of-function mutation that can promote tumorigenesis in mouse models. This mutant is also more responsive to treatment with doxorubicin than its wild-type counterparts. While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R273 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-06 19:01:07.6724	2015-02-06 19:01:07.6724
128	43	V173G/A	While the prognostic impact of individual TP53 mutations is influenced by the cohort being studied, it has been suggested that the R249 mutants have been correlated with worse overall survival in breast cancer patients when compared to wild-type.	2015-02-06 19:01:07.682746	2015-02-06 19:01:07.682746
129	44	FRAMESHIFT TRUNCATION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-06 19:01:07.695013	2015-02-06 19:01:07.695013
130	44	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-06 19:01:07.70566	2015-02-06 19:01:07.70566
131	45	LOSS-OF-FUNCTION	In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.	2015-02-06 19:01:07.716477	2015-02-06 19:01:07.716477
132	46	Q157P/R	U2AF1 Q157P/R has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is less common than the S34F mutation, occurs in the second zinc finger domain of U2AF1 and has been demonstrated to alter splicing. Although this mutation has not been determined to provide prognostic information in AML, patients with MDS were shown to be at an increased risk of transformation to secondary AML. However, the prognostic importance of U2AF1 mutations in MDS has been debated.	2015-02-06 19:01:07.725439	2015-02-06 19:01:07.725439
133	46	S34Y/F	U2AF1 S34Y/F has been shown to be a recurrent mutation in acute myeloid leukemia (AML), myelodysplastic syndromes (MDS) and lung adenocarcinomas. This mutation is is the most commonly identified variant in MDS, occurs in the first zinc finger domain of U2AF1 and has been demonstrated to alter splicing. Although this mutation has not been determined to provide prognostic information in AML, patients with MDS were shown to be at an increased risk of transformation to secondary AML. However, the prognostic importance of U2AF1 mutations in MDS has been debated.	2015-02-06 19:01:07.734495	2015-02-06 19:01:07.734495
134	47	EXON 7	WT1 exon 7 mutations have been shown to be recurrent in acute myeloid leukemia. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 7 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-06 19:01:07.745588	2015-02-06 19:01:07.745588
135	47	EXON 9	WT1 exon 9 mutations have been shown to be recurrent in acute myeloid leukemia, although at a less frequent rate than their exon 7 counterparts. Many sources have examined the prognostic impact of these, agreeing that the mutant exon 9 cohort is correlated with worse overall survival and a number of poor prognistic outcomes. This may be the result of an overall poor response to chemotherapy from WT1 mutant tumors.	2015-02-06 19:01:07.754249	2015-02-06 19:01:07.754249
\.


--
-- Data for Name: evidence_items; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY evidence_items (id, text, clinical_significance, evidence_direction, evidence_type_id, evidence_level_id, drug_id, disease_id, source_id, variant_id, created_at, updated_at, rating) FROM stdin;
1	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	1	1	1	1	40	131	2015-02-06 19:01:07.874267	2015-02-06 19:01:07.874267	3
2	Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.	Sensitivity	Supports	1	1	1	1	40	130	2015-02-06 19:01:07.926327	2015-02-06 19:01:07.926327	3
3	A patient with metastatic bladder cancer that responded well to the mTOR inhibitor everolimus was shown to harbor a TSC1 frameshift truncation mutation (c.1907_1908del, p.Glu636fs) and a nonsense NF2 mutation. 2/3 additional patients with TSC1 mutations also had mild responses to everolimus.	Sensitivity	Supports	1	1	1	2	41	129	2015-02-06 19:01:07.952746	2015-02-06 19:01:07.952746	3
4	Following rapamycin treatment of mice with KrasG12D-induced lung tumors, mice with Tsc1 deletion (exon17/18 deletion resulting in frameshift) in the lung exhibit increased overall survival, increased apoptosis, reduced proliferation and reduced tumor burden compared to Tsc1 wildtype mice.	Sensitivity	Supports	1	2	2	3	42	129	2015-02-06 19:01:07.980456	2015-02-06 19:01:07.980456	3
5	Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.	Sensitivity	Supports	1	2	3	4	43	115	2015-02-06 19:01:08.00953	2015-02-06 19:01:08.00953	4
6	PTEN nonsense  , including R233*, has been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.	Poor Outcome	Supports	2	1	3	5	44	115	2015-02-06 19:01:08.040226	2015-02-06 19:01:08.040226	3
7	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	4	6	45	10	2015-02-06 19:01:08.065506	2015-02-06 19:01:08.065506	5
8	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	4	6	45	14	2015-02-06 19:01:08.087405	2015-02-06 19:01:08.087405	5
9	Patients with other BRAF V600 mutations also respond well to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	4	6	45	15	2015-02-06 19:01:08.10956	2015-02-06 19:01:08.10956	5
10	A single patient with BRAF V600E/V600M bi-allelic mutation responded to the V600E drug dabrafenib.	Sensitivity	Supports	1	1	4	6	46	12	2015-02-06 19:01:08.132391	2015-02-06 19:01:08.132391	3
11	Complete remission rates were higher and event-free survival was longer for normal karyotype AML patients with Exon 12 NPM1  .	Better Outcome	Supports	2	1	3	7	47	92	2015-02-06 19:01:08.157725	2015-02-06 19:01:08.157725	5
12	NPM1 mutations were not associated with M2 FAB subtypes	Negative	Supports	3	1	3	7	47	92	2015-02-06 19:01:08.184369	2015-02-06 19:01:08.184369	4
13	NPM1 mutations were associated with M4, M5a and M5b FAB subtypes	Positive	Supports	3	1	3	7	47	92	2015-02-06 19:01:08.219284	2015-02-06 19:01:08.219284	4
14	NPM1 mutation without FLT3-ITD was associated with reduced relapse risk and increased overall survival in young adult AML patients (median age 43)	Sensitivity	Supports	1	1	3	7	48	92	2015-02-06 19:01:08.241498	2015-02-06 19:01:08.241498	4
15	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	2	1	3	7	49	92	2015-02-06 19:01:08.266752	2015-02-06 19:01:08.266752	4
16	Younger (16-60), normal karyotype AML patients with Exon 12 NPM1 mutation and without FLT3-ITD have increased overall survival, relapse-free survival and response to induction chemotherapy	Better Outcome	Supports	2	1	3	7	50	92	2015-02-06 19:01:08.290381	2015-02-06 19:01:08.290381	4
17	NPM1 mutations were associated with increased complete remission rates as well as longer overall disease free survival in normal karyotype AML patients >59 years old, particularly in those >69 years old	Better Outcome	Supports	2	1	3	7	51	92	2015-02-06 19:01:08.317579	2015-02-06 19:01:08.317579	4
18	NPM1 mutation was associated with higher complete remission rates, lower cumulative incidence of relapse and higher overall survival in intermediate risk AML patients	Better Outcome	Supports	2	1	3	7	52	92	2015-02-06 19:01:08.342028	2015-02-06 19:01:08.342028	4
19	Coocurrence of FLT3-ITD mutations significantly reduced overall survival of normal karyotype AML patients	Poor Outcome	Supports	2	1	3	7	53	92	2015-02-06 19:01:08.368428	2015-02-06 19:01:08.368428	4
20	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	2	1	3	7	54	92	2015-02-06 19:01:08.392035	2015-02-06 19:01:08.392035	4
21	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	2	1	3	7	54	92	2015-02-06 19:01:08.412006	2015-02-06 19:01:08.412006	4
22	No NPM1 mutations were identified in patients with favorable risk cytogenetics (79/215 patients)	Negative	Does Not Support	3	1	3	7	55	93	2015-02-06 19:01:08.436436	2015-02-06 19:01:08.436436	4
23	NPM1 mutations were associated with intermediate risk cytogenetics (including normal karyotype)	Positive	Supports	3	1	3	7	55	93	2015-02-06 19:01:08.458113	2015-02-06 19:01:08.458113	4
24	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	1	2	5	7	56	92	2015-02-06 19:01:08.482307	2015-02-06 19:01:08.482307	3
25	NSC348884 combined with ATRA treatment induces apoptosis in primary AML cells harboring mutant NPM1 but not cells with wildtype NPM1 or both mutant NPM1 and FLT3-ITD	Sensitivity	Supports	1	2	6	7	56	92	2015-02-06 19:01:08.49283	2015-02-06 19:01:08.49283	3
26	ATRA treatment did not effect overall survival in patients <60 years old with NPM1 mutation regardless of FLT3-ITD status	Resistance or Non-Response	Supports	1	1	6	7	57	92	2015-02-06 19:01:08.514785	2015-02-06 19:01:08.514785	3
27	Frequency of NPM1 mutations was not different in normal karyotype AML patients with CEPBA, NRAS or KIT mutations.	Positive	Does Not Support	3	1	3	7	47	92	2015-02-06 19:01:08.536096	2015-02-06 19:01:08.536096	3
28	NPM1 mutations were not associated with M3 FAB subtype (0/55).	Negative	Does Not Support	3	1	3	7	53	92	2015-02-06 19:01:08.558435	2015-02-06 19:01:08.558435	3
29	NPM1 mutations were associated with M5a and M5b FAB subtypes	Positive	Supports	3	1	3	7	53	92	2015-02-06 19:01:08.581644	2015-02-06 19:01:08.581644	3
30	NPM1 mutations were associated with normal karyotype in older (>60) patients	Positive	Supports	3	1	3	7	58	92	2015-02-06 19:01:08.604376	2015-02-06 19:01:08.604376	3
31	Patients with NPM1 mutations were associated with improved complete remission rates as well as overall and disease free survival when all ages and karyotype were analyzed; however, when only normal karyotype patients were analyzed, only complete remission rates were improved	Sensitivity	Supports	1	1	3	7	58	92	2015-02-06 19:01:08.624585	2015-02-06 19:01:08.624585	3
32	Mutation status of IDH1 did not change event-free survival of patients with an NPM1 mutation	N/A	Does Not Support	2	1	3	7	59	92	2015-02-06 19:01:08.646043	2015-02-06 19:01:08.646043	3
33	Complete remission rates were higher and both disease-free and overall survival was longer for patients with Exon 12 NPM1  .	Better Outcome	Supports	2	1	3	7	53	92	2015-02-06 19:01:08.666972	2015-02-06 19:01:08.666972	3
34	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	2	1	3	7	60	92	2015-02-06 19:01:08.689247	2015-02-06 19:01:08.689247	3
35	Normal karyotype patients with NPM1 mutation and without FLT3-ITD had improved overall and relapse-free survival, complete remission rates and remission duration compared to all other combinations of mutants	Better Outcome	Supports	2	1	3	7	61	92	2015-02-06 19:01:08.713828	2015-02-06 19:01:08.713828	3
36	NPM1 mutation was associated with increased overall, event-free and relapse-free survival and remission after induction therapy in the entire cohort and in the cytogenetically normal subset of patients	Better Outcome	Supports	2	1	3	7	62	92	2015-02-06 19:01:08.735841	2015-02-06 19:01:08.735841	3
37	TET2 mutation reduces overall survival of normal karyotype patients with either an NPM1 mutation or NPM1 mutation without FLT3-ITD	Poor Outcome	Supports	2	1	3	7	63	92	2015-02-06 19:01:08.7576	2015-02-06 19:01:08.7576	3
38	Patients with NPM1 mutations and low levels of FLT3-ITD have worse overall survival and cumulative incidence of relapse than those with NPM1 mutations alone in intermediate risk AML patients	Poor Outcome	Supports	2	1	3	7	52	92	2015-02-06 19:01:08.781313	2015-02-06 19:01:08.781313	3
39	Mutation (Type A, W288fs) causes cytoplasmic localization of NPM when transfected into a non-hematopoietic cell line (293T cells). Cytoplasmic localization of NPM in AML patients was associated with good response to induction therapy.	Sensitivity	Supports	1	3	3	7	64	93	2015-02-06 19:01:08.809148	2015-02-06 19:01:08.809148	3
40	ATRA treatment improved overall and relapse-free survival in older (>60) patients with mutant NPM1 and without FLT3-ITD	Sensitivity	Supports	1	1	6	7	58	92	2015-02-06 19:01:08.832331	2015-02-06 19:01:08.832331	2
41	Young patients (18-60) with NPM1 mutations had improved overall survival following high-dose versus standard dose daunorubicin (p=0.01) in univariate analysis. However, the adjusted p-value for multiple testing is 0.11.	Sensitivity	Supports	1	1	7	7	65	92	2015-02-06 19:01:08.85999	2015-02-06 19:01:08.85999	2
42	For patients with NPM1 mutation that achieved CR following induction therapy, relapse-free survival was improved following treatment with valproic acid	Sensitivity	Supports	1	1	8	7	66	92	2015-02-06 19:01:08.884215	2015-02-06 19:01:08.884215	2
43	NPM1 mutations were associated with de novo AML as well as M4 and M5 FAB subtypes	Positive	Supports	3	1	3	7	48	92	2015-02-06 19:01:08.904655	2015-02-06 19:01:08.904655	2
44	Monitoring NPM1 mutation by cDNA-based RT-PCR effectively predicted relapse in 62/93 patients by >1 log increase in NPM1 levels (N=57) or lack of reduction in NPM1 levels following first line therapy (N=15)	Resistance or Non-Response	Supports	1	1	3	7	67	92	2015-02-06 19:01:08.929374	2015-02-06 19:01:08.929374	2
45	Intermediate risk patients with mutant NPM1 had improved overall survival with the presence either IDH1 or IDH2 mutations than those wildtype for both IDH1 and IDH2	Sensitivity	Supports	1	1	3	7	65	92	2015-02-06 19:01:08.952428	2015-02-06 19:01:08.952428	2
46	CD33 expression was significantly increased in patients with NPM1 mutation with or without FLT3-ITD, indicating these patients may respond to anti-CD33 therapy	Sensitivity	Supports	1	3	3	7	68	92	2015-02-06 19:01:08.974508	2015-02-06 19:01:08.974508	2
47	CD33 and CD123 expression was significantly increased in patients with NPM1 mutation with FLT3-ITD, indicating these patients may respond to combined anti-CD33 and anti-CD123 therapy	Sensitivity	Supports	1	3	3	7	68	92	2015-02-06 19:01:08.995565	2015-02-06 19:01:08.995565	2
48	NPM1 mutation without FLT3-ITD was significantly associated with complete remission	Better Outcome	Supports	2	1	3	7	69	92	2015-02-06 19:01:09.023659	2015-02-06 19:01:09.023659	3
49	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP but does not show resistance to MEK inhibitors.	N/A	Does Not Support	1	2	3	6	70	11	2015-02-06 19:01:09.046734	2015-02-06 19:01:09.046734	3
50	In the setting of BRAF(V600E), NF1 loss resulted in elevated activation of RAS-GTP and resistance to RAF inhibitors.	Resistance or Non-Response	Supports	1	2	3	6	70	11	2015-02-06 19:01:09.071033	2015-02-06 19:01:09.071033	3
51	Medullary thyroid cancer cells with RET C634W mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	1	2	9	8	71	117	2015-02-06 19:01:09.098163	2015-02-06 19:01:09.098163	3
52	Medullary thyroid cancer cells with RET M918T mutation are insensitive to motesanib, compared to wild-type RET.	Resistance or Non-Response	Supports	1	2	9	8	71	118	2015-02-06 19:01:09.120574	2015-02-06 19:01:09.120574	3
53	In patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.	Resistance or Non-Response	Supports	1	1	10	9	72	2	2015-02-06 19:01:09.148117	2015-02-06 19:01:09.148117	4
54	In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in "anti-tumor activitiy".	Sensitivity	Supports	1	1	11	10	73	57	2015-02-06 19:01:09.189933	2015-02-06 19:01:09.189933	3
55	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	12	11	74	52	2015-02-06 19:01:09.215959	2015-02-06 19:01:09.215959	3
56	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	12	11	74	54	2015-02-06 19:01:09.237883	2015-02-06 19:01:09.237883	3
57	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	12	11	74	55	2015-02-06 19:01:09.262968	2015-02-06 19:01:09.262968	3
58	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	12	10	74	53	2015-02-06 19:01:09.285921	2015-02-06 19:01:09.285921	3
59	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	12	12	74	59	2015-02-06 19:01:09.308075	2015-02-06 19:01:09.308075	3
60	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	12	13	74	56	2015-02-06 19:01:09.332339	2015-02-06 19:01:09.332339	3
61	In cell lines harboring this fusion, pazopanib exhibited anti-neoplastic properties.	Sensitivity	Supports	1	2	12	14	74	58	2015-02-06 19:01:09.353901	2015-02-06 19:01:09.353901	3
62	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	12	15	73	60	2015-02-06 19:01:09.375855	2015-02-06 19:01:09.375855	3
63	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	11	15	73	60	2015-02-06 19:01:09.385431	2015-02-06 19:01:09.385431	3
64	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	12	10	73	60	2015-02-06 19:01:09.406381	2015-02-06 19:01:09.406381	3
65	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	11	10	73	60	2015-02-06 19:01:09.416676	2015-02-06 19:01:09.416676	3
66	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	12	16	73	60	2015-02-06 19:01:09.443232	2015-02-06 19:01:09.443232	3
67	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	11	16	73	60	2015-02-06 19:01:09.453143	2015-02-06 19:01:09.453143	3
68	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	12	17	73	60	2015-02-06 19:01:09.476106	2015-02-06 19:01:09.476106	3
69	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	11	17	73	60	2015-02-06 19:01:09.487019	2015-02-06 19:01:09.487019	3
70	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	12	18	73	60	2015-02-06 19:01:09.508964	2015-02-06 19:01:09.508964	3
71	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	11	18	73	60	2015-02-06 19:01:09.518084	2015-02-06 19:01:09.518084	3
72	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	12	19	73	60	2015-02-06 19:01:09.543262	2015-02-06 19:01:09.543262	3
73	In a 50 year old male patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor "tumor shrinkage". This fusion is also shown to be reccurent in multiple other cancer types.	Sensitivity	Supports	1	1	11	19	73	60	2015-02-06 19:01:09.552734	2015-02-06 19:01:09.552734	3
74	In acute myloid leukemia patients, D816 mutation is associated with earlier relapse and poorer prognosis than wildtype KIT	Poor Outcome	Supports	2	1	3	20	75	72	2015-02-06 19:01:09.578889	2015-02-06 19:01:09.578889	4
75	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	2	10	6	76	78	2015-02-06 19:01:09.606227	2015-02-06 19:01:09.606227	3
76	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	2	13	6	76	78	2015-02-06 19:01:09.617303	2015-02-06 19:01:09.617303	3
77	The melanoma cell line WM3211, which harbors the L576P mutation, is not sensitive to imatinib, nilotinib, and sorafenib. Molecular modeling indicated the L576P mutation alters the conformation of KIT and reduces the binding affinity of imatinib.	Resistance or Non-Response	Supports	1	2	14	6	76	78	2015-02-06 19:01:09.627772	2015-02-06 19:01:09.627772	3
78	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	2	15	3	77	78	2015-02-06 19:01:09.656686	2015-02-06 19:01:09.656686	3
79	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	2	10	3	77	78	2015-02-06 19:01:09.667887	2015-02-06 19:01:09.667887	3
80	BaF3 cells harboring KIT L576P mutation are sensitive to dasantinib, nilotinib and imatinib.	Sensitivity	Supports	1	2	13	3	77	78	2015-02-06 19:01:09.678388	2015-02-06 19:01:09.678388	3
81	In GIST tumor cells and patients harboring KIT V654A mutation, SU11248 is effective for those that are refractory to imatinib.	Sensitivity	Supports	1	1	16	21	78	79	2015-02-06 19:01:09.703815	2015-02-06 19:01:09.703815	3
82	The melanoma cell line WM3211, which harbors the L576P mutation, is sensitive to dasatinib. Molecular modeling indicated the L576P mutation alters the conformation of KIT but did not alter the binding affinity of dasatinib.	Sensitivity	Supports	1	2	15	6	76	78	2015-02-06 19:01:09.724361	2015-02-06 19:01:09.724361	3
83	KIT V654A results in imatinib resistance in GIST patient-derived cell lines.	Resistance or Non-Response	Supports	1	2	10	21	79	79	2015-02-06 19:01:09.748238	2015-02-06 19:01:09.748238	3
84	In a case study, a patient with anal melanoma harboring an exon 11 KIT mutation showed marked response 4 months after imatinib treatment.	Sensitivity	Supports	1	1	10	6	80	78	2015-02-06 19:01:09.771793	2015-02-06 19:01:09.771793	3
85	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	17	11	81	47	2015-02-06 19:01:09.796379	2015-02-06 19:01:09.796379	5
86	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	18	11	81	47	2015-02-06 19:01:09.808585	2015-02-06 19:01:09.808585	5
87	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	17	11	81	48	2015-02-06 19:01:09.831261	2015-02-06 19:01:09.831261	5
88	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	2	18	11	81	48	2015-02-06 19:01:09.841991	2015-02-06 19:01:09.841991	5
89	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	4	17	11	81	49	2015-02-06 19:01:09.866418	2015-02-06 19:01:09.866418	5
90	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	4	18	11	81	49	2015-02-06 19:01:09.875928	2015-02-06 19:01:09.875928	5
91	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	4	17	11	81	50	2015-02-06 19:01:09.898006	2015-02-06 19:01:09.898006	5
92	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	4	18	11	81	50	2015-02-06 19:01:09.908572	2015-02-06 19:01:09.908572	5
93	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	4	17	11	81	51	2015-02-06 19:01:09.930486	2015-02-06 19:01:09.930486	5
94	5 ligand binding domain mutations have been shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.	Sensitivity	Supports	1	4	18	11	81	51	2015-02-06 19:01:09.940583	2015-02-06 19:01:09.940583	5
95	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.	Sensitivity	Supports	1	2	3	11	82	47	2015-02-06 19:01:09.96345	2015-02-06 19:01:09.96345	3
96	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.	Sensitivity	Supports	1	1	3	11	82	48	2015-02-06 19:01:09.985272	2015-02-06 19:01:09.985272	3
97	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.	Sensitivity	Supports	1	1	3	11	82	49	2015-02-06 19:01:10.008463	2015-02-06 19:01:10.008463	3
98	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.	Sensitivity	Supports	1	1	3	11	82	50	2015-02-06 19:01:10.030698	2015-02-06 19:01:10.030698	3
99	Surveys implicate ligand binding domain mutants in mediating clinical resistance to hormonal therapy and suggest that more potent ER antagonists may be of substantial therapeutic benefit.	Sensitivity	Supports	1	1	3	11	82	51	2015-02-06 19:01:10.063588	2015-02-06 19:01:10.063588	3
100	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from an ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	3	1	3	22	83	114	2015-02-06 19:01:10.088298	2015-02-06 19:01:10.088298	4
101	A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from an ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.	Positive	Supports	3	1	3	22	83	32	2015-02-06 19:01:10.111033	2015-02-06 19:01:10.111033	4
102	Ba/F3 cells expressing the EML4-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing EML4-ALK without this mutation.	Resistance or Non-Response	Supports	1	2	19	3	84	7	2015-02-06 19:01:10.134616	2015-02-06 19:01:10.134616	3
103	In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	1	4	19	3	85	6	2015-02-06 19:01:10.159903	2015-02-06 19:01:10.159903	4
104	In patients with non-small cell lung cancer harboring EML4-ALK fusion, this variant has been shown to confer resistance to crizotinib.	Resistance or Non-Response	Supports	1	4	19	3	85	5	2015-02-06 19:01:10.186321	2015-02-06 19:01:10.186321	4
105	In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective.	Sensitivity	Supports	1	4	19	3	85	4	2015-02-06 19:01:10.215972	2015-02-06 19:01:10.215972	4
106	Fusion protein undergoes proteolysis in APL cells upon ATRA treatment in cell lines.	Sensitivity	Supports	1	1	6	23	86	113	2015-02-06 19:01:10.239656	2015-02-06 19:01:10.239656	5
107	Fusion protein degradation is accelerated upon ATRA treatment at pharmalogical concentrations.	Sensitivity	Supports	1	1	6	23	86	116	2015-02-06 19:01:10.256214	2015-02-06 19:01:10.256214	5
108	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	2	10	24	87	31	2015-02-06 19:01:10.276891	2015-02-06 19:01:10.276891	3
109	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	2	20	24	87	31	2015-02-06 19:01:10.28794	2015-02-06 19:01:10.28794	3
110	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	2	10	24	87	90	2015-02-06 19:01:10.309031	2015-02-06 19:01:10.309031	3
111	Cell lines with MEF2D-CSF1R fusion show sensitivity to Tyrosine Kinase Inhibitors Imatinib and GW-2580	Sensitivity	Supports	1	2	20	24	87	90	2015-02-06 19:01:10.319507	2015-02-06 19:01:10.319507	3
112	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	2	3	3	25	88	119	2015-02-06 19:01:10.339308	2015-02-06 19:01:10.339308	3
113	Patients with SF3B1 mutations had a statistically significant longer overall survival as well as event free-survival. Both before and after adjustment for age, karyotype and sex.	Better Outcome	Supports	2	3	3	25	88	120	2015-02-06 19:01:10.356847	2015-02-06 19:01:10.356847	3
114	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML	N/A	Does Not Support	2	1	3	7	89	134	2015-02-06 19:01:10.37409	2015-02-06 19:01:10.37409	4
115	Rates of complete remission and refractory disease are not different in patients with WT1 mutations (69% exon 7, 15% exon 9) than those without in young (16-60) patients with cytogenetically normal AML.	N/A	Does Not Support	2	1	3	7	89	135	2015-02-06 19:01:10.390221	2015-02-06 19:01:10.390221	4
116	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	1	1	3	7	90	134	2015-02-06 19:01:10.408223	2015-02-06 19:01:10.408223	3
117	No difference in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N/A	Does Not Support	2	1	3	7	89	134	2015-02-06 19:01:10.424097	2015-02-06 19:01:10.424097	3
118	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	2	1	3	7	90	134	2015-02-06 19:01:10.439867	2015-02-06 19:01:10.439867	3
119	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	2	1	3	7	90	134	2015-02-06 19:01:10.458015	2015-02-06 19:01:10.458015	3
120	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	2	1	3	7	91	134	2015-02-06 19:01:10.475664	2015-02-06 19:01:10.475664	3
121	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	2	1	3	7	89	134	2015-02-06 19:01:10.493947	2015-02-06 19:01:10.493947	3
122	WT1 mutations were associated with an inferior response to induction chemotherapy with a higher rate of resistant disease in young (15-60+, median 45) patients with cytogenetically normal AML	Resistance or Non-Response	Supports	1	1	3	7	90	135	2015-02-06 19:01:10.509964	2015-02-06 19:01:10.509964	3
123	No difference in relapse-free or overall survival were identified between young (16-60) patients with or without WT1 mutations (69% exon 7, 15% exon 9) in cytogenetically normal AML	N/A	Does Not Support	2	1	3	7	89	135	2015-02-06 19:01:10.525637	2015-02-06 19:01:10.525637	3
124	WT1 mutations were associated with a higher cumulative incidence of relapse and shorter relapse free survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	2	1	3	7	90	135	2015-02-06 19:01:10.541861	2015-02-06 19:01:10.541861	3
125	WT1 mutations were a negative prognostic factor for overall survival in young (15-60+, median 45) patients with cytogenetically normal AML	Poor Outcome	Supports	2	1	3	7	90	135	2015-02-06 19:01:10.559085	2015-02-06 19:01:10.559085	3
126	WT mutations were associated with shorter overall and disease free survival in a cohort of cytogenetically normal, young (<60) AML patients	Poor Outcome	Supports	2	1	3	7	91	135	2015-02-06 19:01:10.575055	2015-02-06 19:01:10.575055	3
127	Patients with WT1 (69% exon 7, 15% exon 9) and FLT3-ITD mutations had significantly lower complete remission and higher refractory disease rates than those with wildtype WT1 and without FLT3-ITD following induction therapy in young (16-60) patients with cytogenetically normal AML	Poor Outcome	Supports	2	1	3	7	89	135	2015-02-06 19:01:10.591833	2015-02-06 19:01:10.591833	3
128	in WT1 were associated with a worse overall prognosis than patients wildtype for WT1 in young patients (15-50), primarily because of increased risk of disease recurrence	Poor Outcome	Supports	2	1	3	7	92	134	2015-02-06 19:01:10.612241	2015-02-06 19:01:10.612241	2
129	in WT1 were associated with increased risk of recurrence in young patients (15-50) with cytogenetically normal AML	Poor Outcome	Supports	2	1	3	7	92	134	2015-02-06 19:01:10.628405	2015-02-06 19:01:10.628405	2
130	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF Q157P/R mutations and those who harbor wild type U2AF.	Negative	Does Not Support	3	3	3	7	93	132	2015-02-06 19:01:10.645566	2015-02-06 19:01:10.645566	3
131	Age, sex, FAB subtype and karyotypes were not statistically significant between AML patients with U2AF S34Y/F mutations and those who harbor wild type U2AF.	Negative	Does Not Support	3	3	3	7	93	133	2015-02-06 19:01:10.661952	2015-02-06 19:01:10.661952	3
132	After adjust for age and cancer stage, presence of U2AF mutation such as Q157P/R is prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	2	3	3	25	94	132	2015-02-06 19:01:10.679353	2015-02-06 19:01:10.679353	3
133	After adjust for age and cancer stage, presence of U2AF mutation such as S34Y/F is prognostic for poorer survival outcomes in patients with MDS.	Poor Outcome	Supports	2	3	3	25	94	133	2015-02-06 19:01:10.695239	2015-02-06 19:01:10.695239	3
134	In patients with AML, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N/A	Does Not Support	2	3	3	7	93	132	2015-02-06 19:01:10.713269	2015-02-06 19:01:10.713269	2
135	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N/A	Does Not Support	2	3	3	7	93	133	2015-02-06 19:01:10.729404	2015-02-06 19:01:10.729404	2
136	In patients with MDS, those who harbor Q157P/R mutation of U2AF do not show statistical significance in complete remission rate compared to those who harbor wild type U2AF.	N/A	Does Not Support	2	3	3	25	93	132	2015-02-06 19:01:10.749494	2015-02-06 19:01:10.749494	2
137	In patients with AML, complete remission rates are not different between patients who harbor the U2AF1 S34Y/F mutation and those with wild type U2AF1.	N/A	Does Not Support	2	3	3	25	93	133	2015-02-06 19:01:10.770431	2015-02-06 19:01:10.770431	2
138	Breast tumor with R175H mutation are more responsive to doxorubicin than breast tumor with wild type TP53.	Sensitivity	Supports	1	2	21	11	95	121	2015-02-06 19:01:10.790643	2015-02-06 19:01:10.790643	3
139	Breast cancer patients who harbor R175H mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	2	1	3	11	96	121	2015-02-06 19:01:10.809749	2015-02-06 19:01:10.809749	3
140	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R248 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	2	1	3	11	97	122	2015-02-06 19:01:10.827516	2015-02-06 19:01:10.827516	3
141	Breast cancer patients who harbor R248Q mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	2	1	3	11	96	122	2015-02-06 19:01:10.844991	2015-02-06 19:01:10.844991	3
142	In breast cancer patients harboring R248W mutation, the prognosis is worse than any other hotspot TP53 mutation, as well as worse than patients with wild type TP53.	Poor Outcome	Supports	2	1	3	11	96	123	2015-02-06 19:01:10.873019	2015-02-06 19:01:10.873019	3
143	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as R249 is prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	2	1	3	11	97	124	2015-02-06 19:01:10.889054	2015-02-06 19:01:10.889054	3
144	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as R249 is prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	2	1	3	11	97	125	2015-02-06 19:01:10.905463	2015-02-06 19:01:10.905463	3
145	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R273 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	2	1	3	11	97	126	2015-02-06 19:01:10.922316	2015-02-06 19:01:10.922316	3
146	Breast cancer patients who harbor R273C mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	2	1	3	11	96	126	2015-02-06 19:01:10.937933	2015-02-06 19:01:10.937933	3
147	In breast cancer patients  harboring TP53 mutation, mutations in DNA contact region such as R273 is prognostic for a worse relapse-free survival compared to other non-silent mutations in TP53.	Poor Outcome	Supports	2	1	3	11	97	127	2015-02-06 19:01:10.954692	2015-02-06 19:01:10.954692	3
148	Breast cancer patients who harbor R273H mutation has worse overall survival than those with wild type TP53, but have better prognosis than those with R248W mutation.	Poor Outcome	Supports	2	1	3	11	96	127	2015-02-06 19:01:10.971994	2015-02-06 19:01:10.971994	3
149	In breast cancer patients harboring TP53 mutation, mutations in conserved region such as V173 is prognostic for a worse overall survival compared to those harboring wild-type TP53.	Poor Outcome	Supports	2	1	3	11	97	128	2015-02-06 19:01:10.988282	2015-02-06 19:01:10.988282	3
150	In patients with medullary carcinoma, the presence of RET M918T mutation is associated with increased probability of lymph node metastases.	Positive	Supports	3	1	3	8	98	118	2015-02-06 19:01:11.007255	2015-02-06 19:01:11.007255	5
151	Compared to those who harbor a wild type RET, patients with RET M918T mutation develop larger and more aggressive medullary thyroid cancer.	Poor Outcome	Supports	2	1	3	8	99	118	2015-02-06 19:01:11.025746	2015-02-06 19:01:11.025746	4
152	The use of AZD1480 in vitro led to strong repression of tyroid cancer cell growth.	Sensitivity	Supports	1	2	22	8	100	118	2015-02-06 19:01:11.044251	2015-02-06 19:01:11.044251	3
153	MAP2K1 P124S confers increased resistance to inhibition by AZD6244 inhibition by about 5 fold.	Resistance or Non-Response	Supports	1	2	23	6	101	88	2015-02-06 19:01:11.065343	2015-02-06 19:01:11.065343	3
154	MAP2K1 Q56P confers increased resistance to inhibition by AZD6244 by 100 fold.	Resistance or Non-Response	Supports	1	2	23	6	101	89	2015-02-06 19:01:11.082198	2015-02-06 19:01:11.082198	3
155	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	2	24	11	102	108	2015-02-06 19:01:11.100644	2015-02-06 19:01:11.100644	4
156	Breast cancer cell lines with E542K/E545K mutations showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	2	24	11	102	109	2015-02-06 19:01:11.11608	2015-02-06 19:01:11.11608	4
157	Breast cancer cell lines with H1047R mutation showed increased sensitivity to CH5132799 than cells with wild-type PIK3CA gene.	Sensitivity	Supports	1	2	24	11	102	112	2015-02-06 19:01:11.131706	2015-02-06 19:01:11.131706	4
158	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	25	26	103	108	2015-02-06 19:01:11.151506	2015-02-06 19:01:11.151506	4
159	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	25	26	103	109	2015-02-06 19:01:11.170988	2015-02-06 19:01:11.170988	4
160	PIK3CA mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	N/A	Does Not Support	1	1	25	26	103	112	2015-02-06 19:01:11.188524	2015-02-06 19:01:11.188524	4
161	The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	26	26	104	108	2015-02-06 19:01:11.208063	2015-02-06 19:01:11.208063	3
162	The presence of mutation in exon 9 (such as E542K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	27	26	104	108	2015-02-06 19:01:11.218699	2015-02-06 19:01:11.218699	3
163	The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	26	26	104	109	2015-02-06 19:01:11.234963	2015-02-06 19:01:11.234963	3
164	The presence of mutation in exon 9 (such as E545K) or exon 20 may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	27	26	104	109	2015-02-06 19:01:11.244376	2015-02-06 19:01:11.244376	3
165	The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	26	26	104	112	2015-02-06 19:01:11.263127	2015-02-06 19:01:11.263127	3
166	The presence of mutation in exon 9 or exon 20 (such as H1047R) may increase resistance to drugs such as  panitumumab or cetuximab.	Resistance or Non-Response	Supports	1	3	27	26	104	112	2015-02-06 19:01:11.274302	2015-02-06 19:01:11.274302	3
167	Chemotherapy-refractory patients with colorectal cancer harboring Exon 20 PIK3CA mutations had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	Resistance or Non-Response	Supports	1	1	26	26	105	110	2015-02-06 19:01:11.294923	2015-02-06 19:01:11.294923	3
168	In chemotherapy-refractory patients with colorectal cancer, Exon 9 PIK3CA mutations (Exon 9) had no significant impact on response rate, disease control rate, progression free surivival or overall survival following cetuximab plus chemotherapy than those with wildtype PIK3CA	N/A	Does Not Support	1	1	26	26	105	111	2015-02-06 19:01:11.31143	2015-02-06 19:01:11.31143	3
169	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	2	2	11	106	112	2015-02-06 19:01:11.328348	2015-02-06 19:01:11.328348	3
170	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	2	2	11	106	108	2015-02-06 19:01:11.34417	2015-02-06 19:01:11.34417	3
171	Rapamycin inhibits transformation induced by mutation in PIK3CA	Sensitivity	Supports	1	2	2	11	106	109	2015-02-06 19:01:11.360133	2015-02-06 19:01:11.360133	3
172	The presence of exon 9 (such as E542K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	2	3	3	26	104	108	2015-02-06 19:01:11.375744	2015-02-06 19:01:11.375744	3
173	The presence of exon 9 (such as E545K) or exon 20 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	2	3	3	26	104	109	2015-02-06 19:01:11.397497	2015-02-06 19:01:11.397497	3
174	The presence of exon 20 (such as H1047R) or exon 9 mutation may result in poorer survival compared to those with wild-type PIK3CA mutation.	Poor Outcome	Supports	2	3	3	26	104	112	2015-02-06 19:01:11.415092	2015-02-06 19:01:11.415092	3
175	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	2	3	3	26	107	108	2015-02-06 19:01:11.432745	2015-02-06 19:01:11.432745	2
176	Coexistence of exon 9 (E542K or E545K) and exon 20 mutations but not either alone, is prognostic for worse overall survival and cancer specific survival.	Poor Outcome	Supports	2	3	3	26	107	109	2015-02-06 19:01:11.44835	2015-02-06 19:01:11.44835	2
177	In CHO cells with PDGFRA D842I mutation that have shown imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	28	27	108	103	2015-02-06 19:01:11.473233	2015-02-06 19:01:11.473233	4
178	In CHO cells with PDGFRA D842V mutation resulting in imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	28	27	108	104	2015-02-06 19:01:11.489353	2015-02-06 19:01:11.489353	4
179	In CHO cells with PDGFRA D842Y mutation resulting in imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	28	27	108	105	2015-02-06 19:01:11.506121	2015-02-06 19:01:11.506121	4
180	In CHO cells with PDGFRA deletion I843 mutation resulting in imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	28	27	108	106	2015-02-06 19:01:11.524577	2015-02-06 19:01:11.524577	4
181	In CHO cells with PDGFRA DI842-843IM mutation resulting in imatinib resistance, crenolanib showed relatively high potency.	Sensitivity	Supports	1	2	28	27	108	107	2015-02-06 19:01:11.540893	2015-02-06 19:01:11.540893	4
182	While cancer with PVGFRA V561D mutation is known to be sensitive to Imatinib, double mutation of V561D and D842V mutants are resistant to imatinib.	Resistance or Non-Response	Supports	1	1	10	27	79	104	2015-02-06 19:01:11.557304	2015-02-06 19:01:11.557304	4
183	GIST cancer with D842V mutation is resistant to imatinib.	Resistance or Non-Response	Supports	1	1	10	27	109	104	2015-02-06 19:01:11.576434	2015-02-06 19:01:11.576434	4
184	GIST tumor harboring PDGFRA D842V mutation are likely to be benign	Negative	Supports	3	1	3	27	110	104	2015-02-06 19:01:11.599509	2015-02-06 19:01:11.599509	3
185	Vemurafenib resistance is associated with gain of Q61 mutation in NRAS.	Resistance or Non-Response	Supports	1	1	29	6	111	100	2015-02-06 19:01:11.620818	2015-02-06 19:01:11.620818	4
186	In chemotherapy-refractory patients with colorectal cancer, NRAS mutation status (primarily Q61) was not informative for disease control rate, progression free survival or overall survival following cetuximab plus chemotherapy.	N/A	Does Not Support	1	1	26	26	105	100	2015-02-06 19:01:11.649861	2015-02-06 19:01:11.649861	3
187	Chemotherapy-refractory patients with colorectal cancer harboring NRAS mutation (primarily Q61) have a significantly lower response rate to cetuximab than patients wildtype for NRAS.	Resistance or Non-Response	Supports	1	1	26	26	105	100	2015-02-06 19:01:11.665805	2015-02-06 19:01:11.665805	3
188	The presence of NRAS mutation in AML patient does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death)	N/A	Does Not Support	2	1	3	7	112	95	2015-02-06 19:01:11.686693	2015-02-06 19:01:11.686693	3
189	NRAS mutation status does not impact overall, event-free or disease-free survival in patients with AML	N/A	Does Not Support	2	1	3	7	113	98	2015-02-06 19:01:11.703901	2015-02-06 19:01:11.703901	3
190	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.	Poor Outcome	Supports	2	1	3	26	114	95	2015-02-06 19:01:11.721819	2015-02-06 19:01:11.721819	3
191	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.	Poor Outcome	Supports	2	1	3	26	114	96	2015-02-06 19:01:11.739434	2015-02-06 19:01:11.739434	3
192	Patients with colorectal cancer harboring NRAS mutation has poorer survival outcome and worse prognosis.	Poor Outcome	Supports	2	1	3	26	114	97	2015-02-06 19:01:11.759402	2015-02-06 19:01:11.759402	3
193	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	2	1	3	6	115	94	2015-02-06 19:01:11.779708	2015-02-06 19:01:11.779708	3
194	In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS	Poor Outcome	Supports	2	1	3	6	115	95	2015-02-06 19:01:11.799383	2015-02-06 19:01:11.799383	3
195	Melanoma associated with NRAS Q61 mutation was more often associated with those at the extremity than those at the trunk	Positive	Supports	3	1	3	6	116	100	2015-02-06 19:01:11.817334	2015-02-06 19:01:11.817334	3
196	Due to increased reliance of mutant NRAS on HSP90 for stabilization, inhibition of HSP90 by 17-AAG is shown to be effective in a patient with metastatic malignant melenoma with a NRAS G13D mutation	Sensitivity	Supports	1	1	30	6	117	99	2015-02-06 19:01:11.836094	2015-02-06 19:01:11.836094	2
197	In a melanoma patient with NRAS Q61L mutation, treatment with temozolomide resulted in disease free survival of 14 months	Sensitivity	Supports	1	1	31	6	118	101	2015-02-06 19:01:11.857709	2015-02-06 19:01:11.857709	2
198	In a melanoma patient with Q61R mutation, treatment with temozolomide resulted in overall survival of 16 months	Sensitivity	Supports	1	1	31	6	118	102	2015-02-06 19:01:11.874236	2015-02-06 19:01:11.874236	2
199	"AML with mutated NPM1" is a provisional entity in WHO classification of acute myeloid leukemia (AML). This mutation should be tested for in clinical trials and is recommended for testing in patients with cytogentically normal AML.	Positive	Supports	3	4	3	7	119	92	2015-02-06 19:01:11.891841	2015-02-06 19:01:11.891841	5
200	In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increase survival.	Sensitivity	Supports	1	4	32	5	120	91	2015-02-06 19:01:11.912921	2015-02-06 19:01:11.912921	5
201	In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.	Sensitivity	Supports	1	3	33	5	121	91	2015-02-06 19:01:11.933885	2015-02-06 19:01:11.933885	4
202	MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.	Sensitivity	Supports	1	1	34	5	121	91	2015-02-06 19:01:11.954978	2015-02-06 19:01:11.954978	4
203	Patients with colorectal cancer who harbor KRAS mutation have low response rate to cetuximab	Resistance or Non-Response	Supports	1	1	26	26	122	80	2015-02-06 19:01:11.972564	2015-02-06 19:01:11.972564	4
204	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	1	2	35	3	123	85	2015-02-06 19:01:11.994195	2015-02-06 19:01:11.994195	4
205	The use of NVP-BEZ235 in conjunction with ARRY-142886, but not as monotherapy, in lung cancer model with KRAS G12D mutation led to marked tumor regression.	Sensitivity	Supports	1	2	36	3	123	85	2015-02-06 19:01:12.006875	2015-02-06 19:01:12.006875	4
206	Cells harboring KRAS G12V mutation were insensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft models	Resistance or Non-Response	Supports	1	2	26	28	124	87	2015-02-06 19:01:12.02854	2015-02-06 19:01:12.02854	4
207	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Sensitivity	Supports	1	1	26	28	124	87	2015-02-06 19:01:12.044488	2015-02-06 19:01:12.044488	4
208	Cells harboring KRAS G13D mutation were sensitive to cetuximab treatment in isogenic SW48 cells and in a mouse xenograft models	Sensitivity	Supports	1	2	26	28	124	87	2015-02-06 19:01:12.063776	2015-02-06 19:01:12.063776	4
209	KRAS G13D mutation is associated with better response to Cetuximab with longer progression-free and overall survival in colorectal patients compared to other KRAS	Better Outcome	Supports	2	1	26	28	124	87	2015-02-06 19:01:12.08256	2015-02-06 19:01:12.08256	4
210	KRAS mutation status was not predictive of response to regorafenib treatment in patients that had received standard therapy and progressed within 3 months of their last treatment	Sensitivity	Does Not Support	1	1	25	26	103	81	2015-02-06 19:01:12.099746	2015-02-06 19:01:12.099746	4
211	In patients with stage III colorectal cancer undergoing chemotherapy, KRAS G12 mutation did not impact overall or disease free survival.	N/A	Does Not Support	2	1	3	28	125	82	2015-02-06 19:01:12.117026	2015-02-06 19:01:12.117026	4
212	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	1	1	37	3	126	81	2015-02-06 19:01:12.138229	2015-02-06 19:01:12.138229	3
213	In NSCLC, exon 2 KRAS mutations were associated with resistance to the EGFR kinase inhibitors gefitinib and erlotinib	Resistance or Non-Response	Supports	1	1	38	3	126	81	2015-02-06 19:01:12.149519	2015-02-06 19:01:12.149519	3
214	KRAS mutations were significantly associated with lack of response to cetuximab in patients with advanced colorectal cancer	Resistance or Non-Response	Supports	1	2	26	26	127	83	2015-02-06 19:01:12.167854	2015-02-06 19:01:12.167854	3
215	Chemotherapy-refractory patients with colorectal cancer harboring KRAS mutations (primarily G12/G13) had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype KRAS	Resistance or Non-Response	Supports	1	1	26	26	105	83	2015-02-06 19:01:12.185846	2015-02-06 19:01:12.185846	3
216	KRAS mutations were significantly associated with reduced progression-free and overall survival in patients with advanced colorectal cancer	Poor Outcome	Supports	2	2	3	26	127	83	2015-02-06 19:01:12.202512	2015-02-06 19:01:12.202512	3
217	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	3	1	3	29	128	82	2015-02-06 19:01:12.221156	2015-02-06 19:01:12.221156	3
218	Children with early age leukaemia who had second hand smoke exposure are more likely to harbor KRAS mutation.	Positive	Supports	3	1	3	29	128	86	2015-02-06 19:01:12.237067	2015-02-06 19:01:12.237067	3
219	KRAS G12D mutation occurs in never smokers significantly more often than in smokers	Positive	Supports	3	1	3	30	129	85	2015-02-06 19:01:12.256816	2015-02-06 19:01:12.256816	3
220	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	2	1	3	31	130	82	2015-02-06 19:01:12.275703	2015-02-06 19:01:12.275703	3
221	The presence of KRAS mutations in MM patients is prognostic for shorter overall and progression free survival	Poor Outcome	Supports	2	1	3	32	130	86	2015-02-06 19:01:12.294042	2015-02-06 19:01:12.294042	3
222	KRAS mutations in lung patients are not associated with smoking history, age and gender	Positive	Does Not Support	3	1	3	3	131	82	2015-02-06 19:01:12.311706	2015-02-06 19:01:12.311706	3
223	There is no strong association between KRAS mutation status and overall or progression free survival in patients with NSCLC	N/A	Does Not Support	2	1	3	3	132	82	2015-02-06 19:01:12.331567	2015-02-06 19:01:12.331567	3
224	A meta-analysis showed KRAS mutation is associated with worse outcome in patients with NSCLC	Poor Outcome	Supports	2	1	3	3	133	82	2015-02-06 19:01:12.351823	2015-02-06 19:01:12.351823	3
225	KRAS mutations in lung patients are not associated with smoking history, age and gender	Negative	Does Not Support	3	1	3	3	131	86	2015-02-06 19:01:12.379465	2015-02-06 19:01:12.379465	3
226	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	1	1	4	3	134	85	2015-02-06 19:01:12.399395	2015-02-06 19:01:12.399395	2
227	KRAS G12C occur more frequently in women than men	Positive	Supports	3	1	3	30	129	84	2015-02-06 19:01:12.416552	2015-02-06 19:01:12.416552	2
228	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	2	1	10	6	135	73	2015-02-06 19:01:12.43407	2015-02-06 19:01:12.43407	3
229	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	2	1	10	6	135	74	2015-02-06 19:01:12.450033	2015-02-06 19:01:12.450033	3
230	Melanoma patients with KIT mutation but not KIT amplification showed response to imatinib treatment in a small cohort of patients.	Better Outcome	Supports	2	1	10	6	135	76	2015-02-06 19:01:12.468844	2015-02-06 19:01:12.468844	3
231	KIT mutations were identified in morphologically benign, incidentally discovered GISTs at a rate similar to that seen in advanced, metastatic GIST patients indicating is not a prognostic marker for GIST	N/A	Does Not Support	2	1	3	21	136	73	2015-02-06 19:01:12.48746	2015-02-06 19:01:12.48746	3
232	There is no significant association between KIT mutations in exon 11, exon 9 or wild type in survival among GIST patients.	N/A	Does Not Support	2	1	3	21	137	73	2015-02-06 19:01:12.505237	2015-02-06 19:01:12.505237	3
233	KIT mutation is associated with worse overall and cause-specific prognosis in patients with GIST compared to patients with wildtype KIT	Poor Outcome	Supports	2	1	3	21	138	73	2015-02-06 19:01:12.524362	2015-02-06 19:01:12.524362	3
234	GIST patients with exon 14 KIT mutations had reduced overall survival compared to patients wiltype for KIT	Poor Outcome	Supports	2	1	3	21	137	75	2015-02-06 19:01:12.541752	2015-02-06 19:01:12.541752	3
235	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	3	1	3	21	138	76	2015-02-06 19:01:12.557587	2015-02-06 19:01:12.557587	3
236	There is no significant association between KIT mutations in exon 11, exon 9 or wild type in survival among GIST patients.	N/A	Does Not Support	2	1	3	21	137	77	2015-02-06 19:01:12.57683	2015-02-06 19:01:12.57683	3
237	KIT mutation is associated with larger, more invasive tumors, greater pathologic histology and older patients compared to tumors with wildtype KIT	Positive	Supports	3	1	3	21	138	73	2015-02-06 19:01:12.595765	2015-02-06 19:01:12.595765	2
238	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N/A	Does Not Support	2	1	3	30	139	73	2015-02-06 19:01:12.614369	2015-02-06 19:01:12.614369	2
239	KIT mutations detected in 5/60 patients showed no prognostic significance in patients with small cell lung cancer	N/A	Does Not Support	2	1	3	30	139	77	2015-02-06 19:01:12.630072	2015-02-06 19:01:12.630072	2
240	In patients with JAK2 V617F, the use of peglated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.	Sensitivity	Supports	1	1	39	33	140	71	2015-02-06 19:01:12.652938	2015-02-06 19:01:12.652938	4
241	JAK2 V617F not associatted wth  lymphoid leukemia.	Positive	Does Not Support	3	1	3	9	141	71	2015-02-06 19:01:12.670803	2015-02-06 19:01:12.670803	4
242	JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).	Positive	Supports	3	1	3	9	141	71	2015-02-06 19:01:12.689272	2015-02-06 19:01:12.689272	4
243	JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).	Negative	Supports	3	1	3	34	141	71	2015-02-06 19:01:12.707933	2015-02-06 19:01:12.707933	4
244	rs10974944 SNP in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer	Positive	Supports	3	1	3	35	142	71	2015-02-06 19:01:12.726621	2015-02-06 19:01:12.726621	4
245	TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells	Sensitivity	Supports	1	2	40	36	143	71	2015-02-06 19:01:12.748173	2015-02-06 19:01:12.748173	3
246	In patients with IDH2 R140Q/L mutation, the presence of mutation do not impact overall survival or disease free survival.	N/A	Does Not Support	2	1	3	7	144	69	2015-02-06 19:01:12.767835	2015-02-06 19:01:12.767835	3
247	In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2	Better Outcome	Supports	2	1	3	7	145	69	2015-02-06 19:01:12.787	2015-02-06 19:01:12.787	3
248	In AML, patients with IDH2 R172K mutation have worse overall survival compared to those with wild-type IDH2	Poor Outcome	Supports	2	1	3	7	145	70	2015-02-06 19:01:12.805021	2015-02-06 19:01:12.805021	3
249	R140Q mutation in IDH2 does not have prognostic value in patients with MDS.	N/A	Does Not Support	2	1	3	25	146	69	2015-02-06 19:01:12.823416	2015-02-06 19:01:12.823416	3
250	AML patients with IDH2 mutations such as R140Q/L have event free survival and overall survival similar to those with wild-type IDH2	N/A	Does Not Support	2	3	3	7	147	69	2015-02-06 19:01:12.840669	2015-02-06 19:01:12.840669	2
251	AML patients with IDH2 mutations such as R172K have event free survival and overall survival similar to those with wild-type IDH2.	N/A	Does Not Support	2	3	3	7	147	70	2015-02-06 19:01:12.856695	2015-02-06 19:01:12.856695	2
252	In patients suffering from MDS, the presence of IDH2 mutation such as R140Q/L do not confer prognostic value (overall survival)	N/A	Does Not Support	2	3	3	25	148	69	2015-02-06 19:01:12.873936	2015-02-06 19:01:12.873936	2
253	In patients suffering from MDS, the presence of IDH2 mutation such as R172K do not confer prognostic value (overall survival)	N/A	Does Not Support	2	3	3	25	148	70	2015-02-06 19:01:12.890289	2015-02-06 19:01:12.890289	2
254	IDH1 R132 mutation in patients suffering from myelodysplastic syndromes is associated with worse overall survival	Poor Outcome	Supports	2	1	3	37	149	64	2015-02-06 19:01:12.90949	2015-02-06 19:01:12.90949	4
255	IDH1 R132 mutation is associated with patients of older age, high platelet count during diagnosis, cytogenic normalcy and NPM1 mutation	Positive	Supports	3	1	3	7	150	65	2015-02-06 19:01:12.92929	2015-02-06 19:01:12.92929	3
256	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	2	1	3	7	150	67	2015-02-06 19:01:12.947432	2015-02-06 19:01:12.947432	3
257	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	2	1	3	7	150	68	2015-02-06 19:01:12.965103	2015-02-06 19:01:12.965103	3
258	In patients with astrocytoma, the presence of IDH1 R132 mutation is prognostic for better survival compared to patients who harbor wild type mutation.	Better Outcome	Supports	2	1	3	38	151	64	2015-02-06 19:01:12.988208	2015-02-06 19:01:12.988208	3
259	In patients with glioblastoma, those harboring IDH1 R132 mutation has higher overall survival compared to those who do not have IDH1 mutation	Better Outcome	Supports	2	1	3	39	152	64	2015-02-06 19:01:13.010024	2015-02-06 19:01:13.010024	3
260	In leukemia patients, IDH1 R132 mutation is associated with adults and normal karyotype.	Positive	Supports	3	1	3	7	153	64	2015-02-06 19:01:13.030701	2015-02-06 19:01:13.030701	2
261	The complete remission and overall survival rates in patients with IDH1 R132 mutation is not significantly different from those who do not have this mutation	N/A	Does Not Support	2	1	3	7	154	64	2015-02-06 19:01:13.049175	2015-02-06 19:01:13.049175	2
262	IDH1 R132 mutation in patients with AML is not associated with any prognostic value compared to patients with wild-type IDH1.	N/A	Does Not Support	2	1	3	7	150	65	2015-02-06 19:01:13.065828	2015-02-06 19:01:13.065828	2
263	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	1	1	41	40	155	63	2015-02-06 19:01:13.099913	2015-02-06 19:01:13.099913	4
264	Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.	Sensitivity	Supports	1	1	42	40	155	63	2015-02-06 19:01:13.110699	2015-02-06 19:01:13.110699	4
265	In AML patients with FLT3-ITD  , concurrent DNMT3A mutations were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	2	1	3	7	55	61	2015-02-06 19:01:13.126644	2015-02-06 19:01:13.126644	5
266	Sorafenib is effective in patients with FLT3 internal tandem repeat , but not in wild type or D835 mutation.	Sensitivity	Supports	1	2	14	7	156	61	2015-02-06 19:01:13.146866	2015-02-06 19:01:13.146866	4
267	Emergence of D835 mutation in AML patients with FLT3 tandem repeat is associated with sorafenib resistance	Resistance or Non-Response	Supports	1	1	14	7	157	62	2015-02-06 19:01:13.167697	2015-02-06 19:01:13.167697	4
268	Sorafenib is effective in patients with FLT3 internal tandem repeat , but not in wild type or D835 mutation.	Resistance or Non-Response	Supports	1	2	14	7	156	62	2015-02-06 19:01:13.183786	2015-02-06 19:01:13.183786	4
269	CEP701 is effective in AML patients with FLT3 mutation	Sensitivity	Supports	1	1	43	7	158	61	2015-02-06 19:01:13.202379	2015-02-06 19:01:13.202379	4
270	CEP701 is effective in AML patients with FLT3 mutation	Sensitivity	Supports	1	1	43	7	158	62	2015-02-06 19:01:13.219768	2015-02-06 19:01:13.219768	4
271	AG1296 is effective in triggering apoptosis in Ba/F3 and primary AML cells with FLT3 internal tandem repeat	Sensitivity	Supports	1	2	44	7	159	61	2015-02-06 19:01:13.238816	2015-02-06 19:01:13.238816	4
272	FLT3-ITD mutation without NPM1 was associated with increased relapse risk and reduced overall survival in young adult AML patients (median age 43)	Resistance or Non-Response	Supports	1	1	3	7	48	61	2015-02-06 19:01:13.254972	2015-02-06 19:01:13.254972	4
273	In young AML patients (<60 years old), DNMT3A mutation status was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	2	1	3	7	49	61	2015-02-06 19:01:13.271549	2015-02-06 19:01:13.271549	4
274	FLT3-ITD duplication length did not effect prognosis in patients with an NPM1 mutation	N/A	Does Not Support	2	1	3	7	54	61	2015-02-06 19:01:13.288118	2015-02-06 19:01:13.288118	4
275	Simultaneous tandem repeat of FLT3 with loss of wild type FLT3 leads to poor survival and inferior disease free survival in patients with AML.	Poor Outcome	Supports	2	1	3	7	160	61	2015-02-06 19:01:13.30563	2015-02-06 19:01:13.30563	4
276	Patients with NPM1 mutation and FLT3-ITD had reduced event-free survival compared to patients with wildtype FLT3	Poor Outcome	Supports	2	1	3	7	54	61	2015-02-06 19:01:13.322194	2015-02-06 19:01:13.322194	4
277	Using a ratio of FLT3-ITD/FLT3 wildtype levels as a continuous variable, a higher ratio was associated with reduced event-free survival in patients with an NPM1 mutation	Poor Outcome	Supports	2	1	3	7	54	61	2015-02-06 19:01:13.338278	2015-02-06 19:01:13.338278	4
278	A ratio of >0.49 FLT3-ITD/FLT3 wildtype level was associated with reduced event-free and overall survival in old (>59) and younger patients (<60) with an NPM1 mutation	Poor Outcome	Supports	2	1	3	7	54	61	2015-02-06 19:01:13.354201	2015-02-06 19:01:13.354201	4
279	In AML patients, FLT3-TKD mutation is associated with poorer disease free survival compared to patients with wild type FLT3	Poor Outcome	Supports	2	1	3	7	161	62	2015-02-06 19:01:13.374559	2015-02-06 19:01:13.374559	4
280	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	1	2	45	7	162	61	2015-02-06 19:01:13.394035	2015-02-06 19:01:13.394035	3
281	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to the protein tyrosine kinase (PTK) inhibitor SU5614 than cells with FLT3-ITD alone or FLT3 ITD and TKD mutations on opposing alleles.	Resistance or Non-Response	Supports	1	2	45	7	162	62	2015-02-06 19:01:13.409487	2015-02-06 19:01:13.409487	3
282	ATRA treatment did not effect overall survival in patients <60 years old with FLT3-ITD mutation	Resistance or Non-Response	Supports	1	1	6	7	57	61	2015-02-06 19:01:13.425487	2015-02-06 19:01:13.425487	3
283	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	1	2	7	7	162	61	2015-02-06 19:01:13.443179	2015-02-06 19:01:13.443179	3
284	Ba/F3 cells with both FLT3-ITD and TKD mutations on the same allele have more resistance to daunorubicin than cells with FLT3-ITD alone.	Resistance or Non-Response	Supports	1	2	7	7	162	62	2015-02-06 19:01:13.462576	2015-02-06 19:01:13.462576	3
285	WBC counts were higher in AML patients with internal tandem repeat of Flt3 compared to wild type	Positive	Supports	3	1	3	7	163	61	2015-02-06 19:01:13.480595	2015-02-06 19:01:13.480595	3
286	Increasing level of FLT3-ITD mutant correlated with reduced disease-free and overall survival as well as increased relapse rate	Resistance or Non-Response	Supports	1	1	3	7	48	61	2015-02-06 19:01:13.499259	2015-02-06 19:01:13.499259	3
287	AML patients with internal tandem repeat of FLT3 has worse overall survival than patients with wild type FLT3	Poor Outcome	Supports	2	1	3	7	163	61	2015-02-06 19:01:13.518156	2015-02-06 19:01:13.518156	3
288	WBC counts were not significantly different in AML patients with FLT3 D835 mutations compared to wild type	Positive	Does Not Support	3	1	3	7	163	62	2015-02-06 19:01:13.534283	2015-02-06 19:01:13.534283	3
289	AML patients with D835 mutation of FLT3 did not have worse overall survival than patients with wild type FLT3	N/A	Does Not Support	2	1	3	7	163	62	2015-02-06 19:01:13.552267	2015-02-06 19:01:13.552267	3
290	Event-free survival was significantly improved in patients with both NPM1 and FLT3-TKD mutations compared to patients with 1) FLT3-TKD mutations alone 2) either NPM1 or FLT3-TKD when FLT3-ITD patients were excluded from the analysis 3) neither FLT3-TKD or NPM1 mutations or FLT3-TKD alone when only de novo, normal karyotype patients were analyzed	Better Outcome	Supports	2	1	3	7	60	62	2015-02-06 19:01:13.568944	2015-02-06 19:01:13.568944	3
291	Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate	N/A	Does Not Support	2	1	3	7	48	61	2015-02-06 19:01:13.585445	2015-02-06 19:01:13.585445	2
292	Meta-analysis of studies involving cytogentically normal younger (<60) patients showed reduced overall and relapse-free survival for patients with FLT3-ITD	Poor Outcome	Supports	2	4	3	7	164	61	2015-02-06 19:01:13.603537	2015-02-06 19:01:13.603537	2
293	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	46	11	165	38	2015-02-06 19:01:13.624684	2015-02-06 19:01:13.624684	5
294	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	46	11	165	39	2015-02-06 19:01:13.642527	2015-02-06 19:01:13.642527	5
295	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	46	11	165	40	2015-02-06 19:01:13.659014	2015-02-06 19:01:13.659014	5
296	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	46	11	165	41	2015-02-06 19:01:13.675168	2015-02-06 19:01:13.675168	5
297	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	46	11	165	43	2015-02-06 19:01:13.691364	2015-02-06 19:01:13.691364	5
298	This mutation is sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	46	11	165	45	2015-02-06 19:01:13.708661	2015-02-06 19:01:13.708661	5
299	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	46	11	165	46	2015-02-06 19:01:13.737453	2015-02-06 19:01:13.737453	5
300	All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.	Sensitivity	Supports	1	2	46	11	165	37	2015-02-06 19:01:13.756338	2015-02-06 19:01:13.756338	4
301	In NSCLC patients treated with tyrosine kinase inhibitors, the presence of L858R mutation is prognostic for better progression free survival.	Sensitivity	Supports	1	1	3	3	166	35	2015-02-06 19:01:13.775483	2015-02-06 19:01:13.775483	4
302	Patients with NSCLC harboring EGFR T790M mutation has statistically worse overall survival compared to patients with L858R or other exon 19 activating mutation.	Poor Outcome	Supports	2	1	3	3	167	36	2015-02-06 19:01:13.794699	2015-02-06 19:01:13.794699	4
303	In NSCLC patients with T790M and another activating mutation, their progression free survival is shorter compared to those who do not possess T790M mutation.	Poor Outcome	Supports	2	1	3	3	168	36	2015-02-06 19:01:13.812987	2015-02-06 19:01:13.812987	4
304	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective.	Sensitivity	Supports	1	1	38	3	169	36	2015-02-06 19:01:13.832262	2015-02-06 19:01:13.832262	3
305	In a NSCLC patient with T790M and L858R mutation, combination treatment with erlotinib and premetrexed has shown to be effective.	Sensitivity	Supports	1	1	47	3	169	36	2015-02-06 19:01:13.843023	2015-02-06 19:01:13.843023	3
306	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.	N/A	Does Not Support	1	1	37	3	170	35	2015-02-06 19:01:13.862191	2015-02-06 19:01:13.862191	3
307	There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib.	N/A	Does Not Support	1	1	38	3	170	35	2015-02-06 19:01:13.871617	2015-02-06 19:01:13.871617	3
308	Geftinib added to pemetrexed and carboplatin showed greater effectiveness than premetrexed and carboplatin alone in treating NSCLC patients with L858R mutation.	Sensitivity	Supports	1	1	37	3	171	35	2015-02-06 19:01:13.8891	2015-02-06 19:01:13.8891	3
309	In non-small cell lung cancer, the appearance of T790M mutation leads to resistance to gefitinib.	Resistance or Non-Response	Supports	1	1	37	3	172	36	2015-02-06 19:01:13.906741	2015-02-06 19:01:13.906741	3
310	Median survival of patients with EGFR L858R mutation is better than those with wild type EGFR.	Better Outcome	Supports	2	1	3	3	173	35	2015-02-06 19:01:13.924221	2015-02-06 19:01:13.924221	3
311	In NSCLC containing T790M mutation, staurosporine may be inhibitive of EGFR, especially when L858R mutation is also present.	Sensitivity	Supports	1	3	48	3	174	36	2015-02-06 19:01:13.944875	2015-02-06 19:01:13.944875	1
312	DNMT3A mutations (59% of which were R882) were associated with an intermediate risk cytogenetic profile, normal cytogenetic profile, and M4 and M5 FAB subtypes	Positive	Supports	3	1	3	7	55	33	2015-02-06 19:01:13.96141	2015-02-06 19:01:13.96141	5
313	AML patients with DNMT3A mutations (59% of which were R882) showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	2	1	3	7	55	33	2015-02-06 19:01:13.980274	2015-02-06 19:01:13.980274	5
314	In AML patients with FLT3-ITD  , concurrent DNMT3A mutations (including R882) were associated with worse overall survival compared to those without DNMT3A mutation.	Poor Outcome	Supports	2	1	3	7	55	33	2015-02-06 19:01:13.997133	2015-02-06 19:01:13.997133	5
315	In cytogenetically normal AML patients, DNMT3A R882 mutations are associated with lower overall and disease free survival as compared to patients with wild type DNMT3A.	Poor Outcome	Supports	2	1	3	7	175	33	2015-02-06 19:01:14.016764	2015-02-06 19:01:14.016764	5
316	Idarubicin increases the overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	Sensitivity	Supports	1	1	49	7	176	33	2015-02-06 19:01:14.036067	2015-02-06 19:01:14.036067	4
317	Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.	NA	Does Not Support	1	1	7	7	176	33	2015-02-06 19:01:14.052549	2015-02-06 19:01:14.052549	4
318	Complete remission rates did not differ between patients with wildtype or mutant DNMT3A (62% of which affected R882) and cytogenetically normal AML	N/A	Does Not Support	2	1	3	7	177	33	2015-02-06 19:01:14.070679	2015-02-06 19:01:14.070679	4
319	In young AML patients (<60 years old), DNMT3A mutation status (60% of which were R882) was not predictive of overall and relapse free survival in patients with NPM1 mutations and wildtype FLT3 or wildtype NPM1 and FLT3-ITD	N/A	Does Not Support	2	1	3	7	49	33	2015-02-06 19:01:14.087008	2015-02-06 19:01:14.087008	4
320	There is no difference in the complete remission rate of de novo AML patients with DNMT3A mutation compared to those who are wild type for DNMT3A.	N/A	Does Not Support	2	1	3	7	176	33	2015-02-06 19:01:14.103498	2015-02-06 19:01:14.103498	4
321	DNMT3A mutations (62% of which were R882) were associated with reduced disease-free survival in patients with cytogenetically normal AML.	Poor Outcome	Supports	2	1	3	7	177	33	2015-02-06 19:01:14.119447	2015-02-06 19:01:14.119447	4
322	In young AML patients (<60 years old), DNMT3A mutations were associated with significantly reduced overall survival and relapse free survival in patients wildtype for NPM1 and FLT3	Poor Outcome	Supports	2	1	3	7	49	33	2015-02-06 19:01:14.139085	2015-02-06 19:01:14.139085	4
323	De novo AML patients with DNMT3A D882 mutation showed worse survival (event-free and overall) outcome than those without DNMT3A mutation.	Poor Outcome	Supports	2	1	3	7	55	33	2015-02-06 19:01:14.15691	2015-02-06 19:01:14.15691	4
324	In older cytogenetically normal AML patients (>59 years), DNMT3A R882 mutation is prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	2	1	3	7	177	33	2015-02-06 19:01:14.173555	2015-02-06 19:01:14.173555	4
325	In younger cytogenetically normal AML patients (<60 years), DNMT3A mutations other than R882 are prognostic for shorter disease free survival and overall survival compared to patients without the mutation.	Poor Outcome	Supports	2	1	3	7	177	33	2015-02-06 19:01:14.18973	2015-02-06 19:01:14.18973	4
326	Young AML patients (<60 years old) with DNMT3A mutations (60% of which were R882) were older in age, had higher white blood cell counts and had higher platelet counts than patients wildtype for DNMT3A	Positive	Supports	3	1	3	7	49	33	2015-02-06 19:01:14.206018	2015-02-06 19:01:14.206018	3
327	DNMT3A mutations (64.5% R882) were associated with older age, higher white blood cell count and cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	3	1	3	7	178	33	2015-02-06 19:01:14.226955	2015-02-06 19:01:14.226955	3
328	Therapy-related AML was less common in patients with DNMT3A mutations (64.5% of which were R882) than patients wildtype for DNMT3A in a large cohort of younger (18-60) AML patients	Negative	Supports	3	1	3	7	178	33	2015-02-06 19:01:14.243853	2015-02-06 19:01:14.243853	3
329	DNMT3A R882 mutations were associated with cytogenetically normal AML in a large cohort of younger (18-60) AML patients	Positive	Supports	3	1	3	7	178	33	2015-02-06 19:01:14.259998	2015-02-06 19:01:14.259998	3
330	DNMT3A R882 mutations occur most often in de novo AML patients with intermediate cytogenic risk.	Positive	Supports	3	1	3	7	176	33	2015-02-06 19:01:14.277578	2015-02-06 19:01:14.277578	3
331	DNMT3A R882 mutations were associated with older age, higher white blood cell count, and FAB M4 and M5 subtypes compared to wildtype DNMT3A in a cohort of cytogenetically normal AML patients	Positive	Supports	3	1	3	7	175	33	2015-02-06 19:01:14.295812	2015-02-06 19:01:14.295812	3
332	In a large cohort of AML patients (mean = 48 years), DNMT3A mutation (64.5% of which were R882) had no prognostic value on overall, relapse free and event free survival.	N/A	Does Not Support	2	1	3	7	178	33	2015-02-06 19:01:14.31398	2015-02-06 19:01:14.31398	3
333	In a large cohort of cytogenetically normal AML patients (18-60 years old), DNMT3A mutation status (65.4% of which were R882) had no prognostic value for overall, relapse free and event free survival	N/A	Does Not Support	2	1	3	7	178	33	2015-02-06 19:01:14.331771	2015-02-06 19:01:14.331771	3
334	DNMT3A mutations were associated with achievement of complete remission in a large cohort of younger (18-60) AML patients	Better Outcome	Supports	2	1	3	7	178	33	2015-02-06 19:01:14.362425	2015-02-06 19:01:14.362425	3
335	DNMT3A R882 mutation was associated with reduced relapse free and overall survival in ELN-unfavorable cytogenetically normal AMLs	Poor Outcome	Supports	2	1	3	7	178	33	2015-02-06 19:01:14.380964	2015-02-06 19:01:14.380964	3
336	Young AML patients (<60 years old) with DNMT3A mutation have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	2	1	3	7	49	33	2015-02-06 19:01:14.39736	2015-02-06 19:01:14.39736	3
337	Complete remission rate was not different between young AML patients (<60 years old) with or without DNMT3A mutations (60% of which were R882)	Poor Outcome	Supports	2	1	3	7	49	33	2015-02-06 19:01:14.414824	2015-02-06 19:01:14.414824	3
338	Young AML patients (<60 years old) with DNMT3A R882 mutations have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A	Poor Outcome	Supports	2	1	3	7	49	33	2015-02-06 19:01:14.431917	2015-02-06 19:01:14.431917	3
339	In a large cohort of young AML patients (18-60 years old), DNMT3A R882 mutations were associated with reduced relapse free survival in the entire cohort as well as the subset of patients with cytogenetically normal AML	Poor Outcome	Supports	2	4	3	7	178	33	2015-02-06 19:01:14.448462	2015-02-06 19:01:14.448462	3
340	CEBPA mutation status had no impact on ATRA treatment response in older (>60) patients with AML	N/A	Does Not Support	1	1	3	7	57	30	2015-02-06 19:01:14.465865	2015-02-06 19:01:14.465865	4
341	CEBPA mutation was significantly associated with complete remission	Better Outcome	Supports	2	1	3	7	69	30	2015-02-06 19:01:14.482915	2015-02-06 19:01:14.482915	4
342	CEBPA mutation was associated with improved overall survival in older (>60) patients with AML	Better Outcome	Supports	2	1	3	7	57	30	2015-02-06 19:01:14.499335	2015-02-06 19:01:14.499335	4
343	Overall survival was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA mutations	Better Outcome	Supports	2	1	3	7	179	29	2015-02-06 19:01:14.522808	2015-02-06 19:01:14.522808	3
344	Remission duration was significantly longer in younger (16-60), cytogenetically normal AML patients with CEBPA	Better Outcome	Supports	2	1	3	7	179	29	2015-02-06 19:01:14.540013	2015-02-06 19:01:14.540013	3
345	FLT3 mutations did not alter overall survival in younger (16-60), cytogenetically normal AML patients with CEBPA	N/A	Does Not Support	2	1	3	7	179	29	2015-02-06 19:01:14.556725	2015-02-06 19:01:14.556725	2
346	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence.	Poor Outcome	Supports	2	1	3	3	180	28	2015-02-06 19:01:14.578058	2015-02-06 19:01:14.578058	3
347	In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter overall survival.	Poor Outcome	Supports	2	1	3	3	180	28	2015-02-06 19:01:14.594378	2015-02-06 19:01:14.594378	3
348	In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	1	1	50	41	181	27	2015-02-06 19:01:14.615785	2015-02-06 19:01:14.615785	4
349	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	2	1	51	41	182	27	2015-02-06 19:01:14.638003	2015-02-06 19:01:14.638003	3
382	V600E is correlated with disease recurrence in both age cohorts (>65 and <65 yo).	Poor Outcome	Supports	2	1	3	14	200	11	2015-02-06 19:01:15.303851	2015-02-06 19:01:15.303851	3
350	In a patient-derived xenograft of a PIK3CA mutant, the combination of PI3K and CDK 4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.	Sensitivity	Supports	1	1	50	41	181	26	2015-02-06 19:01:14.655674	2015-02-06 19:01:14.655674	4
351	Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.	Better Outcome	Supports	2	1	51	41	182	26	2015-02-06 19:01:14.674628	2015-02-06 19:01:14.674628	3
352	Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.	Poor Outcome	Supports	2	1	3	11	183	25	2015-02-06 19:01:14.693949	2015-02-06 19:01:14.693949	4
353	Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.	Poor Outcome	Supports	2	1	3	42	184	25	2015-02-06 19:01:14.713355	2015-02-06 19:01:14.713355	4
354	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	2	1	3	42	184	25	2015-02-06 19:01:14.73124	2015-02-06 19:01:14.73124	4
355	Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.	Poor Outcome	Supports	2	1	3	30	184	25	2015-02-06 19:01:14.751344	2015-02-06 19:01:14.751344	4
356	Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.	Sensitivity	Supports	1	2	52	43	185	24	2015-02-06 19:01:14.772623	2015-02-06 19:01:14.772623	3
357	Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.	Poor Outcome	Supports	2	2	3	44	186	21	2015-02-06 19:01:14.794627	2015-02-06 19:01:14.794627	4
358	Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.	Poor Outcome	Supports	2	2	3	44	186	21	2015-02-06 19:01:14.811184	2015-02-06 19:01:14.811184	4
359	Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.	Poor Outcome	Supports	2	2	3	44	186	21	2015-02-06 19:01:14.827646	2015-02-06 19:01:14.827646	4
360	Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.	Poor Outcome	Supports	2	2	3	44	186	21	2015-02-06 19:01:14.843785	2015-02-06 19:01:14.843785	4
361	CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.	Negative	Supports	3	2	3	44	187	22	2015-02-06 19:01:14.864263	2015-02-06 19:01:14.864263	3
362	Treatment of CCND2-negative cells with 5-Aza-2'-deoxycytidine restored cyclin D2 expression.	Sensitivity	Supports	1	2	3	44	187	22	2015-02-06 19:01:14.88061	2015-02-06 19:01:14.88061	2
363	dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	1	1	4	6	188	11	2015-02-06 19:01:14.899824	2015-02-06 19:01:14.899824	5
364	dabrafenib with trametinib provides higher response rate and lower toxicity-as compared to chemotherapy-in patients with melanoma.	Sensitivity	Supports	1	1	53	6	188	11	2015-02-06 19:01:14.912263	2015-02-06 19:01:14.912263	5
365	V600E is associated with adverse pathological features of colorectal cancer. This can be concluded as poor prognosis.	Poor Outcome	Supports	2	4	3	26	189	11	2015-02-06 19:01:14.932659	2015-02-06 19:01:14.932659	5
366	thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.	Positive	Supports	3	1	3	14	190	11	2015-02-06 19:01:14.951915	2015-02-06 19:01:14.951915	5
367	thyroid nodule with BRAF V600E mutation is highly correlated with thyroid cancer.	Positive	Supports	3	1	3	14	191	11	2015-02-06 19:01:15.04179	2015-02-06 19:01:15.04179	5
368	Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.	Poor Outcome	Does Not Support	2	1	3	14	192	11	2015-02-06 19:01:15.06234	2015-02-06 19:01:15.06234	5
369	COLO201 and COLO206F cell harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244) resistant. The mechanim of this resistence was shown to be amplification of the BRAF V600E gene.	Resistance or Non-Response	Supports	1	3	23	26	193	13	2015-02-06 19:01:15.080441	2015-02-06 19:01:15.080441	4
370	BRAF mutations are associated with melanoma arising in nonchronic sun damage skin and with superficial spreading melanoma	Positive	Supports	3	1	3	6	194	16	2015-02-06 19:01:15.101087	2015-02-06 19:01:15.101087	4
371	In metastatic colorectal cancer patients with wildtype KRAS, BRAF mutations were associated with poor progression free survival regardless of treatment (panitumumab with best supportive care or best supportive care alone)	Resistance or Non-Response	Supports	1	1	27	26	195	16	2015-02-06 19:01:15.11928	2015-02-06 19:01:15.11928	3
372	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	1	2	54	28	196	11	2015-02-06 19:01:15.138823	2015-02-06 19:01:15.138823	3
373	Combined PD0325901 and PLX4720 administration to NSG mice subcutanousely injected with colorectal cell lines with a BRAF V600E mutation effectively inhibited tumor growth and reduced cellular proliferation	Sensitivity	Supports	1	2	55	28	196	11	2015-02-06 19:01:15.150058	2015-02-06 19:01:15.150058	3
374	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	2	26	28	197	11	2015-02-06 19:01:15.172207	2015-02-06 19:01:15.172207	3
375	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	2	27	28	197	11	2015-02-06 19:01:15.184215	2015-02-06 19:01:15.184215	3
376	Cetuximab or panitumumab may be ineffective in patients with BRAF mutation unless BRAF inhibitor such as Sorafenib is introduced.	Resistance or Non-Response	Supports	1	2	14	28	197	11	2015-02-06 19:01:15.19449	2015-02-06 19:01:15.19449	3
377	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	1	1	4	6	198	16	2015-02-06 19:01:15.214792	2015-02-06 19:01:15.214792	3
378	Melanoma patients with BRAF V600E/K mutations had longer progression-free survival, increased tumor regression,  and increased duration of response to combined dabrafenib and trametinib treatment compared to dabrafenib alone	Sensitivity	Supports	1	1	53	6	198	16	2015-02-06 19:01:15.227053	2015-02-06 19:01:15.227053	3
379	Chemotherapy-refractory patients with colorectal cancer harboring BRAF mutations had lower response and disease control rates and shorter progression free and overall survival following cetuximab plus chemotherapy than those with wildtype BRAF	Resistance or Non-Response	Supports	1	1	26	26	105	11	2015-02-06 19:01:15.24585	2015-02-06 19:01:15.24585	3
380	BRAF V600E is correlated with shorter Disease-Free and Overall Survival in a Spanish cohort of Melanoma patients.	Poor Outcome	Supports	2	1	3	6	199	11	2015-02-06 19:01:15.266314	2015-02-06 19:01:15.266314	3
381	BRAF V600E is shown to be associated with the tall-cell variant of PTC	Positive	Supports	3	1	3	14	200	11	2015-02-06 19:01:15.285101	2015-02-06 19:01:15.285101	3
383	BRAF V600E is correlated with poor prognosis in papillary thyroid cancer in a study of 187 patients with PTC and other thyroid diseases.	Poor Outcome	Supports	2	1	3	14	201	11	2015-02-06 19:01:15.322093	2015-02-06 19:01:15.322093	3
384	BRAF mutation correlated with poor prognosis in papillary thyroid cancer in both older (>65 yo) and younger (<65 yo) cohorts.	Poor Outcome	Supports	2	1	3	14	200	11	2015-02-06 19:01:15.33999	2015-02-06 19:01:15.33999	3
385	BRAF V600E resistance is triggered by ERK dependent or independent pathways.	Resistance or Non-Response	Supports	1	2	3	6	202	11	2015-02-06 19:01:15.358659	2015-02-06 19:01:15.358659	3
386	treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g. dabrafenib) appears to be beneficial, and better than vice versa.	Sensitivity	Supports	1	1	56	6	203	11	2015-02-06 19:01:15.381092	2015-02-06 19:01:15.381092	2
387	treatment with immunotherapy (e.g. Ipilimumab) prior to RAF inhibitor (e.g. dabrafenib) appears to be beneficial, and better than vice versa.	Sensitivity	Supports	1	1	4	6	203	11	2015-02-06 19:01:15.392257	2015-02-06 19:01:15.392257	2
388	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N/A	Does Not Support	1	1	57	6	204	11	2015-02-06 19:01:15.414139	2015-02-06 19:01:15.414139	2
389	BRAF status does not predict prognosis in patients treated with dacarbazine or temozolomide.	N/A	Does Not Support	1	1	32	6	204	11	2015-02-06 19:01:15.425017	2015-02-06 19:01:15.425017	2
390	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	2	58	26	205	11	2015-02-06 19:01:15.449332	2015-02-06 19:01:15.449332	2
391	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	2	29	26	205	11	2015-02-06 19:01:15.459088	2015-02-06 19:01:15.459088	2
392	Mouse xenografts using HT29 cells harboring the BRAF V600E mutation treated with combination therapy (capecitabine, vemurafenib, bevacizumab) showed increased survival and reduced tumor burden compared to single and double agent therapies.	Sensitivity	Supports	1	2	59	26	205	11	2015-02-06 19:01:15.473214	2015-02-06 19:01:15.473214	2
393	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	1	2	55	28	206	11	2015-02-06 19:01:15.491671	2015-02-06 19:01:15.491671	2
394	Combined nutlin-3 and PLX4720 administration to athymic nude mice subcutanousely injected with the A357 colorectal cell line with a BRAF V600E mutation effectively inhibited tumor growth significantly more than single agent therapy	Sensitivity	Supports	1	2	60	28	206	11	2015-02-06 19:01:15.503697	2015-02-06 19:01:15.503697	2
395	In a patient with V600E-positive NSCLC, KRAS G12D seemed to confer resistance to dabrafenib.	Resistance or Non-Response	Supports	1	1	4	3	134	11	2015-02-06 19:01:15.521599	2015-02-06 19:01:15.521599	2
396	Cell lines expressing the BRAF V600E mutation responded better to vemurafenib treatment than cells wildtype for BRAF as measured by reduced cellular proliferation and inhibition of MET and ERK phosphorylation	Sensitivity	Supports	1	2	29	26	205	11	2015-02-06 19:01:15.539	2015-02-06 19:01:15.539	2
397	Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.	Positive	Supports	3	1	3	11	207	19	2015-02-06 19:01:15.559469	2015-02-06 19:01:15.559469	4
398	In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.	Poor Outcome	Supports	2	1	3	11	207	19	2015-02-06 19:01:15.579481	2015-02-06 19:01:15.579481	4
399	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.	Poor Outcome	Supports	2	1	3	45	208	19	2015-02-06 19:01:15.599306	2015-02-06 19:01:15.599306	4
400	Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.	Poor Outcome	Supports	2	1	3	46	209	19	2015-02-06 19:01:15.618924	2015-02-06 19:01:15.618924	4
401	Inceased copy number of CCND1 is associated with poorer overall survival.	Poor Outcome	Supports	2	1	3	3	210	17	2015-02-06 19:01:15.636798	2015-02-06 19:01:15.636798	3
402	Three studies have associated CCND1 expression with poorer survival.	Poor Outcome	Supports	2	1	3	3	210	18	2015-02-06 19:01:15.655564	2015-02-06 19:01:15.655564	3
403	Three studies have found no significant survival impact for CCND1 in lung cancer.	Poor Outcome	Supports	2	1	3	3	210	18	2015-02-06 19:01:15.672504	2015-02-06 19:01:15.672504	3
404	Trametinib can inhibit ARF-mediated ERK phosphorylation and soft agar colony formation of AALE cells in vitro	Sensitivity	Supports	1	2	53	3	211	9	2015-02-06 19:01:15.69275	2015-02-06 19:01:15.69275	3
405	Sorafenib can inhibit ARF-mediated MEK phosphorylation and soft agar colony formation of AALE cells in vitro	Sensitivity	Supports	1	2	14	3	211	9	2015-02-06 19:01:15.72574	2015-02-06 19:01:15.72574	3
406	In one patient with S214C mutation, the use of sorafenib has led to more than 5 years of survival and near remission.	Sensitivity	Supports	1	1	14	3	211	9	2015-02-06 19:01:15.742768	2015-02-06 19:01:15.742768	2
407	SH-SY5Y and Kelly cells with mutant ALK (F1174L) and Ba/F3 cells overexpressing ALK harboring the F1174L mutation are sensitive to TAE684-mediated growth inhibition	Sensitivity	Supports	1	2	61	47	212	7	2015-02-06 19:01:15.763565	2015-02-06 19:01:15.763565	4
408	CH5424802 is effective in inhibiting the activity of the F1174L ALK mutant in a kinase activity assay and proliferation assay using neuroblastoma KELLY cells.	Sensitivity	Supports	1	2	62	47	213	7	2015-02-06 19:01:15.782844	2015-02-06 19:01:15.782844	3
409	CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or AML4-ALK with the L1196M mutation	Sensitivity	Supports	1	2	62	3	213	6	2015-02-06 19:01:15.799053	2015-02-06 19:01:15.799053	3
410	TAE684 inhibits growth of Ba/F3 cells expressing ALK with the R1275Q mutation at a higher IC50 than those expressing F1174L and the SMS-KCNR cell line harboring the R1275Q mutation was resistant to TAE684	NA	Does Not Support	1	2	61	47	212	8	2015-02-06 19:01:15.815039	2015-02-06 19:01:15.815039	3
411	Ba/F3 cells expressing the RANBP2-ALK fusion containing an F1174L mutation were more resistant to crizotinib treatment than Ba/F3 cells expressing RANBP2-ALK without this mutation.	Resistance or Non-Response	Supports	1	2	19	48	84	7	2015-02-06 19:01:15.83558	2015-02-06 19:01:15.83558	3
412	Cells lines expressing ALK harboring the F1174L mutation are less sensitive to growth inhibition by crizotinib than cells expressing the R1275Q mutation.	Resistance or Non-Response	Supports	1	2	19	47	214	7	2015-02-06 19:01:15.854477	2015-02-06 19:01:15.854477	3
413	High levels of crizotinib can overcome drug resistance of Ba/F3 cells expressing the EML4-ALK fusion containing the F1174L mutation.	Sensitivity	Supports	1	2	19	47	215	7	2015-02-06 19:01:15.872568	2015-02-06 19:01:15.872568	3
414	PIK3CA mutation, but not AKT1 E17K, has been shown to confer sensitiity to the AKT inhibitor MK-2206 in breast cancer cell lines.	Sensitivity	Does Not Support	1	1	63	49	216	3	2015-02-06 19:01:15.893813	2015-02-06 19:01:15.893813	3
415	Treatment of Philadelphia Chromosome positive leukemias with Arsenic Trioxide results in high rates of complete remission in patients.	Sensitivity	Supports	1	4	64	7	217	1	2015-02-06 19:01:15.914298	2015-02-06 19:01:15.914298	5
416	The clinical use of imatinib in patients with BCR-ABL fusion has drastic sensitivity to the drug.	Sensitivity	Supports	1	4	10	9	72	1	2015-02-06 19:01:15.931016	2015-02-06 19:01:15.931016	5
417	BCR-ABL inhibitors such as imatinib have lead to "significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens."	Better Outcome	Supports	2	4	10	9	72	1	2015-02-06 19:01:15.947655	2015-02-06 19:01:15.947655	5
418	The use of second-generation BCR-ABL targeted therapies has shown signifcantly greater response in patients resistant to imatinib.	Sensitivity	Supports	1	1	13	9	72	1	2015-02-06 19:01:15.964429	2015-02-06 19:01:15.964429	4
419	The use of second-generation BCR-ABL targeted therapies has shown signifcantly greater response in patients resistant to imatinib.	Sensitivity	Supports	1	1	15	9	72	1	2015-02-06 19:01:15.974008	2015-02-06 19:01:15.974008	4
420	The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.	Positive	Supports	3	4	3	9	72	1	2015-02-06 19:01:15.989859	2015-02-06 19:01:15.989859	4
421	BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.	Resistance or Non-Response	Supports	1	1	10	7	217	1	2015-02-06 19:01:16.009504	2015-02-06 19:01:16.009504	3
422	Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.	Positive	Does Not Support	3	2	3	9	218	1	2015-02-06 19:01:16.028732	2015-02-06 19:01:16.028732	2
\.


--
-- Name: evidence_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_items_id_seq', 422, true);


--
-- Name: evidence_levels_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_levels_id_seq', 4, true);


--
-- Name: evidence_types_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('evidence_types_id_seq', 3, true);


--
-- Name: genes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('genes_id_seq', 47, true);


--
-- Data for Name: genes_sources; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY genes_sources (gene_id, source_id, created_at, updated_at) FROM stdin;
1	1	2015-02-06 19:00:57.491661	2015-02-06 19:00:57.491661
1	2	2015-02-06 19:00:57.503566	2015-02-06 19:00:57.503566
2	3	2015-02-06 19:00:57.720682	2015-02-06 19:00:57.720682
2	4	2015-02-06 19:00:57.725428	2015-02-06 19:00:57.725428
3	5	2015-02-06 19:00:57.90187	2015-02-06 19:00:57.90187
3	6	2015-02-06 19:00:57.906579	2015-02-06 19:00:57.906579
4	7	2015-02-06 19:00:58.081936	2015-02-06 19:00:58.081936
4	8	2015-02-06 19:00:58.086511	2015-02-06 19:00:58.086511
5	9	2015-02-06 19:00:58.29851	2015-02-06 19:00:58.29851
5	10	2015-02-06 19:00:58.303001	2015-02-06 19:00:58.303001
6	11	2015-02-06 19:00:58.482377	2015-02-06 19:00:58.482377
6	12	2015-02-06 19:00:58.486734	2015-02-06 19:00:58.486734
7	11	2015-02-06 19:00:58.659155	2015-02-06 19:00:58.659155
7	12	2015-02-06 19:00:58.662184	2015-02-06 19:00:58.662184
8	11	2015-02-06 19:00:58.835328	2015-02-06 19:00:58.835328
8	12	2015-02-06 19:00:58.838379	2015-02-06 19:00:58.838379
9	13	2015-02-06 19:00:59.087271	2015-02-06 19:00:59.087271
10	14	2015-02-06 19:00:59.279329	2015-02-06 19:00:59.279329
10	15	2015-02-06 19:00:59.283999	2015-02-06 19:00:59.283999
11	14	2015-02-06 19:00:59.461529	2015-02-06 19:00:59.461529
11	15	2015-02-06 19:00:59.464606	2015-02-06 19:00:59.464606
12	16	2015-02-06 19:00:59.660036	2015-02-06 19:00:59.660036
12	17	2015-02-06 19:00:59.664975	2015-02-06 19:00:59.664975
13	18	2015-02-06 19:00:59.844123	2015-02-06 19:00:59.844123
13	19	2015-02-06 19:00:59.848677	2015-02-06 19:00:59.848677
13	20	2015-02-06 19:00:59.852815	2015-02-06 19:00:59.852815
14	21	2015-02-06 19:01:00.037588	2015-02-06 19:01:00.037588
15	21	2015-02-06 19:01:00.255123	2015-02-06 19:01:00.255123
16	22	2015-02-06 19:01:00.454399	2015-02-06 19:01:00.454399
16	23	2015-02-06 19:01:00.457909	2015-02-06 19:01:00.457909
17	24	2015-02-06 19:01:00.646649	2015-02-06 19:01:00.646649
17	25	2015-02-06 19:01:00.651048	2015-02-06 19:01:00.651048
18	26	2015-02-06 19:01:00.832386	2015-02-06 19:01:00.832386
18	27	2015-02-06 19:01:00.83705	2015-02-06 19:01:00.83705
19	28	2015-02-06 19:01:01.066444	2015-02-06 19:01:01.066444
19	29	2015-02-06 19:01:01.071193	2015-02-06 19:01:01.071193
20	30	2015-02-06 19:01:01.287659	2015-02-06 19:01:01.287659
20	31	2015-02-06 19:01:01.291876	2015-02-06 19:01:01.291876
21	30	2015-02-06 19:01:01.466993	2015-02-06 19:01:01.466993
21	31	2015-02-06 19:01:01.47042	2015-02-06 19:01:01.47042
22	32	2015-02-06 19:01:01.69427	2015-02-06 19:01:01.69427
22	18	2015-02-06 19:01:01.697205	2015-02-06 19:01:01.697205
33	33	2015-02-06 19:01:03.771454	2015-02-06 19:01:03.771454
33	18	2015-02-06 19:01:03.775261	2015-02-06 19:01:03.775261
40	34	2015-02-06 19:01:05.114423	2015-02-06 19:01:05.114423
40	35	2015-02-06 19:01:05.119199	2015-02-06 19:01:05.119199
42	36	2015-02-06 19:01:05.504588	2015-02-06 19:01:05.504588
42	37	2015-02-06 19:01:05.5093	2015-02-06 19:01:05.5093
46	38	2015-02-06 19:01:06.238394	2015-02-06 19:01:06.238394
46	39	2015-02-06 19:01:06.242889	2015-02-06 19:01:06.242889
\.


--
-- Data for Name: ratings; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY ratings (id, value, evidence_item_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Name: ratings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('ratings_id_seq', 1, false);


--
-- Data for Name: roles; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles (id, name, description, created_at, updated_at) FROM stdin;
\.


--
-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('roles_id_seq', 1, false);


--
-- Data for Name: roles_users; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY roles_users (role_id, user_id, created_at, updated_at) FROM stdin;
\.


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY schema_migrations (version) FROM stdin;
20150205210725
20140805184308
20141021192035
20141022180523
20141022195329
20141108234203
20141108234935
20141109000136
20141114222425
20150107205621
20150115231117
20150116220720
20150116230539
20150116230632
20150204192653
20150205000010
20150205205734
\.


--
-- Name: sources_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('sources_id_seq', 218, true);


--
-- Data for Name: suggested_changes; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY suggested_changes (id, suggested_changes, moderated_id, moderated_type, user_id, status, created_at, updated_at) FROM stdin;
\.


--
-- Name: suggested_changes_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('suggested_changes_id_seq', 1, false);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('users_id_seq', 1, false);


--
-- Data for Name: variant_groups; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_groups (id, name, description, created_at, updated_at) FROM stdin;
1	TSC Loss	\N	2015-02-06 19:01:07.904949	2015-02-06 19:01:07.904949
2	PTEN Loss-of-Function	\N	2015-02-06 19:01:08.019423	2015-02-06 19:01:08.019423
3	Other V600's	\N	2015-02-06 19:01:08.07451	2015-02-06 19:01:08.07451
4	NPM1 exon 12	\N	2015-02-06 19:01:08.168366	2015-02-06 19:01:08.168366
5	NF1 loss	\N	2015-02-06 19:01:09.057035	2015-02-06 19:01:09.057035
6	Motesanib Resistance	\N	2015-02-06 19:01:09.107607	2015-02-06 19:01:09.107607
7	Imatinib Resistance	\N	2015-02-06 19:01:09.169945	2015-02-06 19:01:09.169945
8	FGFR fusions	\N	2015-02-06 19:01:09.199378	2015-02-06 19:01:09.199378
9	Exon 17	\N	2015-02-06 19:01:09.587953	2015-02-06 19:01:09.587953
10	Exon 11	\N	2015-02-06 19:01:09.638362	2015-02-06 19:01:09.638362
11	ESR1 Ligand-Binding Domain	\N	2015-02-06 19:01:09.817651	2015-02-06 19:01:09.817651
12	DNAJB1 Fusions	\N	2015-02-06 19:01:10.0974	2015-02-06 19:01:10.0974
13	Crizotinib Resistance	\N	2015-02-06 19:01:10.143999	2015-02-06 19:01:10.143999
14	ALK Fusions	\N	2015-02-06 19:01:10.169952	2015-02-06 19:01:10.169952
\.


--
-- Data for Name: variant_group_variants; Type: TABLE DATA; Schema: public; Owner: acoffman
--

COPY variant_group_variants (variant_id, variant_group_id, created_at, updated_at) FROM stdin;
131	1	2015-02-06 19:01:07.913816	2015-02-06 19:01:07.913816
130	1	2015-02-06 19:01:07.938185	2015-02-06 19:01:07.938185
129	1	2015-02-06 19:01:07.962671	2015-02-06 19:01:07.962671
129	1	2015-02-06 19:01:07.992568	2015-02-06 19:01:07.992568
115	2	2015-02-06 19:01:08.021419	2015-02-06 19:01:08.021419
115	2	2015-02-06 19:01:08.050302	2015-02-06 19:01:08.050302
10	3	2015-02-06 19:01:08.076534	2015-02-06 19:01:08.076534
14	3	2015-02-06 19:01:08.096857	2015-02-06 19:01:08.096857
15	3	2015-02-06 19:01:08.119142	2015-02-06 19:01:08.119142
12	3	2015-02-06 19:01:08.141962	2015-02-06 19:01:08.141962
92	4	2015-02-06 19:01:08.170399	2015-02-06 19:01:08.170399
92	4	2015-02-06 19:01:08.209041	2015-02-06 19:01:08.209041
92	4	2015-02-06 19:01:08.229043	2015-02-06 19:01:08.229043
92	4	2015-02-06 19:01:08.252647	2015-02-06 19:01:08.252647
92	4	2015-02-06 19:01:08.276387	2015-02-06 19:01:08.276387
92	4	2015-02-06 19:01:08.301955	2015-02-06 19:01:08.301955
92	4	2015-02-06 19:01:08.328977	2015-02-06 19:01:08.328977
92	4	2015-02-06 19:01:08.354502	2015-02-06 19:01:08.354502
92	4	2015-02-06 19:01:08.378108	2015-02-06 19:01:08.378108
92	4	2015-02-06 19:01:08.401526	2015-02-06 19:01:08.401526
92	4	2015-02-06 19:01:08.42137	2015-02-06 19:01:08.42137
93	4	2015-02-06 19:01:08.446969	2015-02-06 19:01:08.446969
93	4	2015-02-06 19:01:08.46769	2015-02-06 19:01:08.46769
92	4	2015-02-06 19:01:08.501925	2015-02-06 19:01:08.501925
92	4	2015-02-06 19:01:08.525706	2015-02-06 19:01:08.525706
92	4	2015-02-06 19:01:08.54589	2015-02-06 19:01:08.54589
92	4	2015-02-06 19:01:08.568258	2015-02-06 19:01:08.568258
92	4	2015-02-06 19:01:08.591183	2015-02-06 19:01:08.591183
92	4	2015-02-06 19:01:08.613611	2015-02-06 19:01:08.613611
92	4	2015-02-06 19:01:08.633961	2015-02-06 19:01:08.633961
92	4	2015-02-06 19:01:08.655472	2015-02-06 19:01:08.655472
92	4	2015-02-06 19:01:08.676397	2015-02-06 19:01:08.676397
92	4	2015-02-06 19:01:08.700456	2015-02-06 19:01:08.700456
92	4	2015-02-06 19:01:08.722939	2015-02-06 19:01:08.722939
92	4	2015-02-06 19:01:08.744942	2015-02-06 19:01:08.744942
92	4	2015-02-06 19:01:08.768601	2015-02-06 19:01:08.768601
92	4	2015-02-06 19:01:08.794474	2015-02-06 19:01:08.794474
93	4	2015-02-06 19:01:08.820607	2015-02-06 19:01:08.820607
92	4	2015-02-06 19:01:08.844193	2015-02-06 19:01:08.844193
92	4	2015-02-06 19:01:08.870336	2015-02-06 19:01:08.870336
92	4	2015-02-06 19:01:08.89373	2015-02-06 19:01:08.89373
92	4	2015-02-06 19:01:08.914081	2015-02-06 19:01:08.914081
92	4	2015-02-06 19:01:08.939011	2015-02-06 19:01:08.939011
92	4	2015-02-06 19:01:08.961962	2015-02-06 19:01:08.961962
92	4	2015-02-06 19:01:08.983975	2015-02-06 19:01:08.983975
92	4	2015-02-06 19:01:09.007208	2015-02-06 19:01:09.007208
92	4	2015-02-06 19:01:09.033183	2015-02-06 19:01:09.033183
11	5	2015-02-06 19:01:09.058953	2015-02-06 19:01:09.058953
11	5	2015-02-06 19:01:09.080539	2015-02-06 19:01:09.080539
117	6	2015-02-06 19:01:09.109588	2015-02-06 19:01:09.109588
118	6	2015-02-06 19:01:09.130845	2015-02-06 19:01:09.130845
2	7	2015-02-06 19:01:09.172638	2015-02-06 19:01:09.172638
57	8	2015-02-06 19:01:09.201204	2015-02-06 19:01:09.201204
52	8	2015-02-06 19:01:09.226618	2015-02-06 19:01:09.226618
54	8	2015-02-06 19:01:09.249017	2015-02-06 19:01:09.249017
55	8	2015-02-06 19:01:09.273459	2015-02-06 19:01:09.273459
53	8	2015-02-06 19:01:09.295254	2015-02-06 19:01:09.295254
59	8	2015-02-06 19:01:09.318392	2015-02-06 19:01:09.318392
56	8	2015-02-06 19:01:09.341859	2015-02-06 19:01:09.341859
58	8	2015-02-06 19:01:09.362995	2015-02-06 19:01:09.362995
60	8	2015-02-06 19:01:09.394704	2015-02-06 19:01:09.394704
60	8	2015-02-06 19:01:09.42738	2015-02-06 19:01:09.42738
60	8	2015-02-06 19:01:09.462531	2015-02-06 19:01:09.462531
60	8	2015-02-06 19:01:09.496111	2015-02-06 19:01:09.496111
60	8	2015-02-06 19:01:09.529381	2015-02-06 19:01:09.529381
60	8	2015-02-06 19:01:09.563452	2015-02-06 19:01:09.563452
72	9	2015-02-06 19:01:09.590513	2015-02-06 19:01:09.590513
78	10	2015-02-06 19:01:09.640189	2015-02-06 19:01:09.640189
78	10	2015-02-06 19:01:09.688121	2015-02-06 19:01:09.688121
79	10	2015-02-06 19:01:09.713244	2015-02-06 19:01:09.713244
78	10	2015-02-06 19:01:09.734176	2015-02-06 19:01:09.734176
79	10	2015-02-06 19:01:09.758846	2015-02-06 19:01:09.758846
78	10	2015-02-06 19:01:09.781167	2015-02-06 19:01:09.781167
47	11	2015-02-06 19:01:09.819558	2015-02-06 19:01:09.819558
48	11	2015-02-06 19:01:09.851497	2015-02-06 19:01:09.851497
49	11	2015-02-06 19:01:09.885693	2015-02-06 19:01:09.885693
50	11	2015-02-06 19:01:09.918366	2015-02-06 19:01:09.918366
51	11	2015-02-06 19:01:09.950332	2015-02-06 19:01:09.950332
47	11	2015-02-06 19:01:09.973253	2015-02-06 19:01:09.973253
48	11	2015-02-06 19:01:09.995222	2015-02-06 19:01:09.995222
49	11	2015-02-06 19:01:10.018496	2015-02-06 19:01:10.018496
50	11	2015-02-06 19:01:10.052212	2015-02-06 19:01:10.052212
51	11	2015-02-06 19:01:10.073244	2015-02-06 19:01:10.073244
114	12	2015-02-06 19:01:10.099365	2015-02-06 19:01:10.099365
32	12	2015-02-06 19:01:10.120411	2015-02-06 19:01:10.120411
7	13	2015-02-06 19:01:10.14593	2015-02-06 19:01:10.14593
6	14	2015-02-06 19:01:10.171959	2015-02-06 19:01:10.171959
6	13	2015-02-06 19:01:10.174689	2015-02-06 19:01:10.174689
5	14	2015-02-06 19:01:10.198843	2015-02-06 19:01:10.198843
5	13	2015-02-06 19:01:10.202046	2015-02-06 19:01:10.202046
4	14	2015-02-06 19:01:10.226033	2015-02-06 19:01:10.226033
\.


--
-- Name: variant_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variant_groups_id_seq', 14, true);


--
-- Name: variants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: acoffman
--

SELECT pg_catalog.setval('variants_id_seq', 135, true);


--
-- PostgreSQL database dump complete
--

